<!DOCTYPE html>
<html>
<head><meta charset="utf-8" />

<title>JSON_DMW_Reviewer_for_Midterms</title>

<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>



<style type="text/css">
    /*!
*
* Twitter Bootstrap
*
*/
/*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 */
/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
html {
  font-family: sans-serif;
  -ms-text-size-adjust: 100%;
  -webkit-text-size-adjust: 100%;
}
body {
  margin: 0;
}
article,
aside,
details,
figcaption,
figure,
footer,
header,
hgroup,
main,
menu,
nav,
section,
summary {
  display: block;
}
audio,
canvas,
progress,
video {
  display: inline-block;
  vertical-align: baseline;
}
audio:not([controls]) {
  display: none;
  height: 0;
}
[hidden],
template {
  display: none;
}
a {
  background-color: transparent;
}
a:active,
a:hover {
  outline: 0;
}
abbr[title] {
  border-bottom: 1px dotted;
}
b,
strong {
  font-weight: bold;
}
dfn {
  font-style: italic;
}
h1 {
  font-size: 2em;
  margin: 0.67em 0;
}
mark {
  background: #ff0;
  color: #000;
}
small {
  font-size: 80%;
}
sub,
sup {
  font-size: 75%;
  line-height: 0;
  position: relative;
  vertical-align: baseline;
}
sup {
  top: -0.5em;
}
sub {
  bottom: -0.25em;
}
img {
  border: 0;
}
svg:not(:root) {
  overflow: hidden;
}
figure {
  margin: 1em 40px;
}
hr {
  box-sizing: content-box;
  height: 0;
}
pre {
  overflow: auto;
}
code,
kbd,
pre,
samp {
  font-family: monospace, monospace;
  font-size: 1em;
}
button,
input,
optgroup,
select,
textarea {
  color: inherit;
  font: inherit;
  margin: 0;
}
button {
  overflow: visible;
}
button,
select {
  text-transform: none;
}
button,
html input[type="button"],
input[type="reset"],
input[type="submit"] {
  -webkit-appearance: button;
  cursor: pointer;
}
button[disabled],
html input[disabled] {
  cursor: default;
}
button::-moz-focus-inner,
input::-moz-focus-inner {
  border: 0;
  padding: 0;
}
input {
  line-height: normal;
}
input[type="checkbox"],
input[type="radio"] {
  box-sizing: border-box;
  padding: 0;
}
input[type="number"]::-webkit-inner-spin-button,
input[type="number"]::-webkit-outer-spin-button {
  height: auto;
}
input[type="search"] {
  -webkit-appearance: textfield;
  box-sizing: content-box;
}
input[type="search"]::-webkit-search-cancel-button,
input[type="search"]::-webkit-search-decoration {
  -webkit-appearance: none;
}
fieldset {
  border: 1px solid #c0c0c0;
  margin: 0 2px;
  padding: 0.35em 0.625em 0.75em;
}
legend {
  border: 0;
  padding: 0;
}
textarea {
  overflow: auto;
}
optgroup {
  font-weight: bold;
}
table {
  border-collapse: collapse;
  border-spacing: 0;
}
td,
th {
  padding: 0;
}
/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */
@media print {
  *,
  *:before,
  *:after {
    background: transparent !important;
    box-shadow: none !important;
    text-shadow: none !important;
  }
  a,
  a:visited {
    text-decoration: underline;
  }
  a[href]:after {
    content: " (" attr(href) ")";
  }
  abbr[title]:after {
    content: " (" attr(title) ")";
  }
  a[href^="#"]:after,
  a[href^="javascript:"]:after {
    content: "";
  }
  pre,
  blockquote {
    border: 1px solid #999;
    page-break-inside: avoid;
  }
  thead {
    display: table-header-group;
  }
  tr,
  img {
    page-break-inside: avoid;
  }
  img {
    max-width: 100% !important;
  }
  p,
  h2,
  h3 {
    orphans: 3;
    widows: 3;
  }
  h2,
  h3 {
    page-break-after: avoid;
  }
  .navbar {
    display: none;
  }
  .btn > .caret,
  .dropup > .btn > .caret {
    border-top-color: #000 !important;
  }
  .label {
    border: 1px solid #000;
  }
  .table {
    border-collapse: collapse !important;
  }
  .table td,
  .table th {
    background-color: #fff !important;
  }
  .table-bordered th,
  .table-bordered td {
    border: 1px solid #ddd !important;
  }
}
@font-face {
  font-family: 'Glyphicons Halflings';
  src: url('../components/bootstrap/fonts/glyphicons-halflings-regular.eot');
  src: url('../components/bootstrap/fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('../components/bootstrap/fonts/glyphicons-halflings-regular.woff2') format('woff2'), url('../components/bootstrap/fonts/glyphicons-halflings-regular.woff') format('woff'), url('../components/bootstrap/fonts/glyphicons-halflings-regular.ttf') format('truetype'), url('../components/bootstrap/fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
}
.glyphicon {
  position: relative;
  top: 1px;
  display: inline-block;
  font-family: 'Glyphicons Halflings';
  font-style: normal;
  font-weight: normal;
  line-height: 1;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
.glyphicon-asterisk:before {
  content: "\002a";
}
.glyphicon-plus:before {
  content: "\002b";
}
.glyphicon-euro:before,
.glyphicon-eur:before {
  content: "\20ac";
}
.glyphicon-minus:before {
  content: "\2212";
}
.glyphicon-cloud:before {
  content: "\2601";
}
.glyphicon-envelope:before {
  content: "\2709";
}
.glyphicon-pencil:before {
  content: "\270f";
}
.glyphicon-glass:before {
  content: "\e001";
}
.glyphicon-music:before {
  content: "\e002";
}
.glyphicon-search:before {
  content: "\e003";
}
.glyphicon-heart:before {
  content: "\e005";
}
.glyphicon-star:before {
  content: "\e006";
}
.glyphicon-star-empty:before {
  content: "\e007";
}
.glyphicon-user:before {
  content: "\e008";
}
.glyphicon-film:before {
  content: "\e009";
}
.glyphicon-th-large:before {
  content: "\e010";
}
.glyphicon-th:before {
  content: "\e011";
}
.glyphicon-th-list:before {
  content: "\e012";
}
.glyphicon-ok:before {
  content: "\e013";
}
.glyphicon-remove:before {
  content: "\e014";
}
.glyphicon-zoom-in:before {
  content: "\e015";
}
.glyphicon-zoom-out:before {
  content: "\e016";
}
.glyphicon-off:before {
  content: "\e017";
}
.glyphicon-signal:before {
  content: "\e018";
}
.glyphicon-cog:before {
  content: "\e019";
}
.glyphicon-trash:before {
  content: "\e020";
}
.glyphicon-home:before {
  content: "\e021";
}
.glyphicon-file:before {
  content: "\e022";
}
.glyphicon-time:before {
  content: "\e023";
}
.glyphicon-road:before {
  content: "\e024";
}
.glyphicon-download-alt:before {
  content: "\e025";
}
.glyphicon-download:before {
  content: "\e026";
}
.glyphicon-upload:before {
  content: "\e027";
}
.glyphicon-inbox:before {
  content: "\e028";
}
.glyphicon-play-circle:before {
  content: "\e029";
}
.glyphicon-repeat:before {
  content: "\e030";
}
.glyphicon-refresh:before {
  content: "\e031";
}
.glyphicon-list-alt:before {
  content: "\e032";
}
.glyphicon-lock:before {
  content: "\e033";
}
.glyphicon-flag:before {
  content: "\e034";
}
.glyphicon-headphones:before {
  content: "\e035";
}
.glyphicon-volume-off:before {
  content: "\e036";
}
.glyphicon-volume-down:before {
  content: "\e037";
}
.glyphicon-volume-up:before {
  content: "\e038";
}
.glyphicon-qrcode:before {
  content: "\e039";
}
.glyphicon-barcode:before {
  content: "\e040";
}
.glyphicon-tag:before {
  content: "\e041";
}
.glyphicon-tags:before {
  content: "\e042";
}
.glyphicon-book:before {
  content: "\e043";
}
.glyphicon-bookmark:before {
  content: "\e044";
}
.glyphicon-print:before {
  content: "\e045";
}
.glyphicon-camera:before {
  content: "\e046";
}
.glyphicon-font:before {
  content: "\e047";
}
.glyphicon-bold:before {
  content: "\e048";
}
.glyphicon-italic:before {
  content: "\e049";
}
.glyphicon-text-height:before {
  content: "\e050";
}
.glyphicon-text-width:before {
  content: "\e051";
}
.glyphicon-align-left:before {
  content: "\e052";
}
.glyphicon-align-center:before {
  content: "\e053";
}
.glyphicon-align-right:before {
  content: "\e054";
}
.glyphicon-align-justify:before {
  content: "\e055";
}
.glyphicon-list:before {
  content: "\e056";
}
.glyphicon-indent-left:before {
  content: "\e057";
}
.glyphicon-indent-right:before {
  content: "\e058";
}
.glyphicon-facetime-video:before {
  content: "\e059";
}
.glyphicon-picture:before {
  content: "\e060";
}
.glyphicon-map-marker:before {
  content: "\e062";
}
.glyphicon-adjust:before {
  content: "\e063";
}
.glyphicon-tint:before {
  content: "\e064";
}
.glyphicon-edit:before {
  content: "\e065";
}
.glyphicon-share:before {
  content: "\e066";
}
.glyphicon-check:before {
  content: "\e067";
}
.glyphicon-move:before {
  content: "\e068";
}
.glyphicon-step-backward:before {
  content: "\e069";
}
.glyphicon-fast-backward:before {
  content: "\e070";
}
.glyphicon-backward:before {
  content: "\e071";
}
.glyphicon-play:before {
  content: "\e072";
}
.glyphicon-pause:before {
  content: "\e073";
}
.glyphicon-stop:before {
  content: "\e074";
}
.glyphicon-forward:before {
  content: "\e075";
}
.glyphicon-fast-forward:before {
  content: "\e076";
}
.glyphicon-step-forward:before {
  content: "\e077";
}
.glyphicon-eject:before {
  content: "\e078";
}
.glyphicon-chevron-left:before {
  content: "\e079";
}
.glyphicon-chevron-right:before {
  content: "\e080";
}
.glyphicon-plus-sign:before {
  content: "\e081";
}
.glyphicon-minus-sign:before {
  content: "\e082";
}
.glyphicon-remove-sign:before {
  content: "\e083";
}
.glyphicon-ok-sign:before {
  content: "\e084";
}
.glyphicon-question-sign:before {
  content: "\e085";
}
.glyphicon-info-sign:before {
  content: "\e086";
}
.glyphicon-screenshot:before {
  content: "\e087";
}
.glyphicon-remove-circle:before {
  content: "\e088";
}
.glyphicon-ok-circle:before {
  content: "\e089";
}
.glyphicon-ban-circle:before {
  content: "\e090";
}
.glyphicon-arrow-left:before {
  content: "\e091";
}
.glyphicon-arrow-right:before {
  content: "\e092";
}
.glyphicon-arrow-up:before {
  content: "\e093";
}
.glyphicon-arrow-down:before {
  content: "\e094";
}
.glyphicon-share-alt:before {
  content: "\e095";
}
.glyphicon-resize-full:before {
  content: "\e096";
}
.glyphicon-resize-small:before {
  content: "\e097";
}
.glyphicon-exclamation-sign:before {
  content: "\e101";
}
.glyphicon-gift:before {
  content: "\e102";
}
.glyphicon-leaf:before {
  content: "\e103";
}
.glyphicon-fire:before {
  content: "\e104";
}
.glyphicon-eye-open:before {
  content: "\e105";
}
.glyphicon-eye-close:before {
  content: "\e106";
}
.glyphicon-warning-sign:before {
  content: "\e107";
}
.glyphicon-plane:before {
  content: "\e108";
}
.glyphicon-calendar:before {
  content: "\e109";
}
.glyphicon-random:before {
  content: "\e110";
}
.glyphicon-comment:before {
  content: "\e111";
}
.glyphicon-magnet:before {
  content: "\e112";
}
.glyphicon-chevron-up:before {
  content: "\e113";
}
.glyphicon-chevron-down:before {
  content: "\e114";
}
.glyphicon-retweet:before {
  content: "\e115";
}
.glyphicon-shopping-cart:before {
  content: "\e116";
}
.glyphicon-folder-close:before {
  content: "\e117";
}
.glyphicon-folder-open:before {
  content: "\e118";
}
.glyphicon-resize-vertical:before {
  content: "\e119";
}
.glyphicon-resize-horizontal:before {
  content: "\e120";
}
.glyphicon-hdd:before {
  content: "\e121";
}
.glyphicon-bullhorn:before {
  content: "\e122";
}
.glyphicon-bell:before {
  content: "\e123";
}
.glyphicon-certificate:before {
  content: "\e124";
}
.glyphicon-thumbs-up:before {
  content: "\e125";
}
.glyphicon-thumbs-down:before {
  content: "\e126";
}
.glyphicon-hand-right:before {
  content: "\e127";
}
.glyphicon-hand-left:before {
  content: "\e128";
}
.glyphicon-hand-up:before {
  content: "\e129";
}
.glyphicon-hand-down:before {
  content: "\e130";
}
.glyphicon-circle-arrow-right:before {
  content: "\e131";
}
.glyphicon-circle-arrow-left:before {
  content: "\e132";
}
.glyphicon-circle-arrow-up:before {
  content: "\e133";
}
.glyphicon-circle-arrow-down:before {
  content: "\e134";
}
.glyphicon-globe:before {
  content: "\e135";
}
.glyphicon-wrench:before {
  content: "\e136";
}
.glyphicon-tasks:before {
  content: "\e137";
}
.glyphicon-filter:before {
  content: "\e138";
}
.glyphicon-briefcase:before {
  content: "\e139";
}
.glyphicon-fullscreen:before {
  content: "\e140";
}
.glyphicon-dashboard:before {
  content: "\e141";
}
.glyphicon-paperclip:before {
  content: "\e142";
}
.glyphicon-heart-empty:before {
  content: "\e143";
}
.glyphicon-link:before {
  content: "\e144";
}
.glyphicon-phone:before {
  content: "\e145";
}
.glyphicon-pushpin:before {
  content: "\e146";
}
.glyphicon-usd:before {
  content: "\e148";
}
.glyphicon-gbp:before {
  content: "\e149";
}
.glyphicon-sort:before {
  content: "\e150";
}
.glyphicon-sort-by-alphabet:before {
  content: "\e151";
}
.glyphicon-sort-by-alphabet-alt:before {
  content: "\e152";
}
.glyphicon-sort-by-order:before {
  content: "\e153";
}
.glyphicon-sort-by-order-alt:before {
  content: "\e154";
}
.glyphicon-sort-by-attributes:before {
  content: "\e155";
}
.glyphicon-sort-by-attributes-alt:before {
  content: "\e156";
}
.glyphicon-unchecked:before {
  content: "\e157";
}
.glyphicon-expand:before {
  content: "\e158";
}
.glyphicon-collapse-down:before {
  content: "\e159";
}
.glyphicon-collapse-up:before {
  content: "\e160";
}
.glyphicon-log-in:before {
  content: "\e161";
}
.glyphicon-flash:before {
  content: "\e162";
}
.glyphicon-log-out:before {
  content: "\e163";
}
.glyphicon-new-window:before {
  content: "\e164";
}
.glyphicon-record:before {
  content: "\e165";
}
.glyphicon-save:before {
  content: "\e166";
}
.glyphicon-open:before {
  content: "\e167";
}
.glyphicon-saved:before {
  content: "\e168";
}
.glyphicon-import:before {
  content: "\e169";
}
.glyphicon-export:before {
  content: "\e170";
}
.glyphicon-send:before {
  content: "\e171";
}
.glyphicon-floppy-disk:before {
  content: "\e172";
}
.glyphicon-floppy-saved:before {
  content: "\e173";
}
.glyphicon-floppy-remove:before {
  content: "\e174";
}
.glyphicon-floppy-save:before {
  content: "\e175";
}
.glyphicon-floppy-open:before {
  content: "\e176";
}
.glyphicon-credit-card:before {
  content: "\e177";
}
.glyphicon-transfer:before {
  content: "\e178";
}
.glyphicon-cutlery:before {
  content: "\e179";
}
.glyphicon-header:before {
  content: "\e180";
}
.glyphicon-compressed:before {
  content: "\e181";
}
.glyphicon-earphone:before {
  content: "\e182";
}
.glyphicon-phone-alt:before {
  content: "\e183";
}
.glyphicon-tower:before {
  content: "\e184";
}
.glyphicon-stats:before {
  content: "\e185";
}
.glyphicon-sd-video:before {
  content: "\e186";
}
.glyphicon-hd-video:before {
  content: "\e187";
}
.glyphicon-subtitles:before {
  content: "\e188";
}
.glyphicon-sound-stereo:before {
  content: "\e189";
}
.glyphicon-sound-dolby:before {
  content: "\e190";
}
.glyphicon-sound-5-1:before {
  content: "\e191";
}
.glyphicon-sound-6-1:before {
  content: "\e192";
}
.glyphicon-sound-7-1:before {
  content: "\e193";
}
.glyphicon-copyright-mark:before {
  content: "\e194";
}
.glyphicon-registration-mark:before {
  content: "\e195";
}
.glyphicon-cloud-download:before {
  content: "\e197";
}
.glyphicon-cloud-upload:before {
  content: "\e198";
}
.glyphicon-tree-conifer:before {
  content: "\e199";
}
.glyphicon-tree-deciduous:before {
  content: "\e200";
}
.glyphicon-cd:before {
  content: "\e201";
}
.glyphicon-save-file:before {
  content: "\e202";
}
.glyphicon-open-file:before {
  content: "\e203";
}
.glyphicon-level-up:before {
  content: "\e204";
}
.glyphicon-copy:before {
  content: "\e205";
}
.glyphicon-paste:before {
  content: "\e206";
}
.glyphicon-alert:before {
  content: "\e209";
}
.glyphicon-equalizer:before {
  content: "\e210";
}
.glyphicon-king:before {
  content: "\e211";
}
.glyphicon-queen:before {
  content: "\e212";
}
.glyphicon-pawn:before {
  content: "\e213";
}
.glyphicon-bishop:before {
  content: "\e214";
}
.glyphicon-knight:before {
  content: "\e215";
}
.glyphicon-baby-formula:before {
  content: "\e216";
}
.glyphicon-tent:before {
  content: "\26fa";
}
.glyphicon-blackboard:before {
  content: "\e218";
}
.glyphicon-bed:before {
  content: "\e219";
}
.glyphicon-apple:before {
  content: "\f8ff";
}
.glyphicon-erase:before {
  content: "\e221";
}
.glyphicon-hourglass:before {
  content: "\231b";
}
.glyphicon-lamp:before {
  content: "\e223";
}
.glyphicon-duplicate:before {
  content: "\e224";
}
.glyphicon-piggy-bank:before {
  content: "\e225";
}
.glyphicon-scissors:before {
  content: "\e226";
}
.glyphicon-bitcoin:before {
  content: "\e227";
}
.glyphicon-btc:before {
  content: "\e227";
}
.glyphicon-xbt:before {
  content: "\e227";
}
.glyphicon-yen:before {
  content: "\00a5";
}
.glyphicon-jpy:before {
  content: "\00a5";
}
.glyphicon-ruble:before {
  content: "\20bd";
}
.glyphicon-rub:before {
  content: "\20bd";
}
.glyphicon-scale:before {
  content: "\e230";
}
.glyphicon-ice-lolly:before {
  content: "\e231";
}
.glyphicon-ice-lolly-tasted:before {
  content: "\e232";
}
.glyphicon-education:before {
  content: "\e233";
}
.glyphicon-option-horizontal:before {
  content: "\e234";
}
.glyphicon-option-vertical:before {
  content: "\e235";
}
.glyphicon-menu-hamburger:before {
  content: "\e236";
}
.glyphicon-modal-window:before {
  content: "\e237";
}
.glyphicon-oil:before {
  content: "\e238";
}
.glyphicon-grain:before {
  content: "\e239";
}
.glyphicon-sunglasses:before {
  content: "\e240";
}
.glyphicon-text-size:before {
  content: "\e241";
}
.glyphicon-text-color:before {
  content: "\e242";
}
.glyphicon-text-background:before {
  content: "\e243";
}
.glyphicon-object-align-top:before {
  content: "\e244";
}
.glyphicon-object-align-bottom:before {
  content: "\e245";
}
.glyphicon-object-align-horizontal:before {
  content: "\e246";
}
.glyphicon-object-align-left:before {
  content: "\e247";
}
.glyphicon-object-align-vertical:before {
  content: "\e248";
}
.glyphicon-object-align-right:before {
  content: "\e249";
}
.glyphicon-triangle-right:before {
  content: "\e250";
}
.glyphicon-triangle-left:before {
  content: "\e251";
}
.glyphicon-triangle-bottom:before {
  content: "\e252";
}
.glyphicon-triangle-top:before {
  content: "\e253";
}
.glyphicon-console:before {
  content: "\e254";
}
.glyphicon-superscript:before {
  content: "\e255";
}
.glyphicon-subscript:before {
  content: "\e256";
}
.glyphicon-menu-left:before {
  content: "\e257";
}
.glyphicon-menu-right:before {
  content: "\e258";
}
.glyphicon-menu-down:before {
  content: "\e259";
}
.glyphicon-menu-up:before {
  content: "\e260";
}
* {
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
}
*:before,
*:after {
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
}
html {
  font-size: 10px;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
}
body {
  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
  font-size: 13px;
  line-height: 1.42857143;
  color: #000;
  background-color: #fff;
}
input,
button,
select,
textarea {
  font-family: inherit;
  font-size: inherit;
  line-height: inherit;
}
a {
  color: #337ab7;
  text-decoration: none;
}
a:hover,
a:focus {
  color: #23527c;
  text-decoration: underline;
}
a:focus {
  outline: 5px auto -webkit-focus-ring-color;
  outline-offset: -2px;
}
figure {
  margin: 0;
}
img {
  vertical-align: middle;
}
.img-responsive,
.thumbnail > img,
.thumbnail a > img,
.carousel-inner > .item > img,
.carousel-inner > .item > a > img {
  display: block;
  max-width: 100%;
  height: auto;
}
.img-rounded {
  border-radius: 3px;
}
.img-thumbnail {
  padding: 4px;
  line-height: 1.42857143;
  background-color: #fff;
  border: 1px solid #ddd;
  border-radius: 2px;
  -webkit-transition: all 0.2s ease-in-out;
  -o-transition: all 0.2s ease-in-out;
  transition: all 0.2s ease-in-out;
  display: inline-block;
  max-width: 100%;
  height: auto;
}
.img-circle {
  border-radius: 50%;
}
hr {
  margin-top: 18px;
  margin-bottom: 18px;
  border: 0;
  border-top: 1px solid #eeeeee;
}
.sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  border: 0;
}
.sr-only-focusable:active,
.sr-only-focusable:focus {
  position: static;
  width: auto;
  height: auto;
  margin: 0;
  overflow: visible;
  clip: auto;
}
[role="button"] {
  cursor: pointer;
}
h1,
h2,
h3,
h4,
h5,
h6,
.h1,
.h2,
.h3,
.h4,
.h5,
.h6 {
  font-family: inherit;
  font-weight: 500;
  line-height: 1.1;
  color: inherit;
}
h1 small,
h2 small,
h3 small,
h4 small,
h5 small,
h6 small,
.h1 small,
.h2 small,
.h3 small,
.h4 small,
.h5 small,
.h6 small,
h1 .small,
h2 .small,
h3 .small,
h4 .small,
h5 .small,
h6 .small,
.h1 .small,
.h2 .small,
.h3 .small,
.h4 .small,
.h5 .small,
.h6 .small {
  font-weight: normal;
  line-height: 1;
  color: #777777;
}
h1,
.h1,
h2,
.h2,
h3,
.h3 {
  margin-top: 18px;
  margin-bottom: 9px;
}
h1 small,
.h1 small,
h2 small,
.h2 small,
h3 small,
.h3 small,
h1 .small,
.h1 .small,
h2 .small,
.h2 .small,
h3 .small,
.h3 .small {
  font-size: 65%;
}
h4,
.h4,
h5,
.h5,
h6,
.h6 {
  margin-top: 9px;
  margin-bottom: 9px;
}
h4 small,
.h4 small,
h5 small,
.h5 small,
h6 small,
.h6 small,
h4 .small,
.h4 .small,
h5 .small,
.h5 .small,
h6 .small,
.h6 .small {
  font-size: 75%;
}
h1,
.h1 {
  font-size: 33px;
}
h2,
.h2 {
  font-size: 27px;
}
h3,
.h3 {
  font-size: 23px;
}
h4,
.h4 {
  font-size: 17px;
}
h5,
.h5 {
  font-size: 13px;
}
h6,
.h6 {
  font-size: 12px;
}
p {
  margin: 0 0 9px;
}
.lead {
  margin-bottom: 18px;
  font-size: 14px;
  font-weight: 300;
  line-height: 1.4;
}
@media (min-width: 768px) {
  .lead {
    font-size: 19.5px;
  }
}
small,
.small {
  font-size: 92%;
}
mark,
.mark {
  background-color: #fcf8e3;
  padding: .2em;
}
.text-left {
  text-align: left;
}
.text-right {
  text-align: right;
}
.text-center {
  text-align: center;
}
.text-justify {
  text-align: justify;
}
.text-nowrap {
  white-space: nowrap;
}
.text-lowercase {
  text-transform: lowercase;
}
.text-uppercase {
  text-transform: uppercase;
}
.text-capitalize {
  text-transform: capitalize;
}
.text-muted {
  color: #777777;
}
.text-primary {
  color: #337ab7;
}
a.text-primary:hover,
a.text-primary:focus {
  color: #286090;
}
.text-success {
  color: #3c763d;
}
a.text-success:hover,
a.text-success:focus {
  color: #2b542c;
}
.text-info {
  color: #31708f;
}
a.text-info:hover,
a.text-info:focus {
  color: #245269;
}
.text-warning {
  color: #8a6d3b;
}
a.text-warning:hover,
a.text-warning:focus {
  color: #66512c;
}
.text-danger {
  color: #a94442;
}
a.text-danger:hover,
a.text-danger:focus {
  color: #843534;
}
.bg-primary {
  color: #fff;
  background-color: #337ab7;
}
a.bg-primary:hover,
a.bg-primary:focus {
  background-color: #286090;
}
.bg-success {
  background-color: #dff0d8;
}
a.bg-success:hover,
a.bg-success:focus {
  background-color: #c1e2b3;
}
.bg-info {
  background-color: #d9edf7;
}
a.bg-info:hover,
a.bg-info:focus {
  background-color: #afd9ee;
}
.bg-warning {
  background-color: #fcf8e3;
}
a.bg-warning:hover,
a.bg-warning:focus {
  background-color: #f7ecb5;
}
.bg-danger {
  background-color: #f2dede;
}
a.bg-danger:hover,
a.bg-danger:focus {
  background-color: #e4b9b9;
}
.page-header {
  padding-bottom: 8px;
  margin: 36px 0 18px;
  border-bottom: 1px solid #eeeeee;
}
ul,
ol {
  margin-top: 0;
  margin-bottom: 9px;
}
ul ul,
ol ul,
ul ol,
ol ol {
  margin-bottom: 0;
}
.list-unstyled {
  padding-left: 0;
  list-style: none;
}
.list-inline {
  padding-left: 0;
  list-style: none;
  margin-left: -5px;
}
.list-inline > li {
  display: inline-block;
  padding-left: 5px;
  padding-right: 5px;
}
dl {
  margin-top: 0;
  margin-bottom: 18px;
}
dt,
dd {
  line-height: 1.42857143;
}
dt {
  font-weight: bold;
}
dd {
  margin-left: 0;
}
@media (min-width: 541px) {
  .dl-horizontal dt {
    float: left;
    width: 160px;
    clear: left;
    text-align: right;
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
  }
  .dl-horizontal dd {
    margin-left: 180px;
  }
}
abbr[title],
abbr[data-original-title] {
  cursor: help;
  border-bottom: 1px dotted #777777;
}
.initialism {
  font-size: 90%;
  text-transform: uppercase;
}
blockquote {
  padding: 9px 18px;
  margin: 0 0 18px;
  font-size: inherit;
  border-left: 5px solid #eeeeee;
}
blockquote p:last-child,
blockquote ul:last-child,
blockquote ol:last-child {
  margin-bottom: 0;
}
blockquote footer,
blockquote small,
blockquote .small {
  display: block;
  font-size: 80%;
  line-height: 1.42857143;
  color: #777777;
}
blockquote footer:before,
blockquote small:before,
blockquote .small:before {
  content: '\2014 \00A0';
}
.blockquote-reverse,
blockquote.pull-right {
  padding-right: 15px;
  padding-left: 0;
  border-right: 5px solid #eeeeee;
  border-left: 0;
  text-align: right;
}
.blockquote-reverse footer:before,
blockquote.pull-right footer:before,
.blockquote-reverse small:before,
blockquote.pull-right small:before,
.blockquote-reverse .small:before,
blockquote.pull-right .small:before {
  content: '';
}
.blockquote-reverse footer:after,
blockquote.pull-right footer:after,
.blockquote-reverse small:after,
blockquote.pull-right small:after,
.blockquote-reverse .small:after,
blockquote.pull-right .small:after {
  content: '\00A0 \2014';
}
address {
  margin-bottom: 18px;
  font-style: normal;
  line-height: 1.42857143;
}
code,
kbd,
pre,
samp {
  font-family: monospace;
}
code {
  padding: 2px 4px;
  font-size: 90%;
  color: #c7254e;
  background-color: #f9f2f4;
  border-radius: 2px;
}
kbd {
  padding: 2px 4px;
  font-size: 90%;
  color: #888;
  background-color: transparent;
  border-radius: 1px;
  box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.25);
}
kbd kbd {
  padding: 0;
  font-size: 100%;
  font-weight: bold;
  box-shadow: none;
}
pre {
  display: block;
  padding: 8.5px;
  margin: 0 0 9px;
  font-size: 12px;
  line-height: 1.42857143;
  word-break: break-all;
  word-wrap: break-word;
  color: #333333;
  background-color: #f5f5f5;
  border: 1px solid #ccc;
  border-radius: 2px;
}
pre code {
  padding: 0;
  font-size: inherit;
  color: inherit;
  white-space: pre-wrap;
  background-color: transparent;
  border-radius: 0;
}
.pre-scrollable {
  max-height: 340px;
  overflow-y: scroll;
}
.container {
  margin-right: auto;
  margin-left: auto;
  padding-left: 0px;
  padding-right: 0px;
}
@media (min-width: 768px) {
  .container {
    width: 768px;
  }
}
@media (min-width: 992px) {
  .container {
    width: 940px;
  }
}
@media (min-width: 1200px) {
  .container {
    width: 1140px;
  }
}
.container-fluid {
  margin-right: auto;
  margin-left: auto;
  padding-left: 0px;
  padding-right: 0px;
}
.row {
  margin-left: 0px;
  margin-right: 0px;
}
.col-xs-1, .col-sm-1, .col-md-1, .col-lg-1, .col-xs-2, .col-sm-2, .col-md-2, .col-lg-2, .col-xs-3, .col-sm-3, .col-md-3, .col-lg-3, .col-xs-4, .col-sm-4, .col-md-4, .col-lg-4, .col-xs-5, .col-sm-5, .col-md-5, .col-lg-5, .col-xs-6, .col-sm-6, .col-md-6, .col-lg-6, .col-xs-7, .col-sm-7, .col-md-7, .col-lg-7, .col-xs-8, .col-sm-8, .col-md-8, .col-lg-8, .col-xs-9, .col-sm-9, .col-md-9, .col-lg-9, .col-xs-10, .col-sm-10, .col-md-10, .col-lg-10, .col-xs-11, .col-sm-11, .col-md-11, .col-lg-11, .col-xs-12, .col-sm-12, .col-md-12, .col-lg-12 {
  position: relative;
  min-height: 1px;
  padding-left: 0px;
  padding-right: 0px;
}
.col-xs-1, .col-xs-2, .col-xs-3, .col-xs-4, .col-xs-5, .col-xs-6, .col-xs-7, .col-xs-8, .col-xs-9, .col-xs-10, .col-xs-11, .col-xs-12 {
  float: left;
}
.col-xs-12 {
  width: 100%;
}
.col-xs-11 {
  width: 91.66666667%;
}
.col-xs-10 {
  width: 83.33333333%;
}
.col-xs-9 {
  width: 75%;
}
.col-xs-8 {
  width: 66.66666667%;
}
.col-xs-7 {
  width: 58.33333333%;
}
.col-xs-6 {
  width: 50%;
}
.col-xs-5 {
  width: 41.66666667%;
}
.col-xs-4 {
  width: 33.33333333%;
}
.col-xs-3 {
  width: 25%;
}
.col-xs-2 {
  width: 16.66666667%;
}
.col-xs-1 {
  width: 8.33333333%;
}
.col-xs-pull-12 {
  right: 100%;
}
.col-xs-pull-11 {
  right: 91.66666667%;
}
.col-xs-pull-10 {
  right: 83.33333333%;
}
.col-xs-pull-9 {
  right: 75%;
}
.col-xs-pull-8 {
  right: 66.66666667%;
}
.col-xs-pull-7 {
  right: 58.33333333%;
}
.col-xs-pull-6 {
  right: 50%;
}
.col-xs-pull-5 {
  right: 41.66666667%;
}
.col-xs-pull-4 {
  right: 33.33333333%;
}
.col-xs-pull-3 {
  right: 25%;
}
.col-xs-pull-2 {
  right: 16.66666667%;
}
.col-xs-pull-1 {
  right: 8.33333333%;
}
.col-xs-pull-0 {
  right: auto;
}
.col-xs-push-12 {
  left: 100%;
}
.col-xs-push-11 {
  left: 91.66666667%;
}
.col-xs-push-10 {
  left: 83.33333333%;
}
.col-xs-push-9 {
  left: 75%;
}
.col-xs-push-8 {
  left: 66.66666667%;
}
.col-xs-push-7 {
  left: 58.33333333%;
}
.col-xs-push-6 {
  left: 50%;
}
.col-xs-push-5 {
  left: 41.66666667%;
}
.col-xs-push-4 {
  left: 33.33333333%;
}
.col-xs-push-3 {
  left: 25%;
}
.col-xs-push-2 {
  left: 16.66666667%;
}
.col-xs-push-1 {
  left: 8.33333333%;
}
.col-xs-push-0 {
  left: auto;
}
.col-xs-offset-12 {
  margin-left: 100%;
}
.col-xs-offset-11 {
  margin-left: 91.66666667%;
}
.col-xs-offset-10 {
  margin-left: 83.33333333%;
}
.col-xs-offset-9 {
  margin-left: 75%;
}
.col-xs-offset-8 {
  margin-left: 66.66666667%;
}
.col-xs-offset-7 {
  margin-left: 58.33333333%;
}
.col-xs-offset-6 {
  margin-left: 50%;
}
.col-xs-offset-5 {
  margin-left: 41.66666667%;
}
.col-xs-offset-4 {
  margin-left: 33.33333333%;
}
.col-xs-offset-3 {
  margin-left: 25%;
}
.col-xs-offset-2 {
  margin-left: 16.66666667%;
}
.col-xs-offset-1 {
  margin-left: 8.33333333%;
}
.col-xs-offset-0 {
  margin-left: 0%;
}
@media (min-width: 768px) {
  .col-sm-1, .col-sm-2, .col-sm-3, .col-sm-4, .col-sm-5, .col-sm-6, .col-sm-7, .col-sm-8, .col-sm-9, .col-sm-10, .col-sm-11, .col-sm-12 {
    float: left;
  }
  .col-sm-12 {
    width: 100%;
  }
  .col-sm-11 {
    width: 91.66666667%;
  }
  .col-sm-10 {
    width: 83.33333333%;
  }
  .col-sm-9 {
    width: 75%;
  }
  .col-sm-8 {
    width: 66.66666667%;
  }
  .col-sm-7 {
    width: 58.33333333%;
  }
  .col-sm-6 {
    width: 50%;
  }
  .col-sm-5 {
    width: 41.66666667%;
  }
  .col-sm-4 {
    width: 33.33333333%;
  }
  .col-sm-3 {
    width: 25%;
  }
  .col-sm-2 {
    width: 16.66666667%;
  }
  .col-sm-1 {
    width: 8.33333333%;
  }
  .col-sm-pull-12 {
    right: 100%;
  }
  .col-sm-pull-11 {
    right: 91.66666667%;
  }
  .col-sm-pull-10 {
    right: 83.33333333%;
  }
  .col-sm-pull-9 {
    right: 75%;
  }
  .col-sm-pull-8 {
    right: 66.66666667%;
  }
  .col-sm-pull-7 {
    right: 58.33333333%;
  }
  .col-sm-pull-6 {
    right: 50%;
  }
  .col-sm-pull-5 {
    right: 41.66666667%;
  }
  .col-sm-pull-4 {
    right: 33.33333333%;
  }
  .col-sm-pull-3 {
    right: 25%;
  }
  .col-sm-pull-2 {
    right: 16.66666667%;
  }
  .col-sm-pull-1 {
    right: 8.33333333%;
  }
  .col-sm-pull-0 {
    right: auto;
  }
  .col-sm-push-12 {
    left: 100%;
  }
  .col-sm-push-11 {
    left: 91.66666667%;
  }
  .col-sm-push-10 {
    left: 83.33333333%;
  }
  .col-sm-push-9 {
    left: 75%;
  }
  .col-sm-push-8 {
    left: 66.66666667%;
  }
  .col-sm-push-7 {
    left: 58.33333333%;
  }
  .col-sm-push-6 {
    left: 50%;
  }
  .col-sm-push-5 {
    left: 41.66666667%;
  }
  .col-sm-push-4 {
    left: 33.33333333%;
  }
  .col-sm-push-3 {
    left: 25%;
  }
  .col-sm-push-2 {
    left: 16.66666667%;
  }
  .col-sm-push-1 {
    left: 8.33333333%;
  }
  .col-sm-push-0 {
    left: auto;
  }
  .col-sm-offset-12 {
    margin-left: 100%;
  }
  .col-sm-offset-11 {
    margin-left: 91.66666667%;
  }
  .col-sm-offset-10 {
    margin-left: 83.33333333%;
  }
  .col-sm-offset-9 {
    margin-left: 75%;
  }
  .col-sm-offset-8 {
    margin-left: 66.66666667%;
  }
  .col-sm-offset-7 {
    margin-left: 58.33333333%;
  }
  .col-sm-offset-6 {
    margin-left: 50%;
  }
  .col-sm-offset-5 {
    margin-left: 41.66666667%;
  }
  .col-sm-offset-4 {
    margin-left: 33.33333333%;
  }
  .col-sm-offset-3 {
    margin-left: 25%;
  }
  .col-sm-offset-2 {
    margin-left: 16.66666667%;
  }
  .col-sm-offset-1 {
    margin-left: 8.33333333%;
  }
  .col-sm-offset-0 {
    margin-left: 0%;
  }
}
@media (min-width: 992px) {
  .col-md-1, .col-md-2, .col-md-3, .col-md-4, .col-md-5, .col-md-6, .col-md-7, .col-md-8, .col-md-9, .col-md-10, .col-md-11, .col-md-12 {
    float: left;
  }
  .col-md-12 {
    width: 100%;
  }
  .col-md-11 {
    width: 91.66666667%;
  }
  .col-md-10 {
    width: 83.33333333%;
  }
  .col-md-9 {
    width: 75%;
  }
  .col-md-8 {
    width: 66.66666667%;
  }
  .col-md-7 {
    width: 58.33333333%;
  }
  .col-md-6 {
    width: 50%;
  }
  .col-md-5 {
    width: 41.66666667%;
  }
  .col-md-4 {
    width: 33.33333333%;
  }
  .col-md-3 {
    width: 25%;
  }
  .col-md-2 {
    width: 16.66666667%;
  }
  .col-md-1 {
    width: 8.33333333%;
  }
  .col-md-pull-12 {
    right: 100%;
  }
  .col-md-pull-11 {
    right: 91.66666667%;
  }
  .col-md-pull-10 {
    right: 83.33333333%;
  }
  .col-md-pull-9 {
    right: 75%;
  }
  .col-md-pull-8 {
    right: 66.66666667%;
  }
  .col-md-pull-7 {
    right: 58.33333333%;
  }
  .col-md-pull-6 {
    right: 50%;
  }
  .col-md-pull-5 {
    right: 41.66666667%;
  }
  .col-md-pull-4 {
    right: 33.33333333%;
  }
  .col-md-pull-3 {
    right: 25%;
  }
  .col-md-pull-2 {
    right: 16.66666667%;
  }
  .col-md-pull-1 {
    right: 8.33333333%;
  }
  .col-md-pull-0 {
    right: auto;
  }
  .col-md-push-12 {
    left: 100%;
  }
  .col-md-push-11 {
    left: 91.66666667%;
  }
  .col-md-push-10 {
    left: 83.33333333%;
  }
  .col-md-push-9 {
    left: 75%;
  }
  .col-md-push-8 {
    left: 66.66666667%;
  }
  .col-md-push-7 {
    left: 58.33333333%;
  }
  .col-md-push-6 {
    left: 50%;
  }
  .col-md-push-5 {
    left: 41.66666667%;
  }
  .col-md-push-4 {
    left: 33.33333333%;
  }
  .col-md-push-3 {
    left: 25%;
  }
  .col-md-push-2 {
    left: 16.66666667%;
  }
  .col-md-push-1 {
    left: 8.33333333%;
  }
  .col-md-push-0 {
    left: auto;
  }
  .col-md-offset-12 {
    margin-left: 100%;
  }
  .col-md-offset-11 {
    margin-left: 91.66666667%;
  }
  .col-md-offset-10 {
    margin-left: 83.33333333%;
  }
  .col-md-offset-9 {
    margin-left: 75%;
  }
  .col-md-offset-8 {
    margin-left: 66.66666667%;
  }
  .col-md-offset-7 {
    margin-left: 58.33333333%;
  }
  .col-md-offset-6 {
    margin-left: 50%;
  }
  .col-md-offset-5 {
    margin-left: 41.66666667%;
  }
  .col-md-offset-4 {
    margin-left: 33.33333333%;
  }
  .col-md-offset-3 {
    margin-left: 25%;
  }
  .col-md-offset-2 {
    margin-left: 16.66666667%;
  }
  .col-md-offset-1 {
    margin-left: 8.33333333%;
  }
  .col-md-offset-0 {
    margin-left: 0%;
  }
}
@media (min-width: 1200px) {
  .col-lg-1, .col-lg-2, .col-lg-3, .col-lg-4, .col-lg-5, .col-lg-6, .col-lg-7, .col-lg-8, .col-lg-9, .col-lg-10, .col-lg-11, .col-lg-12 {
    float: left;
  }
  .col-lg-12 {
    width: 100%;
  }
  .col-lg-11 {
    width: 91.66666667%;
  }
  .col-lg-10 {
    width: 83.33333333%;
  }
  .col-lg-9 {
    width: 75%;
  }
  .col-lg-8 {
    width: 66.66666667%;
  }
  .col-lg-7 {
    width: 58.33333333%;
  }
  .col-lg-6 {
    width: 50%;
  }
  .col-lg-5 {
    width: 41.66666667%;
  }
  .col-lg-4 {
    width: 33.33333333%;
  }
  .col-lg-3 {
    width: 25%;
  }
  .col-lg-2 {
    width: 16.66666667%;
  }
  .col-lg-1 {
    width: 8.33333333%;
  }
  .col-lg-pull-12 {
    right: 100%;
  }
  .col-lg-pull-11 {
    right: 91.66666667%;
  }
  .col-lg-pull-10 {
    right: 83.33333333%;
  }
  .col-lg-pull-9 {
    right: 75%;
  }
  .col-lg-pull-8 {
    right: 66.66666667%;
  }
  .col-lg-pull-7 {
    right: 58.33333333%;
  }
  .col-lg-pull-6 {
    right: 50%;
  }
  .col-lg-pull-5 {
    right: 41.66666667%;
  }
  .col-lg-pull-4 {
    right: 33.33333333%;
  }
  .col-lg-pull-3 {
    right: 25%;
  }
  .col-lg-pull-2 {
    right: 16.66666667%;
  }
  .col-lg-pull-1 {
    right: 8.33333333%;
  }
  .col-lg-pull-0 {
    right: auto;
  }
  .col-lg-push-12 {
    left: 100%;
  }
  .col-lg-push-11 {
    left: 91.66666667%;
  }
  .col-lg-push-10 {
    left: 83.33333333%;
  }
  .col-lg-push-9 {
    left: 75%;
  }
  .col-lg-push-8 {
    left: 66.66666667%;
  }
  .col-lg-push-7 {
    left: 58.33333333%;
  }
  .col-lg-push-6 {
    left: 50%;
  }
  .col-lg-push-5 {
    left: 41.66666667%;
  }
  .col-lg-push-4 {
    left: 33.33333333%;
  }
  .col-lg-push-3 {
    left: 25%;
  }
  .col-lg-push-2 {
    left: 16.66666667%;
  }
  .col-lg-push-1 {
    left: 8.33333333%;
  }
  .col-lg-push-0 {
    left: auto;
  }
  .col-lg-offset-12 {
    margin-left: 100%;
  }
  .col-lg-offset-11 {
    margin-left: 91.66666667%;
  }
  .col-lg-offset-10 {
    margin-left: 83.33333333%;
  }
  .col-lg-offset-9 {
    margin-left: 75%;
  }
  .col-lg-offset-8 {
    margin-left: 66.66666667%;
  }
  .col-lg-offset-7 {
    margin-left: 58.33333333%;
  }
  .col-lg-offset-6 {
    margin-left: 50%;
  }
  .col-lg-offset-5 {
    margin-left: 41.66666667%;
  }
  .col-lg-offset-4 {
    margin-left: 33.33333333%;
  }
  .col-lg-offset-3 {
    margin-left: 25%;
  }
  .col-lg-offset-2 {
    margin-left: 16.66666667%;
  }
  .col-lg-offset-1 {
    margin-left: 8.33333333%;
  }
  .col-lg-offset-0 {
    margin-left: 0%;
  }
}
table {
  background-color: transparent;
}
caption {
  padding-top: 8px;
  padding-bottom: 8px;
  color: #777777;
  text-align: left;
}
th {
  text-align: left;
}
.table {
  width: 100%;
  max-width: 100%;
  margin-bottom: 18px;
}
.table > thead > tr > th,
.table > tbody > tr > th,
.table > tfoot > tr > th,
.table > thead > tr > td,
.table > tbody > tr > td,
.table > tfoot > tr > td {
  padding: 8px;
  line-height: 1.42857143;
  vertical-align: top;
  border-top: 1px solid #ddd;
}
.table > thead > tr > th {
  vertical-align: bottom;
  border-bottom: 2px solid #ddd;
}
.table > caption + thead > tr:first-child > th,
.table > colgroup + thead > tr:first-child > th,
.table > thead:first-child > tr:first-child > th,
.table > caption + thead > tr:first-child > td,
.table > colgroup + thead > tr:first-child > td,
.table > thead:first-child > tr:first-child > td {
  border-top: 0;
}
.table > tbody + tbody {
  border-top: 2px solid #ddd;
}
.table .table {
  background-color: #fff;
}
.table-condensed > thead > tr > th,
.table-condensed > tbody > tr > th,
.table-condensed > tfoot > tr > th,
.table-condensed > thead > tr > td,
.table-condensed > tbody > tr > td,
.table-condensed > tfoot > tr > td {
  padding: 5px;
}
.table-bordered {
  border: 1px solid #ddd;
}
.table-bordered > thead > tr > th,
.table-bordered > tbody > tr > th,
.table-bordered > tfoot > tr > th,
.table-bordered > thead > tr > td,
.table-bordered > tbody > tr > td,
.table-bordered > tfoot > tr > td {
  border: 1px solid #ddd;
}
.table-bordered > thead > tr > th,
.table-bordered > thead > tr > td {
  border-bottom-width: 2px;
}
.table-striped > tbody > tr:nth-of-type(odd) {
  background-color: #f9f9f9;
}
.table-hover > tbody > tr:hover {
  background-color: #f5f5f5;
}
table col[class*="col-"] {
  position: static;
  float: none;
  display: table-column;
}
table td[class*="col-"],
table th[class*="col-"] {
  position: static;
  float: none;
  display: table-cell;
}
.table > thead > tr > td.active,
.table > tbody > tr > td.active,
.table > tfoot > tr > td.active,
.table > thead > tr > th.active,
.table > tbody > tr > th.active,
.table > tfoot > tr > th.active,
.table > thead > tr.active > td,
.table > tbody > tr.active > td,
.table > tfoot > tr.active > td,
.table > thead > tr.active > th,
.table > tbody > tr.active > th,
.table > tfoot > tr.active > th {
  background-color: #f5f5f5;
}
.table-hover > tbody > tr > td.active:hover,
.table-hover > tbody > tr > th.active:hover,
.table-hover > tbody > tr.active:hover > td,
.table-hover > tbody > tr:hover > .active,
.table-hover > tbody > tr.active:hover > th {
  background-color: #e8e8e8;
}
.table > thead > tr > td.success,
.table > tbody > tr > td.success,
.table > tfoot > tr > td.success,
.table > thead > tr > th.success,
.table > tbody > tr > th.success,
.table > tfoot > tr > th.success,
.table > thead > tr.success > td,
.table > tbody > tr.success > td,
.table > tfoot > tr.success > td,
.table > thead > tr.success > th,
.table > tbody > tr.success > th,
.table > tfoot > tr.success > th {
  background-color: #dff0d8;
}
.table-hover > tbody > tr > td.success:hover,
.table-hover > tbody > tr > th.success:hover,
.table-hover > tbody > tr.success:hover > td,
.table-hover > tbody > tr:hover > .success,
.table-hover > tbody > tr.success:hover > th {
  background-color: #d0e9c6;
}
.table > thead > tr > td.info,
.table > tbody > tr > td.info,
.table > tfoot > tr > td.info,
.table > thead > tr > th.info,
.table > tbody > tr > th.info,
.table > tfoot > tr > th.info,
.table > thead > tr.info > td,
.table > tbody > tr.info > td,
.table > tfoot > tr.info > td,
.table > thead > tr.info > th,
.table > tbody > tr.info > th,
.table > tfoot > tr.info > th {
  background-color: #d9edf7;
}
.table-hover > tbody > tr > td.info:hover,
.table-hover > tbody > tr > th.info:hover,
.table-hover > tbody > tr.info:hover > td,
.table-hover > tbody > tr:hover > .info,
.table-hover > tbody > tr.info:hover > th {
  background-color: #c4e3f3;
}
.table > thead > tr > td.warning,
.table > tbody > tr > td.warning,
.table > tfoot > tr > td.warning,
.table > thead > tr > th.warning,
.table > tbody > tr > th.warning,
.table > tfoot > tr > th.warning,
.table > thead > tr.warning > td,
.table > tbody > tr.warning > td,
.table > tfoot > tr.warning > td,
.table > thead > tr.warning > th,
.table > tbody > tr.warning > th,
.table > tfoot > tr.warning > th {
  background-color: #fcf8e3;
}
.table-hover > tbody > tr > td.warning:hover,
.table-hover > tbody > tr > th.warning:hover,
.table-hover > tbody > tr.warning:hover > td,
.table-hover > tbody > tr:hover > .warning,
.table-hover > tbody > tr.warning:hover > th {
  background-color: #faf2cc;
}
.table > thead > tr > td.danger,
.table > tbody > tr > td.danger,
.table > tfoot > tr > td.danger,
.table > thead > tr > th.danger,
.table > tbody > tr > th.danger,
.table > tfoot > tr > th.danger,
.table > thead > tr.danger > td,
.table > tbody > tr.danger > td,
.table > tfoot > tr.danger > td,
.table > thead > tr.danger > th,
.table > tbody > tr.danger > th,
.table > tfoot > tr.danger > th {
  background-color: #f2dede;
}
.table-hover > tbody > tr > td.danger:hover,
.table-hover > tbody > tr > th.danger:hover,
.table-hover > tbody > tr.danger:hover > td,
.table-hover > tbody > tr:hover > .danger,
.table-hover > tbody > tr.danger:hover > th {
  background-color: #ebcccc;
}
.table-responsive {
  overflow-x: auto;
  min-height: 0.01%;
}
@media screen and (max-width: 767px) {
  .table-responsive {
    width: 100%;
    margin-bottom: 13.5px;
    overflow-y: hidden;
    -ms-overflow-style: -ms-autohiding-scrollbar;
    border: 1px solid #ddd;
  }
  .table-responsive > .table {
    margin-bottom: 0;
  }
  .table-responsive > .table > thead > tr > th,
  .table-responsive > .table > tbody > tr > th,
  .table-responsive > .table > tfoot > tr > th,
  .table-responsive > .table > thead > tr > td,
  .table-responsive > .table > tbody > tr > td,
  .table-responsive > .table > tfoot > tr > td {
    white-space: nowrap;
  }
  .table-responsive > .table-bordered {
    border: 0;
  }
  .table-responsive > .table-bordered > thead > tr > th:first-child,
  .table-responsive > .table-bordered > tbody > tr > th:first-child,
  .table-responsive > .table-bordered > tfoot > tr > th:first-child,
  .table-responsive > .table-bordered > thead > tr > td:first-child,
  .table-responsive > .table-bordered > tbody > tr > td:first-child,
  .table-responsive > .table-bordered > tfoot > tr > td:first-child {
    border-left: 0;
  }
  .table-responsive > .table-bordered > thead > tr > th:last-child,
  .table-responsive > .table-bordered > tbody > tr > th:last-child,
  .table-responsive > .table-bordered > tfoot > tr > th:last-child,
  .table-responsive > .table-bordered > thead > tr > td:last-child,
  .table-responsive > .table-bordered > tbody > tr > td:last-child,
  .table-responsive > .table-bordered > tfoot > tr > td:last-child {
    border-right: 0;
  }
  .table-responsive > .table-bordered > tbody > tr:last-child > th,
  .table-responsive > .table-bordered > tfoot > tr:last-child > th,
  .table-responsive > .table-bordered > tbody > tr:last-child > td,
  .table-responsive > .table-bordered > tfoot > tr:last-child > td {
    border-bottom: 0;
  }
}
fieldset {
  padding: 0;
  margin: 0;
  border: 0;
  min-width: 0;
}
legend {
  display: block;
  width: 100%;
  padding: 0;
  margin-bottom: 18px;
  font-size: 19.5px;
  line-height: inherit;
  color: #333333;
  border: 0;
  border-bottom: 1px solid #e5e5e5;
}
label {
  display: inline-block;
  max-width: 100%;
  margin-bottom: 5px;
  font-weight: bold;
}
input[type="search"] {
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
}
input[type="radio"],
input[type="checkbox"] {
  margin: 4px 0 0;
  margin-top: 1px \9;
  line-height: normal;
}
input[type="file"] {
  display: block;
}
input[type="range"] {
  display: block;
  width: 100%;
}
select[multiple],
select[size] {
  height: auto;
}
input[type="file"]:focus,
input[type="radio"]:focus,
input[type="checkbox"]:focus {
  outline: 5px auto -webkit-focus-ring-color;
  outline-offset: -2px;
}
output {
  display: block;
  padding-top: 7px;
  font-size: 13px;
  line-height: 1.42857143;
  color: #555555;
}
.form-control {
  display: block;
  width: 100%;
  height: 32px;
  padding: 6px 12px;
  font-size: 13px;
  line-height: 1.42857143;
  color: #555555;
  background-color: #fff;
  background-image: none;
  border: 1px solid #ccc;
  border-radius: 2px;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  -webkit-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  -o-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
}
.form-control:focus {
  border-color: #66afe9;
  outline: 0;
  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);
  box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);
}
.form-control::-moz-placeholder {
  color: #999;
  opacity: 1;
}
.form-control:-ms-input-placeholder {
  color: #999;
}
.form-control::-webkit-input-placeholder {
  color: #999;
}
.form-control::-ms-expand {
  border: 0;
  background-color: transparent;
}
.form-control[disabled],
.form-control[readonly],
fieldset[disabled] .form-control {
  background-color: #eeeeee;
  opacity: 1;
}
.form-control[disabled],
fieldset[disabled] .form-control {
  cursor: not-allowed;
}
textarea.form-control {
  height: auto;
}
input[type="search"] {
  -webkit-appearance: none;
}
@media screen and (-webkit-min-device-pixel-ratio: 0) {
  input[type="date"].form-control,
  input[type="time"].form-control,
  input[type="datetime-local"].form-control,
  input[type="month"].form-control {
    line-height: 32px;
  }
  input[type="date"].input-sm,
  input[type="time"].input-sm,
  input[type="datetime-local"].input-sm,
  input[type="month"].input-sm,
  .input-group-sm input[type="date"],
  .input-group-sm input[type="time"],
  .input-group-sm input[type="datetime-local"],
  .input-group-sm input[type="month"] {
    line-height: 30px;
  }
  input[type="date"].input-lg,
  input[type="time"].input-lg,
  input[type="datetime-local"].input-lg,
  input[type="month"].input-lg,
  .input-group-lg input[type="date"],
  .input-group-lg input[type="time"],
  .input-group-lg input[type="datetime-local"],
  .input-group-lg input[type="month"] {
    line-height: 45px;
  }
}
.form-group {
  margin-bottom: 15px;
}
.radio,
.checkbox {
  position: relative;
  display: block;
  margin-top: 10px;
  margin-bottom: 10px;
}
.radio label,
.checkbox label {
  min-height: 18px;
  padding-left: 20px;
  margin-bottom: 0;
  font-weight: normal;
  cursor: pointer;
}
.radio input[type="radio"],
.radio-inline input[type="radio"],
.checkbox input[type="checkbox"],
.checkbox-inline input[type="checkbox"] {
  position: absolute;
  margin-left: -20px;
  margin-top: 4px \9;
}
.radio + .radio,
.checkbox + .checkbox {
  margin-top: -5px;
}
.radio-inline,
.checkbox-inline {
  position: relative;
  display: inline-block;
  padding-left: 20px;
  margin-bottom: 0;
  vertical-align: middle;
  font-weight: normal;
  cursor: pointer;
}
.radio-inline + .radio-inline,
.checkbox-inline + .checkbox-inline {
  margin-top: 0;
  margin-left: 10px;
}
input[type="radio"][disabled],
input[type="checkbox"][disabled],
input[type="radio"].disabled,
input[type="checkbox"].disabled,
fieldset[disabled] input[type="radio"],
fieldset[disabled] input[type="checkbox"] {
  cursor: not-allowed;
}
.radio-inline.disabled,
.checkbox-inline.disabled,
fieldset[disabled] .radio-inline,
fieldset[disabled] .checkbox-inline {
  cursor: not-allowed;
}
.radio.disabled label,
.checkbox.disabled label,
fieldset[disabled] .radio label,
fieldset[disabled] .checkbox label {
  cursor: not-allowed;
}
.form-control-static {
  padding-top: 7px;
  padding-bottom: 7px;
  margin-bottom: 0;
  min-height: 31px;
}
.form-control-static.input-lg,
.form-control-static.input-sm {
  padding-left: 0;
  padding-right: 0;
}
.input-sm {
  height: 30px;
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
}
select.input-sm {
  height: 30px;
  line-height: 30px;
}
textarea.input-sm,
select[multiple].input-sm {
  height: auto;
}
.form-group-sm .form-control {
  height: 30px;
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
}
.form-group-sm select.form-control {
  height: 30px;
  line-height: 30px;
}
.form-group-sm textarea.form-control,
.form-group-sm select[multiple].form-control {
  height: auto;
}
.form-group-sm .form-control-static {
  height: 30px;
  min-height: 30px;
  padding: 6px 10px;
  font-size: 12px;
  line-height: 1.5;
}
.input-lg {
  height: 45px;
  padding: 10px 16px;
  font-size: 17px;
  line-height: 1.3333333;
  border-radius: 3px;
}
select.input-lg {
  height: 45px;
  line-height: 45px;
}
textarea.input-lg,
select[multiple].input-lg {
  height: auto;
}
.form-group-lg .form-control {
  height: 45px;
  padding: 10px 16px;
  font-size: 17px;
  line-height: 1.3333333;
  border-radius: 3px;
}
.form-group-lg select.form-control {
  height: 45px;
  line-height: 45px;
}
.form-group-lg textarea.form-control,
.form-group-lg select[multiple].form-control {
  height: auto;
}
.form-group-lg .form-control-static {
  height: 45px;
  min-height: 35px;
  padding: 11px 16px;
  font-size: 17px;
  line-height: 1.3333333;
}
.has-feedback {
  position: relative;
}
.has-feedback .form-control {
  padding-right: 40px;
}
.form-control-feedback {
  position: absolute;
  top: 0;
  right: 0;
  z-index: 2;
  display: block;
  width: 32px;
  height: 32px;
  line-height: 32px;
  text-align: center;
  pointer-events: none;
}
.input-lg + .form-control-feedback,
.input-group-lg + .form-control-feedback,
.form-group-lg .form-control + .form-control-feedback {
  width: 45px;
  height: 45px;
  line-height: 45px;
}
.input-sm + .form-control-feedback,
.input-group-sm + .form-control-feedback,
.form-group-sm .form-control + .form-control-feedback {
  width: 30px;
  height: 30px;
  line-height: 30px;
}
.has-success .help-block,
.has-success .control-label,
.has-success .radio,
.has-success .checkbox,
.has-success .radio-inline,
.has-success .checkbox-inline,
.has-success.radio label,
.has-success.checkbox label,
.has-success.radio-inline label,
.has-success.checkbox-inline label {
  color: #3c763d;
}
.has-success .form-control {
  border-color: #3c763d;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
}
.has-success .form-control:focus {
  border-color: #2b542c;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
}
.has-success .input-group-addon {
  color: #3c763d;
  border-color: #3c763d;
  background-color: #dff0d8;
}
.has-success .form-control-feedback {
  color: #3c763d;
}
.has-warning .help-block,
.has-warning .control-label,
.has-warning .radio,
.has-warning .checkbox,
.has-warning .radio-inline,
.has-warning .checkbox-inline,
.has-warning.radio label,
.has-warning.checkbox label,
.has-warning.radio-inline label,
.has-warning.checkbox-inline label {
  color: #8a6d3b;
}
.has-warning .form-control {
  border-color: #8a6d3b;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
}
.has-warning .form-control:focus {
  border-color: #66512c;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #c0a16b;
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #c0a16b;
}
.has-warning .input-group-addon {
  color: #8a6d3b;
  border-color: #8a6d3b;
  background-color: #fcf8e3;
}
.has-warning .form-control-feedback {
  color: #8a6d3b;
}
.has-error .help-block,
.has-error .control-label,
.has-error .radio,
.has-error .checkbox,
.has-error .radio-inline,
.has-error .checkbox-inline,
.has-error.radio label,
.has-error.checkbox label,
.has-error.radio-inline label,
.has-error.checkbox-inline label {
  color: #a94442;
}
.has-error .form-control {
  border-color: #a94442;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
}
.has-error .form-control:focus {
  border-color: #843534;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #ce8483;
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #ce8483;
}
.has-error .input-group-addon {
  color: #a94442;
  border-color: #a94442;
  background-color: #f2dede;
}
.has-error .form-control-feedback {
  color: #a94442;
}
.has-feedback label ~ .form-control-feedback {
  top: 23px;
}
.has-feedback label.sr-only ~ .form-control-feedback {
  top: 0;
}
.help-block {
  display: block;
  margin-top: 5px;
  margin-bottom: 10px;
  color: #404040;
}
@media (min-width: 768px) {
  .form-inline .form-group {
    display: inline-block;
    margin-bottom: 0;
    vertical-align: middle;
  }
  .form-inline .form-control {
    display: inline-block;
    width: auto;
    vertical-align: middle;
  }
  .form-inline .form-control-static {
    display: inline-block;
  }
  .form-inline .input-group {
    display: inline-table;
    vertical-align: middle;
  }
  .form-inline .input-group .input-group-addon,
  .form-inline .input-group .input-group-btn,
  .form-inline .input-group .form-control {
    width: auto;
  }
  .form-inline .input-group > .form-control {
    width: 100%;
  }
  .form-inline .control-label {
    margin-bottom: 0;
    vertical-align: middle;
  }
  .form-inline .radio,
  .form-inline .checkbox {
    display: inline-block;
    margin-top: 0;
    margin-bottom: 0;
    vertical-align: middle;
  }
  .form-inline .radio label,
  .form-inline .checkbox label {
    padding-left: 0;
  }
  .form-inline .radio input[type="radio"],
  .form-inline .checkbox input[type="checkbox"] {
    position: relative;
    margin-left: 0;
  }
  .form-inline .has-feedback .form-control-feedback {
    top: 0;
  }
}
.form-horizontal .radio,
.form-horizontal .checkbox,
.form-horizontal .radio-inline,
.form-horizontal .checkbox-inline {
  margin-top: 0;
  margin-bottom: 0;
  padding-top: 7px;
}
.form-horizontal .radio,
.form-horizontal .checkbox {
  min-height: 25px;
}
.form-horizontal .form-group {
  margin-left: 0px;
  margin-right: 0px;
}
@media (min-width: 768px) {
  .form-horizontal .control-label {
    text-align: right;
    margin-bottom: 0;
    padding-top: 7px;
  }
}
.form-horizontal .has-feedback .form-control-feedback {
  right: 0px;
}
@media (min-width: 768px) {
  .form-horizontal .form-group-lg .control-label {
    padding-top: 11px;
    font-size: 17px;
  }
}
@media (min-width: 768px) {
  .form-horizontal .form-group-sm .control-label {
    padding-top: 6px;
    font-size: 12px;
  }
}
.btn {
  display: inline-block;
  margin-bottom: 0;
  font-weight: normal;
  text-align: center;
  vertical-align: middle;
  touch-action: manipulation;
  cursor: pointer;
  background-image: none;
  border: 1px solid transparent;
  white-space: nowrap;
  padding: 6px 12px;
  font-size: 13px;
  line-height: 1.42857143;
  border-radius: 2px;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
.btn:focus,
.btn:active:focus,
.btn.active:focus,
.btn.focus,
.btn:active.focus,
.btn.active.focus {
  outline: 5px auto -webkit-focus-ring-color;
  outline-offset: -2px;
}
.btn:hover,
.btn:focus,
.btn.focus {
  color: #333;
  text-decoration: none;
}
.btn:active,
.btn.active {
  outline: 0;
  background-image: none;
  -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
  box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
}
.btn.disabled,
.btn[disabled],
fieldset[disabled] .btn {
  cursor: not-allowed;
  opacity: 0.65;
  filter: alpha(opacity=65);
  -webkit-box-shadow: none;
  box-shadow: none;
}
a.btn.disabled,
fieldset[disabled] a.btn {
  pointer-events: none;
}
.btn-default {
  color: #333;
  background-color: #fff;
  border-color: #ccc;
}
.btn-default:focus,
.btn-default.focus {
  color: #333;
  background-color: #e6e6e6;
  border-color: #8c8c8c;
}
.btn-default:hover {
  color: #333;
  background-color: #e6e6e6;
  border-color: #adadad;
}
.btn-default:active,
.btn-default.active,
.open > .dropdown-toggle.btn-default {
  color: #333;
  background-color: #e6e6e6;
  border-color: #adadad;
}
.btn-default:active:hover,
.btn-default.active:hover,
.open > .dropdown-toggle.btn-default:hover,
.btn-default:active:focus,
.btn-default.active:focus,
.open > .dropdown-toggle.btn-default:focus,
.btn-default:active.focus,
.btn-default.active.focus,
.open > .dropdown-toggle.btn-default.focus {
  color: #333;
  background-color: #d4d4d4;
  border-color: #8c8c8c;
}
.btn-default:active,
.btn-default.active,
.open > .dropdown-toggle.btn-default {
  background-image: none;
}
.btn-default.disabled:hover,
.btn-default[disabled]:hover,
fieldset[disabled] .btn-default:hover,
.btn-default.disabled:focus,
.btn-default[disabled]:focus,
fieldset[disabled] .btn-default:focus,
.btn-default.disabled.focus,
.btn-default[disabled].focus,
fieldset[disabled] .btn-default.focus {
  background-color: #fff;
  border-color: #ccc;
}
.btn-default .badge {
  color: #fff;
  background-color: #333;
}
.btn-primary {
  color: #fff;
  background-color: #337ab7;
  border-color: #2e6da4;
}
.btn-primary:focus,
.btn-primary.focus {
  color: #fff;
  background-color: #286090;
  border-color: #122b40;
}
.btn-primary:hover {
  color: #fff;
  background-color: #286090;
  border-color: #204d74;
}
.btn-primary:active,
.btn-primary.active,
.open > .dropdown-toggle.btn-primary {
  color: #fff;
  background-color: #286090;
  border-color: #204d74;
}
.btn-primary:active:hover,
.btn-primary.active:hover,
.open > .dropdown-toggle.btn-primary:hover,
.btn-primary:active:focus,
.btn-primary.active:focus,
.open > .dropdown-toggle.btn-primary:focus,
.btn-primary:active.focus,
.btn-primary.active.focus,
.open > .dropdown-toggle.btn-primary.focus {
  color: #fff;
  background-color: #204d74;
  border-color: #122b40;
}
.btn-primary:active,
.btn-primary.active,
.open > .dropdown-toggle.btn-primary {
  background-image: none;
}
.btn-primary.disabled:hover,
.btn-primary[disabled]:hover,
fieldset[disabled] .btn-primary:hover,
.btn-primary.disabled:focus,
.btn-primary[disabled]:focus,
fieldset[disabled] .btn-primary:focus,
.btn-primary.disabled.focus,
.btn-primary[disabled].focus,
fieldset[disabled] .btn-primary.focus {
  background-color: #337ab7;
  border-color: #2e6da4;
}
.btn-primary .badge {
  color: #337ab7;
  background-color: #fff;
}
.btn-success {
  color: #fff;
  background-color: #5cb85c;
  border-color: #4cae4c;
}
.btn-success:focus,
.btn-success.focus {
  color: #fff;
  background-color: #449d44;
  border-color: #255625;
}
.btn-success:hover {
  color: #fff;
  background-color: #449d44;
  border-color: #398439;
}
.btn-success:active,
.btn-success.active,
.open > .dropdown-toggle.btn-success {
  color: #fff;
  background-color: #449d44;
  border-color: #398439;
}
.btn-success:active:hover,
.btn-success.active:hover,
.open > .dropdown-toggle.btn-success:hover,
.btn-success:active:focus,
.btn-success.active:focus,
.open > .dropdown-toggle.btn-success:focus,
.btn-success:active.focus,
.btn-success.active.focus,
.open > .dropdown-toggle.btn-success.focus {
  color: #fff;
  background-color: #398439;
  border-color: #255625;
}
.btn-success:active,
.btn-success.active,
.open > .dropdown-toggle.btn-success {
  background-image: none;
}
.btn-success.disabled:hover,
.btn-success[disabled]:hover,
fieldset[disabled] .btn-success:hover,
.btn-success.disabled:focus,
.btn-success[disabled]:focus,
fieldset[disabled] .btn-success:focus,
.btn-success.disabled.focus,
.btn-success[disabled].focus,
fieldset[disabled] .btn-success.focus {
  background-color: #5cb85c;
  border-color: #4cae4c;
}
.btn-success .badge {
  color: #5cb85c;
  background-color: #fff;
}
.btn-info {
  color: #fff;
  background-color: #5bc0de;
  border-color: #46b8da;
}
.btn-info:focus,
.btn-info.focus {
  color: #fff;
  background-color: #31b0d5;
  border-color: #1b6d85;
}
.btn-info:hover {
  color: #fff;
  background-color: #31b0d5;
  border-color: #269abc;
}
.btn-info:active,
.btn-info.active,
.open > .dropdown-toggle.btn-info {
  color: #fff;
  background-color: #31b0d5;
  border-color: #269abc;
}
.btn-info:active:hover,
.btn-info.active:hover,
.open > .dropdown-toggle.btn-info:hover,
.btn-info:active:focus,
.btn-info.active:focus,
.open > .dropdown-toggle.btn-info:focus,
.btn-info:active.focus,
.btn-info.active.focus,
.open > .dropdown-toggle.btn-info.focus {
  color: #fff;
  background-color: #269abc;
  border-color: #1b6d85;
}
.btn-info:active,
.btn-info.active,
.open > .dropdown-toggle.btn-info {
  background-image: none;
}
.btn-info.disabled:hover,
.btn-info[disabled]:hover,
fieldset[disabled] .btn-info:hover,
.btn-info.disabled:focus,
.btn-info[disabled]:focus,
fieldset[disabled] .btn-info:focus,
.btn-info.disabled.focus,
.btn-info[disabled].focus,
fieldset[disabled] .btn-info.focus {
  background-color: #5bc0de;
  border-color: #46b8da;
}
.btn-info .badge {
  color: #5bc0de;
  background-color: #fff;
}
.btn-warning {
  color: #fff;
  background-color: #f0ad4e;
  border-color: #eea236;
}
.btn-warning:focus,
.btn-warning.focus {
  color: #fff;
  background-color: #ec971f;
  border-color: #985f0d;
}
.btn-warning:hover {
  color: #fff;
  background-color: #ec971f;
  border-color: #d58512;
}
.btn-warning:active,
.btn-warning.active,
.open > .dropdown-toggle.btn-warning {
  color: #fff;
  background-color: #ec971f;
  border-color: #d58512;
}
.btn-warning:active:hover,
.btn-warning.active:hover,
.open > .dropdown-toggle.btn-warning:hover,
.btn-warning:active:focus,
.btn-warning.active:focus,
.open > .dropdown-toggle.btn-warning:focus,
.btn-warning:active.focus,
.btn-warning.active.focus,
.open > .dropdown-toggle.btn-warning.focus {
  color: #fff;
  background-color: #d58512;
  border-color: #985f0d;
}
.btn-warning:active,
.btn-warning.active,
.open > .dropdown-toggle.btn-warning {
  background-image: none;
}
.btn-warning.disabled:hover,
.btn-warning[disabled]:hover,
fieldset[disabled] .btn-warning:hover,
.btn-warning.disabled:focus,
.btn-warning[disabled]:focus,
fieldset[disabled] .btn-warning:focus,
.btn-warning.disabled.focus,
.btn-warning[disabled].focus,
fieldset[disabled] .btn-warning.focus {
  background-color: #f0ad4e;
  border-color: #eea236;
}
.btn-warning .badge {
  color: #f0ad4e;
  background-color: #fff;
}
.btn-danger {
  color: #fff;
  background-color: #d9534f;
  border-color: #d43f3a;
}
.btn-danger:focus,
.btn-danger.focus {
  color: #fff;
  background-color: #c9302c;
  border-color: #761c19;
}
.btn-danger:hover {
  color: #fff;
  background-color: #c9302c;
  border-color: #ac2925;
}
.btn-danger:active,
.btn-danger.active,
.open > .dropdown-toggle.btn-danger {
  color: #fff;
  background-color: #c9302c;
  border-color: #ac2925;
}
.btn-danger:active:hover,
.btn-danger.active:hover,
.open > .dropdown-toggle.btn-danger:hover,
.btn-danger:active:focus,
.btn-danger.active:focus,
.open > .dropdown-toggle.btn-danger:focus,
.btn-danger:active.focus,
.btn-danger.active.focus,
.open > .dropdown-toggle.btn-danger.focus {
  color: #fff;
  background-color: #ac2925;
  border-color: #761c19;
}
.btn-danger:active,
.btn-danger.active,
.open > .dropdown-toggle.btn-danger {
  background-image: none;
}
.btn-danger.disabled:hover,
.btn-danger[disabled]:hover,
fieldset[disabled] .btn-danger:hover,
.btn-danger.disabled:focus,
.btn-danger[disabled]:focus,
fieldset[disabled] .btn-danger:focus,
.btn-danger.disabled.focus,
.btn-danger[disabled].focus,
fieldset[disabled] .btn-danger.focus {
  background-color: #d9534f;
  border-color: #d43f3a;
}
.btn-danger .badge {
  color: #d9534f;
  background-color: #fff;
}
.btn-link {
  color: #337ab7;
  font-weight: normal;
  border-radius: 0;
}
.btn-link,
.btn-link:active,
.btn-link.active,
.btn-link[disabled],
fieldset[disabled] .btn-link {
  background-color: transparent;
  -webkit-box-shadow: none;
  box-shadow: none;
}
.btn-link,
.btn-link:hover,
.btn-link:focus,
.btn-link:active {
  border-color: transparent;
}
.btn-link:hover,
.btn-link:focus {
  color: #23527c;
  text-decoration: underline;
  background-color: transparent;
}
.btn-link[disabled]:hover,
fieldset[disabled] .btn-link:hover,
.btn-link[disabled]:focus,
fieldset[disabled] .btn-link:focus {
  color: #777777;
  text-decoration: none;
}
.btn-lg,
.btn-group-lg > .btn {
  padding: 10px 16px;
  font-size: 17px;
  line-height: 1.3333333;
  border-radius: 3px;
}
.btn-sm,
.btn-group-sm > .btn {
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
}
.btn-xs,
.btn-group-xs > .btn {
  padding: 1px 5px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
}
.btn-block {
  display: block;
  width: 100%;
}
.btn-block + .btn-block {
  margin-top: 5px;
}
input[type="submit"].btn-block,
input[type="reset"].btn-block,
input[type="button"].btn-block {
  width: 100%;
}
.fade {
  opacity: 0;
  -webkit-transition: opacity 0.15s linear;
  -o-transition: opacity 0.15s linear;
  transition: opacity 0.15s linear;
}
.fade.in {
  opacity: 1;
}
.collapse {
  display: none;
}
.collapse.in {
  display: block;
}
tr.collapse.in {
  display: table-row;
}
tbody.collapse.in {
  display: table-row-group;
}
.collapsing {
  position: relative;
  height: 0;
  overflow: hidden;
  -webkit-transition-property: height, visibility;
  transition-property: height, visibility;
  -webkit-transition-duration: 0.35s;
  transition-duration: 0.35s;
  -webkit-transition-timing-function: ease;
  transition-timing-function: ease;
}
.caret {
  display: inline-block;
  width: 0;
  height: 0;
  margin-left: 2px;
  vertical-align: middle;
  border-top: 4px dashed;
  border-top: 4px solid \9;
  border-right: 4px solid transparent;
  border-left: 4px solid transparent;
}
.dropup,
.dropdown {
  position: relative;
}
.dropdown-toggle:focus {
  outline: 0;
}
.dropdown-menu {
  position: absolute;
  top: 100%;
  left: 0;
  z-index: 1000;
  display: none;
  float: left;
  min-width: 160px;
  padding: 5px 0;
  margin: 2px 0 0;
  list-style: none;
  font-size: 13px;
  text-align: left;
  background-color: #fff;
  border: 1px solid #ccc;
  border: 1px solid rgba(0, 0, 0, 0.15);
  border-radius: 2px;
  -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
  background-clip: padding-box;
}
.dropdown-menu.pull-right {
  right: 0;
  left: auto;
}
.dropdown-menu .divider {
  height: 1px;
  margin: 8px 0;
  overflow: hidden;
  background-color: #e5e5e5;
}
.dropdown-menu > li > a {
  display: block;
  padding: 3px 20px;
  clear: both;
  font-weight: normal;
  line-height: 1.42857143;
  color: #333333;
  white-space: nowrap;
}
.dropdown-menu > li > a:hover,
.dropdown-menu > li > a:focus {
  text-decoration: none;
  color: #262626;
  background-color: #f5f5f5;
}
.dropdown-menu > .active > a,
.dropdown-menu > .active > a:hover,
.dropdown-menu > .active > a:focus {
  color: #fff;
  text-decoration: none;
  outline: 0;
  background-color: #337ab7;
}
.dropdown-menu > .disabled > a,
.dropdown-menu > .disabled > a:hover,
.dropdown-menu > .disabled > a:focus {
  color: #777777;
}
.dropdown-menu > .disabled > a:hover,
.dropdown-menu > .disabled > a:focus {
  text-decoration: none;
  background-color: transparent;
  background-image: none;
  filter: progid:DXImageTransform.Microsoft.gradient(enabled = false);
  cursor: not-allowed;
}
.open > .dropdown-menu {
  display: block;
}
.open > a {
  outline: 0;
}
.dropdown-menu-right {
  left: auto;
  right: 0;
}
.dropdown-menu-left {
  left: 0;
  right: auto;
}
.dropdown-header {
  display: block;
  padding: 3px 20px;
  font-size: 12px;
  line-height: 1.42857143;
  color: #777777;
  white-space: nowrap;
}
.dropdown-backdrop {
  position: fixed;
  left: 0;
  right: 0;
  bottom: 0;
  top: 0;
  z-index: 990;
}
.pull-right > .dropdown-menu {
  right: 0;
  left: auto;
}
.dropup .caret,
.navbar-fixed-bottom .dropdown .caret {
  border-top: 0;
  border-bottom: 4px dashed;
  border-bottom: 4px solid \9;
  content: "";
}
.dropup .dropdown-menu,
.navbar-fixed-bottom .dropdown .dropdown-menu {
  top: auto;
  bottom: 100%;
  margin-bottom: 2px;
}
@media (min-width: 541px) {
  .navbar-right .dropdown-menu {
    left: auto;
    right: 0;
  }
  .navbar-right .dropdown-menu-left {
    left: 0;
    right: auto;
  }
}
.btn-group,
.btn-group-vertical {
  position: relative;
  display: inline-block;
  vertical-align: middle;
}
.btn-group > .btn,
.btn-group-vertical > .btn {
  position: relative;
  float: left;
}
.btn-group > .btn:hover,
.btn-group-vertical > .btn:hover,
.btn-group > .btn:focus,
.btn-group-vertical > .btn:focus,
.btn-group > .btn:active,
.btn-group-vertical > .btn:active,
.btn-group > .btn.active,
.btn-group-vertical > .btn.active {
  z-index: 2;
}
.btn-group .btn + .btn,
.btn-group .btn + .btn-group,
.btn-group .btn-group + .btn,
.btn-group .btn-group + .btn-group {
  margin-left: -1px;
}
.btn-toolbar {
  margin-left: -5px;
}
.btn-toolbar .btn,
.btn-toolbar .btn-group,
.btn-toolbar .input-group {
  float: left;
}
.btn-toolbar > .btn,
.btn-toolbar > .btn-group,
.btn-toolbar > .input-group {
  margin-left: 5px;
}
.btn-group > .btn:not(:first-child):not(:last-child):not(.dropdown-toggle) {
  border-radius: 0;
}
.btn-group > .btn:first-child {
  margin-left: 0;
}
.btn-group > .btn:first-child:not(:last-child):not(.dropdown-toggle) {
  border-bottom-right-radius: 0;
  border-top-right-radius: 0;
}
.btn-group > .btn:last-child:not(:first-child),
.btn-group > .dropdown-toggle:not(:first-child) {
  border-bottom-left-radius: 0;
  border-top-left-radius: 0;
}
.btn-group > .btn-group {
  float: left;
}
.btn-group > .btn-group:not(:first-child):not(:last-child) > .btn {
  border-radius: 0;
}
.btn-group > .btn-group:first-child:not(:last-child) > .btn:last-child,
.btn-group > .btn-group:first-child:not(:last-child) > .dropdown-toggle {
  border-bottom-right-radius: 0;
  border-top-right-radius: 0;
}
.btn-group > .btn-group:last-child:not(:first-child) > .btn:first-child {
  border-bottom-left-radius: 0;
  border-top-left-radius: 0;
}
.btn-group .dropdown-toggle:active,
.btn-group.open .dropdown-toggle {
  outline: 0;
}
.btn-group > .btn + .dropdown-toggle {
  padding-left: 8px;
  padding-right: 8px;
}
.btn-group > .btn-lg + .dropdown-toggle {
  padding-left: 12px;
  padding-right: 12px;
}
.btn-group.open .dropdown-toggle {
  -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
  box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
}
.btn-group.open .dropdown-toggle.btn-link {
  -webkit-box-shadow: none;
  box-shadow: none;
}
.btn .caret {
  margin-left: 0;
}
.btn-lg .caret {
  border-width: 5px 5px 0;
  border-bottom-width: 0;
}
.dropup .btn-lg .caret {
  border-width: 0 5px 5px;
}
.btn-group-vertical > .btn,
.btn-group-vertical > .btn-group,
.btn-group-vertical > .btn-group > .btn {
  display: block;
  float: none;
  width: 100%;
  max-width: 100%;
}
.btn-group-vertical > .btn-group > .btn {
  float: none;
}
.btn-group-vertical > .btn + .btn,
.btn-group-vertical > .btn + .btn-group,
.btn-group-vertical > .btn-group + .btn,
.btn-group-vertical > .btn-group + .btn-group {
  margin-top: -1px;
  margin-left: 0;
}
.btn-group-vertical > .btn:not(:first-child):not(:last-child) {
  border-radius: 0;
}
.btn-group-vertical > .btn:first-child:not(:last-child) {
  border-top-right-radius: 2px;
  border-top-left-radius: 2px;
  border-bottom-right-radius: 0;
  border-bottom-left-radius: 0;
}
.btn-group-vertical > .btn:last-child:not(:first-child) {
  border-top-right-radius: 0;
  border-top-left-radius: 0;
  border-bottom-right-radius: 2px;
  border-bottom-left-radius: 2px;
}
.btn-group-vertical > .btn-group:not(:first-child):not(:last-child) > .btn {
  border-radius: 0;
}
.btn-group-vertical > .btn-group:first-child:not(:last-child) > .btn:last-child,
.btn-group-vertical > .btn-group:first-child:not(:last-child) > .dropdown-toggle {
  border-bottom-right-radius: 0;
  border-bottom-left-radius: 0;
}
.btn-group-vertical > .btn-group:last-child:not(:first-child) > .btn:first-child {
  border-top-right-radius: 0;
  border-top-left-radius: 0;
}
.btn-group-justified {
  display: table;
  width: 100%;
  table-layout: fixed;
  border-collapse: separate;
}
.btn-group-justified > .btn,
.btn-group-justified > .btn-group {
  float: none;
  display: table-cell;
  width: 1%;
}
.btn-group-justified > .btn-group .btn {
  width: 100%;
}
.btn-group-justified > .btn-group .dropdown-menu {
  left: auto;
}
[data-toggle="buttons"] > .btn input[type="radio"],
[data-toggle="buttons"] > .btn-group > .btn input[type="radio"],
[data-toggle="buttons"] > .btn input[type="checkbox"],
[data-toggle="buttons"] > .btn-group > .btn input[type="checkbox"] {
  position: absolute;
  clip: rect(0, 0, 0, 0);
  pointer-events: none;
}
.input-group {
  position: relative;
  display: table;
  border-collapse: separate;
}
.input-group[class*="col-"] {
  float: none;
  padding-left: 0;
  padding-right: 0;
}
.input-group .form-control {
  position: relative;
  z-index: 2;
  float: left;
  width: 100%;
  margin-bottom: 0;
}
.input-group .form-control:focus {
  z-index: 3;
}
.input-group-lg > .form-control,
.input-group-lg > .input-group-addon,
.input-group-lg > .input-group-btn > .btn {
  height: 45px;
  padding: 10px 16px;
  font-size: 17px;
  line-height: 1.3333333;
  border-radius: 3px;
}
select.input-group-lg > .form-control,
select.input-group-lg > .input-group-addon,
select.input-group-lg > .input-group-btn > .btn {
  height: 45px;
  line-height: 45px;
}
textarea.input-group-lg > .form-control,
textarea.input-group-lg > .input-group-addon,
textarea.input-group-lg > .input-group-btn > .btn,
select[multiple].input-group-lg > .form-control,
select[multiple].input-group-lg > .input-group-addon,
select[multiple].input-group-lg > .input-group-btn > .btn {
  height: auto;
}
.input-group-sm > .form-control,
.input-group-sm > .input-group-addon,
.input-group-sm > .input-group-btn > .btn {
  height: 30px;
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
}
select.input-group-sm > .form-control,
select.input-group-sm > .input-group-addon,
select.input-group-sm > .input-group-btn > .btn {
  height: 30px;
  line-height: 30px;
}
textarea.input-group-sm > .form-control,
textarea.input-group-sm > .input-group-addon,
textarea.input-group-sm > .input-group-btn > .btn,
select[multiple].input-group-sm > .form-control,
select[multiple].input-group-sm > .input-group-addon,
select[multiple].input-group-sm > .input-group-btn > .btn {
  height: auto;
}
.input-group-addon,
.input-group-btn,
.input-group .form-control {
  display: table-cell;
}
.input-group-addon:not(:first-child):not(:last-child),
.input-group-btn:not(:first-child):not(:last-child),
.input-group .form-control:not(:first-child):not(:last-child) {
  border-radius: 0;
}
.input-group-addon,
.input-group-btn {
  width: 1%;
  white-space: nowrap;
  vertical-align: middle;
}
.input-group-addon {
  padding: 6px 12px;
  font-size: 13px;
  font-weight: normal;
  line-height: 1;
  color: #555555;
  text-align: center;
  background-color: #eeeeee;
  border: 1px solid #ccc;
  border-radius: 2px;
}
.input-group-addon.input-sm {
  padding: 5px 10px;
  font-size: 12px;
  border-radius: 1px;
}
.input-group-addon.input-lg {
  padding: 10px 16px;
  font-size: 17px;
  border-radius: 3px;
}
.input-group-addon input[type="radio"],
.input-group-addon input[type="checkbox"] {
  margin-top: 0;
}
.input-group .form-control:first-child,
.input-group-addon:first-child,
.input-group-btn:first-child > .btn,
.input-group-btn:first-child > .btn-group > .btn,
.input-group-btn:first-child > .dropdown-toggle,
.input-group-btn:last-child > .btn:not(:last-child):not(.dropdown-toggle),
.input-group-btn:last-child > .btn-group:not(:last-child) > .btn {
  border-bottom-right-radius: 0;
  border-top-right-radius: 0;
}
.input-group-addon:first-child {
  border-right: 0;
}
.input-group .form-control:last-child,
.input-group-addon:last-child,
.input-group-btn:last-child > .btn,
.input-group-btn:last-child > .btn-group > .btn,
.input-group-btn:last-child > .dropdown-toggle,
.input-group-btn:first-child > .btn:not(:first-child),
.input-group-btn:first-child > .btn-group:not(:first-child) > .btn {
  border-bottom-left-radius: 0;
  border-top-left-radius: 0;
}
.input-group-addon:last-child {
  border-left: 0;
}
.input-group-btn {
  position: relative;
  font-size: 0;
  white-space: nowrap;
}
.input-group-btn > .btn {
  position: relative;
}
.input-group-btn > .btn + .btn {
  margin-left: -1px;
}
.input-group-btn > .btn:hover,
.input-group-btn > .btn:focus,
.input-group-btn > .btn:active {
  z-index: 2;
}
.input-group-btn:first-child > .btn,
.input-group-btn:first-child > .btn-group {
  margin-right: -1px;
}
.input-group-btn:last-child > .btn,
.input-group-btn:last-child > .btn-group {
  z-index: 2;
  margin-left: -1px;
}
.nav {
  margin-bottom: 0;
  padding-left: 0;
  list-style: none;
}
.nav > li {
  position: relative;
  display: block;
}
.nav > li > a {
  position: relative;
  display: block;
  padding: 10px 15px;
}
.nav > li > a:hover,
.nav > li > a:focus {
  text-decoration: none;
  background-color: #eeeeee;
}
.nav > li.disabled > a {
  color: #777777;
}
.nav > li.disabled > a:hover,
.nav > li.disabled > a:focus {
  color: #777777;
  text-decoration: none;
  background-color: transparent;
  cursor: not-allowed;
}
.nav .open > a,
.nav .open > a:hover,
.nav .open > a:focus {
  background-color: #eeeeee;
  border-color: #337ab7;
}
.nav .nav-divider {
  height: 1px;
  margin: 8px 0;
  overflow: hidden;
  background-color: #e5e5e5;
}
.nav > li > a > img {
  max-width: none;
}
.nav-tabs {
  border-bottom: 1px solid #ddd;
}
.nav-tabs > li {
  float: left;
  margin-bottom: -1px;
}
.nav-tabs > li > a {
  margin-right: 2px;
  line-height: 1.42857143;
  border: 1px solid transparent;
  border-radius: 2px 2px 0 0;
}
.nav-tabs > li > a:hover {
  border-color: #eeeeee #eeeeee #ddd;
}
.nav-tabs > li.active > a,
.nav-tabs > li.active > a:hover,
.nav-tabs > li.active > a:focus {
  color: #555555;
  background-color: #fff;
  border: 1px solid #ddd;
  border-bottom-color: transparent;
  cursor: default;
}
.nav-tabs.nav-justified {
  width: 100%;
  border-bottom: 0;
}
.nav-tabs.nav-justified > li {
  float: none;
}
.nav-tabs.nav-justified > li > a {
  text-align: center;
  margin-bottom: 5px;
}
.nav-tabs.nav-justified > .dropdown .dropdown-menu {
  top: auto;
  left: auto;
}
@media (min-width: 768px) {
  .nav-tabs.nav-justified > li {
    display: table-cell;
    width: 1%;
  }
  .nav-tabs.nav-justified > li > a {
    margin-bottom: 0;
  }
}
.nav-tabs.nav-justified > li > a {
  margin-right: 0;
  border-radius: 2px;
}
.nav-tabs.nav-justified > .active > a,
.nav-tabs.nav-justified > .active > a:hover,
.nav-tabs.nav-justified > .active > a:focus {
  border: 1px solid #ddd;
}
@media (min-width: 768px) {
  .nav-tabs.nav-justified > li > a {
    border-bottom: 1px solid #ddd;
    border-radius: 2px 2px 0 0;
  }
  .nav-tabs.nav-justified > .active > a,
  .nav-tabs.nav-justified > .active > a:hover,
  .nav-tabs.nav-justified > .active > a:focus {
    border-bottom-color: #fff;
  }
}
.nav-pills > li {
  float: left;
}
.nav-pills > li > a {
  border-radius: 2px;
}
.nav-pills > li + li {
  margin-left: 2px;
}
.nav-pills > li.active > a,
.nav-pills > li.active > a:hover,
.nav-pills > li.active > a:focus {
  color: #fff;
  background-color: #337ab7;
}
.nav-stacked > li {
  float: none;
}
.nav-stacked > li + li {
  margin-top: 2px;
  margin-left: 0;
}
.nav-justified {
  width: 100%;
}
.nav-justified > li {
  float: none;
}
.nav-justified > li > a {
  text-align: center;
  margin-bottom: 5px;
}
.nav-justified > .dropdown .dropdown-menu {
  top: auto;
  left: auto;
}
@media (min-width: 768px) {
  .nav-justified > li {
    display: table-cell;
    width: 1%;
  }
  .nav-justified > li > a {
    margin-bottom: 0;
  }
}
.nav-tabs-justified {
  border-bottom: 0;
}
.nav-tabs-justified > li > a {
  margin-right: 0;
  border-radius: 2px;
}
.nav-tabs-justified > .active > a,
.nav-tabs-justified > .active > a:hover,
.nav-tabs-justified > .active > a:focus {
  border: 1px solid #ddd;
}
@media (min-width: 768px) {
  .nav-tabs-justified > li > a {
    border-bottom: 1px solid #ddd;
    border-radius: 2px 2px 0 0;
  }
  .nav-tabs-justified > .active > a,
  .nav-tabs-justified > .active > a:hover,
  .nav-tabs-justified > .active > a:focus {
    border-bottom-color: #fff;
  }
}
.tab-content > .tab-pane {
  display: none;
}
.tab-content > .active {
  display: block;
}
.nav-tabs .dropdown-menu {
  margin-top: -1px;
  border-top-right-radius: 0;
  border-top-left-radius: 0;
}
.navbar {
  position: relative;
  min-height: 30px;
  margin-bottom: 18px;
  border: 1px solid transparent;
}
@media (min-width: 541px) {
  .navbar {
    border-radius: 2px;
  }
}
@media (min-width: 541px) {
  .navbar-header {
    float: left;
  }
}
.navbar-collapse {
  overflow-x: visible;
  padding-right: 0px;
  padding-left: 0px;
  border-top: 1px solid transparent;
  box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.1);
  -webkit-overflow-scrolling: touch;
}
.navbar-collapse.in {
  overflow-y: auto;
}
@media (min-width: 541px) {
  .navbar-collapse {
    width: auto;
    border-top: 0;
    box-shadow: none;
  }
  .navbar-collapse.collapse {
    display: block !important;
    height: auto !important;
    padding-bottom: 0;
    overflow: visible !important;
  }
  .navbar-collapse.in {
    overflow-y: visible;
  }
  .navbar-fixed-top .navbar-collapse,
  .navbar-static-top .navbar-collapse,
  .navbar-fixed-bottom .navbar-collapse {
    padding-left: 0;
    padding-right: 0;
  }
}
.navbar-fixed-top .navbar-collapse,
.navbar-fixed-bottom .navbar-collapse {
  max-height: 340px;
}
@media (max-device-width: 540px) and (orientation: landscape) {
  .navbar-fixed-top .navbar-collapse,
  .navbar-fixed-bottom .navbar-collapse {
    max-height: 200px;
  }
}
.container > .navbar-header,
.container-fluid > .navbar-header,
.container > .navbar-collapse,
.container-fluid > .navbar-collapse {
  margin-right: 0px;
  margin-left: 0px;
}
@media (min-width: 541px) {
  .container > .navbar-header,
  .container-fluid > .navbar-header,
  .container > .navbar-collapse,
  .container-fluid > .navbar-collapse {
    margin-right: 0;
    margin-left: 0;
  }
}
.navbar-static-top {
  z-index: 1000;
  border-width: 0 0 1px;
}
@media (min-width: 541px) {
  .navbar-static-top {
    border-radius: 0;
  }
}
.navbar-fixed-top,
.navbar-fixed-bottom {
  position: fixed;
  right: 0;
  left: 0;
  z-index: 1030;
}
@media (min-width: 541px) {
  .navbar-fixed-top,
  .navbar-fixed-bottom {
    border-radius: 0;
  }
}
.navbar-fixed-top {
  top: 0;
  border-width: 0 0 1px;
}
.navbar-fixed-bottom {
  bottom: 0;
  margin-bottom: 0;
  border-width: 1px 0 0;
}
.navbar-brand {
  float: left;
  padding: 6px 0px;
  font-size: 17px;
  line-height: 18px;
  height: 30px;
}
.navbar-brand:hover,
.navbar-brand:focus {
  text-decoration: none;
}
.navbar-brand > img {
  display: block;
}
@media (min-width: 541px) {
  .navbar > .container .navbar-brand,
  .navbar > .container-fluid .navbar-brand {
    margin-left: 0px;
  }
}
.navbar-toggle {
  position: relative;
  float: right;
  margin-right: 0px;
  padding: 9px 10px;
  margin-top: -2px;
  margin-bottom: -2px;
  background-color: transparent;
  background-image: none;
  border: 1px solid transparent;
  border-radius: 2px;
}
.navbar-toggle:focus {
  outline: 0;
}
.navbar-toggle .icon-bar {
  display: block;
  width: 22px;
  height: 2px;
  border-radius: 1px;
}
.navbar-toggle .icon-bar + .icon-bar {
  margin-top: 4px;
}
@media (min-width: 541px) {
  .navbar-toggle {
    display: none;
  }
}
.navbar-nav {
  margin: 3px 0px;
}
.navbar-nav > li > a {
  padding-top: 10px;
  padding-bottom: 10px;
  line-height: 18px;
}
@media (max-width: 540px) {
  .navbar-nav .open .dropdown-menu {
    position: static;
    float: none;
    width: auto;
    margin-top: 0;
    background-color: transparent;
    border: 0;
    box-shadow: none;
  }
  .navbar-nav .open .dropdown-menu > li > a,
  .navbar-nav .open .dropdown-menu .dropdown-header {
    padding: 5px 15px 5px 25px;
  }
  .navbar-nav .open .dropdown-menu > li > a {
    line-height: 18px;
  }
  .navbar-nav .open .dropdown-menu > li > a:hover,
  .navbar-nav .open .dropdown-menu > li > a:focus {
    background-image: none;
  }
}
@media (min-width: 541px) {
  .navbar-nav {
    float: left;
    margin: 0;
  }
  .navbar-nav > li {
    float: left;
  }
  .navbar-nav > li > a {
    padding-top: 6px;
    padding-bottom: 6px;
  }
}
.navbar-form {
  margin-left: 0px;
  margin-right: 0px;
  padding: 10px 0px;
  border-top: 1px solid transparent;
  border-bottom: 1px solid transparent;
  -webkit-box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.1), 0 1px 0 rgba(255, 255, 255, 0.1);
  box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.1), 0 1px 0 rgba(255, 255, 255, 0.1);
  margin-top: -1px;
  margin-bottom: -1px;
}
@media (min-width: 768px) {
  .navbar-form .form-group {
    display: inline-block;
    margin-bottom: 0;
    vertical-align: middle;
  }
  .navbar-form .form-control {
    display: inline-block;
    width: auto;
    vertical-align: middle;
  }
  .navbar-form .form-control-static {
    display: inline-block;
  }
  .navbar-form .input-group {
    display: inline-table;
    vertical-align: middle;
  }
  .navbar-form .input-group .input-group-addon,
  .navbar-form .input-group .input-group-btn,
  .navbar-form .input-group .form-control {
    width: auto;
  }
  .navbar-form .input-group > .form-control {
    width: 100%;
  }
  .navbar-form .control-label {
    margin-bottom: 0;
    vertical-align: middle;
  }
  .navbar-form .radio,
  .navbar-form .checkbox {
    display: inline-block;
    margin-top: 0;
    margin-bottom: 0;
    vertical-align: middle;
  }
  .navbar-form .radio label,
  .navbar-form .checkbox label {
    padding-left: 0;
  }
  .navbar-form .radio input[type="radio"],
  .navbar-form .checkbox input[type="checkbox"] {
    position: relative;
    margin-left: 0;
  }
  .navbar-form .has-feedback .form-control-feedback {
    top: 0;
  }
}
@media (max-width: 540px) {
  .navbar-form .form-group {
    margin-bottom: 5px;
  }
  .navbar-form .form-group:last-child {
    margin-bottom: 0;
  }
}
@media (min-width: 541px) {
  .navbar-form {
    width: auto;
    border: 0;
    margin-left: 0;
    margin-right: 0;
    padding-top: 0;
    padding-bottom: 0;
    -webkit-box-shadow: none;
    box-shadow: none;
  }
}
.navbar-nav > li > .dropdown-menu {
  margin-top: 0;
  border-top-right-radius: 0;
  border-top-left-radius: 0;
}
.navbar-fixed-bottom .navbar-nav > li > .dropdown-menu {
  margin-bottom: 0;
  border-top-right-radius: 2px;
  border-top-left-radius: 2px;
  border-bottom-right-radius: 0;
  border-bottom-left-radius: 0;
}
.navbar-btn {
  margin-top: -1px;
  margin-bottom: -1px;
}
.navbar-btn.btn-sm {
  margin-top: 0px;
  margin-bottom: 0px;
}
.navbar-btn.btn-xs {
  margin-top: 4px;
  margin-bottom: 4px;
}
.navbar-text {
  margin-top: 6px;
  margin-bottom: 6px;
}
@media (min-width: 541px) {
  .navbar-text {
    float: left;
    margin-left: 0px;
    margin-right: 0px;
  }
}
@media (min-width: 541px) {
  .navbar-left {
    float: left !important;
    float: left;
  }
  .navbar-right {
    float: right !important;
    float: right;
    margin-right: 0px;
  }
  .navbar-right ~ .navbar-right {
    margin-right: 0;
  }
}
.navbar-default {
  background-color: #f8f8f8;
  border-color: #e7e7e7;
}
.navbar-default .navbar-brand {
  color: #777;
}
.navbar-default .navbar-brand:hover,
.navbar-default .navbar-brand:focus {
  color: #5e5e5e;
  background-color: transparent;
}
.navbar-default .navbar-text {
  color: #777;
}
.navbar-default .navbar-nav > li > a {
  color: #777;
}
.navbar-default .navbar-nav > li > a:hover,
.navbar-default .navbar-nav > li > a:focus {
  color: #333;
  background-color: transparent;
}
.navbar-default .navbar-nav > .active > a,
.navbar-default .navbar-nav > .active > a:hover,
.navbar-default .navbar-nav > .active > a:focus {
  color: #555;
  background-color: #e7e7e7;
}
.navbar-default .navbar-nav > .disabled > a,
.navbar-default .navbar-nav > .disabled > a:hover,
.navbar-default .navbar-nav > .disabled > a:focus {
  color: #ccc;
  background-color: transparent;
}
.navbar-default .navbar-toggle {
  border-color: #ddd;
}
.navbar-default .navbar-toggle:hover,
.navbar-default .navbar-toggle:focus {
  background-color: #ddd;
}
.navbar-default .navbar-toggle .icon-bar {
  background-color: #888;
}
.navbar-default .navbar-collapse,
.navbar-default .navbar-form {
  border-color: #e7e7e7;
}
.navbar-default .navbar-nav > .open > a,
.navbar-default .navbar-nav > .open > a:hover,
.navbar-default .navbar-nav > .open > a:focus {
  background-color: #e7e7e7;
  color: #555;
}
@media (max-width: 540px) {
  .navbar-default .navbar-nav .open .dropdown-menu > li > a {
    color: #777;
  }
  .navbar-default .navbar-nav .open .dropdown-menu > li > a:hover,
  .navbar-default .navbar-nav .open .dropdown-menu > li > a:focus {
    color: #333;
    background-color: transparent;
  }
  .navbar-default .navbar-nav .open .dropdown-menu > .active > a,
  .navbar-default .navbar-nav .open .dropdown-menu > .active > a:hover,
  .navbar-default .navbar-nav .open .dropdown-menu > .active > a:focus {
    color: #555;
    background-color: #e7e7e7;
  }
  .navbar-default .navbar-nav .open .dropdown-menu > .disabled > a,
  .navbar-default .navbar-nav .open .dropdown-menu > .disabled > a:hover,
  .navbar-default .navbar-nav .open .dropdown-menu > .disabled > a:focus {
    color: #ccc;
    background-color: transparent;
  }
}
.navbar-default .navbar-link {
  color: #777;
}
.navbar-default .navbar-link:hover {
  color: #333;
}
.navbar-default .btn-link {
  color: #777;
}
.navbar-default .btn-link:hover,
.navbar-default .btn-link:focus {
  color: #333;
}
.navbar-default .btn-link[disabled]:hover,
fieldset[disabled] .navbar-default .btn-link:hover,
.navbar-default .btn-link[disabled]:focus,
fieldset[disabled] .navbar-default .btn-link:focus {
  color: #ccc;
}
.navbar-inverse {
  background-color: #222;
  border-color: #080808;
}
.navbar-inverse .navbar-brand {
  color: #9d9d9d;
}
.navbar-inverse .navbar-brand:hover,
.navbar-inverse .navbar-brand:focus {
  color: #fff;
  background-color: transparent;
}
.navbar-inverse .navbar-text {
  color: #9d9d9d;
}
.navbar-inverse .navbar-nav > li > a {
  color: #9d9d9d;
}
.navbar-inverse .navbar-nav > li > a:hover,
.navbar-inverse .navbar-nav > li > a:focus {
  color: #fff;
  background-color: transparent;
}
.navbar-inverse .navbar-nav > .active > a,
.navbar-inverse .navbar-nav > .active > a:hover,
.navbar-inverse .navbar-nav > .active > a:focus {
  color: #fff;
  background-color: #080808;
}
.navbar-inverse .navbar-nav > .disabled > a,
.navbar-inverse .navbar-nav > .disabled > a:hover,
.navbar-inverse .navbar-nav > .disabled > a:focus {
  color: #444;
  background-color: transparent;
}
.navbar-inverse .navbar-toggle {
  border-color: #333;
}
.navbar-inverse .navbar-toggle:hover,
.navbar-inverse .navbar-toggle:focus {
  background-color: #333;
}
.navbar-inverse .navbar-toggle .icon-bar {
  background-color: #fff;
}
.navbar-inverse .navbar-collapse,
.navbar-inverse .navbar-form {
  border-color: #101010;
}
.navbar-inverse .navbar-nav > .open > a,
.navbar-inverse .navbar-nav > .open > a:hover,
.navbar-inverse .navbar-nav > .open > a:focus {
  background-color: #080808;
  color: #fff;
}
@media (max-width: 540px) {
  .navbar-inverse .navbar-nav .open .dropdown-menu > .dropdown-header {
    border-color: #080808;
  }
  .navbar-inverse .navbar-nav .open .dropdown-menu .divider {
    background-color: #080808;
  }
  .navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
    color: #9d9d9d;
  }
  .navbar-inverse .navbar-nav .open .dropdown-menu > li > a:hover,
  .navbar-inverse .navbar-nav .open .dropdown-menu > li > a:focus {
    color: #fff;
    background-color: transparent;
  }
  .navbar-inverse .navbar-nav .open .dropdown-menu > .active > a,
  .navbar-inverse .navbar-nav .open .dropdown-menu > .active > a:hover,
  .navbar-inverse .navbar-nav .open .dropdown-menu > .active > a:focus {
    color: #fff;
    background-color: #080808;
  }
  .navbar-inverse .navbar-nav .open .dropdown-menu > .disabled > a,
  .navbar-inverse .navbar-nav .open .dropdown-menu > .disabled > a:hover,
  .navbar-inverse .navbar-nav .open .dropdown-menu > .disabled > a:focus {
    color: #444;
    background-color: transparent;
  }
}
.navbar-inverse .navbar-link {
  color: #9d9d9d;
}
.navbar-inverse .navbar-link:hover {
  color: #fff;
}
.navbar-inverse .btn-link {
  color: #9d9d9d;
}
.navbar-inverse .btn-link:hover,
.navbar-inverse .btn-link:focus {
  color: #fff;
}
.navbar-inverse .btn-link[disabled]:hover,
fieldset[disabled] .navbar-inverse .btn-link:hover,
.navbar-inverse .btn-link[disabled]:focus,
fieldset[disabled] .navbar-inverse .btn-link:focus {
  color: #444;
}
.breadcrumb {
  padding: 8px 15px;
  margin-bottom: 18px;
  list-style: none;
  background-color: #f5f5f5;
  border-radius: 2px;
}
.breadcrumb > li {
  display: inline-block;
}
.breadcrumb > li + li:before {
  content: "/\00a0";
  padding: 0 5px;
  color: #5e5e5e;
}
.breadcrumb > .active {
  color: #777777;
}
.pagination {
  display: inline-block;
  padding-left: 0;
  margin: 18px 0;
  border-radius: 2px;
}
.pagination > li {
  display: inline;
}
.pagination > li > a,
.pagination > li > span {
  position: relative;
  float: left;
  padding: 6px 12px;
  line-height: 1.42857143;
  text-decoration: none;
  color: #337ab7;
  background-color: #fff;
  border: 1px solid #ddd;
  margin-left: -1px;
}
.pagination > li:first-child > a,
.pagination > li:first-child > span {
  margin-left: 0;
  border-bottom-left-radius: 2px;
  border-top-left-radius: 2px;
}
.pagination > li:last-child > a,
.pagination > li:last-child > span {
  border-bottom-right-radius: 2px;
  border-top-right-radius: 2px;
}
.pagination > li > a:hover,
.pagination > li > span:hover,
.pagination > li > a:focus,
.pagination > li > span:focus {
  z-index: 2;
  color: #23527c;
  background-color: #eeeeee;
  border-color: #ddd;
}
.pagination > .active > a,
.pagination > .active > span,
.pagination > .active > a:hover,
.pagination > .active > span:hover,
.pagination > .active > a:focus,
.pagination > .active > span:focus {
  z-index: 3;
  color: #fff;
  background-color: #337ab7;
  border-color: #337ab7;
  cursor: default;
}
.pagination > .disabled > span,
.pagination > .disabled > span:hover,
.pagination > .disabled > span:focus,
.pagination > .disabled > a,
.pagination > .disabled > a:hover,
.pagination > .disabled > a:focus {
  color: #777777;
  background-color: #fff;
  border-color: #ddd;
  cursor: not-allowed;
}
.pagination-lg > li > a,
.pagination-lg > li > span {
  padding: 10px 16px;
  font-size: 17px;
  line-height: 1.3333333;
}
.pagination-lg > li:first-child > a,
.pagination-lg > li:first-child > span {
  border-bottom-left-radius: 3px;
  border-top-left-radius: 3px;
}
.pagination-lg > li:last-child > a,
.pagination-lg > li:last-child > span {
  border-bottom-right-radius: 3px;
  border-top-right-radius: 3px;
}
.pagination-sm > li > a,
.pagination-sm > li > span {
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
}
.pagination-sm > li:first-child > a,
.pagination-sm > li:first-child > span {
  border-bottom-left-radius: 1px;
  border-top-left-radius: 1px;
}
.pagination-sm > li:last-child > a,
.pagination-sm > li:last-child > span {
  border-bottom-right-radius: 1px;
  border-top-right-radius: 1px;
}
.pager {
  padding-left: 0;
  margin: 18px 0;
  list-style: none;
  text-align: center;
}
.pager li {
  display: inline;
}
.pager li > a,
.pager li > span {
  display: inline-block;
  padding: 5px 14px;
  background-color: #fff;
  border: 1px solid #ddd;
  border-radius: 15px;
}
.pager li > a:hover,
.pager li > a:focus {
  text-decoration: none;
  background-color: #eeeeee;
}
.pager .next > a,
.pager .next > span {
  float: right;
}
.pager .previous > a,
.pager .previous > span {
  float: left;
}
.pager .disabled > a,
.pager .disabled > a:hover,
.pager .disabled > a:focus,
.pager .disabled > span {
  color: #777777;
  background-color: #fff;
  cursor: not-allowed;
}
.label {
  display: inline;
  padding: .2em .6em .3em;
  font-size: 75%;
  font-weight: bold;
  line-height: 1;
  color: #fff;
  text-align: center;
  white-space: nowrap;
  vertical-align: baseline;
  border-radius: .25em;
}
a.label:hover,
a.label:focus {
  color: #fff;
  text-decoration: none;
  cursor: pointer;
}
.label:empty {
  display: none;
}
.btn .label {
  position: relative;
  top: -1px;
}
.label-default {
  background-color: #777777;
}
.label-default[href]:hover,
.label-default[href]:focus {
  background-color: #5e5e5e;
}
.label-primary {
  background-color: #337ab7;
}
.label-primary[href]:hover,
.label-primary[href]:focus {
  background-color: #286090;
}
.label-success {
  background-color: #5cb85c;
}
.label-success[href]:hover,
.label-success[href]:focus {
  background-color: #449d44;
}
.label-info {
  background-color: #5bc0de;
}
.label-info[href]:hover,
.label-info[href]:focus {
  background-color: #31b0d5;
}
.label-warning {
  background-color: #f0ad4e;
}
.label-warning[href]:hover,
.label-warning[href]:focus {
  background-color: #ec971f;
}
.label-danger {
  background-color: #d9534f;
}
.label-danger[href]:hover,
.label-danger[href]:focus {
  background-color: #c9302c;
}
.badge {
  display: inline-block;
  min-width: 10px;
  padding: 3px 7px;
  font-size: 12px;
  font-weight: bold;
  color: #fff;
  line-height: 1;
  vertical-align: middle;
  white-space: nowrap;
  text-align: center;
  background-color: #777777;
  border-radius: 10px;
}
.badge:empty {
  display: none;
}
.btn .badge {
  position: relative;
  top: -1px;
}
.btn-xs .badge,
.btn-group-xs > .btn .badge {
  top: 0;
  padding: 1px 5px;
}
a.badge:hover,
a.badge:focus {
  color: #fff;
  text-decoration: none;
  cursor: pointer;
}
.list-group-item.active > .badge,
.nav-pills > .active > a > .badge {
  color: #337ab7;
  background-color: #fff;
}
.list-group-item > .badge {
  float: right;
}
.list-group-item > .badge + .badge {
  margin-right: 5px;
}
.nav-pills > li > a > .badge {
  margin-left: 3px;
}
.jumbotron {
  padding-top: 30px;
  padding-bottom: 30px;
  margin-bottom: 30px;
  color: inherit;
  background-color: #eeeeee;
}
.jumbotron h1,
.jumbotron .h1 {
  color: inherit;
}
.jumbotron p {
  margin-bottom: 15px;
  font-size: 20px;
  font-weight: 200;
}
.jumbotron > hr {
  border-top-color: #d5d5d5;
}
.container .jumbotron,
.container-fluid .jumbotron {
  border-radius: 3px;
  padding-left: 0px;
  padding-right: 0px;
}
.jumbotron .container {
  max-width: 100%;
}
@media screen and (min-width: 768px) {
  .jumbotron {
    padding-top: 48px;
    padding-bottom: 48px;
  }
  .container .jumbotron,
  .container-fluid .jumbotron {
    padding-left: 60px;
    padding-right: 60px;
  }
  .jumbotron h1,
  .jumbotron .h1 {
    font-size: 59px;
  }
}
.thumbnail {
  display: block;
  padding: 4px;
  margin-bottom: 18px;
  line-height: 1.42857143;
  background-color: #fff;
  border: 1px solid #ddd;
  border-radius: 2px;
  -webkit-transition: border 0.2s ease-in-out;
  -o-transition: border 0.2s ease-in-out;
  transition: border 0.2s ease-in-out;
}
.thumbnail > img,
.thumbnail a > img {
  margin-left: auto;
  margin-right: auto;
}
a.thumbnail:hover,
a.thumbnail:focus,
a.thumbnail.active {
  border-color: #337ab7;
}
.thumbnail .caption {
  padding: 9px;
  color: #000;
}
.alert {
  padding: 15px;
  margin-bottom: 18px;
  border: 1px solid transparent;
  border-radius: 2px;
}
.alert h4 {
  margin-top: 0;
  color: inherit;
}
.alert .alert-link {
  font-weight: bold;
}
.alert > p,
.alert > ul {
  margin-bottom: 0;
}
.alert > p + p {
  margin-top: 5px;
}
.alert-dismissable,
.alert-dismissible {
  padding-right: 35px;
}
.alert-dismissable .close,
.alert-dismissible .close {
  position: relative;
  top: -2px;
  right: -21px;
  color: inherit;
}
.alert-success {
  background-color: #dff0d8;
  border-color: #d6e9c6;
  color: #3c763d;
}
.alert-success hr {
  border-top-color: #c9e2b3;
}
.alert-success .alert-link {
  color: #2b542c;
}
.alert-info {
  background-color: #d9edf7;
  border-color: #bce8f1;
  color: #31708f;
}
.alert-info hr {
  border-top-color: #a6e1ec;
}
.alert-info .alert-link {
  color: #245269;
}
.alert-warning {
  background-color: #fcf8e3;
  border-color: #faebcc;
  color: #8a6d3b;
}
.alert-warning hr {
  border-top-color: #f7e1b5;
}
.alert-warning .alert-link {
  color: #66512c;
}
.alert-danger {
  background-color: #f2dede;
  border-color: #ebccd1;
  color: #a94442;
}
.alert-danger hr {
  border-top-color: #e4b9c0;
}
.alert-danger .alert-link {
  color: #843534;
}
@-webkit-keyframes progress-bar-stripes {
  from {
    background-position: 40px 0;
  }
  to {
    background-position: 0 0;
  }
}
@keyframes progress-bar-stripes {
  from {
    background-position: 40px 0;
  }
  to {
    background-position: 0 0;
  }
}
.progress {
  overflow: hidden;
  height: 18px;
  margin-bottom: 18px;
  background-color: #f5f5f5;
  border-radius: 2px;
  -webkit-box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
  box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
}
.progress-bar {
  float: left;
  width: 0%;
  height: 100%;
  font-size: 12px;
  line-height: 18px;
  color: #fff;
  text-align: center;
  background-color: #337ab7;
  -webkit-box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.15);
  box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.15);
  -webkit-transition: width 0.6s ease;
  -o-transition: width 0.6s ease;
  transition: width 0.6s ease;
}
.progress-striped .progress-bar,
.progress-bar-striped {
  background-image: -webkit-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: -o-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-size: 40px 40px;
}
.progress.active .progress-bar,
.progress-bar.active {
  -webkit-animation: progress-bar-stripes 2s linear infinite;
  -o-animation: progress-bar-stripes 2s linear infinite;
  animation: progress-bar-stripes 2s linear infinite;
}
.progress-bar-success {
  background-color: #5cb85c;
}
.progress-striped .progress-bar-success {
  background-image: -webkit-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: -o-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
}
.progress-bar-info {
  background-color: #5bc0de;
}
.progress-striped .progress-bar-info {
  background-image: -webkit-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: -o-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
}
.progress-bar-warning {
  background-color: #f0ad4e;
}
.progress-striped .progress-bar-warning {
  background-image: -webkit-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: -o-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
}
.progress-bar-danger {
  background-color: #d9534f;
}
.progress-striped .progress-bar-danger {
  background-image: -webkit-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: -o-linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
}
.media {
  margin-top: 15px;
}
.media:first-child {
  margin-top: 0;
}
.media,
.media-body {
  zoom: 1;
  overflow: hidden;
}
.media-body {
  width: 10000px;
}
.media-object {
  display: block;
}
.media-object.img-thumbnail {
  max-width: none;
}
.media-right,
.media > .pull-right {
  padding-left: 10px;
}
.media-left,
.media > .pull-left {
  padding-right: 10px;
}
.media-left,
.media-right,
.media-body {
  display: table-cell;
  vertical-align: top;
}
.media-middle {
  vertical-align: middle;
}
.media-bottom {
  vertical-align: bottom;
}
.media-heading {
  margin-top: 0;
  margin-bottom: 5px;
}
.media-list {
  padding-left: 0;
  list-style: none;
}
.list-group {
  margin-bottom: 20px;
  padding-left: 0;
}
.list-group-item {
  position: relative;
  display: block;
  padding: 10px 15px;
  margin-bottom: -1px;
  background-color: #fff;
  border: 1px solid #ddd;
}
.list-group-item:first-child {
  border-top-right-radius: 2px;
  border-top-left-radius: 2px;
}
.list-group-item:last-child {
  margin-bottom: 0;
  border-bottom-right-radius: 2px;
  border-bottom-left-radius: 2px;
}
a.list-group-item,
button.list-group-item {
  color: #555;
}
a.list-group-item .list-group-item-heading,
button.list-group-item .list-group-item-heading {
  color: #333;
}
a.list-group-item:hover,
button.list-group-item:hover,
a.list-group-item:focus,
button.list-group-item:focus {
  text-decoration: none;
  color: #555;
  background-color: #f5f5f5;
}
button.list-group-item {
  width: 100%;
  text-align: left;
}
.list-group-item.disabled,
.list-group-item.disabled:hover,
.list-group-item.disabled:focus {
  background-color: #eeeeee;
  color: #777777;
  cursor: not-allowed;
}
.list-group-item.disabled .list-group-item-heading,
.list-group-item.disabled:hover .list-group-item-heading,
.list-group-item.disabled:focus .list-group-item-heading {
  color: inherit;
}
.list-group-item.disabled .list-group-item-text,
.list-group-item.disabled:hover .list-group-item-text,
.list-group-item.disabled:focus .list-group-item-text {
  color: #777777;
}
.list-group-item.active,
.list-group-item.active:hover,
.list-group-item.active:focus {
  z-index: 2;
  color: #fff;
  background-color: #337ab7;
  border-color: #337ab7;
}
.list-group-item.active .list-group-item-heading,
.list-group-item.active:hover .list-group-item-heading,
.list-group-item.active:focus .list-group-item-heading,
.list-group-item.active .list-group-item-heading > small,
.list-group-item.active:hover .list-group-item-heading > small,
.list-group-item.active:focus .list-group-item-heading > small,
.list-group-item.active .list-group-item-heading > .small,
.list-group-item.active:hover .list-group-item-heading > .small,
.list-group-item.active:focus .list-group-item-heading > .small {
  color: inherit;
}
.list-group-item.active .list-group-item-text,
.list-group-item.active:hover .list-group-item-text,
.list-group-item.active:focus .list-group-item-text {
  color: #c7ddef;
}
.list-group-item-success {
  color: #3c763d;
  background-color: #dff0d8;
}
a.list-group-item-success,
button.list-group-item-success {
  color: #3c763d;
}
a.list-group-item-success .list-group-item-heading,
button.list-group-item-success .list-group-item-heading {
  color: inherit;
}
a.list-group-item-success:hover,
button.list-group-item-success:hover,
a.list-group-item-success:focus,
button.list-group-item-success:focus {
  color: #3c763d;
  background-color: #d0e9c6;
}
a.list-group-item-success.active,
button.list-group-item-success.active,
a.list-group-item-success.active:hover,
button.list-group-item-success.active:hover,
a.list-group-item-success.active:focus,
button.list-group-item-success.active:focus {
  color: #fff;
  background-color: #3c763d;
  border-color: #3c763d;
}
.list-group-item-info {
  color: #31708f;
  background-color: #d9edf7;
}
a.list-group-item-info,
button.list-group-item-info {
  color: #31708f;
}
a.list-group-item-info .list-group-item-heading,
button.list-group-item-info .list-group-item-heading {
  color: inherit;
}
a.list-group-item-info:hover,
button.list-group-item-info:hover,
a.list-group-item-info:focus,
button.list-group-item-info:focus {
  color: #31708f;
  background-color: #c4e3f3;
}
a.list-group-item-info.active,
button.list-group-item-info.active,
a.list-group-item-info.active:hover,
button.list-group-item-info.active:hover,
a.list-group-item-info.active:focus,
button.list-group-item-info.active:focus {
  color: #fff;
  background-color: #31708f;
  border-color: #31708f;
}
.list-group-item-warning {
  color: #8a6d3b;
  background-color: #fcf8e3;
}
a.list-group-item-warning,
button.list-group-item-warning {
  color: #8a6d3b;
}
a.list-group-item-warning .list-group-item-heading,
button.list-group-item-warning .list-group-item-heading {
  color: inherit;
}
a.list-group-item-warning:hover,
button.list-group-item-warning:hover,
a.list-group-item-warning:focus,
button.list-group-item-warning:focus {
  color: #8a6d3b;
  background-color: #faf2cc;
}
a.list-group-item-warning.active,
button.list-group-item-warning.active,
a.list-group-item-warning.active:hover,
button.list-group-item-warning.active:hover,
a.list-group-item-warning.active:focus,
button.list-group-item-warning.active:focus {
  color: #fff;
  background-color: #8a6d3b;
  border-color: #8a6d3b;
}
.list-group-item-danger {
  color: #a94442;
  background-color: #f2dede;
}
a.list-group-item-danger,
button.list-group-item-danger {
  color: #a94442;
}
a.list-group-item-danger .list-group-item-heading,
button.list-group-item-danger .list-group-item-heading {
  color: inherit;
}
a.list-group-item-danger:hover,
button.list-group-item-danger:hover,
a.list-group-item-danger:focus,
button.list-group-item-danger:focus {
  color: #a94442;
  background-color: #ebcccc;
}
a.list-group-item-danger.active,
button.list-group-item-danger.active,
a.list-group-item-danger.active:hover,
button.list-group-item-danger.active:hover,
a.list-group-item-danger.active:focus,
button.list-group-item-danger.active:focus {
  color: #fff;
  background-color: #a94442;
  border-color: #a94442;
}
.list-group-item-heading {
  margin-top: 0;
  margin-bottom: 5px;
}
.list-group-item-text {
  margin-bottom: 0;
  line-height: 1.3;
}
.panel {
  margin-bottom: 18px;
  background-color: #fff;
  border: 1px solid transparent;
  border-radius: 2px;
  -webkit-box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
}
.panel-body {
  padding: 15px;
}
.panel-heading {
  padding: 10px 15px;
  border-bottom: 1px solid transparent;
  border-top-right-radius: 1px;
  border-top-left-radius: 1px;
}
.panel-heading > .dropdown .dropdown-toggle {
  color: inherit;
}
.panel-title {
  margin-top: 0;
  margin-bottom: 0;
  font-size: 15px;
  color: inherit;
}
.panel-title > a,
.panel-title > small,
.panel-title > .small,
.panel-title > small > a,
.panel-title > .small > a {
  color: inherit;
}
.panel-footer {
  padding: 10px 15px;
  background-color: #f5f5f5;
  border-top: 1px solid #ddd;
  border-bottom-right-radius: 1px;
  border-bottom-left-radius: 1px;
}
.panel > .list-group,
.panel > .panel-collapse > .list-group {
  margin-bottom: 0;
}
.panel > .list-group .list-group-item,
.panel > .panel-collapse > .list-group .list-group-item {
  border-width: 1px 0;
  border-radius: 0;
}
.panel > .list-group:first-child .list-group-item:first-child,
.panel > .panel-collapse > .list-group:first-child .list-group-item:first-child {
  border-top: 0;
  border-top-right-radius: 1px;
  border-top-left-radius: 1px;
}
.panel > .list-group:last-child .list-group-item:last-child,
.panel > .panel-collapse > .list-group:last-child .list-group-item:last-child {
  border-bottom: 0;
  border-bottom-right-radius: 1px;
  border-bottom-left-radius: 1px;
}
.panel > .panel-heading + .panel-collapse > .list-group .list-group-item:first-child {
  border-top-right-radius: 0;
  border-top-left-radius: 0;
}
.panel-heading + .list-group .list-group-item:first-child {
  border-top-width: 0;
}
.list-group + .panel-footer {
  border-top-width: 0;
}
.panel > .table,
.panel > .table-responsive > .table,
.panel > .panel-collapse > .table {
  margin-bottom: 0;
}
.panel > .table caption,
.panel > .table-responsive > .table caption,
.panel > .panel-collapse > .table caption {
  padding-left: 15px;
  padding-right: 15px;
}
.panel > .table:first-child,
.panel > .table-responsive:first-child > .table:first-child {
  border-top-right-radius: 1px;
  border-top-left-radius: 1px;
}
.panel > .table:first-child > thead:first-child > tr:first-child,
.panel > .table-responsive:first-child > .table:first-child > thead:first-child > tr:first-child,
.panel > .table:first-child > tbody:first-child > tr:first-child,
.panel > .table-responsive:first-child > .table:first-child > tbody:first-child > tr:first-child {
  border-top-left-radius: 1px;
  border-top-right-radius: 1px;
}
.panel > .table:first-child > thead:first-child > tr:first-child td:first-child,
.panel > .table-responsive:first-child > .table:first-child > thead:first-child > tr:first-child td:first-child,
.panel > .table:first-child > tbody:first-child > tr:first-child td:first-child,
.panel > .table-responsive:first-child > .table:first-child > tbody:first-child > tr:first-child td:first-child,
.panel > .table:first-child > thead:first-child > tr:first-child th:first-child,
.panel > .table-responsive:first-child > .table:first-child > thead:first-child > tr:first-child th:first-child,
.panel > .table:first-child > tbody:first-child > tr:first-child th:first-child,
.panel > .table-responsive:first-child > .table:first-child > tbody:first-child > tr:first-child th:first-child {
  border-top-left-radius: 1px;
}
.panel > .table:first-child > thead:first-child > tr:first-child td:last-child,
.panel > .table-responsive:first-child > .table:first-child > thead:first-child > tr:first-child td:last-child,
.panel > .table:first-child > tbody:first-child > tr:first-child td:last-child,
.panel > .table-responsive:first-child > .table:first-child > tbody:first-child > tr:first-child td:last-child,
.panel > .table:first-child > thead:first-child > tr:first-child th:last-child,
.panel > .table-responsive:first-child > .table:first-child > thead:first-child > tr:first-child th:last-child,
.panel > .table:first-child > tbody:first-child > tr:first-child th:last-child,
.panel > .table-responsive:first-child > .table:first-child > tbody:first-child > tr:first-child th:last-child {
  border-top-right-radius: 1px;
}
.panel > .table:last-child,
.panel > .table-responsive:last-child > .table:last-child {
  border-bottom-right-radius: 1px;
  border-bottom-left-radius: 1px;
}
.panel > .table:last-child > tbody:last-child > tr:last-child,
.panel > .table-responsive:last-child > .table:last-child > tbody:last-child > tr:last-child,
.panel > .table:last-child > tfoot:last-child > tr:last-child,
.panel > .table-responsive:last-child > .table:last-child > tfoot:last-child > tr:last-child {
  border-bottom-left-radius: 1px;
  border-bottom-right-radius: 1px;
}
.panel > .table:last-child > tbody:last-child > tr:last-child td:first-child,
.panel > .table-responsive:last-child > .table:last-child > tbody:last-child > tr:last-child td:first-child,
.panel > .table:last-child > tfoot:last-child > tr:last-child td:first-child,
.panel > .table-responsive:last-child > .table:last-child > tfoot:last-child > tr:last-child td:first-child,
.panel > .table:last-child > tbody:last-child > tr:last-child th:first-child,
.panel > .table-responsive:last-child > .table:last-child > tbody:last-child > tr:last-child th:first-child,
.panel > .table:last-child > tfoot:last-child > tr:last-child th:first-child,
.panel > .table-responsive:last-child > .table:last-child > tfoot:last-child > tr:last-child th:first-child {
  border-bottom-left-radius: 1px;
}
.panel > .table:last-child > tbody:last-child > tr:last-child td:last-child,
.panel > .table-responsive:last-child > .table:last-child > tbody:last-child > tr:last-child td:last-child,
.panel > .table:last-child > tfoot:last-child > tr:last-child td:last-child,
.panel > .table-responsive:last-child > .table:last-child > tfoot:last-child > tr:last-child td:last-child,
.panel > .table:last-child > tbody:last-child > tr:last-child th:last-child,
.panel > .table-responsive:last-child > .table:last-child > tbody:last-child > tr:last-child th:last-child,
.panel > .table:last-child > tfoot:last-child > tr:last-child th:last-child,
.panel > .table-responsive:last-child > .table:last-child > tfoot:last-child > tr:last-child th:last-child {
  border-bottom-right-radius: 1px;
}
.panel > .panel-body + .table,
.panel > .panel-body + .table-responsive,
.panel > .table + .panel-body,
.panel > .table-responsive + .panel-body {
  border-top: 1px solid #ddd;
}
.panel > .table > tbody:first-child > tr:first-child th,
.panel > .table > tbody:first-child > tr:first-child td {
  border-top: 0;
}
.panel > .table-bordered,
.panel > .table-responsive > .table-bordered {
  border: 0;
}
.panel > .table-bordered > thead > tr > th:first-child,
.panel > .table-responsive > .table-bordered > thead > tr > th:first-child,
.panel > .table-bordered > tbody > tr > th:first-child,
.panel > .table-responsive > .table-bordered > tbody > tr > th:first-child,
.panel > .table-bordered > tfoot > tr > th:first-child,
.panel > .table-responsive > .table-bordered > tfoot > tr > th:first-child,
.panel > .table-bordered > thead > tr > td:first-child,
.panel > .table-responsive > .table-bordered > thead > tr > td:first-child,
.panel > .table-bordered > tbody > tr > td:first-child,
.panel > .table-responsive > .table-bordered > tbody > tr > td:first-child,
.panel > .table-bordered > tfoot > tr > td:first-child,
.panel > .table-responsive > .table-bordered > tfoot > tr > td:first-child {
  border-left: 0;
}
.panel > .table-bordered > thead > tr > th:last-child,
.panel > .table-responsive > .table-bordered > thead > tr > th:last-child,
.panel > .table-bordered > tbody > tr > th:last-child,
.panel > .table-responsive > .table-bordered > tbody > tr > th:last-child,
.panel > .table-bordered > tfoot > tr > th:last-child,
.panel > .table-responsive > .table-bordered > tfoot > tr > th:last-child,
.panel > .table-bordered > thead > tr > td:last-child,
.panel > .table-responsive > .table-bordered > thead > tr > td:last-child,
.panel > .table-bordered > tbody > tr > td:last-child,
.panel > .table-responsive > .table-bordered > tbody > tr > td:last-child,
.panel > .table-bordered > tfoot > tr > td:last-child,
.panel > .table-responsive > .table-bordered > tfoot > tr > td:last-child {
  border-right: 0;
}
.panel > .table-bordered > thead > tr:first-child > td,
.panel > .table-responsive > .table-bordered > thead > tr:first-child > td,
.panel > .table-bordered > tbody > tr:first-child > td,
.panel > .table-responsive > .table-bordered > tbody > tr:first-child > td,
.panel > .table-bordered > thead > tr:first-child > th,
.panel > .table-responsive > .table-bordered > thead > tr:first-child > th,
.panel > .table-bordered > tbody > tr:first-child > th,
.panel > .table-responsive > .table-bordered > tbody > tr:first-child > th {
  border-bottom: 0;
}
.panel > .table-bordered > tbody > tr:last-child > td,
.panel > .table-responsive > .table-bordered > tbody > tr:last-child > td,
.panel > .table-bordered > tfoot > tr:last-child > td,
.panel > .table-responsive > .table-bordered > tfoot > tr:last-child > td,
.panel > .table-bordered > tbody > tr:last-child > th,
.panel > .table-responsive > .table-bordered > tbody > tr:last-child > th,
.panel > .table-bordered > tfoot > tr:last-child > th,
.panel > .table-responsive > .table-bordered > tfoot > tr:last-child > th {
  border-bottom: 0;
}
.panel > .table-responsive {
  border: 0;
  margin-bottom: 0;
}
.panel-group {
  margin-bottom: 18px;
}
.panel-group .panel {
  margin-bottom: 0;
  border-radius: 2px;
}
.panel-group .panel + .panel {
  margin-top: 5px;
}
.panel-group .panel-heading {
  border-bottom: 0;
}
.panel-group .panel-heading + .panel-collapse > .panel-body,
.panel-group .panel-heading + .panel-collapse > .list-group {
  border-top: 1px solid #ddd;
}
.panel-group .panel-footer {
  border-top: 0;
}
.panel-group .panel-footer + .panel-collapse .panel-body {
  border-bottom: 1px solid #ddd;
}
.panel-default {
  border-color: #ddd;
}
.panel-default > .panel-heading {
  color: #333333;
  background-color: #f5f5f5;
  border-color: #ddd;
}
.panel-default > .panel-heading + .panel-collapse > .panel-body {
  border-top-color: #ddd;
}
.panel-default > .panel-heading .badge {
  color: #f5f5f5;
  background-color: #333333;
}
.panel-default > .panel-footer + .panel-collapse > .panel-body {
  border-bottom-color: #ddd;
}
.panel-primary {
  border-color: #337ab7;
}
.panel-primary > .panel-heading {
  color: #fff;
  background-color: #337ab7;
  border-color: #337ab7;
}
.panel-primary > .panel-heading + .panel-collapse > .panel-body {
  border-top-color: #337ab7;
}
.panel-primary > .panel-heading .badge {
  color: #337ab7;
  background-color: #fff;
}
.panel-primary > .panel-footer + .panel-collapse > .panel-body {
  border-bottom-color: #337ab7;
}
.panel-success {
  border-color: #d6e9c6;
}
.panel-success > .panel-heading {
  color: #3c763d;
  background-color: #dff0d8;
  border-color: #d6e9c6;
}
.panel-success > .panel-heading + .panel-collapse > .panel-body {
  border-top-color: #d6e9c6;
}
.panel-success > .panel-heading .badge {
  color: #dff0d8;
  background-color: #3c763d;
}
.panel-success > .panel-footer + .panel-collapse > .panel-body {
  border-bottom-color: #d6e9c6;
}
.panel-info {
  border-color: #bce8f1;
}
.panel-info > .panel-heading {
  color: #31708f;
  background-color: #d9edf7;
  border-color: #bce8f1;
}
.panel-info > .panel-heading + .panel-collapse > .panel-body {
  border-top-color: #bce8f1;
}
.panel-info > .panel-heading .badge {
  color: #d9edf7;
  background-color: #31708f;
}
.panel-info > .panel-footer + .panel-collapse > .panel-body {
  border-bottom-color: #bce8f1;
}
.panel-warning {
  border-color: #faebcc;
}
.panel-warning > .panel-heading {
  color: #8a6d3b;
  background-color: #fcf8e3;
  border-color: #faebcc;
}
.panel-warning > .panel-heading + .panel-collapse > .panel-body {
  border-top-color: #faebcc;
}
.panel-warning > .panel-heading .badge {
  color: #fcf8e3;
  background-color: #8a6d3b;
}
.panel-warning > .panel-footer + .panel-collapse > .panel-body {
  border-bottom-color: #faebcc;
}
.panel-danger {
  border-color: #ebccd1;
}
.panel-danger > .panel-heading {
  color: #a94442;
  background-color: #f2dede;
  border-color: #ebccd1;
}
.panel-danger > .panel-heading + .panel-collapse > .panel-body {
  border-top-color: #ebccd1;
}
.panel-danger > .panel-heading .badge {
  color: #f2dede;
  background-color: #a94442;
}
.panel-danger > .panel-footer + .panel-collapse > .panel-body {
  border-bottom-color: #ebccd1;
}
.embed-responsive {
  position: relative;
  display: block;
  height: 0;
  padding: 0;
  overflow: hidden;
}
.embed-responsive .embed-responsive-item,
.embed-responsive iframe,
.embed-responsive embed,
.embed-responsive object,
.embed-responsive video {
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  height: 100%;
  width: 100%;
  border: 0;
}
.embed-responsive-16by9 {
  padding-bottom: 56.25%;
}
.embed-responsive-4by3 {
  padding-bottom: 75%;
}
.well {
  min-height: 20px;
  padding: 19px;
  margin-bottom: 20px;
  background-color: #f5f5f5;
  border: 1px solid #e3e3e3;
  border-radius: 2px;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
}
.well blockquote {
  border-color: #ddd;
  border-color: rgba(0, 0, 0, 0.15);
}
.well-lg {
  padding: 24px;
  border-radius: 3px;
}
.well-sm {
  padding: 9px;
  border-radius: 1px;
}
.close {
  float: right;
  font-size: 19.5px;
  font-weight: bold;
  line-height: 1;
  color: #000;
  text-shadow: 0 1px 0 #fff;
  opacity: 0.2;
  filter: alpha(opacity=20);
}
.close:hover,
.close:focus {
  color: #000;
  text-decoration: none;
  cursor: pointer;
  opacity: 0.5;
  filter: alpha(opacity=50);
}
button.close {
  padding: 0;
  cursor: pointer;
  background: transparent;
  border: 0;
  -webkit-appearance: none;
}
.modal-open {
  overflow: hidden;
}
.modal {
  display: none;
  overflow: hidden;
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 1050;
  -webkit-overflow-scrolling: touch;
  outline: 0;
}
.modal.fade .modal-dialog {
  -webkit-transform: translate(0, -25%);
  -ms-transform: translate(0, -25%);
  -o-transform: translate(0, -25%);
  transform: translate(0, -25%);
  -webkit-transition: -webkit-transform 0.3s ease-out;
  -moz-transition: -moz-transform 0.3s ease-out;
  -o-transition: -o-transform 0.3s ease-out;
  transition: transform 0.3s ease-out;
}
.modal.in .modal-dialog {
  -webkit-transform: translate(0, 0);
  -ms-transform: translate(0, 0);
  -o-transform: translate(0, 0);
  transform: translate(0, 0);
}
.modal-open .modal {
  overflow-x: hidden;
  overflow-y: auto;
}
.modal-dialog {
  position: relative;
  width: auto;
  margin: 10px;
}
.modal-content {
  position: relative;
  background-color: #fff;
  border: 1px solid #999;
  border: 1px solid rgba(0, 0, 0, 0.2);
  border-radius: 3px;
  -webkit-box-shadow: 0 3px 9px rgba(0, 0, 0, 0.5);
  box-shadow: 0 3px 9px rgba(0, 0, 0, 0.5);
  background-clip: padding-box;
  outline: 0;
}
.modal-backdrop {
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 1040;
  background-color: #000;
}
.modal-backdrop.fade {
  opacity: 0;
  filter: alpha(opacity=0);
}
.modal-backdrop.in {
  opacity: 0.5;
  filter: alpha(opacity=50);
}
.modal-header {
  padding: 15px;
  border-bottom: 1px solid #e5e5e5;
}
.modal-header .close {
  margin-top: -2px;
}
.modal-title {
  margin: 0;
  line-height: 1.42857143;
}
.modal-body {
  position: relative;
  padding: 15px;
}
.modal-footer {
  padding: 15px;
  text-align: right;
  border-top: 1px solid #e5e5e5;
}
.modal-footer .btn + .btn {
  margin-left: 5px;
  margin-bottom: 0;
}
.modal-footer .btn-group .btn + .btn {
  margin-left: -1px;
}
.modal-footer .btn-block + .btn-block {
  margin-left: 0;
}
.modal-scrollbar-measure {
  position: absolute;
  top: -9999px;
  width: 50px;
  height: 50px;
  overflow: scroll;
}
@media (min-width: 768px) {
  .modal-dialog {
    width: 600px;
    margin: 30px auto;
  }
  .modal-content {
    -webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
    box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
  }
  .modal-sm {
    width: 300px;
  }
}
@media (min-width: 992px) {
  .modal-lg {
    width: 900px;
  }
}
.tooltip {
  position: absolute;
  z-index: 1070;
  display: block;
  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
  font-style: normal;
  font-weight: normal;
  letter-spacing: normal;
  line-break: auto;
  line-height: 1.42857143;
  text-align: left;
  text-align: start;
  text-decoration: none;
  text-shadow: none;
  text-transform: none;
  white-space: normal;
  word-break: normal;
  word-spacing: normal;
  word-wrap: normal;
  font-size: 12px;
  opacity: 0;
  filter: alpha(opacity=0);
}
.tooltip.in {
  opacity: 0.9;
  filter: alpha(opacity=90);
}
.tooltip.top {
  margin-top: -3px;
  padding: 5px 0;
}
.tooltip.right {
  margin-left: 3px;
  padding: 0 5px;
}
.tooltip.bottom {
  margin-top: 3px;
  padding: 5px 0;
}
.tooltip.left {
  margin-left: -3px;
  padding: 0 5px;
}
.tooltip-inner {
  max-width: 200px;
  padding: 3px 8px;
  color: #fff;
  text-align: center;
  background-color: #000;
  border-radius: 2px;
}
.tooltip-arrow {
  position: absolute;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
}
.tooltip.top .tooltip-arrow {
  bottom: 0;
  left: 50%;
  margin-left: -5px;
  border-width: 5px 5px 0;
  border-top-color: #000;
}
.tooltip.top-left .tooltip-arrow {
  bottom: 0;
  right: 5px;
  margin-bottom: -5px;
  border-width: 5px 5px 0;
  border-top-color: #000;
}
.tooltip.top-right .tooltip-arrow {
  bottom: 0;
  left: 5px;
  margin-bottom: -5px;
  border-width: 5px 5px 0;
  border-top-color: #000;
}
.tooltip.right .tooltip-arrow {
  top: 50%;
  left: 0;
  margin-top: -5px;
  border-width: 5px 5px 5px 0;
  border-right-color: #000;
}
.tooltip.left .tooltip-arrow {
  top: 50%;
  right: 0;
  margin-top: -5px;
  border-width: 5px 0 5px 5px;
  border-left-color: #000;
}
.tooltip.bottom .tooltip-arrow {
  top: 0;
  left: 50%;
  margin-left: -5px;
  border-width: 0 5px 5px;
  border-bottom-color: #000;
}
.tooltip.bottom-left .tooltip-arrow {
  top: 0;
  right: 5px;
  margin-top: -5px;
  border-width: 0 5px 5px;
  border-bottom-color: #000;
}
.tooltip.bottom-right .tooltip-arrow {
  top: 0;
  left: 5px;
  margin-top: -5px;
  border-width: 0 5px 5px;
  border-bottom-color: #000;
}
.popover {
  position: absolute;
  top: 0;
  left: 0;
  z-index: 1060;
  display: none;
  max-width: 276px;
  padding: 1px;
  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
  font-style: normal;
  font-weight: normal;
  letter-spacing: normal;
  line-break: auto;
  line-height: 1.42857143;
  text-align: left;
  text-align: start;
  text-decoration: none;
  text-shadow: none;
  text-transform: none;
  white-space: normal;
  word-break: normal;
  word-spacing: normal;
  word-wrap: normal;
  font-size: 13px;
  background-color: #fff;
  background-clip: padding-box;
  border: 1px solid #ccc;
  border: 1px solid rgba(0, 0, 0, 0.2);
  border-radius: 3px;
  -webkit-box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
}
.popover.top {
  margin-top: -10px;
}
.popover.right {
  margin-left: 10px;
}
.popover.bottom {
  margin-top: 10px;
}
.popover.left {
  margin-left: -10px;
}
.popover-title {
  margin: 0;
  padding: 8px 14px;
  font-size: 13px;
  background-color: #f7f7f7;
  border-bottom: 1px solid #ebebeb;
  border-radius: 2px 2px 0 0;
}
.popover-content {
  padding: 9px 14px;
}
.popover > .arrow,
.popover > .arrow:after {
  position: absolute;
  display: block;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
}
.popover > .arrow {
  border-width: 11px;
}
.popover > .arrow:after {
  border-width: 10px;
  content: "";
}
.popover.top > .arrow {
  left: 50%;
  margin-left: -11px;
  border-bottom-width: 0;
  border-top-color: #999999;
  border-top-color: rgba(0, 0, 0, 0.25);
  bottom: -11px;
}
.popover.top > .arrow:after {
  content: " ";
  bottom: 1px;
  margin-left: -10px;
  border-bottom-width: 0;
  border-top-color: #fff;
}
.popover.right > .arrow {
  top: 50%;
  left: -11px;
  margin-top: -11px;
  border-left-width: 0;
  border-right-color: #999999;
  border-right-color: rgba(0, 0, 0, 0.25);
}
.popover.right > .arrow:after {
  content: " ";
  left: 1px;
  bottom: -10px;
  border-left-width: 0;
  border-right-color: #fff;
}
.popover.bottom > .arrow {
  left: 50%;
  margin-left: -11px;
  border-top-width: 0;
  border-bottom-color: #999999;
  border-bottom-color: rgba(0, 0, 0, 0.25);
  top: -11px;
}
.popover.bottom > .arrow:after {
  content: " ";
  top: 1px;
  margin-left: -10px;
  border-top-width: 0;
  border-bottom-color: #fff;
}
.popover.left > .arrow {
  top: 50%;
  right: -11px;
  margin-top: -11px;
  border-right-width: 0;
  border-left-color: #999999;
  border-left-color: rgba(0, 0, 0, 0.25);
}
.popover.left > .arrow:after {
  content: " ";
  right: 1px;
  border-right-width: 0;
  border-left-color: #fff;
  bottom: -10px;
}
.carousel {
  position: relative;
}
.carousel-inner {
  position: relative;
  overflow: hidden;
  width: 100%;
}
.carousel-inner > .item {
  display: none;
  position: relative;
  -webkit-transition: 0.6s ease-in-out left;
  -o-transition: 0.6s ease-in-out left;
  transition: 0.6s ease-in-out left;
}
.carousel-inner > .item > img,
.carousel-inner > .item > a > img {
  line-height: 1;
}
@media all and (transform-3d), (-webkit-transform-3d) {
  .carousel-inner > .item {
    -webkit-transition: -webkit-transform 0.6s ease-in-out;
    -moz-transition: -moz-transform 0.6s ease-in-out;
    -o-transition: -o-transform 0.6s ease-in-out;
    transition: transform 0.6s ease-in-out;
    -webkit-backface-visibility: hidden;
    -moz-backface-visibility: hidden;
    backface-visibility: hidden;
    -webkit-perspective: 1000px;
    -moz-perspective: 1000px;
    perspective: 1000px;
  }
  .carousel-inner > .item.next,
  .carousel-inner > .item.active.right {
    -webkit-transform: translate3d(100%, 0, 0);
    transform: translate3d(100%, 0, 0);
    left: 0;
  }
  .carousel-inner > .item.prev,
  .carousel-inner > .item.active.left {
    -webkit-transform: translate3d(-100%, 0, 0);
    transform: translate3d(-100%, 0, 0);
    left: 0;
  }
  .carousel-inner > .item.next.left,
  .carousel-inner > .item.prev.right,
  .carousel-inner > .item.active {
    -webkit-transform: translate3d(0, 0, 0);
    transform: translate3d(0, 0, 0);
    left: 0;
  }
}
.carousel-inner > .active,
.carousel-inner > .next,
.carousel-inner > .prev {
  display: block;
}
.carousel-inner > .active {
  left: 0;
}
.carousel-inner > .next,
.carousel-inner > .prev {
  position: absolute;
  top: 0;
  width: 100%;
}
.carousel-inner > .next {
  left: 100%;
}
.carousel-inner > .prev {
  left: -100%;
}
.carousel-inner > .next.left,
.carousel-inner > .prev.right {
  left: 0;
}
.carousel-inner > .active.left {
  left: -100%;
}
.carousel-inner > .active.right {
  left: 100%;
}
.carousel-control {
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  width: 15%;
  opacity: 0.5;
  filter: alpha(opacity=50);
  font-size: 20px;
  color: #fff;
  text-align: center;
  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.6);
  background-color: rgba(0, 0, 0, 0);
}
.carousel-control.left {
  background-image: -webkit-linear-gradient(left, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0.0001) 100%);
  background-image: -o-linear-gradient(left, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0.0001) 100%);
  background-image: linear-gradient(to right, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0.0001) 100%);
  background-repeat: repeat-x;
  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);
}
.carousel-control.right {
  left: auto;
  right: 0;
  background-image: -webkit-linear-gradient(left, rgba(0, 0, 0, 0.0001) 0%, rgba(0, 0, 0, 0.5) 100%);
  background-image: -o-linear-gradient(left, rgba(0, 0, 0, 0.0001) 0%, rgba(0, 0, 0, 0.5) 100%);
  background-image: linear-gradient(to right, rgba(0, 0, 0, 0.0001) 0%, rgba(0, 0, 0, 0.5) 100%);
  background-repeat: repeat-x;
  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);
}
.carousel-control:hover,
.carousel-control:focus {
  outline: 0;
  color: #fff;
  text-decoration: none;
  opacity: 0.9;
  filter: alpha(opacity=90);
}
.carousel-control .icon-prev,
.carousel-control .icon-next,
.carousel-control .glyphicon-chevron-left,
.carousel-control .glyphicon-chevron-right {
  position: absolute;
  top: 50%;
  margin-top: -10px;
  z-index: 5;
  display: inline-block;
}
.carousel-control .icon-prev,
.carousel-control .glyphicon-chevron-left {
  left: 50%;
  margin-left: -10px;
}
.carousel-control .icon-next,
.carousel-control .glyphicon-chevron-right {
  right: 50%;
  margin-right: -10px;
}
.carousel-control .icon-prev,
.carousel-control .icon-next {
  width: 20px;
  height: 20px;
  line-height: 1;
  font-family: serif;
}
.carousel-control .icon-prev:before {
  content: '\2039';
}
.carousel-control .icon-next:before {
  content: '\203a';
}
.carousel-indicators {
  position: absolute;
  bottom: 10px;
  left: 50%;
  z-index: 15;
  width: 60%;
  margin-left: -30%;
  padding-left: 0;
  list-style: none;
  text-align: center;
}
.carousel-indicators li {
  display: inline-block;
  width: 10px;
  height: 10px;
  margin: 1px;
  text-indent: -999px;
  border: 1px solid #fff;
  border-radius: 10px;
  cursor: pointer;
  background-color: #000 \9;
  background-color: rgba(0, 0, 0, 0);
}
.carousel-indicators .active {
  margin: 0;
  width: 12px;
  height: 12px;
  background-color: #fff;
}
.carousel-caption {
  position: absolute;
  left: 15%;
  right: 15%;
  bottom: 20px;
  z-index: 10;
  padding-top: 20px;
  padding-bottom: 20px;
  color: #fff;
  text-align: center;
  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.6);
}
.carousel-caption .btn {
  text-shadow: none;
}
@media screen and (min-width: 768px) {
  .carousel-control .glyphicon-chevron-left,
  .carousel-control .glyphicon-chevron-right,
  .carousel-control .icon-prev,
  .carousel-control .icon-next {
    width: 30px;
    height: 30px;
    margin-top: -10px;
    font-size: 30px;
  }
  .carousel-control .glyphicon-chevron-left,
  .carousel-control .icon-prev {
    margin-left: -10px;
  }
  .carousel-control .glyphicon-chevron-right,
  .carousel-control .icon-next {
    margin-right: -10px;
  }
  .carousel-caption {
    left: 20%;
    right: 20%;
    padding-bottom: 30px;
  }
  .carousel-indicators {
    bottom: 20px;
  }
}
.clearfix:before,
.clearfix:after,
.dl-horizontal dd:before,
.dl-horizontal dd:after,
.container:before,
.container:after,
.container-fluid:before,
.container-fluid:after,
.row:before,
.row:after,
.form-horizontal .form-group:before,
.form-horizontal .form-group:after,
.btn-toolbar:before,
.btn-toolbar:after,
.btn-group-vertical > .btn-group:before,
.btn-group-vertical > .btn-group:after,
.nav:before,
.nav:after,
.navbar:before,
.navbar:after,
.navbar-header:before,
.navbar-header:after,
.navbar-collapse:before,
.navbar-collapse:after,
.pager:before,
.pager:after,
.panel-body:before,
.panel-body:after,
.modal-header:before,
.modal-header:after,
.modal-footer:before,
.modal-footer:after,
.item_buttons:before,
.item_buttons:after {
  content: " ";
  display: table;
}
.clearfix:after,
.dl-horizontal dd:after,
.container:after,
.container-fluid:after,
.row:after,
.form-horizontal .form-group:after,
.btn-toolbar:after,
.btn-group-vertical > .btn-group:after,
.nav:after,
.navbar:after,
.navbar-header:after,
.navbar-collapse:after,
.pager:after,
.panel-body:after,
.modal-header:after,
.modal-footer:after,
.item_buttons:after {
  clear: both;
}
.center-block {
  display: block;
  margin-left: auto;
  margin-right: auto;
}
.pull-right {
  float: right !important;
}
.pull-left {
  float: left !important;
}
.hide {
  display: none !important;
}
.show {
  display: block !important;
}
.invisible {
  visibility: hidden;
}
.text-hide {
  font: 0/0 a;
  color: transparent;
  text-shadow: none;
  background-color: transparent;
  border: 0;
}
.hidden {
  display: none !important;
}
.affix {
  position: fixed;
}
@-ms-viewport {
  width: device-width;
}
.visible-xs,
.visible-sm,
.visible-md,
.visible-lg {
  display: none !important;
}
.visible-xs-block,
.visible-xs-inline,
.visible-xs-inline-block,
.visible-sm-block,
.visible-sm-inline,
.visible-sm-inline-block,
.visible-md-block,
.visible-md-inline,
.visible-md-inline-block,
.visible-lg-block,
.visible-lg-inline,
.visible-lg-inline-block {
  display: none !important;
}
@media (max-width: 767px) {
  .visible-xs {
    display: block !important;
  }
  table.visible-xs {
    display: table !important;
  }
  tr.visible-xs {
    display: table-row !important;
  }
  th.visible-xs,
  td.visible-xs {
    display: table-cell !important;
  }
}
@media (max-width: 767px) {
  .visible-xs-block {
    display: block !important;
  }
}
@media (max-width: 767px) {
  .visible-xs-inline {
    display: inline !important;
  }
}
@media (max-width: 767px) {
  .visible-xs-inline-block {
    display: inline-block !important;
  }
}
@media (min-width: 768px) and (max-width: 991px) {
  .visible-sm {
    display: block !important;
  }
  table.visible-sm {
    display: table !important;
  }
  tr.visible-sm {
    display: table-row !important;
  }
  th.visible-sm,
  td.visible-sm {
    display: table-cell !important;
  }
}
@media (min-width: 768px) and (max-width: 991px) {
  .visible-sm-block {
    display: block !important;
  }
}
@media (min-width: 768px) and (max-width: 991px) {
  .visible-sm-inline {
    display: inline !important;
  }
}
@media (min-width: 768px) and (max-width: 991px) {
  .visible-sm-inline-block {
    display: inline-block !important;
  }
}
@media (min-width: 992px) and (max-width: 1199px) {
  .visible-md {
    display: block !important;
  }
  table.visible-md {
    display: table !important;
  }
  tr.visible-md {
    display: table-row !important;
  }
  th.visible-md,
  td.visible-md {
    display: table-cell !important;
  }
}
@media (min-width: 992px) and (max-width: 1199px) {
  .visible-md-block {
    display: block !important;
  }
}
@media (min-width: 992px) and (max-width: 1199px) {
  .visible-md-inline {
    display: inline !important;
  }
}
@media (min-width: 992px) and (max-width: 1199px) {
  .visible-md-inline-block {
    display: inline-block !important;
  }
}
@media (min-width: 1200px) {
  .visible-lg {
    display: block !important;
  }
  table.visible-lg {
    display: table !important;
  }
  tr.visible-lg {
    display: table-row !important;
  }
  th.visible-lg,
  td.visible-lg {
    display: table-cell !important;
  }
}
@media (min-width: 1200px) {
  .visible-lg-block {
    display: block !important;
  }
}
@media (min-width: 1200px) {
  .visible-lg-inline {
    display: inline !important;
  }
}
@media (min-width: 1200px) {
  .visible-lg-inline-block {
    display: inline-block !important;
  }
}
@media (max-width: 767px) {
  .hidden-xs {
    display: none !important;
  }
}
@media (min-width: 768px) and (max-width: 991px) {
  .hidden-sm {
    display: none !important;
  }
}
@media (min-width: 992px) and (max-width: 1199px) {
  .hidden-md {
    display: none !important;
  }
}
@media (min-width: 1200px) {
  .hidden-lg {
    display: none !important;
  }
}
.visible-print {
  display: none !important;
}
@media print {
  .visible-print {
    display: block !important;
  }
  table.visible-print {
    display: table !important;
  }
  tr.visible-print {
    display: table-row !important;
  }
  th.visible-print,
  td.visible-print {
    display: table-cell !important;
  }
}
.visible-print-block {
  display: none !important;
}
@media print {
  .visible-print-block {
    display: block !important;
  }
}
.visible-print-inline {
  display: none !important;
}
@media print {
  .visible-print-inline {
    display: inline !important;
  }
}
.visible-print-inline-block {
  display: none !important;
}
@media print {
  .visible-print-inline-block {
    display: inline-block !important;
  }
}
@media print {
  .hidden-print {
    display: none !important;
  }
}
/*!
*
* Font Awesome
*
*/
/*!
 *  Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
 *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
 */
/* FONT PATH
 * -------------------------- */
@font-face {
  font-family: 'FontAwesome';
  src: url('../components/font-awesome/fonts/fontawesome-webfont.eot?v=4.7.0');
  src: url('../components/font-awesome/fonts/fontawesome-webfont.eot?#iefix&v=4.7.0') format('embedded-opentype'), url('../components/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'), url('../components/font-awesome/fonts/fontawesome-webfont.woff?v=4.7.0') format('woff'), url('../components/font-awesome/fonts/fontawesome-webfont.ttf?v=4.7.0') format('truetype'), url('../components/font-awesome/fonts/fontawesome-webfont.svg?v=4.7.0#fontawesomeregular') format('svg');
  font-weight: normal;
  font-style: normal;
}
.fa {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
/* makes the font 33% larger relative to the icon container */
.fa-lg {
  font-size: 1.33333333em;
  line-height: 0.75em;
  vertical-align: -15%;
}
.fa-2x {
  font-size: 2em;
}
.fa-3x {
  font-size: 3em;
}
.fa-4x {
  font-size: 4em;
}
.fa-5x {
  font-size: 5em;
}
.fa-fw {
  width: 1.28571429em;
  text-align: center;
}
.fa-ul {
  padding-left: 0;
  margin-left: 2.14285714em;
  list-style-type: none;
}
.fa-ul > li {
  position: relative;
}
.fa-li {
  position: absolute;
  left: -2.14285714em;
  width: 2.14285714em;
  top: 0.14285714em;
  text-align: center;
}
.fa-li.fa-lg {
  left: -1.85714286em;
}
.fa-border {
  padding: .2em .25em .15em;
  border: solid 0.08em #eee;
  border-radius: .1em;
}
.fa-pull-left {
  float: left;
}
.fa-pull-right {
  float: right;
}
.fa.fa-pull-left {
  margin-right: .3em;
}
.fa.fa-pull-right {
  margin-left: .3em;
}
/* Deprecated as of 4.4.0 */
.pull-right {
  float: right;
}
.pull-left {
  float: left;
}
.fa.pull-left {
  margin-right: .3em;
}
.fa.pull-right {
  margin-left: .3em;
}
.fa-spin {
  -webkit-animation: fa-spin 2s infinite linear;
  animation: fa-spin 2s infinite linear;
}
.fa-pulse {
  -webkit-animation: fa-spin 1s infinite steps(8);
  animation: fa-spin 1s infinite steps(8);
}
@-webkit-keyframes fa-spin {
  0% {
    -webkit-transform: rotate(0deg);
    transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(359deg);
    transform: rotate(359deg);
  }
}
@keyframes fa-spin {
  0% {
    -webkit-transform: rotate(0deg);
    transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(359deg);
    transform: rotate(359deg);
  }
}
.fa-rotate-90 {
  -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=1)";
  -webkit-transform: rotate(90deg);
  -ms-transform: rotate(90deg);
  transform: rotate(90deg);
}
.fa-rotate-180 {
  -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=2)";
  -webkit-transform: rotate(180deg);
  -ms-transform: rotate(180deg);
  transform: rotate(180deg);
}
.fa-rotate-270 {
  -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=3)";
  -webkit-transform: rotate(270deg);
  -ms-transform: rotate(270deg);
  transform: rotate(270deg);
}
.fa-flip-horizontal {
  -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=0, mirror=1)";
  -webkit-transform: scale(-1, 1);
  -ms-transform: scale(-1, 1);
  transform: scale(-1, 1);
}
.fa-flip-vertical {
  -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=2, mirror=1)";
  -webkit-transform: scale(1, -1);
  -ms-transform: scale(1, -1);
  transform: scale(1, -1);
}
:root .fa-rotate-90,
:root .fa-rotate-180,
:root .fa-rotate-270,
:root .fa-flip-horizontal,
:root .fa-flip-vertical {
  filter: none;
}
.fa-stack {
  position: relative;
  display: inline-block;
  width: 2em;
  height: 2em;
  line-height: 2em;
  vertical-align: middle;
}
.fa-stack-1x,
.fa-stack-2x {
  position: absolute;
  left: 0;
  width: 100%;
  text-align: center;
}
.fa-stack-1x {
  line-height: inherit;
}
.fa-stack-2x {
  font-size: 2em;
}
.fa-inverse {
  color: #fff;
}
/* Font Awesome uses the Unicode Private Use Area (PUA) to ensure screen
   readers do not read off random characters that represent icons */
.fa-glass:before {
  content: "\f000";
}
.fa-music:before {
  content: "\f001";
}
.fa-search:before {
  content: "\f002";
}
.fa-envelope-o:before {
  content: "\f003";
}
.fa-heart:before {
  content: "\f004";
}
.fa-star:before {
  content: "\f005";
}
.fa-star-o:before {
  content: "\f006";
}
.fa-user:before {
  content: "\f007";
}
.fa-film:before {
  content: "\f008";
}
.fa-th-large:before {
  content: "\f009";
}
.fa-th:before {
  content: "\f00a";
}
.fa-th-list:before {
  content: "\f00b";
}
.fa-check:before {
  content: "\f00c";
}
.fa-remove:before,
.fa-close:before,
.fa-times:before {
  content: "\f00d";
}
.fa-search-plus:before {
  content: "\f00e";
}
.fa-search-minus:before {
  content: "\f010";
}
.fa-power-off:before {
  content: "\f011";
}
.fa-signal:before {
  content: "\f012";
}
.fa-gear:before,
.fa-cog:before {
  content: "\f013";
}
.fa-trash-o:before {
  content: "\f014";
}
.fa-home:before {
  content: "\f015";
}
.fa-file-o:before {
  content: "\f016";
}
.fa-clock-o:before {
  content: "\f017";
}
.fa-road:before {
  content: "\f018";
}
.fa-download:before {
  content: "\f019";
}
.fa-arrow-circle-o-down:before {
  content: "\f01a";
}
.fa-arrow-circle-o-up:before {
  content: "\f01b";
}
.fa-inbox:before {
  content: "\f01c";
}
.fa-play-circle-o:before {
  content: "\f01d";
}
.fa-rotate-right:before,
.fa-repeat:before {
  content: "\f01e";
}
.fa-refresh:before {
  content: "\f021";
}
.fa-list-alt:before {
  content: "\f022";
}
.fa-lock:before {
  content: "\f023";
}
.fa-flag:before {
  content: "\f024";
}
.fa-headphones:before {
  content: "\f025";
}
.fa-volume-off:before {
  content: "\f026";
}
.fa-volume-down:before {
  content: "\f027";
}
.fa-volume-up:before {
  content: "\f028";
}
.fa-qrcode:before {
  content: "\f029";
}
.fa-barcode:before {
  content: "\f02a";
}
.fa-tag:before {
  content: "\f02b";
}
.fa-tags:before {
  content: "\f02c";
}
.fa-book:before {
  content: "\f02d";
}
.fa-bookmark:before {
  content: "\f02e";
}
.fa-print:before {
  content: "\f02f";
}
.fa-camera:before {
  content: "\f030";
}
.fa-font:before {
  content: "\f031";
}
.fa-bold:before {
  content: "\f032";
}
.fa-italic:before {
  content: "\f033";
}
.fa-text-height:before {
  content: "\f034";
}
.fa-text-width:before {
  content: "\f035";
}
.fa-align-left:before {
  content: "\f036";
}
.fa-align-center:before {
  content: "\f037";
}
.fa-align-right:before {
  content: "\f038";
}
.fa-align-justify:before {
  content: "\f039";
}
.fa-list:before {
  content: "\f03a";
}
.fa-dedent:before,
.fa-outdent:before {
  content: "\f03b";
}
.fa-indent:before {
  content: "\f03c";
}
.fa-video-camera:before {
  content: "\f03d";
}
.fa-photo:before,
.fa-image:before,
.fa-picture-o:before {
  content: "\f03e";
}
.fa-pencil:before {
  content: "\f040";
}
.fa-map-marker:before {
  content: "\f041";
}
.fa-adjust:before {
  content: "\f042";
}
.fa-tint:before {
  content: "\f043";
}
.fa-edit:before,
.fa-pencil-square-o:before {
  content: "\f044";
}
.fa-share-square-o:before {
  content: "\f045";
}
.fa-check-square-o:before {
  content: "\f046";
}
.fa-arrows:before {
  content: "\f047";
}
.fa-step-backward:before {
  content: "\f048";
}
.fa-fast-backward:before {
  content: "\f049";
}
.fa-backward:before {
  content: "\f04a";
}
.fa-play:before {
  content: "\f04b";
}
.fa-pause:before {
  content: "\f04c";
}
.fa-stop:before {
  content: "\f04d";
}
.fa-forward:before {
  content: "\f04e";
}
.fa-fast-forward:before {
  content: "\f050";
}
.fa-step-forward:before {
  content: "\f051";
}
.fa-eject:before {
  content: "\f052";
}
.fa-chevron-left:before {
  content: "\f053";
}
.fa-chevron-right:before {
  content: "\f054";
}
.fa-plus-circle:before {
  content: "\f055";
}
.fa-minus-circle:before {
  content: "\f056";
}
.fa-times-circle:before {
  content: "\f057";
}
.fa-check-circle:before {
  content: "\f058";
}
.fa-question-circle:before {
  content: "\f059";
}
.fa-info-circle:before {
  content: "\f05a";
}
.fa-crosshairs:before {
  content: "\f05b";
}
.fa-times-circle-o:before {
  content: "\f05c";
}
.fa-check-circle-o:before {
  content: "\f05d";
}
.fa-ban:before {
  content: "\f05e";
}
.fa-arrow-left:before {
  content: "\f060";
}
.fa-arrow-right:before {
  content: "\f061";
}
.fa-arrow-up:before {
  content: "\f062";
}
.fa-arrow-down:before {
  content: "\f063";
}
.fa-mail-forward:before,
.fa-share:before {
  content: "\f064";
}
.fa-expand:before {
  content: "\f065";
}
.fa-compress:before {
  content: "\f066";
}
.fa-plus:before {
  content: "\f067";
}
.fa-minus:before {
  content: "\f068";
}
.fa-asterisk:before {
  content: "\f069";
}
.fa-exclamation-circle:before {
  content: "\f06a";
}
.fa-gift:before {
  content: "\f06b";
}
.fa-leaf:before {
  content: "\f06c";
}
.fa-fire:before {
  content: "\f06d";
}
.fa-eye:before {
  content: "\f06e";
}
.fa-eye-slash:before {
  content: "\f070";
}
.fa-warning:before,
.fa-exclamation-triangle:before {
  content: "\f071";
}
.fa-plane:before {
  content: "\f072";
}
.fa-calendar:before {
  content: "\f073";
}
.fa-random:before {
  content: "\f074";
}
.fa-comment:before {
  content: "\f075";
}
.fa-magnet:before {
  content: "\f076";
}
.fa-chevron-up:before {
  content: "\f077";
}
.fa-chevron-down:before {
  content: "\f078";
}
.fa-retweet:before {
  content: "\f079";
}
.fa-shopping-cart:before {
  content: "\f07a";
}
.fa-folder:before {
  content: "\f07b";
}
.fa-folder-open:before {
  content: "\f07c";
}
.fa-arrows-v:before {
  content: "\f07d";
}
.fa-arrows-h:before {
  content: "\f07e";
}
.fa-bar-chart-o:before,
.fa-bar-chart:before {
  content: "\f080";
}
.fa-twitter-square:before {
  content: "\f081";
}
.fa-facebook-square:before {
  content: "\f082";
}
.fa-camera-retro:before {
  content: "\f083";
}
.fa-key:before {
  content: "\f084";
}
.fa-gears:before,
.fa-cogs:before {
  content: "\f085";
}
.fa-comments:before {
  content: "\f086";
}
.fa-thumbs-o-up:before {
  content: "\f087";
}
.fa-thumbs-o-down:before {
  content: "\f088";
}
.fa-star-half:before {
  content: "\f089";
}
.fa-heart-o:before {
  content: "\f08a";
}
.fa-sign-out:before {
  content: "\f08b";
}
.fa-linkedin-square:before {
  content: "\f08c";
}
.fa-thumb-tack:before {
  content: "\f08d";
}
.fa-external-link:before {
  content: "\f08e";
}
.fa-sign-in:before {
  content: "\f090";
}
.fa-trophy:before {
  content: "\f091";
}
.fa-github-square:before {
  content: "\f092";
}
.fa-upload:before {
  content: "\f093";
}
.fa-lemon-o:before {
  content: "\f094";
}
.fa-phone:before {
  content: "\f095";
}
.fa-square-o:before {
  content: "\f096";
}
.fa-bookmark-o:before {
  content: "\f097";
}
.fa-phone-square:before {
  content: "\f098";
}
.fa-twitter:before {
  content: "\f099";
}
.fa-facebook-f:before,
.fa-facebook:before {
  content: "\f09a";
}
.fa-github:before {
  content: "\f09b";
}
.fa-unlock:before {
  content: "\f09c";
}
.fa-credit-card:before {
  content: "\f09d";
}
.fa-feed:before,
.fa-rss:before {
  content: "\f09e";
}
.fa-hdd-o:before {
  content: "\f0a0";
}
.fa-bullhorn:before {
  content: "\f0a1";
}
.fa-bell:before {
  content: "\f0f3";
}
.fa-certificate:before {
  content: "\f0a3";
}
.fa-hand-o-right:before {
  content: "\f0a4";
}
.fa-hand-o-left:before {
  content: "\f0a5";
}
.fa-hand-o-up:before {
  content: "\f0a6";
}
.fa-hand-o-down:before {
  content: "\f0a7";
}
.fa-arrow-circle-left:before {
  content: "\f0a8";
}
.fa-arrow-circle-right:before {
  content: "\f0a9";
}
.fa-arrow-circle-up:before {
  content: "\f0aa";
}
.fa-arrow-circle-down:before {
  content: "\f0ab";
}
.fa-globe:before {
  content: "\f0ac";
}
.fa-wrench:before {
  content: "\f0ad";
}
.fa-tasks:before {
  content: "\f0ae";
}
.fa-filter:before {
  content: "\f0b0";
}
.fa-briefcase:before {
  content: "\f0b1";
}
.fa-arrows-alt:before {
  content: "\f0b2";
}
.fa-group:before,
.fa-users:before {
  content: "\f0c0";
}
.fa-chain:before,
.fa-link:before {
  content: "\f0c1";
}
.fa-cloud:before {
  content: "\f0c2";
}
.fa-flask:before {
  content: "\f0c3";
}
.fa-cut:before,
.fa-scissors:before {
  content: "\f0c4";
}
.fa-copy:before,
.fa-files-o:before {
  content: "\f0c5";
}
.fa-paperclip:before {
  content: "\f0c6";
}
.fa-save:before,
.fa-floppy-o:before {
  content: "\f0c7";
}
.fa-square:before {
  content: "\f0c8";
}
.fa-navicon:before,
.fa-reorder:before,
.fa-bars:before {
  content: "\f0c9";
}
.fa-list-ul:before {
  content: "\f0ca";
}
.fa-list-ol:before {
  content: "\f0cb";
}
.fa-strikethrough:before {
  content: "\f0cc";
}
.fa-underline:before {
  content: "\f0cd";
}
.fa-table:before {
  content: "\f0ce";
}
.fa-magic:before {
  content: "\f0d0";
}
.fa-truck:before {
  content: "\f0d1";
}
.fa-pinterest:before {
  content: "\f0d2";
}
.fa-pinterest-square:before {
  content: "\f0d3";
}
.fa-google-plus-square:before {
  content: "\f0d4";
}
.fa-google-plus:before {
  content: "\f0d5";
}
.fa-money:before {
  content: "\f0d6";
}
.fa-caret-down:before {
  content: "\f0d7";
}
.fa-caret-up:before {
  content: "\f0d8";
}
.fa-caret-left:before {
  content: "\f0d9";
}
.fa-caret-right:before {
  content: "\f0da";
}
.fa-columns:before {
  content: "\f0db";
}
.fa-unsorted:before,
.fa-sort:before {
  content: "\f0dc";
}
.fa-sort-down:before,
.fa-sort-desc:before {
  content: "\f0dd";
}
.fa-sort-up:before,
.fa-sort-asc:before {
  content: "\f0de";
}
.fa-envelope:before {
  content: "\f0e0";
}
.fa-linkedin:before {
  content: "\f0e1";
}
.fa-rotate-left:before,
.fa-undo:before {
  content: "\f0e2";
}
.fa-legal:before,
.fa-gavel:before {
  content: "\f0e3";
}
.fa-dashboard:before,
.fa-tachometer:before {
  content: "\f0e4";
}
.fa-comment-o:before {
  content: "\f0e5";
}
.fa-comments-o:before {
  content: "\f0e6";
}
.fa-flash:before,
.fa-bolt:before {
  content: "\f0e7";
}
.fa-sitemap:before {
  content: "\f0e8";
}
.fa-umbrella:before {
  content: "\f0e9";
}
.fa-paste:before,
.fa-clipboard:before {
  content: "\f0ea";
}
.fa-lightbulb-o:before {
  content: "\f0eb";
}
.fa-exchange:before {
  content: "\f0ec";
}
.fa-cloud-download:before {
  content: "\f0ed";
}
.fa-cloud-upload:before {
  content: "\f0ee";
}
.fa-user-md:before {
  content: "\f0f0";
}
.fa-stethoscope:before {
  content: "\f0f1";
}
.fa-suitcase:before {
  content: "\f0f2";
}
.fa-bell-o:before {
  content: "\f0a2";
}
.fa-coffee:before {
  content: "\f0f4";
}
.fa-cutlery:before {
  content: "\f0f5";
}
.fa-file-text-o:before {
  content: "\f0f6";
}
.fa-building-o:before {
  content: "\f0f7";
}
.fa-hospital-o:before {
  content: "\f0f8";
}
.fa-ambulance:before {
  content: "\f0f9";
}
.fa-medkit:before {
  content: "\f0fa";
}
.fa-fighter-jet:before {
  content: "\f0fb";
}
.fa-beer:before {
  content: "\f0fc";
}
.fa-h-square:before {
  content: "\f0fd";
}
.fa-plus-square:before {
  content: "\f0fe";
}
.fa-angle-double-left:before {
  content: "\f100";
}
.fa-angle-double-right:before {
  content: "\f101";
}
.fa-angle-double-up:before {
  content: "\f102";
}
.fa-angle-double-down:before {
  content: "\f103";
}
.fa-angle-left:before {
  content: "\f104";
}
.fa-angle-right:before {
  content: "\f105";
}
.fa-angle-up:before {
  content: "\f106";
}
.fa-angle-down:before {
  content: "\f107";
}
.fa-desktop:before {
  content: "\f108";
}
.fa-laptop:before {
  content: "\f109";
}
.fa-tablet:before {
  content: "\f10a";
}
.fa-mobile-phone:before,
.fa-mobile:before {
  content: "\f10b";
}
.fa-circle-o:before {
  content: "\f10c";
}
.fa-quote-left:before {
  content: "\f10d";
}
.fa-quote-right:before {
  content: "\f10e";
}
.fa-spinner:before {
  content: "\f110";
}
.fa-circle:before {
  content: "\f111";
}
.fa-mail-reply:before,
.fa-reply:before {
  content: "\f112";
}
.fa-github-alt:before {
  content: "\f113";
}
.fa-folder-o:before {
  content: "\f114";
}
.fa-folder-open-o:before {
  content: "\f115";
}
.fa-smile-o:before {
  content: "\f118";
}
.fa-frown-o:before {
  content: "\f119";
}
.fa-meh-o:before {
  content: "\f11a";
}
.fa-gamepad:before {
  content: "\f11b";
}
.fa-keyboard-o:before {
  content: "\f11c";
}
.fa-flag-o:before {
  content: "\f11d";
}
.fa-flag-checkered:before {
  content: "\f11e";
}
.fa-terminal:before {
  content: "\f120";
}
.fa-code:before {
  content: "\f121";
}
.fa-mail-reply-all:before,
.fa-reply-all:before {
  content: "\f122";
}
.fa-star-half-empty:before,
.fa-star-half-full:before,
.fa-star-half-o:before {
  content: "\f123";
}
.fa-location-arrow:before {
  content: "\f124";
}
.fa-crop:before {
  content: "\f125";
}
.fa-code-fork:before {
  content: "\f126";
}
.fa-unlink:before,
.fa-chain-broken:before {
  content: "\f127";
}
.fa-question:before {
  content: "\f128";
}
.fa-info:before {
  content: "\f129";
}
.fa-exclamation:before {
  content: "\f12a";
}
.fa-superscript:before {
  content: "\f12b";
}
.fa-subscript:before {
  content: "\f12c";
}
.fa-eraser:before {
  content: "\f12d";
}
.fa-puzzle-piece:before {
  content: "\f12e";
}
.fa-microphone:before {
  content: "\f130";
}
.fa-microphone-slash:before {
  content: "\f131";
}
.fa-shield:before {
  content: "\f132";
}
.fa-calendar-o:before {
  content: "\f133";
}
.fa-fire-extinguisher:before {
  content: "\f134";
}
.fa-rocket:before {
  content: "\f135";
}
.fa-maxcdn:before {
  content: "\f136";
}
.fa-chevron-circle-left:before {
  content: "\f137";
}
.fa-chevron-circle-right:before {
  content: "\f138";
}
.fa-chevron-circle-up:before {
  content: "\f139";
}
.fa-chevron-circle-down:before {
  content: "\f13a";
}
.fa-html5:before {
  content: "\f13b";
}
.fa-css3:before {
  content: "\f13c";
}
.fa-anchor:before {
  content: "\f13d";
}
.fa-unlock-alt:before {
  content: "\f13e";
}
.fa-bullseye:before {
  content: "\f140";
}
.fa-ellipsis-h:before {
  content: "\f141";
}
.fa-ellipsis-v:before {
  content: "\f142";
}
.fa-rss-square:before {
  content: "\f143";
}
.fa-play-circle:before {
  content: "\f144";
}
.fa-ticket:before {
  content: "\f145";
}
.fa-minus-square:before {
  content: "\f146";
}
.fa-minus-square-o:before {
  content: "\f147";
}
.fa-level-up:before {
  content: "\f148";
}
.fa-level-down:before {
  content: "\f149";
}
.fa-check-square:before {
  content: "\f14a";
}
.fa-pencil-square:before {
  content: "\f14b";
}
.fa-external-link-square:before {
  content: "\f14c";
}
.fa-share-square:before {
  content: "\f14d";
}
.fa-compass:before {
  content: "\f14e";
}
.fa-toggle-down:before,
.fa-caret-square-o-down:before {
  content: "\f150";
}
.fa-toggle-up:before,
.fa-caret-square-o-up:before {
  content: "\f151";
}
.fa-toggle-right:before,
.fa-caret-square-o-right:before {
  content: "\f152";
}
.fa-euro:before,
.fa-eur:before {
  content: "\f153";
}
.fa-gbp:before {
  content: "\f154";
}
.fa-dollar:before,
.fa-usd:before {
  content: "\f155";
}
.fa-rupee:before,
.fa-inr:before {
  content: "\f156";
}
.fa-cny:before,
.fa-rmb:before,
.fa-yen:before,
.fa-jpy:before {
  content: "\f157";
}
.fa-ruble:before,
.fa-rouble:before,
.fa-rub:before {
  content: "\f158";
}
.fa-won:before,
.fa-krw:before {
  content: "\f159";
}
.fa-bitcoin:before,
.fa-btc:before {
  content: "\f15a";
}
.fa-file:before {
  content: "\f15b";
}
.fa-file-text:before {
  content: "\f15c";
}
.fa-sort-alpha-asc:before {
  content: "\f15d";
}
.fa-sort-alpha-desc:before {
  content: "\f15e";
}
.fa-sort-amount-asc:before {
  content: "\f160";
}
.fa-sort-amount-desc:before {
  content: "\f161";
}
.fa-sort-numeric-asc:before {
  content: "\f162";
}
.fa-sort-numeric-desc:before {
  content: "\f163";
}
.fa-thumbs-up:before {
  content: "\f164";
}
.fa-thumbs-down:before {
  content: "\f165";
}
.fa-youtube-square:before {
  content: "\f166";
}
.fa-youtube:before {
  content: "\f167";
}
.fa-xing:before {
  content: "\f168";
}
.fa-xing-square:before {
  content: "\f169";
}
.fa-youtube-play:before {
  content: "\f16a";
}
.fa-dropbox:before {
  content: "\f16b";
}
.fa-stack-overflow:before {
  content: "\f16c";
}
.fa-instagram:before {
  content: "\f16d";
}
.fa-flickr:before {
  content: "\f16e";
}
.fa-adn:before {
  content: "\f170";
}
.fa-bitbucket:before {
  content: "\f171";
}
.fa-bitbucket-square:before {
  content: "\f172";
}
.fa-tumblr:before {
  content: "\f173";
}
.fa-tumblr-square:before {
  content: "\f174";
}
.fa-long-arrow-down:before {
  content: "\f175";
}
.fa-long-arrow-up:before {
  content: "\f176";
}
.fa-long-arrow-left:before {
  content: "\f177";
}
.fa-long-arrow-right:before {
  content: "\f178";
}
.fa-apple:before {
  content: "\f179";
}
.fa-windows:before {
  content: "\f17a";
}
.fa-android:before {
  content: "\f17b";
}
.fa-linux:before {
  content: "\f17c";
}
.fa-dribbble:before {
  content: "\f17d";
}
.fa-skype:before {
  content: "\f17e";
}
.fa-foursquare:before {
  content: "\f180";
}
.fa-trello:before {
  content: "\f181";
}
.fa-female:before {
  content: "\f182";
}
.fa-male:before {
  content: "\f183";
}
.fa-gittip:before,
.fa-gratipay:before {
  content: "\f184";
}
.fa-sun-o:before {
  content: "\f185";
}
.fa-moon-o:before {
  content: "\f186";
}
.fa-archive:before {
  content: "\f187";
}
.fa-bug:before {
  content: "\f188";
}
.fa-vk:before {
  content: "\f189";
}
.fa-weibo:before {
  content: "\f18a";
}
.fa-renren:before {
  content: "\f18b";
}
.fa-pagelines:before {
  content: "\f18c";
}
.fa-stack-exchange:before {
  content: "\f18d";
}
.fa-arrow-circle-o-right:before {
  content: "\f18e";
}
.fa-arrow-circle-o-left:before {
  content: "\f190";
}
.fa-toggle-left:before,
.fa-caret-square-o-left:before {
  content: "\f191";
}
.fa-dot-circle-o:before {
  content: "\f192";
}
.fa-wheelchair:before {
  content: "\f193";
}
.fa-vimeo-square:before {
  content: "\f194";
}
.fa-turkish-lira:before,
.fa-try:before {
  content: "\f195";
}
.fa-plus-square-o:before {
  content: "\f196";
}
.fa-space-shuttle:before {
  content: "\f197";
}
.fa-slack:before {
  content: "\f198";
}
.fa-envelope-square:before {
  content: "\f199";
}
.fa-wordpress:before {
  content: "\f19a";
}
.fa-openid:before {
  content: "\f19b";
}
.fa-institution:before,
.fa-bank:before,
.fa-university:before {
  content: "\f19c";
}
.fa-mortar-board:before,
.fa-graduation-cap:before {
  content: "\f19d";
}
.fa-yahoo:before {
  content: "\f19e";
}
.fa-google:before {
  content: "\f1a0";
}
.fa-reddit:before {
  content: "\f1a1";
}
.fa-reddit-square:before {
  content: "\f1a2";
}
.fa-stumbleupon-circle:before {
  content: "\f1a3";
}
.fa-stumbleupon:before {
  content: "\f1a4";
}
.fa-delicious:before {
  content: "\f1a5";
}
.fa-digg:before {
  content: "\f1a6";
}
.fa-pied-piper-pp:before {
  content: "\f1a7";
}
.fa-pied-piper-alt:before {
  content: "\f1a8";
}
.fa-drupal:before {
  content: "\f1a9";
}
.fa-joomla:before {
  content: "\f1aa";
}
.fa-language:before {
  content: "\f1ab";
}
.fa-fax:before {
  content: "\f1ac";
}
.fa-building:before {
  content: "\f1ad";
}
.fa-child:before {
  content: "\f1ae";
}
.fa-paw:before {
  content: "\f1b0";
}
.fa-spoon:before {
  content: "\f1b1";
}
.fa-cube:before {
  content: "\f1b2";
}
.fa-cubes:before {
  content: "\f1b3";
}
.fa-behance:before {
  content: "\f1b4";
}
.fa-behance-square:before {
  content: "\f1b5";
}
.fa-steam:before {
  content: "\f1b6";
}
.fa-steam-square:before {
  content: "\f1b7";
}
.fa-recycle:before {
  content: "\f1b8";
}
.fa-automobile:before,
.fa-car:before {
  content: "\f1b9";
}
.fa-cab:before,
.fa-taxi:before {
  content: "\f1ba";
}
.fa-tree:before {
  content: "\f1bb";
}
.fa-spotify:before {
  content: "\f1bc";
}
.fa-deviantart:before {
  content: "\f1bd";
}
.fa-soundcloud:before {
  content: "\f1be";
}
.fa-database:before {
  content: "\f1c0";
}
.fa-file-pdf-o:before {
  content: "\f1c1";
}
.fa-file-word-o:before {
  content: "\f1c2";
}
.fa-file-excel-o:before {
  content: "\f1c3";
}
.fa-file-powerpoint-o:before {
  content: "\f1c4";
}
.fa-file-photo-o:before,
.fa-file-picture-o:before,
.fa-file-image-o:before {
  content: "\f1c5";
}
.fa-file-zip-o:before,
.fa-file-archive-o:before {
  content: "\f1c6";
}
.fa-file-sound-o:before,
.fa-file-audio-o:before {
  content: "\f1c7";
}
.fa-file-movie-o:before,
.fa-file-video-o:before {
  content: "\f1c8";
}
.fa-file-code-o:before {
  content: "\f1c9";
}
.fa-vine:before {
  content: "\f1ca";
}
.fa-codepen:before {
  content: "\f1cb";
}
.fa-jsfiddle:before {
  content: "\f1cc";
}
.fa-life-bouy:before,
.fa-life-buoy:before,
.fa-life-saver:before,
.fa-support:before,
.fa-life-ring:before {
  content: "\f1cd";
}
.fa-circle-o-notch:before {
  content: "\f1ce";
}
.fa-ra:before,
.fa-resistance:before,
.fa-rebel:before {
  content: "\f1d0";
}
.fa-ge:before,
.fa-empire:before {
  content: "\f1d1";
}
.fa-git-square:before {
  content: "\f1d2";
}
.fa-git:before {
  content: "\f1d3";
}
.fa-y-combinator-square:before,
.fa-yc-square:before,
.fa-hacker-news:before {
  content: "\f1d4";
}
.fa-tencent-weibo:before {
  content: "\f1d5";
}
.fa-qq:before {
  content: "\f1d6";
}
.fa-wechat:before,
.fa-weixin:before {
  content: "\f1d7";
}
.fa-send:before,
.fa-paper-plane:before {
  content: "\f1d8";
}
.fa-send-o:before,
.fa-paper-plane-o:before {
  content: "\f1d9";
}
.fa-history:before {
  content: "\f1da";
}
.fa-circle-thin:before {
  content: "\f1db";
}
.fa-header:before {
  content: "\f1dc";
}
.fa-paragraph:before {
  content: "\f1dd";
}
.fa-sliders:before {
  content: "\f1de";
}
.fa-share-alt:before {
  content: "\f1e0";
}
.fa-share-alt-square:before {
  content: "\f1e1";
}
.fa-bomb:before {
  content: "\f1e2";
}
.fa-soccer-ball-o:before,
.fa-futbol-o:before {
  content: "\f1e3";
}
.fa-tty:before {
  content: "\f1e4";
}
.fa-binoculars:before {
  content: "\f1e5";
}
.fa-plug:before {
  content: "\f1e6";
}
.fa-slideshare:before {
  content: "\f1e7";
}
.fa-twitch:before {
  content: "\f1e8";
}
.fa-yelp:before {
  content: "\f1e9";
}
.fa-newspaper-o:before {
  content: "\f1ea";
}
.fa-wifi:before {
  content: "\f1eb";
}
.fa-calculator:before {
  content: "\f1ec";
}
.fa-paypal:before {
  content: "\f1ed";
}
.fa-google-wallet:before {
  content: "\f1ee";
}
.fa-cc-visa:before {
  content: "\f1f0";
}
.fa-cc-mastercard:before {
  content: "\f1f1";
}
.fa-cc-discover:before {
  content: "\f1f2";
}
.fa-cc-amex:before {
  content: "\f1f3";
}
.fa-cc-paypal:before {
  content: "\f1f4";
}
.fa-cc-stripe:before {
  content: "\f1f5";
}
.fa-bell-slash:before {
  content: "\f1f6";
}
.fa-bell-slash-o:before {
  content: "\f1f7";
}
.fa-trash:before {
  content: "\f1f8";
}
.fa-copyright:before {
  content: "\f1f9";
}
.fa-at:before {
  content: "\f1fa";
}
.fa-eyedropper:before {
  content: "\f1fb";
}
.fa-paint-brush:before {
  content: "\f1fc";
}
.fa-birthday-cake:before {
  content: "\f1fd";
}
.fa-area-chart:before {
  content: "\f1fe";
}
.fa-pie-chart:before {
  content: "\f200";
}
.fa-line-chart:before {
  content: "\f201";
}
.fa-lastfm:before {
  content: "\f202";
}
.fa-lastfm-square:before {
  content: "\f203";
}
.fa-toggle-off:before {
  content: "\f204";
}
.fa-toggle-on:before {
  content: "\f205";
}
.fa-bicycle:before {
  content: "\f206";
}
.fa-bus:before {
  content: "\f207";
}
.fa-ioxhost:before {
  content: "\f208";
}
.fa-angellist:before {
  content: "\f209";
}
.fa-cc:before {
  content: "\f20a";
}
.fa-shekel:before,
.fa-sheqel:before,
.fa-ils:before {
  content: "\f20b";
}
.fa-meanpath:before {
  content: "\f20c";
}
.fa-buysellads:before {
  content: "\f20d";
}
.fa-connectdevelop:before {
  content: "\f20e";
}
.fa-dashcube:before {
  content: "\f210";
}
.fa-forumbee:before {
  content: "\f211";
}
.fa-leanpub:before {
  content: "\f212";
}
.fa-sellsy:before {
  content: "\f213";
}
.fa-shirtsinbulk:before {
  content: "\f214";
}
.fa-simplybuilt:before {
  content: "\f215";
}
.fa-skyatlas:before {
  content: "\f216";
}
.fa-cart-plus:before {
  content: "\f217";
}
.fa-cart-arrow-down:before {
  content: "\f218";
}
.fa-diamond:before {
  content: "\f219";
}
.fa-ship:before {
  content: "\f21a";
}
.fa-user-secret:before {
  content: "\f21b";
}
.fa-motorcycle:before {
  content: "\f21c";
}
.fa-street-view:before {
  content: "\f21d";
}
.fa-heartbeat:before {
  content: "\f21e";
}
.fa-venus:before {
  content: "\f221";
}
.fa-mars:before {
  content: "\f222";
}
.fa-mercury:before {
  content: "\f223";
}
.fa-intersex:before,
.fa-transgender:before {
  content: "\f224";
}
.fa-transgender-alt:before {
  content: "\f225";
}
.fa-venus-double:before {
  content: "\f226";
}
.fa-mars-double:before {
  content: "\f227";
}
.fa-venus-mars:before {
  content: "\f228";
}
.fa-mars-stroke:before {
  content: "\f229";
}
.fa-mars-stroke-v:before {
  content: "\f22a";
}
.fa-mars-stroke-h:before {
  content: "\f22b";
}
.fa-neuter:before {
  content: "\f22c";
}
.fa-genderless:before {
  content: "\f22d";
}
.fa-facebook-official:before {
  content: "\f230";
}
.fa-pinterest-p:before {
  content: "\f231";
}
.fa-whatsapp:before {
  content: "\f232";
}
.fa-server:before {
  content: "\f233";
}
.fa-user-plus:before {
  content: "\f234";
}
.fa-user-times:before {
  content: "\f235";
}
.fa-hotel:before,
.fa-bed:before {
  content: "\f236";
}
.fa-viacoin:before {
  content: "\f237";
}
.fa-train:before {
  content: "\f238";
}
.fa-subway:before {
  content: "\f239";
}
.fa-medium:before {
  content: "\f23a";
}
.fa-yc:before,
.fa-y-combinator:before {
  content: "\f23b";
}
.fa-optin-monster:before {
  content: "\f23c";
}
.fa-opencart:before {
  content: "\f23d";
}
.fa-expeditedssl:before {
  content: "\f23e";
}
.fa-battery-4:before,
.fa-battery:before,
.fa-battery-full:before {
  content: "\f240";
}
.fa-battery-3:before,
.fa-battery-three-quarters:before {
  content: "\f241";
}
.fa-battery-2:before,
.fa-battery-half:before {
  content: "\f242";
}
.fa-battery-1:before,
.fa-battery-quarter:before {
  content: "\f243";
}
.fa-battery-0:before,
.fa-battery-empty:before {
  content: "\f244";
}
.fa-mouse-pointer:before {
  content: "\f245";
}
.fa-i-cursor:before {
  content: "\f246";
}
.fa-object-group:before {
  content: "\f247";
}
.fa-object-ungroup:before {
  content: "\f248";
}
.fa-sticky-note:before {
  content: "\f249";
}
.fa-sticky-note-o:before {
  content: "\f24a";
}
.fa-cc-jcb:before {
  content: "\f24b";
}
.fa-cc-diners-club:before {
  content: "\f24c";
}
.fa-clone:before {
  content: "\f24d";
}
.fa-balance-scale:before {
  content: "\f24e";
}
.fa-hourglass-o:before {
  content: "\f250";
}
.fa-hourglass-1:before,
.fa-hourglass-start:before {
  content: "\f251";
}
.fa-hourglass-2:before,
.fa-hourglass-half:before {
  content: "\f252";
}
.fa-hourglass-3:before,
.fa-hourglass-end:before {
  content: "\f253";
}
.fa-hourglass:before {
  content: "\f254";
}
.fa-hand-grab-o:before,
.fa-hand-rock-o:before {
  content: "\f255";
}
.fa-hand-stop-o:before,
.fa-hand-paper-o:before {
  content: "\f256";
}
.fa-hand-scissors-o:before {
  content: "\f257";
}
.fa-hand-lizard-o:before {
  content: "\f258";
}
.fa-hand-spock-o:before {
  content: "\f259";
}
.fa-hand-pointer-o:before {
  content: "\f25a";
}
.fa-hand-peace-o:before {
  content: "\f25b";
}
.fa-trademark:before {
  content: "\f25c";
}
.fa-registered:before {
  content: "\f25d";
}
.fa-creative-commons:before {
  content: "\f25e";
}
.fa-gg:before {
  content: "\f260";
}
.fa-gg-circle:before {
  content: "\f261";
}
.fa-tripadvisor:before {
  content: "\f262";
}
.fa-odnoklassniki:before {
  content: "\f263";
}
.fa-odnoklassniki-square:before {
  content: "\f264";
}
.fa-get-pocket:before {
  content: "\f265";
}
.fa-wikipedia-w:before {
  content: "\f266";
}
.fa-safari:before {
  content: "\f267";
}
.fa-chrome:before {
  content: "\f268";
}
.fa-firefox:before {
  content: "\f269";
}
.fa-opera:before {
  content: "\f26a";
}
.fa-internet-explorer:before {
  content: "\f26b";
}
.fa-tv:before,
.fa-television:before {
  content: "\f26c";
}
.fa-contao:before {
  content: "\f26d";
}
.fa-500px:before {
  content: "\f26e";
}
.fa-amazon:before {
  content: "\f270";
}
.fa-calendar-plus-o:before {
  content: "\f271";
}
.fa-calendar-minus-o:before {
  content: "\f272";
}
.fa-calendar-times-o:before {
  content: "\f273";
}
.fa-calendar-check-o:before {
  content: "\f274";
}
.fa-industry:before {
  content: "\f275";
}
.fa-map-pin:before {
  content: "\f276";
}
.fa-map-signs:before {
  content: "\f277";
}
.fa-map-o:before {
  content: "\f278";
}
.fa-map:before {
  content: "\f279";
}
.fa-commenting:before {
  content: "\f27a";
}
.fa-commenting-o:before {
  content: "\f27b";
}
.fa-houzz:before {
  content: "\f27c";
}
.fa-vimeo:before {
  content: "\f27d";
}
.fa-black-tie:before {
  content: "\f27e";
}
.fa-fonticons:before {
  content: "\f280";
}
.fa-reddit-alien:before {
  content: "\f281";
}
.fa-edge:before {
  content: "\f282";
}
.fa-credit-card-alt:before {
  content: "\f283";
}
.fa-codiepie:before {
  content: "\f284";
}
.fa-modx:before {
  content: "\f285";
}
.fa-fort-awesome:before {
  content: "\f286";
}
.fa-usb:before {
  content: "\f287";
}
.fa-product-hunt:before {
  content: "\f288";
}
.fa-mixcloud:before {
  content: "\f289";
}
.fa-scribd:before {
  content: "\f28a";
}
.fa-pause-circle:before {
  content: "\f28b";
}
.fa-pause-circle-o:before {
  content: "\f28c";
}
.fa-stop-circle:before {
  content: "\f28d";
}
.fa-stop-circle-o:before {
  content: "\f28e";
}
.fa-shopping-bag:before {
  content: "\f290";
}
.fa-shopping-basket:before {
  content: "\f291";
}
.fa-hashtag:before {
  content: "\f292";
}
.fa-bluetooth:before {
  content: "\f293";
}
.fa-bluetooth-b:before {
  content: "\f294";
}
.fa-percent:before {
  content: "\f295";
}
.fa-gitlab:before {
  content: "\f296";
}
.fa-wpbeginner:before {
  content: "\f297";
}
.fa-wpforms:before {
  content: "\f298";
}
.fa-envira:before {
  content: "\f299";
}
.fa-universal-access:before {
  content: "\f29a";
}
.fa-wheelchair-alt:before {
  content: "\f29b";
}
.fa-question-circle-o:before {
  content: "\f29c";
}
.fa-blind:before {
  content: "\f29d";
}
.fa-audio-description:before {
  content: "\f29e";
}
.fa-volume-control-phone:before {
  content: "\f2a0";
}
.fa-braille:before {
  content: "\f2a1";
}
.fa-assistive-listening-systems:before {
  content: "\f2a2";
}
.fa-asl-interpreting:before,
.fa-american-sign-language-interpreting:before {
  content: "\f2a3";
}
.fa-deafness:before,
.fa-hard-of-hearing:before,
.fa-deaf:before {
  content: "\f2a4";
}
.fa-glide:before {
  content: "\f2a5";
}
.fa-glide-g:before {
  content: "\f2a6";
}
.fa-signing:before,
.fa-sign-language:before {
  content: "\f2a7";
}
.fa-low-vision:before {
  content: "\f2a8";
}
.fa-viadeo:before {
  content: "\f2a9";
}
.fa-viadeo-square:before {
  content: "\f2aa";
}
.fa-snapchat:before {
  content: "\f2ab";
}
.fa-snapchat-ghost:before {
  content: "\f2ac";
}
.fa-snapchat-square:before {
  content: "\f2ad";
}
.fa-pied-piper:before {
  content: "\f2ae";
}
.fa-first-order:before {
  content: "\f2b0";
}
.fa-yoast:before {
  content: "\f2b1";
}
.fa-themeisle:before {
  content: "\f2b2";
}
.fa-google-plus-circle:before,
.fa-google-plus-official:before {
  content: "\f2b3";
}
.fa-fa:before,
.fa-font-awesome:before {
  content: "\f2b4";
}
.fa-handshake-o:before {
  content: "\f2b5";
}
.fa-envelope-open:before {
  content: "\f2b6";
}
.fa-envelope-open-o:before {
  content: "\f2b7";
}
.fa-linode:before {
  content: "\f2b8";
}
.fa-address-book:before {
  content: "\f2b9";
}
.fa-address-book-o:before {
  content: "\f2ba";
}
.fa-vcard:before,
.fa-address-card:before {
  content: "\f2bb";
}
.fa-vcard-o:before,
.fa-address-card-o:before {
  content: "\f2bc";
}
.fa-user-circle:before {
  content: "\f2bd";
}
.fa-user-circle-o:before {
  content: "\f2be";
}
.fa-user-o:before {
  content: "\f2c0";
}
.fa-id-badge:before {
  content: "\f2c1";
}
.fa-drivers-license:before,
.fa-id-card:before {
  content: "\f2c2";
}
.fa-drivers-license-o:before,
.fa-id-card-o:before {
  content: "\f2c3";
}
.fa-quora:before {
  content: "\f2c4";
}
.fa-free-code-camp:before {
  content: "\f2c5";
}
.fa-telegram:before {
  content: "\f2c6";
}
.fa-thermometer-4:before,
.fa-thermometer:before,
.fa-thermometer-full:before {
  content: "\f2c7";
}
.fa-thermometer-3:before,
.fa-thermometer-three-quarters:before {
  content: "\f2c8";
}
.fa-thermometer-2:before,
.fa-thermometer-half:before {
  content: "\f2c9";
}
.fa-thermometer-1:before,
.fa-thermometer-quarter:before {
  content: "\f2ca";
}
.fa-thermometer-0:before,
.fa-thermometer-empty:before {
  content: "\f2cb";
}
.fa-shower:before {
  content: "\f2cc";
}
.fa-bathtub:before,
.fa-s15:before,
.fa-bath:before {
  content: "\f2cd";
}
.fa-podcast:before {
  content: "\f2ce";
}
.fa-window-maximize:before {
  content: "\f2d0";
}
.fa-window-minimize:before {
  content: "\f2d1";
}
.fa-window-restore:before {
  content: "\f2d2";
}
.fa-times-rectangle:before,
.fa-window-close:before {
  content: "\f2d3";
}
.fa-times-rectangle-o:before,
.fa-window-close-o:before {
  content: "\f2d4";
}
.fa-bandcamp:before {
  content: "\f2d5";
}
.fa-grav:before {
  content: "\f2d6";
}
.fa-etsy:before {
  content: "\f2d7";
}
.fa-imdb:before {
  content: "\f2d8";
}
.fa-ravelry:before {
  content: "\f2d9";
}
.fa-eercast:before {
  content: "\f2da";
}
.fa-microchip:before {
  content: "\f2db";
}
.fa-snowflake-o:before {
  content: "\f2dc";
}
.fa-superpowers:before {
  content: "\f2dd";
}
.fa-wpexplorer:before {
  content: "\f2de";
}
.fa-meetup:before {
  content: "\f2e0";
}
.sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  border: 0;
}
.sr-only-focusable:active,
.sr-only-focusable:focus {
  position: static;
  width: auto;
  height: auto;
  margin: 0;
  overflow: visible;
  clip: auto;
}
.sr-only-focusable:active,
.sr-only-focusable:focus {
  position: static;
  width: auto;
  height: auto;
  margin: 0;
  overflow: visible;
  clip: auto;
}
/*!
*
* IPython base
*
*/
.modal.fade .modal-dialog {
  -webkit-transform: translate(0, 0);
  -ms-transform: translate(0, 0);
  -o-transform: translate(0, 0);
  transform: translate(0, 0);
}
code {
  color: #000;
}
pre {
  font-size: inherit;
  line-height: inherit;
}
label {
  font-weight: normal;
}
/* Make the page background atleast 100% the height of the view port */
/* Make the page itself atleast 70% the height of the view port */
.border-box-sizing {
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
}
.corner-all {
  border-radius: 2px;
}
.no-padding {
  padding: 0px;
}
/* Flexible box model classes */
/* Taken from Alex Russell http://infrequently.org/2009/08/css-3-progress/ */
/* This file is a compatability layer.  It allows the usage of flexible box 
model layouts accross multiple browsers, including older browsers.  The newest,
universal implementation of the flexible box model is used when available (see
`Modern browsers` comments below).  Browsers that are known to implement this 
new spec completely include:

    Firefox 28.0+
    Chrome 29.0+
    Internet Explorer 11+ 
    Opera 17.0+

Browsers not listed, including Safari, are supported via the styling under the
`Old browsers` comments below.
*/
.hbox {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
}
.hbox > * {
  /* Old browsers */
  -webkit-box-flex: 0;
  -moz-box-flex: 0;
  box-flex: 0;
  /* Modern browsers */
  flex: none;
}
.vbox {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
}
.vbox > * {
  /* Old browsers */
  -webkit-box-flex: 0;
  -moz-box-flex: 0;
  box-flex: 0;
  /* Modern browsers */
  flex: none;
}
.hbox.reverse,
.vbox.reverse,
.reverse {
  /* Old browsers */
  -webkit-box-direction: reverse;
  -moz-box-direction: reverse;
  box-direction: reverse;
  /* Modern browsers */
  flex-direction: row-reverse;
}
.hbox.box-flex0,
.vbox.box-flex0,
.box-flex0 {
  /* Old browsers */
  -webkit-box-flex: 0;
  -moz-box-flex: 0;
  box-flex: 0;
  /* Modern browsers */
  flex: none;
  width: auto;
}
.hbox.box-flex1,
.vbox.box-flex1,
.box-flex1 {
  /* Old browsers */
  -webkit-box-flex: 1;
  -moz-box-flex: 1;
  box-flex: 1;
  /* Modern browsers */
  flex: 1;
}
.hbox.box-flex,
.vbox.box-flex,
.box-flex {
  /* Old browsers */
  /* Old browsers */
  -webkit-box-flex: 1;
  -moz-box-flex: 1;
  box-flex: 1;
  /* Modern browsers */
  flex: 1;
}
.hbox.box-flex2,
.vbox.box-flex2,
.box-flex2 {
  /* Old browsers */
  -webkit-box-flex: 2;
  -moz-box-flex: 2;
  box-flex: 2;
  /* Modern browsers */
  flex: 2;
}
.box-group1 {
  /*  Deprecated */
  -webkit-box-flex-group: 1;
  -moz-box-flex-group: 1;
  box-flex-group: 1;
}
.box-group2 {
  /* Deprecated */
  -webkit-box-flex-group: 2;
  -moz-box-flex-group: 2;
  box-flex-group: 2;
}
.hbox.start,
.vbox.start,
.start {
  /* Old browsers */
  -webkit-box-pack: start;
  -moz-box-pack: start;
  box-pack: start;
  /* Modern browsers */
  justify-content: flex-start;
}
.hbox.end,
.vbox.end,
.end {
  /* Old browsers */
  -webkit-box-pack: end;
  -moz-box-pack: end;
  box-pack: end;
  /* Modern browsers */
  justify-content: flex-end;
}
.hbox.center,
.vbox.center,
.center {
  /* Old browsers */
  -webkit-box-pack: center;
  -moz-box-pack: center;
  box-pack: center;
  /* Modern browsers */
  justify-content: center;
}
.hbox.baseline,
.vbox.baseline,
.baseline {
  /* Old browsers */
  -webkit-box-pack: baseline;
  -moz-box-pack: baseline;
  box-pack: baseline;
  /* Modern browsers */
  justify-content: baseline;
}
.hbox.stretch,
.vbox.stretch,
.stretch {
  /* Old browsers */
  -webkit-box-pack: stretch;
  -moz-box-pack: stretch;
  box-pack: stretch;
  /* Modern browsers */
  justify-content: stretch;
}
.hbox.align-start,
.vbox.align-start,
.align-start {
  /* Old browsers */
  -webkit-box-align: start;
  -moz-box-align: start;
  box-align: start;
  /* Modern browsers */
  align-items: flex-start;
}
.hbox.align-end,
.vbox.align-end,
.align-end {
  /* Old browsers */
  -webkit-box-align: end;
  -moz-box-align: end;
  box-align: end;
  /* Modern browsers */
  align-items: flex-end;
}
.hbox.align-center,
.vbox.align-center,
.align-center {
  /* Old browsers */
  -webkit-box-align: center;
  -moz-box-align: center;
  box-align: center;
  /* Modern browsers */
  align-items: center;
}
.hbox.align-baseline,
.vbox.align-baseline,
.align-baseline {
  /* Old browsers */
  -webkit-box-align: baseline;
  -moz-box-align: baseline;
  box-align: baseline;
  /* Modern browsers */
  align-items: baseline;
}
.hbox.align-stretch,
.vbox.align-stretch,
.align-stretch {
  /* Old browsers */
  -webkit-box-align: stretch;
  -moz-box-align: stretch;
  box-align: stretch;
  /* Modern browsers */
  align-items: stretch;
}
div.error {
  margin: 2em;
  text-align: center;
}
div.error > h1 {
  font-size: 500%;
  line-height: normal;
}
div.error > p {
  font-size: 200%;
  line-height: normal;
}
div.traceback-wrapper {
  text-align: left;
  max-width: 800px;
  margin: auto;
}
div.traceback-wrapper pre.traceback {
  max-height: 600px;
  overflow: auto;
}
/**
 * Primary styles
 *
 * Author: Jupyter Development Team
 */
body {
  background-color: #fff;
  /* This makes sure that the body covers the entire window and needs to
       be in a different element than the display: box in wrapper below */
  position: absolute;
  left: 0px;
  right: 0px;
  top: 0px;
  bottom: 0px;
  overflow: visible;
}
body > #header {
  /* Initially hidden to prevent FLOUC */
  display: none;
  background-color: #fff;
  /* Display over codemirror */
  position: relative;
  z-index: 100;
}
body > #header #header-container {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding: 5px;
  padding-bottom: 5px;
  padding-top: 5px;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
}
body > #header .header-bar {
  width: 100%;
  height: 1px;
  background: #e7e7e7;
  margin-bottom: -1px;
}
@media print {
  body > #header {
    display: none !important;
  }
}
#header-spacer {
  width: 100%;
  visibility: hidden;
}
@media print {
  #header-spacer {
    display: none;
  }
}
#ipython_notebook {
  padding-left: 0px;
  padding-top: 1px;
  padding-bottom: 1px;
}
[dir="rtl"] #ipython_notebook {
  margin-right: 10px;
  margin-left: 0;
}
[dir="rtl"] #ipython_notebook.pull-left {
  float: right !important;
  float: right;
}
.flex-spacer {
  flex: 1;
}
#noscript {
  width: auto;
  padding-top: 16px;
  padding-bottom: 16px;
  text-align: center;
  font-size: 22px;
  color: red;
  font-weight: bold;
}
#ipython_notebook img {
  height: 28px;
}
#site {
  width: 100%;
  display: none;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  overflow: auto;
}
@media print {
  #site {
    height: auto !important;
  }
}
/* Smaller buttons */
.ui-button .ui-button-text {
  padding: 0.2em 0.8em;
  font-size: 77%;
}
input.ui-button {
  padding: 0.3em 0.9em;
}
span#kernel_logo_widget {
  margin: 0 10px;
}
span#login_widget {
  float: right;
}
[dir="rtl"] span#login_widget {
  float: left;
}
span#login_widget > .button,
#logout {
  color: #333;
  background-color: #fff;
  border-color: #ccc;
}
span#login_widget > .button:focus,
#logout:focus,
span#login_widget > .button.focus,
#logout.focus {
  color: #333;
  background-color: #e6e6e6;
  border-color: #8c8c8c;
}
span#login_widget > .button:hover,
#logout:hover {
  color: #333;
  background-color: #e6e6e6;
  border-color: #adadad;
}
span#login_widget > .button:active,
#logout:active,
span#login_widget > .button.active,
#logout.active,
.open > .dropdown-togglespan#login_widget > .button,
.open > .dropdown-toggle#logout {
  color: #333;
  background-color: #e6e6e6;
  border-color: #adadad;
}
span#login_widget > .button:active:hover,
#logout:active:hover,
span#login_widget > .button.active:hover,
#logout.active:hover,
.open > .dropdown-togglespan#login_widget > .button:hover,
.open > .dropdown-toggle#logout:hover,
span#login_widget > .button:active:focus,
#logout:active:focus,
span#login_widget > .button.active:focus,
#logout.active:focus,
.open > .dropdown-togglespan#login_widget > .button:focus,
.open > .dropdown-toggle#logout:focus,
span#login_widget > .button:active.focus,
#logout:active.focus,
span#login_widget > .button.active.focus,
#logout.active.focus,
.open > .dropdown-togglespan#login_widget > .button.focus,
.open > .dropdown-toggle#logout.focus {
  color: #333;
  background-color: #d4d4d4;
  border-color: #8c8c8c;
}
span#login_widget > .button:active,
#logout:active,
span#login_widget > .button.active,
#logout.active,
.open > .dropdown-togglespan#login_widget > .button,
.open > .dropdown-toggle#logout {
  background-image: none;
}
span#login_widget > .button.disabled:hover,
#logout.disabled:hover,
span#login_widget > .button[disabled]:hover,
#logout[disabled]:hover,
fieldset[disabled] span#login_widget > .button:hover,
fieldset[disabled] #logout:hover,
span#login_widget > .button.disabled:focus,
#logout.disabled:focus,
span#login_widget > .button[disabled]:focus,
#logout[disabled]:focus,
fieldset[disabled] span#login_widget > .button:focus,
fieldset[disabled] #logout:focus,
span#login_widget > .button.disabled.focus,
#logout.disabled.focus,
span#login_widget > .button[disabled].focus,
#logout[disabled].focus,
fieldset[disabled] span#login_widget > .button.focus,
fieldset[disabled] #logout.focus {
  background-color: #fff;
  border-color: #ccc;
}
span#login_widget > .button .badge,
#logout .badge {
  color: #fff;
  background-color: #333;
}
.nav-header {
  text-transform: none;
}
#header > span {
  margin-top: 10px;
}
.modal_stretch .modal-dialog {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
  min-height: 80vh;
}
.modal_stretch .modal-dialog .modal-body {
  max-height: calc(100vh - 200px);
  overflow: auto;
  flex: 1;
}
.modal-header {
  cursor: move;
}
@media (min-width: 768px) {
  .modal .modal-dialog {
    width: 700px;
  }
}
@media (min-width: 768px) {
  select.form-control {
    margin-left: 12px;
    margin-right: 12px;
  }
}
/*!
*
* IPython auth
*
*/
.center-nav {
  display: inline-block;
  margin-bottom: -4px;
}
[dir="rtl"] .center-nav form.pull-left {
  float: right !important;
  float: right;
}
[dir="rtl"] .center-nav .navbar-text {
  float: right;
}
[dir="rtl"] .navbar-inner {
  text-align: right;
}
[dir="rtl"] div.text-left {
  text-align: right;
}
/*!
*
* IPython tree view
*
*/
/* We need an invisible input field on top of the sentense*/
/* "Drag file onto the list ..." */
.alternate_upload {
  background-color: none;
  display: inline;
}
.alternate_upload.form {
  padding: 0;
  margin: 0;
}
.alternate_upload input.fileinput {
  position: absolute;
  display: block;
  width: 100%;
  height: 100%;
  overflow: hidden;
  cursor: pointer;
  opacity: 0;
  z-index: 2;
}
.alternate_upload .btn-xs > input.fileinput {
  margin: -1px -5px;
}
.alternate_upload .btn-upload {
  position: relative;
  height: 22px;
}
::-webkit-file-upload-button {
  cursor: pointer;
}
/**
 * Primary styles
 *
 * Author: Jupyter Development Team
 */
ul#tabs {
  margin-bottom: 4px;
}
ul#tabs a {
  padding-top: 6px;
  padding-bottom: 4px;
}
[dir="rtl"] ul#tabs.nav-tabs > li {
  float: right;
}
[dir="rtl"] ul#tabs.nav.nav-tabs {
  padding-right: 0;
}
ul.breadcrumb a:focus,
ul.breadcrumb a:hover {
  text-decoration: none;
}
ul.breadcrumb i.icon-home {
  font-size: 16px;
  margin-right: 4px;
}
ul.breadcrumb span {
  color: #5e5e5e;
}
.list_toolbar {
  padding: 4px 0 4px 0;
  vertical-align: middle;
}
.list_toolbar .tree-buttons {
  padding-top: 1px;
}
[dir="rtl"] .list_toolbar .tree-buttons .pull-right {
  float: left !important;
  float: left;
}
[dir="rtl"] .list_toolbar .col-sm-4,
[dir="rtl"] .list_toolbar .col-sm-8 {
  float: right;
}
.dynamic-buttons {
  padding-top: 3px;
  display: inline-block;
}
.list_toolbar [class*="span"] {
  min-height: 24px;
}
.list_header {
  font-weight: bold;
  background-color: #EEE;
}
.list_placeholder {
  font-weight: bold;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 7px;
  padding-right: 7px;
}
.list_container {
  margin-top: 4px;
  margin-bottom: 20px;
  border: 1px solid #ddd;
  border-radius: 2px;
}
.list_container > div {
  border-bottom: 1px solid #ddd;
}
.list_container > div:hover .list-item {
  background-color: red;
}
.list_container > div:last-child {
  border: none;
}
.list_item:hover .list_item {
  background-color: #ddd;
}
.list_item a {
  text-decoration: none;
}
.list_item:hover {
  background-color: #fafafa;
}
.list_header > div,
.list_item > div {
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 7px;
  padding-right: 7px;
  line-height: 22px;
}
.list_header > div input,
.list_item > div input {
  margin-right: 7px;
  margin-left: 14px;
  vertical-align: text-bottom;
  line-height: 22px;
  position: relative;
  top: -1px;
}
.list_header > div .item_link,
.list_item > div .item_link {
  margin-left: -1px;
  vertical-align: baseline;
  line-height: 22px;
}
[dir="rtl"] .list_item > div input {
  margin-right: 0;
}
.new-file input[type=checkbox] {
  visibility: hidden;
}
.item_name {
  line-height: 22px;
  height: 24px;
}
.item_icon {
  font-size: 14px;
  color: #5e5e5e;
  margin-right: 7px;
  margin-left: 7px;
  line-height: 22px;
  vertical-align: baseline;
}
.item_modified {
  margin-right: 7px;
  margin-left: 7px;
}
[dir="rtl"] .item_modified.pull-right {
  float: left !important;
  float: left;
}
.item_buttons {
  line-height: 1em;
  margin-left: -5px;
}
.item_buttons .btn,
.item_buttons .btn-group,
.item_buttons .input-group {
  float: left;
}
.item_buttons > .btn,
.item_buttons > .btn-group,
.item_buttons > .input-group {
  margin-left: 5px;
}
.item_buttons .btn {
  min-width: 13ex;
}
.item_buttons .running-indicator {
  padding-top: 4px;
  color: #5cb85c;
}
.item_buttons .kernel-name {
  padding-top: 4px;
  color: #5bc0de;
  margin-right: 7px;
  float: left;
}
[dir="rtl"] .item_buttons.pull-right {
  float: left !important;
  float: left;
}
[dir="rtl"] .item_buttons .kernel-name {
  margin-left: 7px;
  float: right;
}
.toolbar_info {
  height: 24px;
  line-height: 24px;
}
.list_item input:not([type=checkbox]) {
  padding-top: 3px;
  padding-bottom: 3px;
  height: 22px;
  line-height: 14px;
  margin: 0px;
}
.highlight_text {
  color: blue;
}
#project_name {
  display: inline-block;
  padding-left: 7px;
  margin-left: -2px;
}
#project_name > .breadcrumb {
  padding: 0px;
  margin-bottom: 0px;
  background-color: transparent;
  font-weight: bold;
}
.sort_button {
  display: inline-block;
  padding-left: 7px;
}
[dir="rtl"] .sort_button.pull-right {
  float: left !important;
  float: left;
}
#tree-selector {
  padding-right: 0px;
}
#button-select-all {
  min-width: 50px;
}
[dir="rtl"] #button-select-all.btn {
  float: right ;
}
#select-all {
  margin-left: 7px;
  margin-right: 2px;
  margin-top: 2px;
  height: 16px;
}
[dir="rtl"] #select-all.pull-left {
  float: right !important;
  float: right;
}
.menu_icon {
  margin-right: 2px;
}
.tab-content .row {
  margin-left: 0px;
  margin-right: 0px;
}
.folder_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f114";
}
.folder_icon:before.fa-pull-left {
  margin-right: .3em;
}
.folder_icon:before.fa-pull-right {
  margin-left: .3em;
}
.folder_icon:before.pull-left {
  margin-right: .3em;
}
.folder_icon:before.pull-right {
  margin-left: .3em;
}
.notebook_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f02d";
  position: relative;
  top: -1px;
}
.notebook_icon:before.fa-pull-left {
  margin-right: .3em;
}
.notebook_icon:before.fa-pull-right {
  margin-left: .3em;
}
.notebook_icon:before.pull-left {
  margin-right: .3em;
}
.notebook_icon:before.pull-right {
  margin-left: .3em;
}
.running_notebook_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f02d";
  position: relative;
  top: -1px;
  color: #5cb85c;
}
.running_notebook_icon:before.fa-pull-left {
  margin-right: .3em;
}
.running_notebook_icon:before.fa-pull-right {
  margin-left: .3em;
}
.running_notebook_icon:before.pull-left {
  margin-right: .3em;
}
.running_notebook_icon:before.pull-right {
  margin-left: .3em;
}
.file_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f016";
  position: relative;
  top: -2px;
}
.file_icon:before.fa-pull-left {
  margin-right: .3em;
}
.file_icon:before.fa-pull-right {
  margin-left: .3em;
}
.file_icon:before.pull-left {
  margin-right: .3em;
}
.file_icon:before.pull-right {
  margin-left: .3em;
}
#notebook_toolbar .pull-right {
  padding-top: 0px;
  margin-right: -1px;
}
ul#new-menu {
  left: auto;
  right: 0;
}
#new-menu .dropdown-header {
  font-size: 10px;
  border-bottom: 1px solid #e5e5e5;
  padding: 0 0 3px;
  margin: -3px 20px 0;
}
.kernel-menu-icon {
  padding-right: 12px;
  width: 24px;
  content: "\f096";
}
.kernel-menu-icon:before {
  content: "\f096";
}
.kernel-menu-icon-current:before {
  content: "\f00c";
}
#tab_content {
  padding-top: 20px;
}
#running .panel-group .panel {
  margin-top: 3px;
  margin-bottom: 1em;
}
#running .panel-group .panel .panel-heading {
  background-color: #EEE;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 7px;
  padding-right: 7px;
  line-height: 22px;
}
#running .panel-group .panel .panel-heading a:focus,
#running .panel-group .panel .panel-heading a:hover {
  text-decoration: none;
}
#running .panel-group .panel .panel-body {
  padding: 0px;
}
#running .panel-group .panel .panel-body .list_container {
  margin-top: 0px;
  margin-bottom: 0px;
  border: 0px;
  border-radius: 0px;
}
#running .panel-group .panel .panel-body .list_container .list_item {
  border-bottom: 1px solid #ddd;
}
#running .panel-group .panel .panel-body .list_container .list_item:last-child {
  border-bottom: 0px;
}
.delete-button {
  display: none;
}
.duplicate-button {
  display: none;
}
.rename-button {
  display: none;
}
.move-button {
  display: none;
}
.download-button {
  display: none;
}
.shutdown-button {
  display: none;
}
.dynamic-instructions {
  display: inline-block;
  padding-top: 4px;
}
/*!
*
* IPython text editor webapp
*
*/
.selected-keymap i.fa {
  padding: 0px 5px;
}
.selected-keymap i.fa:before {
  content: "\f00c";
}
#mode-menu {
  overflow: auto;
  max-height: 20em;
}
.edit_app #header {
  -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
}
.edit_app #menubar .navbar {
  /* Use a negative 1 bottom margin, so the border overlaps the border of the
    header */
  margin-bottom: -1px;
}
.dirty-indicator {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  width: 20px;
}
.dirty-indicator.fa-pull-left {
  margin-right: .3em;
}
.dirty-indicator.fa-pull-right {
  margin-left: .3em;
}
.dirty-indicator.pull-left {
  margin-right: .3em;
}
.dirty-indicator.pull-right {
  margin-left: .3em;
}
.dirty-indicator-dirty {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  width: 20px;
}
.dirty-indicator-dirty.fa-pull-left {
  margin-right: .3em;
}
.dirty-indicator-dirty.fa-pull-right {
  margin-left: .3em;
}
.dirty-indicator-dirty.pull-left {
  margin-right: .3em;
}
.dirty-indicator-dirty.pull-right {
  margin-left: .3em;
}
.dirty-indicator-clean {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  width: 20px;
}
.dirty-indicator-clean.fa-pull-left {
  margin-right: .3em;
}
.dirty-indicator-clean.fa-pull-right {
  margin-left: .3em;
}
.dirty-indicator-clean.pull-left {
  margin-right: .3em;
}
.dirty-indicator-clean.pull-right {
  margin-left: .3em;
}
.dirty-indicator-clean:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f00c";
}
.dirty-indicator-clean:before.fa-pull-left {
  margin-right: .3em;
}
.dirty-indicator-clean:before.fa-pull-right {
  margin-left: .3em;
}
.dirty-indicator-clean:before.pull-left {
  margin-right: .3em;
}
.dirty-indicator-clean:before.pull-right {
  margin-left: .3em;
}
#filename {
  font-size: 16pt;
  display: table;
  padding: 0px 5px;
}
#current-mode {
  padding-left: 5px;
  padding-right: 5px;
}
#texteditor-backdrop {
  padding-top: 20px;
  padding-bottom: 20px;
}
@media not print {
  #texteditor-backdrop {
    background-color: #EEE;
  }
}
@media print {
  #texteditor-backdrop #texteditor-container .CodeMirror-gutter,
  #texteditor-backdrop #texteditor-container .CodeMirror-gutters {
    background-color: #fff;
  }
}
@media not print {
  #texteditor-backdrop #texteditor-container .CodeMirror-gutter,
  #texteditor-backdrop #texteditor-container .CodeMirror-gutters {
    background-color: #fff;
  }
}
@media not print {
  #texteditor-backdrop #texteditor-container {
    padding: 0px;
    background-color: #fff;
    -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
    box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  }
}
.CodeMirror-dialog {
  background-color: #fff;
}
/*!
*
* IPython notebook
*
*/
/* CSS font colors for translated ANSI escape sequences */
/* The color values are a mix of
   http://www.xcolors.net/dl/baskerville-ivorylight and
   http://www.xcolors.net/dl/euphrasia */
.ansi-black-fg {
  color: #3E424D;
}
.ansi-black-bg {
  background-color: #3E424D;
}
.ansi-black-intense-fg {
  color: #282C36;
}
.ansi-black-intense-bg {
  background-color: #282C36;
}
.ansi-red-fg {
  color: #E75C58;
}
.ansi-red-bg {
  background-color: #E75C58;
}
.ansi-red-intense-fg {
  color: #B22B31;
}
.ansi-red-intense-bg {
  background-color: #B22B31;
}
.ansi-green-fg {
  color: #00A250;
}
.ansi-green-bg {
  background-color: #00A250;
}
.ansi-green-intense-fg {
  color: #007427;
}
.ansi-green-intense-bg {
  background-color: #007427;
}
.ansi-yellow-fg {
  color: #DDB62B;
}
.ansi-yellow-bg {
  background-color: #DDB62B;
}
.ansi-yellow-intense-fg {
  color: #B27D12;
}
.ansi-yellow-intense-bg {
  background-color: #B27D12;
}
.ansi-blue-fg {
  color: #208FFB;
}
.ansi-blue-bg {
  background-color: #208FFB;
}
.ansi-blue-intense-fg {
  color: #0065CA;
}
.ansi-blue-intense-bg {
  background-color: #0065CA;
}
.ansi-magenta-fg {
  color: #D160C4;
}
.ansi-magenta-bg {
  background-color: #D160C4;
}
.ansi-magenta-intense-fg {
  color: #A03196;
}
.ansi-magenta-intense-bg {
  background-color: #A03196;
}
.ansi-cyan-fg {
  color: #60C6C8;
}
.ansi-cyan-bg {
  background-color: #60C6C8;
}
.ansi-cyan-intense-fg {
  color: #258F8F;
}
.ansi-cyan-intense-bg {
  background-color: #258F8F;
}
.ansi-white-fg {
  color: #C5C1B4;
}
.ansi-white-bg {
  background-color: #C5C1B4;
}
.ansi-white-intense-fg {
  color: #A1A6B2;
}
.ansi-white-intense-bg {
  background-color: #A1A6B2;
}
.ansi-default-inverse-fg {
  color: #FFFFFF;
}
.ansi-default-inverse-bg {
  background-color: #000000;
}
.ansi-bold {
  font-weight: bold;
}
.ansi-underline {
  text-decoration: underline;
}
/* The following styles are deprecated an will be removed in a future version */
.ansibold {
  font-weight: bold;
}
.ansi-inverse {
  outline: 0.5px dotted;
}
/* use dark versions for foreground, to improve visibility */
.ansiblack {
  color: black;
}
.ansired {
  color: darkred;
}
.ansigreen {
  color: darkgreen;
}
.ansiyellow {
  color: #c4a000;
}
.ansiblue {
  color: darkblue;
}
.ansipurple {
  color: darkviolet;
}
.ansicyan {
  color: steelblue;
}
.ansigray {
  color: gray;
}
/* and light for background, for the same reason */
.ansibgblack {
  background-color: black;
}
.ansibgred {
  background-color: red;
}
.ansibggreen {
  background-color: green;
}
.ansibgyellow {
  background-color: yellow;
}
.ansibgblue {
  background-color: blue;
}
.ansibgpurple {
  background-color: magenta;
}
.ansibgcyan {
  background-color: cyan;
}
.ansibggray {
  background-color: gray;
}
div.cell {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
  border-radius: 2px;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  border-width: 1px;
  border-style: solid;
  border-color: transparent;
  width: 100%;
  padding: 5px;
  /* This acts as a spacer between cells, that is outside the border */
  margin: 0px;
  outline: none;
  position: relative;
  overflow: visible;
}
div.cell:before {
  position: absolute;
  display: block;
  top: -1px;
  left: -1px;
  width: 5px;
  height: calc(100% +  2px);
  content: '';
  background: transparent;
}
div.cell.jupyter-soft-selected {
  border-left-color: #E3F2FD;
  border-left-width: 1px;
  padding-left: 5px;
  border-right-color: #E3F2FD;
  border-right-width: 1px;
  background: #E3F2FD;
}
@media print {
  div.cell.jupyter-soft-selected {
    border-color: transparent;
  }
}
div.cell.selected,
div.cell.selected.jupyter-soft-selected {
  border-color: #ababab;
}
div.cell.selected:before,
div.cell.selected.jupyter-soft-selected:before {
  position: absolute;
  display: block;
  top: -1px;
  left: -1px;
  width: 5px;
  height: calc(100% +  2px);
  content: '';
  background: #42A5F5;
}
@media print {
  div.cell.selected,
  div.cell.selected.jupyter-soft-selected {
    border-color: transparent;
  }
}
.edit_mode div.cell.selected {
  border-color: #66BB6A;
}
.edit_mode div.cell.selected:before {
  position: absolute;
  display: block;
  top: -1px;
  left: -1px;
  width: 5px;
  height: calc(100% +  2px);
  content: '';
  background: #66BB6A;
}
@media print {
  .edit_mode div.cell.selected {
    border-color: transparent;
  }
}
.prompt {
  /* This needs to be wide enough for 3 digit prompt numbers: In[100]: */
  min-width: 14ex;
  /* This padding is tuned to match the padding on the CodeMirror editor. */
  padding: 0.4em;
  margin: 0px;
  font-family: monospace;
  text-align: right;
  /* This has to match that of the the CodeMirror class line-height below */
  line-height: 1.21429em;
  /* Don't highlight prompt number selection */
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -khtml-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  /* Use default cursor */
  cursor: default;
}
@media (max-width: 540px) {
  .prompt {
    text-align: left;
  }
}
div.inner_cell {
  min-width: 0;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
  /* Old browsers */
  -webkit-box-flex: 1;
  -moz-box-flex: 1;
  box-flex: 1;
  /* Modern browsers */
  flex: 1;
}
/* input_area and input_prompt must match in top border and margin for alignment */
div.input_area {
  border: 1px solid #cfcfcf;
  border-radius: 2px;
  background: #f7f7f7;
  line-height: 1.21429em;
}
/* This is needed so that empty prompt areas can collapse to zero height when there
   is no content in the output_subarea and the prompt. The main purpose of this is
   to make sure that empty JavaScript output_subareas have no height. */
div.prompt:empty {
  padding-top: 0;
  padding-bottom: 0;
}
div.unrecognized_cell {
  padding: 5px 5px 5px 0px;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
}
div.unrecognized_cell .inner_cell {
  border-radius: 2px;
  padding: 5px;
  font-weight: bold;
  color: red;
  border: 1px solid #cfcfcf;
  background: #eaeaea;
}
div.unrecognized_cell .inner_cell a {
  color: inherit;
  text-decoration: none;
}
div.unrecognized_cell .inner_cell a:hover {
  color: inherit;
  text-decoration: none;
}
@media (max-width: 540px) {
  div.unrecognized_cell > div.prompt {
    display: none;
  }
}
div.code_cell {
  /* avoid page breaking on code cells when printing */
}
@media print {
  div.code_cell {
    page-break-inside: avoid;
  }
}
/* any special styling for code cells that are currently running goes here */
div.input {
  page-break-inside: avoid;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
}
@media (max-width: 540px) {
  div.input {
    /* Old browsers */
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-box-align: stretch;
    display: -moz-box;
    -moz-box-orient: vertical;
    -moz-box-align: stretch;
    display: box;
    box-orient: vertical;
    box-align: stretch;
    /* Modern browsers */
    display: flex;
    flex-direction: column;
    align-items: stretch;
  }
}
/* input_area and input_prompt must match in top border and margin for alignment */
div.input_prompt {
  color: #303F9F;
  border-top: 1px solid transparent;
}
div.input_area > div.highlight {
  margin: 0.4em;
  border: none;
  padding: 0px;
  background-color: transparent;
}
div.input_area > div.highlight > pre {
  margin: 0px;
  border: none;
  padding: 0px;
  background-color: transparent;
}
/* The following gets added to the <head> if it is detected that the user has a
 * monospace font with inconsistent normal/bold/italic height.  See
 * notebookmain.js.  Such fonts will have keywords vertically offset with
 * respect to the rest of the text.  The user should select a better font.
 * See: https://github.com/ipython/ipython/issues/1503
 *
 * .CodeMirror span {
 *      vertical-align: bottom;
 * }
 */
.CodeMirror {
  line-height: 1.21429em;
  /* Changed from 1em to our global default */
  font-size: 14px;
  height: auto;
  /* Changed to auto to autogrow */
  background: none;
  /* Changed from white to allow our bg to show through */
}
.CodeMirror-scroll {
  /*  The CodeMirror docs are a bit fuzzy on if overflow-y should be hidden or visible.*/
  /*  We have found that if it is visible, vertical scrollbars appear with font size changes.*/
  overflow-y: hidden;
  overflow-x: auto;
}
.CodeMirror-lines {
  /* In CM2, this used to be 0.4em, but in CM3 it went to 4px. We need the em value because */
  /* we have set a different line-height and want this to scale with that. */
  /* Note that this should set vertical padding only, since CodeMirror assumes
       that horizontal padding will be set on CodeMirror pre */
  padding: 0.4em 0;
}
.CodeMirror-linenumber {
  padding: 0 8px 0 4px;
}
.CodeMirror-gutters {
  border-bottom-left-radius: 2px;
  border-top-left-radius: 2px;
}
.CodeMirror pre {
  /* In CM3 this went to 4px from 0 in CM2. This sets horizontal padding only,
    use .CodeMirror-lines for vertical */
  padding: 0 0.4em;
  border: 0;
  border-radius: 0;
}
.CodeMirror-cursor {
  border-left: 1.4px solid black;
}
@media screen and (min-width: 2138px) and (max-width: 4319px) {
  .CodeMirror-cursor {
    border-left: 2px solid black;
  }
}
@media screen and (min-width: 4320px) {
  .CodeMirror-cursor {
    border-left: 4px solid black;
  }
}
/*

Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org>
Adapted from GitHub theme

*/
.highlight-base {
  color: #000;
}
.highlight-variable {
  color: #000;
}
.highlight-variable-2 {
  color: #1a1a1a;
}
.highlight-variable-3 {
  color: #333333;
}
.highlight-string {
  color: #BA2121;
}
.highlight-comment {
  color: #408080;
  font-style: italic;
}
.highlight-number {
  color: #080;
}
.highlight-atom {
  color: #88F;
}
.highlight-keyword {
  color: #008000;
  font-weight: bold;
}
.highlight-builtin {
  color: #008000;
}
.highlight-error {
  color: #f00;
}
.highlight-operator {
  color: #AA22FF;
  font-weight: bold;
}
.highlight-meta {
  color: #AA22FF;
}
/* previously not defined, copying from default codemirror */
.highlight-def {
  color: #00f;
}
.highlight-string-2 {
  color: #f50;
}
.highlight-qualifier {
  color: #555;
}
.highlight-bracket {
  color: #997;
}
.highlight-tag {
  color: #170;
}
.highlight-attribute {
  color: #00c;
}
.highlight-header {
  color: blue;
}
.highlight-quote {
  color: #090;
}
.highlight-link {
  color: #00c;
}
/* apply the same style to codemirror */
.cm-s-ipython span.cm-keyword {
  color: #008000;
  font-weight: bold;
}
.cm-s-ipython span.cm-atom {
  color: #88F;
}
.cm-s-ipython span.cm-number {
  color: #080;
}
.cm-s-ipython span.cm-def {
  color: #00f;
}
.cm-s-ipython span.cm-variable {
  color: #000;
}
.cm-s-ipython span.cm-operator {
  color: #AA22FF;
  font-weight: bold;
}
.cm-s-ipython span.cm-variable-2 {
  color: #1a1a1a;
}
.cm-s-ipython span.cm-variable-3 {
  color: #333333;
}
.cm-s-ipython span.cm-comment {
  color: #408080;
  font-style: italic;
}
.cm-s-ipython span.cm-string {
  color: #BA2121;
}
.cm-s-ipython span.cm-string-2 {
  color: #f50;
}
.cm-s-ipython span.cm-meta {
  color: #AA22FF;
}
.cm-s-ipython span.cm-qualifier {
  color: #555;
}
.cm-s-ipython span.cm-builtin {
  color: #008000;
}
.cm-s-ipython span.cm-bracket {
  color: #997;
}
.cm-s-ipython span.cm-tag {
  color: #170;
}
.cm-s-ipython span.cm-attribute {
  color: #00c;
}
.cm-s-ipython span.cm-header {
  color: blue;
}
.cm-s-ipython span.cm-quote {
  color: #090;
}
.cm-s-ipython span.cm-link {
  color: #00c;
}
.cm-s-ipython span.cm-error {
  color: #f00;
}
.cm-s-ipython span.cm-tab {
  background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAMCAYAAAAkuj5RAAAAAXNSR0IArs4c6QAAAGFJREFUSMft1LsRQFAQheHPowAKoACx3IgEKtaEHujDjORSgWTH/ZOdnZOcM/sgk/kFFWY0qV8foQwS4MKBCS3qR6ixBJvElOobYAtivseIE120FaowJPN75GMu8j/LfMwNjh4HUpwg4LUAAAAASUVORK5CYII=);
  background-position: right;
  background-repeat: no-repeat;
}
div.output_wrapper {
  /* this position must be relative to enable descendents to be absolute within it */
  position: relative;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
  z-index: 1;
}
/* class for the output area when it should be height-limited */
div.output_scroll {
  /* ideally, this would be max-height, but FF barfs all over that */
  height: 24em;
  /* FF needs this *and the wrapper* to specify full width, or it will shrinkwrap */
  width: 100%;
  overflow: auto;
  border-radius: 2px;
  -webkit-box-shadow: inset 0 2px 8px rgba(0, 0, 0, 0.8);
  box-shadow: inset 0 2px 8px rgba(0, 0, 0, 0.8);
  display: block;
}
/* output div while it is collapsed */
div.output_collapsed {
  margin: 0px;
  padding: 0px;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
}
div.out_prompt_overlay {
  height: 100%;
  padding: 0px 0.4em;
  position: absolute;
  border-radius: 2px;
}
div.out_prompt_overlay:hover {
  /* use inner shadow to get border that is computed the same on WebKit/FF */
  -webkit-box-shadow: inset 0 0 1px #000;
  box-shadow: inset 0 0 1px #000;
  background: rgba(240, 240, 240, 0.5);
}
div.output_prompt {
  color: #D84315;
}
/* This class is the outer container of all output sections. */
div.output_area {
  padding: 0px;
  page-break-inside: avoid;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
}
div.output_area .MathJax_Display {
  text-align: left !important;
}
div.output_area .rendered_html table {
  margin-left: 0;
  margin-right: 0;
}
div.output_area .rendered_html img {
  margin-left: 0;
  margin-right: 0;
}
div.output_area img,
div.output_area svg {
  max-width: 100%;
  height: auto;
}
div.output_area img.unconfined,
div.output_area svg.unconfined {
  max-width: none;
}
div.output_area .mglyph > img {
  max-width: none;
}
/* This is needed to protect the pre formating from global settings such
   as that of bootstrap */
.output {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: vertical;
  -moz-box-align: stretch;
  display: box;
  box-orient: vertical;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: column;
  align-items: stretch;
}
@media (max-width: 540px) {
  div.output_area {
    /* Old browsers */
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-box-align: stretch;
    display: -moz-box;
    -moz-box-orient: vertical;
    -moz-box-align: stretch;
    display: box;
    box-orient: vertical;
    box-align: stretch;
    /* Modern browsers */
    display: flex;
    flex-direction: column;
    align-items: stretch;
  }
}
div.output_area pre {
  margin: 0;
  padding: 1px 0 1px 0;
  border: 0;
  vertical-align: baseline;
  color: black;
  background-color: transparent;
  border-radius: 0;
}
/* This class is for the output subarea inside the output_area and after
   the prompt div. */
div.output_subarea {
  overflow-x: auto;
  padding: 0.4em;
  /* Old browsers */
  -webkit-box-flex: 1;
  -moz-box-flex: 1;
  box-flex: 1;
  /* Modern browsers */
  flex: 1;
  max-width: calc(100% - 14ex);
}
div.output_scroll div.output_subarea {
  overflow-x: visible;
}
/* The rest of the output_* classes are for special styling of the different
   output types */
/* all text output has this class: */
div.output_text {
  text-align: left;
  color: #000;
  /* This has to match that of the the CodeMirror class line-height below */
  line-height: 1.21429em;
}
/* stdout/stderr are 'text' as well as 'stream', but execute_result/error are *not* streams */
div.output_stderr {
  background: #fdd;
  /* very light red background for stderr */
}
div.output_latex {
  text-align: left;
}
/* Empty output_javascript divs should have no height */
div.output_javascript:empty {
  padding: 0;
}
.js-error {
  color: darkred;
}
/* raw_input styles */
div.raw_input_container {
  line-height: 1.21429em;
  padding-top: 5px;
}
pre.raw_input_prompt {
  /* nothing needed here. */
}
input.raw_input {
  font-family: monospace;
  font-size: inherit;
  color: inherit;
  width: auto;
  /* make sure input baseline aligns with prompt */
  vertical-align: baseline;
  /* padding + margin = 0.5em between prompt and cursor */
  padding: 0em 0.25em;
  margin: 0em 0.25em;
}
input.raw_input:focus {
  box-shadow: none;
}
p.p-space {
  margin-bottom: 10px;
}
div.output_unrecognized {
  padding: 5px;
  font-weight: bold;
  color: red;
}
div.output_unrecognized a {
  color: inherit;
  text-decoration: none;
}
div.output_unrecognized a:hover {
  color: inherit;
  text-decoration: none;
}
.rendered_html {
  color: #000;
  /* any extras will just be numbers: */
}
.rendered_html em {
  font-style: italic;
}
.rendered_html strong {
  font-weight: bold;
}
.rendered_html u {
  text-decoration: underline;
}
.rendered_html :link {
  text-decoration: underline;
}
.rendered_html :visited {
  text-decoration: underline;
}
.rendered_html h1 {
  font-size: 185.7%;
  margin: 1.08em 0 0 0;
  font-weight: bold;
  line-height: 1.0;
}
.rendered_html h2 {
  font-size: 157.1%;
  margin: 1.27em 0 0 0;
  font-weight: bold;
  line-height: 1.0;
}
.rendered_html h3 {
  font-size: 128.6%;
  margin: 1.55em 0 0 0;
  font-weight: bold;
  line-height: 1.0;
}
.rendered_html h4 {
  font-size: 100%;
  margin: 2em 0 0 0;
  font-weight: bold;
  line-height: 1.0;
}
.rendered_html h5 {
  font-size: 100%;
  margin: 2em 0 0 0;
  font-weight: bold;
  line-height: 1.0;
  font-style: italic;
}
.rendered_html h6 {
  font-size: 100%;
  margin: 2em 0 0 0;
  font-weight: bold;
  line-height: 1.0;
  font-style: italic;
}
.rendered_html h1:first-child {
  margin-top: 0.538em;
}
.rendered_html h2:first-child {
  margin-top: 0.636em;
}
.rendered_html h3:first-child {
  margin-top: 0.777em;
}
.rendered_html h4:first-child {
  margin-top: 1em;
}
.rendered_html h5:first-child {
  margin-top: 1em;
}
.rendered_html h6:first-child {
  margin-top: 1em;
}
.rendered_html ul:not(.list-inline),
.rendered_html ol:not(.list-inline) {
  padding-left: 2em;
}
.rendered_html ul {
  list-style: disc;
}
.rendered_html ul ul {
  list-style: square;
  margin-top: 0;
}
.rendered_html ul ul ul {
  list-style: circle;
}
.rendered_html ol {
  list-style: decimal;
}
.rendered_html ol ol {
  list-style: upper-alpha;
  margin-top: 0;
}
.rendered_html ol ol ol {
  list-style: lower-alpha;
}
.rendered_html ol ol ol ol {
  list-style: lower-roman;
}
.rendered_html ol ol ol ol ol {
  list-style: decimal;
}
.rendered_html * + ul {
  margin-top: 1em;
}
.rendered_html * + ol {
  margin-top: 1em;
}
.rendered_html hr {
  color: black;
  background-color: black;
}
.rendered_html pre {
  margin: 1em 2em;
  padding: 0px;
  background-color: #fff;
}
.rendered_html code {
  background-color: #eff0f1;
}
.rendered_html p code {
  padding: 1px 5px;
}
.rendered_html pre code {
  background-color: #fff;
}
.rendered_html pre,
.rendered_html code {
  border: 0;
  color: #000;
  font-size: 100%;
}
.rendered_html blockquote {
  margin: 1em 2em;
}
.rendered_html table {
  margin-left: auto;
  margin-right: auto;
  border: none;
  border-collapse: collapse;
  border-spacing: 0;
  color: black;
  font-size: 12px;
  table-layout: fixed;
}
.rendered_html thead {
  border-bottom: 1px solid black;
  vertical-align: bottom;
}
.rendered_html tr,
.rendered_html th,
.rendered_html td {
  text-align: right;
  vertical-align: middle;
  padding: 0.5em 0.5em;
  line-height: normal;
  white-space: normal;
  max-width: none;
  border: none;
}
.rendered_html th {
  font-weight: bold;
}
.rendered_html tbody tr:nth-child(odd) {
  background: #f5f5f5;
}
.rendered_html tbody tr:hover {
  background: rgba(66, 165, 245, 0.2);
}
.rendered_html * + table {
  margin-top: 1em;
}
.rendered_html p {
  text-align: left;
}
.rendered_html * + p {
  margin-top: 1em;
}
.rendered_html img {
  display: block;
  margin-left: auto;
  margin-right: auto;
}
.rendered_html * + img {
  margin-top: 1em;
}
.rendered_html img,
.rendered_html svg {
  max-width: 100%;
  height: auto;
}
.rendered_html img.unconfined,
.rendered_html svg.unconfined {
  max-width: none;
}
.rendered_html .alert {
  margin-bottom: initial;
}
.rendered_html * + .alert {
  margin-top: 1em;
}
[dir="rtl"] .rendered_html p {
  text-align: right;
}
div.text_cell {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
}
@media (max-width: 540px) {
  div.text_cell > div.prompt {
    display: none;
  }
}
div.text_cell_render {
  /*font-family: "Helvetica Neue", Arial, Helvetica, Geneva, sans-serif;*/
  outline: none;
  resize: none;
  width: inherit;
  border-style: none;
  padding: 0.5em 0.5em 0.5em 0.4em;
  color: #000;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
}
a.anchor-link:link {
  text-decoration: none;
  padding: 0px 20px;
  visibility: hidden;
}
h1:hover .anchor-link,
h2:hover .anchor-link,
h3:hover .anchor-link,
h4:hover .anchor-link,
h5:hover .anchor-link,
h6:hover .anchor-link {
  visibility: visible;
}
.text_cell.rendered .input_area {
  display: none;
}
.text_cell.rendered .rendered_html {
  overflow-x: auto;
  overflow-y: hidden;
}
.text_cell.rendered .rendered_html tr,
.text_cell.rendered .rendered_html th,
.text_cell.rendered .rendered_html td {
  max-width: none;
}
.text_cell.unrendered .text_cell_render {
  display: none;
}
.text_cell .dropzone .input_area {
  border: 2px dashed #bababa;
  margin: -1px;
}
.cm-header-1,
.cm-header-2,
.cm-header-3,
.cm-header-4,
.cm-header-5,
.cm-header-6 {
  font-weight: bold;
  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
}
.cm-header-1 {
  font-size: 185.7%;
}
.cm-header-2 {
  font-size: 157.1%;
}
.cm-header-3 {
  font-size: 128.6%;
}
.cm-header-4 {
  font-size: 110%;
}
.cm-header-5 {
  font-size: 100%;
  font-style: italic;
}
.cm-header-6 {
  font-size: 100%;
  font-style: italic;
}
/*!
*
* IPython notebook webapp
*
*/
@media (max-width: 767px) {
  .notebook_app {
    padding-left: 0px;
    padding-right: 0px;
  }
}
#ipython-main-app {
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  height: 100%;
}
div#notebook_panel {
  margin: 0px;
  padding: 0px;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  height: 100%;
}
div#notebook {
  font-size: 14px;
  line-height: 20px;
  overflow-y: hidden;
  overflow-x: auto;
  width: 100%;
  /* This spaces the page away from the edge of the notebook area */
  padding-top: 20px;
  margin: 0px;
  outline: none;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  min-height: 100%;
}
@media not print {
  #notebook-container {
    padding: 15px;
    background-color: #fff;
    min-height: 0;
    -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
    box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  }
}
@media print {
  #notebook-container {
    width: 100%;
  }
}
div.ui-widget-content {
  border: 1px solid #ababab;
  outline: none;
}
pre.dialog {
  background-color: #f7f7f7;
  border: 1px solid #ddd;
  border-radius: 2px;
  padding: 0.4em;
  padding-left: 2em;
}
p.dialog {
  padding: 0.2em;
}
/* Word-wrap output correctly.  This is the CSS3 spelling, though Firefox seems
   to not honor it correctly.  Webkit browsers (Chrome, rekonq, Safari) do.
 */
pre,
code,
kbd,
samp {
  white-space: pre-wrap;
}
#fonttest {
  font-family: monospace;
}
p {
  margin-bottom: 0;
}
.end_space {
  min-height: 100px;
  transition: height .2s ease;
}
.notebook_app > #header {
  -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
}
@media not print {
  .notebook_app {
    background-color: #EEE;
  }
}
kbd {
  border-style: solid;
  border-width: 1px;
  box-shadow: none;
  margin: 2px;
  padding-left: 2px;
  padding-right: 2px;
  padding-top: 1px;
  padding-bottom: 1px;
}
.jupyter-keybindings {
  padding: 1px;
  line-height: 24px;
  border-bottom: 1px solid gray;
}
.jupyter-keybindings input {
  margin: 0;
  padding: 0;
  border: none;
}
.jupyter-keybindings i {
  padding: 6px;
}
.well code {
  background-color: #ffffff;
  border-color: #ababab;
  border-width: 1px;
  border-style: solid;
  padding: 2px;
  padding-top: 1px;
  padding-bottom: 1px;
}
/* CSS for the cell toolbar */
.celltoolbar {
  border: thin solid #CFCFCF;
  border-bottom: none;
  background: #EEE;
  border-radius: 2px 2px 0px 0px;
  width: 100%;
  height: 29px;
  padding-right: 4px;
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
  /* Old browsers */
  -webkit-box-pack: end;
  -moz-box-pack: end;
  box-pack: end;
  /* Modern browsers */
  justify-content: flex-end;
  display: -webkit-flex;
}
@media print {
  .celltoolbar {
    display: none;
  }
}
.ctb_hideshow {
  display: none;
  vertical-align: bottom;
}
/* ctb_show is added to the ctb_hideshow div to show the cell toolbar.
   Cell toolbars are only shown when the ctb_global_show class is also set.
*/
.ctb_global_show .ctb_show.ctb_hideshow {
  display: block;
}
.ctb_global_show .ctb_show + .input_area,
.ctb_global_show .ctb_show + div.text_cell_input,
.ctb_global_show .ctb_show ~ div.text_cell_render {
  border-top-right-radius: 0px;
  border-top-left-radius: 0px;
}
.ctb_global_show .ctb_show ~ div.text_cell_render {
  border: 1px solid #cfcfcf;
}
.celltoolbar {
  font-size: 87%;
  padding-top: 3px;
}
.celltoolbar select {
  display: block;
  width: 100%;
  height: 32px;
  padding: 6px 12px;
  font-size: 13px;
  line-height: 1.42857143;
  color: #555555;
  background-color: #fff;
  background-image: none;
  border: 1px solid #ccc;
  border-radius: 2px;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  -webkit-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  -o-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  height: 30px;
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
  width: inherit;
  font-size: inherit;
  height: 22px;
  padding: 0px;
  display: inline-block;
}
.celltoolbar select:focus {
  border-color: #66afe9;
  outline: 0;
  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);
  box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);
}
.celltoolbar select::-moz-placeholder {
  color: #999;
  opacity: 1;
}
.celltoolbar select:-ms-input-placeholder {
  color: #999;
}
.celltoolbar select::-webkit-input-placeholder {
  color: #999;
}
.celltoolbar select::-ms-expand {
  border: 0;
  background-color: transparent;
}
.celltoolbar select[disabled],
.celltoolbar select[readonly],
fieldset[disabled] .celltoolbar select {
  background-color: #eeeeee;
  opacity: 1;
}
.celltoolbar select[disabled],
fieldset[disabled] .celltoolbar select {
  cursor: not-allowed;
}
textarea.celltoolbar select {
  height: auto;
}
select.celltoolbar select {
  height: 30px;
  line-height: 30px;
}
textarea.celltoolbar select,
select[multiple].celltoolbar select {
  height: auto;
}
.celltoolbar label {
  margin-left: 5px;
  margin-right: 5px;
}
.tags_button_container {
  width: 100%;
  display: flex;
}
.tag-container {
  display: flex;
  flex-direction: row;
  flex-grow: 1;
  overflow: hidden;
  position: relative;
}
.tag-container > * {
  margin: 0 4px;
}
.remove-tag-btn {
  margin-left: 4px;
}
.tags-input {
  display: flex;
}
.cell-tag:last-child:after {
  content: "";
  position: absolute;
  right: 0;
  width: 40px;
  height: 100%;
  /* Fade to background color of cell toolbar */
  background: linear-gradient(to right, rgba(0, 0, 0, 0), #EEE);
}
.tags-input > * {
  margin-left: 4px;
}
.cell-tag,
.tags-input input,
.tags-input button {
  display: block;
  width: 100%;
  height: 32px;
  padding: 6px 12px;
  font-size: 13px;
  line-height: 1.42857143;
  color: #555555;
  background-color: #fff;
  background-image: none;
  border: 1px solid #ccc;
  border-radius: 2px;
  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
  -webkit-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  -o-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
  height: 30px;
  padding: 5px 10px;
  font-size: 12px;
  line-height: 1.5;
  border-radius: 1px;
  box-shadow: none;
  width: inherit;
  font-size: inherit;
  height: 22px;
  line-height: 22px;
  padding: 0px 4px;
  display: inline-block;
}
.cell-tag:focus,
.tags-input input:focus,
.tags-input button:focus {
  border-color: #66afe9;
  outline: 0;
  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);
  box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);
}
.cell-tag::-moz-placeholder,
.tags-input input::-moz-placeholder,
.tags-input button::-moz-placeholder {
  color: #999;
  opacity: 1;
}
.cell-tag:-ms-input-placeholder,
.tags-input input:-ms-input-placeholder,
.tags-input button:-ms-input-placeholder {
  color: #999;
}
.cell-tag::-webkit-input-placeholder,
.tags-input input::-webkit-input-placeholder,
.tags-input button::-webkit-input-placeholder {
  color: #999;
}
.cell-tag::-ms-expand,
.tags-input input::-ms-expand,
.tags-input button::-ms-expand {
  border: 0;
  background-color: transparent;
}
.cell-tag[disabled],
.tags-input input[disabled],
.tags-input button[disabled],
.cell-tag[readonly],
.tags-input input[readonly],
.tags-input button[readonly],
fieldset[disabled] .cell-tag,
fieldset[disabled] .tags-input input,
fieldset[disabled] .tags-input button {
  background-color: #eeeeee;
  opacity: 1;
}
.cell-tag[disabled],
.tags-input input[disabled],
.tags-input button[disabled],
fieldset[disabled] .cell-tag,
fieldset[disabled] .tags-input input,
fieldset[disabled] .tags-input button {
  cursor: not-allowed;
}
textarea.cell-tag,
textarea.tags-input input,
textarea.tags-input button {
  height: auto;
}
select.cell-tag,
select.tags-input input,
select.tags-input button {
  height: 30px;
  line-height: 30px;
}
textarea.cell-tag,
textarea.tags-input input,
textarea.tags-input button,
select[multiple].cell-tag,
select[multiple].tags-input input,
select[multiple].tags-input button {
  height: auto;
}
.cell-tag,
.tags-input button {
  padding: 0px 4px;
}
.cell-tag {
  background-color: #fff;
  white-space: nowrap;
}
.tags-input input[type=text]:focus {
  outline: none;
  box-shadow: none;
  border-color: #ccc;
}
.completions {
  position: absolute;
  z-index: 110;
  overflow: hidden;
  border: 1px solid #ababab;
  border-radius: 2px;
  -webkit-box-shadow: 0px 6px 10px -1px #adadad;
  box-shadow: 0px 6px 10px -1px #adadad;
  line-height: 1;
}
.completions select {
  background: white;
  outline: none;
  border: none;
  padding: 0px;
  margin: 0px;
  overflow: auto;
  font-family: monospace;
  font-size: 110%;
  color: #000;
  width: auto;
}
.completions select option.context {
  color: #286090;
}
#kernel_logo_widget .current_kernel_logo {
  display: none;
  margin-top: -1px;
  margin-bottom: -1px;
  width: 32px;
  height: 32px;
}
[dir="rtl"] #kernel_logo_widget {
  float: left !important;
  float: left;
}
.modal .modal-body .move-path {
  display: flex;
  flex-direction: row;
  justify-content: space;
  align-items: center;
}
.modal .modal-body .move-path .server-root {
  padding-right: 20px;
}
.modal .modal-body .move-path .path-input {
  flex: 1;
}
#menubar {
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  margin-top: 1px;
}
#menubar .navbar {
  border-top: 1px;
  border-radius: 0px 0px 2px 2px;
  margin-bottom: 0px;
}
#menubar .navbar-toggle {
  float: left;
  padding-top: 7px;
  padding-bottom: 7px;
  border: none;
}
#menubar .navbar-collapse {
  clear: left;
}
[dir="rtl"] #menubar .navbar-toggle {
  float: right;
}
[dir="rtl"] #menubar .navbar-collapse {
  clear: right;
}
[dir="rtl"] #menubar .navbar-nav {
  float: right;
}
[dir="rtl"] #menubar .nav {
  padding-right: 0px;
}
[dir="rtl"] #menubar .navbar-nav > li {
  float: right;
}
[dir="rtl"] #menubar .navbar-right {
  float: left !important;
}
[dir="rtl"] ul.dropdown-menu {
  text-align: right;
  left: auto;
}
[dir="rtl"] ul#new-menu.dropdown-menu {
  right: auto;
  left: 0;
}
.nav-wrapper {
  border-bottom: 1px solid #e7e7e7;
}
i.menu-icon {
  padding-top: 4px;
}
[dir="rtl"] i.menu-icon.pull-right {
  float: left !important;
  float: left;
}
ul#help_menu li a {
  overflow: hidden;
  padding-right: 2.2em;
}
ul#help_menu li a i {
  margin-right: -1.2em;
}
[dir="rtl"] ul#help_menu li a {
  padding-left: 2.2em;
}
[dir="rtl"] ul#help_menu li a i {
  margin-right: 0;
  margin-left: -1.2em;
}
[dir="rtl"] ul#help_menu li a i.pull-right {
  float: left !important;
  float: left;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu > .dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
}
[dir="rtl"] .dropdown-submenu > .dropdown-menu {
  right: 100%;
  margin-right: -1px;
}
.dropdown-submenu:hover > .dropdown-menu {
  display: block;
}
.dropdown-submenu > a:after {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  display: block;
  content: "\f0da";
  float: right;
  color: #333333;
  margin-top: 2px;
  margin-right: -10px;
}
.dropdown-submenu > a:after.fa-pull-left {
  margin-right: .3em;
}
.dropdown-submenu > a:after.fa-pull-right {
  margin-left: .3em;
}
.dropdown-submenu > a:after.pull-left {
  margin-right: .3em;
}
.dropdown-submenu > a:after.pull-right {
  margin-left: .3em;
}
[dir="rtl"] .dropdown-submenu > a:after {
  float: left;
  content: "\f0d9";
  margin-right: 0;
  margin-left: -10px;
}
.dropdown-submenu:hover > a:after {
  color: #262626;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left > .dropdown-menu {
  left: -100%;
  margin-left: 10px;
}
#notification_area {
  float: right !important;
  float: right;
  z-index: 10;
}
[dir="rtl"] #notification_area {
  float: left !important;
  float: left;
}
.indicator_area {
  float: right !important;
  float: right;
  color: #777;
  margin-left: 5px;
  margin-right: 5px;
  width: 11px;
  z-index: 10;
  text-align: center;
  width: auto;
}
[dir="rtl"] .indicator_area {
  float: left !important;
  float: left;
}
#kernel_indicator {
  float: right !important;
  float: right;
  color: #777;
  margin-left: 5px;
  margin-right: 5px;
  width: 11px;
  z-index: 10;
  text-align: center;
  width: auto;
  border-left: 1px solid;
}
#kernel_indicator .kernel_indicator_name {
  padding-left: 5px;
  padding-right: 5px;
}
[dir="rtl"] #kernel_indicator {
  float: left !important;
  float: left;
  border-left: 0;
  border-right: 1px solid;
}
#modal_indicator {
  float: right !important;
  float: right;
  color: #777;
  margin-left: 5px;
  margin-right: 5px;
  width: 11px;
  z-index: 10;
  text-align: center;
  width: auto;
}
[dir="rtl"] #modal_indicator {
  float: left !important;
  float: left;
}
#readonly-indicator {
  float: right !important;
  float: right;
  color: #777;
  margin-left: 5px;
  margin-right: 5px;
  width: 11px;
  z-index: 10;
  text-align: center;
  width: auto;
  margin-top: 2px;
  margin-bottom: 0px;
  margin-left: 0px;
  margin-right: 0px;
  display: none;
}
.modal_indicator:before {
  width: 1.28571429em;
  text-align: center;
}
.edit_mode .modal_indicator:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f040";
}
.edit_mode .modal_indicator:before.fa-pull-left {
  margin-right: .3em;
}
.edit_mode .modal_indicator:before.fa-pull-right {
  margin-left: .3em;
}
.edit_mode .modal_indicator:before.pull-left {
  margin-right: .3em;
}
.edit_mode .modal_indicator:before.pull-right {
  margin-left: .3em;
}
.command_mode .modal_indicator:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: ' ';
}
.command_mode .modal_indicator:before.fa-pull-left {
  margin-right: .3em;
}
.command_mode .modal_indicator:before.fa-pull-right {
  margin-left: .3em;
}
.command_mode .modal_indicator:before.pull-left {
  margin-right: .3em;
}
.command_mode .modal_indicator:before.pull-right {
  margin-left: .3em;
}
.kernel_idle_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f10c";
}
.kernel_idle_icon:before.fa-pull-left {
  margin-right: .3em;
}
.kernel_idle_icon:before.fa-pull-right {
  margin-left: .3em;
}
.kernel_idle_icon:before.pull-left {
  margin-right: .3em;
}
.kernel_idle_icon:before.pull-right {
  margin-left: .3em;
}
.kernel_busy_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f111";
}
.kernel_busy_icon:before.fa-pull-left {
  margin-right: .3em;
}
.kernel_busy_icon:before.fa-pull-right {
  margin-left: .3em;
}
.kernel_busy_icon:before.pull-left {
  margin-right: .3em;
}
.kernel_busy_icon:before.pull-right {
  margin-left: .3em;
}
.kernel_dead_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f1e2";
}
.kernel_dead_icon:before.fa-pull-left {
  margin-right: .3em;
}
.kernel_dead_icon:before.fa-pull-right {
  margin-left: .3em;
}
.kernel_dead_icon:before.pull-left {
  margin-right: .3em;
}
.kernel_dead_icon:before.pull-right {
  margin-left: .3em;
}
.kernel_disconnected_icon:before {
  display: inline-block;
  font: normal normal normal 14px/1 FontAwesome;
  font-size: inherit;
  text-rendering: auto;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  content: "\f127";
}
.kernel_disconnected_icon:before.fa-pull-left {
  margin-right: .3em;
}
.kernel_disconnected_icon:before.fa-pull-right {
  margin-left: .3em;
}
.kernel_disconnected_icon:before.pull-left {
  margin-right: .3em;
}
.kernel_disconnected_icon:before.pull-right {
  margin-left: .3em;
}
.notification_widget {
  color: #777;
  z-index: 10;
  background: rgba(240, 240, 240, 0.5);
  margin-right: 4px;
  color: #333;
  background-color: #fff;
  border-color: #ccc;
}
.notification_widget:focus,
.notification_widget.focus {
  color: #333;
  background-color: #e6e6e6;
  border-color: #8c8c8c;
}
.notification_widget:hover {
  color: #333;
  background-color: #e6e6e6;
  border-color: #adadad;
}
.notification_widget:active,
.notification_widget.active,
.open > .dropdown-toggle.notification_widget {
  color: #333;
  background-color: #e6e6e6;
  border-color: #adadad;
}
.notification_widget:active:hover,
.notification_widget.active:hover,
.open > .dropdown-toggle.notification_widget:hover,
.notification_widget:active:focus,
.notification_widget.active:focus,
.open > .dropdown-toggle.notification_widget:focus,
.notification_widget:active.focus,
.notification_widget.active.focus,
.open > .dropdown-toggle.notification_widget.focus {
  color: #333;
  background-color: #d4d4d4;
  border-color: #8c8c8c;
}
.notification_widget:active,
.notification_widget.active,
.open > .dropdown-toggle.notification_widget {
  background-image: none;
}
.notification_widget.disabled:hover,
.notification_widget[disabled]:hover,
fieldset[disabled] .notification_widget:hover,
.notification_widget.disabled:focus,
.notification_widget[disabled]:focus,
fieldset[disabled] .notification_widget:focus,
.notification_widget.disabled.focus,
.notification_widget[disabled].focus,
fieldset[disabled] .notification_widget.focus {
  background-color: #fff;
  border-color: #ccc;
}
.notification_widget .badge {
  color: #fff;
  background-color: #333;
}
.notification_widget.warning {
  color: #fff;
  background-color: #f0ad4e;
  border-color: #eea236;
}
.notification_widget.warning:focus,
.notification_widget.warning.focus {
  color: #fff;
  background-color: #ec971f;
  border-color: #985f0d;
}
.notification_widget.warning:hover {
  color: #fff;
  background-color: #ec971f;
  border-color: #d58512;
}
.notification_widget.warning:active,
.notification_widget.warning.active,
.open > .dropdown-toggle.notification_widget.warning {
  color: #fff;
  background-color: #ec971f;
  border-color: #d58512;
}
.notification_widget.warning:active:hover,
.notification_widget.warning.active:hover,
.open > .dropdown-toggle.notification_widget.warning:hover,
.notification_widget.warning:active:focus,
.notification_widget.warning.active:focus,
.open > .dropdown-toggle.notification_widget.warning:focus,
.notification_widget.warning:active.focus,
.notification_widget.warning.active.focus,
.open > .dropdown-toggle.notification_widget.warning.focus {
  color: #fff;
  background-color: #d58512;
  border-color: #985f0d;
}
.notification_widget.warning:active,
.notification_widget.warning.active,
.open > .dropdown-toggle.notification_widget.warning {
  background-image: none;
}
.notification_widget.warning.disabled:hover,
.notification_widget.warning[disabled]:hover,
fieldset[disabled] .notification_widget.warning:hover,
.notification_widget.warning.disabled:focus,
.notification_widget.warning[disabled]:focus,
fieldset[disabled] .notification_widget.warning:focus,
.notification_widget.warning.disabled.focus,
.notification_widget.warning[disabled].focus,
fieldset[disabled] .notification_widget.warning.focus {
  background-color: #f0ad4e;
  border-color: #eea236;
}
.notification_widget.warning .badge {
  color: #f0ad4e;
  background-color: #fff;
}
.notification_widget.success {
  color: #fff;
  background-color: #5cb85c;
  border-color: #4cae4c;
}
.notification_widget.success:focus,
.notification_widget.success.focus {
  color: #fff;
  background-color: #449d44;
  border-color: #255625;
}
.notification_widget.success:hover {
  color: #fff;
  background-color: #449d44;
  border-color: #398439;
}
.notification_widget.success:active,
.notification_widget.success.active,
.open > .dropdown-toggle.notification_widget.success {
  color: #fff;
  background-color: #449d44;
  border-color: #398439;
}
.notification_widget.success:active:hover,
.notification_widget.success.active:hover,
.open > .dropdown-toggle.notification_widget.success:hover,
.notification_widget.success:active:focus,
.notification_widget.success.active:focus,
.open > .dropdown-toggle.notification_widget.success:focus,
.notification_widget.success:active.focus,
.notification_widget.success.active.focus,
.open > .dropdown-toggle.notification_widget.success.focus {
  color: #fff;
  background-color: #398439;
  border-color: #255625;
}
.notification_widget.success:active,
.notification_widget.success.active,
.open > .dropdown-toggle.notification_widget.success {
  background-image: none;
}
.notification_widget.success.disabled:hover,
.notification_widget.success[disabled]:hover,
fieldset[disabled] .notification_widget.success:hover,
.notification_widget.success.disabled:focus,
.notification_widget.success[disabled]:focus,
fieldset[disabled] .notification_widget.success:focus,
.notification_widget.success.disabled.focus,
.notification_widget.success[disabled].focus,
fieldset[disabled] .notification_widget.success.focus {
  background-color: #5cb85c;
  border-color: #4cae4c;
}
.notification_widget.success .badge {
  color: #5cb85c;
  background-color: #fff;
}
.notification_widget.info {
  color: #fff;
  background-color: #5bc0de;
  border-color: #46b8da;
}
.notification_widget.info:focus,
.notification_widget.info.focus {
  color: #fff;
  background-color: #31b0d5;
  border-color: #1b6d85;
}
.notification_widget.info:hover {
  color: #fff;
  background-color: #31b0d5;
  border-color: #269abc;
}
.notification_widget.info:active,
.notification_widget.info.active,
.open > .dropdown-toggle.notification_widget.info {
  color: #fff;
  background-color: #31b0d5;
  border-color: #269abc;
}
.notification_widget.info:active:hover,
.notification_widget.info.active:hover,
.open > .dropdown-toggle.notification_widget.info:hover,
.notification_widget.info:active:focus,
.notification_widget.info.active:focus,
.open > .dropdown-toggle.notification_widget.info:focus,
.notification_widget.info:active.focus,
.notification_widget.info.active.focus,
.open > .dropdown-toggle.notification_widget.info.focus {
  color: #fff;
  background-color: #269abc;
  border-color: #1b6d85;
}
.notification_widget.info:active,
.notification_widget.info.active,
.open > .dropdown-toggle.notification_widget.info {
  background-image: none;
}
.notification_widget.info.disabled:hover,
.notification_widget.info[disabled]:hover,
fieldset[disabled] .notification_widget.info:hover,
.notification_widget.info.disabled:focus,
.notification_widget.info[disabled]:focus,
fieldset[disabled] .notification_widget.info:focus,
.notification_widget.info.disabled.focus,
.notification_widget.info[disabled].focus,
fieldset[disabled] .notification_widget.info.focus {
  background-color: #5bc0de;
  border-color: #46b8da;
}
.notification_widget.info .badge {
  color: #5bc0de;
  background-color: #fff;
}
.notification_widget.danger {
  color: #fff;
  background-color: #d9534f;
  border-color: #d43f3a;
}
.notification_widget.danger:focus,
.notification_widget.danger.focus {
  color: #fff;
  background-color: #c9302c;
  border-color: #761c19;
}
.notification_widget.danger:hover {
  color: #fff;
  background-color: #c9302c;
  border-color: #ac2925;
}
.notification_widget.danger:active,
.notification_widget.danger.active,
.open > .dropdown-toggle.notification_widget.danger {
  color: #fff;
  background-color: #c9302c;
  border-color: #ac2925;
}
.notification_widget.danger:active:hover,
.notification_widget.danger.active:hover,
.open > .dropdown-toggle.notification_widget.danger:hover,
.notification_widget.danger:active:focus,
.notification_widget.danger.active:focus,
.open > .dropdown-toggle.notification_widget.danger:focus,
.notification_widget.danger:active.focus,
.notification_widget.danger.active.focus,
.open > .dropdown-toggle.notification_widget.danger.focus {
  color: #fff;
  background-color: #ac2925;
  border-color: #761c19;
}
.notification_widget.danger:active,
.notification_widget.danger.active,
.open > .dropdown-toggle.notification_widget.danger {
  background-image: none;
}
.notification_widget.danger.disabled:hover,
.notification_widget.danger[disabled]:hover,
fieldset[disabled] .notification_widget.danger:hover,
.notification_widget.danger.disabled:focus,
.notification_widget.danger[disabled]:focus,
fieldset[disabled] .notification_widget.danger:focus,
.notification_widget.danger.disabled.focus,
.notification_widget.danger[disabled].focus,
fieldset[disabled] .notification_widget.danger.focus {
  background-color: #d9534f;
  border-color: #d43f3a;
}
.notification_widget.danger .badge {
  color: #d9534f;
  background-color: #fff;
}
div#pager {
  background-color: #fff;
  font-size: 14px;
  line-height: 20px;
  overflow: hidden;
  display: none;
  position: fixed;
  bottom: 0px;
  width: 100%;
  max-height: 50%;
  padding-top: 8px;
  -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  /* Display over codemirror */
  z-index: 100;
  /* Hack which prevents jquery ui resizable from changing top. */
  top: auto !important;
}
div#pager pre {
  line-height: 1.21429em;
  color: #000;
  background-color: #f7f7f7;
  padding: 0.4em;
}
div#pager #pager-button-area {
  position: absolute;
  top: 8px;
  right: 20px;
}
div#pager #pager-contents {
  position: relative;
  overflow: auto;
  width: 100%;
  height: 100%;
}
div#pager #pager-contents #pager-container {
  position: relative;
  padding: 15px 0px;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
}
div#pager .ui-resizable-handle {
  top: 0px;
  height: 8px;
  background: #f7f7f7;
  border-top: 1px solid #cfcfcf;
  border-bottom: 1px solid #cfcfcf;
  /* This injects handle bars (a short, wide = symbol) for 
        the resize handle. */
}
div#pager .ui-resizable-handle::after {
  content: '';
  top: 2px;
  left: 50%;
  height: 3px;
  width: 30px;
  margin-left: -15px;
  position: absolute;
  border-top: 1px solid #cfcfcf;
}
.quickhelp {
  /* Old browsers */
  display: -webkit-box;
  -webkit-box-orient: horizontal;
  -webkit-box-align: stretch;
  display: -moz-box;
  -moz-box-orient: horizontal;
  -moz-box-align: stretch;
  display: box;
  box-orient: horizontal;
  box-align: stretch;
  /* Modern browsers */
  display: flex;
  flex-direction: row;
  align-items: stretch;
  line-height: 1.8em;
}
.shortcut_key {
  display: inline-block;
  width: 21ex;
  text-align: right;
  font-family: monospace;
}
.shortcut_descr {
  display: inline-block;
  /* Old browsers */
  -webkit-box-flex: 1;
  -moz-box-flex: 1;
  box-flex: 1;
  /* Modern browsers */
  flex: 1;
}
span.save_widget {
  height: 30px;
  margin-top: 4px;
  display: flex;
  justify-content: flex-start;
  align-items: baseline;
  width: 50%;
  flex: 1;
}
span.save_widget span.filename {
  height: 100%;
  line-height: 1em;
  margin-left: 16px;
  border: none;
  font-size: 146.5%;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  border-radius: 2px;
}
span.save_widget span.filename:hover {
  background-color: #e6e6e6;
}
[dir="rtl"] span.save_widget.pull-left {
  float: right !important;
  float: right;
}
[dir="rtl"] span.save_widget span.filename {
  margin-left: 0;
  margin-right: 16px;
}
span.checkpoint_status,
span.autosave_status {
  font-size: small;
  white-space: nowrap;
  padding: 0 5px;
}
@media (max-width: 767px) {
  span.save_widget {
    font-size: small;
    padding: 0 0 0 5px;
  }
  span.checkpoint_status,
  span.autosave_status {
    display: none;
  }
}
@media (min-width: 768px) and (max-width: 991px) {
  span.checkpoint_status {
    display: none;
  }
  span.autosave_status {
    font-size: x-small;
  }
}
.toolbar {
  padding: 0px;
  margin-left: -5px;
  margin-top: 2px;
  margin-bottom: 5px;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
}
.toolbar select,
.toolbar label {
  width: auto;
  vertical-align: middle;
  margin-right: 2px;
  margin-bottom: 0px;
  display: inline;
  font-size: 92%;
  margin-left: 0.3em;
  margin-right: 0.3em;
  padding: 0px;
  padding-top: 3px;
}
.toolbar .btn {
  padding: 2px 8px;
}
.toolbar .btn-group {
  margin-top: 0px;
  margin-left: 5px;
}
.toolbar-btn-label {
  margin-left: 6px;
}
#maintoolbar {
  margin-bottom: -3px;
  margin-top: -8px;
  border: 0px;
  min-height: 27px;
  margin-left: 0px;
  padding-top: 11px;
  padding-bottom: 3px;
}
#maintoolbar .navbar-text {
  float: none;
  vertical-align: middle;
  text-align: right;
  margin-left: 5px;
  margin-right: 0px;
  margin-top: 0px;
}
.select-xs {
  height: 24px;
}
[dir="rtl"] .btn-group > .btn,
.btn-group-vertical > .btn {
  float: right;
}
.pulse,
.dropdown-menu > li > a.pulse,
li.pulse > a.dropdown-toggle,
li.pulse.open > a.dropdown-toggle {
  background-color: #F37626;
  color: white;
}
/**
 * Primary styles
 *
 * Author: Jupyter Development Team
 */
/** WARNING IF YOU ARE EDITTING THIS FILE, if this is a .css file, It has a lot
 * of chance of beeing generated from the ../less/[samename].less file, you can
 * try to get back the less file by reverting somme commit in history
 **/
/*
 * We'll try to get something pretty, so we
 * have some strange css to have the scroll bar on
 * the left with fix button on the top right of the tooltip
 */
@-moz-keyframes fadeOut {
  from {
    opacity: 1;
  }
  to {
    opacity: 0;
  }
}
@-webkit-keyframes fadeOut {
  from {
    opacity: 1;
  }
  to {
    opacity: 0;
  }
}
@-moz-keyframes fadeIn {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}
@-webkit-keyframes fadeIn {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}
/*properties of tooltip after "expand"*/
.bigtooltip {
  overflow: auto;
  height: 200px;
  -webkit-transition-property: height;
  -webkit-transition-duration: 500ms;
  -moz-transition-property: height;
  -moz-transition-duration: 500ms;
  transition-property: height;
  transition-duration: 500ms;
}
/*properties of tooltip before "expand"*/
.smalltooltip {
  -webkit-transition-property: height;
  -webkit-transition-duration: 500ms;
  -moz-transition-property: height;
  -moz-transition-duration: 500ms;
  transition-property: height;
  transition-duration: 500ms;
  text-overflow: ellipsis;
  overflow: hidden;
  height: 80px;
}
.tooltipbuttons {
  position: absolute;
  padding-right: 15px;
  top: 0px;
  right: 0px;
}
.tooltiptext {
  /*avoid the button to overlap on some docstring*/
  padding-right: 30px;
}
.ipython_tooltip {
  max-width: 700px;
  /*fade-in animation when inserted*/
  -webkit-animation: fadeOut 400ms;
  -moz-animation: fadeOut 400ms;
  animation: fadeOut 400ms;
  -webkit-animation: fadeIn 400ms;
  -moz-animation: fadeIn 400ms;
  animation: fadeIn 400ms;
  vertical-align: middle;
  background-color: #f7f7f7;
  overflow: visible;
  border: #ababab 1px solid;
  outline: none;
  padding: 3px;
  margin: 0px;
  padding-left: 7px;
  font-family: monospace;
  min-height: 50px;
  -moz-box-shadow: 0px 6px 10px -1px #adadad;
  -webkit-box-shadow: 0px 6px 10px -1px #adadad;
  box-shadow: 0px 6px 10px -1px #adadad;
  border-radius: 2px;
  position: absolute;
  z-index: 1000;
}
.ipython_tooltip a {
  float: right;
}
.ipython_tooltip .tooltiptext pre {
  border: 0;
  border-radius: 0;
  font-size: 100%;
  background-color: #f7f7f7;
}
.pretooltiparrow {
  left: 0px;
  margin: 0px;
  top: -16px;
  width: 40px;
  height: 16px;
  overflow: hidden;
  position: absolute;
}
.pretooltiparrow:before {
  background-color: #f7f7f7;
  border: 1px #ababab solid;
  z-index: 11;
  content: "";
  position: absolute;
  left: 15px;
  top: 10px;
  width: 25px;
  height: 25px;
  -webkit-transform: rotate(45deg);
  -moz-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  -o-transform: rotate(45deg);
}
ul.typeahead-list i {
  margin-left: -10px;
  width: 18px;
}
[dir="rtl"] ul.typeahead-list i {
  margin-left: 0;
  margin-right: -10px;
}
ul.typeahead-list {
  max-height: 80vh;
  overflow: auto;
}
ul.typeahead-list > li > a {
  /** Firefox bug **/
  /* see https://github.com/jupyter/notebook/issues/559 */
  white-space: normal;
}
ul.typeahead-list  > li > a.pull-right {
  float: left !important;
  float: left;
}
[dir="rtl"] .typeahead-list {
  text-align: right;
}
.cmd-palette .modal-body {
  padding: 7px;
}
.cmd-palette form {
  background: white;
}
.cmd-palette input {
  outline: none;
}
.no-shortcut {
  min-width: 20px;
  color: transparent;
}
[dir="rtl"] .no-shortcut.pull-right {
  float: left !important;
  float: left;
}
[dir="rtl"] .command-shortcut.pull-right {
  float: left !important;
  float: left;
}
.command-shortcut:before {
  content: "(command mode)";
  padding-right: 3px;
  color: #777777;
}
.edit-shortcut:before {
  content: "(edit)";
  padding-right: 3px;
  color: #777777;
}
[dir="rtl"] .edit-shortcut.pull-right {
  float: left !important;
  float: left;
}
#find-and-replace #replace-preview .match,
#find-and-replace #replace-preview .insert {
  background-color: #BBDEFB;
  border-color: #90CAF9;
  border-style: solid;
  border-width: 1px;
  border-radius: 0px;
}
[dir="ltr"] #find-and-replace .input-group-btn + .form-control {
  border-left: none;
}
[dir="rtl"] #find-and-replace .input-group-btn + .form-control {
  border-right: none;
}
#find-and-replace #replace-preview .replace .match {
  background-color: #FFCDD2;
  border-color: #EF9A9A;
  border-radius: 0px;
}
#find-and-replace #replace-preview .replace .insert {
  background-color: #C8E6C9;
  border-color: #A5D6A7;
  border-radius: 0px;
}
#find-and-replace #replace-preview {
  max-height: 60vh;
  overflow: auto;
}
#find-and-replace #replace-preview pre {
  padding: 5px 10px;
}
.terminal-app {
  background: #EEE;
}
.terminal-app #header {
  background: #fff;
  -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
  box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.2);
}
.terminal-app .terminal {
  width: 100%;
  float: left;
  font-family: monospace;
  color: white;
  background: black;
  padding: 0.4em;
  border-radius: 2px;
  -webkit-box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.4);
  box-shadow: 0px 0px 12px 1px rgba(87, 87, 87, 0.4);
}
.terminal-app .terminal,
.terminal-app .terminal dummy-screen {
  line-height: 1em;
  font-size: 14px;
}
.terminal-app .terminal .xterm-rows {
  padding: 10px;
}
.terminal-app .terminal-cursor {
  color: black;
  background: white;
}
.terminal-app #terminado-container {
  margin-top: 20px;
}
/*# sourceMappingURL=style.min.css.map */
    </style>
<style type="text/css">
    pre { line-height: 125%; }
td.linenos .normal { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
span.linenos { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
td.linenos .special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
span.linenos.special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
.highlight .hll { background-color: #ffffcc }
.highlight { background: #f8f8f8; }
.highlight .c { color: #408080; font-style: italic } /* Comment */
.highlight .err { border: 1px solid #FF0000 } /* Error */
.highlight .k { color: #008000; font-weight: bold } /* Keyword */
.highlight .o { color: #666666 } /* Operator */
.highlight .ch { color: #408080; font-style: italic } /* Comment.Hashbang */
.highlight .cm { color: #408080; font-style: italic } /* Comment.Multiline */
.highlight .cp { color: #BC7A00 } /* Comment.Preproc */
.highlight .cpf { color: #408080; font-style: italic } /* Comment.PreprocFile */
.highlight .c1 { color: #408080; font-style: italic } /* Comment.Single */
.highlight .cs { color: #408080; font-style: italic } /* Comment.Special */
.highlight .gd { color: #A00000 } /* Generic.Deleted */
.highlight .ge { font-style: italic } /* Generic.Emph */
.highlight .gr { color: #FF0000 } /* Generic.Error */
.highlight .gh { color: #000080; font-weight: bold } /* Generic.Heading */
.highlight .gi { color: #00A000 } /* Generic.Inserted */
.highlight .go { color: #888888 } /* Generic.Output */
.highlight .gp { color: #000080; font-weight: bold } /* Generic.Prompt */
.highlight .gs { font-weight: bold } /* Generic.Strong */
.highlight .gu { color: #800080; font-weight: bold } /* Generic.Subheading */
.highlight .gt { color: #0044DD } /* Generic.Traceback */
.highlight .kc { color: #008000; font-weight: bold } /* Keyword.Constant */
.highlight .kd { color: #008000; font-weight: bold } /* Keyword.Declaration */
.highlight .kn { color: #008000; font-weight: bold } /* Keyword.Namespace */
.highlight .kp { color: #008000 } /* Keyword.Pseudo */
.highlight .kr { color: #008000; font-weight: bold } /* Keyword.Reserved */
.highlight .kt { color: #B00040 } /* Keyword.Type */
.highlight .m { color: #666666 } /* Literal.Number */
.highlight .s { color: #BA2121 } /* Literal.String */
.highlight .na { color: #7D9029 } /* Name.Attribute */
.highlight .nb { color: #008000 } /* Name.Builtin */
.highlight .nc { color: #0000FF; font-weight: bold } /* Name.Class */
.highlight .no { color: #880000 } /* Name.Constant */
.highlight .nd { color: #AA22FF } /* Name.Decorator */
.highlight .ni { color: #999999; font-weight: bold } /* Name.Entity */
.highlight .ne { color: #D2413A; font-weight: bold } /* Name.Exception */
.highlight .nf { color: #0000FF } /* Name.Function */
.highlight .nl { color: #A0A000 } /* Name.Label */
.highlight .nn { color: #0000FF; font-weight: bold } /* Name.Namespace */
.highlight .nt { color: #008000; font-weight: bold } /* Name.Tag */
.highlight .nv { color: #19177C } /* Name.Variable */
.highlight .ow { color: #AA22FF; font-weight: bold } /* Operator.Word */
.highlight .w { color: #bbbbbb } /* Text.Whitespace */
.highlight .mb { color: #666666 } /* Literal.Number.Bin */
.highlight .mf { color: #666666 } /* Literal.Number.Float */
.highlight .mh { color: #666666 } /* Literal.Number.Hex */
.highlight .mi { color: #666666 } /* Literal.Number.Integer */
.highlight .mo { color: #666666 } /* Literal.Number.Oct */
.highlight .sa { color: #BA2121 } /* Literal.String.Affix */
.highlight .sb { color: #BA2121 } /* Literal.String.Backtick */
.highlight .sc { color: #BA2121 } /* Literal.String.Char */
.highlight .dl { color: #BA2121 } /* Literal.String.Delimiter */
.highlight .sd { color: #BA2121; font-style: italic } /* Literal.String.Doc */
.highlight .s2 { color: #BA2121 } /* Literal.String.Double */
.highlight .se { color: #BB6622; font-weight: bold } /* Literal.String.Escape */
.highlight .sh { color: #BA2121 } /* Literal.String.Heredoc */
.highlight .si { color: #BB6688; font-weight: bold } /* Literal.String.Interpol */
.highlight .sx { color: #008000 } /* Literal.String.Other */
.highlight .sr { color: #BB6688 } /* Literal.String.Regex */
.highlight .s1 { color: #BA2121 } /* Literal.String.Single */
.highlight .ss { color: #19177C } /* Literal.String.Symbol */
.highlight .bp { color: #008000 } /* Name.Builtin.Pseudo */
.highlight .fm { color: #0000FF } /* Name.Function.Magic */
.highlight .vc { color: #19177C } /* Name.Variable.Class */
.highlight .vg { color: #19177C } /* Name.Variable.Global */
.highlight .vi { color: #19177C } /* Name.Variable.Instance */
.highlight .vm { color: #19177C } /* Name.Variable.Magic */
.highlight .il { color: #666666 } /* Literal.Number.Integer.Long */
    </style>


<style type="text/css">
/* Overrides of notebook CSS for static HTML export */
body {
  overflow: visible;
  padding: 8px;
}

div#notebook {
  overflow: visible;
  border-top: none;
}@media print {
  div.cell {
    display: block;
    page-break-inside: avoid;
  } 
  div.output_wrapper { 
    display: block;
    page-break-inside: avoid; 
  }
  div.output { 
    display: block;
    page-break-inside: avoid; 
  }
}
</style>

<!-- Custom stylesheet, it must be in the same directory as the html file -->
<link rel="stylesheet" href="custom.css">

<!-- Loading mathjax macro -->
<!-- Load mathjax -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS_HTML"></script>
    <!-- MathJax configuration -->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [ ['$','$'], ["\\(","\\)"] ],
            displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
            processEscapes: true,
            processEnvironments: true
        },
        // Center justify equations in code and markdown cells. Elsewhere
        // we use CSS to left justify single line equations in code cells.
        displayAlign: 'center',
        "HTML-CSS": {
            styles: {'.MathJax_Display': {"margin": 0}},
            linebreaks: { automatic: true }
        }
    });
    </script>
    <!-- End of mathjax configuration --></head>

 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.css">

<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.9.1/jquery-ui.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://rawgit.com/ipython-contrib/jupyter_contrib_nbextensions/master/src/jupyter_contrib_nbextensions/nbextensions/toc2/main.css">

<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

<script src="https://rawgit.com/ipython-contrib/jupyter_contrib_nbextensions/master/src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.js"></script>

<script>
$( document ).ready(function(){

            var cfg = {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": false, "toc_position": {}, "toc_section_display": true, "toc_window_display": false};

            // fire the main function with these parameters
            require(['nbextensions/toc2/toc2'], function (toc2) {
                toc2.table_of_contents(cfg);
            });
    });
</script>
<body>
  <div tabindex="-1" id="notebook" class="border-box-sizing">
    <div class="container" id="notebook-container">

<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Reviewer-for-Midterms-in-DMW">Reviewer for Midterms in DMW<a class="anchor-link" href="#Reviewer-for-Midterms-in-DMW">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>This reviewer is meant to supplement the Reader’s existing reviewers and notes. The author of this notebook shall not assume any responsibility on the accuracy and completeness of any information provided in this reviewer. The Reader is
expected to exercise prudence in reading through the reviewer and is free to comment on its accuracy and
completeness for everyone's benefit.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[4]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="kn">import</span> <span class="nn">pandas</span> <span class="k">as</span> <span class="nn">pd</span>
<span class="kn">import</span> <span class="nn">json</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="kn">from</span> <span class="nn">datetime</span> <span class="kn">import</span> <span class="n">datetime</span><span class="p">,</span> <span class="n">timedelta</span>
<span class="kn">from</span> <span class="nn">xml.etree</span> <span class="kn">import</span> <span class="n">ElementTree</span> <span class="k">as</span> <span class="n">ET</span>
</pre></div>

    </div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h2 id="JSON-Fileformats-Reviewer-(Basics)">JSON Fileformats Reviewer (Basics)<a class="anchor-link" href="#JSON-Fileformats-Reviewer-(Basics)">&#182;</a></h2><p>This notebook was adopated from the examples discussed in this <a href="https://towardsdatascience.com/how-to-best-work-with-json-in-python-2c8989ff0390">website</a>.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="How-to-read-JSON-files-and-JSON-strings">How to read JSON files and JSON strings<a class="anchor-link" href="#How-to-read-JSON-files-and-JSON-strings">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[11]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Read JSON files using the json library</span>
<span class="c1"># Use json.load()</span>
<span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="s1">&#39;superheroes.json&#39;</span><span class="p">,</span> <span class="s1">&#39;r&#39;</span><span class="p">)</span> <span class="k">as</span> <span class="n">f</span><span class="p">:</span>
    <span class="n">super_heroes_file</span> <span class="o">=</span> <span class="n">json</span><span class="o">.</span><span class="n">load</span><span class="p">(</span><span class="n">f</span><span class="p">)</span>

<span class="n">super_heroes_file</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[11]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>{&#39;squadName&#39;: &#39;Super Hero Squad&#39;,
 &#39;homeTown&#39;: &#39;Metro City&#39;,
 &#39;formed&#39;: 2016,
 &#39;secretBase&#39;: &#39;Super tower&#39;,
 &#39;active&#39;: True,
 &#39;members&#39;: [{&#39;name&#39;: &#39;Molecule Man&#39;,
   &#39;age&#39;: 29,
   &#39;secretIdentity&#39;: &#39;Dan Jukes&#39;,
   &#39;powers&#39;: [&#39;Radiation resistance&#39;, &#39;Turning tiny&#39;, &#39;Radiation blast&#39;]},
  {&#39;name&#39;: &#39;Madame Uppercut&#39;,
   &#39;age&#39;: 39,
   &#39;secretIdentity&#39;: &#39;Jane Wilson&#39;,
   &#39;powers&#39;: [&#39;Million tonne punch&#39;,
    &#39;Damage resistance&#39;,
    &#39;Superhuman reflexes&#39;]},
  {&#39;name&#39;: &#39;Eternal Flame&#39;,
   &#39;age&#39;: 1000000,
   &#39;secretIdentity&#39;: &#39;Unknown&#39;,
   &#39;powers&#39;: [&#39;Immortality&#39;,
    &#39;Heat Immunity&#39;,
    &#39;Inferno&#39;,
    &#39;Teleportation&#39;,
    &#39;Interdimensional travel&#39;]}]}</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[6]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">super_heroes_json</span> <span class="o">=</span> <span class="s2">&quot;&quot;&quot;{</span>
<span class="s2">  &quot;squadName&quot; : &quot;Super Hero Squad&quot;,</span>
<span class="s2">  &quot;homeTown&quot; : &quot;Metro City&quot;,</span>
<span class="s2">  &quot;formed&quot; : 2016,</span>
<span class="s2">  &quot;secretBase&quot; : &quot;Super tower&quot;,</span>
<span class="s2">  &quot;active&quot; : true,</span>
<span class="s2">  &quot;members&quot; : [</span>
<span class="s2">    {</span>
<span class="s2">      &quot;name&quot; : &quot;Molecule Man&quot;,</span>
<span class="s2">      &quot;age&quot; : 29,</span>
<span class="s2">      &quot;secretIdentity&quot; : &quot;Dan Jukes&quot;,</span>
<span class="s2">      &quot;powers&quot; : [</span>
<span class="s2">        &quot;Radiation resistance&quot;,</span>
<span class="s2">        &quot;Turning tiny&quot;,</span>
<span class="s2">        &quot;Radiation blast&quot;</span>
<span class="s2">      ]</span>
<span class="s2">    },</span>
<span class="s2">    {</span>
<span class="s2">      &quot;name&quot; : &quot;Madame Uppercut&quot;,</span>
<span class="s2">      &quot;age&quot; : 39,</span>
<span class="s2">      &quot;secretIdentity&quot; : &quot;Jane Wilson&quot;,</span>
<span class="s2">      &quot;powers&quot; : [</span>
<span class="s2">        &quot;Million tonne punch&quot;,</span>
<span class="s2">        &quot;Damage resistance&quot;,</span>
<span class="s2">        &quot;Superhuman reflexes&quot;</span>
<span class="s2">      ]</span>
<span class="s2">    },</span>
<span class="s2">    {</span>
<span class="s2">      &quot;name&quot; : &quot;Eternal Flame&quot;,</span>
<span class="s2">      &quot;age&quot; : 1000000,</span>
<span class="s2">      &quot;secretIdentity&quot; : &quot;Unknown&quot;,</span>
<span class="s2">      &quot;powers&quot; : [</span>
<span class="s2">        &quot;Immortality&quot;,</span>
<span class="s2">        &quot;Heat Immunity&quot;,</span>
<span class="s2">        &quot;Inferno&quot;,</span>
<span class="s2">        &quot;Teleportation&quot;,</span>
<span class="s2">        &quot;Interdimensional travel&quot;</span>
<span class="s2">      ]</span>
<span class="s2">    }</span>
<span class="s2">  ]</span>
<span class="s2">}</span>
<span class="s2">&quot;&quot;&quot;</span>
</pre></div>

    </div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[12]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># What if the json file is given as a string?</span>
<span class="c1"># Use json.loads()</span>
<span class="n">super_heroes_string</span> <span class="o">=</span> <span class="n">json</span><span class="o">.</span><span class="n">loads</span><span class="p">(</span><span class="n">super_heroes_json</span><span class="p">)</span>
<span class="n">super_heroes_string</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[12]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>{&#39;squadName&#39;: &#39;Super Hero Squad&#39;,
 &#39;homeTown&#39;: &#39;Metro City&#39;,
 &#39;formed&#39;: 2016,
 &#39;secretBase&#39;: &#39;Super tower&#39;,
 &#39;active&#39;: True,
 &#39;members&#39;: [{&#39;name&#39;: &#39;Molecule Man&#39;,
   &#39;age&#39;: 29,
   &#39;secretIdentity&#39;: &#39;Dan Jukes&#39;,
   &#39;powers&#39;: [&#39;Radiation resistance&#39;, &#39;Turning tiny&#39;, &#39;Radiation blast&#39;]},
  {&#39;name&#39;: &#39;Madame Uppercut&#39;,
   &#39;age&#39;: 39,
   &#39;secretIdentity&#39;: &#39;Jane Wilson&#39;,
   &#39;powers&#39;: [&#39;Million tonne punch&#39;,
    &#39;Damage resistance&#39;,
    &#39;Superhuman reflexes&#39;]},
  {&#39;name&#39;: &#39;Eternal Flame&#39;,
   &#39;age&#39;: 1000000,
   &#39;secretIdentity&#39;: &#39;Unknown&#39;,
   &#39;powers&#39;: [&#39;Immortality&#39;,
    &#39;Heat Immunity&#39;,
    &#39;Inferno&#39;,
    &#39;Teleportation&#39;,
    &#39;Interdimensional travel&#39;]}]}</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="&quot;Prettify&quot;-Printing-of-Read-JSON-File-or-JSON-String">"Prettify" Printing of Read JSON File or JSON String<a class="anchor-link" href="#&quot;Prettify&quot;-Printing-of-Read-JSON-File-or-JSON-String">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[67]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># To prettify printing of the json file</span>
<span class="c1"># Use json.dumps with the argument &quot;indent&quot;</span>
<span class="n">pretty_json</span> <span class="o">=</span> <span class="n">json</span><span class="o">.</span><span class="n">dumps</span><span class="p">(</span><span class="n">super_heroes_file</span><span class="p">,</span> <span class="n">indent</span><span class="o">=</span><span class="mi">4</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="n">pretty_json</span><span class="p">)</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt"></div>


<div class="output_subarea output_stream output_stdout output_text">
<pre>{
    &#34;active&#34;: true,
    &#34;formed&#34;: 2016,
    &#34;homeTown&#34;: &#34;Metro City&#34;,
    &#34;members&#34;: [
        {
            &#34;age&#34;: 29,
            &#34;name&#34;: &#34;Molecule Man&#34;,
            &#34;powers&#34;: [
                &#34;Radiation resistance&#34;,
                &#34;Turning tiny&#34;,
                &#34;Radiation blast&#34;
            ],
            &#34;secretIdentity&#34;: &#34;Dan Jukes&#34;
        },
        {
            &#34;age&#34;: 39,
            &#34;name&#34;: &#34;Madame Uppercut&#34;,
            &#34;powers&#34;: [
                &#34;Million tonne punch&#34;,
                &#34;Damage resistance&#34;,
                &#34;Superhuman reflexes&#34;
            ],
            &#34;secretIdentity&#34;: &#34;Jane Wilson&#34;
        },
        {
            &#34;age&#34;: 1000000,
            &#34;name&#34;: &#34;Eternal Flame&#34;,
            &#34;powers&#34;: [
                &#34;Immortality&#34;,
                &#34;Heat Immunity&#34;,
                &#34;Inferno&#34;,
                &#34;Teleportation&#34;,
                &#34;Interdimensional travel&#34;
            ],
            &#34;secretIdentity&#34;: &#34;Unknown&#34;
        }
    ],
    &#34;secretBase&#34;: &#34;Super tower&#34;,
    &#34;squadName&#34;: &#34;Super Hero Squad&#34;
}
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[69]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># It is also possible to sort the keys</span>
<span class="c1"># Use json.dumps with the argument &quot;sort_keys&quot;</span>
<span class="n">pretty_json_sorted</span> <span class="o">=</span> <span class="n">json</span><span class="o">.</span><span class="n">dumps</span><span class="p">(</span><span class="n">super_heroes_file</span><span class="p">,</span> <span class="n">indent</span><span class="o">=</span><span class="mi">4</span><span class="p">,</span> <span class="n">sort_keys</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="n">pretty_json</span><span class="p">)</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt"></div>


<div class="output_subarea output_stream output_stdout output_text">
<pre>{
    &#34;active&#34;: true,
    &#34;formed&#34;: 2016,
    &#34;homeTown&#34;: &#34;Metro City&#34;,
    &#34;members&#34;: [
        {
            &#34;age&#34;: 29,
            &#34;name&#34;: &#34;Molecule Man&#34;,
            &#34;powers&#34;: [
                &#34;Radiation resistance&#34;,
                &#34;Turning tiny&#34;,
                &#34;Radiation blast&#34;
            ],
            &#34;secretIdentity&#34;: &#34;Dan Jukes&#34;
        },
        {
            &#34;age&#34;: 39,
            &#34;name&#34;: &#34;Madame Uppercut&#34;,
            &#34;powers&#34;: [
                &#34;Million tonne punch&#34;,
                &#34;Damage resistance&#34;,
                &#34;Superhuman reflexes&#34;
            ],
            &#34;secretIdentity&#34;: &#34;Jane Wilson&#34;
        },
        {
            &#34;age&#34;: 1000000,
            &#34;name&#34;: &#34;Eternal Flame&#34;,
            &#34;powers&#34;: [
                &#34;Immortality&#34;,
                &#34;Heat Immunity&#34;,
                &#34;Inferno&#34;,
                &#34;Teleportation&#34;,
                &#34;Interdimensional travel&#34;
            ],
            &#34;secretIdentity&#34;: &#34;Unknown&#34;
        }
    ],
    &#34;secretBase&#34;: &#34;Super tower&#34;,
    &#34;squadName&#34;: &#34;Super Hero Squad&#34;
}
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="Read-JSON-Files-and-JSON-strings-are-returned-as-dictionaries">Read JSON Files and JSON strings are returned as dictionaries<a class="anchor-link" href="#Read-JSON-Files-and-JSON-strings-are-returned-as-dictionaries">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[13]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># what is the type of the read json file?</span>
<span class="c1"># the type is a dictinary!</span>
<span class="nb">type</span><span class="p">(</span><span class="n">super_heroes_file</span><span class="p">),</span> <span class="nb">type</span><span class="p">(</span><span class="n">super_heroes_string</span><span class="p">)</span>

<span class="c1"># note that the output is a dictionary with keys and values!</span>
<span class="c1"># note that nested dictionaries could be observed from the output</span>
<span class="c1"># you can manipulate as usual using what we have learned with dictionaries</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[13]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>(dict, dict)</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="Accessing-of-Data-in-a-JSON-File-or-JSON-String">Accessing of Data in a JSON File or JSON String<a class="anchor-link" href="#Accessing-of-Data-in-a-JSON-File-or-JSON-String">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[14]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># what are the elements under the super_heroes_file?</span>
<span class="c1"># Use .keys()</span>
<span class="n">super_heroes_file</span><span class="o">.</span><span class="n">keys</span><span class="p">()</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[14]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>dict_keys([&#39;squadName&#39;, &#39;homeTown&#39;, &#39;formed&#39;, &#39;secretBase&#39;, &#39;active&#39;, &#39;members&#39;])</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[15]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># what are the values of the keys under the super_heroes_file?</span>
<span class="c1"># Use .values()</span>
<span class="n">super_heroes_file</span><span class="o">.</span><span class="n">values</span><span class="p">()</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[15]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>dict_values([&#39;Super Hero Squad&#39;, &#39;Metro City&#39;, 2016, &#39;Super tower&#39;, True, [{&#39;name&#39;: &#39;Molecule Man&#39;, &#39;age&#39;: 29, &#39;secretIdentity&#39;: &#39;Dan Jukes&#39;, &#39;powers&#39;: [&#39;Radiation resistance&#39;, &#39;Turning tiny&#39;, &#39;Radiation blast&#39;]}, {&#39;name&#39;: &#39;Madame Uppercut&#39;, &#39;age&#39;: 39, &#39;secretIdentity&#39;: &#39;Jane Wilson&#39;, &#39;powers&#39;: [&#39;Million tonne punch&#39;, &#39;Damage resistance&#39;, &#39;Superhuman reflexes&#39;]}, {&#39;name&#39;: &#39;Eternal Flame&#39;, &#39;age&#39;: 1000000, &#39;secretIdentity&#39;: &#39;Unknown&#39;, &#39;powers&#39;: [&#39;Immortality&#39;, &#39;Heat Immunity&#39;, &#39;Inferno&#39;, &#39;Teleportation&#39;, &#39;Interdimensional travel&#39;]}]])</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[25]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Who are the members of the Super Hero Squad?</span>
<span class="c1"># use for loop!</span>
<span class="n">super_hero_members</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">member</span> <span class="ow">in</span> <span class="n">super_heroes_file</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">]:</span>
    <span class="n">super_hero_members</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">member</span><span class="p">[</span><span class="s1">&#39;name&#39;</span><span class="p">])</span>
<span class="n">super_hero_members</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[25]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>[&#39;Molecule Man&#39;, &#39;Madame Uppercut&#39;, &#39;Eternal Flame&#39;]</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[26]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># How many members are there in the Super Hero Squad?</span>
<span class="c1"># use for loop and get the length of super_hero_members</span>
<span class="n">super_hero_members</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">member</span> <span class="ow">in</span> <span class="n">super_heroes_file</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">]:</span>
    <span class="n">super_hero_members</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">member</span><span class="p">[</span><span class="s1">&#39;name&#39;</span><span class="p">])</span>
<span class="nb">len</span><span class="p">(</span><span class="n">super_hero_members</span><span class="p">)</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[26]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>3</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[63]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Return the secret identity &quot;Jane Wilson&quot;</span>
<span class="c1"># Note that the secret identity &quot;Jane Wilson&quot; is under the 2nd dictionary in the list</span>
<span class="n">super_heroes_file</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">]</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[63]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>[{&#39;name&#39;: &#39;Molecule Man&#39;,
  &#39;age&#39;: 29,
  &#39;secretIdentity&#39;: &#39;Dan Jukes&#39;,
  &#39;powers&#39;: [&#39;Radiation resistance&#39;, &#39;Turning tiny&#39;, &#39;Radiation blast&#39;]},
 {&#39;name&#39;: &#39;Madame Uppercut&#39;,
  &#39;age&#39;: 39,
  &#39;secretIdentity&#39;: &#39;Jane Wilson&#39;,
  &#39;powers&#39;: [&#39;Million tonne punch&#39;,
   &#39;Damage resistance&#39;,
   &#39;Superhuman reflexes&#39;]},
 {&#39;name&#39;: &#39;Eternal Flame&#39;,
  &#39;age&#39;: 1000000,
  &#39;secretIdentity&#39;: &#39;Unknown&#39;,
  &#39;powers&#39;: [&#39;Immortality&#39;,
   &#39;Heat Immunity&#39;,
   &#39;Inferno&#39;,
   &#39;Teleportation&#39;,
   &#39;Interdimensional travel&#39;]}]</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[64]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># To return the secret identity &quot;Jane Wilson&quot;, access the 2nd element in the list</span>
<span class="n">super_heroes_file</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">][</span><span class="mi">1</span><span class="p">][</span><span class="s1">&#39;secretIdentity&#39;</span><span class="p">]</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[64]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>&#39;Jane Wilson&#39;</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="Read-JSON-Files-and-JSON-strings-as-pandas.DataFrame">Read JSON Files and JSON strings as pandas.DataFrame<a class="anchor-link" href="#Read-JSON-Files-and-JSON-strings-as-pandas.DataFrame">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[29]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Read JSON file as pandas.DataFrame</span>
<span class="c1"># Use pd.read_json</span>
<span class="n">df_json</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">read_json</span><span class="p">(</span><span class="s1">&#39;superheroes.json&#39;</span><span class="p">)</span>
<span class="n">df_json</span>

<span class="c1"># note that the column &#39;members&#39; has dictionaries as its values!</span>
<span class="c1"># this tells us that the read json file need to be flattened</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[29]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>squadName</th>
      <th>homeTown</th>
      <th>formed</th>
      <th>secretBase</th>
      <th>active</th>
      <th>members</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
      <td>{'name': 'Molecule Man', 'age': 29, 'secretIde...</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
      <td>{'name': 'Madame Uppercut', 'age': 39, 'secret...</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
      <td>{'name': 'Eternal Flame', 'age': 1000000, 'sec...</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[30]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># JSON files read as pandas.DataFrame can also come from URLs</span>
<span class="c1"># Use pd.read_json</span>
<span class="n">df_json_url</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">read_json</span><span class="p">(</span><span class="s1">&#39;https://mdn.github.io/learning-area/javascript/&#39;</span>
                           <span class="s1">&#39;oojs/json/superheroes.json&#39;</span><span class="p">)</span>
<span class="n">df_json_url</span>

<span class="c1"># note that df_json and df_json_url are exactly the same dataframe!</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[30]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>squadName</th>
      <th>homeTown</th>
      <th>formed</th>
      <th>secretBase</th>
      <th>active</th>
      <th>members</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
      <td>{'name': 'Molecule Man', 'age': 29, 'secretIde...</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
      <td>{'name': 'Madame Uppercut', 'age': 39, 'secret...</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
      <td>{'name': 'Eternal Flame', 'age': 1000000, 'sec...</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="How-to-flatten-read-JSON-files-in-a-dataframe">How to flatten read JSON files in a dataframe<a class="anchor-link" href="#How-to-flatten-read-JSON-files-in-a-dataframe">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p><strong>Method 1</strong>: Use <code>.apply(pd.Series)</code></p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[34]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># How do I flatten the &quot;members&quot; column?</span>
<span class="c1"># First use &quot;.apply(pd.Series)&quot;</span>
<span class="n">df_json_members</span> <span class="o">=</span> <span class="n">df_json</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">]</span><span class="o">.</span><span class="n">apply</span><span class="p">(</span><span class="n">pd</span><span class="o">.</span><span class="n">Series</span><span class="p">)</span>
<span class="n">df_json_members</span>

<span class="c1"># note that the values under the &quot;powers&quot; column are lists!</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[34]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>name</th>
      <th>age</th>
      <th>secretIdentity</th>
      <th>powers</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Molecule Man</td>
      <td>29</td>
      <td>Dan Jukes</td>
      <td>[Radiation resistance, Turning tiny, Radiation...</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Madame Uppercut</td>
      <td>39</td>
      <td>Jane Wilson</td>
      <td>[Million tonne punch, Damage resistance, Super...</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Eternal Flame</td>
      <td>1000000</td>
      <td>Unknown</td>
      <td>[Immortality, Heat Immunity, Inferno, Teleport...</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[48]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Then combine dataframes</span>
<span class="n">df_json_flat</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">concat</span><span class="p">([</span><span class="n">df_json_members</span><span class="p">,</span> <span class="n">df_json</span><span class="o">.</span><span class="n">drop</span><span class="p">(</span><span class="s1">&#39;members&#39;</span><span class="p">,</span> <span class="n">axis</span> <span class="o">=</span> <span class="mi">1</span><span class="p">)],</span> <span class="n">axis</span> <span class="o">=</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">df_json_flat</span>

<span class="c1"># don&#39;t forget to drop the &quot;members&quot; column of df_json</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[48]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>name</th>
      <th>age</th>
      <th>secretIdentity</th>
      <th>powers</th>
      <th>squadName</th>
      <th>homeTown</th>
      <th>formed</th>
      <th>secretBase</th>
      <th>active</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Molecule Man</td>
      <td>29</td>
      <td>Dan Jukes</td>
      <td>[Radiation resistance, Turning tiny, Radiation...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Madame Uppercut</td>
      <td>39</td>
      <td>Jane Wilson</td>
      <td>[Million tonne punch, Damage resistance, Super...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Eternal Flame</td>
      <td>1000000</td>
      <td>Unknown</td>
      <td>[Immortality, Heat Immunity, Inferno, Teleport...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p><strong>Method 2</strong>: Use <code>pd.json_normalize()</code></p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[47]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Method 2 uses pd.json_normalize()</span>
<span class="c1"># The code below demonstrates how this is implemented.</span>
<span class="n">df_json_flat_norm</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">json_normalize</span><span class="p">(</span><span class="n">super_heroes_file</span><span class="p">,</span> <span class="n">record_path</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">])</span>
<span class="n">df_json_flat_norm</span>

<span class="c1"># note that &quot;super_heroes_file&quot; should be a JSON file returned as a dictionary</span>
<span class="c1"># [&#39;members&#39;] --&gt; target column to be flattened</span>
<span class="c1"># note also that this method returned the same output as .apply(pd.Series)!</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[47]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>name</th>
      <th>age</th>
      <th>secretIdentity</th>
      <th>powers</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Molecule Man</td>
      <td>29</td>
      <td>Dan Jukes</td>
      <td>[Radiation resistance, Turning tiny, Radiation...</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Madame Uppercut</td>
      <td>39</td>
      <td>Jane Wilson</td>
      <td>[Million tonne punch, Damage resistance, Super...</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Eternal Flame</td>
      <td>1000000</td>
      <td>Unknown</td>
      <td>[Immortality, Heat Immunity, Inferno, Teleport...</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[49]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># How do we use pd.json_normalize to return a complete and flattened dataframe from the read json_file?</span>
<span class="c1"># Get first the keys of the json file read as a dictionary</span>
<span class="n">super_heroes_file</span><span class="o">.</span><span class="n">keys</span><span class="p">()</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[49]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>dict_keys([&#39;squadName&#39;, &#39;homeTown&#39;, &#39;formed&#39;, &#39;secretBase&#39;, &#39;active&#39;, &#39;members&#39;])</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[57]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># use the &quot;meta&quot; argument and input remaining keys under the json file read as a dictionary</span>
<span class="c1"># don&#39;t forget to not include the column to be flattened in the &quot;meta&quot; argument</span>
<span class="n">df_json_flat_norm_full</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">json_normalize</span><span class="p">(</span>
    <span class="n">super_heroes_file</span><span class="p">,</span> <span class="n">record_path</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">],</span>
<span class="n">meta</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;squadName&#39;</span><span class="p">,</span> <span class="s1">&#39;homeTown&#39;</span><span class="p">,</span> <span class="s1">&#39;formed&#39;</span><span class="p">,</span> <span class="s1">&#39;secretBase&#39;</span><span class="p">,</span> <span class="s1">&#39;active&#39;</span><span class="p">])</span>

<span class="n">df_json_flat_norm_full</span>

<span class="c1"># note that the meta argument is case sensitive!</span>
<span class="c1"># note that this is same output as Method 1!</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[57]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>name</th>
      <th>age</th>
      <th>secretIdentity</th>
      <th>powers</th>
      <th>squadName</th>
      <th>homeTown</th>
      <th>formed</th>
      <th>secretBase</th>
      <th>active</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Molecule Man</td>
      <td>29</td>
      <td>Dan Jukes</td>
      <td>[Radiation resistance, Turning tiny, Radiation...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Madame Uppercut</td>
      <td>39</td>
      <td>Jane Wilson</td>
      <td>[Million tonne punch, Damage resistance, Super...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Eternal Flame</td>
      <td>1000000</td>
      <td>Unknown</td>
      <td>[Immortality, Heat Immunity, Inferno, Teleport...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[60]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># We can also add a prefix to the flattened columns via the &quot;meta_prefix&quot; argument</span>
<span class="n">df_json_flat_norm_full_prefix</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">json_normalize</span><span class="p">(</span>
    <span class="n">super_heroes_file</span><span class="p">,</span> <span class="n">record_path</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;members&#39;</span><span class="p">],</span>
<span class="n">meta</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;squadName&#39;</span><span class="p">,</span> <span class="s1">&#39;homeTown&#39;</span><span class="p">,</span> <span class="s1">&#39;formed&#39;</span><span class="p">,</span> <span class="s1">&#39;secretBase&#39;</span><span class="p">,</span> <span class="s1">&#39;active&#39;</span><span class="p">],</span>
<span class="n">meta_prefix</span><span class="o">=</span><span class="s1">&#39;members_&#39;</span><span class="p">)</span>

<span class="n">df_json_flat_norm_full_prefix</span>

<span class="c1"># note that whatever string you put in the &quot;meta_prefix&quot; will be added as prefix to the flattened columns</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[60]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>name</th>
      <th>age</th>
      <th>secretIdentity</th>
      <th>powers</th>
      <th>members_squadName</th>
      <th>members_homeTown</th>
      <th>members_formed</th>
      <th>members_secretBase</th>
      <th>members_active</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>Molecule Man</td>
      <td>29</td>
      <td>Dan Jukes</td>
      <td>[Radiation resistance, Turning tiny, Radiation...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Madame Uppercut</td>
      <td>39</td>
      <td>Jane Wilson</td>
      <td>[Million tonne punch, Damage resistance, Super...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Eternal Flame</td>
      <td>1000000</td>
      <td>Unknown</td>
      <td>[Immortality, Heat Immunity, Inferno, Teleport...</td>
      <td>Super Hero Squad</td>
      <td>Metro City</td>
      <td>2016</td>
      <td>Super tower</td>
      <td>True</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h2 id="JSON-Fileformats-Reviewer-(Advanced)">JSON Fileformats Reviewer (Advanced)<a class="anchor-link" href="#JSON-Fileformats-Reviewer-(Advanced)">&#182;</a></h2>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="JSON-Files-Having-Multiple-Lines-Need-to-Be-Read-Line-per-Line">JSON Files Having Multiple Lines Need to Be Read Line per Line<a class="anchor-link" href="#JSON-Files-Having-Multiple-Lines-Need-to-Be-Read-Line-per-Line">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[5]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># JSON files having multiple lines will resort to an error</span>
<span class="c1"># so the file needs to be read line per line</span>
<span class="n">part1</span> <span class="o">=</span> <span class="s1">&#39;/mnt/data/public/covid19-lake/alleninstitute/CORD19/json/&#39;</span>
<span class="n">part2</span> <span class="o">=</span> <span class="s1">&#39;metadata/part-00000-81803174-7752-4489-8eeb-081318af9653&#39;</span>
<span class="n">part3</span> <span class="o">=</span> <span class="s1">&#39;-c000.json&#39;</span>
<span class="n">file_path</span> <span class="o">=</span> <span class="n">part1</span> <span class="o">+</span> <span class="n">part2</span> <span class="o">+</span> <span class="n">part3</span>
<span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="n">file_path</span><span class="p">,</span> <span class="s1">&#39;r&#39;</span><span class="p">)</span> <span class="k">as</span> <span class="n">f</span><span class="p">:</span>
    <span class="n">journal_json</span> <span class="o">=</span> <span class="n">json</span><span class="o">.</span><span class="n">load</span><span class="p">(</span><span class="n">f</span><span class="p">)</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt"></div>


<div class="output_subarea output_text output_error">
<pre>
<span class="ansi-red-fg">---------------------------------------------------------------------------</span>
<span class="ansi-red-fg">JSONDecodeError</span>                           Traceback (most recent call last)
<span class="ansi-green-fg">&lt;ipython-input-5-dfdd6a81b9f8&gt;</span> in <span class="ansi-cyan-fg">&lt;module&gt;</span>
<span class="ansi-green-intense-fg ansi-bold">      4</span> file_path <span class="ansi-blue-fg">=</span> part1 <span class="ansi-blue-fg">+</span> part2 <span class="ansi-blue-fg">+</span> part3
<span class="ansi-green-intense-fg ansi-bold">      5</span> <span class="ansi-green-fg">with</span> open<span class="ansi-blue-fg">(</span>file_path<span class="ansi-blue-fg">,</span> <span class="ansi-blue-fg">&#39;r&#39;</span><span class="ansi-blue-fg">)</span> <span class="ansi-green-fg">as</span> f<span class="ansi-blue-fg">:</span>
<span class="ansi-green-fg">----&gt; 6</span><span class="ansi-red-fg">     </span>journal_json <span class="ansi-blue-fg">=</span> json<span class="ansi-blue-fg">.</span>load<span class="ansi-blue-fg">(</span>f<span class="ansi-blue-fg">)</span>

<span class="ansi-green-fg">/opt/conda/lib/python3.8/json/__init__.py</span> in <span class="ansi-cyan-fg">load</span><span class="ansi-blue-fg">(fp, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)</span>
<span class="ansi-green-intense-fg ansi-bold">    291</span>     kwarg<span class="ansi-blue-fg">;</span> otherwise<span class="ansi-red-fg"> </span><span class="ansi-red-fg">`</span><span class="ansi-red-fg">`</span>JSONDecoder<span class="ansi-red-fg">`</span><span class="ansi-red-fg">`</span> <span class="ansi-green-fg">is</span> used<span class="ansi-blue-fg">.</span>
<span class="ansi-green-intense-fg ansi-bold">    292</span>     &#34;&#34;&#34;
<span class="ansi-green-fg">--&gt; 293</span><span class="ansi-red-fg">     return loads(fp.read(),
</span><span class="ansi-green-intense-fg ansi-bold">    294</span>         cls<span class="ansi-blue-fg">=</span>cls<span class="ansi-blue-fg">,</span> object_hook<span class="ansi-blue-fg">=</span>object_hook<span class="ansi-blue-fg">,</span>
<span class="ansi-green-intense-fg ansi-bold">    295</span>         parse_float<span class="ansi-blue-fg">=</span>parse_float<span class="ansi-blue-fg">,</span> parse_int<span class="ansi-blue-fg">=</span>parse_int<span class="ansi-blue-fg">,</span>

<span class="ansi-green-fg">/opt/conda/lib/python3.8/json/__init__.py</span> in <span class="ansi-cyan-fg">loads</span><span class="ansi-blue-fg">(s, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)</span>
<span class="ansi-green-intense-fg ansi-bold">    355</span>             parse_int <span class="ansi-green-fg">is</span> <span class="ansi-green-fg">None</span> <span class="ansi-green-fg">and</span> parse_float <span class="ansi-green-fg">is</span> <span class="ansi-green-fg">None</span> <span class="ansi-green-fg">and</span>
<span class="ansi-green-intense-fg ansi-bold">    356</span>             parse_constant is None and object_pairs_hook is None and not kw):
<span class="ansi-green-fg">--&gt; 357</span><span class="ansi-red-fg">         </span><span class="ansi-green-fg">return</span> _default_decoder<span class="ansi-blue-fg">.</span>decode<span class="ansi-blue-fg">(</span>s<span class="ansi-blue-fg">)</span>
<span class="ansi-green-intense-fg ansi-bold">    358</span>     <span class="ansi-green-fg">if</span> cls <span class="ansi-green-fg">is</span> <span class="ansi-green-fg">None</span><span class="ansi-blue-fg">:</span>
<span class="ansi-green-intense-fg ansi-bold">    359</span>         cls <span class="ansi-blue-fg">=</span> JSONDecoder

<span class="ansi-green-fg">/opt/conda/lib/python3.8/json/decoder.py</span> in <span class="ansi-cyan-fg">decode</span><span class="ansi-blue-fg">(self, s, _w)</span>
<span class="ansi-green-intense-fg ansi-bold">    338</span>         end <span class="ansi-blue-fg">=</span> _w<span class="ansi-blue-fg">(</span>s<span class="ansi-blue-fg">,</span> end<span class="ansi-blue-fg">)</span><span class="ansi-blue-fg">.</span>end<span class="ansi-blue-fg">(</span><span class="ansi-blue-fg">)</span>
<span class="ansi-green-intense-fg ansi-bold">    339</span>         <span class="ansi-green-fg">if</span> end <span class="ansi-blue-fg">!=</span> len<span class="ansi-blue-fg">(</span>s<span class="ansi-blue-fg">)</span><span class="ansi-blue-fg">:</span>
<span class="ansi-green-fg">--&gt; 340</span><span class="ansi-red-fg">             </span><span class="ansi-green-fg">raise</span> JSONDecodeError<span class="ansi-blue-fg">(</span><span class="ansi-blue-fg">&#34;Extra data&#34;</span><span class="ansi-blue-fg">,</span> s<span class="ansi-blue-fg">,</span> end<span class="ansi-blue-fg">)</span>
<span class="ansi-green-intense-fg ansi-bold">    341</span>         <span class="ansi-green-fg">return</span> obj
<span class="ansi-green-intense-fg ansi-bold">    342</span> 

<span class="ansi-red-fg">JSONDecodeError</span>: Extra data: line 2 column 1 (char 1734)</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p><strong>Method 1</strong>: Read by using a <code>for loop</code> and <code>.append</code>.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[9]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># use for loop with .append</span>
<span class="n">part1</span> <span class="o">=</span> <span class="s1">&#39;/mnt/data/public/covid19-lake/alleninstitute/CORD19/json/&#39;</span>
<span class="n">part2</span> <span class="o">=</span> <span class="s1">&#39;metadata/part-00000-81803174-7752-4489-8eeb-081318af9653&#39;</span>
<span class="n">part3</span> <span class="o">=</span> <span class="s1">&#39;-c000.json&#39;</span>
<span class="n">file_path</span> <span class="o">=</span> <span class="n">part1</span> <span class="o">+</span> <span class="n">part2</span> <span class="o">+</span> <span class="n">part3</span>
<span class="n">json_lines</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">line</span> <span class="ow">in</span> <span class="nb">open</span><span class="p">(</span><span class="n">file_path</span><span class="p">,</span> <span class="s1">&#39;r&#39;</span><span class="p">):</span>
    <span class="n">json_lines</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">json</span><span class="o">.</span><span class="n">loads</span><span class="p">(</span><span class="n">line</span><span class="p">))</span>
<span class="n">json_lines</span>

<span class="c1"># result is a list of dictionaries</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[9]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>[{&#39;cord_uid&#39;: &#39;vho70jcx&#39;,
  &#39;sha&#39;: &#39;f056da9c64fbf00a4645ae326e8a4339d015d155&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SIANN: Strain Identification by Alignment to Near Neighbors&#39;,
  &#39;doi&#39;: &#39;10.1101/001727&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.&#39;,
  &#39;publish_time&#39;: &#39;2014-01-10&#39;,
  &#39;authors&#39;: &#39;Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/001727&#39;},
 {&#39;cord_uid&#39;: &#39;i9tbix2v&#39;,
  &#39;sha&#39;: &#39;daf32e013d325a6feb80e83d15aabc64a48fae33&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Spatial epidemiology of networked metapopulation: An overview&#39;,
  &#39;doi&#39;: &#39;10.1101/003889&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.&#39;,
  &#39;publish_time&#39;: &#39;2014-06-04&#39;,
  &#39;authors&#39;: &#39;Lin WANG; Xiang Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/003889&#39;},
 {&#39;cord_uid&#39;: &#39;62gfisc6&#39;,
  &#39;sha&#39;: &#39;f33c6d94b0efaa198f8f3f20e644625fa3fe10d2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity&#39;,
  &#39;doi&#39;: &#39;10.1101/006866&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.&#39;,
  &#39;publish_time&#39;: &#39;2014-07-03&#39;,
  &#39;authors&#39;: &#39;Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/006866&#39;},
 {&#39;cord_uid&#39;: &#39;058r9486&#39;,
  &#39;sha&#39;: &#39;4da8a87e614373d56070ed272487451266dce919&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Bayesian mixture analysis for metagenomic community profiling.&#39;,
  &#39;doi&#39;: &#39;10.1101/007476&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix&#39;,
  &#39;publish_time&#39;: &#39;2014-07-25&#39;,
  &#39;authors&#39;: &#39;Sofia Morfopoulou; Vincent Plagnol&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/007476&#39;},
 {&#39;cord_uid&#39;: &#39;wich35l7&#39;,
  &#39;sha&#39;: &#39;eccef80cfbe078235df22398f195d5db462d8000&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Mapping a viral phylogeny onto outbreak trees to improve host transmission inference&#39;,
  &#39;doi&#39;: &#39;10.1101/010389&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus&#39; spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.&#34;,
  &#39;publish_time&#39;: &#39;2014-11-11&#39;,
  &#39;authors&#39;: &#39;Stephen P Velsko; Jonathan E Allen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/010389&#39;},
 {&#39;cord_uid&#39;: &#39;z3tgnzth&#39;,
  &#39;sha&#39;: &#39;c41fdb2efd6d61384a92a84cbba3f8233629a41b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The infant airway microbiome in health and disease impacts later asthma development&#39;,
  &#39;doi&#39;: &#39;10.1101/012070&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.&#39;,
  &#39;publish_time&#39;: &#39;2014-12-02&#39;,
  &#39;authors&#39;: &#39;Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochkov; Kristine Grindle; Sebastian L Johnston; James E Gern; Peter D Sly; Patrick G Holt; Kathryn E Holt; Michael Inouye&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/012070&#39;},
 {&#39;cord_uid&#39;: &#39;1xxrnpg3&#39;,
  &#39;sha&#39;: &#39;1dd898b5ca1ae70ec0e3cad89fc87a165002a99e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Using heterogeneity in the population structure of U.S. swine farms to compare transmission models for porcine epidemic diarrhoea&#39;,
  &#39;doi&#39;: &#39;10.1101/017178&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTIn 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread.&#39;,
  &#39;publish_time&#39;: &#39;2015-03-27&#39;,
  &#39;authors&#39;: &#39;Eamon B. O’Dea; Harry Snelson; Shweta Bansal&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/017178&#39;},
 {&#39;cord_uid&#39;: &#39;8ilzm51q&#39;,
  &#39;sha&#39;: &#39;33565294e6bc67fb7ee14dcae6cfdb08148f4ea5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Big city, small world: Density, contact rates, and transmission of dengue across Pakistan.&#39;,
  &#39;doi&#39;: &#39;10.1101/018481&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Macroscopic descriptions of populations commonly assume that encounters between individuals are well mixed; i.e., each individual has an equal chance of coming into contact with any other individual. Relaxing this assumption can be challenging though, due to the difficulty of acquiring detailed knowledge about the non-random nature of encounters. Here, we fitted a mathematical model of dengue virus transmission to spatial time series data from Pakistan and compared maximum-likelihood estimates of “mixing parameters” when disaggregating data across an urban-rural gradient. We show that dynamics across this gradient are subject not only to differing transmission intensities but also to differing strengths of nonlinearity due to differences in mixing. We furthermore show that neglecting spatial variation in mixing can lead to substantial underestimates of the level of effort needed to control a pathogen with vaccines or other control efforts. We complement this analysis with relevant contemporary environmental drivers of dengue.&#39;,
  &#39;publish_time&#39;: &#39;2015-04-27&#39;,
  &#39;authors&#39;: &#39;Moritz U. G. Kraemer; T. Alex Perkins; Derek A.T. Cummings; Rubeena Zakar; Simon I. Hay; David L. Smith; Robert C. Reiner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/018481&#39;},
 {&#39;cord_uid&#39;: &#39;wafvnbdu&#39;,
  &#39;sha&#39;: &#39;3461d71f6890f7e5ba53bf168be3945cdb16d901&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;MERS-CoV recombination: implications about the reservoir and potential for adaptation&#39;,
  &#39;doi&#39;: &#39;10.1101/020834&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.&#39;,
  &#39;publish_time&#39;: &#39;2015-06-12&#39;,
  &#39;authors&#39;: &#39;Gytis Dudas; Andrew Rambaut&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/020834&#39;},
 {&#39;cord_uid&#39;: &#39;4xocqn6o&#39;,
  &#39;sha&#39;: &#39;1f9d3f9a1a0e8db6a086e0a2b5ba50cf9f235dae&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;On the causes of evolutionary transition:transversion bias&#39;,
  &#39;doi&#39;: &#39;10.1101/027722&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the &#34;conservative transitions&#34; hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.&#39;,
  &#39;publish_time&#39;: &#39;2015-09-28&#39;,
  &#39;authors&#39;: &#39;Arlin Stoltzfus; Ryan W. Norris&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/027722&#39;},
 {&#39;cord_uid&#39;: &#39;q5iylws9&#39;,
  &#39;sha&#39;: &#39;01e3b313e78a352593be2ff64927192af66619b5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Viruses are a dominant driver of protein adaptation in mammals&#39;,
  &#39;doi&#39;: &#39;10.1101/029397&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation against viruses has only been studied in a few proteins specialized in antiviral defense. Whether adaptation to viruses typically involves only specialized antiviral proteins or affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 virus-interacting proteins manually curated from a set of 9,900 proteins conserved across mammals. We show that viruses (i) use the more evolutionarily constrained proteins from the cellular functions they hijack and that (ii) despite this high constraint, virus-interacting proteins account for a high proportion of all protein adaptation in humans and other mammals. Adaptation is elevated in virus-interacting proteins across all functional categories, including both immune and non-immune functions. Our results demonstrate that viruses are one of the most dominant drivers of evolutionary change across mammalian and human proteomes.&#39;,
  &#39;publish_time&#39;: &#39;2015-10-18&#39;,
  &#39;authors&#39;: &#39;David Enard; Le Cai; Carina Gwenapp; Dmitri A Petrov&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/029397&#39;},
 {&#39;cord_uid&#39;: &#39;nljskxut&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#34;What&#39;s in my pot? Real-time species identification on the MinION&#34;,
  &#39;doi&#39;: &#39;10.1101/030742&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What′s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies′ MinION ™ sensing device with a real-time species identification bioinformatics application.&#39;,
  &#39;publish_time&#39;: &#39;2015-11-06&#39;,
  &#39;authors&#39;: &#39;Sissel Juul; Fernando Izquierdo; Adam Hurst; Xiaoguang Dai; Amber Wright; Eugene Kulesha; Roger Pettett; Daniel J Turner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/030742&#39;},
 {&#39;cord_uid&#39;: &#39;lfm6erzy&#39;,
  &#39;sha&#39;: &#39;3b73c8abb7528c958cec0bae1890828af007b599&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;How domain growth is implemented determines the long term behaviour of a cell population through its effect on spatial correlations&#39;,
  &#39;doi&#39;: &#39;10.1101/041509&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDomain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations.&#39;,
  &#39;publish_time&#39;: &#39;2016-02-26&#39;,
  &#39;authors&#39;: &#39;Robert J. H. Ross; R. E. Baker; C. A. Yates&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/041509&#39;},
 {&#39;cord_uid&#39;: &#39;lcpp5fim&#39;,
  &#39;sha&#39;: &#39;8cfb8dfc2f8b2c6e1d60ff48da02de6670694869&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Productive infection of field strains of avian coronavirus infectious bronchitis virus in chicken peripheral blood-derived monocyte&#39;,
  &#39;doi&#39;: &#39;10.1101/041558&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood-derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis.&#39;,
  &#39;publish_time&#39;: &#39;2016-02-26&#39;,
  &#39;authors&#39;: &#39;Yueting Zhang; Gary Whittaker&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/041558&#39;},
 {&#39;cord_uid&#39;: &#39;gjxumrmm&#39;,
  &#39;sha&#39;: &#39;6499f5c865c3db3bdfc1e969ea940a80d547a5bb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Zika virus outbreak in the Americas: Is Aedes albopictus an overlooked culprit?&#39;,
  &#39;doi&#39;: &#39;10.1101/044594&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Summary / AbstractCodon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors.&#39;,
  &#39;publish_time&#39;: &#39;2016-03-30&#39;,
  &#39;authors&#39;: &#39;Azeem Mehmood Butt; Izza Nasrullah; Raheel Qamar; Yigang Tong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/044594&#39;},
 {&#39;cord_uid&#39;: &#39;csokkcqq&#39;,
  &#39;sha&#39;: &#39;a59906b732bf4a489e282c3e4f499d4166c622e7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim&#39;,
  &#39;doi&#39;: &#39;10.1101/046532&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundIn silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics algorithms. Sequenced datasets include background organisms, creating ambiguity in the true source organism for each read. Bacterial and viral datasets were created with even and staggered coverage to evaluate organism identification, read mapping, and gene identification capabilities of available algorithms. These truth datasets are provided as a resource for the development and refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform decision makers on strengths and weaknesses of available algorithms and how the results may be best leveraged for bacterial and viral organism identification and characterization.Source organisms were selected to mirror communities described in the Human Microbiome Project as well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler.ResultsAlgorithms were evaluated on runtime, true positive organisms identified to the genus and species levels, false positive organisms identified to genus and species level, read mapping, relative abundance estimation, and gene calling. No algorithm out performed the others in all categories, and the algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria and LMAT for viruses. The algorithms were ranked by overall performance using a normalized weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken and LMAT.ConclusionsSimulated FASTQ datasets with well-characterized truth data about microbial community composition reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms evaluated. The simulated datasets are available to download from the Sequence Read Archive (SRP062063).&#39;,
  &#39;publish_time&#39;: &#39;2016-03-31&#39;,
  &#39;authors&#39;: &#39;Anna Shcherbina; Darrell O. Ricke; Nelson Chiu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/046532&#39;},
 {&#39;cord_uid&#39;: &#39;08mep8hm&#39;,
  &#39;sha&#39;: &#39;99dab788e984601ee4efcd1138926caf10c00dcf&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension&#39;,
  &#39;doi&#39;: &#39;10.1101/048801&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (fs) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm)α with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.&#39;,
  &#39;publish_time&#39;: &#39;2016-04-15&#39;,
  &#39;authors&#39;: &#39;Naoto Hori; Natalia A. Denesyuk; D. Thirumalai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/048801&#39;},
 {&#39;cord_uid&#39;: &#39;ejqq2wmi&#39;,
  &#39;sha&#39;: &#39;97339c9d365437d54ea33be1a1b634d35f460208&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study&#39;,
  &#39;doi&#39;: &#39;10.1101/053421&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;BackgroundThe ongoing yellow fever (YF) epidemic in Angola strains the global vaccine supply, prompting WHO to adopt dose sparing for its vaccination campaign in Kinshasa in July-August 2016. Although a 5-fold fractional-dose vaccine is similar to standard-dose vaccine in safety and immunogenicity, efficacy is untested. There is an urgent need to ensure the robustness of fractional-dose vaccination by elucidating the conditions under which dose fractionation would reduce transmission.MethodsWe estimate the effective reproductive number for YF in Angola using disease natural history and case report data. With simple mathematical models of YF transmission, we calculate the infection attack rate (IAR, the proportion of population infected over the course of an epidemic) under varying levels of transmissibility and five-fold fractional-dose vaccine efficacy for two vaccination scenarios: (i) random vaccination in a hypothetical population that is completely susceptible; (ii) the Kinshasa vaccination campaign in July-August 2016 with different age cutoff for fractional-dose vaccines.FindingsWe estimate the effective reproductive number early in the Angola outbreak was between 5·2 and 7·1. If vaccine action is all-or-nothing (i.e. a proportion VE of vaccinees receives complete and the remainder receive no protection), n-fold fractionation can dramatically reduce IAR as long as efficacy VE exceeds 1/n. This benefit threshold becomes more stringent if vaccine action is leaky (i.e. the susceptibility of each vaccinee is reduced by a factor that is equal to the vaccine efficacy VE). The age cutoff for fractional-dose vaccines chosen by the WHO for the Kinshasa vaccination campaign (namely, 2 years) provides the largest reduction in IAR if the efficacy of five-fold fractional-dose vaccines exceeds 20%.InterpretationDose fractionation is a very effective strategy for reducing infection attack rate that would be robust with a large margin for error in case fractional-dose VE is lower than expected.FundingNIH-MIDAS, HMRF-Hong Kong&#39;,
  &#39;publish_time&#39;: &#39;2016-05-14&#39;,
  &#39;authors&#39;: &#39;Joseph T. Wu; Corey M. Peak; Gabriel M. Leung; Marc Lipsitch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/053421&#39;},
 {&#39;cord_uid&#39;: &#39;2eh0bhee&#39;,
  &#39;sha&#39;: &#39;1c0b25c8e3c89243fd8797e2e964e12689bc8556&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;50-valent inactivated rhinovirus vaccine is broadly immunogenic in rhesus macaques&#39;,
  &#39;doi&#39;: &#39;10.1101/053967&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAs the predominant etiological agent of the common cold, human rhinovirus (HRV) is the leading cause of human infectious disease. Early studies showed monovalent formalin-inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated with protection. However, co-circulation of many HRV types discouraged further vaccine efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, we show that serum nAb against many rhinovirus types can be induced by polyvalent, inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient quantity of input antigens, and valency was not a major factor for potency or breadth of the response. We for the first time generated a vaccine capable of inducing nAb responses to numerous and diverse HRV types.&#39;,
  &#39;publish_time&#39;: &#39;2016-05-17&#39;,
  &#39;authors&#39;: &#39;Sujin Lee; Minh Trang Nguyen; Michael G. Currier; Joe B. Jenkins; Elizabeth A. Strobert; Adriana E. Kajon; Ranjna Madan-Lala; Yury A. Bochkov; James E. Gern; Krishnendu Roy; Xiaoyan Lu; Dean D. Erdman; Paul Spearman; Martin L. Moore&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/053967&#39;},
 {&#39;cord_uid&#39;: &#39;76i8yvb4&#39;,
  &#39;sha&#39;: &#39;d1b26388670660f66249b55ece2beccee4c18d1a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evidence for reassortment of highly divergent novel rotaviruses from bats in Cameroon, without evidence for human interspecies transmissions&#39;,
  &#39;doi&#39;: &#39;10.1101/054072&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBats are an important reservoir for pathogenic human respiratory and hemorrhagic viruses but only little is known about bat viruses causing gastroenteritis in humans, including rotavirus A strains (RVA). Only three RVA strains have been reported in bats in Kenya (straw-colored fruit bat) and in China (lesser horseshoe and a stoliczka’s trident bat), being highly divergent from each other. To further elucidate the potential of bat RVAs to cause gastroenteritis in humans we started by investigating the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using metagenomics. Five samples contained significant numbers of RVA Illumina reads, sufficient to obtain their (near) complete genomes. A single RVA strain showed a close phylogenetic relationship with the Kenyan bat RVA strain in six gene segments, including VP7 (G25), whereas the other gene segments represented novel genotypes as ratified by the RCWG. The 4 other RVA strains were highly divergent from known strains (but very similar among each other) possessing all novel genotypes. Only the VP7 and VP4 genes showed a significant variability representing multiple novel G and P genotypes, indicating the frequent occurrence of reassortment events.Comparing these bat RVA strains with currently used human RVA screening primers indicated that several of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel VP6 based screening primers matching both human and bat RVAs were developed and used to screen samples from 25 infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, Sanger sequencing did not indicate evidence of interspecies transmissions.This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans.&#39;,
  &#39;publish_time&#39;: &#39;2016-05-18&#39;,
  &#39;authors&#39;: &#39;Claude Kwe Yinda; Mark Zeller; Nádia Conceição-Neto; Piet Maes; Ward Deboutte; Leen Beller; Elisabeth Heylen; Stephen Mbigha Ghogomu; Marc Van Ranst; Jelle Matthijnssens&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/054072&#39;},
 {&#39;cord_uid&#39;: &#39;omtmy9z6&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The ATP synthase subunit β (ATP5B) is an entry factor for the hepatitis E virus&#39;,
  &#39;doi&#39;: &#39;10.1101/060434&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractHepatitis E occurs sporadically and as outbreaks due to contamination of drinking water. The causative agent, hepatitis E virus (HEV) is a hepatotropic non-enveloped RNA virus, which grows poorly in vitro. Consequently, many aspects of HEV biology are poorly characterized, including its cellular receptor and entry mechanism(s). Previous studies from our laboratory have shown that heparan sulfate proteoglycans (HSPGs) act as attachment factors for the virus. In the absence of purified high titer infectious virus, we have used hepatitis E virus-like particles (HEV-LPs) expressed and purified from E. coli to identify HEV entry factor(s) on liver cells in culture. Using a pull down and mass spectrometric approach, we identified the ATP synthase subunit β (ATP5B) to bind the HEV capsid protein. Its role in the entry of HEV was then validated using antibody and siRNA mediated approaches, and infectious HEV from the stools of a hepatitis E patient. Though ATP synthase is largely a mitochondrial protein, the cell surface expressed form of ATP5B is implicated in other viral infections.&#39;,
  &#39;publish_time&#39;: &#39;2016-06-23&#39;,
  &#39;authors&#39;: &#39;Zulfazal Ahmed; Prasida Holla; Imran Ahmad; Shahid Jameel&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/060434&#39;},
 {&#39;cord_uid&#39;: &#39;2cerplno&#39;,
  &#39;sha&#39;: &#39;53442eacc3f233078507fa37b78267399e8c1e3b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/061788&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTZika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes.&#39;,
  &#39;publish_time&#39;: &#39;2016-07-01&#39;,
  &#39;authors&#39;: &#39;Arunachalam Ramaiah; Deisy Contreras; Vineela Gangalapudi; Masumi Sameer Padhye; Jie Tang; Vaithilingaraja Arumugaswami&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/061788&#39;},
 {&#39;cord_uid&#39;: &#39;t3tm6f9p&#39;,
  &#39;sha&#39;: &#39;c04e10858012bdd666d50c0d69c3d1e7224ccbea&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Design: An assay based on single-polypeptide-chain heterodimeric A2AR/D2R and non-oligomerized fusions for in vivo analysis of their allosteric receptor-receptor interactions&#39;,
  &#39;doi&#39;: &#39;10.1101/065250&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundThe adenosine A2A receptor (A2AR) heteromerizes with the dopamine D2 receptor (D2R). In order to explore their functional interaction, we engineered previously stable single-polypeptide-chain (sc) A2AR/D2LR: whether the molecular entity of the striatal A2AR/D2R antagonism, i.e., scA2AR/D2Rs are just A2AR/D2R with the antagonism, remains unresolved.New MethodTo further clarify the heteromerization through the scA2AR/D2LR, we here designed supramolecularly ‘exclusive’ monomers and dimers, using the Cε2 domain of IgE-Fc or apoproteins of the bacterial light-harvesting antenna complex.ResultsA concept of the recptor protein assembly regulation, i.e., the selective monomer/non-obligate dimer formation was obtained. Although none of these new fusions were constructed or tested, we could aim at obtaining heterodimer-specific agents, using the scA2AR/D2R. Whether the resulting designs were explained feasibly and rationally was addressed. The structure and function of the non-obligate dimer were here discussed through scA2AR/D2R, focusing on the procedure of the membrane protein design and methods for transient protein-protein interactions.Summary and OutlookGiven that upon being expressed and allosteric regulation occurs regardless of specific signal to non-specific noise (S/N) ratio, the supramolecular designs, allowing us to express selectively monomer/non-obligate dimer of class A GPCR, are experimentally testable and will be used to confirm in vivo that such low S/N ratio interaction between A2AR and D2LR functions in the dopamine neurotransmission in the striatum.Abbreviations:A2ARadenosine A2A receptor3HA-A2ARA2AR tagged with a triple HA epitopeBchlbacteriochlorophyllBRETbioluminescence resonance energy transferCcarboxy-terminalCDcluster of differentiationD2LR and D2SRthe long and short form of dopamine D2 receptor, respectivelyFabantigen binding fragmentFcFc fragmentFcεRIhigh affinity receptor for IgEFRETfluorescence resonance energy transferG4San amino acid sequence consisting of a four-glycine-repeat followed by a serine residueGABAγ-aminobutyric acidGABABGABA type B receptorGPCRG protein-coupled receptorGttransducinHAhemagglutininHIVhuman immunodeficiency virusICintracellular loopsIgimmunoglobulinLHlight-harvesting antenna complexmAbmonoclonal antibodyMrmolecular weightNamino-terminalPDParkinson’s diseasePSphotosystemRCreaction centerRlucRenilla luciferasescsingle-chainTMtransmembrane3Dthree-dimensional&#39;,
  &#39;publish_time&#39;: &#39;2016-07-26&#39;,
  &#39;authors&#39;: &#39;Toshio Kamiya; Takashi Masuko; Dasiel Oscar Borroto-Escuela; Haruo Okado; Hiroyasu Nakata&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/065250&#39;},
 {&#39;cord_uid&#39;: &#39;57sp9d9l&#39;,
  &#39;sha&#39;: &#39;5570cb9ff2905fda60cfd6e94f3c32774b765265&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The genome of the crustacean Parhyale hawaiensis: a model for animal development, regeneration, immunity and lignocellulose digestion&#39;,
  &#39;doi&#39;: &#39;10.1101/065789&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe amphipod crustacean Parhyale hawaiensis is a blossoming model system for studies of developmental mechanisms and more recently regeneration. We have sequenced the genome allowing annotation of all key signaling pathways, transcription factors, and non-coding RNAs that will enhance ongoing functional studies. Parhyale is a member of the Malacostraca clade, which includes crustacean food crop species. We analysed the immunity related genes of Parhyale as an important comparative system for these species, where immunity related aquaculture problems have increased as farming has intensified. We also find that Parhyale and other species within Multicrustacea contain the enzyme sets necessary to perform lignocellulose digestion (“wood eating”), suggesting this ability may predate the diversification of this lineage. Our data provide an essential resource for further development of Parhyale as an experimental model. The first malacostracan genome will underpin ongoing comparative work in food crop species and research investigating lignocellulose as an energy source.&#39;,
  &#39;publish_time&#39;: &#39;2016-07-25&#39;,
  &#39;authors&#39;: &#39;Damian Kao; Alvina G. Lai; Evangelia Stamataki; Silvana Rosic; Nikolaos Konstantinides; Erin Jarvis; Alessia Di Donfrancesco; Natalia Pouchkina-Stantcheva; Marie Sémon; Marco Grillo; Heather Bruce; Suyash Kumar; Igor Siwanowicz; Andy Le; Andrew Lemire; Michael B. Eisen; Cassandra Extavour; William E. Browne; Carsten Wolff; Michalis Averof; Nipam H. Patel; Peter Sarkies; Anastasios Pavlopoulos; A. Aziz Aboobaker&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/065789&#39;},
 {&#39;cord_uid&#39;: &#39;tiyjmih7&#39;,
  &#39;sha&#39;: &#39;45f184e736c4a857bc0e3df0d55962f743c1493a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Similar ratios of introns to intergenic sequence across animal genomes&#39;,
  &#39;doi&#39;: &#39;10.1101/068627&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractOne central goal of genome biology is to understand how the usage of the genome differs between organisms. Our knowledge of genome composition, needed for downstream inferences, is critically dependent on gene annotations, yet problems associated with gene annotation and assembly errors are usually ignored in comparative genomics. Here we analyze the genomes of 68 species across 12 animal phyla and some single-cell eukaryotes for general trends in genome composition and transcription, taking into account problems of gene annotation. We show that, regardless of genome size, the ratio of introns to intergenic sequence is comparable across essentially all animals, with nearly all deviations dominated by increased intergenic sequence. Genomes of model organisms have ratios much closer to 1:1, suggesting that the majority of published genomes of non-model organisms are underannotated and consequently omit substantial numbers of genes, with likely negative impact on evolutionary interpretations. Finally, our results also indicate that most animals transcribe half or more of their genomes arguing against differences in genome usage between animal groups, and also suggesting that the transcribed portion is more dependent on genome size than previously thought.&#39;,
  &#39;publish_time&#39;: &#39;2016-08-09&#39;,
  &#39;authors&#39;: &#39;Warren R. Francis; Gert Wörheide&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/068627&#39;},
 {&#39;cord_uid&#39;: &#39;30kwl4rj&#39;,
  &#39;sha&#39;: &#39;ffbf8ea9948d73572fd052a74afa01b19e6758a3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Planning horizon affects prophylactic decision-making and epidemic dynamics&#39;,
  &#39;doi&#39;: &#39;10.1101/069013&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;AbstractHuman behavior can change the spread of infectious disease. There is limited understanding of how the time in the future over which individuals make a behavioral decision, their planning horizon, affects epidemic dynamics. We developed an agent-based model (along with an ODE analog) to explore the decision-making of self-interested individuals on adopting prophylactic behavior. The decision-making process incorporates prophylaxis efficacy and disease prevalence with individuals&#39; payoffs and planning horizon. Our results show that for short and long planning horizons individuals do not consider engaging in prophylactic behavior. In contrast, individuals adopt prophylactic behavior when considering intermediate planning horizons. Such adoption, however, is not always monotonically associated with the prevalence of the disease, depending on the perceived protection efficacy and the disease parameters. Adoption of prophylactic behavior reduces the peak size while prolonging the epidemic and potentially generates secondary waves of infection. These effects can be made stronger by increasing the behavioral decision frequency or distorting an individual’s perceived risk of infection.&#34;,
  &#39;publish_time&#39;: &#39;2016-08-12&#39;,
  &#39;authors&#39;: &#39;Luis G. Nardin; Craig R. Miller; Benjamin J. Ridenhour; Stephen M. Krone; Paul Joyce; Bert O. Baumgaertner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/069013&#39;},
 {&#39;cord_uid&#39;: &#39;1znuxoj5&#39;,
  &#39;sha&#39;: &#39;fd3fc2c49f5cc27e4262261d0c9045911d65cb6e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;XRN1 is a Species-Specific Virus Restriction Factor in Yeasts&#39;,
  &#39;doi&#39;: &#39;10.1101/069799&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn eukaryotes, the degradation of cellular mRNAs is accomplished by Xrn1p and the cytoplasmic exosome. Because viral RNAs often lack canonical caps or poly-A tails, they can also be vulnerable to degradation by these host exonucleases. Yeast lack sophisticated mechanisms of innate and adaptive immunity, but do use RNA degradation as an antiviral defense mechanism. One model is that the RNA of yeast viruses is subject to degradation simply as a side effect of the intrinsic exonuclease activity of proteins involved in RNA metabolism. Contrary to this model, we find a highly refined, species-specific relationship between Xrn1p and the “L-A” totiviruses of different Saccharomyces yeast species. We show that the gene XRN1 has evolved rapidly under positive natural selection in Saccharomyces yeast, resulting in high levels of Xrn1p protein sequence divergence from one yeast species to the next. We also show that these sequence differences translate to differential interactions with the L-A virus, where Xrn1p from S. cerevisiae is most efficient at controlling the L-A virus that chronically infects S. cerevisiae, and Xrn1p from S. kudriavzevii is most efficient at controlling the L-A-like virus that we have discovered within S. kudriavzevii. All Xrn1p orthologs are equivalent in their interaction with another virus-like parasite, the Ty1 retrotransposon. Thus, the activity of Xrn1p against totiviruses is not simply an incidental consequence of the enzymatic activity of Xrn1p, but rather Xrn1p co-evolves with totiviruses to maintain its potent antiviral activity and limit viral propagation in Saccharomyces yeasts. Consistent with this, we demonstrated that Xrn1p physically interacts with the Gag protein encoded by the L-A virus, suggesting a host-virus interaction that is more complicated than just Xrn1p-mediated nucleolytic digestion of viral RNAs.&#39;,
  &#39;publish_time&#39;: &#39;2016-08-16&#39;,
  &#39;authors&#39;: &#39;Paul A. Rowley; Brandon Ho; Sarah Bushong; Arlen Johnson; Sara L. Sawyer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/069799&#39;},
 {&#39;cord_uid&#39;: &#39;pxpujrhx&#39;,
  &#39;sha&#39;: &#39;44a440cc1c135c938d2216ea672f8ef4f9c01296&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genome-wide Prediction of microRNAs in Zika virus Genomes Reveals Possible Interactions with Human Genes Involved in the Nervous System Development&#39;,
  &#39;doi&#39;: &#39;10.1101/070656&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractZika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered a large epidemic in Brazil and spread across Latin America. In November of that year, the Brazilian Ministry of Health reported a 20-fold increase in cases of neonatal microcephaly, which corresponds geographically and temporally to the ZIKV outbreak. ZIKV was isolated from the brain tissue of a fetus diagnosed with microcephaly, and recent studies in mice models revealed that ZIKV infection may cause brain defects by influencing brain cell developments. Unfortunately, the mechanisms by which ZIKV alters neurophysiological development remain unknown. MicroRNAs (miRNAs) are small noncoding RNAs that regulate post-transcriptional gene expression by translational repression. In order to gain insight into the possible role of ZIKV-mediated miRNA signaling dysfunction in brain-tissue development, we computationally predicted new miRNAs encoded by the ZIKV genome and their effective hybridization with transcripts from human genes previously shown to be involved in microcephalia. The results of these studies suggest a possible role of these miRNAs on the expression of human genes associated with this disease. Besides, a new ZIKV miRNA was predicted in the 3’stem loop (3’ SL) of the 3’untranslated region (3’UTR) of the ZIKV genome, suggesting the role of the 3’UTR of flaviviruses as a source of miRNAs.&#39;,
  &#39;publish_time&#39;: &#39;2016-08-21&#39;,
  &#39;authors&#39;: &#39;Juan Cristina; Natalia Echeverría; Fabiana Gambaro; Alvaro Fajardo; Pilar Moreno&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/070656&#39;},
 {&#39;cord_uid&#39;: &#39;2j4z5rp8&#39;,
  &#39;sha&#39;: &#39;9b5f5119bbfbded3245acc37859cefde967458e7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Containing Emerging Epidemics: a Quantitative Comparison of Quarantine and Symptom Monitoring&#39;,
  &#39;doi&#39;: &#39;10.1101/072652&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTStrategies for containing an emerging infectious disease outbreak must be non-pharmaceutical when drugs or vaccines for the pathogen do not yet exist or are unavailable. The success of these non-pharmaceutical strategies will depend not only on the effectiveness of quarantine or other isolation measures but also on the epidemiological characteristics of the infection. However, there is currently no systematic framework to assess the relationship between different containment strategies and the natural history and epidemiological dynamics of the pathogen. Here, we compare the effectiveness of quarantine and symptom monitoring, implemented via contact tracing, in controlling epidemics using an agent-based branching model. We examine the relationship between epidemic containment and the disease dynamics of symptoms and infectiousness for seven case study diseases with diverse natural histories including Ebola, Influenza A, and Severe Acute Respiratory Syndrome (SARS). We show that the comparative effectiveness of symptom monitoring and quarantine depends critically on the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. The benefit of quarantine over symptom monitoring is generally maximized for fast-course diseases, but we show the conditions under which symptom monitoring alone can control certain outbreaks. This quantitative framework can guide policy-makers on how best to use non-pharmaceutical interventions to contain emerging outbreaks and prioritize research during an outbreak of a novel pathogen.SIGNIFICANCEQuarantine and symptom monitoring of contacts with suspected exposure to an infectious disease are key interventions for the control of emerging epidemics; however, there does not yet exist a quantitative framework for comparing the control performance of each. Here, we use a mathematical model of seven case study diseases to show how the choice of intervention is influenced by the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. We use this information to identify the most important characteristics of the disease and setting that need to be characterized for an emerging pathogen in order to make an informed decision between quarantine and symptom monitoring.&#39;,
  &#39;publish_time&#39;: &#39;2016-08-31&#39;,
  &#39;authors&#39;: &#39;Corey M Peak; Lauren M Childs; Yonatan H Grad; Caroline O Buckee&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/072652&#39;},
 {&#39;cord_uid&#39;: &#39;kxe65hd7&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Zika virus infection as a cause of congenital brain abnormalities and Guillain-Barré syndrome: systematic review&#39;,
  &#39;doi&#39;: &#39;10.1101/073098&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundThe World Health Organization stated in March 2016 that there was scientific consensus that the mosquito-borne Zika virus was a cause of the neurological disorder Guillain-Barré syndrome and of microcephaly and other congenital brain abnormalities, based on rapid evidence assessments. Decisions about causality require systematic assessment to guide public health actions. The objectives of this study were: to update and re-assess the evidence for causality through a rapid and systematic review about links between Zika virus infection and a) congenital brain abnormalities, including microcephaly, in the foetuses and offspring of pregnant women and b) Guillain-Barré syndrome in any population; and to describe the process and outcomes of an expert assessment of the evidence about causality.Methods and findingsThe study had three linked components. First, in February 2016, we developed a causality framework that defined questions about the relationship between Zika virus infection and each of the two clinical outcomes in 10 dimensions; temporality, biological plausibility, strength of association, alternative explanations, cessation, dose-response, animal experiments, analogy, specificity and consistency. Second, we did a systematic review (protocol number CRD42016036693). We searched multiple online sources up to May 30, 2016 to find studies that directly addressed either outcome and any causality dimension, used methods to expedite study selection, data extraction and quality assessment, and summarised evidence descriptively. Third, a multidisciplinary panel of experts assessed the review findings and reached consensus on causality. We found 1091 unique items up to May 30, 2016. For congenital brain abnormalities, including microcephaly, we included 72 items; for eight of 10 causality dimensions (all except dose-response relationship and specificity) we found that more than half the relevant studies supported a causal association with Zika virus infection. For Guillain-Barré syndrome, we included 36 items, of which more than half the relevant studies supported a causal association in seven of ten dimensions (all except dose-response relationship, specificity and animal experimental evidence). Articles identified non-systematically from May 30-July 29, 2016 strengthened the review findings. The expert panel concluded that: a) the most likely explanation of available evidence from outbreaks of Zika virus infection and clusters of microcephaly is that Zika virus infection during pregnancy is a cause of congenital brain abnormalities including 61 microcephaly; and b) the most likely explanation of available evidence from outbreaks of Zika virus infection and Guillain-Barré syndrome is that Zika virus infection is a trigger of Guillain-Barré syndrome. The expert panel recognised that Zika virus alone may not be sufficient to cause either congenital brain abnormalities or Guillain-Barré syndrome but agreed that the evidence was sufficient to recommend increased public health measures. Weaknesses are the limited assessment of the role of dengue virus and other possible co-factors, the small number of comparative epidemiological studies, and the difficulty in keeping the review up to date with the pace of publication of new research.ConclusionsRapid and systematic reviews with frequent updating and open dissemination are now needed, both for appraisal of the evidence about Zika virus infection and for the next public health threats that will emerge. This rapid systematic review found sufficient evidence to say that Zika virus is a cause of congenital abnormalities and is a trigger of Guillain-Barré situation.&#39;,
  &#39;publish_time&#39;: &#39;2016-09-02&#39;,
  &#39;authors&#39;: &#39;Fabienne Krauer; Maurane Riesen; Ludovic Reveiz; Olufemi T Oladapo; Ruth Martínez-Vega; Teegwendé V. Porgo; Anina Haefliger; Nathalie J Broutet; Nicola Low&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/073098&#39;},
 {&#39;cord_uid&#39;: &#39;8elmjre9&#39;,
  &#39;sha&#39;: &#39;8f4b98da50277e6c9516aef0227f5cca9af6b5e8&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Identification of quercetin from fruits to immediately fight Zika&#39;,
  &#39;doi&#39;: &#39;10.1101/074559&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Zika virus is spread mainly by the bite of an infected mosquito, which can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika in the near future, but unfortunately so far there is no vaccine or medicine for Zika. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the key role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. In response to the current global health emergency triggered by the Zika outbreak, we successfully obtained several active forms of Zika NS2B-NS3pro and further attempted to discover its inhibitors from eatable plants and traditional herbal medicines to immediately fight Zika. Here, for the first time, we discovered that quercetin, a flavonoid extensively existing in many fruits and vegetables, effectively inhibits Zika NS2B-NS3pro. We further quantify its inhibitory activity with IC50 of 26 uM; and Ki of 23 uM. As quercetin has been extensively found in fruits, vegetables, leaves and grains, our discovery would benefit the public to immediately fight Zika.&#39;,
  &#39;publish_time&#39;: &#39;2016-09-11&#39;,
  &#39;authors&#39;: &#39;Amrita Roy; Liangzhong Lim; Jianxing Song&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/074559&#39;},
 {&#39;cord_uid&#39;: &#39;l82aakrk&#39;,
  &#39;sha&#39;: &#39;f9b108c052ded463773a9838a8157ac7d5ba6d1a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations&#39;,
  &#39;doi&#39;: &#39;10.1101/078113&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTZika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors.&#39;,
  &#39;publish_time&#39;: &#39;2016-09-28&#39;,
  &#39;authors&#39;: &#39;Amrita Roy; Liangzhong Lim; Shagun Srivastava; Jianxing Song&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/078113&#39;},
 {&#39;cord_uid&#39;: &#39;gfiukh90&#39;,
  &#39;sha&#39;: &#39;5b27a16123f46b5aaeda3171d22e075dcbd36ee6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Identification of a Zika NS2B-NS3pro pocket susceptible to allosteric inhibition by small molecules including qucertin rich in edible plants&#39;,
  &#39;doi&#39;: &#39;10.1101/078543&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTIt has been recently estimated that one-third of the world population will be infected by Zika virus, but unfortunately so far there is no vaccine or medicine available. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the irreplaceable role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. Very recently we identified two small molecules inhibit Zika NS2B-NS3pro in non-competitive mode, with Ki values of 0.57 and 2.02 µM respective for p-Nitrophenyl-p-guanidino benzoate and qucertin. Here, by molecular docking, we show that although one is designed compound while another is a natural product, both molecules bind to the same pocket on the back of the substrate-binding pocket of Zika NS2B-NS3pro. As the two inhibitors fundamentally differ from cn-716, the only known peptidomimetic boronic acid inhibitor in both structure scaffolds and inhibitory modes, our discovery might open up a new avenue for the future development of allosteric inhibitors, which is highly demanded to achieve therapeutic inhibition of flaviviral NS2B-NS3pro complexes. Furthermore, as qucertin is abundant in many vegetables and fruits such caper, lovage, tea and red onion, our results should benefit the public to immediately fight Zika virus.&#39;,
  &#39;publish_time&#39;: &#39;2016-10-01&#39;,
  &#39;authors&#39;: &#39;Liangzhong Lim; Amrita Roy; Jianxing Song&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/078543&#39;},
 {&#39;cord_uid&#39;: &#39;mkoyc7wh&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Comparative analysis of variation and selection in the HCV genome&#39;,
  &#39;doi&#39;: &#39;10.1101/078584&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractGenotype 1 of the hepatitis C virus (HCV) is the most prevalent of the variants of this virus. Its two main subtypes, HCV-1a and HCV-1b, are associated to differences in epidemic features and risk groups, despite sharing similar features in most biological properties. We have analyzed the impact of positive selection on the evolution of these variants using complete genome coding regions, and compared the levels of genetic variability and the distribution of positively selected sites. We have also compared the distributions of positively selected and conserved sites considering different factors such as RNA secondary structure, the presence of different epitopes (antibody, CD4 and CD8), and secondary protein structure. Less than 10% of the genome was found to be under positive selection, and purifying selection was the main evolutionary force in both subtypes. We found differences in the number of positively selected sites between subtypes in several genes (Core, HVR2 in E2, P7, helicase in NS3 and NS4a). Heterozygosity values in positively selected sites and the rate of non-synonymous substitutions were significantly higher in subtype HCV-1b. Logistic regression analyses revealed that similar selective forces act at the genome level in both subtypes: RNA secondary structure and CD4 T-cell epitopes are associated with conservation, while CD8 T-cell epitopes are associated with positive selection in both subtypes. These results indicate that similar selective constraints are acting along HCV-1a and HCV-1b genomes, despite some differences in the distribution of positively selected sites at independent genes.&#39;,
  &#39;publish_time&#39;: &#39;2016-09-30&#39;,
  &#39;authors&#39;: &#39;Juan Ángel Patiño-Galindo; Fernando González-Candelas&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/078584&#39;},
 {&#39;cord_uid&#39;: &#39;lj0ar2v5&#39;,
  &#39;sha&#39;: &#39;28b107243576723248ad4053261000311a22f134&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genetic, transcriptome, proteomic and epidemiological evidence for blood brain barrier disruption and polymicrobial brain invasion as determinant factors in Alzheimer’s disease&#39;,
  &#39;doi&#39;: &#39;10.1101/080333&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMultiple pathogens have been detected in Alzheimer’s disease (AD) brains. A bioinformatics approach was used to assess relationships between pathogens and AD genes (GWAS), the AD hippocampal transcriptome and plaque or tangle proteins. Host/pathogen interactomes (C.albicans, C.Neoformans, Bornavirus, B.Burgdorferri, cytomegalovirus, Ebola virus, HSV-1, HERV-W, HIV-1, Epstein-Barr, hepatitis C, influenza, C.Pneumoniae, P.Gingivalis, H.Pylori, T.Gondii, T.Cruzi) significantly overlap with misregulated AD hippocampal genes, with plaque and tangle proteins and, except Bornavirus, Ebola and HERV-W, with AD genes. Upregulated AD hippocampal genes match those upregulated by multiple bacteria, viruses, fungi or protozoa in immunocompetent blood cells. AD genes are enriched in bone marrow and immune locations and in GWAS datasets reflecting pathogen diversity, suggesting selection for pathogen resistance. The age of AD patients implies resistance to infections afflicting the younger. APOE4 protects against malaria and hepatitis C, and immune/inflammatory gain of function applies to APOE4, CR1, TREM2 and presenilin variants. 30/78 AD genes are expressed in the blood brain barrier (BBB), which is disrupted by AD risk factors (ageing, alcohol, aluminium, concussion, cerebral hypoperfusion, diabetes, homocysteine, hypercholesterolaemia, hypertension, obesity, pesticides, pollution, physical inactivity, sleep disruption and smoking). The BBB and AD benefit from statins, NSAIDs, oestrogen, melatonin and the Mediterranean diet. Polymicrobial involvement is supported by the upregulation of pathogen sensors/defenders (bacterial, fungal, viral) in the AD brain, blood or CSF. Cerebral pathogen invasion permitted by BBB inadequacy, activating a hyper-efficient immune/inflammatory system, betaamyloid and other antimicrobial defence may be responsible for AD which may respond to antibiotic, antifungal or antiviral therapy.&#39;,
  &#39;publish_time&#39;: &#39;2016-10-12&#39;,
  &#39;authors&#39;: &#39;C.J. Carter&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/080333&#39;},
 {&#39;cord_uid&#39;: &#39;w49x7kca&#39;,
  &#39;sha&#39;: &#39;e6fe63419fe399f9c77513c1c11a043db006a2f3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Accurate prediction of human essential genes using only nucleotide composition and association information&#39;,
  &#39;doi&#39;: &#39;10.1101/084129&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThree groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (https://cefg.uestc.edu.cn/Pheg), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements.&#39;,
  &#39;publish_time&#39;: &#39;2016-10-28&#39;,
  &#39;authors&#39;: &#39;Feng-Biao Biao; Chuan Dong; Hong-Li Hua; Shuo Liu; Hao Luo; Hong-Wan Zhang; Yan-Ting Jin; Kai-Yue Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/084129&#39;},
 {&#39;cord_uid&#39;: &#39;phmm596b&#39;,
  &#39;sha&#39;: &#39;dff716159738acd6bd53b4be12af45358a58752e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;MYCOPLASMA FAUCIUM AND BREAST CANCER&#39;,
  &#39;doi&#39;: &#39;10.1101/089128&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Viruses and bacteria are the cause of a large number of different human diseases. It is believed that some of them may even contribute to the development of cancer. The present work is dedicated to the identification of mycoplasmas in patients with breast cancer. Mycoplasmas may participate in the development of several human diseases including chronic fatigue syndrome, acquired immunodeficiency syndrome, atypical pneumonia, etc. Moreover, there is a reason to believe that mycoplasma can participate in the development of cancer, leukemia and lymphoma. DNA samples from blood, saliva and tumor tissues of the Oncology Institute of Moldova patients diagnosed with breast cancer were analyzed. Mycoplasma testing was performed using nested PCR method. For Mycoplasma spp. detection, we used primers from the region of the 16S-23S RNA genes. The identification of Mycoplasma faucium, Mycoplasma salivarium and Mycoplasma orale was performed by nested PCR with primers for RNA polymerase beta subunit gene corresponding to mycoplasma. M.faucium and M.salivarius was found in saliva at about 100%, and M.orale at a frequency of about 50%. Only M.faucium was found with the frequency of about 60% in the tissue of the patients. Moreover, a fairly high rate of detection of mycoplasma is observed both in the cases when primers for RNA polymerase gene and primers for 16S-23S RNA were used. We found M.faucium in tumor tissues of patients diagnosed with breast cancer. It is known that mycoplasmas are able to stimulate the synthesis of certain cytokines, which act as mitogenes on the cell. We assume that M.faucium can stimulate the mitogenes synthesis in breast tissues (e.g., cytokines) which, in turn, stimulate cell division and thus participate in the initiation of breast cancer.&#39;,
  &#39;publish_time&#39;: &#39;2016-11-22&#39;,
  &#39;authors&#39;: &#39;Valentin Mitin; Lidia Tumanova; Natalia Botnariuc&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/089128&#39;},
 {&#39;cord_uid&#39;: &#39;iedh762s&#39;,
  &#39;sha&#39;: &#39;8c900409a679dc1f16ef13c73500cbb53605b683&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/090936&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe severity and outcome of respiratory viral infections is partially determined by the cellular response mounted by infected lung epithelial cells. Disease prevention and treatment is dependent on our understanding of the shared and unique responses elicited by diverse viruses, yet few studies compare host responses to different viruses while controlling other experimental parameters. We compared changes in gene expression of murine lung epithelial cells infected individually by three respiratory viruses causing mild (rhinovirus, RV1B), moderate (coronavirus, MHV-1), and severe (influenza A virus, PR8) disease in mice. RV1B infection caused numerous gene expression changes, but the differential effect peaked at 12 hours post-infection. PR8 altered an intermediate number of genes whose expression continued to change through 24 hours. MHV-1 had comparatively few effects on host gene expression. The viruses elicited highly overlapping responses in antiviral genes, though MHV-1 induced a lower type I interferon response than the other two viruses. Signature genes were identified for each virus and included host defense genes for PR8, tissue remodeling genes for RV1B, and transcription factors for MHV-1. Our comparative approach identified universal and specific transcriptional signatures of virus infection that can be used to discover mechanisms of pathogenesis in the respiratory tract.&#39;,
  &#39;publish_time&#39;: &#39;2016-12-02&#39;,
  &#39;authors&#39;: &#39;James T. VanLeuven; Benjamin J. Ridenhour; Craig R. Miller; Tanya A. Miura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/090936&#39;},
 {&#39;cord_uid&#39;: &#39;2179jaes&#39;,
  &#39;sha&#39;: &#39;6aa2716c00d89f4b760a9167b15d6203303ae6b9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Vitrification after multiple rounds of sample application and blotting improves particle density on cryo-electron microscopy grids&#39;,
  &#39;doi&#39;: &#39;10.1101/094623&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Single particle cryo-electron microscopy (cryoEM) is becoming widely adopted as a tool for structural characterization of biomolecules at near-atomic resolution. Vitrification of the sample to obtain a dense distribution of particles within a single field of view remains a major bottleneck for the success of such experiments. Here, we describe a simple and cost-effective method to increase the density of frozen-hydrated particles on grids with holey carbon support films. It relies on performing multiple rounds of sample application and blotting prior to plunge freezing in liquid ethane. We show that this approach is generally applicable and significantly increases particle density for a range of samples, such as small protein complexes, viruses and filamentous assemblies. The method is versatile, easy to implement, minimizes sample requirements and can enable characterization of samples that would otherwise resist structural studies using single particle cryoEM.&#39;,
  &#39;publish_time&#39;: &#39;2016-12-15&#39;,
  &#39;authors&#39;: &#39;Joost Snijder; Andrew J. Borst; Annie Dosey; Alexandra C. Walls; Anika Burrell; Vijay S. Reddy; Justin M. Kollman; David Veesler&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/094623&#39;},
 {&#39;cord_uid&#39;: &#39;ahdz5078&#39;,
  &#39;sha&#39;: &#39;8eb99586e9103599ab2f58cc33ca94cd0b60ae53&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and ecological modelling reveal diversity in upper respiratory tract microbial population structures from a cross-sectional community swabbing study&#39;,
  &#39;doi&#39;: &#39;10.1101/099069&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTRespiratory tract infections (RTI) are responsible for over 4 million deaths per year worldwide with pathobiont carriage a required precursor to infection. Through a cross-sectional community-based nasal self-swabbing study we sought to determine carriage epidemiology for respiratory pathogens amongst bacteria (Streptococcus pneumoniae, Haemophilus influenza, Moraxella catarrhalis, Staphylococcus aureus, Pseudomonas aeruginosa and Neisseria meningitidis) and viruses (RSV, Influenza viruses A and B, Rhinovirus/Enterovirus, Coronavirus, Parainfluenza viruses 1-3 and Adenovirus (ADV)). Carriage of bacterial and viral species was shown to vary with participant age, recent RTI and the presence of other species. The spatial structure of microbial respiratory communities was less nested (more disordered) in the young (0-4 years) and those with recent RTI. Species frequency distributions were flatter than random expectation in young individuals (X2 = 20.42, p = 0.002), indicating spatial clumping of species consistent with facilitative relationships amongst them. Deviations from a neutral model of ecological niches were observed for samples collected in the summer and from older individuals (those aged 5-17, 18-64 and ≥65 years) but not in samples collected from winter, younger individuals (those aged 0-4 years), individuals with recent RTI and individuals without recent RTI, demonstrating the importance of both neutral and niche processes in respiratory community assembly. The application of epidemiological methods and ecological theory to sets of respiratory tract samples has yielded novel insights into the factors that drive microbial community composition, such as seasonality and age, as well as species patterns and interactions within the nose.&#39;,
  &#39;publish_time&#39;: &#39;2017-01-09&#39;,
  &#39;authors&#39;: &#39;Abigail L. Coughtrie; Denise E. Morris; Rebecca Anderson; Nelupha Begum; David W. Cleary; Saul N. Faust; Johanna M. Jefferies; Alex R. Kraaijeveld; Michael V. Moore; Mark A. Mullee; Paul J. Roderick; Andrew Tuck; Robert N. Whittaker; Ho Ming Yuen; C. Patrick Doncaster; Stuart C. Clarke&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/099069&#39;},
 {&#39;cord_uid&#39;: &#39;pl7t3hca&#39;,
  &#39;sha&#39;: &#39;e0fd364797eb02a280e25e2649dfe6d6c0d33e05&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species&#39;,
  &#39;doi&#39;: &#39;10.1101/101774&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTaxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus.&#39;,
  &#39;publish_time&#39;: &#39;2017-01-19&#39;,
  &#39;authors&#39;: &#39;Gary R. Whittaker; Nicole M. André; Jean Kaoru Millet&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/101774&#39;},
 {&#39;cord_uid&#39;: &#39;km0b1z64&#39;,
  &#39;sha&#39;: &#39;d418cdee18b07a0adfb553d43b4a08f7736fa9f3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Metagenomic Sequencing Detects Respiratory Pathogens in Hematopoietic Cellular Transplant Patients&#39;,
  &#39;doi&#39;: &#39;10.1101/102798&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTRATIONALECurrent microbiologic diagnostics often fail to identify the etiology of lower respiratory tract infections (LRTI) in hematopoietic cellular transplant recipients (HCT), which precludes the implementation of targeted therapies.OBJECTIVESTo address the need for improved LRTI diagnostics, we evaluated the utility of metagenomic next generation sequencing (mNGS) of bronchoalveolar lavage (BAL) to detect microbial pathogens in HCT patients with acute respiratory illnesses.METHODSWe enrolled 22 post-HCT adults ages 19-69 years with acute respiratory illnesses who underwent BAL at the University of Michigan between January 2012 and May 2013. mNGS was performed on BAL fluid to detect microbes and simultaneously assess the host transcriptional response. Results were compared against conventional microbiologic assays.MEASUREMENTS &amp; MAIN RESULTSmNGS demonstrated 100% sensitivity for detecting respiratory microbes (human metapneumovirus, respiratory syncytial virus, Stenotrophomonas maltophilia, human herpesvirus 6 and cytomegalovirus) when compared to standard testing. Previously unrecognized LRTI pathogens were identified in six patients for whom standard testing was negative (human coronavirus 229E, human rhinovirus A, Corynebacterium propinquum and Streptococcus mitis); findings were confirmed by independent PCR and 16S rRNA sequencing. Relative to patients without infection, patients with infection had increased expression of immunity related genes (p=0.022) and significantly lower diversity of their respiratory microbiome (p=0.017).CONCLUSIONSCompared to conventional diagnostics, mNGS enhanced detection of pathogens in BAL fluid from HCT patients. Furthermore, our results suggest that combining unbiased microbial pathogen detection with assessment of host gene biomarkers of immune response may hold promise for enhancing the diagnosis of post-HCT respiratory infections.&#39;,
  &#39;publish_time&#39;: &#39;2017-01-24&#39;,
  &#39;authors&#39;: &#39;C Langelier; MS Zinter; K Kalantar; GA Yanik; S Christenson; B Odonovan; C White; M Wilson; A Sapru; CC Dvorak; S Miller; CY Chiu; JL DeRisi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/102798&#39;},
 {&#39;cord_uid&#39;: &#39;ois9fiuq&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The immunomodulatory CEA cell adhesion molecule 6 (CEACAM6/CD66c) is a candidate receptor for the influenza A virus&#39;,
  &#39;doi&#39;: &#39;10.1101/104026&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTo establish a productive infection in host cells, viruses often use one or multiple host membrane glycoprotein as their receptors. For Influenza A virus (IAV) such a glycoprotein receptor has not been described, to date. Here we show that IAV is using the host membrane glycoprotein CD66c as a receptor for entry into human epithelial lung cells. Neuraminidase (NA), a viral spike protein binds to CD66c on the cell surface during IAV entry into the host cells. Lung cells overexpressing CD66c showed an increase in virus binding and subsequent entry into the cell. Upon comparison, CD66c demonstrated higher binding capacity than other membrane glycoproteins (EGFR and DC-SIGN) reported earlier to facilitate IAV entry into host cells. siRNA mediated knockdown of CD66c from lung cells inhibited virus binding on cell surface and entry into cells. Blocking CD66c by antibody on the cell surface resulted in decreased virus entry. We found CD66c is a specific glycoprotein receptor for influenza A virus that did not affect entry of non-IAV RNA virus (Hepatitis C virus). Finally, IAV pre-incubated with recombinant CD66c protein when administered intranasally in mice showed decreased cytopathic effects in mice lungs. This publication is the first to report CD66c (CEACAM6) as a glycoprotein receptor for Influenza A virus.Significance StatementCells are enclosed by a semipermeable membrane that allows selective exchange of biomolecules between cells and their surroundings. A set of specialized proteins in this semipermeable membrane, work like gatekeepers to the cell and regulate entry of these biomolecules. One class of such surface proteins is termed as receptors. Viruses bind to one or more of these receptors and manipulate gatekeepers for their own successful entry into host-cells. A membrane protein that influenza A virus (Flu virus) uses for entry into the cells was not discovered till date. This study reports for the first time, a receptor for influenza A virus, that was sought after by researchers for decades. The viral receptor is a promising target that can be used to inhibit virus entry into host cells.&#39;,
  &#39;publish_time&#39;: &#39;2017-01-30&#39;,
  &#39;authors&#39;: &#39;Shah Kamranur Rahman; Mairaj Ahmed Ansari; Pratibha Gaur; Imtiyaz Ahmad; Chandrani Chakravarty; Dileep Kumar Verma; Sanjay Chhibber; Naila Nehal; Shanmugaapriya Sellathanby; Dagmar Wirth; Gulam Waris; Sunil K. Lal&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/104026&#39;},
 {&#39;cord_uid&#39;: &#39;9pa51d9x&#39;,
  &#39;sha&#39;: &#39;95754ec84bdee0637765a5ee34983a5f6cc86eed&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;PGC-1α coordinates with Bcl-2 to control cell cycle in U251 cells through reducing ROS&#39;,
  &#39;doi&#39;: &#39;10.1101/111567&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractB-cell lymphoma 2 (Bcl-2) has a dual function, acting both as an oncogene and an anti-tumor gene. It is well known that Bcl-2 exerts its tumor promoting function through the mitochondrial pathway. However, the mechanism by which Bcl-2 suppresses tumor formation is not well understood. We have previously shown that Bcl-2 inhibits cell cycle progression from the G0/G1 to the S phase after serum starvation, and that quiescent Bcl-2 expressing cells maintained a significant lower level of mitochondrial reactive oxygen species (ROS) than the control cells. Based on the fact that ROS mediate cell cycle progression, and are controlled by peroxisome proliferator-activated receptor-γ co-activator 1α (PGC-1α), a key molecule induced by prolonged starvation and involved in mitochondrial metabolism, we hypothesized that PGC-1α might be related with the cell cycle function of Bcl-2. Here, we showed that PGC-1α was upregulated upon Bcl-2 overexpression and downregulated following Bcl-2 knockdown during serum starvation. Knockdown of PGC-1α activated Bcl-2 expression. Taken together, our results suggest that after serum depletion, PGC-la might coordinate with Bcl-2 to reduce ROS, which in turn delay cell cycle progression.Summary statementPGC-1α coordinate with Bcl-2 delay cell cycle progression to reduce ROS after serum depletion in human glioma U251 cells.&#39;,
  &#39;publish_time&#39;: &#39;2017-02-24&#39;,
  &#39;authors&#39;: &#39;Kun Yao; Xufeng Fu; Du Xing; Yan Li; Bing Han; Chen Zexi; Shanshan Yang; Ran Wei; Jiaqi Zhou; Qinghua Cui&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/111567&#39;},
 {&#39;cord_uid&#39;: &#39;mudv1ejd&#39;,
  &#39;sha&#39;: &#39;b4fc44eabb1f640124fff98990c85dc5c9984aee&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The translational landscape of Zika virus during infection of mammalian and insect cells&#39;,
  &#39;doi&#39;: &#39;10.1101/112904&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractZika virus is a single-stranded, positive-sense RNA virus of the family Flaviviridae, which has recently undergone a rapid expansion among humans in the Western Hemisphere. Here, we report a high-resolution map of ribosomal occupancy of the Zika virus genome during infection of mammalian and insect cells, obtained by ribosome profiling. In contrast to some other flaviviruses such as West Nile, we find no evidence for substantial frameshift-induced ribosomal drop-off during translation of the viral polyprotein, indicating that Zika virus must use alternative mechanisms to downregulate levels of catalytically active viral polymerase. We also show that high levels of ribosome-protected fragments map in-frame to two previously overlooked upstream open reading frames (uORFs) initiating at CUG and UUG codons, with likely consequences for the efficiency of polyprotein expression. Curiously, in African isolates of Zika virus, the two uORFs are fused in-frame into a single uORF. A parallel RNA-Seq analysis reveals the 5′ end position of the subgenomic flavivirus RNA in mammalian and insect cells. Together, these provide the first analysis of flavivirus gene expression by ribosome profiling.Author SummaryRecent Zika virus outbreaks have been associated with congenital diseases and neurological complications. An enhanced understanding of the molecular biology of this pathogen may contribute towards the development of improved treatment and control methods. We present a single-codon resolution analysis of Zika virus translation in mammalian and mosquito cells using ribosome profiling. The analysis revealed two hitherto uncharacterized uORFs in the 5′ leader of Zika virus Brazilian isolate PE243, both of which are occupied by ribosomes during infection. In contrast, these two uORFs are fused into a single uORF in African isolates. This observation provides a new avenue for further investigations into potential factors involved in the emergence of Zika virus from a rarely detected pathogen into a major epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2017-03-02&#39;,
  &#39;authors&#39;: &#39;Nerea Irigoyen; Adam M. Dinan; Luke W. Meredith; Ian Goodfellow; Ian Brierley; Andrew E. Firth&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/112904&#39;},
 {&#39;cord_uid&#39;: &#39;d0y1hs88&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Myeloablation-associated deletion of ORF4 in a human coronavirus 229E infection&#39;,
  &#39;doi&#39;: &#39;10.1101/113423&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWe describe metagenomic next-generation sequencing (mNGS) of a human coronavirus 229E from a patient with AML and persistent upper respiratory symptoms, who underwent hematopoietic cell transplantation (HCT). mNGS revealed a 548-nucleotide deletion, which comprised the near entirety of the ORF4 gene, and no minor allele variants were detected to suggest a mixed infection. As part of her pre-HCT conditioning regimen, the patient received myeloablative treatment with cyclophosphamide and 12 Gy total body irradiation. Iterative sequencing and RT-PCR confirmation of 4 respiratory samples over the 4-week peritransplant period revealed that the pre-conditioning strain contained an intact ORF4 gene, while the deletion strain appeared just after conditioning and persisted over a 2.5-week period. This sequence represents one of the largest genomic deletions detected in a human RNA virus and describes large-scale viral mutation associated with myeloablation for HCT.&#39;,
  &#39;publish_time&#39;: &#39;2017-03-03&#39;,
  &#39;authors&#39;: &#39;Alexander L. Greninger; Gregory Pepper; Ryan C. Shean; Anne Cent; Isabel Palileo; Jane M. Kuypers; Joshua T. Schiffer; Keith R. Jerome&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/113423&#39;},
 {&#39;cord_uid&#39;: &#39;0abutzb3&#39;,
  &#39;sha&#39;: &#39;d91120d80916351170b34715ea4ab141f02c51fb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Prophage genomics reveals patterns in phage genome organization and replication&#39;,
  &#39;doi&#39;: &#39;10.1101/114819&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTemperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution.&#39;,
  &#39;publish_time&#39;: &#39;2017-03-07&#39;,
  &#39;authors&#39;: &#39;Han Suh Kang; Katelyn McNair; Daniel A. Cuevas; Barbara A. Bailey; Anca M. Segall; Robert A. Edwards&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/114819&#39;},
 {&#39;cord_uid&#39;: &#39;jopeo9gs&#39;,
  &#39;sha&#39;: &#39;0cb9c296684ca5e71462d825cab2827854a01544&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;p53 is not necessary for DUX4 pathology&#39;,
  &#39;doi&#39;: &#39;10.1101/118315&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractsFSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4.Summary StatementDUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion.&#39;,
  &#39;publish_time&#39;: &#39;2017-03-19&#39;,
  &#39;authors&#39;: &#39;Darko Bosnakovski; Erik A. Toso; Olivia O. Recht; Anja Cucak; Abhinav K Jain; Michelle C. Barton; Michael Kyba&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/118315&#39;},
 {&#39;cord_uid&#39;: &#39;4it5c9n2&#39;,
  &#39;sha&#39;: &#39;f122e125c2cd3580b1a38e27c780b99f671fd78f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An amino acid motif in HLA-DRβ1 distinguishes patients with uveitis in juvenile idiopathic arthritis&#39;,
  &#39;doi&#39;: &#39;10.1101/140954&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractUveitis is a visually-debilitating disorder that affects up to 30% of children with juvenile idiopathic arthritis (JIA). To identify genetic susceptibility loci for uveitis in JIA, we conducted a genome-wide association study comparing 192 JIA-associated uveitis cases with 330 JIA individuals without uveitis. Two cohorts of JIA patients underwent genotyping and quality control. We used an HLA-specific imputation panel to impute HLA-specific amino acids and HLA types, and identified the amino acid serine at position 11 (serine-11) in HLA- DRB1 as associated to increased risk of uveitis (OR = 2.60, p = 5.43×10−10). The signal at serine-11 was female-specific (interaction of sex and serine-11, p = 0.0096). Serine-11 resides in the YST-motif (positions 10-12) in the peptide binding groove of HLA-DRB1. Quantitative binding affinity predictions revealed peptide-binding preferences that distinguish HLA-DRB1 allotypes with the YST-motif. Our findings highlight a genetically distinct, sexually-dimorphic feature of JIA-associated uveitis.&#39;,
  &#39;publish_time&#39;: &#39;2017-05-22&#39;,
  &#39;authors&#39;: &#39;A.J.W. Haasnoot; M.W. Schilham; S.S.M. Kamphuis; P.C.E. Hissink Muller; A. Heiligenhaus; D. Foell; R.A. Ophoff; T.R.D.J. Radstake; A.I. Den Hollander; T.H.C.M. Reinards; S. Hiddingh; N. Schalij-Delfos; E.P.A.H. Hoppenreijs; M.A.J. van Rossum; C. Wouters; R.K. Saurenmann; N. Wulffraat; R. ten Cate; J.H. de Boer; S.L. Pulit; J.J.W. Kuiper&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/140954&#39;},
 {&#39;cord_uid&#39;: &#39;pt0kes0s&#39;,
  &#39;sha&#39;: &#39;88e3b43319399c339186c02c44d369228c35eef4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An anti-cancer binary system activated by bacteriophage HK022 Integrase&#39;,
  &#39;doi&#39;: &#39;10.1101/147736&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTCancer gene therapy is a great promising tool for cancer therapeutics due to the specific targeting based on the cancerous gene expression background. Binary systems based on site-specific recombination are one of the most effective potential approaches for cancer gene therapy. In these systems, a cancer specific promoter expresses a site-specific recombinase/integrase that in turn controls the expression of a toxin gene. In the current study, we have developed a new HK022 bacteriophage Integrase (Int) based binary system activating a Diphtheria toxin (DTA) gene expression specifically in cancer cells. We have demonstrated the efficiency, and the high specificity of the system in vitro in cell cultures and in vivo in a lung cancer mouse model. Strikingly, different apoptotic and anti-apoptotic factors demonstrated a remarkable efficacy killing capability of the Int-based binary system compared to the conventional hTERT-DTA mono system in the LLC-Kat lung cancer mice model; we observed that the active hTERT promoter down regulation by the transcription factors Mad-1 is the cornerstone of this phenomenon. The new Int-based binary system offers advantages over already known counterparts and may therefore be developed into a safer and efficient cancer treatment technology.&#39;,
  &#39;publish_time&#39;: &#39;2017-06-12&#39;,
  &#39;authors&#39;: &#39;Amer Elias; Itay Spector; Natasha Gritsenko; Yael Zilberstein; Rena Gorovits; Gali Prag; Mikhail Kolot&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/147736&#39;},
 {&#39;cord_uid&#39;: &#39;9e3z9xdd&#39;,
  &#39;sha&#39;: &#39;0b4b4e5bb8d0d5167eec1e203b5dad283bd364a5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Doubling healthy lifespan using drug synergy&#39;,
  &#39;doi&#39;: &#39;10.1101/153205&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryPharmacological interventions that target human ageing would extend individual healthspan and result in dramatic economic benefits to rapidly ageing societies worldwide. For such interventions to be contemplated they need to comprise drugs that are efficacious when given to adults and for which extensive human safety data are available. Here we show that dramatic lifespan extension can be achieved in C.elegans by targeting multiple, evolutionarily conserved ageing pathways using drugs that are already in human use. By targeting multiple synergistic ageing pathways, we are able to slow ageing rate, double lifespan and improves healthspan while minimize developmental and fitness trade-offs. Moreover, we established that there is no synergistic benefit in a daf-2 or daf-7 background, implying the involvement of the TGFβ and IGF pathways in this synergy. Employing lipidomics and transcriptomics analysis we found lipid metabolism to be affected resulting in increased monounsaturated fatty acids (MUFA) and decrease membrane peroxidation index. Our best drug combination showed a conserved lifespan extension in fruit flies. To the best of our knowledge, this is the largest lifespan effect ever reported for any adult-onset drug treatment in C. elegans. This drug-repurposing approach, using drugs already approved for humans to target multiple conserved aging pathways simultaneously, could lead to interventions that prevent age-related diseases and overall frailty in a rapidly ageing population.&#39;,
  &#39;publish_time&#39;: &#39;2017-06-21&#39;,
  &#39;authors&#39;: &#39;Tesfahun Dessale; Krishna Chaithanya Batchu; Diogo Barardo; Ng Li Fang; Vanessa Yuk Man Lam; Linfan Xiao; Markus R. Wenk; Nicholas S. Tolwinski; Jan Gruber&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/153205&#39;},
 {&#39;cord_uid&#39;: &#39;9y8r277c&#39;,
  &#39;sha&#39;: &#39;90e28ff462882ca7a9329beb879f73c2e99430e4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evolutionary proteomics uncovers ciliary signaling components&#39;,
  &#39;doi&#39;: &#39;10.1101/153437&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTCilia are organelles specialized for movement and signaling. To infer when during animal evolution signaling pathways became associated with cilia, we characterized the proteomes of cilia from three organisms: sea urchins, sea anemones and choanoflagellates. From these ciliomes, we identified 437 high confidence ciliary candidate proteins conserved in mammals, including known regulators of Hh, GPCR and TRP channel signaling. The phylogenetic profiles of their ciliary association indicate that the Hh and GPCR pathways were linked to cilia before the origin of bilateria and TRP channels before the origin of animals. We demonstrated that some of the candidates not previously implicated in ciliary biology localized to cilia and further investigated ENKUR, a TRP channel-interacting protein that we identified in the cilia of all three organisms. In animals, ENKUR is expressed by cells with motile cilia, ENKUR localizes to cilia in diverse organisms and, in both Xenopus laevis and mice, ENKUR is required for patterning the left/right axis. Moreover, mutation of ENKUR causes situs inversus in humans. Thus, proteomic profiling of cilia from diverse eukaryotes defines a conserved ciliary proteome, reveals ancient connections to Hh, GPCR and TRP channel signaling, and uncovers a novel ciliary protein that controls vertebrate development and human disease.&#39;,
  &#39;publish_time&#39;: &#39;2017-06-22&#39;,
  &#39;authors&#39;: &#39;Monika Abedin Sigg; Tabea Menchen; Jeffery Johnson; Chanjae Lee; Semil P. Choksi; Galo Garcia; Henriette Busengdal; Gerard Dougherty; Petra Pennekamp; Claudius Werner; Fabian Rentzsch; Nevan Krogan; John B. Wallingford; Heymut Omran; Jeremy F. Reiter&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/153437&#39;},
 {&#39;cord_uid&#39;: &#39;7xmn8yvc&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Mapping the drivers of within-host pathogen evolution using massive data sets&#39;,
  &#39;doi&#39;: &#39;10.1101/155242&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDifferences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations.&#39;,
  &#39;publish_time&#39;: &#39;2017-06-25&#39;,
  &#39;authors&#39;: &#39;Duncan S. Palmer; Isaac Turner; Sarah Fidler; John Frater; Philip Goulder; Dominique Goedhals; Kuan-Hsiang Gary Huang; Annette Oxenius; Rodney Phillips; Roger Shapiro; Cloete van Vuuren; Angela R. McLean; Gil McVean&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/155242&#39;},
 {&#39;cord_uid&#39;: &#39;jhwjh12k&#39;,
  &#39;sha&#39;: &#39;7a51b191fd6575831585eee67a0ac67f209d8fcb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Nuclease escape elements protect messenger RNA against cleavage by multiple viral endonucleases&#39;,
  &#39;doi&#39;: &#39;10.1101/155457&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTDuring lytic Kaposi’s sarcoma-associated herpesvirus (KSHV) infection, the viral endonu-clease SOX promotes widespread degradation of cytoplasmic messenger RNA (mRNA). However, select mRNAs, including the transcript encoding interleukin-6 (IL-6), escape SOX-induced cleavage. IL-6 escape is mediated through a 3’ UTR RNA regulatory element that overrides the SOX targeting mechanism. Here, we reveal that this protective RNA element functions to broadly restrict cleavage by a range of homologous and non-homologous viral endonucleases. However, it does not impede cleavage by cellular endonucleases. The IL-6 protective sequence may be representative of a larger class of nuclease escape elements, as we identified a similar protective element in the GADD45B mRNA. The IL-6 and GADD45B-derived elements display similarities in their sequence, putative structure, and several associated RNA binding proteins. However, the overall composition of their ribonucleoprotein complexes appears distinct, leading to differences in the breadth of nucleases restricted. These findings highlight how RNA elements can selectively control transcript abundance in the background of widespread virus-induced mRNA degradation.AUTHOR SUMMARYThe ability of viruses to control the host gene expression environment is crucial to promote viral infection. Many viruses express factors that reduce host gene expression through widespread mRNA decay. However, some mRNAs escape this fate, like the transcript encoding the immunoregulatory cytokine IL-6 during KSHV infection. IL-6 escape relies on an RNA regulatory element located in its 3’UTR and involves the recruitment of a protective protein complex. Here, we show that this escape extends beyond KSHV to a variety of related and unrelated viral endonucleases. However, the IL-6 element does not protect against cellular endonucleases, revealing for the first time a virus-specific nuclease escape element. We identified a related escape element in the GADD45B mRNA, which displays several similarities with the IL-6 element. However, these elements assemble a largely distinct complex of proteins, leading to differences in the breadth of their protective capacity. Collectively, these findings reveal how a putative new class of RNA elements function to control RNA fate in the background of widespread mRNA degradation by viral endonucleases.&#39;,
  &#39;publish_time&#39;: &#39;2017-06-26&#39;,
  &#39;authors&#39;: &#39;Mandy Muller; Britt A. Glaunsinger&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/155457&#39;},
 {&#39;cord_uid&#39;: &#39;h89scli5&#39;,
  &#39;sha&#39;: &#39;b5161b031c7f720562e94735a018d1c3c8be3ae5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Quantifying the Risk and Cost of Active Monitoring for Infectious Diseases&#39;,
  &#39;doi&#39;: &#39;10.1101/156497&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDuring outbreaks of deadly emerging pathogens (e.g., Ebola, MERS-CoV) and bioterror threats (e.g., smallpox), actively monitoring potentially infected individuals aims to limit disease transmission and morbidity. Guidance issued by CDC on active monitoring was a cornerstone of its response to the West Africa Ebola outbreak. There are limited data on how to balance the costs and performance of this important public health activity. We present a framework that estimates the risks and costs of specific durations of active monitoring for pathogens of significant public health concern. We analyze data from New York City’s Ebola active monitoring program over a 16-month period in 2014-2016. For monitored individuals, we identified unique durations of active monitoring that minimize expected costs for those at “low (but not zero) risk” and “some or high risk”: 21 and 31 days, respectively. Extending our analysis to smallpox and MERS-CoV, we found that the optimal length of active monitoring relative to the median incubation period was reduced compared to Ebola due to less variable incubation periods. Active monitoring can save lives but is expensive. Resources can be most effectively allocated by using exposure-risk categories to modify the duration or intensity of active monitoring.&#39;,
  &#39;publish_time&#39;: &#39;2017-06-28&#39;,
  &#39;authors&#39;: &#39;Nicholas G Reich; Justin Lessler; Jay K Varma; Neil M Vora&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/156497&#39;},
 {&#39;cord_uid&#39;: &#39;iisqysqm&#39;,
  &#39;sha&#39;: &#39;138d4617f4f4984bec9cf2c7540fed533d9e4d6e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies&#39;,
  &#39;doi&#39;: &#39;10.1101/157156&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractHealth-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray® Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.&#39;,
  &#39;publish_time&#39;: &#39;2017-07-31&#39;,
  &#39;authors&#39;: &#39;Lindsay Meyers; Christine C. Ginocchio; Aimie N. Faucett; Frederick S. Nolte; Per H. Gesteland; Amy Leber; Diane Janowiak; Virginia Donovan; Jennifer Dien Bard; Silvia Spitzer; Kathleen A. Stellrecht; Hossein Salimnia; Rangaraj Selvarangan; Stefan Juretschko; Judy A. Daly; Jeremy C. Wallentine; Kristy Lindsey; Franklin Moore; Sharon L. Reed; Maria Aguero-Rosenfeld; Paul D. Fey; Gregory A. Storch; Steve J. Melnick; Christine C. Robinson; Jennifer F. Meredith; Camille V. Cook; Robert K. Nelson; Jay D. Jones; Samuel V. Scarpino; Benjamin M. Althouse; Kirk M. Ririe; Bradley A. Malin; Mark A. Poritz&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/157156&#39;},
 {&#39;cord_uid&#39;: &#39;j5gjipdn&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Self-assembled Star-shaped Chiroplasmonic Gold Nanoparticles for Ultrasensitive Chiro-immunosensor of Viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/162412&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Near field optics and optical tunneling light-matter interaction in the superstructure of chiral nanostructures and semiconductor quantum dots exhibit strong optical rotation activity that may open a new window for chiral-based bioanalytes detection. Herein we report an ultrasensitive, chiro-immunosensor using superstructure of chiral gold nanohybrids (CAu NPs) and quantum dots (QDs). Self-assembling techniques were employed to create asymmetric plasmonic chiral nanostructures for extending the spectral range of circular dichroism (CD) response for obtaining superior plasmonic resonant coupling with the QDs excitonic state; this may help to achieve lower the limit of detection (LOD) values. As a result, the designed probe exhibited avian influenza A (H5N1) viral concentration at picomolar level, a significant improvement in sensitivity in comparison to a non-assembled CAu NPs based chiroassay. The practicability of the proposed sensing system was successfully demonstrated on several virus cultures including, avian influenza A (H4N6) virus, fowl adenovirus and coronavirus in blood samples. The results of our study highlights that exciton-plasmon interaction changes chirality and the self-assembled nanostructures are an efficient strategy for enhancing the sensitivity of plasmonic nanosensors.&#39;,
  &#39;publish_time&#39;: &#39;2017-07-12&#39;,
  &#39;authors&#39;: &#39;Syed Rahin Ahmed; Éva Nagy; Suresh Neethirajan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/162412&#39;},
 {&#39;cord_uid&#39;: &#39;jzzhpw7h&#39;,
  &#39;sha&#39;: &#39;3b22eecad8a582436c52284a4db2198a98a94e18&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;DUSP1 regulates apoptosis and cell migration, but not the JIP1-protected cytokine response, during Respiratory Syncytial Virus and Sendai Virus infection&#39;,
  &#39;doi&#39;: &#39;10.1101/163360&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe host antiviral response involves the induction of interferons and proinflammatory cytokines, but also the activation of cell death pathways, including apoptosis, to limit viral replication and spreading. This host defense is strictly regulated to eliminate the infection while limiting tissue damage that is associated with virus pathogenesis. Post-translational modifications, most notably phosphorylation, are key regulators of the antiviral defense implying an important role of protein phosphatases. Here, we investigated the role of the dual-specificity phosphatase 1 (DUSP1) in the host defense against human respiratory syncytial virus (RSV), a pathogenic virus of the Pneumoviridae family, and Sendai virus (SeV), a model virus being developed as a vector for anti-RSV vaccine. We found that DUSP1 is upregulated before being subjected to proteasomal degradation. DUSP1 does not inhibit the antiviral response, but negatively regulates virus-induced JNK/p38 MAPK phosphorylation. Interaction with the JNK-interacting protein 1 scaffold protein prevents dephosphorylation of JNK by DUSP1, likely explaining that AP-1 activation and downstream cytokine production are protected from DUSP1 inhibition. Importantly, DUSP1 promotes SeV-induced apoptosis and suppresses cell migration in RSV-infected cells. Collectively, our data unveil a previously unrecognized selective role of DUSP1 in the regulation of tissue damage and repair during infections by RSV and SeV.&#39;,
  &#39;publish_time&#39;: &#39;2017-07-13&#39;,
  &#39;authors&#39;: &#39;Alexa C. Robitaille; Elise Caron; Nicolas Zucchini; Espérance Mukawera; Damien Adam; Mélissa K. Mariani; Anaïs Gélinas; Audray Fortin; Emmanuelle Brochiero; Nathalie Grandvaux&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/163360&#39;},
 {&#39;cord_uid&#39;: &#39;yqa7ggqg&#39;,
  &#39;sha&#39;: &#39;5d9e490190986584d799e2828c6fd9679e5ae159&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics&#39;,
  &#39;doi&#39;: &#39;10.1101/167999&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.Significance StatementBats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection.&#39;,
  &#39;publish_time&#39;: &#39;2017-07-24&#39;,
  &#39;authors&#39;: &#39;Pamela C. De La Cruz-Rivera; Mohammed Kanchwala; Hanquan Liang; Ashwani Kumar; Lin-Fa Wang; Chao Xing; John W. Schoggins&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/167999&#39;},
 {&#39;cord_uid&#39;: &#39;8xcplab3&#39;,
  &#39;sha&#39;: &#39;fb7371ad31cd501854660876a282060402082537&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;MERS-CoV spillover at the camel-human interface&#39;,
  &#39;doi&#39;: &#39;10.1101/173211&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.&#39;,
  &#39;publish_time&#39;: &#39;2017-08-10&#39;,
  &#39;authors&#39;: &#39;Gytis Dudas; Luiz Max Carvalho; Andrew Rambaut; Trevor Bedford; Ali M. Somily; Mazin Barry; Sarah S. Al Subaie; Abdulaziz A. BinSaeed; Fahad A. Alzamil; Waleed Zaher; Theeb Al Qahtani; Khaldoon Al Jerian; Scott J.N. McNabb; Imad A. Al-Jahdali; Ahmed M. Alotaibi; Nahid A. Batarfi; Matthew Cotten; Simon J. Watson; Spela Binter; Paul Kellam&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/173211&#39;},
 {&#39;cord_uid&#39;: &#39;v7asfaxc&#39;,
  &#39;sha&#39;: &#39;656ef218f7dcea5555a59cdc29d0f9ec8d5c2a82&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;MERS-CoV NSP16 necessary for IFN resistance and viral pathogenesis&#39;,
  &#39;doi&#39;: &#39;10.1101/173286&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronaviruses encode a mix of highly conserved and novel genes as well as genetic elements necessary for infection and pathogenesis, raising the possibility for common targets for attenuation and therapeutic design. In this study, we focus on the highly conserved nonstructural protein (NSP) 16, a viral 2’O methyl-transferase (MTase) that encodes critical functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif in the conserved KDKE motif of MERS NSP16 (D130A) and evaluated the effect on viral infection and pathogenesis. While the absence of 2’O MTase activity had only marginal impact on propagation and replication in Vero cells, the MERS dNSP16 mutant demonstrated significant attenuation relative to control both in primary human airway cultures and in vivo. Further examination indicated the MERS dNSP16 mutant had a type I IFN based attenuation and was partially restored in the absence of IFIT molecules. Importantly, the robust attenuation permitted use of MERS dNSP16 as a live attenuated vaccine platform protecting from challenge with a mouse adapted MERS-CoV strain. These studies demonstrate the importance of the conserved 2’O MTase activity for CoV pathogenesis and highlight NSP16 as a conserved universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting.SignificanceCoronavirus emergence in both human and livestock represents a significant threat to global public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and swine delta coronavirus in the 21st century. These studies describe an approach that effectively targets the highly conserved 2’O methyl-transferase activity of coronaviruses for attenuation. With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable target for a live attenuated vaccine platform as well as therapeutic development for both current and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad applicability of targeting viral 2’O MTase function across coronaviruses.&#39;,
  &#39;publish_time&#39;: &#39;2017-08-08&#39;,
  &#39;authors&#39;: &#39;Vineet D. Menachery; Lisa E. Gralinski; Hugh D. Mitchell; Kenneth H. Dinnon; Sarah R. Leist; Boyd L. Yount; Rachel L. Graham; Eileen T. McAnarney; Kelly G. Stratton; Adam S. Cockrell; Kari Debbink; Amy C. Sims; Katrina M. Waters; Ralph S. Baric&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/173286&#39;},
 {&#39;cord_uid&#39;: &#39;vqtx98xm&#39;,
  &#39;sha&#39;: &#39;f126b7f9c8ba02a0f872b51cf95d08086e0bd4ac&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations&#39;,
  &#39;doi&#39;: &#39;10.1101/175562&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.IMPORTANCEUnique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.&#39;,
  &#39;publish_time&#39;: &#39;2017-08-11&#39;,
  &#39;authors&#39;: &#39;Kevin W. Graepel; Xiaotao Lu; James Brett Case; Nicole R. Sexton; Everett Clinton Smith; Mark R. Denison&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/175562&#39;},
 {&#39;cord_uid&#39;: &#39;pgfdy23h&#39;,
  &#39;sha&#39;: &#39;7116ddab89281e7641fb659019c90ee3715f5039&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The impact of persistent bacterial bronchitis on the pulmonary microbiome of children&#39;,
  &#39;doi&#39;: &#39;10.1101/181982&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPersistent bacterial bronchitis is a leading cause of chronic wet cough in young children. This study aimed to characterise the respiratory bacterial microbiota of healthy children and to assess the impact of the changes associated with the development of persistent bacterial bronchitis.Blind, protected brushings were obtained from 20 healthy controls and 24 children with persistent bacterial bronchitis, with an additional directed sample obtained from persistent bacterial bronchitis patients. DNA was extracted, quantified using a 16S rRNA gene quantitative PCR assay prior to microbial community analysis by 16S rRNA gene sequencing.No significant difference in bacterial diversity or community composition (R2 = 0.01, P = 0.36) was observed between paired blind and non-blind brushes, showing that blind brushings are a valid means of accessing the airway microbiota. This has important implications for collecting lower respiratory samples from healthy children. A significant decrease in bacterial diversity (P &lt; 0.001) and change in community composition (R2 = 0.08, P = 0.004) was observed between controls and patients. Bacterial communities within patients with PBB were dominated by Proteobacteria, and indicator species analysis showed that Haemophilus and Neisseria were significantly associated with the patient group. In 15 (52.9%) cases the dominant organism by sequencing was not identified by standard routine clinical culture.The bacteria present in the lungs of patients with persistent bacterial bronchitis were less diverse in terms of richness and evenness. The results validate the clinical diagnosis, and suggest that more attention to bacterial communities in children with chronic cough may lead to more rapid recognition of this condition with earlier treatment and reduction in disease burden.&#39;,
  &#39;publish_time&#39;: &#39;2017-08-29&#39;,
  &#39;authors&#39;: &#39;Leah Cuthbertson; Vanessa Craven; Lynne Bingle; William O.C.M. Cookson; Mark L. Everard; Miriam F. Moffatt&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/181982&#39;},
 {&#39;cord_uid&#39;: &#39;p3b4yk5d&#39;,
  &#39;sha&#39;: &#39;3d690de86ca6e200f4f9cac61b004e47ad753aae&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Mouse hepatitis virus nsp14 exoribonuclease activity is required for resistance to innate immunity&#39;,
  &#39;doi&#39;: &#39;10.1101/182196&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;ABSTRACTCoronaviruses (CoV) are positive-sense RNA viruses that infect numerous mammalian and avian species and are capable of causing severe and lethal disease in humans. CoVs encode several innate immune antagonists that interact with the host innate immune response to facilitate efficient viral replication. CoV non-structural protein 14 (nsp14) encodes 3&#39;-to-5&#39; exoribonuclease activity (ExoN), which performs a proofreading function and is required for high-fidelity replication. Outside of the order Nidovirales, arenaviruses are the only RNA viruses that encode an ExoN, which functions to degrade dsRNA replication intermediates. In this study, we tested the hypothesis that CoV ExoN may also function to antagonize the innate immune response. We demonstrate that viruses lacking ExoN activity [ExoN(-)] are sensitive to cellular pretreatment with interferon beta (IFN-β) in a dose-dependent manner. In addition, ExoN(-) virus replication was attenuated in wild-type bone marrow-derived macrophages (BMMs) and partially restored in interferon alpha/beta receptor deficient (IFNAR-/-) BMMs. ExoN(-) virus replication did not result in IFN-β gene expression, and in the presence of an IFN-β-mediated antiviral state, ExoN(-) viral RNA levels were not substantially reduced relative to untreated. However, ExoN(-) virus generated from IFN-β pretreated cells had reduced specific infectivity and decreased relative fitness, suggesting that ExoN(-) virus generated during an antiviral state is less viable to establish a subsequent infection. Overall, our data suggest MHV ExoN activity is required for resistance to the innate immune response and antiviral mechanisms affecting the viral RNA sequence and/or an RNA modification act on viruses lacking ExoN activity.&#34;,
  &#39;publish_time&#39;: &#39;2017-08-29&#39;,
  &#39;authors&#39;: &#39;James Brett Case; Yize Li; Ruth Elliott; Xiaotao Lu; Kevin W. Graepel; Nicole R. Sexton; Everett Clinton Smith; Susan R. Weiss; Mark R. Denison&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/182196&#39;},
 {&#39;cord_uid&#39;: &#39;g55d5ijx&#39;,
  &#39;sha&#39;: &#39;08a22278486e12768ce186677a6a89663d24586f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Complemented palindrome small RNAs first discovered from SARS coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/185876&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes.&#39;,
  &#39;publish_time&#39;: &#39;2017-09-07&#39;,
  &#39;authors&#39;: &#39;Chang Liu; Ze Chen; Wenyuan Shen; Deshui Yu; Siyu Li; Yue Hu; Haishuo Ji; Wenjun Bu; Qingsong Wang; Shan Gao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/185876&#39;},
 {&#39;cord_uid&#39;: &#39;vok1dni9&#39;,
  &#39;sha&#39;: &#39;090b6c8b3df30bc248221869f673a2d970caa1b9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Quantifying the success of measles vaccination campaigns in the Rohingya refugee camps&#39;,
  &#39;doi&#39;: &#39;10.1101/19008417&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;In the wake of the Rohingya population&#39;s mass migration from Myanmar, one of the world&#39;s largest refugee settlements was constructed in Cox&#39;s Bazar, Bangladesh to accommodate nearly 900,000 new refugees. Refugee populations are particularly vulnerable to infectious disease outbreaks due to many population and environmental factors. A large measles outbreak, with over 2,500 cases, occurred among the Rohingya population between September and December 2017. Here, we estimate key epidemiological parameters and use a dynamic mathematical model of measles transmission to evaluate the effectiveness of the reactive vaccination campaigns in the refugee camps. We also estimate the potential for subsequent outbreaks under different vaccination coverage scenarios. Our modeling results highlight the success of the vaccination campaigns in rapidly curbing transmission and emphasize the public health importance of maintaining high levels of vaccination in this population, where high birth rates and historically low vaccination coverage rates create suitable conditions for future measles outbreaks.&#34;,
  &#39;publish_time&#39;: &#39;2019-10-05&#39;,
  &#39;authors&#39;: &#39;Taylor Chin; Caroline O. Buckee; Ayesha S. Mahmud&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/19008417&#39;},
 {&#39;cord_uid&#39;: &#39;jwesa12u&#39;,
  &#39;sha&#39;: &#39;10525ac89e46be4cb9cb9fd1131d28411a902047&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Using Digital Surveillance Tools for Near Real-Time Mapping of the Risk of International Infectious Disease Spread: Ebola as a Case Study&#39;,
  &#39;doi&#39;: &#39;10.1101/19011940&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In our increasingly interconnected world, it is crucial to understand the risk of an outbreak originating in one country or region and spreading to the rest of the world. Digital disease surveillance tools such as ProMED and HealthMap have the potential to serve as important early warning systems as well as complement the field surveillance during an ongoing outbreak. Here we present a flexible statistical model that uses data produced from digital surveillance tools (ProMED and HealthMap) to forecast short term incidence trends in a spatially explicit manner. The model was applied to data collected by ProMED and HealthMap during the 2013-2016 West African Ebola epidemic. The model was able to predict each instance of international spread 1 to 4 weeks in advance. Our study highlights the potential and limitations of using publicly available digital surveillance data for assessing outbreak dynamics in real-time.&#39;,
  &#39;publish_time&#39;: &#39;2019-11-15&#39;,
  &#39;authors&#39;: &#39;Sangeeta Bhatia; Britta Lassmann; Emily Cohn; Malwina Carrion; Moritz U.G. Kraemer; Mark Herringer; John Brownstein; Larry Madoff; Anne Cori; Pierre Nouvellet&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/19011940&#39;},
 {&#39;cord_uid&#39;: &#39;npd2qf7m&#39;,
  &#39;sha&#39;: &#39;44b6eea3629b4e13595dc49813ee39a0d20dc5f2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evolutionary rate shifts suggest species-specific adaptation events in HIV-1 and SIV&#39;,
  &#39;doi&#39;: &#39;10.1101/190769&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe process of molecular adaptation following a cross-species virus transmission event is currently poorly understood. Here, we identified 137 protein sites that experienced deceleration in their rate of evolution along the HIV-1/SIV phylogeny, likely indicating gain-of-function and consequent adaptation. The majority of such events occurred in parallel to cross-species transmission events and varied between HIV-1 groups, indicating independent adaptation strategies. The evolutionary rate decelerations we found were particularly prominent in accessory proteins that counteract host antiviral restriction factors, suggesting that these factors are a major barrier to viral adaptation to a new host. Surprisingly, we observed that the non-pandemic HIV-1 group O, derived from gorillas, exhibited more rate deceleration events than the pandemic group M, derived from chimpanzees. We suggest that the species barrier is higher when the genetic distance of the hosts increases. Our approach paves the way for subsequent studies on cross-species transfers in other major pathogens.&#39;,
  &#39;publish_time&#39;: &#39;2017-09-19&#39;,
  &#39;authors&#39;: &#39;Maoz Gelbart; Adi Stern&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/190769&#39;},
 {&#39;cord_uid&#39;: &#39;6riyqn4k&#39;,
  &#39;sha&#39;: &#39;aa5a99e67d3e887b3fa88575fb0d4a662d0a60ce&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Modeling Epidemics: A Primer and Numerus Software Implementation&#39;,
  &#39;doi&#39;: &#39;10.1101/191601&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractEpidemiological models are dominated by SEIR (Susceptible, Exposed, Infected and Removed) dynamical systems formulations and their elaborations. These formulations can be continuous or discrete, deterministic or stochastic, or spatially homogeneous or heterogeneous, the latter often embracing a network formulation. Here we review the continuous and discrete deterministic and discrete stochastic formulations of the SEIR dynamical systems models, and we outline how they can be easily and rapidly constructed using the Numerus Model Builder, a graphically-driven coding platform. We also demonstrate how to extend these models to a metapopulation setting using both the Numerus Model Builder network and geographical mapping tools.&#39;,
  &#39;publish_time&#39;: &#39;2017-09-22&#39;,
  &#39;authors&#39;: &#39;Wayne M. Getz; Richard Salter; Oliver Muellerklein; Hyun S. Yoon; Krti Tallam&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/191601&#39;},
 {&#39;cord_uid&#39;: &#39;uw76y16g&#39;,
  &#39;sha&#39;: &#39;9b17a9fa6be11d8b3dbed9f3dea2516020083bd3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Fast Virome Explorer: A Pipeline for Virus and Phage Identification and Abundance Profiling in Metagenomics Data&#39;,
  &#39;doi&#39;: &#39;10.1101/196998&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIdentifying viruses and phages in a metagenomics sample has important implication in improving human health, preventing viral outbreaks, and developing personalized medicine. With the rapid increase in data files generated by next generation sequencing, existing tools for identifying and annotating viruses and phages in metagenomics samples suffer from expensive running time. In this paper, we developed a stand-alone pipeline, FastViromeExplorer, for rapid identification and abundance quantification of viruses and phages in big metagenomic data. Both real and simulated data validated FastViromeExplorer as a reliable tool to accurately identify viruses and their abundances in large data, as well as in a time efficient manner.&#39;,
  &#39;publish_time&#39;: &#39;2017-10-02&#39;,
  &#39;authors&#39;: &#39;Saima Sultana Tithi; Roderick V. Jensen; Liqing Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/196998&#39;},
 {&#39;cord_uid&#39;: &#39;ifla4aix&#39;,
  &#39;sha&#39;: &#39;7395de48b2402e5e1561a93b99e8aae8728c6ff1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain and is required for ubiquitination&#39;,
  &#39;doi&#39;: &#39;10.1101/200410&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTTRIM25 is a novel RNA-binding protein and a member of the Tripartite Motif (TRIM) family of E3 ubiquitin ligases, which plays a pivotal role in the innate immune response. Almost nothing is known about its RNA-related roles in cell biology. Furthermore, its RNA-binding domain has not been characterized. Here, we reveal that RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain, which we postulate to be a novel RNA-binding domain. Using CLIP-seq and SILAC-based co-immunoprecipitation assays, we uncover TRIM25’s endogenous RNA targets and protein binding partners. Finally, we show that the RNA-binding activity of TRIM25 is important for its ubiquitin ligase function. These results reveal new insights into the molecular roles and characteristics of RNA-binding E3 ubiquitin ligases and demonstrate that RNA could be an essential factor for their biological functions.&#39;,
  &#39;publish_time&#39;: &#39;2017-10-09&#39;,
  &#39;authors&#39;: &#39;Nila Roy Choudhury; Gregory Heikel; Maryia Trubitsyna; Peter Kubik; Jakub Stanislaw Nowak; Shaun Webb; Sander Granneman; Christos Spanos; Juri Rappsilber; Alfredo Castello; Gracjan Michlewski&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/200410&#39;},
 {&#39;cord_uid&#39;: &#39;v4spn75m&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;IFITM proteins inhibit HIV-1 protein synthesis&#39;,
  &#39;doi&#39;: &#39;10.1101/201525&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractInterferon induced transmembrane proteins (IFITMs) inhibit the cellular entry of a broad range of viruses, but it has been suspected that for HIV-1 IFITMs may also inhibit a post-integration replicative step. We show that IFITM expression reduces HIV-1 viral protein synthesis by preferentially excluding viral mRNA transcripts from translation and thereby restricts viral production. Codon-optimization of proviral DNA rescues viral translation, implying that IFITM-mediated restriction requires recognition of viral RNA elements. In addition, we find that expression of the viral accessory protein Nef can help overcome the IFITM-mediated inhibition of virus production. Our studies identify a novel role for IFITMs in inhibiting HIV replication at the level of translation, but show that the effects can be overcome by the lentiviral protein Nef.&#39;,
  &#39;publish_time&#39;: &#39;2017-10-11&#39;,
  &#39;authors&#39;: &#39;Wing-Yiu Jason Lee; Chen Liang; Richard D Sloan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/201525&#39;},
 {&#39;cord_uid&#39;: &#39;43cxzudl&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Delving Below the Species Level To Characterize the Ecological Diversity within the Global Virome: An Exploration of West Nile Virus&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.12.874214&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryEfforts to describe the diversity of viruses have largely focused on classifying viruses at the species level. However, substantial ecological diversity, both in virulence level and host range, is known within virus species. Here we demonstrate a proof of concept for easily discovering ecological diversity within a virus species taxon. We have focused on the West Nile Virus to take advantage of its broad host range in nature. We produced a genome-based phylogeny of world diversity of WNV and then used Ecotype Simulation 2 to hypothesize demarcation of genomes into 69 putative ecotypes (ecologically distinct populations), based only on clustering of genome sequences. Then we looked for evidence of ecological divergence among ecotypes based on differences in host bird associations within the Connecticut-New York region. Our results indicated significant heterogeneity among ecotypes for their associations with different bird hosts. Ecological diversity within other zoonotic viruses could be easily discovered using this approach. Opportunities for extending this line of research to human associations of virus ecotypes are limited by missing geographic metadata on human samples.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-13&#39;,
  &#39;authors&#39;: &#39;Tong Kong; Kelly Mei; Ammie Wang; Danny Krizanc; Frederick M. Cohan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.12.874214&#39;},
 {&#39;cord_uid&#39;: &#39;2fcl6f4i&#39;,
  &#39;sha&#39;: &#39;97027e08cb413795cf19064338efd049fe5d491b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.13.876037&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAdenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-13&#39;,
  &#39;authors&#39;: &#39;Alexander M. Price; Katharina E. Hayer; Daniel P. Depledge; Angus C. Wilson; Matthew D. Weitzman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.13.876037&#39;},
 {&#39;cord_uid&#39;: &#39;bqwmx5dy&#39;,
  &#39;sha&#39;: &#39;93f67ffe7803061de9b19c4dfa346b3aa97aa4eb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.15.877092&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSingle particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-16&#39;,
  &#39;authors&#39;: &#39;Ali Punjani; Haowei Zhang; David J. Fleet&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.15.877092&#39;},
 {&#39;cord_uid&#39;: &#39;eohjkqc4&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.16.875872&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-16&#39;,
  &#39;authors&#39;: &#39;Kelsey T. Young; Kevin K. Lahmers; Holly S. Sellers; David E. Stallknecht; Rebecca L. Poulson; Jerry T. Saliki; S. Mark Tompkins; Ian Padykula; Chris Siepker; Elizabeth W. Howerth; Michelle Todd; James B. Stanton&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.16.875872&#39;},
 {&#39;cord_uid&#39;: &#39;t2azgyp8&#39;,
  &#39;sha&#39;: &#39;b0899a264af548a89d649154aec569889717b295&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Stochastic challenges to interrupting helminth transmission&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.17.19013490&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Predicting the effect of different programmes designed to control both the morbidity induced by helminth infections and parasite transmission is greatly facilitated by the use of mathematical models of transmission and control impact. In such models, it is essential to account for as many sources of uncertainty -- natural, or otherwise -- to ensure robustness in prediction and to accurately depict variation around an expected outcome. In this paper, we investigate how well the standard deterministic models match the predictions made using individual-based stochastic simulations. We also explore how well concepts which derive from deterministic models, such as ‘breakpoints’ in transmission, apply in the stochastic world. Employing an individual based stochastic model framework we also investigate how transmission and control are affected by the migration of infected people into a defined community. To give our study focus we consider the control of soil-transmitted helminths (STH) by mass drug administration (MDA), though our methodology is readily applicable to the other helminth species such as the schistosome parasites and the filarial worms. We show it is possible to define a ‘stochastic breakpoint’ where much noise surrounds the expected deterministic breakpoint. We also discuss the concept of the ‘interruption of transmission’ independent of the ‘breakpoint’ concept where analyses of model behaviour illustrate the current limitations of deterministic models to account for the ‘fade-out’ or transmission extinction behaviour in simulations. The analyses based on migration confirm a relationship between the infected human migration rate per unit of time and the death rate of infective stages that are released into the free-living environment (eggs or larvae depending on the STH species) that create the reservoir of infection which in turn determines the likelihood that control activities aim at chemotherapeutic treatment of the human host will eliminate transmission. The development of a new stochastic simulation code for STH in the form of a publicly-available open-source python package which includes features to incorporate many population stratifications, different control interventions including mass drug administration (with defined frequency, coverage levels and compliance patterns) and inter-village human migration is also described.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-18&#39;,
  &#39;authors&#39;: &#39;Robert J Hardwick; Marleen Werkman; James E Truscott; Roy M Anderson&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.17.19013490&#39;},
 {&#39;cord_uid&#39;: &#39;bi49e7ne&#39;,
  &#39;sha&#39;: &#39;a555fbeeffc44dfdb5087c83395f0f743654cbba&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A novel totivirus alters gene expression and vacuolar morphology in Malassezia cells and induces a TLR3-mediated inflammatory immune response&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.17.880526&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMost fungal viruses have been identified in plant pathogens, whereas the presence of viral particles in human pathogenic fungi is less well studied. In the present study, we observed extrachromosomal double-stranded RNA (dsRNA) segments in various clinical isolates of Malassezia species. Malassezia is the most dominant fungal genus on the human skin surface, and species in this group are considered to be etiological factors in various skin diseases including dandruff, seborrheic dermatitis, and atopic dermatitis. We identified novel dsRNA segments and our sequencing results revealed that the virus, named MrV40, belongs to the Totiviridae family and contains an additional satellite dsRNA segment encoding a novel protein. The transcriptome of virus-infected M. restricta cells was compared to that of virus-free cells, and the results showed that transcripts involved in ribosomal biosynthesis were down regulated and those involved in energy production and programmed cell death were increased in abundance. Moreover, transmission electron microscopy revealed significantly larger vacuoles for virus-infected M. restricta cells, indicating that MrV40 infection dramatically altered M. restricta physiology. Our analysis also revealed that a viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs) and this result suggests that a viral element contributes to the pathogenesis of Malassezia.ImportanceMalassezia is the most dominant fungal genus on the human skin surface and is associated with various skin diseases including dandruff and seborrheic dermatitis. Among Malassezia species, M. restricta is the most widely observed species on the human skin. In the current study, we identified a novel dsRNA virus, named MrV40, in M. restricta and characterized the sequences and structure of the viral genome along with an independent satellite dsRNA viral segment. Moreover, we found altered expression of genes involved in ribosomal synthesis and programmed cell death, indicating that virus infection altered the physiology of the fungal host cells. Our data also showed that the viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs), indicating that a viral element likely contributes to the pathogenesis of Malassezia. This is the first study to identify and characterize a novel mycovirus in Malassezia.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-19&#39;,
  &#39;authors&#39;: &#39;Minji Park; Yong-Joon Cho; Donggyu Kim; Chul-Su Yang; Shi Mun Lee; Thomas L. Dawson; Satoshi Nakamizo; Kenji Kabashima; Yang Won Lee; Won Hee Jung&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.17.880526&#39;},
 {&#39;cord_uid&#39;: &#39;e5p4metz&#39;,
  &#39;sha&#39;: &#39;8c223fd63ba6a924ecd3df8d32bcf0556758cb95&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.18.880849&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial (E. coli) and archaeal (H. volcanii, P. furiosus) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-19&#39;,
  &#39;authors&#39;: &#39;Felix Grünberger; Robert Knüppel; Michael Jüttner; Martin Fenk; Andreas Borst; Robert Reichelt; Winfried Hausner; Jörg Soppa; Sebastien Ferreira-Cerca; Dina Grohmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.18.880849&#39;},
 {&#39;cord_uid&#39;: &#39;qw2tsrxm&#39;,
  &#39;sha&#39;: &#39;73d80c8f5780d70bd8d343188c56e898e91557b6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Ca2+ ions promote fusion of Middle East Respiratory Syndrome coronavirus with host cells and increase infectivity&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.18.881391&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a major emerging zoonotic infectious disease. Since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. To date, there are no efficacious drugs and vaccines against MERS-CoV, increasing its potential to cause a public health emergency. A critical step in the life cycle of MERS-CoV is the fusion with the host cell with its spike (S) protein as main determinant of viral entry. Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca2+) plays an important role for fusogenic activity via a Ca2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudo-particles (PPs) on host cells without and with Ca2+. Our results demonstrate that intracellular Ca2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca2+, as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca2+ ions interacting with the FP has implications on the fusion dynamics of the virus.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-19&#39;,
  &#39;authors&#39;: &#39;Marco R. Straus; Tiffany Tang; Alex L. Lai; Annkatrin Flegel; Miya Bidon; Jack H. Freed; Susan Daniel; Gary R. Whittaker&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.18.881391&#39;},
 {&#39;cord_uid&#39;: &#39;7a5nxxar&#39;,
  &#39;sha&#39;: &#39;63781701d3aac3b1640caae24b2c8c2a06c8e6ab&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Comparison of alternative models of human movement and the spread of disease&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.19.882175&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPredictive models for the spatial spread of infectious diseases has received much attention in recent years as tools for the management of infectious diseas outbreaks. Prominently, various versions of the so-called gravity model, borrowed from transportation theory, have been used. However, the original literature suggests that the model has some potential misspecifications inasmuch as it fails to capture higher-order interactions among population centers. The fields of economics, geography and network sciences holds alternative formulations for the spatial coupling within and among conurbations. These includes Stouffer’s rank model, Fotheringham’s competing destinations model and the radiation model of Simini et al. Since the spread of infectious disease reflects mobility through the filter of age-specific susceptibility and infectivity and since, moreover, disease may alter spatial behavior, it is essential to confront with epidemiological data on spread. To study their relative merit we, accordingly, fit variants of these models to the uniquely detailed dataset of prevaccination measles in the 954 cities and towns of England and Wales over the years 1944-65 and compare them using a consistent likelihood framework. We find that while the gravity model is a reasonable first approximation, both Stouffer’s rank model, an extended version of the radiation model and the Fotheringham competing destinations model provide significantly better fits, Stouffer’s model being the best. Through a new method of spatially disaggregated likelihoods we identify areas of relatively poorer fit, and show that it is indeed in densely-populated conurbations that higher order spatial interactions are most important. Our main conclusion is that it is premature to narrow in on a single class of models for predicting spatial spread of infectious disease. The supplemental materials contain all code for reproducing the results and applying the methods to other data sets.Author summaryThe ability to predict how infectious disease will spread is of great importance in the face of the numerous emergent and re-emergent pathogens that currently threatening human well-being. We identified a variety of alternative models that predict human mobility as as a function of population distribution across a landscape. These consider some models that account for pair-wise interactions between population centers, as well as some that allow for higher-order interactions. We trained the models using a uniquely rich spatiotemporal data set on pre-vaccination measles in England and wales (1944-65), which comprises more than a million records from 954 cities and towns. Likelihood rankings of the different models reveal strong evidence for higher-order interactions in the form of competition among cities as destinations for travelers and, thus, dilution of spatial transmission. The currently most commonly used so-called ‘gravity’ models were far from the best in capturing spatial disease dynamics.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-19&#39;,
  &#39;authors&#39;: &#39;Ottar N. Bjørnstad; Bryan T. Grenfell; Cecile Viboud; Aaron A. King&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.19.882175&#39;},
 {&#39;cord_uid&#39;: &#39;8b2ookby&#39;,
  &#39;sha&#39;: &#39;d83e3c028de1950c4f8dedae21eb90f90c4ed6c3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Translational profiling of macrophages infected with Leishmania donovani identifies mTOR- and eIF4A-sensitive immune-related transcripts&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.20.884338&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe protozoan parasite Leishmania donovani (L. donovani) causes visceral leishmaniasis, a chronic infection which is fatal when untreated. While previous studies showed that L. donovani reprograms transcription to subvert host cell functions, it remains unclear whether the parasite also alters host mRNA translation to establish a successful infection. To assess this, we compared transcriptome-wide translation in primary mouse macrophages infected with L. donovani promastigotes or amastigotes using polysome-profiling. This identified ample selective changes in translation (3,127 transcripts) which were predicted to target central cellular functions by inducing synthesis of proteins related to chromatin remodeling and RNA metabolism while inhibiting those related to intracellular trafficking and antigen presentation. Parallel quantification of protein and mRNA levels for a set of transcripts whose translation was activated upon L. donovani infection (Papbpc1, Eif2ak2, and Tgfb) confirmed, as indicated by polysome-profiling, increased protein levels despite largely unaltered mRNA levels. Mechanistic in silico analyses suggested activated translation depending on the kinase mTOR (e.g. Pabpc1) and the RNA helicase eIF4A (e.g. Tgfb) during infection. Accordingly, treatment with mTOR inhibitors torin-1 or rapamycin reversed L. donovani-induced PABPC1 without affecting corresponding transcript levels. Similarly, the production of TGF-β decreased in presence of the eIF4A inhibitor silvestrol despite unaltered Tgfb mRNA levels. Consistent with parasite modulation of host eIF4A-sensitive translation to promote infection, silvestrol suppressed L. donovani replication within macrophages. In contrast, parasite survival was favored under mTOR inhibition. In summary, infection-associated changes in translation of mTOR- and eIF4A-sensitive mRNAs contribute to modulate mRNA metabolism and immune responses in L. donovani-infected macrophages. Although the net outcome of such translation programs favours parasite propagation, individual translation programs appear to have opposing roles during L. donovani infection, thereby suggesting their selective targeting as key for therapeutic effects.Author SummaryFine-tuning the efficiency of mRNA translation into proteins allows cells to tailor their responses to stress without the need for synthesizing new mRNA molecules. It is well established that the protozoan parasite Leishmania donovani alters transcription of specific genes to subvert host cell functions. However, discrepancies between transcriptomic and proteomic data suggest that post-transcriptional regulatory mechanisms also contribute to modulate host gene expression programs during L. donovani infection. Herein, we report that one third of protein-coding mRNAs expressed in macrophages are differentially translated upon infection with L. donovani. Our computational analyses reveal that subsets of mRNAs encoding functionally related proteins share the same directionality of translational regulation, which is likely to impact metabolic and microbicidal activity of infected cells. We also show that upregulated translation of transcripts that encode central regulators of mRNA metabolism and inflammation is sensitive to the activation of mTOR or eIF4A during infection. Finally, we observe that inhibition of eIF4A activity reduces parasite survival within macrophages while selective blockade of mTOR has the opposite effect. Thus, our study points to a dual role for translational control of host gene expression during L. donovani infection and suggests that novel regulatory nodes could be targeted for therapeutic intervention.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-20&#39;,
  &#39;authors&#39;: &#39;Visnu Chaparro; Louis-Philippe Leroux; Laia Masvidal; Julie Lorent; Tyson E. Graber; Aude Zimmermann; Guillermo Arango Duque; Albert Descoteaux; Tommy Alain; Ola Larsson; Maritza Jaramillo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.20.884338&#39;},
 {&#39;cord_uid&#39;: &#39;ahs2wdus&#39;,
  &#39;sha&#39;: &#39;48df5b06710467cbe5b41dd851e946fc936ecf15&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Low basal expression and slow induction of IFITM3 puts immune cells at risk of influenza A infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.20.885590&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe interferon-induced transmembrane protein, IFITM3, has been shown to restrict influenza virus infection in murine and in vitro settings for ten years, but no explanation has been found to explain why this virus infection is so highly contagious and infects most individuals it comes in contact with. We confirm that the expression level of IFITM3 plays a role in determining the level of viral infection through manipulation of IFITM3 levels with interferon (IFN) stimulation and overexpression systems. Low basal expression may put some immune cells, including lymphocytes and lung-resident macrophages, at risk of influenza virus infection. Investigating the induction of IFITM3 by IFN, we find a strong preference for Type I IFN in IFITM3 induction in both cell lines and primary human cells. While myeloid cells can increase expression following stimulation by Type I IFN, lymphocytes show minimal induction of IFITM3 following IFN stimulation, suggesting that they are always at risk of viral infection. Surprisingly, we found that the time it takes for maximal induction of IFITM3 is relatively slow for an interferon-stimulated gene at around 36 hours. Low basal expression and slow induction of IFITM3 could increase the risk of influenza virus infection in selected immune cells.ImportanceInfluenza virus infection remains one of the top ten threats to global health, causing significant deaths and hospitalisations across the world each year. Understanding mechanisms for controlling influenza virus infection remain a priority. The interferon-induced transmembrane protein IFITM3 can restrict influenza infection by limiting replication of the virus. The precise mechanisms of how IFITM3 reduced replication of influenza are unknown, although it is predicted to prevent release of viral contents into the cytosol by preventing pore formation on the endosomal compartments where it is suggested to reside. Here we have shown that the expression level of IFITM3 is important in determining the control of influenza virus infection. We find an expression pattern for IFITM3 that varies based on cell type, tissue locality, differentiation state and cell naivety, all of which highlights cells that may be at the highest risk of influenza infection.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-23&#39;,
  &#39;authors&#39;: &#39;Dannielle Wellington; Zixi Yin; Liwei Zhang; Jessica Forbester; Kerry Kite; Henry Laurenson-Schafer; Shokouh Makvandi-Nejad; Boquan Jin; Emma Bowes; Krishnageetha Manoharan; David Maldonado-Perez; Clare Verill; Ian Humphreys; Tao Dong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.20.885590&#39;},
 {&#39;cord_uid&#39;: &#39;ts91ivwf&#39;,
  &#39;sha&#39;: &#39;106f10eaae4fb67aca5b2bd737ce2bf1c27fa243&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community&#39;,
  &#39;doi&#39;: &#39;10.1101/2019.12.21.885921&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMost emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens.&#39;,
  &#39;publish_time&#39;: &#39;2019-12-23&#39;,
  &#39;authors&#39;: &#39;Daniel J. Becker; Kelly A. Speer; Alexis M. Brown; M. Brock Fenton; Alex D. Washburne; Sonia Altizer; Daniel G. Streicker; Raina K. Plowright; Vladimir E. Chizhikov; Nancy B. Simmons; Dmitriy V. Volokhov&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2019.12.21.885921&#39;},
 {&#39;cord_uid&#39;: &#39;2vecg9op&#39;,
  &#39;sha&#39;: &#39;09c9fcabc66a106e01ef42247cbd86b6d85bd67f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Mono-ADP-ribosylation by ARTD10 restricts Chikungunya virus replication by interfering with the proteolytic activity of nsP2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.07.896977&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA subset of intracellular mono-ADP-ribosyltransferases diphtheria toxin-like (ARTDs, aka mono-PARPs) is induced by type I interferons. Some of these mono-ARTDs feature antiviral activity while certain RNA viruses, including Chikungunya virus (CHIKV), encode mono-ADP-ribosylhydrolases, suggesting a role for mono-ADP-ribosylation (MARylation) in host-virus conflicts. CHIKV expresses four non-structural proteins (nsP1-nsP4), with nsP3 containing a macrodomain that hydrolyzes and thereby reverses protein MARylation in vitro and in cells. This de-MARylation activity is essential as hydrolase inactivating mutations result in replication defective virus. However, the substrates of MARylation during CHIKV infection are unknown and thus it is unclear how the macrodomain contributes to virus replication and how mono-ARTD-dependent MARylation confers antiviral immunity. We identified ARTD10 and ARTD12 as restriction factors for CHIKV replication in a catalytic activity-dependent manner. CHIKV replication requires processing of the non-structural polyprotein nsP1-4 by the nsP2-encoded protease and the assembly of the four individual nsPs into a functional replication complex. Expression of ARTD10 and ARTD12 resulted in a reduction of processed nsPs. Similarly, MAR hydrolase inactive CHIKV replicon mutants revealed a decrease in processed nsPs, comparable to an nsP2 protease defective mutant. This suggested that the macrodomain contributes to nsP2 protease activity. In support, a hydrolase-deficient virus was complemented by a protease-deficient virus. We hypothesized that MARylation regulates the proteolytic function of nsP2. Indeed, we found that nsP2 is MARylated by ARTD10. This inhibited nsP2 protease activity, thereby preventing polyprotein processing and consequently virus replication. This inhibition was antagonized by the MAR hydrolase activity of nsP3. Together, our findings provide a mechanistic explanation for the need of the viral MAR hydrolase for efficient replication of CHIKV.Author SummaryInfectious diseases still pose major health threats. Especially fast evolving viruses find ever new strategies to manipulate the immune response. With climate warming and increased human mobility vector-borne pathogens like Chikungunya virus (CHIKV) spread and cause world-wide epidemics. Beyond the acute phase, CHIKV patients regularly suffer from chronic rheumatism. This entails a decline in life quality and an economic burden. To date no drugs are approved and the mode of pathogenesis remains elusive. Here we describe a mechanistic function of the CHIKV nsP3 macrodomain. We found that the viral nsP2 is mono-ADP-ribosylated interfering with its auto-proteolytic function. The nsP3 macrodomain removes this modification and restores the protease activity that is essential for replication. Because macrodomains are highly conserved they might represent broad antiviral targets.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-08&#39;,
  &#39;authors&#39;: &#39;Sarah Krieg; Fabian Pott; Laura Eckei; Maud Verheirstraeten; Mareike Bütepage; Barbara Lippok; Christine Goffinet; Bernhard Lüscher; Patricia Verheugd&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.07.896977&#39;},
 {&#39;cord_uid&#39;: &#39;2f9nc2to&#39;,
  &#39;sha&#39;: &#39;b80d9de7750d627a0189fcd2dfd82c7efcb1c079&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An interaction network of RNA-binding proteins involved in Drosophila oogenesis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.08.899146&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDuring Drosophila oogenesis, the localization and translational regulation of maternal transcripts relies on RNA-binding proteins (RBPs). Many of these RBPs localize several mRNAs and may have additional direct interaction partners to regulate their functions. Using immunoprecipitation from whole Drosophila ovaries coupled to mass spectrometry, we examined protein-protein associations of 6 GFP-tagged RBPs expressed at physiological levels. Analysis of the interaction network and further validation in human cells allowed us to identify 26 previously unknown associations, besides recovering several well characterized interactions. We identified interactions between RBPs and several splicing factors, providing links between nuclear and cytoplasmic events of mRNA regulation. Additionally, components of the translational and RNA decay machineries were selectively co-purified with some baits, suggesting a mechanism for how RBPs may regulate maternal transcripts. Given the evolutionary conservation of the studied RBPs, the interaction network presented here provides the foundation for future functional and structural studies of mRNA localization across metazoans.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-09&#39;,
  &#39;authors&#39;: &#39;Prashali Bansal; Johannes Madlung; Kristina Schaaf; Boris Macek; Fulvia Bono&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.08.899146&#39;},
 {&#39;cord_uid&#39;: &#39;8iet67u3&#39;,
  &#39;sha&#39;: &#39;e18620d5cff022d18aa23da2fa4634331fa54860&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Altered polarization of PAR-2 signaling during airway epithelial remodeling&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.09.900555&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTBackgroundProtease-activated receptor 2 (PAR-2) is activated by proteases involved in allergy and triggers airway epithelial secretion and inflammation. PAR-2 is normally expressed basolaterally in differentiated nasal ciliated cells.ObjectiveWe tested if epithelial remodeling during diseases characterized by loss of cilia and squamous metaplasia may alter PAR-2 polarization.MethodsEndogenous PAR-2 responses were measured by live cell calcium and cilia imaging, measurement of fluid secretion, and quantification of cytokines. We utilized airway squamous cell lines, primary differentiated air-liquid interface cultures, and tissue explants. Cells were exposed to disease-related modifiers that alter epithelial morphology, including IL-13, cigarette smoke condensate, and retinoic acid deficiency. We used concentrations and exposure times that altered epithelial morphology without causing breakdown of the epithelial barrier, likely reflecting early disease states.ResultsPAR-2 signaling in airway squamous cells activated calcium and inflammatory responses. Squamous cells cultured at air liquid interface (ALI) responded to PAR-2 agonists applied both apically and basolaterally. Primary well-differentiated nasal epithelial ALI cultures responded only to basolateral PAR-2 stimulation. Primary cultures exposed to IL-13, cigarette smoke condensate, or reduced retinoic acid responded to both apical and basolateral PAR-2 stimulation. Nasal polyp tissue, but not control middle turbinate, exhibited apical calcium responses to PAR-2 stimulation. However, isolated ciliated cells from both polyp and turbinate maintained basolateral PAR-2 polarization.ConclusionsSquamous metaplasia and/or loss of cilia enhances apical PAR-2 responses. Altered PAR-2 polarization in dedifferentiated or remodeled epithelia may contribute to increased sensitivity to inhaled protease allergens in inflammatory airway diseases.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-09&#39;,
  &#39;authors&#39;: &#39;Ryan M. Carey; Jenna R. Freund; Benjamin M. Hariri; Nithin D. Adappa; James N. Palmer; Robert J. Lee&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.09.900555&#39;},
 {&#39;cord_uid&#39;: &#39;wtarrhf0&#39;,
  &#39;sha&#39;: &#39;0624a12abfe85c8b5070850d912a2db4cd453236&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;TALC: Transcription Aware Long Read Correction&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.10.901728&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTMotivationLong-read sequencing technologies are invaluable for determining complex RNA transcript architectures but are error-prone. Numerous “hybrid correction” algorithms have been developed for genomic data that correct long reads by exploiting the accuracy and depth of short reads sequenced from the same sample. These algorithms are not suited for correcting more complex transcriptome sequencing data.ResultsWe have created a novel algorithm called TALC (Transcription Aware Long Read Correction) which models changes in RNA expression and isoform representation in a weighted De-Bruijn graph to correct long reads from transcriptome studies. We show that transcription aware correction by TALC improves the accuracy of the whole spectrum of downstream RNA-seq applications and is thus necessary for transcriptome analyses that use long read technology.Availability and ImplementationTALC is implemented in C++ and available at https://gitlab.igh.cnrs.fr/lbroseus/TALC.Contactwilliam.ritchie@igh.cnrs.fr&#39;,
  &#39;publish_time&#39;: &#39;2020-01-11&#39;,
  &#39;authors&#39;: &#39;Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.10.901728&#39;},
 {&#39;cord_uid&#39;: &#39;ahrrphm8&#39;,
  &#39;sha&#39;: &#39;0015023cc06b5362d332b3baf348d11567ca2fbb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The RNA pseudoknots in foot-and-mouth disease virus are dispensable for genome replication but essential for the production of infectious virus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.10.901801&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe positive stranded RNA genomes of picornaviruses comprise a single large open reading frame flanked by 5′ and 3′ untranslated regions (UTRs). Foot-and-mouth disease virus (FMDV) has an unusually large 5′ UTR (1.3 kb) containing five structural domains. These include the internal ribosome entry site (IRES), which facilitates initiation of translation, and the cis-acting replication element (cre). Less well characterised structures are a 5′ terminal 360 nucleotide stem-loop, a variable length poly-C-tract of approximately 100-200 nucleotides and a series of two to four tandemly repeated pseudoknots (PKs). We investigated the structures of the PKs by selective 2′ hydroxyl acetylation analysed by primer extension (SHAPE) analysis and determined their contribution to genome replication by mutation and deletion experiments. SHAPE and mutation experiments confirmed the importance of the previously predicted PK structures for their function. Deletion experiments showed that although PKs are not essential for replication, they provide genomes with a competitive advantage. However, although replicons and full-length genomes lacking all PKs were replication competent, no infectious virus was rescued from genomes containing less than one PK copy. This is consistent with our earlier report describing the presence of putative packaging signals in the PK region.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-11&#39;,
  &#39;authors&#39;: &#39;Joseph C. Ward; Lidia Lasecka-Dykes; Chris Neil; Oluwapelumi Adeyemi; Sarah Gold; Niall McLean; Caroline Wright; Morgan R. Herod; David Kealy; Emma Warner; Donald P. King; Tobias J. Tuthill; David J. Rowlands; Nicola J. Stonehouse&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.10.901801&#39;},
 {&#39;cord_uid&#39;: &#39;8kced06y&#39;,
  &#39;sha&#39;: &#39;f8e8a9c6f5662c1739be031e6134284ef08ea8b2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Programmable low-cost DNA-based platform for viral RNA detection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.12.902452&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractViral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-16&#39;,
  &#39;authors&#39;: &#39;Lifeng Zhou; Arun Richard Chandrasekaran; Jibin Abraham Punnoose; Gaston Bonenfant; Stephon Charles; Oksana Levchenko; Pheonah Badu; Cassandra Cavaliere; Cara T. Pager; Ken Halvorsen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.12.902452&#39;},
 {&#39;cord_uid&#39;: &#39;sjicr24s&#39;,
  &#39;sha&#39;: &#39;de3ae7971891058db2f4b36cd216531e6f533332&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.15.908608&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I-β2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and β2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.IMPORTANCEUrsidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-16&#39;,
  &#39;authors&#39;: &#39;Hongyu Yuan; Lizhen Ma; Lijie Zhang; Xiaoying Li; Chun Xia&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.15.908608&#39;},
 {&#39;cord_uid&#39;: &#39;bf4qpsy7&#39;,
  &#39;sha&#39;: &#39;26cb6703ca72bf97887abbc29a40b1bd9d7890f4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Hotspots of aberrant enhancer activity in fibrolamellar carcinoma reveal molecular mechanisms of oncogenesis and intrinsic drug resistance&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.18.911297&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractFibrolamellar carcinoma (FLC) is a rare, therapeutically intractable liver cancer that disproportionately affects youth. Although FLC tumors exhibit a distinct gene expression profile, the causative transcriptional mechanisms remain unclear. Here we used chromatin run-on sequencing to discover approximately 7,000 enhancers and 141 enhancer hotspots activated in FLC relative to non-malignant liver. Detailed bioinformatic analyses revealed aberrant ERK/MEK signaling and candidate master transcriptional regulators. We also defined the genes most strongly associated with aberrant FLC enhancer activity, including CA12 and SLC16A14. Treatment of FLC cell models with inhibitors of CA12 or SLC16A14 independently reduced cell viability and/or significantly enhanced the effect of MEK inhibitor cobimetinib. These findings highlight new molecular targets for drug development as well as novel drug combination approaches.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-18&#39;,
  &#39;authors&#39;: &#39;Timothy A. Dinh; Ramja Sritharan; F. Donelson Smith; Adam B. Francisco; Rosanna K. Ma; Rodica P. Bunaciu; Matt Kanke; Charles G. Danko; Andrew P. Massa; John D. Scott; Praveen Sethupathy&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.18.911297&#39;},
 {&#39;cord_uid&#39;: &#39;v4mbry22&#39;,
  &#39;sha&#39;: &#39;8b88d01563538e4abd29c3ba493c534af30664cc&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A mathematical model for simulating the transmission of Wuhan novel Coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.19.911669&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-19&#39;,
  &#39;authors&#39;: &#39;Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.19.911669&#39;},
 {&#39;cord_uid&#39;: &#39;3qds5j1p&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.20.913368&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDetailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-21&#39;,
  &#39;authors&#39;: &#39;Ning Dong; Xuemei Yang; Lianwei Ye; Kaichao Chen; Edward Wai-Chi Chan; Mengsu Yang; Sheng Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.20.913368&#39;},
 {&#39;cord_uid&#39;: &#39;t9bt70f2&#39;,
  &#39;sha&#39;: &#39;a05179c57d5e1cdddc839ff3c0b70a614efbb6cb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.21.914044&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it.One Sentence SummaryIt is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. Applied to the Wuhan 2019 Novel Coronavirus, our prediction demonstrated that several vertebrate-infectious coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-24&#39;,
  &#39;authors&#39;: &#39;Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.21.914044&#39;},
 {&#39;cord_uid&#39;: &#39;f1ixbzx8&#39;,
  &#39;sha&#39;: &#39;403d885a33958731c4607ea632453e25a89e2b57&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Novel ionophores active against La Crosse virus identified through rapid antiviral screening&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.21.914929&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBunyaviruses are significant human pathogens, causing diseases ranging from hemorrhagic fevers to encephalitis. Among these viruses, La Crosse virus (LACV), a member of the California serogroup, circulates in the eastern and midwestern United States. While LACV infection is often asymptomatic, dozens of cases of encephalitis are reported yearly. Unfortunately, no antivirals have been approved to treat LACV infection. Here, we developed a method to rapidly test potential antivirals against LACV infection. From this screen, we identified several potential antiviral molecules, including known antivirals. Additionally, we identified many novel antivirals that exhibited antiviral activity without affecting cellular viability. Valinomycin, a potassium ionophore, was among our top targets. We found that valinomycin exhibited potent anti-LACV activity in multiple cell types in a dose-dependent manner. Valinomycin did not affect particle stability or infectivity, suggesting that it may preclude virus replication by altering cellular potassium ions, a known determinant of LACV entry. We extended these results to other ionophores and found that the antiviral activity of valinomycin extended to other viral families including bunyaviruses (Rift Valley fever virus, Keystone virus), enteroviruses (Coxsackievirus, rhinovirus), flavirivuses (Zika), and coronaviruses (229E and MERS-CoV). In all viral infections, we observed significant reductions in virus titer in valinomycin-treated cells. In sum, we demonstrate the importance of potassium ions to virus infection, suggesting a potential therapeutic target to disrupt virus replication.ImportanceNo antivirals are approved for the treatment of bunyavirus infection. The ability to rapidly screen compounds and identify novel antivirals is one means to accelerate drug discovery for viruses with no approved treatments. We used this approach to screen hundreds of compounds against La Crosse virus, an emerging bunyavirus that causes significant disease, including encephalitis. We identified several known and previously unidentified antivirals. We focused on a potassium ionophore, valinomycin, due to its promising in vitro antiviral activity. We demonstrate that valinomycin, as well as a selection of other ionophores, exhibits activity against La Crosse virus as well as several other distantly related bunyaviruses. We finally observe that valinomycin has activity against a wide array of human viral pathogens, suggesting that disrupting potassium ion homeostasis with valinomycin may be a potent host pathway to target to quell virus infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-23&#39;,
  &#39;authors&#39;: &#39;Zachary J. Sandler; Michelle N. Vu; Vineet D. Menachery; Bryan C. Mounce&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.21.914929&#39;},
 {&#39;cord_uid&#39;: &#39;o47v5vgw&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.22.914952&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Since the SARS outbreak 18 years ago, a large number of severe acute respiratory syndrome related coronaviruses (SARSr-CoV) have been discovered in their natural reservoir host, bats1-4. Previous studies indicated that some of those bat SARSr-CoVs have the potential to infect humans5-7. Here we report the identification and characterization of a novel coronavirus (nCoV-2019) which caused an epidemic of acute respiratory syndrome in humans, in Wuhan, China. The epidemic, started from December 12th, 2019, has caused 198 laboratory confirmed infections with three fatal cases by January 20th, 2020. Full-length genome sequences were obtained from five patients at the early stage of the outbreak. They are almost identical to each other and share 79.5% sequence identify to SARS-CoV. Furthermore, it was found that nCoV-2019 is 96% identical at the whole genome level to a bat coronavirus. The pairwise protein sequence analysis of seven conserved non-structural proteins show that this virus belongs to the species of SARSr-CoV. The nCoV-2019 virus was then isolated from the bronchoalveolar lavage fluid of a critically ill patient, which can be neutralized by sera from several patients. Importantly, we have confirmed that this novel CoV uses the same cell entry receptor, ACE2, as SARS-CoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-23&#39;,
  &#39;authors&#39;: &#39;Peng Zhou; Xing-Lou Yang; Xian-Guang Wang; Ben Hu; Lei Zhang; Wei Zhang; Hao-Rui Si; Yan Zhu; Bei Li; Chao-Lin Huang; Hui-Dong Chen; Jing Chen; Yun Luo; Hua Guo; Ren-Di Jiang; Mei-Qin Liu; Ying Chen; Xu-Rui Shen; Xi Wang; Xiao-Shuang Zheng; Kai Zhao; Quan-Jiao Chen; Fei Deng; Lin-Lin Liu; Bing Yan; Fa-Xian Zhan; Yan-Yi Wang; Geng-Fu Xiao; Zheng-Li Shi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.22.914952&#39;},
 {&#39;cord_uid&#39;: &#39;tyhwtt7j&#39;,
  &#39;sha&#39;: &#39;9daed5e675ac113d1939056dcebbb1f32ec0d7ec&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.22.915660&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractOver the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-22&#39;,
  &#39;authors&#39;: &#39;Michael Letko; Vincent Munster&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.22.915660&#39;},
 {&#39;cord_uid&#39;: &#39;mt8h4rk5&#39;,
  &#39;sha&#39;: &#39;6cb02c7565f6f74a3d165e14196de5e9e87d2d04&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.23.20018549&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since first identified, the epidemic scale of the recently emerged novel coronavirus (2019-nCoV) in Wuhan, China, has increased rapidly, with cases arising across China and other countries and regions. using a transmission model, we estimate a basic reproductive number of 3.11 (95%CI, 2.39-4.13); 58-76% of transmissions must be prevented to stop increasing; Wuhan case ascertainment of 5.0% (3.6-7.4); 21022 (11090-33490) total infections in Wuhan 1 to 22 January.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-24&#39;,
  &#39;authors&#39;: &#39;Jonathan M Read; Jessica RE Bridgen; Derek AT Cummings; Antonia Ho; Chris P Jewell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.23.20018549&#39;},
 {&#39;cord_uid&#39;: &#39;px70bft2&#39;,
  &#39;sha&#39;: &#39;24ff82a421f1a33db3ebaa22652bd62e0cb77c57&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.23.916395&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundsAn ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.MethodsAccounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (γ), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI.FindingsThe early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.ConclusionThe mean estimate of R0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-24&#39;,
  &#39;authors&#39;: &#39;Shi Zhao; Qianyin Lin; Jinjun Ran; Salihu S Musa; Guangpu Yang; Weiming Wang; Yijun Lou; Daozhou Gao; Lin Yang; Daihai He; Maggie H Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.23.916395&#39;},
 {&#39;cord_uid&#39;: &#39;59rcogfc&#39;,
  &#39;sha&#39;: &#39;fe66306680e5be9820e2cc3742b663a782eb1b31&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.23.916726&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;We present a timely evaluation of the Chinese 2019-nCov epidemic in its initial phase, where 2019-nCov demonstrates comparable transmissibility but lower fatality rates than SARS and MERS. A quick diagnosis that leads to case isolation and integrated interventions will have a major impact on its future trend. Nevertheless, as China is facing its Spring Festival travel rush and the epidemic has spread beyond its borders, further investigation on its potential spatiotemporal transmission pattern and novel intervention strategies are warranted.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-25&#39;,
  &#39;authors&#39;: &#39;Mingwang Shen; Zhihang Peng; Yanni Xiao; Lei Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.23.916726&#39;},
 {&#39;cord_uid&#39;: &#39;jjkf5gg3&#39;,
  &#39;sha&#39;: &#39;671b44ae9bec89cacf7ae9e7b2a79649a9b66773&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Pattern of early human-to-human transmission of Wuhan 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.23.917351&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTOn December 31, 2019, the World Health Organization was notified about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China. Chinese authorities later identified a new coronavirus (2019-nCoV) as the causative agent of the outbreak. As of January 23, 2020, 655 cases have been confirmed in China and several other countries. Understanding the transmission characteristics and the potential for sustained human-to-human transmission of 2019-nCoV is critically important for coordinating current screening and containment strategies, and determining whether the outbreak constitutes a public health emergency of international concern (PHEIC). We performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date. We found the basic reproduction number, R0, to be around 2.2 (90% high density interval 1.4—3.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of a similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and the 1918 pandemic influenza. These findings underline the importance of heightened screening, surveillance and control efforts, particularly at airports and other travel hubs, in order to prevent further international spread of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-24&#39;,
  &#39;authors&#39;: &#39;Julien Riou; Christian L. Althaus&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.23.917351&#39;},
 {&#39;cord_uid&#39;: &#39;j99qg2a0&#39;,
  &#39;sha&#39;: &#39;e4cb4af2f4ec6f023096afcf7b3d4410584e1393&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Designing Effective small interfering RNA for Post-Transcriptional Silencing of Human GREM1: A Comprehensive Bioinformatics Approach&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.23.917559&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTHuman gremlin-1 is a physiologically versatile signaling molecule that has been associated with several human diseases including cancer. The ability of gremlin-1 to induce fibrosis in organs and transduce angiogenesis makes it a target for cancer therapy. RNAi-based therapy has proven to be very efficient and specific in tumor growth inhibition. The efficacy and specificity of siRNA-mediated gene silencing depends on the designing approaches. Here, empirical guidelines for siRNA design and comprehensive target site availability analysis were used to select effective siRNA from a plethora of potential candidates designed using several computation algorithms. Then, the selected siRNA candidates were subjected to stringent similarity searches in order to obtain siRNA candidates with reduced off-target effects (high specificity). The best candidates were compared to experimentally successful gremlin-1 siRNAs in order to predict the silencing potency of the selected siRNAs. siRNA-6 (sense strand: 5’-CCAAGAAAUUCACUACCAU-3’), siRNA-7 (sense strand: 5’-CCAUGAUGGUCACACUCAA-3’) and siRNA-47 (sense strand: 5’-GGCCCAGCACAAUGACUCA-3’) were predicted to be highly effective siRNA candidates for gremlin-1 silencing. These siRNAs can be considered for RNAi-based therapy because off-target effects are predicted to be minimal.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-24&#39;,
  &#39;authors&#39;: &#39;Asiedu Ebenezer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.23.917559&#39;},
 {&#39;cord_uid&#39;: &#39;u2w2bb2p&#39;,
  &#39;sha&#39;: &#39;a3116a1ea2e157c8855023d968cc0f0bd6c9846f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The 2019-new coronavirus epidemic: evidence for virus evolution&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.24.915157&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThere is concern about a new coronavirus, the 2019-nCoV, as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequence of 2019-nCoV and 12 whole genome sequences highly similar sequences available in gene bank (5 from SARS, 2 from MERS and 5 from Bat SARS-like Coronavirus). FUBAR analysis shows that the Nucleocapsid and the Spike Glycoprotein has some sites under positive pressure while homology modelling helped to explain some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019.nCoV significantly clustered with Bat SARS-like Coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in S and nucleocapsid proteins. From these results, 2019nCoV could be considered a coronavirus distinct from SARS virus, probably transmitted from bats or another host where mutations conferred upon it the ability to infect humans.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-24&#39;,
  &#39;authors&#39;: &#39;Domenico Benvenuto; Marta Giovannetti; Alessandra Ciccozzi; Silvia Spoto; Silvia Angeletti; Massimo Ciccozzi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.24.915157&#39;},
 {&#39;cord_uid&#39;: &#39;qpaig5tm&#39;,
  &#39;sha&#39;: &#39;b6e92eda6c26b456165422d4e5552b81754c4a82&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;2019-20 Wuhan coronavirus outbreak: Intense surveillance is vital for preventing sustained transmission in new locations&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.24.919159&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe outbreak of pneumonia originating in Wuhan, China, has generated 830 confirmed cases, including 26 deaths, as of 24 January 2020. The virus (2019-nCoV) has spread elsewhere in China and to other countries, including South Korea, Thailand, Japan and USA. Fortunately, there has not yet been evidence of sustained human-to-human transmission outside of China. Here we assess the risk of sustained transmission whenever the coronavirus arrives in other countries. Data describing the times from symptom onset to hospitalisation for 47 patients infected in the current outbreak are used to generate an estimate for the probability that an imported case is followed by sustained human-to-human transmission. Under the assumptions that the imported case is representative of the patients in China, and that the 2019-nCoV is similarly transmissible to the SARS coronavirus, the probability that an imported case is followed by sustained human-to-human transmission is 0.37. However, if the mean time from symptom onset to hospitalisation can be halved by intense surveillance, then the probability that an imported case leads to sustained transmission is only 0.005. This emphasises the importance of current surveillance efforts in countries around the world, to ensure that the ongoing outbreak will not become a large global epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-25&#39;,
  &#39;authors&#39;: &#39;R.N. Thompson&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.24.919159&#39;},
 {&#39;cord_uid&#39;: &#39;5jai745w&#39;,
  &#39;sha&#39;: &#39;bb97f90ff5bfd41f2f1148c5cbdb257fdba535df&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.24.919183&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1–3. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-25&#39;,
  &#39;authors&#39;: &#39;Fan Wu; Su Zhao; Bin Yu; Yan-Mei Chen; Wen Wang; Yi Hu; Zhi-Gang Song; Zhao-Wu Tao; Jun-Hua Tian; Yuan-Yuan Pei; Ming-Li Yuan; Yu-Ling Zhang; Fa-Hui Dai; Yi Liu; Qi-Min Wang; Jiao-Jiao Zheng; Lin Xu; Edward C. Holmes; Yong-Zhen Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.24.919183&#39;},
 {&#39;cord_uid&#39;: &#39;hab8h5gz&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.24.919241&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;This manuscript has been withdrawn as it was submitted without the full consent of all the authors. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-25&#39;,
  &#39;authors&#39;: &#39;Zeliang Chen; Wenjun Zhang; Yi Lu; Cheng Guo; Zhongmin Guo; Conghui Liao; Xi Zhang; Yi Zhang; Xiaohu Han; Qianlin Li; W. Ian Lipkin; Jiahai Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.24.919241&#39;},
 {&#39;cord_uid&#39;: &#39;9d8k7b23&#39;,
  &#39;sha&#39;: &#39;e29b287d326be9c2c0a7dcd7a81cb49cc2125360&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.24.919282&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.ImportanceSince 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-25&#39;,
  &#39;authors&#39;: &#39;Sean M. Boyles; Carla N. Mavian; Esteban Finol; Maria Ukhanova; Caroline J. Stephenson; Gabriela Hamerlinck; Seokyoung Kang; Caleb Baumgartner; Mary Geesey; Israel Stinton; Kate Williams; Derrick K. Mathias; Mattia Prosperi; Volker Mai; Marco Salemi; Eva A. Buckner; John A. Lednicky; Adam R. Rivers; Rhoel R. Dinglasan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.24.919282&#39;},
 {&#39;cord_uid&#39;: &#39;t1iagum7&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;MRCA time and epidemic dynamics of the 2019 novel coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.25.919688&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe 2019 novel coronavirus (2019-nCoV) have emerged from Wuhan, China. Studying the epidemic dynamics is crucial for further surveillance and control of the outbreak. We employed a Bayesian framework to infer the time-calibrated phylogeny and the epidemic dynamics represented by the effective reproductive number (Re) changing over time from 33 genomic sequences available from GISAID. The time of the most recent common ancestor (MRCA) was December 17, 2019 (95% HPD: December 7, 2019 – December 23, 2019). The median estimate of Re shifted from 1.6 to 1.1 on around January 1, 2020. This study provides an early insight of the 2019-nCoV epidemic. However, due to limited amount of data, one should be cautious when interpreting the results at this stage.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-26&#39;,
  &#39;authors&#39;: &#39;Chi Zhang; Mei Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.25.919688&#39;},
 {&#39;cord_uid&#39;: &#39;3e2soc6w&#39;,
  &#39;sha&#39;: &#39;abcfffafab399149d4adadd6bb458c4994e2025d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.25.919787&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTRationaleSeveral studies have estimated basic production number of novel coronavirus pneumonia (NCP). However, the time-varying transmission dynamics of NCP during the outbreak remain unclear.ObjectivesWe aimed to estimate the basic and time-varying transmission dynamics of NCP across China, and compared them with SARS.MethodsData on NCP cases by February 7, 2020 were collected from epidemiological investigations or official websites. Data on severe acute respiratory syndrome (SARS) cases in Guangdong Province, Beijing and Hong Kong during 2002-2003 were also obtained. We estimated the doubling time, basic reproduction number (R0) and time-varying reproduction number (Rt) of NCP and SARS.Measurements and main resultsAs of February 7, 2020, 34,598 NCP cases were identified in China, and daily confirmed cases decreased after February 4. The doubling time of NCP nationwide was 2.4 days which was shorter than that of SARS in Guangdong (14.3 days), Hong Kong (5.7 days) and Beijing (12.4 days). The R0 of NCP cases nationwide and in Wuhan were 4.5 and 4.4 respectively, which were higher than R0 of SARS in Guangdong (R0=2.3), Hongkong (R0=2.3), and Beijing (R0=2.6). The Rt for NCP continuously decreased especially after January 16 nationwide and in Wuhan. The R0 for secondary NCP cases in Guangdong was 0.6, and the Rt values were less than 1 during the epidemic.ConclusionsNCP may have a higher transmissibility than SARS, and the efforts of containing the outbreak are effective. However, the efforts are needed to persist in for reducing time-varying reproduction number below one.At a Glance CommentaryScientific Knowledge on the SubjectSince December 29, 2019, pneumonia infection with 2019-nCoV, now named as Novel Coronavirus Pneumonia (NCP), occurred in Wuhan, Hubei Province, China. The disease has rapidly spread from Wuhan to other areas. As a novel virus, the time-varying transmission dynamics of NCP remain unclear, and it is also important to compare it with SARS.What This Study Adds to the FieldWe compared the transmission dynamics of NCP with SARS, and found that NCP has a higher transmissibility than SARS. Time-varying production number indicates that rigorous control measures taken by governments are effective across China, and persistent efforts are needed to be taken for reducing instantaneous reproduction number below one.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-26&#39;,
  &#39;authors&#39;: &#39;Tao Liu; Jianxiong Hu; Jianpeng Xiao; Guanhao He; Min Kang; Zuhua Rong; Lifeng Lin; Haojie Zhong; Qiong Huang; Aiping Deng; Weilin Zeng; Xiaohua Tan; Siqing Zeng; Zhihua Zhu; Jiansen Li; Dexin Gong; Donghua Wan; Shaowei Chen; Lingchuan Guo; Yan Li; Limei Sun; Wenjia Liang; Tie Song; Jianfeng He; Wenjun Ma&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.25.919787&#39;},
 {&#39;cord_uid&#39;: &#39;ykofrn9i&#39;,
  &#39;sha&#39;: &#39;cbc05d14c57b91081970a232ab83bc993f998fe2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.26.20018754&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Natalie M. Linton; Tetsuro Kobayashi; Yichi Yang; Katsuma Hayashi; Andrei R. Akhmetzhanov; Sung-mok Jung; Baoyin Yuan; Ryo Kinoshita; Hiroshi Nishiura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.26.20018754&#39;},
 {&#39;cord_uid&#39;: &#39;n93cm6ws&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological identification of a novel infectious disease in real time: Analysis of the atypical pneumonia outbreak in Wuhan, China, 2019-20&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.26.20018887&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: Virological tests indicate that a novel coronavirus is the most likely explanation for the 2019-20 pneumonia outbreak in Wuhan, China. We demonstrate that non-virological descriptive characteristics could have determined that the outbreak is caused by a novel pathogen in advance of virological testing. Methods: Characteristics of the ongoing outbreak were collected in real time from two medical social media sites. These were compared against characteristics of ten existing pathogens that can induce atypical pneumonia. The probability that the current outbreak is due to &#34;Disease X&#34; (i.e., previously unknown etiology) as opposed to one of the known pathogens was inferred, and this estimate was updated as the outbreak continued. Results: The probability that Disease X is driving the outbreak was assessed as over 32% on 31 December 2019, one week before virus identification. After some specific pathogens were ruled out by laboratory tests on 5 Jan 2020, the inferred probability of Disease X was over 59%. Conclusions: We showed quantitatively that the emerging outbreak of atypical pneumonia cases is consistent with causation by a novel pathogen. The proposed approach, that uses only routinely-observed non-virological data, can aid ongoing risk assessments even before virological test results become available. Keywords: Epidemic; Causation; Bayes\&#39; theorem; Diagnosis; Prediction; Statistical model&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Sung-mok Jung; Ryo Kinoshita; Robin N. Thompson; Katsuma Hayashi; Natalie M. Linton; Yichi Yang; Andrei R. Akhmetzhanov; Hiroshi Nishiura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.26.20018887&#39;},
 {&#39;cord_uid&#39;: &#39;4pgish5x&#39;,
  &#39;sha&#39;: &#39;68e509d42a7d3349550512d80eca4e2b2123613f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.26.919985&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA novel coronavirus (2019-nCov) was identified in Wuhan, Hubei Province, China in December of 2019. This new coronavirus has resulted in thousands of cases of lethal disease in China, with additional patients being identified in a rapidly growing number internationally. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public database and the state-of-the-art single-cell RNA-Seq technique, we analyzed the ACE2 RNA expression profile in the normal human lungs. The result indicates that the ACE2 virus receptor expression is concentrated in a small population of type II alveolar cells (AT2). Surprisingly, we found that this population of ACE2-expressing AT2 also highly expressed many other genes that positively regulating viral reproduction and transmission. A comparison between eight individual samples demonstrated that the Asian male one has an extremely large number of ACE2-expressing cells in the lung. This study provides a biological background for the epidemic investigation of the 2019-nCov infection disease, and could be informative for future anti-ACE2 therapeutic strategy development.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-26&#39;,
  &#39;authors&#39;: &#39;Yu Zhao; Zixian Zhao; Yujia Wang; Yueqing Zhou; Yu Ma; Wei Zuo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.26.919985&#39;},
 {&#39;cord_uid&#39;: &#39;k2qj6o7j&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.26.920249&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundA novel coronavirus (2019-nCoV) associated with human to human transmission and severe human infection has been recently reported from the city of Wuhan in China. Our objectives were to characterize the genetic relationships of the 2019-nCoV and to search for putative recombination within the subgenus of sarbecovirus.MethodsPutative recombination was investigated by RDP4 and Simplot v3.5.1 and discordant phylogenetic clustering in individual genomic fragments was confirmed by phylogenetic analysis using maximum likelihood and Bayesian methods.ResultsOur analysis suggests that the 2019-nCoV although closely related to BatCoV RaTG13 sequence throughout the genome (sequence similarity 96.3%), shows discordant clustering with the Bat-SARS-like coronavirus sequences. Specifically, in the 5’-part spanning the first 11,498 nucleotides and the last 3’-part spanning 24,341-30,696 positions, 2019-nCoV and RaTG13 formed a single cluster with Bat-SARS-like coronavirus sequences, whereas in the middle region spanning the 3’-end of ORF1a, the ORF1b and almost half of the spike regions, 2019-nCoV and RaTG13 grouped in a separate distant lineage within the sarbecovirus branch.ConclusionsThe levels of genetic similarity between the 2019-nCoV and RaTG13 suggest that the latter does not provide the exact variant that caused the outbreak in humans, but the hypothesis that 2019-nCoV has originated from bats is very likely. We show evidence that the novel coronavirus (2019-nCov) is not-mosaic consisting in almost half of its genome of a distinct lineage within the betacoronavirus. These genomic features and their potential association with virus characteristics and virulence in humans need further attention.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-27&#39;,
  &#39;authors&#39;: &#39;D. Paraskevis; E.G. Kostaki; G. Magiorkinis; G. Panayiotakopoulos; S. Tsiodras&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.26.920249&#39;},
 {&#39;cord_uid&#39;: &#39;u8goc7io&#39;,
  &#39;sha&#39;: &#39;12fac9aedb1a09a3922a3c084ce4723708e463d6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The incubation period of 2019-nCoV infections among travellers from Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.27.20018986&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Currently, a novel coronavirus 2019-nCoV causes an outbreak of viral pneumonia in Wuhan, China. Little is known about its epidemiological characteristics. Using the travel history and symptom onset of 88 confirmed cases that were detected outside Wuhan, we estimate the mean incubation period to be 6.4 (5.6 - 7.7, 95% CI) days, ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile). These values help to inform case definitions for 2019-nCoV and appropriate durations for quarantine.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Jantien A. Backer; Don Klinkenberg; Jacco Wallinga&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.27.20018986&#39;},
 {&#39;cord_uid&#39;: &#39;2ygtr69i&#39;,
  &#39;sha&#39;: &#39;029c1c588047f1d612a219ee15494d2d19ff7439&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Protective Population Behavior Change in Outbreaks of Emerging Infectious Disease&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.27.921536&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTDuring outbreaks of emerging infections, the lack of effective drugs and vaccines increases reliance on non-pharmacologic public health interventions and behavior change to limit human-to-human transmission. Interventions that increase the speed with which infected individuals remove themselves from the susceptible population are paramount, particularly isolation and hospitalization. Ebola virus disease (EVD), Severe Acute Respiratory Syndrome (SARS), and Middle East Respiratory Syndrome (MERS) are zoonotic viruses that have caused significant recent outbreaks with sustained human-to-human transmission. This investigation quantified changing mean removal rates (MRR) and days from symptom onset to hospitalization (DSOH) of infected individuals from the population in seven different outbreaks of EVD, SARS, and MERS, to test for statistically significant differences in these metrics between outbreaks. We found that epidemic week and viral serial interval were correlated with the speed with which populations developed and maintained health behaviors in each outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Evans K. Lodge; Annakate M. Schatz; John M. Drake&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.27.921536&#39;},
 {&#39;cord_uid&#39;: &#39;zgdbvfi4&#39;,
  &#39;sha&#39;: &#39;06a1002f9fbea7179ac3572843f66b14568af6e4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.27.921627&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Abstract2019-nCov has caused more than 80 deaths as of 27 January 2020 in China, and infection cases have been reported in more than 10 countries. However, there is no approved drug to treat the disease. 2019-nCov Mpro is a potential drug target to combat the virus. We built homology models based on SARS Mpro structures, and docked 1903 small molecule drugs to the models. Based on the docking score and the 3D similarity of the binding mode to the known Mpro ligands, 4 drugs were selected for binding free energy calculations. Both MM/GBSA and SIE methods voted for nelfinavir, with the binding free energy of −24.69±0.52 kcal/mol and −9.42±0.04 kcal/mol, respectively. Therefore, we suggested that nelfinavir might be a potential inhibitor against 2019-nCov Mpro.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Zhijian Xu; Cheng Peng; Yulong Shi; Zhengdan Zhu; Kaijie Mu; Xiaoyu Wang; Weiliang Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.27.921627&#39;},
 {&#39;cord_uid&#39;: &#39;ely200x3&#39;,
  &#39;sha&#39;: &#39;aef0734eb2538d3624d650a4ae20c3f66cace5f2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Chronic hM3Dq signaling in microglia ameliorates neuroinflammation in male mice&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.27.921809&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMicroglia express muscarinic G protein-coupled receptors (GPCRs) that sense cholinergic activity and are activated by acetylcholine to potentially regulate microglial functions. Knowledge about how distinct types of muscarinic GPCR signaling regulate microglia function in vitro and in vivo is still poor, partly due to the fact that some of these receptors are also present in astrocytes and neurons. We generated mice expressing the hM3Dq Designer Receptor Exclusively Activated by Designer Drugs (DREADD) selectively in microglia to investigate the role of muscarinic M3Gq-linked signaling. We show that activation of hM3Dq using clozapine N-oxide (CNO) elevated intracellular calcium levels and increased phagocytosis of FluoSpheres in vitro. Acute treatment with CNO in vivo did not affect male mouse behavior, however chronic CNO treatment decreased sickness behavior triggered by lipopolysaccharide (LPS) treatment. Interestingly, whereas acute treatment with CNO increased synthesis of cytokine mRNA, chronic treatment attenuated LPS-induced cytokine mRNA changes in the brain, likely explaining the improvement in sickness behavior by chronic hM3Dq activation. No effect of CNO was observed in DREADD-negative mice. These results suggest that chronic activation of M3 muscarinic receptors (the hM3Dq progenitor) in microglia, and potentially other Gq-coupled GPCRs, preconditions microglia to decrease their response to further immunological challenges. Our results indicate that hM3Dq can be a useful tool to modulate neuroinflammation and study microglial immunological memory in vivo, which may be applicable for manipulations of neuroinflammation in neurodegenerative and psychiatric diseases.HighlightsMicroglial function was manipulated by specific activation of hM3Dq signaling.Chronic hM3Dq activation prevented LPS-induced sickness behavior in mice.Microglial hM3Dq signaling modulated expression of pro-inflammatory cytokines.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;William Binning; Aja E. Hogan-Cann; Diana Yae Sakae; Matthew Maksoud; Valeriy Ostapchenko; Mohammed Al-Onaizi; Sara Matovic; Wei-Yang Lu; Marco A. M. Prado; Wataru Inoue; Vania F. Prado&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.27.921809&#39;},
 {&#39;cord_uid&#39;: &#39;si486lqi&#39;,
  &#39;sha&#39;: &#39;754315299d847600d6c5d414665c728d40bf731d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Breaking down of the healthcare system: Mathematical modelling for controlling the novel coronavirus (2019-nCoV) outbreak in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.27.922443&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA novel coronavirus pneumonia initially identified in Wuhan, China and provisionally named 2019-nCoV has surged in the public. In anticipation of substantial burdens on healthcare system following this human-to-human spread, we aim to scrutinise the currently available information and evaluate the burden of healthcare systems during this outbreak in Wuhan. We applied a modified SIR model to project the actual number of infected cases and the specific burdens on isolation wards and intensive care units, given the scenarios of different diagnosis rates as well as different public health intervention efficacy. Our estimates suggest the actual number of infected cases could be much higher than the reported, with estimated 26,701 cases (as of 28th January 2020) assuming 50% diagnosis rate if no public health interventions were implemented. The estimated burdens on healthcare system could be largely reduced if at least 70% efficacy of public health intervention is achieved.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Wai-Kit Ming; Jian Huang; Casper J. P. Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.27.922443&#39;},
 {&#39;cord_uid&#39;: &#39;x22rc60j&#39;,
  &#39;sha&#39;: &#39;30b5c7b8faf95265f67ae59f4686eaf9b2772893&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimated effectiveness of traveller screening to prevent international spread of 2019 novel coronavirus (2019-nCoV)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.28.20019224&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Traveller screening is being used to limit further global spread of 2019 novel coronavirus (nCoV) following its recent emergence. Here, we analyze the expected impact of different travel screening programs given remaining uncertainty around the values of key nCoV life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss around half of infected travellers. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. These findings emphasize the need for measures to track travellers who become ill after being missed by a travel screening program. We make our model available for interactive use so stakeholders can explore scenarios of interest using the most up-to-date information. We hope these findings contribute to evidence-based policy to combat the spread of nCoV, and to prospective planning to mitigate future emerging pathogens.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-30&#39;,
  &#39;authors&#39;: &#39;Katelyn Gostic; Ana C. R. Gomez; Riley O. Mummah; Adam J. Kucharski; James O. Lloyd-Smith&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.28.20019224&#39;},
 {&#39;cord_uid&#39;: &#39;mbdj3r0m&#39;,
  &#39;sha&#39;: &#39;957e71389292e0881dff2bbe0e87fe6bf8b2d381&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Risk for Transportation of 2019 Novel Coronavirus (COVID-19) from Wuhan to Cities in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.28.20019299&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;On January 23, 2020, China quarantined Wuhan to contain an emerging coronavirus (COVID-19). We estimated the probability of transportation of COVID-19 from Wuhan to 369 cities in China before the quarantine. The expected risk is &gt;50% in 130 (95% CI 89-190) cities and &gt;99% in the 4 largest metropolitan areas of China.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-30&#39;,
  &#39;authors&#39;: &#39;Zhanwei Du; Ling Wang; Simon Cauchemez; Xiaoke Xu; Xianwen Wang; Benjamin J Cowling; Lauren Ancel Meyers&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.28.20019299&#39;},
 {&#39;cord_uid&#39;: &#39;635lbedk&#39;,
  &#39;sha&#39;: &#39;d49f8ba7212183bc1b7f2cb6dd709dd197303b3d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Spreading predictability in complex networks&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.28.922757&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSpreading dynamics analysis is an important and interesting topic since it has many applications such as rumor or disease controlling, viral marketing and information recommending. Many state-of-the-art researches focus on predicting infection scale or threshold. Few researchers pay attention to the predicting of infection nodes from a snapshot. With developing of precision marketing, recommending and, controlling, how to predict infection nodes precisely from snapshot becomes a key issue in spreading dynamics analysis. In this paper, a probability based prediction model is presented so as to estimate the infection nodes from a snapshot of spreading. Experimental results on synthetic and real networks demonstrate that the model proposed could predict the infection nodes precisely in the sense of probability.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Na Zhao; Jian Wang; Yong Yu; Jun-Yan Zhao; Duan-Bing Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.28.922757&#39;},
 {&#39;cord_uid&#39;: &#39;iu2ziccm&#39;,
  &#39;sha&#39;: &#39;c6ec1380338cb2467dc6fc14add09ca5bea18411&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.28.922922&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Abstract2019 Novel Coronavirus (2019-nCoV) is a virus identified as the cause of the outbreak of pneumonia first detected in Wuhan, China. Investigations on the transmissibility, severity, and other features associated with this virus are ongoing. Currently, there is no vaccine or therapeutic antibody to prevent the infection, and more time is required to develop an effective immune strategy against the pathogen. In contrast, specific inhibitors targeting the key protease involved in replication and proliferation of the virus are the most effective means to alleviate the epidemic. The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development. In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-29&#39;,
  &#39;authors&#39;: &#39;Yan Li; Jinyong Zhang; Ning Wang; Haibo Li; Yun Shi; Gang Guo; Kaiyun Liu; Hao Zeng; Quanming Zou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.28.922922&#39;},
 {&#39;cord_uid&#39;: &#39;lzda8kyn&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.28.923011&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe newly identified 2019 novel coronavirus (2019-nCoV) has caused more than 800 laboratory-confirmed human infections, including 25 deaths, posing a serious threat to human health. Currently, however, there is no specific antiviral treatment or vaccine. Considering the relatively high identity of receptor binding domain (RBD) in 2019-nCoV and SARS-CoV, it is urgent to assess the cross-reactivity of anti-SARS-CoV antibodies with 2019-nCoV spike protein, which could have important implications for rapid development of vaccines and therapeutic antibodies against 2019-nCoV. Here, we report for the first time that a SARS-CoV-specific human monoclonal antibody, CR3022, could bind potently with 2019-nCoV RBD (KD of 6.3 nM). The epitope of CR3022 does not overlap with the ACE2 binding site within 2019-nCoV RBD. Therefore, CR3022 has the potential to be developed as candidate therapeutics, alone or in combination with other neutralizing antibodies, for the prevention and treatment of 2019-nCoV infections. Interestingly, some of the most potent SARS-CoV-specific neutralizing antibodies (e.g., m396, CR3014) that target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein, indicating that the difference in the RBD of SARS-CoV and 2019-nCoV has a critical impact for the cross-reactivity of neutralizing antibodies, and that it is still necessary to develop novel monoclonal antibodies that could bind specifically to 2019-nCoV RBD.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Xiaolong Tian; Cheng Li; Ailing Huang; Shuai Xia; Sicong Lu; Zhengli Shi; Lu Lu; Shibo Jiang; Zhenlin Yang; Yanling Wu; Tianlei Ying&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.28.923011&#39;},
 {&#39;cord_uid&#39;: &#39;x443k65a&#39;,
  &#39;sha&#39;: &#39;f24d3b4b4af138be06b7452b7acefc8948bc1056&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Beware of asymptomatic transmission: Study on 2019-nCoV prevention and control measures based on extended SEIR model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.28.923169&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryBackgroundThe 2019 new coronavirus, “2019-nCoV”, was discovered from Wuhan Viral Pneumonia cases in December 2019, and was named by the World Health Organization on January 12, 2020. In the early stage, people knows little about the 2019-nCoV virus was not clear, and the spread period was encountering China’s annual spring migration, which made the epidemic spread rapidly from Wuhan to almost all provinces in China.MethodsThis study builds a SEIRD model that considers the movement of people across regions, revealing the effects of three measures on controlling the spread of the epidemic.Based on MATLAB R2017a, computational experiments were performed to simulate the epidemic prevention and control measures.FindingsThe research results show that current prevention and control measures in China are very necessary. This study further validates the concerns of international and domestic experts regarding asymptomatic transmission (E-status).InterpretationThe results of this study are applicable to explore the impact of the implementation of relevant measures on the prevention and control of epidemic spread, and to identify key individuals that may exist during the spread of the epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-28&#39;,
  &#39;authors&#39;: &#39;Peng Shao; Yingji Shan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.28.923169&#39;},
 {&#39;cord_uid&#39;: &#39;rr5qhsam&#39;,
  &#39;sha&#39;: &#39;2afff5fe89a3e3ec270a41c2b929dfeec68d7d50&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Real time estimation of the risk of death from novel coronavirus (2019-nCoV) infection: Inference using exported cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.29.20019547&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The exported cases of 2019 novel coronavirus (2019-nCoV) infection who were confirmed in other countries provide a chance to estimate the cumulative incidence and confirmed case fatality risk (cCFR) in China. Knowledge of the cCFR is critical to characterize the severity and understand pandemic potential of 2019-nCoV in the early stage of epidemic. Using the exponential growth rate of the incidence, the present study statistically estimated the cCFR and the basic reproduction number, i.e., the average number of secondary cases generated by a single primary case in a naive population. As of 24 January 2020, with 23 exported cases, and estimating the growth rate from 8 December 2019 (scenario 1) and using the data since growth of exported cases (scenario 2), the cumulative incidence in China was estimated at 5433 cases (95% confidence interval (CI): 3883, 7160) and 17780 cases (95% CI: 9646, 28724), respectively. The latest estimates of the cCFR were 4.6% (95% CI: 3.1-6.6) for scenario 1 and 7.7% (95% CI: 4.9-11.3%) for scenario 2, respectively. The basic reproduction number was estimated to be 2.2 (95% CI: 2.1, 2.3) and 3.7 (95% CI: 3.1, 4.3) for scenarios 1 and 2, respectively. Based on the results, we note that current 2019-nCoV epidemic has a substation potential to cause a pandemic. The proposed approach can provide insights into early risk assessment using only publicly available data.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Sung-mok Jung; Andrei R. Akhmetzhanov; Katsuma Hayashi; Natalie M. Linton; Yichi Yang; Baoyin Yuan; Tetsuro Kobayashi; Ryo Kinoshita; Hiroshi Nishiura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.29.20019547&#39;},
 {&#39;cord_uid&#39;: &#39;yd7tu7u4&#39;,
  &#39;sha&#39;: &#39;c04c7fb330a409a00f67040dde0f83b3da88eacb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.29.924100&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractStarting from December 2019, a novel coronavirus, named 2019-nCoV, was found to cause Severe Acute Respiratory (SARI) symptoms and rapid pandemic in China. With the hope to identify candidate drugs for 2019-nCoV, we adopted a computational approach to screen for available commercial medicines which may function as inhibitors for the Mpro of 2019-nCoV. Up to 10 commercial medicines that may form hydrogen bounds to key residues within the binding pocket of 2019-nCoV Mpro were identified, which may have higher mutation tolerance than lopinavir/ritonavir and may also function as inhibitors for other coronaviruses with similar Mpro binding sites and pocket structures.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-29&#39;,
  &#39;authors&#39;: &#39;Xin Liu; Xiu-Jie Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.29.924100&#39;},
 {&#39;cord_uid&#39;: &#39;ac00tai9&#39;,
  &#39;sha&#39;: &#39;b9dbb79c9e06164c3b7fefce67c11c5d6caf1fcd&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Interpretable detection of novel human viruses from genome sequencing data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.29.925354&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-30&#39;,
  &#39;authors&#39;: &#39;Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.29.925354&#39;},
 {&#39;cord_uid&#39;: &#39;xxcx7hg5&#39;,
  &#39;sha&#39;: &#39;acb0e11e5763dbf7588ec435751a2ab705a783f1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.29.925867&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-30&#39;,
  &#39;authors&#39;: &#39;Arunachalam Ramaiah; Vaithilingaraja Arumugaswami&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.29.925867&#39;},
 {&#39;cord_uid&#39;: &#39;90tccg03&#39;,
  &#39;sha&#39;: &#39;a6af28a7f0a0d8d1e4d5b06f5ae38412298bbcb4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Direct Measurement of Rates of Asymptomatic Infection and Clinical Care-Seeking for Seasonal Coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.20019612&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The pandemic potential of the novel coronavirus (nCoV) that emerged in Wuhan, China, during December 2019 is strongly tied to the number and contagiousness of undocumented human infections. Here we present findings from a proactive longitudinal sampling study of acute viral respiratory infections that documents rates of asymptomatic infection and clinical care seeking for seasonal coronavirus. We find that the majority of infections are asymptomatic by most symptom definitions and that only 4% of individuals experiencing a seasonal coronavirus infection episode sought medical care for their symptoms. These numbers indicate that a very high percentage of seasonal coronavirus infections are undocumented and provide a reference for understanding the spread of the emergent nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-03&#39;,
  &#39;authors&#39;: &#39;Jeffrey Shaman; Marta Galanti&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.20019612&#39;},
 {&#39;cord_uid&#39;: &#39;81j4347d&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modelling the epidemic trend of the 2019-nCOV outbreak in Hubei Province, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.20019828&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;As of 8am 30th January (Beijing Time) 2020, Approximate 8000 cases across the world have been confirmed. It&#39;s necessary to simulate epidemic trend of the 2019-nCOV outbreak in Hubei Province, the hardest-hit area. By SEIR simulation, the predicted epidemic peak in Hubei will be within 28th January 2020 to 7th February 2020, up to 7000-9000 infectious cases in total. The estimate above was based on some assumptions and limitations exited.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Lizhe Ai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.20019828&#39;},
 {&#39;cord_uid&#39;: &#39;mm4ngrla&#39;,
  &#39;sha&#39;: &#39;f17c6ac4986aecba2dc22f67b32f2d16b369117b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission and epidemiological characteristics of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected Pneumonia (COVID-19): preliminary evidence obtained in comparison with 2003-SARS&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.20019836&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives: Latest epidemic data of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected Pneumonia (COVID-19) was collected and a detailed statistical analysis was carried out to make comparison with 2003-SARS in order to provide scientific reference for the prevention and control of COVID-19. Methods: The information of COVID-19 and 2003-SARS from websites of NHCPRC and the World Health Organization was collected, and then the transmission dynamics of the two kinds of infectious diseases were analyzed. The information of 853 confirmed COVID-19 patients obtained from the website of health committees of 18 provinces. A descriptive epidemiological analysis method was employed to carefully analyze the epidemic characteristics. Subsequently, the COVID-19 epidemic data in Wuhan and other inland regions of China was analyzed separately and compared. A multivariate function model was constructed based on the confirmed COVID-19 case data. Results: The growth rate of new cases and deaths of COVID-19 were significantly faster than those of 2003-SARS. The number of confirmed cases in Wuhan and other inland areas both showed increasing trends. 853 confirmed COVID-19 cases aged 1 months to 94 years and the average age was (45.05 ± 17.22) years. The gender ratio (M: F) was 1.12: 1. Conclusions: The fatality rate of COVID-19 is lower than that of 2003-SARS and the cure rate is higher. The age of COVID-19 patients is mainly concentrated in the 30-50 years old (60.61%). The harm of the first-generation COVID-19 patients is higher than that of secondary cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Rongqiang Zhang; Hui Liu; Fengying Li; Bei Zhang; Qiling Liu; Xiangwen Li; Limei Luo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.20019836&#39;},
 {&#39;cord_uid&#39;: &#39;e85xdrcw&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The impact of transmission control measures during the first 50 days of the COVID-19 epidemic in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.20019844&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Respiratory illness caused by a novel coronavirus (COVID-19) appeared in China during December 2019. Attempting to contain infection, China banned travel to and from Wuhan city on 23 January and implemented a national emergency response. Here we evaluate the spread and control of the epidemic based on a unique synthesis of data including case reports, human movement and public health interventions. The Wuhan shutdown slowed the dispersal of infection to other cities by an estimated 2.91 days (95%CI: 2.54-3.29), delaying epidemic growth elsewhere in China. Other cities that implemented control measures pre-emptively reported 33.3% (11.1-44.4%) fewer cases in the first week of their outbreaks (13.0; 7.1-18.8) compared with cities that started control later (20.6; 14.5-26.8). Among interventions investigated here, the most effective were suspending intra-city public transport, closing entertainment venues and banning public gatherings. The national emergency response delayed the growth and limited the size of the COVID-19 epidemic and, by 19 February (day 50), had averted hundreds of thousands of cases across China.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Huaiyu Tian; Yonghong Liu; Yidan Li; Chieh-Hsi Wu; Bin Chen; Moritz U. G. Kraemer; Bingying Li; Jun Cai; Bo Xu; Qiqi Yang; Ben Wang; Peng Yang; Yujun Cui; Yimeng Song; Pai Zheng; Quanyi Wang; Ottar N Bjornstad; Ruifu Yang; Bryan Grenfell; Oliver Pybus; Christopher Dye&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.20019844&#39;},
 {&#39;cord_uid&#39;: &#39;p5aj5k2g&#39;,
  &#39;sha&#39;: &#39;0ff4ad5359ee2df3568605385a8616d8da66bf2c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.20019877&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number Ro -- the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of Ro vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of Ro across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate $r$, the mean generation interval $\\bar G$, and the generation-interval dispersion $\\kappa$. We then apply our framework to early estimates of Ro for the SARS-CoV-2 outbreak. We show that many early Ro estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of Ro, including the shape of the generation-interval distribution, in efforts to estimate Ro at the outset of an epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Sang Woo Park; Benjamin M. Bolker; David Champredon; David J.D. Earn; Michael Li; Joshua S. Weitz; Bryan T. Grenfell; Jonathan Dushoff&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.20019877&#39;},
 {&#39;cord_uid&#39;: &#39;ua6aeg5a&#39;,
  &#39;sha&#39;: &#39;18dc6a67045d2687d2b5c11c85797f42824ed243&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evolution and variation of 2019-novel coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.926477&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1·05 × 10−2 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 × 10−4 to 2.72 × 10−2) for N, 5.34 × 10−3 (5.10 × 10−4, 1.28 × 10−2) for S, 1.69 × 10−3 (3.94 × 10−4, 3.60 × 10−3) for P, 1.65 × 10−3 (4.47 × 10−4, 3.24 × 10−3) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-30&#39;,
  &#39;authors&#39;: &#39;Chenglong Xiong; Lufang Jiang; Yue Chen; Qingwu Jiang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.926477&#39;},
 {&#39;cord_uid&#39;: &#39;nfpw89fu&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.926881&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDeubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-31&#39;,
  &#39;authors&#39;: &#39;Wioletta Rut; Mikołaj Żmudziński; Scott J. Snipas; Miklos Bekes; Tony T. Huang; Marcin Drag&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.926881&#39;},
 {&#39;cord_uid&#39;: &#39;dpcpg5c1&#39;,
  &#39;sha&#39;: &#39;ffc9bc4ac6548cb1c03c2c6f088ab1c1c2067c6f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Nucleotide Analogues as Inhibitors of Viral Polymerases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.927574&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryCoronaviruses such as the newly discovered virus from Wuhan, China, 2019-nCoV, and the viruses that cause SARS and MERS, have resulted in regional and global public health emergencies. Based on our molecular insight that the hepatitis C virus and the coronavirus use a similar viral genome replication mechanism, we reasoned that the FDA-approved drug EPCLUSA (Sofosbuvir/Velpatasvir) for the treatment of hepatitis C will also inhibit the above coronaviruses, including 2019-nCoV. To develop broad spectrum anti-viral agents, we further describe a novel strategy to design and synthesize viral polymerase inhibitors, by combining the ProTide Prodrug approach used in the development of Sofosbuvir with the use of 3’-blocking groups that we have previously built into nucleotide analogues that function as polymerase terminators.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-31&#39;,
  &#39;authors&#39;: &#39;Jingyue Ju; Shiv Kumar; Xiaoxu Li; Steffen Jockusch; James J. Russo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.927574&#39;},
 {&#39;cord_uid&#39;: &#39;vpcx2t3w&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.927806&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince December 2019, a newly identified coronavirus (2019 novel coronavirus, 2019-nCov) is causing outbreak of pneumonia in one of largest cities, Wuhan, in Hubei province of China and has draw significant public health attention. The same as severe acute respiratory syndrome coronavirus (SARS-CoV), 2019-nCov enters into host cells via cell receptor angiotensin converting enzyme II (ACE2). In order to dissect the ACE2-expressing cell composition and proportion and explore a potential route of the 2019-nCov infection in digestive system infection, 4 datasets with single-cell transcriptomes of lung, esophagus, gastric, ileum and colon were analyzed. The data showed that ACE2 was not only highly expressed in the lung AT2 cells, esophagus upper and stratified epithelial cells but also in absorptive enterocytes from ileum and colon. These results indicated along with respiratory systems, digestive system is a potential routes for 2019-nCov infection. In conclusion, this study has provided the bioinformatics evidence of the potential route for infection of 2019-nCov in digestive system along with respiratory tract and may have significant impact for our healthy policy setting regards to prevention of 2019-nCoV infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-31&#39;,
  &#39;authors&#39;: &#39;Hao Zhang; Zijian Kang; Haiyi Gong; Da Xu; Jing Wang; Zifu Li; Xingang Cui; Jianru Xiao; Tong Meng; Wang Zhou; Jianmin Liu; Huji Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.927806&#39;},
 {&#39;cord_uid&#39;: &#39;1qniriu0&#39;,
  &#39;sha&#39;: &#39;6ae20454d1a9f228864de24660c2460becbc8151&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Machine intelligence design of 2019-nCoV drugs&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.30.927889&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWuhan coronavirus, called 2019-nCoV, is a newly emerged virus that infected more than 9692 people and leads to more than 213 fatalities by January 30, 2020. Currently, there is no effective treatment for this epidemic. However, the viral protease of a coronavirus is well-known to be essential for its replication and thus is an effective drug target. Fortunately, the sequence identity of the 2019-nCoV protease and that of severe-acute respiratory syndrome virus (SARS-CoV) is as high as 96.1%. We show that the protease inhibitor binding sites of 2019-nCoV and SARS-CoV are almost identical, which means all potential anti-SARS-CoV chemotherapies are also potential 2019-nCoV drugs. Here, we report a family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC). The potential effectiveness of treating 2019-nCoV by using some existing HIV drugs is also analyzed.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-04&#39;,
  &#39;authors&#39;: &#39;Kaifu Gao; Duc Duy Nguyen; Rui Wang; Guo-Wei Wei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.30.927889&#39;},
 {&#39;cord_uid&#39;: &#39;2wshgzjk&#39;,
  &#39;sha&#39;: &#39;a01672035e0b9d21f9a934b9c9071610513325f2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effectiveness of airport screening at detecting travellers infected with 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.20019265&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Billy Quilty; Sam Clifford; Stefan Flasche; Rosalind M Eggo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.20019265&#39;},
 {&#39;cord_uid&#39;: &#39;gmi1ewc2&#39;,
  &#39;sha&#39;: &#39;a44334e676e43c1889d282b0f0a8365f0f1e0c52&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Early dynamics of transmission and control of COVID-19: a mathematical modelling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.20019901&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: An outbreak of the novel coronavirus SARS-CoV-2 has led to 46,997 confirmed cases as of 13th February 2020. Understanding the early transmission dynamics of the infection and evaluating the effectiveness of control measures is crucial for assessing the potential for sustained transmission to occur in new areas. Methods: We combined a stochastic transmission model with data on cases of novel coronavirus disease (COVID-19) in Wuhan and international cases that originated in Wuhan to estimate how transmission had varied over time during January and February 2020. Based on these estimates, we then calculated the probability that newly introduced cases might generate outbreaks in other areas. Findings: We estimated that the median daily reproduction number, Rt , declined from 2.35 (95% CI: 1.15-4.77) one week before travel restrictions were introduced on 23rd January to 1.05 (95% CI: 0.413-2.39) one week after. Based on our estimates of Rt,we calculated that in locations with similar transmission potential as Wuhan in early January, once there are at least four independently introduced cases, there is a more than 50% chance the infection will establish within that population. Interpretation: Our results show that COVID-19 transmission likely declined in Wuhan during late January 2020, coinciding with the introduction of control measures. As more cases arrive in international locations with similar transmission potential to Wuhan pre-control, it is likely many chains of transmission will fail to establish initially, but may still cause new outbreaks eventually.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Adam J Kucharski; Timothy W Russell; Charlie Diamond; Yang Liu; John Edmunds; Sebastian Funk; Rosalind M Eggo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.20019901&#39;},
 {&#39;cord_uid&#39;: &#39;gxstlzuk&#39;,
  &#39;sha&#39;: &#39;0c27c0ddaa4761f6155838df81a88d24619720f8&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.928796&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTSummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.AvailabilityAvailable online: https://www.genomedetective.com/app/typingtool/cov*Contactkoen@emweb.be and deoliveira@ukzn.ac.zaSupplementary informationSupplementary data is available online.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.928796&#39;},
 {&#39;cord_uid&#39;: &#39;rv26qfmf&#39;,
  &#39;sha&#39;: &#39;eae11a3f01b79e76a968bfb84886432f04584596&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.929042&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe emergence of a novel, highly pathogenic coronavirus, 2019-nCoV, in China, and its rapid national and international spread pose a global health emergency. Coronaviruses use their spike proteins to select and enter target cells and insights into nCoV-2019 spike (S)-driven entry might facilitate assessment of pandemic potential and reveal therapeutic targets. Here, we demonstrate that 2019-nCoV-S uses the SARS-coronavirus receptor, ACE2, for entry and the cellular protease TMPRSS2 for 2019-nCoV-S priming. A TMPRSS2 inhibitor blocked entry and might constitute a treatment option. Finally, we show that the serum form a convalescent SARS patient neutralized 2019-nCoV-S-driven entry. Our results reveal important commonalities between 2019-nCoV and SARS-coronavirus infection, which might translate into similar transmissibility and disease pathogenesis. Moreover, they identify a target for antiviral intervention.One sentence summaryThe novel 2019 coronavirus and the SARS-coronavirus share central biological properties which can guide risk assessment and intervention.&#39;,
  &#39;publish_time&#39;: &#39;2020-01-31&#39;,
  &#39;authors&#39;: &#39;Markus Hoffmann; Hannah Kleine-Weber; Nadine Krüger; Marcel Müller; Christian Drosten; Stefan Pöhlmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.929042&#39;},
 {&#39;cord_uid&#39;: &#39;a1mcnta5&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Highly Distinguished Amino Acid Sequences of 2019-nCoV (Wuhan Coronavirus)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.929497&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractUsing a method for pathogen screening in DNA synthesis orders, we have identified a number of amino acid sequences that distinguish 2019-nCoV (Wuhan Coronavirus) from all other known viruses in Coronaviridae. We find three main regions of unique sequence: two in the 1ab polyprotein QHO60603.1, one in surface glycoprotein QHO60594.1.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Jacob Beal; Thomas Mitchell; Daniel Wyschogrod; Jeff Manthey; Adam Clore&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.929497&#39;},
 {&#39;cord_uid&#39;: &#39;cszqykpu&#39;,
  &#39;sha&#39;: &#39;ff54e3e961a72eb1d2500166809b3651b2f98cf6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.929547&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &lt; 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Bo Ram Beck; Bonggun Shin; Yoonjung Choi; Sungsoo Park; Keunsoo Kang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.929547&#39;},
 {&#39;cord_uid&#39;: &#39;vbgf50os&#39;,
  &#39;sha&#39;: &#39;9e94f9379fd74fcacc4f3a57e03cbe9035efee8e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Molecular Modeling Evaluation of the Binding Effect of Ritonavir, Lopinavir and Darunavir to Severe Acute Respiratory Syndrome Coronavirus 2 Proteases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.01.31.929695&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThree anti-HIV drugs, ritonavir, lopinavir and darunavir, might have therapeutic effect on coronavirus disease 2019 (COVID-19). In this study, the structure models of two severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteases, coronavirus endopeptidase C30 (CEP_C30) and papain like viral protease (PLVP), were built by homology modeling. Ritonavir, lopinavir and darunavir were then docked to the models, respectively, followed by energy minimization of the protease-drug complexes. In the simulations, ritonavir can bind to CEP_C30 most suitably, and induce significant conformation changes of CEP_C30; lopinavir can also bind to CEP_C30 suitably, and induce significant conformation changes of CEP_C30; darunavir can bind to PLVP suitably with slight conformation changes of PLVP. It is suggested that the therapeutic effect of ritonavir and lopinavir on COVID-19 may be mainly due to their inhibitory effect on CEP_C30, while ritonavir may have stronger efficacy; the inhibitory effect of darunavir on SARS-CoV-2 and its potential therapeutic effect may be mainly due to its inhibitory effect on PLVP.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-03&#39;,
  &#39;authors&#39;: &#39;Shen Lin; Runnan Shen; Jingdong He; Xinhao Li; Xushun Guo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.01.31.929695&#39;},
 {&#39;cord_uid&#39;: &#39;aeuy92bx&#39;,
  &#39;sha&#39;: &#39;a8e53c77bc72f467723a7d5b8020875c0e96f8aa&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Potent neutralization of 2019 novel coronavirus by recombinant ACE2-Ig&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.01.929976&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Abstract2019-nCoV, which is a novel coronavirus emerged in Wuhan, China, at the end of 2019, has caused at least infected 11,844 as of Feb 1, 2020. However, there is no specific antiviral treatment or vaccine currently. Very recently report had suggested that novel CoV would use the same cell entry receptor, ACE2, as the SARS-CoV. In this report, we generated a novel recombinant protein by connecting the extracellular domain of human ACE2 to the Fc region of the human immunoglobulin IgG1. An ACE2 mutant with low catalytic activity was also used in the study. The fusion proteins were then characterized. Both fusion proteins has high affinity binding to the receptor-binding domain (RBD) of SARS-CoV and 2019-nCoV and exerted desired pharmacological properties. Moreover, fusion proteins potently neutralized SARS-CoV and 2019-nCoV in vitro. As these fusion proteins exhibit cross-reactivity against coronaviruses, they could have potential applications for diagnosis, prophylaxis, and treatment of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-02&#39;,
  &#39;authors&#39;: &#39;Changhai Lei; Wenyan Fu; Kewen Qian; Tian Li; Sheng Zhang; Min Ding; Shi Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.01.929976&#39;},
 {&#39;cord_uid&#39;: &#39;eecyduzq&#39;,
  &#39;sha&#39;: &#39;836190d366ba8e34c3a5df8a24d17a36fd503505&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.01.930537&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreaks of 2002/2003 SARS, 2012/2015 MERS and 2019/2020 Wuhan respiratory syndrome clearly indicate that genome evolution of an animal coronavirus (CoV) may enable it to acquire human transmission ability, and thereby to cause serious threats to global public health. It is widely accepted that CoV human transmission is driven by the interactions of its spike protein (S-protein) with human receptor on host cell surface; so, quantitative evaluation of these interactions may be used to assess the human transmission capability of CoVs. However, quantitative methods directly using viral genome data are still lacking. Here, we perform large-scale protein-protein docking to quantify the interactions of 2019-nCoV S-protein receptor-binding domain (S-RBD) with human receptor ACE2, based on experimental SARS-CoV S-RBD-ACE2 complex structure. By sampling a large number of thermodynamically probable binding conformations with Monte Carlo algorithm, this approach successfully identified the experimental complex structure as the lowest-energy receptor-binding conformations, and hence established an experiment-based strength reference for evaluating the receptor-binding affinity of 2019-nCoV via comparison with SARS-CoV. Our results show that this binding affinity is about 73% of that of SARS-CoV, supporting that 2019-nCoV may cause human transmission similar to that of SARS-CoV. Thus, this study presents a method for rapidly assessing the human transmission capability of a newly emerged CoV and its mutant strains, and demonstrates that post-genome analysis of protein-protein interactions may provide early scientific guidance for viral prevention and control.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-03&#39;,
  &#39;authors&#39;: &#39;Qiang Huang; Andreas Herrmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.01.930537&#39;},
 {&#39;cord_uid&#39;: &#39;uqls3p01&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genomic variance of the 2019-nCoV coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.02.931162&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThere is rising global concern for the recently emerged novel Coronavirus (2019-nCov). Full genomic sequences have been released by the worldwide scientific community in the last few weeks in order to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and SARS. We confirm high sequence similarity (&gt;99%) between all sequenced 2019-nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019-nCoV. Despite the low heterogeneity of the 2019-nCoV genomes, we could identify at least two hyper-variable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8-encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti-coronavirus approaches.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-04&#39;,
  &#39;authors&#39;: &#39;Carmine Ceraolo; Federico M. Giorgi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.02.931162&#39;},
 {&#39;cord_uid&#39;: &#39;rj9k9awy&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evidence and characteristics of human-to-human transmission of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20019141&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Min Kang; Jie Wu; Wenjun Ma; Jianfeng He; Jing Lu; Tao Liu; Baisheng Li; Shujiang Mei; Feng Ruan; Lifeng Lin; Lirong Zou; Changwen Ke; Haojie Zhong; Yingtao Zhang; Xuguang Chen; Zhe Liu; Qi Zhu; Jianpeng Xiao; Jianxiang Yu; Jianxiong Hu; Weilin Zeng; Xing Li; Yuhuang Liao; Xiujuan Tang; Songjian Xiao; Ying Wang; Yingchao Song; Xue Zhuang; Lijun Liang; Siqing Zeng; Guanhao He; Peng Lin; Huihong Deng; Tie Song&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20019141&#39;},
 {&#39;cord_uid&#39;: &#39;vspnuxz9&#39;,
  &#39;sha&#39;: &#39;a1bff76ce360e8990b0a4ee2a5228a6e6e63d9c1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Serial interval of novel coronavirus (2019-nCoV) infections&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20019497&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Hiroshi Nishiura; Natalie M Linton; Andrei R. Akhmetzhanov&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20019497&#39;},
 {&#39;cord_uid&#39;: &#39;mjhez5im&#39;,
  &#39;sha&#39;: &#39;5d3612cd95331a2f0c43ef9d03ce583b5ff8c995&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20020206&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The 2019-nCoV is reported to share the same entry (ACE2) as SARS-CoV according to the updated findings. Analyzing the distribution and expression level of the route of coronavirus may help reveal underlying mechanisms of viral susceptibility and post-infection modulation. In this study, we found that the expression of ACE2 in healthy populations and patients with underlying diseases was not significantly different, suggesting relatively similar susceptibility, which was consistent with current clinical observations. Moreover, based on the expression of ACE2 in smoking individuals, we inferred that long-term smoking might be a risk factor for 2019-nCoV. Analyzing the ACE2 in SARS-CoV infected cells suggested that ACE2 was more than just a receptor but also participated in post-infection regulation, including immune response, cytokine secretion, and viral genome replication. We also constructed Protein-protein interaction (PPI) networks and identified hub genes in viral activity and cytokine secretion. Our findings could explain the clinical symptoms so far and help clinicians and researchers understand the pathogenesis and design therapeutic strategies for 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Xiang He; Lei Zhang; Qin Ran; Anying Xiong; Junyi Wang; Dehong Wu; Feng Chen; Guoping Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20020206&#39;},
 {&#39;cord_uid&#39;: &#39;f3h74j1n&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of the asymptomatic ratio of novel coronavirus infections (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20020248&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A total of 565 Japanese citizens were evacuated from Wuhan, China to Japan. All passengers were screened for symptoms and also undertook reverse transcription polymerase chain reaction testing, identifying 5 asymptomatic and 7 symptomatic passengers testing positive for 2019-nCoV. We show that the screening result is suggestive of the asymptomatic ratio at 41.6%.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Hiroshi Nishiura; Tetsuro Kobayashi; Takeshi Miyama; Ayako Suzuki; Sungmok Jung; Katsuma Hayashi; Ryo Kinoshita; Yichi Yang; Baoyin Yuan; Andrei R. Akhmetzhanov; Natalie M Linton&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20020248&#39;},
 {&#39;cord_uid&#39;: &#39;b4mdiont&#39;,
  &#39;sha&#39;: &#39;21100aba41a4bfb48d7dc37f1bf5dbb38bf3867a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Network-based Drug Repurposing for Human Coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20020263&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Human Coronaviruses (HCoVs), including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle east respiratory syndrome coronavirus (MERS-CoV), and 2019 novel coronavirus (2019-nCoV), lead global epidemics with high morbidity and mortality. However, there are currently no effective drugs targeting 2019-nCoV. Drug repurposing, represented as an effective drug discovery strategy from existing drugs, could shorten the time and reduce the cost compared to de novo drug discovery. In this study, we present an integrative, antiviral drug repurposing methodology implementing a systems pharmacology-based network medicine platform, quantifying the interplay between the HCoV-host interactome and drug targets in the human protein-protein interaction network. Phylogenetic analyses of 15 HCoV whole genomes reveal that 2019-nCoV has the highest nucleotide sequence identity with SARS-CoV (79.7%) among the six other known pathogenic HCoVs. Specifically, the envelope and nucleocapsid proteins of 2019-nCoV are two evolutionarily conserved regions, having the sequence identities of 96% and 89.6%, respectively, compared to SARS-CoV. Using network proximity analyses of drug targets and known HCoV-host interactions in the human protein-protein interactome, we computationally identified 135 putative repurposable drugs for the potential prevention and treatment of HCoVs. In addition, we prioritized 16 potential anti-HCoV repurposable drugs (including melatonin, mercaptopurine, and sirolimus) that were further validated by enrichment analyses of drug-gene signatures and HCoV-induced transcriptomics data in human cell lines. Finally, we showcased three potential drug combinations (including sirolimus plus dactinomycin, mercaptopurine plus melatonin, and toremifene plus emodin) captured by the Complementary Exposure pattern: the targets of the drugs both hit the HCoV-host subnetwork, but target separate neighborhoods in the human protein-protein interactome network. In summary, this study offers powerful network-based methodologies for rapid identification of candidate repurposable drugs and potential drug combinations toward future clinical trials for HCoVs.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Yadi Zhou; Yuan Hou; Jiayu Shen; Yin Huang; William Martin; Feixiong Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20020263&#39;},
 {&#39;cord_uid&#39;: &#39;4sw1hrnf&#39;,
  &#39;sha&#39;: &#39;5c525cc04075b7c82e52b1e80d96bb394b1a0907&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Getting to zero quickly in the 2019-nCov epidemic with vaccines or rapid testing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20020271&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Any plan for stopping the ongoing 2019-nCov epidemic must be based on a quantitative understanding of the proportion of the at-risk population that needs to be protected by effective control measures in order for transmission to decline sufficiently and quickly enough for the epidemic to end. Using an SEIR-type transmission model, we contrasted two alternate strategies by modeling the proportion of the population that needs to be protected from infection by one-time vaccination (assuming 100% effectiveness) or by testing with isolation and treatment of individuals within six, 24, or 48 hours of symptom onset. If R is currently 2.2, vaccination at the herd immunity coverage of 55% would drive R just below 1, but transmission could persist for years. Over 80% of coverage is required to end the epidemic in 6 months with population-wide vaccination. The epidemic could be ended in just under a year if testing with isolation and treatment reached 80% of symptomatically infected patients within 24 hours of symptom onset (assuming 10% asymptomatic transmission). The epidemic could be ended in six months if testing with isolation and treatment reached 90% of symptomatic patients. If 90% of symptomatic patients could be tested within six hours of symptoms appearing, the epidemic could be ended in under four months.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Gerardo Chowell; Ranu Dhillon; Devabhaktuni Srikrishna&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20020271&#39;},
 {&#39;cord_uid&#39;: &#39;bnv38la3&#39;,
  &#39;sha&#39;: &#39;b51d1c7287ac75790a52752489577d33d522898b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Diarrhea may be underestimated: a missing link in 2019 novel coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.20020289&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of pneumonia caused by the 2019 Novel Coronavirus (2019-nCoV) was reported in Wuhan City, China. However, the clinical symptoms varied in different reports. Based on the results of inter-group difference test, we found that the incidence of diarrhea differed in three recent reports. As 2019-nCoV utilizes the same cell entry receptor ACE2 as severe acute respiratory syndrome coronavirus (SARS-CoV) and ACE2 tightly controls intestinal inflammation, to trace the route of infection mediated by 2019-nCoV, we used the single-cell RNA sequencing data for analysis. We found that the ACE2 mRNA was highly expressed in the healthy human small intestine rather than the lung. Besides, single-cell RNA sequencing data showed that ACE2 was significantly elevated in the proximal and distal enterocytes, where the small intestinal epithelium is exposed to the foreign pathogen. Thus, we suspect that ACE2-expressing small intestinal epithelium cells might be vulnerable to 2019-nCoV infection when people eat infected wild animals and diarrhea may serve as an indicator for infection, suggesting that clinicians should pay more attention to patients with diarrhea during the outbreak of pneumonia.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Weicheng Liang; Zhijie Feng; Shitao Rao; Cuicui Xiao; Zexiao Lin; Qi Zhang; Wei Qi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.20020289&#39;},
 {&#39;cord_uid&#39;: &#39;xo5u02d7&#39;,
  &#39;sha&#39;: &#39;6c91b00faa16142426820016ad3a7847fc77c8e4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.931766&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA newly identified coronavirus, 2019-nCoV, has been posing significant threats to public health since December 2019. ACE2, the host cell receptor for severe acute respiratory syndrome coronavirus (SARS), has recently been demonstrated in mediating 2019-nCoV infection. Interestingly, besides the respiratory system, substantial proportion of SARS and 2019-nCoV patients showed signs of various degrees of liver damage, the mechanism and implication of which have not yet been determined. Here, we performed an unbiased evaluation of cell type specific expression of ACE2 in healthy liver tissues using single cell RNA-seq data of two independent cohorts, and identified specific expression in cholangiocytes. The results indicated that virus might directly bind to ACE2 positive cholangiocytes but not necessarily hepatocytes. This finding suggested the liver abnormalities of SARS and 2019-nCoV patients may not be due to hepatocyte damage, but cholangiocyte dysfunction and other causes such as drug induced and systemic inflammatory response induced liver injury. Our findings indicate that special care of liver dysfunction should be installed in treating 2019-nCoV patients during the hospitalization and shortly after cure.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-04&#39;,
  &#39;authors&#39;: &#39;Xiaoqiang Chai; Longfei Hu; Yan Zhang; Weiyu Han; Zhou Lu; Aiwu Ke; Jian Zhou; Guoming Shi; Nan Fang; Jia Fan; Jiabin Cai; Jue Fan; Fei Lan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.931766&#39;},
 {&#39;cord_uid&#39;: &#39;cetdqgff&#39;,
  &#39;sha&#39;: &#39;bb8e3d331bb2975e1c644c8729b842797da2d626&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.932350&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAs of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-04&#39;,
  &#39;authors&#39;: &#39;Gurjit S. Randhawa; Maximillian P.M. Soltysiak; Hadi El Roz; Camila P.E. de Souza; Kathleen A. Hill; Lila Kari&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.932350&#39;},
 {&#39;cord_uid&#39;: &#39;7i52vltp&#39;,
  &#39;sha&#39;: &#39;181b7b57851e6f58a601b68e613d10c10616f774&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.03.933226&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-04&#39;,
  &#39;authors&#39;: &#39;Syed Faraz Ahmed; Ahmed A. Quadeer; Matthew R. McKay&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.03.933226&#39;},
 {&#39;cord_uid&#39;: &#39;f5yjz0zp&#39;,
  &#39;sha&#39;: &#39;1b531d266b4933bf8040739ed199914655b075d6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Population movement, city closure and spatial transmission of the 2019-nCoV infection in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020339&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of pneumonia caused by a novel coronavirus (2019-nCoV) in Wuhan City of China obtained global concern, the population outflow from Wuhan has contributed to spatial expansion in other parts of China. We examined the effects of population outflow from Wuhan on the 2019-nCoV transmission in other provinces and cities of China, as well as the impacts of the city closure in Wuhan. We observed a significantly positive association between population movement and the number of cases. Further analysis revealed that if the city closure policy was implemented two days earlier, 1420 (95% CI: 1059, 1833) cases could be prevented, and if two days later, 1462 (95% CI: 1090, 1886) more cases would be possible. Our findings suggest that population movement might be one important trigger of the 2019-nCoV infection transmission in China, and the policy of city closure is effective to prevent the epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Siqi Ai; Guanghu Zhu; Fei Tian; Huan Li; Yuan Gao; Yinglin Wu; Qiyong Liu; Hualiang Lin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020339&#39;},
 {&#39;cord_uid&#39;: &#39;xv42jvd6&#39;,
  &#39;sha&#39;: &#39;e3f19f88e7e3d1279b4d93e4db041e44a8d35dbb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological parameter review and comparative dynamics of influenza, respiratory syncytial virus, rhinovirus, human coronavirus, and adenovirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020404&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Influenza-like illness (ILI) accounts for a large burden of annual morbidity and mortality worldwide. A finer-grained knowledge of the parameters and dynamics of the viruses commonly underlying ILI is needed for modeling, diagnostic, and intervention efforts. We conducted an extensive literature review for epidemiological parameter values for influenza, respiratory syncytial virus (RSV), rhinovirus, human coronavirus (HCoV), and adenovirus. We also developed a deterministic SEIR model for ILI, and derived an expression for R0. We here report ranges and means for parameters for these five common viruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Julie Spencer; Deborah P Shutt; Sarah K Moser; Hannah Clegg; Helen J Wearing; Harshini Mukundan; Carrie A Manore&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020404&#39;},
 {&#39;cord_uid&#39;: &#39;kctgr529&#39;,
  &#39;sha&#39;: &#39;c9533866dd77deaa65ee1015b57bca5486f93326&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The impact of traffic isolation in Wuhan on the spread of 2019-nCov&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020438&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The 2019-nCoV outbreak occurred near the Chinese Spring Festival transport period in Wuhan. As an important transportation center, the migration of Wuhan accelerated the spread of 2019-nCoV across mainland China. Based on the cumulative Baidu migration index (CBMI), we first analyzed the proportion of Wuhan&#39;s migrant population to other cities. Our results confirm that there is a significant correlation between the export population of Wuhan and reported cases in various regions. We subsequently found that the mortality rate in Hubei Province was much higher than that in other regions of mainland China, while the investigation of potential cases in Wuhan was far behind other provinces in Mainland China, which indicates the effectiveness of early isolation.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Gehui Jin; Jiayu Yu; Liyuan Han; Shiwei Duan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020438&#39;},
 {&#39;cord_uid&#39;: &#39;8kdtpwbv&#39;,
  &#39;sha&#39;: &#39;164b0678afc42f1923aa100b624ca969a53fb3b2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Forecasting the Wuhan coronavirus (2019-nCoV) epidemics using a simple (simplistic) model - update (Feb. 8, 2020)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020461&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Confirmed infection cases in mainland China were analyzed using the data up to January 28, 2020 (first 13 days of reliable confirmed cases). For the first period the cumulative number of cases followed an exponential function. However, from January 28, we discerned a downward deviation from the exponential growth. This slower-than-exponential growth was also confirmed by a steady decline of the effective reproduction number. A backtrend analysis suggested the original basic reproduction number R0 to be about 2.4 to 2.5. As data become available, we subsequently analyzed them during three consecutive periods obtaining a sequence of model predictions. All available data up were processed the same way. We used a simple logistic growth model that fitted very well with all data. Using this model and the three sets of data, we estimated maximum cases as about 21,000, 28,000 and 35,000 cases refining these predictions in near-real time. With slightly different approach (linearization in time) the estimate of maximum cases was even higher (about 65,000). Although the estimates of maximum cases increase as more data were reported all models show reaching a peak in mid-February in contrast to the unconfined exponential growth. These predictions do not account for any possible other secondary sources of infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Slav W Hermanowicz&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020461&#39;},
 {&#39;cord_uid&#39;: &#39;iat80b4l&#39;,
  &#39;sha&#39;: &#39;013d9d1cba8a54d5d3718c229b812d7cf91b6c89&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing spread risk of Wuhan novel coronavirus within and beyond China, January-April 2020: a travel network-based modelling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020479&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: A novel coronavirus (2019-nCoV) emerged in Wuhan City, China, at the end of 2019 and has caused an outbreak of human-to-human transmission with a Public Health Emergency of International Concern declared by the World Health Organization on January 30, 2020. Aim: We aimed to estimate the potential risk and geographic range of Wuhan novel coronavirus (2019-nCoV) spread within and beyond China from January through to April, 2020. Methods: A series of domestic and international travel network-based connectivity and risk analyses were performed, by using de-identified and aggregated mobile phone data, air passenger itinerary data, and case reports. Results: The cordon sanitaire of Wuhan is likely to have occurred during the latter stages of peak population numbers leaving the city before Lunar New Year (LNY), with travellers departing into neighbouring cities and other megacities in China. We estimated that 59,912 air passengers, of which 834 (95% UI: 478 - 1349) had 2019-nCoV infection, travelled from Wuhan to 382 cities outside of mainland China during the two weeks prior to the lockdown of Wuhan. The majority of these cities were in Asia, but major hubs in Europe, the US and Australia were also prominent, with strong correlation seen between predicted importation risks and reported cases. Because significant spread has already occurred, a large number of airline travellers (3.3 million under the scenario of 75% travel reduction from normal volumes) may be required to be screened at origin high-risk cities in China and destinations across the globe for the following three months of February to April, 2020 to effectively limit spread beyond its current extent. Conclusion: Further spread of 2019-nCoV within China and international exportation is likely to occur. All countries, especially vulnerable regions, should be prepared for efforts to contain the 2019-nCoV infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Shengjie Lai; Isaac Bogoch; Nick Ruktanonchai; Alexander Watts; Xin Lu; Weizhong Yang; Hongjie Yu; Kamran Khan; Andrew J Tatem&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020479&#39;},
 {&#39;cord_uid&#39;: &#39;9fd5a49o&#39;,
  &#39;sha&#39;: &#39;158e546658e42ab0a89d174bc9facb437a034df6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Using predicted imports of 2019-nCoV cases to determine locations that may not be identifying all imported cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020495&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Cases from the ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (2019-nCoV) exported from mainland China can lead to self-sustained outbreaks in other populations. Internationally imported cases are currently being reported in several different locations. Early detection of imported cases is critical for containment of the virus. Based on air travel volume estimates from Wuhan to international destinations and using a generalized linear regression model we identify locations which may potentially have undetected internationally imported cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Pablo M De Salazar; Rene Niehus; Aimee Taylor; Caroline O Buckee; Marc Lipsitch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020495&#39;},
 {&#39;cord_uid&#39;: &#39;ecu579el&#39;,
  &#39;sha&#39;: &#39;136ee862ad53b606c8f2bea917b9705f5079e88e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Risk assessment of novel coronavirus COVID-19 outbreaks outside China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.20020503&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We developed a computational tool to assess the risk of novel coronavirus outbreaks outside China. We estimate the dependence of the risk of a major outbreak in a country from imported cases on key parameters such as: (i) the evolution of the cumulative number of cases in mainland China outside the closed areas; (ii) the connectivity of the destination country with China, including baseline travel frequencies, the effect of travel restrictions, and the efficacy of entry screening at destination; (iii) the efficacy of control measures in the destination country (expressed by the local reproduction number Rloc). We found that in countries with low connectivity to China but with relatively high Rloc, the most beneficial control measure to reduce the risk of outbreaks is a further reduction in their importation number either by entry screening or travel restrictions. Countries with high connectivity but low Rloc benefit the most from policies that further reduce Rloc. Countries in the middle should consider a combination of such policies. Risk assessments were illustrated for selected groups of countries from America, Asia and Europe, and we investigated how their risks depend on those parameters, and how the risk is increasing in time as the number of cases in China is growing.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-05&#39;,
  &#39;authors&#39;: &#39;Peter Boldog; Tamas Tekeli; Zsolt Vizi; Attila Denes; Ferenc Bartha; Gergely Rost&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.20020503&#39;},
 {&#39;cord_uid&#39;: &#39;mtv80pjo&#39;,
  &#39;sha&#39;: &#39;8a1fde8c65e439496ac5810504de23ef77312f28&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.933135&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAs the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al. (https://doi.org/10.1002/jmv.25682) and by Pradhan et al. (https://doi.org/10.1101/2020.01.30.927871), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-08&#39;,
  &#39;authors&#39;: &#39;Chengxin Zhang; Wei Zheng; Xiaoqiang Huang; Eric W. Bell; Xiaogen Zhou; Yang Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.933135&#39;},
 {&#39;cord_uid&#39;: &#39;6ojmmmuj&#39;,
  &#39;sha&#39;: &#39;74b00f19c3af87d1081644f02490ba250f57b7ca&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.04.934232&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundNew endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.MethodSeveral techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.ResultsExtensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.ConclusionT cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Miyssa I. Abdelmageed; Abdelrahman H. Abdelmoneim; Mujahed I. Mustafa; Nafisa M. Elfadol; Naseem S. Murshed; Shaza W. Shantier; Abdelrafie M. Makhawi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.04.934232&#39;},
 {&#39;cord_uid&#39;: &#39;uqkcbs2s&#39;,
  &#39;sha&#39;: &#39;59eab95c43fdea01481fdbf9bae45dfe28ffc693&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Bulk and single-cell transcriptomics identify tobacco-use disparity in lung gene expression of ACE2, the receptor of 2019-nCov&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.20020107&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;In current severe global emergency situation of 2019-nCov outbreak, it is imperative to identify vulnerable and susceptible groups for effective protection and care. Recently, studies found that 2019-nCov and SARS-nCov share the same receptor, ACE2. In this study, we analyzed five large-scale bulk transcriptomic datasets of normal lung tissue and two single-cell transcriptomic datasets to investigate the disparities related to race, age, gender and smoking status in ACE2 gene expression and its distribution among cell types. We didn&#39;t find significant disparities in ACE2 gene expression between racial groups (Asian vs Caucasian), age groups (&gt;60 vs &lt;60) or gender groups (male vs female). However, we observed significantly higher ACE2 gene expression in former smoker&#39;s lung compared to non-smoker&#39;s lung. Also, we found higher ACE2 gene expression in Asian current smokers compared to non-smokers but not in Caucasian current smokers, which may indicate an existence of gene-smoking interaction. In addition, we found that ACE2 gene is expressed in specific cell types related to smoking history and location. In bronchial epithelium, ACE2 is actively expressed in goblet cells of current smokers and club cells of non-smokers. In alveoli, ACE2 is actively expressed in remodelled AT2 cells of former smokers. Together, this study indicates that smokers especially former smokers may be more susceptible to 2019-nCov and have infection paths different with non-smokers. Thus, smoking history may provide valuable information in identifying susceptible population and standardizing treatment regimen.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Guoshuai Cai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.20020107&#39;},
 {&#39;cord_uid&#39;: &#39;xtmozkf6&#39;,
  &#39;sha&#39;: &#39;f5d9ab798adf8face1655d476644ebb75fbfef18&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.20020545&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Respiratory disease caused by the 2019 novel coronavirus (2019-nCoV) pneumonia first emerged in Wuhan, Hubei Province, China, in December 2019 and spread rapidly to other provinces and other countries. Angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV and has been suggested to be also the receptor for 2019-nCoV. Paradoxically, ACE2 expression in the lung protects mice from SARS-CoV spike protein induced lung injury by attenuating the renin-angiotensin system. In the intestine, ACE2 also suppresses intestinal inflammation by maintaining amino acid homeostasis, antimicrobial peptide expression and ecology of the gut microbiome. Upon analysis of single cell-RNA sequencing data from control subjects and those with colitis or inflammatory bowel disease (IBD), we found that ACE2 expression in the colonocytes was positively associated with genes regulating viral infection, innate and cellular immunity, but was negatively associated with viral transcription, protein translation, humoral immunity, phagocytosis and complement activation. In summary, we suggest that ACE2 may play dual roles in mediating the susceptibility and immunity of 2019-nCoV infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-07&#39;,
  &#39;authors&#39;: &#39;Jun Wang; Shanmeizi Zhao; Ming Liu; Zhiyao Zhao; Yiping Xu; Ping Wang; Meng Lin; Yanhui Xu; Bing Huang; Xiaoyu Zuo; Zhanghua Chen; Fan Bai; Jun Cui; Andrew M Lew; Jincun Zhao; Yan Zhang; Haibin Luo; Yuxia Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.20020545&#39;},
 {&#39;cord_uid&#39;: &#39;vpouyyev&#39;,
  &#39;sha&#39;: &#39;5f6a410a4ba086b296100a8a5a10df59eb337c90&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemic doubling time of the COVID-19 epidemic by Chinese province&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.20020750&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 epidemic doubling time by Chinese province was increasing from January 20 through February 9, 2020. Yet, the harmonic mean doubling time was relatively short, ranging from 1.4 (Hunan, 95% CI, 1.2-2.0) to 3.0 (Xinjiang, 95% CI, 2.0-4.9) days, with an estimate of 2.5 days (95% CI, 2.4-2.7) for Hubei.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-06&#39;,
  &#39;authors&#39;: &#39;Kamalich Muniz-Rodriguez; Gerardo Chowell; Chi-Hin Cheung; Dongyu Jia; Po-Ying Lai; Yiseul Lee; Manyun Liu; Sylvia K. Ofori; Kimberlyn M. Roosa; Lone Simonsen; Cecile G. Viboud; Isaac Chun-Hai Fung&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.20020750&#39;},
 {&#39;cord_uid&#39;: &#39;4i97fpzc&#39;,
  &#39;sha&#39;: &#39;8762adc9311e5803d1de438493772269acfe7fa3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preparedness and vulnerability of African countries against introductions of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.20020792&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus (2019-nCoV) epidemic has spread to 23 countries from China. Local cycles of transmission already occurred in 7 countries following case importation. No African country has reported cases yet. The management and control of 2019-nCoV introductions heavily relies on the public health capacity of a country. Here we evaluate the preparedness and vulnerability of African countries against their risk of importation of 2019-nCoV. We used data on air travel volumes departing from airports in the infected provinces in China and directed to Africa to estimate the risk of introduction per country. We determined the countries capacity to detect and respond to cases with two indicators: preparedness, using the WHO International Health Regulation Monitoring and Evaluation Framework; and vulnerability, with the Infectious Disease Vulnerability Index. Countries were clustered according to the Chinese regions contributing the most to their risk. Findings: Countries at the highest importation risk (Egypt, Algeria, Republic of South Africa) have moderate to high capacity to respond to outbreaks. Countries at moderate risk (Nigeria, Ethiopia, Sudan, Angola, Tanzania, Ghana, Kenya) have variable capacity and high vulnerability. Three clusters of countries are identified that share the same exposure to the risk originating from the provinces of Guangdong, Fujian, and Beijing, respectively. Interpretation: Several countries in Africa are stepping up their preparedness to detect and cope with 2019-nCoV importations. Resources and intensified surveillance and capacity capacity should be urgently prioritized towards countries at moderate risk that may be ill-prepared to face the importation and to limit onward transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-07&#39;,
  &#39;authors&#39;: &#34;Marius Gilbert; Giulia Pullano; Francesco Pinotti; Eugenio Valdano; Chiara Poletto; Pierre-Yves Boelle; Eric D&#39;Ortenzio; Yazdan Yazdanpanah; Serge Paul Eholie; Mathias Altmann; Bernardo Gutierrez; Moritz U. G. Kraemer; Vittoria Colizza&#34;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.20020792&#39;},
 {&#39;cord_uid&#39;: &#39;jawwrsa5&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A database resource for Genome-wide dynamics analysis of Coronaviruses on a historical and global scale&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.920009&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe recent outbreak of a new zoonotic origin Coronavirus has ring the bell for the potential spread of epidemic Coronavirus crossing the species. With the urgent needs to assist the control of the Coronavirus spread and to provide valuable scientific information, we developed a coronavirus database (CoVdb), an online genomics and proteomics analysis platform. Based on public available coronavirus genomic information, the database annotates the genome of every strain and identifies 780 possible ORFs of all strains available in Genebank. In addition, the comprehensive evaluation of all the published genomes of Coronavirus strains, including population genetics analysis, functional analysis and structural analysis on a historical and global scale were presented in the CoVdb. In the database, the researcher can easily obtain the basic information of a Coronavirus gene with the distribution of the gene among strains, conserved or high mutation regions, possible subcellular location and topology of the gene. Moreover, sliding windows for population genetics analysis results is provided, thereby facilitating genetics and evolutional analysis at the genomic level. CoVdb can be accessed freely at http://covdb.popgenetics.net.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-07&#39;,
  &#39;authors&#39;: &#39;Zhenglin Zhu; Kaiwen Meng; Geng Meng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.920009&#39;},
 {&#39;cord_uid&#39;: &#39;7vrm081c&#39;,
  &#39;sha&#39;: &#39;73c8af41cfdbf52c0dfba37727e3b94cb56b495e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.935072&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Bishajit Sarkar; Md. Asad Ullah; Fatema Tuz Johora; Masuma Afrin Taniya; Yusha Araf&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.935072&#39;},
 {&#39;cord_uid&#39;: &#39;py6qu4tl&#39;,
  &#39;sha&#39;: &#39;dca8ced82157924ed86c698a7dd482be81b4b266&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Teicoplanin potently blocks the cell entry of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.935387&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince December 2019, the outbreak of a new coronavirus, named 2019-nCoV, has greatly threatened the public health in China and raised great concerns worldwide. No specific treatment for this infection is currently available. We previously reported that teicoplanin, a glycopeptide antibiotic which has routinely been used in the clinic to treat bacterial infection with low toxicity, significantly inhibits the invasion of cells by Ebola virus, SARS-CoV and MERS-CoV, via specifically inhibiting the activity of cathepsin L. Here, we tested the efficacy of teicoplanin against 2019-nCoV virus infection and found that teicoplanin potently prevents the entrance of 2019-nCoV-Spike-pseudoviruses into the cytoplasm, with an IC50 of 1.66 μM. Although the inhibitory effect upon the replication of wildtype viruses ex vivo and in vivo remains to be determined, our preliminary result indicates that the potential antiviral activity of teicoplanin could be applied for the treatment of 2019-nCoV virus infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Junsong Zhang; Xiancai Ma; Fei Yu; Jun Liu; Fan Zou; Ting Pan; Hui Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.935387&#39;},
 {&#39;cord_uid&#39;: &#39;g5wpa2ee&#39;,
  &#39;sha&#39;: &#39;0aff00101d5ccc6592987185ab833f95d842f98b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Potentially highly potent drugs for 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.05.936013&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe World Health Organization (WHO) has declared the 2019 novel coronavirus (2019-nCoV) infection outbreak a global health emergency. Currently, there is no effective anti-2019-nCoV medication. The sequence identity of the 3CL proteases of 2019-nCoV and SARS is 96%, which provides a sound foundation for structural-based drug repositioning (SBDR). Based on a SARS 3CL protease X-ray crystal structure, we construct a 3D homology structure of 2019-nCoV 3CL protease. Based on this structure and existing experimental datasets for SARS 3CL protease inhibitors, we develop an SBDR model based on machine learning and mathematics to screen 1465 drugs in the DrugBank that have been approved by the U.S. Food and Drug Administration (FDA). We found that many FDA approved drugs are potentially highly potent to 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Duc Duy Nguyen; Kaifu Gao; Jiahui Chen; Rui Wang; Guo-Wei Wei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.05.936013&#39;},
 {&#39;cord_uid&#39;: &#39;ra3t6kmm&#39;,
  &#39;sha&#39;: &#39;c85f571a674c7fed0ccb9176e9cf9f3d3659ca32&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis of the epidemic growth of the early 2019-nCoV outbreak using internationally confirmed cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.06.20020941&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: On January 23, 2020, a quarantine was imposed on travel in and out of Wuhan, where the 2019 novel coronavirus (2019-nCoV) outbreak originated from. Previous analyses estimated the basic epidemiological parameters using symptom onset dates of the confirmed cases in Wuhan and outside China. Methods: We obtained information on the 46 coronavirus cases who traveled from Wuhan before January 23 and have been subsequently confirmed in Hong Kong, Japan, Korea, Macau, Singapore, and Taiwan as of February 5, 2020. Most cases have detailed travel history and disease progress. Compared to previous analyses, an important distinction is that we used this data to informatively simulate the infection time of each case using the symptom onset time, previously reported incubation interval, and travel history. We then fitted a simple exponential growth model with adjustment for the January 23 travel ban to the distribution of the simulated infection time. We used a Bayesian analysis with diffuse priors to quantify the uncertainty of the estimated epidemiological parameters. We performed sensitivity analysis to different choices of incubation interval and the hyperparameters in the prior specification. Results: We found that our model provides good fit to the distribution of the infection time. Assuming the travel rate to the selected countries and regions is constant over the study period, we found that the epidemic was doubling in size every 2.9 days (95% credible interval [CrI], 2 days--4.1 days). Using previously reported serial interval for 2019-nCoV, the estimated basic reproduction number is 5.7 (95% CrI, 3.4--9.2). The estimates did not change substantially if we assumed the travel rate doubled in the last 3 days before January 23, when we used previously reported incubation interval for severe acute respiratory syndrome (SARS), or when we changed the hyperparameters in our prior specification. Conclusions: Our estimated epidemiological parameters are higher than an earlier report using confirmed cases in Wuhan. This indicates the 2019-nCoV could have been spreading faster than previous estimates.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-09&#39;,
  &#39;authors&#39;: &#39;Qingyuan Zhao; Yang Chen; Dylan S Small&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.06.20020941&#39;},
 {&#39;cord_uid&#39;: &#39;tovfd9lw&#39;,
  &#39;sha&#39;: &#39;dfb0fedbeed56bd2b795a67faab28295afc14c96&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 2019 novel coronavirus infection in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.06.20020974&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since December 2019, acute respiratory disease (ARD) due to 2019 novel coronavirus (2019-nCoV) emerged in Wuhan city and rapidly spread throughout China. We sought to delineate the clinical characteristics of these cases. Methods: We extracted the data on 1,099 patients with laboratory-confirmed 2019-nCoV ARD from 552 hospitals in 31 provinces/provincial municipalities through January 29th, 2020. Results: The median age was 47.0 years, and 41.90% were females. Only 1.18% of patients had a direct contact with wildlife, whereas 31.30% had been to Wuhan and 71.80% had contacted with people from Wuhan. Fever (87.9%) and cough (67.7%) were the most common symptoms. Diarrhea is uncommon. The median incubation period was 3.0 days (range, 0 to 24.0 days). On admission, ground-glass opacity was the typical radiological finding on chest computed tomography (50.00%). Significantly more severe cases were diagnosed by symptoms plus reverse-transcriptase polymerase-chain-reaction without abnormal radiological findings than non-severe cases (23.87% vs. 5.20%, P&lt;0.001). Lymphopenia was observed in 82.1% of patients. 55 patients (5.00%) were admitted to intensive care unit and 15 (1.36%) succumbed. Severe pneumonia was independently associated with either the admission to intensive care unit, mechanical ventilation, or death in multivariate competing-risk model (sub-distribution hazards ratio, 9.80; 95% confidence interval, 4.06 to 23.67). Conclusions: The 2019-nCoV epidemic spreads rapidly by human-to-human transmission. Normal radiologic findings are present among some patients with 2019-nCoV infection. The disease severity (including oxygen saturation, respiratory rate, blood leukocyte/lymphocyte count and chest X-ray/CT manifestations) predict poor clinical outcomes.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-09&#39;,
  &#39;authors&#39;: &#39;Wei-jie Guan; Zheng-yi Ni; Yu Hu; Wen-hua Liang; Chun-quan Ou; Jian-xing He; Lei Liu; Hong Shan; Chun-liang Lei; David SC Hui; Bin Du; Lan-juan Li; Guang Zeng; Kowk-Yung Yuen; Ru-chong Chen; Chun-li Tang; Tao Wang; Ping-yan Chen; Jie Xiang; Shi-yue Li; Jin-lin Wang; Zi-jing Liang; Yi-xiang Peng; Li Wei; Yong Liu; Ya-hua Hu; Peng Peng; Jian-ming Wang; Ji-yang Liu; Zhong Chen; Gang Li; Zhi-jian Zheng; Shao-qin Qiu; Jie Luo; Chang-jiang Ye; Shao-yong Zhu; Nan-shan Zhong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.06.20020974&#39;},
 {&#39;cord_uid&#39;: &#39;77nsidzc&#39;,
  &#39;sha&#39;: &#39;4c24ee4a86de1d5207d93ce39132f7b660cf82e0&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Freshwater monitoring by nanopore sequencing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.06.936302&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTClean freshwater lies at the heart of human society and monitoring its quality is paramount. In addition to chemical controls, traditional microbiological water tests focus on the detection of specific bacterial pathogens. The direct tracing of all aquatic DNA poses a more profound alternative. Yet, this has hitherto been underused due to challenges in cost and logistics. Here we present a simple, fast, inexpensive and comprehensive freshwater diagnostics workflow centred around portable nanopore DNA sequencing. Using defined bacterial compositions and spatiotemporal microbiota from surface water of an example river in Cambridge (UK), our study shows how nanopore sequencing can be readily integrated for the assessment of aquatic bacterial diversity and pollution. We provide a computational benchmark that features more than ten taxonomic classification tools to derive guidelines for bacterial DNA analyses with nanopore data. Through complementary physicochemical measurements, we find that nanopore metagenomics can depict fine temporal gradients along the main hydrological axis of an urban-rural interface and yield high-resolution pathogen maps that address concerns of public health.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-07&#39;,
  &#39;authors&#39;: &#39;Lara Urban; Andre Holzer; J Jotautas Baronas; Michael Hall; Philipp Braeuninger-Weimer; Michael J Scherm; Daniel J Kunz; Surangi N Perera; Daniel E Martin-Herranz; Edward T Tipper; Susannah J Salter; Maximilian R Stammnitz&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.06.936302&#39;},
 {&#39;cord_uid&#39;: &#39;q14x0i2c&#39;,
  &#39;sha&#39;: &#39;ac35eeb6eaaf382fe0e123719f7dc409046e5d87&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Incorporating Human Movement Data to Improve Epidemiological Estimates for 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.20021071&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Estimating the key epidemiological features of the novel coronavirus (2019-nCoV) epidemic proves to be challenging, given incompleteness and delays in early data reporting, in particular, the severe under-reporting bias in the epicenter, Wuhan, Hubei Province, China. As a result, the current literature reports widely varying estimates. We developed an alternative geo-stratified debiasing estimation framework by incorporating human mobility with case reporting data in three stratified zones, i.e., Wuhan, Hubei Province excluding Wuhan, and mainland China excluding Hubei. We estimated the latent infection ratio to be around 0.12% (18,556 people) and the basic reproduction number to be 3.24 in Wuhan before the city&#39;s lockdown on January 23, 2020. The findings based on this debiasing framework have important implications to prioritization of control and prevention efforts.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-09&#39;,
  &#39;authors&#39;: &#39;Zhidong Cao; Qingpeng Zhang; Xin Lu; Dirk Pfeiffer; Lei Wang; Hongbing Song; Tao Pei; Zhongwei Jia; Daniel Dajun Zeng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.20021071&#39;},
 {&#39;cord_uid&#39;: &#39;kevrp8rg&#39;,
  &#39;sha&#39;: &#39;d5f14e63b86239f491c9dd1749cff01dc4d633cb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Networks of information token recurrences derived from genomic sequences may reveal hidden patterns in epidemic outbreaks: A case study of the 2019-nCoV coronavirus.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.20021139&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and suggest touching points and temporal pathways that are of interest for deeper investigation by geneticists and epidemiologists.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Markus Luczak-Roesch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.20021139&#39;},
 {&#39;cord_uid&#39;: &#39;45g12waw&#39;,
  &#39;sha&#39;: &#39;36a5f6d55d7c5f67d4344e36da0a72856ad3dda0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.20021154&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Steven Sanche; Yen Ting Lin; Chonggang Xu; Ethan Romero-Severson; Nick Hengartner; Ruian Ke&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.20021154&#39;},
 {&#39;cord_uid&#39;: &#39;7axgotby&#39;,
  &#39;sha&#39;: &#39;ba92c589f077b55e9cd13264d619c2ffe9d3ce7a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission Dynamics of 2019-nCoV in Malaysia&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.20021188&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This paper focuses on the formulation of a deterministic 2019-nCov transmission model by considering the exposed and recovered populations with immunity. The scenario of the simulation is depicted based on the patient zero in Malaysia. The transmission model is found to be able to predict the next confirmed case given a single case introduced in a fully susceptible population. The mathematical model is developed based on the SEIR model which has susceptible, exposed, infectious and recovered populations. The system of equations which were obtained were solved numerically and the simulation results were analyzed. The analysis includes the impact of the disease if no control is taken.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Jane Labadin; Boon Hao Hong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.20021188&#39;},
 {&#39;cord_uid&#39;: &#39;y4we4rrk&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Tracking the spread of novel coronavirus (2019-nCoV) based on big data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.20021196&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus (2019-nCoV) appeared in Wuhan in late 2019 have infected 34,598 people, and killed 723 among them until 8th February 2020. The new virus has spread to at least 316 cities (until 1st February 2020) in China. We used the traffic flow data from Baidu Map, and number of air passengers who left Wuhan from 1st January to 26th January, to quantify the potential infectious people. We developed multiple linear models with local population and air passengers as predicted variables to explain the variance of confirmed cases in every city across China. We found the contribution of air passengers from Wuhan was decreasing gradually, but the effect of local population was increasing, indicating the trend of local transmission. However, the increase of local transmission is slow during the early stage of novel coronavirus, due to the super strict control measures carried out by government agents and communities.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Xumao Zhao; Xiang Liu; Xinhai Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.20021196&#39;},
 {&#39;cord_uid&#39;: &#39;ptbk69rk&#39;,
  &#39;sha&#39;: &#39;3e5633a9a76a8c03e4ec57e4659872a63f045117&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.937862&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined. The independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying the entire (virus) species to complement research focused on individual pathogenic viruses of immediate significance. This research will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Alexander E. Gorbalenya; Susan C. Baker; Ralph S. Baric; Raoul J. de Groot; Christian Drosten; Anastasia A. Gulyaeva; Bart L. Haagmans; Chris Lauber; Andrey M Leontovich; Benjamin W. Neuman; Dmitry Penzar; Stanley Perlman; Leo L.M. Poon; Dmitry Samborskiy; Igor A. Sidorov; Isabel Sola; John Ziebuhr&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.937862&#39;},
 {&#39;cord_uid&#39;: &#39;kh06egig&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Exploring the coronavirus epidemic using the new WashU Virus Genome Browser&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.939124&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince its debut in mid-December, 2019, the novel coronavirus (2019-nCoV) has rapidly spread from its origin in Wuhan, China, to several countries across the globe, leading to a global health crisis. As of February 7, 2020, 44 strains of the virus have been sequenced and uploaded to NCBI’s GenBank [1], providing insight into the virus’s evolutionary history and pathogenesis. Here, we present the WashU Virus Genome Browser, a web-based portal for viewing virus genomic data. The browser is home to 16 complete 2019-nCoV genome sequences, together with hundreds of related viral sequences including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and Ebola virus. In addition, the browser features unique customizability, supporting user-provided upload of novel viral sequences in various formats. Sequences can be viewed in both a track-based representation as well as a phylogenetic tree-based view, allowing the user to easily compare sequence features across multiple strains. The WashU Virus Genome Browser inherited many features and track types from the WashU Epigenome Browser, and additionally incorporated a new type of SNV track to address the specific needs of viral research. Our Virus Browser portal can be accessed at https://virusgateway.wustl.edu, and documentation is available at https://virusgateway.readthedocs.io/.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Jennifer A. Flynn; Deepak Purushotham; Mayank NK Choudhary; Xiaoyu Zhuo; Changxu Fan; Gavriel Matt; Daofeng Li; Ting Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.939124&#39;},
 {&#39;cord_uid&#39;: &#39;dnxhtbxn&#39;,
  &#39;sha&#39;: &#39;adb09e4e5c7331b2aa661b3d3bb0a643e00d11bc&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.939207&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYA novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Matthew C. Wong; Sara J. Javornik Cregeen; Nadim J. Ajami; Joseph F. Petrosino&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.939207&#39;},
 {&#39;cord_uid&#39;: &#39;25uy4lrw&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.07.939389&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSevere acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the Corona Virus Disease 2019 (COVID-19) cases in China has become a public health emergency of international concern (PHEIC). Based on angiotensin converting enzyme 2 (ACE2) as cell entry receptor of SARS-CoV, we used the hACE2 transgenic mice infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant lymphocytes and monocytes in alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, alveolar macrophages and alveolar epithelia. The phenomenon was not found in wild type mice with SARS-CoV-2 infection. The pathogenicity of SARS-CoV-2 in hACE2 mice was clarified and the Koch’s postulates were fulfilled as well, and the mouse model may facilitate the development of therapeutics and vaccines against SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Linlin Bao; Wei Deng; Baoying Huang; Hong Gao; Jiangning Liu; Lili Ren; Qiang Wei; Pin Yu; Yanfeng Xu; Feifei Qi; Yajin Qu; Fengdi Li; Qi Lv; Wenling Wang; Jing Xue; Shuran Gong; Mingya Liu; Guanpeng Wang; Shunyi Wang; Zhiqi Song; Linna Zhao; Peipei Liu; Li Zhao; Fei Ye; Huijuan Wang; Weimin Zhou; Na Zhu; Wei Zhen; Haisheng Yu; Xiaojuan Zhang; Li Guo; Lan Chen; Conghui Wang; Ying Wang; Xinming Wang; Yan Xiao; Qiangming Sun; Hongqi Liu; Fanli Zhu; Chunxia Ma; Lingmei Yan; Mengli Yang; Jun Han; Wenbo Xu; Wenjie Tan; Xiaozhong Peng; Qi Jin; Guizhen Wu; Chuan Qin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.07.939389&#39;},
 {&#39;cord_uid&#39;: &#39;a8ig607t&#39;,
  &#39;sha&#39;: &#39;2cc809ed4d5c3640ee2351fdb1876e8ff6c01b51&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Feasibility of controlling 2019-nCoV outbreaks by isolation of cases and contacts&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.20021162&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: To assess the viability of isolation and contact tracing to control onwards transmission from imported cases of 2019-nCoV. Methods: We developed a stochastic transmission model, parameterised to the 2019-nCoV outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a 2019 nCoV-like pathogen. We considered scenarios that varied in: the number of initial cases; the basic reproduction number R0; the delay from symptom onset to isolation; the probability contacts were traced; the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreaks were deemed controlled if transmission ended within 12 weeks or before 5000 cases in total. We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort. Findings: While simulated outbreaks starting with only 5 initial cases, R0 of 1.5 and little transmission before symptom onset could be controlled even with low contact tracing probability, the prospects of controlling an outbreak dramatically dropped with the number of initial cases, with higher R0, and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1.5 were controllable with under 50% of contacts successfully traced. For R0 of 2.5 and 3.5, more than 70% and 90% of contacts respectively had to be traced to control the majority of outbreaks. The delay between symptom onset and isolation played the largest role in determining whether an outbreak was controllable for lower values of R0. For higher values of R0 and a large initial number of cases, contact tracing and isolation was only potentially feasible when less than 1% of transmission occurred before symptom onset. Interpretation: We found that in most scenarios contact tracing and case isolation alone is unlikely to control a new outbreak of 2019-nCov within three months. The probability of control decreases with longer delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Joel Hellewell; Sam Abbott; Amy Gimma; Nikos I Bosse; Christopher I Jarvis; Timothy W Russell; James D Munday; Adam J Kucharski; W John Edmunds; CMMID nCoV working group; Sebastian Funk; Rosalind M Eggo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.20021162&#39;},
 {&#39;cord_uid&#39;: &#39;0qwwycnc&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Caution on Kidney Dysfunctions of 2019-nCoV Patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.20021212&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Until 24:00 of February 7th 2020, 31774 laboratory-confirmed cases of novel coronavirus (2019-nCoV) infection have been reported, including 6101 severe cases in critical conditions and 722 deaths. The critical and urgent need at this moment is to find an effective treatment strategy with available means to prevent these thousands of severe inpatients from worsening and dying. It has been recently known that the 2019-nCoV shares a common cellular mechanism with the severe acute respiratory syndrome-associated coronavirus (SARS-CoV). Thus, we surveyed a previous retrospective case study on SARS which showed that acute renal impairment was uncommon in SARS but carried a formidably high mortality (91.7%, 33 of 36 cases). Here we report an ongoing case study on kidney functions in 59 patients infected by 2019-nCoV (including 28 diagnosed as severe cases and 3 deaths). 63% (32/51) of the patients exhibited proteinuria, indicative of renal impairment. 19% (11/59) and 27% (16/59) of the patients had an elevated level of plasma creatinine and urea nitrogen respectively. The computerized tomography (CT) scan showed radiographic abnormalities of the kidneys in 100% (27/27) of the patients. Together, these multiple lines of evidence point to the idea that renal impairment is common in 2019-nCov patients, which may be one of the major causes of the illness by the virus infection and also may contribute to multi-organ failure and death eventually. Therefore, we strongly suggest exercising a high degree of caution in monitoring the kidney functions of 2019-nCoV patients and, very importantly, that applying potential interventions including continuous renal replacement therapies (CRRT) for protecting kidney functions as early as possible, particular for those with plasma creatinine rising, is key to preventing fatality.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Zhen Li; Ming Wu; Jie Guo; Jiwei Yao; Xiang Liao; Siji Song; Min Han; Jiali Li; Guangjie Duan; Yuanxiu Zhou; Xiaojun Wu; Zhansong Zhou; Taojiao Wang; Ming Hu; Xianxiang Chen; Yu Fu; Chong Lei; Hailong Dong; Yi Zhou; Hongbo Jia; Xiaowei Chen; Junan Yan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.20021212&#39;},
 {&#39;cord_uid&#39;: &#39;juiomzb3&#39;,
  &#39;sha&#39;: &#39;5bbc428de7fc2f10c24d8439edda26d84b319687&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of the Time-Varying Reproduction Number of COVID-19 Outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.20021253&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The 2019-nCoV outbreak in Wuhan, China has attracted world-wide attention. As of February 11, 2020, a total of 44730 cases of novel coronavirus-infected pneumonia associated with COVID-19 were confirmed by the National Health Commission of China. Methods: Three approaches, namely Poisson likelihood-based method (ML), exponential growth rate-based method (EGR) and stochastic Susceptible-Infected-Removed dynamic model-based method (SIR), were implemented to estimate the basic and controlled reproduction numbers. Results: A total of 71 chains of transmission together with dates of symptoms onset and 67 dates of infections were identified among 5405 confirmed cases outside Hubei as reported by February 2, 2020. Based on this information, we find the serial interval having an average of 4.41 days with a standard deviation of 3.17 days and the infectious period having an average of 10.91 days with a standard deviation of 3.95 days. Conclusions: The controlled reproduction number is declining. It is lower than one in most regions of China, but is still larger than one in Hubei Province. Sustained efforts are needed to further reduce the Rc to below one in order to end the current epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Chong You; Yuhao Deng; Wenjie Hu; Jiarui Sun; Qiushi Lin; Feng Zhou; Cheng Heng Pang; Yuan Zhang; Zhengchao Chen; Xiao-Hua Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.20021253&#39;},
 {&#39;cord_uid&#39;: &#39;2g8rfzf4&#39;,
  &#39;sha&#39;: &#39;5cfb44ecdfb61b2898b6cf7315b1e810874a9560&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing the plausibility of subcritical transmission of 2019-nCoV in the United States&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.20021311&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract: The 2019-nCoV outbreak has raised concern of global spread. While person-to-person transmission within the Wuhan district has led to a large outbreak, the transmission potential outside of the region remains unclear. Here we present a simple approach for determining whether the upper limit of the confidence interval for the reproduction number exceeds one for transmission in the United States, which would allow endemic transmission. As of February 7, 2020, the number of cases in the United states support subcritical transmission, rather than ongoing transmission. However, this conclusion can change if pre-symptomatic cases resulting from human-to-human transmission have not yet been identified.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Seth Blumberg; Thomas M Lietman; Travis C Porco&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.20021311&#39;},
 {&#39;cord_uid&#39;: &#39;pm8usc55&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.926006&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAt the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China1, even more spread than SARS-CoV infection2. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion3,4. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear5,6. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Tong Meng; Hao Cao; Hao Zhang; Zijian Kang; Da Xu; Haiyi Gong; Jing Wang; Zifu Li; Xingang Cui; Huji Xu; Haifeng Wei; Xiuwu Pan; Rongrong Zhu; Jianru Xiao; Wang Zhou; Liming Cheng; Jianmin Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.926006&#39;},
 {&#39;cord_uid&#39;: &#39;hwvr0v8c&#39;,
  &#39;sha&#39;: &#39;329dfdd2fbe4b3c510c5329219bfd33764ab1710&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Domestication reprogrammed the budding yeast life cycle&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.939314&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDomestication of plants and animals is the foundation for feeding the world population. We report that domestication of the model yeast S. cerevisiae reprogrammed its life cycle entirely. We tracked growth, gamete formation and cell survival across many environments for nearly 1000 genome sequenced isolates and found a remarkable dichotomy between domesticated and wild yeasts. Wild yeasts near uniformly trigger meiosis and sporulate when encountering nutrient depletions, whereas domestication relaxed selection on sexual reproduction and favoured survival as quiescent cells. Domestication also systematically enhanced fermentative over respiratory traits while decreasing stress tolerance. We show that this yeast domestication syndrome was driven by aneuploidies and gene function losses that emerged independently in multiple domesticated lineages during the specie’s recent evolutionary history. We found domestication to be the most dramatic event in budding yeast evolution, raising questions on how much domestication has distorted our understanding of this key model species.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-09&#39;,
  &#39;authors&#39;: &#39;Matteo De Chiara; Benjamin Barré; Karl Persson; Amadi Onyetuga Chioma; Agurtzane Irizar; Joseph Schacherer; Jonas Warringer; Gianni Liti&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.939314&#39;},
 {&#39;cord_uid&#39;: &#39;xetzg7gp&#39;,
  &#39;sha&#39;: &#39;17307ab6eda64e1538da204c02114dd36a9b7fd6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.939553&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, used mutations to evade attack from the human immune system. Overall, we present an immunological resource for SARS-CoV-2 that could promote both therapeutic development and mechanistic research.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.939553&#39;},
 {&#39;cord_uid&#39;: &#39;emr0eh0i&#39;,
  &#39;sha&#39;: &#39;a22ceac8d5e1cfaf2e79557d3c794458d0e1cdbe&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Identification of a Pangolin Niche for a 2019-nCoV-like Coronavirus via an Extensive Meta-metagenomic Search&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.939660&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn numerous instances, tracking the biological significance of a nucleic acid sequence can be augmented through the identification of environmental niches in which the sequence of interest is present. Many metagenomic datasets are now available, with deep sequencing of samples from diverse biological niches. While any individual metagenomic dataset can be readily queried using web-based tools, meta-searches through all such datasets are less accessible. In this brief communication, we demonstrate such a meta-meta-genomic approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the keyword “virome”. In addition to the homology to bat coronaviruses observed in descriptions of the 2019-nCoV sequence (F. Wu et al. 2020, Nature, doi.org/10.1038/s41586-020-2008-3; P. Zhou et al. 2020, Nature, doi.org/10.1038/s41586-020-2012-7), we note a strong homology to numerous sequence reads in a metavirome dataset generated from the lungs of deceased Pangolins reported by Liu et al. (Viruses 11:11, 2019, http://doi.org/10.3390/v11110979). Our observations are relevant to discussions of the derivation of 2019-nCoV and illustrate the utility and limitations of meta-metagenomic search tools in effective and rapid characterization of potentially significant nucleic acid sequences.ImportanceMeta-metagenomic searches allow for high-speed, low-cost identification of potentially significant biological niches for sequences of interest.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-14&#39;,
  &#39;authors&#39;: &#39;Lamia Wahba; Nimit Jain; Andrew Z. Fire; Massa J. Shoura; Karen L. Artiles; Matthew J. McCoy; Dae-Eun Jeong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.939660&#39;},
 {&#39;cord_uid&#39;: &#39;mm2sp48e&#39;,
  &#39;sha&#39;: &#39;b970b7a807f37678f91d4b989ffdb7d7606d754d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.08.939892&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince December 2019, a novel coronavirus named 2019 coronavirus (2019-nCoV) has emerged in Wuhan of China and spread to several countries worldwide within just one month. Apart from fever and respiratory complications, acute kidney injury has been observed in some patients with 2019-nCoV. In a short period of time, angiotensin converting enzyme II (ACE2), have been proposed to serve as the receptor for the entry of 2019-nCoV, which is the same for severe acute respiratory syndrome coronavirus (SARS). To investigate the possible cause of kidney damage in 2019-nCoV patients, we used both published kidney and bladder cell atlas data and an independent unpublished kidney single cell RNA-Seq data generated in-house to evaluate ACE2 gene expressions in all cell types in healthy kidneys and bladders.Our results showed the enriched expression of all subtypes of proximal tubule cells of kidney and low but detectable levels of expression in bladder epithelial cells. These results indicated the urinary system is a potential route for 2019-nCoV infection, along with the respiratory system and digestion system. Our findings suggested the kidney abnormalities of SARS and 2019-nCoV patients may be due to proximal tubule cells damage and subsequent systematic inflammatory response induced kidney injury. Beyond that, laboratory tests of viruses and related indicators in urine may be needed in some special patients of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Wei Lin; Longfei Hu; Yan Zhang; Joshua D. Ooi; Ting Meng; Peng Jin; Xiang Ding; Longkai Peng; Lei Song; Zhou Xiao; Xiang Ao; Xiangcheng Xiao; Qiaoling Zhou; Ping Xiao; Jue Fan; Yong Zhong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.08.939892&#39;},
 {&#39;cord_uid&#39;: &#39;f87h5qh6&#39;,
  &#39;sha&#39;: &#39;5c31897d01f3edc7f58a0f03fceec2373fcfdc3d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The effect of travel restrictions on the spread of the 2019 novel coronavirus (2019-nCoV) outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.09.20021261&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Motivated by the rapid spread of a novel coronavirus (2019-nCoV) in Mainland China, we use a global metapopulation disease transmission model to project the impact of both domestic and international travel limitations on the national and international spread of the epidemic. The model is calibrated on the evidence of internationally imported cases before the implementation of the travel quarantine of Wuhan. By assuming a generation time of 7.5 days, the reproduction number is estimated to be 2.4 [90% CI 2.2-2.6]. The median estimate for number of cases before the travel ban implementation on January 23, 2020 is 58,956 [90% CI 40,759 - 87,471] in Wuhan and 3,491 [90% CI 1,924 - 7,360] in other locations in Mainland China. The model shows that as of January 23, most Chinese cities had already received a considerable number of infected cases, and the travel quarantine delays the overall epidemic progression by only 3 to 5 days. The travel quarantine has a more marked effect at the international scale, where we estimate the number of case importations to be reduced by 80% until the end of February. Modeling results also indicate that sustained 90% travel restrictions to and from Mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher reduction of transmission in the community.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Matteo Chinazzi; Jessica T. Davis; Marco Ajelli; Corrado Gioannini; Maria Litvinova; Stefano Merler; Ana Pastore y Piontti; Luca Rossi; Kaiyuan Sun; Cécile Viboud; Xinyue Xiong; Hongjie Yu; M. Elizabeth Halloran; Ira M. Longini; Alessandro Vespignani&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.09.20021261&#39;},
 {&#39;cord_uid&#39;: &#39;6u9q0ox9&#39;,
  &#39;sha&#39;: &#39;2913d91f13fd59c698f68ba63008d8e0550c0607&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Spatially Explicit Modeling of 2019-nCoV Epidemic Trend based on Mobile Phone Data in Mainland China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.09.20021360&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;IAs of February 11, 2020, all prefecture-level cities in mainland China have reported confirmed cases of 2019 novel coronavirus (2019-nCoV), but the city-level epidemical dynamics is unknown. The aim of this study is to model the current dynamics of 2019-nCoV at city level and predict the trend in the next 30 days under three possible scenarios in mainland China. We developed a spatially explicit epidemic model to consider the unique characteristics of the virus transmission in individual cities. Our model considered that the rate of virus transmission among local residents is different from those with Wuhan travel history due to the self-isolation policy. We introduced a decay rate to quantify the effort of each city to gradually control the disease spreading. We used mobile phone data to obtain the number of individuals in each city who have travel history to Wuhan. This city-level model was trained using confirmed cases up to February 10, 2020 and validated by new confirmed cases on February 11, 2020. We used the trained model to predict the future dynamics up to March 12, 2020 under different scenarios: the current trend maintained, control efforts expanded, and person-to-person contact increased due to work resuming. We estimated that the total infections in mainland China would be 72172, 54348, and 149774 by March 12, 2020 under each scenario respectively. Under the current trend, all cities will show the peak point of daily new infections by February 21. This date can be advanced to February 14 with control efforts expanded or postponed to February 26 under pressure of work resuming. Except Wuhan that cannot eliminate the disease by March 12, our model predicts that 95.4%, 100%, and 75.7% cities will have no new infections by the end of February under three scenarios. The spatial pattern of our prediction could help the government allocate resources to cities that have a more serious epidemic in the next 30 days.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Xiaolin Zhu; Aiyin Zhang; Shuai Xu; Pengfei Jia; Xiaoyue Tan; Jiaqi Tian; Tao Wei; Zhenxian Quan; Jiali Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.09.20021360&#39;},
 {&#39;cord_uid&#39;: &#39;5dd89gnm&#39;,
  &#39;sha&#39;: &#39;d2609c4d8837558f4b8f32170422ce1cd12425a6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing the Tendency of 2019-nCoV (COVID-19) Outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.09.20021444&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since December 8, 2019, the spread of COVID-19 is increasing every day. It is particularly important to predict the trend of the epidemic for the timely adjustment of the economy and industries. We proposed a Flow-SEHIR model in this paper, based on which we further analyzed the trends of 2019-nCoV (COVID-19) in China. The results show that the basic reproductive numbers R0 of COVID-19 is 3.56 (95% CI: 2.31-4.81). The number of daily confirmed new cases reaches the inflection point on Feb. 6-10 outside Hubei. For the maximum of infected cases number, the predicted peak value in China except Hubei was estimated to be 13806 (95% CI: 11926-15845). The peak arrival time is on March 3-9. The temporal number of patients in most areas of China outside Hubei will peak from March 12 to March 15. The peak values of more than 73.5% provinces or regions in China will be controlled within 1000. According to Flow-SEHIR model and estimations from the data of evacuation of nationals from Wuhan, the peak cumulative number of patients in Hubei was estimated to be 403481 (95% CI: 143284-1166936).&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Qinghe Liu; Zhicheng Liu; Deqiang Li; Zefei Gao; Junkai Zhu; Junyan Yang; Qiao Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.09.20021444&#39;},
 {&#39;cord_uid&#39;: &#39;uqgrjjwp&#39;,
  &#39;sha&#39;: &#39;dcd7a1235ea74e3ef71d051103bf8a64c3c8f457&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The lockdown of Hubei Province causing different transmission dynamics of the novel coronavirus (2019-nCoV) in Wuhan and Beijing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.09.20021477&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: After the outbreak of novel coronavirus (2019-nCoV) starting in late 2019, a number of researchers have reported the predicted the virus transmission dynamics. However, under the strict control policy the novel coronavirus does not spread naturally outside Hubei Province, and none of the prediction closes to the real situation. Methods and findings: We used the traditional SEIR model, fully estimated the effect of control measures, to predict the virus transmission in Wuhan, the capital city of Hubei Province, and Beijing. We forecast that the outbreak of 2019-nCoV would reach its peak around March 6±10 in Wuhan and March 20±16 in Beijing, respectively. The infectious population in Beijing would be much less (only 0.3%) than those in Wuhan at the peak of this transmission wave. The number of confirmed cases in cities inside Hubei Province grow exponentially, whereas those in cities outside the province increase linearly. Conclusions: The unprecedented province lockdown substantially suspends the national and global outbreak of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Xinhai Li; Xumao Zhao; Yuehua Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.09.20021477&#39;},
 {&#39;cord_uid&#39;: &#39;er3zmcz2&#39;,
  &#39;sha&#39;: &#39;815745bf1b522d33fd7371cc9a6561a2a93ef87e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Simulating the infected population and spread trend of 2019-nCov under different policy by EIR model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021519&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported official data, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Hao Xiong; Huili Yan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021519&#39;},
 {&#39;cord_uid&#39;: &#39;pd70i3d8&#39;,
  &#39;sha&#39;: &#39;48656efc59191537073975938f25f201524971af&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021584&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Severe ill patients with 2019 novel coronavirus (2019-nCoV) infection progressed rapidly to acute respiratory failure. We aimed to select the most useful prognostic factor for severe illness incidence. Methods: The study prospectively included 61 patients with 2019-nCoV infection treated at Beijing Ditan Hospital from January 13, 2020 to January 31, 2020. Prognostic factor of severe illness was selected by the LASSO COX regression analyses, to predict the severe illness probability of 2019-CoV pneumonia. The predictive accuracy was evaluated by concordance index, calibration curve, decision curve and clinical impact curve. Results: The neutrophil-to-lymphocyte ratio (NLR) was identified as the independent risk factor for severe illness in patients with 2019-nCoV infection. The NLR had a c-index of 0.807 (95% confidence interval, 0.676-0.38), the calibration curves fitted well, and the decision curve and clinical impact curve showed that the NLR had superior standardized net benefit. In addition, the incidence of severe illness was 9.1% in age ≥ 50 and NLR &lt; 3.13 patients, and half of patients with age ≥ 50 and NLR ≥ 3.13 would develop severe illness. Based on the risk stratification of NLR with age, the study developed a 2019-nCoV pneumonia management process. Conclusions: The NLR was the early identification of risk factors for 2019-nCoV severe illness. Patients with age ≥ 50 and NLR ≥ 3.13 facilitated severe illness, and they should rapidly access to intensive care unit if necessary.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Jingyuan Liu; Yao Liu; Pan Xiang; Lin Pu; Haofeng Xiong; Chuansheng Li; Ming Zhang; Jianbo Tan; Yanli Xu; Rui Song; Meihua Song; Lin Wang; Wei Zhang; Bing Han; Li Yang; Xiaojing Wang; Guiqin Zhou; Ting Zhang; Ben Li; Yanbin Wang; Zhihai Chen; Xianbo Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021584&#39;},
 {&#39;cord_uid&#39;: &#39;172tl3fb&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and clinical features of the 2019 novel coronavirus outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021675&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Our manuscript was based on surveillance cases of COVID-19 identified before January 26, 2020. As of February 20, 2020, the total number of confirmed cases in mainland China has reached 18 times of the number in our manuscript. While the methods and the main conclusions in our original analyses remain solid, we decided to withdraw this preprint for the time being, and will replace it with a more up-to-date version shortly. Should you have any comments or suggestions, please feel free to contact the corresponding author.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;Yang Yang; Qingbin Lu; Mingjin Liu; Yixing Wang; Anran Zhang; Neda Jalali; Natalie Dean; Ira Longini; M. Elizabeth Halloran; Bo Xu; Xiaoai Zhang; Liping Wang; Wei Liu; Liqun Fang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021675&#39;},
 {&#39;cord_uid&#39;: &#39;zbqbogwu&#39;,
  &#39;sha&#39;: &#39;06c1b3535b83251cf92c01258b5048beeab7a460&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Beyond R0: the importance of contact tracing when predicting epidemics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021725&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The basic reproductive number --- R0 --- is one of the most common and most commonly misapplied numbers in public health. Nevertheless, estimating R0 for every transmissible pathogen, emerging or endemic, remains a priority for epidemiologists the world over. Although often used to compare outbreaks and forecast pandemic risk, this single number belies the complexity that two different pathogens can exhibit, even when they have the same R0. Here, we show how predicting outbreak size requires both an estimate of R0 and an estimate of the heterogeneity in the number of secondary infections. To facilitate rapid determination of outbreak risk, we propose a reformulation of a classic result from random network theory that relies on contact tracing data to simultaneously determine the first moment (R0) and the higher moments (representing the heterogeneity) in the distribution of secondary infections. Further, we show how this framework is robust in the face of the typically limited amount of data for emerging pathogens. Lastly, we demonstrate that without data on the heterogeneity in secondary infections for emerging pathogens like 2019-nCoV, the uncertainty in outbreak size ranges dramatically, in the case of 2019-nCoV from 5-40% of susceptible individuals. Taken together, our work highlights the critical need for contact tracing during emerging infectious disease outbreaks and the need to look beyond R0 when predicting epidemic size.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Laurent Hébert-Dufresne; Benjamin M. Althouse; Samuel V. Scarpino; Antoine Allard&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021725&#39;},
 {&#39;cord_uid&#39;: &#39;5td4p2gd&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Statistical Inference for Coronavirus Infected Patients in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021774&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: The new coronavirus outbreak has seriously affected the quality of life in China. Wuhan is the disaster area, where the number of cases has increased rapidly. However, the current measures of infected patients in Wuhan are still underestimated. Objective: To estimate the overall infected patients in Wuhan from several sampled data. The correct estimated infected patients can be helpful for the government to arrange the needed beds in hospital wards to meet the actual needs. Design: We proposed to use the sampling survey to estimate the overall infected patients in Wuhan. The sampling survey is a kind of non-comprehensive survey. It selected some units from all the survey objects to carry out the survey and made the estimation and inference to all the survey objects. Sampling surveys can obtain information that reflects the overall situation, although it is not a comprehensive survey. Setting: We estimated the overall infection rate in Wenzhou city, which has a better data collection system. Simultaneously, another different samples of Wuhan tourists to Singapore will be used to validate the infection rate in Wenzhou city. Combined these two samples, we give the estimation of the number of infected patients in Wuhan and other prefecture-level cities in Hubei Province. Participants: The number of people who returned from Wuhan to Wenzhou was selected from the daily notification of the pneumonia epidemic caused by a new coronavirus infection in the city. Exposures for observational studies: The daily rate of the pneumonia epidemic caused by the new coronavirus infection in Wenzhou City. The numerator is the number of people diagnosed and whether each person diagnosed had a history of living in Wuhan. The denominator is the total number of people returning to Wenzhou from Wuhan. Based on this rate, it is reasonable to predict the number of the infected patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Yongdao Zhou; Jianghu Dong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021774&#39;},
 {&#39;cord_uid&#39;: &#39;o7pz0rst&#39;,
  &#39;sha&#39;: &#39;eec5b4ce2de8d9d00be664e6af79cac476991bbf&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Distribution of the 2019-nCoV Epidemic and Correlation with Population Emigration from Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021824&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUNDS: The ongoing new coronavirus (2019-nCoV) pneumonia outbreak is spreading in China and has not reached its peak. Five millions of people had emigrated from Wuhan before the city lockdown, which potentially represent a source of virus spreaders. Case distribution and its correlation with population emigration from Wuhan in early epidemic are of great importance for early warning and prevention of future outbreak. METHODS: The officially reported cases of 2019-nCoV pneumonia were collected as of January 30, 2020. Time and location information of these cases were extracted analyzed with ArcGIS and WinBUGS. Population migration data of Wuhan City and Hubei province were extracted from Baidu Qianxi and analyzed for their correlation with case number. FINDINGS: The 2019-nCoV pneumonia cases were predominantly distributed in Hubei and other provinces of South China. Hot spot provinces included Sichuan and Yunnan provinces that are adjacent to Hubei. While Wuhan city has the highest number of cases, the time risk is relatively stable. Numbers of cases in some cities are relatively low, but the time risks are continuously rising. The case numbers of different provinces and cities of Hubei province were highly correlated with the emigrated populations from Wuhan. Lockdown of 19 cities of Hubei province, and implementation of nationwide control measures efficiently prevented the exponential growth of case number. INTERPRETATION: Population emigrated from Wuhan was the main infection source for other cities and provinces. Some cities with low case number but were in rapid increase. Due to the upcoming Spring Festival return transport wave, understanding of the trends of risks in different regions is of great significance for preparedness for both individuals and institutions.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Zeliang Chen; Qi Zhang; Yi Lu; Xi Zhang; Wenjun Zhang; Cheng Guo; Conghui Liao; Qianlin Li; Xiaohu Han; Jiahai Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021824&#39;},
 {&#39;cord_uid&#39;: &#39;qng05jry&#39;,
  &#39;sha&#39;: &#39;15ad534bb2ec7bc4b0ee83ddac11a1b837d743c6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.20021832&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: To explore the cellular immunity and cytokines status of NCP patients and to predict the correlation between the cellular immunity levels, cytokines and the severity of patients. Methods: 123 NCP patients were divided into mild and severe groups. Peripheral blood was collected, lymphocyte subsets and cytokines were detected. Correlation analysis was performed on the lymphocyte subsets and cytokines, and the differences between the indexes of the two groups were analyzed. Results: 102 mild and 21 severe patients were included. Lymphocyte subsets were reduced in two groups. The proportion of CD8 + T reduction in the mild and severe group was 28.43% and 61.9%, respectively; The proportion of B cell reduction was 25.49% and 28.57%; The proportion of NK cell reduction was 34.31% and 47.62%; The detection value of IL-6 was 0 in 55.88% of the mild group, mild group has a significantly lower proportion of patients with IL-6 higher than normal than severe group; There was no significant linear correlation between the lymphocyte subsets and cytokines, while significant differences were noticed between the two groups in CD4 + T, CD8 + T, IL-6 and IL-10. Conclusions: Low levels of CD4+T and CD8+T are common in severe NCP. IL-6 and IL-10 levels were higher in severe patients. T cell subsets and cytokines can be used as one of the basis for predicting the transition from mild to severe. Large number of samples are still needed to confirm the &#34;warning value&#34; of CD4 + T, CD8 + T IL-6 and IL-10.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Suxin Wan; Qingjie Yi; Shibing Fan; Jinglong Lv; Xianxiang Zhang; Lian Guo; Chunhui Lang; Qing Xiao; Kaihu Xiao; Zhengjun Yi; Mao Qiang; Jianglin Xiang; Bangshuo Zhang; Yongping Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.20021832&#39;},
 {&#39;cord_uid&#39;: &#39;7n8p9okf&#39;,
  &#39;sha&#39;: &#39;83323659de2f2d5ed3ca6ada7cf137a8a17bb014&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.936898&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate. Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs. In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases. Six crystal structures of protease:inhibitor complexes were determined as part of this study. Compounds synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic. Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera. The best near-equipotent inhibitors, 11u (P2 = cyclopentylmethyl) and 11r (P2 = cyclohexylmethyl), display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures. In Huh7 cells, 11r exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-10&#39;,
  &#39;authors&#39;: &#39;Linlin Zhang; Daizong Lin; Yuri Kusov; Yong Nian; Qingjun Ma; Jiang Wang; Albrecht von Brunn; Pieter Leyssen; Kristina Lanko; Johan Neyts; Adriaan de Wilde; Eric J. Snijder; Hong Liu; Rolf Hilgenfeld&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.936898&#39;},
 {&#39;cord_uid&#39;: &#39;z3x0c37y&#39;,
  &#39;sha&#39;: &#39;7f8c673d136b9bc0aeda55e49cc72aacaa5688de&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structural genomics and interactomics of 2019 Wuhan novel coronavirus, 2019-nCoV, indicate evolutionary conserved functional regions of viral proteins&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.942136&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDuring its first month, the recently emerged 2019 Wuhan novel coronavirus (2019-nCoV) has already infected many thousands of people in mainland China and worldwide and took hundreds of lives. However, the swiftly spreading virus also caused an unprecedentedly rapid response from the research community facing the unknown health challenge of potentially enormous proportions. Unfortunately, the experimental research to understand the molecular mechanisms behind the viral infection and to design a vaccine or antivirals is costly and takes months to develop. To expedite the advancement of our knowledge we leverage the data about the related coronaviruses that is readily available in public databases, and integrate these data into a single computational pipeline. As a result, we provide a comprehensive structural genomics and interactomics road-maps of 2019-nCoV and use these information to infer the possible functional differences and similarities with the related SARS coronavirus. All data are made publicly available to the research community at http://korkinlab.org/wuhan&#39;,
  &#39;publish_time&#39;: &#39;2020-02-14&#39;,
  &#39;authors&#39;: &#39;Hongzhu Cui; Ziyang Gao; Ming Liu; Senbao Lu; Sun Mo; Winnie Mkandawire; Oleksandr Narykov; Suhas Srinivasan; Dmitry Korkin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.942136&#39;},
 {&#39;cord_uid&#39;: &#39;xa6kwguo&#39;,
  &#39;sha&#39;: &#39;7ab9f9fcea519ebce527c3ede8091beedbb26ad9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.942185&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Javier A. Jaimes; Nicole M. André; Jean K. Millet; Gary R. Whittaker&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.942185&#39;},
 {&#39;cord_uid&#39;: &#39;bgdcm1i1&#39;,
  &#39;sha&#39;: &#39;8c210df7072bc88fa7d2da72b25c84430c16f5e8&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Recombination and lineage-specific mutations led to the emergence of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.942748&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The recent outbreak of a new coronavirus (SARS-CoV-2) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS-CoV and MERS-CoV, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the SARS-CoV-2 involving the RBD. As a result of this recombination event, SARS-CoV and SARS-CoV-2 share a similar genotype in RBD, including two insertions (positions 432-436 and 460-472), and alleles 427N and 436Y. Both 427N and 436Y belong to a helix that interacts with the human ACE2 receptor. Ancestral state analyses revealed that SARS-CoV-2 differentiated from its most recent common ancestor with RaTG13 by accumulating a significant number of amino acid changes in the RBD. In sum, we propose a two-hit scenario in the emergence of the SARS-CoV-2 virus whereby the SARS-CoV-2 ancestors in bats first acquired genetic characteristics of SARS-CoV by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, the lineage that led to SARS-CoV-2 accumulated further, unique changes specifically in the RBD.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Juan Angel Patino-Galindo; Ioan Filip; Mohammed AlQuraishi; Raul Rabadan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.942748&#39;},
 {&#39;cord_uid&#39;: &#39;mpb4fy16&#39;,
  &#39;sha&#39;: &#39;35ac99bfeb665ce0a2ae361dba1161e69eb8b7d5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;ΦX174 Attenuation by Whole Genome Codon Deoptimization&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.10.942847&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractNatural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-11&#39;,
  &#39;authors&#39;: &#39;James T. Van Leuven; Martina M. Ederer; Katelyn Burleigh; LuAnn Scott; Randall A. Hughes; Vlad Codrea; Andrew D. Ellington; Holly Wichman; Craig Miller&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.10.942847&#39;},
 {&#39;cord_uid&#39;: &#39;6r6zwfoy&#39;,
  &#39;sha&#39;: &#39;2cfe7a4cc1fceed0ad7ffbd6244a61e0b2c3ac8b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Facemask shortage and the coronavirus disease (COVID-19) outbreak: Reflection on public health measures&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20020735&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;To the best of our knowledge, this is the first study to investigate the facemask shortage during the novel coronavirus pneumonia (COVID-19) outbreak in China. We have summarized in detail the management strategies implemented by the Chinese governments during the outbreaks. By considering three scenarios for the outbreak development, we simulated the facemasks availability from late-December 2019 to late-April 2020 and estimated the duration of sufficient facemask supplies. Our findings showed that if the COVID-19 outbreak occurred only in Wuhan city or Hubei province, facemask shortage would not appear with the existing public health measures. However, if the outbreak occurred in the whole of China, a shortage of facemask could be substantial assuming no alternative public health measures. Supplies of facemasks in the whole of China would have been sufficient for both healthcare workers and the general population if the COVID-19 outbreak only occurred in Wuhan city or Hubei province. However, if the outbreak occurred in the whole of China, facemask supplies in China could last for 5 days if under the existing public health measures and a shortage of 853 million facemasks is expected by 30 Apr 2020. Assuming a gradually decreased import volume, we estimated that dramatic increase in productivity (42.7 times of the usual level) is needed to mitigate the facemask crisis by the end of April. In light of the COVID-19 outbreak in China, a shortage of facemasks and other medical resources can considerably compromise the efficacy of public health measures. Effective public health measures should also consider the adequacy and affordability of medical resources. Global collaboration should be strengthened to prevent the development of a global pandemic from a regional epidemic via easing the medical resources crisis in the affected countries.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Huailiang Wu; Jian Huang; Casper JP Zhang; Zonglin He; Wai-kit Ming&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20020735&#39;},
 {&#39;cord_uid&#39;: &#39;5xki1ulf&#39;,
  &#39;sha&#39;: &#39;8b88901d0bb033af3a18ebcdf7f2aa7110b21504&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluating the accuracy of different respiratory specimens in the laboratory diagnosis and monitoring the viral shedding of 2019-nCoV infections&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20021493&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The outbreak of novel coronavirus pneumonia (NCP) caused by 2019-nCoV spread rapidly, and elucidation the diagnostic accuracy of different respiratory specimens is crucial for the control and treatment of this diseases. Methods: Respiratory samples including nasal swabs, throat swabs, sputum and bronchoalveolar lavage fluid (BALF) were collected from Guangdong CDC confirmed NCP patients, and viral RNAs were detected using a CFDA approved detection kit. Results were analyzed in combination with sample collection date and clinical information. Finding: Except for BALF, the sputum possessed the highest positive rate (74.4%~88.9%), followed by nasal swabs (53.6%~73.3%) for both severe and mild cases during the first 14 days after illness onset (d.a.o). For samples collected ≥ 15 d.a.o, sputum and nasal swabs still possessed a high positive rate ranging from 42.9%~61.1%. The positive rate of throat swabs collected ≥ 8 d.a.o was low, especially in samples from mild cases. Viral RNAs could be detected in all the lower respiratory tract of severe cases, but not the mild cases. CT scan of cases 02, 07 and 13 showed typical viral pneumonia with ground glass opacity, while no viral RNAs were detected in first three or all the upper respiratory samples. Interpretation: Sputum is most accurate for laboratory diagnosis of NCP, followed by nasal swabs. Detection of viral RNAs in BLAF is necessary for diagnosis and monitoring of viruses in severe cases. CT scan could serve as an important make up for the diagnosis of NCP. Funding National Science and Technology Major Project, Sanming Project of Medicine and China Postdoctoral Science Foundation.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Yang Yang; Minghui Yang; Chenguang Shen; Fuxiang Wang; Jing Yuan; Jinxiu Li; Mingxia Zhang; Zhaoqin Wang; Li Xing; Jinli Wei; Ling Peng; Gary Wong; Haixia Zheng; Mingfeng Liao; Kai Feng; Jianming Li; Qianting Yang; Juanjuan Zhao; Zheng Zhang; Lei Liu; Yingxia Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20021493&#39;},
 {&#39;cord_uid&#39;: &#39;w5t1ni4l&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Ophthalmologic evidence against the interpersonal transmission of 2019 novel coronavirus through conjunctiva&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20021956&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The emerging 2019 novel coronavirus (2019−nCoV) has pushed several countries into state of emergency all over the world. The possible transmission of 2019−nCoV by conjunctiva is controversial and has substantial public health implications. Methods: A retrospective cohort study was initiated to investigate the possible transmission of 2019−nCoV through aerosol contact with conjunctiva. We enrolled 67 cases of confirmed or suspected cases of novel coronavirus pneumonia (NCP) during 17−28 Jan 2020. Nasopharyngeal and conjunctival swabs were collected for real time RT-PCR analysis to detect 2019−nCoV. Results: 63 patients were identified as laboratory−confirmed NCP and the remaining four were suspected NCP. Conjunctival swab samples from one NCP patient yielded positive PCR results and two NCP patients yielded probable positive PCR results. None of the three patients had ocular symptoms. The only one NCP patient with conjunctivitis as the first symptom had negative conjunctival sac 2019−nCoV test. Conjunctival swab samples from the four suspected cases of NCIP were negative. Conclusion: 2019−nCoV can be detected in the conjunctival sac of patients with NCP. Through clinical analysis, viral transmission via the conjunctival route was not supported by the data. Good clinical protection can effectively cut off the transmission path.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Yunyun Zhou; Yuyang Zeng; Yongqing Tong; ChangZheng Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20021956&#39;},
 {&#39;cord_uid&#39;: &#39;w4zad1zt&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20022053&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An increasing number of cases of novel coronavirus pneumonia (NCP) infected with 2019-nCoV have been identified in Wuhan and other cities in China, since December 2019. We analyzed data on the 17 confirmed cases in Dazhou to provide the epidemiologic characteristics of NCP outside Wuhan. Among them, 12 patients were still quarantined in the hospital, 5 patients were discharged NCP patients according to the national standards. Compared with non-discharged NCP patients, the discharged NCP patients had younger ages. Moreover, discharged NCP patients had higher heart rate, lymphocytes levels and monocytes levels than non-discharged NCP patients on admission to the hospital. Notably, all of 17 patients had abnormal increased C-reactive protein levels, and 16 patients had abnormal computed tomography images. This study provided some information that younger age, higher lymphocytes levels and monocytes levels at the diagnoses of 2019-nCoV may contributed to faster recovery and better therapeutic outcome.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Jie Li; Shilin Li; Yurui Cai; Qin Liu; Xue Li; Zhaoping Zeng; Yanpeng Chu; Fangcheng Zhu; Fanxin Zeng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20022053&#39;},
 {&#39;cord_uid&#39;: &#39;1f5hn3ig&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#34;Primary Care Practitioners&#39; Response to 2019 Novel Coronavirus Outbreak in China&#34;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20022095&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;None&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Zhijie Xu; Yi Qian; Lizheng Fang; Mi Yao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20022095&#39;},
 {&#39;cord_uid&#39;: &#39;qbemfymp&#39;,
  &#39;sha&#39;: &#39;004f0f8bb66cf446678dc13cf2701feec4f36d76&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Healthcare-resource-adjusted vulnerabilities towards the 2019-nCoV epidemic across China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20022111&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We integrate the human movement and healthcare resource data to identify cities with high vulnerability towards the 2019-nCoV epidemic with respect to available health resources. The results inform public health responses in multiple ways.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Hanchu Zhou; Jianan Yang; Kaichen Tang; Qingpeng Zhang; zhidong cao; Dirk Pfeiffer; Daniel Dajun Zeng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20022111&#39;},
 {&#39;cord_uid&#39;: &#39;ssdx8xwi&#39;,
  &#39;sha&#39;: &#39;8bcbb998e7067bab377471601a3e72e50c2fe24f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Data-Based Analysis, Modelling and Forecasting of the COVID-19 outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20022186&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since the first suspected case of coronavirus disease-2019 (COVID-19) on December 1st, 2019, in Wuhan, Hubei Province, China, a total of 40,235 confirmed cases and 909 deaths have been reported in China up to February 10, 2020, evoking fear locally and internationally. Here, based on the publicly available epidemiological data for Hubei, China from January 11 to February 10, 2020, we provide estimates of the main epidemiological parameters. In particular, we provide an estimation of the case fatality and case recovery ratios, along with their 90% confidence intervals as the outbreak evolves. On the basis of a Susceptible-Infected-Recovered-Dead (SIDR) model, we provide estimations of the basic reproduction number (R0), and the per day infection mortality and recovery rates. By calibrating the parameters of the SIRD model to the reported data, we also attempt to forecast the evolution of the of the outbreak at the epicenter three weeks ahead, i.e. until February 29. As the number of infected individuals, especially of those with asymptomatic or mild courses, is suspected to be much higher than the official numbers, which can be considered only as a subset of the actual numbers of infected and recovered cases in the total population, we have repeated the calculations under a second scenario that considers twenty times the number of confirmed infected cases and forty times the number of recovered, leaving the number of deaths unchanged. Based on the reported data, the expected value of R0 as computed considering the period from the 11th of January until the 18th of January, using the official counts of confirmed cases was found to be ~4.6, while the one computed under the second scenario was found to be ~3.2. Thus, based on the SIRD simulations, the estimated average value of R0 was found to be ~2.6 based on confirmed cases and ~2 based on the second scenario. Our forecasting flashes a note of caution for the presently unfolding outbreak in China. Based on the official counts for confirmed cases, the simulations suggest that the cumulative number of infected could reach 180,000 (with lower bound of 45,000) by February 29. Regarding the number of deaths, simulations forecast that on the basis of the up to the 10th of February reported data, the death toll might exceed 2,700 (as a lower bound) by February 29. Our analysis further reveals a significant decline of the case fatality ratio from January 26 to which various factors may have contributed, such as the severe control measures taken in Hubei, China (e.g. quarantine and hospitalization of infected individuals), but mainly because of the fact that the actual cumulative numbers of infected and recovered cases in the population most likely are much higher than the reported ones. Thus, in a scenario where we have taken twenty times the confirmed number of infected and forty times the confirmed number of recovered cases, the case fatality ratio is around 0.15% in the total population. Importantly, based on this scenario, simulations suggest a slow down of the outbreak in Hubei at the end of February.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Cleo Anastassopoulou; Lucia Russo; Athanasios Tsakris; Constantinos Siettos&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20022186&#39;},
 {&#39;cord_uid&#39;: &#39;dbwfn27p&#39;,
  &#39;sha&#39;: &#39;fbc41a8e025cd6eb2d7c156aad3b6923af2349c4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCoV, in the nasal tissue&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20022228&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A novel coronavirus (2019-nCoV) was first identified in Wuhan, Hubei Province, and then spreads to the other Provinces of China. WHO decides to determine a Public Health Emergency of International Concern (PHEIC) of 2019-nCoV. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public single-cell RNA-Seq datasets, we analyzed the ACE2 RNA expression profile in the tissues at different locations of the respiratory tract. The result indicates that the ACE2 expression appears in nasal epithelial cells. We found that the size of this population of ACE2-expressing nasal epithelial cells is comparable with the size of the population of ACE2-expression type II alveolar cells (AT2) in the Asian sample reported by Yu Zhao et al. We further detected 2019-nCoV by polymerase chain reaction (PCR) from the nasal-swab and throat-swab of seven suspected cases. We found that 2019-nCoV tends to have a higher concentration in the nasal-swab comparing to the throat-swab, which could attribute to the ACE2-expressing nasal epithelial cells. We hope this study could be informative for virus-prevention strategy development, especially the treatment of nasal mucus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;CHAO WU; Shufa Zheng; Yu Chen; Min Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20022228&#39;},
 {&#39;cord_uid&#39;: &#39;tzdgc3dw&#39;,
  &#39;sha&#39;: &#39;a726e1844e86c7c17b8b4461dd4c29ad3094c792&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Lockdown may partially halt the spread of 2019 novel coronavirus in Hubei province, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.20022236&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We present a timely evaluation of the impact of lockdown on the 2019-nCov epidemic in Hubei province, China. The implementation appears to be effective in reducing about 60% of new infections and deaths, and its effect also appears to be sustainable even after its removal. Delaying its implementation reduces its effectiveness. However, the direct economic cost of such a lockdown remains to be seen and whether the model is replicable in other Chinese regions remains a matter of further investigation.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Mingwang Shen; Zhihang Peng; Yuming Guo; Yanni Xiao; Lei Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.20022236&#39;},
 {&#39;cord_uid&#39;: &#39;prqeufwg&#39;,
  &#39;sha&#39;: &#39;fc15f433048e4d9464964930c36a1e7057ba1088&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.944462&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-15&#39;,
  &#39;authors&#39;: &#39;Daniel Wrapp; Nianshuang Wang; Kizzmekia S. Corbett; Jory A. Goldsmith; Ching-Lin Hsieh; Olubukola Abiona; Barney S. Graham; Jason S. McLellan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.944462&#39;},
 {&#39;cord_uid&#39;: &#39;cv36vc8i&#39;,
  &#39;sha&#39;: &#39;4d1090e239baa1ea54af521f4b8bd8b5ab51fd35&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A Generalized Discrete Dynamic Model for Human Epidemics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.11.944728&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA discrete dynamic model for human epidemics was developed in present study. The model included major parameters as transmission strength and its decline parameters, mean incubation period, hospitalization time, non-hospitalization daily mortality, non-hospitalization daily recovery rate, and hospitalization proportion, etc. Sensitivity analysis of the model indicated the total cumulative cases significantly increased with initial transmission strength, hospitalization time. The total cumulative cases significantly decreased with transmission strength’s decline and hospitalization proportion, and linearly decreased with non-hospitalization daily mortality and non-hospitalization daily recovery rate. In a certain range, the total cumulative cases significantly increased with mean incubation period. Sensitivity analysis demonstrated that dynamic change of transmission strength is one of the most important and controllable factors. In addition, reducing the delay for hospitalization is much effective in weakening disease epidemic. Non-hospitalization recovery rate is of importance for enhancing immunity to recover from the disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-12&#39;,
  &#39;authors&#39;: &#39;Wenjun Zhang; Zeliang Chen; Yi Lu; Zhongmin Guo; Yanhong Qi; Guoling Wang; Jiahai Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.11.944728&#39;},
 {&#39;cord_uid&#39;: &#39;7t41wvw3&#39;,
  &#39;sha&#39;: &#39;d3c2e2839498c613ee95739dce7052109750362c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Long-Term Persistence of IgG Antibodies in SARS-CoV Infected Healthcare Workers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20021386&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND: The ongoing worldwide outbreak of the 2019-nCoV is markedly similar to the severe acute respiratory syndrome (SARS) outbreak 17 years ago. During the 2002-2003 SARS outbreak, healthcare workers formed a special population of patients. Although virus-specific IgG play important roles in virus neutralization and prevention against future infection, limited information is available regarding the long term persistence of IgG after infection with SARS-like coronavirus. METHODS: A long-term prospective cohort study followed 34 SARS-CoV-infected healthcare workers from a hospital with clustered infected cases during the 2002-2003 SARS outbreak in Guangzhou, China, with a 13-year follow-up. Serum samples were collected annually from 2003-2015. Twenty SARS-CoV-infected and 40 non-infected healthcare workers were enrolled in 2015, and their serum samples were collected. All sera were tested for IgG antibodies with ELISA using whole virus and a recombinant nucleocapsid protein of SARS-CoV, as a diagnostic antigen. RESULTS: Anti SARS-CoV IgG was found to persist for up to 12 years. IgG titers typically peaked in 2004, declining rapidly from 2004-2006, and then continued to decline at a slower rate. IgG titers in SARS-CoV-infected healthcare workers remained at a significantly high level until 2015. Patients treated with corticosteroids at the time of infection were found to have lower IgG titers than those without. CONCLUSIONS: IgG antibodies against SARS-CoV can persist for at least 12 years. The presence of SARS-CoV IgG might provide protection against SARS-CoV and other betacoronavirus. This study provides valuable information regarding humoral immune responses against SARS-CoV and the 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-14&#39;,
  &#39;authors&#39;: &#39;Xiaoqin Guo; Zhongmin Guo; Chaohui Duan; Zeliang chen; Guoling Wang; Yi Lu; Mengfeng Li; Jiahai Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20021386&#39;},
 {&#39;cord_uid&#39;: &#39;bhm2un6v&#39;,
  &#39;sha&#39;: &#39;d34a47b592e9c6e84b5b4115474349acbc0fb925&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Statistics based predictions of coronavirus 2019-nCoV spreading in mainland China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20021931&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background. The epidemic outbreak cased by coronavirus 2019-nCoV is of great interest to researches because of the high rate of spread of the infection and the significant number of fatalities. A detailed scientific analysis of the phenomenon is yet to come, but the public is already interested in the questions of the duration of the epidemic, the expected number of patients and deaths. For long time predictions, the complicated mathematical models are necessary which need many efforts for unknown parameters identification and calculations. In this article, some preliminary estimates will be presented. Objective. Since the reliable long time data are available only for mainland China, we will try to predict the epidemic characteristics only in this area. We will estimate some of the epidemic characteristics and present the most reliable dependences for victim numbers, infected and removed persons versus time. Methods. In this study we use the known SIR model for the dynamics of an epidemic, the known exact solution of the linear equations and statistical approach developed before for investigation of the children disease, which occurred in Chernivtsi (Ukraine) in 1988-1989. Results. The optimal values of the SIR model parameters were identified with the use of statistical approach. The numbers of infected, susceptible and removed persons versus time were predicted. Conclusions. Simple mathematical model was used to predict the characteristics of the epidemic caused by coronavirus 2019-nCoV in mainland China. The further research should focus on updating the predictions with the use of fresh data and using more complicated mathematical models.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Igor Nesteruk&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20021931&#39;},
 {&#39;cord_uid&#39;: &#39;lvjveaob&#39;,
  &#39;sha&#39;: &#39;5fb2cce255dd56b944ae52cec664bd7f4f222c66&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the daily trend in the size of the COVID-19 infected population in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022277&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;There has been an outbreak of coronavirus disease (COVID-19) in Wuhan city, Hubei province, China since December 2019. Cases have been exported to other parts of China and more than 20 countries. We provide estimates of the daily trend in the size of the epidemic in Wuhan based on detailed information of 10,940 confirmed cases outside Hubei province.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Qiushi Lin; Taojun Hu; Xiao-Hua Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022277&#39;},
 {&#39;cord_uid&#39;: &#39;j60j7e9b&#39;,
  &#39;sha&#39;: &#39;5585d267f9b505b068f9f72bff5f12c248472f0a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#34;Epidemic size of novel coronavirus-infected pneumonia in the Epicenter Wuhan: using data of five-countries&#39; evacuation action&#34;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022285&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Since late December 2019, novel coronavirus-infected pneumonia (NCP) emerged in Wuhan, Hubei province, China. Meanwhile, NCP rapidly spread from China to other countries, and several countries&#39; government rush to evacuate their citizens from Wuhan. We analyzed the infection rate of the evacuees and extrapolated the results in Wuhan&#39;s NCP incidence estimation. Methods: We collected the total number and confirmed cases of 2019-nCov infection in the evacuation of Korea, Japan, Germany, Singapore, and France and estimated the infection rate of the 2019 novel coronavirus (2019-nCov) among people who were evacuated from Wuhan with a meta-analysis. NCP incidence of Wuhan was indirectly estimated based on data of evacuation. Results: From Jan 29 to Feb 2, 2020, 1916 people have been evacuated from Wuhan, among them 17 have been confirmed 2019-nCov infected. The infection rate is estimated to be 1.1% (95% CI 0.4%-3.1%) using one group meta-analysis method with random effect model. We then estimated that almost 110,000 (95% CI: 40,000-310,000) people were infected with 2019-nCov in Wuhan around Feb 2, 2020, assuming the infection risk of evacuees is close to Chinese citizens in Wuhan. Conclusions: At the beginning of the outbreak, incidence of NCP may be vastly underestimated. Our result emphasizes that 2019-nCov has proposed a huge public health threats in Wuhan. We need to respond more rapidly, take large-scale public health interventions and draconian measures to limiting population mobility and control the epidemic.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Hongxin Zhao; Sailimai Man; Bo Wang; Yi Ning&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022285&#39;},
 {&#39;cord_uid&#39;: &#39;78qyuk5x&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical diagnosis of 8274 samples with 2019-novel coronavirus in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022327&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: 2019-Novel coronavirus (2019-nCoV) outbreaks create challenges for hospital laboratories because thousands of samples must be evaluated each day. Sample types, interpretation methods, and corresponding laboratory standards must be established. The possibility of other infections should be assessed to provide a basis for clinical classification, isolation, and treatment. Accordingly, in the present study, we evaluated the testing methods for 2019-nCoV and co-infections. Methods: We used a fluorescence-based quantitative PCR kit urgently distributed by the Chinese CDC to detect 8274 close contacts in the Wuhan region against two loci on the 2019-nCoV genome. We also analyzed 613 patients with fever who underwent multiple tests for 13 respiratory pathogens; 316 subjects were also tested for 2019-nCoV. Findings: Among the 8274 subjects, 2745 (33.2%) had 2019-nCoV infection; 5277 (63.8%) subjects showed negative results in the 2019-nCoV nucleic acid test (non-019-nCoV); and 252 cases (3.0%) because only one target was positive, the diagnosis was not definitive. Sixteen patients who originally had only one positive target were re-examined a few days later; 14 patients (87.5%) were finally defined as 2019-nCoV-positive, and 2 (12.5%) were finally defined as negative. The positive rates of nCoV-NP and nCovORF1ab were 34.7% and 34.7%, respectively. nCoV-NP-positive only and nCovORF1ab-positive cases accounted for 1.5% and 1.5%, respectively. In the 316 patients with multiple respiratory pathogens, 104 were positive for 2019-nCov and 6/104 had co-infection with coronavirus (3/104), influenza A virus (2/104), rhinovirus (2/104), and influenza A H3N2 (1/104); the remaining 212 patients had influenza A virus (11/202), influenza A H3N2 (11/202), rhinovirus (10/202), respiratory syncytial virus (7/202), influenza B virus (6/202), metapneumovirus (4/202), and coronavirus (2/202). Interpretation: Clinical testing methods for 2019-nCoV require improvement. Importantly, 5.8% of 2019-nCoV infected and 18.4% of non-2019-nCoV-infected patients had other pathogen infections. It is important to treat combined infections and perform rapid screening to avoid cross-contamination of patients. A test that quickly and simultaneously screens as many pathogens as possible is needed.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Ming Wang; Qing Wu; Wanzhou Xu; Bin Qiao; Jingwei Wang; Hongyun Zheng; Shupeng Jiang; Junchi Mei; Zegang Wu; Yayun Deng; Fangyuan Zhou; Wei Wu; Yan Zhang; Zhihua Lv; Jingtao Huang; Xiaoqian Guo; Lina Feng; Zunen Xia; Di Li; Zhiliang Xu; Tiangang Liu; Pingan Zhang; Yongqing Tong; Yan Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022327&#39;},
 {&#39;cord_uid&#39;: &#39;vv1wjen0&#39;,
  &#39;sha&#39;: &#39;409c431edf54d022463f40f2d31a0dfab42f5324&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#34;Can Search Query Forecast successfully in China&#39;s 2019-nCov pneumonia?&#34;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022400&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Recently the novel coronavirus (2019-nCov) pneumonia outbreak in China then the world, and the Number of infections and death continues to increases. Search Query performs well in forecasting the epidemics. It is still a question whether search engine data can forecast the drift and the inflexion in 2019-nCov pneumonia. Based on the Baidu Search Index, we propose three prediction models: composite Index, composite Index with filtering and suspected NCP(Novel Coronavirus Pneumonia). The result demonstrates that the predictive model of composite index with filtering performs the best while the model of suspected NCP has the highest forecast error. We further predict the out-of-the-set NCP confirmed cases and monitor that the next peak of new diagnoses will occur on February 16th and 17th.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Li Xiaoxuan; Wu Qi; Lv Benfu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022400&#39;},
 {&#39;cord_uid&#39;: &#39;s5w8rf6t&#39;,
  &#39;sha&#39;: &#39;c672f5b0d7c0a40f7cc1bdf079066d72eb8f64ec&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022418&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;In December 2019 and January 2020, novel coronavirus (2019-nCoV) - infected pneumonia (NCIP) occurred in Wuhan, and has already posed a serious threat to public health. ACE2 (Angiotensin Converting Enzyme 2) has been shown to be one of the major receptors that mediate the entry of 2019-nCoV into human cells, which also happens in severe acute respiratory syndrome coronavirus (SARS). Several researches have indicated that some patients have abnormal renal function or even kidney damage in addition to injury in respiratory system, and the related mechanism is unknown. This arouses our interest in whether coronavirus infection will affect the urinary and male reproductive systems. Here in this study, we used the online datasets to analyze ACE2 expression in different human organs. The results indicate that ACE2 highly expresses in renal tubular cells, Leydig cells and cells in seminiferous ducts in testis. Therefore, virus might directly bind to such ACE2 positive cells and damage the kidney and testicular tissue of patients. Our results indicate that renal function evaluation and special care should be performed in 2019-nCoV patients during clinical work, because of the kidney damage caused by virus and antiviral drugs with certain renal toxicity. In addition, due to the potential pathogenicity of the virus to testicular tissues, clinicians should pay attention to the risk of testicular lesions in patients during hospitalization and later clinical follow-up, especially the assessment and appropriate intervention in young patients&#39; fertility.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Caibin Fan; Kai Li; Yanhong Ding; Wei Lu Lu; Jianqing Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022418&#39;},
 {&#39;cord_uid&#39;: &#39;y7d3ux28&#39;,
  &#39;sha&#39;: &#39;11f96d05db6854ef95312aa3a4736724ce1f02d6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022426&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country. Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival &amp; traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% &amp; 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Samuel J Clifford; Carl A B Pearson; Petra Klepac; Kevin Van Zandvoort; Billy J Quilty; Rosalind M Eggo; Stefan Flasche&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022426&#39;},
 {&#39;cord_uid&#39;: &#39;kfsuiz5t&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Early epidemiological assessment of the transmission potential and virulence of coronavirus disease 2019 (COVID-19) in Wuhan City: China, January-February, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022434&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since the first cluster of cases was identified in Wuhan City, China, in December, 2019, coronavirus disease 2019 (COVID-19) has rapidly spread across China, causing multiple introductions in 109 countries/territories/areas as of March 10th. Despite the scarcity of publicly available data, scientists around the world have made strides in estimating the magnitude of the epidemic, the basic reproduction number, and transmission patterns. Recently more evidence suggests that a substantial fraction of the infected individuals with the novel coronavirus show little if any symptoms, which suggest the need to reassess the transmission potential of this emerging disease. In this study, we derive estimates of the transmissibility and virulence of COVID-19 in Wuhan City, China, by reconstructing the underlying transmission dynamics using multiple data sources. Methods: We employ statistical methods and publicly available epidemiological datasets to jointly derive estimates of transmissibility and severity associated with the novel coronavirus. For this purpose, the daily series of laboratory-confirmed COVID-19 cases and deaths in Wuhan City and epidemiological data of Japanese evacuees from Wuhan City on board government-chartered flights were integrated into our analysis. Results: Our posterior estimates of basic reproduction number (R) in Wuhan City, China in 2019-2020 reached values as high as 5.20 (95%CrI: 5.04-5.47) and the enhanced public health intervention after January 23rd in 2020 was associated with a declined R at 0.58 (95%CrI: 0.51-0.64), with the total number of infections (i.e. cumulative infections) estimated at 1905526 (95%CrI: 1350283-2655936) in Wuhan City, raising the proportion of infected individuals to 19.1% (95%CrI: 13.5-26.6%). We also found that most recent crude infection fatality ratio (IFR) and time-delay adjusted IFR is estimated to be 0.04% (95% CrI: 0.03-0.06%) and 0.12% (95%CrI: 0.08-0.17%), which is several orders of magnitude smaller than the crude CFR estimated at 4.19% Conclusions: We have estimated key epidemiological parameters of the transmissibility and virulence of COVID-19 in Wuhan, China during January-February, 2020 using an ecological modelling approach. The power of our approach lies in the ability to infer epidemiological parameters with quantified uncertainty from partial observations collected by surveillance systems.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Kenji Mizumoto; Katsushi Kagaya; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022434&#39;},
 {&#39;cord_uid&#39;: &#39;zvngy7zz&#39;,
  &#39;sha&#39;: &#39;90b5ecf991032f3918ad43b252e17d1171b4ea63&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The role of absolute humidity on transmission rates of the COVID-19 outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022467&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A novel coronavirus (COVID-19) was identified in Wuhan, Hubei Province, China, in December 2019 and has caused over 40,000 cases worldwide to date. Previous studies have supported an epidemiological hypothesis that cold and dry (low absolute humidity) environments facilitate the survival and spread of droplet-mediated viral diseases, and warm and humid (high absolute humidity) environments see attenuated viral transmission (i.e., influenza). However, the role of absolute humidity in transmission of COVID-19 has not yet been established. Here, we examine province-level variability of the basic reproductive numbers of COVID-19 across China and find that changes in weather alone (i.e., increase of temperature and humidity as spring and summer months arrive in the North Hemisphere) will not necessarily lead to declines in COVID-19 case counts without the implementation of extensive public health interventions.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Wei Luo; Maimuna S Majumder; Dianbo Liu; Canelle Poirier; Kenneth D Mandl; Marc Lipsitch; Mauricio Santillana&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022467&#39;},
 {&#39;cord_uid&#39;: &#39;x4qdiln9&#39;,
  &#39;sha&#39;: &#39;05d99c07db59b6948e39bfa62c2cbbf62944059a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A spatial model of CoVID-19 transmission in England and Wales: early spread and peak timing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.20022566&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: An outbreak of a novel coronavirus, named CoVID-19, was first reported in China on 31 December 2019. As of 9 February 2020, cases have been reported in 25 countries, including probable cases of human-to-human transmission in England. Methods: We adapted an existing national-scale metapopulation model to capture the spread of CoVID-19 in England and Wales. We used 2011 census data to capture population sizes and population movement, together with parameter estimates from the current outbreak in China. Results: We predict that a CoVID-19 outbreak will peak 126 to 147 days (~4 months) after the start of person-to-person transmission in England and Wales in the absence of controls, assuming biological parameters remain unchanged. Therefore, if person-to-person transmission persists from February, we predict the epidemic peak would occur in June. The starting location has minimal impact on peak timing, and model stochasticity varies peak timing by 10 days. Incorporating realistic parameter uncertainty leads to estimates of peak time ranging from 78 days to 241 days after person-to-person transmission has been established. Seasonal changes in transmission rate substantially impact the timing and size of the epidemic peak, as well as the total attack rate. Discussion: We provide initial estimates of the potential course of CoVID-19 in England and Wales in the absence of control measures. These results can be refined with improved estimates of epidemiological parameters, and permit investigation of control measures and cost effectiveness analyses. Seasonal changes in transmission rate could shift the timing of the peak into winter months, which will have important implications for healthcare capacity planning.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-14&#39;,
  &#39;authors&#39;: &#39;Leon Danon; Ellen Brooks-Pollock; Mick Bailey; Matt J Keeling&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.20022566&#39;},
 {&#39;cord_uid&#39;: &#39;anls8gri&#39;,
  &#39;sha&#39;: &#39;3579a2b0053bef55f93716e444173727a6c94c6d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.945576&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Pathogenic human coronavirus infections, such as severe acute respiratory syndrome CoV (SARS-CoV) and Middle East respiratory syndrome CoV (MERS-CoV), cause high morbidity and mortality 1,2. Recently, a severe pneumonia-associated respiratory syndrome caused by a new coronavirus was reported at December 2019 (2019-nCoV) in the city Wuhan, Hubei province, China3–5, which was also named as pneumonia-associated respiratory syndrome (PARS)6. Up to 9th of February 2020, at least 37, 251 cases have been reported with 812 fatal cases according to the report from China CDC. However, the immune mechanism that potential orchestrated acute mortality from patients of 2019-nCoV is still unknown. Here we show that after the 2019-nCoV infection, CD4+T lymphocytes are rapidly activated to become pathogenic T helper (Th) 1 cells and generate GM-CSF etc. The cytokines environment induces inflammatory CD14+CD16+ monocytes with high expression of IL-6 and accelerates the inflammation. These aberrant and excessive immune cells may enter the pulmonary circulation in huge numbers and play an immune damaging role to causing lung functional disability and quick mortality. Our results demonstrate that excessive non-effective host immune responses by pathogenic T cells and inflammatory monocytes may associate with severe lung pathology. Therefore, we suggest that monoclonal antibody that targets the GM-CSF or interleukin 6 receptor may potentially curb immunopathology caused by 2019-nCoV and consequently win more time for virus clearance.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Yonggang Zhou; Binqing Fu; Xiaohu Zheng; Dongsheng Wang; Changcheng Zhao; Yingjie qi; Rui Sun; Zhigang Tian; Xiaoling Xu; Haiming Wei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.945576&#39;},
 {&#39;cord_uid&#39;: &#39;8p1agcm2&#39;,
  &#39;sha&#39;: &#39;f6f1ec12291f743de1504003929ea654217f2af6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Candidate targets for immune responses to 2019-Novel Coronavirus (nCoV): sequence homology- and bioinformatic-based predictions&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.946087&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTEffective countermeasures against the recent emergence and rapid expansion of the 2019-Novel Coronavirus (2019-nCoV) require the development of data and tools to understand and monitor viral spread and immune responses. However, little information about the targets of immune responses to 2019-nCoV is available. We used the Immune Epitope Database and Analysis Resource (IEDB) resource to catalog available data related to other coronaviruses, including SARS-CoV, which has high sequence similarity to 2019-nCoV, and is the best-characterized coronavirus in terms of epitope responses. We identified multiple specific regions in 2019-nCoV that have high homology to SARS virus. Parallel bionformatic predictions identified a priori potential B and T cell epitopes for 2019-nCoV. The independent identification of the same regions using two approaches reflects the high probability that these regions are targets for immune recognition of 2019-nCoV.ONE SENTENCE SUMMARYWe identified potential targets for immune responses to 2019-nCoV and provide essential information for understanding human immune responses to this virus and evaluation of diagnostic and vaccine candidates.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Alba Grifoni; John Sidney; Yun Zhang; Richard H Scheuermann; Bjoern Peters; Alessandro Sette&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.946087&#39;},
 {&#39;cord_uid&#39;: &#39;er6lz09f&#39;,
  &#39;sha&#39;: &#39;4b56993a4d0f091e70ef5dec2e5a61c3043f752e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.12.946772&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMotivationAccurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm.ResultsWe trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours.AvailabilityThe repository of this project will be published.Contactdongsi@uw.eduSupplementary informationSupplementary data will be available at Bioinformatics online.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-13&#39;,
  &#39;authors&#39;: &#39;Jonas Pfab; Dong Si&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.12.946772&#39;},
 {&#39;cord_uid&#39;: &#39;z8ix4kz7&#39;,
  &#39;sha&#39;: &#39;38bab3f17b186bd8ee289e5d135bb7d500ef500b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Probabilistic reconstruction of measles transmission clusters from routinely collected surveillance data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20020891&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Pockets of susceptibility resulting from spatial or social heterogeneity in vaccine coverage can drive measles outbreaks, as cases imported into such pockets are likely to cause further transmission and lead to large transmission clusters. Characterising the dynamics of transmission is essential for identifying which individuals and regions might be most at risk. As data from detailed contact tracing investigations are not available in many settings, we combined age, location, genotype, and onset date of cases in order to probabilistically reconstruct the importation status and transmission clusters within a newly developed R package called o2geosocial. We compared our inferred cluster size distributions to 737 transmission clusters identified through detailed contact-tracing in the United States between 2001 and 2016. We were able to reconstruct the importation status of the cases and found good agreement between the inferred and reference clusters. The results were improved when the contact-tracing investigations were used to set the importation status before running the model. Spatial heterogeneity in vaccine coverage is difficult to measure directly. Our approach was able to highlight areas with potential for local transmission using a minimal number of variables and could be applied to assess the intensity of ongoing transmission in a region.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-15&#39;,
  &#39;authors&#39;: &#39;Alexis Robert; Adam J Kucharski; Paul A Gastañaduy; Prabasaj Paul; Sebastian Funk&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20020891&#39;},
 {&#39;cord_uid&#39;: &#39;lmoekp1m&#39;,
  &#39;sha&#39;: &#39;7ed2c0786eef3d39fa0435b461aa34ab454a4753&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Understanding the present status and forecasting of COVID―19 in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022251&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The present status of COVID―19 is analyzed and the end of the disease is forecasted. The peak of the epidemic is different in three regions, Wuhan, Hubei province except Wuhan, and mainland China except Hubei. In two regions except Wuhan, the peak of the epidemic passed ten days ago. If the trend until February 11 does not change, the disease may end by the end of February. In Wuhan, the epidemic reached a peak but the reported number of newly infected patients fluctuates largely. We need to know the reason for the big fluctuation to forecast the end of the disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-14&#39;,
  &#39;authors&#39;: &#39;Toshihisa Tomie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022251&#39;},
 {&#39;cord_uid&#39;: &#39;nqxo3f45&#39;,
  &#39;sha&#39;: &#39;8b71ce92ec03e066eca7792bfd17e2f210d50ce3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022673&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP). We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS. We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene. Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Jing-Wen Ai; Hao-Cheng Zhang; Teng Xu; Jing Wu; Mengqi Zhu; Yi-Qi Yu; Han-Yue Zhang; Zhongliang Shen; Yang Li; Xian Zhou; Guo-Qing Zang; Jie Xu; Wen-Jing Chen; Yong-Jun Li; De-Sheng Xie; Ming-Zhe Zhou; Jing-Ying Sun; Jia-Zhen Chen; Wen-Hong Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022673&#39;},
 {&#39;cord_uid&#39;: &#39;h0h0d93t&#39;,
  &#39;sha&#39;: &#39;adb3a6501ad731eda95d0a1a182a793a0dcd58b0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022707&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore. We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% - 64%) of Singapore′s capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 - 4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% - 67%), among intermediate surveillance locations it is 37% (95% HPDI 18% - 68%), and among low surveillance locations it is 11% (95% HPDI 0% - 42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers may be underestimated by several fold, and severity correspondingly overestimated by several fold. Undetected cases are likely in countries around the world, with greater risk in countries of low detection capacity and high connectivity to the epicenter of the outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-14&#39;,
  &#39;authors&#39;: &#39;Rene Niehus; Pablo M De Salazar; Aimee Taylor; Marc Lipsitch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022707&#39;},
 {&#39;cord_uid&#39;: &#39;60qmiwjm&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis of meteorological conditions and prediction of epidemic trend of 2019-nCoV infection in 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022715&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To investigate the meteorological condition for incidence and spread of 2019-nCoV infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease. Methods: The meteorological factors during the outbreak period of the novel coronavirus pneumonia in Wuhan in 2019 were collected and analyzed, and were confirmed with those of Severe Acute Respiratory Syndrome (SARS) in China in 2003. Data of patients infected with 2019-nCoV and SARS coronavirus were collected from WHO website and other public sources. Results: This study found that the suitable temperature range for 2019-nCoV coronavirus survival is (13-24 degree Celsius), among which 19 degree Celsius lasting about 60 days is conducive to the spread between the vector and humans; the humidity range is 50%-80%, of which about 75% humidity is conducive to the survival of the coronavirus; the suitable precipitation range is below 30 mm/ month. Cold air and continuous low temperature over one week are helpful for the elimination of the virus. The prediction results show that with the approach of spring, the temperature in north China gradually rises, and the coronavirus spreads to middle and high latitudes along the temperature line of 13-18 degree Celsius. The population of new coronavirus infections is concentrated in Beijing, Tianjin, Hebei, Jiangsu, Zhejiang, Shanghai and other urban agglomerations. Starting from May 2020, the Beijing-Tianjin-Hebei urban agglomeration, the Central China Zhengzhou-Wuhan urban agglomeration, the eastern Jiangsu-Zhejiang-Shanghai urban agglomeration, and the southern Pearl River Delta urban agglomeration are all under a high temperature above 24 degree Celsius, which is not conducive to the survival and reproduction of coronaviruses, so the epidemic is expected to end. Conclusions: A wide range of continuous warm and dry weather is conducive to the survival of 2019-nCoV. The coming of spring, in addition to the original Wuhan-Zhengzhou urban agglomeration in central China, means that the prevention and control measures in big cities located in mid-latitude should be strengthened, especially the monitoring of transportation hubs. The Pearl River Delta urban agglomeration is a concentrated area of population in south China, with a faster temperature rise than those in mid-high latitudes, and thus the prevention in this area should be prioritized. From a global perspective, cities with a mean temperature below 24 degree Celsius are all high-risk cities for 2019-nCoV transmission before June.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Jin Bu; Dong-Dong Peng; Hui Xiao; Qian Yue; Yan Han; Yu Lin; Gang Hu; Jing Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022715&#39;},
 {&#39;cord_uid&#39;: &#39;3p2dl8yf&#39;,
  &#39;sha&#39;: &#39;1218f278a4f8d83dac14b23c8f698062812ef9d5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Potential impact of seasonal forcing on a SARS-CoV-2 pandemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022806&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Richard A Neher; Robert Dyrdak; Valentin Druelle; Emma B Hodcroft; Jan Albert&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022806&#39;},
 {&#39;cord_uid&#39;: &#39;ibx89gqw&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the distribution of the incubation period of 2019 novel coronavirus (COVID-19) infection between travelers to Hubei, China and non-travelers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022822&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives: Amid the continuing spread of the novel coronavirus (COVID-19), the incubation period of COVID-19 should be regularly re-assessed as more information is available upon the increase in reported cases. The present work estimated the distribution of incubation periods of patients infected in and outside Hubei province of China. Methods: Clinical data were collected from the individual cases reported by the media as they were not fully available on the official pages of the Chinese health authorities. MLE was used to estimate the distributions of the incubation period. Results: It was found that the incubation period of patients with no travel history to Hubei was longer and more volatile. Conclusion: It is recommended that the duration of quarantine should be extended to at least 3 weeks.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Char Leung&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022822&#39;},
 {&#39;cord_uid&#39;: &#39;hlhfx3fm&#39;,
  &#39;sha&#39;: &#39;47b07518dc0f608e53198bb5f8a95caa1c164fe5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A simple laboratory parameter facilitates early identification of COVID-19 patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.20022830&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Qilin Li; Xiuli Ding; Geqing Xia; Zhi Geng; Fenghua Chen; Lin Wang; Zheng Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.20022830&#39;},
 {&#39;cord_uid&#39;: &#39;5f42du0b&#39;,
  &#39;sha&#39;: &#39;3319392ca8bd2b8f2e2d00bccdb80deaa8a77c2e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.13.945485&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Tommy Tsan-Yuk Lam; Marcus Ho-Hin Shum; Hua-Chen Zhu; Yi-Gang Tong; Xue-Bing Ni; Yun-Shi Liao; Wei Wei; William Yiu-Man Cheung; Wen-Juan Li; Lian-Feng Li; Gabriel M Leung; Edward C. Holmes; Yan-Ling Hu; Yi Guan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.13.945485&#39;},
 {&#39;cord_uid&#39;: &#39;rdgyej4p&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20021535&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background:A Novel pneumonia associated with the 2019 coronavirus infected pneumonia (NCIP) suddenly broke out in Wuhan, China in December 2019. 37287 confirmed cases and 813 death case in China (Until 8th/Feb/2019) have been reported in just fortnight. Although this risky pneumonia with high infection rates and high mortality rates need to be resolved immediately, major gaps in our knowledge of clinical characters of it were still not be established. The aim of this study is to summaries and analysis the clinical characteristics of 2019-nCoV pneumonia. Methods: Literature have been systematically performed a search on PubMed, Embase, Web of Science, GreyNet International, and The Cochrane Library from inception up to February 8, 2020. The Newcastle-Ottawa Scale was used to assess quality, and publication bias was analyzed by Egger test. In the single-arm meta-analysis, A fix-effects model was used to obtain a pooled incidence rate. We conducted subgroup analysis according to geographic region and research scale. Results: A total of nine studies including 356 patients were included in this study, the mean age was 52.4 years and 221 (62.1%) were male. The pooled incidences rate of symptoms as follows: pharyngalgia (12.2%, 95% CI: 0.087-0.167), diarrhea (9.2%, 95% CI: 0.062-0.133) and headache (8.9%, 95% CI: 0.063-0.125). Meanwhile, 5.7% (95% CI: 0.027-0.114) of patients were found without any symptoms although they were diagnosed by RT-PCR. In the terms of CT imaging examination, the most of patients showed bilateral mottling or ground-glass opacity, 8.6% (95% CI: 0.048-0.148) of patients with crazy-paving pattern, and 11.5% (95% CI: 0.064-0.197) of patients without obvious CT imaging presentations. The pooled incidence of mortality was 8.9% (95% CI: 0.062-0.126). Conclusions: To our knowledge, this is the first evidence-based medicine research to further elaborate the clinical characteristics of NCIP, which is beneficial to the next step of prevention and treatment.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Kai Qian; Yi Deng; Yonghang Tai; Jun Peng; Hao Peng; Lihong Jiang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20021535&#39;},
 {&#39;cord_uid&#39;: &#39;j1yqn9xg&#39;,
  &#39;sha&#39;: &#39;f4d76fb8161f4cee6987d7d412798e1276348858&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing the impact of reduced travel on exportation dynamics of novel coronavirus infection (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20022897&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The impact of the drastic reduction in travel volume within mainland China in January and February 2020 was quantified with respect to reports of novel coronavirus (COVID-19) infections outside China. Data on confirmed cases diagnosed outside China were analyzed using statistical models to estimate the impact of travel reduction on three epidemiological outcome measures: (i) the number of exported cases, (ii) the probability of a major epidemic, and (iii) the time delay to a major epidemic. From 28 January to 7 February 2020, we estimated that 226 exported cases (95% confidence interval: 86, 449) were prevented, corresponding to a 70.4% reduction in incidence compared to the counterfactual scenario. The reduced probability of a major epidemic ranged from 7% to 20% in Japan, which resulted in a median time delay to a major epidemic of two days. Depending on the scenario, the estimated delay may be less than one day. As the delay is small, the decision to control travel volume through restrictions on freedom of movement should be balanced between the resulting estimated epidemiological impact and predicted economic fallout.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Asami Anzai; Tetsuro Kobayashi; Natalie M. Linton; Ryo Kinoshita; Katsuma Hayashi; Ayako Suzuki; Yichi Yang; Sungmok Jung; Takeshi Miyama; Andrei R. Akhmetzhanov; Hiroshi Nishiura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20022897&#39;},
 {&#39;cord_uid&#39;: &#39;6f6t558e&#39;,
  &#39;sha&#39;: &#39;b1b9d81726e56c2781ce6b63ae5015cf2f314c6f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the Efficacy of Traffic Blockage and Quarantine for the Epidemic Caused by 2019-nCoV (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20022913&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since the 2019-nCoV (COVID-19) outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The quarantine and traffic blockage can alleviate the risk of the epidemic and the infections, henceforth evaluating the efficacy of such actions is essential to inform policy makers and raise the public awareness of the importance of self-isolation and quarantine. Method: We collected confirmed case data and the migration data, and introduced the quarantine factor and traffic blockage factor to the Flow-SEIR model. By varying the quarantine factor and traffic blockage factor, we simulated the change of the peak number and arrival time of infections, then the efficacy of these two intervation measures can be analyzed in our simulation. In our study, the self-protection at home is also included in quarantine. Results: In the simulated results, the quarantine and traffic blockage are effective for epidemic control. For Hubei province, the current quarantine factor is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine, and the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who had flowed out from Hubei. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, and the current traffic blockage factor is estimated to be 0.26. With the quarantine and traffic blockage factor increasing, the number of infections decrease dramatically. We also simulated the start dates of quarantine and traffic blockage at four time points, the simulated results show that the early of warning is also effective for epidemic containing. However, provincial level traffic blockage can only alleviate 21.06% - 22.38% of the peak number of infections. In general, the quarantine is much more effective than the traffic blockage control. Conclusion: Both of quarantine and traffic blockage are effective ways to control the spread of COVID-19. However, the eff icacy of quarantine is found to be much stronger than that of traffic blockage. Considering traffic blockage may also cause huge losses of economy, we propose to gradually deregulate the traffic blockage, and improve quarantine instead. Also, there might be a large number of asymptomatic carriers of COVID-19, the quarantine should be continued for a long time until the epidemic is totally under control.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Deqiang Li; Zhicheng Liu; Qinghe Liu; Zefei Gao; Junkai Zhu; Junyan Yang; Qiao Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20022913&#39;},
 {&#39;cord_uid&#39;: &#39;4ah705nc&#39;,
  &#39;sha&#39;: &#39;f4f4b045dce8ff5ff1f5aaa48a6bdbaaa7b05179&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A new transmission route for the propagation of the SARS-CoV-2 coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20022939&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Antoine Danchin; Tuen Wai Patrick Ng; Gabriel TURINICI&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20022939&#39;},
 {&#39;cord_uid&#39;: &#39;u2uswgy3&#39;,
  &#39;sha&#39;: &#39;080660f20f078c10524f6186bca263327094acbb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20023028&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19&#39;s graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control. Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation. Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Shuai Wang; Bo Kang; Jinlu Ma; Xianjun Zeng; Mingming Xiao; Jia Guo; Mengjiao Cai; Jingyi Yang; Yaodong Li; Xiangfei Meng; Bo Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20023028&#39;},
 {&#39;cord_uid&#39;: &#39;rgzmpoxv&#39;,
  &#39;sha&#39;: &#39;f7891683eecd12b3908862fc7ef1d341ca7a293c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Efficacy of Contact Tracing for the Containment of the 2019 Novel Coronavirus (COVID-19).&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20023036&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Contact tracing is a central public health response to infectious disease outbreaks, especially in the early stages of an outbreak when specific treatments are limited. Importation of novel Coronavirus (COVID-19) from China and elsewhere into the United Kingdom highlights the need to understand the impact of contact tracing as a control measure. Using detailed survey information on social encounters coupled to predictive models, we investigate the likely efficacy of the current UK definition of a close contact (within 2 meters for 15 minutes or more) and the distribution of secondary cases that may go untraced. Taking recent estimates for COVID-19 transmission, we show that less than 1 in 5 cases will generate any subsequent untraced cases, although this comes at a high logistical burden with an average of 36.1 individuals (95th percentiles 0-182) traced per case. Changes to the definition of a close contact can reduce this burden, but with increased risk of untraced cases; we estimate that any definition where close contact requires more than 4 hours of contact is likely to lead to uncontrolled spread.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Matt J Keeling; T. Deirdre Hollingsworth; Jonathan M Read&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20023036&#39;},
 {&#39;cord_uid&#39;: &#39;3nipr212&#39;,
  &#39;sha&#39;: &#39;0eda8331214ca028350b07e2953702f3078a105e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20023127&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Estimation of the fraction and contagiousness of undocumented novel coronavirus (COVID-19) infections is critical for understanding the overall prevalence and pandemic potential of this disease. Many mild infections are typically not reported and, depending on their contagiousness, may support stealth transmission and the spread of documented infection. Methods Here we use observations of reported infection and spread within China in conjunction with mobility data, a networked dynamic metapopulation model and Bayesian inference, to infer critical epidemiological characteristics associated with the emerging coronavirus, including the fraction of undocumented infections and their contagiousness. Results We estimate 86% of all infections were undocumented (95% CI: [82%-90%]) prior to the Wuhan travel shutdown (January 23, 2020). Per person, these undocumented infections were 52% as contagious as documented infections ([44%-69%]) and were the source of infection for two-thirds of documented cases. Our estimate of the reproductive number (2.23; [1.77-3.00]) aligns with earlier findings; however, after travel restrictions and control measures were imposed this number falls considerably. Conclusions A majority of COVID-19 infections were undocumented prior to implementation of control measures on January 23, and these undocumented infections substantially contributed to virus transmission. These findings explain the rapid geographic spread of COVID-19 and indicate containment of this virus will be particularly challenging. Our findings also indicate that heightened awareness of the outbreak, increased use of personal protective measures, and travel restriction have been associated with reductions of the overall force of infection; however, it is unclear whether this reduction will be sufficient to stem the virus spread.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;Ruiyun Li; Sen Pei; Bin Chen; Yimeng Song; Tao Zhang; Wan Yang; Jeffrey Shaman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20023127&#39;},
 {&#39;cord_uid&#39;: &#39;rcbw54xc&#39;,
  &#39;sha&#39;: &#39;57e01ad2a4961cd5cc6a3733f5f8c013a8946f3c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A model simulation study on effects of intervention measures in Wuhan COVID-19 epidemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.14.20023168&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: In the beginning of January 2020, new unknown virus pneumonia cases started to emerge in local hospitals in Wuhan, China. This virus epidemic quickly became a public health emergency of international concern by the WHO. Enormous amount of medical supplies as well as healthcare personals from other provinces were mobilized to support Wuhan. This current work tent to help people understanding how infectious disease spread and the purpose and consequences of various efforts based on simulation model. Method: a simulation model was created using known parameters. R0 set to 3 and mean incubation time to be 7.5days. the epidemic was divided to 3 periods. Simulation would run 50 times to mimic different patient0 status. Personal activity index was used to mimic different level of control measures. 141427709 simulated patients were created. Cumulation number of patients at the end of period 1 (day50) is 2868.7 ± 1739.0. Total infected patients could be 913396.5 ± 559099.9 by the end of period 2 (day70) in free transmission state. And at day90, total patients number is 913396.5 ± 559099.9. Conclusion: COVID-19 is a novel severe respiratory disease. This will put great burden on the shoulder of healthcare workers as well as on medical hardware and supplements. Current strict control measures help to contain disease from spreading. An early detecting, reporting and fast reacting system needs to be setup to prevent future unknown infectious disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Guopeng ZHOU; Chunhua CHI&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.14.20023168&#39;},
 {&#39;cord_uid&#39;: &#39;y9sel25v&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimate number of individuals infected with the 2019-novel coronavirus in South Korea due to the influx of international students from countries with virus risk: a simulation study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.15.20023234&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: In March 2020, overall, 37,000 international students from the country at risk of the 2019-novel coronavirus (COVID-19) infection will arrive in Seoul, South Korea. Individuals from the country at risk of COVID-19 infection have been included in a home-quarantine program, but the efficacy of the program is uncertain. Methods: To estimate the possible number of infected individuals within the large influx of international students, we used a deterministic compartmental model for epidemic and perform a simulation-based search of different rates of compliance with home-quarantine. Results: Under the home-quarantine program, the total number of the infected individuals would reach 24-53 from March 17-March 20, 50-86 from March 18-March 16, and 234-343 from March 4-March 23 with the arrival of 0.1%, 0.2%, and 1% of pre-infectious individuals, in Seoul, South Korea, respectively. Our findings indicated when incoming international students showed strict compliance with quarantine, epidemics were less likely to occur in Seoul, South Korea. Conclusion: To mitigate possible epidemics, additional efforts to improve the compliance of home-quarantine are warranted along with other containment policies.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Sukhyun Ryu; Sheikh Taslim Ali; Jun-sik Lim; Byung Chul Chun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.15.20023234&#39;},
 {&#39;cord_uid&#39;: &#39;1rq612hm&#39;,
  &#39;sha&#39;: &#39;91098f6fe46a21565bf0cb06fe960cdb2c3f5e38&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The role of institutional trust in preventive and treatment-seeking behaviors during the 2019 novel coronavirus (2019-nCoV) outbreak among residents in Hubei, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.15.20023333&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Since December 2019, pneumonia associated with the 2019 novel coronavirus (2019-nCoV) has emerged in Wuhan, China. The exponential increase of the confirmed number of cases of 2019n-CoV is of great concern to the global community. The fears and panic among residents in the epicenters have prompted diverse responses, which are understudied. During such a crisis, community trust and support for the government and health authorities are important to contain the outbreak. We aimed to investigate the influence of institutional trust on public responses to the 2019-nCoV outbreak. Methods An anonymous Internet-based, cross-sectional survey was administered on January 29, 2020. The study population comprised all residents currently residing or working in the province of Hubei, where Wuhan is the capital city. The level of trust in information provision and preventive instructions, individual preventive behaviors and treatment-seeking behaviors were queried. Findings The majority of the participants expressed a great extent of trust in the information and preventive instructions provided by the central government than by the local government. A high uptake of 2019-nCoV preventive measures was found, particularly among people who had been placed under quarantine. Being under quarantine (adjusted odds ratio [OR] = 2.35, 95% confidence interval [CI] 1.80 to 3.08) and having a high institutional trust score (OR = 2.23, 95% CI 1.96 to 2.53) were both strong and significant determinants of higher preventive behavior scores. The majority of study participants (85.7%, n = 3,640) reported that they would seek hospital treatment if they suspected themselves to have been infected with 2019 n-CoV. Few of the participants from Wuhan (16.6%, n = 475) and those participants who were under quarantine (13.8%, n = 550) expressed an unwillingness to seek hospital treatment. Similarly, being under quarantine (OR = 2.36, 95% CI 1.80 to 3.09) and having a high institutional trust score (OR = 2.20, 95% CI 1.96 to 2.49) were two strong significant determinants of hospital treatment-seeking. Interpretation The results of this study suggest that institutional trust is an important factor influencing adequate preventive behavior and seeking formal medical care during an outbreak. In view of the 2019-nCoV being highly pathogenic and extremely contagious, our findings also underscore the importance of public health intervention to reach individuals with poor adherence to preventive measures and who are reluctant to seek treatment at formal health services. Funding National Key R&amp;D Program of China, Ningbo Health Branding Subject Fund, Sanming Project of Medicine in Shenzhen, K.C. Wong Magna Fund in Ningbo University, National Natural Science Foundation of China, Fundamental Research Funds for the Central Universities, China Postdoctoral Science Foundation, and Natural Science Basic Research Program of Shanxi Province. Keywords: 2019-nCoV; institutional trust; preventive behaviors&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Li Ping Wong; Qun hong Wu; Xi Chen; Zhuo Chen; Haridah Alias; Mingwang Shen; Jing cen Hu; Shiwei Duan; Jin jie Zhang; Liyuan Han&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.15.20023333&#39;},
 {&#39;cord_uid&#39;: &#39;ehm4fgos&#39;,
  &#39;sha&#39;: &#39;317943dec06b88c1c62ef0ecd832f0d644b1d417&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.15.20023440&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Can Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.15.20023440&#39;},
 {&#39;cord_uid&#39;: &#39;3r0nqzgu&#39;,
  &#39;sha&#39;: &#39;73998a3f565b661816d48aa7c20fa5ae617864d5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Profiling ACE2 expression in colon tissue of healthy adults and colorectal cancer patients by single-cell transcriptome analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.15.20023457&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A newly identified novel coronavirus (2019-nCoV) has caused numerous acute respiratory syndrome cases in Wuhan China from December 2019 to Feb 2020. Its fast spreading to other provinces in China and overseas is very likely causing a pandemic. Since the novel coronavirus has been reported to be capable of endangering thousands of lives, it is extremely important to find out how the coronavirus is transmitted in human organs. Apart from fever and respiratory complications, gastrointestinal symptoms are observed in some patients with 2019-nCoV but the significance remains undetermined. The cell receptor angiotensin covering enzyme II (ACE2), which is the major receptor of SARS-nCoV, has been reported to be a cellular entry receptor of 2019-nCoV as well. Here, to more precisely explore the potential pathogen transmission route of the 2019-nCoV infections in the gastrointestinal tract, we analyzed the ACE2 RNA expression profile in the colon tissue of healthy adults and colorectal cancer patients of our cohort and other databases. The data indicates that ACE2 is mainly expressed in epithelial cells of the colon. The expression of ACE2 is gradually increased from healthy control, adenoma to colorectal cancer patients in our cohort as well as in the external Asian datasets. According to the expression profile of ACE2 in colon epithelial cells, we speculate adenoma and colorectal cancer patients are more likely to be infected with 2019-nCoV than healthy people. Our data may provide a theoretical basis for the classification and management of future 2019-nCoV susceptibility people in clinical application.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Haoyan Chen; Baoqin Xuan; Yuqing Yan; Xiaoqiang Zhu; Chaoqin Shen; Gang Zhao; Linhua Ji; Danhua Xu; Hua Xiong; TaChung Yu; Xiaobo Li; Qiang Liu; Yingxuan Chen; Yun Cui; Jie Hong; Jing-Yuan Fang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.15.20023457&#39;},
 {&#39;cord_uid&#39;: &#39;b4kzgubs&#39;,
  &#39;sha&#39;: &#39;f9c6a656d0ce352cc1fa0df8d83e6d4a1f53d7c2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the Case Fatality Risk of COVID-19 using Cases from Outside China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.15.20023499&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;There is large uncertainty around the case fatality risk (CFR) for COVID-19 in China. Therefore, we considered symptomatic cases outside of China (countries/settings with 20+ cases) and the proportion who are in intensive care units (4.0%, 14/349 on 13 February 2020). Given what is known about CFRs for ICU patients with severe respiratory conditions from a meta-analysis, we estimated a CFR of 1.37% (95%CI: 0.57% to 3.22%) for COVID-19 cases outside of China.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Nick Wilson; Amanda Kvalsvig; Lucy Telfar Barnard; Michael Baker&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.15.20023499&#39;},
 {&#39;cord_uid&#39;: &#39;9aegg5sd&#39;,
  &#39;sha&#39;: &#39;1df9ce60e358f6cef0be3f6e15fe2a9c6f93f39c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Multivariate Analyses of Codon Usage of SARS-CoV-2 and other betacoronaviruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.15.950568&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus which also includes severe acute respiratory syndrome related coronavirus (SARSr-CoV) and Middle East respiratory syndrome related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses can help identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, global correspondence analysis (CA), within-group correspondence analysis (WCA) and between-group correspondence analysis (BCA) were performed among different genes in coronavirus viral sequences. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Haogao Gu; Daniel Chu; Malik Peiris; Leo L.M. Poon&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.15.950568&#39;},
 {&#39;cord_uid&#39;: &#39;m87tapjp&#39;,
  &#39;sha&#39;: &#39;10138816c67a562b55bc9c2b882b62c81fc4b062&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemic analysis of COVID-19 in China by dynamical modeling&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023465&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of the novel coronavirus (2019-nCoV) epidemic has attracted world- wide attention. Herein, we propose a mathematical model to analyzes this epidemic, based on a dynamic mechanism that incorporating the intrinsic impact of hidden la- tent and infectious cases on the entire process of transmission. Meanwhile, this model is validated by data correlation analysis, predicting the recent public data, and back- tracking, as well as sensitivity analysis. The dynamical model reveals the impact of various measures on the key parameters of the epidemic. According to the public data of NHCs from 01/20 to 02/09, we predict the epidemic peak and possible end time for 5 different regions. The epidemic in Beijing and Shanghai, Mainland/Hubei and Hubei/Wuhan, are expected to end before the end of February, and before mid- March respectively. The model indicates that, the outbreak in Wuhan is predicted to be ended in the early April. As a result, more effective policies and more efforts on clinical research are demanded. Moreover, through the backtracking simulation, we infer that the outbreak of the epidemic in Mainland/Hubei, Hubei/Wuhan, and Wuhan can be dated back to the end of December 2019 or the beginning of January 2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Liangrong Peng; Wuyue Yang; Dongyan Zhang; Changjing Zhuge; Liu Hong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023465&#39;},
 {&#39;cord_uid&#39;: &#39;9j5yda3k&#39;,
  &#39;sha&#39;: &#39;b117016a887f05386acc4387dab5bf8c769cf90e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023564&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Xinmiao Fu; Qi Ying; Tieyong Zeng; Tao Long; Yan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023564&#39;},
 {&#39;cord_uid&#39;: &#39;ndnkhj42&#39;,
  &#39;sha&#39;: &#39;8bd8158e4a47c65b12a462b49b4b8b4af1488dc3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of the final size of the coronavirus epidemic by the logistic model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023606&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In the note, the logistic growth regression model is used for the estimation of the final size and its peak time of the coronavirus epidemic. Based on available data estimation the final size will be about 90 000 cases and the peak time was on 10 Feb 2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;milan batista&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023606&#39;},
 {&#39;cord_uid&#39;: &#39;oknxic4q&#39;,
  &#39;sha&#39;: &#39;d069dfb7f0aefcdc2c890a1bbe773ebd26b01a55&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Utilize State Transition Matrix Model to Predict the Novel Corona Virus Infection Peak and Patient Distribution&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023614&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Since December 2019, a pneumonia caused by the 2019 novel coronavirus (2019-nCoV) has broken out in Wuhan, Hubei province, China. The continuous rising of infected cases has imposed overwhelming pressure on public health decision and medical resource allocation in China. We managed to forecast the infection peak time in Hubei province and the severe and critical case distribution. Methods: We used data resource according to cases reported by the National Health Commission of the People&#39;s Republic of China (Jan 25, 2019, to Feb 28, 2020) as the training set to deduce the arrival of the peak infection time and the number of severe and critical cases in Wuhan on subsequent days. Medical observation, discharge, infected, non-Severe, infected and severe, cure and death data were collected and analyzed. Using this state transition matrix model, we will be able predict when the inflection peak time (the maximum open infection cases) in Hubei Province will occur. Also, we can use this model to predict the patient distribution (severe, non-severe) to better allocate medical resource. Under relative pessimistic scenario, the inflection peak time is April 6-April 14. The numbers of critically ill and critically ill patients will lie between 8300-9800 and 2200-2700, respectively. Results: In very optimistic scenarios (daily NCC decay rate of -10%), the peak time of open inflection cases will arrive around February 23-February 26. At the same time, there will be a peak in the numbers of severely ill and critically ill patients, between 6800-7200 and 1800-2000, respectively. In a relative optimistic scenario (daily NCC decay rate of -5%), the inflection case peak time will arrive around February 28-March 2. The numbers of critically ill and critically ill patients will lie between 7100-7800 and 1900-2200, respectively. In a relatively pessimistic scenario (daily NCC decay rate of -1%), the inflection peak time does not arrive around the end of March. Estimated time is April 6-April 14. The numbers of critically ill and critically ill patients will lie between 8300-9800 and 2200-2700, respectively. We are using the diagnosis rate, mortality rate, cure rate as the 2/8 data. There should be room for improvement, if these metrics continue to improve. In that case, the peak time will arrive earlier than our estimation. Also, the severe and critical case ratios are likely to decline as the virus becomes less toxic and medical conditions improve. If that happens, the peak numbers will be lower than predicted above. Conclusion: We can infer that we are still not close to the end of this outbreak and the number of critically ill patients is still climbing. Assisting critical care resources in Hubei province requires the government to consider further tilt, and it is vital to make reasonable management of doctors and medical assistance systems to curb the transmission trend.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Ke Wu; Junhua Zheng; Jian Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023614&#39;},
 {&#39;cord_uid&#39;: &#39;c800ynvc&#39;,
  &#39;sha&#39;: &#39;3da1a03cb675f463b0d8e30744aa33512241dd29&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023655&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Pengpeng Shi; Shengli Cao; Peihua Feng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023655&#39;},
 {&#39;cord_uid&#39;: &#39;9xmeyrtu&#39;,
  &#39;sha&#39;: &#39;7325363d061f0f4699e29bfa544438e14e7c207d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023671&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The dynamic changes of lymphocyte subsets and cytokines profiles of patients with novel coronavirus disease (COVID-19) and their correlation with the disease severity remain unclear. Method: Peripheral blood samples were longitudinally collected from 40 confirmed COVID-19 patients and examined for lymphocyte subsets by flow cytometry and cytokine profiles by specific immunoassays. Findings: Of the 40 COVID-19 patients enrolled, 13 severe cases showed significant and sustained decreases in lymphocyte counts but increases in neutrophil counts than 27 mild cases. Further analysis demonstrated significant decreases in the counts of T cells, especially CD8 + T cells, as well as increases in IL-6, IL-10, IL-2 and IFN-γ levels in the peripheral blood in the severe cases compared to those in the mild cases. T cell counts and cytokine levels in severe COVID-19 patients who survived the disease gradually recovered at later time points to levels that were comparable to those of the mild cases. Moreover, the neutrophil-to-CD8+ T cell ratio (N8R) were identified as the most powerful prognostic factor affecting the prognosis for severe COVID-19. Conclusion: The degree of lymphopenia and a proinflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity. N8R may serve as a useful prognostic factor for early identification of severe COVID-19 cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Jing Liu; Sumeng Li; Jia Liu; Boyun Liang; Xiaobei Wang; Hua Wang; Wei Li; Qiaoxia Tong; Jianhua Yi; Lei Zhao; Lijuan Xiong; Chunxia Guo; Jin Tian; Jinzhuo Luo; Jinghong Yao; Ran Pang; Hui Shen; Cheng Peng; Ting Liu; Qian Zhang; Jun Wu; Ling Xu; Sihong Lu; Baoju Wang; Zhihong Weng; Chunrong Han; Huabing Zhu; Ruxia Zhou; Helong Zhou; Xiliu Chen; Pian Ye; Bin Zhu; Shengsong He; Yongwen He; Shenghua Jie; Ping Wei; Jianao Zhang; Yinping Lu; Weixian Wang; Li Zhang; Ling Li; Fengqin Zhou; Jun Wang; Ulf Dittmer; Mengji Lu; Yu Hu; Dongliang Yang; Xin Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023671&#39;},
 {&#39;cord_uid&#39;: &#39;fugb778l&#39;,
  &#39;sha&#39;: &#39;6445688d3f59cb7e02f2a8b28450cdf118a8a373&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Contacts in context: large-scale setting-specific social mixing matrices from the BBC Pandemic project&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023754&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Social mixing patterns are crucial in driving transmission of infectious diseases and informing public health interventions to contain their spread. Age-specific social mixing is often inferred from surveys of self-recorded contacts which by design often have a very limited number of participants. In addition, such surveys are rare, so public health interventions are often evaluated by considering only one such study. Here we report detailed population contact patterns for United Kingdom based self-reported contact data from over 36,000 volunteers that participated in the massive citizen science project BBC Pandemic. The amount of data collected allows us generate fine-scale age-specific population contact matrices by context (home, work, school, other) and type (conversational or physical) of contact that took place. These matrices are highly relevant for informing prevention and control of new outbreaks, and evaluating strategies that reduce the amount of mixing in the population (such as school closures, social distancing, or working from home). In addition, they finally provide the possibility to use multiple sources of social mixing data to evaluate the uncertainty that stems from social mixing when designing public health interventions.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Petra Klepac; Adam J Kucharski; Andrew JK Conlan; Stephen Kissler; Maria Tang; Hannah Fry; Julia R Gog&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023754&#39;},
 {&#39;cord_uid&#39;: &#39;asyrijyg&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Study on SARS-COV-2 transmission and the effects of control measures in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023770&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Bo Zhang; Hongwei Zhou; Fang Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023770&#39;},
 {&#39;cord_uid&#39;: &#39;32vs98ya&#39;,
  &#39;sha&#39;: &#39;e62e453735dc2022f4d81aa9b22769de30d0bb90&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;When will the battle against novel coronavirus end in Wuhan: a SEIR modeling analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023804&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Recent outbreak of 2019-nCoV in Wuhan raised serious public health concerns. By February 15, 2020 in Wuhan, the total number of confirmed infection cases has reached 37,914, and the number of deaths has reached 1123, accounting for 56.9% of the total confirmed cases and 73.7% of the total deaths in China. People are eager to know when the epidemic will be completely controlled and when people&#39;s work and life will be on the right track. In this study we analyzed the epidemic dynamics and trend of 2019-nCoV in Wuhan by using the data after the closure of Wuhan city till February 12, 2020 based on the SEIR modeling method. The optimal parameters were estimated as R0=1.44 (interquartile range: 1.40-1.47)，TI=14 (interquartile range: 14-14) and TE=3.0 (interquartile range: 2.8-3.1). Based on these parameters, the number of infected individuals in Wuhan city may reach the peak around February 19 at about 45,000 people. Once entering March, the epidemic would gradually decline, and end around the late March. It is worth noting that the above prediction is based on the assumption that the number of susceptible population N = 200,000 will not increase. If the epidemic situation is not properly controlled, the peak of infected number can be further increased and the peak time will be a little postponed. It was expected that the epidemic would subside in early March, and disappear gradually towards the late March.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;lianglu zhang; kangkang wan; jing chen; changming lu; lanlan dong; zhicheng wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023804&#39;},
 {&#39;cord_uid&#39;: &#39;jljjqs6m&#39;,
  &#39;sha&#39;: &#39;33bd85ea9ca61c4d0674c43ae5b901b271b46f1e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Fractal kinetics of COVID-19 pandemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023820&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We give an update to the original paper posted on 2/17/20 -- now (as of 3/1/20) the China deaths are rapidly decreasing, and we find an exponential decline to the power law similar to the that predicted by the network model of \\citet{vazquez_polynomial_2006}. At the same time, we see non-China deaths increasing rapidly, and similar to the early behavior of the China statistics. Thus, we see three stages of the spread of the disease in terms of number of deaths: exponential growth, power-law behavior, and then exponential decline in the daily rate. (Original abstract) The novel coronavirus (COVID-19) continues to grow rapidly in China and is spreading in other parts of the world. The classic epidemiological approach in studying this growth is to quantify a reproduction number and infection time, and this is the approach followed by many studies on the epidemiology of this disease. However, this assumption leads to exponential growth, and while the growth rate is high, it is not following exponential behavior. One approach that is being used is to simply keep adjusting the reproduction number to match the dynamics. Other approaches use rate equations such as the SEIR and logistical models. Here we show that the current growth closely follows power-law kinetics, indicative of an underlying fractal or small-world network of connections between susceptible and infected individuals. Positive deviations from this growth law might indicate either a failure of the current containment efforts while negative deviations might indicate the beginnings of the end of the pandemic. We cannot predict the ultimate extent of the pandemic but can get an estimate of the growth of the disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Anna L. Ziff; Robert M. Ziff&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023820&#39;},
 {&#39;cord_uid&#39;: &#39;z5i8hyja&#39;,
  &#39;sha&#39;: &#39;fbebe4b66073c44cface2e842754bce26e3e2913&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Risk map of the novel coronavirus (2019-nCoV) in China: proportionate control is needed&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023838&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background China is running a national level antivirus campaign against the novel coronavirus (2019-nCoV). Strict control measures are being enforced in either the populated areas and remote regions. While the virus is closed to be under control, tremendous economic loss has been caused. Methods and findings We assessed the pandemic risk of 2019-nCoV for all cities/regions in China using the random forest algorithm, taking into account the effect of five factors: the accumulative and increased numbers of confirmed cases, total population, population density, and GDP. We defined four levels of the risk, corresponding to the four response levels to public health emergencies in China. The classification system has good consistency among cities in China, as the error rate of the confusion matrix is 1.58%. Conclusions The pandemic risk of 2019-nCoV is dramatically different among the 442 cities/regions. We recommend to adopt proportionate control policy according to the risk level to reduce unnecessary economic loss.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Xinhai Li; Xumao Zhao; Yingqiang Lou; Yuehua Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023838&#39;},
 {&#39;cord_uid&#39;: &#39;bsgqsdbx&#39;,
  &#39;sha&#39;: &#39;d4cfb1fc4fc53abbc1d4b0ca60276c6af6632c3c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.20023903&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Since late December, 2019, an outbreak of pneumonia cases caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, and continued to spread throughout China and across the globe. To date, few data on immunologic features of Coronavirus Disease 2019 (COVID-19) have been reported. Methods In this single-centre retrospective study, a total of 21 patients with pneumonia who were laboratory-confirmed to be infected with SARS-CoV-2 in Wuhan Tongji hospital were included from Dec 19, 2019 to Jan 27, 2020. The immunologic characteristics as well as their clinical, laboratory, radiological features were compared between 11 severe cases and 10 moderate cases. Results Of the 21 patients with COVID-19, only 4 (19%) had a history of exposure to the Huanan seafood market. 7 (33.3%) patients had underlying conditions. The average age of severe and moderate cases was 63.9 and 51.4 years, 10 (90.9%) severe cases and 7 (70.0%) moderate cases were male. Common clinical manifestations including fever (100%, 100%), cough (70%, 90%), fatigue (100%, 70%) and myalgia (50%, 30%) in severe cases and moderate cases. PaO2/FiO2 ratio was significantly lower in severe cases (122.9) than moderate cases (366.2). Lymphocyte counts were significantly lower in severe cases (7000 million/L) than moderate cases (11000 million/L). Alanine aminotransferase, lactate dehydrogenase levels, high-sensitivity C-reactive protein and ferritin were significantly higher in severe cases (41.4 U/L, 567.2 U/L, 135.2 mg/L and 1734.4 ug/L) than moderate cases (17.6 U/L, 234.4 U/L, 51.4 mg/L and 880.2 ug /L). IL-2R, TNF-α and IL-10 concentrations on admission were significantly higher in severe cases (1202.4 pg/mL, 10.9 pg/mL and 10.9 pg/mL) than moderate cases (441.7 pg/mL, 7.5 pg/mL and 6.6 pg/mL). Absolute number of total T lymphocytes, CD4+T cells and CD8+T cells decreased in nearly all the patients, and were significantly lower in severe cases (332.5, 185.6 and 124.3 million/L) than moderate cases (676.5, 359.2 and 272.0 million/L). The expressions of IFN-γ by CD4+T cells tended to be lower in severe cases (14.6%) than moderate cases (23.6%). Conclusion The SARS-CoV-2 infection may affect primarily T lymphocytes, particularly CD4+T cells, resulting in significant decrease in number as well as IFN-γ production, which may be associated with disease severity. Together with clinical characteristics, early immunologic indicators including diminished T lymphocytes and elevated cytokines may serve as potential markers for prognosis in COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Guang Chen; Di Wu; Wei Guo; Yong Cao; Da Huang; Hongwu Wang; Tao Wang; Xiaoyun Zhang; Huilong Chen; Haijing Yu; Xiaoping Zhang; Minxia Zhang; Shiji Wu; Jianxin Song; Tao Chen; Meifang Han; Shusheng Li; Xiaoping Luo; Jianping Zhao; Qin Ning&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.20023903&#39;},
 {&#39;cord_uid&#39;: &#39;tjxtifao&#39;,
  &#39;sha&#39;: &#39;56a81431a148344381a1da80c877885f3f3d6d59&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.946699&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAlternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-17&#39;,
  &#39;authors&#39;: &#39;V Vern Lee; Louise M. Judd; Aaron R. Jex; Kathryn E. Holt; Christopher J. Tonkin; Stuart A. Ralph&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.946699&#39;},
 {&#39;cord_uid&#39;: &#39;nhq0oq8y&#39;,
  &#39;sha&#39;: &#39;a388cca261e8af7936cf3b79c35f43442f65706b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.951723&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses’ similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Chunyun Sun; Long Chen; Ji Yang; Chunxia Luo; Yanjing Zhang; Jing Li; Jiahui Yang; Jie Zhang; Liangzhi Xie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.951723&#39;},
 {&#39;cord_uid&#39;: &#39;rymj6lkj&#39;,
  &#39;sha&#39;: &#39;22f3be8bbf6d792549b7a6088df7440e425cb34f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.16.951913&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe new coronavirus (2019-nCoV) outbreak from December 2019 in Wuhan, Hubei, China, has been declared a global public health emergency. Angiotensin I converting enzyme 2 (ACE2), is the host receptor by 2019-nCov to infect human cells. Although ACE2 is reported to be expressed in lung, liver, stomach, ileum, kidney and colon, its expressing levels are rather low, especially in the lung. 2019-nCoV may use co-receptors/auxiliary proteins as ACE2 partner to facilitate the virus entry. To identify the potential candidates, we explored the single cell gene expression atlas including 119 cell types of 13 human tissues and analyzed the single cell co-expression spectrum of 51 reported RNA virus receptors and 400 other membrane proteins. Consistent with other recent reports, we confirmed that ACE2 was mainly expressed in lung AT2, liver cholangiocyte, colon colonocytes, esophagus keratinocytes, ileum ECs, rectum ECs, stomach epithelial cells, and kidney proximal tubules. Intriguingly, we found that the candidate co-receptors, manifesting the most similar expression patterns with ACE2 across 13 human tissues, are all peptidases, including ANPEP, DPP4 and ENPEP. Among them, ANPEP and DPP4 are the known receptors for human CoVs, suggesting ENPEP as another potential receptor for human CoVs. We also conducted “CellPhoneDB” analysis to understand the cell crosstalk between CoV-targets and their surrounding cells across different tissues. We found that macrophages frequently communicate with the CoVs targets through chemokine and phagocytosis signaling, highlighting the importance of tissue macrophages in immune defense and immune pathogenesis.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Furong Qi; Shen Qian; Shuye Zhang; Zheng Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.16.951913&#39;},
 {&#39;cord_uid&#39;: &#39;x2bl5lcn&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the case fatality ratio of the COVID-19 epidemic in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20023630&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Corona Virus Disease 2019 (COVID-19) due to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan city and rapidly spread throughout China since late December 2019. Crude case fatality ratio (CFR) with dividing the number of known deaths by the number of confirmed cases does not represent the true CFR and might be off by orders of magnitude. We aim to provide a precise estimate of the CFR of COVID-19 using statistical models at the early stage of the epidemic. Methods: We extracted data from the daily released epidemic report published by the National Health Commission P. R. China from 20 Jan 2020, to 1 March 2020. Competing risk model was used to obtain the cumulative hazards for death, cure, and cure-death hazard ratio. Then the CFR was estimated based on the slope of the last piece in joinpoint regression model, which reflected the most recent trend of the epidemic. Results: As of 1 March 2020, totally 80,369 cases were diagnosed as COVID-19 in China. The CFR of COVID-19 were estimated to be 70.9% (95% CI: 66.8%-75.6%) during Jan 20-Feb 2, 20.2% (18.6%-22.1%) during Feb 3-14, 6.9% (6.4%-7.4%) during Feb 15-23, 1.5% (1.4%-1.6%) during Feb 24-March 1 in Hubei province, and 20.3% (17.0%-25.3%) during Jan 20-28, 1.9% (1.8%-2.1%) during Jan 29-Feb 12, 0.9% (0.8%-1.1%) during Feb 13-18, 0.4% (0.4%-0.5%) during Feb 19-March 1 in other areas of China, respectively. Conclusions: Based on analyses of public data, we found that the CFR in Hubei was much higher than that of other regions in China, over 3 times in all estimation. The CFR would follow a downwards trend based on our estimation from recently released data. Nevertheless, at early stage of outbreak, CFR estimates should be viewed cautiously because of limited data source on true onset and recovery time.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Xing Wang; Zihui Ma; Yi Ning; Chen Chen; Rujin Chen; Qiwen Chen; Heng Zhang; Chunming Li; Yan He; Tao Wang; Cheng Tong; Junqing Wu; Yuyan Li; Handong Ma; Shaodian Zhang; Hongxin Zhao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20023630&#39;},
 {&#39;cord_uid&#39;: &#39;7odpslba&#39;,
  &#39;sha&#39;: &#39;fe685aa676e739bd52ba2585a7e5b27c55e2d0d6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evidence for gastrointestinal infection of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20023721&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The new coronavirus (SARS-CoV-2) outbreak originating from Wuhan, China, poses a threat to global health. While it&#39;s evident that the virus invades respiratory tract and transmits from human to human through airway, other viral tropisms and transmission routes remain unknown. We tested viral RNA in stool from 73 SARS-CoV-2-infected hospitalized patients using rRT-PCR. 53.42% of the patients tested positive in stool. 23.29% of the patients remained positive in feces even after the viral RNA decreased to undetectable level in respiratory tract. The viral RNA was also detected in gastrointestinal tissues. Furthermore, gastric, duodenal and rectal epithelia showed positive immunofluorescent staining of viral host receptor ACE2 and viral nucleocapsid protein in a case of SARS-CoV-2 infection. Our results provide evidence for gastrointestinal infection of SARS-CoV-2, highlighting its potential fecal-oral transmission route.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Fei Xiao; Meiwen Tang; Xiaobin Zheng; Chunna Li; Jianzhong He; Zhongsi Hong; Siwen Huang; Zhenyi Zhang; Xianqi Lin; Zhaoxiong Fang; Renxu Lai; Shoudeng Chen; Jing Liu; Jin Huang; Jinyu Xia; Zhonghe Li; Guanmin Jiang; Ye Liu; Xiaofeng Li; Hong Shan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20023721&#39;},
 {&#39;cord_uid&#39;: &#39;q6pz6l2b&#39;,
  &#39;sha&#39;: &#39;0cad40f50e41144e8b64744251515eca8cfac5f7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The reproductive number R0 of COVID-19 based on estimate of a statistical time delay dynamical system&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20023747&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;In this paper, we estimate the reproductive number R0 of COVID-19 based on Wallinga and Lipsitch framework {11} and a novel statistical time delay dynamic system. We use the observed data reported in CCDC&#39;s paper to estimate distribution of the generation interval of the infection and apply the simulation results from the time delay dynamic system as well as released data from CCDC to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate r of COVID-19 is almost in [0.30, 0.32] which is larger than the growth rate 0.1 estimated by CCDC {9}, and the reproductive number R0 of COVID-19 is estimated by 3.25≤ R0 ≤3.4 if we simply use R=1+r*Tc with Tc=7.5, which is bigger than that of SARS. Some evolutions and predictions are listed.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Nian Shao; Jin Cheng; Wenbin Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20023747&#39;},
 {&#39;cord_uid&#39;: &#39;nav88l73&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 in a Designated Infectious Diseases HospitalOutside Hubei Province,China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20024018&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background A new type of novel coronavirus infection (COVID-19) occurred in Wuhan, Hubei Province. Previous investigations reported patients in Wuhan city often progressed into severe or critical and had a high mortality rate.The clinical characteristics of affected patients outside the epicenter of Hubei province are less well understood. Methods All confirmed COVID-19 case treated in the Third People&#39;s Hospital of Shenzhen,from January 11, 2020 to February 6, 2020, were included in this study. We analyzed the epidemiological and clinical features of these cases to better inform patient management in normal hospital settings. Results Among the 298 confirmed cases, 233(81.5%) had been to Hubei while 42(14%) had not clear epidemiological history. Only 192(64%) cases presented with fever as initial symptom. The lymphocyte count decreased in 38% patients after admission. The number (percent) of cases classified as non-severe and severe was 240(80.6%) and 58(19.4%) respectively. Thirty-two patients (10.7%) needed ICU care. Compared to the non-severe cases, severe cases were associated with older age, underlying diseases, as well as higher levels of CRP, IL-6 and ESR. The median (IRQ) duration of positive viral test were 14(10-19). Slower clearance of virus was associated with higher risk of progression to severe clinical condition. As of February 14, 2020, 66(22.1%) patients were discharged and the overall mortality rate remains 0. Conclusions In a designated hospital outside the Hubei Province, COVID-19 patients were mainly characterized by mild symptoms and could be effectively manage by properly using the existing hospital system.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Qingxian Cai; Deliang Huang; Pengcheng Ou; Hong Yu; Zhibin Zhu; Zhang Xia; Yinan Su; Zhenghua Ma; Yiming Zhang; Zhiwei Li; Qing He; Yang Fu; Lei Liu; Jun Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20024018&#39;},
 {&#39;cord_uid&#39;: &#39;s12vwz8m&#39;,
  &#39;sha&#39;: &#39;dac1b1607ae72b9509ab26367e0d55016e8132a8&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemic Situation of Novel Coronavirus Pneumonia in China mainland&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20024034&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;[Objective] Analyze the occurrence of novel coronavirus pneumonia(NCP) in China mainland, explore the epidemiological rules, and evaluate the effect of prevention and control. [Methods] From December 1, 2019 to February 14, 2020, Analysis of 66,492 confirmed cases of NCP in China mainland. [Results] From December 1, 2019 to February 14, 2020, a total of 66,492 cases of NCP were confirmed in China mainland, a total of 54,406 cases were confirmed in Hubei Province, a total of 37,914 cases were confirmed in Wuhan city. On February 5, 2020, the number of suspected cases of NCP in China mainland reached a maximum of 5,328. Since then, the suspected cases have shown a significant downward trend. On February 4, 2020, the number of confirmed cases has gradually decreased in China mainland since then. On February 12, 2020, the number of confirmed cases of NCP in China mainland increased explosively to 15,152, and then began to decline. From February 3, 2020, except for Hubei Province, In China mainland, the number of newly confirmed cases of NCP has continued to decline; From December 1, 2019 to February 14, 2020, a total of 1,523 cases of NCP deaths in China mainland, a cumulative cure of 8096 cases, and mortality and cure rate were 2.29% (1523/66492) and 12.18% (8096/66492) respectively; Starting from January 27, 2020, The spread index of NCP gradually declined, and the extinction index of NCP rose little by little from January 29, 2020. [Conclusion] Starting from February 5, 2020, the number of suspected cases of the NCP is gradually decrease in China mainland. From February 3, 2020, the number of newly confirmed cases of NCP is continuous decline in China mainland except Hubei Province, This shows that the control of the transmission of NCP has achieved. Judged from the number of confirmed cases of NCP, controlling the outbreak in China mainland is to control the epidemic in Hubei province, and the key to controlling the outbreak in Hubei province is to control the epidemic in Wuhan city; Judging from the gradual increase in the number and cure rate of the NCP, and the gradual decline in the number of deaths and mortality, the trend of outbreak control and treatment is getting better; A decrease of the NCP spread index indicates a slowdown in the spread of the virus, but the cumulative confirmed cases is still increase, so the epidemic will continue for some time. However, the turning point of the epidemic in mainland China has not yet occurred. On February 12, 2020, the turning point of the epidemic in China mainland except Hubei province, has turned up.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Liu youbin; Li bao hong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20024034&#39;},
 {&#39;cord_uid&#39;: &#39;nnkholfe&#39;,
  &#39;sha&#39;: &#39;541e7d329fa466fe7b961263666386dddec75894&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A quantitative framework to define the end of an outbreak: application to Ebola Virus Disease&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20024042&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Declaring the end of an outbreak is an important step in controlling infectious disease outbreaks. An objective estimation of the probability of cases arising in the future is important to reduce the risk of post-declaration flare-ups. We developed a simulation-based model to quantify that probability. We tested it on simulated Ebola Virus Disease (EVD) data and found this probability was most sensitive to the instantaneous reproduction number, the reporting rate, and the delay between symptom onset and recovery or death of the last detected case. For EVD, our results suggest that the current WHO criterion of 42 days since the outcome of the last detected case is too short and very sensitive to underreporting. The 90 days of enhanced surveillance period after the end-of-outbreak declaration is therefore crucial to capture potential flare-ups of cases. Hence, we suggest a shift to a preliminary end-of-outbreak declaration after 63 days from the symptom onset day of the last detected case. This should be followed by a 90-day enhanced surveillance, after which the official end-of-outbreak can be declared. This corresponds to less than 5% probability of flare ups in most of the scenarios examined. Our quantitative framework could be adapted to define end-of-outbreak criteria for other infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Bimandra A Djaafara; Natsuko Imai; Esther Hamblion; Benido Impouma; Christl A Donnelly; Anne Cori&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20024042&#39;},
 {&#39;cord_uid&#39;: &#39;2w3bx6p8&#39;,
  &#39;sha&#39;: &#39;a6d2875c8b70a41ea815511417546a0689816964&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20024075&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Haoyang Sun; Borame Lee Dickens; Mark Chen; Alex Richard Cook; Hannah Eleanor Clapham&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20024075&#39;},
 {&#39;cord_uid&#39;: &#39;mlkxcn5k&#39;,
  &#39;sha&#39;: &#39;f0e75c4697317cbd0e3a5cdd722fe0526595db64&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical features and progression of acute respiratory distress syndrome in coronavirus disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20024166&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) results in a cluster of coronavirus disease 2019 (COVID-19). We reported the clinical characteristics of COVID-19 patients with acute respiratory distress syndrome (ARDS), and further investigated the treatment and progression of ARDS in COVID-19. Methods: This study enrolled 109 patients with COVID-19 admitted to the Central Hospital of Wuhan, a designated hospital in Wuhan, China, from January 2 to February 1, 2020. Patients were followed up to February 12, 2020. The clinical data were collected from the electronic medical records. The differences in the treatment and progression with the time and the severity of ARDS were determined. Findings: Among 109 patients, mean age was 55 years, and 59 patients were male. With a median 15 days (range, 4 to 30 days) follow-up period, 31 patients (28.4%) died, while 78 (71.6%) survived and discharged. Of all patients, 53 (48.6%) developed ARDS. Compared to non-ARDS patients, ARDS patients were elder (mean age, 61 years vs. 49 years), and more likely to have the coexistent conditions, including diabetes (20.8% vs. 1.8%), cerebrovascular disease (11.3% vs. 0%), and chronic kidney disease (15.1% vs. 3.6%). Compared to mild ARDS patients, those with moderate and severe ARDS had higher mortality rates. No significant effect of antivirus, glucocorticoid, or immunoglobulin treatment on survival was observed in patients with ARDS. Interpretation: The mortality rate increased with the severity of ARDS in COVID-19, and the effects of current therapies on the survival for these patients were not satisfactory, which needs more attention from clinicians. Funding: Health and Family Planning Commission of Wuhan Municipality.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Yanli Liu; Wenwu Sun; Jia Li; Liangkai Chen; Yujun Wang; Lijuan Zhang; Li Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20024166&#39;},
 {&#39;cord_uid&#39;: &#39;t2x4yfrv&#39;,
  &#39;sha&#39;: &#39;341fe593ea84b66223d50924bddbba252c37bc4e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Tracking and Predicting COVID-19 Epidemic in China Mainland&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.20024257&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;By proposing a varying coefficient Susceptible-Infected-Removal model (vSIR), we track the epidemic of COVID-19 in 30 provinces in China and 15 cities in Hubei province, the epicenter of the outbreak. It is found that the spread of COVID-19 has been significantly slowing down within the two weeks from January 27 to February 10th with 87.0% and 84.3% reductions in the reproduction number R0 among the 30 provinces and 15 Hubei cities, respectively. This suggests the extreme control measures implemented since January 23, which include cutting off Wuhan and many other cities and towns, a great public awareness and high level of self isolation at home, have contributed to a substantial decline in the reproductivity of the COVID-19 in China. We predict that Hubei province will reach its peak between February 20 and 22, 2020, and if the removal rate can be increased to 0.1, the epidemic outside Hubei province will end in May 2020, and inside Hubei in early June.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Haoxuan Sun; Yumou Qiu; Han Yan; Yaxuan Huang; Yuru Zhu; Song Xi Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.20024257&#39;},
 {&#39;cord_uid&#39;: &#39;t9f87kk0&#39;,
  &#39;sha&#39;: &#39;a8be6ba53508e988c2c8c153478ff615b0d77687&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Modulation of metabolic functions through Cas13d-mediated gene knockdown in liver&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.945014&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRNA knockdown in vivo carries significant potential for disease modelings and therapies. Despite the emerging approaches of CRISPR/Cas9-mediated permanent knock out of targeted genes, strategies targeting RNA for disruption are advantageous in the treatment of acquired metabolic disorders when permanent modification of the genome DNA is not appropriate, and RNA virus infection diseases when pathogenic DNA is not available (such as SARS-Cov-2 and MERS infections). Recently, Cas13d, a family of RNA-targeting CRISPR effectors, has been shown to accomplish robust down-regulation of cellular RNAs in mammalian cells in vitro. Among the various Cas13d subtypes, CasRx (RfxCas13d) showed the most potent RNA knockdown efficiency in HEK293T cells. However, the RNA-targeting activity of Cas13d still needs to be verified in vivo. In this study, the CasRx system was demonstrated to efficiently and functionally knock down genes related to metabolism functions, including Pten, Pcsk9 and lncLstr, in mouse hepatocytes. CasRx-mediated simultaneous knockdown of multiple genes was also achieved by sgRNA arrays, providing a useful strategy to modulate complex metabolism networks. Moreover, the AAV (adeno-associated virus)-mediated delivery of CasRx and Pcsk9 sgRNAs into mouse liver successfully decreased serum PCSK9, resulting in significant reduction of serum cholesterol levels. Importantly, CasRx-mediated knockdown of Pcsk9 is reversible and Pcsk9 could be repeatedly down-regulated, providing an effective strategy to reversibly modulate metabolic genes. The present work supplies a successful proof-of-concept trial that suggests efficient and regulatory knockdown of target metabolic genes for a designed metabolism modulation in the liver.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Bingbing He; Wenbo Peng; Jia Huang; Hang Zhang; Yingsi Zhou; Xiali Yang; Jing Liu; Zhijie Li; Chunlong Xu; Mingxing Xue; Hui Yang; Pengyu Huang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.945014&#39;},
 {&#39;cord_uid&#39;: &#39;mw0wb8a8&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.951335&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreak of 2019-nCoV in the central Chinese city of Wuhan at the end of 2019 poses unprecedent public health challenges to both China and the rest world1. The new coronavirus shares high sequence identity to SARS-CoV and a newly identified bat coronavirus2. While bats may be the reservoir host for various coronaviruses, whether 2019-nCoV has other hosts is still ambiguous. In this study, one coronavirus isolated from Malayan pangolins showed 100%, 98.2%, 96.7% and 90.4% amino acid identity with 2019-nCoV in the E, M, N and S genes, respectively. In particular, the receptor-binding domain of the S protein of the Pangolin-CoV is virtually identical to that of 2019-nCoV, with one amino acid difference. Comparison of available genomes suggests 2019-nCoV might have originated from the recombination of a Pangolin-CoV-like virus with a Bat-CoV-RaTG13-like virus. Infected pangolins showed clinical signs and histopathological changes, and the circulating antibodies reacted with the S protein of 2019-nCoV. The isolation of a coronavirus that is highly related to 2019-nCoV in the pangolins suggests that these animals have the potential to act as the intermediate host of 2019-nCoV. The newly identified coronavirus in the most-trafficked mammal could represent a continuous threat to public health if wildlife trade is not effectively controlled.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Kangpeng Xiao; Junqiong Zhai; Yaoyu Feng; Niu Zhou; Xu Zhang; Jie-Jian Zou; Na Li; Yaqiong Guo; Xiaobing Li; Xuejuan Shen; Zhipeng Zhang; Fanfan Shu; Wanyi Huang; Yu Li; Ziding Zhang; Rui-Ai Chen; Ya-Jiang Wu; Shi-Ming Peng; Mian Huang; Wei-Jun Xie; Qin-Hui Cai; Fang-Hui Hou; Yahong Liu; Wu Chen; Lihua Xiao; Yongyi Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.951335&#39;},
 {&#39;cord_uid&#39;: &#39;sty7myg4&#39;,
  &#39;sha&#39;: &#39;b705170c7527f5bc6842d90bf443fa9f9ee00bf9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structure of dimeric full-length human ACE2 in complex with B0AT1&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.951848&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAngiotensin-converting enzyme 2 (ACE2) is the surface receptor for SARS coronavirus (SARS-CoV), directly interacting with the spike glycoprotein (S protein). ACE2 is also suggested to be the receptor for the new coronavirus (2019-nCoV), which is causing a serious epidemic in China manifested with severe respiratory syndrome. B0AT1 (SLC6A19) is a neutral amino acid transporter whose surface expression in intestinal cells requires ACE2. Here we present the 2.9 Å resolution cryo-EM structure of full-length human ACE2 in complex with B0AT1. The complex, assembled as a dimer of ACE2-B0AT1 heterodimers, exhibits open and closed conformations due to the shifts of the peptidase domains (PDs) of ACE2. A newly resolved Collectrin-like domain (CLD) on ACE2 mediates homo-dimerization. Structural modelling suggests that the ACE2-B0AT1 complex can bind two S proteins simultaneously, providing important clues to the molecular basis for coronavirus recognition and infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-18&#39;,
  &#39;authors&#39;: &#39;Renhong Yan; Yuanyuan Zhang; Yaning Li; Lu Xia; Qiang Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.951848&#39;},
 {&#39;cord_uid&#39;: &#39;os758lsd&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.951939&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;A recombinant adenovirus vaccine against the SARS Coronavirus (SARS-CoV) was constructed, which contains fragments from the S, N, and Orf8 genes. Rhesus Macaques immunized with the recombinant adenovirus generated antigen-specific humoral and cellular response. Furthermore, the vaccine provided significant protection against subsequent live SARS-CoV challenge. In contrast, three out of four monkeys immunized with placebo suffered severe alveolar damage and pulmonary destruction.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Yiyou Chen; Qiang Wei; Ruobing Li; Hong Gao; Hua Zhu; Wei Deng; Linlin Bao; Wei Tong; Zhe Cong; Hong Jiang; Chuan Qin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.951939&#39;},
 {&#39;cord_uid&#39;: &#39;xhachzby&#39;,
  &#39;sha&#39;: &#39;cc5e36a8d22f6b1708b1c43764806b1722aa1754&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;X-ray Structure of Main Protease of the Novel Coronavirus SARS-CoV-2 Enables Design of α-Ketoamide Inhibitors&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.952879&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA novel coronavirus has been identified as the causative agent of a massive outbreak of atypical pneumonia originating at Wuhan, Hubei province, China. Involved in the formation of the coronavirus replication complex, the viral main protease (Mpro, also called 3CLpro) represents an attractive target for therapy. We determined the crystal structure of the unliganded Mpro at 1.75 Å resolution and used this structure to guide optimization of a series of alpha-ketoamide inhibitors. The main goal of the optimization efforts was improvement of the pharmacokinetic properties of the compounds. We further describe 1.95- and 2.20-Å crystal structures of the complex between the enzyme and the most potent alpha-ketoamide optimized this way. These structures will form the basis for further development of these compounds to antiviral drugs.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Linlin Zhang; Daizong Lin; Xinyuanyuan Sun; Katharina Rox; Rolf Hilgenfeld&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.952879&#39;},
 {&#39;cord_uid&#39;: &#39;p4q64ksa&#39;,
  &#39;sha&#39;: &#39;1b4ab28a9391d516e1753874c73d2d244104f825&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.952895&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells1. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar2, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-23. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)4-6, the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection7,8. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride9,10) or the PBM (SB2035805), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Intikhab alam; Allan Kamau; Maxat Kulmanov; Stefan T. Arold; Arnab Pain; Takashi Gojobori; Carlos M. Duarte&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.952895&#39;},
 {&#39;cord_uid&#39;: &#39;hicqy5sj&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Molecular mechanism of evolution and human infection with the novel coronavirus (2019-nCoV)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.17.952903&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince December, 2019, an outbreak of pneumonia caused by the new coronavirus (2019-nCoV) has hit the city of Wuhan in the Hubei Province. With the continuous development of the epidemic, it has become a national public health crisis and calls for urgent antiviral treatments or vaccines. The spike protein on the coronavirus envelope is critical for host cell infection and virus vitality. Previous studies showed that 2019-nCoV is highly homologous to human SARS-CoV and attaches host cells though the binding of the spike receptor binding domain (RBD) domain to the angiotensin-converting enzyme II (ACE2). However, the molecular mechanisms of 2019-nCoV binding to human ACE2 and evolution of 2019-nCoV remain unclear. In this study, we have extensively studied the RBD-ACE2 complex, spike protein, and free RBD systems of 2019-nCoV and SARS-CoV using protein-protein docking and molecular dynamics (MD) simulations. It was shown that the RBD-ACE2 binding free energy for 2019-nCoV is significantly lower than that for SARS-CoV, which is consistent with the fact that 2019-nCoV is much more infectious than SARS-CoV. In addition, the spike protein of 2019-nCoV shows a significantly lower free energy than that of SARS-CoV, suggesting that 2019-nCoV is more stable and able to survive a higher temperature than SARS-CoV. This may also provide insights into the evolution of 2019-nCoV because SARS-like coronaviruses are thought to have originated in bats that are known to have a higher body-temperature than humans. It was also revealed that the RBD of 2019-nCoV is much more flexible especially near the binding site and thus will have a higher entropy penalty upon binding ACE2, compared to the RBD of SARS-CoV. That means that 2019-nCoV will be much more temperature-sensitive in terms of human infection than SARS-CoV. With the rising temperature, 2019-nCoV is expected to decrease its infection ability much faster than SARS-CoV, and get controlled more easily. The present findings are expected to be helpful for the disease prevention and control as well as drug and vaccine development of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Jiahua He; Huanyu Tao; Yumeng Yan; Sheng-You Huang; Yi Xiao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.17.952903&#39;},
 {&#39;cord_uid&#39;: &#39;ic2h01gm&#39;,
  &#39;sha&#39;: &#39;6186828135d9e5e7e50f66567505d02e910b3ee2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Association between 2019-nCoV transmission and N95 respirator use&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20021881&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;2019-nCoV had caused pneumonia outbreak in Wuhan. Existing evidence have confirmed the human-to-human transmission of 2019-nCoV. We retrospectively collected infection data from 2 January to 22 January at six departments from Zhongnan Hospital of Wuhan University. In our study, we found N95 respirators, disinfection and hand washing can help to reduce the risk of 2019-nCoV infection in medical staffs. Our results call for re-emphasizing strict occupational protection code in battling this novel contagious disease. The risk of 2019-nCoV infection was higher in the open area than in the quarantined area. N95 may be more effective for 2019-nCoV infections.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Xinghuan Wang; Zhenyu Pan; Zhenshun Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20021881&#39;},
 {&#39;cord_uid&#39;: &#39;ahxnhutv&#39;,
  &#39;sha&#39;: &#39;db142fe07544dcf0e454f8a9f39452f0aa92a6ca&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Kidney impairment is associated with in-hospital death of COVID-19 patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20023242&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Information on kidney impairment in patients with coronavirus disease 2019 (COVID-19) is limited. This study aims to assess the prevalence and impact of abnormal urine analysis and kidney dysfunction in hospitalized COVID-19 patients in Wuhan. Method: We conducted a consecutive cohort study of COVID-19 patients admitted in a tertiary teaching hospital with 3 branches following a major outbreak in Wuhan in 2020. Hematuria, proteinuria, serum creatinine concentration and other clinical parameters were extracted from the electronic hospitalization databases and laboratory databases. Incidence rate for acute kidney injury (AKI) was examined during the study period. Association between kidney impairment and in-hospital death was analyzed. Results: We included 710 consecutive COVID19 patients, 89 (12.3%) of whom died in hospital. The median age of the patients was 63 years (inter quartile range, 51-71), including 374 men and 336 women. On admission, 44% of patients have proteinuria hematuria and 26.9% have hematuria, and the prevalence of elevated serum creatinine and blood urea nitrogen were 15.5% and 14.1% respectively. During the study period, AKI occurred in 3.2% patients. Kaplan-Meier analysis demonstrated that patients with kidney impairment have higher risk for in-hospital death. Cox proportional hazard regression confirmed that elevated serum creatinine, elevated urea nitrogen, AKI, proteinuria and hematuria was an independent risk factor for in-hospital death after adjusting for age, sex, disease severity, leukocyte count and lymphocyte count. Conclusion: The prevalence of kidney impairment (hematuria, proteinuria and kidney dysfunction) in hospitalized COVID-19 patients was high. After adjustment for confounders, kidney impairment indicators were associated with higher risk of in-hospital death. Clinicians should increase their awareness of kidney impairment in hospitalized COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Yichun Cheng; Ran Luo; Kun Wang; Meng Zhang; Zhixiang Wang; Lei Dong; Junhua Li; Ying Yao; Shuwang Ge; Gang Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20023242&#39;},
 {&#39;cord_uid&#39;: &#39;uo7fb10k&#39;,
  &#39;sha&#39;: &#39;b9548530d954b407b35fb266e2b4192402c982a1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Phase adjusted estimation of the number of 2019 novel coronavirus cases in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024281&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV / SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of Rt. In the first assumption, Rt was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with Rt = 1.9, 2.6 or 3.1. The number of infections would reach 11,044, 70,258 and 227,989, respectively, by 29 February 2020. In the second assumption, Rt was assumed to gradually decrease at different phases from high level of transmission (Rt = 3.1, 2.6 and 1.9) to below 1 (Rt = 0.9 or 0.5) owing to increasingly implemented public heath intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce Rt to an ideal level and control the infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Huwen Wang; Zezhou Wang; Yinqiao Dong; Ruijie Chang; Chen Xu; Xiaoyue Yu; Shuxian Zhang; Lhakpa Tsamlag; Meili Shang; Jinyan Huang; Ying Wang; Gang Xu; Tian Shen; Xinxin Zhang; Yong Cai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024281&#39;},
 {&#39;cord_uid&#39;: &#39;nzynerfu&#39;,
  &#39;sha&#39;: &#39;36521caf90f471c9da1a4e84f8562440d73ead9a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of the epidemic properties of the 2019 novel coronavirus: A mathematical modeling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024315&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The 2019 novel Coronavirus (COVID-19) emerged in Wuhan, China in December 2019 and has been spreading rapidly in China. Decisions about its pandemic threat and the appropriate level of public health response depend heavily on estimates of its basic reproduction number and assessments of interventions conducted in the early stages of the epidemic. Methods We conducted a mathematical modeling study using five independent methods to assess the basic reproduction number (R0) of COVID-19, using data on confirmed cases obtained from the China National Health Commission for the period 10th January to 8th February. We analyzed the data for the period before the closure of Wuhan city (10th January to 23rd January) and the post-closure period (23rd January to 8th February) and for the whole period, to assess both the epidemic risk of the virus and the effectiveness of the closure of Wuhan city on spread of COVID-19. Findings Before the closure of Wuhan city the basic reproduction number of COVID-19 was 4.38 (95% CI: 3.63-5.13), dropping to 3.41 (95% CI: 3.16-3.65) after the closure of Wuhan city. Over the entire epidemic period COVID-19 had a basic reproduction number of 3.39 (95% CI: 3.09-3.70), indicating it has a very high transmissibility. Interpretation COVID-19 is a highly transmissible virus with a very high risk of epidemic outbreak once it emerges in metropolitan areas. The closure of Wuhan city was effective in reducing the severity of the epidemic, but even after closure of the city and the subsequent expansion of that closure to other parts of Hubei the virus remained extremely infectious. Emergency planners in other cities should consider this high infectiousness when considering responses to this virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Jinghua Li; Yijing Wang; Stuart Gilmour; Mengying Wang; Daisuke Yoneoka; Ying Wang; Xinyi You; Jing Gu; Chun Hao; Liping Peng; Zhicheng Du; Dong Roman Xu; Yuantao Hao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024315&#39;},
 {&#39;cord_uid&#39;: &#39;yj25zkx6&#39;,
  &#39;sha&#39;: &#39;4609a2dd3568a7241548b7303e0d836e33d4d5ca&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Reduction and Functional Exhaustion of T Cells in Patients with Coronavirus Disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024364&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND The outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has posed great threat to human health, which has been declared a public health emergency of international concern (PHEIC) by the WHO. T cells play a critical role in antiviral immunity but their numbers and functional state in COVID-19 patients remain largely unclear. METHODS We retrospectively reviewed the counts of total T cells, CD4+, CD8+ T cell subsets, and serum cytokine concentration from inpatient data of 522 patients with laboratory-confirmed COVID-19, admitted into two hospitals in Wuhan from December 2019 to January 2020, and 40 healthy controls, who came to the hospitals for routine physical examination. In addition, the expression of T cell exhaustion markers PD-1 and Tim-3 were measured by flow cytometry in the peripheral blood of 14 COVID-19 cases. RESULTS The number of total T cells, CD4+ and CD8+ T cells were dramatically reduced in COVID-19 patients, especially among elderly patients (≥60 years of age) and in patients requiring Intensive Care Unit (ICU) care. Counts of total T cells, CD8+T cells or CD4+T cells lower than 800/μL, 300/μL, or 400/μL, respectively, are negatively correlated with patient survival. Statistical analysis demonstrated that T cell numbers are negatively correlated to serum IL-6, IL-10 and TNF-α concentration, with patients in decline period showing reduced IL-6, IL-10 and TNF-α concentrations and restored T cell counts. Finally, T cells from COVID-19 patients have significantly higher levels of the exhausted marker PD-1 as compared to health controls. Moreover, increasing PD-1 and Tim-3 expression on T cells could be seen as patients progressed from prodromal to overtly symptomatic stages, further indicative of T cell exhaustion. CONCLUSIONS T cell counts are reduced significantly in COVID-19 patients, and the surviving T cells appear functionally exhausted. Non-ICU patients, with total T cells, CD8+T cells CD4+T cells counts lower than 800/μL, 300/μL, and 400/μL, respectively, may still require aggressive intervention even in the immediate absence of more severe symptoms due to a high risk for further deterioration in condition.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Bo Diao; Chenhui Wang; Yingjun Tan; Xiewan Chen; Ying Liu; Lifeng Ning; Li Chen; Min Li; Yueping Liu; Gang Wang; Zilin Yuan; Zeqing Feng; Yuzhang Wu; Yongwen Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024364&#39;},
 {&#39;cord_uid&#39;: &#39;j0nm444m&#39;,
  &#39;sha&#39;: &#39;fc7a6b5d1852c5ecce2d20fd0d73d5f957ed7055&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effective containment explains sub-exponential growth in confirmed cases of recent COVID-19 outbreak in Mainland China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024414&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The recent outbreak of COVID-19 in Mainland China is characterized by a distinctive algebraic, sub-exponential increase of confirmed cases with time during the early phase of the epidemic, contrasting an initial exponential growth expected for an unconstrained outbreak with sufficiently large reproduction rate. Although case counts vary significantly between affected provinces in Mainland China, the scaling law t^μ is surprisingly universal, with a range of exponents μ = 2.1 ± 0.3. The universality of this behavior indicates that, in spite of social, regional, demographical, geographical, and socio-economical heterogeneities of affected Chinese provinces, this outbreak is dominated by fundamental mechanisms that are not captured by standard epidemiological models. We show that the observed scaling law is a direct consequence of containment policies that effectively deplete the susceptible population. To this end we introduce a parsimonious model that captures both, quarantine of symptomatic infected individuals as well as population wide isolation in response to mitigation policies or behavioral changes. For a wide range of parameters, the model reproduces the observed scaling law in confirmed cases and explains the observed exponents. Quantitative fits to empirical data permit the identification of peak times in the number of asymptomatic or oligo-symptomatic, unidentified infected individuals, as well as estimates of local variations in the basic reproduction number. The model implies that the observed scaling law in confirmed cases is a direct signature of effective contaiment strategies and/or systematic behavioral changes that affect a substantial fraction of the susceptible population. These insights may aid the implementation of containment strategies in potential export induced COVID-19 secondary outbreaks elsewhere or similar future outbreaks of other emergent infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Benjamin F Maier; Dirk Brockmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024414&#39;},
 {&#39;cord_uid&#39;: &#39;28ddm9um&#39;,
  &#39;sha&#39;: &#39;ba1c4c0de19352d0b9ed7db6199803b621f0a8f2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Psychological responses, behavioral changes and public perceptions during the early phase of the COVID-19 outbreak in China: a population based cross-sectional survey&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024448&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To investigate psychological and behavioral responses to the threat of SARS-CoV-2 infections and their associations with public perceptions in China Design: Cross sectional population-based telephone survey via random digital dialing between 1 and 10 February, 2020 Setting: Wuhan (the epicentre and quarantined city), and Shanghai (a typical major city with close transportation link with Wuhan) Participants: Random sample of 510 residents in Wuhan and 501 residents in Shanghai aged above 18 Main outcome measures: Anxiety (measured by the 7-item generalized anxiety disorder [GAD-7] scale), recommended and avoidance behaviors (engaged in all six behaviors such as increasing surface cleaning and reducing going out). Results: The prevalence rates of moderate or severe anxiety (score ≥10 on GAD-7) were 32.7% (n=167) among Wuhan participants and 20.4% (n=102) among Shanghai participants. 78.6% (n=401) of Wuhan participants and 63.9% (n=320) of Shanghai participants had carried out all six precautionary behaviors. For both measures, Wuhan participants were more responsive to the outbreak (p&lt;0.001). Controlling for personal characteristics, logistic regression results suggested that risks of moderate or severe anxiety were positively associated with perceived susceptibility (odds ratio 1.6, 95% confidence interval 1.3-1.8) and severity of the disease (1.6, 1.4-1.9) and confusion about information reliability (1.6, 1.5-1.9). Having confidence in taking measures to protect oneself against the disease was associated with a lower risk (0.6, 0.5-0.7). The strongest predictor of behavioral change was perceived severity (1.2, 1.1-1.4), followed by confusion about information reliability (1.1, 1.0-1.3). Conclusions: Psychological and behavioral responses to COVID-19 have been dramatic during the rising phase of the outbreak. Our results support efforts for timely dissemination of accurate and reliable information to address the high anxiety level.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Mengcen Qian; Qianhui Wu; Peng Wu; Zhiyuan Hou; Yuxia Liang; Benjamin J Cowling; Hongjie Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024448&#39;},
 {&#39;cord_uid&#39;: &#39;od8s0zhm&#39;,
  &#39;sha&#39;: &#39;537009c0181c3269f40753f75ddca4d17ae8abae&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the cure rate and case fatality rate of the ongoing epidemic COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024513&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The epidemic caused by the novel coronavirus COVID-19 in Wuhan at the end of 2019 has become an urgent public event of worldwide concern. However, due to the changing data of the epidemic, there is no scientific estimate of the cure rate and case fatality rate of the epidemic. This study proposes a method to estimate the cure rate and case fatality rate of COVID-19. The ratio of cumulative discharges on a given day to the sum of cumulative discharges on a given day and cumulative deaths before j days is used to estimate the cure rate. Moreover, the case fatality ratio can also be estimated. After simulation calculations, j is statistically appropriate when it is 8-10, and it is also clinically appropriate. When j is 9, based on the available data, it is inferred that the cure rate of this epidemic is about 93% and the case fatality rate is about 7%. This method of estimating the cure rate can be used to evaluate the effectiveness of treatment in different medical schemes and different regions, and has great value and significance for decision-making in the epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Ying Diao; Xiaoyun Liu; Tao Wang; Xiaofei Zeng; Chen Dong; Changlong Zhou; Yuanming Zhang; Xuan She; Dingfu Liu; Zhongli Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024513&#39;},
 {&#39;cord_uid&#39;: &#39;aoi4iqkf&#39;,
  &#39;sha&#39;: &#39;9bf1677e5f822110e736076a4ba6960a80f2f348&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 50466 patients with 2019-nCoV infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024539&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: We aim to summarize reliable evidences of evidence-based medicine for the treatment and prevention of the 2019 novel coronavirus (2019-nCoV) by analyzing all the published studies on the clinical characteristics of patients with 2019-nCoV. Methods: PubMed, Cochrane Library, Embase, and other databases were searched. Several studies on the clinical characteristics of 2019-nCoV infection were collected for Meta-analysis. Results: Ten studies were included in Meta-analysis, including a total number of 50466 patients with 2019-nCoV infection. Meta-analysis shows that, among these patients, the incidence of fever was 89.1%, the incidence of cough was 72.2%, and the incidence of muscle soreness or fatigue was 42.5%. The incidence of acute respiratory distress syndrome (ARDS) was 14.8%, the incidence of abnormal chest computer tomography (CT) was 96.6%, the percentage of severe cases in all infected cases was 18.1%, and the case fatality rate of patients with 2019-nCoV infection was 4.3%. Conclusion: Fever and cough are the most common symptoms in patients with 2019-nCoV infection, and most of these patients have abnormal chest CT examination. Several people have muscle soreness or fatigue as well as ARDS. Diarrhea, hemoptysis, headache, sore throat, shock, and other symptoms only occur in a small number of patients. The case fatality rate of patients with 2019-nCoV infection is lower than that of Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS).&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Pengfei Sun; Shuyan Qie; Zongjan Liu; Jizhen Ren; Jianing Jianing Xi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024539&#39;},
 {&#39;cord_uid&#39;: &#39;tc8eru1w&#39;,
  &#39;sha&#39;: &#39;672ca9b6e50e26c49fc7be2ef3b4f685badf9db1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modeling and Prediction of the 2019 Coronavirus Disease Spreading in China Incorporating Human Migration Data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024570&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This study integrates the daily intercity migration data with the classic Susceptible-Exposed-Infected-Removed (SEIR) model to construct a new model suitable for describing the dynamics of epidemic spreading of Coronavirus Disease 2019 (COVID-19) in China. Daily intercity migration data for 367 cities in China are collected from Baidu Migration, a mobile-app based human migration tracking data system. Historical data of infected, recovered and death cases from official source are used for model fitting. The set of model parameters obtained from best data fitting using a constrained nonlinear optimization procedure is used for estimation of the dynamics of epidemic spreading in the coming weeks. Our results show that the number of infections in most cities in China will peak between mid February to early March 2020, with about 0.8%, less than 0.1% and less than 0.01% of the population eventually infected in Wuhan, Hubei Province and the rest of China, respectively.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Choujun Zhan; Chi K. Tse; Yuxia Fu; Zhikang Lai; Haijun Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024570&#39;},
 {&#39;cord_uid&#39;: &#39;toizrp89&#39;,
  &#39;sha&#39;: &#39;a09c1ad0f05bbcc256cd667ad7411e6690134370&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Association of Population Migration and Coronavirus Disease 2019 Epidemic Control&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.20024661&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background and Objective To analyze the impact of different patterns of migration flow in two cities, Hefei and Shenzhen, on the epidemic and disease control of Coronavirus Disease 2019 (COVID-19), in order to provide insight for making differentiated controlling policies. Methods We collected demographic and epidemiological information of confirmed COVID-19 cases in Hefei and Shenzhen between January 19 and February 11, 2020, from data officially published by the provincial and municipal Centers for Disease Control and Prevention (CDC). From these data we calculated basic reproduction number R0 to reflect the rate of spread of COVID-19 in these cities. Aggregated data of population migration during the same period was extracted from Baidu Migration. The change of R0 in the two cites were analyzed and compared. Spearman correlation analysis between R0 and population inflow from epidemic focus were performed. Results A total of 157 confirmed cases was identified in Hefei by 24:00 February 11, 2020, with an average age of 44.4±15.6 years, 74 female (47.1%) and 386 confirmed cases were identified in Shenzhen, with an average age of 45.15±17.99 years, 202 female (52.3%). Significant difference in the proportion of imported cases between the two cities was observed (Hefei vs Shenzhen, 24.2% vs 74.9%, p=0.000). Before January 31 2020, during the initial stage of the Level 1 Response to Major Public Health Emergencies, there was no significant association observed in Shenzhen between R0 and the proportion of population inflow from the epidemic focus (P =0.260, r=-0.452); meanwhile in Hefei, such association was strong (P =0.000, r=1.0). However, after the initial stage of response, the situation reversed. A weak association was observed in Shenzhen between be R0 and the proportion of population inflow from the epidemic focus (P=0.073, r=0.536) but not in Hefei (P =0.498, r=0.217). Conclusion Following Level 1 Response, consistent decline of R0 of COVID-19 was observed in both Hefei and Shenzhen. Different patterns of disease spread were observed in the two cities, driven by different patterns of population migration. This indicated that population migration should be taken into consideration when we set controlling policy of a novel infectious disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Yu Ding; Sihui Luo; Xueying Zheng; Ping Ling; Tong Yue; Zhirong Liu; Jianping Weng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.20024661&#39;},
 {&#39;cord_uid&#39;: &#39;2inlyd0t&#39;,
  &#39;sha&#39;: &#39;cb3974f33a1d54369d630e62052d603148bb12e7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.954628&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.954628&#39;},
 {&#39;cord_uid&#39;: &#39;ukh84gvu&#39;,
  &#39;sha&#39;: &#39;c2af4db8591a8ef84229eb08359c0a50efa24544&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structure and immune recognition of the porcine epidemic diarrhea virus spike protein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.955195&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPorcine epidemic diarrhea virus is an alphacoronavirus responsible for significant morbidity and mortality in pigs. A key determinant of viral tropism and entry, the PEDV spike protein is a key target for the host antibody response and a good candidate for a protein-based vaccine immunogen. We used electron microscopy to evaluate the PEDV spike structure, as well as pig polyclonal antibody responses to viral infection. The structure of the PEDV spike reveals a configuration similar to that of HuCoV-NL63. Several PEDV protein-protein interfaces are mediated by non-protein components including a glycan at Asn264 and two bound palmitoleic acid molecules. The polyclonal antibody response to PEDV infection shows a dominance of epitopes in the S1 region. This structural and immune characterization provides new insights into coronavirus spike stability determinants and explores the immune landscape of viral spike proteins.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-19&#39;,
  &#39;authors&#39;: &#39;Robert N. Kirchdoerfer; Mahesh Bhandari; Olnita Martini; Leigh M. Sewall; Sandhya Bangaru; Kyoung-Jin Yoon; Andrew B. Ward&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.955195&#39;},
 {&#39;cord_uid&#39;: &#39;deu7lrxq&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;CasRx-mediated RNA targeting prevents choroidal neovascularization in a mouse model of age-related macular degeneration&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.18.955286&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe smallest Cas13 family protein, CasRx, has a high cleavage activity and targeting specificity, offering attractive opportunity for therapeutic applications. Here we report that delivery of CasRx by adeno-associated virus via intravitreal injection could efficiently knockdown Vegfa transcripts and significantly reduce the area of laser-induced choroidal neovascularization in a mouse model of age-related macular degeneration. Thus, RNA-targeting CRISPR system could be used for in vivo gene therapy.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Changyang Zhou; Xinde Hu; Cheng Tang; Wenjia Liu; Shaoran Wang; Yingsi Zhou; Qimeng Zhao; Qiyu Bo; Linyu Shi; Xiaodong Sun; Haibo Zhou; Hui Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.18.955286&#39;},
 {&#39;cord_uid&#39;: &#39;chhemr7s&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Early Phylogenetic Estimate Of The Effective Reproduction Number Of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20024851&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;To reconstruct the evolutionary dynamics of the 2019 novel coronavirus, 52 2019−nCOV genomes available on 04 February 2020 at GISAID were analysed. The two models used to estimate the reproduction number (coalescent−based exponential growth and a birth−death skyline method) indicated an estimated mean evolutionary rate of 7.8 x 10−4 subs/site/year (range 1.1x10−4−15x10−4). The estimated R value was 2.6 (range 2.1−5.1), and increased from 0.8 to 2.4 in December 2019. The estimated mean doubling time of the epidemic was between 3.6 and 4.1 days. This study proves the usefulness of phylogeny in supporting the surveillance of emerging new infections even as the epidemic is growing.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Alessia Lai; Annalisa Bergna; Carla Acciarri; Massimo Galli; Gianguglielmo Zehender&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20024851&#39;},
 {&#39;cord_uid&#39;: &#39;5ge7ozpd&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Comparative study of the lymphocyte change between COVID-19 and non-COVID-19 pneumonia cases suggesting uncontrolled inflammation might not be the main reason of tissue injury&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20024885&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The authors have withdrawn this manuscript because the statistical methods in our manuscript need to be modified and we are going to improve the statistical methods and try to give more precise model. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Yishan Zheng; Zhen Huang; Guoping Ying; Xia Zhang; Wei Ye; Zhiliang Hu; Chunmei Hu; Hongxia Wei; Yi Zeng; Yun Chi; Cong Cheng; Feishen Lin; Hu Lu; Lingyan Xiao; Yan Song; Chunming Wang; Yongxiang Yi; Lei Dong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20024885&#39;},
 {&#39;cord_uid&#39;: &#39;fmymklz6&#39;,
  &#39;sha&#39;: &#39;f3ff1ecae96700f41b83d2a034a3a959428388b0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The cross-sectional study of hospitalized coronavirus disease 2019 patients in Xiangyang, Hubei province&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025023&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Summary Objective To describe the epidemiological and clinical characteristics of the Coronavirus Disease 2019 (COVID-19) hospitalized patients and to offer suggestions to the urgent needs of COVID-19 prevention, diagnosis and treatment. Methods We included 102 confirmed COVID-19 cases hospitalized in Xiangyang No.1 peoples hospital, Hubei, China until Feb 9th, 2020. Demographic data, laboratory findings and chest computed tomographic (CT) images were obtained and analyzed. Findings All cases were confirmed by real-time RT-PCR, including 52 males and 50 females with a mean age of 50.38 years (SD 16.86). Incubation time ranged from one to twenty days with a mean period of 8.09 days (SD 4.99). Fever (86[84.3%] of 102 patients), cough (58[57%]), fatigue (28[27%]), shortness of breath (24[23%]), diarrhea (15[15%]), expectoration (13[12%]), inappetence (11[10%]) were common clinical manifestations. We observed a decreased blood leukocyte count and lymphopenia in 21 (20.6%) and 56 (54.9%) patients, respectively. There were 66 (68%) of 97 patients with elevated C-reactive protein levels and 49 (57.6%) of 85 with increased erythrocytes sedimentation rate. Higher levels of procalcitonin and ferritin were observed in 19 (25.3%) of 75 and 12 (92.3%) of 13 patients, respectively. Eight patients were admitted to intensive care unit (ICU), six developed respiratory failure, three had multiple organ failure and three died. The cumulative positivity rate over three rounds of real-time RT-PCR was 96%. One-hundred patients were found with typical radiological abnormalities in two rounds of chest CT scans, indicating a 98% consistency with real-time RT-PCR results. Interpretation Most COVID-19 patients in Xiangyang were secondary cases without sex difference, and the rate of severe cases and death was low. Middle-to-old-age individuals were more susceptible to the virus infection and the subsequent development of severe/fatal consequences. The average incubation period was longer among our patients. We recommend prolonging the quarantine period to three weeks. Three times real-time RT-PCR plus two times CT scans is a practical clinical diagnosis strategy at present and should be used to increase the accuracy of diagnosis, thereby controlling the source of infection more effectively. Key Words SARS-CoV-2; COVID-19; epidemiological and clinical features; diagnosis&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Jinwei Ai; Junwen Chen; Yong Wang; Xiaoyun Liu; Wufeng Fan; Gaojing Qu; Meiling Zhang; Shengduo Polo Pei; Bowen Tang; Shuai Yuan; Yang Li; Lisha Wang; Guoxin Huang; Bin Pei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025023&#39;},
 {&#39;cord_uid&#39;: &#39;dbzrd23n&#39;,
  &#39;sha&#39;: &#39;eb8ac60527db35b10881cb4fd86b8a6e21983d02&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A descriptive study of the impact of diseases control and prevention on the epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai, lessons learned for metropolis epidemics prevention&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025031&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Objective: To describe and evaluate the impact of diseases control and prevention on epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai. Design: A retrospective descriptive study Setting: China Participants: Epidemiology information was collected from publicly accessible database. 265 patients admitted to Shanghai Public Health Center with confirmed COVID-19 were enrolled for clinical features analysis. Main outcome measure: Prevention and control measures taken by Shanghai government, epidemiological, demographic, clinical, laboratory and radiology data were collected. Weibull distribution, Chi-square test, Fisher&#39;s exact test, t test or Mann-Whitney U test were used in statistical analysis. Results: COVID-19 transmission rate within Shanghai had reduced over 99% than previous speculated, and the exponential growth has been stopped so far. Epidemic was characterized by the first stage mainly composed of imported cases and the second stage where &gt;50% of cases were local. The incubation period was 6.4 (95% CI 5.3 to 7.6) days and the mean onset-admission interval was 5.5 days (95% CI, 5.1 to 5.9). Median time for COVID-19 progressed to severe diseases were 8.5 days (IQR: 4.8-11.0 days). By February 11th, proportion of patients being mild, moderate, severe and critically ill were 1.9%(5/265), 89.8%(238/265), 3.8%(10/265), 4.5%(12/265), respectively; 47 people in our cohort were discharged, and 1 patient died. Conclusion: Strict controlling of the transmission rate at the early stage of an epidemic in metropolis can quickly prohibit the spread of the diseases. Controlling local clusters is the key to prevent outbreaks from imported cases. Most COVID-19 severe cases progressed within 14 days of disease onset. Multiple systemic laboratory abnormalities had been observed before significant respiratory dysfunction. Keyword: COVID-19, SARS-CoV-2, epidemics dynamics, diseases control, clinical features&#34;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Hongzhou Lu; Jingwen Ai; Yinzhong Shen; Yang Li; Tao Li; Xian Zhou; Haocheng Zhang; Qiran Zhang; Yun Ling; Sheng Wang; Hongping Qu; Yuan Gao; Yingchuan Li; Kanglong Yu; Duming Zhu; Hecheng Zhu; Rui Tian; Mei Zeng; Qiang Li; Yuanlin Song; Xiangyang Li; Jinfu Xu; Jie Xu; Enqiang Mao; Bijie Hu; Xin Li; Lei Zhu; Wenhong Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025031&#39;},
 {&#39;cord_uid&#39;: &#39;hpid3fzc&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Trends and prediction in daily incidence of novel coronavirus infection in China, Hubei Province and Wuhan City: an application of Farr law&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025148&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The recent outbreak of novel coronavirus (2019-nCoV) has infected tens of thousands of patients in China. Studies have forecasted future trends of the incidence of 2019-nCoV infection, but appeared unsuccessful. Farr law is a classic epidemiology theory/practice for predicting epidemics. Therefore, we used and validated a model based on Farr law to predict the daily-incidence of 2019-nCoV infection in China and 2 regions of high-incidence. Methods: We extracted the 2019-nCoV incidence data of China, Hubei Province and Wuhan City from websites of the Chinese and Hubei health commissions. A model based on Farr law was developed using the data available on Feb. 8, 2020, and used to predict daily-incidence of 2019-nCoV infection in China, Hubei Province and Wuhan City afterward. Results: We observed 50,995 (37001 on or before Feb. 8) incident cases in China from January 16 to February 15, 2020. The daily-incidence has peaked in China, Hubei Providence and Wuhan City, but with different downward slopes. If no major changes occur, our model shows that the daily-incidence of 2019-nCoV will drop to single-digit by February 25 for China and Hubei Province, but by March 8 for Wuhan city. However, predicted 75% confidence intervals of daily-incidence in all 3 regions of interest had an upward trend. The predicted trends overall match the prospectively-collected data, confirming usefulness of these models. Conclusions: This study shows the daily-incidence of 2019-nCoV in China, Hubei Province and Wuhan City has reached the peak and was decreasing. However, there is a possibility of upward trend.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Jie Xu; Yajiao Cheng; Xiaoling Yuan; Wei V. Li; Lanjing Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025148&#39;},
 {&#39;cord_uid&#39;: &#39;6kpgt70s&#39;,
  &#39;sha&#39;: &#39;712f1a74a5083d690832eed5fd9ff16a04434d8d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025155&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-24&#39;,
  &#39;authors&#39;: &#39;Laura E Lamb; Sarah N Bartolone; Elijah Ward; Michael B Chancellor&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025155&#39;},
 {&#39;cord_uid&#39;: &#39;02qvv8le&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the risk of 2019 Novel Coronavirus death during the course of the outbreak in China, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025163&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since the first case of Novel Coronavirus (2019-nCov) was identified in December 2019 in Wuhan City, China, the number of cases continues to grow across China and multiple cases have been exported to other countries. The cumulative number of reported deaths is at 637 as of February 7, 2020. Here we statistically estimated the time-delay adjusted death risk for Wuhan as well as for China excluding Wuhan to interpret the current severity of the epidemic in China. We found that the latest estimates of the death risk in Wuhan could be as high as 20% in the epicenter of the epidemic whereas we estimate it ~1% in the relatively mildly-affected areas. Because the elevated death risk estimates are likely associated with a breakdown of the medical/health system, enhanced public health interventions including social distancing and movement restrictions should be effectively implemented to bring the epidemic under control.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Kenji Mizumoto; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025163&#39;},
 {&#39;cord_uid&#39;: &#39;qg2o0ug4&#39;,
  &#39;sha&#39;: &#39;3c94f582610453b6c1389c3a5d6ad57bcf0ba033&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025239&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Summary Background The pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is a highly infectious disease, which was occurred in Wuhan, Hubei Province, China in December 2019. As of February 13, 2020, a total of 59883 cases of COVID-19 in China have been confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. Methods The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. Findings The mean age of the dead was 71.48 years, the average course of the disease was 10.56 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients&#39; last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients&#39; PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%). Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-Dimer and LDH levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Xun Li; Luwen Wang; Shaonan Yan; Fan Yang; Longkui Xiang; Jiling Zhu; Bo Shen; Zuojiong Gong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025239&#39;},
 {&#39;cord_uid&#39;: &#39;goqgv2cc&#39;,
  &#39;sha&#39;: &#39;b463aa228141ea5c920755e57bf9c49a55adf141&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Note on COVID-19 Diagnosis Number Prediction Model in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025262&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: To predict the diagnosed COVID-19 patients and the trend of the epidemic in China. It may give the public some scientific information to ease the fear of the epidemic. Objective: In December 2019, pneumonia infected with the novel coronavirus burst in Wuhan, China. We aimed to use a mathematical model to predict number of diagnosed patients in future to ease anxiety on the emergent situation. Design: According to all diagnosis number from WHO website and combining with the transmission mode of infectious diseases, the mathematical model was fitted to predict future trend of outbreak. Setting: Our model was based on the epidemic situation in China, which could provide referential significance for disease prediction in other countries, and provide clues for prevention and intervention of relevant health authorities. Participants: In this retrospective, all diagnosis number from Jan 21 to Feb 10, 2020 reported from China was included and downloaded from WHO website. Main Outcome(s) and Measure(s): We develop a simple but accurate formula to predict the next day diagnosis number:N_i/N_(i-1) =[(N_(i-1)/N_(i-2) )]^α，where Ni is the total diagnosed patient till the ith day, and α was estimated as 0.904 at Feb 10. Results: Based on this model, it is predicted that the rate of disease infection will decrease exponentially. The total number of infected people is limited; thus, the disease will have limited impact. However, new diagnosis will last to March. Conclusions and Relevance: Through the establishment of our model, we can better predict the trend of the epidemic in China.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Yi Li; Xianhong Yin; Meng Liang; Xiaoyu Liu; Meng Hao; Yi Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025262&#39;},
 {&#39;cord_uid&#39;: &#39;368pz09b&#39;,
  &#39;sha&#39;: &#39;1e45cd5feb7928bbc8bcd524d9c6b7adeae08a6b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025288&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Lunwen Wang; Xun Li; Hui Chen; Shaonan Yan; Yan Li; Dong Li; Zuojiong Gong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025288&#39;},
 {&#39;cord_uid&#39;: &#39;j0ufth5d&#39;,
  &#39;sha&#39;: &#39;4e2bc97a6164191ba3d53abfa49ccd973aa80326&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Early Prediction of Disease Progression in 2019 Novel Coronavirus Pneumonia Patients Outside Wuhan with CT and Clinical Characteristics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025296&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To determine the predictive value of CT and clinical characteristics for short-term disease progression in patients with 2019 novel coronavirus pneumonia (NCP). Materials and Methods: 224 patients with confirmed 2019 novel coronavirus (COVID-19) infection outside Wuhan who had chest CT examinations were retrospectively screened. Clinical data were obtained from electronic medical records. CT images were reviewed and scored for lesion distribution, lobe and segment involvement, ground-glass opacities, consolidation, and interstitial thickening. All included patients with moderate NCP were observed for at least 14 days from admission to determine whether they exacerbated to severe NCP (progressive group) or not (stable group). CT and clinical characteristics between the two groups were compared, and multivariate logistic regression and sensitivity analyses were performed to identify the risk factors for developing severe NCP. Results: A total of 141 patients with moderate NCP were included, of which 15 (10.6%) patients developed severe NCP during hospitalization and assigned to the progressive group. Multivariate logistic regression analysis showed that higher neutrophil-to-lymphocyte ratio (NLR) (odds ratio [OR] and 95% confidence interval [CI], 1.26 [1.04-1.53]; P = 0.018) and CT severity score (OR and 95% CI, 1.25 [1.08-1.46]; P = 0.004) on admission were independent predictors for progression to severe NCP, and sensitivity analysis confirmed the consistent results in nonimported patients but not in imported patients. However, no significant difference in lung involvement was found on CT between imported and nonimported patients (all P &gt; 0.05). Patients who were admitted more than 4 days from symptom onset tended to have more severe lung involvement. Spearman correlation analysis showed the close association between CT severity score and inflammatory indexes (r = 0.17~0.47, all P &lt; 0.05). Conclusion: CT severity score was associated with inflammatory levels and higher NLR and CT severity score on admission were independent risk factors for short-term progression in patients with NCP outside Wuhan. Furthermore, early admission and surveillance by CT should be recommended to improve clinical outcomes.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Zhichao Feng; Qizhi Yu; Shanhu Yao; Lei Luo; Junhong Duan; Zhimin Yan; Min Yang; Hongpei Tan; Mengtian Ma; Ting Li; Dali Yi; Ze Mi; Huafei Zhao; Yi Jiang; Zhenhu He; Huiling Li; Wei Nie; Yin Liu; Jing Zhao; Muqing Luo; Xuanhui Liu; Pengfei Rong; Wei Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025296&#39;},
 {&#39;cord_uid&#39;: &#39;8o3l3rsf&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effectiveness of control strategies for Coronavirus Disease 2019: a SEIR dynamic modeling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025387&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Since its first cases occurrence in Wuhan, China, the Coronavirus Disease 2019 (COVID-19) has been spreading rapidly to other provinces and neighboring countries. A series of intervention strategies have been implemented, but didn&#39;t stop its spread. Methods: Two mathematical models have been developed to simulate the current epidemic situation in the city of Wuhan and in other parts of China. Special considerations were given to the mobility of people for the estimation and forecast the number of asymptomatic infections, symptomatic infections, and the infections of super-spreading events (Isse). Findings: The basic reproductive number (R0) was calculated for the period between 18 January 2020 and 16 February 2020: R0 declined from 5.75 to 1.69 in Wuhan and from 6.22 to 1.67 in the entire country (not including the Wuhan area). At the same time, Wuhan is estimated to observe a peak in the number of confirmed cases around 6 February 2020. The number of infected individuals in the entire country (not including the Wuhan area) peaked around February 3. The results also show that the peak of new asymptomatic cases per day in Wuhan occurred on February 6, and the peak of new symptomatic infections have occurred on February 3. Concurrently, while the number of confirmed cases nationwide would continue to decline, the number of real-time COVID-19 inpatients in Wuhan has reached a peak of 13,030 on February 14 before it decreases. The model further shows that the COVID-19 cases will gradually wane by the end of April 2020, both in Wuhan and the other parts of China. The number of confirmed cases would reach the single digit on March 27 in Wuhan and March 19 in the entire country. The five cities with top risk index in China with the exclusion of Wuhan are: Huanggang, Xiaogan, Jingzhou, Chongqing, and Xiangyang city. Interpretations: Although the national peak time has been reached, a significant proportion of asymptomatic patients and the infections of super-spreading events (Isse) still exist in the population, indicating the potential difficulty for the prevention and control of the disease. As the Return-to-Work tide is approaching and upgrading, further measures (e.g., escalatory quarantine, mask wearing when going out, and sit apart when taking vehicles) will be particularly crucial to stop the COVID-19 in other cities outside of Wuhan.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Jinhua Pan; Ye Yao; Zhixi Liu; Mengyi Li; Ying Wang; Weizhen Dong; Haidong Kan; Weibing Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025387&#39;},
 {&#39;cord_uid&#39;: &#39;h5pit94p&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Generalized anxiety disorder, depressive symptoms and sleep quality during COVID-19 epidemic in China: a web-based cross-sectional survey&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025395&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: China has been severely affected by COVID-19 (Corona Virus Disease 2019) since December, 2019. This study aimed to assess the population mental health burden during the epidemic, and to explore the potential influence factors. Methods: Using a web-based cross-sectional survey, we collected data from 7,236 self-selected volunteers assessed with demographic information, COVID-19 related knowledge, Generalized Anxiety Disorder-7 (GAD-7), Center for Epidemiology Scale for Depression (CES-D), and Pittsburgh Sleep Quality Index (PSQI). Logistic regressions were used to identify influence factors associated with mental health problem. Results: Of the total sample analyzed, the overall prevalence of GAD, depressive symptoms, and sleep quality were 35.1%, 20.1%, and 18.2%, respectively. Young people reported a higher prevalence of GAD and depressive symptoms than older people (P&lt;0.001). Compared with other occupational group, healthcare workers have the highest rate of poor sleep quality (P&lt;0.001). Multivariate logistic regression showed that age (&lt; 35 years) and times to focus on the COVID-19 (≥ 3 hours per day) were associated with GAD, and healthcare workers were associated with poor sleep quality. Conclusions: Our study identified a major mental health burden of the public during COVID-19 epidemic in China. Young people, people who spent too much time on the epidemic, and healthcare workers were at high risk for mental illness. Continuous surveillance and monitoring of the psychological consequences for outbreaks should become routine as part of preparedness efforts worldwide.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Yeen Huang; Ning Zhao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025395&#39;},
 {&#39;cord_uid&#39;: &#39;st5vs6gq&#39;,
  &#39;sha&#39;: &#39;7bfc745e4edcef81527e38d69589e7bcb5f725c2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The serial interval of COVID-19 from publicly reported confirmed cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.20025452&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.6% of reports indicating pre-symptomatic transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Zhanwei Du; Xiaoke Xu; Ye Wu; Lin Wang; Benjamin J Cowling; Lauren Ancel Meyers&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.20025452&#39;},
 {&#39;cord_uid&#39;: &#39;ao7bkcv5&#39;,
  &#39;sha&#39;: &#39;aa507fcd3876f8c422993c8dd13847b1b8e77471&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Pangolin homology associated with 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.950253&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTo explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Tao Zhang; Qunfu Wu; Zhigang Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.950253&#39;},
 {&#39;cord_uid&#39;: &#39;j6y806qu&#39;,
  &#39;sha&#39;: &#39;9f5625d9287d8c6729215a20add99c8cd4ba08cb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Potential T-cell and B-cell Epitopes of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.955484&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data.Significance StatementThe novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Ethan Fast; Russ B. Altman; Binbin Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.955484&#39;},
 {&#39;cord_uid&#39;: &#39;dc8mbllr&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.956235&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA novel and highly pathogenic coronavirus (2019-nCoV) has caused an outbreak in Wuhan city, Hubei province of China since December 2019, and soon spread nationwide and spilled over to other countries around the world. To better understand the initial step of infection at atomic-level, we determined the crystal structure of the 2019-nCoV spike receptor-binding domain (RBD) bound with the cell receptor ACE2 at 2.45 Å resolution. The overall ACE2-binding mode of the 2019-nCoV RBD is nearly identical to that of the SARS-CoV RBD, which also utilizes ACE2 as the cell receptor. Structural analysis identified residues in 2019-nCoV RBD critical for ACE2 binding, and majority of which are either highly conserved or shared similar side chain properties with those in the SARS-CoV RBD. Such similarity in structure and sequence strongly argue for a convergent evolution between 2019-nCoV and SARS-CoV RBD for improved binding to ACE2 despite of being segregated in different genetic lineages in the betacoronavirus genus. The epitopes of two SARS-CoV antibodies targeting the RBD are also analyzed with the 2019-nCoV RBD, providing insights into future identification of cross-reactive antibodies.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Jun Lan; Jiwan Ge; Jinfang Yu; Sisi Shan; Huan Zhou; Shilong Fan; Qi Zhang; Xuanling Shi; Qisheng Wang; Linqi Zhang; Xinquan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.956235&#39;},
 {&#39;cord_uid&#39;: &#39;yaow639d&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.956581&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYThe recent emergence of a novel coronavirus associated with an ongoing outbreak of pneumonia (Covid-2019) resulted in infections of more than 72,000 people and claimed over 1,800 lives. Coronavirus spike (S) glycoprotein trimers promote entry into cells and are the main target of the humoral immune response. We show here that SARS-CoV-2 S mediates entry in VeroE6 cells and in BHK cells transiently transfected with human ACE2, establishing ACE2 as a functional receptor for this novel coronavirus. We further demonstrate that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, which correlates with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S1/S2 subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and other SARS-related CoVs. We determined a cryo-electron microscopy structure of the SARS-CoV-2 S ectodomain trimer, demonstrating spontaneous opening of the receptor-binding domain, and providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal sera potently inhibited SARS-CoV-2 S-mediated entry into target cells, thereby indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Alexandra C. Walls; Young-Jun Park; M. Alexandra Tortorici; Abigail Wall; Andrew T. McGuire; David Veesler&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.956581&#39;},
 {&#39;cord_uid&#39;: &#39;elo7kfun&#39;,
  &#39;sha&#39;: &#39;3a6544f6f247366d4c12167f7fba743e9d981c63&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structural basis for the recognition of the 2019-nCoV by human ACE2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.956946&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAngiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 (COVID-19). Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B0AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 Å in the presence of B0AT1. The local resolution at the ACE2-RBD interface is 3.5 Å, allowing analysis of the detailed interactions between the RBD and the receptor. Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Renhong Yan; Yuanyuan Zhang; Yingying Guo; Lu Xia; Qiang Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.956946&#39;},
 {&#39;cord_uid&#39;: &#39;99sbhbuo&#39;,
  &#39;sha&#39;: &#39;70c7260fc0702bd4a554421ac68972c991313e56&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Mucin 4 Protects Female Mice from Coronavirus Pathogenesis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.19.957118&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractUsing incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4-/- mice and found that female, but not male, Muc4-/- mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4-/- mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4’s impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis.ImportanceGiven the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-20&#39;,
  &#39;authors&#39;: &#39;Jessica A. Plante; Kenneth S. Plante; Lisa E. Gralinski; Anne Beall; Martin T. Ferris; Daniel Bottomly; Richard Green; Shannon K. McWeeney; Mark T. Heise; Ralph S. Baric; Vineet D. Menachery&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.19.957118&#39;},
 {&#39;cord_uid&#39;: &#39;bojhhrvb&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Novel Coronavirus 2019 (Covid-19) epidemic scale estimation: topological network-based infection dynamic model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20023572&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Backgrounds: An ongoing outbreak of novel coronavirus pneumonia (Covid-19) hit Wuhan and hundreds of cities, 29 territories globally. We present a method for scale estimation in dynamic while most of the researchers used static parameters. Methods: We use historical data and the SEIR model for important parameters assumption. And according to the timeline, we use dynamic parameters for infection topology network building. Also, the migration data is used for the Non-Wuhan area estimation which can be cross-validated for the Wuhan model. All data are from the public. Results: The estimated number of infections is 61,596 (95%CI: 58,344.02-64,847.98) by 25 Jan in Wuhan. And the estimation number of the imported cases from Wuhan of Guangzhou was 170 (95%CI: 161.27-179.26), infection scale in Guangzhou is 315 (95%CI: 109.20-520.79), while the imported cases are 168 and the scale of the infection is 339 published by the authority. Conclusions: Using dynamic network models and dynamic parameters for different time periods is an effective way of infection scale modeling.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Keke Tang; Yining Huang; Meilian Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20023572&#39;},
 {&#39;cord_uid&#39;: &#39;bxzprauu&#39;,
  &#39;sha&#39;: &#39;373b3b78e296f0046af3506452c6934b71084b58&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 in Wuhan: Immediate Psychological Impact on 5062 Health Workers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025338&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND: The outbreak of COVID-19 has laid unprecedented psychological stress on health workers (HWs). We aimed to assess the immediate psychological impact on HWs at Tongji Hospital in Wuhan, China. METHODS: We conducted a single-center, cross-sectional survey of HWs via online questionnaires between February 8th and 10th, 2020. We evaluated stress, depression and anxiety by Impact of Event Scale-Revised (IES-R), Patient Health Questionnaire-9 (PHQ-9), and Generalized Anxiety Disorder 7-item (GAD-7), respectively. We also designed a questionnaire to assess the effect of psychological protective measures taken by Tongji Hospital. Multivariate logistic regression was used to identify predictors of acute stress, depression, and anxiety. RESULTS: We received 5062 completed questionnaires (response rate, 77.1 percent). 1509 (29.8 percent), 681 (13.5 percent) and 1218 (24.1 percent) HWs reported stress, depression and anxiety symptoms. Women (hazard ratio[HR], 1.31; P=0.032), years of working&gt; 10 years (HR, 2.02; P&lt;0.001), concomitant chronic diseases (HR, 1.51; P&lt;0.001), history of mental disorders (HR, 3.27; P&lt;0.001), and family members or relatives confirmed or suspected (HR, 1.23; P=0.030) were risk factors for stress, whereas care provided by hospital and department administrators(odds ratio [OR], 0.76; P=0.024) and full coverage of all departments with protective measures (OR, 0.69; P=0.004) were protective factors. CONCLUSIONS: Women and those who have more than 10 years of working, concomitant chronic diseases, history of mental disorders, and family members or relatives confirmed or suspected are susceptible to stress, depression and anxiety among HWs during the COVID-19 pandemic. Psychological protective measures implemented by the hospital could be helpful.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Zhou Zhu; Shabei Xu; Hui Wang; Zheng Liu; Jianhong Wu; Guo Li; Jinfeng Miao; Chenyan Zhang; Yuan Yang; Wenzhe Sun; Suiqiang Zhu; Yebin Fan; Junbo Hu; Jihong Liu; Wei Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025338&#39;},
 {&#39;cord_uid&#39;: &#39;eu3rry7p&#39;,
  &#39;sha&#39;: &#39;f854d7b750541d65c48df953a65028a50ce57881&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;ACP risk grade: a simple mortality index for patients with confirmed or suspected severe acute respiratory syndrome coronavirus 2 disease (COVID-19) during the early stage of outbreak in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025510&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) disease (COVID-19) outbreaks in Wuhan, China, healthcare systems capacities in highly endemic areas have been overwhelmed. Approaches to efficient management are urgently needed and key to a quicker control of the outbreaks and casualties. We aimed to characterize the clinical features of hospitalized patients with confirmed or suspected COVID-19, and develop a mortality risk index for COVID-19 patients. Methods: In this retrospective one-centre cohort study, we included all the confirmed or suspected COVID-19 patients hospitalized in a COVID-19-designated hospital from January 21 to February 5, 2020. Demographic, clinical, laboratory, radiological and clinical outcome data were collected from the hospital information system, nursing records and laboratory reports. Results: Of 577 patients with at least one post-admission evaluation, the median age was 55 years (interquartile range [IQR], 39 - 66); 254 (44.0%) were men; 22.8% (100/438) were severe pneumonia on admission, and 37.7% (75/199) patients were SARS-CoV-2 positive. The clinical, laboratory and radiological data were comparable between positive and negative SARS-CoV-2 patients. During a median follow-up of 8.4 days (IQR, 5.8 - 12.0), 39 patients died with a 12-day cumulative mortality of 8.7% (95% CI, 5.9% to 11.5%). A simple mortality risk index (called ACP index), composed of Age and C-reactive Protein, was developed. By applying the ACP index, patients were categorized into three grades. The 12-day cumulative mortality in grade three (age ≥ 60 years and CRP ≥ 34 mg/L) was 33.2% (95% CI, 19.8% to 44.3%), which was significantly higher than those of grade two (age ≥ 60 years and CRP &lt; 34 mg/L; age &lt; 60 years and CRP ≥ 34 mg/L; 5.6% [95% CI, 0 to 11.3%]) and grade one (age &lt; 60 years and CRP &lt; 34 mg/L, 0%) (P &lt;0.001), respectively. Conclusion: The ACP index can predict COVID-19 related short-term mortality, which may be a useful and convenient tool for quickly establishing a COVID-19 hierarchical management system that can greatly reduce the medical burden and therefore mortality in highly endemic areas.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Jiatao Lu; Shufang Hu; Rong Fan; Zhihong Liu; Xueru Yin; Qiongya Wang; Qingquan Lv; Zhifang Cai; Haijun Li; Yuhai Hu; Ying Han; Hongping Hu; Wenyong Gao; Shibo Feng; Qiongfang Liu; Hui Li; Jian Sun; Jie Peng; Xuefeng Yi; Zixiao Zhou; Yabing Guo; Jinlin Hou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025510&#39;},
 {&#39;cord_uid&#39;: &#39;yqiggzk9&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 51 patients discharged from hospital with COVID-19 in Chongqing，China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025536&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Background：Since December 2019, Severe acute respiratory syndrome coronavirus 2（SARS-CoV-2）-infected disease (Coronavirus Disease 2019，COVID-19) emerged in Wuhan , China，and rapidly spread throughout China，even throughout the world. We try to describe the epidemiological and clinical characteristics of COVID-19 in non-Wuhan area，and explore its effective treatment. Methods：Retrospective, single-center case series of the 51 hospitalized patients with confirmed COVID-19 at Chongqing University Three Gorges Hospital in Chongqing, China, from January 20 to February 3, 2020；The discharge time was from January 29 to February 11, 2020. The main results and indicators of epidemiology, demography, clinical manifestation, laboratory examination, imaging data and treatment data of 51 patients with covid-19 were collected and analyzed. The changes of blood routine and biochemical indexes at discharge and admission were compared. Compare the clinical characteristics of severe patients (including severe and critical patients) and non- severe patients (general patients). Results： Of 51 hospitalized patients with COVID-19, the median age was 45 years (interquartile range, 34-51; range, 16-68 years) and 32 (62.7%) were men.43（84.3%）patients had been to Wuhan or Other Hubei areas outside Wuhan，and 4（7.7%） patients had a clear contact history of COVID-19 patients before the onset of the disease, and 4（7.7%） patients had no clear epidemiological history of COVID-19.Common symptoms included fever (43 [84.3%]), cough (38 [74.5%]) and fatigue (22 [43.1%]). Lymphopenia was observed in 26 patients (51.0%), and elevated C-reactive protein level in 32 patients (62.7%). Ground-glass opacity was the typical radiological finding on chest computed tomography (41 [80.4%])，Local consolidation of pneumonia in some patients(17 [33.3%]).Most of the patients were treated with traditional Chinese medicine decoction (28 [54.9%])，all of them received aerosol inhalation of recombinant human interferon a-1b for injection and oral antiviral therapy with Lopinavir and Ritonavir tablets (51 [100%]); Most of the patients were given Bacillus licheniformis capsules regulated intestinal flora treatment (44 [86.3%]). 10 patients (19.6%) received short-term (3-5 days) glucocorticoid treatment. Compared with non-severe patients (n = 44), severe patients (n = 7) were older (median age, 52 years vs 44 years), had a higher proportion of diabetes mellitus (4 [57.1%] vs 0 [0.0%]), most of them needed antibiotic treatment (7 [100%] vs 4 [9.1%], most of them needed nutritional diet (6 [85.7%) vs 0 [0.0%], and were more likely to have dyspnea (6 [85.7%] vs 5 [11.4%])，most of them needed noninvasive mechanical ventilation (6 [85.7%] vs 0 [0.0%]). Except one patient died, the remaining 50 patients were discharged according to the discharge standard, the common clinical symptoms disappeared basically, the lymphocyte increased significantly (P=0.008), CRP decreased significantly (P &lt;0.001). The median length of stay was 12 days (IQR, 9-13). Conclusion：In 51 single center cases confirmed as COVID-19 and discharged from the hospital, 13.7% of the patients were severe. The main clinical symptoms of patients with COVID-19 were fever, cough and asthenia，Some patients had obvious dyspnea. They had clinical laboratory and radiologic characteristics. There is no specific drug treatment for the disease. For the treatment of COVID-19, in addition to oxygen inhalation and antiviral treatment, attention should be paid to the dialectical treatment of traditional Chinese medicine, regulation of intestinal flora, nutritional support treatment and other comprehensive treatment.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;liu lei; Gao Jian-ya&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025536&#39;},
 {&#39;cord_uid&#39;: &#39;4pe9o59g&#39;,
  &#39;sha&#39;: &#39;8d6d3be5f9353eb3358b44e646c83f1a90861b4d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The efficacy of convalescent plasma for the treatment of severe influenza&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025593&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background. Administration of convalescent plasma may be of clinical benefit for treatment of severe acute viral respiratory infections. However, no clear evidence exists to support or oppose convalescent plasma use in clinical practice. We conducted a systematic review and meta-analysis to assess the evidence of randomized controlled trials (RCTs) in the convalescent plasma for the treatment of severe influenza. Methods. Healthcare databases were searched in February 2020. All records were screened against the eligibility criteria. Data extraction and risk of bias assessments were undertaken. The primary outcome was case fatality rates by influenza. Results. We identified 5 RCTs of severe influenza. The pooled analyses showed no evidence for a reduction in mortality (Odds Ratio (OR) 1.06; p = 0.87; I2 = 35%). We also found non significant reductions in days in ICU and hospital, and days on mechanical ventilation. There seemed to have a biological benefit of increasing HAI titer levels and decreasing influenza B virus loads and cytokines after convalescent plasma treatment. No serious adverse events was reported between two groups. Studies were commonly of low risk of bias with high quality. Conclusions. Convalescent plasma appears safe but may not reduce mortality in severe influenza. This therapy should be studied within the context of a well designed clinical trial for treatment of SARS Cov 2 infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Zhiheng Xu; Jianmeng Zhou; Yongbo Huang; Xuesong Liu; Yonghao Xu; Sibei Chen; Dongdong Liu; Zhimin Lin; Xiaoqing Liu; Yimin Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025593&#39;},
 {&#39;cord_uid&#39;: &#39;rmia5w3x&#39;,
  &#39;sha&#39;: &#39;6fcf4019b0be24071210f97eda4baba53a6a79de&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical and pathological characteristics of 2019 novel coronavirus disease (COVID-19): a systematic review&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025601&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: In 2002-2003, a severe pulmonary infectious disease occurred in Guangdong, China. The disease was caused by SARS-CoV, 17 years apart, also happen in China, and SARS-CoV-2, this epidemic has posed a significant hazard to people health both China and the whole world. Objective: Summarized the latest epidemiological changes, clinical manifestations, auxiliary examination and pathological characteristics of COVID-19. Evidence Review: PubMed database were searched from 2019 to 2020 using the index terms novel coronavirus or COVID-19 or 2019-nCoV or SARS-CoV-2 and synonyms. Articles that reported clinical characteristics, laboratory results, imageological diagnosis and pathologic condition were included and were retrospectively reviewed for these cases. This paper adopts the method of descriptive statistics. Results: 34 COVID-19-related articles were eligible for this systematic review;Four of the articles were related to pathology. We found that Fever (86.0%), cough (63.9%) and Malaise/Fatigue (34.7%) were the most common symptoms in COVID-19. But in general, the clinical symptoms and signs of COVID-19 were not obvious. Compared with SARS, COVID-19 was transmitted in a more diverse way. The mortality rates of COVID-19 were 2.5%, and the overall infection rate of healthcare worker of COVID-19 was 3.9%. We also found that the pathological features of COVID-19 have greatly similar with SARS, which manifested as ARDS. But the latest pathological examination of COVID-19 revealed the obvious mucinous secretions in the lungs. Interpretation: The clinical and pathological characteristics of SARS and COVID-19 in China are very similar, but also difference. The latest finds of pathological examination on COVID-19 may upend existing treatment schemes, so the early recognition of disease by healthcare worker is very important.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;mao yaqian; Wei Lin; Junping Wen; Gang Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025601&#39;},
 {&#39;cord_uid&#39;: &#39;ofoqk100&#39;,
  &#39;sha&#39;: &#39;a36bfdd1c9a1666401269fb0c08a3c2922bd6812&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of 24 Asymptomatic Infections with COVID-19 Screened among Close Contacts in Nanjing, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025619&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Previous studies have showed clinical characteristics of patients with the 2019 novel coronavirus disease (COVID-19) and the evidence of person-to-person transmission. Limited data are available for asymptomatic infections. This study aims to present the clinical characteristics of 24 cases with asymptomatic infection screened from close contacts and to show the transmission potential of asymptomatic COVID-19 virus carriers. Methods: Epidemiological investigations were conducted among all close contacts of COVID-19 patients (or suspected patients) in Nanjing, Jiangsu Province, China, from Jan 28 to Feb 9, 2020, both in clinic and in community. Asymptomatic carriers were laboratory-confirmed positive for the COVID-19 virus by testing the nucleic acid of the pharyngeal swab samples. Their clinical records, laboratory assessments, and chest CT scans were reviewed. Findings: None of the 24 asymptomatic cases presented any obvious symptoms before nucleic acid screening. Five cases (20.8%) developed symptoms (fever, cough, fatigue and etc.) during hospitalization. Twelve (50.0%) cases showed typical CT images of ground-glass chest and five (20.8%) presented stripe shadowing in the lungs. The remaining seven (29.2%) cases showed normal CT image and had no symptoms during hospitalization. These seven cases were younger (median age: 14.0 years; P = 0.012) than the rest. None of the 24 cases developed severe COVID-19 pneumonia or died. The median communicable period, defined as the interval from the first day of positive nucleic acid tests to the first day of continuous negative tests, was 9.5 days (up to 21 days among the 24 asymptomatic cases). Through epidemiological investigation, we observed a typical asymptomatic transmission to the cohabiting family members, which even caused severe COVID-19 pneumonia. Interpretation: The asymptomatic carriers identified from close contacts were prone to be mildly ill during hospitalization. However, the communicable period could be up to three weeks and the communicated patients could develop severe illness. These results highlighted the importance of close contact tracing and longitudinally surveillance via virus nucleic acid tests. Further isolation recommendation and continuous nucleic acid tests may also be recommended to the patients discharged.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Zhiliang Hu; Ci Song; Chuanjun Xu; Guangfu Jin; Yaling Chen; Xin Xu; Hongxia Ma; Wei Chen; Yuan Lin; Yishan Zheng; Jianming Wang; zhibin hu; Yongxiang Yi; Hongbing Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025619&#39;},
 {&#39;cord_uid&#39;: &#39;kqanoog7&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025841&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We report the kinetics of the immune response in relation to clinical and virological features of a patient with mild-to-moderate coronavirus disease-19 (COVID-19) requiring hospitalisation. Increased antibody-secreting cells, follicular T-helper cells, activated CD4+ and CD8+ T-cells and IgM/IgG SARS-CoV-2-binding antibodies were detected in blood, prior to symptomatic recovery. These immunological changes persisted for at least 7 days following full resolution of symptoms, indicating substantial anti-viral immunity in this non-severe COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Irani Thevarajan; Thi HO Nguyen; Marios Koutsakos; Julian Druce; Leon Caly; Carolien E van de Sandt; Xiaoxiao Jia; Suellen Nicholson; Mike Catton; Benjamin Cowie; Steven Tong; Sharon Lewin; Katherine Kedzierska&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025841&#39;},
 {&#39;cord_uid&#39;: &#39;atnz63pk&#39;,
  &#39;sha&#39;: &#39;207dcb6e3cd43cdea91d29a15be1ee34068bd54e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the Asymptomatic Proportion of 2019 Novel Coronavirus onboard the Princess Cruises Ship, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025866&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The potential infectiousness of asymptomatic COVID-19 cases together with a substantial fraction of asymptomatic infections among all infections, have been highlighted in clinical studies. We conducted statistical modeling analysis to derive the delay-adjusted asymptomatic proportion of the positive COVID-19 infections onboard the Princess Cruises ship along with the timeline of infections. We estimated the asymptomatic proportion at 17.9% (95% CrI: 15.5%-20.2%), with most of the infections occurring before the start of the 2-week quarantine.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Kenji Mizumoto; Katsushi Kagaya; Alexander Zarebski; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025866&#39;},
 {&#39;cord_uid&#39;: &#39;s7uqawbd&#39;,
  &#39;sha&#39;: &#39;bd8f57ea4aadeda075d557f566a1909d68e658c4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025874&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-24&#39;,
  &#39;authors&#39;: &#39;Lin Yu; Shanshan Wu; Xiaowen Hao; Xuelong Li; Xiyang Liu; Shenglong Ye; Heng Han; Xue Dong; Xin Li; Jiyao Li; Jianmin Liu; Na Liu; Wanzhong Zhang; Vicent Pelechano; Wei-Hua Chen; Xiushan Yin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025874&#39;},
 {&#39;cord_uid&#39;: &#39;86opxdjd&#39;,
  &#39;sha&#39;: &#39;ba3efcd6b74e55327fd7db470d824fc18943f30e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluating the impact of international airline suspensions on the early global spread of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025882&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Global airline networks play a key role in the global importation of emerging infectious diseases. Detailed information on air traffic between international airports has been demonstrated to be useful in retrospectively validating and prospectively predicting case emergence in other countries. In this paper, we use a well-established metric known as effective distance on the global air traffic data from IATA to quantify risk of emergence for different countries as a consequence of direct importation from China, and compare it against arrival times for the first 24 countries. Using this model trained on official first reports from WHO, we estimate time of arrival (ToA) for all other countries. We then incorporate data on airline suspensions to recompute the effective distance and assess the effect of such cancellations in delaying the estimated arrival time for all other countries. Finally we use the infectious disease vulnerability indices to explain some of the estimated reporting delays.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Aniruddha Adiga; Srinivasan Venkatramanan; James Schlitt; Akhil Peddireddy; Allan Dickerman; Andrei Bura; Andrew Warren; Brian D Klahn; Chunhong Mao; Dawen Xie; Dustin Machi; Erin Raymond; Fanchao Meng; Golda Barrow; Henning Mortveit; Jiangzhuo Chen; Jim Walke; Joshua Goldstein; Mandy L Wilson; Mark Orr; Przemyslaw Porebski; Pyrros A Telionis; Richard Beckman; Stefan Hoops; Stephen Eubank; Young Yun Baek; Bryan Lewis; Madhav Marathe; Chris Barrett&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025882&#39;},
 {&#39;cord_uid&#39;: &#39;w9a0jopu&#39;,
  &#39;sha&#39;: &#39;9896bc65559e6406d0d3cb35e9b01b953d61959d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;From Isolation to Coordination: How Can Telemedicine Help Combat the COVID-19 Outbreak?&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025957&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The rapid spread of Coronavirus disease 2019 (COVID-19) presents China with a critical challenge. As normal capacity of the Chinese hospitals is exceeded, healthcare professionals struggling to manage this unprecedented crisis face the difficult question of how best to coordinate the medical resources used in highly separated locations. Responding rapidly to this crisis, the National Telemedicine Center of China (NTCC), located in Zhengzhou, Henan Province, has established the Emergency Telemedicine Consultation System (ETCS), a telemedicine-enabled outbreak alert and response network. ETCS is built upon a doctor-to-doctor (D2D) approach, in which health services can be accessed remotely through terminals across hospitals. The system architecture of ETCS comprises three major architectural layers: (1) telemedicine service platform layer, (2) telemedicine cloud layer, and (3) telemedicine service application layer. Our ETCS has demonstrated substantial benefits in terms of the effectiveness of consultations and remote patient monitoring, multidisciplinary care, and prevention education and training.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Yunkai Zhai; Yichuan Wang; Minhao Zhang; Jody Hoffer Gittell; Shuai Jiang; Baozhan Chen; Fangfang Cui; Xianying He; Jie Zhao; Xiaojun Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025957&#39;},
 {&#39;cord_uid&#39;: &#39;4gaa14ly&#39;,
  &#39;sha&#39;: &#39;ae6fc64042b050df93ebb8f8045892952f18510f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing the impact of a symptom-based mass screening and testing intervention during a novel infectious disease outbreak: The case of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025973&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A symptom-based mass screening and testing intervention (MSTI) can identify a large fraction of infected individuals during an infectious disease outbreak. China is currently using this strategy for the COVID-19 outbreak. However, MSTI might lead to increased transmission if not properly implemented. We investigate under which conditions MSTI is beneficial.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Yang Ge; Brian Kenneth McKay; Shengzhi Sun; Feng Zhang; Andreas Handel&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025973&#39;},
 {&#39;cord_uid&#39;: &#39;1s0exznp&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Generation of antibodies against COVID-19 virus for development of diagnostic tools&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.20025999&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Maohua Li; Ronghua Jin; Ya Peng; Cuiyan Wang; Wenlin Ren; Fudong Lv; Sitao Gong; Feng Fang; Qianyun Wang; Jianli Li; Tong Shen; Hunter Sun; Lei Zhou; Yali Cui; Hao Song; Le Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.20025999&#39;},
 {&#39;cord_uid&#39;: &#39;bnnt05fn&#39;,
  &#39;sha&#39;: &#39;d01e580a45c457c7c200cd884e746b4d836e2ea9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Vulnerabilities in coronavirus glycan shields despite extensive glycosylation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.957472&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSevere acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses (CoVs) are zoonotic pathogens with high fatality rates and pandemic potential. Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Yasunori Watanabe; Zachary T. Berndsen; Jayna Raghwani; Gemma E. Seabright; Joel D. Allen; Jason S. McLellan; Ian A. Wilson; Thomas A. Bowden; Andrew B. Ward; Max Crispin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.957472&#39;},
 {&#39;cord_uid&#39;: &#39;535lw99y&#39;,
  &#39;sha&#39;: &#39;78e49fdb6f0aa9924a5b510341d52b618fff0ca6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A Multiscale and Comparative Model for Receptor Binding of 2019 Novel Coronavirus and the Implication of its Life Cycle in Host Cells&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.958272&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe respiratory syndrome caused by a new type of coronavirus has been emerging from China and caused more than 1000 death globally since December 2019. This new virus, called 2019 novel coronavirus (2019-nCoV) uses the same receptor called Angiotensinconverting enzyme 2 (ACE2) to attack humans as the coronavirus that caused the severe acute respiratory syndrome (SARS) seventeen years ago. Both viruses recognize ACE2 through the spike proteins (S-protein) on their surfaces. It was found that the S-protein from the SARS coronavirus (SARS-CoV) bind stronger to ACE2 than 2019-nCoV. However, function of a bio-system is often under kinetic, rather than thermodynamic, control. To address this issue, we constructed a structural model for complex formed between ACE2 and the S-protein from 2019-nCoV, so that the rate of their association can be estimated and compared with the binding of S-protein from SARS-CoV by a multiscale simulation method. Our simulation results suggest that the association of new virus to the receptor is slower than SARS, which is consistent with the experimental data obtained very recently. We further integrated this difference of association rate between virus and receptor into a mathematical model which describes the life cycle of virus in host cells and its interplay with the innate immune system. Interestingly, we found that the slower association between virus and receptor can result in longer incubation period, while still maintaining a relatively higher level of viral concentration in human body. Our computational study therefore provides, from the molecular level, one possible explanation that the new disease by far spread much faster than SARS.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Zhaoqian Su; Yinghao Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.958272&#39;},
 {&#39;cord_uid&#39;: &#39;0vjs2w3l&#39;,
  &#39;sha&#39;: &#39;9aef9e1c22896809852ec6c94e413aff33b3d9cf&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Inactivating porcine coronavirus before nuclei acid isolation with the temperature higher than 56 °C damages its genome integrity seriously&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.20.958785&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Abstract2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019. Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the positive rate is only 30-50%. Currently, fluorescent quantitative RT-PCR technology is mainly used to detect 2019-nCoV. According to “The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)” issued by National Health and Commission of China and “The Experts’ Consensus on Nucleic Acid Detection of 2019-nCoV” released by Chinese Society of Laboratory Medicine, the human samples must be placed under 56°C or higher to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated as the template of qRT-PCR. In this study, we demonstrated that the virus inactivation treatment disrupts its genome integrity seriously when using porcine epidemic diarrhea virus (vaccine), a kind of coronavirus, as a model. Our results showed that only 50.11% of the detectable viral templates left after the inactivation of 56 °C for 30 minutes and only 3.36% left after the inactivation of 92 °C for 5 minutes when the samples were preserved by Hank’s solution, one of an isotonic salt solutions currently suggested. However, the detectable templates of viral nucleic acids can be unchanged after the samples were incubated at 56 °C or higher if the samples were preserved with an optimized solution to protect the RNA from being disrupted. We therefore highly recommend to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV genome and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-22&#39;,
  &#39;authors&#39;: &#39;Qingxin Zhang; Qingshun Zhao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.20.958785&#39;},
 {&#39;cord_uid&#39;: &#39;u6b8iwr0&#39;,
  &#39;sha&#39;: &#39;233978852b45429db49d2aabe1c9c9bfe73b1884&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;CoVID-19 in Japan: What could happen in the future?&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026070&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 has been impacting on the whole world critically and constantly Since December 2019. We have independently developed a novel statistical time delay dynamic model on the basis of the distribution models from CCDC. Based only on the numbers of confirmed cases in different regions in China, the model can clearly reveal that the containment of the epidemic highly depends on early and effective isolation. We apply the model on the epidemic in Japan and conclude that there could be a rapid outbreak in Japan if no effective quarantine measures are carried out immediately.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Nian Shao; Hanshuang Pan; Xingjie Li; Weijia Li; Shufen Wang; Yan Xuan; Yue Yan; Yu Jiang; Keji Liu; Yu Chen; Boxi Xu; Xinyue Luo; Christopher Y. Shen; Min Zhong; Xiang Xu; Xu Chen; Shuai Lu; Guanghong Ding; Jin Cheng; Wenbin Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026070&#39;},
 {&#39;cord_uid&#39;: &#39;1mxjklgx&#39;,
  &#39;sha&#39;: &#39;091a8e9a61e19e88caeb039f0e3888d111b20439&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological characteristics of 1212 COVID-19 patients in Henan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026112&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Based on publicly released data for 1212 patients, we investigated the epidemiological characteristics of COVID-19 in Henan of China. The following findings are obtained: 1) COVID-19 patients in Henan show gender (55% vs 45%) and age (81% aged between 21 and 60) preferences, possible causes were explored; 2) Statistical analysis on 483 patients reveals that the estimated average, mode and median incubation periods are 7.4, 4 and 7 days; Incubation periods of 92% patients were no more than 14 days; 3) The epidemic of COVID-19 in Henan has undergone three stages and showed high correlations with the numbers of patients that recently return from Wuhan; 4) Network analysis on the aggregate outbreak phenomena of COVID-19 revealed that 208 cases were clustering infected, and various people&#39;s Hospital are the main force in treating patients. The related investigations have potential implications for the prevention and control of COVID-19.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Pei Wang; Junan Lu; Yanyu Jin; Mengfan Zhu; Lingling Wang; Shunjie Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026112&#39;},
 {&#39;cord_uid&#39;: &#39;aoyyk5fl&#39;,
  &#39;sha&#39;: &#39;eaf0b485f290fa884dd18de71be79d86de20eb31&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Public Exposure to Live Animals, Behavioural Change, and Support in Containment Measures in response to COVID-19 Outbreak: a population-based cross sectional survey in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026146&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background In response to the COVID-19 outbreak, we aimed to investigate behavioural change on exposure to live animals before and during the outbreak, and public support and confidence for governmental containment measures. Methods A population-based cross-sectional telephone survey via random dialing was conducted in Wuhan (the epicentre) and Shanghai (an affected city with imported cases) between 1 and 10 February, 2020. 510 residents in Wuhan and 501 residents in Shanghai were randomly sampled. Differences of outcome measures were compared before and during the outbreak, and between two cities. Findings Proportion of respondents visiting wet markets at usual was 23.3% (119/510) in Wuhan and 20.4% (102/501) in Shanghai. During the outbreak, it decreased to 3.1% (16) in Wuhan (p&lt;0.001), and 4.4% (22) in Shanghai (p&lt;0.001). Proportion of those consuming wild animal products declined from 10.2% (52) to 0.6% (3) in Wuhan (p&lt;0.001), and from 5.2% (26) to 0.8% (4) in Shanghai (p&lt;0.001). 79.0% (403) of respondents in Wuhan and 66.9% (335) of respondents in Shanghai supported permanent closure of wet markets (P&lt;0.001). 95% and 92% of respondents supported banning wild animal trade and quarantining Wuhan, and 75% were confident towards containment measures. Females and the more educated were more supportive for the above containment measures. Interpretation The public responded quickly to the outbreak, and reduced exposure to live animals, especially in Wuhan. With high public support in containment measures, better regulation of wet markets and healthy diets should be promoted.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Zhiyuan Hou; Leesa Lin; Liang Lu; Fanxing Du; Mengcen Qian; Yuxia Liang; Juanjuan Zhang; Hongjie Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026146&#39;},
 {&#39;cord_uid&#39;: &#39;t05ummop&#39;,
  &#39;sha&#39;: &#39;d12b31daa935e9043a9f526a2547b6216ada7cde&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;2019 Novel Coronavirus can be detected in urine, blood, anal swabs and oropharyngeal swabs samples&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026179&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We tested samples collected from nine patients diagnosed with coronavirus disease 2019 (COVID-19). The virus was found in urine, blood, anal swabs and oropharyngeal swabs. It is the first time for SARS-CoV-2 found in urine, though no urinary irritation was found.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Liang Peng; Jing Liu; Wenxiong Xu; Qiumin Luo; Keji Deng; Bingliang Lin; Zhiliang Gao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026179&#39;},
 {&#39;cord_uid&#39;: &#39;z4u017sa&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026187&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Background: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Diagnostic test based on real-time reverse transcription polymerase chain reaction assay (qRT-PCR) was the main means of confirmation, and sample collection was mostly throat swabs, which was easy to miss the diagnosis. It is necessary to seek specimen types with higher detection efficiency and accuracy. Methods: Paired specimens of throat swabs and sputum were obtained from 54 cases, and RNA was extracted and tested for 2019-nCoV (equated with SARS-CoV-2) by qRT-PCR assay. Results: The positive rates of 2019-nCoV from sputum specimens and throat swabs were 76.9% and 44.2%, respectively. Sputum specimens showed a significantly higher positive rate than throat swabs in detecting viral nucleic acid using qRT-PCR assay (P=0.001). Conclusions: The detection rates of 2019-nCoV from sputum specimens are significantly higher than throat swabs. We suggest that sputum would benefit for the detection of 2019-nCoV in patients who produce sputum. The results can facilitate the selection of specimens and increase the accuracy of diagnosis.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Chenyao Lin; Jie Xiang; Mingzhe Yan; Hongze Li; Shuang Huang; Changxin Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026187&#39;},
 {&#39;cord_uid&#39;: &#39;2vvqczj7&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological Development of Novel Coronavirus Pneumonia in China and Its Forecast&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026229&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;ABSTRACT BACKGROUND AND OBJECTIVE The novel coronavirus (SARS-Cov-2) infected coronavirus disease 2019 (COVID-19) was broken out in Wuhan and Hubei province for more than a month. It severely threats people&#39;s health of thousands in Chin and even other countries. In order to prevent its wide spread, it is necessary to understand the development of the epidemic with precise mathematical language. METHODS The various data of novel coronavirus pneumonia were collected from the official websites of the National Health Committee of the People&#39;s Republic of China. According to epidemic and administrative division, three groups were divided to analyze the data, Hubei Province (including Wuhan), nationwide without Hubei and Henan Province. With classic SIR models, the fitting epidemiological curves of incidence have made, and basic reproduction number (R0) was also calculated as well. Therefore the disease&#39;s infection intensity, peak time and the epidemiological end time can be deduced.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Shan shan Wu; Pan pan Sun; Rui ling Li; Liang Zhao; Yan li Wang; Li fang Jiang; Jin Bo Deng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026229&#39;},
 {&#39;cord_uid&#39;: &#39;ragcpbl6&#39;,
  &#39;sha&#39;: &#39;87b89b038b28d2ba0bcfd2203845bf15f922c1b5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026328&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy. Methods We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level. Results The median age of the cases was 44 years, with an increasing of cases in younger age groups and the elderly as the epidemic progressed. The delay from symptom onset to hospital admission decreased from 4.4 days (95%CI: 0.0-14.0) until January 27 to 2.6 days (0.0-9.0) from January 28 to February 17. The mean incubation period was estimated at 5.2 days (1.8-12.4) and the mean serial interval at 5.1 days (1.3-11.6). The epidemic dynamics in provinces outside Hubei was highly variable, but consistently included a mix of case importations and local transmission. We estimate that the epidemic was self-sustained for less than three weeks with Rt reaching peaks between 1.40 (1.04-1.85) in Shenzhen City of Guangdong Province and 2.17 (1.69-2.76) in Shandong Province. In all the analyzed locations (n=10) Rt was estimated to be below the epidemic threshold since the end of January. Conclusion Our findings suggest that the strict containment measures and movement restrictions in place may contribute to the interruption of local COVID-19 transmission outside Hubei Province. The shorter serial interval estimated here implies that transmissibility is not as high as initial estimates suggested.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Juanjuan Zhang; Maria Litvinova; Wei Wang; Yan Wang; Xiaowei Deng; Xinghui Chen; Mei Li; Wen Zheng; Lan Yi; Xinhua Chen; Qianhui Wu; Yuxia Liang; Xiling Wang; Juan Yang; Kaiyuan Sun; Ira M. Longini; M. Elizabeth Halloran; Peng Wu; Benjamin J. Cowling; Stefano Merler; Cecile Viboud; Alessandro Vespignani; Marco Ajelli; Hongjie Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026328&#39;},
 {&#39;cord_uid&#39;: &#39;vw3h1m5y&#39;,
  &#39;sha&#39;: &#39;b842a645116edb86df997aa960699a7513f2196a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026435&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Amna Tariq; Yiseul Lee; Kimberlyn Roosa; Seth Blumberg; Ping Yan; Stefan Ma; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026435&#39;},
 {&#39;cord_uid&#39;: &#39;eedk54xf&#39;,
  &#39;sha&#39;: &#39;29bc34aad802ab6b35074f23c631540a84b16d0d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Trends in Transmissibility of 2019 Novel Coronavirus-infected Pneumonia in Wuhan and 29 Provinces in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026468&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The 2019 novel coronavirus infected pneumonia (COVID- 19) represents a significant public health threat. The COVID-19 emerged in December 2019 in Wuhan, China and rapidly spread to other regions and countries. The variation in transmission patterns and disease spread in regard to time or among different locations, partially reflecting the public health in- tervention effects, remains to be quantified. As most transmissibility-related epidemic parameters are unknown, we sought, with minimal assumptions, to estimate real-time transmissibility and forecast new cases using dynamic modelling. Method: Using the cases reported from the National Health Commission of China and transportation data, including the total number of travelling hours through railway, airplane, and car outbound from Wuhan, we have built a time-series model to estimate real-time underlying transmission rates of newly generated cases sequentially from January 20, 2020 to Feb 13, 2020 in Wuhan, Hubei province and other 28 provinces in China. We quantified the instantaneous transmission rate and relative reproduction number (R_t) of COVID-19, and evaluated whether public health intervention affected the case transmissibility in each province. Based on the current estimates, we have predicted the trend of disease spread with a high level of certainty. Findings: We estimated that R_t declined from the range of 4 to 5 towards 1 and remained below unity, while there was an initial growth followed by a decline in a shorter period in Hubei and other provinces. The ratio of transmission rates decreased dramatically from January 23 to 27 likely due to the rigorous public health intervention implemented by the government beginning on January 23, 2020. The mean duration of the infectious period was 6 to 9 days. We have predicted the trend of infection sizes which be- came stable in provinces around February 19 to 24, 2020, and the date of containment would be one-week later in Wuhan. Interpretation: Public health interventions implemented at both the so- cial and personal levels are effective in preventing outbreaks of COVID-19 in Wuhan and other provinces. Model prediction results suggested that COVID- 19 will be contained around the end of February 2020 in China. Keywords: Coronavirus, COVID-19, transmissibility, dynamic reproduction number R, statistical modelling, pneumonia outbreak&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Huazhen Lin; Wei Liu; Hong Gao; Jinyu Nie; Qiao Fan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026468&#39;},
 {&#39;cord_uid&#39;: &#39;jydxk6j1&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the serial interval of the novel coronavirus disease (COVID-19): A statistical analysis using the public data in Hong Kong from January 16 to February 15, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.20026559&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Backgrounds: The emerging virus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has caused a large outbreak of novel coronavirus disease (COVID-19) in Wuhan, China since December 2019. Based on the publicly available surveillance data, we identified 21 transmission chains in Hong Kong and estimated the serial interval (SI) of COVID-19. Methods: Index cases were identified and reported after symptoms onset, and contact tracing was conducted to collect the data of the associated secondary cases. An interval censored likelihood framework is adopted to fit a Gamma distribution function to govern the SI of COVID-19. Findings: Assuming a Gamma distributed model, we estimated the mean of SI at 4.4 days (95%CI: 2.9−6.7) and SD of SI at 3.0 days (95%CI: 1.8−5.8) by using the information of all 21 transmission chains in Hong Kong. Conclusion: The SI of COVID-19 may be shorter than the preliminary estimates in previous works. Given the likelihood that SI could be shorter than the incubation period, pre-symptomatic transmission may occur, and extra efforts on timely contact tracing and quarantine are recommended in combating the COVID-19 outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Shi Zhao; Daozhou Gao; Zian Zhuang; Marc Chong; Yongli Cai; Jinjun Ran; Peihua Cao; Kai Wang; Yijun Lou; Weiming Wang; Lin Yang; Daihai He; Maggie Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.20026559&#39;},
 {&#39;cord_uid&#39;: &#39;if9lar1c&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.959817&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractReverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability1-3. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-24 in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-21&#39;,
  &#39;authors&#39;: &#39;Tran Thi Nhu Thao; Fabien Labroussaa; Nadine Ebert; Philip V’kovski; Hanspeter Stalder; Jasmine Portmann; Jenna Kelly; Silvio Steiner; Melle Holwerda; Annika Kratzel; Mitra Gultom; Laura Laloli; Linda Hüsser; Manon Wider; Stephanie Pfaender; Dagny Hirt; Valentina Cippà; Silvia Crespo-Pomar; Simon Schröder; Doreen Muth; Daniela Niemeyer; Marcel A. Müller; Christian Drosten; Ronald Dijkman; Joerg Jores; Volker Thiel&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.959817&#39;},
 {&#39;cord_uid&#39;: &#39;v0yzur5h&#39;,
  &#39;sha&#39;: &#39;cc41c51293ed24a57ddf467e462ec9399463ee33&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.959973&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-22&#39;,
  &#39;authors&#39;: &#39;Anton Nekrutenko; Sergei L Kosakovsky Pond&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.959973&#39;},
 {&#39;cord_uid&#39;: &#39;ppl3xnm9&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Molecular Survey for Selected Viral Pathogens in Wild Leopard Cats (Prionailurus bengalensis) in Taiwan with an Emphasis on the Spatial and Temporal Dynamics of Carnivore Protoparvovirus 1&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.21.960492&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe leopard cat (Prionailurus bengalensis) has been listed as an endangered species under the Wildlife Conservation Act in Taiwan since 2009. In this study, we targeted viral pathogens, included carnivore protoparvovirus 1 (CPPV-1), feline leukemia virus (FeLV), feline immunodeficiency virus (FIV), coronavirus (CoV), and canine morbillivirus (CMV), using molecular screening. The spatial and temporal dynamics of the target pathogens were evaluated. Through sequencing and phylogenetic analysis, we aimed to clarify the phylogenetic relationship of isolated viral pathogens between leopard cats and domestic carnivores. Samples from 23 and 29 leopard cats that were live-trapped and found dead, respectively, were collected from Miaoli County from 2015 to 2019 in northwestern Taiwan. CPPV-1 and coronavirus were detected in leopard cats. The prevalence (95% confidence interval) of CPPV-1, and CoV was 63.5% (50.4%–76.6%) and 8.8% (0%–18.4%), respectively. The majority of sequences of each CPPV-1 strain amplified from Taiwanese leopard cats and domestic carnivores were identical. All the amplified CoV sequences from leopard cats were identified as feline coronavirus. The spatial and temporal aggregation of CPPV-1 infection in leopard cats was not determined in the sampling area, which indicated a wide distribution of CPPV-1 in the leopard cat habitat. We consider sympatric domestic carnivores to be the probable primary reservoir for the pathogens identified. We strongly recommend establishing efforts to manage CPPV-1 and FCoV in the leopard cat habitat, with an emphasis on vaccination programs and population control measures for free-roaming dogs and cats.IMPORTANCEThe leopard cat (Prionailurus bengalensis) is an endangered species in Taiwan. The effects of infectious diseases on the wildlife population have increasingly been recognized. In this study, we targeted highly pathogenic viral pathogens in wild cat species, included carnivore protoparvovirus 1 (CPPV-1), feline leukemia virus (FeLV), feline immunodeficiency virus (FIV), coronavirus (CoV), and canine morbillivirus (CMV), using molecular screening. Furthermore, we collected the epidemiological and phylogenetic data to understand the spatial and temporal dynamics of the target pathogens in the wild leopard cat population and identified the possible origin of target pathogens. Based on our study, we consider sympatric domestic carnivores to be the probable primary reservoir for the pathogens identified. Our study provides a deeper understanding related to the distribution of target viral pathogens in the wild leopard cats. The information is essential for leopard cat conservation and pathogen management.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Chen-Chih Chen; Ai-Mei Chang; Wan-Jhen Chen; Po-Jen Chang; Yu-Ching Lai; Hsu-Hsun Lee&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.21.960492&#39;},
 {&#39;cord_uid&#39;: &#39;o3hytzwu&#39;,
  &#39;sha&#39;: &#39;54d2fd5689ac3255e8041c36b5fb333a3bd3f9b2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Novel Method for the Estimation of a Dynamic Effective Reproduction Number (Dynamic-R) in the CoViD-19 Outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20023267&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The CoViD-19 outbreak has escalated to a pandemic in the last few weeks, with no signs of stopping. Pharmaceutical solutions based upon virologic studies, at this point, remain inconclusive. In contrast, this paper looks towards epidemiological models during this phase of viral growth, in particular, by providing a responsive, timely model of the R value based on the previous few days&#39; results. Such an R value, although bearing less statistical precision due to limited sampling, could allow R to become a more effective, responsive standalone measure of infectious transmission. It demonstrates that the R value can be used as a dynamic, time-dependent indicator without the use of curve-fitting, and also estimates the most recent R-value of the CoViD-19 outbreak to be about 4.29, based on the data from the previous 3 days.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Yi Chen Chong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20023267&#39;},
 {&#39;cord_uid&#39;: &#39;y24qtqjj&#39;,
  &#39;sha&#39;: &#39;4f8d24c531d2c334969e09e4b5aed66dcc925c4b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Association of radiologic findings with mortality of patients infected with 2019 novel coronavirus in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20024927&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Radiologic characteristics of 2019 novel coronavirus (2019-nCoV) infected pneumonia (NCIP) which had not been fully understood are especially important for diagnosing and predicting prognosis. We retrospective studied 27 consecutive patients who were confirmed NCIP, the clinical characteristics and CT image findings were collected, and the association of radiologic findings with mortality of patients was evaluated. 27 patients included 12 men and 15 women, with median age of 60 years (IQR 47-69). 17 patients discharged in recovered condition and 10 patients died in hospital. The median age of mortality group was higher compared to survival group (68 (IQR 63-73) vs 55 (IQR 35-60), P = 0.003). The comorbidity rate in mortality group was significantly higher than in survival group (80% vs 29%, P = 0.018). The predominant CT characteristics consisted of ground glass opacity (67%), bilateral sides involved (86%), both peripheral and central distribution (74%), and lower zone involvement (96%). The median CT score of mortality group was higher compared to survival group (30 (IQR 7-13) vs 12 (IQR 11-43), P = 0.021), with more frequency of consolidation (40% vs 6%, P = 0.047) and air bronchogram (60% vs 12%, P = 0.025). An optimal cutoff value of a CT score of 24.5 had a sensitivity of 85.6% and a specificity of 84.5% for the prediction of mortality. 2019-nCoV was more likely to infect elderly people with chronic comorbidities. CT findings of NCIP were featured by predominant ground glass opacities mixed with consolidations, mainly peripheral or combined peripheral and central distributions, bilateral and lower lung zones being mostly involved. A simple CT scoring method was capable to predict mortality.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Mingli Yuan; Wen Yin; Zhaowu Tao; Weijun Tan; Yi Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20024927&#39;},
 {&#39;cord_uid&#39;: &#39;pi5uq90z&#39;,
  &#39;sha&#39;: &#39;5a9164999237a46d3d6aed53bd59b17d91a9e6d0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 transmission in cancer patients of a tertiary hospital in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20025320&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In December 2019, an outbreak of atypical pneumonia known as 2019 novel coronavirus disease (COVID-19) occurred in Wuhan, China. This new type of pneumonia is characterized by rapid human-to-human transmission. Among the different disease types, cancer patients are often recalled to the hospital for treatment and disease surveillance, and the majority of cancer treatments such as chemotherapy and radiotherapy are immunosuppressive. This prompts us to consider if cancer patients were at an elevated risk of SARS-CoV-2 infection. A total of 1,524 cancer patients who were managed at our tertiary cancer institution-Zhongnan hospital of Wuhan University were reviewed during the period of Dec 30, 2019 to Feb 17, 2020. It was found that cancer patients had an estimated 2-fold increased risk of COVID-19 than the general population. We identified twelve patients who were infected with SARS-CoV-2, with two recorded deaths (16.7%), albeit one patient passed away from a COVID-19 unrelated cause. Interestingly, only five of these patients were ongoing treatment at the time of contracting the virus, suggesting that hospital visitation was the likely factor contributing to the elevated incidence in cancer patients. Moreover, we also observed that the incidence of severe COVID-19 was not higher than in the general population. Consequently, for cancer patients who require treatment, proper isolation protocols must be in place to mitigate the risk of SARS-CoV-2 infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Jing Yu; Wen Ouyang; Melvin L.K. Chua; Conghua Xie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20025320&#39;},
 {&#39;cord_uid&#39;: &#39;hth8f5sn&#39;,
  &#39;sha&#39;: &#39;04030bba3035a58c7725ae267973206f6eb6c0b4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20025460&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third zoonotic CoV has raised great public health concern. The demand for rapid and accurate diagnosis of this novel pathogen brought significant clinical and technological challenges. Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR (RT-PCR) are the most widely used molecular diagnostics. Methods: 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was analyzed and used for the design and development of an isothermal, CRISPR-based diagnostic for this novel virus. The diagnostic performance of CRISPR-nCoV was assessed and compared across three technology platforms (mNGS, RT-PCR and CRISPR). Results: 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral RNA was observed in clinical specimens. CRISPR-nCoV demonstrated a near single-copy sensitivity and great clinical sensitivity with a shorter turn-around time than RT-PCR. Conclusion: CRISPR-nCoV presents as a promising diagnostic option for the emerging pathogen.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-23&#39;,
  &#39;authors&#39;: &#39;Tieying Hou; Weiqi Zeng; Minling Yang; Wenjing Chen; Lili Ren; Jingwen Ai; Ji Wu; Yalong Liao; Xuejing Gou; Yongjun Li; Xiaorui Wang; Hang Su; Jianwei Wang; Bing Gu; Teng Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20025460&#39;},
 {&#39;cord_uid&#39;: &#39;qc1mpzli&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Temperature significant change COVID-19 Transmission in 429 cities&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20025791&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background There is no evidence supporting that temperature changes COVID-19 transmission. Methods We collected the cumulative number of confirmed cases of all cities and regions affected by COVID-19 in the world from January 20 to February 4, 2020, and calculated the daily means of the average, minimum and maximum temperatures in January. Then, restricted cubic spline function and generalized linear mixture model were used to analyze the relationships. Results There were in total 24,232 confirmed cases in China and 26 overseas countries. In total, 16,480 cases (68.01%) were from Hubei Province. The lgN rose as the average temperature went up to a peak of 8.72℃ and then slowly declined. The apexes of the minimum temperature and the maximum temperature were 6.70℃ and 12.42℃ respectively. The curves shared similar shapes. Under the circumstance of lower temperature, every 1℃ increase in average, minimum and maximum temperatures led to an increase of the cumulative number of cases by 0.83, 0.82 and 0.83 respectively. In the single-factor model of the higher-temperature group, every 1℃ increase in the minimum temperature led to a decrease of the cumulative number of cases by 0.86. Conclusion The study found that, to certain extent, temperature could significant change COVID-19 transmission, and there might be a best temperature for the viral transmission, which may partly explain why it first broke out in Wuhan. It is suggested that countries and regions with a lower temperature in the world adopt the strictest control measures to prevent future reversal.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Mao Wang; Aili Jiang; Lijuan Gong; Lina Luo; Wenbin Guo; Chuyi Li; Jing Zheng; Chaoyong Li; Bixing Yang; Jietong Zeng; Youping Chen; Ke Zheng; Hongyan Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20025791&#39;},
 {&#39;cord_uid&#39;: &#39;niw61l9r&#39;,
  &#39;sha&#39;: &#39;7fd337f16780aba8f1e599ce7516dc9b1d80a546&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Neurological Manifestations of Hospitalized Patients with COVID-19 in Wuhan, China: a retrospective case series study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20026500&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;OBJECTIVE: To study the neurological manifestations of patients with coronavirus disease 2019 (COVID-19). DESIGN: Retrospective case series SETTING: Three designated COVID-19 care hospitals of the Union Hospital of Huazhong University of Science and Technology in Wuhan, China. PARTICIPANTS: Two hundred fourteen hospitalized patients with laboratory confirmed diagnosis of severe acute respiratory syndrome from coronavirus 2 (SARS-CoV-2) infection. Data were collected from 16 January 2020 to 19 February 2020. MAIN OUTCOME MEASURES: Clinical data were extracted from electronic medical records and reviewed by a trained team of physicians. Neurological symptoms fall into three categories: central nervous system (CNS) symptoms or diseases (headache, dizziness, impaired consciousness, ataxia, acute cerebrovascular disease, and epilepsy), peripheral nervous system (PNS) symptoms (hypogeusia, hyposmia, hypopsia, and neuralgia), and skeletal muscular symptoms. Data of all neurological symptoms were checked by two trained neurologists. RESULTS: Of 214 patients studied, 88 (41.1%) were severe and 126 (58.9%) were non-severe patients. Compared with non-severe patients, severe patients were older (58.7 ± 15.0 years vs 48.9 ± 14.7 years), had more underlying disorders (42 [47.7%] vs 41 [32.5%]), especially hypertension (32 [36.4%] vs 19 [15.1%]), and showed less typical symptoms such as fever (40 [45.5%] vs 92 [73%]) and cough (30 [34.1%] vs 77 [61.1%]). Seventy-eight (36.4%) patients had neurologic manifestations. More severe patients were likely to have neurologic symptoms (40 [45.5%] vs 38 [30.2%]), such as acute cerebrovascular diseases (5 [5.7%] vs 1 [0.8%]), impaired consciousness (13 [14.8%] vs 3 [2.4%]) and skeletal muscle injury (17 [19.3%] vs 6 [4.8%]). CONCLUSION: Compared with non-severe patients with COVID-19, severe patients commonly had neurologic symptoms manifested as acute cerebrovascular diseases, consciousness impairment and skeletal muscle symptoms.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Ling Mao; Mengdie Wang; Shanghai Chen; Quanwei He; Jiang Chang; Candong Hong; Yifan Zhou; David Wang; Yanan Li; Huijuan Jin; Bo Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20026500&#39;},
 {&#39;cord_uid&#39;: &#39;tb37l9yp&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Peptide-based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Corona Virus Disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20026617&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A respiratory illness has been spreading rapidly in China, since its outbreak in Wuhan city, Hubei province in December 2019. The illness was caused by a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Clinical manifestations related to SARS-CoV-2 infection ranged from no symptom to fatal pneumonia. World Health Organization (WHO) named the diseases associated with SARS-CoV-2 infection as COVID-19. Real time RT-PCR is the only laboratory test available till now to confirm the infection. However, the accuracy of real time RT-PCR depends on many factors, including sampling location and of methods, quality of RNA extraction and training of operators etc. Variations in these factors might significantly lower the sensitivity of the detection. We developed a peptide-based luminescent immunoassay to detect IgG and IgM. Cut-off value of this assay was determined by the detection of 200 healthy sera and 167 sera from patients infected with other pathogens than SARS-CoV-2. To evaluate the performance of this assay, we detected IgG and IgM in the 276 sera from confirmed patients. The positive rate of IgG and IgM were 71.4% (197/276) and 57.2% (158/276) respectively. By combining with real time RT-PCR detection, this assay might help to enhance the accuracy of diagnosis of SARS-CoV-2 infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Xuefei Cai; Juan Chen; Jieli Hu; Quanxin Long; Haijun Deng; Kai Fan; Pu Liao; Beizhong Liu; Guicheng Wu; Yaokai Chen; Zhijie Li; Kun Wang; Xiaoli Zhang; Wenguang Tian; Jianglin Xiang; Hongxin Du; Jing Wang; Yuan Hu; Ni Tang; Yong Lin; Jihua Ren; Luyi Huang; Jie Wei; Chunyang Gan; Yanmeng Chen; Qingzhu Gao; Amei Chen; Changlong He; Daoxin Wang; Peng Hu; Fachun Zhou; Ailong Huang; Ping Liu; Deqiang Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20026617&#39;},
 {&#39;cord_uid&#39;: &#39;sjod3q3j&#39;,
  &#39;sha&#39;: &#39;eeae21c913759e8cf4c345e561457fe7ef84480f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Applying chemical reaction transition theory to predict the latent transmission dynamics of coronavirus outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.20026815&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The recent outbreak of the Covid-19 suggests a rather long latent phase that precludes public health officials to predict the pandemic transmission on time. Here we apply mass action laws and chemical transition theory to propose a kinetic model that accounts for viral transmission dynamics at the latent phase. This model is useful for authorities to make early preventions and control measurements that stop the spread of a deadly new virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Peng Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.20026815&#39;},
 {&#39;cord_uid&#39;: &#39;dqxfcwyu&#39;,
  &#39;sha&#39;: &#39;eb7fc5004191e8fce0d6e4ff98de6865872e665f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.951178&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Tamina Park; Sang-Yeop Lee; Seil Kim; Mi Jeong Kim; Hong Gi Kim; Sangmi Jun; Seung Il Kim; Bum Tae Kim; Edmond Changkyun Park; Daeui Park&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.951178&#39;},
 {&#39;cord_uid&#39;: &#39;nbu6bsb3&#39;,
  &#39;sha&#39;: &#39;7207df5b61e93fa5a2b6e720c91aed05270e81be&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.22.961268&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;1AbstractThe ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Zhen Zhao; Haodong Cui; Wenxing Song; Xiaoling Ru; Wenhua Zhou; Xuefeng Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.22.961268&#39;},
 {&#39;cord_uid&#39;: &#39;ne0qwv56&#39;,
  &#39;sha&#39;: &#39;8b2e50eb3ea84225580fbdbccccb4fcd3f062feb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Intrinsic growth rules of patients infected, dead and cured with 2019 novel coronavirus in mainland China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20024802&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background An outbreak of a novel coronavirus (SARS-CoV-2)-infected pneumonia (COVID-19) was first occurred in Wuhan, China, in December 2019 and then spread rapidly to other regions. It has been declared that at least one confirmed case infected by SARS-CoV-2 was found in each province of China by late January 2020. Methods We collected the time series data of the cumulative number of confirmed infected, dead, and cured cases from the health commissions in 31 provinces in mainland China. A simple descriptive model in a logistic form was formulated to infer the intrinsic epidemic rules of COVID-19. Furthermore, we compared the intrinsic epidemic rules of COVID-19 in Hubei with that of the severe acute respiratory syndrome (SARS) in Beijing, which was obtained from the Ministry of Public Health of China in 2003. Results The feature of data of three population groups in all the provinces could be well captured (R-square&gt;0.95) by the proposed descriptive model with slight variation among different provinces. By comparing fitted parameters, we found that the fitted curve of the infected case is the earliest to be saturated and has the lowest semi-saturation period in both COVID-19 and SARS. While all three population groups of SARS are later to saturate and have a longer semi-saturation period than that of COVID-19. Conclusions Our model is robust and stable to depict the intrinsic growth rule for the cumulative number of confirmed infected, dead, and cured cases in 30 provinces, which could depict the epidemic situation in mainland China very well. Despite the virus caused SARS (SARS-CoV) and the virus caused COVID-19 (SARS-CoV-2) are homologous, their epidemiologic characteristics are quite different. The duration of the outbreak would be shorter for COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Chuanliang Han; Yimeng Liu; Saini Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20024802&#39;},
 {&#39;cord_uid&#39;: &#39;oly7a3c2&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Conjunctival polymerase chain reaction-tests of 2019 novel coronavirus in patients in Shenyang,China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20024935&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Purpose: The 2019 novel coronavirus(COVID-19) mainly transmitted by person-to-person through inhalation of respiratory droplets. We report the laboratory results of conjunctival PCR-tests and some clinical features of these patients in shenyang China. Design: This is a cross-sectional non-randomized study. Subjects: The study include 14 confirmly diagnosed cases, 16 suspected cases and some medical observed patients. Methods: All patients with diagnosed and suspected COVID-19 were admitted to a designated hospital in Shenyang,China. We collected conjunctival samples of these patients to do the laboratory tests by real time RT-PCR.Medical observed patients were enrolled if they had clinical symptoms. Then we analysed the PCR results and clinical data from eletronic medical records in order to find some relationships. Main Outcome Measures: Clinical condition and PCR results of conjunctival swabs compared with other specimens. Results: One of the identified case coverted from suspected case without typical clinical symptoms. Twenty-two medical observed cases were removed because none of them converted to identified cases.One of the suspected converted to identified case recently.The included cases in our study are imported cases with less underlying diseases and the severity of their infection was relatively moderate. All the conjunctival results of PCR-test were negative. Two cases had typical clinical symptoms but were finally confirmed by repeated pharynx swab tests. Conclusion: Conjunctiva may be a transmission way of COVID-19. And ocular conjunctival swabs in combination with PCR test could be a non-invasive,convenient and feasible diagnostic method for identifying the infection of COVID-19. Emphasis on the false-negative results is vital.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;li Xu; Xinyue Zhang; Wei Song; Baijun Sun; Jinping Mu; Xue Dong; Bing Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20024935&#39;},
 {&#39;cord_uid&#39;: &#39;8l1vfsbc&#39;,
  &#39;sha&#39;: &#39;bbba5ff9ab9eccef99a0bbc6ea557a373158bbcd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20026690&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus SARS-CoV-2, etiological agent of recently named Coronavirus infected disease (COVID-19) by WHO, has caused more than 2, 000 deaths worldwide since its emergency in Wuhan City, Hubei province, China, in December, 2019. The symptoms of COVID-19 varied from modest, mild to acute respiratory distress syndrome (ARDS), and the latter of which is generally associated with deregulated immune cytokine production; however, we currently know little as to the interplay between the extent of clinical symptoms and the compositions of lung immune microenvironment. Here, we comprehensively characterized the lung immune microenvironment with the bronchoalveolar lavage fluid (BALF) from 3 severe and 3 mild COVID-19 patients and 8 previously reported healthy lung controls through single-cell RNA sequence (scRNA-seq) combined with TCR-seq. Our data shows that monocyte-derived FCN1+ macrophages, whereas notFABP4+ alveolar macrophages that represent a predominant macrophage subset in BALF from patients with mild diseases, overwhelm in the severely damaged lungs from patients with ARDS. These cells are highly inflammatory and enormous chemokine producers implicated in cytokine storm. Furthermore, the formation of tissue resident, highly expanded clonal CD8+ T cells in the lung microenvironment of mild symptom patients suggests a robust adaptive immune response connected to a better control of COVID-19. This study first reported the cellular atlas of lung bronchoalveolar immune microenvironment in COVID-19 patients at the single-cell resolution, and unveiled the potential immune mechanisms underlying disease progression and protection in COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Minfeng Liao; Yang Liu; Jin Yuan; Yanling Wen; Gang Xu; Juanjuan Zhao; Lin Chen; Jinxiu Li; Xin Wang; Fuxiang Wang; Lei Liu; Shuye Zhang; Zheng Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20026690&#39;},
 {&#39;cord_uid&#39;: &#39;jxtch47t&#39;,
  &#39;sha&#39;: &#39;a9c58042555c0d56b8e94bc5558db0c3dd697dcc&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiologic and Clinical Characteristics of 91 Hospitalized Patients with COVID-19 in Zhejiang, China: A retrospective, multi-centre case series&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20026856&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Recent studies have focused initial clinical and Epidemiologic characteristics on the COVID-19, mainly revealing situation in Wuhan, Hubei. Aim To reveal more data on the epidemiologic and clinical characteristics of COVID-19 patients outside of Wuhan, in Zhejiang, China. Design Retrospective case series. Methods 88 cases of laboratory-confirmed and 3 cases of clinical-confirmed COVID-19 were admitted to five hospitals in Zhejiang province, China. Data were collected from 20 January 2020 to 11 February 2020. Results Of all 91 patients, 88 (96.70%) were laboratory-confirmed COVID-19 with throat swab samples that tested positive for SARS-Cov-2 while 3 (3.30%) were clinical-diagnosed COVID-19 cases. The median age of the patients was 50 (36.5-57) years, and female accounted for 59.34%. In this sample 40 (43.96%) patients had contracted the diseases from local cases, 31 (34.07%) patients had been to Wuhan/Hubei, 8 (8.79%) cases had contacted with people from Wuhan, 11 (12.09%) cases were confirmed aircraft transmission. In particular within the city of Ningbo, 60.52% cases can be traced back to an event held in a temple. The most common symptoms were fever (71.43%), cough (60.44%) and fatigue (43.96%). The median of incubation period was 6 (IQR, 3-8) days and the median time from first visit to a doctor to confirmed diagnosis was 1 (1-2) days. According to the Chest computed tomography scans, 67.03% cases had bilateral pneumonia. Conclusions Social activity cluster, family cluster and travel by airplane were how COVID-19 patients get transmitted and could be rapidly diagnosed COVID-19 in Zhejiang.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Guo-Qing Qian; Nai-Bin Yang; Feng Ding; Ada Hoi Yan Ma; Zong-Yi Wang; Yue-Fei Shen; Chun-Wei Shi; Xiang Lian; Jin-Guo Chu; Lei Chen; Zhi-Yu Wang; Da-Wei Ren; Guo-Xiang Li; Xue-Qin Chen; Hua-Jiang Shen; Xiao-Min Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20026856&#39;},
 {&#39;cord_uid&#39;: &#39;oeqf7uk9&#39;,
  &#39;sha&#39;: &#39;bf39a81c9aab9e0569f33d93d5e203d320dac024&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Gender differences in patients with COVID-19: Focus on severity and mortality&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20026864&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: The recent outbreak of Novel Coronavirus (SARS-CoV-2) Disease (COVID-19) has put the world on alert, that is reminiscent of the SARS outbreak seventeen years ago. Objective: We aim to compare the severity and mortality between male and female patients with both COVID-19 and SARS, to explore the most useful prognostic factors for individualized assessment. Design, Setting, and Participants: We extracted the data from a case series of 43 hospitalized patients we treated, a public data set of the first 37 cases died of COVID-19 in Wuhan city and 1019 survived patients from six cities in China. We also analyzed the data of 524 patients with SARS, including 139 deaths, from Beijing city in early 2003. Main Outcomes and Measures: Severity and mortality. Results: Older age and high number of comorbidities were associated with higher severity and mortality in patients with both COVID-19 and SARS. The percentages of older age (≥65 years) were much higher in the deceased group than in the survived group in patients with both COVID-19 (83.8 vs. 13.2, P&lt;0.001) and SARS (37.4 vs. 4.9, P&lt;0.001). In the case series, men tend to be more serious than women (P=0.035), although age was comparable between men and women. In the public data set, age was also comparable between men and women in the deceased group or the survived group in patients with COVID-19. Meanwhile, gender distribution was exactly symmetrical in the 1019 survivors of COVID-19. However, the percentage of male were higher in the deceased group than in the survived group (70.3 vs. 50.0, P=0.015). The gender role in mortality was also observed in SARS patients. Survival analysis showed that men (hazard ratio [95% CI] 1.47 [1.05-2.06, P= 0.025) had a significantly higher mortality rate than women in patients with SARS. Conclusions and Relevance: Older age and male gender are risk factors for worse outcome in patients with COVID. While men and women have the same susceptibility to both SARS-CoV-2 and SARS-CoV, men may be more prone to have higher severity and mortality independent of age and susceptibility.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Jian-Min Jin; Peng Bai; Wei He; Fei Wu; Xiao-Fang Liu; De-Min Han; Shi Liu; Jin-Kui Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20026864&#39;},
 {&#39;cord_uid&#39;: &#39;mlyg4hj0&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Mental health status and coping strategy of medical workers in China during The COVID-19 outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20026872&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The authors have withdrawn their manuscript whilst they perform additional experiments to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Chen Siyu; Min Xia; Weiping Wen; Liqian Cui; Weiqiang Yang; Shaokun Liu; Jiahua Fan Fan; Huijun Yue; Shangqing Tang; Bingjie Tang; Xiaoling Li; Lin Chen; Zili Qin; Kexing Lv; Xueqin Guo; Yu Lin; Yihui Wen; Wenxiang Gao; Ying Zheng; Wei Xu; Yun Li; Yang Xu; Li Ling; Wenbin Lei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20026872&#39;},
 {&#39;cord_uid&#39;: &#39;zqsy6r4i&#39;,
  &#39;sha&#39;: &#39;6bdd6867205d9b09f833f01c65f1bbfdae469164&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Deep learning Enables Accurate Diagnosis of Novel Coronavirus (COVID-19) with CT images&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20026930&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background A novel coronavirus (COVID-19) has emerged recently as an acute respiratory syndrome. The outbreak was originally reported in Wuhan, China, but has subsequently been spread world-widely. As the COVID-19 continues to spread rapidly across the world, computed tomography (CT) has become essentially important for fast diagnoses. Thus, it is urgent to develop an accurate computer-aided method to assist clinicians to identify COVID-19-infected patients by CT images. Materials and Methods We collected chest CT scans of 88 patients diagnosed with the COVID-19 from hospitals of two provinces in China, 101 patients infected with bacteria pneumonia, and 86 healthy persons for comparison and modeling. Based on the collected dataset, a deep learning-based CT diagnosis system (DeepPneumonia) was developed to identify patients with COVID-19. Results The experimental results showed that our model can accurately identify the COVID-19 patients from others with an excellent AUC of 0.99 and recall (sensitivity) of 0.93. In addition, our model was capable of discriminating the COVID-19 infected patients and bacteria pneumonia-infected patients with an AUC of 0.95, recall (sensitivity) of 0.96. Moreover, our model could localize the main lesion features, especially the ground-glass opacity (GGO) that is of great help to assist doctors in diagnosis. The diagnosis for a patient could be finished in 30 seconds, and the implementation on Tianhe-2 supercompueter enables a parallel executions of thousands of tasks simultaneously. An online server is available for online diagnoses with CT images by http://biomed.nscc-gz.cn/server/Ncov2019. Conclusions The established models can achieve a rapid and accurate identification of COVID-19 in human samples, thereby allowing identification of patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Ying Song; Shuangjia Zheng; Liang Li; Xiang Zhang; Xiaodong Zhang; Ziwang Huang; Jianwen Chen; Huiying Zhao; Yusheng Jie; Ruixuan Wang; Yutian Chong; Jun Shen; Yunfei Zha; Yuedong Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20026930&#39;},
 {&#39;cord_uid&#39;: &#39;jtsxpjhc&#39;,
  &#39;sha&#39;: &#39;e4701bdf5d414ea8ca38a4d8d3f69540805293b0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical features and laboratory inspection of novel coronavirus pneumonia (COVID-19) in Xiangyang, Hubei&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.20026963&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since December 2019, a novel coronavirus pneumonia (COVID-19) rapidly spread in China, reached multiple continents currently.We aimed to reveal the infectious characteristics of COVID-19 that provide more information for the research of novel coronavirus. Methods: We performed a retrospective study on the clinical characteristics of 128 COVID-19 cases with laboratory-confirmed from Xiangyang No 1 Hospitalad during January 2020 to 16 February 2020. Results: Female patients account for 53.1%. The aged below 20 years that accounts for 1.6% of overall patients. The aged in 21~50, 51~65, over 66 years were accounts for 44.5%, 35.1%,18.8%, respectively. In the difference age spectrum, all severe groups compared with non-severe groups were difference significantly ( P &lt; 0.01 ). Fever ( 89.8% ) and Cough ( 67.2% ) were common clinical symptoms. The rate of patients with sore throats (14.1%) was rare. The rate of chest computed tomography scan showing ground glass opacity in overall, non-severe, severe groups were 63.3%, 60.7%, 76.2%, respectively. White blood cell counts in the normal range of overall patients, but severe group patients were increased significantly ( P &lt; 0.01). Lymphocytes of overall patients were decreased. Alanine transaminase (ALT) and aspartate transaminase (AST) in the normal range of overall patients, but its were elevated in the severe group. Creatinine (CR) and blood urea nitrogen (BUN) of overall patients in the normal range. C-reactive protein (CRP) level of all patients were increased markedly, but it in the severe group was significantly higher than that in the non-severe group ( P &lt; 0.01 ). Conclusions: Our data provide more information that advanced age, lower lymphocytes levels at the diagnosed COVID-19 patients may be a risk factor for unfavourable prognosis. The white blood cells and C-reactive protein level elevated in severe COVID-19 patients may be accompanying bacterial infection. 2019-nCov may be carries a risk factor of impaired liver and kidney function.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Weiliang Cao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.20026963&#39;},
 {&#39;cord_uid&#39;: &#39;25va2cvt&#39;,
  &#39;sha&#39;: &#39;09b6706748f0c1ae0da436ac2dfac9052b84e4ea&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Cryo-EM structures of HKU2 and SADS-CoV spike glycoproteins and insights into coronavirus evolution&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.23.961912&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-24&#39;,
  &#39;authors&#39;: &#39;Jinfang Yu; Shuyuan Qiao; Runyu Guo; Xinquan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.23.961912&#39;},
 {&#39;cord_uid&#39;: &#39;nqn1qzcu&#39;,
  &#39;sha&#39;: &#39;1d882d90109fb4496088342cf2bcaa4fec40cf9f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Inferring transmission trees to guide targeting of interventions against visceral leishmaniasis and post-kala-azar dermal leishmaniasis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20023325&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Understanding of spatiotemporal transmission of infectious diseases has improved significantly in recent years. Advances in Bayesian inference methods for individual-level geo-located epidemiological data have enabled reconstruction of transmission trees and quantification of disease spread in space and time, while accounting for uncertainty in missing data. However, these methods have rarely been applied to endemic diseases or ones in which asymptomatic infection plays a role, for which novel estimation methods are required. Here, we develop such methods to analyse longitudinal incidence data on visceral leishmaniasis (VL), and its sequela, post-kala-azar dermal leishmaniasis (PKDL), in a highly endemic community in Bangladesh. Incorporating recent data on infectiousness of VL and PKDL, we show that while VL cases drive transmission when incidence is high, the contribution of PKDL increases significantly as VL incidence declines (reaching 55% in this setting). Transmission is highly focal: &gt;85% of mean distances from inferred infectors to their secondary VL cases were &lt;300m, and estimated average times from infector onset to secondary case infection were &lt;4 months for 90% of VL infectors, but up to 2.75yrs for PKDL infectors. Estimated numbers of secondary VL cases per VL and PKDL case varied from 0-6 and were strongly correlated with the infector&#39;s duration of symptoms. Counterfactual simulations suggest that prevention of PKDL could have reduced VL incidence by up to a quarter. These results highlight the need for prompt detection and treatment of PKDL to achieve VL elimination in the Indian subcontinent and provide quantitative estimates to guide spatiotemporally-targeted interventions against VL.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Lloyd A. C. Chapman; Simon E. F. Spencer; Timothy M. Pollington; Chris P. Jewell; Dinesh Mondal; Jorge Alvar; T. Deirdre Hollingsworth; Mary M. Cameron; Caryn Bern; Graham F. Medley&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20023325&#39;},
 {&#39;cord_uid&#39;: &#39;f21dknmb&#39;,
  &#39;sha&#39;: &#39;01d162d7fae6aaba8e6e60e563ef4c2fca7b0e18&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20025437&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the &#34;coronavirus&#34; should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Yiqing Xue; Chunli Zhang; Guangming Tan; Gang Niu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20025437&#39;},
 {&#39;cord_uid&#39;: &#39;r31pgnu0&#39;,
  &#39;sha&#39;: &#39;47f65bff75b7752a38ccbb540fbd74955b3addea&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Demonstration of four entities of appendicitis in China through studying cluster/outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20026021&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To differentiate different entities of appendicitis through studying cluster/outbreak, ascertain common setting of cluster/outbreak, and provide the epidemiological evidences of infectious etiology of appendicitis. Background: Differential diagnosis and management for perforated appendicitis and non-perforated appendicitis and the infectious etiology of appendicitis are current hot topics. Methods: Field investigation for Tibetan students were carried out and reports published in English and Chinese medical journals were reviewed. Results: The literature review included 473 patients in 7 cluster/outbreaks of appendicitis in 6 provinces and autonomous regions. All the clusters/outbreaks occurred in group living units. We found two classic entities of appendicitis with natural history from non perforated appendicitis to perforated appendicitis and two entities of non-perforated appendicitis . In classic entities, one may represent majority of sporadic patients and the other may represent partial sporadic patients with obvious gastrointestinal manifestation. In entities of non-perforated appendicitis, one was identical to features of sporadic non-perforated appendicitis and the other one is identical to the following Tibetan students and associated with Fusobacterium. The field investigation for 120 Tibetan students with appendicitis showed that the resected appendices exhibited diffuse or focal hemorrhages and infiltration by eosinophils and by lymphocytes. Most patients had normal body temperature, white blood cell count and neutrophil count. This is a new entity of appendicitis. The clusters/outbreaks of appendicitis showed the features of infectious disease in epidemiology. The entity of perforated appendicitis was not found. Conclusion: Studying cluster/outbreak is a good method to differentiate different entities of appendicitis and infectious etiology.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Yi-Tian Guo; De-qiang Ye; Gui-fang Yang; Guo-zhen Liu; Xiao-chen Cui; Shi-Yun Tan; Yi Guo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20026021&#39;},
 {&#39;cord_uid&#39;: &#39;g3asp1ps&#39;,
  &#39;sha&#39;: &#39;99a46ba28f5811c112876ec77f570d6c10778db6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20026682&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using &#39;Feidian&#39;, Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Wenjun Wang; Yikai Wang; Xin Zhang; Yaping Li; Xiaoli Jia; Shuangsuo Dang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20026682&#39;},
 {&#39;cord_uid&#39;: &#39;rr7r7ef0&#39;,
  &#39;sha&#39;: &#39;0fd300aefb704c20f32152b97b6194015f1c74e7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Characterizing the transmission and identifying the control strategy for COVID-19 through epidemiological modeling&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20026773&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Ke K. Zhang; Linglin Xie; Lauren Lawless; Huijuan Zhou; Guannan Gao; Chengbin Xue&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20026773&#39;},
 {&#39;cord_uid&#39;: &#39;tkvgzbuk&#39;,
  &#39;sha&#39;: &#39;5b1a2b3207e3f674b94f6f271b92c29f3a8a6754&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical and radiographic features of cardiac injury in patients with 2019 novel coronavirus pneumonia&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027052&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To investigate the correlation between clinical characteristics and cardiac injury of COVID-2019 pneumonia. Methods: In this retrospective, single-center study, 41 consecutive corona virus disease 2019 (COVID-2019) patients (including 2 deaths) of COVID-2019 in Beijing Youan Hospital, China Jan 21 to Feb 03, 2020, were involved in this study. The high risk factors of cardiac injury in different COVID-2019 patients were analyzed. Computed tomographic (CT) imaging of epicardial adipose tissue (EAT) has been used to demonstrate the cardiac inflammation of COVID-2019. Results：Of the 41 COVID-2019 patients, 2 (4.88%), 32 (78.05%), 4 (9.75%) and 3 (7.32%) patients were clinically diagnosed as light, mild, severe and critical cases, according to the 6th guidance issued by the National Health Commission of China. 10 (24.4%) patients had underlying complications, such as hypertension, CAD, type 2 diabetes mellites and tumor. The peak value of TnI in critical patients is 40-fold more than normal value. 2 patients in the critical group had the onset of atrial fibrillation, and the peak heart rates reached up to 160 bpm. CT scan showed low EAT density in severe and critical patients. Conclusion: Our results indicated that cardiac injury of COVID-2019 was rare in light and mild patients, while common in severe and critical patients. Therefore, the monitoring of the heart functions of COVID-2019 patients and applying potential interventions for those with abnormal cardiac injury related characteristics, is vital to prevent the fatality.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Hui Hui; Yingqian Zhang; Xin Yang; Xi Wang; Bingxi He; Li Li; Hongjun Li; Jie Tian; Yundai Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027052&#39;},
 {&#39;cord_uid&#39;: &#39;b2znv6pa&#39;,
  &#39;sha&#39;: &#39;d9eeeb81d17be6a722b11081f13576de197d249f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;2019 novel coronavirus disease in hemodialysis (HD) patients: Report from one HD center in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027201&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of COVID-19 originated in Wuhan has become a global epidemic of contagious diseases, which poses a serious threat to human life and health, especially for those with underlined diseases. However, Impacts of COVID-19 epidemic on HD center and HD patients are still unknown. In this report, we reviewed the whole course of the epidemic emerged in the HD center of Renmin Hospital, Wuhan University from January 14, 2020, the day the first case was confirmed, to February 17, 2020, the day the epidemic extinction. There are totally 37 cases among 230 HD patients and 4 cases among 33 staff being diagnosed with COVID-19. The epidemiology, clinical presentation and immune profile of dialysis patients contracted COVID-19 were further studied. We found that the two key measures we took in response to the epidemic, one was upgrading level of prevention and protection on January 21 and the other one starting universal screening, isolating, and distributing the infected cases on February 4, were effective in the epidemic control. No new COVID-19 case had been diagnosed since February 13. During the epidemic, 7 HD patients died, including 6 with COVID-19 and 1 without COVID-19. The presumed causes of death were not directly related to pneumonia, but due to cardiovascular and cerebrovascular diseases, hyperkalemia, etc. Most of the leukocytes in the peripheral blood of the HD patients infected with COVID-19 decreased, and the CT images of the chest mostly showed the ground glass like changes on the right side. The symptoms of most of the patients were mild, and there were no cases admitted to ICU. The frequency of lymphocytes in PBMCs and the serum level of inflammatory cytokines were assessed in HD patients contracted COVID-19 or not, non-HD COVID-19 patients, as well as healthy volunteers. The results showed that lymphocytes of T cell, Th cells, killer T cells, as well as NK cells in PBMCs of HD patients decreased significantly than other groups. HD patients with COVID-19 also displayed remarkable lower serum level of inflammatory cytokines than other COVID-19 patients. Our study indicates that HD patients are the highly susceptible population and HD centers are high risk area in the outbreak of COVID-19 epidemic. Measures of prevention, protection, screening, isolation, and distribution are essential in the epidemic management and should be taken in the early stage. HD Patients with COVID-19 are mostly clinical mild and unlikely progress to severe pneumonia due to the impaired cellular immune function and incapability of mounting cytokines storm. More attention should be paid to prevent cardiovascular events, which may be the collateral impacts of COVID-19 epidemic on HD patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Yiqiong Ma; Bo Diao; Xifeng Lv; Jili Zhu; Wei Liang; Lei Liu; Wenduo Bu; Huiling Cheng; Sihao Zhang; Lianhua Yang; Ming Shi; Guohua Ding; Bo Shen; Huiming Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027201&#39;},
 {&#39;cord_uid&#39;: &#39;9tgnx7du&#39;,
  &#39;sha&#39;: &#39;6f53b9dff58818839c45410d11b58f0f3e327d8a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of risk factors for COVID-19 mortality - preliminary results&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027268&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since late December 2019 a new epidemic outbreak has emerged from Whuhan, China. Rapidly the new coronavirus has spread worldwide. China CDC has reported results of a descriptive exploratory analysis of all cases diagnosed until the 11th February 2020, presenting the epidemiologic curves and geo-temporal spread of COVID-19 along with case fatality rate according to some baseline characteristics, such as age, gender and several well-established high prevalence comorbidities. Despite this, we intend to increase even further the predictive value of that manuscript by presenting the odds ratio for mortality due to COVID-19 adjusted for the presence of those comorbidities and baseline characteristics such as age and gender. Besides, we present a way to determine the risk of each particular patient, given his characteristics. We found that age is the variable that presents higher risk of COVID-19 mortality, where 60 or older patients have an OR = 18.8161 (CI95%[7.1997; 41.5517]). Regarding comorbidities, cardiovascular disease appears to be the riskiest (OR=12.8328 CI95%[10.2736; 15.8643], along with chronic respiratory disease (OR=7.7925 CI95%[5.5446; 10.4319]). Males are more likely to die from COVID-19 (OR=1.8518 (CI95%[1.5996; 2.1270]). Some limitations such as the lack of information about the correct prevalence of gender per age or about comorbidities per age and gender or the assumption of independence between risk factors are expected to have a small impact on results. A final point of paramount importance is that the equation presented here can be used to determine the probability of dying from COVID-19 for a particular patient, given its age interval, gender and comorbidities associated.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Francisco Caramelo; Nuno Ferreira; Barbara Oliveiros&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027268&#39;},
 {&#39;cord_uid&#39;: &#39;ad3tct5b&#39;,
  &#39;sha&#39;: &#39;53326420d7eb607538a248c0da27d3b092f651e2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Lessons learnt from 288 COVID-19 international cases: importations over time, effect of interventions, underdetection of imported cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027326&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;288 cases have been confirmed out of China from January 3 to February 13, 2020. We collected and synthesized all available information on these cases from official sources and media. We analyzed importations that were successfully isolated and those leading to onward transmission. We modeled their number over time, in relation to the origin of travel (Hubei province, other Chinese provinces, other countries) and interventions. We characterized importations timeline to assess the rapidity of isolation, and epidemiologically linked clusters to estimate the rate of detection. We found a rapid exponential growth of importations from Hubei, combined with a slower growth from the other areas. We predicted a rebound of importations from South East Asia in the upcoming weeks. Time from travel to detection has considerably decreased since the first importation, however 6 cases out of 10 were estimated to go undetected. Countries outside China should be prepared for the possible emergence of several undetected clusters of chains of local transmissions.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Francesco Pinotti; Laura Di Domenico; Ernesto Ortega; Marco Mancastroppa; Giulia Pullano; Eugenio Valdano; Pierre-Yves Boelle; Chiara Poletto; Vittoria Colizza&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027326&#39;},
 {&#39;cord_uid&#39;: &#39;of9wlhga&#39;,
  &#39;sha&#39;: &#39;ffbd7555a337706238c211197b221795e4e35146&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of COVID-2019 burden and potential for international dissemination of infection from Iran&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027375&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The Coronavirus Disease 2019 (COVID-19) epidemic began in Wuhan, China in late 2019 and continues to spread globally, with exported cases confirmed in 28 countries at the time of writing. During the interval between February 19 and 23, 2020, Iran reported its first 43 cases with eight deaths. Three exported cases originating in Iran were identified, suggesting a underlying burden of disease in that country than is indicated by reported cases. A large epidemic in Iran could further fuel global dissemination of COVID-19. We sought to estimate COVID-19 outbreak size in Iran based on known exported case counts and air travel links between Iran and other countries, and to anticipate where infections originating in Iran may spread to next. We assessed interconnectivity between Iran and other countries using using International Air Transport Association (IATA) data. We used the methods of Fraser et al. to estimate the size of the underlying epidemic that would result in cases being observed in the United Arab Emirates (UAE), Lebanon, and Canada. Time at risk estimates were based on a presumed 6 week epidemic age, and length of stay data for visitors to Iran derived from the United Nations World Tourism Organization (UNWTO). We evaluated the relationship between the strength of travel links with Iran, and destination country rankings on the Infectious Disease Vulnerability Index (IDVI), a validated metric that estimates the capacity of a country to respond to an infectious disease outbreak . Scores range between 0-1, with higher scores reflecting greater capacity to manage infectious outbreaks. UAE, Lebanon, and Canada ranked 3rd, 21st, and 31st, respectively, for outbound air travel volume from Iran in February 2019. We estimated that 18,300 (95% confidence interval: 3770 to 53,470) COVID-19 cases would have had to occur in Iran, assuming an outbreak duration of 1.5 months in the country, in order to observe these three internationally exported cases reported at the time of writing. Results were robust under varying assumptions about undiagnosed case numbers in Syria, Azerbaijan and Iraq. Even if it were assumed that all cases were identified in all countries with certainty, the &#34;best case&#34; outbreak size was substantial (1820, 95% CI: 380-5320 cases), and far higher than reported case counts. Given the low volumes of air travel to countries with identified cases of COVID-19 with origin in Iran (such as Canada), it is likely that Iran is currently experiencing a COVID-19 epidemic of significant size for such exportations to be occurring. This is concerning, both for public health in Iran itself, and because of the high likelihood for outward dissemination of the epidemic to neighbouring countries with lower capacity to respond to infectious diseases epidemics.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Ashleigh R. Tuite; Isaac Bogoch; Ryan Sherbo; Alexander Watts; David N. Fisman; Kamran Khan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027375&#39;},
 {&#39;cord_uid&#39;: &#39;vkgnwxzc&#39;,
  &#39;sha&#39;: &#39;c1ae608c7ffb926a0f50a6a34c0780983274ea74&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimate the incubation period of coronavirus 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027474&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Motivation: Wuhan pneumonia is an acute infectious disease caused by the 2019 novel coronavirus (COVID-19). It is being treated as a Class A infectious disease though it was classified as Class B according to the Infectious Disease Prevention Act of China. Accurate estimation of the incubation period of the coronavirus is essential to the prevention and control. However, it remains unclear about its exact incubation period though it is believed that symptoms of COVID-19 can appear in as few as 2 days or as long as 14 or even more after exposure. The accurate incubation period calculation requires original chain-of-infection data that may not be fully available in the Wuhan regions. In this study, we aim to accurately calculate the incubation period of COVID-19 by taking advantage of the chain-of-infection data, which is well-documented and epidemiologically informative, outside the Wuhan regions. Methods: We acquired and collected officially reported COVID-19 data from 10 regions in China except for Hubei province. To achieve the accurate calculation of the incubation period, we only involved the officially confirmed cases with a clear history of exposure and time of onset. We excluded those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time. We proposed a Monte Caro simulation approach to estimate the incubation of COVID-19 as well as employed nonparametric ways. We also employed manifold learning and related statistical analysis to decipher the incubation relationships between different age/gender groups. Result: The incubation period of COVID-19 did not follow general incubation distributions such as lognormal, Weibull, and Gamma distributions. We estimated that the mean and median of its incubation were 5.84 and 5.0 days via bootstrap and proposed Monte Carlo simulations. We found that the incubation periods of the groups with age&gt;=40 years and age&lt;40 years demonstrated a statistically significant difference. The former group had a longer incubation period and a larger variance than the latter. It further suggested that different quarantine time should be applied to the groups for their different incubation periods. Our machine learning analysis also showed that the two groups were linearly separable. incubation of COVID-19 along with previous statistical analysis. Our results further indicated that the incubation difference between males and females did not demonstrate a statistical significance.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Henry Han&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027474&#39;},
 {&#39;cord_uid&#39;: &#39;mopydeq5&#39;,
  &#39;sha&#39;: &#39;7b5be48b7769ccad2f44a8c54efcea5281a6c633&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Spread and control of COVID-19 in China and their associations with population movement, public health emergency measures, and medical resources&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027623&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;ABSTRACT BACKGROUND The COVID-19 epidemic, first emerged in Wuhan during December 2019, has spread globally. While the mass population movement for Chinese New Year has significantly influenced spreading the disease, little direct evidence exists about the relevance to epidemic and its control of population movement from Wuhan, local emergency response, and medical resources in China. METHODS Spearman&#39;s correlation analysis was performed between official data of confirmed COVID-19 cases from Jan 20th to Feb 19th, 2020 and real-time travel data and health resources data. RESULTS There were 74,675 confirmed COVID-19 cases in China by Feb 19th, 2020. The overall fatality rate was 2.84%, much higher in Hubei than in other regions (3.27% vs 0.73%). The index of population inflow from Hubei was positively correlated with total (Provincial r=0.9159, p&lt;0.001; City r=0.6311, p&lt;0.001) and primary cases (Provincial r=0.8702, p&lt;0.001; City r=0.6358, p&lt;0.001). The local health emergency measures (eg, city lockdown and traffic control) were associated with reduced infections nationwide. Moreover, the number of public health employees per capita was inversely correlated with total cases (r=-0.6295, p&lt;0.001) and infection rates (r=-0.4912, p&lt;0.01). Similarly, cities with less medical resources had higher fatality (r=-0.4791, p&lt;0.01) and lower cure rates (r=0.5286, p&lt;0.01) among the confirmed cases. CONCLUSIONS The spread of the COVID-19 in China in its early phase was attributed primarily to population movement from Hubei, and effective governmental health emergency measures and adequate medical resources played important roles in subsequent control of epidemic and improved prognosis of affected individuals.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Songmin Ying; Fei Li; Xinwei Geng; Zhouyang Li; Xufei Du; Haixia Chen; Sisi Chen; Min Zhang; Zhehua Shao; Yinfang Wu; Madiha Zahra Syeda; Fugui Yan; Luanqing Che; Bin Zhang; Jian Lou; Shaobin Wang; Zhengming Chen; Wen Li; Ye Shen; Zhihua Chen; Huahao Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027623&#39;},
 {&#39;cord_uid&#39;: &#39;eoxrh2sl&#39;,
  &#39;sha&#39;: &#39;d7701711485f822c0f43df58b48faf6a433028c4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission potential of the novel coronavirus (COVID-19) onboard the Diamond Princess Cruises Ship, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.20027649&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An outbreak of COVID-19 developed aboard the Princess Cruises Ship during January-February 2020. Using mathematical modeling and time-series incidence data describing the trajectory of the outbreak among passengers and crew members, we characterize how the transmission potential varied over the course of the outbreak. Our estimate of the mean reproduction number in the confined setting reached values as high as ~11, which is higher than mean estimates reported from community-level transmission dynamics in China and Singapore (approximate range: 1.1-7). Our findings suggest that Rt decreased substantially compared to values during the early phase after the Japanese government implemented an enhanced quarantine control. Most recent estimates of Rt reached values largely below the epidemic threshold, indicating that a secondary outbreak of the novel coronavirus was unlikely to occur aboard the Diamond Princess Ship.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Kenji Mizumoto; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.20027649&#39;},
 {&#39;cord_uid&#39;: &#39;0n5bxp93&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;IFN-λ4 increases the risk of gastrointestinal infections and malaria in Malian children&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.962688&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTGenetic polymorphisms within the IFNL3/IFNL4 genomic region encoding type III interferons have been strongly associated with impaired clearance of hepatitis C virus (HCV) infection. We hypothesized that this association might extend to the immune response to other pathogens as well. In a cohort of 914 Malian children enrolled at birth, we analyzed episodes of malaria, gastrointestinal and respiratory infections in relation to two genetic polymorphisms functionally affecting type III interferons - rs368234815 (IFN-λ4) and rs4803217 (IFN-λ3), using information for 30,626 clinic visits from birth through 1-5 years of follow-up. Compared to children with the rs368234815-TT/TT genotype (IFN-λ4-Null), each copy of the rs368234815-dG allele was associated with an earlier first episode of a gastrointestinal infection (p=0.004), malaria (p=0.044) or respiratory infection (p=0.045). The risk of ever experiencing an infection during the follow-up was also significantly increased with each copy of the rs368234815-dG allele – for gastrointestinal infections (OR=1.5, 95%CI (1.11-2.03), p=0.0079) and malaria (OR=1.32, 95%CI (1.04-1.68, p=0.022), but not respiratory infections (p=0.58). IFNL4-rs368234815 and IFNL3-rs4803217 were in moderate linkage disequilibrium in this population (r2=0.78). All the associations for rs4803217 were weaker and lost significance after adjusting for rs368234815, implicating IFN-λ4 and not IFN-λ3 as the primary cause of these associations. Thus, our results suggest the role of IFN-λ4 in several infections in young children.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Ludmila Prokunina-Olsson; Robert L. Morrison; Adeola Obajemu; Almahamoudou Mahamar; Sungduk Kim; Oumar Attaher; Oscar Florez-Vargas; Youssoufa Sidibe; Olusegun O. Onabajo; Amy A. Hutchinson; Michelle Manning; Jennifer Kwan; Nathan Brand; Alassane Dicko; Michal Fried; Paul S. Albert; Sam M. Mbulaiteye; Patrick E. Duffy&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.962688&#39;},
 {&#39;cord_uid&#39;: &#39;epy7774j&#39;,
  &#39;sha&#39;: &#39;c8b6f2752a842dd9eb9c50a82112748ef10ba259&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.24.963348&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe ongoing outbreak of a new coronavirus (2019-nCoV) causes an epidemic of acute respiratory syndrome in humans. 2019-nCoV rapidly spread to national regions and multiple other countries, thus, pose a serious threat to public health. Recent studies show that spike (S) proteins of 2019-nCoV and SARS-CoV may use the same host cell receptor called angiotensin-converting enzyme 2 (ACE2) for entering into host cells. The affinity between ACE2 and 2019-nCoV S is much higher than ACE2 binding to SARS-CoV S protein, explaining that why 2019-nCoV seems to be more readily transmitted from the human to human. Here, we reported that ACE2 can be significantly upregulated after infection of various viruses including SARS-CoV and MERS-CoV. Basing on findings here, we propose that coronavirus infection can positively induce its cellular entry receptor to accelerate their replication and spread, thus drugs targeting ACE2 expression may be prepared for the future emerging infectious diseases caused by this cluster of viruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Pei-Hui Wang; Yun Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.24.963348&#39;},
 {&#39;cord_uid&#39;: &#39;z4q0eyjo&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Deep learning-based model for detecting 2019 novel coronavirus pneumonia on high-resolution computed tomography: a prospective study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20021568&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Computed tomography (CT) is the preferred imaging method for diagnosing 2019 novel coronavirus (COVID19) pneumonia. Our research aimed to construct a system based on deep learning for detecting COVID-19 pneumonia on high resolution CT, relieve working pressure of radiologists and contribute to the control of the epidemic. Methods: For model development and validation, 46,096 anonymous images from 106 admitted patients, including 51 patients of laboratory confirmed COVID-19 pneumonia and 55 control patients of other diseases in Renmin Hospital of Wuhan University (Wuhan, Hubei province, China) were retrospectively collected and processed. Twenty-seven consecutive patients undergoing CT scans in Feb, 5, 2020 in Renmin Hospital of Wuhan University were prospectively collected to evaluate and compare the efficiency of radiologists against 2019-CoV pneumonia with that of the model. Findings: The model achieved a per-patient sensitivity of 100%, specificity of 93.55%, accuracy of 95.24%, PPV of 84.62%, and NPV of 100%; a per-image sensitivity of 94.34%, specificity of 99.16%, accuracy of 98.85%, PPV of 88.37%, and NPV of 99.61% in retrospective dataset. For 27 prospective patients, the model achieved a comparable performance to that of expert radiologist. With the assistance of the model, the reading time of radiologists was greatly decreased by 65%. Conclusion: The deep learning model showed a comparable performance with expert radiologist, and greatly improve the efficiency of radiologists in clinical practice. It holds great potential to relieve the pressure of frontline radiologists, improve early diagnosis, isolation and treatment, and thus contribute to the control of the epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Jun Chen; Lianlian Wu; Jun Zhang; Liang Zhang; Dexin Gong; Yilin Zhao; Shan Hu; Yonggui Wang; Xiao Hu; Biqing Zheng; Kuo Zhang; Huiling Wu; Zehua Dong; Youming Xu; Yijie Zhu; Xi Chen; Lilei Yu; Honggang Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20021568&#39;},
 {&#39;cord_uid&#39;: &#39;acmfhbtm&#39;,
  &#39;sha&#39;: &#39;da5ecffba8c5d1b7afee057ff61149fb976501ce&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Highland of COVID-19 outside Hubei: epidemic characteristics, control and projections of Wenzhou, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20024398&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In late December 2019, Chinese authorities reported a cluster of pneumonia cases of unknown aetiology in Wuhan, China. Wenzhou, as a southeast coastal city with the most cases outside Hubei Province, its policy control and epidemic projections have certain references for national and worldwide epidemic prevention and control. The maximum number of illness onset in Wenzhou appeared on January 26 (38 cases). Since January 27, the incidence per day in Wenzhou has continued to decline is likely to be due to isolation of people return to Wenzhou, implementation of region quarantine and suspension of public transportation system. Under the current control efforts, we estimate that the daily incidence by March 3-9, 2020 will drop to 0 in Wenzhou using SEIR model. The total number of affected people is 538.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Liangde Xu; Jian Yuan; Yaru Zhang; Guosi Zhang; Fan Lu; Jianzhong Su; Jia Qu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20024398&#39;},
 {&#39;cord_uid&#39;: &#39;dy28eo89&#39;,
  &#39;sha&#39;: &#39;bca15ad98e1d08c6a62c9c0486de84bc15a9f661&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20024711&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background. The clinical presentation of 2019 Novel Coronavirus (2019-nCov) infected pneumonia (NCIP) resembles that of other etiologies of community-acquired pneumonia (CAP). We aimed to identify clinical laboratory features to distinguish NCIP from CAP. Methods. We compared the ability of the hematological and biochemical features of 84 patients with NCIP at hospital admission and 316 patients with CAP. Parameters independently predictive of NCIP were calculated by multivariate logistic regression. The receiver operating characteristic (ROC) curves were generated and the area under the ROC curve (AUC) was measured to evaluate the discriminative ability. Results. Most hematological and biochemical indexes of patients with NCIP were significantly different from patients with CAP. Nine laboratory parameters were identified to be highly predictive of a diagnosis of NCIP by multivariate analysis. The AUCs demonstrated good discriminatory ability for red cell distribution width (RDW) with an AUC of 0.88 and Hemoglobin (HGB) with an AUC of 0.82. Red blood cell (RBC), albumin (ALB), eosinophil (EO), hematocrit (HCT), alkaline phosphatase (ALP), and white blood cell (WBC) had fair discriminatory ability. Combinations of any two parameters performed better than did the RDW alone. Conclusions. Routine laboratory examinations may be helpful for the diagnosis of NCIP. Application of laboratory tests may help to optimize the use of isolation rooms for patients when they present with unexplained febrile respiratory illnesses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-25&#39;,
  &#39;authors&#39;: &#39;Yunbao Pan; Guangming Ye; Xiantao Zeng; Guohong Liu; Xiaojiao Zeng; Xianghu Jiang; Jin Zhao; Liangjun Chen; Shuang Guo; Qiaoling Deng; Xiaoyue Hong; Ying Yang; Yirong Li; Xinghuan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20024711&#39;},
 {&#39;cord_uid&#39;: &#39;370fqkat&#39;,
  &#39;sha&#39;: &#39;c8d206a4f9af0709b6e9ee90c4d854d482cb0784&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20025643&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Aim: The new coronavirus pneumonia (COVID-19) outbreaking at the end of 2019 is highly contagious. Crude mortality rate reached 49% in critical patients. Inflammation matters on disease progression. This study analyzed blood inflammation indicators among mild, severe and critical patients, helping to identify severe or critical patients early. Methods: In this cross-sectional study, 100 patients were included and divided to mild, severe or critical groups. Correlation of peripheral blood inflammation-related indicators with disease criticality was analyzed. Cut-off values for critically ill patients were speculated through the ROC curve. Results：Significantly, disease severity were associated with age (R=-0.564, P&lt;0.001), interleukin-2 receptor (IL2R) (R=-0.534, P&lt;0.001), interleukin-6 (IL-6) (R=-0.535, P&lt;0.001), interleukin-8 (IL-8) (R=-0.308, P&lt;0.001), interleukin-10 (IL-10) (R=-0.422, P&lt;0.001), tumor necrosis factor α (TNFα) (R=-0.322, P&lt;0.001), C-reactive protein (CRP) (R=-0.604, P&lt;0.001), ferroprotein (R=-0.508, P&lt;0.001), procalcitonin (R=-0.650, P&lt;0.001), white cell counts (WBC) (R=-0.54, P&lt;0.001), lymphocyte counts (LC) (R=-0.56, P&lt;0.001), neutrophil count (NC) (R=-0.585, P&lt;0.001) and eosinophil counts (EC) (R=-0.299, P=0.01). Conclusion：With following parameters such as age &gt;67.5 years, IL2R &gt;793.5U/mL, CRP &gt;30.7ng/mL, ferroprotein &gt;2252μg/L, WBC&gt;9.5*10^9/L or NC &gt;7.305*10^9/L, the progress of COVID-19 to critical stage should be closely observed and possibly prevented. Inflammation is closely related to severity of COVID-19, and IL-6, TNFα and IL-8 might be promising therapeutic targets.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Jing Gong; Hui Dong; Song Qing Xia; Yi Zhao Huang; Dingkun Wang; Yan Zhao; Wenhua Liu; Shenghao Tu; Mingmin Zhang; Qi Wang; Fuer Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20025643&#39;},
 {&#39;cord_uid&#39;: &#39;qtrquuvv&#39;,
  &#39;sha&#39;: &#39;7d3dcbbc2b0741fc544693d7b3fc185f0bd50eac&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Open-source analytics tools for studying the COVID-19 coronavirus outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027433&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;To provide convenient access to epidemiological data on the coronavirus outbreak, we developed an R package, nCov2019 (https://github.com/GuangchuangYu/nCov2019). Besides detailed real-time statistics, it offers access to three data sources with detailed daily statistics from December 1, 2019, for 43 countries and more than 500 Chinese cities. We also developed a web app (http://www.bcloud.org/e/) with interactive plots and simple time-series forecasts. These analytics tools could be useful in informing the public and studying how this and similar viruses spread in populous countries.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Tianzhi Wu; Xijin Ge; Guangchuang Yu; Erqiang Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027433&#39;},
 {&#39;cord_uid&#39;: &#39;aoqyx8mk&#39;,
  &#39;sha&#39;: &#39;160ecaaa5a766c289fc2f6b5499f0dfe7aab971c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Stochastic discrete epidemic modeling of COVID-19 transmission in the Province of Shaanxi incorporating public health intervention and case importation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027615&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Before the lock-down of Wuhan/Hubei/China, on January 23rd 2020, a large number of individuals infected by COVID-19 moved from the epicenter Wuhan and the Hubei province due to the Spring Festival, resulting in an epidemic in the other provinces including the Shaanxi province. The epidemic scale in Shaanxi was comparatively small and with half of cases being imported from the epicenter. Based on the complete epidemic data including the symptom onset time and transmission chains, we calculate the control reproduction number (1.48-1.69) in Xian. We could also compute the time transition, for each imported or local case, from the latent, to infected, to hospitalized compartment, as well as the effective reproduction number. This calculation enables us to revise our early deterministic transmission model to a stochastic discrete epidemic model with case importation and parameterize it. Our model-based analyses reveal that the newly generated infections decay to zero quickly; the cumulative number of case-driven quarantined individuals via contact tracing stabilize at a manageable level, indicating that the intervention strategies implemented in the Shaanxi province have been effective. Risk analyses, important for the consideration of resumption of work, show that a large second outbreak is expected if the level of case importation remains at the same level as between January 10th and February 4th 2020. However, if the case importation decreases by 30%, 60% and 90%, the second outbreak if happening will be of small-scale assuming contact tracing and quarantine/isolation remain as effective as before. Finally, we consider the effects of intermittent inflow with a Poisson distribution on the likelihood of multiple outbreaks. We believe the developed methodology and stochastic model provide an important model framework for the evaluation of revising travel restriction rules in the consideration of resuming social-economic activities while managing the disease control with potential case importation.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Sanyi Tang; Biao Tang; Nicola Luigi Bragazzi; Fan Xia; Tangjuan Li; Sha He; Pengyu Ren; Xia Wang; Zhihang Peng; Yanni Xiao; Jianhong Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027615&#39;},
 {&#39;cord_uid&#39;: &#39;7s9ot4vq&#39;,
  &#39;sha&#39;: &#39;875b7c463f00772fa0dc18ada678bc1ff16a4274&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Comorbidity and its impact on 1,590 patients with COVID-19 in China: A Nationwide Analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027664&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To evaluate the spectrum of comorbidities and its impact on the clinical outcome in patients with coronavirus disease 2019 (COVID-19). Design: Retrospective case studies Setting: 575 hospitals in 31 province/autonomous regions/provincial municipalities across China Participants: 1,590 laboratory-confirmed hospitalized patients. Data were collected from November 21st, 2019 to January 31st, 2020. Main outcomes and measures: Epidemiological and clinical variables (in particular, comorbidities) were extracted from medical charts. The disease severity was categorized based on the American Thoracic Society guidelines for community-acquired pneumonia. The primary endpoint was the composite endpoints, which consisted of the admission to intensive care unit (ICU), or invasive ventilation, or death. The risk of reaching to the composite endpoints was compared among patients with COVID-19 according to the presence and number of comorbidities. Results: Of the 1,590 cases, the mean age was 48.9 years. 686 patients (42.7%) were females. 647 (40.7%) patients were managed inside Hubei province, and 1,334 (83.9%) patients had a contact history of Wuhan city. Severe cases accounted for 16.0% of the study population. 131 (8.2%) patients reached to the composite endpoints. 399 (25.1%) reported having at least one comorbidity. 269 (16.9%), 59 (3.7%), 30 (1.9%), 130 (8.2%), 28 (1.8%), 24 (1.5%), 21 (1.3%), 18 (1.1%) and 3 (0.2%) patients reported having hypertension, cardiovascular diseases, cerebrovascular diseases, diabetes, hepatitis B infections, chronic obstructive pulmonary disease, chronic kidney diseases, malignancy and immunodeficiency, respectively. 130 (8.2%) patients reported having two or more comorbidities. Patients with two or more comorbidities had significantly escalated risks of reaching to the composite endpoint compared with those who had a single comorbidity, and even more so as compared with those without (all P&lt;0.05). After adjusting for age and smoking status, patients with COPD (HR 2.681, 95%CI 1.424-5.048), diabetes (HR 1.59, 95%CI 1.03-2.45), hypertension (HR 1.58, 95%CI 1.07-2.32) and malignancy (HR 3.50, 95%CI 1.60-7.64) were more likely to reach to the composite endpoints than those without. As compared with patients without comorbidity, the HR (95%CI) was 1.79 (95%CI 1.16-2.77) among patients with at least one comorbidity and 2.59 (95%CI 1.61-4.17) among patients with two or more comorbidities. Conclusion: Comorbidities are present in around one fourth of patients with COVID-19 in China, and predispose to poorer clinical outcomes.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Wei-jie Guan; Wen-hua Liang; Yi Zhao; Heng-rui Liang; Zi-sheng Chen; Yi-min Li; Xiao-qing Liu; Ru-chong Chen; Chun-li Tang; Tao Wang; Chun-quan Ou; Li Li; Ping-yan Chen; Ling Sang; Wei Wang; Jian-fu Li; Cai-chen Li; Li-min Ou; Bo Cheng; Shan Xiong; Zheng-yi Ni; Yu Hu; Jie Xiang; Lei Liu; Hong Shan; Chun-liang Lei; Yi-xiang Peng; Li Wei; Yong Liu; Ya-hua Hu; Peng Peng; Jian-ming Wang; Ji-yang Liu; Zhong Chen; Gang Li; Zhi-jian Zheng; Shao-qin Qiu; Jie Luo; Chang-jiang Ye; Shao-yong Zhu; Lin-ling Cheng; Feng Ye; Shi-yue Li; Jin-ping Zheng; Nuo-fu Zhang; Nan-shan Zhong; Jian-xing He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027664&#39;},
 {&#39;cord_uid&#39;: &#39;gvjiexfm&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Age-dependent risks of Incidence and Mortality of COVID-19 in Hubei Province and Other Parts of China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027672&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;New coronavirus SARS-CoV-2 poses a big challenge for global public health in early 2020. Coronavirus Disease 2019 (COVID-19) caused by the virus rapidly spreads all over the world and takes thousands of lives in just two months. It is critical to refine the incidence and mortality risks of COVID-19 for the effective management of the general public and patients in the outbreak. In this report, we investigate the incidence and mortality risks of the infection by analyzing the age composition of 5319 infected patients, 76 fatal cases, and 1,144,648 individuals of the general public in China. Our result shows a relatively low incidence risk for young people but a very high mortality risk for seniors. Notably, mortality risk could be as high as 0.48 for people older than 80 years. Furthermore, our study suggests that a good medical service can effectively reduce the mortality rate of the viral infection to 1% or less.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Hongdou Li; Shuang Wang; Fan Zhong; Wuyin Bao; Yipeng Li; Lei Liu; Hongyan Wang; Yungang He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027672&#39;},
 {&#39;cord_uid&#39;: &#39;65plab8b&#39;,
  &#39;sha&#39;: &#39;e046a06f262d74d025272ad697d358c4afb5d9bd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027755&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19. Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Guanmin Jiang; Xiaoshuai Ren; Yan Liu; Hongtao Chen; Wei Liu; Zhaowang Guo; Yaqin Zhang; Chaoqun Chen; Jianhui Zhou; Qiang Xiao; Hong Shan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027755&#39;},
 {&#39;cord_uid&#39;: &#39;cr0s5d1j&#39;,
  &#39;sha&#39;: &#39;0eee9760aee0ca2fdf8d1d215c3689d5f8d84df5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prevalence and clinical features of 2019 novel coronavirus disease (COVID-19) in the Fever Clinic of a teaching hospital in Beijing: a single-center, retrospective study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027763&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background With the spread of COVID-19 from Wuhan, Hubei Province to other areas of the country, medical staff in Fever Clinics faced the challenge of identifying suspected cases among patients with respiratory infections manifested with fever. We aimed to describe the prevalence and clinical features of COVID-19 as compared to pneumonias of other etiologies in a Fever Clinic in Beijing. Methods In this single-center, retrospective study, 342 cases of pneumonia were diagnosed in Fever Clinic in Peking University Third Hospital between January 21 to February 15, 2020. From these patients, 88 were reviewed by panel discussion as possible or probable cases of COVID-19, and received 2019-nCoV detection by RT-PCR. COVID-19 was confirmed by positive 2019-nCoV in 19 cases, and by epidemiological, clinical and CT features in 2 cases (the COVID-19 Group, n=21), while the remaining 67 cases served as the non-COVID-19 group. Demographic and epidemiological data, symptoms, laboratory and lung CT findings were collected, and compared between the two groups. Findings The prevalence of COVID-19 in all pneumonia patients during the study period was 6.14% (21/342). Compared with the non-COVID-19 group, more patients with COVID-19 had an identified epidemiological history (90.5% versus 32.8%, P&lt;0.001). The COVID-19 group had lower WBC [5.19×10^9/L (±1.47) versus 7.21×10^9/L (±2.94), P&lt;0.001] and neutrophil counts [3.39×10^9/L (±1.48) versus 5.38×10^9/L (±2.85), P&lt;0.001] in peripheral blood. However, the percentage and count of lymphocytes were not different. On lung CT scans, involvement of 4 or more lobes was more common in the COVID-19 group (45% versus 16.4%, P=0.008). Interpretation In the period of COVID-19 epidemic outside Hubei Province, the prevalence of COVID-19 in patients with pneumonia visiting to our Fever Clinic in Beijing was 6.14%. Epidemiological evidence was important for prompt case finding, and lower blood WBC and neutrophil counts may be useful for differentiation from pneumonia of other etiologies.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Ying Liang; Jingjin Liang; Qingtao Zhou; Xiaoguang Li; Fei Lin; Zhonghua Deng; Biying Zhang; Lu Li; Xiaohua Wang; Hong Zhu; Qingbian Ma; Xiaomei Tong; Jie Xu; Yongchang Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027763&#39;},
 {&#39;cord_uid&#39;: &#39;a12708qd&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Genomic variations of SARS-CoV-2 suggest multiple outbreak sources of transmission&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20027953&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We examined 169 genomes of SARS-CoV-2 and found that they can be classified into two major genotypes, Type I and Type II. Type I can be further divided into Type IA and IB. Our phylogenetic analysis showed that the Type IA resembles the ancestral SARS-CoV-2 most. Type II was likely evolved from Type I and predominant in the infections. Our results suggest that Type II SARS-CoV-2 was the source of the outbreak in the Wuhan Huanan market and it was likely originated from a super-spreader. The outbreak caused by the Type I virus should have occurred somewhere else, because the patients had no direct link to the market. Furthermore, by analyzing three genomic sites that distinguish Type I and Type II strains, we found that synonymous changes at two of the three sites confer higher protein translational efficiencies in Type II strains than in Type I strains, which might explain why Type II strains are predominant, implying that Type II is more contagious (transmissible) than Type I. These findings could be valuable for the current epidemic prevention and control.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Liangsheng Zhang; Jian-Rong Yang; Zhenguo Zhang; Zhenguo Lin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20027953&#39;},
 {&#39;cord_uid&#39;: &#39;lad31adx&#39;,
  &#39;sha&#39;: &#39;eca1cf50bd67a7efbcaf78864f5f5b3b850243d6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Data on Hospital Environmental Hygiene Monitoring and Medical Staff Protection during the Coronavirus Disease 2019 Outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.20028043&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: The outbreak of coronavirus disease 2019 (COVID-19) has placed unprecedented challenges on hospital environmental hygiene and medical staff protection. It is crucial to assess hospital environmental hygiene to understand the most important environmental issues for controlling the spread of COVID-19 in hospitals. Objective: To detect the presence of COVID-19 in the samples from the area at risk of contamination in the First Hospital of Jilin University. Methods: Viruses in the air were collected by natural sedimentation and air particle sampler methods. Predetermined environmental surfaces were sampled using swabs at seven o&#39;clock in the morning before disinfection. The real-time reverse-transcription PCR method was used to detect the existence of COVID-19 pathogens. Results: Viruses could be detected on the surfaces of the nurse station in the isolation area with suspected patients and in the air of the isolation ward with an intensive care patient. Conclusion: Comprehensive monitoring of hospital environmental hygiene during pandemic outbreaks is conducive to the refinement of hospital infection control. It is of great significance to ensure the safety of medical treatment and the quality of hospital infection control through the monitoring of environmental hygiene.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Yanfang Jiang; Haifeng Wang; Yukun Chen; Jiaxue He; Liguo Chen; Yong Liu; Xinyuan Hu; Ang Li; Siwen Liu; Peng Zhang; Hongyan Zou; Shucheng Hua&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.20028043&#39;},
 {&#39;cord_uid&#39;: &#39;i55tm3hh&#39;,
  &#39;sha&#39;: &#39;a2a6e262098539eb875a26800d9f6d3d0d5d1875&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.963546&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within COVID-19 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;CV Herst; S Burkholz; J Sidney; A Sette; PE Harris; S Massey; T Brasel; E Cunha-Neto; DS Rosa; WCH Chao; R Carback; T Hodge; L Wang; S Ciotlos; P Lloyd; R Rubsamen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.963546&#39;},
 {&#39;cord_uid&#39;: &#39;2hyiped2&#39;,
  &#39;sha&#39;: &#39;86b6b0c1b2777541feb83116bcb7a5cb12a52310&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.964775&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Yu Jin Jung; Gun-Soo Park; Jun Hye Moon; Keunbon Ku; Seung-Hwa Beak; Seil Kim; Edmond Changkyun Park; Daeui Park; Jong-Hwan Lee; Cheol Woo Byeon; Joong Jin Lee; Jin-Soo Maeng; Seong Jun Kim; Seung Il Kim; Bum-Tae Kim; Min Jun Lee; Hong Gi Kim&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.964775&#39;},
 {&#39;cord_uid&#39;: &#39;e9vq3fe3&#39;,
  &#39;sha&#39;: &#39;949feecec7c10e71532e860f7f1230bd696e8234&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.965434&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Lin Li; Ting Sun; Yufei He; Wendong Li; Yubo Fan; Jing Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.965434&#39;},
 {&#39;cord_uid&#39;: &#39;qefiw4ho&#39;,
  &#39;sha&#39;: &#39;dbefc8ad2a3de5d1696b7e604de8bce1da2ea8cd&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Screening of FDA-approved drugs using a MERS-CoV clinical isolate from South Korea identifies potential therapeutic options for COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.25.965582&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn 2015, the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) reached the Republic of Korea through nosocomial transmission and was the largest epidemic outside of the Arabian Peninsula. To date, despite various strategies to identify CoV interventions, only limited therapeutic options are available. To address these unmet medical needs, we used a South Korean MERS-CoV clinical isolate and screened 5,406 compounds, including United States Food and Drug Administration (FDA)-approved drugs and bioactive molecules, for their activity against the isolate. The primary assay confirmed 221 hits by dose-response curve analysis and identified 54 hits with a therapeutic index (TI) greater than 6. Time-of-addition studies with 12 FDA-approved drugs demonstrated that 8 and 4 therapeutics act on the early and late stages of the viral life cycle, respectively. Among the drugs were e.g., three cardiotonic agents (ouabain, digitoxin, digoxin) with a TI greater than 100, an anti-malaria drug (atovaquone; TI &gt;34), an inhalable corticosteroid (ciclesonide; TI &gt;6), etc. Together, our results identify potential therapeutic options for treating MERS-CoV infections and could provide a basis for agents against a wider range of coronavirus-related illnesses, including the currently emerging Coronavirus Disease 2019 (COVID-19) outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Meehyun Ko; So Young Chang; Soo Young Byun; Inhee Choi; Anne-Laure Pham Hung d’Alexandry d’Orengiani; David Shum; Ji-Young Min; Marc P. Windisch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.25.965582&#39;},
 {&#39;cord_uid&#39;: &#39;v0ln3wfa&#39;,
  &#39;sha&#39;: &#39;baabfb35a321ea12028160e0d2c1552a2fda2dd5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Features of COVID-19 Related Liver Damage&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20026971&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND: A recent outbreak of SARS-CoV-2 infection occurs mainly in China, with rapidly increasing the number of cases (namely COVID-19). Abnormal liver functions are frequently present in these patients, here we aimed to clarify the clinical features of COVID-19-related liver damage to provide some references for the clinical treatment. METHODS: In this retrospective, single-center study, we included all confirmed COVID-19 cases in Shanghai Public Health Clinical Center from January 20 to January 31, 2020. The outcomes were followed up until February 19, 2020. A total of 148 cases were analyzed for clinical features, laboratory parameters (including liver function tests), medications and the length of stay. FINDINGS: Of 148 confirmed SARS-CoV-2-infected patients, 49.3% were females and 50.7% were males. The median age was 50.5 years (interquartile range, 36-64). Patients had clinical manifestations of fever (70.1%), cough (45.3%), expectoration (26.7%) at admission. 75 patients (50.7%) showed abnormal liver functions at admission. Patients (n = 75) who had elevated liver function index were more likely to have a moderate-high degree fever (44% vs 27.4%; p = 0.035) and significantly present in male patients (62.67% vs 38.36%; p = 0.005). The numbers of CD4+ and CD8+ T cells were significantly lower in abnormal liver function group than those in normal liver function group. There was no statistical difference in prehospital medications between normal and abnormal liver function groups, while the utilization rate of lopinavir/ritonavir after admission was significantly higher in patients with emerging liver injury than that in patients with normal liver functions. Importantly, the emerging abnormal liver functions after admission caused a prolonged length of stay. INTERPRETATION: SARS-CoV-2 may cause the liver function damage and the Lopinavir/ritonavir should be applied carefully for the treatment of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Zhenyu Fan; Liping Chen; Jun Li; Cheng Tian; Yajun Zhang; Shaoping Huang; Zhanju Liu; Jilin Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20026971&#39;},
 {&#39;cord_uid&#39;: &#39;r7yyffgt&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20026989&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;IMPORTANCE: COVID-19-infected pneumonia patients with severe immune abnormalities and risk of cytokine release syndrome. The definition, prevention, and treatment of COVID-19-infected pneumonia in critically ill patients with cytokine release syndrome symptoms is an important problem.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Wenjun Wang; Jianxing He; puyi Lie; liyan Huang; Sipei Wu; yongping lin; xiaoqing liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20026989&#39;},
 {&#39;cord_uid&#39;: &#39;m0d72ntg&#39;,
  &#39;sha&#39;: &#39;7344c6de7ce2258e74ce131bed72e6ad465f0ce8&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Relations of parameters for describing the epidemic of COVID―19 by the Kermack―McKendrick model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20027797&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In order to quantitatively characterize the epidemic of COVID―19, useful relations among parameters describing an epidemic in general are derived based on the Kermack-McKendrick model. The first relation is 1/τgrow=1/τtrans−1/τinf, where τgrow is the time constant of the exponential growth of an epidemic, τtrans is the time for a pathogen to be transmitted from one patient to uninfected person, and the infectious time τinf is the time during which the pathogen keeps its power of transmission. The second relation p(∞) ≈1−exp(−(R0−1)/0.60) is the relation between p(∞), the final size of the disaster defined by the ratio of the total infected people to the population of the society,and the basic reproduction number, R0, which is the number of persons infected by the transmission of the pathogen from one infected person during the infectious time. The third relation 1/τend=1/τinf−(1−p(∞))/τtrans gives the decay time constant τend at the ending stage of the epidemic. Derived relations are applied to influenza in Japan in 2019 for characterizing the epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Toshihisa Tomie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20027797&#39;},
 {&#39;cord_uid&#39;: &#39;pytu6sy0&#39;,
  &#39;sha&#39;: &#39;c10661c9b35e068691b879661bcebeba3bd6aad9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The infection evidence of SARS-COV-2 in ocular surface： a single-center cross-sectional study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20027938&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Purpose: The aim of this study was to identify whether SARS-COV-2 infected in ocular surface. Methods: Cross-sectional study of patients presenting for who received a COVID-19 diagnosis, from December 30, 2019 to February 7, 2020, at Tongji hospital, Tongji medical college, Huazhong University of Science and Technology. Demographics, temperature was recorded, blood routine test (Rt), chest Computed Tomography (CT) were took intermittently, and SARS-COV-2 real-time reverse-transcriptase-polymerase-chain-reaction (RT-PCR) assay were arranged for the nasopharyngeal and conjunctival swab samples. Results: A total of 102 patients (48 Male [50%] and 54 Female [50%]) with clinical symptoms, Rt, and chest Computed Tomography (CT) abnormalities were identified with a clinical diagnosis of COVID-19. Patients had a mean [SD] gestational age of 57.63 [14.90] years. Of a total of 102 patients identified, 72 patients (36 men [50%] and 36 women [50%]; mean [SD] age, 58.68 [14.81] years) confirmed by laboratory diagnosis with SARS-COV-2 RT-PCR assay. Only two patients (2.78%) with conjunctivitis was identified from 72 patients with a laboratory confirmed COVID-19. However, SARS-COV-2 RNA fragments was found in ocular discharges by SARS-COV-2 RT-PCR only in one patient with conjunctivitis. Conclusions: Although we suspect the incidence of SARS-COV-2 infection through the ocular surface is extremely low, the nosocomial infection of SARS-CoV-2 through the eyes after occupational exposure is a potential route. The inefficient diagnostic method and the sampling time lag may contribute to the lower positive rate of conjunctival swab samples of SARS-COV-2. Therefore, to lower the SARS-COV-2 nosocomial infection, the protective goggles should be wore in all the health care workers.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Xufang Sun; Xian Zhang; Xuhui Chen; Liwen Chen; Chaohua Deng; Xiaojing Zou; Weiyong Liu; Huimin Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20027938&#39;},
 {&#39;cord_uid&#39;: &#39;hk491mr6&#39;,
  &#39;sha&#39;: &#39;46634d03aa169aab9c372746ebaf3aaa65d7c7d9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Case fatality rate of novel coronavirus disease 2019 in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028076&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Background: A pandemic of coronavirus disease 2019 (COVID-19) which have caused more than 80 thousand persons infected globally is still ongoing. This study aims to calculate its case fatality rate (CFR). Methods: The method, termed as converged CFR calculation, was based on the formula of dividing the number of known deaths by the number of confirmed cases T days before, where T was an average time period from case confirmation to death. It was found that supposing a T, if it was smaller (bigger) than the true T, calculated CFRs would gradually increase (decrease) to infinitely near the true T with time went on. According to the law, the true T value could be determined by trends of daily CFRs calculated with different assumed T values (left of true T is decreasing, right is increasing). Then the CFR could be calculated. Results: CFR of COVID-19 in China except Hubei Province was 0.8% to 0.9%. So far, the CFR had accurately predicted the death numbers more than 3 weeks. CFR in Hubei of China was 5.4% by which the calculated death number corresponded with the reported number for 2 weeks. Conclusion: The method could be used for CFR calculating while pandemics are still ongoing. Dynamic monitoring of the daily CFRs trends could help outbreak-controller to have a clear vision in the timeliness of the case confirmation.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-26&#39;,
  &#39;authors&#39;: &#39;Rui Qi; Chao Ye; Xiang-rong Qin; Xue-Jie Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028076&#39;},
 {&#39;cord_uid&#39;: &#39;011k6mm0&#39;,
  &#39;sha&#39;: &#39;c954675ee859e2f7b8f352a398a67469b50f05de&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluation of the clinical characteristics of suspected or confirmed cases of COVID-19 during home care with isolation: A new retrospective analysis based on O2O&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028084&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Summary Background The recent outbreak of the novel coronavirus in December 2019 (COVID-19) has activated top-level response nationwide. We developed a new treatment model based on the online-to-offline (O2O) model for the home isolated patients, because in the early stages the medical staff were insufficient to cope with so many patients. Methods In this single-centered, retrospective study, we enrolled 48 confirmed/suspected COVID-19 patients who underwent home isolation in Wuhan between January 6 and January 31, 2020. By WeChat and online document editing all patients were treated with medical observation scale. The clinical indications such as Fever, Muscle soreness, Dyspnea and Lack of strength were collected with this system led by medical staff in management, medicine, nursing, rehabilitation and psychology. Findings The mean(SD) age of 48 patients was 39.08(13.88) years, 35(72.9%) were women. Compared with non-hospitalized patients, inpatients were older(≥8805;70years, 2.4% vs 33.3%, P&lt;0.04). All inpatients had fever, 50% inpatients had coughs and showed infiltration in both lungs at the time of diagnosis. 33.3% inpatients exhibited negative changes in their CT results at initial diagnosis. The body temperature of non-hospitalized patients with mild symptoms returned to normal by day 4-5. While dyspnea peaked on day 6 for non-hospitalized patients with mild symptoms, it persisted in hospitalized patients and exacerbated over time. The lack of strength and muscle soreness were both back to normal by day 4 for non-hospitalized patients. Interpretation Monitoring the trends of symptoms is more important for identifying severe cases. Excessive laboratory data and physical examination are not necessary for the evaluation of patients with mild symptoms. The system we developed is the first to convert the subjective symptoms of patients into objective scores. This type of O2O, subjective-to-objective strategy may be used in regions with similar highly infectious diseases to minimize the possibility of infection among medical staff.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Hui Xu; Sufang Huang; Shangkun Liu; Juan Deng; Bo Jiao; Ling Ai; Yaru Xiao; Li Yan; Shusheng Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028084&#39;},
 {&#39;cord_uid&#39;: &#39;96wkqutc&#39;,
  &#39;sha&#39;: &#39;2536b83acf84368d7c13be81fe07aa0575115da7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of country-level basic reproductive ratios for novel Coronavirus (COVID-19) using synthetic contact matrices&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028167&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of novel coronavirus (COVID-19) has the potential for global spread, infecting large numbers in all countries. In this case, estimating the country-specific basic reproductive ratio is a vital first step in public-health planning. The basic reproductive ratio (R0) is determined by both the nature of pathogen and the network of contacts through which the disease can spread - with this network determined by socio-demographics including age-structure and household composition. Here we focus on the age-structured transmission within the population, using data from China to inform age-dependent susceptibility and synthetic age-mixing matrices to inform the contact network. This allows us to determine the country-specific basic reproductive ratio as a multiplicative scaling of the value from China. We predict that R0 will be highest across Eastern Europe and Japan, and lowest across Africa, Central America and South-Western Asia. This pattern is largely driven by the ratio of children to older adults in each country and the observed propensity of clinical cases in the elderly.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Joe Hilton; Matt J Keeling&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028167&#39;},
 {&#39;cord_uid&#39;: &#39;fcmzdcuh&#39;,
  &#39;sha&#39;: &#39;a625e5d6fc8956c231c500a40e887e248a69de61&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 82 death cases with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028191&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center. Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms. Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of &gt;5 (94.5%), high systemic immune-inflammation index of &gt;500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (&gt;10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed. Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Bicheng Zhang; Xiaoyang Zhou; Yanru Qiu; Fan Feng; Jia Feng; Yifan Jia; Hengcheng Zhu; Ke Hu; Jiasheng Liu; Zaiming Liu; Shihong Wang; Yiping Gong; Chenliang Zhou; Ting Zhu; Yanxiang Cheng; Zhichao Liu; Hongping Deng; Fenghua Tao; Yijun Ren; Biheng Cheng; Ling Gao; Xiongfei Wu; Lilei Yu; Zhixin Huang; Zhangfan Mao; Qibin Song; Bo Zhu; Jun Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028191&#39;},
 {&#39;cord_uid&#39;: &#39;87pamw52&#39;,
  &#39;sha&#39;: &#39;7aa49aa2b76ea92b9265ad9d5a9d9ef0ea585b24&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Community responses during the early phase of the COVID-19 epidemic in Hong Kong: risk perception, information exposure and preventive measures&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028217&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Community responses are important for outbreak management during the early phase when non-pharmaceutical interventions are the major preventive options. Therefore, this study aims to examine the psychological and behavioral responses of the community during the early phase of the COVID-19 epidemic in Hong Kong. Method: A cross-sectional online survey was launched within 36 hours after confirmed COVID-19 cases were first reported. Councilors of all 452 district council constituency areas were approached for survey dissemination. Respondent demographics, anxiety level, risk perception, sources to retrieve COVID-19 information, actual adoption and perceived efficacy of precautionary measures were collected. Result: Analysis from 1715 complete responses indicated high perceived susceptibility (89%) and high perceived severity (97%). Most respondents were worried about COVID-19 (97%), and had their daily routines disrupted (slightly/greatly: 98%). The anxiety level, measured by the Hospital Anxiety and Depression Scale, was borderline abnormal (9.01). Nearly all respondents were alert to the disease progression (99.5%). The most trusted information sources were doctors (84%), followed by broadcast (57%) and newspaper (54%), but they were not common information sources (doctor: 5%; broadcast: 34%; newspaper: 40%). Only 16% respondents found official websites reliable. Enhanced personal hygiene practices and travel avoidance to China were frequently adopted (&gt;77%) and considered effective (&gt;90%). The adoption of social-distancing measures was lower (39%-88%), and their drivers for greater adoption include: being female (adjusted odds ratio [aOR]:1.27), living in the New Territories (aOR:1.32-1.55), perceived as having good understanding of COVID-19 (aOR:1.84) and being more anxious (aOR:1.07). Discussion: Risk perception towards COVID-19 in the community was high. Most respondents are alert to the disease progression, and adopt self-protective measures. This study contributes by examining the psycho-behavioral responses of hosts, in addition to the largely studied mechanistic aspects, during the early phase of the current COVID-19 epidemic. The timely psychological and behavioral assessment of the community is useful to inform subsequent interventions and risk communication strategies as the epidemic progresses.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Kin On Kwok; Kin Kit Li; Ho Hin Chan; Yuan Yuan Yi; Arthur Tang; Wan In Wei; Yeung Shan Wong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028217&#39;},
 {&#39;cord_uid&#39;: &#39;dlivkn8l&#39;,
  &#39;sha&#39;: &#39;111b9a6e91c938696fcdb4cb128b8ae739dbe11c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028225&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2. Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients&#39; sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient&#39;s partner was uninfected and the other patient&#39;s partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.&#34;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Pengfei Cui; Zhe Chen; Tian Wang; Jun Dai; Jinjin Zhang; Ting Ding; Jingjing Jiang; Jia Liu; Cong Zhang; Wanying Shan; Sheng Wang; Yueguang Rong; Jiang Chang; Xiaoping Miao; Xiangyi Ma; Shixuan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028225&#39;},
 {&#39;cord_uid&#39;: &#39;yk9u4fvp&#39;,
  &#39;sha&#39;: &#39;15e5dd0eb126bfe384c25ba551317eb9ba2ad1ef&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Perceptions of the Adult US Population regarding the Novel Coronavirus Outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028308&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: COVID-19 outbreak is spreading globally. Although the risk of infection in the US is currently low, it is important to understand the public perception of risk and trust in sources of information to better inform public health messaging. In this study, we surveyed the adult US population to understand their risk perceptions about the COVID-19 outbreak. Methods and Findings: We used an online platform to survey 718 adults in the US in early February 2020 using a questionnaire that we developed. Our sample was fairly similar to the general adult US population in terms of age, gender, race, ethnicity and education. We found that 69% of the respondents wanted the scientific/public health leadership (either the CDC Director or NIH Director) to lead the US response to COVID-19 outbreak as compared to 14% who wanted the political leadership (either the president or the Congress) to lead the response. Risk perception was low (median score of 5 out of 10) with the respondents trusting health professionals and health officials for information on COVID-19. Majority of the respondents were in favor of strict infection prevention policies to control the outbreak. Conclusion: Given our results, the public health/scientific leadership should be at the forefront of the COVID-19 response to promote trust.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;SarahAnn M McFadden; Amyn A Malik; Obianuju G Aguolu; Kathryn S Willebrand; Saad B Omer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028308&#39;},
 {&#39;cord_uid&#39;: &#39;cv3qgno3&#39;,
  &#39;sha&#39;: &#39;0fbb18050e29ca78191625d42576b1c574027377&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028373&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028373&#39;},
 {&#39;cord_uid&#39;: &#39;px0twvhs&#39;,
  &#39;sha&#39;: &#39;4bd697fdded7ee65154e28def586a7e9b60e7078&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission characteristics of the COVID-19 outbreak in China: a study driven by data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028431&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days. It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-01&#39;,
  &#39;authors&#39;: &#39;Meili Li; Pian Chen; Qianqian Yuan; Baojun Song; Junling Ma&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028431&#39;},
 {&#39;cord_uid&#39;: &#39;onkdv388&#39;,
  &#39;sha&#39;: &#39;54ae20515fcef8417e255b19df1a5bf6f2ab40bb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modelling the coronavirus disease (COVID-19) outbreak on the Diamond Princess ship using the public surveillance data from January 20 to February 20, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028449&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus disease 2019 (COVID-19) outbreak on the Diamond Princess ship has caused over 634 cases as of February 20, 2020. We model the transmission process on the ship with a stochastic model and estimate the basic reproduction number at 2.2 (95%CI: 2.1−2.4). We estimate a large dispersion parameter than other coronaviruses, which implies that the virus is difficult to go extinction. The epidemic doubling time is at 4.6 days (95%CI: 3.0−9.3), and thus timely actions were crucial. The lesson learnt on the ship is generally applicable in other settings.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Shi Zhao; Peihua Cao; Daozhou Gao; Zian Zhuang; Marc Chong; Yongli Cai; Jinjun Ran; Kai Wang; Yijun Lou; Weiming Wang; Lin Yang; Daihai He; Maggie H Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028449&#39;},
 {&#39;cord_uid&#39;: &#39;k260c04b&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Effects of &#34;Fangcang, Huoshenshan, and Leishenshan&#34; Makeshift Hospitals and Temperature on the Mortality of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028472&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background In December 2019, a novel coronavirus disease (COVID-19) broke out in Wuhan, China, however, the factors affecting the mortality remain unclear. Methods Thirty-two days of data that were shared by China National Health Commission and China Weather Net were collected using standard forms. The difference in the mortality of confirmed and severe cases before and after the use of Fangcang, Huoshenshan, and Leishenshan makeshift hospitals (MSHs) was tested using Mann-Whitney U test. We also studied whether air temperature (AT) could affect the above outcomes of COVID-19 cases by performing Spearman analysis. Results The mortality of confirmed cases was significantly decreased both in Wuhan (U = 1, P &lt; 0.001) and Hubei (U = 0, P &lt; 0.001), while in non-Hubei regions, as a contrast, the mortality of confirmed cases remained unchanged (U = 40, P = 0.139). However, another eight days later, changes in the mortality in non-Hubei regions also became significant (U = 73, P = 0.039). Mortality of confirmed cases was found to be significantly correlated with temperature both in Wuhan (r = -0.441, P = 0.012) and Hubei (r = -0.440, P = 0.012). Conclusions Our findings indicated that both the use of MSHs and the rise of AT were beneficial to the survival of COVID-19 cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Yuwen Cai; Tianlun Huang; Xin Liu; Gaosi Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028472&#39;},
 {&#39;cord_uid&#39;: &#39;zriuh5q5&#39;,
  &#39;sha&#39;: &#39;691da9d5576aa959e22114a1ebb35548efd276a0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Infection Dynamics of Coronavirus Disease 2019 (Covid-19) Modeled with the Integration of the Eyring Rate Process Theory and Free Volume Concept&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028571&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The Eyrings rate process theory and free volume concept, two very popular theories in chemistry and physics fields, are employed to treat infectious disease transmissions. The susceptible individ- uals are assumed to move stochastically from one place to another. The virus particle transmission rate is assumed to obey the Eyring rate process theory and also controlled by how much free volume available in a system. The transmission process is considered to be a sequential chemical reaction, and the concentrations or fractions of four epidemiological compartments, the susceptible, the exposed, the infected, and the removed, can be derived and calculated. The obtained equations show that the basic reproduction number, R0, is not a constant, dependent on the volume frac- tion of virus particles, virus particle size, and virus particle packing structure, the energy barrier associated with susceptible individuals, and environment temperature. The developed models are applied to treat coronavirus disease 2019 (Covid-19) transmission and make predictions on peak time, peak infected, and R0. Our work provides a simple and straightforward approach to estimate how infection diseases evolve and how many people may be infected.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Tian Hao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028571&#39;},
 {&#39;cord_uid&#39;: &#39;nsp2t0i6&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Heart injury signs are associated with higher and earlier mortality in coronavirus disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.20028589&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: Heart injury can be easily induced by viral infection such as adenovirus and enterovirus. However, whether coronavirus disease 2019 (COVID-19) causes heart injury and hereby impacts mortality has not yet been fully evaluated. Objective: To explore whether heart injury occurs in COVID-19 on admission and hereby aggravates mortality later. Design, Setting, and Participants A single-center retrospective cohort study including 188 COVID-19 patients admitted from December 25, 2019 to January 27, 2020 in Wuhan Jinyintan Hospital, China; follow up was completed on February 11, 2020. Exposures: High levels of heart injury indicators on admission (hs-TNI; CK; CK-MB; LDH; α-HBDH). Main Outcomes and Measures: Mortality in hospital and days from admission to mortality (survival days). Results: Of 188 patients with COVID-19, the mean age was 51.9 years (standard deviation: 14.26; range: 21~83 years) and 119 (63.3%) were male. Increased hs-TnI levels on admission tended to occur in older patients and patients with comorbidity (especially hypertension). High hs-TnI on admission (≥ 6.126 pg/mL), even within the clinical normal range (0~28 pg/mL), already can be associated with higher mortality. High hs-TnI was associated with increased inflammatory levels (neutrophils, IL-6, CRP, and PCT) and decreased immune levels (lymphocytes, monocytes, and CD4+ and CD8+ T cells). CK was not associated with mortality. Increased CK-MB levels tended to occur in male patients and patients with current smoking. High CK-MB on admission was associated with higher mortality. High CK-MB was associated with increased inflammatory levels and decreased lymphocytes. Increased LDH and α-HBDH levels tended to occur in older patients and patients with hypertension. Both high LDH and α-HBDH on admission were associated with higher mortality. Both high LDH and α-HBDH were associated with increased inflammatory levels and decreased immune levels. hs-TNI level on admission was negatively correlated with survival days (r= -0.42, 95% CI= -0.64~-0.12, P=0.005). LDH level on admission was negatively correlated with survival days (r= -0.35, 95% CI= -0.59~-0.05, P=0.022). Conclusions and Relevance: Heart injury signs arise in COVID-19, especially in older patients, patients with hypertension and male patients with current smoking. COVID-19 virus might attack heart via inducing inflammatory storm. High levels of heart injury indicators on admission are associated with higher mortality and shorter survival days. COVID-19 patients with signs of heart injury on admission must be early identified and carefully managed by cardiologists, because COVID-19 is never just confined to respiratory injury.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Chaomin Wu; Xianglin Hu; Jianxin Song; Chunling Du; Jie Xu; Dong Yang; Dechang Chen; Ming Zhong; Jinjun Jiang; Weining Xiong; Ke Lang; Yuye Zhang; Guohua Shi; Lei Xu; Yuanlin Song; Xin Zhou; Ming Wei; Junhua Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.20028589&#39;},
 {&#39;cord_uid&#39;: &#39;fvig79k3&#39;,
  &#39;sha&#39;: &#39;11875dfda6cd04d5032611edf0e340582ceb620f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Predictions for the binding domain and potential new drug targets of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.961938&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAn outbreak of new SARS-like viral in Wuhan, China has been named 2019-nCoV. The current state of the epidemic is increasingly serious, and there has been the urgent necessity to develop an effective new drug. In previous studies, it was found that the conformation change in CTD1 was the region where SARS-CoV bound to human ACE2. Although there are mutations of the 2019-nCoV, the binding energy of ACE2 remains high. The surface glycoprotein of 2019-nCoV was coincident with the CTD1 region of the S-protein by comparing the I-TASSER prediction model with the actual SARS model, which suggests that 2019-nCoV may bind to the ACE2 receptor through conformational changes. Furthermore, site prediction on the surface glycoprotein of 2019-nCoV suggests some core amino acid area may be a novel drug target against 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Zehua Zeng; Zhi Luo; Hongwu Du&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.961938&#39;},
 {&#39;cord_uid&#39;: &#39;wx7nheay&#39;,
  &#39;sha&#39;: &#39;313dd0869f169624e78d770963eacb0d1641cafa&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structure of Mpro from COVID-19 virus and discovery of its inhibitors&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.964882&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYA new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan1-4. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus5,6. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited strong antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Zhenming Jin; Xiaoyu Du; Yechun Xu; Yongqiang Deng; Meiqin Liu; Yao Zhao; Bing Zhang; Xiaofeng Li; Leike Zhang; Chao Peng; Yinkai Duan; Jing Yu; Lin Wang; Kailin Yang; Fengjiang Liu; Rendi Jiang; Xinglou Yang; Tian You; Xiaoce Liu; Xiuna Yang; Fang Bai; Hong Liu; Xiang Liu; Luke W. Guddat; Wenqing Xu; Gengfu Xiao; Chengfeng Qin; Zhengli Shi; Hualiang Jiang; Zihe Rao; Haitao Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.964882&#39;},
 {&#39;cord_uid&#39;: &#39;egntml7e&#39;,
  &#39;sha&#39;: &#39;0a32446730827ad8152c6a61e4738e4e0b231412&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Artesunate interacts with Vitamin D receptor to reverse mouse model of sepsis-induced immunosuppression via enhancing autophagy&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.966143&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackground and PurposeImmunosuppression is the predominant cause of mortality for sepsis due to failure to eradicate invading pathogens. Unfortunately, no effective and specific drugs capable of reversing immunosuppression are available for clinical use. Increasing evidence implicates vitamin D receptor (VDR) involved in sepsis-induced immunosuppression. Herein, artesunate (AS) was discovered to reverse sepsis-induced immunosuppression and its molecular mechanism is investigated.Experimental ApproachEffect of artesunate on sepsis-induced immunosuppression was investigated in mice and in vitro. VDR was predicted to be an interacted candidate of AS by bioinformatics predict, then identified using PCR and immunoblotting. VDR, ATG16L1 and NF-κB p65 were modified to investigate the alteration of AS’s effect on pro-inflammatory cytokines release, bacteria clearance and autophagy activities in sepsis-induced immunosuppression.Key ResultsAS significantly reduced the mortality of cecal ligation and puncture (CLP)-induced sepsis immunosuppression mice challenged with Pseudomonas Aeruginosa, and enhanced proinflammatory cytokines release and bacterial clearance to reverse sepsis-induced immunosuppression in vivo and in vitro. Mechanically, AS interacted with VDR thereby inhibited the nuclear translocation of VDR, then influencing ATG16L1 transcription and subsequent autophagy activity. In addition, AS inhibited physical interaction between VDR and NF-κB p65 in LPS tolerance macrophages, then promoted nuclear translocation of NF-κB p65, which activated the transcription of NF-κB p65 target genes such as pro-inflammatory cytokines.Conclusion and ImplicationsOur findings provide an evidence that AS interacted with VDR to reverse sepsis-induced immunosuppression in an autophagy and NF-κB dependent way, highlighting a novel approach for sepsis treatment and drug repurposing of AS in the future.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Shenglan Shang; Jiaqi Wu; Xiaoli Li; Xin Liu; Pan Li; Chunli Zheng; Yonghua Wang; Songqing Liu; Jiang Zheng; Hong Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.966143&#39;},
 {&#39;cord_uid&#39;: &#39;7tkvl894&#39;,
  &#39;sha&#39;: &#39;35604e3a5d38d6794543372e4dd9ccf1d8d9cd2a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.26.967026&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Hayden C. Metsky; Catherine A. Freije; Tinna-Solveig F. Kosoko-Thoroddsen; Pardis C. Sabeti; Cameron Myhrvold&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.26.967026&#39;},
 {&#39;cord_uid&#39;: &#39;jadzias9&#39;,
  &#39;sha&#39;: &#39;ae61300f4e7a12a68baf114f199eb51940c7aad3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20027169&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background and Objective: To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. Design and Methods: We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. Results: The peak time of the curve for the infected newly diagnosed as COVID-19 should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei. It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese COVID-19 epidemic can be completely controlled in May,2020. Conclusions: COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global SARS-CoV-2 cases rise with the great efforts by Chinese government and its people.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Rui Li; Wenliang Lu; Xifei Yang; Peihua Feng; Ozarina Muqimova; Xiaoping Chen; Gang Wei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20027169&#39;},
 {&#39;cord_uid&#39;: &#39;2zm4dq0u&#39;,
  &#39;sha&#39;: &#39;2e41aee6651cf4cab4edf5aa67cc401c2bef9d67&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Sex differences in clinical findings among patients with coronavirus disease 2019 (COVID-19) and severe condition&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20027524&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To compare the sex differences in the clinical findings among patients with severe coronavirus disease 2019 (COVID-19). Methods: We retrospectively collected data of 47 patients diagnosed as severe type of COVID-19 from February 8 to 22, 2020, including demographics, illness history, physical examination, laboratory test, management, and compared differences between men and women. Results: Of the 47 patients, 28 (59.6%) were men. The median age was 62 years, and 30 (63.8%) had comorbidities. The initial symptoms were mainly fever (34 [72.3%]), cough (36 [76.6%]), myalgia (5 [10.6%]) and fatigue (7 [14.9%]). Procalcitonin level was higher in men than in women (0.08 vs. 0.04ng/ml, p=0.002). N-terminal-pro brain natriuretic peptide increased in 16 (57.1%) men and 5 (26.3%) women (p=0.037). Five men (17.9%) had detected positive influenza A antibody, but no women. During 2-week admission, 5 (17.9%) men and 1 (5.3%) woman were reclassified into the critical type due to deterioration. Mortality was 3.6% in men and 0 in women respectively. Four (21.1%) women and one man (3.6%) recovered and discharged from hospital. Conclusion: Sex differences may exist in COVID-19 patients of severe type. Men are likely to have more complicated clinical condition and worse in-hospital outcomes as compared to women.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Jing Li; Yinghua Zhang; Fang Wang; Bing Liu; Hui Li; Guodong Tang; Zhigang Chang; Aihua Liu; Chunyi Fu; Jing Gao; Jing Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20027524&#39;},
 {&#39;cord_uid&#39;: &#39;owjgs6ja&#39;,
  &#39;sha&#39;: &#39;2d0285c90976dfa43d58eca6b7d1d7ef29994fc2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Therapeutic effects of dipyridamole on COVID-19 patients with coagulation dysfunction&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20027557&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The human coronavirus HCoV-19 infection can cause acute respiratory distress syndrome (ARDS), hypercoagulability, hypertension, extrapulmonary multiorgan dysfunction. Effective antiviral and anti-coagulation agents with safe clinical profiles are urgently needed to improve the overall prognosis. We screened an FDA approved drug library and found that an anticoagulant agent dipyridamole (DIP) suppressed HCoV-19 replication at an EC50 of 100 nM in vitro. It also elicited potent type I interferon responses and ameliorated lung pathology in a viral pneumonia model. In analysis of twelve HCoV-19 infected patients with prophylactic anti-coagulation therapy, we found that DIP supplementation was associated with significantly increased platelet and lymphocyte counts and decreased D-dimer levels in comparison to control patients. Two weeks after initiation of DIP treatment, 3 of the 6 severe cases (60%) and all 4 of the mild cases (100%) were discharged from the hospital. One critically ill patient with extremely high levels of D-dimer and lymphopenia at the time of receiving DIP passed away. All other patients were in clinical remission. In summary, HCoV-19 infected patients could potentially benefit from DIP adjunctive therapy by reducing viral replication, suppressing hypercoagulability and enhancing immune recovery. Larger scale clinical trials of DIP are needed to validate these therapeutic effects.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Xiaoyan Liu; Zhe Li; Shuai Liu; Zhanghua Chen; Zhiyao Zhao; Yi-you Huang; Qingling Zhang; Jun Wang; Yinyi Shi; Yanhui Xu; Jing Sun; Huifang Xian; Rongli Fang; Fan Bai; Changxing Ou; Bei Xiong; Andrew M Lew; Jun Cui; Hui Huang; Jincun Zhao; Xuechuan Hong; Yuxia Zhang; Fulin Zhou; Hai-Bin Luo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20027557&#39;},
 {&#39;cord_uid&#39;: &#39;s9j21zsy&#39;,
  &#39;sha&#39;: &#39;bd2f7613f5818fc82f76c7d68bdac08825957fb9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A machine learning-based model for survival prediction in patients with severe COVID-19 infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20028027&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The sudden increase of COVID-19 cases is putting a high pressure on healthcare services worldwide. At the current stage, fast, accurate and early clinical assessment of the disease severity is vital. To support decision making and logistical planning in healthcare systems, this study leverages a database of blood samples from 404 infected patients in the region of Wuhan, China to identify crucial predictive biomarkers of disease severity. For this purpose, machine learning tools selected three biomarkers that predict the survival of individual patients with more than 90% accuracy: lactic dehydrogenase (LDH), lymphocyte and high-sensitivity C-reactive protein (hs-CRP). In particular, relatively high levels of LDH alone seem to play a crucial role in distinguishing the vast majority of cases that require immediate medical attention. This finding is consistent with current medical knowledge that high LDH levels are associated with tissue breakdown occurring in various diseases, including pulmonary disorders such as pneumonia. Overall, this paper suggests a simple and operable formula to quickly predict patients at the highest risk, allowing them to be prioritised and potentially reducing the mortality rate.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-01&#39;,
  &#39;authors&#39;: &#39;Li Yan; Hai-Tao Zhang; Jorge Goncalves; Yang Xiao; Maolin Wang; Yuqi Guo; Chuan Sun; Xiuchuan Tang; Liang Jin; Mingyang Zhang; Xiang Huang; Ying Xiao; Haosen Cao; Yanyan Chen; Tongxin Ren; Fang Wang; Yaru Xiao; Sufang Huang; Xi Tan; Niannian Huang; Bo Jiao; Yong Zhang; Ailin Luo; Laurent Mombaerts; Junyang Jin; Zhiguo Cao; Shusheng Li; Hui Xu; Ye Yuan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20028027&#39;},
 {&#39;cord_uid&#39;: &#39;xiynfpuw&#39;,
  &#39;sha&#39;: &#39;264a816e7099246c13a5dad3a581a474d85d50bd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Hypokalemia and Clinical Implications in Patients with Coronavirus Disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20028530&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND: SARS-CoV-2 has caused a series of COVID-19 globally. SARS-CoV-2 binds angiotensin I converting enzyme 2 (ACE2) of renin-angiotensin system (RAS) and causes prevalent hypokalemia METHODS: The patients with COVID-19 were classified into severe hypokalemia, hypokalemia, and normokalemia group. The study aimed to determine the relationship between hypokalemia and clinical features, the underlying causes and clinical implications of hypokalemia. RESULTS: By Feb 15, 2020, 175 patients with COVID-19 (92 women and 83 men; median age, 46 [IQR, 34-54] years) were admitted to hospital in Wenzhou, China, consisting 39 severe hypokalemia-, 69 hypokalemia-, and 67 normokalemia patients. Gastrointestinal symptoms were not associated with hypokalemia among 108 hypokalemia patients (P&gt;0.05). Body temperature, CK, CK-MB, LDH, and CRP were significantly associated with the severity of hypokalemia (P&lt;0.01). 93% of severe and critically ill patients had hypokalemia which was most common among elevated CK, CK-MB, LDH, and CRP. Urine K+ loss was the primary cause of hypokalemia. severe hypokalemia patients was given 3 g/day, adding up to an average of 34 (SD=4) g potassium during hospital stay. The exciting finding was that patients responded well to K+ supplements when they were inclined to recovery. CONCLUSIONS: Hypokalemia is prevailing in patients with COVID-19. The correction of hypokalemia is challenging because of continuous renal K+ loss resulting from the degradation of ACE2. The end of urine K+ loss indicates a good prognosis and may be a reliable, in-time, and sensitive biomarker directly reflecting the end of adverse effect on RAS system.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;dong chen; Xiaokuni Li; qifa song; Chenchan Hu; Feifei Su; Jianyi Dai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20028530&#39;},
 {&#39;cord_uid&#39;: &#39;ry3m6x8c&#39;,
  &#39;sha&#39;: &#39;75e68869f9b65bca661e768402395d9dede6de2c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modeling the Epidemic Dynamics and Control of COVID-19 Outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20028639&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The coronavirus disease 2019 (COVID-19) is rapidly spreading over China and more than 30 countries in last two months. COVID-19 has multiple characteristics distinct from other infectious diseases, including a high infectivity during incubation, time delay between real dynamics and daily observed case numbers, and the effects from multiple quarantine and control measures. We develop a model SUQC to adequately characterizes the dynamics of COVID-19 and explicitly model the control by artificial measures, which is more suitable for analysis than other existing epidemic models. The SUQC model is applied to the daily released data of the confirmed infected to analyze the outbreak of COVID-19 in Wuhan, Hubei (excluding Wuhan), China (excluding Hubei) and four first-tier cities of China. We find that, before January 30, 2020, all these regions except Beijing have a reproductive number R&gt;1, and after January 30, all regions have a reproductive number R&lt;1, indicating the effectiveness of the quarantine and control measures in inhibiting COVID-19. The confirmation rate of Wuhan is 0.0643, significantly lower than 0.1914 of Hubei (excluding Wuhan) and 0.2189 of China (excluding Hubei), but increases to 0.3229 after Feb 12th when clinical diagnosis was adopted. The un-quarantined infected individuals in Wuhan on February 12, 2020 is as high as 3,509 and decreases to 334 on February 21th, 2020. After fitting the model with recent data, we predict that the end times of COVID-19 of Wuhan and Hubei are around late-March, of China (excluding Hubei) around mid-March, and of the four tier-one cities before March 2020. A total of 80,511 individuals of the whole country are infected, among which 49,510 are from Wuhan, 17,679 from Hubei(excluding Wuhan), and the rest 13,322 from other regions of China (excluding Hubei). We suggest the rigorous quarantine and control measures should be kept before March in Beijing, Shanghai, Guangzhou and Shenzhen, and before late-March in Hubei. The model can also be useful to predict the trend of epidemic and provide quantitative guide for other counties in a high risk of outbreak, such as South Korea, Japan and Iran.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Shilei Zhao; Hua Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20028639&#39;},
 {&#39;cord_uid&#39;: &#39;rbnfh89u&#39;,
  &#39;sha&#39;: &#39;b579c3547bba2a33057373d57b7c05f37dd4cfc3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20028787&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited. METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG). CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be greatly relieved.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-01&#39;,
  &#39;authors&#39;: &#39;Jie Xiang; Mingzhe Yan; Hongze Li; Ting Liu; Chenyao Lin; Shuang Huang; Changxin Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20028787&#39;},
 {&#39;cord_uid&#39;: &#39;z7a3g6e8&#39;,
  &#39;sha&#39;: &#39;905603e9db862f9b4b42a95f51e9c8d0015d1dff&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission potential of COVID-19 in South Korea&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20028829&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since the first identified individual of 2019 novel coronavirus (COVID-19) infection on Jan 20, 2020 in South Korea, the number of confirmed cases rapidly increased. As of Feb 26, 2020, 1,261 cases of COVID-19 including 12 deaths were confirmed in South Korea. Using the incidence data of COVID-19, we estimate the reproduction number at 1.5 (95% CI: 1.4-1.6), which indicates sustained transmission and support the implementation of social distancing measures to rapidly control the outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Eunha Shim; Amna Tariq; Wongyeong Choi; Yiseul Lee; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20028829&#39;},
 {&#39;cord_uid&#39;: &#39;hgn6o5yf&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A simple ecological model captures the transmission pattern of the coronavirus COVID-19 outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20028928&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The rapid spread of the 2019 novel coronavirus (COVID-19), initially reported in the city of Wuhan in China, and quickly transmitted to the entire nation and beyond, has become an international public health emergency. Estimating the final number of infection cases and the turning point (time with the fastest spreading rate) is crucial to assessing and improving the national and international control measures currently being applied. In this paper we develop a simple model based on infectious growth with a time-varying infection rate, and estimate the final number of infections and the turning point using data updated daily from 3 February 2020, when China escalated its initial public health measures, to 10 February. Our model provides an extremely good fit to the existing data and therefore a reasonable estimate of the time-varying infection rate that has largely captured the transmission pattern of this epidemic outbreak. Our estimation suggests that (i) the final number of infections in China could reach 78,000 with an upper 95% confidence limit of 88,880; (ii) the turning point of the fastest spread was on the 4th or the 5th of February; and (iii) the projected period for the end of the outbreak (i.e., when 95% of the final predicted number of infection is reached) will be the 24th of February, with an upper 95% confidence limit on the 19th of March. This suggests that the current control measures in China are excellent, and more than sufficient to contain the spread of this highly infectious novel coronavirus, and that the application of such measures could be considered internationally for the global control of this outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Feng Zhang; Jinmei Zhang; Menglan Cao; Cang Hui&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20028928&#39;},
 {&#39;cord_uid&#39;: &#39;9hh17k86&#39;,
  &#39;sha&#39;: &#39;68ed7bb581b0cf32116ee46767c08e3180b05c34&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.20029009&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Although the outbreak of Coronavirus disease 2019 (COVID-19) has caused over 2200 deaths in China, there was no study about death yet. We aimed to describe the clinical characteristics of non-survivors with COVID-19. Methods For this retrospective, single-center study, we included 36 non-survivors with COVID-19 in the Fifth Hospital of Wuhan. Cases were confirmed by real-time RT-PCR between Jan 21 and Feb 10, 2020 according to the recommended protocol. The epidemiological, demographic, clinical, laboratory, radiological and treatment data were collected and analyzed. Outcomes were followed up until Feb 14, 2020. This study was approved by the ethics commissions of the Fifth Hospital of Wuhan, with a waiver of informed consent due to a public health outbreak investigation. Results We included 36 patients who died from COVID-19. The mean age of the patients was 69.22 years (SD 9.64, range 50-90). 25(69.44%) patients were males, and 11 (30.56%) female. 26 (72.22%) patients had chronic diseases, mainly including hypertension, cardiovascular disease and diabetes. Patients had common clinical symptoms of fever (34 [94.44%] patients), cough (28 [77.78%] patients), shortness of breath (21 [58.33%] patients), and fatigue (17 [47.22%] patient). Chest computed tomographic scans showed that 31 (96.88%) patients had bilateral pneumonia. Lymphopenia occurred in 24 patients (70.59%), decreased albumin (30.18, [SD, 4.76]) in 25 patients (80.65%), elevated D-dimer (8.64 [IQR, 2.39-20]) in 27 patients (100%), and elevated lactate dehydrogenase (502.5 U/L [IQR, 410-629]) in 26 patients (100%). Nearly all of the patients have elevated CRP (106.3 mg/L [IQR, 60.83-225.3]), PCT (0.61 ng/ml [IQR, 0.16-2.10]) and IL-6 (100.6 pg/ml [IQR, 51.51-919.5]). Most patients received antiviral therapy and antibiotic therapy, and more than half of patients received glucocorticoid therapy (25 [69.44%]). All the patients had acute respiratory distress syndrome (ARDS). The median time from onset to ARDS was 11 days. One (2.78%) patient presented with acute renal injury. The median time from onset to death was 17 days. Interpretation Lots of patients died from COVID-19 till now. The median survival time of these non-survivors from onset to death was about 2 weeks. Most patients were older males with comorbidities. They finally progressed to ARDS. The median time from onset to ARDS was 11 days. Gradually decreased lymphocytes and increased inflammation biomarkers were common, and need to be monitored in the routine treatment.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Ying Huang; Rui Yang; Ying Xu; Ping Gong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.20029009&#39;},
 {&#39;cord_uid&#39;: &#39;618glydc&#39;,
  &#39;sha&#39;: &#39;1b9fed2824e97db8cb35912549dd2e9cde7b6c18&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Strategies for vaccine design for corona virus using Immunoinformatics techniques&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.967422&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Anamika Basu; Anasua Sarkar; Ujjwal Maulik&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.967422&#39;},
 {&#39;cord_uid&#39;: &#39;eaglecq7&#39;,
  &#39;sha&#39;: &#39;92c020779e68e82d9a5f5898435e618caa63a099&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.967588&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractFaced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the “coronavirus” should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Chunli Zhang; Yiqing Xue; Guangming Tan; Gang Niu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.967588&#39;},
 {&#39;cord_uid&#39;: &#39;1f9kjtmm&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The SARS-CoV-2 receptor ACE2 expression of maternal-fetal interface and fetal organs by single cell transcriptome study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.967760&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Recent studies have demonstrated that SARS-CoV-2 cell entry depends on both ACE2 and TMPRSS2 genes (DOI: 10.1016/j.cell.2020.02.052), but our current work only focus on ACE2, which is insufficient to support the conclusion of this paper. So the authors have withdrawn their manuscript whilst they perform additional experiments and analysis to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-27&#39;,
  &#39;authors&#39;: &#39;Mengmeng Li; Liang Chen; Chenglong Xiong; Xiangjie Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.967760&#39;},
 {&#39;cord_uid&#39;: &#39;9ruhvpbv&#39;,
  &#39;sha&#39;: &#39;ae66f22f886d3b019934ae237c11a5fb53dbefb2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Prediction of receptorome for human-infecting virome&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.967885&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-28&#39;,
  &#39;authors&#39;: &#39;Zheng Zhang; Sifan Ye; Aiping Wu; Taijiao Jiang; Yousong Peng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.967885&#39;},
 {&#39;cord_uid&#39;: &#39;mp3a9c9u&#39;,
  &#39;sha&#39;: &#39;7445c94870da80c81cd17f6a4c1358004956ee36&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Molecular Dynamics Simulations Indicate the SARS-CoV-2 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.968008&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe coronavirus outbreak took place in December 2019 and continues to spread worldwide. In the absence of an effective vaccine, inhibitor repurposing may seem a fruitful attempt. Here, we compared Mpros from SARS-CoV-2 and SARS-CoV. Despite a high level of sequence similarity, the binding sites of analysed proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, the analysis of the pockets’ conformational changes during the simulation time indicates their flexibility, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the SARS-CoV-2 Mpro with respect to mutations of the binding cavity and adjacent flexible loops indicates that the protein’s mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Maria Bzówka; Karolina Mitusińska; Agata Raczyńska; Aleksandra Samol; Jack Tuszyński; Artur Góra&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.968008&#39;},
 {&#39;cord_uid&#39;: &#39;3h1o0oz3&#39;,
  &#39;sha&#39;: &#39;6d0127f985edbe088bc279865bef25a31f54a066&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.27.969006&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.27.969006&#39;},
 {&#39;cord_uid&#39;: &#39;2vdvswda&#39;,
  &#39;sha&#39;: &#39;24e17488d399c436305c819953beae2961214771&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and clinical features of COVID-19 patients with and without pneumonia in Beijing, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20028068&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background:SARS-CoV-2-caused coronavirus disease (COVID-19) is posinga large casualty. The features of COVID-19patients withand without pneumonia,SARS-CoV-2 transmissibility in asymptomatic carriers, and factors predicting disease progression remain unknown. Methods: We collected information on clinical characteristics, exposure history, andlaboratory examinations of all laboratory-confirmed COVID-19 patients admitted to PLA General Hospital. Cox regression analysis was applied to identify prognostic factors. The last follow-up was February 18, 2020. Results:We characterized 55 consecutive COVID-19 patients. The mean incubation was 8.42(95% confidence interval [CI], 6.55-10.29) days. The mean SARS-CoV-2-positive duration from first positive test to clearance was 9.71(95%CI, 8.21-11.22) days. COVID-19 course was approximately 2 weeks. Asymptomatic carriers might transmit SARS-CoV-2. Compared with patients without pneumonia, those with pneumonia were 15 years older and had a higher rate of hypertension, higher frequencies of having a fever and cough, and higher levels of interleukin-6 (14.61 vs. 8.06pg/mL, P=0.040), B lymphocyte proportion (13.0% vs.10.0%, P=0.024), low account (&lt;190/μL) of CD8+ T cells (33.3% vs. 0, P=0.019). Multivariate Cox regression analysis indicated that circulating interleukin-6 andlactate independently predicted COVID-19 progression, with a hazard ratio (95%CI) of 1.052 (1.000-1.107) and 1.082 (1.013-1.155), respectively. During disease course,T lymphocytes were generally lower,neutrophils higher, in pneumonia patients than in pneumonia-free patients. CD8+ lymphocytes did not increase at the 20th days after illness onset. Conclusion: The epidemiological features areimportant for COVID-19 prophylaxis. Circulating interleukin-6 and lactateare independent prognostic factors. CD8+ T cell exhaustion might be critical in the development of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Penghui Yang; Yibo Ding; Zhe Xu; Rui Pu; Ping Li; Jin Yan; Jiluo Liu; Fanping Meng; Lei Huang; Lei Shi; Tianjun Jiang; Enqiang Qin; Min Zhao; Dawei Zhang; Peng Zhao; Lingxiang Yu; Zhaohai Wang; Zhixian Hong; Zhaohui Xiao; Qing Xi; Dexi Zhao; Peng Yu; Caizhong Zhu; Zhu Chen; Shaogeng Zhang; Junsheng Ji; Guangwen Cao; Fusheng Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20028068&#39;},
 {&#39;cord_uid&#39;: &#39;73gxq2w6&#39;,
  &#39;sha&#39;: &#39;54cfc3c68e1e4832fee5b3294e5673e37978ebdf&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Risk factors related to hepatic injury in patients with corona virus disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20028514&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Aims: Corona virus disease 2019 (COVID-19) has rapidly become the most severe public health issue all over the world. Despite respiratory symptoms, hepatic injury has also been observed in clinical settings. This study aimed to investigate the risk factors involved with hepatic injury in the patients with COVID-19. Methods: A total of 85 hospitalized patients who were diagnosed with COVID-19 in Beijing Youan Hospital were retrospectively analyzed. According to liver function, they were divided into ALT normal group (n=52) and ALT elevation group (n=33). Clinical features and laboratory data were compared between the two groups. The independent risk factors for liver injury were analyzed. Results: There were 33 patients with hepatic injury in our study, accounting for 38.8% (33/85). The patients in ALT elevation group were older than those in ALT normal group. The levels of lactic acid, CRP, myoglobin, and neutrophils were significantly higher in ALT elevation group. The lymphocytes and albumin were significantly lower in ALT elevation group. The proportion of severe and critical patients in ALT elevation group was significantly higher. Multivariate logistic regression analysis showed CRP ≥20 mg/L and lymphocyte count&lt; 1.1*10^9/L were independently related to hepatic injury. Conclusions: Lymphopenia and CRP may serve as the risk factors related to hepatic injury in patients with COVID-19, which might be related to inflammatory cytokine storm in liver injury. Early detection and timely treatment of hepatic injury in patients with COVID-19 are necessary.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Lu Li; Shuang Li; Manman Xu; Pengfei Yu; Sujun Zheng; Zhongping Duan; Jing Liu; Yu Chen; Junfeng Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20028514&#39;},
 {&#39;cord_uid&#39;: &#39;3wvcuboc&#39;,
  &#39;sha&#39;: &#39;d75c48f51b2a5c26a41fe615339e5947d5e9cb30&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis of epidemiological characteristics of coronavirus 2019 infection and preventive measures in Shenzhen China: a heavy population city&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20028555&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Coronavirus 2019 infection (COVID-19) outbroke in Wuhan, Hubei and spread to all provinces in China and other countries. Shenzhen ranked the top cities outside Wuhan with reported 416 confirmed cases by February 20, 2020. Here, we analyzed the epidemiological characteristics of COVID-19 in Shenzhen and potential link to the preventive strategies for the whole city and inside hospitals. Based on the daily new cases, the epidemic of COVID-19 in Shenzhen can be classified into three phases: the slow increase phase from January 19 to January 28, the rapid increase and plateau phase from January 29 to February 5 and the decline phase since February 6. In the three phases, the number of patients from Hubei decreased, and the number of familial clustering cases increased. The newly diagnosed COVID-19 cases reached its peak around January 31, which was 7 days after the peak date of cases arrival at Shenzhen. A series of early preventive strategies were implemented since January 19, which included detection of body temperature at all entrances of main traffic and buildings, outpatients service specially for patients with fever in all main hospitals in Shenzhen. All the patients with fever were screened with nasal or throat swab PCR detection of coronavirus 2019, Chest CT and blood lymphocyte counting in order to find out early case of COVID-19. Observation wards were established in every main hospital and a designated hospital was responsible for admission and medical care of all confirmed cases. Protection procedure was established for all medical staff involved in the screening and care of suspected and confirmed cases. 14 days isolated observation of all subjects arrived at Shenzhen from Hubei was implemented in February 2. After the implementation of all these strategies and measures, the COVID-19 cases started to decline since February 6. There were almost no community transmission and nosocomial infection occurred in Shenzhen. In conclusion, in situation of major outbreak of respiratory infectious disease, such as COVID-19, in nearby province of Hubei, Shenzhen, a high population density, high proportion of external population and high mobility city, has to face the imported cases and risk of spreading the outbreak into Shenzhen city. The implementation of early preventive strategies and measures in Shenzhen were successful in early identification of COVID-19 cases and prevented major outbreak occurred in Shenzhen. Early identification of imported cases, prevention of family clustering transmission, preventive measures in the public area and very strict infection control procedure in hospital setting are crucial for the successful control of outbreak in Shenzhen.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Kai Yang; Lingwei Wang; Furong Li; Dandan Chen; Xi Li; Chen Qiu; Rongchang Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20028555&#39;},
 {&#39;cord_uid&#39;: &#39;py38bel4&#39;,
  &#39;sha&#39;: &#39;8575c06c64fdd0bb43efc46cd994f557698c755f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical significance of IgM and IgG test for diagnosis of highly suspected COVID-19 infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20029025&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Quick, simple and accurate diagnosis of suspected COVID-19 is very important for the screening and therapy of patients. Although several methods were performed in clinical practice, however, the IgM and IgG diagnostic value evaluation was little performed. 57 suspected COVID-19 infection patients were enrolled in our study. 24 patients with positive and 33 patients with negative nucleic acid test. The positive rate of COVID-19 nucleic acid was 42.10%. The positive detection rate of combination of IgM and IgG for patients with COVID-19 negative and positive nucleic acid test was 72.73% and 87.50%. The results were significantly higher than the nucleic acid or IgM, IgG single detection. hsCRP in the COVID-19 nucleic acid negative group showed significantly higher than the positive groups (P=0.0298). AST in the COVID-19 IgM negative group showed significantly lower than the positive groups (P=0.0365). We suggest a quick, simple, accurate aided detection method for the suspected patients and on-site screening in close contact with the population.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Xingwang Jia; Pengjun Zhang; Yaping Tian; Junli Wang; Huadong Zeng; Jun Wang; Liu Jiao; Zeyan Chen; Lijun Zhang; Haihong He; Kunlun He; Yajie Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20029025&#39;},
 {&#39;cord_uid&#39;: &#39;p5ecpcll&#39;,
  &#39;sha&#39;: &#39;a5039336577ac0fed15b784adb912df2a6a143f3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis on the Clinical Characteristics of 36 Cases of Novel Coronavirus Pneumonia in Kunming&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20029173&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Objective: To analyze the clinical characteristics of patients with novel coronavirus pneumonia in Kunming City, and to study the correlation between nutritional status and immune function. Methods: Clinical data of 36 patients with novel coronavirus pneumonia in isolation area of Kunming Third People&#39;s Hospital from January 31 to February 15, 2020 were collected, and the basic situation, clinical characteristics, laboratory examination and CT imaging characteristics were analyzed. Serum albumin (ALB), prealbumin (PAB), hypersensitive c-reactive protein (hs-crp), CD3T cells, CD4T cells, CD8T cells and normal control group were analyzed. A simple linear regression analysis of the relationship between proalbumin and T cell subpopulation counts in the blood of patients. Results: (1) The patients with new coronavirus pneumonia in Kunming were mainly of common type. (2) 50% of the patients&#39; first symptoms were fever and cough; (3) The total number of white blood cells in peripheral blood was normal or decreased in 23 cases (79%), and the lymphocyte count decreased in 5 cases (13.89%), without anemia. Hypersensitive c-reactive protein increased in 19 (52.78%) cases, and procalcitonin increased in 1 case. Albumin decreased in 5 cases (13.89%), proalbumin decreased in 15 cases (41.67%), alanine transaminase increased slightly in 4 cases (11.11%), alanine transaminase increased slightly in 4 cases (11.11%), total bilirubin increased slightly in 11 cases (30.56%), and renal function and blood coagulation were normal. Absolute value of CD3+T cells is with a decrease in 21 cases (58.3%), CD4+T in 28 cases (77.8%), CD8+T in 17 cases (47.2%), and CD4+/ CD8+ inverse in 6 cases (16.7%). (4) The prealbumin, CD3 T cells, CD4 T cells and CD8 T cells in the new coronavirus pneumonia group were significantly lower than those in the normal control group, and the hypersensitive c-reactive protein was higher than that in the normal control group. (5) The levels of PAB in the serum of the patients were linearly correlated with hs-crp, CD3 T cells, CD4 T cells and CD8 T cells, and the correlation coefficients were -0.474, 0.558, 0.467 and 0.613, respectively, showing statistical differences. Conclusion: The clinical characteristics of the novel coronavirus pneumonia in Kunming are different from those in Wuhan. The changes of serum proalbumin and T cell subsets are relatively obvious. Changes in serum proalbumin may contribute to the early warning of novel coronavirus pneumonia. The nutritional status of patients with common and mild pneumonia should be considered.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-01&#39;,
  &#39;authors&#39;: &#39;Haiyan Fu; Hongjuan Li; Xiaoqing Tang; Xiang Li; Jie Shen; Yujun Zhou; Bing Xu; Yu Luo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20029173&#39;},
 {&#39;cord_uid&#39;: &#39;6uaj8fb7&#39;,
  &#39;sha&#39;: &#39;2af91bbb625d983d5ad3aadb9e5f1d82268a0f1f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Highly ACE2 Expression in Pancreas May Cause Pancreas Damage After SARS-CoV-2 Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20029181&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The ongoing outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) started in the end of 2019 in China has triggered a global public health crisis. Previous studies have shown that SARS-CoV-2 infects cells by binding angiotensin-converting enzyme 2 (ACE2), which is the same as SARS-CoV. The expression and distribution of ACE2 in the pancreas are unknown. At the same time, the injury of pancreas after SARS-CoV-2 infection has not been concerned. Here, we collected public datasets (bulk RNA-seq and single-cell RNA-seq) to indicate the expression and the distribution of ACE2 in pancreas (in both exocrine glands and islets). And further, clinical data including mild and severe patients with COVID-19 demonstrated there existed mild pancreatitis. In the 67 severe cases, 11 patients (16.41%) showed elevated levels of both amylase and lipase, and 5 patients (7.46%) showed imaging alterations. Only one patient (1.85%) showed elevated levels of both amylase and lipase in 54 mild cases, without imaging changes. Our study revealed the phenomenon and possible cause of mild pancreatic injury in patients with COVID-19. This suggests that pancreatitis after SARS-CoV-2 infection should also be paid attention in clinical work.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Furong Liu; Xin Long; Wenbin Zou; Minghao Fang; Wenjuan Wu; Wei Li; Bixiang Zhang; Wanguang Zhang; Xiaoping Chen; Zhanguo Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20029181&#39;},
 {&#39;cord_uid&#39;: &#39;zr34wsae&#39;,
  &#39;sha&#39;: &#39;530f3ba1a2c044305d73123fd8d2246f63120bc3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Impact of the COVID-19 Outbreak on the Medical Treatment of Chinese Children with Chronic Kidney Disease (CKD)：A Multicenter Cross-section Study in the Context of a Public Health Emergency of International Concern&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20029199&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To investigate the impact of the COVID-19 outbreak on the medical advice seeking of Chinese children with chronic kidney disease (CKD). Materials and Methods: An anonymous online questionnaire survey was conducted in 17 pediatric nephropathy diagnosis and treatment centers in China. The questions collected basic information on the patients and their parents and data on changes in the approach to medical treatment and their needs in the context of the outbreak etc. This is a Multicenter Cross-section Study. Results: A total of 735 valid questionnaires were collected. 555 patients (75.5%) and their parents said that the outbreak had a significant influence on their medical treatment: 264 patients (47.6%) said that it would be delayed by 2 to 4 weeks and 199 patients (35.9%) by 4 to 8 weeks. 510 patients (84.16%) hoped to get in touch with specialists through online consultation, and 528 patients (84.5%) hoped that online consultation could be implemented and that medication could be delivered to them.. A total of 458 patients (62.3%) said that their greatest concern was that the CKD would be aggravated or that they would experience a relapse; only 203 patients were infected by 2019-nCoV. A total of 313 patients (42.5%) experienced anxiety and thus required the intervention of psychologists. Conclusion: The COVID-19 outbreak has affected the medical treatment of children with CKD. Online consultation, medication delivery and psychological counselling are the greatest needs reported by patients and their families and could especially provide solutions for the management of low income children with CKD in remote rural areas in the context of the COVID-19 epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Gaofu Zhang; Haiping Yang; Aihua Zhang; Qian Shen; Li Wang; Zhijuan Li; Yuhong Li; Lijun Zhao; Yue Du; Liangzhong Sun; Bo Zhao; Hongtao Zhu; Haidong Fu; Xiaoyan Li; Xiaojie Gao; Sheng Hao; Juanjuan Ding; Zongwen Chen; Zhiquan Xu; Xiaorong Liu; Daoqi Wu; Mingsi Gao; Mo Wang; Qiu Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20029199&#39;},
 {&#39;cord_uid&#39;: &#39;p3dex2to&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Closed environments facilitate secondary transmission of coronavirus disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.20029272&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Commissioned by the Minister of the Ministry of Health, Labour, and Welfare of Japan, we collected secondary transmission data with the aim of identifying high risk transmission settings. We show that closed environments contribute to secondary transmission of COVID-19 and promote superspreading events. Closed environments are consistent with large-scale COVID-19 transmission events such as that of the ski chalet-associated cluster in France and the church- and hospital-associated clusters in South Korea. Our findings are also consistent with the declining incidence of COVID-19 cases in China, as gathering in closed environments was prohibited in the wake of the rapid spread of the disease. Reduction of unnecessary close contact in closed environments may help prevent large case clusters and superspreading events.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Hiroshi Nishiura; Hitoshi Oshitani; Tetsuro Kobayashi; Tomoya Saito; Tomimasa Sunagawa; Tamano Matsui; Takaji Wakita; Motoi Suzuki&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.20029272&#39;},
 {&#39;cord_uid&#39;: &#39;5hkd80eh&#39;,
  &#39;sha&#39;: &#39;da3aa20131ac2805c0d9e1b29f094683479ab5b7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Ruler elements in chromatin remodelers set nucleosome array spacing and phasing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.969618&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Arrays of regularly spaced nucleosomes dominate chromatin and are often phased by alignment to reference sites like active promoters. How the distances between nucleosomes (spacing), and between phasing sites and nucleosomes are determined remains unclear, and specifically, how ATP dependent chromatin remodelers impact these features. Here, we used genome-wide reconstitution to probe how Saccharomyces cerevisiae ATP dependent remodelers generate phased arrays of regularly spaced nucleosomes. We find that remodelers bear a functional element named the ‘ruler’ that determines spacing and phasing in a remodeler-specific way. We use structure-based mutagenesis to identify and tune the ruler element residing in the Nhp10 and Arp8 modules of the INO80 remodeler complex. Generally, we propose that a remodeler ruler regulates nucleosome sliding direction bias in response to (epi)genetic information. This finally conceptualizes how remodeler-mediated nucleosome dynamics determine stable steady-state nucleosome positioning relative to other nucleosomes, DNA bound factors, DNA ends and DNA sequence elements.&#39;,
  &#39;publish_time&#39;: &#39;2020-02-29&#39;,
  &#39;authors&#39;: &#39;Elisa Oberbeckmann; Vanessa Niebauer; Shinya Watanabe; Lucas Farnung; Manuela Moldt; Andrea Schmid; Patrick Cramer; Craig L. Peterson; Sebastian Eustermann; Karl-Peter Hopfner; Philipp Korber&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.969618&#39;},
 {&#39;cord_uid&#39;: &#39;48mtdwuv&#39;,
  &#39;sha&#39;: &#39;3d498994d143a4ed7cc381700a19f2790d46bb36&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.969782&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYIn the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification1,2,3,4. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers5. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes6,7,8. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification9,10. Feature extraction techniques from natural language processing have previously been analogized to genomics data11; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at https://github.com/mriglobal/vorpal.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.969782&#39;},
 {&#39;cord_uid&#39;: &#39;2uvibr2j&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.28.970343&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Muhammad Tahir ul Qamar; Abdur Rehman; Usman Ali Ashfaq; Muhammad Qasim Awan; Israr Fatima; Farah Shahid; Ling-Ling Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.28.970343&#39;},
 {&#39;cord_uid&#39;: &#39;0ga5rel6&#39;,
  &#39;sha&#39;: &#39;0b460e73926eb107001c95c0dfeae2362b4251b0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Precautions are Needed for COVID-19 Patients with Coinfection of Common Respiratory Pathogens&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20027698&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: With the ongoing outbreak of Coronavirus Disease 2019 (COVID-19), infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan. Methods: We conducted a double-centre study recruiting 68 patients with severe acute respiratory coronavirus 2 (SARS-CoV-2) infection confirmed by nucleic acid testing in Qingdao and Wuhan from January 17 to February 16, 2020. Indirect immunofluorescence was performed to detect the specific IgM antibody against common respiratory pathogens in collected acute phase serum. Results: Of the 68 patients with SARS-CoV-2 infection, 30 (44.12%) were from Qingdao. The median age of Qingdao and Wuhan patients were 50 (IQR: 37-59) and 31 (IQR: 28-38) years, respectively, and the majority of patients were female in Qingdao (60.00%) and Wuhan (55.26%). Among COVID-19 patients in Qingdao, 24 (80.00%) of them had IgM antibodies against at least one respiratory pathogen, whereas only one (2.63%) of the patients in Wuhan had positive results for serum IgM antibody detection (P&lt;0.0001). The most common respiratory pathogens detected in Qingdao COVID-19 patients were influenza virus A (60.00%) and influenza virus B (53.33%), followed by mycoplasma pneumoniae (23.33%) and legionella pneumophila (20.00%). While the pattern for coinfection in patients with community-acquired pneumonia in Qingdao was quite different, with a positive rate of only 20.90%. Interpretation: We reported a large proportion of COVID-19 patients with coinfection of seasonal respiratory pathogens in Qingdao, northeast China, which differed greatly from the patients in Wuhan, central China. Precautions are needed when dealing with COVID-19 patients beyond the epidemic centre who have coinfection with other respiratory pathogens. We highly recommend adding SARS-CoV-2 to routine diagnostic testing in capable hospitals to prevent misdetection of the virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Quansheng Xing; Guoju Li; Yuhan Xing; Ting Chen; Wenjie Li; Wei Ni; Kai Deng; Ruqin Gao; Changzheng Chen; Yang Gao; Qiang Li; Guiling Yu; Jianning Tong; Wei Li; Guiliang Hao; Yue Sun; Ai Zhang; Qin Wu; Zipu Li; Silin Pan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20027698&#39;},
 {&#39;cord_uid&#39;: &#39;s43mous6&#39;,
  &#39;sha&#39;: &#39;14783283da3482bf2aab4e6100fe4ea5c29d8db7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Vicarious traumatization in the general public, members, and non-members of medical teams aiding in COVID-19 control&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029322&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since December 2019, more than 79,000 people have been diagnosed with infection of the Corona Virus Disease 2019 (COVID-19). A large number of medical staff were dispersed for Wuhan city and Hubei province to aid COVID-19 control. Psychological stress, especially vicarious traumatization (VT) caused by the COVID-19 pandemic, should not be ignored. To address this concern, the study employed a total of 214 general public (GP) and 526 nurses to evaluate VT scores via a mobile app-based questionnaire. Results showed that the VT scores slightly increased across periods of aiding COVID-19 control, although no statistical difference was noted (P = 0.083). However, the study found lower scores for VT in nurses [median = 69; interquartile range (IQR) = 56-85] than those of the GP (median = 75.5; IQR = 62-88.3) (P = 0.017). In addition, the VT scores for front-line nurses (FLNs; median = 64; IQR = 52-75), including scores for physiological and psychological responses, were significantly lower than those of non-front-line nurses (nFLNs; median = 75.5; IQR = 63-92) (P &lt; 0.001). Interestingly, the VT scores of the GP were significantly higher than those of the FLNs (P &lt; 0.001). However, no statistical difference was observed compared with those of nFLNs (P &gt; 0.05). Importantly, nFLNs are more likely to suffer from VT, which might be related to two factors, namely, gender [odds ratio (OR) = 3.1717; 95% confidence interval (CI) = 4.247-18.808; P = 0.002] and fertility [OR = 2.072; 95%CI = 0.626-24.533; P = 0.039]. Therefore, increased attention should be paid to the psychological problems of the medical staff, especially nFLNs, and GP under the situation of the spread and control of COVID-19. Early strategies that aim to prevent and treat VT in medical staff and GP are extremely necessary.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Zhenyu Li; Jingwu Ge; Meiling Yang; Jianping Feng; Mei Qiao; Riyue Jiang; Jiangjiang Bi; Gaofeng Zhan; Xiaolin Xu; Long Wang; Qin Zhou; Chenliang Zhou; Yinbing Pan; Shijiang Liu; Haiwei Zhang; Jianjun Yang; Bin Zhu; Yimin Hu; Kenji Hashimoto; Yan Jia; Haofei Wang; Rong Wang; Cunming Liu; Chun Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029322&#39;},
 {&#39;cord_uid&#39;: &#39;9lissxsy&#39;,
  &#39;sha&#39;: &#39;5241581d168b90832f88103bd40c8db2d843a6ea&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Association of Cardiovascular Manifestations with In-hospital Outcomes in Patients with COVID-19: A Hospital Staff Data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029348&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The outbreaks of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) remain a huge threat to the public health worldwide. Clinical data is limited up to now regarding the risk factors in favor of severe conversion of non-severe case with COVID-19. Aims: This study analyzed a hospital staff data to figure out general clinical features of COVID-19 in terms of the association of cardiovascular manifestations (CVMs) with in-hospital outcomes of COVID-19 cases. Methods: Retrospective, single-center case series of 41 consecutive hospitalized health staff with confirmed COVID-19 were collected at the Central Hospital of Wuhan in Wuhan, China, from January 15 to January 24, 2020. Epidemiological, demographic, clinical, laboratory, radiological, treatment data and in-hospital adverse events were collected and analyzed. Final date of follow-up was March 3, 2020. A comparative study was applied between cases with CVMs and those without CVMs. Results: Of all, clinicians and clinical nurses accounted for 80.5%, while 87.8% of all had history of patient contact. The population was presented with a mean age of 39.1 +- 9.2 and less comorbidities than community population. The three most frequent symptoms of COVID-19 cases analyzed were fever (82.9%), myalgia or fatigue (80.5%) and cough (63.4%). While, the three most frequent initial symptoms were myalgia or fatigue (80.5%), fever (73.2%) and cough (41.5%). There were 95.1% cases featured as non-severe course of disease according to the official standard in China. Patients with CVMs and those without CVMs accounted for 58.5% and 41.5%, respectively. Compared with cases without CVMs, patients with CVMs were presented with lower baseline lymphocyte count (0.99 +- 0.43 and 1.55 +- 0.61, P&lt;0.001), more who had at least once positive nucleic acid detection of throat swab during admission (50.0% and 11.8%, P=0.011), and more received oxygen support (79.2% and 23.5%, P&lt;0.001). The rate of in-hospital adverse events was significantly higher in patients with CVMs group (75.0% and 23.5%, P=0.001). Multivariable logistic regression model indicated that, coexisting with CVMs in COVID-19 patients was not independently associated with in-hospital adverse events. Conclusions: Most of hospital staff with COVID-19 had history of patient contact, featured non-severe course of disease. Cases with CVMs suffered from more in-hospital adverse events than those without CVMs. But concomitant CVMs were not independently associated with in-hospital adverse events in COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Ru Liu; Xiaoyan Ming; Ou Xu; Jianli Zhou; Hui Peng; Ning Xiang; Jiaming Zhang; Hong Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029348&#39;},
 {&#39;cord_uid&#39;: &#39;qs7dsxn5&#39;,
  &#39;sha&#39;: &#39;f87eec8279bc79900914c5689bb42f49f41aace5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The spatiotemporal estimation of the dynamic risk and the international transmission of 2019 Novel Coronavirus (COVID-19) outbreak: A global perspective&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029413&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;An ongoing novel coronavirus SARS-CoV-2 pneumonia infection outbreak called COVID-19 started in Wuhan, Hubei Province, China, in December 2019. It both spread rapidly to all provinces in China and started spreading around the world quickly through international human movement from January 2020. Currently, the spatiotemporal epidemic transmission patterns, prediction models, and possible risk analysis for the future are insufficient for COVID-19 but we urgently need relevant information, particularly from the global perspective. We have developed a novel two-stage simulation model to simulate the spatiotemporal changes in the number of COVID-19 cases and estimate the future worldwide risk. Based on the connectivity of countries to China and the country&#39;s medical and epidemic prevention capabilities, different scenarios are generated to analyze the possible transmission throughout the world and use this information to evaluate each country&#39;s vulnerability to and the dynamic risk of COVID-19. Countries&#39; vulnerability to the COVID-19 outbreak from China is calculated for 63 countries around the world. Taiwan, South Korea, Hong Kong, and Japan are the most vulnerable areas. The relationship between each country&#39;s vulnerability and days before the first imported case occurred shows a very high exponential decrease. The cumulative number of cases in each country also has a linear relationship with vulnerability, which can compare and quantify the initial epidemic prevention capabilities to various countries&#39; management strategies. In total, 1,000 simulation results of future cases around the world are generated for the spatiotemporal risk assessment. According to the simulation results of this study, if there is no specific medicine for it, it will likely form a global pandemic. This method can be used as a preliminary risk assessment of the spatiotemporal spread for a new global epidemic. * Note: This study was completed on February 15, 2020.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Yuan-Chien Lin; Wan-Ju Chi; Yu-Ting Lin; Chun-Yeh Lai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029413&#39;},
 {&#39;cord_uid&#39;: &#39;m9icky9z&#39;,
  &#39;sha&#39;: &#39;c12ad8b411c24fe215be91d1f990645743e55ab5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;An epidemiological forecast model and software assessing interventions on COVID-19 epidemic in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029421&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We develop a health informatics toolbox that enables public health workers to timely analyze and evaluate the time-course dynamics of the novel coronavirus (COVID-19) infection using the public available data from the China CDC. This toolbox is built upon a hierarchical epidemiological model in which two observed time series of daily proportions of infected and removed cases are emitted from the underlying infection dynamics governed by a Markov SIR infectious disease process. We extend the SIR model to incorporate various types of time-varying quarantine protocols, including government-level macro isolation policies and community-level micro inspection measures. We develop a calibration procedure for under-reported infected cases. This toolbox provides forecast, in both online and offline forms, of turning points of interest, including the time when daily infected proportion becomes smaller than the previous ones and the time when daily infected proportions becomes smaller than that of daily removed proportion, as well as the ending time of the epidemic. An R software is made available for the public, and examples on the use of this software are illustrated. Some possible extensions of our novel epidemiological models are discussed.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Peter X Song; Lili Wang; Yiwang Zhou; Jie He; Bin Zhu; Fei Wang; Lu Tang; Marisa Eisenberg&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029421&#39;},
 {&#39;cord_uid&#39;: &#39;b8eezxv3&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029439&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Tao Suo; Xinjin Liu; Ming Guo; Jiangpeng Feng; Wenjia Hu; Yang Yang; Qiuhan Zhang; Xin Wang; Muhanmmad Sajid; Dong Guo; Zhixiang Huang; Liping Deng; Tielong Chen; Fang Liu; Ke Xu; Yuan Liu; Qi Zhang; Yingle Liu; Yong Xiong; Guozhong Guo; Yu Chen; Ke Lan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029439&#39;},
 {&#39;cord_uid&#39;: &#39;ndn6iwre&#39;,
  &#39;sha&#39;: &#39;1ad1d4b84aea4ceaf05c62a1ad04e7150f7f4684&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029462&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Nicholas Easom; Peter Moss; Gavin Barlow; Anda Samson; Tom Taynton; Kate Adams; Monica Ivan; Phillipa Burns; Kavitha Gajee; Kirstine Eastick; Patrick Lillie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029462&#39;},
 {&#39;cord_uid&#39;: &#39;g41pd9uz&#39;,
  &#39;sha&#39;: &#39;7852aafdfb9e59e6af78a47af796325434f8922a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029520&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Although the SARS-CoV-2 viral load detection of respiratory specimen has been widely used for novel coronavirus disease (COVID-19) diagnosis, it is undeniable that serum SARS-CoV-2 nucleic acid (RNAaemia) could be detected in a fraction of the COVID-19 patients. However, it is not clear that if the incidence of RNAaemia could be correlated with the occurrence of cytokine storm or with the specific class of patients. Methods: This study enrolled 48 patients with COVID-19 admitted to the General Hospital of Central Theater Command, PLA, a designated hospital in Wuhan, China. The patients were divided into three groups according to the Diagnosis and Treatment of New Coronavirus Pneumonia (version 6) published by the National Health Commission of China. The clinical and laboratory data were collected. The serum viral load detection and serum IL-6 levels were determined. Except for routine statistical analysis, Generalized Linear Models (GLMs) analysis was used to establish a patient status prediction model based on real-time RT-PCR Ct value. Findings: The Result showed that cases with RNAaemia were exclusively confirmed in critically ill patients group and appeared to reflect the illness severity. Further more, the inflammatory cytokine IL-6 levels were significantly elevated in critically ill patients, which is almost 10-folds higher than those in other patients. More importantly, the extremely high IL-6 level was closely correlated with the incidence of RNAaemia (R=0.902) and the vital signs of COVID-19 patients (R= -0.682). Interpretation: Serum SARS-CoV-2 viral load (RNAaemia) is strongly associated with cytokine storm and can be used to predict the poor prognosis of COVID-19 patients. Moreover, our results strongly suggest that cytokine IL-6 should be considered as a therapeutic target in critically ill patients with excessive inflammatory response.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Xiaohua Chen; Binghong Zhao; Yueming Qu; Yurou Chen; Jie Xiong; Yong Feng; Dong Men; Qianchuan Huang; Ying Liu; Bo Yang; Jinya Ding; Feng Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029520&#39;},
 {&#39;cord_uid&#39;: &#39;3kp3ptbn&#39;,
  &#39;sha&#39;: &#39;0bd443591b5e10934beef050b516afeca6b668fe&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A simple model to assess Wuhan lock-down effect and region efforts during COVID-19 epidemic in China Mainland&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029561&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since COVID-19 emerged in early December, 2019 in Wuhan and swept across China Mainland, a series of large-scale public health interventions, especially Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement, have been taken by the government to control the epidemic. Based on Baidu Migration data and the confirmed cases data, we identified two key factors affecting the later (e.g February 27, 2020) cumulative confirmed cases in non-Wuhan region (y). One is the sum travelers from Wuhan during January 20 to January 26 (x1), which had higher infected probability but lower transmission ability because the human-to-human transmission risk of COVID-19 was confirmed and announced on January 20. The other is the seed cases from Wuhan before January 19, which had higher transmission ability and could be represented with the confirmed cases before January 29 (x2) due to a mean 10-day delay between infection and detection. A simple yet effective regression model then was established as follow: y= 70.0916+0.0054*x1+2.3455*x2 (n = 44, R2 = 0.9330, P&lt;10-7). Even the lock-down date only delay or in advance 3 days, the estimated confirmed cases by February 27 in non-Wuhan region will increase 35.21% or reduce 30.74% - 48.59%. Although the above interventions greatly reduced the human mobility, Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement do have a determining effect on the ongoing spread of COVID-19 across China Mainland. The strategy adopted by China has changed the fast-rising curve of newly diagnosed cases, the international community should learn from lessons of Wuhan and experience from China. Efforts of 29 Provinces and 44 prefecture-level cities against COVID-19 were also assessed preliminarily according to the interpretive model. Big data has played and will continue playing an important role in public health.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Yuan zheming; Chen Yuan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029561&#39;},
 {&#39;cord_uid&#39;: &#39;2s4ifz7i&#39;,
  &#39;sha&#39;: &#39;b8bb4db131a25b1bbb30d4205b16dc9ef988d22f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Machine learning-based CT radiomics model for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection: A multicenter study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.20029603&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Objectives To develop and test machine learning-based CT radiomics models for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection. Design Cross-sectional Setting Multicenter Participants A total of 52 patients with laboratory-confirmed SARS-CoV-2 infection and their initial CT images were enrolled from 5 designated hospitals in Ankang, Lishui, Zhenjiang, Lanzhou, and Linxia between January 23, 2020 and February 8, 2020. As of February 20, patients remained in hospital or with non-findings in CT were excluded. Therefore, 31 patients with 72 lesion segments were included in the final analysis. Intervention CT radiomics models based on logistic regression (LR) and random forest (RF) were developed on features extracted from pneumonia lesions in training and inter-validation datasets. The predictive performance was further evaluated in test dataset on lung lobe- and patients-level. Main outcomes Short-term hospital stay (≤10 days) and long-term hospital stay (&gt;10 days). Results The CT radiomics models based on 6 second-order features were effective in discriminating short- and long-term hospital stay in patients with pneumonia associated with SARS-CoV-2 infection, with areas under the curves of 0.97 (95%CI 0.83-1.0) and 0.92 (95%CI 0.67-1.0) by LR and RF, respectively, in the test dataset. The LR model showed a sensitivity and specificity of 1.0 and 0.89, and the RF model showed similar performance with sensitivity and specificity of 0.75 and 1.0 in test dataset. Conclusions The machine learning-based CT radiomics models showed feasibility and accuracy for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Xiaolong Qi; Zicheng Jiang; QIAN YU; Chuxiao Shao; Hongguang Zhang; Hongmei Yue; Baoyi Ma; Yuancheng Wang; Chuan Liu; Xiangpan Meng; Shan Huang; Jitao Wang; Dan Xu; Junqiang Lei; Guanghang Xie; Huihong Huang; Jie Yang; Jiansong Ji; Hongqiu Pan; Shengqiang Zou; Shenghong Ju&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.20029603&#39;},
 {&#39;cord_uid&#39;: &#39;1yka7ej9&#39;,
  &#39;sha&#39;: &#39;38e796bbec2f90b1b802c14dc922102e96f6361e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The within-host viral kinetics of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.965418&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Chentong Li; Jinhu Xu; Jiawei Liu; Yicang Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.965418&#39;},
 {&#39;cord_uid&#39;: &#39;0d77ojnb&#39;,
  &#39;sha&#39;: &#39;07e833d0917cace550853f72923856d0fe1a7120&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The 2019 coronavirus (SARS-CoV-2) surface protein (Spike) S1 Receptor Binding Domain undergoes conformational change upon heparin binding&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.971093&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMany pathogens take advantage of the dependence of the host on the interaction of hundreds of extracellular proteins with the glycosaminoglycans heparan sulfate to regulate homeostasis and use heparan sulfate as a means to adhere and gain access to cells. Moreover, mucosal epithelia such as that of the respiratory tract are protected by a layer of mucin polysaccharides, which are usually sulfated. Consequently, the polydisperse, natural products of heparan sulfate and the allied polysaccharide, heparin have been found to be involved and prevent infection by a range of viruses including S-associated coronavirus strain HSR1. Here we use surface plasmon resonance and circular dichroism to measure the interaction between the SARS-CoV-2 Spike S1 protein receptor binding domain (SARS-CoV-2 S1 RBD) and heparin. The data demonstrate an interaction between the recombinant surface receptor binding domain and the polysaccharide. This has implications for the rapid development of a first-line therapeutic by repurposing heparin and for next-generation, tailor-made, GAG-based antivirals.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Courtney Mycroft-West; Dunhao Su; Stefano Elli; Scott Guimond; Gavin Miller; Jeremy Turnbull; Edwin Yates; Marco Guerrini; David Fernig; Marcelo Lima; Mark Skidmore&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.971093&#39;},
 {&#39;cord_uid&#39;: &#39;jmrg4oeb&#39;,
  &#39;sha&#39;: &#39;2ea102f58147dab02e4dea90eb90dbc67149f678&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Mutations, Recombination and Insertion in the Evolution of 2019-nCoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.971101&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.ConclusionsOur studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.One Sentence SummaryNovel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Aiping Wu; Peihua Niu; Lulan Wang; Hangyu Zhou; Xiang Zhao; Wenling Wang; Jingfeng Wang; Chengyang Ji; Xiao Ding; Xianyue Wang; Roujian Lu; Sarah Gold; Saba Aliyari; Shilei Zhang; Ellee Vikram; Angela Zou; Emily Lenh; Janet Chen; Fei Ye; Na Han; Yousong Peng; Haitao Guo; Guizhen Wu; Taijiao Jiang; Wenjie Tan; Genhong Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.971101&#39;},
 {&#39;cord_uid&#39;: &#39;gzubfnqg&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.02.29.971127&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Curti Lucia; Pereyra-Bonnet Federico; Gimenez Carla Alejandra&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.02.29.971127&#39;},
 {&#39;cord_uid&#39;: &#39;tne83uu0&#39;,
  &#39;sha&#39;: &#39;7e9cd4bbf0fba1cc0bcded40041fd01b9dfb683f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiologic Characteristics of COVID-19 in Guizhou, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20028944&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;At the end of 2019, a coronavirus disease 2019 (COVID-19) outbroke in Wuhan, China, and spread to Guizhou province on January of 2020. To acquire the epidemiologic characteristics of COVID-19 in Guizhou, China, we collected data on 162 laboratory-confirmed cases related to COVID-19. We described the demographic characteristics of the cases and estimated the incubation period, serial interval and basic reproduction number. We also presented two representative case studies in Guizhou province -- Case Study 1 was an example of asymptomatic carrier; and Case Study 2 was an example of a large and complex infection chain that involved four different districts spanning three provinces and eight families. With an estimation of 8 days incubation period and 6 days serial interval, our results indicate that there may exist infectiousness during the incubation period for 2019-nCoV. This increases the difficulty of screening or identifying cases related to COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Kaike Ping&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20028944&#39;},
 {&#39;cord_uid&#39;: &#39;hhsfq8bz&#39;,
  &#39;sha&#39;: &#39;f19e2df932f90f387fe71533f3802875d9042160&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029074&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Coronavirus disease-2019 (COVID-19) is a rapidly escalating epidemic caused by SARS-CoV-2. Identification of a simple and effective indicator to assess disease severity and outcome is urgently needed. Methods: This study retrospectively analyzed dynamic changes of lymphocyte percentage (LYM%) in 15 death cases, 15 severe cases as well as 40 moderate cases of COVID-19 patients. Next, prognostic role of lymphopenia in COVID-19 were verified in 92 hospitalized cases. Results: Our results from death and severe cases showed that LYM% in blood test were inversely associated with the progression and severity of COVID-19. LYM% in patients with moderate COVID-19 remained higher than 20% 10-12 days after symptom onset. In contrast, LYM% was lower than 20% in severe cases. However, LYM% in severe cases was higher than 5% 17-19 days after the onset of the disease, while it fell below 5% in death cases. Therefore, we established a reliable Time from symptom onset-LYM% model (TLM), which could be used to evaluate disease severity and predict the outcomes of hospitalized patients with COVID-19. Conclusion: Lymphopenia can be used to indicate clinical course, treatment effect and outcomes of COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Li Tan; Qi Wang; Duanyang Zhang; Jinya Ding; Qianchuan Huang; Yi-Quan Tang; Qiongshu Wang; Hongming Miao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029074&#39;},
 {&#39;cord_uid&#39;: &#39;nsrm0axa&#39;,
  &#39;sha&#39;: &#39;510ecab01c66bc2a911abcf67451ec482ed18a05&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and clinical features of 2019-nCoV acute respiratory disease cases in Chongqing municipality, China: a retrospective, descriptive, multiple-center study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029397&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background：In January 19, 2020, first case of 2019 novel coronavirus (2019-nCoV) pneumonia (COVID-19) was confirmed in Chongqing municipality, China. Methods：In this retrospective, descriptive, multiple-center study, total of 267 patients with COVID-19 confirmed by real-time RT-PCR in Chongqing from Jan 19 to Feb 16, 2020 were recruited. Epidemiological, demographic, clinical, radiological characteristics, laboratory examinations, and treatment regimens were collected on admission. Clinical outcomes were followed up until Feb 16, 2020. Results：267 laboratory-confirmed COVID-19 patients admitted to 3 designated-hospitals in Chongqing provincial municipality from January 19 to February 16, 2020 were enrolled and categorized on admission. 217 (81.27%) and 50 (18.73%) patients were categorized into non-severe and severe subgroups, respectively. The median age of patients was 48.0 years (IQR, 35.0-65.0), with 129 (48.3%) of the patients were more than 50 years of age. 149 (55.8%) patients were men. Severe patients were significantly older (median age, 71.5 years [IQR, 65.8-77.0] vs 43.0 years [IQR, 32.5-57.0]) and more likely to be male (110 [50.7%] vs 39 [78.0%]) and have coexisting disorders (15 [30.0%] vs 26 [12.0%]). 41 (15.4%) patients had a recent travel to Hubei province, and 139 (52.1%) patients had a history of contact with patients from Hubei. On admission, the most common symptoms of COVID-19 were fever 225(84.3%), fatigue (208 [77.9%]), dry cough (189 [70.8%]), myalgia or arthralgia (136 [50.9%]). Severe patients were more likely to present dyspnea (17 [34.0%] vs 26 [12.0%]) and confusion (10 [20.0%] vs 15 [6.9%]). Rales (32 [12.0%]) and wheezes (20 [7.5%]) are not common noted for COVID-19 patients, especially for the non-severe (11 [5.1%], 10 [4.6%]). 118 (44.2%). Most severe patients demonstrated more laboratory abnormalities. 231 (86.5%), 61 (22.8%) patients had lymphopenia, leukopenia and thrombocytopenia, respectively. CD4+T cell counts decrease was observed in 77.1 % of cases, especially in the severe patients (45, 100%). 53.1% patients had decreased CD+3 T cell counts, count of CD8+T cells was lower than the normal range in part of patients (34.4%). More severe patients had lower level of CD4+ T cells and CD+3 T cells (45 [100.0%] vs 29[56.9%], 31 [68.9%] vs 20 [39.2%]). Most patients had normal level of IL-2, IL-4, TNF-α and INF-γ, while high level of IL-6 and IL-17A was common in COVID-19 patients (47 [70.1%], 35 [52.2%]). Level of IL-6, IL-17A and TNF-α was remarkably elevated in severe patients (32 [84.2%] vs 15 [51.7%], 25 [65.8%] vs 10 [34.5%], 17 [44.7%] vs 5 [17.2%]). All patients received antiviral therapy (267, 100%). A portion of severe patients (38, 76.0%) received systemic corticosteroid therapy. Invasive mechanical ventilation in prone position, non-invasive mechanical ventilation, high-flow nasal cannula oxygen therapy was adopted only in severe patients with respiratory failure (5[10.0%], 35[70.0%], 12[24.0%]). Traditional Chinese medicine was adopted to most of severe patients (43,86.0%). Conclusion:Our study firstly demonstrated the regional disparity of COVID-19 in Chongqing municipality and further thoroughly compared the differences between severe and non-severe patients. The 28-day mortality of COVID-19 patients from 3 designed hospitals of Chongqing is 1.5%, lower than that of Hubei province and mainland China including Hubei province. However, the 28-mortality of severe patients was relatively high, with much higher when complications occurred. Notably, the 28-mortality of critically severe patients complicated with severe ARDS is considerably as high as 44.4%. Therefore, early diagnosis and intensive care of critically severe COVID-19 cases, especially those combined with ARDS, will be considerably essential to reduce mortality.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Di Qi; Xiaofeng Yan; Xumao Tang; Junnan Peng; Qian Yu; Longhua Feng; Guodan Yuan; An Zhang; Yaokai Chen; Jing Yuan; Xia Huang; Xianxiang Zhang; Peng Hu; Yuyan Song; Chunfang Qian; Qiangzhong Sun; Daoxin Wang; Jin Tong; Jianglin Xiang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029397&#39;},
 {&#39;cord_uid&#39;: &#39;iwsa760n&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Systematic Review of the Registered Clinical Trials of Coronavirus Disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029611&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since the outbreak of coronavirus disease 2019 (COVID-19), many researchers in China have immediately carried out clinical research scheme of the COVID-19. But, there is still a lack of systematic review of registered clinical trials. Therefore, we conducted a systematic review of the clinical trials of COVID-19 to summarize the characteristics of the COVID-19 registered clinical trials. Methods: This study is based on the recommendations of the PRISMA in the Cochrane handbook. The databases from the Chinese Clinical Registration Center and the ClinicalTrials.gov were searched to collect the registered clinical trials of COVID-19. The retrieval inception date is February 9, 2020. Two researchers independently selected the literature based on inclusion and exclusion criteria, extracted data and evaluated the risk of bias. Results: A total of 75 registered clinical trials (63 interventional studies and 12 observational studies) of COVID-19 were obtained. A majority of clinical trials were sponsored by Chinese hospitals. Only 11 trials have begun to recruit patients, and none of the registered clinical trials had been completed; 34 trials were early clinical exploratory trials or in a pre-experiment stage, 15 trials belonged to phrase Ⅲ and 4 trials were phrase Ⅳ. The methods of intervention included traditional Chinese medicine involving 26 trials, Western medicine involving 30 trials, and integrated traditional Chinese medicine and Western medicine involving 19 trials. The subjects were mainly non-critical adult patients (≥ 18 years old). The median sample size of the trials was 100 (IQR: 60 - 200), and the median execute time of the trials was 179 d (IQR: 94 - 366 d). The main outcomes were clinical observation and examinations. Overall, both the methodology quality of interventional trials and observational studies were low. Conclusions: Disorderly and intensive clinical trials of COVID-19 using traditional Chinese medicine and western medicine are ongoing or will being carried out in China. However, based on the low methodology quality and small sample size and long studies execute time, we will not be able to obtain reliable, high-quality clinical evidence about COVID-19 treatment in the near future. Improving the quality of study design, prioritizing promising drugs, and using different designs and statistical methods are worth advocating and recommending for the clinical trials of COVID-19 in China.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Rui-fang Zhu; Ru-lu Gao; Sue-Ho Robert; Jin-ping Gao; Shi-gui Yang; Changtai Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029611&#39;},
 {&#39;cord_uid&#39;: &#39;3e1ji2mw&#39;,
  &#39;sha&#39;: &#39;4bc43084ccdb8d3bb704ca577473169cd1a64f9c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Risk estimation and prediction by modeling the transmission of the novel coronavirus (COVID-19) in mainland China excluding Hubei province&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029629&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: In December 2019, an outbreak of novel coronavirus disease (COVID-19) emerged in Wuhan, China and has swiftly spread to other parts of China and a number of oversea countries. Our aim is to evaluate the effectiveness of the evolution of interventions and self-protection measures, estimate the risk of partial lifting control measures and predict the epidemic trend of the virus in mainland China excluding Hubei province based on the published data and a novel mathematical model. Methods: A novel COVID-19 transmission dynamic model incorporating the intervention measures implemented in China is proposed. We parameterize the model by using the Markov Chain Monte Carlo (MCMC) method and estimate the control reproduction number Rc, as well as the effective daily reproduction ratio Re(t), of the disease transmission in mainland China excluding Hubei province. Results: The estimation outcomes indicate that the control reproduction number is 3.36 (95% CI 3.20-3.64) and Re(t) has dropped below 1 since January 31st, 2020, which implies that the containment strategies implemented by the Chinese government in mainland China excluding Hubei province are indeed effective and magnificently suppressed COVID-19 transmission. Moreover, our results show that relieving personal protection too early may lead to the spread of disease for a longer time and more people would be infected, and may even cause epidemic or outbreak again. By calculating the effective reproduction ratio, we proved that the contact rate should be kept at least less than 30% of the normal level by April, 2020. Conclusions: To ensure the epidemic ending rapidly, it is necessary to maintain the current integrated restrict interventions and self-protection measures, including travel restriction, quarantine of entry, contact tracing followed by quarantine and isolation and reduction of contact, like wearing masks, etc. People should be fully aware of the real-time epidemic situation and keep sufficient personal protection until April. If all the above conditions are met, the outbreak is expected to be ended by April in mainland China apart from Hubei province.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Hui Wan; Jing-an Cui; Guo-Jing Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029629&#39;},
 {&#39;cord_uid&#39;: &#39;4k1i6y98&#39;,
  &#39;sha&#39;: &#39;5f72fd41d7ba8d4d7de7aa1f83fc5c90ffa50b03&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;How does the outbreak of 2019-nCoV spread in mainland China? A retrospective analysis of the dynamic transmission routes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029645&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The fourth outbreak of the Coronaviruses, known as the 2019-nCoV, has occurred in Wuhan city of Hubei province in China in December 2019. We propose a time-varying sparse vector autoregressive (VAR) model to retrospectively analyze and visualize the dyamic transmission routes of this outbreak in mainland China over January 31 - February 19, 2020. Our results demonstrate that the influential inter-province routes from Hubei have become unidentifiable since February 4, whereas the self-transmission in each province was accelerating over February 4-15. From February 16, all routes became less detectable, and no influential transmissions could be identified on February 18 and 19. Such evidence supports the effectiveness of government interventions, including the travel restrictions in Hubei. Implications of our results suggest that in addition to the origin of the outbreak, virus preventions are of crucial importance in provinces with the largest migrant workers percentages (e.g., Jiangxi, Henan and Anhui) to controlling the spread of 2019-nCoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Xiandeng Jiang; Le Chang; Yanlin Shi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029645&#39;},
 {&#39;cord_uid&#39;: &#39;kbd2h4l0&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The potential role of IL-6 in monitoring severe case of coronavirus disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029769&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract. Background: The outbreak of coronavirus disease 2019 (COVID-19) in Wuhan City, China spreads rapidly since December, 2019. Most patients show mild symptoms, but some of them develop into severe disease. There is currently no specific medication. The purpose of this study is to to explore changes of markers in peripheral blood of severe COVID-19 patients, which may be of value in disease monitoring. Methods Clinical data of patients with nonsevere and severe type COVID-19 diagnosed by laboratory test in our institution were collected. The relationship between peripheral blood cells and cytokines, clinical manifestation and outcome was analyzed. Results A total of 69 severe type COVID-19 patients were included. On admission, the median age of severe cases was 56-year old, with 52.17% female patient. The most common symptoms were fever (79.72%), cough (63.77%), shortness of breath (57.97%) and fatigue (50.72%). Diarrhea is less common. The most common comorbidity is hypertension. Upon admission, the proportion of bilateral pulmonary involvement and interstitial abnormalities evidenced by chest computed tomography (CT) imaging in severe cases was 60.87% and 27.54%, respectively. Compared with patients with nonsevere disease, those with severe disease showed obvious lymphocytopenia. Elevated level of lactate dehydrogenase (LDH), C-reactive protein (CRP), ferritin and D-dimer was found in most cases. Two patients (2.9%) needed transfer to the intensive care unit. Baseline immunological parameters and most of the inflammatory parameters were basically within the normal range. However, baseline interleukin-6 (IL-6) was significantly increased in severe type, which was closely related to the maximal body temperature during hospitalization and to CT findings. Baseline IL-6 was also significantly related to the increase of baseline level of CRP, LDH, ferritin and D-dimer. The increase of baseline IL-6 level suggests that it may positively correlate with the severity of COVID-19. Among the 30 severe type patients whose IL-6 was assessed before and after treatment, significant decrease in IL-6 and improved CT assessment was found in 25 patients after treatment. Whereas the IL-6 level was further increased in 3 cases, which was closely related to disease progression. It is suggested that IL-6 may be used as a marker for disease monitoring in severe COVID-19 patients. Conclusions On admission, the baseline level of IL-6, CRP, LDH and ferritin was closely related to the severity of COVID-19, and the elevated IL-6 was significantly related to the clinical manifestation of severe type patients. The decrease of IL-6 was closely related to treatment effectiveness, while the increase of IL-6 indicated disease exacerbation. Collectively, the dynamic change of IL-6 level can be used as a marker for disease monitoring in patients with severe COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Tao Liu; Jieying Zhang; Yuhui Yang; Hong Ma; Zhengyu Li; Jiaoyue Zhang; Ji Cheng; Xiaoyun Zhang; Yanxia Zhao; Zihan Xia; Liling Zhang; Gang Wu; Jianhua Yi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029769&#39;},
 {&#39;cord_uid&#39;: &#39;a5x5ga60&#39;,
  &#39;sha&#39;: &#39;c8437a45bfb84fb206fe03fd18d28858bae32651&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029785&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of the novel coronavirus disease 2019 (COVID-19) infection began in December 2019 in Wuhan, and rapidly spread to many provinces in China. The number of cases has increased markedly in Anhui, but information on the clinical characteristics of patients is limited. We reported 75 patients with COVID-19 in the First Affiliated Hospital of USTC from Jan 21 to Feb 16, 2020, Hefei, Anhui Province, China. COVID-19 infection was confirmed by real-time RT-PCR of respiratory nasopharyngeal swab samples. Epidemiological, clinical and laboratory data were collected and analyzed. Of the 75 patients with COVID-19, 61 (81.33%) had a direct or indirect exposure history to Wuhan. Common symptoms at onset included fever (66 [88.0%] of 75 patients) and dry cough (62 [82.67%]). Of the patients without fever, cough could be the only or primary symptom. The most prominent laboratory abnormalities were lymphopenia, decreased percentage of lymphocytes (LYM%), decreased CD4+ and CD8+ T cell counts, elevated C-reactive protein (CRP) and lactate dehydrogenase (LDH). Patients with elevated interleukin 6 (IL-6) showed significant decreases in the LYM%, CD4+ and CD8+ T cell counts. Besides, the percentage of neutrophils, CRP, LDH and Procalcitonin levels increased significantly. We concluded that COVID-19 could cause different degrees of hematological abnormalities and damage of internal organs. Hematological profiles including LYM, LDH, CRP and IL-6 could be indicators of diseases severity and evaluation of treatment effectiveness. Antiviral treatment requires a comprehensive and supportive approach. Further targeted therapy should be determined based on individual clinical manifestations and laboratory indicators.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Zonghao Zhao; Jiajia Xie; Ming Yin; Yun Yang; Hongliang He; Tengchuan Jin; Wenting Li; Xiaowu Zhu; Jing Xu; Changcheng Zhao; Lei Li; Yi Li; Hylemariam Mihiretie Mengist; Ayesha Zahid; Ziqin Yao; Chengchao Ding; Yingjie Qi; Yong Gao; Xiaoling Ma&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029785&#39;},
 {&#39;cord_uid&#39;: &#39;38f8ftmh&#39;,
  &#39;sha&#39;: &#39;2fe6b550f737baa47a5f2c8ab64cc3d9271c308a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluation of the potential incidence of COVID-19 and effectiveness of contention measures in Spain: a data-driven approach&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029801&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Our society is currently experiencing an unprecedented challenge, managing and containing an outbreak of a new coronavirus disease known as COVID-19. While China - were the outbreak started - seems to have been able to contain the growth of the epidemic, different outbreaks are nowadays being detected in multiple countries. Much is currently unknown about the natural history of the disease, such as a possible asymptomatic spreading or the role of age in both the susceptibility and mortality of the disease. Nonetheless, authorities have to take action and implement contention measures, even if not everything is known. To facilitate this task, we have studied the effect of different containment strategies that can be put into effect. Our work specifically refers to the situation in Spain as of February 28th, 2020, where a few dozens of cases have been detected. We implemented an SEIR-metapopulation model that allows tracing explicitly the spatial spread of the disease through data-driven stochastic simulations. Our results are in line with the most recent recommendations from the World Health Organization, namely, that the best strategy is the early detection and isolation of individuals with symptoms, followed by interventions and public recommendations aimed at reducing the transmissibility of the disease, which although not efficacious for disease eradication, would produce as a second-order effect a delay of several days in the raise of the number of infected cases&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Alberto Aleta; Yamir Moreno&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029801&#39;},
 {&#39;cord_uid&#39;: &#39;7reqkx3h&#39;,
  &#39;sha&#39;: &#39;d7111b44f2d13b24a11d55c8be2d2bbe6b245eac&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 Epidemic Outside China: 34 Founders and Exponential Growth&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029819&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: In December 2019, pneumonia infected with a novel coronavirus burst in Wuhan, China. Now the situation is almost controlled in China but is worse outside China. We aimed to build a mathematical model to capture the global trend of epidemics outside China. Methods: In this retrospective, outside-China diagnosis number reported from Jan 21 to Feb 28, 2020 was downloaded from WHO website. We develop a simple regression model on these numbers: log10 (Nt+34)=0.0515*t+2.075 where Nt is the total diagnosed patient till the ith day, t=1 at Feb 1. Findings: Based on this model, we estimate that there have been about 34 unobserved founder patients at the beginning of spread outside China. The global trend is approximately exponential, with the rate of 10 folds every 19 days.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Yi Li; Meng Liang; Xianhong Yin; Xiaoyu Liu; Meng Hao; Zixin Hu; Yi Wang; Li Jin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029819&#39;},
 {&#39;cord_uid&#39;: &#39;j58f1lwa&#39;,
  &#39;sha&#39;: &#39;f8360b0e64d444215d21a61e056b070f12417e53&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preliminary epidemiological analysis on children and adolescents with novel coronavirus disease 2019 outside Hubei Province, China: an observational study utilizing crowdsourced data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.20029884&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The outbreak of coronavirus disease 2019 (COVID-19) continues to expand across the world. Though both the number of cases and mortality rate in children and adolescents is reported to be low in comparison to adults, limited data has been reported on the outbreak with respect to pediatric patients. To elucidate information, we utilized crowdsourced data to perform a preliminary epidemiologic analysis of pediatric patients with COVID-19 Methods: In this observational study, data was collected from two open-access, line list crowdsourced online databases. Pediatric cases of COVID-19 were defined as patients ≤19 years of age with a laboratory confirmed diagnosis. The primary outcomes were case counts and cumulative case counts. Secondary outcomes included days between symptoms onset and first medical care and days between first medical care and reporting. Tertiary outcomes were rate of travel to Wuhan, rate of infected family members and rates of symptoms. Results: A total of 82 patients were included. The median age was 10 [IQR: 5-15] years. Patients from mainland China (outside Hubei) accounted for 46.3% of cases, while the remaining 53.7% of cases were international. Males and females accounted for 52.4% and 32.9% of cases, respectively, with the remaining 14.6% being designated as unknown. A male skew persisted across subgroup analyses by age group (p=1.0) and location (inside/outside China) (p=0.22). While the number of reported international cases has been steadily increasing over the study period, the number of reported cases in China rapidly decreased from the start point. The median reporting delay was 3 [IQR: 2-4.8] days. The median delay between symptom onset and first seeking medical care was 1 [IQR: 0-3.25] day. In international cases, time to first seeking medical care was a median of 2.5 days longer than in China (p=0.04). When clinical features were reported, fever was the most common presentation (68.0%), followed by cough (36.0%). Conclusions: The number of reported international pediatric COVID-19 cases is rapidly increasing. COVID-19 infections are, to-date, more common in males than females in both the children and adolescent age groups. Additionally, this male predominance remains the case both inside and outside of China. Crowdsourced data enabled early analysis of epidemiologic variables in pediatric patients with COVID-19. Further data sharing is required to enable analyses that are required to understand the course of this infection in children.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Brandon Michael Henry; Maria Helena S Oliveira&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.20029884&#39;},
 {&#39;cord_uid&#39;: &#39;6om1y33o&#39;,
  &#39;sha&#39;: &#39;a891d059692c951d1fb0fb36914fe451f973684a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Kallikrein 13: a new player in coronaviral infections&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.01.971499&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTHuman coronavirus HKU1 (HCoV-HKU1) is associated with respiratory disease and is prevalent worldwide, but in vitro model for virus replication is lacking. Interaction between the coronaviral spike (S) protein and its receptor is the major determinant of virus tissue and host specificity, but virus entry is a complex process requiring a concerted action of multiple cellular elements. Here, we show that KLK13 is required for the infection of the human respiratory epithelium and is sufficient to mediate the entry of HCoV-HKU1 to non-permissive RD cells. We also demonstrated HCoV-HKU1 S protein cleavage by KLK13 in the S1/S2 region, proving that KLK13 is the priming enzyme for this virus. Summarizing, we show for the first time that protease distribution and specificity predetermines the tissue and cell specificity of the virus and may also regulate interspecies transmission. It is also of importance that presented data may be relevant for the emerging coronaviruses, including SARS-CoV-2 and may help to understand the differences in their zoonotic potential.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Aleksandra Milewska; Katherine Falkowski; Magdalena Kalinska; Ewa Bielecka; Antonina Naskalska; Pawel Mak; Adam Lesner; Marek Ochman; Maciej Urlik; Jan Potempa; Tomasz Kantyka; Krzysztof Pyrc&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.01.971499&#39;},
 {&#39;cord_uid&#39;: &#39;yu89n6rq&#39;,
  &#39;sha&#39;: &#39;d810f78f8850d9386abc6fdd8d652f2f85191753&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The effect of human mobility and control measures on the COVID-19 epidemic in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20026708&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The ongoing COVID-19 outbreak has expanded rapidly throughout China. Major behavioral, clinical, and state interventions are underway currently to mitigate the epidemic and prevent the persistence of the virus in human populations in China and worldwide. It remains unclear how these unprecedented interventions, including travel restrictions, have affected COVID-19 spread in China. We use real-time mobility data from Wuhan and detailed case data including travel history to elucidate the role of case importation on transmission in cities across China and ascertain the impact of control measures. Early on, the spatial distribution of COVID-19 cases in China was well explained by human mobility data. Following the implementation of control measures, this correlation dropped and growth rates became negative in most locations, although shifts in the demographics of reported cases are still indicative of local chains of transmission outside Wuhan. This study shows that the drastic control measures implemented in China have substantially mitigated the spread of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Moritz U.G. Kraemer; Chia-Hung Yang; Bernardo Gutierrez; Chieh-Hsi Wu; Brennan Klein; David M. Pigott; Louis du Plessis; Nuno R Faria; Ruoran Li; William P. Hanage; John S Brownstein; Maylis Layan; Alessandro Vespignani; Huaiyu Tian; Christopher Dye; Simon Cauchemez; Oliver Pybus; Samuel V Scarpino&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20026708&#39;},
 {&#39;cord_uid&#39;: &#39;ngbfiws5&#39;,
  &#39;sha&#39;: &#39;d85d142716f7f85cd27b54fc38a9e2968cf14346&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effects of weather-related social distancing on city-scale transmission of respiratory viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20027599&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND: Unusually high snowfall in western Washington State in February 2019 led to widespread school and workplace closures. We assessed the impact of social distancing caused by this extreme weather event on the transmission of respiratory viruses. METHODS: Residual specimens from patients evaluated for acute respiratory illness at hospitals in the Seattle metropolitan area were screened for a panel of respiratory viruses. Transmission models were fit to each virus, with disruption of contact rates and care-seeking informed by data on local traffic volumes and hospital admissions. RESULTS: Disruption in contact patterns reduced effective contact rates during the intervention period by 16% to 95%, and cumulative disease incidence through the remainder of the season by 3% to 9%. Incidence reductions were greatest for viruses that were peaking when the disruption occurred and least for viruses in early epidemic phase. CONCLUSION: High-intensity, short-duration social distancing measures may substantially reduce total incidence in a respiratory virus epidemic if implemented near the epidemic peak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Michael L Jackson; Gregory R Hart; Denise J McCulloch; Amanda Adler; Elisabeth Brandstetter; Kairsten Fay; Peter Han; Kirsten Lacombe; Jover Lee; Thomas Sibley; Deborah A Nickerson; Mark Rieder; Lea Starita; Janet A Englund; Trevor Bedford; Helen Chu; Michael Famulare&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20027599&#39;},
 {&#39;cord_uid&#39;: &#39;rlsdznmv&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Monitoring Disease Transmissibility of 2019 Novel Coronavirus Disease in Zhejiang, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20028704&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We monitored the transmissibility of 2019 novel coronavirus disease in Zhejiang accounting the transmissions from imported cases. Even though Zhejiang is one of the worst-affected provinces, an interruption of disease transmission (i.e. instantaneous reproduction numbers &lt;1) was observed in early/mid-February after an early social-distancing response to the outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-05&#39;,
  &#39;authors&#39;: &#39;Ka Chun Chong; Wei Cheng; Shi Zhao; Feng Ling; Kirran N Mohammad; Maggie Haitian Wang; Benny Chung-ying Zee; Lesley Wei; Xi Xiong; Hengyan Liu; Jingxuan Wang; Enfu Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20028704&#39;},
 {&#39;cord_uid&#39;: &#39;qlkt5fzp&#39;,
  &#39;sha&#39;: &#39;70cc2e5152d3dc4d44494124ff556c9bbe9e6f41&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of Patients with Severe Pneumonia Caused by the 2019 Novel Coronavirus in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20029306&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Background: A new virus broke out in Wuhan, Hubei, China, and was later named 2019 novel coronavirus (2019-nCoV). The clinical characteristics of severe pneumonia caused by 2019-nCoV are still not clear. Objectives: The aim of this study was to explore the clinical characteristics and risk factors of the severe pneumonia caused by the 2019-nCoV in Wuhan, China. Method: The study included patients hospitalized at the central hospital of Wuhan who had been diagnosed with a pneumonia caused by the novel coronavirus. Clinical features, chronic co-morbidities, demographic data, laboratory examinations, and chest computed tomography (CT) scans were reviewed through electronic medical records. SPSS was used for data analysis to explore the clinical characteristics and risk factors of the patients with the severe pneumonia. Results: A total of 110 patients diagnosed with 2019 novel coronavirus pneumonia were included in the study, including 38 with severe pneumonia and 72 with non-severe pneumonia. Statistical analysis showed that advanced age, an increase of D-dimer, and a decrease of lymphocytes were characteristics of the patients with severe pneumonia. Moreover, in the early stage of the disease, chest CT scans of patients with the severe pneumonia showed the illness can progress rapidly. Conclusions: Advanced age, lymphocyte decline, and D-dimer elevation are important characteristics of patients with severe pneumonia. Clinicians should focus on these characteristics to identify high-risk patients at an early stage.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Yafei Wang; Ying Zhou; Zhen Yang; Dongping Xia; Shuang Geng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20029306&#39;},
 {&#39;cord_uid&#39;: &#39;4nmc356g&#39;,
  &#39;sha&#39;: &#39;f82372b109168b411344f537210c63cc6ed323ed&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Seattle Flu Study: a multi-arm community-based prospective study protocol for assessing influenza prevalence, transmission, and genomic epidemiology&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20029595&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Introduction. Influenza epidemics and pandemics cause significant morbidity and mortality. An effective response to a potential pandemic requires the infrastructure to rapidly detect, characterize, and potentially contain new and emerging influenza strains at a population level. The objective of this study is to use data gathered simultaneously from community and hospital sites to develop a model of how influenza enters and spreads in a population. Methods and Analysis. Starting in the 2018-19 season, we have been enrolling individuals with acute respiratory illness from community sites throughout the Seattle metropolitan area, including clinics, childcare facilities, Seattle-Tacoma International Airport, workplaces, college campuses, and homeless shelters. At these sites, we collect clinical data and mid-nasal swabs from individuals with at least two acute respiratory symptoms. Additionally, we collect residual nasal swabs and data from individuals who seek care for respiratory symptoms at four regional hospitals. Samples are tested using a multiplex molecular assay, and influenza whole genome sequencing is performed for samples with influenza detected. Geospatial mapping and computational modeling platforms are in development to characterize the regional spread of influenza and other respiratory pathogens. Ethics and Dissemination. The study was approved by the University of Washington&#39;s Institutional Review Board. Results will be disseminated through talks at conferences, peer-reviewed publications, and on the study website (www.seattleflu.org).&#34;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Helen Y. Chu; Michael Boeckh; Janet A. Englund; Michael Famulare; Barry R. Lutz; Deborah A Nickerson; Mark J. Rieder; Lea M Starita; Amanda Adler; Elisabeth Brandstetter; Chris D. Frazar; Peter D. Han; Reena K. Gularti; James Hadfield; Michael L. Jackson; Anahita Kiavand; Louise E. Kimball; Kirsten Lacombe; Jennifer Logue; Victoria Lyon; Kira L. Newman; Thomas R. Sibley; Monica L. Zigman Suschsland; Caitlin Wolf; Jay Shendure; Trevor Bedford&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20029595&#39;},
 {&#39;cord_uid&#39;: &#39;b10yokyu&#39;,
  &#39;sha&#39;: &#39;5796fdc80a85b6e634a279c57711356055b65880&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#34;Close contacts and household transmission of SARS-CoV-2 in China: a content analysis based on local Heath Commissions&#39; public disclosures.&#34;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20029868&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Public disclosure from Chinese local Health Commissions show that male and mid-aged index patients are the major source of virus from outside to the households. This implies that two important actions should be taken in disease control. First considers the kindergarten and school closure time. Local kindergarten and primary schools should postpone their admission and regular operation, to at least not earlier than local secondary schools. Second, workplace hygiene is of particular important in the next phase of disease control. As travel restrictions within China are being gradually lifted, we suggest that workplace hygiene should be taken serious care in the next month or two.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;xiaoke Xu; Xiaofan Liu; Ye Wu; Sheikh Taslim ALI&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20029868&#39;},
 {&#39;cord_uid&#39;: &#39;c3dxfet9&#39;,
  &#39;sha&#39;: &#39;ebed882da10cbf669cbb86802e9fa07a4a33ef91&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20029975&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of Coronavirus Disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, December 2019, and continuously poses a serious threat to public health. Our previous study has shown that cytokine storm occurred during SARS-CoV-2 infection, while the detailed role of cytokines in the disease severity and progression remained unclear due to the limited case number. In this study, we examined 48 cytokines in the plasma samples from 53 COVID-19 cases, among whom 34 were severe cases, and the others moderate. Results showed that 14 cytokines were significantly elevated upon admission in COVID-19 cases. Moreover, IP-10, MCP-3, and IL-1ra were significantly higher in severe cases, and highly associated with the PaO2/FaO2 and Murray score. Furthermore, the three cytokines were independent predictors for the progression of COVID-19, and the combination of IP-10, MCP-3 and IL-1ra showed the biggest area under the curve (AUC) of the receiver-operating characteristics (ROC) calculations. Serial detection of IP-10, MCP-3 and IL-1ra in 14 severe cases showed that the continuous high levels of these cytokines were associated with disease deterioration and fatal outcome. In conclusion, we report three cytokines that closely associated with disease severity and outcome of COVID-19. These findings add to our understanding of the immunopathologic mechanisms of SARS-CoV-2 infection, which suggested novel therapeutic targets and strategy.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Yang Yang; Chenguang Shen; Jinxiu Li; Jing Yuan; Minghui Yang; Fuxiang Wang; Guobao Li; Yanjie Li; Li Xing; Ling Peng; Jinli Wei; Mengli Cao; Haixia Zheng; Weibo Wu; Rongrong Zou; Delin Li; Zhixiang Xu; Haiyan Wang; Mingxia Zhang; Zheng Zhang; Lei Liu; Yingxia Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20029975&#39;},
 {&#39;cord_uid&#39;: &#39;zav5gksq&#39;,
  &#39;sha&#39;: &#39;c8df44a3612e85e267351e936ddeb8fc5867afa1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The timing of one-shot interventions for epidemic control&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030007&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The apparent early success in China&#39;s large-scale intervention to control the COVID-19 epidemic has led to interest in whether other countries can replicate it as well as concerns about a resurgence of the epidemic if or when China relaxes the interventions. In this paper we look at the impact of a single short-term intervention on an epidemic. We see that if an intervention cannot be sustained long-term, it has the greatest impact if it is imposed once infection levels have become large enough that there is an appreciable number of infections present. For minimising the total number infected it should start close to the peak so that there is no rebound once the intervention is stopped, while to minimise the peak prevalence, it should start earlier, allowing two peaks of comparable size rather than one very large peak. In populations with distinct subgroups, synchronized interventions are less effective than targeting the interventions in each sub-population separately. We do not attempt to clearly determine what makes an intervention sustainable or not. We believe that is a policy question. If an intervention is sustainable, it should be kept in place. Our intent is to offer insight into how best to time an intervention whose impact on society is too great to maintain.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Francesco Di Lauro; István Z Kiss; Joel Miller&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030007&#39;},
 {&#39;cord_uid&#39;: &#39;wtdzad90&#39;,
  &#39;sha&#39;: &#39;d0b33d173f1d32662e2ad8d7e85181f9b23f010c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of COVID-19 outbreak size in Italy based on international case exportations&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030049&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Italy is currently experiencing an epidemic of COVID-19 which emerged in the Lombardy region . During the interval between February 25-29, 2020, we identified 46 cases of COVID-19 reported in 21 countries in Europe, Africa, North America, and South America which were either in individuals with recent travel from Italy, or who had presumed infection by a traveler from Italy 2. In six cases, in four of the affected countries (Switzerland, France, Austria, Croatia), land travel was a likely route of introduction, or was documented to have been the route of introduction. We used air travel volume between Italian cities and cities in other countries as an index of connectedness, using data available from the International Air Transport Association (IATA) for February 2015 (2.61 million total departing international air passengers from Italy). We used the methods of Fraser et al. to estimate the size of the underlying epidemic in Italy necessary in order for these cases to be observed with a reasonable probability. To estimate the time at risk of COVID-19 exposure for travelers departing Italy, we obtained data from the United Nations World Tourism Organization (UNWTO) for the proportion of international travelers that are non-residents of Italy (63%) and the average length of stay of tourists to Italy (3.4 days), and assumed the Italian epidemic began one month prior to February 29, 2020. We also performed sensitivity analyses in which we included outbound travel to all countries regardless of reported case importations, inflated travel volumes by 35%, to account for the relative increase in flight numbers from 2015-2019, and excluded cases in bordering countries and which were documented to have been introduced by overland travel. When all cases were considered we estimated a true outbreak size of 3971 cases (95% CI 2907-5297), as compared to a reported case count of 1128 on February 29, 2020, suggesting non-identification of 72% (61-79%) of cases. In sensitivity analyses, outbreak sizes varied from 1552 to 4533 cases (implying non-identification of 27-75% of cases). We recently used similar methods to estimate a much larger epidemic size in Iran, with a far greater degree of under-reporting, based on many fewer exported cases. The reason for this difference relates to the relatively high volume of travel from Italy, relative to Iran. In summary, we suggest that the numerous COVID-19 case exportations from Italy in recent days suggest an epidemic that is larger than official case counts suggest, and which is approximately on a par with that currently occurring in South Korea, which reports 3526 cases (and fewer deaths) as of February 29, 2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Ashleigh Tuite; Victoria Ng; Erin Rees; David Fisman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030049&#39;},
 {&#39;cord_uid&#39;: &#39;xfxf1vse&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Forecasting the Cumulative Number of COVID-19 Deaths in China: a Boltzmann Function-based Modeling Study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030064&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The COVID-19 outbreak is on-going in China. Here we estimated the potential total numbers of COVID-19 deaths in China, outside Hubei (in China), Hubei Province, Wuhan City and outside Wuhan (in Hubei) by Boltzmann function-based analyses, which are 3342 (95% CI, 3214, 3527), 111 (109, 114), 3245 (3100, 3423), 2613 (2498, 2767) and 627 (603, 654), respectively. The results may help to evaluate the severity of COVID-19 outbreaks and facilitate timely mental service for the families of passed patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Cheng Long; Qi Ying; Xinmiao Fu; Zhongyan Li; Yuanyuan Gao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030064&#39;},
 {&#39;cord_uid&#39;: &#39;4f84pc83&#39;,
  &#39;sha&#39;: &#39;3b181a741d4beafda3ba3a8fc68239493d49f6aa&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of local novel coronavirus (COVID-19) cases in Wuhan, China from off-site reported cases and population flow data from different sources&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030080&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Backgrounds: In December 2019, a novel coronavirus (COVID-19) pneumonia hit Wuhan, Hubei Province, China and spread to the rest of China and overseas. The emergence of this virus coincided with the Spring Festival Travel Rush in China. It is possible to estimate total number of cases of COVID-19 in Wuhan, by 23 January 2020, given the cases reported in other cities and population flow data between cities. Methods: We built a model to estimate the total number of cases in Wuhan by 23 January 2020, based on the number of cases detected outside Wuhan city in China, with the assumption that if the same screening effort used in other cities applied in Wuhan. We employed population flow data from different sources between Wuhan and other cities/regions by 23 January 2020. The number of total cases was determined by the maximum log likelihood estimation. Findings: From overall cities/regions data, we predicted 1326 (95% CI: 1177, 1484), 1151 (95% CI: 1018, 1292) and 5277 (95% CI: 4732, 5859) as total cases in Wuhan by 23 January 2020, based on different source of data from Changjiang Daily newspaper, Tencent, and Baidu. From separate cities/regions data, we estimated 1059 (95% CI: 918, 1209), 5214 (95% CI: 4659, 5808) as total cases in Wuhan in Wuhan by 23 January 2020, based on different sources of population flow data from Tencent and Baidu. Conclusion: Sources of population follow data and methods impact the estimates of local cases in Wuhan before city lock down. Keyword: COVID-19; mobility; pneumonia; transportation; outbreaks&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Zian Zhuang; Peihua Cao; Shi Zhao; Yijun Lou; Weiming Wang; Shu Yang; Lin Yang; Daihai He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030080&#39;},
 {&#39;cord_uid&#39;: &#39;va4t15mb&#39;,
  &#39;sha&#39;: &#39;267729947ca478946d5a4bffb8e13d50c3545120&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030130&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Corona Virus Disease 2019 (COVID-19) is a recently emerged life-threatening disease caused by SARS-CoV-2. Real- time fluorescent PCR (RT-PCR) is the clinical standard for SARS-CoV-2 nucleic acid detection. To detect SARS-CoV-2 early and control the disease spreading on time, a faster and more convenient method for SARS-CoV-2 nucleic acid detecting, RT-LAMP method (reverse transcription loop-mediated isothermal amplification) was developed. RNA reverse transcription and nucleic acid amplification were performed in one step at 63 ℃ isothermal conditions, and the results can be obtained within 30 minutes. ORF1ab gene, E gene and N gene were detected at the same time. ORF1ab gene was very specific and N gene was very sensitivity, so they can guarantee both sensitivity and specificity for SARS-CoV-2. The sensitivity of RT-LAMP assay is similar to RT-PCR, and specificity was 99% as detecting 208 clinical specimens. The RT-LAMP assay reported here has the advantages of rapid amplification, simple operation, and easy detection, which is useful for the rapid and reliable clinical diagnosis of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-02&#39;,
  &#39;authors&#39;: &#39;Weihua Yang; Xiaofei Dang; Qingxi Wang; Mingjie Xu; Qianqian Zhao; Yunying Zhou; Huailong Zhao; Li Wang; Yihui Xu; Jun Wang; Shuyi Han; Min Wang; Fengyan Pei; Yunshan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030130&#39;},
 {&#39;cord_uid&#39;: &#39;i7tjno0w&#39;,
  &#39;sha&#39;: &#39;b53dd68cad40cbcd698a848f53853e5f03d44a1f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Validity of Wrist and Forehead Temperature in Temperature Screening in the General Population During the Outbreak of 2019 Novel Coronavirus: a prospective real-world study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030148&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Aims: Temperature screening is important in the population during the outbreak of 2019 Novel Coronavirus (COVID-19). This study aimed to compare the accuracy and precision of wrist and forehead temperature with tympanic temperature under different circumstances. Methods: We performed a prospective observational study in a real-life population. We consecutively collected wrist and forehead temperatures in Celsius (C) using a non-contact infrared thermometer (NCIT). We also measured the tympanic temperature using a tympanic thermometers (IRTT) and defined fever as a tympanic temperature ≥37.3C. Results: We enrolled a total of 528 participants including 261 indoor and 267 outdoor participants. We divided outdoor participants into four types according to their means of transportation to the hospital as walk, bicycle, electric vehicle, car, and inside the car. Under different circumstance, the mean difference ranged from -1.72 to -0.56C in different groups for the forehead measurements, and -0.96 to -0.61C for the wrist measurements. Both measurements had high fever screening abilities in inpatients (wrist: AUC 0.790; 95% CI: 0.725-0.854, P &lt;0.001; forehead: AUC 0.816; 95% CI: 0.757-0.876, P &lt;0.001). The cut-off value of wrist measurement for detecting tympanic temperature ≥37.3C was 36.2C with a 86.4% sensitivity and a 67.0% specificity, and the best threshold of forehead measurement was also 36.2C with a 93.2% sensitivity and a 60.0% specificity. Conclusions: Wrist measurement is more stable than forehead measurement under different circumstance. Both measurements have great fever screening abilities for indoor patients. The cut-off value of both measurements was 36.2C.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Ge Chen; Jiarong Xie; Guangli Dai; Peijun Zheng; Xiaqing Hu; Hongpeng Lu; Lei Xu; Xueqin Chen; Xiaomin Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030148&#39;},
 {&#39;cord_uid&#39;: &#39;llqpfhwg&#39;,
  &#39;sha&#39;: &#39;ff067164497bcfbd9145be223dcd2b05f159dd63&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030189&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Summary Background The novel coronavirus SARS-CoV-2 is a newly emerging virus. The antibody response in infected patient remains largely unknown, and the clinical values of antibody testing have not been fully demonstrated. Methods A total of 173 patients with confirmed SARS-CoV-2 infection were enrolled. Their serial plasma samples (n = 535) collected during the hospitalization period were tested for total antibodies (Ab), IgM and IgG against SARS-CoV-2 using immunoassays. The dynamics of antibodies with the progress and severity of disease was analyzed. Findings Among 173 patients, the seroconversion rate for Ab, IgM and IgG was 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively. Twelve patients who had not seroconverted were those only blood samples at the early stage of illness were collected. The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. The presence of antibodies was &lt; 40% among patients in the first 7 days of illness, and then rapidly increased to 100.0%, 94.3% and 79.8% for Ab, IgM and IgG respectively since day 15 after onset. In contrast, the positive rate of RNA decreased from 66.7% (58/87) in samples collected before day 7 to 45.5% (25/55) during days 15 to 39. Combining RNA and antibody detections significantly improved the sensitivity of pathogenic diagnosis for COVID-19 patients (p &lt; 0.001), even in early phase of 1-week since onset (p = 0.007). Moreover, a higher titer of Ab was independently associated with a worse clinical classification (p = 0.006). Interpretation The antibody detection offers vital clinical information during the course of SARS-CoV-2 infection. The findings provide strong empirical support for the routine application of serological testing in the diagnosis and management of COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Juanjuan Zhao; Quan Yuan; Haiyan Wang; Wei Liu; Xuejiao Liao; Yingying Su; Xin Wang; Jing Yuan; Tingdong Li; Jinxiu Li; Shen Qian; Congming Hong; Fuxiang Wang; Yingxia Liu; Zhaoqin Wang; Qing He; Zhiyong Li; Bin He; Tianying Zhang; Shengxiang Ge; Lei Liu; Jun Zhang; Ningshao Xia; Zheng Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030189&#39;},
 {&#39;cord_uid&#39;: &#39;dvrstz6f&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the burden of United States workers exposed to infection or disease: a key factor in containing risk of COVID-19 infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030288&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: With the global spread of COVID-19, there is a compelling public health interest in quantifying who is at increased risk of disease. Occupational characteristics, such as interfacing with the public and being in close quarters with other workers, not only put workers at high risk for disease, but also make them a nexus of disease transmission to the community. This can further be exacerbated through presenteeism, the term used to describe the act of coming to work despite being symptomatic for disease. Understanding which occupational groups are exposed to infection and disease in the workplace can help to inform public health risk response and management for COVID-19, and subsequent infectious disease outbreaks. Methods: To estimate the burden of United States workers exposed to infection and disease in the workplace, national employment data (by Standard Occupational Classification) maintained by the Bureau of Labor Statistics (BLS) was merged with BLS O*NET survey data, which ranks occupations with particular physical, ergonomic, and structural exposures. For this analysis, occupations reporting exposure to infection or disease more than once a month was the focus. Results: Based on our analyses, approximately 10% (14.4 M) of United States workers are employed in occupations where exposure to disease or infection occurs at least once per week. Approximately 18.4% (26.7 M) of all United States workers are employed in occupations where exposure to disease or infection occurs at least once per month. While the majority of exposed workers are employed in healthcare sectors, other occupational sectors also have high proportions of exposed workers. These include protective service occupations (e.g. police officers, correctional officers, firefighters), office and administrative support occupations (e.g. couriers and messengers, patient service representatives), education occupations (e.g. preschool and daycare teachers), community and social services occupations (community health workers, social workers, counselors), and even construction and extraction occupations (e.g. plumbers, septic tank installers, elevator repair). Conclusions: The large number of persons employed in a wide variety of occupations with frequent exposure to infection and disease underscore the importance of all workplaces developing risk response plans for COVID-19. This work also serves as an important reminder that the workplace is a key locus for public health interventions, which could protect both workers and the communities they serve.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Marissa G Baker; Trevor K Peckham; Noah S. Seixas&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030288&#39;},
 {&#39;cord_uid&#39;: &#39;etl9uwt7&#39;,
  &#39;sha&#39;: &#39;7fb7c0c48b30e66dfaea5fdec16d999ed52d1ee0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preliminary estimating the reproduction number of the coronavirus disease (COVID-19) outbreak in Republic of Korea and Italy by 5 March 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030312&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus disease 2019 (COVID-19) outbreak and Italy has caused 6088 cases and 41 deaths in Republic of Korea and 3144 cases and 107 death in Italy by 5 March 2020. We modeled the transmission process in Republic of Korea and Italy with a stochastic model and estimated the basic reproduction number R0 as 2.6 (95% CI: 2.3-2.9) or 3.2 (95% CI: 2.9-3.5) in Republic of Korea, under the assumption that the exponential growth starting on 31 January or 5 February 2020, and 2.6 (95% CI: 2.3-2.9) or 3.3 (95% CI: 3.0-3.6) in Italy, under the assumption that the exponential growth starting on 5 February or 10 February 2020. Estimates of dispersion term (k) were 10 (95% CI: 5-56) or 22 (95% CI: 8-61) in Republic of Korea, and 13 (95% CI: 5-61) or 37 (95% CI: 13-61) in Italy, and all of which imply few super-spreading events.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Zian Zhuang; Shi Zhao; Qianying Lin; Peihua Cao; Yijun Lou; Lin Yang; Shu Yang; Daihai He; Li Xiao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030312&#39;},
 {&#39;cord_uid&#39;: &#39;zclymg50&#39;,
  &#39;sha&#39;: &#39;e5163021a1b88e2c2335cca27fbfcf883f870830&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preliminary estimation of the novel coronavirus disease (COVID-19) cases in Iran: a modelling analysis based on overseas cases and air travel data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030320&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As of 1 March 2020, Iran has reported 987 COVID-19 cases and including 54 associated deaths. At least six neighboring countries (Bahrain, Iraq, Kuwait, Oman, Afghanistan and Pakistan) have reported imported COVID-19 cases from Iran. We used air travel data and the cases from Iran to other Middle East countries and estimated 16533 (95% CI: 5925, 35538) COVID-19 cases in Iran by 25 February, before UAE and other Gulf Cooperation Council countries suspended inbound and outbound flights from Iran.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Zian Zhuang; Shi Zhao; Qianying Lin; Peihua Cao; Yijun Lou; Lin Yang; Daihai He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030320&#39;},
 {&#39;cord_uid&#39;: &#39;jir7n19b&#39;,
  &#39;sha&#39;: &#39;49ac69f362c27acbc6de0c5cbb640267e7a1e797&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical features and outcomes of 221 patients with COVID-19 in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.20030452&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Rationale: In late December 2019, an outbreak of acute respiratory illness, now officially named as COVID-19, or coronavirus disease 2019, emerged in Wuhan, China, now spreading across the whole country and world. More data were needed to understand the clinical characteristics of the disease. Objectives: To study the epidemiology, clinical features and outcomes of patients with COVID-19. Methods: we performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital. Measurements and Main Results: The median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. Conclusions: The COVID-19 epidemic spreads rapidly by human-to-human transmission. Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes. Keywords: coronavirus disease 2019; clinical features; outcomes; severe patients&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Guqin Zhang; Chang Hu; Linjie Luo; Fang Fang; Yongfeng Chen; Jianguo Li; Zhiyong Peng; Huaqin Pan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.20030452&#39;},
 {&#39;cord_uid&#39;: &#39;xq1cxrvf&#39;,
  &#39;sha&#39;: &#39;c15f8c23eedbedf8d48bbc88b8225ae9c9d234bf&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.968388&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTSevere Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 – a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Youngchang Kim; Robert Jedrzejczak; Natalia I. Maltseva; Michael Endres; Adam Godzik; Karolina Michalska; Andrzej Joachimiak&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.968388&#39;},
 {&#39;cord_uid&#39;: &#39;fu3cl7lq&#39;,
  &#39;sha&#39;: &#39;2eac7de1c407d9eab13a6d538142cd21a814ee7b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.968818&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-05&#39;,
  &#39;authors&#39;: &#39;Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.968818&#39;},
 {&#39;cord_uid&#39;: &#39;cr22vp8b&#39;,
  &#39;sha&#39;: &#39;7607a4ad84452e29998a80f44bfd6bb2f5f68a7f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.972927&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-05&#39;,
  &#39;authors&#39;: &#39;Chuang Liu; Yang Yang; Yuanzhu Gao; Chenguang Shen; Bin Ju; Congcong Liu; Xian Tang; Jinli Wei; Xiaomin Ma; Weilong Liu; Shuman Xu; Yingxia Liu; Jing Yuan; Jing Wu; Zheng Liu; Zheng Zhang; Peiyi Wang; Lei Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.972927&#39;},
 {&#39;cord_uid&#39;: &#39;xdsi7oye&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.972935&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Jennifer Harcourt; Azaibi Tamin; Xiaoyan Lu; Shifaq Kamili; Senthil Kumar. Sakthivel; Janna Murray; Krista Queen; Ying Tao; Clinton R. Paden; Jing Zhang; Yan Li; Anna Uehara; Haibin Wang; Cynthia Goldsmith; Hannah A. Bullock; Lijuan Wang; Brett Whitaker; Brian Lynch; Rashi Gautam; Craig Schindewolf; Kumari G. Lokugamage; Dionna Scharton; Jessica A. Plante; Divya Mirchandani; Steven G. Widen; Krishna Narayanan; Shinji Makino; Thomas G. Ksiazek; Kenneth S. Plante; Scott C. Weaver; Stephen Lindstrom; Suxiang Tong; Vineet D. Menachery; Natalie J. Thornburg&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.972935&#39;},
 {&#39;cord_uid&#39;: &#39;vorczspu&#39;,
  &#39;sha&#39;: &#39;bad4192bbc6d41c066bc947eaeeafc53024c98be&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evidence for RNA editing in the transcriptome of 2019 Novel Coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.973255&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe 2019-nCoV outbreak has become a global health risk. Editing by host deaminases is an innate restriction process to counter viruses, and it is not yet known whether it operates against coronaviruses. Here we analyze RNA sequences from bronchoalveolar lavage fluids derived from two Wuhan patients. We identify nucleotide changes that may be signatures of RNA editing: Adenosine-to-Inosine changes from ADAR deaminases and Cytosine-to-Uracil changes from APOBEC ones. A mutational analysis of genomes from different strains of human-hosted Coronaviridae reveals patterns similar to the RNA editing pattern observed in the 2019-nCoV transcriptomes. Our results suggest that both APOBECs and ADARs are involved in Coronavirus genome editing, a process that may shape the fate of both virus and patient.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Salvatore Di Giorgio; Filippo Martignano; Maria Gabriella Torcia; Giorgio Mattiuz; Silvestro G. Conticello&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.973255&#39;},
 {&#39;cord_uid&#39;: &#39;fhqbw32a&#39;,
  &#39;sha&#39;: &#39;1d2947d5c0addcbbcfa71d87ea074a8b6ff2a973&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Speed and strength of an epidemic intervention&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.974048&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAn epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number ℛ). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like ℛ = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of “test-and-treat” interventions against the human immunodeficiency virus (HIV) can be done more clearly on the speed than strength scale, given uncertainty in the proportion of HIV spread that happens early in the course of infection. We suggest that disease modelers should avoid over-emphasizing the reproductive number at the expense of the exponential growth rate, but instead look at these as complementary measures.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Jonathan Dushoff; Sang Woo Park&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.974048&#39;},
 {&#39;cord_uid&#39;: &#39;dbowa5bt&#39;,
  &#39;sha&#39;: &#39;7221cb14c12070a78e4ee5abe9484c40289fceea&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.974139&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryThe unprecedented epidemic of pneumonia caused by a novel coronavirus, HCoV-19, in China and beyond has caused public health concern at a global scale. Although bats are regarded as the most likely natural hosts for HCoV-191,2, the origins of the virus remain unclear. Here, we report a novel bat-derived coronavirus, denoted RmYN02, identified from a metagenomics analysis of samples from 227 bats collected from Yunnan Province in China between May and October, 2019. RmYN02 shared 93.3% nucleotide identity with HCoV-19 at the scale of the complete virus genome and 97.2% identity in the 1ab gene in which it was the closest relative of HCoV-19. In contrast, RmYN02 showed low sequence identity (61.3%) to HCoV-19 in the receptor binding domain (RBD) and might not bind to angiotensin-converting enzyme 2 (ACE2). Critically, however, and in a similar manner to HCoV-19, RmYN02 was characterized by the insertion of multiple amino acids at the junction site of the S1 and S2 subunits of the Spike (S) protein. This provides strong evidence that such insertion events can occur in nature. Together, these data suggest that HCoV-19 originated from multiple naturally occurring recombination events among those viruses present in bats and other wildlife species.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-05&#39;,
  &#39;authors&#39;: &#39;Hong Zhou; Xing Chen; Tao Hu; Juan Li; Hao Song; Yanran Liu; Peihan Wang; Di Liu; Jing Yang; Edward C. Holmes; Alice C. Hughes; Yuhai Bi; Weifeng Shi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.974139&#39;},
 {&#39;cord_uid&#39;: &#39;jjraqr85&#39;,
  &#39;sha&#39;: &#39;20f2a638e2c9a1917b7c001c371335da3d288505&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Partial RdRp sequences offer a robust method for Coronavirus subgenus classification&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.02.974311&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe recent reclassification of the Riboviria, and the introduction of multiple new taxonomic categories including both subfamilies and subgenera for coronaviruses (family Coronaviridae, subfamily Orthocoronavirinae) represents a major shift in how official classifications are used to designate specific viral lineages. While the newly defined subgenera provide much-needed standardisation for commonly cited viruses of public health importance, no method has been proposed for the assignment of subgenus based on partial sequence data, or for sequences that are divergent from the designated holotype reference genomes. Here, we describe the genetic variation of a partial region of the coronavirus RNA-dependent RNA polymerase (RdRp), which is one of the most used partial sequence loci for both detection and classification of coronaviruses in molecular epidemiology. We infer Bayesian phylogenies from more than 7000 publicly available coronavirus sequences and examine clade groupings relative to all subgenus holotype sequences. Our phylogenetic analyses are largely coherent with genome-scale analyses based on designated holotype members for each subgenus. Distance measures between sequences form discrete clusters between taxa, offering logical threshold boundaries that can attribute subgenus or indicate sequences that are likely to belong to unclassified subgenera both accurately and robustly. We thus propose that partial RdRp sequence data of coronaviruses is sufficient for the attribution of subgenus-level taxonomic classifications and we supply the R package, “MyCoV”, which provides a method for attributing subgenus and assessing the reliability of the attribution.Importance StatementThe analysis of polymerase chain reaction amplicons derived from biological samples is the most common modern method for detection and classification of infecting viral agents, such as Coronaviruses. Recent updates to the official standard for taxonomic classification of Coronaviruses, however, may leave researchers unsure as to whether the viral sequences they obtain by these methods can be classified into specific viral taxa due to variations in the sequences when compared to type strains. Here, we present a plausible method for defining genetic dissimilarity cut-offs that will allow researchers to state which taxon their virus belongs to and with what level of certainty. To assist in this, we also provide the R package ‘MyCoV’ which classifies user generated sequences.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;David A Wilkinson; Lea Joffrin; Camille Lebarbenchon; Patrick Mavingui&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.02.974311&#39;},
 {&#39;cord_uid&#39;: &#39;hnx213kp&#39;,
  &#39;sha&#39;: &#39;3a73b880acd8983f87b210ff7fee434fd6ecc8c8&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20028423&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control. Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk. Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 21 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days. Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-04&#39;,
  &#39;authors&#39;: &#39;Qifang Bi; Yongsheng Wu; Shujiang Mei; Chenfei Ye; Xuan Zou; Zhen Zhang; Xiaojian Liu; Lan Wei; Shaun A Truelove; Tong Zhang; Wei Gao; Cong Cheng; Xiujuan Tang; Xiaoliang Wu; Yu Wu; Binbin Sun; Suli Huang; Yu Sun; Juncen Zhang; Ting Ma; Justin Lessler; Teijian Feng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20028423&#39;},
 {&#39;cord_uid&#39;: &#39;jtwb17u8&#39;,
  &#39;sha&#39;: &#39;0f8261c63cdeeb5c2bf9294eba2954e1ca71b244&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20029843&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The COVID-19 outbreak containment strategies in China based on non-pharmaceutical interventions (NPIs) appear to be effective. Quantitative research is still needed however to assess the efficacy of different candidate NPIs and their timings to guide ongoing and future responses to epidemics of this emerging disease across the World. Methods: We built a travel network-based susceptible-exposed-infectious-removed (SEIR) model to simulate the outbreak across cities in mainland China. We used epidemiological parameters estimated for the early stage of outbreak in Wuhan to parameterise the transmission before NPIs were implemented. To quantify the relative effect of various NPIs, daily changes of delay from illness onset to the first reported case in each county were used as a proxy for the improvement of case identification and isolation across the outbreak. Historical and near-real time human movement data, obtained from Baidu location-based service, were used to derive the intensity of travel restrictions and contact reductions across China. The model and outputs were validated using daily reported case numbers, with a series of sensitivity analyses conducted. Results: We estimated that there were a total of 114,325 COVID-19 cases (interquartile range [IQR] 76,776 - 164,576) in mainland China as of February 29, 2020, and these were highly correlated (p&lt;0.001, R2=0.86) with reported incidence. Without NPIs, the number of COVID-19 cases would likely have shown a 67-fold increase (IQR: 44 - 94), with the effectiveness of different interventions varying. The early detection and isolation of cases was estimated to prevent more infections than travel restrictions and contact reductions, but integrated NPIs would achieve the strongest and most rapid effect. If NPIs could have been conducted one week, two weeks, or three weeks earlier in China, cases could have been reduced by 66%, 86%, and 95%, respectively, together with significantly reducing the number of affected areas. However, if NPIs were conducted one week, two weeks, or three weeks later, the number of cases could have shown a 3-fold, 7-fold, and 18-fold increase across China, respectively. Results also suggest that the social distancing intervention should be continued for the next few months in China to prevent case numbers increasing again after travel restrictions were lifted on February 17, 2020. Conclusion: The NPIs deployed in China appear to be effectively containing the COVID-19 outbreak, but the efficacy of the different interventions varied, with the early case detection and contact reduction being the most effective. Moreover, deploying the NPIs early is also important to prevent further spread. Early and integrated NPI strategies should be prepared, adopted and adjusted to minimize health, social and economic impacts in affected regions around the World.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Shengjie Lai; Nick W Ruktanonchai; Liangcai Zhou; Olivia Prosper; Wei Luo; Jessica R Floyd; Amy Wesolowski; Mauricio Santillana; Chi Zhang; Xiangjun Du; Hongjie Yu; Andrew J Tatem&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20029843&#39;},
 {&#39;cord_uid&#39;: &#39;66ulqu11&#39;,
  &#39;sha&#39;: &#39;3028628066ec2401f3981f4e70c5b1acd4cef573&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission interval estimates suggest pre-symptomatic spread of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20029983&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods: We collected outbreak information from Singapore and Tianjin, China, reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring. Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20029983&#39;},
 {&#39;cord_uid&#39;: &#39;5ciaonf0&#39;,
  &#39;sha&#39;: &#39;462cbb326ccd8587cae7a3538c8c6712d9013698&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and clinical features of 291 cases with coronavirus disease 2019 in areas adjacent to Hubei, China: a double-center observational study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030353&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Abstract Background: The clinical outcomes of COVID-19 patients in Hubei and other areas are different. We aim to investigate the epidemiological and clinical characteristics of patient with COVID-19 in Hunan which is adjacent to Hubei. Methods: In this double-center, observational study, we recruited all consecutive patients with laboratory confirmed COVID-19 from January 23 to February 14, 2020 in two designated hospitals in Hunan province, China. Epidemiological and clinical data from patients&#39; electronic medical records were collected and compared between mild, moderate and severe/critical group in detail. Clinical outcomes were followed up to February 20, 2020. Findings: 291 patients with COVID-19 were categorized into mild group (10.0%), moderate group (72.8%) and severe/critical group (17.2%). The median age of all patients was 46 years (49.8% were male). 86.6% patients had an indirect exposure history. The proportion of patients that had been to Wuhan in severe/critical group (48.0% vs 17.2%, p=0.006) and moderate group (43.4% vs 17.2%, p=0.007) were higher than mild group. Fever (68.7%), cough (60.5%), and fatigue (31.6%) were common symptoms especially for severe and critical patients. Typical lung imaging finding were bilateral and unilateral ground glass opacity or consolidation. Leukopenia, lymphopenia and eosinopenia occurred in 36.1%, 22.7% and 50.2% patients respectively. Increased fibrinogen was detected in 45 of 58 (77.6%) patients with available results. 29 of 44 (65.9%) or 22 of 40 (55.0%) patients were positive in Mycoplasma pneumonia or Chlamydia pneumonia antibody test respectively. Compared with mild or moderate group, severe/critical group had a relative higher level of neutrophil, Neutrophil-to-Lymphocyte Ratio, h-CRP, ESR, CK, CK-MB, LDH, D-dimer, and a lower level of lymphocyte, eosinophils, platelet, HDL and sodium (all p&lt;0.01). Most patients received antiviral therapy and Chinese Medicine therapy. As of February 20, 2020, 159 (54.6%) patients were discharged and 2 (0.7%) patients died during hospitalization. The median length of hospital stay in discharged patients was 12 days (IQR: 10-15). Interpretation: The epidemiological and clinical characteristics of COVID-19 patients in Hunan is different from patients in Wuhan. The proportion of patients that had been to Wuhan in severe/critical group and moderate group were higher than mild group. Laboratory and imaging examination can assist in the diagnosis and classification of COVID-19 patients.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Xu Chen; Fang Zheng; Yanhua Qing; Shuizi Ding; Danhui Yang; Cheng Lei; Zhilan Yin; Xianglin Zhou; Dixuan Jiang; Qi Zuo; Jun He; Jianlei Lv; Ping Chen; Yan Chen; Hong Peng; Honghui Li; Yuanlin Xie; Jiyang Liu; Zhiguo Zhou; Hong Luo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030353&#39;},
 {&#39;cord_uid&#39;: &#39;cv32u0ox&#39;,
  &#39;sha&#39;: &#39;e560eb28bb3f35099d2632f80adaa14436516474&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030437&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Xiaoping Chen; Jiaxin Ling; Pingzheng Mo; Yongxi Zhang; Qunqun Jiang; Zhiyong Ma; Qian Cao; Wenjia Hu; Shi Zou; Liangjun Chen; Lei Yao; Mingqi Luo; Tielong Chen; Liping Deng; Ke Liang; Shihui Song; Rongrong Yang; Ruiying Zheng; Shicheng Gao; Xien Gui; Hengning Ke; Wei Hou; Åke Lundkvist; Yong Xiong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030437&#39;},
 {&#39;cord_uid&#39;: &#39;y8hzi1gl&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modelling-based evaluation of the effect of quarantine control by the Chinese government in the coronavirus disease 2019 outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030445&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus disease 2019 (COVID-19) epidemic, which was first identified in Wuhan, China in December 2019, has rapidly spread all over China and across the world. By the end of February 2020, the epidemic outside Hubei province in China has been well controlled, yet the next wave of transmission in other countries may have just begun. A retrospective modeling of the transmission dynamics would provide insights into the epidemiological characteristics of the disease and evaluation of the effectiveness of the strict measures that have been taken by central and local governments of China. Using a refined susceptible-exposed-infectious-removed (SEIR) transmission model and a new strategy of model fitting, we were able to estimate model parameters in a dynamic manner. The resulting parameter estimation can well reflect the prevention policy scenarios. Our simulation results with different degrees of government control suggest that the strictly enforced quarantine and travel ban have significantly decreased the otherwise uncontrollable spread of the disease. Our results suggest similar measures should be considered by other countries that are of high risk of COVID-19 outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Xinkai Zhou; Zhigui Wu; Ranran Yu; Shanni Cao; Wen Fang; Zhen Jiang; Fang Yuan; Chao Yan; Dijun Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030445&#39;},
 {&#39;cord_uid&#39;: &#39;944pn0k9&#39;,
  &#39;sha&#39;: &#39;9701a8c529cd8c18124da4cd61c1165a64b50281&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evolving Epidemiology and Impact of Non-pharmaceutical Interventions on the Outbreak of Coronavirus Disease 2019 in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030593&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND We described the epidemiological features of the coronavirus disease 2019 (Covid-19) outbreak, and evaluated the impact of non-pharmaceutical interventions on the epidemic in Wuhan, China. METHODS Individual-level data on 25,961 laboratory-confirmed Covid-19 cases reported through February 18, 2020 were extracted from the municipal Notifiable Disease Report System. Based on key events and interventions, we divided the epidemic into four periods: before January 11, January 11-22, January 23 - February 1, and February 2-18. We compared epidemiological characteristics across periods and different demographic groups. We developed a susceptible-exposed-infectious-recovered model to study the epidemic and evaluate the impact of interventions. RESULTS The median age of the cases was 57 years and 50.3% were women. The attack rate peaked in the third period and substantially declined afterwards across geographic regions, sex and age groups, except for children (age &lt;20) whose attack rate continued to increase. Healthcare workers and elderly people had higher attack rates and severity risk increased with age. The effective reproductive number dropped from 3.86 (95% credible interval 3.74 to 3.97) before interventions to 0.32 (0.28 to 0.37) post interventions. The interventions were estimated to prevent 94.5% (93.7 to 95.2%) infections till February 18. We found that at least 59% of infected cases were unascertained in Wuhan, potentially including asymptomatic and mild-symptomatic cases. CONCLUSIONS Considerable countermeasures have effectively controlled the Covid-19 outbreak in Wuhan. Special efforts are needed to protect vulnerable populations, including healthcare workers, elderly and children. Estimation of unascertained cases has important implications on continuing surveillance and interventions.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Chaolong Wang; Li Liu; Xingjie Hao; Huan Guo; Qi Wang; Jiao Huang; Na He; Hongjie Yu; Xihong Lin; An Pan; Sheng Wei; Tangchun Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030593&#39;},
 {&#39;cord_uid&#39;: &#39;d4dnz2lk&#39;,
  &#39;sha&#39;: &#39;9c33486a49de4aea64ce61c0a2c21a88c316b6a8&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030627&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: Establishing a social contact data sharing initiative and an interactive tool to assess mitigation strategies for COVID-19. Results: We organized data sharing of published social contact surveys via online repositories and formatting guidelines. We analyzed this social contact data in terms of weighted social contact matrices, next generation matrices, relative incidence and R0. We incorporated location-specific isolation measures (e.g. school closure or telework) and capture their effect on transmission dynamics. All methods have been implemented in an online application based on R Shiny and applied to COVID-19 with age-specific susceptibility and infectiousness. Using our online tool with the available social contact data, we illustrate that social distancing could have a considerable impact on reducing transmission for COVID-19. The effect itself depends on assumptions made about disease-specific characteristics and the choice of intervention(s). Keywords: social contact data, user interface, transmission dynamics, infectious diseases, epidemics, social distancing, behavioral changes, data sharing initiative, open-source, COVID-19&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Lander Willem; Thang Van Hoang; Sebastian Funk; Pietro Coletti; Philippe Beutels; Niel Hens&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030627&#39;},
 {&#39;cord_uid&#39;: &#39;5imbomop&#39;,
  &#39;sha&#39;: &#39;594d23a412649df7dbfef5347a2a0c208ed4417d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Immunodepletion with Hypoxemia: A Potential High Risk Subtype of Coronavirus Disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030650&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of COVID-2019 is becoming a global public health emergency. Although its basic clinical features have been reported, the dynamic characteristics of immune system in COVID-2019 patients, especially those critical patients with refractory hypoxemia, are not yet well understood. We aim to describe the dynamic characteristics of immune system in 3 critical patients with refractory hypoxemia, and discuss the relationship between hypoxemia severity and immune cell levels, and the changes of gut microbes of COVID-2019 patient. Methods This is a retrospective study from 3 patients with 2019-nCoV infection admitted to Renmin Hospital of Wuhan University, a COVID-2019 designated hospital in Wuhan, from January 31 to February 6, 2020. All patients were diagnosed and classified based on the Diagnosis and Treatment of New Coronavirus Pneumonia (6th edition) published by the National Health Commission of China4. We recorded the epidemiological history, demographic features, clinical characteristics, symptoms and signs, treatment and clinical outcome in detail. Blood samples were collected and we determined the expression levels of immune cells (CD3+ T cells, CD4+ T cells, CD8+ T cells, CD19+ B cells, and CD16+56+ NK cells) in different time points. Nanopore Targeted Sequencing was used to determine the alterations of gut microbiota homeostasis. Results Apart from the clinical features described previously4, we found that four patients had decreased immune cells and refractory hypoxemia during the hospitalization, and the severity of hypoxemia was strongly correlated to the expression levels of immune cells. Additionally, we found that the proportion of probiotics was significantly reduced, such as Bifidobacterium, Lactobacillus, and Eubacterium, and the proportion of conditioned pathogenic bacteria was significantly increased, such as Corynebacterium of Actinobacteria and Ruthenibacterium of Firmicutes. Notably, all patients died. Conclusions We discussed the dynamic characteristics of host immune system and the imbalance of gut microbiota in 3 critical patients with COVID-2019. Hypoxemia severity was closely related with host immune cell levels, and the vicious circle between immune disorder and gut microbiota imbalance may be a high risk of fatal pneumonia. To the best of our knowledge, this is the first study which revealing that immunodepletion with refractory hypoxemia is a potential high risk subtype of COVID-2019 and the vicious circle between immune disorder and gut dysbiosis may be a high risk of fatal pneumonia.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Lilei Yu; Yongqing Tong; Gaigai Shen; Aisi Fu; Yanqiu Lai; Xiaoya Zhou; Yuan Yuan; Yuhong Wang; Yuchen Pan; Zhiyao Yu; Yan Li; Tiangang Liu; Hong Jiang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030650&#39;},
 {&#39;cord_uid&#39;: &#39;9ecuca17&#39;,
  &#39;sha&#39;: &#39;f879c69d61f14d7efc51023258d1361bc681d403&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical findings in critical ill patients infected with SARS-Cov-2 in Guangdong Province, China: a multi-center, retrospective, observational study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030668&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Background In December 2019, human infection with a novel coronavirus, known as SARS-CoV-2, was identified in Wuhan, China. The mortality of critical illness was high in Wuhan. Information about critically ill patients with SARS-CoV-2 infection outside of Wuhan is scarce. We aimed to provide the clinical features, treatment, and prognosis of the critically ill patients with SARS-CoV-2 infection in Guangdong Province. Methods In this multi-centered, retrospective, observational study, we enrolled critically ill patients with SARS-CoV-2 pneumonia who were admitted to the intensive care unit (ICU) in Guangdong Province. Demographic data, symptoms, laboratory findings, comorbidities, treatments, and prognosis were collected. Data were compared between patients with and without intubation. Results Forty-five critically ill patients with SARS-CoV-2 pneumonia were identified in 7 ICUs in Guangdong Province. The mean age was 56.7 years, and 29 patients (64.4%) were men. The most common symptoms at the onset of illness were high fever and cough. Majority of patients presented with lymphopenia and elevated lactate dehydrogenase. Treatment with antiviral drugs was initiated in all the patients. Thirty-seven patients (82.2%) had developed acute respiratory distress syndrome, and 13 (28.9%) septic shock. A total of 20 (44.4%) patients required intubation and 9 (20%) required extracorporeal membrane oxygenation. As of February 28th 2020, only one patient (2.2%) had died and half of them had discharged of ICU. Conclusions Infection with SARS-CoV-2 in critical illness is characterized by fever, lymphopenia, acute respiratory failure and multiple organ dysfunction. Compared with critically ill patients infected with SARS-CoV-2 in Wuhan, the mortality of critically ill patients in Guangdong Province was relatively low. These data provide some general understandings and experience for the critical patients with SARS-CoV-2 outside of Wuhan.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Yonghao Xu; Zhiheng Xu; Xuesong Liu; Lihua Cai; Haichong Zheng; Yongbo Huang; Lixin Zhou; Linxi Huang; Yun Lin; Liehua Deng; Jianwei Li; Sibei Chen; Dongdong Liu; Zhimin Lin; Liang Zhou; Weiqun He; Xiaoqing Liu; Yimin Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030668&#39;},
 {&#39;cord_uid&#39;: &#39;6t9kl3hq&#39;,
  &#39;sha&#39;: &#39;e0a53c258ee0483666329ccab591c0753ff49531&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Key Points of Clinical and CT Imaging Features of 2019 Novel Coronavirus (2019-nCoV) Imported Pneumonia Based On 21 Cases Analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030775&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background and Objective: WHO Director-General declared that the 2019-nCoV outbreak constitutes a Public Health Emergency of International Concern,and the outbreak is still on-going.Chest CT had been a key component of the diagnostic workup for patients with suspected infection. In this retrospective study, we attempt to summarize and analyze the chest CT features of 2019-nCov infections, and to identify the typical features to improved the diagnostic accuracy of new coronavirus pneumonia (NCP). Methods:Chest CT scans and Clinical data of 21 patients confirmed NCP in our hospital were enrolled.These patients were divided into mild and sever group according to clinical manifestations described by the 6th clinical practice guideline of NCP in China. Main clinical and chest CT features were analyzed and identify. Results: Fever (85.7%) and cough (80.9%) were the two main symptoms of NCP patients.More significantly higher incidence (85.7%) of shortness of breath in the severe cases. Multiple lesions in both lungs and with incidence of GGO(100%),vascular enlargement (76.5%) and cobblestone/reticular pattern(70.6%) were the major feature.The incidence of consolidation, mixed pattern and vascular enlargement features were up to 100% in the severe group, significantly higher than that of patients in mild group. In addition, the incidence of air-bronchogram, dilated bronchi with thickened wall and fibrosis in the severe group was significantly higher than that in the mild group. Conclusions: Fever and cough are the typical clinical features of NCP patients, and chest CT mainly manifested as multiple lesions in both lungs, often accompanied by GGO, vascular enlargement and cobblestone/reticular pattern.Changes in these main CT features can indicate development of the disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;wenxiu Wu; zhifeng xu; yabin Jin; aizhen Pan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030775&#39;},
 {&#39;cord_uid&#39;: &#39;zhdjtv4j&#39;,
  &#39;sha&#39;: &#39;0ab795fc615df6457551a8e231dce1f268eef9d2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Caution: The clinical characteristics of COVID-19 patients at admission are changing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030833&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background With the emergence of 4rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 (COVID-19) has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Methods Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. Results Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24). Conclusions The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Zhaowei Chen; Jijia Hu; Zongwei Zhang; Shan Jiang; Tao Wang; Zhengli Shi; Zhan Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030833&#39;},
 {&#39;cord_uid&#39;: &#39;1mu1z4xd&#39;,
  &#39;sha&#39;: &#39;5bb89950ec5a06e2b7f69b2a9c4213dda19b1ab0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prediction of New Coronavirus Infection Based on a Modified SEIR Model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030858&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND The outbreak of the new coronavirus infection in Wuhan City, Hubei Province in December 2019, poses a huge threat to China and even global public health security. Respiratory droplets and contact transmission are the main routes of transmission of new coronaviruses. Compared with SARS and Ebola viruses, new coronavirus infections are infectious during the incubation period. Traditional SEIR (susceptibility-exposure-infection-Removal) There are some differences in conditions for the prediction of the epidemic trend of new coronavirus infection. The outbreak of the new coronavirus infection coincided with the Spring Festival before and after the Chinese Spring Festival.It is necessary to make appropriate optimization and amendments to the traditional model to meet the actual evolution of the epidemic situation. METHODS The traditional SEIR model assumes that the virus-infected person is not infectious during the incubation period and that the infected person did not take isolation measures during the illness. The transmission of the new coronavirus no longer meets the basic assumptions of the classical kinetic system. Therefore, this article first establishes a modified SEIR model. Predict and analyze the changing trend of the epidemic situation, then estimate the parameters involved in the infection dynamics model, and then use Matlab to simulate the established dynamic equations based on public data and analyze the results. Recommendations for universal prevention and control of infectious diseases. RESULTS The first case of new coronavirus infection was confirmed in Wuhan on December 8, 2019. When Wuhan City took no action, assuming the average daily number of contacts per infected person k = 5, the number of infected persons will reach about 2,384,803 people; If wuhan adopts the measures of sealing the city on January 22, 2020, under the premise of k=2, the number of infected people decreases by 19,773 compared with that on January 23, and there is no significant change in the time when the number of infected people reaches the peak. Under the premise of k = 1, the number of infected persons was reduced by 14,330 compared with the closure on January 23, and the time to reach the peak of the number of infected persons was reduced by 2 days. If Wuhan City is closed for one day, the number of infected persons will increase from 106,145 to 130,626 under the premise of k = 2; the number of infected persons will increase from 74,369 to 92,010 under the premise of k = 1. CONCLUSIONS Comparing the number of confirmed diagnoses actually notified by the department with the number of infected people obtained from the simulation of the model, it can be seen that the city closure measures adopted by the Wuhan Municipal Government on January 23 and the first-level response measures adopted by the country are effective for the epidemic Prevention and control play a vital role. Wearing a mask when going out and avoiding close contact with people can effectively reduce the infection rate.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Zhou Tang; Xianbin Li; Houqiang Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030858&#39;},
 {&#39;cord_uid&#39;: &#39;itp1hnql&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Psychological impact of the coronavirus disease 2019 (COVID-19) outbreak on healthcare workers in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.20030874&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Introduction Since the outbreak of coronavirus disease 2019 (COVID-19), more than 3000 (including clinical diagnosis) healthcare workers (HCWs) have been infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China. This study is aimed to investigate the risk perception and immediate psychological state of HCWs in the early stage of the COVID-19 epidemic. Methods This study utilized a cross-sectional survey designed on a convenient sample of 4357 HCWs in China. Its data were collected using anonymous structured questionnaires distributed through social software. 6 questions were set to evaluate the participants&#39; risk perception of COVID-19, and a General Health Questionnaire was used to identify the participants&#39; immediate psychological status. Descriptive statistics were used for data analysis. Risk perception and psychological status were compared by demographic characteristics and COVID-19 exposure experiences. Result A total of 4,600 questionnaires were distributed, and 4,357 qualified ones (94.7%) were collected. The main concerns of HCWs are: infection of colleagues (72.5%), infection of family members (63.9%), protective measures (52.3%) and medical violence (48.5%). And 39.1% of the HCWs had psychological distress, especially working in Wuhan, participating in frontline treatments, having been isolated and having family members or colleagues infected. Conclusions The finding indicating that, faced with the COVID-19 epidemic, HCWs, especially in Wuhan, were worried about the risks of infection and protective measures, resulting in psychological distress, so further actions should be taken.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Yuhong Dai; Guangyuan Hu; Huihua Xiong; Hong Qiu; Xianglin Yuan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.20030874&#39;},
 {&#39;cord_uid&#39;: &#39;qeo5dfxg&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.962332&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA new coronavirus SARS-CoV-2, recently discovered in Wuhan, China, has caused over 74000 infection cases and 2000 deaths. Due to the rapidly growing cases and the unavailability of specific therapy, there is a desperate need for vaccines to combat the epidemic of SARS-CoV-2. In the present study, we performed an in silico approach based on the available virus genome to identify the antigenic B-cell epitopes and human-leukocyte-antigen (HLA) restricted T-cell epitopes. A total of 61 B-cell epitopes were initially identified, 19 of which with higher potential immunogenicity were used for vaccine design. 499 T-cell epitopes were predicted that showed affinity with the 34 most popular HLA alleles in Chinese population. Based on these epitopes, 30 vaccine candidates were designed and inspected against safety risks, including potential toxicity, human homologous, pharmaceutical peptides and bioactive peptides. Majority of vaccine peptides contained both B-cell and T-cell epitopes, which may interact with the most prevalent HLA alleles accounting for ~99% of Chinese population. Docking analysis showed stable hydrogen bonds of epitopes with their corresponding HLA alleles. In conclusion, these putative antigenic peptides may elicit the resistance response to the viral infection. In vitro and in vivo experiments are required to validate the effectiveness of these peptide vaccine.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-03&#39;,
  &#39;authors&#39;: &#39;Ye Feng; Min Qiu; Shengmei Zou; Yun Li; Kai Luo; Rongchang Chen; Yingqiang Sun; Kui Wang; Xinlei Zhuang; Shanshan Zhang; Shuqing Chen; Fan Mo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.962332&#39;},
 {&#39;cord_uid&#39;: &#39;qmg58cgr&#39;,
  &#39;sha&#39;: &#39;8349823092836fe397a59e38615d1491423dbe70&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.972133&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Bowen Tang; Fengming He; Dongpeng Liu; Meijuan Fang; Zhen Wu; Dong Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.972133&#39;},
 {&#39;cord_uid&#39;: &#39;3ybko13r&#39;,
  &#39;sha&#39;: &#39;0fa18c47543c3ce69a3c9e4d77958c4d3691b1ed&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Significance of hydrophobic and charged sequence similarities in sodium-bile acid cotransporter and vitamin D-binding protein macrophage activating factor&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.03.975524&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSodium-bile acid cotransporter, also denominated sodium-taurocholate cotransporting polypeptide (NTCP) is an integral membrane protein with multiple hydrophobic transmembrane domains. The third extracellular domain of NTCP presents a stretch of nine aminoacids (KGIVISLVL) that is characterized by pronounced hydrophobicity and serves as receptor for a protein, preS1, showing the hydrophobic epta-peptide sequence NPLGFFP. Vitamin D-binding protein macrophage activating factor (DBP-MAF) is a multifunctional protein that is characterized by two hydrophobic regions able to bind fatty acids and vitamin D, respectively. Here we demonstrate that NTCP and DBP-MAF show significant sequence similarities as far as hydrophobic stretches of aminoacids are concerned. Alignment of the sequence of seven aminoacids preceding the 157-KGIVISLVL-165 stretch of NTCP shows four aminoacids that are identical to those of the corresponding sequence of DBP-MAF, and two that are conserved substitutions. In addition, in the sequence of DBP-MAF that is aligned with the sequence YKGIVISLVL of NTCP, there are two contiguous negatively charged aminoacids (ED) and, in the preceding epta-peptide sequence, there are three negatively charged aminoacids (D-ED), whereas in the corresponding sequence of NTCP there are only two (D--D) that are not contiguous. This concentration of negatively charged aminoacids may be involved in binding of protein inserts characterized by high density of positively charges residues. The alternating hydrophobic and electrostatic interactions described in this paper may help elucidating the biological roles of these proteins as far as protein-protein interactions are concerned.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-05&#39;,
  &#39;authors&#39;: &#39;Ibne Raihan Zunaid; Stefania Pacini; Marco Ruggiero&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.03.975524&#39;},
 {&#39;cord_uid&#39;: &#39;ti9b1etu&#39;,
  &#39;sha&#39;: &#39;3afd5fba7dc182ddfa769c0d766134b525581005&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission and clinical characteristics of coronavirus disease 2019 in 104 outside-Wuhan patients, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20026005&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Cases with coronavirus disease 2019 (COVID-19) emigrated from Wuhan escalated the risk of spreading in other cities. This report focused on the outside-Wuhan patients to assess the transmission and clinical characteristics of this illness. Methods: Contact investigation was conducted on each patient who admitted to the assigned hospitals in Hunan Province (geographically adjacent to Wuhan) from Jan 22, 2020 to Feb 12, 2020. Demographic, clinical, laboratory and radiological characteristics, medication therapy and outcomes were collected and analyzed. Patients were confirmed by PCR test. Results: Of the 104 patients, 48 (46.15%) were imported cases and 56 (53.85%) were indigenous cases; 93 (89.42%) had a definite contact history with infections. Family clusters were the major body of patients. Transmission along the chain of 3 &amp;ldquo:generations” was observed. Mean age was 43 (rang, 8-84) years (including 3 children) and 49 (47.12%) were male. Most patients had typical symptoms, 5 asymptomatic infections were found and 2 of them infected their relatives. The median incubation period was 6 (rang, 1-32) days, of 8 patients ranged from 18 to 32 days. Just 9 of 16 severe patients required ICU care. Until Feb 12, 2020, 40 (38.46%) discharged and 1 (0.96%) died. For the antiviral treatment, 80 (76.92%) patients received traditional Chinese medicine therapy. Conclusions: Family but not community transmission occupied the main body of infections in the two centers. Asymptomatic transmission demonstrated here warned us that it may bring more risk to the spread of COVID-19. The incubation period of 8 patients exceeded 14 days.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Chengfeng Qiu; Qian Xiao; Xin Liao; Ziwei Deng; Huiwen Liu; Yuanlu Shu; Dinghui Zhou; Ye Deng; Hongqiang Wang; Xiang Zhao; Jianliang Zhou; Jin Wang; Zhihua Shi; Long Da&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20026005&#39;},
 {&#39;cord_uid&#39;: &#39;s4d442kx&#39;,
  &#39;sha&#39;: &#39;3deb6393bd43f440a69c84da9ed61986b1781d36&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20029538&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20029538&#39;},
 {&#39;cord_uid&#39;: &#39;dmud2zf7&#39;,
  &#39;sha&#39;: &#39;3fe2552d3d8e6c9ead2c41e30fbb69e917277f53&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20030395&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since mid-December 2019, a cluster of pneumonia-like diseases caused by a novel coronavirus, now designated COVID-19 by the WHO, emerged in Wuhan city and rapidly spread throughout China. Here we identify the clinical characteristics of COVID-19 in a cohort of patients in Shanghai. Methods: Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, laboratory and radiological features. Results: Of 198 patients, the median duration from disease onset to hospital admission was 4 days. The mean age of the patients was 50.1 years, and 51.0% patients were male. The most common symptom was fever. Less than half of the patients presented with respiratory systems including cough, sputum production, itchy or sore throat, shortness of breath, and chest congestion. 5.6% patients had diarrhoea. On admission, T lymphocytes were decreased in 45.8% patients. Ground glass opacity was the most common radiological finding on chest computed tomography. 9.6% were admitted to the ICU because of the development of organ dysfunction. Compared with patients not treated in ICU, patients treated in the ICU were older, had longer waiting time to admission, fever over 38.5o C, dyspnoea, reduced T lymphocytes, elevated neutrophils and organ failure. Conclusions: In this single centre cohort of COVID-19 patients, the most common symptom was fever, and the most common laboratory abnormality was decreased blood T cell counts. Older age, male, fever over 38.5oC, symptoms of dyspnoea, and underlying comorbidity, were the risk factors most associated with severity of disease. Key words: 2019 novel coronavirus; acute respiratory infection; risk factors for disease severity&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Min Cao; Dandan Zhang; Youhua Wang; Yunfei Lu; Xiangdong Zhu; Ying Li; Honghao Xue; Yunxiao Lin; Min Zhang; Yiguo Sun; Zongguo Yang; Jia Shi; Yi Wang; Chang Zhou; Yidan Dong; Ping Liu; Steven M Dudek; Zhen Xiao; Hongzhou Lu; Longping Peng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20030395&#39;},
 {&#39;cord_uid&#39;: &#39;na8odvj7&#39;,
  &#39;sha&#39;: &#39;4380f0251c595d6cb551e643198d2dacd3c6746c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20030916&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Corona Virus Disease 2019 (COVID-19) has spread rapidly to more than 70 countries and regions overseas and over 80000 cases have been infected, resulting in more than three thousand deaths. Rapid diagnosis of patients remains a bottleneck in containing the progress of the epidemic. We used automated chemiluminescent immunoassay to detect serum IgM and IgG antibodies to 2019-nCoV of 736 subjects. COVID-19 patients were becoming reactive(positive) for specific antibodies from 7-12 days after the onset of morbidity. Specific IgM and IgG increased with the progression of the disease. The areas under the ROC curves of IgM and IgG were 0.988 and 1.000, respectively. Specific antibody detection has good sensitivity and specificity. Detection of specific antibodies in patients with fever can be a good distinction between COVID-19 and other diseases, so as to be a complement to nucleic acid diagnosis to early diagnosis of suspected cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Jin Zhang; Jianhua Liu; Na Li; Yong Liu; Rui Ye; Xiaosong Qin; Rui Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20030916&#39;},
 {&#39;cord_uid&#39;: &#39;rscduazd&#39;,
  &#39;sha&#39;: &#39;e066fdbb4a9338a57477424cb66fd4d75b2fd66d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Comparison of severe and non-severe COVID-19 pneumonia: review and meta-analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20030965&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To compare the difference between severe and non-severe COVID-19 pneumonia and figure out the potential symptoms lead to severity. Methods: Articles from PubMed, Embase, Cochrane database, and google up-to 24 February 2020 were systematically reviewed. Eighteen Literatures were identified with cases of COVID-19 pneumonia. The extracted data includes clinical symptoms, age, gender, sample size and region et al were systematic reviewed and meta analyzed. Results: 14 eligible studies including 1,424 patients were analyzed. Symptoms like fever (89.2%), cough (67.2%), fatigue (43.6%) were common, dizziness, hemoptysis, abdominal pain and conjunctival congestion/conjunctivitis were rare. Polypnea/dyspnea in severe patients were significantly higher than non-severe (42.7% vs.16.3%, P&lt;0.0001). Fever and diarrhea were higher in severe patients(p=0.0374and0.0267). Further meta-analysis showed incidence of fever(OR1.70,95%CI 1.01-2.87), polypnea/dyspnea(OR3.53, 95%CI 1.95-6.38) and diarrhea(OR1.80,95%CI 1.06-3.03) was higher in severe patients, which meant the severe risk of patients with fever, polypnea/dyspnea, diarrhea were 1.70, 3.53, 1.80 times higher than those with no corresponding symptoms. Conclusions: Fever, cough and fatigue are common symptoms in COVID-19 pneumonia. Compared with non-severe patients, the symptoms as fever, polypnea/dyspnea and diarrhea are potential symptoms lead to severity.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-09&#39;,
  &#39;authors&#39;: &#39;Weiping Ji; Jing Zhang; Gautam Bishnu; Xudong Du; Xinxin Chen; Hui Xu; Xiaoling Guo; Zhenzhai Cai; Xian Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20030965&#39;},
 {&#39;cord_uid&#39;: &#39;sozs02x0&#39;,
  &#39;sha&#39;: &#39;1a12cc7d49d8521bec5d447ebb413cbcf5aca8f4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Study of the mental health status of medical personnel dealing with new coronavirus pneumonia&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20030973&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: This paper studied the relationship between personality traits and mental health conditions of medical personnel to provide a basis and reference for the implementation of targeted education on mental health. Methods: A self-report inventory, the Symptom Checklist-90 (SCL-90), was used to investigate the mental health status of 548 medical personnel dealing with the new coronavirus pneumonia in eight provinces and cities of China. Results: The overall mean SCL-90 score and mean values of factors (somatization, obsessive-compulsive, anxiety, phobic anxiety, and psychoticism) of the medical personnel were significantly higher than in the norm group (p &lt; 0.05), while their average interpersonal sensitivity score was significantly lower (p &lt; 0.01). In addition, personal factors affecting the mental health status of medical personnel were identified. ( all p &lt; 0.05). Conclusion: The overall mental health status of medical personnel responding to new coronavirus pneumonia is generally higher than that of the norm group in China. The results of this study should contribute to measures to alleviate the psychological pressures on medical personnel dealing with the new coronavirus epidemic in China&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;ning sun; jun xing; jun xu; ling shu geng; qian yu li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20030973&#39;},
 {&#39;cord_uid&#39;: &#39;dop8knqn&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Case fatality risk of novel coronavirus diseases 2019 in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031005&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective The outbreak of novel coronavirus disease 2019 (COVID-19) imposed a substantial health burden in mainland China and remains a global epidemic threat. Our objectives are to assess the case fatality risk (CFR) among COVID-19 patients detected in mainland China, stratified by clinical category and age group. Method We collected individual information on laboratory-confirmed COVID-19 cases from publicly available official sources from December 29, 2019 to February 23, 2020. We explored the risk factors associated with mortality. We used methods accounting for right-censoring and survival analyses to estimate the CFR among detected cases. Results Of 12,863 cases reported outside Hubei, we obtained individual records for 9,651 cases, including 62 deaths and 1,449 discharged cases. The deceased were significantly older than discharged cases (median age: 77 vs 39 years, p&lt;0.001). 58% (36/62) were male. Older age (OR 1.18 per year; 95%CI: 1.14 to 1.22), being male (OR 2.02; 95%CI: 1.02 to 4.03), and being treated in less developed economic regions (e.g., West and Northeast vs. East, OR 3.93; 95%CI: 1.74 to 8.85) were mortality risk factors. The estimated CFR was 0.89-1.24% among all cases. The fatality risk among critical patients was 2-fold higher than that among severe and critical patients, and 24-fold higher than that among moderate, severe and critical patients. Conclusions Our estimates of CFR based on laboratory-confirmed cases ascertained outside of Hubei suggest that COVID-19 is not as severe as severe acute respiratory syndrome and Middle East respiratory syndrome, but more similar to the mortality risk of 2009 H1N1 influenza pandemic in hospitalized patients. The fatality risk of COVID-19 is higher in males and increases with age. Our study improves the severity assessment of the ongoing epidemic and can inform the COVID-19 outbreak response in China and beyond.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Xiaowei Deng; Juan Yang; Wei Wang; Xiling Wang; Jiaxin Zhou; Zhiyuan Chen; Jing Li; Yinzi Chen; Han Yan; Juanjuan Zhang; Yongli Zhang; Yan Wang; Qi Qiu; Hui Gong; Xianglin Wei; Lili Wang; Kaiyuan Sun; Peng Wu; Marco Ajelli; Benjamin J. Cowling; Cecile Viboud; Hongjie Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031005&#39;},
 {&#39;cord_uid&#39;: &#39;vg1vvm6f&#39;,
  &#39;sha&#39;: &#39;dd14a222d0d516266debe1c487e32ad13c105291&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics of 101 non-surviving hospitalized patients with COVID-19: A single center, retrospective study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031039&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Rationale: COVID-19 have deprived many lives in Wuhan, China and caused global concerns. Few studies focused on clinical characteristics non-survivors hospitalized with COVID-19. Objectives: We aimed to clarify the clinical characteristics of died patients with COVID-19 and tracked the causes of the rapid disease progression and death. Methods: 101 non-surviving patients with confirmed COVID-19 were included in this retrospective study. Clinical data were collected and compared between non-survivors who died within 3 days and after 3 days of admission. Multivariable regression analysis was used to analyze risk factors associated with rapid disease progression and death. Measurements and Main Results: Among included patients, median age was 71 years (IQR, 59-80), 60 (59.41%) were men. 82 (81.19%) had one or more comorbidities including hypertension (59 [58.42%]), diabetes (22 [21.78%]) etc. 100 (99.01%) suffered respiratory failure, 53 (52.48%) developed acute cardiac injury, acute kidney and liver injury occurred in 23 (22.77%) and 18 (17.82%) patients, respectively. Multivariable regression analysis showed that elevated high sensitivity troponin I (OR 2.68, 95%CI 1.31-5.49, P=0.007), neutrophils (OR 1.14, 95%CI 1.01-1.28, P=0.033) and depressed oxygen saturation (OR 0.94, 95%CI 0.89-0.99, P=0.027) on admission were associated with rapid death of patients with COVID-19. Conclusions: Older patients (&gt;70 years) with comorbidities had a steeply increased risk of death with COVID-19. Respiratory failure, acute cardiac and kidney injury played a crucial role in the death of patients. Elevated high sensitivity troponin, neutrophils and depressed oxygen saturation predicted the rapid death of patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Qiao Shi; Kailiang Zhao; Jia Yu; Jiarui Feng; Kaiping Zhao; Xiaoyi Zhang; Xiaoyan Chen; Peng Hu; Yupu Hong; Man Li; Fang Liu; Chen Chen; Weixing Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031039&#39;},
 {&#39;cord_uid&#39;: &#39;4ep7nf3q&#39;,
  &#39;sha&#39;: &#39;f47af9fe364fd9bac5e061142208d801667c5aca&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;How to differentiate COVID-19 pneumonia from heart failure with computed tomography at initial medical contact during epidemic period&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031047&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;OBJECTIVES To compare chest CT findings in heart failure with those of Corona Virus Disease 2019 (COVID-19) pneumonia. BACKGROUND During epidemic period, chest computed tomography (CT) has been highly recommended for screening patients with suspected COVID-19. However, the comparison of CT imaging between heart failure and COVID-19 pneumonia has not been fully elucidated. METHODS Patients with heart failure (n=12), COVID-19 pneumonia (n=12) and one patient with both diseases were retrospectively enrolled. Clinical information and imaging of chest CT were collected and analyzed. RESULTS There was no difference of ground glass opacity (GGO), consolidation, crazy paving pattern, lobes affected and septal thickening between heart failure and COVID-19 pneumonia. However, less rounded morphology (8.3% vs. 67%, p=0.003), more peribronchovascular thickening (75% vs. 33%, p=0.041) and fissural thickening (33% vs. 0%, p=0.028), less peripheral distribution (33% vs. 92%, p=0.003) were found in heart failure group than that in COVID-19 group. Importantly, there were also more patients with upper pulmonary vein enlargement (75% vs. 8.3%, p=0.001), subpleural effusion and cardiac enlargement in heart failure group than that in COVID-19 group (50% vs. 0%, p=0.005, separately). Besides, more fibrous lesions were found in COVID-19 group although there was no statistical difference (25% vs. 0%, P=0.064) CONCLUSIONS Although there are some overlaps of CT imaging between heart failure and COVID-19, CT is still a useful tool in differentiating COVID-19 pneumonia.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Zhaowei Zhu; Jianjun Tang; Xiangping Chai; Zhenfei Fang; Qiming Liu; Xinqun Hu; Danyan Xu; Jia He; Liang Tang; Shi Tai; Yuzhi Wu; Shenghua Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031047&#39;},
 {&#39;cord_uid&#39;: &#39;mrsya6wz&#39;,
  &#39;sha&#39;: &#39;29b3eaeacecf2bfc95b9a493dbb1ba4b2323dd46&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Adjusted age-specific case fatality ratio during the COVID-19 epidemic in Hubei, China, January and February 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031104&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The coronavirus disease 2019 (COVID-19) epidemic that originated in Wuhan, China has spread to more than 60 countries. We estimated the age-specific case fatality ratio (CFR) by fitting a transmission model to data from China, accounting for underreporting of cases and the time delay to death. Overall CFR among all infections was 1.6% (1.4-1.8%) and increased considerably for the elderly, highlighting the expected burden for populations with further expansion of the COVID-19 epidemic around the globe.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Julien Riou; Anthony Hauser; Michel J Counotte; Christian L Althaus&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031104&#39;},
 {&#39;cord_uid&#39;: &#39;f4hj35dr&#39;,
  &#39;sha&#39;: &#39;734779fad93249a2f6fede6afd10eeff7b37919b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031112&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Stephen M Kissler; Christine Tedijanto; Edward Goldstein; Yonatan H. Grad; Marc Lipsitch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031112&#39;},
 {&#39;cord_uid&#39;: &#39;bpnqz1cn&#39;,
  &#39;sha&#39;: &#39;ee902c8e20385bed8b5fccb85d4e53996069cc0c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031120&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND The outbreak of a novel coronavirus (SARS-CoV-2, previously provisionally named 2019 novel coronavirus or 2019-nCoV) since December 2019 in Wuhan, China, has become an emergency of major international concern. Apart from the respiratory system, it is unclear whether SARS-CoV-2 can also directly infect other tissues such as the kidney or induce acute renal failure. METHODS We conducted a retrospective analysis of estimated glomerular filtration rate (eGFR) along with other clinical parameters from 85 patients with laboratory-confirmed COVID-19 admitted to a hospital in Wuhan from January 17, 2020 to March 3, 2020. Kidney tissues from six patients with postmortem examinations in the other hospital were analyzed by Hematoxylin and Eosin (H&amp;E) and in situ expression of viral nucleocaspid protein (NP) antigen, immune cell markers (CD8, CD68 and CD56) and the complement C5b-9 was detected by immunohistochemistry. RESULTS 27.06% (23/85) patients exhibited acute renal failure (ARF). The eldery patients and cases with comorbidities such as hypertension and heart failure more easily developed ARF (65.22% vs 24.19%, p&lt; 0.001; 69.57% vs 11.29%, p&lt; 0.001, respectively). H&amp;E staining demonstrated kidney tissues from postmortems have severe acute tubular necrosis and lymphocyte infiltration. Immunohistochemistry showed that SARS-CoV-2 NP antigen was accumulated in kidney tubules. Viral infection not only induces CD68+ macrophages infiltrated into tubulointerstitium, but also enhances complement C5b-9 deposition on tubules. CONCLUSIONS SARS-CoV-2 induces ARF in COVID-19 patients. Viruses directly infect human kidney tubules to induce acute tubular damage. The viruses not only have direct cytotoxicity, but also initiate CD68+ macrophage together with complement C5b-9 deposition to mediate tubular pathogenesis.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Bo Diao; Chenhui Wang; Rongshuai Wang; Zeqing Feng; Yingjun Tan; Huiming Wang; Changsong Wang; Liang Liu; Ying Liu; Yueping Liu; Gang Wang; Zilin Yuan; Liang Ren; Yuzhang Wu; Yongwen Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031120&#39;},
 {&#39;cord_uid&#39;: &#39;utlponna&#39;,
  &#39;sha&#39;: &#39;f40eaec892778166e4f8f8eeb576893e82a591be&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The impact of social distancing and epicenter lockdown on the COVID-19 epidemic in mainland China: A data-driven SEIQR model study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031187&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of coronavirus disease 2019 (COVID-19) which originated in Wuhan, China, constitutes a public health emergency of international concern with a very high risk of spread and impact at the global level. We developed data-driven susceptible-exposed-infectious-quarantine-recovered (SEIQR) models to simulate the epidemic with the interventions of social distancing and epicenter lockdown. Population migration data combined with officially reported data were used to estimate model parameters, and then calculated the daily exported infected individuals by estimating the daily infected ratio and daily susceptible population size. As of Jan 01, 2020, the estimated initial number of latently infected individuals was 380.1 (95%-CI: 379.8~381.0). With 30 days of substantial social distancing, the reproductive number in Wuhan and Hubei was reduced from 2.2 (95%-CI: 1.4~3.9) to 1.58 (95%-CI: 1.34~2.07), and in other provinces from 2.56 (95%-CI: 2.43~2.63) to 1.65 (95%-CI: 1.56~1.76). We found that earlier intervention of social distancing could significantly limit the epidemic in mainland China. The number of infections could be reduced up to 98.9%, and the number of deaths could be reduced by up to 99.3% as of Feb 23, 2020. However, earlier epicenter lockdown would partially neutralize this favorable effect. Because it would cause in situ deteriorating, which overwhelms the improvement out of the epicenter. To minimize the epidemic size and death, stepwise implementation of social distancing in the epicenter city first, then in the province, and later the whole nation without the epicenter lockdown would be practical and cost-effective.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Yuzhen Zhang; Bin Jiang; Jiamin Yuan; Yanyun Tao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031187&#39;},
 {&#39;cord_uid&#39;: &#39;rbsv58fo&#39;,
  &#39;sha&#39;: &#39;bfed3ce6a6df2d442d96d0cab07d4bc1f2347255&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Seattle Flu Study - Swab and Send: Study Protocol for At-Home Surveillance Methods to Estimate the Burden of Respiratory Pathogens on a City-Wide Scale&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031211&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction. While seasonal influenza and other respiratory pathogens cause significant morbidity and mortality each year, the community-based burden of these infections remains incompletely understood. Understanding the prevalence, epidemiology, and transmission dynamics of respiratory pathogen infections among community-dwelling individuals is essential during pandemic and epidemic settings and for developing pandemic-preparedness infrastructure. Methods and Analysis. We present the protocol for a novel, city-wide home-based cross-sectional study in the Seattle Metropolitan area, utilizing rapid delivery systems for self-collection of a nasal swab and return to the laboratory for respiratory pathogen testing. All participation takes place electronically, including recruitment, consent, and data collection. Within 48 hours of participants self-reporting respiratory symptoms, a nasal swab kit is delivered to the household via a courier service. Demographic and illness characteristics are collected at the time of sample collection and recovery and behavioral information collected one week later. Specimens are tested in the laboratory for multiple respiratory pathogens, and results are available on a public website for participants. Ethics and Dissemination. The study was approved by the University of Washington Institutional Review Board (Protocol #00006181). Results will be disseminated through peer-reviewed publications, talks at conferences, and on the Study Website (www.seattleflu.org).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Ashley E Kim; Elisabeth Brandstetter; Chelsey Graham; Jessica Heimonen; Audrey Osterbind; Denise J McCulloch; Peter D Han; Lea M Starita; Deborah A Nickerson; Margaret M Van de Loo; Jennifer Mooney; Mark J Rieder; Misja Ilcisin; Kairsten A Fay; Jover Lee; Thomas R Sibley; Trevor Bedford; Janet A Englund; Michael Boeckh; Helen Y Chu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031211&#39;},
 {&#39;cord_uid&#39;: &#39;myh73pe6&#39;,
  &#39;sha&#39;: &#39;b696d208705fcb1925693c5f0d118733bb557ea6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Exploring diseases/traits and blood proteins causally related to expression of ACE2, the putative receptor of 2019-nCov: A Mendelian Randomization analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031237&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The novel coronavirus 2019-nCoV has caused major outbreaks in many parts of the world. A better understanding of the pathophysiology of COVID-19 is urgently needed. Clinically, it is important to identify who may be susceptible to infection and identify treatments for the disease. There is good evidence that ACE2 is a receptor for 2019-nCoV, and studies also suggested that high expression of ACE2 may increase susceptibility to infection. Here we conducted a phenome-wide Mendelian randomization (MR) study to prioritize diseases/traits and blood proteins that may be causally linked to ACE2 expression in the lung. Expression data was based on GTEx. We also explored drug candidates whose targets overlapped with the top-ranked proteins in MR analysis, as these drugs could potentially alter ACE2 expression and may be clinically relevant. Notably, MR is much less vulnerable to confounding and reverse causality compared to observational studies. The most consistent finding was a tentative causal association between diabetes-related traits and increased ACE2 expression. Based on one of the largest GWAS on type II diabetes (T2DM) to date (N=898,130), we found that T2DM is causally linked to raised ACE2 expression (beta=0.1835, 95% CI 0.0853-0.2817; p=2.49E-4; GSMR method). Significant associations (at nominal level; p&lt;0.05) was also observed across multiple datasets, with different analytic methods, and for both type I and II diabetes. Other diseases/traits having nominal significant associations with increased ACE2 included inflammatory bowel disease, (ER+) breast and lung cancers, asthma, smoking and elevated ALT, among others. We also uncovered a number of plasma/serum proteins potentially linked to altered ACE2 expression, and the top enriched pathways included cytokine-cytokine-receptor interaction, VEGF signaling, JAK-STAT signaling etc. We also explored drugs that target some of the top-ranked proteins in the MR analysis. In conclusion, the current MR analysis reveals diseases/traits and blood proteins that may causally affect ACE2 expression, which in turn may influence susceptibility to the infection. The proteome-wide MR analysis may shed light on the molecular mechanisms underlying ACE2 expression, and may help guide drug repositioning in the future. Nevertheless, we stress that further studies are required to verify our findings due to various limitations and the exploratory nature of some analyses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Shitao Rao; Alexandria Lau; Hon-Cheong So&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031237&#39;},
 {&#39;cord_uid&#39;: &#39;ufcvecwo&#39;,
  &#39;sha&#39;: &#39;374a5295716649a80dca87097ad9ac902d344db6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Outcome reporting from protocols of clinical trials of Coronavirus Disease 2019 (COVID-19): a review&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.20031401&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives: To examine heterogeneity of outcomes in protocols of clinical trials of Coronavirus Disease 2019 (COVID-19) and to identify outcomes for prioritization in developing a core outcome set (COS) in this field. Design: This study is a review. Data sources: Databases of ICMJE-accepted clinical trial registry platform were searched on February 14, 2020. Eligibility Criteria: Randomized controlled trials (RCTs) and non-RCTs of COVID-19 were considered.Conditions of patients include common type, severe type or critical type. Interventions include traditional Chinese medicine (TCM) and Western medicine. We excluded trials that for discharged patients, psychological intervention and complications of COVID-19. Data extraction and synthesis: The general information and outcomes, outcome measurement instruments and measurement times were extracted. The results were analysed by descriptive analysis. Results: 19 registry platforms were searched. A total of 97 protocols were included from 160 protocols. For protocols of TCM clinical trials, 76 outcomes from 16 outcome domains were reported, and almost half (34/76, 44.74%) of outcomes were reported only once; the most frequently reported outcome was time of SARS-CoV-2 RNA turns to negative. 27 (27/76, 35.53%) outcomes were provided one or more outcome measurement instruments. 10 outcomes were provided one or more measurement time frame. For protocols of western medicine clinical trials, 126 outcomes from 17 outcome domains were reported; almost half (62/126, 49.21%) of outcomes were reported only once; the most frequently reported outcome was proportion of patients with negative SARS-CoV-2. 27 outcomes were provided one or more outcome measurement instruments. 40 (40/126, 31.75%) outcomes were provided one or more measurement time frame. Conclusion: Outcome reporting in protocols of clinical trials of COVID-19 is inconsistent. Thus, developing a core outcome set is necessary. Keywords: Outcomes; clinical trials, COVID-19; review.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Ruijin Qiu; Xuxu Wei; Mengzhu Zhao; Changming Zhong; Chen Zhao; Jiayuan Hu; Min Li; Ya Huang; Songjie Han; Tianmai He; Jing Chen; Hongcai Shang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.20031401&#39;},
 {&#39;cord_uid&#39;: &#39;4ihv80au&#39;,
  &#39;sha&#39;: &#39;7bdceb3b947ec934199d0c97fb9ec5a0e2bac776&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.975995&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-04&#39;,
  &#39;authors&#39;: &#39;Yonghua Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.975995&#39;},
 {&#39;cord_uid&#39;: &#39;wynyrumi&#39;,
  &#39;sha&#39;: &#39;6726b273f345838661eaa39e8adb5df4e1815a9a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;In silico study of the spike protein from SARS-CoV-2 interaction with ACE2: similarity with SARS-CoV, hot-spot analysis and effect of the receptor polymorphism&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.976027&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe spread of the COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor via which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. We used homology-based protein-protein docking, binding energy estimation, and decomposition, aiming to predict both qualitative and quantitative aspects of the interaction. Using in silico structural modelling, we brought additional evidence that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than mutation accumulation. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to hACE2 and maintaining the stability of the interface. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Houcemeddine Othman; Zied Bouslama; Jean-Tristan Brandenburg; Jorge da Rocha; Yosr Hamdi; Kais Ghedira; Najet-Srairi Abid; Scott Hazelhurst&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.976027&#39;},
 {&#39;cord_uid&#39;: &#39;erdity4m&#39;,
  &#39;sha&#39;: &#39;5cd48b8ab4ef66ca582df97a293d22a5088fa568&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Cryo-electron microscopy structure of the SADS-CoV spike glycoprotein provides insights into an evolution of unique coronavirus spike proteins&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.976258&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe current outbreak of Coronavirus Disease 2019 (COVID-19) by a novel betacoronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has aroused great public health concern. Coronavirus has a history of causing epidemics in human and animals. In 2017 an outbreak in piglets by a novel coronavirus was emerged designated as swine acute diarrhea syndrome coronavirus (SADS-CoV) which is originated from the same genus of horseshoe bats (Rhinolophus) as Severe Acute Respiratory Syndrome CoV (SARS-CoV) having a broad species tropism. In addition to human cells, it can also infect cell lines from diverse species. Coronavirus host range is determined by its spike glycoprotein (S). Given the importance of S protein in viral entry to cells and host immune responses, here we report the cryo-EM structure of the SADS-CoV S in the prefusion conformation at a resolution of 3.55 Å. Our study reveals that SADS-CoV S structure takes an intra-subunit quaternary packing mode where the NTD and CTD from the same subunit pack together by facing each other. The comparison of NTD and CTD with that of the other four genera suggests the evolutionary process of the SADS-CoV S. Moreover, SADS-CoV S has several characteristic structural features, such as more compact architecture of S trimer, and masking of epitopes by glycan shielding, which may facilitate viral immune evasion. These data provide new insights into the evolutionary relationships of SADS-CoV S and would extend our understanding of structural and functional diversity, which will facilitate to vaccine development.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Hongxin Guan; Youwang Wang; Abdullah F.U.H. Saeed; Jinyu Li; Syed Sajid Jan; Vanja Perčulija; Yu Li; Ping Zhu; Songying Ouyang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.976258&#39;},
 {&#39;cord_uid&#39;: &#39;fnj57b0l&#39;,
  &#39;sha&#39;: &#39;dfb2f07149617b0c2911899ac61563e808e2cee1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genome-wide data inferring the evolution and population demography of the novel pneumonia coronavirus (SARS-CoV-2)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.976662&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSince December 2019, coronavirus disease 2019 (COVID-19) emerged in Wuhan, Central China and rapidly spread throughout China. Up to March 3, 2020, SARS-CoV-2 has infected more than 89,000 people in China and other 66 countries across six continents. In this study, we used 10 new sequenced genomes of SARS-CoV-2 and combined 136 genomes from GISAID database to investigate the genetic variation and population demography through different analysis approaches (e.g. Network, EBSP, Mismatch, and neutrality tests). The results showed that 80 haplotypes had 183 substitution sites, including 27 parsimony-informative and 156 singletons. Sliding window analyses of genetic diversity suggested a certain mutations abundance in the genomes of SARS-CoV-2, which may be explaining the existing widespread and high adaptation of the deadly virus. Phylogenetic analysis showed that the view, pangolin acted as an intermediate host, may be controversial. The network indicated that, in the original haplotype (H14), one patient sample lived near the Huanan seafood market (approximate 2 km), indicating high possibility of the patient having a history of unconscious contact with this market. However, based on this clue, we cannot accurately concluded that whether this market was the origin center of SARS-CoV-2. Additionally, 16 genomes, collected from this market, assigned to 10 haplotypes, indicated a circulating infection within the market in a short term and then leading to the outbreak of SARS-CoV-2 in Wuhan and other areas. The EBSP results showed that the first estimated expansion date of SARS-CoV-2 began from 7 December 2019, which may indicated that the transmission could have begun from person to person in mid to late November.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Bin Fang; Linlin Liu; Xiao Yu; Xiang Li; Guojun Ye; Juan Xu; Ling Zhang; Faxian Zhan; Guiming Liu; Tao Pan; Yilin Shu; Yongzhong Jiang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.976662&#39;},
 {&#39;cord_uid&#39;: &#39;h5ire02h&#39;,
  &#39;sha&#39;: &#39;e350e1b4aab99c73d0e0d03121b50c059c04fd61&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Novel Immunoglobulin Domain Proteins Provide Insights into Evolution and Pathogenesis Mechanisms of SARS-Related Coronaviruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.04.977736&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTA novel coronavirus (SARS-CoV-2) is the causative agent of an emergent severe respiratory disease (COVID-19) in humans that is threatening to result in a global health crisis. By using genomic, sequence, structural and evolutionary analysis, we show that Alpha- and Beta-CoVs possess several novel families of immunoglobulin (Ig) domain proteins, including ORF8 and ORF7a from SARS-related coronaviruses and two protein groups from certain Alpha-CoVs. Among them, ORF8 is distinguished in being rapidly evolving, possessing a unique insert and a hypervariable position among SARS-CoV-2 genomes in its predicted ligand-binding groove. We also uncover many Ig proteins from several metazoan viruses which are distinct in sequence and structure but share an architecture comparable to that of CoV Ig domain proteins. Hence, we propose that deployment of Ig domain proteins is a widely-used strategy by viruses, and SARS-CoV-2 ORF8 is a potential pathogenicity factor which evolves rapidly to counter the immune response and facilitate the transmission between hosts.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Yongjun Tan; Theresa Schneider; Matthew Leong; L Aravind; Dapeng Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.04.977736&#39;},
 {&#39;cord_uid&#39;: &#39;gunn55f9&#39;,
  &#39;sha&#39;: &#39;9bbfd3d34ee18ea1b9f4669331a6cee9c5992893&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20030502&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Roman Woelfel; Victor Max Corman; Wolfgang Guggemos; Michael Seilmaier; Sabine Zange; Marcel A Mueller; Daniela Niemeyer; Patrick Vollmar; Camilla Rothe; Michael Hoelscher; Tobias Bleicker; Sebastian Bruenink; Julia Schneider; Rosina Ehmann; Katrin Zwirglmaier; Christian Drosten; Clemens Wendtner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20030502&#39;},
 {&#39;cord_uid&#39;: &#39;e2p46wa8&#39;,
  &#39;sha&#39;: &#39;c8c052d27aaf8015316dcd2644fa5e0b3870cea1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031088&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Individual quarantine and active monitoring of contacts are core disease control strategies, particularly for emerging infectious diseases such as Coronavirus Disease 2019 (COVID-19). To estimate the comparative efficacy of these interventions to control COVID-19, we fit a stochastic branching model, comparing two sets of reported parameters for the dynamics of the disease. Our results suggest that individual quarantine may contain an outbreak of COVID-19 with a short serial interval (4.8 days) only in settings with high intervention performance where at least three-quarters of infected contacts are individually quarantined. However, in settings where this performance is unrealistically high and the outbreak of COVID-19 continues to grow, so too will the burden of the number of contacts traced for active monitoring or quarantine. In such circumstances where resources are prioritized for scalable interventions such as social distancing, we show active monitoring or individual quarantine of high-risk contacts can contribute synergistically to social distancing. To the extent that interventions based on contact tracing can be implemented, therefore, they can help mitigate the spread of COVID-19. Our model highlights the urgent need for more data on the serial interval and the extent of presymptomatic transmission in order to make data-driven policy decisions regarding the cost-benefit comparisons of individual quarantine vs. active monitoring of contacts.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Corey M Peak; Rebecca Kahn; Yonatan H Grad; Lauren M Childs; Ruoran Li; Marc Lipsitch; Caroline O Buckee&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031088&#39;},
 {&#39;cord_uid&#39;: &#39;nrnc8u28&#39;,
  &#39;sha&#39;: &#39;a7ac40c0f2083c733cc714e563b21c2d606818c0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Appealing for Efficient, Well Organized Clinical Trials on COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031476&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The rapid emergence of clinical trials on COVID-19 stimulated a wave of discussion in scientific community. We reviewed the characteristics of interventional trials from Chinese Clinical Trial Registration (ChiCTR) and ClinicalTrials.gov. A total of 171 COVID-19-related interventional trials were identified on Feb 22nd, 2020. These trials are classified into 4 categories based on treatment modalities, including chemical drugs, biological therapies, traditional Chinese medicine treatments and other therapies. Our analysis focused on the issues of stage, design, randomization, blinding, primary endpoints definition and sample size of these trials. We found some studies with potential defects including unreasonable design, inappropriate primary endpoint definition, insufficient sample size and ethical issue. Clinical trials on COVID-19 should be designed based on scientific rules, ethics and benefits for patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Yang Zhao; Yongyue Wei; Sipeng Shen; Mingzhi Zhang; Feng Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031476&#39;},
 {&#39;cord_uid&#39;: &#39;7wtsw1cf&#39;,
  &#39;sha&#39;: &#39;f49a9b36227a498495c9788a4e5e4d6429b71eab&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characterization and chest CT findings in laboratory-confirmed COVID-19: a systematic review and meta-analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031518&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Imagery techniques have been used as essential parts of diagnostic workup for patients suspected for 2019-nCoV infection, Multiple studies have reported the features of chest computed tomography (CT) scans among a number of 2019-nCoV patients. Method: Study Identification was carried out in databases (PubMed, Embase and Cochrane Library) to identify published studies examining the diagnosis, the 2019 novel coronavirus (2019-nCoV). Heterogeneity among reported prevalence was assessed by computing p-values of Cochrane Q-test and I2-statics. The pooled prevalence of treatment failure was carried out with a fixed effects meta-analysis model, generating the pooled 95% confidence interval. A random-effect model was used to pool the results since this model could incorporate the heterogeneity of the studies and therefore proved a more generalized result. Results: According to the combined results of meta-analysis, the total 55% of corona patients were males. The mean age of the patients was 41.31 (34.14, 48.47). Two prevalent clinical symptoms between patients were fever, cough with prevalence of 85%, and 62%, respectively. Either Ground Glass Opacity GGO or consolidation was seen in 86% but 14% had NO GGO or consolidation. The other rare CT symptoms were pericardial effusion, and pleural effusion with 4, 5, 7% prevalence, respectively. The most prevalent event was Either GGO or consolidation in 85% of patients. Conclusion: The most CT-scan abnormality is Either Ground Glass Opacity GGO or consolidation however in few patients none of them might be observed, so trusting in just CT findings will lead to miss some patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Golnaz Vaseghi; Marjan Mansourian; Raheleh Karimi; Kiyan Heshmat-Ghahdarijani; Sadegh Baradaran Mahdavi; Amirhossein Pezeshki; Behrooz Ataei; Alireza Zandifar; Omid Shafaat; Shaghayegh Haghjoo Javanmard&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031518&#39;},
 {&#39;cord_uid&#39;: &#39;jg6v644y&#39;,
  &#39;sha&#39;: &#39;114ed64f52f503d9d2e2ba1fd1ee62b0a168cd84&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Acute Myocardial Injury of Patients with Coronavirus Disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031591&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since the outbreak of the Coronavirus Disease 2019 (COVID-19) in China, respiratory manifestations of the disease have been observed. However, as a fatal comorbidity, acute myocardial injury (AMI) in COVID-19 patients has not been previously investigated in detail. We investigated the clinical characteristics of COVID-19 patients with AMI and determined the risk factors for AMI in them. Methods: We analyzed data from 53 consecutive laboratory-confirmed and hospitalized COVID-19 patients (28 men, 25 women; age, 19-81 years). We collected information on epidemiological and demographic characteristics, clinical features, routine laboratory tests (including cardiac injury biomarkers), echocardiography, electrocardiography, imaging findings, management methods, and clinical outcomes. Results: Cardiac complications were found in 42 of the 53 (79.25%) patients: tachycardia (n=15), electrocardiography abnormities (n=11), diastolic dysfunction (n=20), elevated myocardial enzymes (n=30), and AMI (n=6). All the six AMI patients were aged &gt;60 years; five of them had two or more underlying comorbidities (hypertension, diabetes, cardiovascular diseases, and chronic obstructive pulmonary disease). Novel coronavirus pneumonia (NCP) severity was higher in the AMI patients than in patients with non-definite AMI (p&lt;0.001). All the AMI patients required care in intensive care unit; of them, three died, two remain hospitalized. Multivariate analyses showed that C-reactive protein (CRP) levels, NCP severity, and underlying comorbidities were the risk factors for cardiac abnormalities in COVID-19 patients. Conclusions: Cardiac complications are common in COVID-19 patients. Elevated CRP levels, underlying comorbidities, and NCP severity are the main risk factors for cardiac complications in COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Huayan Xu; Keke Hou; Hong Xu; Zhenlin Li; Huizhu Chen; Na Zhang; Rong Xu; Hang Fu; Ran Sun; Lingyi Wen; Linjun Xie; Hui Liu; Kun Zhang; Joseph B Selvanayagam; Chuan Fu; Shihua Zhao; Zhigang Yang; Ming Yang; Yingkun Guo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031591&#39;},
 {&#39;cord_uid&#39;: &#39;k13cchxn&#39;,
  &#39;sha&#39;: &#39;0b282573f5c63c943021c10ca39a1ed21acfb429&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A data-driven assessment of early travel restrictions related to the spreading of the novel COVID-19 within mainland China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031740&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Two months after it was firstly reported, the novel coronavirus disease COVID-19 has already spread worldwide. However, the vast majority of reported infections have occurred in China. To assess the effect of early travel restrictions adopted by the health authorities in China, we have implemented an epidemic metapopulation model that is fed with mobility data corresponding to 2019 and 2020. This allows to compare two radically different scenarios, one with no travel restrictions and another in which mobility is reduced by a travel ban. Our findings indicate that i) travel restrictions are an effective measure in the short term, however, ii) they are ineffective when it comes to completely eliminate the disease. The latter is due to the impossibility of removing the risk of seeding the disease to other regions. Our study also highlights the importance of developing more realistic models of behavioral changes when a disease outbreak is unfolding.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Alberto Aleta; Qitong Hu; Jiachen Ye; Peng Ji; Yamir Moreno&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031740&#39;},
 {&#39;cord_uid&#39;: &#39;hwqgotl2&#39;,
  &#39;sha&#39;: &#39;a34d2c1876ffedc4f95f3af13ac06039cd38a5b0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the infection and case fatality ratio for COVID-19 using age-adjusted data from the outbreak on the Diamond Princess cruise ship&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031773&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Adjusting for delay from confirmation-to-death, we estimated case and infection fatality ratios (CFR, IFR) for COVID-19 on the Diamond Princess ship as 2.3% (0.75%-5.3%) and 1.2% (0.38-2.7%). Comparing deaths onboard with expected deaths based on naive CFR estimates using China data, we estimate IFR and CFR in China to be 0.5% (95% CI: 0.2-1.2%) and 1.1% (95% CI: 0.3-2.4%) respectively.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Timothy W Russell; Joel Hellewell; Christopher I Jarvis; Kevin van-Zandvoort; Sam Abbott; Ruwan Ratnayake; Stefan Flasche; Rosalind M Eggo; Adam J Kucharski&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031773&#39;},
 {&#39;cord_uid&#39;: &#39;cq6mivr9&#39;,
  &#39;sha&#39;: &#39;af266fac8970a7960e96630a67d91bec5dda0335&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the generation interval for COVID-19 based on symptom onset data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031815&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies. Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided. Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers. Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32-67%) for Singapore and 62% (95%CI 50-76%) for Tianjin, China. Estimates of the reproduction number based on the generation interval distribution were slightly higher than those based on the serial interval distribution. Conclusions: Estimating generation and serial interval distributions from outbreak data requires careful investigation of the underlying transmission network. Detailed contact tracing information is essential for correctly estimating these quantities.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Tapiwa Ganyani; Cecile Kremer; Dongxuan Chen; Andrea Torneri; Christel Faes; Jacco Wallinga; Niel Hens&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031815&#39;},
 {&#39;cord_uid&#39;: &#39;royt8bo6&#39;,
  &#39;sha&#39;: &#39;a0ecc9b46addbc1faab491434b5331702f7883b8&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A mathematical model for estimating the age-specific transmissibility of a novel coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031849&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: A novel coronavirus named as &#34;SARS-CoV-2&#34; has spread widely in many countries since December 2019, especially in China. This study aimed to quantify the age-specific transmissibility by using a mathematical model. Methods: An age-specific susceptible - exposed - symptomatic - asymptomatic - recovered - seafood market (SEIARW) model was developed based on two suspected transmission routes (from market to person and person to person). The susceptible people from Wuhan City were divided into different age groups. We used the subscript i and j to represent age group 1 to 4 (1: &lt;= 14 years; 2: 15-44 years; 3: 45-64 years; 4: &gt;= 65 years) and 1 to 5 (1: &lt;= 5 years; 2: 6-14 years; 3: 15-24 years; 4: 25-59 years; 4: &gt;= 60 years), respectively. Data of reported COVID-19 cases were collected from one published literature from 26 November to 22 December, 2019 in Wuhan City, China. The age-specific transmissibility of the virus was estimated accordingly secondary attack rate (SAR). Results: The age-specific SEIARW model fitted with the reported data well by dividing the population into four age groups (χ2 = 4.99 × 10-6, P &gt; 0.999), and five age groups (χ2 = 4.85 × 10-6, P &gt; 0.999). Based on the four-age-group SEIARW model, the highest transmissibility occurred from age group 2 to 3 (SAR23 = 17.56 per 10 million persons), followed by from age group 3 to 2 (SAR32 = 10.17 per 10 million persons). The lowest transmissibility occurred from age group 1 to 2 (SAR12 = 0.002 per 10 million persons). Based on the five-age-group SEIARW model, the highest transmissibility occurred from age group 4 to 5 (SAR45 = 12.40 per 10 million persons), followed by from age group 5 to 4 (SAR54 = 6.61 per 10 million persons). The lowest transmissibility occurred from age group 3 to 4 (SAR34 = 0.0002 per 10 million persons). Conclusions: SARS-CoV-2 has high transmissibility among adults and elder people but low transmissibility among children and young people.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Zeyu Zhao; Yuan-Zhao Zhu; Jing-Wen Xu; Qing-Qing Hu; Zhao Lei; Jia Rui; Xingchun Liu; Yao Wang; Li Luo; Shan-Shan Yu; Jia Li; Ruo-Yun Liu; Fang Xie; Ying-Ying Su; Yi-Chen Chiang; Yanhua Su; Benhua Zhao; Tianmu Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031849&#39;},
 {&#39;cord_uid&#39;: &#39;qz2joxys&#39;,
  &#39;sha&#39;: &#39;b4edd81e0d2770b0d6e0e3c93c7fffc43e3caa80&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Role of temperature and humidity in the modulation of the doubling time of COVID-19 cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031872&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 is having a great impact on public health, mortality and economy worldwide, in spite of the efforts to prevent its epidemy. The SARS-CoV-2 genome is different from that of MERS-CoV and SARS-CoV, although also expected to spread differently according to meteorological conditions. Our main goal is to investigate the role of some meteorological variables on the expansion of this outbreak. In this study, an exponential model relating the number of accumulated confirmed cases and time was considered. The rate of COVID-19 spread, using as criterion the doubling time of the number of confirmed cases, was used as dependent variable in a linear model that took four independent meteorological variables: temperature, humidity, precipitation and wind speed. Only China cases were considered, to control both cultural aspects and containment policies. Confirmed cases and the 4 meteorological variables were gathered between January 23 and March 1 (39 days) for the 31 provinces of Mainland China. Several periods of time were sampled for each province, obtaining more than one value for the rate of disease progression. Two different periods of time were tested, of 12 and 15 days, along with 3 and 5 different starting points in time, randomly chosen. The median value for each meteorological variable was computed, using the same time period; models with adjusted R square above 0.75 were selected. The rate of progression and doubling time were computed and used to fit a linear regression model. Models were evaluated using alpha=0.05. Results indicate that the doubling time correlates positively with temperature and inversely with humidity, suggesting that a decrease in the rate of progression of COVID-19 with the arrival of spring and summer in the north hemisphere. A 20oC increase is expected to delay the doubling time in 1.8 days. Those variables explain 18% of the variation in disease doubling time; the remaining 82% may be related to containment measures, general health policies, population density, transportation or cultural aspects.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Barbara Oliveiros; Liliana Caramelo; Nuno C Ferreira; Francisco Caramelo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031872&#39;},
 {&#39;cord_uid&#39;: &#39;yj2jh694&#39;,
  &#39;sha&#39;: &#39;a42ff48d50aa3a0ebfe840d46ed2204c49955442&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Emotional responses and coping strategies of nurses and nursing college students during COVID-19 outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031898&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Affected by a Corona Virus Disease 2019 (COVID-19) outbreak, Since December 2019, there have been more than 76,000 cases of COVID-19 in China, causing more than 3,000 medical staff infections. Due to COVID-19 spreads quickly, is highly contagious, and can be fatal in severe cases, and there are no specific medicines, it poses a huge threat to the life and health of nurses and has a large impact on their emotional responses and coping strategies. Methods: This study conducted an online questionnaire survey from February 1 to 9, 2020 to investigate the current state of emotional responses and coping strategies of nurses and college nursing students in Anhui Province. This study used a modified Brief COPE (Carver, 1997) and a emotional responses scale. Results: The results found that women showed more severe anxiety and fear than men. Participants from cities showed more anxiety and fear than participants from rural, but rural participants showed more sadness than urban participants. The closer COVID-19 is to the participants, the stronger the anxiety and anger. Compared with Nursing college students, nurses have stronger emotional responses and are more willing to use Problem-focused coping. People may have a cycle of &#34;the more fear, the more problem-focused coping&#34;. And people may &#34;The more angry, the more emotion-focused coping&#34;, &#34;the more problem-focused coping, the more anxious, the more angry, the more sadness&#34;. Conclusion: COVID-19 is a pressure source with great influence, both for individuals and for the social public groups. Different individuals and groups may experience different levels of psychological crisis, and those nurses at the core of the incident are affected. Hospitals should focus on providing psychological support to nurses and providing timely psychological assistance and training in coping strategies. Improving nurses\&#39; ability to regulate emotions and effective coping strategies, providing a strong guarantee for resolutely winning the battle against epidemic prevention and control.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Long Huang; Fu ming xu; Hai rong Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031898&#39;},
 {&#39;cord_uid&#39;: &#39;fcvstps9&#39;,
  &#39;sha&#39;: &#39;5129bb31949c66ebd544658c531004046d4102da&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031906&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND Corona Virus Disease 2019 (COVID-19) is spreading worldwide. Effective screening for patients is important to limit the epidemic. However, some defects make the currently applied diagnosis methods are still not very ideal for early warning of patients. We aimed to develop a diagnostic model that allows for the quick screening of highly suspected patients using easy-to-get variables. METHODS A total of 1,311 patients receiving severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) nucleicacid detection were included, whom with a positive result were classified into COVID-19 group. Multivariate logistic regression analyses were performed to construct the diagnostic model. Receiver operating characteristic (ROC) curve analysis were used for model validation. RESULTS After analysis, signs of pneumonia on CT, history of close contact, fever, neutrophil-to-lymphocyte ratio (NLR), Tmax and sex were included in the diagnostic model. Age and meaningful respiratory symptoms were enrolled into COVID-19 early warning score (COVID-19 EWS). The areas under the ROC curve (AUROC) indicated that both of the diagnostic model (training dataset 0.956 [95%CI 0.935-0.977, P &lt; 0.001]; validation dataset 0.960 [95%CI 0.919-1.0, P &lt; 0.001] ) and COVID-19 EWS (training dataset 0.956 [95%CI 0.934-0.978, P &lt; 0.001] ; validate dataset 0.966 [95%CI 0.929-1, P &lt; 0.001]) had good discrimination capacity. In addition, we also obtained the cut-off values of disease severity predictors, such as CT score, CD8+ T cell count, CD4+ T cell count, and so on. CONCLUSIONS The new developed COVID-19 EWS was a considerable tool for early and relatively accurately warning of SARS-CoV-2 infected patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Cong-Ying Song; Jia Xu; Jian-Qin He; Yuan-Qiang Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031906&#39;},
 {&#39;cord_uid&#39;: &#39;1uqjmeic&#39;,
  &#39;sha&#39;: &#39;e05c39ce913edc81f93c9247f9714d6cddd30073&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Sensitive one-step isothermal detection of pathogen-derived RNAs&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20031971&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The recent outbreaks of Ebola, Zika, MERS, and SARS-CoV-2 (2019-nCoV) require fast, simple, and sensitive onsite nucleic acid diagnostics that can be developed rapidly to prevent the spread of diseases. We have developed a SENsitive Splint-based one-step isothermal RNA detection (SENSR) method for rapid and straightforward onsite detection of pathogen RNAs with high sensitivity and specificity. SENSR consists of two simple enzymatic reactions: a ligation reaction by SplintR ligase and subsequent transcription by T7 RNA polymerase. The resulting transcript forms an RNA aptamer that induces fluorescence. Here, we demonstrate that SENSR is an effective and highly sensitive method for the detection of the current epidemic pathogen, severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2). We also show that the platform can be extended to the detection of five other pathogens. Overall, SENSR is a molecular diagnostic method that can be developed rapidly for onsite uses requiring high sensitivity, specificity, and short assaying times.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-09&#39;,
  &#39;authors&#39;: &#39;Chang Ha Woo; Sungho Jang; Giyoung Shin; Gyoo Yeol Jung; Jeong Wook Lee&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20031971&#39;},
 {&#39;cord_uid&#39;: &#39;tvm1rf3d&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The prevalence and influencing factors for anxiety in medical workers fighting COVID-19 in China: A cross-sectional survey&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20032003&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract: Background: The COVID-19 outbreak caused by the SARS-Cov-2 virus has been sustained in China since December 2019, and could become a pandemic if we do not contain it. The mental health of frontline medical staff is a concern. In this study, we aimed to identify the influencing factors on medical worker anxiety in China during the COVID-19 outbreak. Methods: We conducted a cross-sectional study to estimate the prevalence of anxiety among medical staff from 10th February 2020 to 20th February 2020 in China using the Zung Self-rating Anxiety Scale (SAS) to assess anxiety, using the criteria of normal (≤49), mild (50-59), moderate (60-70) and severe anxiety (≥70). We used multivariable linear regression to determine the factors (e.g., having direct contact treating infected patients, being a medical staff worker from Hubei province, being a suspect case) for anxiety. We also used adjusted models to confirm independent factors for anxiety after adjusting for gender, age, education and marital status. Results: Of 512 medical staff from China, 164 healthcare workers (32.03%) had had direct contact by treating infected patients. The prevalence of anxiety was 12.5%, with 53 workers suffering from mild (10.35%), seven workers from moderate (1.36%) and four workers from severe anxiety (0.78%). After adjusting for sociodemographic characteristics (gender, age, education and marital status), medical staff who had had direct contact treating infected patients saw higher anxiety scores than those who had not had direct contact (βvalue=2.33, CI: 0.65 -4.00; p=0.0068). Similar things were observed in medical staff from Hubei province, compared with those from other parts of China (βvalue=3.67, CI: 1.44 -5.89; p=0.0013). The most important variable was suspect cases with high anxiety scores, compared to non-suspect cases (βvalue=4.44, CI: 1.55 -7.33; p=0.0028). Conclusion: Our results highlight that government authorities should make early detection of the high risk of anxiety among medical staff a priority, and implement appropriate psychological intervention programs, to prevent medical staff from developing psychological disorders that could potentially exert an adverse effect on combating the COVID-19 epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Chenyun Liu; Yun-zhi Yang; Xiao Ming Zhang; Xinying Xu; Qing-Li Dou; Wen-Wu Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20032003&#39;},
 {&#39;cord_uid&#39;: &#39;nt5sprdh&#39;,
  &#39;sha&#39;: &#39;93d8f1552a81587e2f88e38580cbe4753b9934f1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.20032011&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Shona C Moore; Rebekah Penrice-Randal; Muhannad Alruwaili; Xiaofeng Dong; Steven T Pullan; Daniel Carter; Kevin Bewley; Qin Zhao; Yani Sun; Catherine Hartley; En-min Zhou; Tom Solomon; Michael B. J. Beadsworth; James Cruise; Debby Bogaert; Derrick W T Crook; David A Matthews; Andrew D. Davidson; Zana Mahmood; Waleed Aljabr; Julian Druce; Richard T Vipond; Lisa Ng; Laurent Renia; Peter Openshaw; J Kenneth Baillie; Miles W Carroll; Calum Semple; Lance Turtle; Julian Alexander Hiscox&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.20032011&#39;},
 {&#39;cord_uid&#39;: &#39;l72x4wn3&#39;,
  &#39;sha&#39;: &#39;278e2c52630f1dbbda6a79db15a2260510586150&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;PhyloFold: Precise and Swift Prediction of RNA Secondary Structures to Incorporate Phylogeny among Homologs&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.975797&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMotivationThe simultaneous consideration of sequence alignment and RNA secondary structure, or structural alignment, is known to help predict more accurate secondary structures of homologs. However, the consideration is heavy and can be done only roughly to decompose structural alignments.ResultsThe PhyloFold method, which predicts secondary structures of homologs considering likely pairwise structural alignments, was developed in this study. The method shows the best prediction accuracy while demanding comparable running time compared to conventional methods.AvailabilityThe source code of the programs implemented in this study is available on “https://github.com/heartsh/phylofold” and “https://github.com/heartsh/phyloalifold“.Contact“tagashira_masaki_17@stu-cbms.k.u-tokyo.ac.jp”.Supplementary informationSupplementary data are available.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-06&#39;,
  &#39;authors&#39;: &#39;Masaki Tagashira&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.975797&#39;},
 {&#39;cord_uid&#39;: &#39;x3hr1y6o&#39;,
  &#39;sha&#39;: &#39;fd14ed7c073b7ff03afa517e9c0fd1e849878252&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Direct RNA sequencing and early evolution of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.976167&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;George Taiaroa; Daniel Rawlinson; Leo Featherstone; Miranda Pitt; Leon Caly; Julian Druce; Damian Purcell; Leigh Harty; Thomas Tran; Jason Roberts; Mike Catton; Deborah Williamson; Lachlan Coin; Sebastian Duchene&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.976167&#39;},
 {&#39;cord_uid&#39;: &#39;plh92ifi&#39;,
  &#39;sha&#39;: &#39;d23c6a066a58dbe5ec5fa57e67f1b795337299f7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;LY6E impairs coronavirus fusion and confers immune control of viral disease&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.05.979260&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTZoonotic coronaviruses (CoVs) are significant threats to global health, as exemplified by the recent emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)1. Host immune responses to CoV are complex and regulated in part through antiviral interferons. However, the interferon-stimulated gene products that inhibit CoV are not well characterized2. Here, we show that interferon-inducible lymphocyte antigen 6 complex, locus E (LY6E) potently restricts cellular infection by multiple CoVs, including SARS-CoV, SARS-CoV-2, and Middle East respiratory syndrome coronavirus (MERS-CoV). Mechanistic studies revealed that LY6E inhibits CoV entry into cells by interfering with spike protein-mediated membrane fusion. Importantly, mice lacking Ly6e in hematopoietic cells were highly susceptible to murine CoV infection. Exacerbated viral pathogenesis in Ly6e knockout mice was accompanied by loss of hepatic and splenic immune cells and reduction in global antiviral gene pathways. Accordingly, we found that Ly6e directly protects primary B cells and dendritic cells from murine CoV infection. Our results demonstrate that LY6E is a critical antiviral immune effector that controls CoV infection and pathogenesis. These findings advance our understanding of immune-mediated control of CoV in vitro and in vivo, knowledge that could help inform strategies to combat infection by emerging CoV.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Stephanie Pfaender; Katrina B. Mar; Eleftherios Michailidis; Annika Kratzel; Dagny Hirt; Philip V’kovski; Wenchun Fan; Nadine Ebert; Hanspeter Stalder; Hannah Kleine-Weber; Markus Hoffmann; H. Heinrich Hoffmann; Mohsan Saeed; Ronald Dijkman; Eike Steinmann; Mary Wight-Carter; Natasha W. Hanners; Stefan Pöhlmann; Tom Gallagher; Daniel Todt; Gert Zimmer; Charles M. Rice; John W. Schoggins; Volker Thiel&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.05.979260&#39;},
 {&#39;cord_uid&#39;: &#39;8874c8jp&#39;,
  &#39;sha&#39;: &#39;dd2cd004a197872cfce1c31a9685d6972f838efa&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Impact of Host-Based Early Warning on Disease Outbreaks&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20029793&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: The detection of communicable pathogens responsible for major outbreaks relies on health care professionals recognition of symptoms manifesting in infectious individuals. Early warning of such communicable diseases before the onset of symptoms could improve both patient care and public health responses. However, the potential impact of such a host-based early warning system on containing the spread of an outbreak and in steering public health response is unknown. Methods: We extend the deterministic SEIR (Susceptible, Exposed, Infectious, Recovered) model to simulate disease outbreak scenarios and to quantify the potential impact of a host-based early warning capability to mitigate pathogen transmission during an outbreak. In particular, we compare and contrast the performance of five different policies: Self-monitoring and reporting (baseline SEIR model), Quarantining the entire population, Quarantine-on-alert (with high sensitivity early warning), Quarantine-on-alert (with high specificity early warning), and Quarantine-on-alert (ideal early warning). We further evaluate these five policy options against four different outbreak scenarios with high or low disease transmission and high or low initial population exposures. Results: For all scenarios, a quarantine-on-alert policy coupled with the near-ideal early warning capability reduces quarantine needs with only a small increase in the number of additional infections. The cost of a highly specific early detection system (i.e., a reduction in false alarms and thus quarantine costs) is an increase in additional infections relative to the near-ideal system. Conversely, a highly sensitive early detection system increases the percentage of the population in quarantine compared to both the ideal and high-specificity early detection system while also reducing the number of additional infections to nearly the numbers seen by quarantining the entire population a priori. Conclusions: Our simulations demonstrate the utility of host-based early warning systems in controlling an outbreak under various outbreak conditions. Our tools also provide a simulation capability for evaluating public health policies enabling quantitative evaluation of their impacts prior to implementation.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Mark Hernandez; Lauren E Milechin; Shakti K Davis; Rich DeLaura; Kajal T Claypool; Albert Swiston&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20029793&#39;},
 {&#39;cord_uid&#39;: &#39;dtoheyo4&#39;,
  &#39;sha&#39;: &#39;8a73ab00fcfa1789da4d0755f67bc5da48846ace&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Evaluation of Sleep Disturbances for Chinese Frontline Medical Workers under the Outbreak of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20031278&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of coronavirus disease 2019 in China remains to be a serious challenge for frontline medical workers (fMW). They are under high risk of being infected and high mental stress, which may lead to sleep disturbances, anxiety, and depression. Methods We conducted a cross-sectional study to evaluate sleep disturbances of fMW and made a comparison with non-fMW. The medical workers from multiple hospitals in Hubei Province, China, were volunteered to participate. An online questionnaire, including Pittsburgh Sleep Quality Index (PSQI), Athens Insomnia Scale (AIS), and Visual Analogue Scale (VAS), was used to evaluate sleep disturbances and mental status of fMW. Sleep disturbances were defined as PSQI&gt;7 points or/and AIS&gt;6 points. We compared the scores of PSQI, AIS, anxiety and depression VAS, and prevalence of sleep disturbances between fMW and non-fMW. Subgroup analysis for different gender in fMW was conducted. Findings A total of 1306 subjects (including 801 fMW and 505 non-fMW) were enrolled. Compared to non-fMW, fMW had significantly higher scores of PSQI (p＜0.0001), AIS (p＜0.0001), anxiety (p＜0.0001) and depression (p=0.0010), and higher prevalence of sleep disturbances with PSQI &gt; 7 points (p＜0.0001) and AIS &gt; 6 points (p＜0.0001). In subgroup analysis, compared to male fMW, female fMW had significantly higher scores of PSQI (p=0.022) and higher prevalence of sleep disturbances with PSQI &gt; 7 points (p&lt;0.0001). Interpretation fMW had higher prevalence of sleep disturbances and worse sleep quality than non-fMW. Female fMW were more vulnerable to having sleep disturbances than male fMW. Funding None.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Jing Qi; Jing Xu; Bozhi Li; Jinsha Huang; Yuan Yang; Zhentao Zhang; Dongai Yao; Qunhui Liu; Min Jia; Daokai Gong; Xiaohong Ni; Qimei Zhang; Furong Shang; Nian Xiong; Chunli Zhu; Tao Wang; Xi Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20031278&#39;},
 {&#39;cord_uid&#39;: &#39;jarnavwl&#39;,
  &#39;sha&#39;: &#39;fd847a97e6fce134345b25ae9240c4ebb680cca5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics on 25 Discharged Patients with COVID-19 Virus Returning&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20031377&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Here we report the clinical features of 25 discharged patients with COVID-19 recovery. Our analysis indicated that there was a significant inverse correlation existed between serum D-Dimer level and the duration of antiviral treatment, while lymphocyte concentration significantly positively correlated with the duration of virus reversal.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Jing Yuan; Shanglong Kou; Yanhua Liang; Jianfeng Zeng; Yanchao Pan; Lei Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20031377&#39;},
 {&#39;cord_uid&#39;: &#39;qgvue8yw&#39;,
  &#39;sha&#39;: &#39;610549c879c62efc42f2ad32f63c65b25d8931c7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20031856&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of the recently emerged novel corona virus disease 2019 (COVID-19) poses a challenge for public health laboratories. We aimed to evaluate the diagnostic value of serological assay for SARS-CoV-2. Methods A newly-developed ELISA assay for IgM and IgG antibodies against N protein of SARS-CoV-2 were used to screen the serums of 238 admitted hospital patients with confirmed or suspected SARS-CoV-2 infection from February 6 to February 14, 2020. SARS-CoV-2 RNA was detected by real time RT-PCR on pharyngeal swab specimens. Findings Of the 238 patients, 194 (81.5%) were detected to be antibody (IgM and/or IgG) positive, which was significantly higher than the positive rate of viral RNA (64.3%). There was no difference in the positive rate of antibody between the confirmed patients (83.0%, 127/153) and the suspected patients (78.8%, 67/85) whose nucleic acid tests were negative. After the patients were defined to the different stages of disease based on the day when the test samples were collected, the analysis results showed that the antibody positive rates were very low in the first five days after initial onset of symptoms, and then rapidly increased as the disease progressed. After 10 days, the antibody positive rates jumped to above 80% from less than 50%. On the contrary, the positive rates of viral RNA kept above 60% in the first 11 days after initial onset of symptoms, and then rapidly decreased. In addition, half of the suspected patients with symptoms for 6-10 days were detected to be antibody positive. Interpretation The suspected patients were most likely infected by SARS-CoV-2. Before the 11th day after initial onset of symptoms, nucleic acid test is important for confirmation of viral infection. The combination of serological assay can greatly improve the diagnostic efficacy. After that, the diagnosis for viral infection should be majorly dependent on serological assay. Keywords. SARS-CoV-2; diagnosis; serological assay; nucleic acid test&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Lei Liu; Wanbing Liu; Shengdian Wang; Shangen Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20031856&#39;},
 {&#39;cord_uid&#39;: &#39;v6wt9mhh&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the scale of COVID-19 Epidemic in the United States: Simulations Based on Air Traffic directly from Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20031880&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: Coronavirus Disease 2019 (COVID-19) infection has been characterized by rapid spread and unusually large case clusters. It is important to have an estimate of the current state of COVID-19 epidemic in the U.S. to develop informed public health strategies. Methods: We estimated the potential scale of the COVID-19 epidemic (as of 03/01/2020) in the U.S. from cases imported directly from Wuhan area. We used simulations based on transmission dynamics parameters estimated from previous studies and air traffic data from Wuhan to the U.S and deliberately built our model based on conservative assumptions. Detection and quarantine of individual COVID-19 cases in the U.S before 03/01/2020 were also taken into account. We. A SEIR model was used to simulate the growth of the number of infected individuals in Wuhan area and in the U.S. Results: With the most likely model, we estimated that there would be 9,484 infected cases (90%CI 2,054-24,241) as of 03/01/2020 if no intervention procedure had been taken to reduce the transmissibility in unidentified cases. Assuming current preventive procedures have reduced 25% of the transmissibility in unidentified cases, the number of infected cases would be 1,043 (90%CI 107-2,474). Conclusion: Our research indicates that, as of 03/01/2020., it is likely that there are already thousands of individuals in the US infected with SARS-CoV-2. Our model is dynamic and is available to the research community to further evaluate as the situation becomes clearer.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Dalin Li; Jun Lv; Gregory Botwin; Jonathan Braun; Weihua Cao; Liming Li; Dermot P.B. McGovern&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20031880&#39;},
 {&#39;cord_uid&#39;: &#39;v3gww4iv&#39;,
  &#39;sha&#39;: &#39;fd96fc1859201e13e35526c188e022fa6e335327&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20031955&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The ongoing outbreak of novel corona virus disease 2019 (COVID-19) in Wuhan, China, is arousing international concern. This study evaluated whether and when the infected but asymptomatic cases during the incubation period could infect others. Methods: We collected data on demographic characteristics, exposure history, and symptom onset day of the confirmed cases, which had been announced by the Chinese local authorities. We evaluated the potential of transmission during the incubation period in 50 infection clusters, including 124 cases. All the secondary cases had a history of contact with their first-generation cases prior to symptom onset. Results: The estimated mean incubation period for COVID-19 was 4.9 days (95% confidence interval [CI], 4.4 to 5.4) days, ranging from 0.8 to 11.1 days (2.5th to 97.5th percentile). The observed mean and standard deviation (SD) of serial interval was 4.1±3.3 days, with the 2.5th and 97.5th percentiles at -1 and 13 days. The infectious curve showed that in 73.0% of the secondary cases, their date of getting infected was before symptom onset of the first-generation cases, particularly in the last three days of the incubation period. Conclusions: The results indicated the transmission of COVID-9 occurs among close contacts during the incubation period, which may lead to a quarantine loophole. Strong and effective countermeasures should be implemented to prevent or mitigate asymptomatic transmission during the incubation period in populations at high risk.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Wei Xia; Jiaqiang Liao; Chunhui Li; Yuanyuan Li; Xi Qian; Xiaojie Sun; Hongbo Xu; Gaga Mahai; Xin Zhao; Lisha Shi; Juan Liu; Ling Yu; Meng Wang; Qianqian Wang; Asmagvl Namat; Ying Li; Jingyu Qu; Qi Liu; Xiaofang Lin; Shuting Cao; Shu Huan; Jiying Xiao; Fengyu Ruan; Hanjin Wang; Qing Xu; Xingjuan Ding; Xingjie Fang; Feng Qiu; Jiaolong Ma; Yu Zhang; Aizhen Wang; Yuling Xing; Shunqing Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20031955&#39;},
 {&#39;cord_uid&#39;: &#39;zio8yhuy&#39;,
  &#39;sha&#39;: &#39;26e75d3c815aae7fd9b094c3e5c74d3f7132ca13&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;CORONAVIRUS IN PREGNANCY AND DELIVERY: RAPID REVIEW AND EXPERT CONSENSUS&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20032144&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND Person to person spread of COIVD-19 in the UK has now been confirmed. There are limited case series reporting the impact on women affected by coronaviruses (CoV) during pregnancy. In women affected by SARS and MERS, the case fatality rate appeared higher in women affected in pregnancy compared with non-pregnant women. We conducted a rapid, review to guide management of women affected by COVID -19 during pregnancy and developed interim practice guidance with the RCOG and RCPCH to inform maternity and neonatal service planning METHODS Searches were conducted in PubMed and MedRxiv to identify primary case reports, case series, observational studies or randomised-controlled trial describing women affected by coronavirus in pregnancy and on neonates. Data was extracted from relevant papers and the review was drafted with representatives of the RCPCH and RCOG who also provided expert consensus on areas where data were lacking RESULTS From 9964 results on PubMed and 600 on MedRxiv, 18 relevant studies (case reports and case series) were identified. There was inconsistent reporting of maternal, perinatal and neonatal outcomes across case reports and series concerning COVID-19, SARS, MERS and other coronaviruses. From reports of 19 women to date affected by COVID-19 in pregnancy, delivering 20 babies, 3 (16%) were asymptomatic, 1 (5%) was admitted to ICU and no maternal deaths have been reported. Deliveries were 17 by caesarean section, 2 by vaginal delivery, 8 (42%) delivered pre-term. There was one neonatal death, in 15 babies who were tested there was no evidence of vertical transmission. CONCLUSIONS Morbidity and mortality from COVID-19 appears less marked than for SARS and MERS, acknowledging the limited number of cases reported to date. Pre-term delivery affected 42% of women hospitalised with COVID-19, which may put considerable pressure on neonatal services if the UK reasonable worse-case scenario of 80% of the population affected is realised. There has been no evidence of vertical transmission to date. The RCOG and RCPCH have provided interim guidance to help maternity and neonatal services plan their response to COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#34;Edward Mullins; David Evans; Russell Viner; Patrick O&#39;Brien; Eddie Morris&#34;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20032144&#39;},
 {&#39;cord_uid&#39;: &#39;r9itwxcp&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluating the secondary transmission pattern and epidemic prediction of the COVID-19 in metropolitan areas of China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20032177&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Understanding the transmission dynamics of COVID-19 is crucial for evaluating the spread pattern of it, especially in metropolitan areas of China which may cause secondary outbreaks outside Wuhan, the center of the new coronavirus disease outbreak. We used reported data from Jan 24, 2020, to Feb 23, 2020, fitted the model of infection, and on the number of cases reported to estimate likely number of infections in four high risk metropolitan areas, as well as facilitate understanding the COVID-19&#39;s spread pattern. A group of SERI model statistical parameters were estimated using Markov Chain Monte Carlo (MCMC) methods, and our modeling integrated the effect of the official quarantine regulation and travel restriction of China. As a result, we estimated that the basic reproductive number R0 \u200bis 3.11 in Beijing, 2.78 in Shanghai, 2.02 in Guangzhou, and 1.75 in Shenzhen. In addition, we inferred the prediction results and compared the results of different level of parameters, For example, In Beijing, the predicated peak number of cases is around 466 at the peak time Feb 29, 2020; however, when the city conducts different levels (strict, mild, or weak) of travel restrictions or regulation measures, the estimation results show that transmission dynamics will change and the peek number of cases shows the changing proportion is between 56%~159%. We concluded that public health interventions would reduce the risks of COVID-19 spreading and more rigorous control and prevention measures will effectively contain its further spread, but risk increases when businesses and social activities returning back before the ending date. Besides, the experiences gained and lessons learned from China are potential to provide evidences supporting for other metropolitan areas and big cities with emerging cases outside China.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Na Hong; Jie He; Yingying Ma; Huizhen Jiang; Lin Han; Longxiang Su; Weiguo Zhu; Yun Long&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20032177&#39;},
 {&#39;cord_uid&#39;: &#39;wuvry51z&#39;,
  &#39;sha&#39;: &#39;ea17aad7692bb5040f3b2ba1d01b8249e4969469&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20032334&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;James P Broughton; Xianding Deng; Guixia Yu; Clare L Fasching; Jasmeet Singh; Jessica Streithorst; Andrea Granados; Alicia Sotomayor-Gonzalez; Kelsey Zorn; Allan Gopez; Elaine Hsu; Wei Gu; Steven Miller; Chao-Yang Pan; Hugo Guevara; Debra Wadford; Janice Chen; Charles Y Chiu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20032334&#39;},
 {&#39;cord_uid&#39;: &#39;23mp5961&#39;,
  &#39;sha&#39;: &#39;b5be10942198c0a79dff8aea2253b77ae2401882&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Early, low-dose and short-term application of corticosteroid treatment in patients with severe COVID-19 pneumonia: single-center experience from Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20032342&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Severe patients with 2019 novel coronavirus (2019-nCoV) pneumonia progressed rapidly to acute respiratory failure. We aimed to evaluate the definite efficacy and safety of corticosteroid in the treatment of severe COVID-19 pneumonia. Methods: Forty-six hospitalized patients with severe COVID-19 pneumonia hospitalized at Wuhan Union Hospital from January 20 to February 25, 2020, were retrospectively reviewed. The patients were divided into two groups based on whether they received corticosteroid treatment. The clinical symptoms and chest computed tomography(CT) results were compared. Results: A total of 26 patients received intravenous administration of methylprednisolone with a dosage of 1-2mg/kg/d for 5-7 days, while the remaining patients not. There was no significant difference in age, sex, comorbidities, clinical or laboratory parameters between the two groups on admission. The average number of days for body temperature back to the normal range was significantly shorter in patients with administration of methylprednisolone when compared to those without administration of methylprednisolone (2.06 ± 0.28 vs. 5.29 ± 0.70, P=0.010). The patients with administration of methylprednisolone had a faster improvement of SpO2, while patients without administration of methylprednisolone had a significantly longer interval of using supplemental oxygen therapy (8.2days[IQR 7.0-10.3] vs. 13.5days(IQR 10.3-16); P&lt;0.001). In terms of chest CT, the absorption degree of the focus was significantly better in patients with administration of methylprednisolone. Conclusion: Our data indicate that in patients with severe COVID-19 pneumonia, early, low-dose and short-term application of corticosteroid was associated with a faster improvement of clinical symptoms and absorption of lung focus.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Yin Wang; Weiwei Jiang; Qi He; Cheng Wang; Baoju Wang; Pan Zhou; Nianguo Dong; Qiaoxia Tong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20032342&#39;},
 {&#39;cord_uid&#39;: &#39;it4ka7v0&#39;,
  &#39;sha&#39;: &#39;6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20032417&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The current outbreak of coronavirus disease 2019 (COVID-19) has quickly spread across countries and become a global crisis. However, one of the most important clinical characteristics in epidemiology, the distribution of the incubation period, remains unclear. Different estimates of the incubation period of COVID-19 were reported in recent published studies, but all have their own limitations. In this study, we propose a novel low-cost and accurate method to estimate the incubation distribution. Methods: We have conducted a cross-sectional and forward follow-up study by identifying those asymptomatic individuals at their time of departure from Wuhan and then following them until their symptoms developed. The renewal process is hence adopted by considering the incubation period as a renewal and the duration between departure and symptom onset as a forward recurrence time. Under mild assumptions, the observations of selected forward times can be used to consistently estimate the parameters in the distribution of the incubation period. Such a method enhances the accuracy of estimation by reducing recall bias and utilizing the abundant and readily available forward time data. Findings: The estimated distribution of forward time fits the observations in the collected data well. The estimated median of incubation period is 8.13 days (95% confidence interval [CI]: 7.37-8.91), the mean is 8.62 days (95% CI: 8.02-9.28), the 90th percentile is 14.65 days (95% CI: 14.00-15.26), and the 99th percentile is 20.59 days (95% CI: 19.47, 21.62). Compared with results in other studies, the incubation period estimated in this study is longer. Interpretation: Based on the estimated incubation distribution in this study, about 10% of patients with COVID-19 would not develop symptoms until 14 days after infection. Further study of the incubation distribution is warranted to directly estimate the proportion with long incubation periods.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Qin Jing; Chong You; Qiushi Lin; Taojun Hu; Shicheng Yu; Xiao-Hua Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20032417&#39;},
 {&#39;cord_uid&#39;: &#39;duy5nvb5&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prevalence and Risk Factors of Acute Posttraumatic Stress Symptoms during the COVID-19 Outbreak in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.20032425&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background A novel coronavirus (SARA-CoV-2) emerged in Wuhan, China, in December 2019. Within a few weeks, the disease caused by SARA-CoV-2, which is named COVID-19, has escalated into an unprecedented ongoing outbreak with frightening speed, becoming a global health emergency. This study aimed to exam the prevalence and risk factors of acute posttraumatic stress symptoms (PTSS) in Chinese people shortly after the massive outbreak of COVID-19. Method An online anonymous questionnaire survey was conducted in mainland China between 30 January and 3 February, 2020. The survey consisted of two self-administered questionnaires: one was designed to require personal information (gender, age, education background), current location, recent exposure history of Wuhan, the classification of population, and subjective sleep quality; the other was the PTSD Checklist for DSM-5 (PCL-5), which was to assess PTSS referring to the outbreak. Results A total of 2091 Chinese participated in the current study. The prevalence of PTSS among the public in mainland China 1 month after the COVID-19 outbreak was 4.6%. Multiple linear regression analysis revealed that gender (p &lt; 0.001), exposure history of Wuhan (p = 0.047), classification of population (p &lt; 0.001), and subjective sleep quality (p &lt; 0.001) could be regarded as predictor factors for PTSS. Conclusions The results showed that some Chinese showed acute PTSS during the COVID-19 outbreak. Therefore, comprehensive psychological intervention needs further implementation. Furthermore, females, people who having recent exposure history of Wuhan, those at high risk of infection or with poor sleep quality deserve special attention.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Luna Sun; Zhuoer Sun; Lili Wu; Zhenwen Zhu; Fan Zhang; Zhilei Shang; Yanpu Jia; Jingwen Gu; Yaoguang Zhou; Yan Wang; Nianqi Liu; Weizhi Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.20032425&#39;},
 {&#39;cord_uid&#39;: &#39;r88hpbxp&#39;,
  &#39;sha&#39;: &#39;837056cb9cb01b2f15c39e7e342a3c47bfcc24cf&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.977876&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreak of coronavirus disease (COVID-19) in China caused by SARS-CoV-2 virus continually lead to worldwide human infections and deaths. It is currently no specific viral protein targeted therapeutics yet. Viral nucleocapsid protein is a potential antiviral drug target, serving multiple critical functions during the viral life cycle. However, the structural information of SARS-CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the 2.7 Å crystal structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. Although overall structure is similar with other reported coronavirus nucleocapsid protein N-terminal domain, the surface electrostatic potential characteristics between them are distinct. Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique potential RNA binding pocket alongside the β-sheet core. Complemented by in vitro binding studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Sisi Kang; Mei Yang; Zhongsi Hong; Liping Zhang; Zhaoxia Huang; Xiaoxue Chen; Suhua He; Ziliang Zhou; Zhechong Zhou; Qiuyue Chen; Yan Yan; Changsheng Zhang; Hong Shan; Shoudeng Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.977876&#39;},
 {&#39;cord_uid&#39;: &#39;vu7u11yk&#39;,
  &#39;sha&#39;: &#39;a33f451d8e2caff57c1133f14198454a64993c47&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.980037&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-07&#39;,
  &#39;authors&#39;: &#39;Zhiqiang Zheng; Vanessa M. Monteil; Sebastian Maurer-Stroh; Chow Wenn Yew; Carol Leong; Nur Khairiah Mohd-Ismail; Suganya Cheyyatraivendran Arularasu; Vincent Tak Kwong Chow; Raymond Lin Tzer Pin; Ali Mirazimi; Wanjin Hong; Yee-Joo Tan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.980037&#39;},
 {&#39;cord_uid&#39;: &#39;ekze7vj0&#39;,
  &#39;sha&#39;: &#39;5cf2bec1530cef38cd0eea173fb90ba1a788e11f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Assessing Topographic Structural Connectivity of the Human Basal Ganglia and Thalamus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.06.981142&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe basal ganglia and thalamus play an important role in cognition, procedural learning, eye movements, control of voluntary motor movements, emotional control, habit development, and are structures that are severely impacted by neurological disorders such as Alzheimer’s disease, Parkinson’s disease, or Tourette syndrome. To understand the structural connectivity of cortical and subcortical circuits in the healthy human brain could thus be of pivotal importance for detecting changes in this circuitry and to start early intervention, to assess the progress of movement rehabilitation, or the effectiveness of therapeutic approaches in neuropsychiatry. While conventional magnetic resonance imaging (MRI), positron emission tomography, or magnetoencephalography are able to provide detailed information about connectivity at the macro level, the sensitivity and specificity these imaging techniques put limits on the amount of detail one can obtain when measuring in vivo connectivity of human basal ganglia and thalamus. In contrast, the multiband diffusion echo planar imaging MRI sequence, which acquires multiple slices of the brain simultaneously, enables high resolution imaging of these brain structures with only short acquisition times at 3-Tesla field strength. Here, we introduce a novel protocol that allows us to generate comprehensive in vivo participant-specific probabilistic patterns and visualizations of the structural connections that exist within basal ganglia and thalamic nuclei. Moreover, we are able to map specific parcellations of these nuclei into sub-territories based on their connectivity with primary motor-, and somatosensory cortex. The detailed subcortical structural connectivity mapping introduced in this work could benefit early intervention and therapy methods for human movement rehabilitation and for treating neuropsychiatric disorders.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Irtiza A. Gilani; Kader K. Oguz; Huseyin Boyaci; Katja Doerschner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.06.981142&#39;},
 {&#39;cord_uid&#39;: &#39;g1kr7s11&#39;,
  &#39;sha&#39;: &#39;c1a08ce8f2c5d5311b19822c3df5aa9d25c4d93c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The epidemiological characteristics of 2019 novel coronavirus diseases (COVID-19) in Jingmen,Hubei,China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20031393&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Summary Background: Some articles have reported the epidemiological and clinical characteristics of coronavirus disease (COVID-19) in Wuhan, but other cities have rarely been reported. This study explored the epidemiology of COVID-19 in Jingmen. Methods: All confirmed cases of COVID-19 in the First People′s Hospital of Jingmen are included from January 12 to February 14,2020. Cases were analyzed for epidemiological data and were confirmed by real-time PCR. Findings: Of the 213 cases (108 men and 105 women) , 88 (41%) had exposure to Wuhan. The median age was 48 years ( range,2-88 years;IQR,35-58.5). Thirty-three severe patients with a median age of 66 years(range,33-82 years,IQR, 57-76) were treated in intensive care units; out of these patients, 66.7 %(22) were men and 19 (57.5%) had chronic diseases, including hypertension, diabetes, heart failure, stroke, and renal insufficiency. Under the controlled measures, the number of new patients gradually decreased and nearly disappeared after 20 days. Interpretation: All people are susceptible to the COVID-19, but older males and those with comorbid conditions are more likely to have severe symptoms. Even though COVID-19 is highly contagious, control measures have proven to be very effective.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Qijun Gao; yingfu hu; zhiguo dai; Jing wu; Feng Xiao; Jing wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20031393&#39;},
 {&#39;cord_uid&#39;: &#39;xle4bx49&#39;,
  &#39;sha&#39;: &#39;c6012a6141f5ad05ac983aa9f1a7c57938363b16&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prognostic value of NT-proBNP in patients with severe COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20031575&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China has been declared a public health emergency of international concern. The cardiac injury was dominate in the process. However, whether N terminal pro B type natriuretic peptide (NT-proBNP) predicted outcome of COVID-19 patients was unknown. The study initially enrolled 102 patients with severe COVID-19 pneumonia from a continuous sample. After screening out the ineligible cases, 54 patients were analyzed in this study. Results found that patients with higher NT-proBNP (above 88.64 pg/mL) level had more risks of in-hospital death. After adjusting for potential cofounders in separate modes, NT-proBNP presented as an independent risk factor of in-hospital death in patients with severe COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Lei Gao; Dan Jiang; Xuesong Wen; Xiaocheng Cheng; Min Sun; Bin He; Lin-na You; Peng Lei; Xiao-wei Tan; Shu Qin; Guoqiang Cai; Dongying Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20031575&#39;},
 {&#39;cord_uid&#39;: &#39;q4i9kh65&#39;,
  &#39;sha&#39;: &#39;598d3eb737dfa7701ce8c29c86bc9f6589d8a581&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Duration of viral detection in throat and rectum of a patient with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20032052&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The rapid spread of coronavirus disease 2019 (COVID-19) raises concern about a global pandemic. Knowledge about the duration of viral shedding remains important for patient management and infection control. We report the duration of viral detection in throat and rectum of a COVID-19 patient treated at the Hospital for Tropical Diseases in Ho Chi Minh City, Vietnam. Despite clinical recovery, SARS-CoV-2 RNA remained detectable by real time RT-PCR in throat and rectal swabs until day 11 and 18 of hospitalization, respectively. Because live SARS-CoV-2 has been successfully isolated from a stool sample from a COVID-19 patient in China, the results demonstrate that COVID-19 patients may remain infectious for long periods, and fecal-oral transmission may be possible. Therefore, our finding has important implications for infection control.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Le Van Tan; Nghiem My Ngoc; Bui Thi Ton That; Le Thi Tam Uyen; Nguyen Thi Thu Hong; Nguyen Thi Phuong Dung; Le Nguyen Truc Nhu; Tran Tan Thanh; Dinh Nguyen Huy Man; Nguyen Thanh Phong; Tran Tinh Hien; Nguyen Thanh Truong; Guy Thwaites; Nguyen Van Vinh Chau&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20032052&#39;},
 {&#39;cord_uid&#39;: &#39;kv77pw7y&#39;,
  &#39;sha&#39;: &#39;74bee8bf3229f4c28991a7231c02d911d24770e9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20032326&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Gustavo Barcelos Barra; Ticiane Henriques Santa Rita; Pedro Goes Mesquita; Rafael Henriques Jacomo; Lidia Freire Abdalla Nery&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20032326&#39;},
 {&#39;cord_uid&#39;: &#39;5xxi33v3&#39;,
  &#39;sha&#39;: &#39;31816b72a378debeb42e76ff7ff2af69727cd049&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Diagnosis of Acute Respiratory Syndrome Coronavirus 2 Infection by Detection of Nucleocapsid Protein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20032524&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND Nucleic acid test and antibody assay have been employed in the diagnosis for SARS-CoV-2 infection, but the use of viral antigen for diagnosis has not been successfully developed. Theoretically, viral antigen is the specific marker of the virus and precedes antibody appearance within the infected population. There is a clear need of detection of viral antigen for rapid and early diagnosis. METHODS We included a cohort of 239 participants with suspected SARS-CoV-2 infection from 7 centers for the study. We measured nucleocapsid protein in nasopharyngeal swab samples in parallel with the nucleic acid test. Nucleic acid test was taken as the reference standard, and statistical evaluation was taken in blind. We detected nucleocapsid protein in 20 urine samples in another center, employing nasopharyngeal swab nucleic acid test as reference standard. RESULTS We developed a fluorescence immunochromatographic assay for detecting nucleocapsid protein of SARS-CoV-2 in nasopharyngeal swab sample and urine within 10 minutes. 100% of nucleocapsid protein positive and negative participants accord with nucleic acid test for same samples. Further, earliest participant after 3 days of fever can be identified by the method. In an additional preliminary study, we detected nucleocapsid protein in urine in 73.6% of diagnosed COVID-19 patients. CONCLUSIONS Those findings indicate that nucleocapsid protein assay is an accurate, rapid, early and simple method for diagnosis of COVID-19. Appearance of nucleocapsid protein in urine coincides our finding of the SARS-CoV-2 invading kidney and might be of diagnostic value.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Bo Diao; Kun Wen; Jian Chen; Yueping Liu; Zilin Yuan; Chao Han; Jiahui Chen; Yuxian Pan; Li Chen; Yunjie Dan; Jing Wang; Yongwen Chen; Guohong Deng; Hongwei Zhou; Yuzhang Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20032524&#39;},
 {&#39;cord_uid&#39;: &#39;kjjlxdpp&#39;,
  &#39;sha&#39;: &#39;19bc9648d12dc6b7ca757f82dcd182c03111691e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of Coronavirus Pneumonia 2019 (COVID-19): An Updated Systematic Review&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20032573&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;OBJECTIVE Clinical characteristics of novel coronavirus pneumonia (COVID-19) have been described in numerous studies but yielded varying results. We aimed to conduct a systematic review on scientific literatures and to synthesize critical data on clinical traits of COVID-19 from its initial outbreak to pandemic. METHODS Systematic searches were conducted to identify retrospective observational study that contained clinical characteristics on COVID-19 through multiple databases. Two reviewers independently evaluated eligible publications. Data on clinical characteristics of COVID-19 were extracted and analyzed. RESULTS Seventy-two retrospective studies demonstrating the clinical characteristics of COVID-19 were included. A total of 3470 COVID-19 patients were synthesized to the final analysis in an unbiased manner. The most common symptom was fever (2878 [83.0%]), and 63.4% of the patients presented fever as onset symptom. There were 2528 [88.2%] of 2866 cases had abnormal lung findings on chest CT scan. Laboratory findings showed that 1498 [62.8%] of 2387 cases had lymphopenia, and 1354 [64.8%] of 2091 cases had an increased level of C-reactive protein (CRP). A total of 185 [11.5%] patients were admitted to intensive care unit (ICU) while the overall case fatality rate (CFR) was 3.7%. Compared to patients admitted outside of Hubei, China, those from Hubei had a significant higher ICU admission rate (21.9% vs. 2.5%, p&lt;0.001). Also, CFR attributed to COVID-19 was significantly higher in Hubei than that of non-Hubei admissions (10.4% vs. 0.6%, p&lt;0.001). INTERPRETATION This large patient-based systematic review presents a more precise profiling of the COVID-19 from its outbreak to current pandemic. Dynamic evolvements of COVID-19 are needed to be characterized in future studies.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Zhangfu Fang; Fang Yi; Kang Wu; Kefang Lai; Xizhuo Sun; Nanshan Zhong; Zhigang Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20032573&#39;},
 {&#39;cord_uid&#39;: &#39;f3indt3b&#39;,
  &#39;sha&#39;: &#39;fbda9162de4c4a881b118dd140f1e976902df15e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis of early renal injury in COVID-19 and diagnostic value of multi-index combined detection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20032599&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives The aim of the study was to analyze the incidence of COVID-19 with early renal injury, and to explore the value of multi-index combined detection in diagnosis of early renal injury in COVID-19. Design The study was an observational, descriptive study. Setting This study was carried out in a tertiary hospital in Guangdong, China. Participants 12 patients diagnosed with COVID-19 from January 20, 2020 to February 20, 2020. Primary and secondary outcome measures The primary outcome was to evaluate the incidence of early renal injury in COVID-19. In this study, the estimated glomerular filtration rate (eGFR), endogenous creatinine clearance (Ccr) and urine microalbumin / urinary creatinine ratio (UACR) were calculated to assess the incidence of early renal injury. Secondary outcomes were the diagnostic value of urine microalbumin (UMA), α1-microglobulin (A1M), urine immunoglobulin-G (IGU), urine transferring (TRU) alone and in combination in diagnosis of COVID-19 with early renal injury. Results While all patients had no significant abnormalities in serum creatinine (Scr) and blood urea nitrogen (BUN), the abnormal rates of eGFR, Ccr, and UACR were 66.7%, 41.7%, and 41.7%, respectively. Urinary microprotein detection indicated that the area under curve (AUC) of multi-index combined to diagnose early renal injury in COVID-19 was 0.875, which was higher than UMA (0,813), A1M (0.813), IGU (0.750) and TRU (0.750) alone. Spearman analysis showed that the degree of early renal injury was significantly related to C-reactive protein (CRP) and neutrophil ratio (NER), suggesting that the more severe the infection, the more obvious the early renal injury. Hypokalemia and hyponatremia were common in patients with COVID-19, and there was a correlation with the degree of renal injury. Conclusions Early renal injury was common in patients with COVID-19. Combined detection of UMA, A1M, IGU, and TRU was helpful for the diagnosis of early renal injury in COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Xu-wei Hong; Ze-pai Chi; Guo-yuan Liu; Hong Huang; Shun-qi Guo; Jing-ru Fan; Xian-wei Lin; Liao-zhun Qu; Rui-lie Chen; Ling-jie Wu; Liang-yu Wang; Qi-chuan Zhang; Su-wu Wu; Ze-qun Pan; Hao Lin; Yu-hua Zhou; Yong-hai Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20032599&#39;},
 {&#39;cord_uid&#39;: &#39;ohba2n2o&#39;,
  &#39;sha&#39;: &#39;9a077c5ff1dcbb9104feeaa832e73ccdcf5fdcc9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical outcomes of 402 patients with COVID-2019 from a single center in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.20032672&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The SARS-CoV-2 outbreak is causing widespread infections and significant mortality. Previous studies describing clinical characteristics of the disease contained small cohorts from individual centers or larger series consisting of mixed cases from different hospitals. We report analyses of mortality and disease severity among 402 patients from a single hospital. The cohort included 297 patients with confirmed and 105 with suspected diagnosis. The latter group met the criteria for clinical diagnosis but nucleic acid tests results were initially interpreted as suspicious. Data were compared between genders and among different age groups. The overall case fatality is 5.2%. However, patients 70 years of age or older suffered a significantly higher mortality (17.8%), associated with more patients having severe or critical illness (57.5%). Patients 50 years of age or older had a mortality of 8.0%, and those younger than 50 years, 1.2%. Male patients had a mortality of 7.6% versus 2.9% in females.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Yingjie Wu; Wei Guo; Huan Liu; Bei Qi; Ke Liang; Bo Xu; Zhiyong Peng; Shu-Yuan Xiao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.20032672&#39;},
 {&#39;cord_uid&#39;: &#39;e8qubwha&#39;,
  &#39;sha&#39;: &#39;3ebbcdddb122f405061bb5e1ddcf94abe9258d58&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Substrate specificity profiling of SARS-CoV-2 Mpro protease provides basis for anti-COVID-19 drug design&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.981928&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn December 2019, the first cases of a novel coronavirus infection were diagnosed in Wuhan, China. Due to international travel and human-to-human transmission, the virus spread rapidly inside and outside of China. Currently, there is no effective antiviral treatment for COVID-19, therefore research efforts are focused on the rapid development of vaccines and antiviral drugs. The SARS-CoV-2 Mpro protease constitutes one of the most attractive antiviral drug targets. To address this emerging problem, we have synthesized a combinatorial library of fluorogenic substrates with glutamine in the P1 position. We used it to determine the substrate preferences of the SARS-CoV and SARS-CoV-2 proteases, using natural and a large panel of unnatural amino acids. The results of our work provide a structural framework for the design of inhibitors as antiviral agents or diagnostic tests.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Wioletta Rut; Katarzyna Groborz; Linlin Zhang; Xinyuanyuan Sun; Mikolaj Zmudzinski; Rolf Hilgenfeld; Marcin Drag&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.981928&#39;},
 {&#39;cord_uid&#39;: &#39;4ezld56u&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Meta-transcriptomic analysis of virus diversity in urban wild birds with paretic disease&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.982207&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWild birds are major natural reservoirs and potential dispersers of a variety of infectious diseases. As such, it is important to determine the diversity of viruses they carry and use this information to help understand the potential risks of spill-over to humans, domestic animals, and other wildlife. We investigated the potential viral causes of paresis in long-standing, but undiagnosed disease syndromes in wild Australian birds. RNA from diseased birds was extracted and pooled based on tissue type, host species and clinical manifestation for metagenomic sequencing. Using a bulk and unbiased meta-transcriptomic approach, combined with careful clinical investigation and histopathology, we identified a number of novel viruses from the families Astroviridae, Picornaviridae, Polyomaviridae, Paramyxoviridae, Parvoviridae, Flaviviridae, and Circoviridae in common urban wild birds including Australian magpies, magpie lark, pied currawongs, Australian ravens, and rainbow lorikeets. In each case the presence of the virus was confirmed by RT-PCR. These data revealed a number of candidate viral pathogens that may contribute to coronary, skeletal muscle, vascular and neuropathology in birds of the Corvidae and Artamidae families, and neuropathology in members of the Psittaculidae. The existence of such a diverse virome in urban avian species highlights the importance and challenges in elucidating the etiology and ecology of wildlife pathogens in urban environments. This information will be increasingly important for managing disease risks and conducting surveillance for potential viral threats to wildlife, livestock and human health. More broadly, our work shows how meta-transcriptomics brings a new utility to pathogen discovery in wildlife diseases.ImportanceWildlife naturally harbor a diverse array of infectious microorganisms and can be a source of novel diseases in domestic animals and human populations. Using unbiased RNA sequencing we identified highly diverse viruses in native birds in Australian urban environments presenting with paresis. This investigation included the clinical investigation and description of poorly understood recurring syndromes of unknown etiology: clenched claw syndrome, and black and white bird disease. As well as identifying a range of potentially disease-causing viral pathogens, this study describes methods that can effectively and efficiently characterize emergent disease syndromes in free ranging wildlife, and promotes further surveillance for specific potential pathogens of potential conservation and zoonotic concern.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-08&#39;,
  &#39;authors&#39;: &#39;Wei-Shan Chang; John-Sebastian Eden; Jane Hall; Mang Shi; Karrie Rose; Edward C. Holmes&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.982207&#39;},
 {&#39;cord_uid&#39;: &#39;2w0zr9c0&#39;,
  &#39;sha&#39;: &#39;d3493e82bfbb4d2dad1a01b5c2bb50df879619a7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 is sensitive to type I interferon pretreatment&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.07.982264&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.ImportanceWith the ongoing outbreak of COVID-19 disease, differences between the SARS-CoV-2 and the original SARS-CoV could be leveraged to inform disease progression and eventual treatment options. In addition, these findings could have key implications for animal model development as well as further research into how SARS-CoV-2 modulates the type I IFN response early during infection.Article SummarySARS-CoV-2 has similar replication kinetics to SARS-CoV, but demonstrates significant sensitivity to type I interferon treatment.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-09&#39;,
  &#39;authors&#39;: &#39;Kumari G. Lokugamage; Adam Hage; Craig Schindewolf; Ricardo Rajsbaum; Vineet D. Menachery&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.07.982264&#39;},
 {&#39;cord_uid&#39;: &#39;txyf05n4&#39;,
  &#39;sha&#39;: &#39;6cf87a546884756094da0d300e85a061c2cc43ea&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A retrospective study of the clinical characteristics of COVID-19 infection in 26 children&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20029710&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The outbreak of novel coronavirus pneumonia in China began in December 2019. Studies on novel coronavirus disease (COVID-19) were less based on pediatric patients. This study aimed to reveal the clinical characteristics of COVID-19 in children. Method: This study retrospectively analyzed the clinical symptoms, laboratory results, chest CT, and treatment of children with laboratory-confirmed COVID-19(ie, with samples that were positive for 2019 novel coronavirus[2019-nCoV]) who were admitted to Shenzhen Center of National Infectious Disease Clinical Medical Research from January 16 to February 8, 2020. Result: Nine patients had no obvious clinical symptom. 11 patients developed fever. Other symptoms, including cough(in eleven of seventeen patients), rhinorrhea(in two), diarrhea(in two), vomiting(in two), were also observed. A small minority of patients had lymphocytopenia. Alanine transaminase or transaminase increased in three cases. According to chest CT scan, 11 patients showed unilateral pneumonia, 8 patients had no pulmonary infiltration. No serious complications such as acute respiratory syndrome and acute lung injury occurred in all patients. Conclusion: The clinical characteristics of 2019-nCoV infection in children were different from adult. The overall condition of children were mild and have a good prognosis.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Anjue Tang; Wenhui Xu; min shen; Peifen Chen; Guobao Li; Yingxia Liu; Lei Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20029710&#39;},
 {&#39;cord_uid&#39;: &#39;uq4eobf0&#39;,
  &#39;sha&#39;: &#39;43f760b5cb9c3b430c6d0b31fa90d42e71a6429c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmission potential of COVID-19 in Iran&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20030643&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We computed reproduction number of COVID-19 epidemic in Iran using two different methods. We estimated R0 at 3.6 (95% CI, 3.2, 4.2) (generalized growth model) and at 3.58 (95% CI, 1.29, 8.46) (estimated epidemic doubling time of 1.20 (95% CI, 1.05, 1.44) days) respectively. Immediate social distancing measures are recommended.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Kamalich Muniz-Rodriguez; Isaac Chun-Hai Fung; Shayesterh R. Ferdosi; Sylvia K. Ofori; Yiseul Lee; Amna Tariq; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20030643&#39;},
 {&#39;cord_uid&#39;: &#39;1k64g7m3&#39;,
  &#39;sha&#39;: &#39;4c96d9cfc6da83e41b0889579d1e8b25765bd355&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20031229&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in Chinese Han population is limited. Our objective is to describe the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in the Chinese Han population. In this case series of 752 patients, the full spectrum of cases is described. Fever was present in 86-90% of the patients. The second most common symptom was cough (49.1-51.0%), fatigue (25.2-27.1%), sputum (20.0-23.1%), and headache (9.8-11.1%). the mortality rate is 4.6% in Wuhan, 1.9% in Beijing, and 0.9% in Shanghai. Our findings showed that the levels of lymphocytes were 0.8（IQR, 0.6-1.1）109/L in Wuhan, 1.0（IQR, 0.7-1.4）109/L in Beijing, and 1.1 (IQR, 0.8-1.5) 109/L in Shanghai before admission to hospitals, respectively, indicating that cellular immune function might relate to the mortality. Based on the reference ranges of normal Chinese Han population and the data of the critically ill patients we have observed, it is recommended that reference ranges of people at high risk of COVID-19 infection are CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500 cells/mm3, and CD8+ lymphocytes below 300 cells/mm3.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Qiang Zeng; Yong-zhe Li; Gang Huang; Wei Wu; Sheng-yong Dong; Yang Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20031229&#39;},
 {&#39;cord_uid&#39;: &#39;glq0lckz&#39;,
  &#39;sha&#39;: &#39;4c84d1153daa335f5ff6d61db665024b32ac8195&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of SARS-CoV-2 Pneumonia Compared to Controls in Chinese Han Population&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20031658&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in Chinese Han population is limited. Our objective is to describe the epidemiological and clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in the Chinese Han population. Methods Retrospective, multi-center case series of the 69 consecutive hospitalized patients with confirmed SARS-CoV-2 pneumonia, from February 7 to February 28, 2020; final date of follow-up was February 29, 2020. Results The study population included 69 hospitalized patients with confirmed SARS-CoV-2 pneumonia without comorbidities and 14,117 normal controls. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. Compared with mild patients (n = 44), severe or critical patients (n = 25) were significantly older (median age, 67 years [IQR, 58-79] vs. 49 years [IQR, 36-60]; P &lt; 0.01). Fever was present in 98.6% of the patients. The second most common symptom was cough (62.3%), fatigue (58.0%), sputum (39.1%), and headache (33.3%). The median incubation period was 4 days (IQR, 2 to 7). Leukocyte count was 74.1% of normal controls and lymphocyte count was 45.9% of normal controls. The phenomenon of lymphocyte depletion (PLD) observed in severe or critical cases in 100%. Levels of lactate dehydrogenase, D-dimer, procalcitonin, and interleukin-6 were showed significant differences between mild and severe or critical cases. Chest computed tomographic scans showed bilateral patchy patterns (49.3%), local patchy shadowing (29.0%), and ground glass opacity (21.7%). 7.3% patients were diagnosed ARDS, 7.3% patients were diagnosed acute cardiac injury (troponin I &gt;28 pg/mL) and 4.4% patients were diagnosed fungal infections or shock. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Conclusions In this multicenter case series of 69 patients without comorbidities, the full spectrum of asymptomatic, mild, severe, and critical cases is described. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Among the 25 patients with severe or critical disease, 12.0% patients were underwent non-invasive mechanical ventilation, 8.0% patients underwent invasive mechanical ventilation, and 4.0% patients died.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Yang Xu; Yi-rong Li; Qiang Zeng; Zhi-bing Lu; Yong-zhe Li; Wei Wu; Sheng-yong Dong; Gang Huang; Xing-huan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20031658&#39;},
 {&#39;cord_uid&#39;: &#39;2u6ki9dv&#39;,
  &#39;sha&#39;: &#39;e5f19b6daf956e815c779228cc0cad1293d65bbb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of Two Human to Human Transmitted Coronaviruses: Corona Virus Disease 2019 versus Middle East Respiratory Syndrome Coronavirus.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20032821&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;After the outbreak of the middle east respiratory syndrome (MERS) worldwide in 2012. Currently, a novel human coronavirus has caused a major disease outbreak, and named corona virus disease 2019 (COVID-19). The emergency of MRES-COV and COVID-19 has caused global panic and threatened health security. Unfortunately, the similarities and differences between the two coronavirus diseases remain to be unknown. The aim of this study, therefore, is to perform a systematic review to compare epidemiological, clinical and laboratory features of COVID-19 and MERS-COV population. We searched PubMed, EMBASE and Cochrane Register of Controlled Trials database to identify potential studies reported COVID-19 or MERS-COV. Epidemiological, clinical and laboratory outcomes, the admission rate of intensive cure unit (ICU), discharge rate and fatality rate were evaluated using GraphPad Prism software. Thirty-two studies involving 3770 patients (COVID-19 = 1062, MERS-COV = 2708) were included in this study. The present study revealed that compared with COVID-19 population, MERS-COV population had a higher rate of ICU admission, discharge and fatality and longer incubation time. It pointed out that fever, cough and generalised weakness and myalgia were main clinical manifestations of both COVID-19 and MERS-COV, whereas ARDS was main complication. The most effective drug for MERS-COV is ribavirin and interferon.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Ping Xu; Guo-Dong Sun; Zhi-Zhong Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20032821&#39;},
 {&#39;cord_uid&#39;: &#39;mr8z65o5&#39;,
  &#39;sha&#39;: &#39;c42a617a00afe6a36bde0a8e3638e0f55bfee4f7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prediction of COVID-19 Spreading Profiles in South Korea, Italy and Iran by Data-Driven Coding&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20032847&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This work applies a data-driven coding method for prediction of the COVID-19 spreading profile in any given population that shows an initial phase of epidemic progression. Based on the historical data collected for COVID-19 spreading in 367 cities in China and the set of parameters of the augmented Susceptible-Exposed-Infected-Removed (SEIR) model obtained for each city, a set of profile codes representing a variety of transmission mechanisms and contact topologies is formed. By comparing the data of an early outbreak of a given population with the complete set of historical profiles, the best fit profiles are selected and the corresponding sets of profile codes are used for prediction of the future progression of the epidemic in that population. Application of the method to the data collected for South Korea, Italy and Iran shows that peaks of infection cases are expected to occur before the end of March 2020, and that the percentage of population infected in each city will be less than 0.01%, 0.05% and 0.02%, for South Korea, Italy and Iran, respectively.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Choujun Zhan; Chi K. Tse; Zhikang Lai; Tianyong Hao; Jingjing Su&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20032847&#39;},
 {&#39;cord_uid&#39;: &#39;6tnkz6hh&#39;,
  &#39;sha&#39;: &#39;c4ce14ce42fa4360dfe3515ec9d1584847381c27&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A deterministic epidemic model for the emergence of COVID-19 in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20032854&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Coronavirus disease (COVID-19) broke out in Wuhan, Hubei province,China, in December 2019 and soon after Chinese health authorities tookunprecedented prevention and control measures to curb the spreading ofthe novel coronavirus-related pneumonia. We develop a mathematicalmodel based on daily updates of reported cases to study the evolutionof the epidemic. With the model, on 95% confidence level, we estimatethe basic reproduction number, R0 = 2.82 ± 0.11, time between March19 and March 21 when the effective reproduction number becoming lessthan one, the epidemic ending after April 2 and the total number ofconfirmed cases approaching 14408 ± 429 on the Chinese mainlandexcluding Hubei province.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Meng Wang; Jingtao Qi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20032854&#39;},
 {&#39;cord_uid&#39;: &#39;bc9retcq&#39;,
  &#39;sha&#39;: &#39;c91b581d38243c502118d93fc6f71ee02ec395e4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Quantifying dynamics of SARS-CoV-2 transmission suggests that epidemic control and avoidance is feasible through instantaneous digital contact tracing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.20032946&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Mobile phone apps implementing algorithmic contact tracing can speed up the process of tracing newly diagnosed individuals, spreading information instantaneously back through a past contact network to inform them that they are at risk of being infected, and thus allow them to take appropriate social distancing and testing measures. The aim of non-pharmaceutical infection prevention is to move a population towards herd protection, a state where a population maintains R\u200b0&lt;\u200b1, thus making it impossible for a pathogen to cause an epidemic. Here, we address epidemiological issues that affect the feasibility of an algorithmic approach to instantaneous contact tracing; ethical and implementation issues are addressed separately. First we quantify the parameters of COVID-19 in a framework that is consistent with the renewal equation formulation of epidemic spread. Second, we use an analytical solution to application of first-degree contact tracing in the renewal equation model to explore combinations of efficacy that can induce herd protection (R\u200b0&lt;\u200b1). With the emergence of the novel viral pathogen SARS-CoV-2, of clear potential for a global pandemic with high fatality rates and incapacitated health systems, the question of prevention has critical priority. We come to the conclusion that isolating symptomatic cases and tracing their contacts in a classical manner is not sufficiently fast to stop the spread of the epidemic and needs to be accompanied by measures of social distancing that are disruptive to a wide number of people. We show that first-degree instantaneous contact tracing, informing users when they can move safely or when to seek medical help and avoid vulnerable individuals, has the potential to stop the spread of the epidemic if used by a sufficiently large number of people with reasonable fidelity.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Luca Ferretti; Chris Wymant; Michelle Kendall; Lele Zhao; Anel Nurtay; David G Bonsall; Christophe Fraser&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.20032946&#39;},
 {&#39;cord_uid&#39;: &#39;te049lxh&#39;,
  &#39;sha&#39;: &#39;b4d9a00412c65725de441654454803bb20ecb931&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.980383&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.Abstract Figure&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Conor Jenkins; Ben Orsburn&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.980383&#39;},
 {&#39;cord_uid&#39;: &#39;h2h4bnd5&#39;,
  &#39;sha&#39;: &#39;2989bf36068372cfedef82f58f4777e60ccbcfaa&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Aerodynamic Characteristics and RNA Concentration of SARS-CoV-2 Aerosol in Wuhan Hospitals during COVID-19 Outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.08.982637&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundThe ongoing outbreak of COVID-19 has spread rapidly and sparked global concern. While the transmission of SARS-CoV-2 through human respiratory droplets and contact with infected persons is clear, the aerosol transmission of SARS-CoV-2 has been little studied.MethodsThirty-five aerosol samples of three different types (total suspended particle, size segregated and deposition aerosol) were collected in Patient Areas (PAA) and Medical Staff Areas (MSA) of Renmin Hospital of Wuhan University (Renmin) and Wuchang Fangcang Field Hospital (Fangcang), and Public Areas (PUA) in Wuhan, China during COVID-19 outbreak. A robust droplet digital polymerase chain reaction (ddPCR) method was employed to quantitate the viral SARS-CoV-2 RNA genome and determine aerosol RNA concentration.ResultsThe ICU, CCU and general patient rooms inside Renmin, patient hall inside Fangcang had undetectable or low airborne SARS-CoV-2 concentration but deposition samples inside ICU and air sample in Fangcang patient toilet tested positive. The airborne SARS-CoV-2 in Fangcang MSA had bimodal distribution with higher concentration than those in Renmin during the outbreak but turned negative after patients number reduced and rigorous sanitization implemented. PUA had undetectable airborne SARS-CoV-2 concentration but obviously increased with accumulating crowd flow.ConclusionsRoom ventilation, open space, proper use and disinfection of toilet can effectively limit aerosol transmission of SARS-CoV-2. Gathering of crowds with asymptomatic carriers is a potential source of airborne SARS-CoV-2. The virus aerosol deposition on protective apparel or floor surface and their subsequent resuspension is a potential transmission pathway and effective sanitization is critical in minimizing aerosol transmission of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Yuan Liu; Zhi Ning; Yu Chen; Ming Guo; Yingle Liu; Nirmal Kumar Gali; Li Sun; Yusen Duan; Jing Cai; Dane Westerdahl; Xinjin Liu; Kin-fai Ho; Haidong Kan; Qingyan Fu; Ke Lan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.08.982637&#39;},
 {&#39;cord_uid&#39;: &#39;pxymczn2&#39;,
  &#39;sha&#39;: &#39;4f53e43ba1bfb84611eaa839994f9297cdd65dc9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The effectiveness of full and partial travel bans against COVID-19 spread in Australia for travellers from China.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20032045&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Australia implemented a travel ban on China on February 1st 2020. Partial lifting of the ban is being considered, given the decline in incidence of COVID-19 in China. We modelled three scenarios to test the impact of travel bans on epidemic control in Australia. Scenario one was no ban, scenario two was the current ban, scenario three was a partial lifting of the current ban to allow over 100,000 university students to enter Australia, but not tourists. We used disease incidence data from China and air travel passenger movements between China and Australia, derived from incoming passenger arrival cards. We estimated the true incidence of disease in China using data on expected proportion of under-ascertainment of cases. We used an age specific deterministic model divided in 18 age stratified groups to model the epidemic in each scenario. The modelled epidemic with the full ban fitted the observed incidence of cases well. The modelled epidemic of the current ban predicts 57 cases on March 6th in Australia, compared to 66 observed on this date. The modelled impact without a travel ban implemented on February the 1st shows the epidemic would continue for more than a year resulting in more than 2000 cases and about 400 deaths. The impact of a partial lifting of a ban is minimal, and may be safe to consider. Travel restrictions were highly effective for containing the COVID-19 epidemic in Australia and averted a much larger epidemic. The epidemic is still containable if other measures are used in tandem as cases surge in other countries. This research can inform decisions on placing or lifting travel bans as a control measure for the COVID-19 epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Valentina Costantino; David James Heslop; Chandini Raina MacIntyre&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20032045&#39;},
 {&#39;cord_uid&#39;: &#39;fm6lf66a&#39;,
  &#39;sha&#39;: &#39;8ed2df683ca3cde36ebee8185754abf4c44b717b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Data-driven discovery of clinical routes for severity detection in COVID-19 pediatric cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20032219&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of COVID-19 epidemic has caused worldwide health concerns since Nov., 2019. A previous study described the demographic, epidemiologic, and clinical features for infected infants. However, compared with adult cases, little attention has been paid to the infected pediatric cases. Severity detection is challenging for children since most of children patients have mild symptoms no matter they are moderately or critically ill therein.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Hui Yu; Jianbo Shao; Yuqi Guo; Yun Xiang; Chuan Sun; Ye Yuan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20032219&#39;},
 {&#39;cord_uid&#39;: &#39;a66sszp2&#39;,
  &#39;sha&#39;: &#39;67a2ae408efcb6686a22ae27435c7e4fa10c4a21&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;First 12 patients with coronavirus disease 2019 (COVID-19) in the United States&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20032896&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: More than 93,000 cases of coronavirus disease (COVID-19) have been reported worldwide. We describe the epidemiology, clinical course, and virologic characteristics of the first 12 U.S. patients with COVID-19. Methods: We collected demographic, exposure, and clinical information from 12 patients confirmed by CDC during January 20-February 5, 2020 to have COVID-19. Respiratory, stool, serum, and urine specimens were submitted for SARS-CoV-2 rRT-PCR testing, virus culture, and whole genome sequencing. Results: Among the 12 patients, median age was 53 years (range: 21-68); 8 were male, 10 had traveled to China, and two were contacts of patients in this series. Commonly reported signs and symptoms at illness onset were fever (n=7) and cough (n=8). Seven patients were hospitalized with radiographic evidence of pneumonia and demonstrated clinical or laboratory signs of worsening during the second week of illness. Three were treated with the investigational antiviral remdesivir. All patients had SARS-CoV-2 RNA detected in respiratory specimens, typically for 2-3 weeks after illness onset, with lowest rRT-PCR Ct values often detected in the first week. SARS-CoV-2 RNA was detected after reported symptom resolution in seven patients. SARS-CoV-2 was cultured from respiratory specimens, and SARS-CoV-2 RNA was detected in stool from 7/10 patients. Conclusions: In 12 patients with mild to moderately severe illness, SARS-CoV-2 RNA and viable virus were detected early, and prolonged RNA detection suggests the window for diagnosis is long. Hospitalized patients showed signs of worsening in the second week after illness onset.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#34;Stephanie A. Kujawski; Karen K Wong; Jennifer P. Collins; Lauren Epstein; Marie E. Killerby; Claire M. Midgley; Glen R. Abedi; N. Seema Ahmed; Olivia Almendares; Francisco N. Alvarez; Kayla N. Anderson; Sharon Balter; Vaughn Barry; Karri Bartlett; Karlyn Beer; Michael A. Ben-Aderet; Isaac Benowitz; Holly Biggs; Alison M. Binder; Stephanie R. Black; Brandon Bonin; Catherine M. Brown; Hollianne Bruce; Jonathan Bryant-Genevier; Alicia Budd; Diane Buell; Rachel Bystritsky; Jordan Cates; E. Matt Charles; Kevin Chatham-Stephens; Nora Chea; Howard Chiou; Demian Christiansen; Victoria Chu; Sara Cody; Max Cohen; Erin Conners; Aaron Curns; Vishal Dasari; Patrick Dawson; Traci DeSalvo; George Diaz; Matthew Donahue; Suzanne Donovan; Lindsey M. Duca; Keith Erickson; Mathew D. Esona; Suzanne Evans; Jeremy Falk; Leora R. Feldstein; Martin Fenstersheib; Marc Fischer; Rebecca Fisher; Chelsea Foo; Marielle J. Fricchione; Oren Friedman; Alicia M. Fry; Romeo R. Galang; Melissa M. Garcia; Susa I. Gerber; Graham Gerrard; Isaac Ghinai; Prabhu Gounder; Jonathan Grein; Cheri Grigg; Jeffrey D. Gunzenhauser; Gary I. Gutkin; Meredith Haddix; Aron J. Hall; George Han; Jennifer Harcourt; Kathleen Harriman; Thomas Haupt; Amber Haynes; Michelle Holshue; Cora Hoover; Jennifer C. Hunter; Max W. Jacobs; Claire Jarashow; Michael A. Jhung; Kiran Joshi; Talar Kamali; Shifaq Kamili; Lindsay Kim; Moon Kim; Jan King; Hannah L. Kirking; Amanda Kita-Yarbro; Rachel Klos; Miwako Kobayashi; Anna Kocharian; Kenneth K. Komatsu; Ram Koppaka; Jennifer E. Layden; Yan Li; Scott Lindquist; Stephen Lindstrom; Ruth Link-Gelles; Joana Lively; Michelle Livingston; Kelly Lo; Jennifer Lo; Xiaoyan Lu; Brian Lynch; Larry Madoff; Lakshmi Malapati; Gregory Marks; Mariel Marlow; Glenn E. Mathisen; Nancy McClung; Olivia McGovern; Tristan D. McPherson; Mitali Mehta; Audrey Meier; Lynn Mello; Sung-sil Moon; Margie Morgan; Ruth N. Moro; Janna&#39; Murray; Rekha Murthy; Shannon Novosad; Sara E. Oliver; Jennifer O&#39;Shea; Massimo Pacilli; Clinton R. Paden; Mark A. Pallansch; Manisha Patel; Sajan Patel; Isabel Pedraza; Satish K. Pillai; Talia Pindyck; Ian Pray; Krista Queen; Nichole Quick; Heather Reese; Brian Rha; Heather Rhodes; Susan Robinson; Philip Robinson; Melissa Rolfes; Janell Routh; Rachel Rubin; Sarah L. Rudman; Senthilkumar K. Sakthivel; Sarah Scott; Christopher Shepherd; Varun Shetty; Ethan A. Smith; Shanon Smith; Bryan Stierman; William Stoecker; Rebecca Sunenshine; Regina Sy-Santos; Azaibi Tamin; Ying Tao; Dawn Terashita; Natalie J. Thornburg; Suxiang Tong; Elizabeth Traub; Ahmet Tural; Anna Uehara; Timothy M. Uyeki; Grace Vahey; Jennifer R. Verani; Elsa Villarino; Megan Wallace; Lijuan Wang; John T. Watson; Matthew Westercamp; Brett Whitaker; Sarah Wilkerson; Rebecca C. Woodruff; Jonathan M. Wortham; Tiffany Wu; Amy Xie; Anna Yousaf; Matthew Zahn; Jing Zhang&#34;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20032896&#39;},
 {&#39;cord_uid&#39;: &#39;fn9b7adi&#39;,
  &#39;sha&#39;: &#39;81f08d610ed4c61197c816bdbe5f23b14670dd47&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The effect of control strategies that reduce social mixing on outcomes of the COVID-19 epidemic in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033050&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND: In December 2019, a novel strain of SARS-CoV-2 emerged in Wuhan, China. Since then, the city of Wuhan has taken unprecedented measures and efforts in response to the outbreak. METHODS: We quantified the effects of control measures on population contact patterns in Wuhan, China, to assess their effects on the progression of the outbreak. We included the latest estimates of epidemic parameters from a transmission model fitted to data on local and internationally exported cases from Wuhan in the age-structured epidemic framework. Further, we looked at the age-distribution of cases. Lastly, we simulated lifting of the control measures by allowing people to return to work in a phased-in way, and looked at the effects of returning to work at different stages of the underlying outbreak. FINDINGS: Changes in mixing patterns may have contributed to reducing the number of infections in mid-2020 by 92% (interquartile range: 66-97%). There are benefits to sustaining these measures until April in terms of reducing the height of the peak, overall epidemic size in mid-2020 and probability that a second peak may occur after return to work. However, the modelled effects of social distancing measures vary by the duration of infectiousness and the role school children play in the epidemic. INTERPRETATION: Restrictions on activities in Wuhan, if maintained until April, would likely contribute to the reduction and delay the epidemic size and peak, respectively. However, there are some limitations to the analysis, including large uncertainties around estimates of R0 and the duration of infectiousness.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Kiesha Prem; Yang Liu; Tim Russell; Adam J Kucharski; Rosalind M Eggo; Nicholas Davies; Mark Jit; Petra Klepac&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033050&#39;},
 {&#39;cord_uid&#39;: &#39;9ynljm2r&#39;,
  &#39;sha&#39;: &#39;221f592feb12c19df1f3e619021205e247d39b46&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033068&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The illness progress of partial patient of COVID-19 is rapid and the mortality rate is high.we aim to describe the clinical features in death cases with COVID-19. Methods In this single center, observational study, We recruited all Death Cases with COVID-19 from Dec 30, 2019 to Feb 16, 2020 in Intensive care unit of Wuhan Jinyintan Hospital.Demographics, basic diseases, X-ray/CT results, possible therapy strategies and test results when their entrance into admission, ICU and 48 h before death were collected and analyzed. Results This study involved 101 COVID-19 dead cases in Intensive care unit of Wuhan Jinyintan Hospital.47 patients went directly to the ICU because of critical condition, and 54 patients were transferred to ICU with aggravated condition.57 (56.44%) were laboratory confirmed by RT-PCR, and 44 (43.6%) were consistent with clinical diagnostic criteria.The cases included 64 males and 37 females with average age of 65.46 years (SD 9.74). The blood type distribution was significantly different, with type A 44.44%, type B 29.29%, type AB 8.08% and type O 18.19%.The clinical manifestations of new coronavirus pneumonia are non-specific,the common symptom was fever (91 [90.10%] of 101 patients),Cough (69[68.32%]) and dyspnea (75[74.26%]). Neutrophils, PCT, CRP,IL-6,D-dimer gradually increased as time went on.Myocardial enzymes were abnormal in most patients at admission,with the progress of the disease, myocardial damage indicators were significantly increased.61(60.40%) used antiviral drugs,59(58.42%) used glucocorticoids, 63.37% used intravenous immunoglobulins, and 44.55% used thymosin preparations. All patients received antibiotic treatment, 63(62.38%) used restricted antibiotics, 23(22.78%) used antifungal drugs.84(83.17%) used non-invasive ventilator or high-flow oxygen therapy equipment, and 76.24% used invasive mechanical ventilation. 7 patients were treated with ECMO and 8 patients were treated with CRRT.The median time from ARDS to invasive mechanical ventilation was 3.00 days(IQR 0.00-6.00). The duration of invasive mechanical ventilation was 5 days (IQR2.00-8.00). Conclusions Critical COVID-19 can cause fatal respiratory distress syndrome and multiple organ failure with high mortality rate. Heart may be the earliest damaged organ except the lungs. Secondary infection in the later period is worthy of attention.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;JIan Chen; Hua Fan; Lin Zhang; Bin Huang; Muxin Zhu; Yong Zhou; WenHu Yu; Liping Zhu; Shaohui Cheng; Xiaogen Tao; Huan Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033068&#39;},
 {&#39;cord_uid&#39;: &#39;hnm54k4r&#39;,
  &#39;sha&#39;: &#39;9ae476404f7ef1ec1ede965f0b898f31a5bf5a81&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological, Clinical Characteristics and Outcome of Medical Staff Infected with COVID-19 in Wuhan, China: A Retrospective Case Series Analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033118&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Backgrounds Since December 2019, a novel coronavirus epidemic has emerged in Wuhan city, China and then rapidly spread to other areas. As of 20 Feb 2020, a total of 2,055 medical staff confirmed with coronavirus disease 2019 (COVID-19) caused by SARS-Cov-2 in China had been reported. We sought to explore the epidemiological, clinical characteristics and prognosis of novel coronavirus-infected medical staff. Methods In this retrospective study, 64 confirmed cases of novel coronavirus-infected medical staff admitted to Union Hospital, Wuhan between 16 Jan, 2020 to 15 Feb, 2020 were included. Two groups concerned were extracted from the subjects based on duration of symptoms: group 1 (&lt;= 10 days) and group 2 (&gt;10 days). Epidemiological and clinical data were analyzed and compared across groups. The Kaplan-Meier plot was used to inspect the change in hospital discharge rate. The Cox regression model was utilized to identify factors associated with hospital discharge. Findings The median age of medical staff included was 35 years old. 64% were female and 67% were nurses. None had an exposure to Huanan seafood wholesale market or wildlife. A small proportion of the cohort had contact with specimens (5%) as well as patients in fever clinics (8%) and isolation wards (5%). Fever (67%) was the most common symptom, followed by cough (47%) and fatigue (34%). The median time interval between symptoms onset and admission was 8.5 days. On admission, 80% of medical staff showed abnormal IL-6 levels and 34% had lymphocytopenia. Chest CT mainly manifested as bilateral (61%), subpleural (80%) and ground-glass (52%) opacities. During the study period, no patients was transferred to intensive care unit or died, and 34 (53%) had been discharged. Higher body mass index (BMI) (HR 0.14; 95% CI 0.03-0.73), fever (HR 0.24; 95% CI 0.09-0.60) and higher levels of IL-6 on admission (HR 0.31; 95% CI 0.11-0.87) were unfavorable factors for discharge. Interpretation In this study, medical staff infected with COVID-19 have relatively milder symptoms and favorable clinical course, which may be partly due to their medical expertise, younger age and less underlying diseases. Smaller BMI, absence of fever symptoms and normal IL-6 levels on admission are favorable for discharge for medical staff. Further studies should be devoted to identifying the exact patterns of SARS-CoV-2 infection among medical staff.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Jie Liu; Liu Ouyang; Pi Guo; Hai sheng Wu; Peng Fu; Yu liang Chen; Dan Yang; Xiao yu Han; Yu kun Cao; Osamah Alwalid; Juan Tao; Shu yi Peng; He shui Shi; Fan Yang; Chuan sheng Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033118&#39;},
 {&#39;cord_uid&#39;: &#39;289fu2yb&#39;,
  &#39;sha&#39;: &#39;3e6dda036f3d57c4b1b5b318225ce2a1fccb6ab7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical features of imported cases of coronavirus disease 2019 in Tibetan patients in the Plateau area&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033126&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Coronavirus disease 2019 (COVID-19), caused by SARS-CoV-2, has rapidly spread throughout China, but the clinical characteristics of Tibetan patients living in the Qinghai-Tibetan plateau are unknown. We aimed to investigate the epidemiological, clinical, laboratory and radiological characteristics of these patients. We included 67 Tibetan patients with confirmed SARS-CoV-2 infection. The patients were divided into two groups based on the presence of clinical symptoms at admission, with 31 and 36 patients in the symptomatic and asymptomatic groups, respectively. The epidemiological, clinical, laboratory and radiological characteristics were extracted and analysed. No patient had a history of exposure to COVID-19 patients from Wuhan or had travelled to Wuhan. The mean age of Tibetan patients was 39.3 years and 59% of the patients were male. Seven patients presented with fever on admission and lymphocytopenia was present in 20 patients. 47 patients had abnormal chest CTs at admission instead of stating that 20 were unchanged. Lactate dehydrogenase levels were increased in 31 patients. Seven patients progressed to severe COVID-19; however, after treatment, their condition was stable. No patients died. Of the 36 asymptomatic patients, the mean age was younger than the symptomatic group (34.4vs 44.9 years, P=0.02). Lymphocyte count and prealbumin levels were higher in the asymptomatic group than the group with clinical symptoms (1.6 vs 1.3 and 241.8 vs 191.9, respectively; P&lt;0.05). Imported cases of COVID-19 in Tibetan patients were generally mild in this high-altitude area. Absence of fever or radiologic abnormalities on initial presentation were common&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Yu Lei; yunping lan; jianli lu; xiaobo huang; bamu silang; fan zeng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033126&#39;},
 {&#39;cord_uid&#39;: &#39;7kzaglt2&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Age specificity of cases and attack rate of novel coronavirus disease (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033142&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Age distribution of the cases with novel coronavirus disease (COVID-19) is rather different from that of influenza. We examined the age distribution of COVID-19 cases in Japan from January to March, 2020. Children are less likely to be diagnosed as cases, and moreover, the risk of disease given exposure among children appears to be low. Both the overall risk and the conditional risk of disease given exposure are likely to be the highest among adults aged from 50-69 years. The most plausible explanation that we believe is immune imprinting to a similar virus among adults.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Kenji Mizumoto; Ryosuke Omori; Hiroshi Nishiura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033142&#39;},
 {&#39;cord_uid&#39;: &#39;0d6o5w8z&#39;,
  &#39;sha&#39;: &#39;ca88735399ff43d0e673876200655099f06f5567&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Ascertainment rate of novel coronavirus disease (COVID-19) in Japan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033183&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We analyzed the epidemiological dataset of confirmed cases with COVID-19 in Japan as of 28 February 2020 and estimated the number of severe and non-severe cases, accounting for under-ascertainment. The ascertainment rate of non-severe cases was estimated at 0.44 (95% confidence interval: 0.37, 0.50), indicating that unbiased number of non-cases would be more than twice the reported count. Severe cases are twice more likely diagnosed and reported than other cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Ryosuke Omori; Kenji Mizumoto; Hiroshi Nishiura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033183&#39;},
 {&#39;cord_uid&#39;: &#39;hiw576og&#39;,
  &#39;sha&#39;: &#39;a48bc1745fa15021b49890385022bbe7c4b1076b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Aerosol and surface stability of HCoV-19 (SARS-CoV-2) compared to SARS-CoV-1&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033217&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A novel human coronavirus, now named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, referred to as HCoV-19 here) that emerged in Wuhan, China in late 2019 is now causing a pandemic. Here, we analyze the aerosol and surface stability of HCoV-19 and compare it with SARS-CoV-1, the most closely related human coronavirus.2 We evaluated the stability of HCoV-19 and SARS-CoV-1 in aerosols and on different surfaces and estimated their decay rates using a Bayesian regression model&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Neeltje van Doremalen; Trenton Bushmaker; Dylan Morris; Myndi Holbrook; Amandine Gamble; Brandi Williamson; Azaibi Tamin; Jennifer Harcourt; Natalie Thornburg; Susan Gerber; Jamie Lloyd-Smith; Emmie de Wit; Vincent Munster&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033217&#39;},
 {&#39;cord_uid&#39;: &#39;k4l45ene&#39;,
  &#39;sha&#39;: &#39;8bb82a543720a74015bce24945dc0955b666ae7d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The demand for inpatient and ICU beds for COVID-19 in the US: lessons from Chinese cities&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033241&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Sustained spread of SARS-CoV-2 has happened in major US cities. Capacity needs in Chinese cities could inform the planning of local healthcare resources. Methods: We described the intensive care unit (ICU) and inpatient bed needs for confirmed COVID-19 patients in two Chinese cities (Wuhan and Guangzhou) from January 10 to February 29, 2020, and compared the timing of disease control measures in relation to the timing of SARS-CoV-2 community spread. We estimated the peak ICU bed needs in US cities if a Wuhan-like outbreak occurs. Results: In Wuhan, strict disease control measures were implemented six weeks after sustained local transmission of SARS-CoV-2. Between January 10 and February 29, COVID-19 patients accounted for an average of 637 ICU patients and 3,454 serious inpatients on each day. During the epidemic peak, 19,425 patients (24.5 per 10,000 adults) were hospitalized, 9,689 (12.2 per 10,000 adults) were considered to be in serious condition, and 2,087 patients (2.6 per 10,000 adults) needed critical care per day. In Guangzhou, strict disease control measures were implemented within one week of case importation. Between January 24 and February 29, COVID-19 accounted for an average of 9 ICU patients and 20 inpatients on each day. During the epidemic peak, 15 patients were in critical condition, and 38 were classified as serious. If a Wuhan-like outbreak were to happen in a US city, the need for healthcare resources may be higher in cities with a higher prevalence of vulnerable populations. Conclusion: Even after the lockdown of Wuhan on January 23, the number of seriously ill COVID-19 patients continued to rise, exceeding local hospitalization and ICU capacities for at least a month. Plans are urgently needed to mitigate the effect of COVID-19 outbreaks on the local healthcare system in US cities.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Ruoran Li; Caitlin Rivers; Qi Tan; Megan B Murray; Eric Toner; Marc Lipsitch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033241&#39;},
 {&#39;cord_uid&#39;: &#39;10n2u1b1&#39;,
  &#39;sha&#39;: &#39;4cc91ce442f6fd13456d4e51508605fbcf891ecd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimates of the severity of COVID-19 disease&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033357&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: A range of case fatality ratio (CFR) estimates for COVID 19 have been produced that differ substantially in magnitude. Methods: We used individual-case data from mainland China and cases detected outside mainland China to estimate the time between onset of symptoms and outcome (death or discharge from hospital). We next obtained age-stratified estimates of the CFR by relating the aggregate distribution of cases by dates of onset to the observed cumulative deaths in China, assuming a constant attack rate by age and adjusting for the demography of the population, and age and location-based under ascertainment. We additionally estimated the CFR from individual linelist data on 1,334 cases identified outside mainland China. We used data on the PCR prevalence in international residents repatriated from China at the end of January 2020 to obtain age-stratified estimates of the infection fatality ratio (IFR). Using data on age stratified severity in a subset of 3,665 cases from China, we estimated the proportion of infections that will likely require hospitalisation. Findings: We estimate the mean duration from onset-of-symptoms to death to be 17.8 days (95% credible interval, crI 16.9,19.2 days) and from onset-of-symptoms to hospital discharge to be 22.6 days (95% crI 21.1,24.4 days). We estimate a crude CFR of 3.67% (95% crI 3.56%,3.80%) in cases from mainland China. Adjusting for demography and under-ascertainment of milder cases in Wuhan relative to the rest of China, we obtain a best estimate of the CFR in China of 1.38% (95% crI 1.23%,1.53%) with substantially higher values in older ages. Our estimate of the CFR from international cases stratified by age (under 60 or 60 and above) are consistent with these estimates from China. We obtain an overall IFR estimate for China of 0.66% (0.39%,1.33%), again with an increasing profile with age. Interpretation: These early estimates give an indication of the fatality ratio across the spectrum of COVID-19 disease and demonstrate a strong age-gradient in risk.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Robert Verity; Lucy C Okell; Ilaria Dorigatti; Peter Winskill; Charles Whittaker; Natsuko Imai; Gina Cuomo-Dannenburg; Hayley Thompson; Patrick Walker; Han Fu; Amy Dighe; Jamie Griffin; Anne Cori; Marc Baguelin; Sangeeta Bhatia; Adhiratha Boonyasiri; Zulma M Cucunuba; Rich Fitzjohn; Katy A M Gaythorpe; Will Green; Arran Hamlet; Wes Hinsley; Daniel Laydon; Gemma Nedjati-Gilani; Steven Riley; Sabine van-Elsand; Erik Volz; Haowei Wang; Yuanrong Wang; Xiayoue Xi; Christl Donnelly; Azra Ghani; Neil Ferguson&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033357&#39;},
 {&#39;cord_uid&#39;: &#39;xckkdci1&#39;,
  &#39;sha&#39;: &#39;b30b43ac5cf19932f2b3c3034e2f4ea34a69e901&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Genomic epidemiology of a densely sampled COVID19 outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033365&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Analysis of genetic sequence data from the pandemic SARS Coronavirus 2 can provide insights into epidemic origins, worldwide dispersal, and epidemiological history. With few exceptions, genomic epidemiological analysis has focused on geographically distributed data sets with few isolates in any given location. Here we report an analysis of 20 whole SARS-CoV 2 genomes from a single relatively small and geographically constrained outbreak in Weifang, People&#39;s Republic of China. Using Bayesian model-based phylodynamic methods, we estimate the reproduction number for the outbreak to be 1.99(95% CI:1.48-3.14). We further estimate the number of infections through time and compare these estimates to confirmed diagnoses by the Weifang Centers for Disease Control. We find that these estimates are consistent with reported cases and there is unlikely to be a large undiagnosed burden of infection over the period we studied.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Erik Volz; Han Fu; Haowei Wang; Xiauoyue Xi; Wei Chen; Dehui Liu; Yingying Chen; Mengmeng Tian; Wei Tan; Junjie Zai; Wanying Zan; Xinguang Li; Qing Nie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033365&#39;},
 {&#39;cord_uid&#39;: &#39;6xu7q927&#39;,
  &#39;sha&#39;: &#39;f734d47a423cbe54ec0cc9b2dee39470cf74fd9b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Novel Coronavirus (COVID-19) Knowledge and Perceptions: A Survey of Healthcare Workers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033381&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: During the first week of March, the surge of coronavirus disease 2019 (COVID-19) cases reached over 100 countries with more than 100,000 cases. Healthcare authorities have already initiated awareness and preparedness activities beyond borders. A poor understanding of the disease among healthcare workers (HCWs) may result in delayed treatment and the rapid spread of infection. This study aimed to investigate the knowledge and perceptions of HCWs about COVID-19. Methods: A cross-sectional, web-based study was conducted among HCWs about COVID-19 during the first week of March 2020. A 23-item survey instrument was developed and distributed randomly to HCWs using social media; it required 5 minutes to complete. A chi-square test was used to investigate the level of association among variables at the significance level of p&lt;0.05. Results: Of 529 participants, a total of 453 HCWs completed the survey (response rate: 85.6%); 51.6% were males, 32.1% were aged 25-34 years, and most were doctors (30.2%) and medical students (29.6%). Regarding COVID-19, most of the participants used social media to obtain information (61%), and a significant proportion of HCWs had poor knowledge of its transmission (61%) and symptom onset (63.6%) and showed positive perceptions of COVID-19 prevention and control. Factors such as age and profession were associated with inadequate knowledge and poor perception of COVID-19. Conclusion: As the global threat of COVID-19 continues to emerge, it is critical to improve the knowledge and perceptions of HCWs. Educational interventions are urgently needed to reach HCWs beyond borders, and further studies are warranted.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Akshaya Srikanth Bhagavathula; Wafa Ali Aldhaleei; Jamal Rahmani; Mohammadjavad Ashrafi Mahabadi; Deepak Kumar Bandari&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033381&#39;},
 {&#39;cord_uid&#39;: &#39;98jz8tox&#39;,
  &#39;sha&#39;: &#39;14220553c9160c06bb99f59d59e6a8ecf3595226&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Impact of School Closure for COVID-19 on the US Healthcare Workforce and the Net Mortality Effects&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033415&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background COVID-19 is leading to the implementation of social distancing policies around the world and in the United States, including school closures. The evidence that mandatory school closures reduce cases and ultimately mortality mostly comes from experience with influenza or from models that do not include the impact of school closure on the healthcare labor supply or the role of the healthcare labor force in reducing the per infection mortality from the pathogen. There is considerable uncertainty of the incremental effect of school closures on transmission and lives saved from school closures. The likely, but uncertain, benefits from school closure need to be weighed against uncertain, and seldom quantified, costs of healthcare worker absenteeism associated with additional child care obligations. Methods We analyze data from the US Current Population Survey to measure the potential child care obligations for US healthcare workers that will need to be addressed if school closures are employed as a social distancing measure. We account for the occupation within the healthcare sector, state, and household structure to identify the segments of the healthcare labor force that are most exposed to child care obligations from school closures. We use these estimates to identify the critical level for the importance of healthcare labor supply in increasing a patient&#39;s COVID-19 survival probability that would undo the benefits of school closures and ultimately increase cumulative mortality. Findings The US healthcare sector has some of the highest child care obligations in the United States. 29% of healthcare provider households must provide care for children 3-12. Assuming non-working adults or a sibling 13 years old or older can provide child care, leaves 15% of healthcare provider households in need of childcare during a school closure, while 7% of healthcare households are single-parent households. We document the substantial variation within the healthcare system. For example, 35% of medical assistants and 31% of nursing, psychiatric, and home health aide households have child care obligations, while only 24% of emergency medical personnel have childcare obligations. Child care obligations can vary between states by over 10 percentage points. A 15% decline in the healthcare labor force, combined with reasonable parameters for COVID-19 such as a 15% case reduction from school closings and 2% baseline mortality rate implies that a 15% loss in the healthcare labor force must decrease the survival probability per percent healthcare worker lost by 17.6% for a school closure to increase cumulative mortality. This means that the per infection mortality rate cannot increase from 2% to 2.35% when the healthcare workforce declines by 15%; otherwise, school closures will lead to a greater number of deaths than they prevent. For school closures to unambiguously provide a net reduction in COVID-19 mortality with these parameters, the school closures must reduce cases by over 25%. Conclusion School closures come with many tradeoffs. Setting aside economic costs, school closures implemented to reduce COVID-19 spread create unintended childcare obligations, which are particularly large in healthcare occupations. Detailed data are provided to help public health officials make informed decisions about the tradeoffs associated with closing schools. The results suggest that it is unclear if the potential contagion prevention from school closures justifies the potential loss of healthcare workers from the standpoint of reducing cummulative mortality.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Jude Bayham; Eli P Fenichel&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033415&#39;},
 {&#39;cord_uid&#39;: &#39;z7r45291&#39;,
  &#39;sha&#39;: &#39;ac3e33b6ed12e43208d76bf455f3b81067e6ac6d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Lessons drawn from China and South Korea for managing COVID-19 epidemic: insights from a comparative modeling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033464&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We conducted a comparative study of COVID-19 epidemic in three different settings: mainland China, the Guangdong province of China and South Korea, by formulating two disease transmission dynamics models incorporating epidemic characteristics and setting-specific interventions, and fitting the models to multi-source data to identify initial and effective reproduction numbers and evaluate effectiveness of interventions. We estimated the initial basic reproduction number for South Korea, the Guangdong province and mainland China as 2.6 (95% confidence interval (CI): (2.5, 2.7)), 3.0 (95%CI: (2.6, 3.3)) and 3.8 (95%CI: (3.5,4.2)), respectively, given a serial interval with mean of 5 days with standard deviation of 3 days. We found that the effective reproduction number for the Guangdong province and mainland China has fallen below the threshold 1 since February 8th and 18th respectively, while the effective reproduction number for South Korea remains high, suggesting that the interventions implemented need to be enhanced in order to halt further infections. We also project the epidemic trend in South Korea under different scenarios where a portion or the entirety of the integrated package of interventions in China is used. We show that a coherent and integrated approach with stringent public health interventions is the key to the success of containing the epidemic in China and specially its provinces outside its epicenter, and we show that this approach can also be effective to mitigate the burden of the COVID-19 epidemic in South Korea. The experience of outbreak control in mainland China should be a guiding reference for the rest of the world including South Korea.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Biao Tang; Fan Xia; Nicola Luigi Bragazzi; Xia Wang; Sha He; Xiaodan Sun; Sanyi Tang; Yanni Xiao; Jianhong Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033464&#39;},
 {&#39;cord_uid&#39;: &#39;z9jk1n7o&#39;,
  &#39;sha&#39;: &#39;f4fecacaff35a0498dee4231408c6b9aa049c51b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modeling the situation of COVID-19 and effects of different containment strategies in China with dynamic differential equations and parameters estimation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033498&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;This paper proposed a quarantine-susceptible-exposed-infectious-resistant (QSEIR) model which considers the unprecedented strict quarantine measures in almost the whole of China to resist the epidemic. We estimated model parameters from published information with the statistical method and stochastic simulation, we found the parameters that achieved the best simulation test result. The next stage involved quantitative predictions of future epidemic developments based on different containment strategies with the QSEIR model, focused on the sensitivity of the outcomes to different parameter choices in mainland China. The main results are as follows. If the strict quarantine measures are being retained, the peak value of confirmed cases would be in the range of [52438, 64090] and the peak date would be expected in the range February 7 to February 19, 2020. During March18-30, 2020, the epidemic would be controlled. The end date would be in the period from August 20 to September 1, 2020. With 80% probability, our prediction on the peak date was 4 days ahead of the real date, the prediction error of the peak value is 0.43%, both estimates are much closer to the observed values compared with published studies. The sensitive analysis indicated that the quarantine measures (or with vaccination) are the most effective containment strategy to control the epidemic, followed by measures to increase the cured rate (like finding special medicine). The long-term simulation result and sensitive analysis in mainland China showed that the QSEIR model is stable and can be empirically validated. It is suggested that the QSEIR model can be applied to predict the development trend of the epidemic in other regions or countries in the world. In mainland China, the quarantine measures can&#39;t be relaxed before the end of March 2020. China can fully resume production with appropriate anti-epidemic measures beginning in early April 2020. The results of this study also implied that other countries now facing the epidemic outbreaks should act more decisively and take in time quarantine measures though it may have negative short-term public and economic consequences.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Xiuli Liu; Geoffrey J.D. Hewings; Shouyang Wang; Minghui Qin; Xin Xiang; Shan Zheng; Xuefeng Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033498&#39;},
 {&#39;cord_uid&#39;: &#39;50zrnau8&#39;,
  &#39;sha&#39;: &#39;f628a6163a739a85fd972c00b7d274823c41bb66&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The time scale of asymptomatic transmission affects estimates of epidemic potential in the COVID-19 outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033514&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We assess the impact of asymptomatic transmission on epidemic potential of novel respiratory pathogens (like COVID-19) -- as measured both by the basic reproduction number (i.e., the expected number of secondary cases generated by an average primary case in a fully susceptible population) and the fraction of new secondary cases attributable to asymptomatic individuals. We show that the impact of asymptomatic transmission depends on generation intervals (i.e., time between when an individual is infected and when that individual infects another person). If the generation-interval distribution of asymptomatic transmission differs from that of symptomatic transmission, then estimates of the basic reproduction number which do not explicitly account for asymptomatic cases may be systematically biased. Specifically, if asymptomatic cases have a shorter generation interval than symptomatic cases, R_0 will be over-estimated, and if they have a longer generation interval, R_0 will be under-estimated. We also show that as the length of asymptomatic generation intervals increase, estimates of the realized proportion of asymptomatic transmission during the exponential phase of the epidemic decrease. Our analysis provides a rationale for assessing the duration of asymptomatic cases of COVID-19 in addition to their prevalence in the population.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Sang Woo Park; Daniel M Cornforth; Jonathan Dushoff; Joshua S Weitz&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033514&#39;},
 {&#39;cord_uid&#39;: &#39;e99athff&#39;,
  &#39;sha&#39;: &#39;6115e9fbaed5e154ff92cb8d02387fec72251dab&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Protocol of a randomized controlled trial testing inhaled Nitric Oxide in mechanically ventilated patients with severe acute respiratory syndrome in COVID-19 (SARS-CoV-2).&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.20033530&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction. Severe acute respiratory syndrome (SARS-CoV-2) due to novel Coronavirus (2019-nCoV) related infection (COVID-19) is characterized by severe ventilation perfusion mismatch leading to refractory hypoxemia. To date, there is no specific treatment available for 2019-nCoV. Nitric oxide is a selective pulmonary vasodilator gas used as a rescue therapy in refractory hypoxemia due to acute respiratory distress syndrome (ARDS). In has also shown in-vitro and clinical evidence that inhaled nitric oxide gas (iNO) has antiviral activity against other strains of coronavirus. The primary aim of this study is to determine whether inhaled NO improves oxygenation in patients with hypoxic SARS-CoV2. This is a multicenter randomized controlled trial with 1:1 individual allocation. Patients will be blinded to the treatment. Methods and analysis. Intubated patients admitted to the intensive care unit with confirmed SARS-CoV-2 infection and severe hypoxemia will be randomized to receive inhalation of NO (treatment group) or not (control group). Treatment will be stopped when patients are free from hypoxemia for more than 24 hours. The primary outcome evaluates levels of oxygenation between the two groups at 48 hours. Secondary outcomes include rate of survival rate at 28 and 90 days in the two groups, time to resolution of severe hypoxemia, time to achieve negativity of SARS-CoV-2 RT-PCR tests. Ethics and dissemination. The study protocol has been approved by the Investigational Review Board of Xijing Hospital (Xian, China) and pending the Partners Human Research Committee (Boston, USA). Recruitment will start after approval of both IRBs and local IRBs at other enrolling centers. Results of this study will be published in scientific journals, presented at scientific meetings, reported through flyers and posters, and published on related website or media in combating against this widespread contagious diseases. Trial registration. Clinicaltrials.gov. NCT submitted&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Chong Lei; Binxiao Su; Hailong Dong; Andrea Bellavia; Raffaele Di Fenza; Bijan Safaee Fakhr; Stefano Gianni; Luigi Giuseppe Grassi; Robert Kacmarek; Caio Cesar Araujo Morais; Riccardo Pinciroli; Emanuele Vassena; Lorenzo Berra&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.20033530&#39;},
 {&#39;cord_uid&#39;: &#39;1qn5y4gc&#39;,
  &#39;sha&#39;: &#39;1759e6c9124b81ef925a0afe855e1b2064918615&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.983064&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractEpidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Gun-Soo Park; Keunbon Ku; Seung-Hwa Beak; Seong Jun Kim; Seung Il Kim; Bum-Tae Kim; Jin-Soo Maeng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.983064&#39;},
 {&#39;cord_uid&#39;: &#39;j5bepvvw&#39;,
  &#39;sha&#39;: &#39;d5a1fed3fc7baf4e31c897ec639995106e9fae78&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.983247&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe recent outbreak of coronavirus disease (COVID-19) caused by SARS-CoV-2 infection in Wuhan, China has posed a serious threat to global public health. To develop specific anti-coronavirus therapeutics and prophylactics, the molecular mechanism that underlies viral infection must first be confirmed. Therefore, we herein used a SARS-CoV-2 spike (S) protein-mediated cell-cell fusion assay and found that SARS-CoV-2 showed plasma membrane fusion capacity superior to that of SARS-CoV. We solved the X-ray crystal structure of six-helical bundle (6-HB) core of the HR1 and HR2 domains in SARS-CoV-2 S protein S2 subunit, revealing that several mutated amino acid residues in the HR1 domain may be associated with enhanced interactions with HR2 domain. We previously developed a pan-coronavirus fusion inhibitor, EK1, which targeted HR1 domain and could inhibit infection by divergent human coronaviruses tested, including SARS-CoV and MERS-CoV. We then generated a series of lipopeptides and found that the EK1C4 was the most potent fusion inhibitor against SARS-CoV-2 S protein-mediated membrane fusion and pseudovirus infection with IC50s of 1.3 and 15.8 nM, about 241- and 149-fold more potent than that of EK1 peptide, respectively. EK1C4 was also highly effective against membrane fusion and infection of other human coronavirus pseudoviruses tested, including SARS-CoV and MERS-CoV, as well as SARSr-CoVs, potently inhibiting replication of 4 live human coronaviruses, including SARS-CoV-2. Intranasal application of EK1C4 before or after challenge with HCoV-OC43 protected mice from infection, suggesting that EK1C4 could be used for prevention and treatment of infection by currently circulating SARS-CoV-2 and emerging SARSr-CoVs.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Shuai Xia; Meiqin Liu; Chao Wang; Wei Xu; Qiaoshuai Lan; Siliang Feng; Feifei Qi; Linlin Bao; Lanying Du; Shuwen Liu; Chuan Qin; Fei Sun; Zhengli Shi; Yun Zhu; Shibo Jiang; Lu Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.983247&#39;},
 {&#39;cord_uid&#39;: &#39;t25skqxk&#39;,
  &#39;sha&#39;: &#39;e476334b4f415f8fde5e2f560fd1ad0fa314af90&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A single locus underlies variation in Caenorhabditis elegans chemotherapeutic responses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.984393&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTPleiotropy, the concept that a single gene controls multiple distinct traits, is prevalent in most organisms and has broad implications for medicine and agriculture. Identifying the molecular mechanisms underlying pleiotropy has the power to unveil previously unknown biological connections between seemingly unrelated traits. Additionally, the discovery of pleiotropic genes increases our understanding of both genetic and phenotypic complexity by characterizing novel gene functions. Quantitative trait locus (QTL) mapping has been used to identify several pleiotropic regions in many organisms. However, gene knockout studies are needed to eliminate the possibility of tightly linked, non-pleiotropic loci. Here, we use a panel of 296 recombinant inbred advanced intercross lines of Caenorhabditis elegans and a high-throughput fitness assay to identify a single large-effect QTL on the center of chromosome V associated with variation in responses to eight chemotherapeutics. We validate this QTL with near-isogenic lines and pair genome-wide gene expression data with drug response traits to perform mediation analysis, leading to the identification of a pleiotropic candidate gene, scb-1. Using deletion strains created by genome editing, we show that scb-1, which was previously implicated in response to bleomycin, also underlies responses to other double-strand DNA break-inducing chemotherapeutics. This finding provides new evidence for the role of scb-1 in the nematode drug response and highlights the power of mediation analysis to identify causal genes.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-09&#39;,
  &#39;authors&#39;: &#39;Kathryn S. Evans; Erik C. Andersen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.984393&#39;},
 {&#39;cord_uid&#39;: &#39;2suz20me&#39;,
  &#39;sha&#39;: &#39;598b8f280571ace476ad56a469e7959918fa7958&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Tilorone: A Broad-Spectrum Antiviral For Emerging Viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.09.984856&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTilorone is a 50-year-old synthetic small-molecule compound with antiviral activity that is proposed to induce interferon after oral administration. This drug is used as a broad-spectrum antiviral in several countries of the Russian Federation. We have recently described activity in vitro and in vivo against the Ebola Virus. After a broad screening of additional viruses, we now describe in vitro activity against Chikungunya virus (CHIK) and Middle Eastern Respiratory Syndrome Coronavirus (MERS-CoV).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-11&#39;,
  &#39;authors&#39;: &#39;Sean Ekins; Peter B. Madrid&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.09.984856&#39;},
 {&#39;cord_uid&#39;: &#39;pc16duhj&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and clinical characteristics of COVID-19 in adolescents and young adults&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20032136&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Adolescents and young adults might play a key role in the worldwide spread of Coronavirus Disease 2019 (COVID-19), because they are more involved in overseas studying, business, working, and travelling. However, the epidemiological and clinical characteristics of them are still unknown. Methods: We collected data of 46 confirmed COVID-19 patients aged 10 to 35 years from the study hospital. The demographics, epidemiological, and clinical data were collected. Several key epidemiological parameters, the asymptomatic cases and transmission to their family members and the clinical characteristics at admission, and during treatment were summarized. RESULTS: Of 46 confirmed patients, 14 patients (47.3%) were aged from 10 to 24 years, and 24 (52.7%) patients were male. The mean incubation period for symptomatic cases was 6.6 days (95% confidence interval (CI) 4.4 - 9.6). The median serial interval was 1.9 days (95% CI 0.4 - 6.2). Three of asymptomatic cases showed the transmission to their family members. Only 1 patient was identified as severe cases at admission. The common symptoms at admission were dry cough (34, 91.0%), and fever (29, 69.0%). Nearly 60% of the patients had showed ground-glass opacity by chest CT findings. Three patients developed acute kidney injury during treatment. Majority of patients (78.3%) were discharged by the end of the follow-up. Conclusions: The adolescent and young adult patients of COVID-19 had a long incubation period, and a short serial interval. The transmission to their family contactors occurred in asymptomatic cases. Few of the study patients have developed complications during treatment.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Jiaqiang Liao; Shibing Fan; Jing Chen; Jianglin Wu; Shunqing Xu; Yuming Guo; Chunhui Li; Xianxiang Zhang; Chuansha Wu; Huaming Mou; Chenxi Song; Feng Li; Guicheng Wu; Jingjing Zhang; Lian Guo; Huawen Liu; Jinglong Lv; Lixin Xu; Chunhui Lang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20032136&#39;},
 {&#39;cord_uid&#39;: &#39;odl1nv4i&#39;,
  &#39;sha&#39;: &#39;9e4b67ef6613c31882bb561d28770203da19771b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effects of Chinese strategies for controlling the diffusion and deterioration of novel coronavirus-infected pneumonia in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20032755&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: In December 2019, an outbreak of new type of coronavirus named COVID-19 occurred in Wuhan, Hubei Province, China. In a very short time, this virus spread rapidly over China, greatly threatening public health and economic development. The Chinese government acted quickly and implemented a series of strategies to prevent diffusion of this disease. We therefore sought to evaluate the effects of these Chinese strategies for controlling the spread of COVID-19. Methods: From the data of cumulative confirmed cases from provincial Health Commission websites of China, we performed model fitting and calculated the growth speed of cumulative confirmed patients. We further analyzed the time when this growth speed, the rate of the number of new cases, reached its maximum (Speedmax). Comparing different times to Speedmax of different areas in China, we calculated the dates at which the growth speed began to decline in different areas. Also, The number of plateaus were analyzed. Results: The quartic model showed the best fit. For almost all areas in mainland China, the speed of infections reached Speedmax and began to decline within 14 days; exceptions were Hebei, Heilongjiang, Hainan, Guizhou, and Hubei. The number of plateaus was significantly correlated with the emigration index. However, the distance from other areas to Hubei and the number of plateaus had little influence on when a province or area arrived at Speedmax. Once strict intervention strategies were implemented, diffusion and deterioration of COVID-19 were inhibited quickly and effectively over China. Conclusion: Our study suggests that Chinese strategies are highly effective on controlling the diffusion and deterioration of the novel coronavirus-infected pneumonia. These strategies supply experience and guidelines for other countries to control the COVID-19 epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Xiaoqiang Wang; Weitian Tian; Xin Lv; Yumiao Shi; Xiaoxin Zhou; Weifeng Yu; Diansan Su; Jie Tian&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20032755&#39;},
 {&#39;cord_uid&#39;: &#39;9zs68dnn&#39;,
  &#39;sha&#39;: &#39;5aa0306fa361d31085f6df03d7e71307534b34c6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Revealing the influence of national public health policies for the outbreak of the SARS-CoV-2 epidemic in Wuhan, China through status dynamic modeling&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20032995&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The epidemic caused by SARS-CoV-2 was first reported in Wuhan, China, and now is spreading worldwide. The Chinese government responded to this epidemic with multiple public health policies including locking down the city of Wuhan, establishing multiple temporary hospitals, and prohibiting public gathering events. Here, we constructed a new real-time status dynamic model of SEIO (MH) to reveal the influence of national public health policies and to model the epidemic in Wuhan. Methods A real-time status dynamic model was proposed to model the population of Wuhan in status Susceptible (S), Exposed (E), Infected with symptoms (I), with Medical care (M), and Out of the system (O) daily. Model parameters were fitted according to the daily report of new infections from Jan. 27th, 2020 to Feb. 2nd, 2020. Using the fitted parameters, the epidemic under different conditions was simulated and compared with the current situation. Finding According to our study, the first patient is most likely appeared on Nov. 29th, 2019. There had already been 4,153 infected people and 6,536 exposed ones with the basic reproduction number R_0 of 2.65 before lockdown, whereas R_0 dropped to 1.98 for the first 30 days after the lockdown. The peak point is Feb. 17th, 2020 with 24,115 infected people and the end point is Jun. 17th, 2020. In total, 77,453 people will be infected. If lockdown imposed 7 days earlier, the total number of infected people would be 21,508, while delaying the lockdown by 1-6 days would expand the infection scale 1.23 to 4.94 times. A delay for 7 days would make the epidemic finally out of control. Doubling the number of beds in hospitals would decrease the total infections by 28%, and further investment in bed numbers would yield a diminishing return. Last, public gathering events that increased the transmission parameter by 5% in one single day would increase 4,243 infected people eventually. Interpretation Our model forecasted that the peak time in Wuhan was Feb. 17th, 2020 and the epidemic in Wuhan is now under control. The outbreak of SARS-CoV-2 is currently a global public health threat for all nations. Multiple countries including South Korea, Japan, Iran, Italy, and the United States are suffering from SARS-CoV-2. Our study, which simulated the epidemic in Wuhan, the first city in the world fighting against SARS-CoV-2, may provide useful guidance for other countries in dealing with similar situations.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;tianyi qiu; Han Xiao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20032995&#39;},
 {&#39;cord_uid&#39;: &#39;wmfcwqfw&#39;,
  &#39;sha&#39;: &#39;48f486ccea11b4ba01186e287124316c962857f7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Protocol for a randomized controlled trial testing inhaled nitric oxide therapy in spontaneously breathing patients with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033522&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: the current worldwide outbreak of Coronavirus disease 2019 (COVID-19) due to a novel coronavirus (SARS-CoV-2) is seriously threatening the public health. The number of infected patients is continuously increasing and the need for Intensive Care Unit admission ranges from 5 to 26%. The mortality is reported to be around 3.4% with higher values for the elderly and in patients with comorbidities. Moreover, this condition is challenging the healthcare system where the outbreak reached its highest value. To date there is still no available treatment for SARS-CoV-2. Clinical and preclinical evidence suggests that nitric oxide (NO) has a beneficial effect on the coronavirus-mediated acute respiratory syndrome, and this can be related to its viricidal effect. The time from the symptoms onset to the development of severe respiratory distress is relatively long. We hypothesize that high concentrations of inhaled NO administered during early phases of COVID-19 infection can prevent the progression of the disease. Methods and analysis: This is a multicenter randomized controlled trial. Spontaneous breathing patients admitted to the hospital for symptomatic COVID-19 infection will be eligible to enter the study. Patients in the treatment group will receive inhaled NO at high doses (140-180 parts per million) for 30 minutes, 2 sessions every day for 14 days in addition to the hospital care. Patient in the control group will receive only hospital care. The primary outcome is the percentage of patients requiring endotracheal intubation due to the progression of the disease in the first 28 days from enrollment in the study. Secondary outcomes include mortality at 28 days, proportion of negative test for SARS-CoV-2 at 7 days and time to clinical recovery. Ethics and dissemination: The trial protocol has been approved at the Investigation Review Boards of Xijing Hospital (Xi an, China) and The Partners Human Research Committee of Massachusetts General Hospital (Boston, USA) is pending. Recruitment is expected to start in March 2020. Results of this study will be published in scientific journals, presented at scientific meetings, and on related website or media in fighting this widespread contagious disease. Trial registration. Clinicaltrials.gov. NCT submitted&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Lorenzo Berra; Chong Lei; Binxiao Su; Hailong Dong; Bijan Safaee Fakhr; Luigi Giuseppe Grassi; Raffaele Di Fenza; Stefano Gianni; Riccardo Pinciroli; Emanuele Vassena; Caio Cesar Araujo Morais; Andrea Bellavia; Stefano Spina; Robert Kacmarek; Ryan Carroll&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033522&#39;},
 {&#39;cord_uid&#39;: &#39;1esupl4q&#39;,
  &#39;sha&#39;: &#39;e08d308e5923cb4f741d34a6cd3548a17a04a2b1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia: a case-control study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033605&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Na Li; Lefei Han; Min Peng; Yuxia Lv; Yin Ouyang; Kui Liu; Linli Yue; Qiannan Li; Guoqiang Sun; Lin Chen; Lin Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033605&#39;},
 {&#39;cord_uid&#39;: &#39;1frc4zya&#39;,
  &#39;sha&#39;: &#39;92413565e9d507fdf5550d5123ef420948ae993c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis clinical features of COVID-19 infection in secondary epidemic area and report potential biomarkers in evaluation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033613&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Objective: Based on the clinical characteristics of infected patients with novel coronavirus in secondary epidemic areas,we aimed to identify potential biomarkers for the evaluation of novel coronavirus-infected patients,guide the diagnosis and treatment of this disease in secondary epidemic areas and provide a reference for the clinical prevention and control of this epidemic situation. Methods: The clinical data of 33 patients with respiratory symptoms caused by the novel coronavirus in Wenzhou city from January 15 to February 12,2020,were thoroughly reviewed. At the onset of the disease,we found that the primary symptoms were fever,cough,fatigue,chest tightness,chest pain and specific blood test results. According to the patients&#39;histories,the patients were divided into two groups: those who spent time in the main epidemic area and those who did not spend time in the main epidemic area. The differences in the clinical manifestations between these two groups were analyzed. Results: The main clinical symptoms of patients infected with novel coronavirus in the secondary epidemic area were respiratory tract ailments and systemic symptoms. After grouping patients based on the presence or absence of residency in or travel history to the main epidemic area, there was no significant difference between the baseline data of these two groups, and there were no significant differences in symptoms and signs between the two groups (P&gt;0.05). Some patients had abnormally increased serum amyloid protein A (SAA). There were statistically significant differences in the leukocyte count/C-reactive protein,monocyte ratio/C-reactive protein,neutrophil count/C-reactive protein,monocyte count/C-reactive protein and hemoglobin/C-reactive protein values between the two groups (P &lt; 0.05). Conclusion: Respiratory tract ailments and systemic symptoms were the primary symptoms of novel coronavirus infection in the secondary epidemic area; these symptoms are not typical. The abnormal increase in serum amyloid protein (SAA) may be used as an auxiliary index for diagnosis and treatment. CRP changes before other blood parameters and thus may be an effective evaluation index for patients with COVID-19 infection.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Weiping Ji; Gautam Bishnu; Zhenzhai Cai; Xian Shen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033613&#39;},
 {&#39;cord_uid&#39;: &#39;ha9fn3pr&#39;,
  &#39;sha&#39;: &#39;c2a84d150958eb9c206a388660a12e4f5fa6fa1c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Prediction for Development of COVID-19 in Global Major Epidemic Areas Through Empirical Trends in China by Utilizing State Transition Matrix Model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033670&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Since pneumonia caused by coronavirus disease 2019 (COVID-19) broke out in Wuhan, Hubei province, China, tremendous infected cases has risen all over the world attributed to high transmissibility. We managed to mathematically forecast the inflection point (IFP) of new cases in South Korea, Italy, and Iran, utilizing the transcendental model from Hubei and non-Hubei in China. Methods: We extracted data from reports released by the National Health Commission of the People&#39;s Republic of China (Dec 31, 2019 to Mar 5, 2020) and World Health Organization (Jan 20, 2020 to Mar 5, 2020) as the training set to deduce the arrival of the IFP of new cases in Hubei and non-Hubei on subsequent days and the data from Mar 6 to Mar 9 as validation set. New close contacts, newly confirmed cases, cumulative confirmed cases, non-severe cases, severe cases, critical cases, cured cases, and death data were collected and analyzed. Using this state transition matrix model, the horizon of the IFP of time (the rate of new increment reaches zero) could be predicted in South Korean, Italy, and Iran. Also, through this model, the global trend of the epidemic will be decoded to allocate international medical resources better and instruct the strategy for quarantine. Results: the optimistic scenario (non-Hubei model, daily increment rate of -3.87%), the relative pessimistic scenario (Hubei model, daily increment rate of -2.20%), and the relatively pessimistic scenario (adjustment, daily increment rate of -1.50%) were inferred and modeling from data in China. Matching and fitting with these scenarios, the IFP of time in South Korea would be Mar 6-Mar 12, Italy Mar 10-Mar 24, and Iran is Mar 10-Mar 24. The numbers of cumulative confirmed patients will reach approximately 20k in South Korea, 209k in Italy, and 226k in Iran under fitting scenarios, respectively. There should be room for improvement if these metrics continue to improve. In that case, the IFP will arrive earlier than our estimation. However, with the adoption of different diagnosis criteria, the variation of new cases could impose various influences in the predictive model. If that happens, the IFP of increment will be higher than predicted above. Conclusion: We can affirm that the end of the burst of the epidemic is still inapproachable, and the number of confirmed cases is still escalating. With the augment of data, the world epidemic trend could be further predicted, and it is imperative to consummate the assignment of global medical resources to manipulate the development of COVID-19.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Zhong Zheng; Ke Wu; Zhixian Yao; Junhua Zheng; Jian Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033670&#39;},
 {&#39;cord_uid&#39;: &#39;kgkcgpmq&#39;,
  &#39;sha&#39;: &#39;ee7f56d556c2063d71905df34a589bbf5f1ff051&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effectiveness of isolation and contact tracing for containment and slowing down a COVID-19 epidemic: a modelling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033738&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Novel coronavirus (SARS-CoV-2) is extending its range of transmission in all parts of the world, with substantial variation in rates of transmission and severity of associated disease. Methods: We evaluated whether and under which conditions it is possible to control and slow down a COVID-19 epidemic in the early stages by isolation and contact tracing. We used a stochastic transmission model in which every person generates novel infections according to a probability distribution that is affected by the incubation period distribution (time from infection to symptoms), distribution of the latent period (time from infection to a person becoming infectious), and overall transmissibility. The model distinguishes between close contacts (e.g., within a household) and other contacts in the population. Findings: The analyses showed that transmissibility and the duration of the latent period relative to the duration of incubation period have strong impact on the controllability of the disease. Delays in diagnosis of cases and proportion of asymptomatic cases are key factors for containment and slowing down the epidemic. Interpretation: Isolation and contact tracing can be an effective means to control early epidemics, but only if transmissibility as measured by R0 is in the lower ranges of reported values. Timeliness as well as completeness of tracing and diagnosis of cases are paramount to achieve containment and effective slowing down of the epidemic growth rate.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Mirjam E Kretzschmar; Ganna Rozhnova; Michiel E van Boven&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033738&#39;},
 {&#39;cord_uid&#39;: &#39;tvxaxfqv&#39;,
  &#39;sha&#39;: &#39;f7b069911c90d9deab85becbad3e9a633e0bf57c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Inferring the number of COVID-19 cases from recently reported deaths&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033761&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We estimate the number of COVID-19 cases from newly reported deaths in a population without previous reports. Our results suggest that by the time a single death occurs, hundreds to thousands of cases are likely to be present in that population. This suggests containment via contact tracing will be challenging at this point, and other response strategies should be considered. Our approach is implemented in a publicly available, user-friendly, online tool.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Thibaut Jombart; Kevin van Zandvoort; Tim Russell; Christopher Jarvis; Amy Gimma; Sam Abbott; Samuel Clifford; Sebastian Funk; Hamish Gibbs; Yang Liu; Carl Pearson; Nikos Bosse; Rosalind M Eggo; Adam J Kucharski; John Edmunds&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033761&#39;},
 {&#39;cord_uid&#39;: &#39;ffm5fww3&#39;,
  &#39;sha&#39;: &#39;33a6aa86bb6fe30251d7f2c1f64c5b7a94c391a0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Active or latent tuberculosis increases susceptibility to COVID-19 and disease severity&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033795&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: Risk factors associated with COVID-19, the viral pneumonia originating in Wuhan, China, in Dec 2019, require clarification so that medical resources can be prioritized for those at highest risk of severe COVID-19 complications. Infection with M. tuberculosis (MTB), the pathogen that causes TB and latently infects ~25% of the global population, may be a risk factor for SARS-CoV-2 infection and severe COVID-19 pneumonia. Objective: To determine if latent or active TB increase susceptibility to SARS-COV-19 infection and disease severity, and lead to more rapid development of COVID-19 pneumonia. Design: An observational case-control study of 36 confirmed COVID-19 cases from Shenyang, China, conducted in Feb 2020. Final date of follow-up: Feb 29, 2020. Cases were grouped according to COVID-19 pneumonia severity (mild/moderate, severe/critical), and MTB infection status compared. Comparisons were made with MTB infection data from another case-control study on bacterial/viral pneumonia at Shenyang Chest Hospital. Setting: Multi-center study involving three primary care hospitals in Shenyang, China. Participants: 86 suspected COVID-19 cases from participating primary-care hospitals in Shenyang. All 36 SARS-CoV-2 +ve cases (based on RT-PCR assay) were included. Disease severity was assessed using the Diagnostic and Treatment Guidelines of the National Health Commission of China (v6). Mean age, 47 years (range: 25-79), gender ratio, 1:1. Exposures: Confirmed COVID-19 pneumonia. Interferon-gamma Release Assays (IGRA) were performed using peripheral blood to determine MTB infection. Main Outcome and Measures: Epidemiological, demographic, clinical, radiological, and laboratory data were collected. Comparison of MTB infection status between patients with mild/moderate and severe/critical COVID-19 pneumonia. Results: Mean age of 36 COVID-19 patients: 47 (range: 25-79); M/F: 18/18; Wuhan/Hubei connection: 42%. Mild/moderate cases: 27 (75%); severe/critical: 9 (25%). MTB infection (IGRA+ve): 13 cases (36.11%), including 7 of 9 severe/critical cases. MTB infection rate: higher in COVID-19 (36.11%) than bacterial pneumonia (20%; p=0.0047) and viral pneumonia patients (16.13%; p=0.024). MTB infection more common than other co-morbidities (36.11% vs diabetes: 25%; hypertension: 22.2%; coronary heart disease: 8.33%; COPD: 5.56%). MTB co-infection linked with disease severity (severe/critical 78% vs mild/moderate cases 22%; p=0.0049), and rate of disease progression: infection to development of symptoms (MTB+SARS-CoV-2: 6.5+/-4.2 days vs SARS-COV-2: 8.9+/-5.2 days; p=0.073); from symptom development to diagnosed as severe (MTB+SARS-CoV-2: 3.4+/-2.0 days vs SARS-COV-2: 7.5+/-0.5 days; p=0.075). Conclusions and Relevance: MTB infection likely increases susceptibility to SARS-CoV-2, and increases COVID-19 severity, but this requires validation in a larger study. MTB infection status of COVID-19 patients should be checked routinely at hospital admission.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Yongyu Liu; Lijun Bi; Yu Chen; Yaguo Wang; Joy Fleming; Yanhong Yu; Ye Gu; Chang Liu; Lichao Fan; Xiaodan Wang; Moxin Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033795&#39;},
 {&#39;cord_uid&#39;: &#39;0xzsa21a&#39;,
  &#39;sha&#39;: &#39;fd6163a04c4ac0100a7c9be19def78d2171eac47&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prediction of the COVID-19 outbreak based on a realistic stochastic model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033803&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The current outbreak of coronavirus disease 2019 (COVID-19) has become a global crisis due to its quick and wide spread over the world. A good understanding of the dynamic of the disease would greatly enhance the control and prevention of COVID-19. However, to the best of our knowledge, the unique features of the outbreak have limited the applications of all existing models. In this paper, a novel stochastic model is proposed which aims to account for the unique transmission dynamics of COVID-19 and capture the effects of intervention measures implemented in Mainland China. We find that, (1) instead of aberration, there is a remarkable amount of asymptomatic individuals, (2) an individual with symptoms is approximately twice more likely to pass the disease to others than that of an asymptomatic patient, (3) the transmission rate has reduced significantly since the implementation of control measures in Mainland China, (4) it is expected that the epidemic outbreak would be contained by early March in the the selected provinces and cities.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Yuan Zhang; Chong You; Zhenghao Cai; Jiarui Sun; Wenjie Hu; Xiao-Hua Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033803&#39;},
 {&#39;cord_uid&#39;: &#39;j181i5pr&#39;,
  &#39;sha&#39;: &#39;11edcc84add7b4bfe1f7cc84068a06ff9b7398b2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Strongly heterogeneous transmission of COVID-19 in mainland China: local and regional variation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033852&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of novel coronavirus disease 2019 (COVID-19) started in the city of Wuhan, China, with a period of rapid initial spread. Transmission on a regional and then national scale was promoted by intense travel during the holiday period of the Chinese New Year. We studied the variation in transmission of COVID-19, locally in Wuhan, as well as on a larger spatial scale, among different cities and even among provinces in mainland China. Methods In addition to reported numbers of new cases, we have been able to assemble detailed contact data for some of the initial clusters of COVID-19. This enabled estimation of the serial interval for clinical cases, as well as reproduction numbers for small and large regions. Findings We estimated the average serial interval was 4.8 days. For early transmission in Wuhan, any infectious case produced as many as four new cases, transmission outside Wuhan was less intense, with reproduction numbers below two. During the rapid growth phase of the outbreak the region of Wuhan city acted as a hot spot, generating new cases upon contact, while locally, in other provinces, transmission was low. Interpretation COVID-19 is capable of spreading very rapidly. The sizes of outbreak in provinces of mainland China mainly depended on the numbers of cases imported from Wuhan as the local reproduction numbers were low. The COVID-19 epidemic should be controllable with appropriate interventions. Funding No specific funding.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Yuke Wang; Peter F.M. Teunis&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033852&#39;},
 {&#39;cord_uid&#39;: &#39;suhqgmlo&#39;,
  &#39;sha&#39;: &#39;f2f114ba9f0b618a8b2396ab87f2d861ba9fd483&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Virus strain of a mild COVID-19 patient in Hangzhou representing a new trend in SARS-CoV-2 evolution related to Furin cleavage site&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.20033944&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We found, in our 788 confirmed COVID-19 patients, the decreased rate of severe/critical type, increased liver/kidney damage and prolonged period of nuclear acid positivity during virus dissemination, when compared with Wuhan. To investigate the underlining mechanism, we isolated one strain of SARS- CoV-2 (ZJ01) in mild COVID-19 patient and found the existence of 35 specific gene mutation by gene alignment. Further phylogenetic analysis and RSCU heat map results suggested that ZJ01 may be a potential evolutionary branch of SARS-CoV-2. We classified 54 strains of viruses worldwide (C/T type) based on the base (C or T) at positions 8824 and 28247. ZJ01 were both T at these two sites, becoming the only TT type currently identified in the world. The prediction of Furin cleavage site (FCS) and the sequence alignment of virus family indicated that FCS may be an important site of coronavirus evolution. ZJ01 had mutations near FCS (F1-2), which caused changes in the structure and the electrostatic distribution of the S protein surface, further affecting the binding capacity of Furin. Single cell sequencing and ACE2-Furin co-expression results confirmed that Furin level was higher in the whole body, especially in glands, liver, kidney and colon while FCS may help SARS-CoV-2 infect these organs. The evolutionary pattern of SARS-CoV-2 towards FCS formation may result in its clinical symptom becoming closer to HKU-1 and OC43 (the source of FCS sequence-PRRA) caused influenza, further showing potential in differentiating into mild COVID-19 subtypes.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Jin Xi; Kangli Xu; Penglei Jiang; Jiangshan Lian; Shaorui Hao; Hongyu Jia; Hangping Yao; Yimin Zhang; Ruoheng Zheng; Dong Chen; Jinmei Yao; Jianhua Hu; Jianguo Gao; Jian Shen; Liang Wen; Yue Ren; Guodong Yu; Xiaoyan Wang; Yingfeng Lu; Xiaopeng Yu; Liang Yu; Dairong Xiang; Lin Zheng; Nanping Wu; Xiangyun Lu; Linfang Cheng; Fumin Liu; Haibo Wu; Changzhong Jin; Xiaofeng Yang; Pengxu Qian; Yunqing Qiu; Jifang Sheng; Tingbo Liang; Lanjuan Li; Yida Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.20033944&#39;},
 {&#39;cord_uid&#39;: &#39;wpywh8c9&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#34;A proposal of alternative primers for the ARTIC Network&#39;s multiplex PCR to improve coverage of SARS-CoV-2 genome sequencing&#34;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.985150&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole-genome analysis of the novel coronavirus, SARS-CoV-2, soon after the start of COVID-19 epidemics was realized. The primer set was adapted by many researchers worldwide and has already contributed to the high-quality and prompt genome epidemiology of this rapidly spreading viral disease. We have also seen the great performance of their primer set and protocol; the primer set amplifies all desired 98 PCR amplicons with fairly small amplification bias from clinical samples with relatively high viral load. However, we also observed an acute drop of reads derived from some amplicons especially amplicon 18 and 76 in &#34;pool 2&#34; as a sample\&#39;s viral load decreases. We suspected this low coverage issue was due to dimer formation between primers targeting those two amplicons. Indeed, replacement of just one of those primers, nCoV-2019_76_RIGHT, to a newly designed primer resulted in a drastic improvement of coverages at both regions targeted by the amplicons 18 and 76. Given this result, we further replaced four primers in &#34;pool 1&#34; with each respective alternative. These modifications also improved coverage in eight amplicons particularly in samples with low viral load. The results of our experiments clearly indicate that primer dimer formation is one critical cause of coverage bias in ARTIC protocol. Importantly, some of the problematic primers are detectable by observing primer dimers in raw NGS sequence reads and replacing them with alternatives as shown in this study. We expect a continuous improvement of the ARTIC primer set will extend the limit for completion of SARS-CoV-2 genomes to samples with lower viral load, that supports better genomic epidemiology and mitigation of spread of this pathogen.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Kentaro Itokawa; Tsuyoshi Sekizuka; Masanori Hashino; Rina Tanaka; Makoto Kuroda&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.985150&#39;},
 {&#39;cord_uid&#39;: &#39;qbi5998f&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;In silico Design of novel Multi-epitope recombinant Vaccine based on Coronavirus surface glycoprotein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.985499&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIt is of special significance to find a safe and effective vaccine against coronavirus disease 2019 (COVID-19) that can induce T cell and B cell -mediated immune responses. There is currently no vaccine to prevent COVID-19. In this project, a novel multi-epitope vaccine for COVID-19 virus based on surface glycoprotein was designed through application of bioinformatics methods. At the first, seventeen potent linear B-cell and T-cell binding epitopes from surface glycoprotein were predicted in silico, then the epitopes were joined together via different linkers. The ability of the selected epitopes to induce interferon-gamma was evaluate using IFNepitope web server. One final vaccine was constructed which composed of 398 amino acids and attached to 50S ribosomal protein L7/L12 as adjuvant. Physicochemical properties, as well as antigenicity in the proposed vaccines, were checked for defining the vaccine stability and its ability to induce cell-mediated immune responses. Three-dimensional structure of the mentioned vaccine was subjected to the molecular docking studies with MHC-I and MHC-II molecules. The results proposed that the multi-epitope vaccine with 50S ribosomal protein L7/L12 was a stable construct with high aliphatic content and high antigenicity.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-10&#39;,
  &#39;authors&#39;: &#39;Mandana Behbahani&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.985499&#39;},
 {&#39;cord_uid&#39;: &#39;y12cq60a&#39;,
  &#39;sha&#39;: &#39;7698ec48a89a682cb52c066e293567fa3e83b7dd&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Larger viral genome size facilitates emergence of zoonotic diseases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.986109&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-11&#39;,
  &#39;authors&#39;: &#39;Richard E. Grewelle&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.986109&#39;},
 {&#39;cord_uid&#39;: &#39;rzda8x62&#39;,
  &#39;sha&#39;: &#39;13bdc1078ca5091bfd346a4ecc5dacb6566480d9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.986190&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Stephen A. Wells&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.986190&#39;},
 {&#39;cord_uid&#39;: &#39;jpkxjn6e&#39;,
  &#39;sha&#39;: &#39;15bad8368522a05f32dee771e5a8259f0e9cbbfd&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.986398&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.One Sentence SummaryMolecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Esther S. Brielle; Dina Schneidman-Duhovny; Michal Linial&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.986398&#39;},
 {&#39;cord_uid&#39;: &#39;c1n994j6&#39;,
  &#39;sha&#39;: &#39;8df83cc50766ef007425f79d6220d66c19f73f63&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Efficient inactivation of SARS-CoV-2 by WHO-recommended hand rub formulations and alcohols&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.10.986711&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe recent emergence of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causing COVID-19 is a major burden for health care systems worldwide. It is important to address if the current infection control instructions based on active ingredients are sufficient. We therefore determined the virucidal activity of two alcohol-based hand rub solutions for hand disinfection recommended by the World Health Organization (WHO), as well as commercially available alcohols. Efficient SARS-CoV-2 inactivation was demonstrated for all tested alcohol-based disinfectants. These findings show the successful inactivation of SARS-CoV-2 for the first time and provide confidence in its use for the control of COVID-19.ImportanceThe current COVID-19 outbreak puts a huge burden on the world’s health care systems. Without effective therapeutics or vaccines being available, effective hygiene measure are of utmost importance to prevent viral spreading. It is therefore crucial to evaluate current infection control strategies against SARS-CoV-2. We show the inactivation of the novel coronavirus for the first time and endorse the importance of disinfectant-based hand hygiene to reduce SARS-CoV-2 transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Annika Kratzel; Daniel Todt; Philip V’kovski; Silvio Steiner; Mitra L. Gultom; Tran Thi Nhu Thao; Nadine Ebert; Melle Holwerda; Jörg Steinmann; Daniela Niemeyer; Ronald Dijkman; Günter Kampf; Christian Drosten; Eike Steinmann; Volker Thiel; Stephanie Pfaender&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.10.986711&#39;},
 {&#39;cord_uid&#39;: &#39;h5zfmhqj&#39;,
  &#39;sha&#39;: &#39;041bae0a6de2b69979d39460b3f2ee8946534ec2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of the final size of the second phase of the coronavirus epidemic by the logistic model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20024901&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In the note, the logistic growth regression model is used for the estimation of the final size and its peak time of the coronavirus epidemic in China, South Korea, and the rest of the World.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;milan batista&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20024901&#39;},
 {&#39;cord_uid&#39;: &#39;qbxx6yae&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical features and outcomes of 2019 novel coronavirus-infected patients with cardiac injury&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20030957&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Aims To explore the epidemiological and clinical features of 2019 novel coronavirus(2019-nCoV)-infected patients with cardiac injury . Methods and results Data were collected from patients medical records, and we defined cardiac injury according to cardiac biomarker troponin I level &gt; 0.03&gt;ug/L. Among the 291 patients, 15 (5.2%) showed evidence of cardiac injury. Of 16 hospitalized patients with cardiac injury, the median age was 62 years, and 11/15 (73.3%) were men. Underlying cardiovascular diseases in some patients were hypertension (n=7, 46.6%), coronary heart disease (n=3, 20%) and diabetes (n=3, 20%). The most common symptoms at illness onset in patients with cardiac injury were fever (n=11, 73.3%), cough (n=7, 46.7%), headache or fatigue (n=5, 33.3%) and dyspnoea (n=4, 26.6%). These patients had higher systolic pressures, and lower lymphocyte counts and platelet counts, compared with patients without cardiac injury, respectively. Bilateral infiltrates on chest X-ray and elevated C-reactive protein occurred in all patients with cardiac injury. Compared with patients without cardiac injury, patients with cardiac injury were more likely to develop acute respiratory distress syndrome (73.3%), and receive mechanical ventilation (53.4%), continuous renal replacement therapy (33.3%), extracorporeal membrane oxygenation (26.7%) and vasopressor therapy (26.7%) and be admitted to the intensive care unit (73.3%). One patient died during the study. Conclusion Cardiac injury is a common condition among patients infected with 2019-nCoV.Compared with patients without cardiac injury, the clinical outcomes of patients with cardiac injury are relatively worse. Keywords: 2019-nCoV, Cardiac injury, Clinical features&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;youbin liu; Jinglong Li; Dehui liu; Huafeng Song; Chunlin chen; Mingfang lv; Xing pei; Zhongwei Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20030957&#39;},
 {&#39;cord_uid&#39;: &#39;5sfojk7y&#39;,
  &#39;sha&#39;: &#39;bb6317ba4c887c1e5d5409d3727c908234f307ea&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Relationship between the ABO Blood Group and the COVID-19 Susceptibility&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20031096&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;OBJECTIVE To investigate the relationship between the ABO blood group and the COVID-19 susceptibility. DESIGN The study was conducted by comparing the blood group distribution in 2,173 patients with COVID-19 confirmed by SARS-CoV-2 test from three hospitals in Wuhan and Shenzhen, China with that in normal people from the corresponding regions. Data were analyzed using one-way ANOVA and 2-tailed χ2 and a meta-analysis was performed by random effects models. SETTING Three tertiary hospitals in Wuhan and Shenzhen, China. PARTICIPANTS A total of 1,775 patients with COVID-19, including 206 dead cases, from Wuhan Jinyintan Hospital, Wuhan, China were recruited. Another 113 and 285 patients with COVID-19 were respectively recruited from Renmin Hospital of Wuhan University, Wuhan and Shenzhen Third People&#39;s Hospital, Shenzhen, China. MAIN OUTCOME MEASURES Detection of ABO blood groups, infection occurrence of SARS-CoV-2, and patient death RESULTS The ABO group in 3694 normal people in Wuhan showed a distribution of 32.16%, 24.90%, 9.10% and 33.84% for A, B, AB and O, respectively, versus the distribution of 37.75%, 26.42%, 10.03% and 25.80% for A, B, AB and O, respectively, in 1775 COVID-19 patients from Wuhan Jinyintan Hospital. The proportion of blood group A and O in COVID-19 patients were significantly higher and lower, respectively, than that in normal people (both P &lt; 0.001). Similar ABO distribution pattern was observed in 398 patients from another two hospitals in Wuhan and Shenzhen. Meta-analyses on the pooled data showed that blood group A had a significantly higher risk for COVID-19 (odds ratio-OR, 1.20; 95% confidence interval-CI 1.02~1.43, P = 0.02) compared with non-A blood groups, whereas blood group O had a significantly lower risk for the infectious disease (OR, 0.67; 95% CI 0.60~0.75, P &lt; 0.001) compared with non-O blood groups.In addition, the influence of age and gender on the ABO blood group distribution in patients with COVID-19 from two Wuhan hospitals (1,888 patients) were analyzed and found that age and gender do not have much effect on the distribution. CONCLUSION People with blood group A have a significantly higher risk for acquiring COVID-19 compared with non-A blood groups, whereas blood group O has a significantly lower risk for the infection compared with non-O blood groups.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Jiao Zhao; Yan Yang; Han-Ping Huang; Dong Li; Dong-Feng Gu; Xiang-Feng Lu; Zheng Zhang; Lei Liu; Ting Liu; Yu-Kun Liu; Yun-Jiao He; Bin Sun; Mei-Lan Wei; Guang-Yu Yang; Xinghuan Wang; Li Zhang; Xiao-Yang Zhou; Ming-Zhao Xing; Peng George Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20031096&#39;},
 {&#39;cord_uid&#39;: &#39;mz06mkqf&#39;,
  &#39;sha&#39;: &#39;f14434d7d3b68f4fe7f68adba747b7700ccdd6ac&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prolonged presence of SARS-CoV-2 in feces of pediatric patients during the convalescent phase&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20033159&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Severe acute respiratory coronavirus 2 (SARS-CoV-2) is a newly identified virus which mainly spreads from person-to-person. Fecal shedding of SARS-CoV-2 has been constantly reported in patients with coronavirus disease 2019 (COVID-19). Most published studies focus on adult populations, whereas data concerning pediatric patients is relatively scarce. Methods: From January 17, 2020 to March 6, 2020, three pediatric cases of COVID-19 were reported in Qingdao, Shandong Province, China. Epidemiological, clinical, laboratory, and radiological characteristics and treatment data of these children were collected. Real-time fluorescence reverse-transcriptase-polymerase-chain reaction (RT-PCR) was performed to detect SARS-CoV-2 RNA in throat swabs and fecal specimens. Results: All the three pediatric cases were household contacts of adults whose symptoms developed earlier. There has been no evidence showing the virus was transmitted from the children to others. Severity of disease of these children was mild to moderate and fever was the most consistent and predominant symptom at onset of illness (two cases had body temperature higher than 38.5 Celsius). All children showed increased lymphocytes (&gt;4.4*109/L) with normal white blood cell counts on admission. One child had elevated serum levels of procalcitonin and C-reaction protein. Radiological changes were not typical for COVID-19. All children showed good response to supportive treatment. Clearance of SARS-CoV-2 in respiratory tract occurred within two weeks after abatement of fever, whereas persistent presence of viral RNA was found in stools of all children. One case had fecal SARS-CoV-2 turned negative 8 days after throat swabs showing negative, while that of another child lagged behind for 20 days. At the time of writing, one child still had positive results for RT-PCR analysis in stools after negative conversion of viral RNA in respiratory samples (over 19 days behind). Conclusions: Pediatric patients with COVID-19 are very different from adult patients in regards to epidemiological, clinical, laboratory, and radiological characteristics. Prolonged shedding of SARS-CoV-2 in stools of infected children indicates the potential for the virus to be transmitted through fecal excretion. Massive efforts should be made at all levels to prevent spreading of the infection among children after reopening of kindergartens and schools.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Yuhan Xing; Wei Ni; Qin Wu; Wenjie Li; Guoju Li; Jianning Tong; Xiufeng Song; Quansheng Xing&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20033159&#39;},
 {&#39;cord_uid&#39;: &#39;44zduv27&#39;,
  &#39;sha&#39;: &#39;2365df72b5a011f1a1a38b615f6cd87a126d311e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluating the effect of public health intervention on the global-wide spread trajectory of Covid-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20033639&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As COVID-19 evolves rapidly, the issues the governments of affected countries facing are whether and when to take public health interventions and what levels of strictness of these interventions should be, as well as when the COVID-19 spread reaches the stopping point after interventions are taken. To help governments with policy-making, we developed modified auto-encoders (MAE) method to forecast spread trajectory of Covid-19 of countries affected, under different levels and timing of intervention strategies. Our analysis showed public health interventions should be executed as soon as possible. Delaying intervention 4 weeks after March 8, 2020 would cause the maximum number of cumulative cases of death increase from 7,174 to 133,608 and the ending points of the epidemic postponed from Jun 25 to Aug 22.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Zixin Hu; Qiyang Ge; Shudi Li; Li Jin; Momiao Xiong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20033639&#39;},
 {&#39;cord_uid&#39;: &#39;qrzt4zva&#39;,
  &#39;sha&#39;: &#39;4a908bb4c3229f4128139e44e69529f0c2d1666d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Social distance and SARS memory: impact on the public awareness of 2019 novel coronavirus (COVID-19) outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20033688&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This study examines publicly available online search data in China to investigate the spread of public awareness of the 2019 novel coronavirus (COVID-19) outbreak. We found that cities that suffered from SARS and have greater migration ties to the epicentre, Wuhan, had earlier, stronger and more durable public awareness of the outbreak. Our data indicate that forty-eight such cities developed awareness up to 19 days earlier than 255 comparable cities, giving them an opportunity to better prepare. This study suggests that it is important to consider memory of prior catastrophic events as they will influence the public response to emerging threats.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Haohui Chen; Weipan Xu; Cecile Paris; Andrew Reeson; Xun Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20033688&#39;},
 {&#39;cord_uid&#39;: &#39;wxm8kqnd&#39;,
  &#39;sha&#39;: &#39;982689f92bf67e90a43fc142c605da9a5045f756&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimation of instant case fatality rate of COVID-19 in Wuhan and Hubei based on daily case notification data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20034215&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of coronavirus disease 2019 (COVID-19) initially appeared and has most rapidly spread in Wuhan, China. The case fatality rate is the most direct indicator to assess the hazards of an infectious disease. We aimed to estimate the instant fatality rate and cure rate of COVID-19 in Wuhan City and its affiliated Hubei Province. Methods We collected the daily case notification data of COVID-19 from Dec 8, 2019 to Mar 10, 2020 in Wuhan City and Hubei Province officially announced by the Chinese authority. The numbers of daily confirmed/deaths/cured cases and the numbers of daily cumulative confirmed/deaths/cured cases were obtained. The death time and cure time of COVID-19 patients were calculated based on the dates of diagnosis, death and discharge of individual cases. Then the estimated diagnosis dates of deaths and cured cases were obtained on the basis of the median death or medium cure time, respectively. Finally, the instant fatality rate of COVID-19 was calculated according to the numbers of deaths and cured cases on the same estimated diagnosis dates. Results From Jan 1, 2020 to Feb 22, 2020 in Wuhan City, the instant case fatality rate of COVID-19 was 3.4%～19.5% and the instant cured rate was 80.0%～96.6%. The average fatality rate reached 11.4% while the average cure rate was 88.6%. During the same period in Hubei Province, the instant case fatality rate was 3.8%～16.6% and the instant cured rate was 83.4%～96.6%. The average fatality rate and the average cure rate were 9.2% and 91.8%, respectively. Conclusions The fatality rate and cure rate of COVID-19 in Wuhan City and Hubei Province were underestimated. Wuhan showed higher fatality rate and cure rate than the whole Hubei Province did.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Lei Cao; Ting-ting Huang; Jun-xia Zhang; Qi Qin; Si-yu Liu; Hui-min Xue; Ya-xin Gong; Chang-hua Ning; Xiao-tong Shen; Jia-xin Yang; Yan-ni Mi; Xue Xiao; Yongxiao Cao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20034215&#39;},
 {&#39;cord_uid&#39;: &#39;pvb5c4ih&#39;,
  &#39;sha&#39;: &#39;d3de3a027f59c0acc13afa851e270c7fef849db5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Predicting the cumulative number of cases for the COVID-19 epidemic in China from early data&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20034314&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We model the COVID-19 coronavirus epidemic in China. We use early reported case data to predict the cumulative number of reported cases to a final size. The key features of our model are the timing of implementation of major public policies restricting social movement, the identification and isolation of unreported cases, and the impact of asymptomatic infectious cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Zhihua Liu; pierre magal; Ousmane Seydi; Glenn Webb&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20034314&#39;},
 {&#39;cord_uid&#39;: &#39;9607dy2o&#39;,
  &#39;sha&#39;: &#39;3298d34e643c7a157f4226fde679f71194e417b5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Generalized logistic growth modeling of the COVID-19 outbreak in 29 provinces in China and in the rest of the world&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20034363&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: the COVID-19 has been successfully contained in China but is spreading all over the world. We use phenomenological models to dissect the development of the epidemics in China and the impact of the drastic control measures both at the aggregate level and within each province. We use the experience from China to analyze the calibration results on Japan, South Korea, Iran, Italy and Europe, and make future scenario projections. Methods: we calibrate the logistic growth model, the generalized logistic growth model, the generalized growth model and the generalized Richards model to the reported number of infected cases from Jan. 19 to March 10 for the whole of China, 29 provinces in China, four severely affected countries and Europe as a whole. The different models provide upper and lower bounds of our scenario predictions. Results: We quantitatively document four phases of the outbreak in China with a detailed analysis on the heterogenous situations across provinces. Based on Chinese experience, we identify a high risk in Japan with estimated total confirmed cases as of March 25 being 1574 (95% CI: [880, 2372]), and 5669 (95% CI: [988, 11340]) by June. For South Korea, we expect the number of infected cases to approach the ceiling, 7928 (95% CI: [6341, 9754]), in 20 days. We estimate 0.15% (95% CI: [0.03%, 0.30%]) of Italian population to be infected in a positive scenario. We would expect 114867 people infected in Europe in 10 days, in a negative but probable scenario, corresponding to 0.015% European population. Conclusions: The extreme containment measures implemented by China were very effective with some instructive variations across provinces. For other countries, it is almost inevitable to see the continuation of the outbreak in the coming months. Japan and Italy are in serious situations with no short-term end to the outbreak to be expected. There is a significant risk concerning the upcoming July 2020 Summer Olympics in Tokyo. Iran&#39;s situation is highly uncertain with unclear and negative future scenarios, while South Korea is approaching the end of the outbreak. Both Europe and the USA are at early stages of the outbreak, posing significant health and economic risks to the world in absence of serious measures.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Ke Wu; Didier Darcet; Qian Wang; Didier Sornette&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20034363&#39;},
 {&#39;cord_uid&#39;: &#39;x88vw3ga&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Method to Model Outbreaks of New Infectious Diseases with Pandemic Potential such as COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20034512&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The emergence of the novel coronavirus (a.k.a. COVID-19, SARS-CoV-2) out of Wuhan, Hubei Province, China caught the world by surprise. As the outbreak began to spread outside of China, too little was known about the virus to model its transmission with any acceptable accuracy. World governments responded to rampant minformation about the virus leading to collatoral disasters, such as plunging financial markets, that could have been avoided if better models of the outbreak had been available. This is an engineering approach to model the spread of a new infectious disease from sparse data when little is known about the infectious agent itself. The most sought after estimate is the projected number of people who may be contagious, but still be within the incubation period thus remaining asymptomatic and undetected. This parameter is useful for estimating the extent of outbreak among a population over time and what to expect. It is thus important for common sense decision making by governments, organizations, and even individuals to contain, prepare, prevent, react, and safeguard. A Monte Carlo approach is implemented by using incubation period and a variable, called testing efficiency as variables. The Coronavirus outbreak in USA will be used to illustrate the implementation of this modeling approach. Among others it is demonstrated that imposing early travel restrictions from infected countries slowed down the outbreak in the USA by about 26 days.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Willem G Odendaal&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20034512&#39;},
 {&#39;cord_uid&#39;: &#39;rike3ukb&#39;,
  &#39;sha&#39;: &#39;254c1e4d661d7fb5954c855387595c3cde0e8156&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Environmental contamination of the SARS-CoV-2 in healthcare premises: An urgent call for protection for healthcare workers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.20034546&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance A large number of healthcare workers (HCWs) were infected by SARS-CoV-2 during the ongoing outbreak of COVID-19 in Wuhan, China. Hospitals are significant epicenters for the human-to-human transmission of the SARS-CoV-2 for HCWs, patients, and visitors. No data has been reported on the details of hospital environmental contamination status in the epicenter of Wuhan. Objective To investigate the extent to which SARS-CoV-2 contaminates healthcare settings, including to identify function zones of the hospital with the highest contamination levels and to identify the most contaminated objects, and personal protection equipment (PPE) in Wuhan, China. Design A field investigation was conducted to collect the surface swabs in various environments in the hospital and a laboratory experiment was conducted to examine the presence of the SARS-CoV-2 RNA. Setting Six hundred twenty-six surface samples were collected within the Zhongnan Medical Center in Wuhan, China in the mist of the COVID-19 outbreak between February 7 - February 27, 2020. Participants Dacron swabs were aseptically collected from the surfaces of 13 hospital function zones, five major objects, and three major personal protection equipment (PPE). The SARS-CoV-2 RNAs were detected by reverse transcription-PCR (RT-PCR). Main Outcomes and Measures SARS-CoV-2 RNAs Results The most contaminated zones were the intensive care unit specialized for taking care of novel coronavirus pneumonia (NCP) (31.9%), Obstetric Isolation Ward specialized for pregnant women with NCP (28.1%), and Isolation Ward for NCP (19.6%). We classified the 13 zones into four contamination levels. The most contaminated objects are self-service printers (20.0%), desktop/keyboard (16.8%), and doorknob (16.0%). Both hand sanitizer dispensers (20.3%) and gloves (15.4%) were most contaminated PPE. Conclusions and Relevance Many surfaces were contaminated with SARS-CoV-2 across the hospital in various patient care areas, commonly used objects, medical equipment, and PPE. The 13 hospital function zones were classified into four contamination levels. These findings emphasize the urgent need to ensure adequate environmental cleaning, strengthen infection prevention training, and improve infection prevention precautions among HCWs during the outbreak of COVID-19. The findings may have important implications for modifying and developing urgently needed policy to better protect healthcare workers during this ongoing pandemic of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Guangming Ye; Hualiang Lin; Liangjun Chen; Shichan Wang; Zhikun Zeng; Wei Wang; Shiyu Zhang; Terri Rebmann; Yirong Li; Zhenyu Pan; Zhonghua Yang; Ying Wang; Fubing Wang; Zhengmin Qian; Xinghuan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.20034546&#39;},
 {&#39;cord_uid&#39;: &#39;hq5um68k&#39;,
  &#39;sha&#39;: &#39;a59a1e31a7d2edf1be120c4a13a454bed64b1593&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Novel and potent inhibitors targeting DHODH, a rate-limiting enzyme in de novo pyrimidine biosynthesis, are broad-spectrum antiviral against RNA viruses including newly emerged coronavirus SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.983056&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractEmerging and re-emerging RNA viruses occasionally cause epidemics and pandemics worldwide, such as the on-going outbreak of coronavirus SARS-CoV-2. Existing direct-acting antiviral (DAA) drugs cannot be applied immediately to new viruses because of virus-specificity, and the development of new DAA drugs from the beginning is not timely for outbreaks. Thus, host-targeting antiviral (HTA) drugs have many advantages to fight against a broad spectrum of viruses, by blocking the viral replication and overcoming the potential viral mutagenesis simultaneously. Herein, we identified two potent inhibitors of DHODH, S312 and S416, with favorable drug-like and pharmacokinetic profiles, which all showed broad-spectrum antiviral effects against various RNA viruses, including influenza A virus (H1N1, H3N2, H9N2), Zika virus, Ebola virus, and particularly against the recent novel coronavirus SARS-CoV-2. Our results are the first to validate that DHODH is an attractive host target through high antiviral efficacy in vivo and low virus replication in DHODH knocking-out cells. We also proposed the drug combination of DAA and HTA was a promising strategy for anti-virus treatment and proved that S312 showed more advantageous than Oseltamivir to treat advanced influenza diseases in severely infected animals. Notably, S416 is reported to be the most potent inhibitor with an EC50 of 17nM and SI value &gt;5882 in SARS-CoV-2-infected cells so far. This work demonstrates that both our self-designed candidates and old drugs (Leflunomide/Teriflunomide) with dual actions of antiviral and immuno-repression may have clinical potentials not only to influenza but also to COVID-19 circulating worldwide, no matter such viruses mutate or not.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Rui Xiong; Leike Zhang; Shiliang Li; Yuan Sun; Minyi Ding; Yong Wang; Yongliang Zhao; Yan Wu; Weijuan Shang; Xiaming Jiang; Jiwei Shan; Zihao Shen; Yi Tong; Liuxin Xu; Chen Yu; Yingle Liu; Gang Zou; Dimitri Lavillete; Zhenjiang Zhao; Rui Wang; Lili Zhu; Gengfu Xiao; Ke Lan; Honglin Li; Ke Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.983056&#39;},
 {&#39;cord_uid&#39;: &#39;qebbkr6d&#39;,
  &#39;sha&#39;: &#39;d070dc3ef2125fd891f3e023598831e23a145880&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A data-driven drug repositioning framework discovered a potential therapeutic agent targeting COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.986836&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe global spread of SARS-CoV-2 requires an urgent need to find effective therapeutics for the treatment of COVID-19. We developed a data-driven drug repositioning framework, which applies both machine learning and statistical analysis approaches to systematically integrate and mine large-scale knowledge graph, literature and transcriptome data to discover the potential drug candidates against SARS-CoV-2. The retrospective study using the past SARS-CoV and MERS-CoV data demonstrated that our machine learning based method can successfully predict effective drug candidates against a specific coronavirus. Our in silico screening followed by wet-lab validation indicated that a poly-ADP-ribose polymerase 1 (PARP1) inhibitor, CVL218, currently in Phase I clinical trial, may be repurposed to treat COVID-19. Our in vitro assays revealed that CVL218 can exhibit effective inhibitory activity against SARS-CoV-2 replication without obvious cytopathic effect. In addition, we showed that CVL218 is able to suppress the CpG-induced IL-6 production in peripheral blood mononuclear cells, suggesting that it may also have anti-inflammatory effect that is highly relevant to the prevention immunopathology induced by SARS-CoV-2 infection. Further pharmacokinetic and toxicokinetic evaluation in rats and monkeys showed a high concentration of CVL218 in lung and observed no apparent signs of toxicity, indicating the appealing potential of this drug for the treatment of the pneumonia caused by SARS-CoV-2 infection. Moreover, molecular docking simulation suggested that CVL218 may bind to the N-terminal domain of nucleocapsid (N) protein of SARS-CoV-2, providing a possible model to explain its antiviral action. We also proposed several possible mechanisms to explain the antiviral activities of PARP1 inhibitors against SARS-CoV-2, based on the data present in this study and previous evidences reported in the literature. In summary, the PARP1 inhibitor CVL218 discovered by our data-driven drug repositioning framework can serve as a potential therapeutic agent for the treatment of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Yiyue Ge; Tingzhong Tian; Suling Huang; Fangping Wan; Jingxin Li; Shuya Li; Hui Yang; Lixiang Hong; Nian Wu; Enming Yuan; Lili Cheng; Yipin Lei; Hantao Shu; Xiaolong Feng; Ziyuan Jiang; Ying Chi; Xiling Guo; Lunbiao Cui; Liang Xiao; Zeng Li; Chunhao Yang; Zehong Miao; Haidong Tang; Ligong Chen; Hainian Zeng; Dan Zhao; Fengcai Zhu; Xiaokun Shen; Jianyang Zeng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.986836&#39;},
 {&#39;cord_uid&#39;: &#39;ymc93p0w&#39;,
  &#39;sha&#39;: &#39;6de59b8f8122ee30e23ef152639bb402264810f7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.987016&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSteroid compounds, which are expected to have dual functions in blocking host inflammation and MERS-CoV replication, were screened from a chemical library. Within this library, ciclesonide, an inhaled corticosteroid, suppressed human coronavirus replication in cultured cells, but did not suppress replication of respiratory syncytial virus or influenza virus. The effective concentration of ciclesonide to block SARS-CoV-2 (the cause of COVID-19) replication (EC90) was 6.3 μM. After the eleventh consecutive MERS-CoV passage in the presence of ciclesonide, a resistant mutation was generated, which resulted in an amino acid substitution (A25V) in nonstructural protein (NSP) 15, as identified using reverse genetics. A recombinant virus with the mutation was also resistant to ciclesonide suppression of viral replication. These observations suggest that the effect of ciclesonide was specific to coronavirus, suggesting this is a candidate drug for treatment of patients suffering MERS or COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Shutoku Matsuyama; Miyuki Kawase; Naganori Nao; Kazuya Shirato; Makoto Ujike; Wataru Kamitani; Masayuki Shimojima; Shuetsu Fukushi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.987016&#39;},
 {&#39;cord_uid&#39;: &#39;v5v9bjgs&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.987222&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;To date, the SARS-CoV-2 genome has been considered genetically more stable than SARS-CoV or MERS-CoV. Here we report a 382-nt deletion covering almost the entire open reading frame 8 (ORF8) of SARS-CoV-2 obtained from eight hospitalized patients in Singapore. The deletion also removes the ORF8 transcription-regulatory sequence (TRS), which in turn enhances the downstream transcription of the N gene. We also found that viruses with the deletion have been circulating for at least four weeks. During the SARS-CoV outbreak in 2003, a number of genetic variants were observed in the human population [1], and similar variation has since been observed across SARS-related CoVs in humans and bats. Overwhelmingly these viruses had mutations or deletions in ORF8, that have been associated with reduced replicative fitness of the virus [2]. This is also consistent with the observation that towards the end of the outbreak sequences obtained from human SARS cases possessed an ORF8 deletion that may be associated with host adaptation [1]. We therefore hypothesise that the major deletion revealed in this study may lead to an attenuated phenotype of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Yvonne CF Su; Danielle E Anderson; Barnaby E Young; Feng Zhu; Martin Linster; Shirin Kalimuddin; Jenny GH Low; Zhuang Yan; Jayanthi Jayakumar; Louisa Sun; Gabriel Z Yan; Ian H Mendenhall; Yee-Sin Leo; David Chien Lye; Lin-Fa Wang; Gavin JD Smith&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.987222&#39;},
 {&#39;cord_uid&#39;: &#39;mlo8zypy&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A human monoclonal antibody blocking SARS-CoV-2 infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.11.987958&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe emergence of the novel human coronavirus SARS-CoV-2 in Wuhan, China has caused a worldwide epidemic of respiratory disease (COVID-19). Vaccines and targeted therapeutics for treatment of this disease are currently lacking. Here we report a human monoclonal antibody that neutralizes SARS-CoV-2 (and SARS-CoV). This cross-neutralizing antibody targets a communal epitope on these viruses and offers potential for prevention and treatment of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Chunyan Wang; Wentao Li; Dubravka Drabek; Nisreen M.A. Okba; Rien van Haperen; Albert D.M.E. Osterhaus; Frank J.M. van Kuppeveld; Bart L. Haagmans; Frank Grosveld; Berend-Jan Bosch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.11.987958&#39;},
 {&#39;cord_uid&#39;: &#39;ll4rxd9p&#39;,
  &#39;sha&#39;: &#39;0e38333bff68345492526fd39b70d1b18969cb83&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Deep Learning-based Detection for COVID-19 from Chest CT using Weak Label&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20027185&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Accurate and rapid diagnosis of COVID-19 suspected cases plays a crucial role in timely quarantine and medical treatment. Developing a deep learning-based model for automatic COVID-19 detection on chest CT is helpful to counter the outbreak of SARS-CoV-2. A weakly-supervised deep learning-based software system was developed using 3D CT volumes to detect COVID-19. For each patient, the lung region was segmented using a pre-trained UNet; then the segmented 3D lung region was fed into a 3D deep neural network to predict the probability of COVID-19 infectious. 499 CT volumes collected from Dec. 13, 2019, to Jan. 23, 2020, were used for training and 131 CT volumes collected from Jan 24, 2020, to Feb 6, 2020, were used for testing. The deep learning algorithm obtained 0.959 ROC AUC and 0.976 PR AUC. There was an operating point with 0.907 sensitivity and 0.911 specificity in the ROC curve. When using a probability threshold of 0.5 to classify COVID-positive and COVID-negative, the algorithm obtained an accuracy of 0.901, a positive predictive value of 0.840 and a very high negative predictive value of 0.982. The algorithm took only 1.93 seconds to process a single patient&#39;s CT volume using a dedicated GPU. Our weakly-supervised deep learning model can accurately predict the COVID-19 infectious probability in chest CT volumes without the need for annotating the lesions for training. The easily-trained and high-performance deep learning algorithm provides a fast way to identify COVID-19 patients, which is beneficial to control the outbreak of SARS-CoV-2. The developed deep learning software is available at \\url{https://github.com/sydney0zq/covid-19-detection}.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Chuansheng Zheng; Xianbo Deng; Qing Fu; Qiang Zhou; Jiapei Feng; Hui Ma; Wenyu Liu; Xinggang Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20027185&#39;},
 {&#39;cord_uid&#39;: &#39;uj0i2anr&#39;,
  &#39;sha&#39;: &#39;2ae710bf5f60c23c489e4c61245a35a4f85178cb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034231&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The detection of serum antibodies to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is emerging as a new tool for the coronavirus disease-2019 (COVID-19) diagnosis. Since many coronaviruses are sensitive to heat, heating inactivation of samples at 56 ℃ prior to testing is considered a possible method to reduce the risk of transmission, but the effect of heating on the measurement of SARS-CoV-2 antibodies is still unclear. By comparing the levels of SARS-CoV-2 antibodies before and after heat inactivation of serum at 56 ℃ for 30 minutes using a quantitative fluorescence immunochromatographic assay, we shown that heat inactivation significantly interferes with the levels of antibodies to SARS-CoV-2. The IgM levels of all the 34 serum samples (100%) from COVID-19 patients decreased by an average level of 53.56%. The IgG levels were decreased in 22 of 34 samples (64.71%) by an average level of 49.54%. Similar changes can also be observed in the non-COVID-19 diseases group (n=9). Of note, 44.12% of the detected IgM levels were dropped below the cut-off value after heating, suggesting heat inactivation can lead to false-negative results of these samples. Our results indicate that heat inactivation of serum at 56 ℃ for 30 minutes interferes with the immunoanalysis of antibodies to SARS-CoV-2. Heat inactivation prior to immunoanalysis is not recommended and the possibility of false-negative results should be considered if the sample was pre-inactivated by heating.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Xiumei Hu; Taixue An; Bo Situ; Yuhai Hu; Zihao Ou; Qiang Li; Xiaojing He; Ye Zhang; Peifu Tian; Dehua Sun; Yongyu Rui; Qian Wang; Dan Ding; Lei Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034231&#39;},
 {&#39;cord_uid&#39;: &#39;qejmwvst&#39;,
  &#39;sha&#39;: &#39;5470929d340ca0fbb030ce8e00a885a0d730adbe&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Rational evaluation of various epidemic models based on the COVID-19 data of China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034595&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;During the study of epidemics, one of the most significant and also challenging problems is to forecast the future trends, on which all follow-up actions of individuals and governments heavily rely. However, to pick out a reliable predictable model/method is far from simple, a rational evaluation of various possible choices is eagerly needed, especially under the severe threat of COVID-19 epidemics which is spreading worldwide right now. In this paper, based on the public COVID-19 data of seven provinces/cities in China reported during the spring of 2020, we make a systematical investigation on the forecast ability of eight widely used empirical functions, four statistical inference methods and five dynamical models widely used in the literature. We highlight the significance of a well balance between model complexity and accuracy, over-fitting and under-fitting, as well as model robustness and sensitivity. We further introduce the Akaike information criterion, root mean square errors and robustness index to quantify these three golden means and to evaluate various epidemic models/methods. Through extensive simulations, we find that the inflection point plays a crucial role in the choice of the size of dataset in forecasting. Before the inflection point, no model considered here could make a reliable prediction. We further notice the Logistic function steadily underestimate the final epidemic size, while the Gomertz&#39;s function makes an overestimation in all cases. Since the methods of sequential Bayesian and time-dependent reproduction number take the non-constant nature of the effective reproduction number with the progression of epidemics into consideration, we suggest to employ them especially in the late stage of an epidemic. The transition-like behavior of exponential growth method from underestimation to overestimation with respect to the inflection point might be useful for constructing a more reliable forecast. Towards the dynamical models based on ODEs, it is observed that the SEIR-QD and SEIR-PO models generally show a better performance than SIR, SEIR and SEIR-AHQ models on the COVID-19 epidemics, whose success could be attributed to the inclusion of self-protection and quarantine, and a proper trade-off between model complexity and fitting accuracy.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Wuyue Yang; Dongyan Zhang; Liangrong Peng; Changjing Zhuge; Liu Hong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034595&#39;},
 {&#39;cord_uid&#39;: &#39;ytsqfqvh&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Impact assessment of non-pharmaceutical interventions against COVID-19 and influenza in Hong Kong: an observational study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034660&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: A range of public health measures have been implemented to delay and reduce local transmission of COVID-19 in Hong Kong, and there have been major changes in behaviours of the general public. We examined the effect of these interventions and behavioral changes on the incidence of COVID-19 as well as on influenza virus infections which may share some aspects of transmission dynamics with COVID-19. Methods: We reviewed policy interventions and measured changes in population behaviours through two telephone surveys, on January 20-23 and February 11-14. We analysed data on laboratory-confirmed COVID-19 cases, influenza surveillance data in outpatients of all ages, and influenza hospitalisations in children. We estimated the daily effective reproduction number (R_t), for COVID-19 and influenza A(H1N1). Findings: COVID-19 transmissibility has remained at or below 1, indicating successful containment to date. Influenza transmission declined substantially after the implementation of social distancing measures and changes in population behaviours in late January, with a 44% (95% confidence interval, CI: 34% to 53%) reduction in transmissibility in the community, and a 33% (95% CI: 24% to 43%) reduction in transmissibility based on paediatric hospitalization rates. In the two surveys we estimated that 74.5% and 97.5% of the general adult population wore masks when going out, and 61.3% and 90.2% avoided going to crowded places, respectively. Implications: Containment measures, social distancing measures and changes in population behaviour have successfully prevented spread of COVID-19. The social distancing measures and behavioural changes led to a substantial reduction in influenza transmission in early February 2020. However, it may be challenging to avoid fatigue and sustain these measures and population behaviours as COVID-19 continues to spread globally. Funding: Health and Medical Research Fund, Hong Kong&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Benjamin J Cowling; Sheikh Taslim Ali; Tiffany W. Y. Ng; Tim K Tsang; Julian C. M Li; Min Whui Fong; Qiuyan Liao; Mike YW Kwan; So Lun Lee; Susan S. Chiu; Joseph T. Wu; Peng Wu; Gabriel M. Leung&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034660&#39;},
 {&#39;cord_uid&#39;: &#39;5zrywg58&#39;,
  &#39;sha&#39;: &#39;622580ea34c3acc011faa7573bc549e34078edde&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Ocular manifestations and clinical characteristics of 534 cases of COVID-19 in China: A cross-sectional study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034678&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: The novel coronavirus disease (COVID-19) was first reported in Wuhan, China in December 2019 and is now pandemic all over the world. Previous study has reported several COVID-19 cases with conjunctivitis. However, the complete profiling of COVID-19 related ocular symptoms and diseases are still missing. We aim to investigate the ocular manifestations and clinical characteristics of COVID-19 patients. Methods: A total of five hundred and thirty-four patients were recruited at Mobile Cabin Hospital and Tongji Hospital. We collected information on demographic characteristics, exposure history, ocular symptoms, systemic concomitant symptoms, eye drop medication, eye protections, radiologic findings, and SARS-CoV-2 detection in nasopharyngeal swabs by RT-PCR from questionnaires and electronic medical records. Results: The median age of patients was 40 and 50 years at Mobile Cabin Hospital and Tongji Hospital, respectively. Of 534 COVID-19 patients, 25 patients (4.68%) presented with conjunctival congestion and 3 patients had conjunctival congestion as the initial symptom. The average duration of conjunctival congestion was 4.9 ± 2.6 days (mean [SD]), ranging from 2 to 10 days. Dry eye (112, 20.97%), blurred vision (68, 12.73%), and foreign body sensation (63, 11.80%) ranked as the top three COVID-19 related ocular symptoms. Notably, a total of 332 COVID-19 patients (62%) had a hand-eye contact history. We also found that some COVID-19 patients had a history of eye disease, including conjunctivitis (33, 6.18%), dry eye (24, 4.49%), keratitis (14, 2.62%), cataract (9, 1.69%), and diabetic retinopathy (5, 0.94%). In consistent with previous studies, the most common clinical symptoms were fever, cough, and fatigue. Patients, 60.5% in Mobile Cabin Hospital and 67.5% in Tongji Hospital, respectively were confirmed with positive SARS-CoV-2 detection. Conclusions: Conjunctival congestion was one of the COVID-19 related ocular symptoms, which may have clinical diagnostic significance. It is essential to provide eye-care equipment and strengthen education on eye protection, as dirty hand-eye contact might be a high risk factor of COVID-19. Further detailed and comprehensive ophthalmological guidance is needed for COVID-19 control.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Liwen Chen; Chaohua Deng; Xuhui Chen; Xian Zhang; Bo Chen; Huimin Yu; Yuanjun Qin; Ke Xiao; Hong Zhang; Xufang Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034678&#39;},
 {&#39;cord_uid&#39;: &#39;awgyxn3t&#39;,
  &#39;sha&#39;: &#39;7ecc2bca3cfb5ebea94cd2601d6d3db60603452f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of 34 Children with Coronavirus Disease-2019 in the West of China: a Multiple-center Case Series&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034686&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;BACKGROUND Up to 9 March, 2020, 109577 patients were diagnosed with coronavirus disease-2019 (COVID-19) globally. The clinical and epidemiological characteristics of adult patients have been revealed recently. However, the information of paediatric patients remains unclear. We describe the clinical and epidemiological characteristics of paediatric patients to provide valuable insight into early diagnosis of COVID-19 in children, as well as epidemic control policy making. METHODS and FINDINGS This retrospective, observational study was a case series performed at 4 hospitals in the west of China. Thirty-four paediatric patients with COVID-19 were included from January 1 to February 25, 2020. And the final follow-up visit was completed by February 28, 2020. Clinical and epidemiological characteristics were analyzed on the basis of demographic data, medical history, laboratory tests, radiological findings, and treatment information. Data analysis was performed on 34 paediatrics patients with COVID-19 aged from 1 to 144 months (median 33.00, IQR 10.00 - 94.25), among whom 14 males (41.18%) were included. 47.60% of patients were noticed without any exposure history. The median incubation period was 10.50 (7.75 - 25.25) days. Infections of other respiratory pathogens were reported in 16 patients (47.06%). The most common initial symptoms were fever (76.47%), cough (58.82%), and expectoration (20.59%). Vomiting (11.76%) and diarrhea (11.76%) were also reported in a considerable portion of cases. A remarkable increase was detected in serum amyloid A for 17 patients (85.00%) and high-sensitivity C-reactive protein for 17 patients (58.62%), while a decrease of prealbumin was noticed in 25 patients (78.13%). In addition, the levels of lactate dehydrogenase was increased significantly in 28 patients (82.35%), as well as α-hydroxybutyrate dehydrogenase in 25 patients (73.53%). Patchy lesions in lobules were detected by chest computed tomographic scans in 28 patients (82.36%). The typical feature of ground-glass opacity for adults was rare in paediatric patients (2.94%). A late-onset pattern of lesions in lobules were also noticed. Stratified analysis of the clinical features were not performed due to relatively limited samples. CONCLUSIONS Our data presented the clinical and epidemiological features of paediatric patients systemically. The findings offer new insight into the early identification and intervention of paediatric patients with COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Che Zhang; Jiaowei Gu; Quanjing Chen; Na Deng; Jingfeng Li; Li Huang; Xihui Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034686&#39;},
 {&#39;cord_uid&#39;: &#39;jjdtuofy&#39;,
  &#39;sha&#39;: &#39;e4b62ae61f0135ded00dcefd923c75ba78630ced&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Spread of SARS-CoV-2 Coronavirus likely to be constrained by climate&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034728&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As new cases of SARS CoV-2 (aka 2019-nCoV) Coronavirus are confirmed throughout the world and millions of people are being put into quarantine, hit the developing world, such as sub-Saharan Africa, potentially leading to a global human calamity. It is still early days, but using existing data we develop a large ensemble of ecological niche models that project monthly variation in climate suitability of SARS-CoV-2 Coronavirus throughout a typical climatological year. The current spread suggests a degree of climate determination with Coronavirus displaying preference for cool and dry conditions. The predecessor SARS-CoV was linked to similar climate conditions. Should the spread of SARS CoV-2 continue to follow current trends, a worst-case scenario of synchronous global pandemic is improbable. More probable is the emergence of asynchronous seasonal global outbreaks much like other respiratory diseases. People in temperate warm and cold climates are more vulnerable. Those in arid climates follow next in vulnerability, while the disease will likely marginally affect the tropics. Our projections minimize uncertainties related with spread of SARS CoV-2, providing critical information for anticipating the adequate social, economic and political responses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Miguel B. Araujo; Babak Naimi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034728&#39;},
 {&#39;cord_uid&#39;: &#39;pkg4rxni&#39;,
  &#39;sha&#39;: &#39;9da357734a3bfc5b51dc2c63b6b4ab28930767ab&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Immunopathological characteristics of coronavirus disease 2019 cases in Guangzhou, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034736&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Coronavirus disease 2019 (COVID-19) is a respiratory disorder caused by the highly contagious SARS-CoV-2. The immunopathological characteristics of COVID-19 patients, either systemic or local, have not been thoroughly studied. In the present study, we analyzed both the changes in the cellularity of various immune cell types as well as cytokines important for immune reactions and inflammation. Our data indicate that patients with severe COVID-19 exhibited an overall decline of lymphocytes including CD4+ and CD8+ T cells, B cells, and NK cells. The number of immunosuppressive regulatory T cells was moderately increased in patients with mild COVID-19. IL-2, IL-6, and IL-10 were remarkably up-regulated in patients with severe COVID-19. The levels of IL-2 and IL-6 relative to the length of hospital stay underwent a similar &#34;rise-decline&#34; pattern, probably reflecting the therapeutic effect. In conclusion, our study shows that the comprehensive decrease of lymphocytes, and the elevation of IL-2 and IL-6 are reliable indicators of severe COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Yaling Shi; Mingkai Tan; Xing Chen; Yanxia Liu; Jide Huang; Jingyi Ou; Xilong Deng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034736&#39;},
 {&#39;cord_uid&#39;: &#39;hr4x664o&#39;,
  &#39;sha&#39;: &#39;6fba5d79bdf83322d365179fa84ca02de2a16ebb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Infrared assessment of human facial temperature in the presence and absence of common cosmetics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034793&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Using a sensitive research-grade infrared camera, we find that common facial cosmetics and lotions mask skin temperature in assays of the human forehead. We test a family of 10 commonly-used cosmetic products and find that volatile liquids and creams lower thermal skin temperature by at least 2 C for up to 5-10 min and at least 1 C for up to 20 min, respectively. Powder and cream that contains brightening agent lower indefinitely the skin temperature sensed by infrared camera. With the qualification that these experiments were performed in a controlled laboratory setting rather than the mass crowd screening environment where infrared temperature sensing of humans sees widespread use, our tests suggest that for human subjects whose face was treated with certain cosmetics and lotions, infrared-based screening for elevated facial temperature (fever) can be unreliable.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-13&#39;,
  &#39;authors&#39;: &#39;Kaikai Zheng; Ruoyu Dong; Huan Wang; Steve Granick&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034793&#39;},
 {&#39;cord_uid&#39;: &#39;xfjexm5b&#39;,
  &#39;sha&#39;: &#39;67ff35bb162142475e1d16e78e56dbcf59ed65bc&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Impact of self-imposed prevention measures and short-term government intervention on mitigating and delaying a COVID-19 epidemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034827&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: With new cases of COVID-19 surging around the world, some countries may have to prepare for moving beyond the containment phase. Prediction of the effectiveness of non-case-based interventions for mitigating, delaying or preventing the epidemic is urgent, especially for countries affected by the increased seasonal influenza activity. Methods: We developed a transmission model to evaluate the impact of self-imposed prevention measures (handwashing, mask-wearing, and social distancing) due to COVID-19 awareness and of short-term government-imposed social distancing on the peak number of diagnoses, attack rate and time until the peak number of diagnoses. Findings: For fast awareness spread in the population, self-imposed measures can significantly reduce the attack rate, diminish and postpone the peak number of diagnoses. A large epidemic can be prevented if the efficacy of these measures exceeds 50%. For slow awareness spread, self-imposed measures reduce the peak number of diagnoses and attack rate but do not affect the timing of the peak. Early implementation of short-term government interventions can only delay the peak. Interpretation: Handwashing, mask-wearing and social distancing as a reaction to information dissemination about COVID-19 can be effective strategies to mitigate and delay the epidemic. We stress the importance of a rapid spread of awareness on the use of self-imposed prevention measures in the population. Early-initiated short-term government-imposed social distancing can buy time for healthcare systems to prepare for an increasing COVID-19 burden. Keywords: SARS-CoV-2, COVID-19, mathematical model, prevention measures, disease awareness, epidemic control, social distancing, handwashing, mask-wearing&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Alexandra Teslya; Thi Mui Pham; Noortje E. Godijk; Mirjam E. Kretzschmar; Martin C.J. Bootsma; Ganna Rozhnova&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034827&#39;},
 {&#39;cord_uid&#39;: &#39;y353snxw&#39;,
  &#39;sha&#39;: &#39;bab88fad90f0f0448f75af01145e2f1b90412527&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Recommendations for standardized management of CML patients in the core epidemic area of COVID-19 (Multi-center survey results in Hubei Province, China)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20034876&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Since late December 2019, the outbreak of the novel coronavirus disease, COVID-19, that began in Wuhan, has become endemic in China and more than 100 countries and regions in the world. There is no report about the prevalence of COVID-19 in CML patients untill now. We aimed to describe the clinical course, outcomes of CML patients with COVID-19 and prevalence of COVID-19 in CML patients. Methods In this multi-center survey, cross-sectional survey, observational study, the clinical data of CML patients with COVID-19 in each center were collected. Simultaneously, an online survey was conducted for information about the CML patients under the management at each center by asking the CML patients to complete a questionnaire,from February 15, 2020 to February 21, 2020. The questionnaire includes demographic data, place of residence, smoking status, CML diagnosis and treatment, comorbidities, combined medications, epidemiological history, symptoms(fever, cough, shortness of breath, etc) during the epidemic. Additional clinical data was collected on respondents suspected or confirmed to have COVID-19. We described and analyzed the prevalence of COVID-19 in CML patients, and focus on the clinical characteristics and outcomes of COVID-19 patients. Data were compared between the CML patients with optimal response and those with non-optimal response. The primary outcome was prevalence of COVID-19 in CML patients, as of Feb 21, 2020. Secondary outcomes included the history of epidemiology of CML patients, the clinical characteristics and outcomes of CML patients with COVID-19 . Findings Of 392 respondents, 223( 56.9%) were males, and 240( 61.2%) were 50 years or younger. Only 10 patients took drugs irregularly due to the influence of the epidemic because of traffic control, pharmacies unable to operate normally, etc. In the history of epidemiology, there were 4 patients with definite contact with COVID-19, of which 3 were remote contact and 1 was close contact. 12 respondents had fever, cough or shortness of breath during the epidemic, 1 case (common type) was confirmed with COVID-19 and cured after treatment. 1 patient was clinically diagnosed and succumbed. 1 of 299 (0.3%) patients with an optimal response was diagnosed with COVID-19. Of the 50 patients who failed to respond to CML treatment or had a poor response, 1 patient (2%) had a clinical diagnosis of COVID-19. Interpretation While the 392 CML respondents required regular referrals to hospitals, they did not have much contact with COVID-19 patients during the outbreak. Patients who failed to achieved an optimal response to CML therapy appear more likely to have a symptomatic infection with SARS-CoV-2. Older patients with comorbidities are at increased risk of death.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Danyu wang; Jing-Ming Guo; Zhuang-Zhi Yang; Yong You; Zhi-Chao Chen; Shi-Ming Chen; Hui Cheng; You-Shan Zhang; Dao-Zi Jiang; Xue-Lan Zuo; Xiao-Jian Zhu; Jun Huang; Hong-Xiang Wang; Qing Li; Qing Wu; Zhi-Ping Huang; Qi-Huan Liu; Ying Bao; Da-Lin Zhang; Xin-Hua Zhang; Zhe Zhao; Ren-Ying Ge; Jie Du; Ya-Ping Wang; Hong-Bo Ren; Hong Han; Yun-Hui Wei; Hang Xiang; Chu-Cheng Wan; You-Fang Zhao; Bin Chen; Guo-Ling Yuan; Meng Li; Wei-ming Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20034876&#39;},
 {&#39;cord_uid&#39;: &#39;68193u0a&#39;,
  &#39;sha&#39;: &#39;97e0efc17b5a10c75f7c83b08423d27585a31df5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20035048&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: A recently emerging respiratory disease named coronavirus disease 2019 (COVID-19) has quickly spread across the world. This disease is initiated by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) and uncontrolled cytokine storm, but it remains unknown as to whether a robust antibody response is related to clinical deterioration and poor outcome in laboratory-confirmed COVID-19 patients. Methods: Anti-SARS-CoV-2 IgG and IgM antibodies were determined by chemiluminescence analysis (CLIA) in COVID-19 patients from a single center in Wuhan. Median IgG and IgM levels in acute and convalescent-phase sera (within 35 days) for all included patients were calculated and compared among severe and nonsevere patients. Immune response phenotyping based on late IgG levels and neutrophil-to-lymphocyte ratio (NLR) was characterized to stratify patients with different disease severities and outcome. Laboratory parameters in patients with different immune response phenotypes and disease severities were analyzed. Findings: A total of 222 patients were included in this study. IgG was first detected on day 4 of illness, and its peak levels occurred in the fourth week. Severe cases were more frequently found in patients with high IgG levels, compared to those who with low IgG levels (51.8% versus 32.3%; p=0.008). Severity rates for patients with NLRhiIgGhi, NLRhiIgGlo, NLRloIgGhi, and NLRloIgGlo phenotype was 72.3%, 48.5%, 33.3%, and 15.6%, respectively (p&lt;0.0001). Furthermore, severe patients with NLRhiIgGhi, NLRhiIgGlo had higher proinflammatory cytokines levels including IL-2, IL-6 and IL-10, and decreased CD4+ T cell count compared to those with NLRloIgGlo phenotype (p&lt;0.05). Recovery rate for severe patients with NLRhiIgGhi, NLRhiIgGlo, NLRloIgGhi, and NLRloIgGlo phenotype was 58.8% (20/34), 68.8% (11/16), 80.0% (4/5), and 100% (12/12), respectively (p=0.0592). Dead cases only occurred in NLRhiIgGhi and NLRhiIgGlo phenotypes. Interpretation: COVID-19 severity is associated with increased IgG response, and an immune response phenotyping based on late IgG response and NLR could act as a simple complementary tool to discriminate between severe and nonsevere COVID-19 patients, and further predict their clinical outcome.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Bicheng Zhang; Xiaoyang Zhou; Chengliang Zhu; Fan Feng; Yanru Qiu; Jia Feng; Qingzhu Jia; Qibin Song; Bo Zhu; Jun Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20035048&#39;},
 {&#39;cord_uid&#39;: &#39;zowo5ojd&#39;,
  &#39;sha&#39;: &#39;33eff3ff8721d2abd9204e3aabd504e167f3f01d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preliminary evaluation of voluntary event cancellation as a countermeasure against the COVID-19 outbreak in Japan as of 11 March, 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.20035220&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: To control COVID−19 outbreak in Japan, sports and entertainment events were canceled in Japan for two weeks from 26 February to 11 March. It has been designated as voluntary event cancellation (VEC). Object: This study predicts the effectiveness of VEC enduring and after its implementation. Method: We applied a simple susceptible−infected−recovery model to data of patients with symptoms in Japan during 14 January to VEC introduction and after VEC introduction to 8 March. We adjusted the reporting delay in the latest few days. Results: Results suggest that the basic reproduction number, R0, before VEC introduced as 2.50 with a 95% confidence interval (CI) was [2.43, 2.55] and the effective reproduction number, Rv, after VEC introduced as 1. 88; its 95% CI was [1.68,2.02]. Discussion and Conclusion: Results demonstrated that VEC can reduce COVID−19 infectiousness by 35%, but R0 remains higher than one.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Yoshiyuki Sugishita; Junko Kurita; Tamie Sugawara; Yasushi Ohkusa&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.20035220&#39;},
 {&#39;cord_uid&#39;: &#39;o6v37h8w&#39;,
  &#39;sha&#39;: &#39;0139ea4ca580af99b602c6435368e7fdbefacb03&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A Combined Evidence Approach to Prioritize Nipah Virus Inhibitors&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.977918&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundNipah Virus (NiV) came into limelight recently due to an outbreak in Kerala, India. NiV causes severe disease and death in people with over 75% case fatality rate. It is a public health concern and has the potential to become a global pandemic. Lack of treatment has forced the containment methods to be restricted to isolation and surveillance. WHO’s ‘R&amp;D Blueprint list of priority diseases’ (2018) indicates that there is an urgent need for accelerated research &amp; development for addressing NiV.Materials &amp; MethodsIn the quest for druglike NiV inhibitors (NVIs) a thorough literature search followed by systematic data curation was conducted. Rigorous data analysis was done with curated NVIs for prioritizing druglike compounds. For the same, more than 1800 descriptors of NVIs were computed and comparative analysis was performed with the FDA approved small molecules and antivirals. These compounds were further evaluated through PAINS filter to study their toxicity profile. Simultaneously, compounds were also prioritized based on the robustness of the assays through which they were identified.ResultsOur efforts lead to the creation of a well-curated structured knowledgebase of 182 NVIs with 98 small molecule inhibitors. The reported IC50/EC50 values for some of these inhibitors are in the nanomolar range – as low as 0.47 nM. In order to prioritize these inhibitors, we performed several tests and applied filters to identify drug-like non-toxic compounds. Of 98, a few compounds passed DruLito &amp; PAINS filters exhibiting drug-like properties and were also prioritized in an independent screen based only the assay robustness. The NVIs have diverse structural features and offer a wide spectrum of ways in which they can be developed further as druglike molecules.ConclusionWe report a knowledgebase for furthering the development of NVIs. The platform has a diverse set of 98 NVIs of which a few have been prioritized based on a combined evidence strategy. The platform has the provision to submit new inhibitors as and when reported by the community for further enhancement of NiV inhibitor landscape.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Nishi Kumari; Ayush Upadhyay; Kishan Kalia; Rakesh Kumar; Kanika Tuteja; Priyanka Rani Paul; Eugenia Covernton; Tina Sharma; Vinod Scaria; Anshu Bhardwaj&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.977918&#39;},
 {&#39;cord_uid&#39;: &#39;0hxan9rw&#39;,
  &#39;sha&#39;: &#39;ac45d1c2865c21a16912e73a857060bacce4c95a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;High sensitivity detection of coronavirus SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.988246&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMany detection methods have been used or reported for the diagnosis and/or surveillance of SARS-CoV-2. Among them, reverse transcription polymerase chain reaction (RT-PCR) is the most sensitive, claiming detection of about 5 copies of viruses. However, it has been reported that only 47-59% of the positive cases were identified by RT-PCR, probably due to loss or degradation of virus RNA in the sampling process, or even mutation of the virus genome. Therefore, developing highly sensitive methods is imperative to ensure robust detection capabilities. With the goal of improving sensitivity and accommodate various application settings, we developed a multiplex-PCR-based method comprised of 172 pairs of specific primers, and demonstrated its efficiency to detect SARS-CoV-2 at low copy numbers. The assay produced clean characteristic target peaks of defined sizes, which allowed for direct identification of positives by electrophoresis. In addition, optional sequencing can provide further confirmation as well as phylogenetic information of the identified virus(es) for specific strain discrimination, which will be of paramount importance for surveillance purposes that represent a global health imperative. Finally, we also developed in parallel a multiplex-PCR-based metagenomic method that is amenable to detect SARS-CoV-2, with the additional benefit of its potential for uncovering mutational diversity and novel pathogens at low sequencing depth.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Chenyu Li; David N. Debruyne; Julia Spencer; Vidushi Kapoor; Lily Y. Liu; Bo Zhou; Lucie Lee; Rounak Feigelman; Grayson Burdon; Jeffrey Liu; Alejandra Oliva; Adam Borcherding; Hongdong Tan; Alexander E. Urban; Guoying Liu; Zhitong Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.988246&#39;},
 {&#39;cord_uid&#39;: &#39;kjjxi27b&#39;,
  &#39;sha&#39;: &#39;a6cba8f9a9f89a08bb87e30a209389c9f1ca8c3d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Establishing Prokaryotic Expression System of Angiotensin-Converting Enzyme 2 (ACE2) gene in pigs&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.988634&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn this paper, ACE2 gene of pigs was cloned and the purified protein was obtained via the prokaryotic expression system. Polyclonal antibody of high titer and sensitivity was obtained using Wastar rats immunization method and is then used to determine of the expression of ACE2 using immunohistochemistry. The sequence of ACE2 in pigs covered 2418 nucleotides and coded 805 amino acid (aa) residues. Sequence homology analysis showed that the ACE2 sequence in pigs is highly conserved among species at the nucleotide and amino acid levels. Genetic evolution analysis revealed that ACE2 gene in pigs has the shortest genetic distance with that in goats while residing in a totally different branch from that in zebra fishes. Analysis of protein structure predicted that ACE2 protein is a transmembrane secreted protein with high hydrophilicity, containing a signal peptide sequences locating between 1aa to 17aa. The ACE2 fusion protein expressed (under the induction with 1.0 mmol/L IPTG for 10 h) was of approximately 100 kDa and mainly existed in inclusion body. Wastar rats immunization showed that the titer of the anti-ACE2 antiserum in rats was 1: 3200. Western blot showed that the antibody binds specifically. Immunohistochemistry showed that the ACE2 protein was expressed in all major tissues of pigs. It is the first time that polyclonal antibody of ACE2 in pigs was obtained and the expression of ACE2 was confirmed. These results will provide a basis for investigating on ACE2’s biological activity in pigs.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-12&#39;,
  &#39;authors&#39;: &#39;Hang Xiao; Xin-Tian Nie; Xiao-Xia Ji; Shu-ping Yan; Bin Zhu; Yuan-Shu Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.988634&#39;},
 {&#39;cord_uid&#39;: &#39;wt4pxu08&#39;,
  &#39;sha&#39;: &#39;9867fe1531c1541ca2b25a82d7adb62d575798be&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The architecture of SARS-CoV-2 transcriptome&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.988865&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3′ tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.HighlightsWe provide a high-resolution map of SARS-CoV-2 transcriptome and epitranscriptome using nanopore direct RNA sequencing and DNA nanoball sequencing.The transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical.In addition to the genomic and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces transcripts encoding unknown ORFs.We discover at least 41 potential RNA modification sites with an AAGAA motif.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Dongwan Kim; Joo-Yeon Lee; Jeong-Sun Yang; Jun Won Kim; V. Narry Kim; Hyeshik Chang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.988865&#39;},
 {&#39;cord_uid&#39;: &#39;hj675z1b&#39;,
  &#39;sha&#39;: &#39;28884786bfd579216f563c278e1730290dd4ccb8&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.12.989186&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummarySARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency.1 Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.2 Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3’-fluoro-3’-deoxythymidine triphosphate and 3’-azido-3’-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Jingyue Ju; Xiaoxu Li; Shiv Kumar; Steffen Jockusch; Minchen Chien; Chuanjuan Tao; Irina Morozova; Sergey Kalachikov; Robert N. Kirchdoerfer; James J. Russo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.12.989186&#39;},
 {&#39;cord_uid&#39;: &#39;42pjc0lo&#39;,
  &#39;sha&#39;: &#39;57a86b3acd182c955877afd4792719a4e9a0ac32&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Mental health status among family members of health care workers in Ningbo, China during the Coronavirus Disease 2019 (COVID-19) outbreak: a Cross-sectional Study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20033290&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: So far, the psychological impact of COVID-19 epidemic among family members of Health care workers (HCWs) in China has been neglected. The present cross-sectional study aimed to investigate the mental health status and related factors of families of HCWs in Designated Hospitals in Ningbo, China. Method: Family members of HCWs working in five designated hospitals in Ningbo, China were recruited between February 10th and 20th, 2020. Information on demographic variables, the COVID-19-related events in the lives, knowledge of COVID-19 and the working status of family members (that is, HCWs) was collected using online self-administered questionnaires. The mental health status were assessed using the Chinese version of Patient Health Questionnare-9 (PHQ-9) and Chinese version of Generalized Anxiety Disorder-7 (GAD-7 ). Multivariable logistic regression analyses were conducted to identify the main factors associated with the mental health conditions. Results: A total of 822 participants completed questionnaires correctly. (response rate of 95.80% ). The overall prevalence of GAD and depressive symptoms were 33.73%, and 29.35%, respectively. More times (hours) to focus on the COVID-19 (OR=1.215, 95%CI:1.061-1.391), family members (that is, HCWs) directly contact with confirmed or suspected COVID-19 patients (OR=1.477, 95%CI:1.069-2.040) were risk factors for GAD, while higher participants self-reported safety score for protective equipment of HCWs (OR=0.807, 95%CI:0.700-0.930) was a protective factor. More times (hours) to focus on the COVID-19 (OR=1.215, 95%CI:1.061-1.391), longer average working times per week for family members (that is, HCWs) (OR=1.017, 95%CI:1.005-1.029), being parents and other next of kin of HCWs were risk factors for depressive symptoms (OR=3.526, 95%CI:1.609-7.728 and OR=1.639, 95%CI:1.096-2.451, respectively). In addition, compared with participants who were HCWs, participants who were enterprise workers and were more likely to develop depressive symptoms(OR=1.750, 95%CI:1.104-2.776), while who were government employees or institutions employees were less likely to suffer depressive symptoms (OR=0.529, 95%CI:0.286-0.977). Conclusions: Psychological responses to COVID-19 have been dramatic among family members of HCWs during the rising phase of the outbreak. Our findings provide strong evidence to pay more attention on the mental health status of this vulnerable but often unseen populations during COVID-19 epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Yuchen Ying; Fanqian Kong; Binbin Zhu; Yunxin Ji; Zhongze Lou; Liemin Ruan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20033290&#39;},
 {&#39;cord_uid&#39;: &#39;2v5wkjrq&#39;,
  &#39;sha&#39;: &#39;bb53a02a8468fa9b33b082328580378680aa0838&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Cybernetics-based Dynamic Infection Model for Analyzing SARS-COV-2 Infection Stability and Predicting Uncontrollable Risks&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20034082&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since December 2019, COVID-19 has raged in Wuhan and subsequently all over China and the world. We propose a Cybernetics-based Dynamic Infection Model (CDIM) to the dynamic infection process with a probability distributed incubation delay and feedback principle. Reproductive trends and the stability of the SARS-COV-2 infection in a city can then be analyzed, and the uncontrollable risks can be forecasted before they really happen. The infection mechanism of a city is depicted using the philosophy of cybernetics and approaches of the control engineering. Distinguished with other epidemiological models, such as SIR, SEIR, etc., that compute the theoretical number of infected people in a closed population, CDIM considers the immigration and emigration population as system inputs, and administrative and medical resources as dynamic control variables. The epidemic regulation can be simulated in the model to support the decision-making for containing the outbreak. City case studies are demonstrated for verification and validation.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Wenlei Xiao; Qiang Liu; J Huan; Pengpeng Sun; Liuquan Wang; Chenxin Zang; Sanying Zhu; Liansheng Gao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20034082&#39;},
 {&#39;cord_uid&#39;: &#39;3rrry27f&#39;,
  &#39;sha&#39;: &#39;aa5475eb5aa3a1091fa01df00aca2f3def543f70&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Health, distress, and life satisfaction of people one-month into COVID-19 outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20034496&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background What are the health and wellbeing conditions of people during the COVID-19 outbreak in China? The epidemiological reports now focus on the confirmed COVID-19 cases, and we aim to assess the health and wellbeing of normal adults living and working after one month of public health emergencies into the COVID-19 outbreak in China. Methods One month into the outbreak, on Feb 20/21, 2020, we sampled 369 adults on the eight dimensions of health (SF12), distress (K6), and life satisfaction in 64 cities that varied in their densities of coronavirus confirmed cases. The participants also reported their work status, whether they had chronic health issues, the number of hours they exercised per day in the past week. Findings In general, life of normal adults was severely disrupted. 33% of the participants had not left their home at all during the one-month period due to the restrictive measures to contain COVID-19 in China. 27% of the participants worked at office; 38% resorted to working from home; and 25% suspended working due to the outbreak. Those who suspended working reported worse health conditions by SF12 as well as distress (K6). The severity of the COVID-19 in individuals location predicts their life satisfaction, and this relationship is moderated by the existing chronic health issues and exercising hour of each individual. Interpretation Our findings support the need to pay attention to the health of people who were not affected by the virus epistemologically. We need to support especially people who suspended working during the outbreak. Our results also highlight that people who are usually active, as indicated by more exercises, might be more suspectable to wellbeing issues due to the lockdown. Policymakers who are thinking of introducing measures to contain COVID-19 may benefit from understanding such health and welling implications.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Stephen X Zhang; Yifei Wang; Andreas Rauch; Feng Wei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20034496&#39;},
 {&#39;cord_uid&#39;: &#39;czpz3yh2&#39;,
  &#39;sha&#39;: &#39;e172b0d49bd083dca5cd1021900477dd97774b84&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;If containment is not possible, how do we minimize mortality for COVID-19 and other emerging infectious disease outbreaks?&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20034892&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;If COVID-19 containment policies fail and social distancing measures cannot be sustained until vaccines becomes available, the next best approach is to use interventions that reduce mortality and prevent excess infections while allowing low-risk individuals to acquire immunity through natural infection until population level immunity is achieved. In such a situation, allowing some infections to occur in lower-risk groups might lead to an overall greater reduction in mortality than trying to protect everyone equally.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Andreas Handel; Joel Miller; Yang Ge; Isaac Chun-Hai Fung&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20034892&#39;},
 {&#39;cord_uid&#39;: &#39;tof0so67&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Triaging patients in the outbreak of the 2019 novel coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035212&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In the end of 2019, the epidemic of a new coronavirus (SARS-CoV-2) occurred in Wuhan and spread rapidly. Changsha, a city located south to the epicenter, was soon impacted. To control the transmission of the coronavirus and avoid nosocomial infection, triage procedures based on epidemiology were implemented in a local hospital of the city. This retrospective study analyzed the data collected during the triage period and found that COVID-19 patients were enriched seven folds into the Section A designated for rapid detection and quarantine. On the other side, roughly triple amounts of visits were received at the Section B for patients without obvious epidemiological history. Eight COVID-19 cases were spotted out of 247 suspected patients. More than 50% of the suspected patients were submitted to multiple rounds of nucleic acid analysis for SARS-CoV-2 infection. Of the 239 patients who were diagnosed as negative of the virus infection,188 were successfully revisited and none was reported as a COVID-19 case. Of the eight COVID-19 patients, three were confirmed only after multiple rounds of nucleic acid analysis. Besides comorbidities, delayed sharing of epidemiological history added another layer of complexity to the diagnosis in practice. While SARS-CoV-2 epidemic is being alerted in many countries, our report will be helpful to other colleagues in rapid identification of COVID-19 cases and controlling the transmission of the disease.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Guoqing Huang; Weiqian Zeng; Wenbo Wang; Yanmin Song; Xiaoye Mo; Jia Li; Ping Wu; Ruolong Wang; Fangyi Zhou; Jing Wu; Bin Yi; Zeng Xiong; Lu Zhou; Fanqi Wang; Yangjing Tian; Wenbao Hu; Xia Xu; Ruonan Zhai; Kai Yuan; Xiangmin Li; Xinjian Qiu; Jian Qiu; Aimin Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035212&#39;},
 {&#39;cord_uid&#39;: &#39;8ozauxlk&#39;,
  &#39;sha&#39;: &#39;de055f09fef2776bc78bac5d58c4131301b2025f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Impacts of social and economic factors on the transmission of coronavirus disease (COVID-19) in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035238&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This paper examines the role of various socioeconomic factors in mediating the local and cross-city transmissions of the novel coronavirus 2019 (COVID-19) in China. We implement a machine learning approach to select instrumental variables that strongly predict virus transmission among the rich exogenous weather characteristics. Our 2SLS estimates show that the stringent quarantine, massive lockdown and other public health measures imposed in late January significantly reduced the transmission rate of COVID-19. By early February, the virus spread had been contained. While many socioeconomic factors mediate the virus spread, a robust government response since late January played a determinant role in the containment of the virus. We also demonstrate that the actual population flow from the outbreak source poses a higher risk to the destination than other factors such as geographic proximity and similarity in economic conditions. The results have rich implications for ongoing global efforts in containment of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Yun Qiu; Xi Chen; Wei Shi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035238&#39;},
 {&#39;cord_uid&#39;: &#39;fkwskhfk&#39;,
  &#39;sha&#39;: &#39;80df1ce37238d763d313bcd0e673716c7c4458c1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Impact of city and residential unit lockdowns on prevention and control of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035253&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;With respect to the asymptomatic transmission characteristics of the novel coronavirus that appeared in 2019 (COVID-19), a susceptible-asymptomatic-infected-recovered-death (SAIRD) model that considered human mobility was constructed in this study. The dissemination of COVID-19 was simulated using computational experiments to identify the mechanisms underlying the impact of city and residential lockdowns on controlling the spread of the epidemic. Results: The implementation of measures to lock down cities led to higher mortality rates in these cities, due to reduced mobility. Moreover, implementing city lockdown along with addition of hospital beds led to improved cure and reduced mortality rates. Stringent implementation and early lockdown of residential units effectively controlled the spread of the epidemic, and reduced the number of hospital bed requirements. Collectively, measures to lock down cities and residential units should be taken to prevent the spread of COVID-19. In addition, medical resources should be increased in cities under lockdown. Implementation of these measures would reduce the spread of the virus to other cities and allow appropriate treatment of patients in cities under lockdown.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Peng Shao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035253&#39;},
 {&#39;cord_uid&#39;: &#39;lwe7whmg&#39;,
  &#39;sha&#39;: &#39;7677310b4e43cfaff6fd4ad53636fa80de589fd2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The effects of border control and quarantine measures on global spread of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035261&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The rapid expansion of coronavirus (COVID-19) has been observed in many parts of the world. Many newly reported cases of this new coronavirus during early outbreak phases have been associated with travel history from an epidemic region (identified as imported cases). For those cases without travel history, the risk of wider spreads through community contact is even higher. However, most population models assume a homogeneous infected population without considering that the imported and secondary cases contracted by the imported cases can pose a different risk to community spread. We have developed an “easy-to-use” mathematical framework extending from a meta-population model embedding city-to-city connections to stratify the dynamics of transmission waves caused by imported, secondary, and others from an outbreak source region when control measures are considered. Using the dynamics of the secondary cases, we are able to determine the probability of community spread. Using the top 10 visiting cities from Wuhan in China as an example, we first demonstrated that the arrival time and the dynamics of the outbreaks at these cities can be successfully predicted under the reproductive number R0 = 2.92 and latent period τ = 5.2 days. Next, we showed that although control measures can gain extra 32.5 and 44.0 days in arrival time through a high intensive border control measure and a shorter time to quarantine under a low R0 (1.4), if the R0 is higher (2.92), only 10 extra days can be gained for each of the same measures. This suggests the importance of lowering the incidence at source regions together with infectious disease control measures in susceptible regions. The study allows us to assess the effects of border control and quarantine measures on the emergence and the global spread in a fully connected world using the dynamics of the secondary cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;M. Pear Hossain; Alvin Junus; Xiaolin Zhu; Pengfei Jia; Tzai-Hung Wen; Dirk Pfeiffer; Hsiang-Yu Yuan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035261&#39;},
 {&#39;cord_uid&#39;: &#39;igfur3l9&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Influence factors of death risk among COVID-19 patients in Wuhan, China: a hospital-based case-cohort study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035329&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background. Coronavirus disease 2019 (COVID-19) triggered by infection with severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) has been widely pandemic all over the world. The aim of this study was to analyze the influence factors of death risk among 200 COVID-19 patients. Methods. Two hundred patients with confirmed SARS-CoV-2 infection were recruited. Demographic data and clinical characteristics were collected from electronic medical records. Biochemical indexes on admission were measured and patient&#39;s prognosis was tracked. The association of demographic data, clinical characteristics and biochemical indexes with death risk was analyzed. Results. Of 200 COVID-19 patients, 163 (81.5%) had at least one of comorbidities, including diabetes, hypertension, hepatic disease, cardiac disease, chronic pulmonary disease and others. Among all patients, critical cases, defined as oxygenation index lower than 200, accounted for 26.2%. Severe cases, oxygenation index from 200 to 300, were 29.7%. Besides, common cases, oxygenation index higher than 300, accounted for 44.1%. At the end of follow-up, 34 (17%) were died on mean 10.9 day after hospitalization. Stratified analysis revealed that older ages, lower oxygenation index and comorbidities elevated death risk of COVID-19 patients. On admission, 85.5% COVID-19 patients were with at least one of extrapulmonary organ injuries. Univariable logistic regression showed that ALT and TBIL, two indexes of hepatic injury, AST, myoglobin and LDH, AST/ALT ratio, several markers of myocardial injury, creatinine, urea nitrogen and uric acid, three indexes of renal injury, were positively associated with death risk of COVID-19 patients. Multivariable logistic regression revealed that AST/ALT ratio, urea nitrogen, TBIL and LDH on admission were positively correlated with death risk of COVID-19 patients. Conclusion. Older age, lower oxygenation index and comorbidities on admission elevate death risk of COVID-19 patients. AST/ALT ratio, urea nitrogen, TBIL and LDH on admission may be potential prognostic indicators. Early hospitalization is of great significance to prevent multiple organ damage and improve the survival of COVID-19 patients.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Lin Fu; Jun Fei; Hui-Xian Xiang; Ying Xiang; Zhu-Xia Tan; Meng-Die Li; Fang-Fang Liu; Hong-Yan Liu; Ling Zheng; Ying Li; Hui Zhao; De-xiang Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035329&#39;},
 {&#39;cord_uid&#39;: &#39;a0pf12jo&#39;,
  &#39;sha&#39;: &#39;46c5f4a01496ab4fe89b1dfba7b7f94445766640&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Forecasting of COVID-19 Confirmed Cases in Different Countries with ARIMA Models&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035345&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The epidemic of a novel coronavirus illness (COVID-19) becomes as a global threat. The aim of this study is first to find the best prediction models for daily confirmed cases in countries with high number of confirmed cases in the world and second to predict confirmed cases with these models in order to have more readiness in healthcare systems. This study was conducted based on daily confirmed cases of COVID-19 that were collected from the official website of Johns Hopkins University from January 22th, 2020 to March 1th, 2020. Auto Regressive Integrated Moving Average (ARIMA) model was used to predict the trend of confirmed cases. Stata version 12 was used. Mainland China and Thailand had almost a stable trend. The trend of South Korea was decreasing and will become stable in near future. Iran and Italy had unstable trends. Mainland China and Thailand were successful in haltering COVID-19 epidemic. Investigating their protocol in this control like quarantine should be in the first line of other countries program&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Tania Dehesh; H.A. Mardani-Fard; Paria Dehesh&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035345&#39;},
 {&#39;cord_uid&#39;: &#39;hxuyaany&#39;,
  &#39;sha&#39;: &#39;ee3aa97a0ad91231296bd0643136c83bb97af88a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Multi-city modeling of epidemics using spatial networks: Application to 2019-nCov (COVID-19) coronavirus in India&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035386&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The ongoing pandemic of 2019-nCov (COVID-19) coronavirus has made reliable epidemiological modeling an urgent necessity. Unfortunately, most of the existing models are either too fine-grained to be efficient or too coarse-grained to be reliable. Here we propose a computationally efficient hybrid approach that uses SIR model for individual cities which are in turn coupled via empirical transportation networks that facilitate migration among them. The treatment presented here differs from existing models in two crucial ways: first, self-consistent determination of coupling parameters so as to maintain the populations of individual cities, and second, the incorporation of distance dependent temporal delays in migration. We apply our model to Indian aviation as well as railway networks taking into account populations of more than 300 cities. Our results project that through the domestic transportation, the significant population is poised to be exposed within 90 days of the onset of epidemic. Thus, serious supervision of domestic transport networks is warranted even after restricting international migration.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Bhalchandra S Pujari; Snehal M Shekatkar&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035386&#39;},
 {&#39;cord_uid&#39;: &#39;18khagfy&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Wuhan and Hubei COVID-19 mortality analysis reveals the critical role of timely supply of medical resources&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035410&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We report that COVID-19 mortality and recovery rates in Hubei Province, China exponentially decays (R2&gt;0.93) and grows (R2&gt;0.95), respectively. A great number of newly supplied medical resources (health workers and beds) enabled overwhelming patients to be treated effectively. This may help other countries to deal with the coming COVID-19 outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Zuqin Zhang; Wei Yao; Yan Wang; Cheng Long; XINMIAO FU&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035410&#39;},
 {&#39;cord_uid&#39;: &#39;s956fh59&#39;,
  &#39;sha&#39;: &#39;3c0e6714183f9935f79f8b2e2a708833b52f00e0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Impact of the contact and exclusion rates on the spread of COVID-19 pandemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.20035485&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In March 2020, Coronavirus Disease 2019 originating from Wuhan, China, has become pandemic. Based on epidemic data from China we constructed and constrained a simple Susceptible-Infected-Infectious-Excluded (SIIE) model. The model is characterized by only three parameters: average incubation period, contact rate rC, and exclusion rate rE. These two rates control the daily multiplication coefficient β describing epidemic growth in its early exponential phase, during which the number of excluded (confirmed) individuals grows as E = E0 exp(α,t) with α = lnβ. The rates can be modified by non-therapeutic interventions: rC can be reduced by quarantine, while rE can be increased by efficient testing, enabling isolation of infectious individuals. In order to contain exponential epidemic growth, quotient rC/rE has to be reduced to 1. Based on the model, we estimate that the quarantine imposed in China on January 23, 2020 resulted in about 50-fold reduction of this quotient, allowing to terminate the exponential growth phase and enter the phase of exponential regression of daily new cases. In contrast, quarantine imposed in northern, most affected, Italian provinces on February 21 resulted in about 3-fold reduction of rC/rE and corresponding decrease of β from about 1.5 to about 1.2; further 5-fold reduction is necessary to terminate exponential growth. Even higher reduction should take place in France, Germany and Spain for which β &gt; 1.3 (as of March 10, 2020). Considering longer-term dynamics in the case when the initial exponential growth is not contained by quarantine, we analyzed a scenario in which rE decreases in response to the surge of daily new cases which convinces people to isolate themselves. In such case, initial abrupt epidemic growth is followed by a slow decrease over years. This scenario, although economically and socially devastating, grants time to develop, produce, and distribute vaccine.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Marek Kochanczyk; Frederic Grabowski; Tomasz Lipniacki&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.20035485&#39;},
 {&#39;cord_uid&#39;: &#39;ih1py5n9&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Rhesus macaques can be effectively infected with SARS-CoV-2 via ocular conjunctival route&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.990036&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe outbreak of Corona Virus Disease 2019 (COVID-19) is highly infectious and transmitted mainly through human-to-human transmission via respiratory droplets and direct or close contact to the patients with SARS-CoV-2. The other potential transmission routes remain to be further researched. In some clinical cases, samples of tears and conjunctival secretions from both SARS-CoV[1] and SARS-CoV-2 patients with conjunctivitis[2] displayed detectable viral RNA. A previous study reported the case of a clinician who was infected with SARS-CoV-2 while working with patients under all safeguards except eye protection [2]. By contrast, no SARS-CoV-2 could be detected by RT-PCR in 114 conjunctival swabs samples from patients with COVID-19 pneumonia [4]. Anatomically, the linkage of the ocular with respiratory tissues is primarily by the nasolacrimal system [5]. The potential extra-respiratory transmissible routes of SARS-CoV-2 via ocular remained unclear. Whether ocular conjunctiva is one of the portals that SARS-CoV-2 enters the host needs to be further research by laboratory-confirmation for providing significant data to oversight and prevention in people.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Wei Deng; Linlin Bao; Hong Gao; Zhiguang Xiang; Yajin Qu; Zhiqi Song; Shunran Gong; Jiayi Liu; Jiangning Liu; Pin Yu; Feifei Qi; Yanfeng Xu; Fengli Li; Chong Xiao; Qi Lv; Jing Xue; Qiang Wei; Mingya Liu; Guanpeng Wang; Shunyi Wang; Haisheng Yu; Xing Liu; Wenjie Zhao; Yunlin Han; Chuan Qin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.990036&#39;},
 {&#39;cord_uid&#39;: &#39;q4m2uj6r&#39;,
  &#39;sha&#39;: &#39;d1696944ff45daaa2f305405c54ba45b16f3c86c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Reinfection could not occur in SARS-CoV-2 infected rhesus macaques&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.990226&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAn outbreak of the Corona Virus Disease 2019 (COVID-19), caused by the severe acute respiratory syndrome CoV-2 (SARS-CoV-2), began in Wuhan and spread globally. Recently, it has been reported that discharged patients in China and elsewhere were testing positive after recovering. However, it remains unclear whether the convalescing patients have a risk of “relapse” or “reinfection”. The longitudinal tracking of re-exposure after the disappeared symptoms of the SARS-CoV-2-infected monkeys was performed in this study. We found that weight loss in some monkeys, viral replication mainly in nose, pharynx, lung and gut, as well as moderate interstitial pneumonia at 7 days post-infection (dpi) were clearly observed in rhesus monkeys after the primary infection. After the symptoms were alleviated and the specific antibody tested positively, the half of infected monkeys were rechallenged with the same dose of SARS-CoV-2 strain. Notably, neither viral loads in nasopharyngeal and anal swabs along timeline nor viral replication in all primary tissue compartments at 5 days post-reinfection (dpr) was found in re-exposed monkeys. Combined with the follow-up virologic, radiological and pathological findings, the monkeys with re-exposure showed no recurrence of COVID-19, similarly to the infected monkey without rechallenge. Taken together, our results indicated that the primary SARS-CoV-2 infection could protect from subsequent exposures, which have the reference of prognosis of the disease and vital implications for vaccine design.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Linlin Bao; Wei Deng; Hong Gao; Chong Xiao; Jiayi Liu; Jing Xue; Qi Lv; Jiangning Liu; Pin Yu; Yanfeng Xu; Feifei Qi; Yajin Qu; Fengdi Li; Zhiguang Xiang; Haisheng Yu; Shuran Gong; Mingya Liu; Guanpeng Wang; Shunyi Wang; Zhiqi Song; Wenjie Zhao; Yunlin Han; Linna Zhao; Xing Liu; Qiang Wei; Chuan Qin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.990226&#39;},
 {&#39;cord_uid&#39;: &#39;c2lljdi7&#39;,
  &#39;sha&#39;: &#39;4cd17ff08e33563f77854f8aba29a012e3b88b0d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.990242&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-CoV-2, distinguishing it from other coronavirus strains, such as MERS-CoV, HCoV-NL63, HCoV-OC43, HCoV-229E, HCoV-HKU1, and SARS-CoV regardless of missing information and errors in sequencing (noise). From a dataset of 553 complete genome non-repeated sequences that vary from 1,260 to 31,029 bps in length, the proposed approach classifies the different coronaviruses with an average accuracy of 98.75% in a 10-fold cross-validation, identifying SARS-CoV-2 with an AUC of 98%, specificity of 0.9939 and sensitivity of 1.00 in a binary classification. Then, using the same basis, we classify SARS-CoV-2 from 384 complete viral genome sequences with human host, that contain the gene ORF1ab from the NCBI with a 10-fold accuracy of 98.17% , a specificity of 0.9797 and sensitivity of 1.00. These preliminary results seem encouraging enough to identify deep learning as a promising research venue to develop assisted detection tests for SARS-CoV-2. At this end the interaction between viromics and deep learning, will hopefully help to solve global infection problems. In addition, we offer our code and processed data to be used for diagnostic purposes by medical doctors, virologists and scientists involved in solving the SARS-CoV-2 pandemic. As more data become available we will update our system.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Alejandro Lopez-Rincon; Alberto Tonda; Lucero Mendoza-Maldonado; Eric Claassen; Johan Garssen; Aletta D. Kraneveld&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.990242&#39;},
 {&#39;cord_uid&#39;: &#39;c08ptb1o&#39;,
  &#39;sha&#39;: &#39;2f6efc4edc0c9fffd3acc810efd8616e7772b28a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.990267&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Ilda D’Annessa; Filippo Marchetti; Giorgio Colombo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.990267&#39;},
 {&#39;cord_uid&#39;: &#39;n7ylgqfu&#39;,
  &#39;sha&#39;: &#39;1aa3e788fc6b03c147e37200c3b011ca7a289a5c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Dark Proteome of Newly Emerged SARS-CoV-2 in Comparison with Human and Bat Coronaviruses&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.990598&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRecently emerged Wuhan’s novel coronavirus designated as SARS-CoV-2, a causative agent of coronavirus disease 2019 (COVID-19) is rapidly spreading its pathogenicity throughout the world now. More than 4000 mortalities have occurred worldwide till the writing of this article and this number is increasing every passing hour. World Health Organization (WHO) has declared it as a global public health emergency. The multiple sequence alignment data correlated with already published reports on SARS-CoV-2 indicated that it is closely related to Bat-Severe Acute Respiratory Syndrome like coronavirus (Bat CoV SARS-like) and well-studied Human SARS. In this study, we have exploited the complementary approach to examine the intrinsically disordered regions in proteome of SARS-CoV-2 using Bat SARS-like and Human SARS CoVs as comparative models. According to our findings, SARS-CoV-2 proteome contains a significant amount of ordered proteins except Nucleocapsid, Nsp8, and ORF6. Further, cleavage sites in replicase 1ab polyprotein are found to be highly disordered. We have extensively investigated the dark proteome in SARS-CoV-2 which will have implications for the structured and unstructured biology of SARS or SARS-like coronaviruses.SignificanceThe infection caused by Novel Coronavirus (SARS-CoV-2) is responsible for the current pandemic that cause severe respiratory disease and pneumonia-like infection in humans. Currently, there is no such in-depth information on protein structure and function available in public domain and no effective anti-viral or vaccines are available for the treatment of this infection. Our study provides comparative order and disorder-based proteome information with Human SARS and Bat CoV that may be useful for structure-based drug discovery.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Rajanish Giri; Taniya Bhardwaj; Meenakshi Shegane; Bhuvaneshwari R. Gehi; Prateek Kumar; Kundlik Gadhave&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.990598&#39;},
 {&#39;cord_uid&#39;: &#39;62izhj8d&#39;,
  &#39;sha&#39;: &#39;0d4432c32daefa6f0d711ca35eca81188e0bccdf&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.991083&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Abstract17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-15&#39;,
  &#39;authors&#39;: &#39;Jamie A. Kelly; Jonathan D. Dinman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.991083&#39;},
 {&#39;cord_uid&#39;: &#39;xieqswct&#39;,
  &#39;sha&#39;: &#39;05d47dd5b46f86428de058db4ecc8bca76a9ad16&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Development of CRISPR as a prophylactic strategy to combat novel coronavirus and influenza&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.991307&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe outbreak of the coronavirus disease 2019 (COVID-19), caused by the Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2), has infected more than 100,000 people worldwide with over 3,000 deaths since December 2019. There is no cure for COVID-19 and the vaccine development is estimated to require 12-18 months. Here we demonstrate a CRISPR-Cas13-based strategy, PAC-MAN (Prophylactic Antiviral CRISPR in huMAN cells), for viral inhibition that can effectively degrade SARS-CoV-2 sequences and live influenza A virus (IAV) genome in human lung epithelial cells. We designed and screened a group of CRISPR RNAs (crRNAs) targeting conserved viral regions and identified functional crRNAs for cleaving SARS-CoV-2. The approach is effective in reducing respiratory cell viral replication for H1N1 IAV. Our bioinformatic analysis showed a group of only six crRNAs can target more than 90% of all coronaviruses. The PAC-MAN approach is potentially a rapidly implementable pan-coronavirus strategy to deal with emerging pandemic strains.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Timothy R. Abbott; Girija Dhamdhere; Yanxia Liu; Xueqiu Lin; Laine Goudy; Leiping Zeng; Augustine Chemparathy; Stephen Chmura; Nicholas S. Heaton; Robert Debs; Tara Pande; Drew Endy; Marie La Russa; David B. Lewis; Lei S. Qi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.991307&#39;},
 {&#39;cord_uid&#39;: &#39;s4cwubkx&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 receptor ACE2 and TMPRSS2 are predominantly expressed in a transient secretory cell type in subsegmental bronchial branches&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.991455&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYThe SARS-CoV-2 pandemic affecting the human respiratory system severely challenges public health and urgently demands for increasing our understanding of COVID-19 pathogenesis, especially host factors facilitating virus infection and replication. SARS-CoV-2 was reported to enter cells via binding to ACE2, followed by its priming by TMPRSS2. Here, we investigate ACE2 and TMPRSS2 expression levels and their distribution across cell types in lung tissue (twelve donors, 39,778 cells) and in cells derived from subsegmental bronchial branches (four donors, 17,521 cells) by single nuclei and single cell RNA sequencing, respectively. While TMPRSS2 is expressed in both tissues, in the subsegmental bronchial branches ACE2 is predominantly expressed in a transient secretory cell type. Interestingly, these transiently differentiating cells show an enrichment for pathways related to RHO GTPase function and viral processes suggesting increased vulnerability for SARS-CoV-2 infection. Our data provide a rich resource for future investigations of COVID-19 infection and pathogenesis.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Soeren Lukassen; Robert Lorenz Chua; Timo Trefzer; Nicolas C. Kahn; Marc A. Schneider; Thomas Muley; Hauke Winter; Michael Meister; Carmen Veith; Agnes W. Boots; Bianca P. Hennig; Michael Kreuter; Christian Conrad; Roland Eils&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.991455&#39;},
 {&#39;cord_uid&#39;: &#39;f03ka7bd&#39;,
  &#39;sha&#39;: &#39;522da4df2c31909547ac81763ad28d318fdb88e6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.13.991570&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe outbreak of COVID-19, which is caused by SARS-CoV-2 virus, continues to spread globally, but there is currently very little understanding of the epitopes on the virus. In this study, we have determined the crystal structure of the receptor-binding domain (RBD) of the SARS-CoV-2 spike (S) protein in complex with CR3022, a neutralizing antibody previously isolated from a convalescent SARS patient. CR3022 targets a highly conserved epitope that enables cross-reactive binding between SARS-CoV-2 and SARS-CoV. Structural modeling further demonstrates that the binding site can only be accessed when at least two RBDs on the trimeric S protein are in the “up” conformation. Overall, this study provides structural and molecular insight into the antigenicity of SARS-CoV-2.ONE SENTENCE SUMMARYStructural study of a cross-reactive SARS antibody reveals a conserved epitope on the SARS-CoV-2 receptor-binding domain.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Meng Yuan; Nicholas C. Wu; Xueyong Zhu; Chang-Chun D. Lee; Ray T. Y. So; Huibin Lv; Chris K. P. Mok; Ian A. Wilson&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.13.991570&#39;},
 {&#39;cord_uid&#39;: &#39;scbbps9f&#39;,
  &#39;sha&#39;: &#39;ad146e228bda4e5a3d56b28a3a1acbd01e0b323e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modelling the epidemic 2019-nCoV event in Italy: a preliminary note&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20034884&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An analysis of the time evolution of the 2019-nCoV outbreak event in Italy is proposed and is based on the preliminary data at disposal (till March 11th, 2020) on one side, and on an epidemiological model recently used to describe the same epidemic event in the Wuhan region (February 2020) on the other side. The equations of the model include the description of compartments like Susceptible (S), exposed (E), infectious but not yet symptomatic (pre-symptomatic) (A), infectious with symptoms (I), hospitalized (H) and recovered (R). Further stratification includes quarantined susceptible (Sq), isolated exposed (Eq) and isolated infected (Iq) compartments. The equations are numerically solved for boundary (initial) conditions tuned on the Italian event. The role of quarantine is specifically emphasized and supports the strategies adopted providing a numerical description of the effects.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Marco Claudio Traini; Carla Caponi; Giuseppe Vittorio De Socio&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20034884&#39;},
 {&#39;cord_uid&#39;: &#39;7umn0vkv&#39;,
  &#39;sha&#39;: &#39;a11f079f3652b13e131525ee5b691525fac47382&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Maximum entropy method for estimating the reproduction number: An investigation for COVID-19 in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20035659&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The key parameter that characterizes the transmissibility of a disease is the reproduction number R. If it exceeds 1, the number of incident cases will inevitably grow over time, and a large epidemic is possible. To prevent the expansion of an epidemic, R must be reduced to a level below 1. To estimate the reproduction number, the probability distribution function of the generation interval of an infectious disease is required to be available; however, this distribution is often unknown. In this letter, given the incomplete information for the generation interval, we propose a maximum entropy method to estimate the reproduction number. Based on this method, given the mean value and variance of the generation interval, we first determine its probability distribution function and in turn estimate the real-time values of reproduction number of COVID-19 in China. By applying these estimated reproduction numbers into the susceptible-infectious-removed epidemic model, we simulate the evolutionary track of the epidemic in China, which is well in accordance with that of the real incident cases. The simulation results predict that China&#39;s epidemic will gradually tend to disappear by May 2020 if the quarantine measures can continue to be executed.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Yong Tao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20035659&#39;},
 {&#39;cord_uid&#39;: &#39;ba3we97o&#39;,
  &#39;sha&#39;: &#39;78ab35770cc98632c434523df48c6ba290182d37&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Excess cases of Influenza like illnesses in France synchronous with COVID19 invasion.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20035741&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Several French regions where COVID19 has been reported currently show a renewed increase in ILI cases in the general practice based Sentinelles network. Here we computed the number of excess cases by region and found correlation with the number of reported COVID19 cases so far. These data suggest larger circulation of SARS-CoV-2 in the French population than apparent from confirmed cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Pierre-Yves BOELLE&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20035741&#39;},
 {&#39;cord_uid&#39;: &#39;kvfau8j0&#39;,
  &#39;sha&#39;: &#39;3701cb3b329a07d9e6fe1986584be12f5eafe111&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Expected impact of COVID-19 outbreak in a major metropolitan area in Brazil&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20035873&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In January 2020 China reported to the World Health Organization an outbreak of pneumonia of undetermined origin in the city of Wuhan, Hubei. In January 30, 2020, the World Health Organization declared the outbreak of COVID-19 as a Public Health Emergency of International Interest (PHEI). Objectives: The aim of this study is to assess the impact of a COVID-19 epidemic in the metropolitan region of Sao Paulo, Brazil. Methods: We used a generalized SEIR (Susceptibles, Exposed, Infectious, Recovered) model, with additional Hospitalized variables (SEIHR model) and age-stratified structure to analyze the expected time evolution during the onset of the epidemic in the metropolitan area of Sao Paulo. The model allows to determine the evolution of the number of cases, the number of patients admitted to hospitals and deaths caused by COVID-19. In order to investigate the sensibility of our results with respect to parameter estimation errors we performed Monte Carlo analysis with 100 000 simulations by sampling parameter values from an uniform distribution in the confidence interval. Results: We estimate 1 368 (IQR: 880, 2 407) cases, 301 (22%) in older people (more than 60 years), 81 (50, 143) hospitalizations, and 14 (9, 26) deaths in the first 30 days, and 38 583 (IQR: 16 698, 113, 163) cases, 8 427 (21.8%) in older people (more than 60 years), 2181 (914, 6392) hospitalizations, and 397(166, 1205) deaths in the first 60 days. Limitations: We supposed a constant transmission probability Pc among different age-groups, and that every severe and critic case will be hospitalized, as well as that the detection capacity in all the primary healthcare services does not change during the outbreak. Conclusion: Supposing the reported parameters in the literature apply in the city of Sao Paulo, our study shows that it is expected that the impact of a COVID-19 outbreak will be important, requiring special planning from the authorities. This is the first study for a major metropolitan center in the south hemisphere, and we believe it can provide policymakers with a prognosis of the burden of the pandemic not only in Brazil, but also in other tropical zones, allowing to estimate total cases, hospitalization, and deaths, in support to the management of the public health emergence caused by COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Tarcisio M Rocha Filho; Fabiana S. Ganem dos Santos; Victor B Gomes; Thiago A.H. Rocha; Julio H.R. Croda; Walter M Ramalho; Wildo N Araujo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20035873&#39;},
 {&#39;cord_uid&#39;: &#39;jmc9yhr0&#39;,
  &#39;sha&#39;: &#39;c2298fc67d299e45f2458eb2b2c4cd37b7a1ef64&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Indications for healthcare surge capacity in European countries facing an exponential increase in COVID19 cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20035980&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;European healthcare systems face rapidly increasing pressure from COVID-19. We calculated pressures on EU healthcare systems by relating both country-specific accumulated COVID-19 deaths (intensity-approach) and active COVID-19 cases (magnitude-approach) to various estimates of hospital beds. On March 14 2020 - relative to Italy on March 11- we found Spain, Luxembourg, Switzerland and France to experience the highest pressure using the intensity-approach, versus Iceland, Denmark, Norway, Sweden, Spain, Switzerland and Slovenia using the magnitude approach.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Frederik Verelst; Elise J. Kuylen; Philippe Beutels&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20035980&#39;},
 {&#39;cord_uid&#39;: &#39;hhj7zte1&#39;,
  &#39;sha&#39;: &#39;f172175615d00fd8aef97db204fc8f23e56aef34&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;CoViD--19: An Automatic, Semiparametric Estimation Method for the Population Infected in Italy&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20036103&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;To date, official data on the number of people infected with the SARS-CoV-2 , responsible for the CoViD19 , have been released by the Italian Government just on the basis of a non representative sample of population which tested positive for the swab. However a reliable estimation of the number of infected, including asymptomatic people, turns out to be crucial in the preparation of operational schemes and to estimate the future number of people, who will require, to different extents, medical attentions. In order to overcome the current data shortcoming, this paper proposes a bootstrap driven, estimation procedure for the number of people infected with the SARSCoV2. This method is designed to be robust, automatic and suitable to generate estimations at regional level. Obtained results show that, while official data at March the 12th report 12.839 cases in Italy, people infected wiyh the SARSCoV2 could be as high as 105.789.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Livio Fenga&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20036103&#39;},
 {&#39;cord_uid&#39;: &#39;aa7slcnc&#39;,
  &#39;sha&#39;: &#39;6e818fd2341af1e16caf6411c7fbb4200a17646d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20036129&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The outbreak of COVID-19 caused by a novel Coronavirus (termed SARS-CoV-2) has spread to over 120 countries around the world. Currently, real-time qPCR (RT-qPCR) is used as the gold standard for diagnostics of SARS-CoV-2. However, the positive rate of RT-qPCR assay of pharyngeal swab samples is reported to be 30~60%. More accurate and sensitive methods are urgently needed. Method：We established a digital PCR (dPCR) protocol to detect SARS-CoV-2 on 194 clinical pharyngeal swab samples, including 103 suspected patients, 75 close contacts and 16 supposed convalescents. Results: The limit of blanks (LoBs) of the dPCR assays are about 1.6, 1.6 and 0.8 copies/reaction for ORF 1ab, N and E gene. The limit of detection (LoD) is 2 copies/reaction. The overall accuracy of ddPCR is 95.5 %. For the fever suspected patients, the accuracy of SARS-CoV-2 detection was significantly improved from 28.2% to 87.4% by dPCR. For close contacts, the suspect rate was greatly decreased from 21% down to 1%. In addition, quantification of the virus load for convalescents by dPCR showed that a longer observation in the hospital is needed for aged patients. Conclusion: dPCR could be a confirmatory method for suspected patients diagnosed by RT-qPCR. Furthermore, dPCR is more sensitive and suitable for low virus load specimens from the both patients under isolation and those under observation who may not be exhibiting clinical symptoms.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;lianhua dong; Junbo Zhou; Chunyan Niu; Quanyi Wang; Yang Pan; Xia Wang; Yongzhuo zhang; Jiayi yang; Manqing Liu; Yang zhao; Tao peng; Jie xie; yunhua gao; Di wang; yun zhao; Xinhua Dai; xiang fang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20036129&#39;},
 {&#39;cord_uid&#39;: &#39;gttuxtw6&#39;,
  &#39;sha&#39;: &#39;67e26529131aa6241f7f15af391c689c34183a4e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Correcting under-reported COVID-19 case numbers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20036178&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The COVID-19 virus has spread worldwide in a matter of a few months. Healthcare systems struggle to monitor and report current cases. Limited capabilities in testing result in difficult to guide policies and mitigate lack of preparation. Since severe cases, which more likely lead to fatal outcomes, are detected at a higher percentage than mild cases, the reported death rates are likely inflated in most countries. Such under-estimation can be attributed to the under-sampling of infection cases and results in systematic death rate estimation biases. The method proposed here utilizes a benchmark country (South Korea) and its reported death rates in combination with population demographics to correct the reported COVID-19 case numbers. By applying a correction, we predict that the number of cases is highly under-reported in most countries. In the case of China, it is estimated that more than 700.000 cases of COVID-19 actually occurred instead of the confirmed 80,932 cases as of 3/13/2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Alexander Lachmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20036178&#39;},
 {&#39;cord_uid&#39;: &#39;nf51yjmj&#39;,
  &#39;sha&#39;: &#39;9856876e0f976c1535272fc84aed92d3e2834bed&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Propagation analysis and prediction of the COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.20036202&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Based on the official data modeling, this paper studies the transmission process of the Corona Virus Disease 2019 (COVID-19). The error between the model and the official data curve is within 3%. At the same time, it realized forward prediction and backward inference of the epidemic situation, and the relevant analysis help relevant countries to make decisions.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;LiXiang Li; ZiHang Yang; ZhongKai Dang; Cui Meng; JingZe Huang; HaoTian Meng; DeYu Wang; GuanHua Chen; JiaXuan Zhang; HaiPeng Peng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.20036202&#39;},
 {&#39;cord_uid&#39;: &#39;5hpbjkft&#39;,
  &#39;sha&#39;: &#39;6ede7d40a9858d89341de780a3be9e8b43291151&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 invades host cells via a novel route: CD147-spike protein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.988345&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYCurrently, COVID-19 caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been widely spread around the world; nevertheless, so far there exist no specific antiviral drugs for treatment of the disease, which poses great challenge to control and contain the virus. Here, we reported a research finding that SARS-CoV-2 invaded host cells via a novel route of CD147-spike protein (SP). SP bound to CD147, a receptor on the host cells, thereby mediating the viral invasion. Our further research confirmed this finding. First, in vitro antiviral tests indicated Meplazumab, an anti-CD147 humanized antibody, significantly inhibited the viruses from invading host cells, with an EC50 of 24.86 μg/mL and IC50 of 15.16 μg/mL. Second, we validated the interaction between CD147 and SP, with an affinity constant of 1.85×10−7M. Co-Immunoprecipitation and ELISA also confirmed the binding of the two proteins. Finally, the localization of CD147 and SP was observed in SARS-CoV-2 infected Vero E6 cells by immuno-electron microscope. Therefore, the discovery of the new route CD147-SP for SARS-CoV-2 invading host cells provides a critical target for development of specific antiviral drugs.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-14&#39;,
  &#39;authors&#39;: &#39;Ke Wang; Wei Chen; Yu-Sen Zhou; Jian-Qi Lian; Zheng Zhang; Peng Du; Li Gong; Yang Zhang; Hong-Yong Cui; Jie-Jie Geng; Bin Wang; Xiu-Xuan Sun; Chun-Fu Wang; Xu Yang; Peng Lin; Yong-Qiang Deng; Ding Wei; Xiang-Min Yang; Yu-Meng Zhu; Kui Zhang; Zhao-Hui Zheng; Jin-Lin Miao; Ting Guo; Ying Shi; Jun Zhang; Ling Fu; Qing-Yi Wang; Huijie Bian; Ping Zhu; Zhi-Nan Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.988345&#39;},
 {&#39;cord_uid&#39;: &#39;fn7l93wh&#39;,
  &#39;sha&#39;: &#39;2c006d09b6fccc527bf5ee3de0f165b018c39e73&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Potential Neutralizing Antibodies Discovered for Novel Corona Virus Using Machine Learning&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.14.992156&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe fast and untraceable virus mutations take lives of thousands of people before the immune system can produce the inhibitory antibody. Recent outbreak of novel coronavirus infected and killed thousands of people in the world. Rapid methods in finding peptides or antibody sequences that can inhibit the viral epitopes of COVID-19 will save the life of thousands. In this paper, we devised a machine learning (ML) model to predict the possible inhibitory synthetic antibodies for Corona virus. We collected 1933 virus-antibody sequences and their clinical patient neutralization response and trained an ML model to predict the antibody response. Using graph featurization with variety of ML methods, we screened thousands of hypothetical antibody sequences and found 8 stable antibodies that potentially inhibit COVID-19. We combined bioinformatics, structural biology, and Molecular Dynamics (MD) simulations to verify the stability of the candidate antibodies that can inhibit the Corona virus.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Rishikesh Magar; Prakarsh Yadav; Amir Barati Farimani&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.14.992156&#39;},
 {&#39;cord_uid&#39;: &#39;t5ar9pp0&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;GENOMIC CHARACTERISATION AND PHYLOGENETIC ANALYSIS OF SARS-COV-2 IN ITALY&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20032870&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This report describes the isolation, the molecular characterization and the phylogenetic analysis of the first three complete genomes of SARS-CoV-2 isolated from three patients involved in the first outbreak of COVID-19 in Lombardy, Italy. Early molecular epidemiological tracing suggests that SARS-CoV-2 was present in Italy weeks before the first reported cases of infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Gianguglielmo Zehender; Alessia Lai; Annalisa Bergna; Luca Meroni; Agostino Riva; Claudia Balotta; Maciej Tarkowski; Arianna Gabrieli; Dario Bernacchia; Stefano Rusconi; Giuliano Rizzardini; Spinello Antinori; Massimo Galli&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20032870&#39;},
 {&#39;cord_uid&#39;: &#39;2okcuviq&#39;,
  &#39;sha&#39;: &#39;f08249cbb9bdfc3b7f103470537a52190c81b974&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Blood single cell immune profiling reveals the interferon-MAPK pathway mediated adaptive immune response for COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20033472&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The coronavirus disease 2019 (COVID-19) outbreak is an ongoing global health emergence, but the pathogenesis remains unclear. We revealed blood cell immune response profiles using 5&#39; mRNA, TCR and BCR V(D)J transcriptome analysis with single-cell resolution. Data from 134,620 PBMCs and 83,387 TCR and 12,601 BCR clones was obtained, and 56 blood cell subtypes and 23 new cell marker genes were identified from 16 participants. The number of specific subtypes of immune cells changed significantly when compared patients with controls. Activation of the interferon-MAPK pathway is the major defense mechanism, but MAPK transcription signaling is inhibited in cured patients. TCR and BCR V(D)J recombination is highly diverse in generating different antibodies against SARS-CoV-2. Therefore, the interferon-MAPK pathway and TCR- and BCR-produced antibodies play important roles in the COVID-19 immune response. Immune deficiency or immune over-response may result in the condition of patients with COVID-19 becoming critical or severe.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Lulin Huang; Yi Shi; Bo Gong; Li Jiang; Xiaoqi Liu; Jialiang Yang; Juan Tang; Chunfang You; Qi Jiang; Bo Long; Tao Zeng; Mei Luo; Fanwei Zeng; Fanxin Zeng; Shuqiang Wang; Xingxiang Yang; Zhenglin Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20033472&#39;},
 {&#39;cord_uid&#39;: &#39;hrrzztt5&#39;,
  &#39;sha&#39;: &#39;1c02f14de3ae4c5e26600b073ec6396caefb3830&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Temporal relationship between outbound traffic from Wuhan and the 2019 coronavirus disease (COVID-19) incidence in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20034199&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The city of Wuhan is the epicenter of the 2019 coronavirus disease (COVID-19) outbreak and a central Chinese hub for transportation and industry. Mass migration prior to the Chinese New Year may have accelerated the spread of COVID-19 across China. This analysis investigated the temporal relationship between daily outbound traffic from Wuhan and the incidence of COVID-19 in 31 Chinese provinces during January-February 2020. Methods: We collected incidence of confirmed COVID-19 cases from National and Provincial Health Committee reports from January 10 to February 29, 2020. Volume of outbound traffic from Wuhan to other provinces was measured by Baidu Migration Index, a widely used metric that tracks migration based on cellphone location. We used cross-correlation function and autoregressive integrated moving average (ARIMA) model to examine time-lagged association between traffic volume and COVID-19 incidence by province. Contributors to the provincial variation in the temporal associations were investigated. Additionally, we estimated the reduction in cumulative incidence of COVID-19 cases following the travel ban for Wuhan. Results: Cross-correlation function analyses suggested that the volume of outbound traffic from Wuhan was positively associated with COVID-19 incidence in all provinces, with correlation coefficients between 0.22-0.78 (all P&lt;0.05). Approximately 42% of provinces showed &lt;1 week of lag between traffic volume and COVID-19 incidence, 39% with 1 week, and 19% with 2-3 weeks. Migration had more prolonged impacts in provinces closer to Wuhan and with more passenger influx from Wuhan, but affected economically advantaged provinces to a lesser extent. We further estimated that the travel ban may have prevented approximately 19,768 COVID-19 cases (95% CI: 13,589, 25,946) outside of Wuhan by February 29, 2020. Conclusions: Outflowing migration from Wuhan facilitated the COVID-19 transmission to other parts of China with varying time-lagged effects dependent on provincial characteristics. The travel ban led to a significant reduction in COVID-19 outside of Wuhan.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Zaixing Shi; Ya Fang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20034199&#39;},
 {&#39;cord_uid&#39;: &#39;vi51uons&#39;,
  &#39;sha&#39;: &#39;1ee4cc235ce07b2d08d989e9001e2f4bd9af3138&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A high efficient hospital emergency responsive mode is the key of successful treatment of 100 COVID-19 patients in Zhuhai&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20034629&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Since December 2019, Coronavirus Disease 2019 (COVID-19) emerged in Wuhan city and rapidly spread throughout China. The mortality of novel coronavirus pneumonia (NCP) in severe and critical cases is very high. Facing this kind of public health emergency, high efficient administrative emergency responsive mode in designated hospital is needed. Method: As an affiliated hospital of Sun Yat-sen University, our hospital is the only designated one for diagnosis and treatment of COVID-19 in Zhuhai, a medium-sized city. Novel coronavirus pneumonia department, which is administrative led by the president of hospital directly, has been established at early stage of epidemic crisis in my hospital. In NCP department, there are core members of Pulmonary and Critical Care Medicine (PCCM) specialist and multidisciplinary team. Don&#39;t stick to national guidelines of NCP, based on professional opinion by respiratory professor and expert group, we focused on individualized treatment and timely adjustment of treatment and management strategies in working about COVID-19 patients. Results: 1. High working efficiency: By Mar 02, 2020, we have completed 2974 citywide consultations and treatment of 366 inpatients, including 101 patients diagnosed with COVID-19. 2. Excellent therapeutic effect：Among 101 hospitalized patients with confirmed COVID-19, all were cured and discharged, except for one death. No secondary hospital infection, no pipeline infection and no pressure sore were found in all patients. 3. Finding and confirming person-to-person transmission characteristic of COVID-19 prior to official release conference: Strengthened protection is key point to zero infection in healthcare group and medical faculty and lower rate of second generation infectious patients. 4. Timely adjustment management and treatment strategy prior to guideline update: The first evidence of digestive tract involvement in COVID-19 has been found, and the earliest clinical trial of chloroquine in the treatment of COVID-19 has been carried out in our hospital. Conclusions: In our hospital, establishment of NCP department, which is administratively led by the president of hospital directly and specialized conduct by respiratory professor, is the key to success in management and treatment of COVID-19 patients. This hospital emergency responsive mode could provide reference for other hospitals and cities in epidemic situation.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Jin Huang; Zhonghe Li; Xiujuan Qu; Xiaobin Zheng; Changli Tu; Meizhu Chen; Cuiyan Tan; Jing Liu; Hong Shan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20034629&#39;},
 {&#39;cord_uid&#39;: &#39;alsiqv04&#39;,
  &#39;sha&#39;: &#39;c952f4000c5163f168b943fcee51e0634066d9fc&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;International expansion of a novel SARS-CoV-2 mutant&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20035204&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Letter to the editor. There is no abstract. The summary was showed: SARS-CoV-2 has inevitably mutated during its pandemic spread to cause unpredictable effects on COVID-19 and complicate epidemic control efforts. Here we report that a novel SARS-CoV-2 mutation (ORF3a) appears to be spreading worldwide, which deserves close attention.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Minjin Wang; Mengjiao Li; Ruotong Ren; Andreas Brave; Sylvie van der Werf; En-Qiang Chen; Zhiyong Zong; Weimin Li; Binwu Ying&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20035204&#39;},
 {&#39;cord_uid&#39;: &#39;vof63qat&#39;,
  &#39;sha&#39;: &#39;179df1e769292dd113cef1b54b0b43213e6b5c97&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Systematic review and meta-analysis of predictive symptoms and comorbidities for severe COVID-19 infection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20035360&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background/introduction COVID−19, a novel coronavirus outbreak starting in China, is now a rapidly developing public health emergency of international concern. The clinical spectrum of COVID−19 disease is varied, and identifying factors associated with severe disease has been described as an urgent research priority. It has been noted that elderly patients with pre-existing comorbidities are more vulnerable to more severe disease. However, the specific symptoms and comorbidities that most strongly predict disease severity are unclear. We performed a systematic review and meta-analysis to identify the symptoms and comorbidities predictive of COVID−19 severity. Method This study was prospectively registered on PROSPERO. A literature search was performed in three databases (MEDLINE, EMBASE and Global Health) for studies indexed up to 5th March 2020. Two reviewers independently screened the literature and both also completed data extraction. Quality appraisal of studies was performed using the STROBE checklist. Random effects meta-analysis was performed for selected symptoms and comorbidities to identify those most associated with severe COVID−19 infection or ICU admission. Results Of the 2259 studies identified, 42 were selected after title and abstract analysis, and 7 studies (including 1813 COVID−19 patients) were chosen for inclusion. The ICU group were older (62.4 years) compared to the non-ICU group (46 years), with a significantly higher proportion of males (67.2% vs. 57.1%, p=0.04). Dyspnoea was the only significant symptom predictive for both severe disease (pOR 3.70, 95% CI 1.83 − 7.46) and ICU admission (pOR 6.55, 95% CI 4.28 − 10.0). Notwithstanding the low prevalence of COPD in severe disease and ICU-admitted groups (4.5% and 9.7%, respectively), COPD was the most strongly predictive comorbidity for both severe disease (pOR 6.42, 95% CI 2.44 − 16.9) and ICU admission (pOR 17.8, 95% CI 6.56 − 48.2). Cardiovascular disease and hypertension were also strongly predictive for both severe disease and ICU admission. Those with CVD and hypertension were 4.4 (95% CI 2.64 − 7.47) and 3.7 (95% CI 2.22 − 5.99) times more likely to have an ICU admission respectively, compared to patients without the comorbidity. Conclusions Dyspnoea was the only symptom strongly predictive for both severe disease and ICU admission, and could be useful in guiding clinical management decisions early in the course of illness. When looking at ICU-admitted patients, who represent the more severe end of the spectrum of clinical severity, COPD patients are particularly vulnerable, and those with cardiovascular disease and hypertension are also at a high-risk of severe illness. To aid clinical assessment, risk stratification, efficient resource allocation, and targeted public health interventions, future research must aim to further define those at high-risk of severe illness with COVID−19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-16&#39;,
  &#39;authors&#39;: &#39;Vageesh Jain; Jin-Min Yuan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20035360&#39;},
 {&#39;cord_uid&#39;: &#39;gv8wlo06&#39;,
  &#39;sha&#39;: &#39;1e574c1272c82d322baf7a2b3a539ebbd66f1697&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Demographic science aids in understanding the spread and fatality rates of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036293&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Governments around the world must rapidly mobilize and make difficult policy decisions to mitigate the COVID-19 pandemic. Because deaths have been concentrated at older ages, we highlight the important role of demography, particularly how the age structure of a population may help explain differences in fatality rates across countries and how transmission unfolds. We examine the role of age structure in deaths thus far in Italy and South Korea and illustrate how the pandemic could unfold in populations with similar population sizes but different age structures, showing a dramatically higher burden of mortality in countries with older versus younger populations. This powerful interaction of demography and current age-specific mortality for COVID-19 suggests that social distancing and other policies to slow transmission should consider both the age composition of local and national contexts as well as the social connectedness of older and younger generations. We also call for countries to provide case and fatality data disaggregated by age and sex to improve real-time targeted nowcasting.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Jennifer Beam Dowd; Valentina Rotondi; Liliana Adriano; David M Brazel; Per Block; Xuejie Ding; Yan Liu; Melinda C Mills&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036293&#39;},
 {&#39;cord_uid&#39;: &#39;jvhrp51s&#39;,
  &#39;sha&#39;: &#39;ce1d3a3103bbd8b5d7164259be03f1d6d3d5d251&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The clinical and epidemiological features and hints of 82 confirmed COVID-19 pediatric cases aged 0-16 in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036319&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Although COVID-19 pediatric patients just account for 1% of the overall cases, they are nonnegligible invisible infection sources. We quantitatively analyzed the clinical and epidemiological features of 82 confirmed cases aged 0-16 admitted to Wuhan Children&#39;s Hospital, which are expected to shed some lights onto the pediatric diagnosis and therapy.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Hui Yu; Qinzhen Cai; Xiang Dai; Xiuzhen Liu; Hongs Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036319&#39;},
 {&#39;cord_uid&#39;: &#39;qjwn1h2g&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Coronavirus disease-19: Summary of 2,370 Contact Investigations of the First 30 Cases in the Republic of Korea&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036350&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Between January 24 and March 10, of a total of 2,370 individuals who had contacted the first 30 cases of COVID19, 13 were found to have COVID19, resulting secondary attack rate of 0.55% (95% CI 0.31 to 0.96). Of 119 household contacts, 9 had infections resulting secondary attack rate of 7.56 (95% CI 3.73 to 14.26).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Young June Choe&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036350&#39;},
 {&#39;cord_uid&#39;: &#39;grx4gx95&#39;,
  &#39;sha&#39;: &#39;4ffcd1d8e0f56c5a238bbd4040ec23410ef71a1b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Coronavirus disease-19: The First 7,755 Cases in the Republic of Korea&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036368&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We report the first 7,755 patients with confirmed COVID19 in Korea as of March 13, 2020. A total of 66 deaths were identified, resulting case fatality proportion of 0.9%. Older people, and those with coexisting medical conditions were at risk for fatal outcomes. The highest number of cases were from Daegu, followed by Gyeongbuk, with elevated age-stratified case fatality. This summary may help to understand the disease dynamics in the early phase of COVID19 outbreak, therefore, to guide future public health measures.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Young June Choe&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036368&#39;},
 {&#39;cord_uid&#39;: &#39;eo2pcgix&#39;,
  &#39;sha&#39;: &#39;eac6f99c5187f0a254687b455c4fa8d59d190b3c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Development and validation of a rapid single-step reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) system potentially to be used for reliable and high-throughput screening of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036376&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The recent pandemic of COVID-19 has involved tens of thousands of patients in numerous countries and the causative virus, SARS COV-2 is highly transmissible. Molecular diagnostic tools are central to containment of the virus and initiating proper clinical care. Rapidity, user-friendliness, and high degree of sensitivity and specificity are desirable features of diagnostic assays for screening purposes. Herein, we present a single step reverse transcriptase LAMP assay (RT-LAMP), which can detect up to 500 viral copies in 30 minutes. We challenged our assay with a large number of clinical samples collected from 47 confirmed cases and 213 negative patients. Our LAMP assay showed a high degree of sensitivity and specificity compared to two commercialized qRT-PCR assay as gold standard. We present a rapid RT-LAMP assay that could extend the capacity of laboratories to process 2.5 more clinical samples relative to qRT-PCR and potentially could be used for high-throughput screening purposes.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Minghua Jiang; Wenjie Fang; Amir Aratehfar; Xiaojing Li; Liyan ling; Hua Fang; Farnaz Farnaz Daneshnia; Jian Yu; Wanqing Liao; Hao Pei; Weihua Pan; Cornelia Lass-Florl&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036376&#39;},
 {&#39;cord_uid&#39;: &#39;y5io06k6&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Routes for COVID-19 importation in Brazil&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036392&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The global outbreak caused by the severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) has been declared a pandemic by the WHO. As the number of imported SARS-CoV-2 cases is on the rise in Brazil, we use incidence and historical air travel data to estimate the most important routes of importation into the country.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Darlan da Silva Candido; Alexander Watts; Leandro Abade; Moritz UG Kraemer; Oliver G Pybus; Julio Croda; Wanderson de Oliveira; Kamran Khan; Ester C Sabino; Nuno R. Faria&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036392&#39;},
 {&#39;cord_uid&#39;: &#39;u1i5onqr&#39;,
  &#39;sha&#39;: &#39;d282b79ca6f7cafe79c135b5d21101eb54312651&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Understanding Epidemic Data and Statistics: A case study of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036418&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The 2019-Novel-Coronavirus (COVID-19) has affected 115 countries and out of more than 118,000 confirmed cases. Understanding the transmission dynamics of the infection in each country which affected on a daily basis and evaluating the effectiveness of control policies is critical for our further actions. To date, the statistics of COVID-10 reported cases show more than 80 percent of infected had a mild case of disease, while around 14 percent of infected experienced a severe one and about 5 percent are categorized as critical disease victims. Today&#39;s report (2020-03-12; daily updates in the prepared website) shows the confirmed cases of COVID-19 in China, South Korea, Italy, and Iran are 80932, 7869, 12462 and 10075; respectively. Calculating the total Case Fatality Rate (CFR) of Italy (2020-03-04), about 7.9% of confirmed cases passed away. Compared to South Korea&#39;s rate of 0.76% (10% lower than Italy) and China&#39;s 3.8% (50% lower than Italy), the CFR of Italy is too high. There are some effective policies that yield significant changes in the trend of cases. The lockdown policy in china and Italy (the effect observed after 11 days), Shutdown of all non-essential companies in Hubei (the effect observed after 5 days), combined policy in South Korea and reducing working hours in Iran.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Amirhoshang Hoseinpour Dehkordi; Majid Alizadeh; Pegah Derakhshan; Peyman Babazadeh; Arash Jahandideh&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036418&#39;},
 {&#39;cord_uid&#39;: &#39;f53i4n02&#39;,
  &#39;sha&#39;: &#39;bbd9d63dc2c733c763770f62205ef9adeceb0127&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effects of temperature variation and humidity on the mortality of COVID-19 in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036426&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Object Meteorological parameters are the important factors influencing the infectious diseases like severe acute respiratory syndrome (SARS). This study aims to explore the association between coronavirus disease (COVID-19) death and weather parameters. Methods In this study, we collected the daily death number of COVID-19, meteorological and air pollutant data from 20 January, 2020 to 29 February, 2020 in Wuhan, China. Then, the generalized additive model was applied to explore the impact of temperature, humidity and diurnal temperature range on daily mortality of COVID-19. Results There were in total 2299 COVID-19 mortality counts in Wuhan. A positive association with COVID-19 mortality was observed for diurnal temperature range (r = 0.44), but negative association for relative humidity (r = −0.32). In addition, each 1 unit increase in diurnal temperature range was only associated with a 2.92% (95% CI: 0.61%, 5.28%) increase in COVID-19 mortality at lag 3. However, both per 1 unit increase of temperature and absolute humidity were related to the decreased COVID-19 mortality at lag 3 and lag 5, respectively. Conclusion In summary, this study suggests the temperature variation and humidity may be important factors affecting the COVID-19 mortality.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Yueling Ma; Yadong Zhao; Jiangtao Liu; Xiaotao He; Bo Wang; Shihua Fu; Jun Yan; Jingping Niu; Bin Luo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036426&#39;},
 {&#39;cord_uid&#39;: &#39;djq0lvr2&#39;,
  &#39;sha&#39;: &#39;d09c0f71b1a404a592d0dcad2c3409d43c375f99&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036533&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of a new coronavirus (SARS-CoV-2) disease (Covid-19) has become pandemic. To be more effectively controlling the disease, it is critical to set up an optimal quarantine period so that about 95% of the cases developing symptoms will be retained for isolation. At the moment, the WHO-established quarantine period is 14 days based on previous reports which had studied small sizes of hospitalized cases (10 and ~100, respectively), however, over 80% of adult- and 95% of child-cases were not necessary to stay in hospitals, and therefore, had not been hospitalized. Therefore, we are questioning if the current-inferred median incubation time is representative for the whole Covid-19 population, and if the current quarantine period is optimal. Methods We compiled and analyzed the patient-level information of 2015 laboratory-confirmed Covid-19 cases including 99 children in 28 Chinese provinces. This cohort represents a wide-range spectrum of Covid-19 disease with both hospitalized and non-hospitalized cases. Results The full range of incubation periods of the Covid-19 cases ranged from 0 to 33 days among 2015 cases. There were 6 (0.13%) symptom-free cases including 4 females with a median age of 25.5 years and 2 males with a median age of 36 years. The median incubation period of both male and female adults was similar (7-day) but significantly shorter than that (9-day) of child cases (P=0.02). This cohort contained 4 transmission generations, and incubation periods of the cases between generations were not significantly different, suggesting that the virus has not been rapidly adapted to human beings. Interestingly, incubation periods of 233 cases (11.6%) were longer than the WHO-established quarantine period (14 days). Data modeling suggested that if adults take an extra 4-day or 7-day of isolation (i.e., a quarantine period of 18 or 21 days), 96.2% or 98.3%, respectively, of the people who are developing symptoms will be more effectively quarantined. Patients transmitted via lunch/dinner parties (i.e., gastrointestinal tract infection through oral transmission) had a significantly longer incubation period (9-day) than other adults transmitted via respiratory droplets or contaminated surfaces and objects (P&lt;0.004). Conclusions The whole Covid-19 population including both hospitalized and non-hospitalized cases had a median incubation period of 7-day for adults, which is 1.8-day longer than the hospitalized cases reported previously. An extension of the adult quarantine period to 18 days or 21 days could be more effective in preventing virus-spreading and controlling the disease. The cases transmitted by lunch/dinner parties could be infected first in the gastrointestinal tract through oral transmission and then infected in the respiratory system so that they had a longer incubation period.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Xue Jiang; Yawei Niu; Xiong Li; Lin Li; Wenxiang Cai; Yucan Chen; Bo Liao; Edwin Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036533&#39;},
 {&#39;cord_uid&#39;: &#39;61o5f82j&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmissibility of coronavirus disease 2019 (COVID-19) in Chinese cities with different transmission dynamics of imported cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036541&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Monitoring the time-varying reproduction number (Rt) of the disease is useful in determining whether there is sustained transmission in a population. In this study, we examined Rt of COVID-19 and compared its transmissibility between different intervention periods in Hangzhou and Shenzhen. Methods: Daily aggregated counts of confirmed imported and local cases between January 1, 2020 and March 13, 2020 were analysed. A likelihood function was constructed to estimate Rt, accounting for imported cases. Results: Although Hangzhou had fewer number of cases than Shenzhen, Shenzhen had higher proportion of imported cases than Hangzhou (83% vs 29%). Since the epidemic of COVID-19 in Shenzhen was dominated by imported cases, Rt was kept below unity through time. On the contrary, Rt was greater than unity in Hangzhou from 16 January to 7 February due to the surge in local cases. Credits to the Wuhan lockdown and outbreak response measures following the local lockdown, Rt decreased steadily and dropped below unity in mid-February. Conclusion: The lockdown measures and local outbreak responses helped reduce the potential of local transmission in Hangzhou and Shenzhen. Meanwhile, cities with similar epidemic trend could have different transmission dynamics given the variation in imported cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Ka Chun Chong; Wei Cheng; Shi Zhao; Feng Ling; Kirran Mohammad; Maggie Wang; Benny Zee; Lei Wei; Xi Xiong; Hengyan Liu; Jingxuan Wang; Enfu Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036541&#39;},
 {&#39;cord_uid&#39;: &#39;fb8mca1h&#39;,
  &#39;sha&#39;: &#39;65ed294672533d6734d1c0d66b1c987101a42fcf&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating unobserved SARS-CoV-2 infections in the United States&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036582&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Countries across the world observed dramatic rises in COVID-19 cases and deaths in March 2020. In the United States, delays in the availability of diagnostic testing have prompted questions about the extent of unobserved community transmission. Using a simulation model informed by reported cases and deaths, we estimated that tens of thousands of people (median: \u200b22,876, 95% posterior predictive interval: 7,451 - 53,044) were infected by the time a national emergency was declared. Our results also indicate that fewer than 10% of locally acquired, symptomatic infections in the US were detected throughout much of late February. These results point to the need for immediate, large-scale efforts to mitigate the impacts of SARS-CoV-2 on the US.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Alex Perkins; Sean M. Cavany; Sean M Moore; Rachel J Oidtman; Anita Lerch; Marya Poterek&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036582&#39;},
 {&#39;cord_uid&#39;: &#39;lh4cs5qz&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effect of large-scale testing platform in prevention and control of the COVID-19 pandemic: an empirical study with a novel numerical model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036624&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: China adopted an unprecedented province-scale quarantine since January 23rd 2020, after the novel coronavirus (COVID-19) broke out in Wuhan in December 2019. Responding to the challenge of limited testing capacity, large-scale standardized and fully-automated laboratory (Huo-Yan) was built as an ad-hoc measure. There was so far no empirical data or mathematical model to reveal the impact of the testing capacity improvement since the quarantine. Methods: We integrated public data released by the Health Commission of Hubei Province and Huo-Yan Laboratory testing data into a novel differential model with non-linear transfer coefficients and competitive compartments, to evaluate the trends of suspected cases under different nucleic acid testing capacities. Results: Without the establishment of Huo-Yan, the suspected cases would increased by 47% to 33,700, the corresponding cost of the quarantine would be doubled, and the turning point of the increment of suspected cases and the achievement of &#34;daily settlement&#34; (all daily new discovered suspected cases were diagnosed according to the nucleic acid testing results) would be delayed for a whole week and 11 days. If the Huo-Yan Laboratory could ran at its full capacity, the number of suspected cases could started to decrease at least a week earlier, the peak of suspected cases would be reduced by at least 44% and the quarantine cost could be reduced by more than 72%. Ideally, if a daily testing capacity of 10,500 could achieved immediately after the Hubei lockdown, &#34;daily settlement&#34; for all suspected cases would be achieved immediately. Conclusions: Large-scale and standardized clinical testing platform with nucleic acid testing, high-throughput sequencing and immunoprotein assessment capabilities need to be implemented simultaneously in order to maximize the effect of quarantine and minimize the duration and cost. Such infrastructure like Huo-Yan, is of great significance for the early prevention and control of infectious diseases for both common times and emergencies.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Qing Xie; Jing Wang; Jianling You; Shida Zhu; Rui Zhou; Zhijian Tian; Hao Wu; Yang Lin; Wei Chen; Lan Xiao; Xin Jin; Jianjuan Li; Jie Dong; Honglong Wu; Wei Zhang; Jing Li; Xun Xu; Ye Yin; Feng Mu; Weijun Chen; Jian Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036624&#39;},
 {&#39;cord_uid&#39;: &#39;rbzl0txp&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Stability of SARS-CoV-2 in different environmental conditions&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036673&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Stability of SARS-CoV-2 in different environmental conditions.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Alex Chin; Julie Chu; Mahen Perera; Kenrie Hui; Hui-Ling Yen; Michael Chan; Malik Peiris; Leo Poon&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036673&#39;},
 {&#39;cord_uid&#39;: &#39;eflwztji&#39;,
  &#39;sha&#39;: &#39;ef69fe3596cf5422a4dce8d21e2892687b1241e7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Temporal dynamics in viral shedding and transmissibility of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036707&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We report temporal patterns of viral shedding in 94 laboratory-confirmed COVID-19 patients and modelled COVID-19 infectiousness profile from a separate sample of 77 infector-infectee transmission pairs. We observed the highest viral load in throat swabs at the time of symptom onset, and inferred that infectiousness peaked on or before symptom onset. We estimated that 44% of transmission could occur before first symptoms of the index. Disease control measures should be adjusted to account for probable substantial pre-symptomatic transmission.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Xi He; Eric HY Lau; Peng Wu; Xilong Deng; Jian Wang; Xinxin Hao; Yiu Chung Lau; Jessica Y Wong; Yujuan Guan; Xinghua Tan; Xiaoneng Mo; Yanqing Chen; Baolin Liao; Weilie Chen; Fengyu Hu; Qing Zhang; Mingqiu Zhong; Yanrong Wu; Lingzhai Zhao; Fuchun Zhang; Benjamin J Cowling; Fang Li; Gabriel M Leung&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036707&#39;},
 {&#39;cord_uid&#39;: &#39;xqs4baou&#39;,
  &#39;sha&#39;: &#39;98629ae2533d73afbb0f5ef9d0cc813bf3afa851&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis of Epidemic Situation of New Coronavirus Infection at Home and Abroad Based on Rescaled Range (R/S) Method&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.20036756&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;1Summary 1.1 Background The outbreak of the new coronavirus infection broke out in Wuhan City, Hubei Province in December 2019, and has spread to 97 countries and regions around the world. Apart from China, there are currently three other severely affected areas, namely Italy, South Korea, and Iran. This poses a huge threat to China&#39;s and even global public health security, challenges scientific research work such as disease surveillance and tracking, clinical treatment, and vaccine development, and it also brings huge uncertainty to the global economy. As of March 11, 2020, the epidemic situation in China is nearing its end, but the epidemic situation abroad is in the outbreak period. Italy has even taken measures to close the city nationwide, with a total of 118,020 cases of infection worldwide. 1.2 Method This article selects the data of newly confirmed cases of COVID-19 at home and abroad as the data sample. Among them: the data of newly confirmed cases abroad is represented by Italy, and the span is from February 13 to March 10. The data of newly confirmed cases at home are divided into two parts: Hubei Province and other provinces except Hubei Province, spanning from January 23 to March 3, and with February 12 as the cutting point, it is divided into two periods, the growth period and the recession period. The rescaled range (R / S) analysis method and the dimensionless fractal Hurst exponent are used to measure the correlation of time series to determine whether the time series conforms to the fractal Brownian motion, that is, a biased random process. Contrast analysis of the meaning of H value in different stages and different overall H values in the same stage. 1.3 Results Based on R / S analysis and calculated Hurst value of newly confirmed cases in Hubei and non-Hubei provinces, it was found that the H value of Hubei Province in the first stage was 0.574, which is greater than 0.5, indicating that the future time series has a positive correlation and Fractal characteristics; The H value in the second stage is 1.368, which is greater than 1, which indicates that the future epidemic situation is completely preventable and controllable, and the second stage has a downward trend characteristic, which indicates that there is a high probability that the future time series will decline. The H values of the first and second stages of non-Hubei Province are 0.223 and 0.387, respectively, which are both less than 0.5, indicating that the time series of confirmed cases in the future is likely to return to historical points, and the H value in the second stage is greater than that in the first stage, indicating that the time series of confirmed cases in the second stage is more long-term memory than the time series of confirmed cases in the first stage. The daily absolute number of newly confirmed cases in Italy was converted to the daily growth rate of confirmed cases to eliminate the volatility of the data. The H value was 1.853, which was greater than 1, indicating that the time series of future confirmed cases is similar to the trend of historical changes. The daily rate of change in cases will continue to rise. 1.4 Conclusion According to the different interpretation of the H value obtained by the R / S analysis method, hierarchical isolation measures are adopted accordingly. When the H value is greater than 0.5, it indicates that the development of the epidemic situation in the area has more long-term memory, that is, when the number of confirmed cases in the past increases rapidly, the probability of the time series of confirmed cases in the future will continue the historical trend. Therefore, it is necessary to formulate strict anti-epidemic measures in accordance with the actual conditions of various countries, to detect, isolate, and treat early to reduce the base of infectious agents.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Xiaofeng Ji; Zhou Tang; Kejian Wang; Xianbin Li; Houqiang Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.20036756&#39;},
 {&#39;cord_uid&#39;: &#39;9fy5ska6&#39;,
  &#39;sha&#39;: &#39;f9a5880da2dbac60b06c520c9af75985d5d6c24c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The biomechanical role of extra-axonemal structures in shaping the flagellar beat of Euglena&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.991331&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWe propose and discuss a model for flagellar mechanics in Euglena gracilis. We show that the peculiar non-planar shapes of its beating flagellum, dubbed “spinning lasso”, arise from the mechanical interactions between two of its inner components, namely, the axoneme and the paraflagellar rod. The spontaneous shape of the axoneme and the resting shape of the paraflagellar rod are incompatible. The complex non-planar configurations of the coupled system emerge as the energetically optimal compromise between the two antagonistic components. The model is able to reproduce the experimentally observed flagellar beats and their characteristic spinning lasso geometric signature, namely, travelling waves of torsion with alternating sing along the length of the flagellum.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-15&#39;,
  &#39;authors&#39;: &#39;Giancarlo Cicconofri; Giovanni Noselli; Antonio DeSimone&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.991331&#39;},
 {&#39;cord_uid&#39;: &#39;unvabosp&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;RBD mutations from circulating SARS-CoV-2 strains enhance the structural stability and human ACE2 affinity of the spike protein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.991844&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;A novel coronavirus SARS-CoV-2 is associated with the current global pandemic of Coronavirus Disease 2019 (COVID-19). Bats and pangolins are suspected as the reservoir and the intermediate host. The receptor binding domain (RBD) of the SARS-CoV-2 S protein plays the key role in the tight binding to human receptor ACE2 for viral entry. Here, we analyzed the worldwide RBD mutants and found 18 mutant strains fell into 8 mutation types under high positive selection pressure during the spread. The equilibrium dissociation constant (KD) of three types of RBD mutants emerging in Wuhan, Shenzhen, Hong Kong and France were two orders of magnitude lower than the prototype Wuhan-Hu-1 strain due to the stabilization of the beta-sheet scaffold of the RBD. This indicated that the mutated viruses may have evolved to acquire remarkably increased infectivity. Five France isolates and one Hong Kong isolate shared the same RBD mutation enhancing the binding affinity, which suggested that they may have originated as a novel sub-lineage. The KD values for the bat and the pangolin SARS-like CoV RBDs indicated that it would be difficult for bat SARS-like CoV to infect humans; however, the pangolin CoV is potentially infectious to humans with respect to its RBD. These analyses of critical RBD mutations provide further insights into the evolutionary trend of SARS-CoV-2 under high selection pressure. The enhancement of the SARS-CoV-2 binding affinity to ACE2 reveals a possible higher risk of more severe virus transmissions during a sustained pandemic of COVID-19 if no effective precautions are implemented.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Junxian Ou; Zhonghua Zhou; Jing Zhang; Wendong Lan; Shan Zhao; Jianguo Wu; Donald Seto; Gong Zhang; Qiwei Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.991844&#39;},
 {&#39;cord_uid&#39;: &#39;izkz1hz4&#39;,
  &#39;sha&#39;: &#39;5dc2c157ddcd97f36c94faab478c38f463f78ae3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An emergent clade of SARS-CoV-2 linked to returned travellers from Iran&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.992818&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe SARS-CoV-2 epidemic has rapidly spread outside China with major outbreaks occurring in Italy, South Korea and Iran. Phylogenetic analyses of whole genome sequencing data identified a distinct SARS-CoV-2 clade linked to travellers returning from Iran to Australia and New Zealand. This study highlights potential viral diversity driving the epidemic in Iran, and underscores the power of rapid genome sequencing and public data sharing to improve the detection and management of emerging infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;John-Sebastian Eden; Rebecca Rockett; Ian Carter; Hossinur Rahman; Joep de Ligt; James Hadfield; Matthew Storey; Xiaoyun Ren; Rachel Tulloch; Kerri Basile; Jessica Wells; Roy Byun; Nicky Gilroy; Matthew V O’Sullivan; Vitali Sintchenko; Sharon C Chen; Susan Maddocks; Tania C Sorrell; Edward C Holmes; Dominic E Dwyer; Jen Kok&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.992818&#39;},
 {&#39;cord_uid&#39;: &#39;ebbzx8yr&#39;,
  &#39;sha&#39;: &#39;ddc40ffc47069f3714c8da65843f36f41ed0f8bb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.992883&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYSARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease—COVID-19— within several months of its initial identification. Comparable to the first SARS-CoV, this novel coronavirus’s surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 Å. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the “closed” prefusion S structure, but is accessible in “open” conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses.HIGHLIGHTSHigh resolution structure of the SARS-CoV-2 Receptor-Binding-Domain (RBD).Recognition of the SARS-CoV-2 RBD by SARS-CoV antibodies.Structure of the SARS-COV-2 RBD in complex with antibody CR3022.Identification of a cryptic site of vulnerability on the SARS-CoV-2 Spike.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;M. Gordon Joyce; Rajeshwer S. Sankhala; Wei-Hung Chen; Misook Choe; Hongjun Bai; Agnes Hajduczki; Lianying Yan; Spencer L. Sterling; Caroline E. Peterson; Ethan C. Green; Clayton Smith; Natalia de Val; Mihret Amare; Paul Scott; Eric D. Laing; Christopher C. Broder; Morgane Rolland; Nelson L. Michael; Kayvon Modjarrad&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.992883&#39;},
 {&#39;cord_uid&#39;: &#39;ukz73rp2&#39;,
  &#39;sha&#39;: &#39;e49c9e5b4f77c5cdbf6fa77343d767d909548dea&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.15.993097&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe World Health Organization has recently declared the ongoing outbreak of COVID-19, which is caused by a novel coronavirus SARS-CoV-2, as pandemic. There is currently a lack of knowledge in the antibody response elicited from SARS-CoV-2 infection. One major immunological question is concerning the antigenic differences between SARS-CoV-2 and SARS-CoV. We address this question by using plasma from patients infected by SARS-CoV-2 or SARS-CoV, and plasma obtained from infected or immunized mice. Our results show that while cross-reactivity in antibody binding to the spike protein is common, cross-neutralization of the live viruses is rare, indicating the presence of non-neutralizing antibody response to conserved epitopes in the spike. Whether these non-neutralizing antibody responses will lead to antibody-dependent disease enhancement needs to be addressed in the future. Overall, this study not only addresses a fundamental question regarding the antigenicity differences between SARS-CoV-2 and SARS-CoV, but also has important implications in vaccine development.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Huibin Lv; Nicholas C. Wu; Owen Tak-Yin Tsang; Meng Yuan; Ranawaka A. P. M. Perera; Wai Shing Leung; Ray T. Y. So; Jacky Man Chun Chan; Garrick K. Yip; Thomas Shiu Hong Chik; Yiquan Wang; Chris Yau Chung Choi; Yihan Lin; Wilson W. Ng; Jincun Zhao; Leo L. M. Poon; J. S. Malik Peiris; Ian A. Wilson; Chris K. P. Mok&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.15.993097&#39;},
 {&#39;cord_uid&#39;: &#39;szg12wfa&#39;,
  &#39;sha&#39;: &#39;00340eea543336d54adda18236424de6a5e91c9d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Regaining perspective on SARS-CoV-2 molecular tracing and its implications&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20034470&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;During the past three months, a new coronavirus (SARS-CoV-2) epidemic has been growing exponentially, affecting over 100 thousand people worldwide, and causing enormous distress to economies and societies of affected countries. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined all full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies. Our analysis clearly shows severe limitations in the present data, in light of which any finding should be considered, at the very best, preliminary and hypothesis-generating. Hence the need for avoiding stigmatization based on partial information, and for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Carla Mavian; Simone Marini; Costanza Manes; Ilaria Capua; Mattia Prosperi; Marco Salemi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20034470&#39;},
 {&#39;cord_uid&#39;: &#39;fk2wc715&#39;,
  &#39;sha&#39;: &#39;b12f20ae397aa504f852fc25caeb3e3b0e5b1942&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Advance of Novel Coronavirus Registration Clinical Trial&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20034934&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background.This research aims to analyze the clinical characteristics of a novel Corona Virus Disease (COVID-19) registered in the Chinese Clinical Trial Registry (ChiCTR), and to provide an information-based data foundation and information reference for curbing the spread of the COVID-19 epidemic. Methods.The ChiCTR database was comprehensively searched, and the COVID-19 research registration information as of February 24, 2020 was counted. Descriptive analysis of registration time trends and geographical distribution. Chi-square test was used to compare the statistical differences between different study types, intervention methods, research phases and major sponsors. Frequency studies were used to analyze the characteristics of CMT and BI therapy. Results.232 COVID-19 studies registered at the ChiCTR Center were collected. The overall number of COVID-19 registrations was increased. Hubei Province, China has the largest number of registrations, followed by Guangdong, Zhejiang and Sichuan. There were significant differences between the number of participants (P=0.000), study duration (P=0.008), study assignment (P=0.000), and blind method (P=0.000) for different study types. Significant differences could be seen in the dimensions of multicenter study(P=0.022), of participants numbe(P=0.000), study duration(P=0.000) and study assignment(P=0.001) for the four intervention methods. There were significant differences in study assignment (P=0.043) between the early and late studies. CMT drugs with high research frequency are chloroquine, lopinavir / ritonavir, and I-IFN, which are all in phase 4; BI was Cell therapy, plasma therapy, Thymosin, and M/P-AB, which are all in early stages. Conclusion.There were problems of unclear classification of research types and irregular registration behavior. Also, within the studies researched, heterogeneity exists for various dimensions. Different research types, intervention methods, and research stages lead to significant differences in some dimensions of the COVID-19 study. Finally, statistical high-frequency research drugs can provide more treatment options and evidence-based evidence for the clinical practice.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;gao song; meng qun cheng; xian wen wei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20034934&#39;},
 {&#39;cord_uid&#39;: &#39;lm9pay2b&#39;,
  &#39;sha&#39;: &#39;984e07e449dde660098275470dd952cc88df3223&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluation of Nucleocapsid and Spike Protein-based ELISAs for detecting antibodies against SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20035014&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: At present, PCR-based nucleic acid detection cannot meet the demands for coronavirus infectious disease (COVID-19) diagnosis. Methods: 214 confirmed COVID-19 patients who were hospitalized in the General Hospital of Central Theater Command of the People&#39;s Liberation Army between January 18 and February 26, 2020, were recruited. Two Enzyme-Linked Immunosorbent Assay (ELISA) kits based on recombinant SARS-CoV-2 nucleocapsid protein (rN) and spike protein (rS) were used for detecting IgM and IgG antibodies, and their diagnostic feasibility was evaluated. Results: Among the 214 patients, 146 (68.2%) and 150 (70.1%) were successfully diagnosed with the rN-based IgM and IgG ELISAs, respectively; 165 (77.1%) and 159 (74.3%) were successfully diagnosed with the rS-based IgM and IgG ELISAs, respectively. The positive rates of the rN-based and rS-based ELISAs for antibody (IgM and/or IgG) detection were 80.4% and 82.2%, respectively. The sensitivity of the rS-based ELISA for IgM detection was significantly higher than that of the rN-based ELISA. We observed an increase in the positive rate for IgM and IgG with an increasing number of days post-disease onset (d.p.o.), but the positive rate of IgM dropped after 35 d.p.o. The positive rate of rN-based and rS-based IgM and IgG ELISAs was less than 60% during the early stage of the illness 0-10 d.p.o., and that of IgM and IgG was obviously increased after 10 d.p.o. Conclusions: ELISA has a high sensitivity, especially for the detection of serum samples from patients after 10 d.p.o, it can be an important supplementary method for COVID-19 diagnosis.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Wanbing Liu; Lei Liu; Guomei Kou; Yaqiong Zheng; Yinjuan Ding; Wenxu Ni; Qiongshu Wang; Li Tan; Wanlei Wu; Shi Tang; Zhou Xiong; Shangen Zheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20035014&#39;},
 {&#39;cord_uid&#39;: &#39;ksbha7kz&#39;,
  &#39;sha&#39;: &#39;6c7c93c25caa6d94c70c891e2446a4a12e729365&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Acute myelitis after SARS-CoV-2 infection: a case report.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20035105&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We firstly reported a case of acute myelitis in a SARS-CoV-2-infected patient. A 66-year-old man with COVID-19 was admitted with acute flaccid paralysis of bilateral lower limbs and urinary and bowel incontinence. All serum microbiological studies were negative except for SARS-CoV-2 nucleic acid testing. Clinical findings could be ascribed to a post-infectious acute myelitis. He was receiving treatment with ganciclovir, lopinavir/ritonavir, moxifloxacin, dexamethasone, human immunoglobulin and mecobalamin. With a diagnosis of post-infectious acute myelitis and comprehensive treatment, paralysis of bilateral lower extremities ameliorated. After two times negative novel coronavirus RNA nasopharyngeal swabs tests, he was discharged and transferred to a designated hospital for isolation treatment and rehabilitation therapy.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Kang Zhao; Jucun Huang; Dan Dai; Yuwei Feng; Liming Liu; Shuke Nie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20035105&#39;},
 {&#39;cord_uid&#39;: &#39;8znnq0rh&#39;,
  &#39;sha&#39;: &#39;9ad252c633b7596a4b64ca9f1699a0db5ce99c05&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20036145&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Currently, there are no approved specific antiviral agents for 2019 novel coronavirus disease (COVID-19). In this study, ten severe patients confirmed by real-time viral RNA test were enrolled prospectively. One dose of 200 mL convalescent plasma (CP) derived from recently recovered donors with the neutralizing antibody titers above 1:640 was transfused to the patients as an addition to maximal supportive care and antiviral agents. The primary endpoint was the safety of CP transfusion. The second endpoints were the improvement of clinical symptoms and laboratory parameters within 3 days after CP transfusion. The median time from onset of illness to CP transfusion was 16.5 days. After CP transfusion, the level of neutralizing antibody increased rapidly up to 1:640 in five cases, while that of the other four cases maintained at a high level (1:640). The clinical symptoms were significantly improved along with increase of oxyhemoglobin saturation within 3 days. Several parameters tended to improve as compared to pre-transfusion, including increased lymphocyte counts (0.65*109/L vs. 0.76*109/L) and decreased C-reactive protein (55.98 mg/L vs. 18.13 mg/L). Radiological examinations showed varying degrees of absorption of lung lesionswithin 7 days. The viral load was undetectable after transfusion in seven patients who had previous viremia. No severe adverse effects were observed. This study showed CP therapy was welltolerated and could potentially improve the clinical outcomes through neutralizing viremia in severe COVID-19 cases. The optimal dose and time point, as well as the clinical benefit of CP therapy, needs further investigation in larger well-controlled trials.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Kai Duan; Bende Liu; Cesheng Li; Huajun Zhang; Ting Yu; Jieming Qu; Min Zhou; Li Chen; Shengli Meng; Yong Hu; Cheng Peng; Mingchao Yuan; Jinyan Huang; Zejun Wang; Jianhong Yu; Xiaoxiao Gao; Dan Wang; Xiaoqi Yu; Li Li; Jiayou Zhang; Xiao Wu; Bei Li; Yanping Yu; Wei Chen; Yan Peng; Yeqin Hu; Lianzhen Lin; Xuefei Liu; Shihe Huang; Zhijun Zhou; Lianghao Zhang; Yue Wang; Zhi Zhang; Kun Deng; Zhiwu Xia; Qin Gong; Wei Zhang; Xiaobei Zheng; Ying Liu; Huichuan Yang; Dongbo Zhou; Ding Yu; Jifeng Hou; Zhengli Shi; Saijuan Chen; Zhu Chen; Xin-xin Zhang; Xiaoming Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20036145&#39;},
 {&#39;cord_uid&#39;: &#39;rn59srwo&#39;,
  &#39;sha&#39;: &#39;584b43ae1d473b93ad7b6c95d681e8f356ba82cd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preprint server use in kidney disease research: a rapid review&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20036590&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Purpose of review: Preprint servers including arXiv and bioRxiv have disrupted the scientific communication landscape by providing rapid access to pre-peer reviewed research. MedRxiv is a recently launched free online repository for preprints in the health sciences. We sought to summarize potential benefits and risks to preprint server use, from both the researcher and end-user perspective, and evaluate the uptake of preprint servers in the nephrology community. Sources of Information: We performed a rapid review of articles describing preprint servers and their use. We approached the 20 highest impact nephrology journals regarding their policy towards the use of preprint servers. We evaluated the average time from study completion to publication of impactful articles in nephrology. Finally, we evaluated the number of nephrology articles submitted to preprint servers. Findings: To date over 600 kidney-related articles have been uploaded to bioRxiv and medRxiv. The average time from study completion to publication was over 10 months. 16 of the top 20 nephrology journals currently accept research submitted to a preprint server. Transparency and collaboration, visibility and recognition, and rapid dissemination of results were identified as benefits of preprint servers. Concerns exist regarding the potential risk of non-peer reviewed medical research being publicly available. Limitations: Preprint servers remain a recent phenomenon in health sciences and their long-term impact on the medical literature remains to be seen. Implications: The quantity of research submitted to preprint servers is likely to continue to grow. The model for dissemination of research results will need to adapt to incorporate preprint servers.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Caitlyn Vlasschaert; Cameron Giles; Swapnil Hiremath; Matthew Lanktree&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20036590&#39;},
 {&#39;cord_uid&#39;: &#39;37dadupn&#39;,
  &#39;sha&#39;: &#39;a5c214b0c76d5a11c11deef9d25adafaffa4bfe9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Building a COVID-19 Vulnerability Index&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20036723&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Dave DeCaprio; Joseph A Gartner; Thadeus Burgess; Sarthak Kothari; Shaayaan Sayed; Carol J McCall&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20036723&#39;},
 {&#39;cord_uid&#39;: &#39;nnlynjoy&#39;,
  &#39;sha&#39;: &#39;430088e2d738c4c20413082888b2666668c9105f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effect of continuous renal replacement therapy on all-cause mortality in COVID-19 patients undergoing invasive mechanical ventilation: a retrospective cohort study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20036780&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since December 2019, when coronavirus disease 2019 (COVID 19) emerged in Wuhan and rapidly spread throughout the world, critically ill patients have a high mortality rate. We aimed to assess the effect of continuous renal replacement therapy (CRRT) on all cause mortality in patients with COVID 19 undergoing invasive mechanical ventilation. Methods: In this retrospective cohort study, we included all patients with COVID 19 undergoing invasive mechanical ventilation at Optical Valley Branch of Tongji Hospital Affiliated with Tongji Medical College, Huazhong University of Science and Technology, Wuhan from February 12th to March 2nd, 2020. Demographic, clinical, laboratory, and treatment data were collected and analyzed. All patients were followed until death or end of follow up (March 9th), and all survivors were followed for at least one week. Findings: Of 36 hospitalized COVID-19 patients with invasive mechanical ventilation, the mean age was 69.4 (SD 1.8) years and 30 (83.3%) were men. 22 (61.1%) patients received CRRT (CRRT group) and 14 cases (38.9%) were managed in conventional strategy (non CRRT group). There was no difference in age, sex, comorbidities, complications, treatments and most of the laboratory findings, except for patients in the CRRT group with higher levels of aspartate aminotransferase and serum creatinine. During the average follow-up period of 10.4 days, 12 of 22 (54.5%) patients in CRRT group and 11 of 14 (78.6%) patients in non CRRT group died. Kaplan Meier analysis demonstrated a prolonged survival in patients in CRRT group than non CRRT group (P=0.032). The association between CRRT treatment and a reduced risk of mortality remained significant after adjusting for confounding factors in seven different models, with an adjusted hazard ratio (aHR) varying between 0.283 and 0.424. Older age, higher levels of IL1b, IL2 receptor, hs cTnI and NT proBNP were independently associated with increased risk of mortality in patients with CRRT treatment. Interpretation: CRRT may be beneficial for the treatment of COVID 19 patients with invasive mechanical ventilation. Further prospective multicenter studies with larger sample sizes are required.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Yi Yang; Jia Shi; Shuwang Ge; Shuiming Guo; Xue Xing; Yanan Wang; Anying Cheng; Qingquan Liu; Junhua Li; Yong Ning; Fan He; Gang Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20036780&#39;},
 {&#39;cord_uid&#39;: &#39;rr8qet8c&#39;,
  &#39;sha&#39;: &#39;2cd8b9a41c271e30c064212f8e113204d3e6e29e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Emergence of evidence during disease outbreaks: lessons learnt from the Zika virus outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20036806&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: Outbreaks of infectious diseases trigger an increase in scientific research and output. Early in outbreaks, evidence is scarce, but it accumulates rapidly. We are continuously facing new disease outbreaks, including the new coronavirus (SARS-nCoV-2) in December 2019.The objective of this study was to describe the accumulation of evidence during the 2013-2016 Zika virus (ZIKV) outbreak in the Pacific and the Americas related to aetiological causal questions about congenital abnormalities and Guillain-Barre syndrome. Methods: We hypothesised that the temporal sequence would follow a pre-specified order, according to study design. We assessed 1) how long it takes before findings from a specific study design appear, 2) how publication of preprints could reduce the time to publication and 3) how time to publication evolves over time. Results: We included 346 publications published between March 6, 2014 and January 1, 2019. In the 2013-2016 ZIKV outbreak, case reports, case series and basic research studies were published first. Case-control and cohort studies appeared between 400-700 days after ZIKV was first detected in the region of the study origin. Delay due to the publication process were lowest at the beginning of the outbreak. Only 4.6% of the publications was available as preprints. Discussion: The accumulation of evidence over time in new causal problems generally followed a hierarchy. Preprints reduced the delay to initial publication. Our methods can be applied to new emerging infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Michel J Counotte; Kaspar W Meili; Nicola Low&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20036806&#39;},
 {&#39;cord_uid&#39;: &#39;i46n70gb&#39;,
  &#39;sha&#39;: &#39;e1200121af466e437f3cf17aa10654b7725edb07&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19: Forecasting short term hospital needs in France&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20036939&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Europe is now considered as the epicenter of the SARS-CoV-2 pandemic, France being among the most impacted country. In France, there is an increasing concern regarding the capacity of the healthcare system to sustain the outbreak, especially regarding intensive care units (ICU). The aim of this study was to estimate the dynamics of the epidemic in France, and to assess its impact on healthcare resources for each French metropolitan Region. We developed a deterministic, age-structured, Susceptible-Exposed-Infectious-Removed (SEIR) model based on catchment areas of each COVID-19 referral hospitals. We performed one month ahead predictions (up to April 14, 2020) for three different scenarios (R0=1.5, R0=2.25, R0=3), where we estimated the daily number of COVID-19 cases, hospitalizations and deaths, the needs in ICU beds per Region and the reaching date of ICU capacity limits. At the national level, the total number of infected cases is expected to range from 22,872 in the best case (R0=1.5) to 161,832 in the worst case (R0=3), while the total number of deaths would vary from 1,021 to 11,032, respectively. At the regional level, all ICU capacities may be overrun in the worst scenario. Only seven Regions may lack ICU beds in the mild scenario (R0=2.25) and only one in the best case. In the three scenarios, Corse may be the first Region to see its ICU capacities overrun. The two other Regions, whose capacity will be overrun shortly after are Grand-Est and Bourgogne-Franche-Comté. Our analysis shows that, even in the best case scenario, the French healthcare system will very soon be overwhelmed. While drastic social distancing measures may temper our results, a massive reorganization leading to an expansion of French ICU capacities seems to be necessary to manage the coming wave of critically affected COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Clément Massonnaud; Jonathan Roux; Pascal Crépey&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20036939&#39;},
 {&#39;cord_uid&#39;: &#39;s4ef9qob&#39;,
  &#39;sha&#39;: &#39;bf3f1752cf136693453f34b968adcbae605fc38b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Transmissibility of 2019 Novel Coronavirus: zoonotic vs. human to human transmission, China, 2019-2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20037036&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Objectives: The novel coronavirus (2019-nCoV) originating from Wuhan has rapidly spread throughout China. While the origin of the outbreak remains uncertain, accumulating evidence links a wet market in Wuhan for the early spread of 2019-nCoV. Similarly, the influence of the marketplace on the early transmission dynamics is yet to be investigated. Methods: Using the daily series of COVID-19 incidenceincluding contact history with the market, we have conducted quantitative modeling analyses to estimate the reproduction numbers (R) for the market-to-human and human-to-human transmission together with the reporting probability and the early effects of public health interventions. Results: Our mean R estimates for China in 2019-2020 are estimated at 0.37 (95%CrI: 0.02-1.78) for market-to-human transmission, and 3.87 (95%CrI: 3.18-4.78) for human-to-human transmission, respectively. Moreover we estimated that the reporting rate cases stemming from market-to-human transmission was 3-31 fold higher than that for cases stemming from human-to-human transmission, suggesting that contact history with the wet market played a key role in identifying COVID-19 cases. Conclusions: Our findings suggest that the proportions of asymptomatic and subclinical patients constitute a substantial component of the epidemic&#39;s magnitude. Findings suggest that the development of rapid diagnostic tests could help bring the epidemic more rapidly under control.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Kenji Mizumoto; Katsushi Kagaya; Gerardo Chowell&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20037036&#39;},
 {&#39;cord_uid&#39;: &#39;tdlcb9bf&#39;,
  &#39;sha&#39;: &#39;50be853b37945ba63f2fa2c4654b8c57ce0c1597&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Hydroxychloroquine and Azithromycin as a treatment of COVID-19: preliminary results of an open-label non-randomized clinical trial&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20037135&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Chloroquine and Hydroxychloroquine have been found to be efficient on COV-19, and reported to be efficient in Chinese patients infected by this virus. We evaluate the role of Hydroxychloroquine on respiratory viral loads. Patients and methods Patients were included in a single arm protocol to receive 600mg of hydroxychloroquine daily and their viral load in nasal swabs was tested daily. Depending on their clinical presentation azithromycin was added to the treatment. Untreated patients from another center and cases refusing the protocol were included as negative control. Presence and absence of virus at Day-6 was considered the end point. Results Twenty cases were treated in this study and showed a significant reduction of the viral carriage at D-6 compared to controls, and much lower than reported average carrying duration of untreated patients in the literature. Azithromycin added to Hydroxychloroquine was significantly more efficient for virus elimination. Conclusion : Hydroxychloroquine is significantly associated with viral load reduction/disappearance in patients with COVID-19 and its effect is reinforced by Azithromycin.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Philippe GAUTRET; Jean Christophe LAGIER; Philippe PAROLA; Van Thuan HOANG; Line MEDDED; Morgan MAILHE; Barbara DOUDIER; Johan COURJON; Valerie GIORDANENGO; Vera ESTEVES VIEIRA; Herve TISSOT DUPONT; Stephane HONORE; Philippe COLSON; Eric CHABRIERE; Bernard LA SCOLA; Jean Marc ROLAIN; Philippe BROUQUI; Didier RAOULT&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20037135&#39;},
 {&#39;cord_uid&#39;: &#39;3svnvozz&#39;,
  &#39;sha&#39;: &#39;e367637e063ec728b13f3685e8aee5f775c553cd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Roles of meteorological conditions in COVID-19 transmission on a worldwide scale&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20037168&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The novel coronavirus (SARS-CoV-2/ 2019-nCoV) identified in Wuhan, China, in December 2019 has caused great damage to public health and economy worldwide with over 140,000 infected cases up to date. Previous research has suggested an involvement of meteorological conditions in the spread of droplet-mediated viral diseases, such as influenza. However, as for the recent novel coronavirus, few studies have discussed systematically about the role of daily weather in the epidemic transmission of the virus. Here, we examine the relationships of meteorological variables with the severity of the outbreak on a worldwide scale. The confirmed case counts, which indicates the severity of COVID-19 spread, and four meteorological variables, i.e., air temperature, relative humidity, wind speed, and visibility, were collected daily between January 20 and March 11 (52 days) for 430 cities and districts all over China, 21 cities/ provinces in Italy, 21 cities/ provinces in Japan, and 51 other countries around the world. Four different time delays of weather (on the day, 3 days ago, 7 days ago, and 14 days ago) as to the epidemic situation were taken for modeling and we finally chose the weather two weeks ago to model against the daily epidemic situation as its correlated with the outbreak best. Taken Chinese cities as a discovery dataset, it was suggested that temperature, wind speed, and relative humidity combined together could best predict the epidemic situation. The meteorological model could well predict the outbreak around the world with a high correlation (r2&gt;0.6) with the real data. Using this model, we further predicted the possible epidemic situation in the future 12 days in several high-latitude cities with potential outbreak. This model could provide more information for government&#39;s future decisions on COVID-19 outbreak control.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Biqing Chen; Hao Liang; Xiaomin Yuan; Yingying Hu; Miao Xu; Yating Zhao; Binfen Zhang; Fang Tian; Xuejun Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20037168&#39;},
 {&#39;cord_uid&#39;: &#39;6fbnyntq&#39;,
  &#39;sha&#39;: &#39;6be177e9b694282436d5d625d2ee3ae86a4db93a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Hundreds of severe pediatric COVID-19 infections in Wuhan prior to the lockdown&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20037176&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Before January 22, 2020, only one pediatric case of COVID-19 was reported in mainland China1,2. However, a retrospective surveillance study3 identified six children who had been hospitalized for COVID-19 in one of three central Wuhan hospitals between January 7th and January 15th. Given that Wuhan has over 395 other hospitals, there may have been far more severe pediatric cases than reported. There were six and 43 children out of 336 who tested positive for COVID-19 and influenza, respectively among all pediatric admissions during the 9-day period3. By using this ratio in a detailed analysis of influenza surveillance data and COVID-19 epidemic dynamics (see Appendix), we estimate that there were 313 [95% CI: 171-520] children hospitalized for COVID-19 in Wuhan during January 7-15, 2020 (Figure). Under an epidemic doubling time of 7.31 days4, we estimate that there were 1105 [95% CI: 592, 1829] cumulative pediatric COVID-19 hospitalizations prior to the January 23rd lockdown, which far surpasses the 425 confirmed cases reported across all age groups, none of which were children under age 151. Children are strikingly absent from COVID-19 reports and limited data suggest that pediatric infections are overwhelmingly mild5. Thus, our estimates for hundreds of severe pediatric cases likely translates to thousands or even tens of thousands of mildly infected children, suggesting that the force of infection from children may be grossly underestimated and the infection fatality rate overestimated from confirmed case counts alone. This highlights the urgent need for more robust surveillance to gauge the true extent and severity of COVID-19 in all ages.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Zhanwei Du; Ciara Nugent; Benjamin J Cowling; Lauren Ancel Meyers&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20037176&#39;},
 {&#39;cord_uid&#39;: &#39;7w1bhaz6&#39;,
  &#39;sha&#39;: &#39;1a9d52bf91d858565ab0ccc8cab03e919ef6e5c3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;High incidence of asymptomatic SARS-CoV-2 infection, Chongqing, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.20037259&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: SARS-CoV-2 has been a global pandemic, but the emergence of asymptomatic patients has caused difficulties in the prevention of the epidemic. Therefore, it is significant to understand the epidemiological characteristics of asymptomatic patients with SARS-CoV-2 infection. Methods: In this single-center, retrospective and observational study, we collected data from 167 patients with SARS-CoV-2 infection treated in Chongqing Public Health Medical Center (Chongqing, China) from January to March 2020. The epidemiological characteristics and variable of these patients were collected and analyzed. Findings: 82.04% of the SARS-CoV-2 infected patients had a travel history in Wuhan or a history of contact with returnees from Wuhan, showing typical characteristics of imported cases, and the proportion of severe Covid-19 patients was 13.2%, of which 59% were imported from Wuhan. For the patients who was returnees from Wuhan, 18.1% was asymptomatic patients. In different infection periods, compared with the proportion after 1/31/2020, the proportion of asymptomatic patient among SARS-CoV-2 infected patient was higher(19% VS 1.5%). In different age groups, the proportion of asymptomatic patient was the highest(28.6%) in children group under 14, next in elder group over 70 (27.3%). Compared with mild and common Covid-19 patients, the mean latency of asymptomatic was longer (11.25 days VS 8.86 days), but the hospital length of stay was shorter (14.3 days VS 16.96 days) . Conclusion: The SARS-CoV-2 prevention needs to focus on the screening of asymptomatic patients in the community with a history of contact with the imported population, especially for children and the elderly population.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Yang Tao; Panke Cheng; Wen Chen; Peng Wan; Yaokai Chen; Guodan Yuan; Junjie Chen; Da Huo; Ge Guan; Dayu Sun; Ju Tan; Guanyuan Yang; Wen Zeng; Chuhong Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.20037259&#39;},
 {&#39;cord_uid&#39;: &#39;edu1zuzt&#39;,
  &#39;sha&#39;: &#39;2757708c4f72260abefb9d74302d868165cbc1a4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.990317&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The newly emerged pandemic coronavirus, SARS-CoV-2, has posed a significant public health threat worldwide. However, the mode of virus transmission and tissue tropism is not well established yet. Recent findings of substantial liver damage in patients and ACE2+ cholangiocytes in healthy liver tissues prompted us to hypothesize that human liver ductal organoids could serve as a model to determine the susceptibility and mechanisms underlining the liver damage upon SARS-CoV-2 infection. By single-cell RNA sequencing, we found that long-term liver ductal organoid culture preserved the human specific ACE2+ population of cholangiocytes. Moreover, human liver ductal organoids were permissive to SARS-CoV-2 infection and support robust replication. Notably, virus infection impaired the barrier and bile acid transporting functions of cholangiocytes through dysregulation of genes involved in tight junction formation and bile acid transportation, which could explain the bile acid accumulation and consequent liver damage in patients. These results indicate that control of liver damage caused directly by viral infection should be valued in treating COVID-19 patients. Our findings also provide an application of human organoids in investigating the tropism and pathogenesis of SARS-CoV-2, which would facilitate novel drug discovery.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Bing Zhao; Chao Ni; Ran Gao; Yuyan Wang; Li Yang; Jinsong Wei; Ting Lv; Jianqing Liang; Qisheng Zhang; Wei Xu; Youhua Xie; Xiaoyue Wang; Zhenghong Yuan; Junbo Liang; Rong Zhang; Xinhua Lin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.990317&#39;},
 {&#39;cord_uid&#39;: &#39;wbyqmvhs&#39;,
  &#39;sha&#39;: &#39;10aa307dc5239e819a19dd696b7f3e5bd34a37b1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.993584&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTCOVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging viruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Minfeng Xiao; Xiaoqing Liu; Jingkai Ji; Min Li; Jiandong Li; Lin Yang; Wanying Sun; Peidi Ren; Guifang Yang; Jincun Zhao; Tianzhu Liang; Huahui Ren; Tian Chen; Huanzi Zhong; Wenchen Song; Yanqun Wang; Ziqing Deng; Yanping Zhao; Zhihua Ou; Daxi Wang; Jielun Cai; Xinyi Cheng; Taiqing Feng; Honglong Wu; Yanping Gong; Huanming Yang; Jian Wang; Xun Xu; Shida Zhu; Fang Chen; Yanyan Zhang; Weijun Chen; Yimin Li; Junhua Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.993584&#39;},
 {&#39;cord_uid&#39;: &#39;fw4pmaoc&#39;,
  &#39;sha&#39;: &#39;c7f56ac99072a397fddf17d3f1d2e34d4146edbe&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evidence of the Recombinant Origin and Ongoing Mutations in Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.993816&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYThe recent global outbreak of viral pneumonia designated as Coronavirus Disease 2019 (COVID-19) by coronavirus (SARS-CoV-2) has threatened global public health and urged to investigate its source. Whole genome analysis of SARS-CoV-2 revealed ~96% genomic similarity with bat CoV (RaTG13) and clustered together in phylogenetic tree. Furthermore, RaTGl3 also showed 97.43% spike protein similarity with SARS-CoV-2 suggesting that RaTGl3 is the closest strain. However, RBD and key amino acid residues supposed to be crucial for human-to-human and cross-species transmission are homologues between SARS-CoV-2 and pangolin CoVs. These results from our analysis suggest that SARS-CoV-2 is a recombinant virus of bat and pangolin CoVs. Moreover, this study also reports mutations in coding regions of 125 SARS-CoV-2 genomes signifying its aptitude for evolution. In short, our findings propose that homologous recombination has been occurred between bat and pangolin CoVs that triggered cross-species transmission and emergence of SARS-CoV-2, and, during the ongoing outbreak, SARS-CoV-2 is still evolving for its adaptability.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Jiao-Mei Huang; Syed Sajid Jan; Xiaobin Wei; Yi Wan; Songying Ouyang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.993816&#39;},
 {&#39;cord_uid&#39;: &#39;i9r77o70&#39;,
  &#39;sha&#39;: &#39;39ab0e9d7a93e6a7483bf4851e69cd66d2653256&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.994152&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins—receptor binding domain (RBD), S1 subunit and S ectodomain—in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Tingting Li; Qingbing Zheng; Hai Yu; Dinghui Wu; Wenhui Xue; Yuyun Zhang; Xiaofen Huang; Lizhi Zhou; Zhigang Zhang; Zhenghui Zha; Tingting Chen; Zhiping Wang; Jie Chen; Hui Sun; Tingting Deng; Yingbin Wang; Yixin Chen; Qinjian Zhao; Jun Zhang; Ying Gu; Shaowei Li; Ningshao Xia&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.994152&#39;},
 {&#39;cord_uid&#39;: &#39;jalijjmg&#39;,
  &#39;sha&#39;: &#39;e05ece9c6fdb3e9942598685bb94748b97d88e16&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.16.994236&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYThe rapid and escalating spread of SARS coronavirus 2 (SARS-CoV-2) poses an immediate public health emergency, and no approved therapeutics or vaccines are currently available. The viral spike protein S binds ACE2 on host cells to initiate molecular events that release the viral genome intracellularly. Soluble ACE2 inhibits entry of both SARS and SARS-2 coronaviruses by acting as a decoy for S binding sites, and is a candidate for therapeutic and prophylactic development. Using deep mutagenesis, variants of ACE2 are identified with increased binding to the receptor binding domain of S at a cell surface. Mutations are found across the interface and also at buried sites where they are predicted to enhance folding and presentation of the interaction epitope. The N90-glycan on ACE2 hinders association. The mutational landscape offers a blueprint for engineering high affinity ACE2 receptors to meet this unprecedented challenge.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Erik Procko&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.16.994236&#39;},
 {&#39;cord_uid&#39;: &#39;lv5xjfk4&#39;,
  &#39;sha&#39;: &#39;e63ae71c6969c43fbe201c471c3f1bc3edfb7948&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Cross-reaction of sera from COVID-19 patients with SARS-CoV assays.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20034454&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The SARS-CoV-2 shares 74.5% genome identity with SARS-CoV, both exhibiting a similar well conserved structure. Therefore, antibodies produced in COVID-19 and SARS patients should not be that dissimilar. We evaluated SARS-CoV test assays to detect for the presence of antibodies to SARS-CoV-2 and tried to determine the timing of appearance of these antibodies by testing serial sera from these patients. Methods: Tests were carried out using ELISA (total antibodies) and indirect immunofluorescence (IIFA) (IgM &amp; IgG) methods on serial sera from patients confirmed with SARS-CoV-2 infection. Results: Cross-reactivity was seen in these two test assays with sera from COVID-19 patients and was detected in 6 out of 7 patients from 7 days after onset of symptoms. Five of the patients had detectable antibodies by the 3rd week into their illness and there was evidence of seroconversion in 4 patients. The IIFA method was marginally more sensitive compared to the ELISA assay, however the IIFA IgM test was not useful in the early phase of the illness with poor sensitivity. Conclusions: Existing diagnostic assays for SARS-CoV can detect antibodies in patients who were diagnosed with COVID-19. These assays maybe be utilized as an interim measure in epidemiological investigations for contact tracing and to determine the extent of community spread of this new emerging virus pending the availability of specific serology tests for SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Wei Yee Wan; Siew Hoon Lim; Eng Hong Seng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20034454&#39;},
 {&#39;cord_uid&#39;: &#39;dptgg05n&#39;,
  &#39;sha&#39;: &#39;17643a57b92abf0d6635d308949e78de2ecc7f66&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Characterization of anti-viral immunity in recovered individuals infected by SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20036640&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The WHO has declared SARS-CoV-2 outbreak a public health emergency of international concern. However, to date, there was hardly any study in characterizing the immune responses, especially adaptive immune responses to SARS-CoV-2 infection. In this study, we collected blood from COVID-19 patients who have recently become virus-free and therefore were discharged, and analyzed their SARS-CoV-2-specific antibody and T cell responses. We observed SARS-CoV-2-specific humoral and cellular immunity in the patients. Both were detected in newly discharged patients, suggesting both participate in immune-mediated protection to viral infection. However, follow-up patients (2 weeks post discharge) exhibited high titers of IgG antibodies, but with low levels of virus-specific T cells, suggesting that they may enter a quiescent state. Our work has thus provided a basis for further analysis of protective immunity to SARS-CoV-2, and understanding the pathogenesis of COVID-19, especially in the severe cases. It has also implications in designing an effective vaccine to protect and treat SARS-CoV-2 infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Chen Dong; Ling Ni; Fang Ye; Meng-Li Chen; Yu Feng; Yong-Qiang Deng; Hui Zhao; Peng Wei; Jiwan Ge; Xiaoli Li; Lin Sun; Pengzhi Wang; Peng Liang; Han Guo; Xinquan Wang; Cheng-Feng Qin; Fang Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20036640&#39;},
 {&#39;cord_uid&#39;: &#39;j5sb3g1u&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Change in outbreak epicenter and its impact on the importation risks of COVID-19 progression: a modelling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20036681&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The outbreak of Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) that originated in the city of Wuhan, China has now spread to every inhabitable continent, but now theattention has shifted from China to other epicenters, especially Italy. This study explored the influence of spatial proximities and travel patterns from Italy on the further spread of SARS-CoV-2 around the globe. We showed that as the epicenter changes, the dynamics of SARS-CoV-2 spread change to reflect spatial proximities.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Oyelola Adegboye; Adeshina Adekunle; Anton Pak; Ezra Gayawan; Denis Leung; Diana Rojas; Faiz Elfaki; Emma McBryde; Damon Eisen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20036681&#39;},
 {&#39;cord_uid&#39;: &#39;7atoko6c&#39;,
  &#39;sha&#39;: &#39;27d2a39228be51b570f2339e44b2cb1a94713683&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Evaluation of recombinant nucleocapsid and spike proteins for serological diagnosis of novel coronavirus disease 2019 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20036954&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The colloidal gold immunochromatography assay (GICA) is a rapid diagnostic tool for novel coronavirus disease 2019 (COVID-19) infections. However, with significant numbers of false negatives, improvements to GICA are needed. Methods: Six recombinant HCoV-19 nucleocapsid and spike proteins were prepared and evaluated. The optimal proteins were employed to develop a sandwich-format GICA strip to detect total antibodies (IgM and IgG) against HCoV-19. GICA performance was assessed with comparison of viral RNA detection. Results: Recombinant HCoV-19 proteins were obtained, including three prokaryotically expressed rN, rN1, rN2 nucleocapsid proteins, and three eukaryotically expressed rS1, rS-RBD, rS-RBD-mFc spike proteins. The recombinant proteins with the highest ELISA titers (rS1 and rS-RBD-mFc) against coronavirus-specific IgM and IgG were chosen for GICA development. The GICA has a sensitivity and specificity of 86.89% (106/122) and 99.39% (656/660), respectively. Furthermore, 65.63% (21/32) of the clinically confirmed but RT-PCR negative samples were GICA positive. Conclusions: The eukaryotically-expressed spike proteins (rS1and rS-RBD-mFc) are more suitable than the prokaryotically expressed nucleocapsid proteins for HCoV-19 serological diagnosis. The GICA sandwich used to detect total antibodies is a powerful complement to the current standard RNA-based tests.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Pingping Zhang; Qi Gao; Tang Wang; Yuehua Ke; Fei Mo; Ruizhong Jia; Wanbing Liu; Lei Liu; Shangen Zheng; Yuzhen Liu; Luping Li; Yao Wang; Lei Xu; Kun Hao; Ruifu Yang; Shiyue Li; Changqing Lin; Yong Zhao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20036954&#39;},
 {&#39;cord_uid&#39;: &#39;vwbpkpxd&#39;,
  &#39;sha&#39;: &#39;b100dcf4e2ec314df8ebad9a20aa245edee6db59&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Machine Learning the Phenomenology of COVID-19 From Early Infection Dynamics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037309&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We present a data-driven machine learning analysis of COVID-19 from its early infection dynamics, with the goal of extracting actionable public health insights. We focus on the transmission dynamics in the USA starting from the first confirmed infection on January 21 2020. We find that COVID-19 has a strong infectious force if left unchecked, with a doubling time of under 3 days. However it is not particularly virulent. Our methods may be of general interest.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Malik Magdon-Ismail&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037309&#39;},
 {&#39;cord_uid&#39;: &#39;s0zdqd6d&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;How to improve adherence with quarantine: Rapid review of the evidence&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037408&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives: The January 2020 outbreak of coronavirus has once again thrown the vexed issue of quarantine into the spotlight, with many countries asking their citizens to self-isolate if they have potentially come into contact with the infection. However, adhering to quarantine is difficult. Decisions on how to apply quarantine should be based on the best available evidence to increase the likelihood of people adhering to protocols. We conducted a rapid review to identify factors associated with adherence to quarantine during infectious disease outbreaks. Study design: Rapid evidence review. Methods: We searched Medline, PsycINFO and Web of Science for published literature on the reasons for and factors associated with adherence to quarantine during an infectious disease outbreak. Results: We found 3163 papers and included 14 in the review. Adherence to quarantine ranged from as little as 0 up to 92.8%. The main factors which influenced or were associated with adherence decisions were the knowledge people had about the disease and quarantine procedure, social norms, perceived benefits of quarantine and perceived risk of the disease, as well as practical issues such as running out of supplies or the financial consequences of being out of work. Conclusions: People vary in their adherence to quarantine during infectious disease outbreaks. To improve this, public health officials should provide a timely, clear rationale for quarantine and information about protocols; emphasise social norms to encourage this altruistic behaviour; increase the perceived benefit that engaging in quarantine will have on public health; and ensure that sufficient supplies of food, medication and other essentials are provided.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Rebecca Webster; Samantha Brooks; Louise Smith; Lisa Woodland; Simon Wessely; James Rubin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037408&#39;},
 {&#39;cord_uid&#39;: &#39;14he8n3u&#39;,
  &#39;sha&#39;: &#39;9731e9ca09f70adcddebc6667f689908bd3b18e0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Favipiravir versus Arbidol for COVID-19: A Randomized Clinical Trial&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037432&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Importance: WHO has made the assessment that coronavirus disease 2019 (COVID-19) can be characterized as a pandemic. But there is no effective antiviral drug for COVID-19 so far. Objective: To compare the efficacy and safety of favipiravir and arbidol to treat COVID-19 patients on 7 day′s clinical recovery rate. Design: Prospective, multicenter, open-label, randomized superiority trial in February, 2020. Setting: Multicenter study. Participants: Patients with confirmed COVID-19 admitted to 3 hospitals from Feb 20, 2020 to Mar 12, 2020. Interventions: Conventional therapy + favipiravir or arbidol. Main Outcomes and Measures: The primary outcome was 7 day′s clinical recovery rate. Duration of fever, cough relief time and auxiliary oxygen therapy or noninvasive mechanical ventilation rate were the secondary outcomes. The patients with chest CT imaging and laboratory-confirmed COVID-19 infection, aged 18 years or older were randomly assigned to receive favipiravir or arbidol. Safety data were collected for a further 1 weeks′ follow-up. Results: 120 patients were assigned to favipiravir group (116 assessed) and 120 to arbidol group (120 assessed). In FAS cohort, for ordinary patients with COVID-19, 7 day′s clinical recovery rate was 55.86% in the arbidol group and 71.43% in the favipiravir group (P = 0.0199). For ordinary COVID-19 patients and COVID-19 patients with hypertension and/or diabetes, the time of fever reduction and cough relief in favipiravir group was significantly shorter than that in arbidol group (both P &lt; 0.001), but there was no statistical difference was observed of auxiliary oxygen therapy or noninvasive mechanical ventilation rate (both P &gt; 0.05). The most possible adverse events were abnormal LFT, psychiatric symptom reactions, digestive tract reactions and raised serum uric acid (3 [2.50 %] in arbidol group vs 16 [13.79%] in favipiravir group, P &lt; 0.0001). Conclusions and Relevance: In ordinary COVID-19 patients untreated with antiviral previously, favipiravir can be considered as a preferred treatment because of its′ higher 7 day′s clinical recovery rate and more effectively reduced incidence of fever, cough except some antiviral-associated adverse effects. Trial Registration: This study is registered with Chictr.org.cn, number ChiCTR200030254.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Chang Chen; Jianying Huang; Zhenshun Cheng; Jianyuan Wu; Song Chen; Yongxi Zhang; Bo Chen; Mengxin Lu; Yongwen Luo; Jingyi Zhang; Ping Yin; Xinghuan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037432&#39;},
 {&#39;cord_uid&#39;: &#39;us7d19p5&#39;,
  &#39;sha&#39;: &#39;9231e9d6dd98dca26e67eaf02eea8c0ad0bcce3b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#34;The impact of unplanned school closure on children&#39;s social contact: Rapid evidence review&#34;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037457&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background Emergency school closures are often used as public health interventions during infectious disease outbreaks in an attempt to minimise the spread of infection. However, if children continue to mix with others outside the home during the closures, these measures are unlikely to be effective. Objectives This review aimed to summarise existing literature on children&#39;s activities and contacts made outside the home during unplanned school closures. Methods We searched four databases from inception to February 2020 for relevant literature. Main findings were extracted. Results 3,343 citations were screened and 19 included in the review. Activities and social contacts appeared to decrease during closures but contact was still common. All studies reported children leaving the house or being looked after by non-household members. There was some evidence that older child age and parental disagreement with closure were predictive of children leaving the house, and mixed evidence regarding the relationship between infection status and leaving the home. Parental agreement with closure was generally high, but some parents disagreed due to perceived low risk of infection and practical issues regarding childcare and financial impact. Conclusions Evidence suggests that many children continue to leave the house and mix with others during school closures despite public health recommendations to avoid social contact. This review of behaviour during unplanned school closures could be used to improve infectious disease modelling.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Samantha Brooks; Louise Smith; Rebecca Webster; Dale Weston; Lisa Woodland; Ian Hall; James Rubin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037457&#39;},
 {&#39;cord_uid&#39;: &#39;sz26gchn&#39;,
  &#39;sha&#39;: &#39;c54a60bad3d8663eaef03c69a465f0cf619429fd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Pandemic dynamics of COVID-19 using epidemic stage, instantaneous reproductive number and pathogen genome identity (GENI) score: modeling molecular epidemiology&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037481&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Global spread of COVID-19 created an unprecedented infectious disease crisis that progressed to a pandemic with &gt;180,000 cases in &gt;100 countries. Reproductive number (R) is an outbreak metric estimating the transmission of a pathogen. Initial R values were published based on the early outbreak in China with limited number of cases with whole genome sequencing. Initial comparisons failed to show a direct relationship viral genomic diversity and epidemic severity was not established for SARS-Cov-2. Methods: Each country&#39;s COVID-19 outbreak status was classified according to epicurve stage (index, takeoff, exponential, decline). Instantaneous R estimates (Wallinga and Teunis method) with a short and standard serial interval examined asymptomatic spread. Whole genome sequences were used to quantify the pathogen genome identity score that were used to estimate transmission time and epicurve stage. Transmission time was estimated based on evolutionary rate of 2 mutations/month. Findings: The country-specific R revealed variable infection dynamics between and within outbreak stages. Outside China, R estimates revealed propagating epidemics poised to move into the takeoff and exponential stages. Population density and local temperatures had variable relationship to the outbreaks. GENI scores differentiated countries in index stage with cryptic transmission. Integration of incidence data with genome variation directly increases in cases with increased genome variation. Interpretation: R was dynamic for each country and during the outbreak stage. Integrating the outbreak dynamic, dynamic R, and genome variation found a direct association between cases and genome variation. Synergistically, GENI provides an evidence-based transmission metric that can be determined by sequencing the virus from each case. We calculated an instantaneous country-specific R at different stages of outbreaks and formulated a novel metric for infection dynamics using viral genome sequences to capture gaps in untraceable transmission. Integrating epidemiology with genome sequencing allows evidence-based dynamic disease outbreak tracking with predictive evidence.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;DJ Darwin R Bandoy; Bart C Weimer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037481&#39;},
 {&#39;cord_uid&#39;: &#39;51b7hss1&#39;,
  &#39;sha&#39;: &#39;ed4a41df2cf4e14a59f8c3899a8b19a2fbf1c291&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Tool to Early Predict Severe 2019-Novel Coronavirus Pneumonia (COVID-19) : A Multicenter Study using the Risk Nomogram in Wuhan and Guangdong, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037515&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Severe cases of coronavirus disease 2019 (COVID-19) rapidly develop acute respiratory distress leading to respiratory failure, with high short-term mortality rates. At present, there is no reliable risk stratification tool for non-severe COVID-19 patients at admission. We aimed to construct an effective model for early identifying cases at high risk of progression to severe COVID-19. Methods SARS-CoV-2 infected patients from one center in Wuhan city and two centers in Guangzhou city, China were included retrospectively. All patients with non-severe COVID-19 during hospitalization were followed for more than 15 days after admission. Patients who deteriorated to severe or critical COVID-19 and patients who kept non-severe state were assigned to the severe and non-severe group, respectively. We compared the demographic, clinical, and laboratory data between severe and non-severe group. Based on baseline data, least absolute shrinkage and selection operator (LASSO) algorithm and logistic regression model were used to construct a nomogram for risk prediction in the train cohort. The predictive accuracy and discriminative ability of nomogram were evaluated by area under the curve (AUC) and calibration curve. Decision curve analysis (DCA) and clinical impact curve analysis (CICA) were conducted to evaluate the clinical applicability of our nomogram. Findings The train cohort consisted of 189 patients, while the two independent validation cohorts consisted of 165 and 18 patients. Among all cases, 72 (19.35%) patients developed severe COVID-19 and 107 (28.76%) patients had one of the following basic disease, including hypertension, diabetes, coronary heart disease, chronic respiratory disease, tuberculosis disease. We found one demographic and six serological indicators (age, serum lactate dehydrogenase, C-reactive protein, the coefficient of variation of red blood cell distribution width (RDW), blood urea nitrogen, albumin, direct bilirubin) are associated with severe COVID-19. Based on these features, we generated the nomogram, which has remarkably high diagnostic accuracy in distinguishing individuals who exacerbated to severe COVID-19 from non-severe COVID-19 (AUC 0.912 [95% CI 0.846-0.978]) in the train cohort with a sensitivity of 85.71 % and specificity of 87.58% ; 0.853 [0.790-0.916] in validation cohort with a sensitivity of 77.5 % and specificity of 78.4%. The calibration curve for probability of severe COVID-19 showed optimal agreement between prediction by nomogram and actual observation. DCA and CICA further indicated that our nomogram conferred significantly high clinical net benefit. Interpretation Our nomogram could help clinicians to early identify patients who will exacerbate to severe COVID-19. And this risk stratification tool will enable better centralized management and early treatment of severe patients, and optimal use of medical resources via patient prioritization and thus significantly reduce mortality rates. The RDW plays an important role in predicting severe COVID-19, implying that the role of RBC in severe disease is underestimated.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Jiao Gong; Jingyi Ou; Xueping Qiu; Yusheng Jie; Yaqiong Chen; Lianxiong Yuan; Jing Cao; Mingkai Tan; Wenxiong Xu; Fang Zheng; Yaling Shi; Bo Hu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037515&#39;},
 {&#39;cord_uid&#39;: &#39;u15wuwkt&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The prevention of nosocomial SARS-CoV2 transmission in endoscopy: a systematic review of recommendations within gastroenterology to identify best practice.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037564&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Endoscopy generates aerosol droplets and fomites, thereby increasing the risk of SARS-CoV2 transmission to healthcare workers and uninfected patients within endoscopy departments. Despite the sharp rise in the incidence of COVID-19, authoritative recommendations to limit the spread of SARS-CoV2 within gastrointestinal endoscopy units are lacking. Therefore, with the primary aim of identifying best practice and scrutinizing its supporting evidence, we conducted a systematic review of literature for articles published between 1 January 2002 and 15 March 2020 in five databases relating to both the current SARS-CoV2 and the previous SARS-CoV outbreaks. Official websites for gastroenterology and endoscopy societies in the 15 most affected countries were also searched. Unfortunately, a paucity of high quality data and heterogeneity of recommendations between countries was observed. Interestingly, not all countries advocated the postponement of non-urgent or elective procedures. Recommendations for patient screening and personal protective equipment were commonly featured in all recommendations but specifics varied. Only 32% (9/28) of all gastroenterology and endoscopy societies issued guidance on endoscopy in the COVID-19 pandemic. In conclusion, stronger evidence to inform current practice and robust guidelines are urgently needed to prevent the transmission of SARS-CoV2 in gastrointestinal endoscopy departments worldwide.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;John Ong; Gail B. Cross; Yock Young Dan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037564&#39;},
 {&#39;cord_uid&#39;: &#39;yx8b2moc&#39;,
  &#39;sha&#39;: &#39;aafa6cdfe96a5cdaf7b7c2f04b11a5dbdd73b2df&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Incidence, clinical characteristics and prognostic factor of patients with COVID-19: a systematic review and meta-analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037572&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Recently, Coronavirus Disease 2019 (COVID-19) outbreak started in Wuhan, China. Although the clinical features of COVID-19 have been reported previously, data regarding the risk factors associated with the clinical outcomes are lacking. Objectives: To summary and analyze the clinical characteristics and identify the predictors of disease severity and mortality. Methods: The PubMed, Web of Science Core Collection, Embase, Cochrane and MedRxiv databases were searched through February 25, 2020. Meta-analysis of Observational Studies in Epidemiology (MOOSE) recommendations were followed. We extracted and pooled data using random-effects meta-analysis to summary the clinical feature of the confirmed COVID-19 patients, and further identify risk factors for disease severity and death. Heterogeneity was evaluated using the I2 method and explained with subgroup analysis and meta-regression. Results: A total of 30 studies including 53000 patients with COVID-19 were included in this study, the mean age was 49.8 years (95% CI, 47.5-52.2 yrs) and 55.5% were male. The pooled incidence of severity and mortality were 20.2% (95% CI, 15.1-25.2%) and 3.1% (95% CI, 1.9-4.2%), respectively. The predictor for disease severity included old age (≥ 50 yrs, odds ratio [OR] = 2.61; 95% CI, 2.29-2.98), male (OR =1.348, 95% CI, 1.195-1.521), smoking (OR =1.734, 95% CI, 1.146-2.626) and any comorbidity (OR = 2.635, 95% CI, 2.098-3.309), especially chronic kidney disease (CKD, OR = 6.017; 95% CI, 2.192-16.514), chronic obstructive pulmonary disease (COPD, OR = 5.323; 95% CI, 2.613-10.847) and cerebrovascular disease (OR = 3.219; 95% CI, 1.486-6.972). In terms of laboratory results, increased lactate dehydrogenase (LDH), C-reactive protein (CRP) and D-dimer and decreased blood platelet and lymphocytes count were highly associated with severe COVID-19 (all for P &lt; 0.001). Meanwhile, old age (≥ 60 yrs, RR = 9.45; 95% CI, 8.09-11.04), followed by cardiovascular disease (RR = 6.75; 95% CI, 5.40-8.43) hypertension (RR = 4.48; 95% CI, 3.69-5.45) and diabetes (RR = 4.43; 95% CI, 3.49-5.61) were found to be independent prognostic factors for the COVID-19 related death. Conclusions: To our knowledge, this is the first evidence-based medicine research to explore the risk factors of prognosis in patients with COVID-19, which is helpful to identify early-stage patients with poor prognosis and adapt effective treatment.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Xianxian Zhao; Bili Zhang; Pan Li; Chaoqun Ma; Jiawei Gu; Pan Hou; Zhifu Guo; Hong Wu; Yuan Bai&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037572&#39;},
 {&#39;cord_uid&#39;: &#39;we62087x&#39;,
  &#39;sha&#39;: &#39;ffbc1762c9e43be58033a7a1f2e21e00d9c7d250&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Sentinel Event Surveillance to Estimate Total SARS-CoV-2 Infections, United States&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037648&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Human infections with a novel coronavirus (SARS-CoV-2) were first identified via syndromic surveillance in December of 2019 in Wuhan China. Since identification, infections (coronavirus disease-2019; COVID-19) caused by this novel pathogen have spread globally, with more than 180,000 confirmed cases as of March 16, 2020. Effective public health interventions, including social distancing, contact tracing, and isolation/quarantine rely on the rapid and accurate identification of confirmed cases. However, testing capacity (having sufficient tests and laboratory throughput) to support these non-pharmaceutical interventions remains a challenge for containment and mitigation of COVID-19 infections. We undertook a sentinel event strategy (where single health events signal emerging trends) to estimate the incidence of COVID-19 in the US. Data from a recent national conference, the Conservative Political Action Conference, (CPAC) near Washington, DC and from the outbreak in Wuhan, China were used to fit a simple exponential growth model to estimate the total number of incident SARS- CoV-2 infections in the United States on March 1, 2020, and to forecast subsequent infections potentially undetected by current testing strategies. Our analysis and forecasting estimates a total of 54,100 SARS-CoV-2 infections (80 % CI 5,600 to 125,300) have occurred in the United States to March 12, 2020. Our forecast predicts that a very substantial number of infections are undetected, and without extensive and far-reaching non-pharmaceutical interventions, the number of infections should be expected to grow at an exponential rate.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Andrew A. Lover; Thomas McAndrew&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037648&#39;},
 {&#39;cord_uid&#39;: &#39;p4cb7xyu&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Simple Mathematics on Covid-19 Expansion&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037663&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We review the Kermack and McKendrick model of epidemics and apply it to Covid-19. Despite the simplicity of this model, solid conclusions are extracted that can assist potential decisions on the strategy to combat the outbreak, essentially configuring a scenario ranging from short-term suppression to long-term mitigation depending on the achieved reduction in the contact number.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Javier Garcia de Abajo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037663&#39;},
 {&#39;cord_uid&#39;: &#39;95ka0p8n&#39;,
  &#39;sha&#39;: &#39;81231d91aecc8f4dd09c6fa8a53e48af7e0c4ba0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modeling and Forecasting Trend of COVID-19 Epidemic in Iran&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037671&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: COVID-19 is an emerging disease and precise data on its epidemiological profile are not available in the world and Iran. In Iran, until March 17, 2020, approximately 14,991 COVID-19 patients were reported. Although modeling and predictions will be typically associated with random error, this study aimed to model and determine the epidemic trend and prediction of COVID-19 in Iran. Methods: This study is a secondary data analysis and mathematical modeling. We used the daily reports of definitive COVID-19 patients released by Iran Ministry of Health and Medical Education. At the beginning of the outbreak, a delayed (stochastic) dynamic system model was used to calculate basic reproduction number (R0). Epidemic Curve along with patient population growth, recovered and deceased individuals was used to predict the epidemic trend. Epidemic projection models of logistic growth differential equations, Gompertz growth, von Bertalanffy and Cubic Polynomial east squared error (LSE) method were used. Matlab software was used to predict the number of cases until April 3, 2020. Results: In the first week of the epidemic, R0 (the mean number of individuals directly infected by an infectious COVID-19 case through the total infectious period, when introduced to a susceptible population) in Iran was estimated to be 4.7 that has now fallen to below 2. The discovery process of COVID-19 cases in Iran is increasing and is based on public behavior and government interventions. Given the assumptions in Models, and three different scenarios, the prediction of the patients on April 3, 2020 using three growth models of Gompertz and von Bertalanffy and least squared error was estimated at 19,500, 27,000, and 48,830, respectively. The number of recovered individuals was estimated 6540 according to Von model, 11800 according to the Gompertz model, and 21000 according to the curve fitting model with least squared error. The number of deceased COVID-19 patients was also estimated to be 1707 individuals using the logistic growth model, 3165 ones by Von model and 6300 ones according to the LSE method. Conclusion: The process of controlling the epidemic is tangible. The most ideal scenario is the Gompertz model, but it is hard to fulfill. If enforcement and public behavior interventions continue with current trends, the control and reduction of the COVID-19 epidemic in Iran will be flat from April 3, 2020 by Von model and new cases are expected to decline from the following Iranian new year. This study suggests that government interventions and people&#39;s behaviors determine the persistence of the epidemic and should be addressed with greater responsibility, accountability, rigor, and quality.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Ali Ahmadi; Majid Shirani; Fereydoon Rahmani&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037671&#39;},
 {&#39;cord_uid&#39;: &#39;fuqtwn5a&#39;,
  &#39;sha&#39;: &#39;28d5f69c84981ebeb76cf7c07b554d06bf85a770&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Tracing DAY-ZERO and Forecasting the Fade out of the COVID-19 Outbreak in Lombardy, Italy: A Compartmental Modelling and Numerical Optimization Approach.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037689&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since the first suspected cluster of cases of coronavirus disease-2019 (COVID-19) on December 1st, 2019, in Wuhan, Hubei Province, China, a total of 195,892 confirmed infected cases, 80,840 recovered and 7,865 deaths have been reported worldwide up to March 16, 2020. After China, Italy is currently at the forefront of the combat against the epidemic that has now spread to all 22 Italian regions. The disease is sweeping through Lombardy, which remains in lockdown since the 8th of March. As of the same day, the isolation measures taken in Lombardy have been extended to the entire country. On March 11, the WHO declared COVID-19 pandemic. Here, we provide estimates for: (a) the DAY-ZERO of the outbreak in Lombardy, Italy; (b) the actual number of exposed/infected cases in the total population; (c) the basic reproduction number (R0); (d) the effective per-day disease transmission and mortality rates; and, importantly, (e) a forecast for the fade out of the outbreak, on the basis of the released data of confirmed cases for Lombardy from February 21 to March 8, the day of lockdown. Methods To deal with the uncertainty in the number of actual exposed/ infected cases in the total population, we address a new compartmental Susceptible/ Exposed/ Infectious/ Recovered/ Dead (SEIRD) model with two compartments of infectious persons: one modelling the total cases in the population and another modelling the confirmed cases. The parameters of the model corresponding to the recovery period, and the time from exposure to the time that an individual starts to be infectious, have been set as reported from clinical studies on COVID-19. For the estimation of the DAY-ZERO of the outbreak in Lombardy, as well as of the effective per-day transmission and mortality rates for which no clinical data are available, we have used the SEIRD simulator to fit the data from February 21 to the 8th of March, the lockdown day of Lombardy (and of all Italy). This was accomplished by solving a mixed-integer optimization problem with the aid of genetic algorithms. Based on the computed values, we also provide an estimation of the basic reproduction number R0. Furthermore, by reducing the estimated transmission rate by 90% on March 8 (to reflect the lockdown of almost all activities), we run the simulator from March 8 to forecast the fade out of the outbreak. Findings Based on the proposed methodological procedure, we estimated that the actual cumulative number of exposed cases in the total population in Lombardy on March 8 was of the order of 15 times the confirmed cumulative number of infected cases. According to this scenario, the DAY-ZERO for the outbreak in Lombardy was the 21st of January 2020. The effective per-day disease transmission rate for the period until March 8 was found to be 0.779 (90% CI: 0.777-0.781), while the &#34;effective&#34; per-day mortality rate was found to be 0.0173 (90% CI: 0.0154-0.0192). Based on these values, the basic reproduction rate R0 was found to be 4.04 (90% CI: 4.03-4.05). Importantly, by reducing the transmission rate by 90% on March 8 to reflect the suspension of almost all activities in Italy, we run the simulator to forecast the fade out of the epidemic. Simulations show that if the measures continue, the complete fade out of the outbreak in Lombardy is expected to occur by the end of May 2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Lucia Russo; Cleo Anastassopoulou; Athanassios Tsakris; Gennaro Nicola Bifulco; Emilio Fortunato Campana; Gerardo Toraldo; Constantinos Siettos&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037689&#39;},
 {&#39;cord_uid&#39;: &#39;ao93c31w&#39;,
  &#39;sha&#39;: &#39;2b34a2b59b93b191b3566d1952d4b5f794ba5e3e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating Spot Prevalence of COVID-19 from Daily Death Data in Italy&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037697&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 like many infectious diseases has a variable severity. Due to limited testing capacity for new virus, the majority of infected showing mild or no symptoms it is likely current prevalence data is an underestimate. We present an estimate of the point prevalence of COVID-19 compared to the number of confirmed case in Italy based on the daily death rate and information about the incubation period, time from symptom onset to death and reported case fatality rate. Our model predicts that when the first 3 infected cases had been identified by Italian authorities there were already nearly 30 cases in Italy, and by the 24th of February 2020 only 0.5% cases had been detected and confirmed by Italian authorities. While official statistics had 132 confirmed case we believe a more accurate estimate would be closer to 26000. With a case-doubling period of about 2.5 days.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Ali Raheem&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037697&#39;},
 {&#39;cord_uid&#39;: &#39;ntra3jhs&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A serological assay to detect SARS-CoV-2 seroconversion in humans&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037713&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: SARS-Cov-2 (severe acute respiratory disease coronavirus 2), which causes Coronavirus Disease 2019 (COVID19) was first detected in China in late 2019 and has since then caused a global pandemic. While molecular assays to directly detect the viral genetic material are available for the diagnosis of acute infection, we currently lack serological assays suitable to specifically detect SARS-CoV-2 antibodies. Methods: Here we describe serological enzyme-linked immunosorbent assays (ELISA) that we developed using recombinant antigens derived from the spike protein of SARS-CoV-2. These assays were developed with negative control samples representing pre-COVID 19 background immunity in the general population and samples from COVID19 patients. Results: The assays are sensitive and specific, allowing for screening and identification of COVID19 seroconverters using human plasma/serum as early as 3 days post symptom onset. Importantly, these assays do not require handling of infectious virus, can be adjusted to detect different antibody types and are amendable to scaling. Conclusion: Serological assays are of critical importance to determine seroprevalence in a given population, define previous exposure and identify highly reactive human donors for the generation of convalescent serum as therapeutic. Sensitive and specific identification of coronavirus SARS-Cov-2 antibody titers will also support screening of health care workers to identify those who are already immune and can be deployed to care for infected patients minimizing the risk of viral spread to colleagues and other patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-18&#39;,
  &#39;authors&#39;: &#39;Fatima Amanat; Thi Nguyen; Veronika Chromikova; Shirin Strohmeier; Daniel Stadlbauer; Andres Javier; Kaijun Jiang; Guha Asthagiri-Arunkumar; Jose Polanco; Maria Bermudez-Gonzalez; Daniel Caplivski; Allen Cheng; Katherine Kedzierska; Olli Vapalahti; Jussi Hepojoki; Viviana Simon; Florian Krammer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037713&#39;},
 {&#39;cord_uid&#39;: &#39;6ymuovl2&#39;,
  &#39;sha&#39;: &#39;493aa8aa2e9122620b1dcaf102ab62e643ad5632&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 Progression Timeline and Effectiveness of Response-to-Spread Interventions across the United States&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037770&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Motivated by the rapid upsurge of COVID-19 cases in the United States beginning March 2020, we forecast the disease spread and assess the effectiveness of containment strategies by using an estalished network-driven epidemic dynamic model. Our model is initialized using the daily counts of active and confirmed COVID-19 cases across the US. Based on our model predictions for the March 14-16 timeframe, the national epidemic peak could be expected to arrive by early June, corresponding to a daily active count of ≈ 7% of the US population, if no containment plans are implemented. Epidemic peaks are expected to arrive in the states of Washington and New York by May 21 and 25, respectively. With a modest 25% reduction in COVID-19 transmissibility via community-level interventions, the epidemic progression could be delayed by up to 34 days. Wholesale interstate traffic restriction is ineffective in delaying the epidemic outbreak, but it does desynchronize the arrival of state-wise epidemic peaks, which could potentially alleviate the burden on limited available medical resources. In addition to forecasting the arrival timeline of the state-wise epidemic peaks, we attempt at informing the optimal timing necessary to enforce community-level interventions. Our findings underscore the pressing need for preparedness and timely interventions in states with a large fraction of the vulnerable uninsured and liquid-asset-poverty populations.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Pai Liu; Payton Beeler; Rajan K Chakrabarty&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037770&#39;},
 {&#39;cord_uid&#39;: &#39;c94vw0vo&#39;,
  &#39;sha&#39;: &#39;798a9a453732be004bdbbcd01e872abe0e3af90f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Intensive Care Unit Resource Planning During COVID-19 Emergency at the Regional Level: the Italian case.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037788&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Severe acute respiratory syndrome Covid-19 (SARS-CoV-2) has been declared a worldwide emergency and a pandemic disease by the World Health Organisation (WHO). It started in China in December 2019, and it is currently rapidly spreading throughout Italy, which is now the most affected country after China. There is great attention for the diffusion and evolution of the CoVid-19 infection which started from the north (particularly in the Lombardia region) and it is now rapidly affecting other Italian regions. We investigate on the impact of patients hospitalisation in Intensive Care Units (ICUs) at a regional and subregional granularity. We propose a model derived from well-known models in epidemic to estimate the needed number of places in intensive care units. The model will help decision-makers to plan resources in the short and medium-term in order to guarantee appropriate treatments to all patients needing it. We analyse Italian data at regional level up to March 15th aiming to: (i) support health and government decision-makers in gathering rapid and efficient decisions on increasing health structures capacities (in terms of ICU slots) and (ii) define a scalable geographic model to plan emergency and future CoVid-19 patients management using reallocating them among health structures. Finally, the here proposed model can be useful in countries where CoVid-19 is not yet strongly diffused.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Pietro Hiram Guzzi; Giuseppe Tradigo; Pierangelo Veltri&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037788&#39;},
 {&#39;cord_uid&#39;: &#39;fq11rhab&#39;,
  &#39;sha&#39;: &#39;5f2e586d719f24d65278de208759dab8213327f3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Reverse transcription loop-mediated isothermal amplification combined with nanoparticles-based biosensor for diagnosis of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037796&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Given the scale and rapid spread of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, known as 2019-nCov) infection (COVID-19), the ongoing global SARS-CoV-2 outbreak has become a huge public health issue. Rapid and precise diagnostic methods are thus immediately needed for diagnosing COVID-19, providing timely treatment and facilitating infection control. A one-step reverse transcription loop-mediated isothermal amplification (RT-LAMP) coupled with nanoparticles-based biosensor (NBS) assay (RT-LAMP-NBS) was successfully established for rapidly and accurately diagnosing COVID-19. A simple equipment (such as heating block) was required for maintaining a constant temperature (63 C) for only 40 min. Using two designed LAMP primer sets, F1ab (opening reading frame 1a/b) and np (nucleoprotein) genes of SARS-CoV-2 were simultaneously amplified and detected in a one-step and single-tube reaction, and the detection results were easily interpreted by NBS. The sensitivity of SARS-CoV-2 RT-LAMP-NBS was 12 copies (each of detection target) per reaction, and no cross-reactivity was generated from non-SARS-CoV-2 templates. Among clinically diagnosed COVID-19 patients, the analytical sensitivity of SARS-CoV-2 was 100% (33/33) in the oropharynx swab samples, and the assay&#39;s specificity was also 100% (96/96) when analyzed the clinical samples collected from non-COVID-19 patients. The total diagnosis test from sample collection to result interpretation only takes approximately 1 h. In sum, the RT-LAMP-NBS is a promising tool for diagnosing the current SARS-CoV-2 infection in first line field, public health and clinical laboratories, especially for resource-challenged regions.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Xiong Zhu; Xiaoxia Wang; Limei Han; Ting Chen; Licheng Wang; Huan Li; Sha Li; Lvfen He; Xiaoying Fu; Shaojin Chen; Xing Mei; Hai Chen; Yi Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037796&#39;},
 {&#39;cord_uid&#39;: &#39;jf36as70&#39;,
  &#39;sha&#39;: &#39;facbc20132a3de7d2038920b62ee0a3a3cb47035&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;AAEDM: Theoretical Dynamic Epidemic Diffusion Model and Covid-19 Korea Pandemic Cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037838&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This paper deals with an advanced analytical epidemic diffusion model which is capable to predict the status of epidemic impacts. This newly propose model well describes an epidemic growth and it could be widely applied into various topics including pathology, epidemiology, business and data sciences. The Advanced Analytical Epidemic Diffusion Model (AAEDM) is a dynamic diffusion prediction model which is theoretically intuitive and its tractable closed formula could be easily adapted into versatile Bigdata driven analytics including the machine learning system. This dynamic model is still an analytical model but the periods of prediction are segmented for adapting the values from the dataset when the data is available. The epidemiologically vital parameters which effect on the AAEDM are also introduced in this paper. The evaluation of this theoretical model based on the Covid-19 data in Korea has been accomplished with relative fair future prediction accuracies. Although this analytical model has been designed from a basic exponential growth model, the performance of the AAEDM is competitive with other Bigdata based simulation models. Since the AAEDM is relatively simple and handy, anyone can use this model into analyzing outbreak situations in his daily life.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Song-Kyoo Kim&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037838&#39;},
 {&#39;cord_uid&#39;: &#39;2gokv7id&#39;,
  &#39;sha&#39;: &#39;7dd547f2db271b15f7ba1ea69583ec5d2b38ef27&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Double-Quencher Probes Improved the Detection Sensitivity of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) by One-Step RT-PCR&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.20037903&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerges in Wuhan City, Hubei Province, spreads worldwide, and threats the human life. The detection of SARS-CoV-2 is important for the prevention of the outbreak and management of patients. Real-time reverse-transcription polymerase chain reaction (RT-PCR) assay detected the virus in clinical laboratory. Methods: This study utilized primers and single-quencher probes in accordance with the Centers for Disease Control and Prevention (CDC) in the USA and the National Institute of Infectious Diseases (NIID) in Japan. Moreover, we designed the double-quencher probes (YCH assay) according to the oligonucleotide sequence established by NIID. Using these assays, we conducted a one-step real-time RT-PCR with serial DNA positive control to assess the detection sensitivity. Results: The threshold cycle (Ct) value of RT-PCR was relatively low in CDC and YCH assays compared to NIID assay. Serial dilution assay showed that both CDC and YCH assays could detect a low-copy number of DNA positive control. The background fluorescent signal at the baseline was lower in YCH than that of NIID. Conclusion: Double-quencher probes decreased background fluorescent signal and improved detection sensitivity of SARS-CoV-2.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Yosuke Hirotsu; Hitoshi Mochizuki; Masao Omata&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.20037903&#39;},
 {&#39;cord_uid&#39;: &#39;d840uu3e&#39;,
  &#39;sha&#39;: &#39;956d0469d076bfebc265cf58743281c109706391&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) System for Rapid Viral DNA Sensing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.994137&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTA fully Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) system is developed for viral DNA detection. This powerful system is patterned with high-aspect ratio micropillars to enhance reporter probe binding. After surface modification and probe immobilization, CRISPR Cas12a/crRNA complex is injected into the fully enclosed system. With the presence of double-stranded DNA target, the CRISPR enzyme is activated and non-specifically cleaves the ssDNA reporters initially immobilized on the micropillars. This collateral cleavage releases fluorescence dyes into the assay, and the intensity is linearly proportional to the target DNA concentration ranging from 0.1 to 10 nM. Importantly, this system does not rely on traditional dye-quencher labeled probe thus eliminating the fluorescence background presented in the assay. Furthermore, our one-step detection protocol is performed at isothermal conditions (37°C) without using complicated and time-consuming off-chip probe hybridization and denaturation. This miniaturized and fully packed IMPACT chip demonstrates rapid, sensitive, and simple nucleic acid detection and is an ideal candidate for the next generation molecular diagnostic platform for point-of-care (POC) applications, responding to emerging and deadly pathogen outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Kenneth N. Hass; Mengdi Bao; Qian He; Myeongkee Park; Peiwu Qin; Ke Du&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.994137&#39;},
 {&#39;cord_uid&#39;: &#39;nwrv0152&#39;,
  &#39;sha&#39;: &#39;7af6a15c35df8fed989119cc36b6679175d2a768&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Effects of a blend of essential oils in milk replacer on performance, rumen fermentation, blood parameters and health scores of dairy heifers&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.995209&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe objective of this study was to evaluate how the inclusion of a blend of essential oils in milk replacer (MR) affects different outcomes of dairy heifers. The outcomes evaluated: feed intake, performance, body development, blood cells and metabolites, insulin-like growth factor-1 (IGF-1), rumen fermentation, fecal scores and respiratory scores. All outcomes were evaluated during pre-weaning (4 – 60 d of age), and carryover effects on post-weaning (61 – 90 d of age) periods. The experimental units utilized were 29 newborn Holstein × Gyr crossbred dairy heifers, with genetic composition of 5/8 or more Holstein and 3/8 or less Gyr and body weight (BW) at birth of 32.2 ± 5.2 kg. Experimental units were randomly assigned to either a control group (CON, n = 15) or a treatment group, consisting of supplementation of a blend of essential oils (BEO, n = 14) with1 g/d/calf (Apex Calf, Adisseo, China). During the pre-weaning phase, all heifers were fed 5 L of MR/d reconstituted at 15% (dry matter basis), divided into two equal meals. Water and starter were provided ad libitum. During the post-weaning, animals received a maximum 3 kg of starter/d, and ad libitum corn silage, divided into two meals. The outcomes feed intake, fecal and respiratory scores were evaluated daily. BW was measured every three days, while body development was recorded weekly. Blood samples were collected on 0, 30 and 60 d of age for total blood cell count, weekly to determinate ß-hydroxybutyrate, urea and glucose, and biweekly for IGF-1. Ruminal parameters (pH, volatile fatty acids, ammonia-N and acetate:proprionate proportion - C2:C3) were measured each 14 days. A randomized complete block design with an interaction between treatment and week was the experimental method of choice to test the hypothesis of effect of BEO on all outcomes. An ANOVA procedure was used for continuous outcomes and a non-parametric test was used for the ordered categorical outcomes both adopting a C.I. = 95%. Results indicated that there was not enough evidence to accept the alternative hypothesis of effect of BEO in MR on: feed intake, performance, body development and blood metabolites during both pre-weaning and post-weaning periods. However, results indicated that the inclusion of BEO in MR significantly affects the proportion of C2:C3 during pre and post-weaning (P ≤ 0.05). Similarly, the effect is significant for basophil (P ≤ 0.001) and platelet (P ≤ 0.05) counts in pre-weaning. The interaction between week and treatment was also significant for lymphocytes (P ≤ 0.001), revealing a long-term treatment immunological effect. Lastly, the effect on fecal scores was also significant (P ≤ 0.05) during pre-weaning, with lower values for BEO. BEO contributed for ruminal manipulation on pre-weaning and carry over effect on post-weaning; immunity improvement and a decrease morbidity of neonatal diarrhea in pre-weaning phase.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Joana P. Campolina; Sandra G. Coelho; Anna Luiza Belli; Fernanda S. Machado; Luiz G. R. Pereira; Thierry R. Tomich; Wanessa A. Carvalho; Rodrigo O. S. Silva; Alessandra L. Voorsluys; David V. Jacob; Mariana M. Campos&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.995209&#39;},
 {&#39;cord_uid&#39;: &#39;zrbesm5b&#39;,
  &#39;sha&#39;: &#39;50be51724c114a2dcfbc9618d94a0166822003de&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Comparative Pathogenesis Of COVID-19, MERS And SARS In A Non-Human Primate Model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.995639&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractA novel coronavirus, SARS-CoV-2, was recently identified in patients with an acute respiratory syndrome, COVID-19. To compare its pathogenesis with that of previously emerging coronaviruses, we inoculated cynomolgus macaques with SARS-CoV-2 or MERS-CoV and compared with historical SARS-CoV infections. In SARS-CoV-2-infected macaques, virus was excreted from nose and throat in absence of clinical signs, and detected in type I and II pneumocytes in foci of diffuse alveolar damage and mucous glands of the nasal cavity. In SARS-CoV-infection, lung lesions were typically more severe, while they were milder in MERS-CoV infection, where virus was detected mainly in type II pneumocytes. These data show that SARS-CoV-2 can cause a COVID-19-like disease, and suggest that the severity of SARS-CoV-2 infection is intermediate between that of SARS-CoV and MERS-CoV.One Sentence SummarySARS-CoV-2 infection in macaques results in COVID-19-like disease with prolonged virus excretion from nose and throat in absence of clinical signs.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-17&#39;,
  &#39;authors&#39;: &#39;Barry Rockx; Thijs Kuiken; Sander Herfst; Theo Bestebroer; Mart M. Lamers; Dennis de Meulder; Geert van Amerongen; Judith van den Brand; Nisreen M.A. Okba; Debby Schipper; Peter van Run; Lonneke Leijten; Ernst Verschoor; Babs Verstrepen; Jan Langermans; Christian Drosten; Martje Fentener van Vlissingen; Ron Fouchier; Rik de Swart; Marion Koopmans; Bart L. Haagmans&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.995639&#39;},
 {&#39;cord_uid&#39;: &#39;kh9whqzd&#39;,
  &#39;sha&#39;: &#39;96250a5e9ece21b2415b83d6756a3fcd07123ee7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Scientometric Trends for Coronaviruses and Other Emerging Viral Infections&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.17.995795&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;COVID-19 is the most rapidly expanding coronavirus outbreak in the past two decades. To provide a swift response to a novel outbreak, prior knowledge from similar outbreaks is essential. Here, we study the volume of research conducted on previous coronavirus outbreaks, specifically SARS and MERS, relative to other infectious diseases by analyzing over 45 million papers from the last 20 years. Our results demonstrate that previous coronavirus outbreaks have been understudied compared to other viruses. We also show that the research volume of emerging infectious diseases is very high after an outbreak and drops drastically upon the containment of the disease. This can yield inadequate research and limited investment in gaining a full understanding of novel coronavirus management and prevention. Independent of the outcome of the current COVID-19 outbreak, we believe that measures should be taken to encourage sustained research in the field.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Dima Kagan; Jacob Moran-Gilad; Michael Fire&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.17.995795&#39;},
 {&#39;cord_uid&#39;: &#39;xsqgrd5l&#39;,
  &#39;sha&#39;: &#39;112e05c31018c816d0bd9ee64336eba79ee786ac&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;High transmissibility of COVID-19 near symptom onset&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20034561&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The dynamics of coronavirus disease 2019 (COVID-19) transmissibility after symptom onset remains unknown. Methods We conducted a prospective case-ascertained study on laboratory-confirmed COVID-19 cases and their contacts. Secondary clinical attack rate (considering symptomatic cases only) was analyzed for different exposure windows after symptom onset of index cases and for different exposure settings. Results Thirty-two confirmed patients were enrolled and 12 paired data (index-secondary cases) were identified among the 1,043 contacts. The secondary clinical attack rate was 0.9% (95% CI 0.5-1.7%). The attack rate was higher among those whose exposure to index cases started within five days of symptom onset (2.4%, 95% CI 1.1-4.5%) than those who were exposed later (zero case from 605 close contacts, 95% CI 0-0.61%). The attack rate was also higher among household contacts (13.6%, 95% CI 4.7-29.5%) and non-household family contacts (8.5%, 95% CI 2.4-20.3%) than that in healthcare or other settings. The higher secondary clinical attack rate for contacts near symptom onset remained when the analysis was restricted to household and family contacts. There was a trend of increasing attack rate with the age of contacts (p for trend &lt; 0.001). Conclusions High transmissibility of COVID-19 near symptom onset suggests that finding and isolating symptomatic patients alone may not suffice to contain the epidemic, and more generalized social distancing measures are required. Rapid reduction of transmissibility over time implies that prolonged hospitalization of mild cases might not be necessary in large epidemics.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-19&#39;,
  &#39;authors&#39;: &#39;Hao-Yuan Cheng; Shu-Wan Jian; Ding-Ping Liu; Ta-Chou Ng; Wan-Ting Huang; Hsien-Ho Lin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20034561&#39;},
 {&#39;cord_uid&#39;: &#39;x2z81cqm&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Development and utilization of an intelligent application for aiding COVID-19 diagnosis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20035816&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;On basis of laboratory examinations indicators from a total of 620 samples, a machine learning-based diagnosis model was developed, and subsequently implemented as a COVID-19 diagnosis aid APP. External validation showed satisfiable model prediction performance (i.e., AUC= 0.872) which guarantees the promising use of this tool for extensive screening.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Zirui Meng; Minjin Wang; Huan Song; Shuo Guo; Yanbing Zhou; Weimin Li; Yongzhao Zhou; Mengjiao Li; Xingbo Song; Yi Zhou; Qingfeng Li; Xiaojun Lu; Binwu Ying&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20035816&#39;},
 {&#39;cord_uid&#39;: &#39;drzphrqj&#39;,
  &#39;sha&#39;: &#39;88bdd99c41f943dd769a81f74a53434f3eb4f371&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Preliminary evidence that higher temperatures are associated with lower incidence of COVID-19, for cases reported globally up to 29th February 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20036731&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Seasonal variation in COVID-19 incidence could impact the trajectory of the pandemic. Using global line-list data on COVID-19 cases reported until 29th February 2020 and global gridded temperature data, and after adjusting for surveillance capacity and time since first imported case, higher average temperature was strongly associated with lower COVID-19 incidence for temperatures of 1°C and higher. However, temperature explained a relatively modest amount of the total variation in COVID-19 incidence. These preliminary findings support stringent containment efforts in Europe and elsewhere.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Melanie Bannister-Tyrrell; Anne Meyer; Celine Faverjon; Angus Cameron&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20036731&#39;},
 {&#39;cord_uid&#39;: &#39;snp8o9fw&#39;,
  &#39;sha&#39;: &#39;1d2e66528cfb3f754157404a225d551f6cdab1ae&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiology of seasonal coronaviruses: Establishing the context for COVID-19 emergence&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20037101&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Public health preparedness for coronavirus disease 2019 (COVID-19) is challenging in the absence of setting-specific epidemiological data. Here we describe the epidemiology of seasonal human coronaviruses (sCoVs) and other cocirculating viruses in the West of Scotland, UK. We analyzed routine diagnostic data for &gt;70,000 episodes of respiratory illness tested molecularly for multiple respiratory viruses between 2005 and 2017. Statistical associations with patient age and sex differed between CoV-229E, CoV-OC43 and CoV-NL63. Furthermore, the timing and magnitude of sCoV outbreaks did not occur concurrently and coinfections were not reported. With respect to other cocirculating respiratory viruses, we found evidence of positive, rather than negative, interactions with sCoVs. These findings highlight the importance of considering cocirculating viruses in the differential diagnosis of COVID-19. Further work is needed to establish the occurrence/degree of cross-protective immunity conferred across sCoVs and with COVID-19, as well as the role of viral coinfection in COVID-19 disease severity.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Sema Nickbakhsh; Antonia Ho; Diogo F.P. Marques; Jim McMenamin; Rory R. Gunson; Pablo Murcia&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20037101&#39;},
 {&#39;cord_uid&#39;: &#39;4fyihdkv&#39;,
  &#39;sha&#39;: &#39;28750ae423bceec7befb82f2b857f9d2ce989e90&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Now-casting the COVID-19 epidemic: The use case of Japan, March 2020&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20037473&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Reporting delays in disease surveillance impair the ability to assess the current dynamic of an epidemic. In continuously updated epidemic curves, case numbers for the most recent epidemic week or day usually appear to be lower than the previous, suggesting a decline of the epidemic. In reality, the epidemic curve may still be on the rise, because reporting delay prevents the most recent cases to appear in the case count. In context of the COVID-19 epidemic and for countries planning large international gatherings, such as the Summer Olympic Games in Japan 2020, the ability to assess the actual stage of an epidemic is of outmost importance. Methods We applied now-casting onto COVID-19 data provided by the nCoV-2019 Data Working Group to evaluate the .true count of cases, by taking into account reporting delays occurring between date of symptom onset and date of confirmation. Findings We calculated a decrease of reporting delay, from a median delay of ten days in calendar week four 2020 to six days in calendar week eight, resulting in an overall mean of 4.3 days. The confidence intervals of the now-casting indicated an increase of cases in the last reporting days, while case country in that same time period suggested a decline. Interpretation As a specific use case this tool may be of particular value for the challenging risk assessment and risk communication in the context of the Summer Olympic Games in Japan 2020 and similar situations elsewhere.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Stephan Gloeckner; Gerard Krause; Michael Hoehle&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20037473&#39;},
 {&#39;cord_uid&#39;: &#39;mrsjhjh4&#39;,
  &#39;sha&#39;: &#39;03482e6a876292352f7aaf1cceeb848e28fb973e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating Preventable COVID19 Infections Related to Elective Outpatient Surgery in Washington State: A Quantitative Model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20037952&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: As the number of suspected and confirmed COVID19 cases in the US continues to rise, the US surgeon general, Centers for Disease Control and Prevention, and several specialty societies have issued recommendations to consider canceling elective surgeries. However, these recommendations have also faced controversy and opposition. Objective: The goal of this study is to provide a quantitative analysis and model for preventable COVID19 infections from elective outpatient or ambulatory surgery cases, which can also be adapted to analyze COVID19 transmission in other healthcare settings. Furthermore, given the controversy over the appropriate handling of elective surgical cases during this pandemic, we hope that our results may have a positive impact on health policy and public health. Methods: Using previously published information on elective ambulatory or outpatient surgical procedures and publicly available data on COVID19 infections in the US and on the Diamond Princess cruise ship, we calculated a transmission rate and generated a mathematical model to predict a lower bound for the number of healthcare-acquired COVID19 infections that could be prevented by canceling or postponing elective outpatient surgeries in Washington state. Results: Our model predicts that over the course of 30 days, at least 2445 preventable patient infections and at least 1557 preventable healthcare worker (HCW) infections would occur in WA state alone if elective outpatient procedures were to continue as usual. The majority of these infections are caused by transmission from HCW who became infected at work. Conclusion: Given the large numbers of COVID19 infections that could be prevented by canceling elective outpatient surgeries, our findings support the recommendations of the US Surgeon General, CDC, American College of Surgeons (ACS), American Society of Anesthesiologists (ASA), and Anesthesia Patient Safety Foundation (APSF) to consider rescheduling or postponing elective surgeries until the COVID19 pandemic is under better control in the US.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Yuemei Zhang; Sheng-Ru Cheng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20037952&#39;},
 {&#39;cord_uid&#39;: &#39;tg5wbwf9&#39;,
  &#39;sha&#39;: &#39;edc4eb3c1259234cb996e7d6cb544ba3d209bab4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20037994&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Coronavirus disease 2019 (COVID-19) is a novel human respiratory disease caused by the SARS-CoV-2 virus. Asymptomatic individuals in the context of COVID-19 are those subjects who are carriers of the virus but display no clinical symptoms. Current evidence reveals that this sub-population is a major contributing factor in the propagation of this disease, while escaping detection by public health surveillance systems. In this manuscript, we present a mathematical model taking into account asymptomatic carriers. Our results indicate a basic reproduction number as high as 26.5. The first three weeks of the model exhibit exponential growth, which is in agreement with average case data collected from thirteen countries with universal health care and robust communicable disease surveillance systems; the average rate of growth in the number of reported cases is 25\\% per day during this period. The model was applied to every county in the US to give estimates of mild, severe, and critical case severity, and also mortality. These estimates can be considered an upper bound.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Jacob B Aguilar; Juan B. Gutierrez&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20037994&#39;},
 {&#39;cord_uid&#39;: &#39;dm8yl4xu&#39;,
  &#39;sha&#39;: &#39;e7e0ab1fe95b766b6e913e9d14f054731224d9f3&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038018&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background We aim to investigate the profile of acute antibody response in COVID-19 patients, and provide proposals for the usage of antibody test in clinical practice. Methods A multi-center cross-section study (285 patients) and a single-center follow-up study (63 patients) were performed to investigate the feature of acute antibody response to SARS-CoV-2. A cohort of 52 COVID-19 suspects and 64 close contacts were enrolled to evaluate the potentiality of the antibody test. Results The positive rate for IgG reached 100% around 20 days after symptoms onset. The median day of serocon-version for both lgG and IgM was 13 days after symptoms onset. Seroconversion of IgM occurred at the same time, or earlier, or later than that of IgG. IgG levels in 100% patients (19/19) entered a platform within 6 days after seroconversion. The criteria of IgG seroconversion and ≥ 4-fold increase in the IgG titers in sequential samples together diagnosed 82.9% (34/41) of the patients. Antibody test aided to confirm 4 patients with COVID-19 from 52 suspects who failed to be confirmed by RT-PCR and 7 patients from 148 close contacts with negative RT-PCR. Conclusion IgM and IgG should be detected simultaneously at the early phase of infection. The serological diagnosis criterion of seroconversion or ≥ 4-fold increase in the IgG titer is suitable for a majority of COVID-19 patients. Serologic test is helpful for the diagnosis of SARS-CoV-2 infection in suspects and close contacts.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Quan-xin Long; Hai-jun Deng; Juan Chen; Jieli Hu; Bei-zhong Liu; Pu Liao; Yong Lin; Li-hua Yu; Zhan Mo; Yin-yin Xu; Fang Gong; Gui-cheng Wu; Xian-xiang Zhang; Yao-kai Chen; Zhi-jie Li; Kun Wang; Xiao-li Zhang; Wen-guang Tian; Chang-chun Niu; Qing-jun Yang; Jiang-lin Xiang; Hong-xin Du; Hua-wen Liu; Chunhui Lang; Xiao-he Luo; Shao-bo Wu; Xiao-ping Cui; Zheng Zhou; Jing Wang; Cheng-jun Xue; Xiao-feng Li; Li Wang; Xiao-jun Tang; Yong Zhang; Jing-fu Qiu; Xia-mao Liu; Jin-jing Li; De-chun Zhang; Fan Zhang; Xue-fei Cai; Deqiang Wang; Yuan Hu; Ji-hua Ren; Ni Tang; Ping Liu; Qin Li; Ai-long Huang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038018&#39;},
 {&#39;cord_uid&#39;: &#39;q856rx6b&#39;,
  &#39;sha&#39;: &#39;59ebacae3048d5db81d51968371b6350658e8b94&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Chinese Public Attention to COVID-19 Epidemic: Based on Social Media&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038026&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Since the new coronavirus epidemic in China in December 2019, information and discussions about COVID-19 have spread rapidly on the Internet and have quickly become the focus of worldwide attention, especially on social media. Objective: This study aims to investigate and analyze the public&#39;s attention to COVID-19-related events in China at the beginning of the COVID-19 epidemic in China (December 31, 2019, to February 20, 2020) through the Sina Microblog hot search list. Methods: We collected topics related to the COVID-19 epidemic on the Sina Microblog hot search list from December 31, 2019, to February 20, 2020 and described the trend of public attention on COVID-19 epidemic-related topics. ROST CM6.0 (ROST Content Mining System Version 6.0) was used to analyze the collected text for word segmentation, word frequency, and sentiment analysis. We further described the hot topic keywords and sentiment trends of public attention. We used VOSviewer to implement a visual cluster analysis of hot keywords and build a social network of public opinion content. Results: The study has four main findings. First, we analyzed the changing trend of the public&#39;s attention to the COVID-19 epidemic, which can be divided into three stages. Second, the hot topic keywords of public attention at each stage are slightly different. In addition, the emotional tendency of the public toward the COVID-19 epidemic-related hot topics has changed from negative to neutral, with negative emotions weakening and positive emotions increasing as a whole. Finally, we divided the COVID-19 topics with the most public concern into five categories: (1) new COVID-19 epidemics and their impact; (2) frontline reporting of the epidemic and prevention and control measures; (3) expert interpretation and discussion on the source of infection; (4) medical services on the frontline of the epidemic; and (5) focus on the global epidemic and the search for suspected cases. Conclusions: This is the first study of public attention on the COVID-19 epidemic using a Chinese social media platform (i.e., Sina Microblog). Our study found that social media (e.g., Sina Microblog) can be used to measure public attention to public health emergencies. During the epidemic of the novel coronavirus, a large amount of information about the COVID-19 epidemic was disseminated on Sina Microblog and received widespread public attention. We have learned about the hotspots of public concern regarding the COVID-19 epidemic. These findings can help the government and health departments better communicate with the public on health and translate public health needs into practice to create targeted measures to prevent and control the spread of COVID-19. Keywords: COVID-19; Sina Microblog; Public attention; Social media; China; Public health emergency&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Yuxin Zhao; Huilan Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038026&#39;},
 {&#39;cord_uid&#39;: &#39;9595vm0k&#39;,
  &#39;sha&#39;: &#39;1e9cafd54968d935da52bdcae678df1e6ec2b650&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2 specific antibody responses in COVID-19 patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038059&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A new coronavirus, SARS-CoV-2, has recently emerged to cause a human pandemic. Whereas molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are important for contact tracing, identifying the viral reservoir and epidemiological studies. Here, we developed serological assays for the detection of SARS-CoV-2 neutralizing, spike- and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed infections of SARS-CoV-2, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrate that most PCR-confirmed SARS-CoV-2 infected individuals seroconverted, as revealed by sensitive and specific in-house ELISAs. We found that commercial S1 IgG or IgA ELISAs were of lower specificity while sensitivity varied between the two, with IgA showing higher sensitivity. Overall, the validated assays described here can be instrumental for the detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiological and vaccine evaluation studies.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;NISREEN M.A. OKBA; Marcel A Muller; Wentao Li; Chunyan Wang; Corine H. GeurtsvanKessel; Victor M. Corman; Mart M. Lamers; Reina S. Sikkema; Erwin de Bruin; Felicity D. Chandler; Yazdan Yazdanpanah; Quentin Le Hingrat; Diane Descamps; Nadhira Houhou-Fidouh; Chantal B. E. M. Reusken; Berend-Jan Bosch; Christian Drosten; Marion P.G. Koopmans; Bart L. Haagmans&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038059&#39;},
 {&#39;cord_uid&#39;: &#39;mlwrfkm4&#39;,
  &#39;sha&#39;: &#39;7352452a89aa55529c073dfece3ea425bb91d7d2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the Risks from COVID-19 Infection in Adult Chemotherapy Patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038067&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The SARS-CoV-2 (COVID-19) novel coronavirus represents a significant health risk, particularly in older patients. Cancer is one of the leading causes of death in most rich countries, and delivering chemotherapy may be associated with increased risk in the presence of a pandemic infection. Estimating this risk is crucial in making decisions about balancing risks and benefits from administering chemotherapy. However, there are no specific data about chemotherapy risks per se. Here we develop a simple model to estimate the potential harms in patients undergoing chemotherapy during a COVID outbreak. We use age-related case fatality rates as a basis for estimating risk, and use previous data from risk of death during influenza outbreaks to estimate the additional risk associated with chemotherapy. We use data from randomised trials to estimate benefit across a range of curative and palliative settings, and address the balance of benefit against the risk of harm. We then use those data to estimate the impact on national chemotherapy delivery patterns.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Matt Williams; Kerlann Le Calvez; Ella Mi; Jiarong Chen; Seema Dadhania; Lillie Pakzad-Shahabi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038067&#39;},
 {&#39;cord_uid&#39;: &#39;qpt5dj8p&#39;,
  &#39;sha&#39;: &#39;7adf41d2120f3398e13817367c2120ebac6b99b6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Predicting the epidemic trend of COVID-19 in China and across the world using the machine learning approach&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038117&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Although the COVID-19 has been well controlled in China, it is rapidly spreading outside China that may lead to catastrophic results globally without implementation of necessary mitigation measures. Because the COVID-19 outbreak has made comprehensive and profound impacts on the world, an accurate prediction of its epidemic trend is significant. Although many studies have predicted the COVID-19 epidemic trend, most of these studies have used the early stage data and focused on the China cases. Methods: We predicted the COVID-19 epidemic trend in China and across the world using the machine learning approach. We first built the models for predicting the daily numbers of cumulative confirmed cases (CCCs), new cases (NCs), and death cases (DCs) of COVID-19 in China based on the data from Jan 20, 2020, to Mar 1, 2020. Furthermore, we built the models, derived from the models for the China cases, for predicting the epidemic trend across the world (outside China). Findings: The COVID-19 outbreak will peak on Feb 22, 2020 in China and April 10, 2020 across the world. It will be basically under control early April, 2020 in China and mid-June, 2020 across the world. The total number of COVID-19 cases will reach around 89, 000 in China and 403,000 across the world during the epidemic. Around 4,000 and 18,300 people will died of COVID-19 in China and across the world, respectively. The COVID-19 mortality rate is estimated to be around 4% all over the world. Interpretation: The COVID-19 outbreak is controllable in the foreseeable future if comprehensive and stringent control measures are taken.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Mengyuan Li; Zhilan Zhang; Shanmei Jiang; Qian Liu; Canping Chen; Yue Zhang; Xiaosheng Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038117&#39;},
 {&#39;cord_uid&#39;: &#39;6su2x8mk&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Non-severe vs severe symptomatic COVID-19: 104 cases from the outbreak on the cruise ship “Diamond Princess” in Japan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038125&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The ongoing outbreak of 2019 novel coronavirus disease (COVID-19) is a global threat. Clarifying its clinical features such as risk factors for disease progression is a pressing issue. We analyzed the difference between non-severe and severe cases with COVID-19 as a result of the mass infection on a cruise ship “Diamond Princess” in Japan. Methods In this retrospective, single-center study, total 104 cases of laboratory-confirmed COVID-19 were enrolled from the mass infection on the cruise ship from 11th to 25th Feb, 2020, at Self-Defense Forces Central Hospital in Japan. Clinical records, laboratory data, and radiological findings were collected and analyzed. Clinical outcomes were followed up until 26th Feb, 2020. Findings Of the 104 patients, 47 were male. The median age was 68 years. During the observation period, eight patients deteriorated into the severe cases. Finally, 76 and 28 patients were classified as non-severe (asymptomatic, mild), and severe cases, respectively. Chest CT abnormalities were found in 43 in non-severe cases and 23 in severe cases. The prevalence of consolidation on chest CT scan and lymphopenia on the admission day was significantly higher in severe cases and in the eight cases which worsened into severe disease during the observation period. Interpretation The high proportion of non-severe cases corrects the assessment of the trend of the outbreak. Consolidation on chest CT scan and lymphopenia were possible risk factors for deterioration of COVID-19 and contribute to the clinical management. Funding Not applicable.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Sakiko Tabata; Kazuo Imai; Shuichi Kawano; Mayu Ikeda; Tatsuya Kodama; Kazuyasu Miyoshi; Hirofumi Obinata; Satoshi Mimura; Tsutomu Kodera; Manabu Kitagaki; Michiya Sato; Satoshi Suzuki; Toshimitsu Ito; Yasuhide Uwabe; Kaku Tamura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038125&#39;},
 {&#39;cord_uid&#39;: &#39;rfni76sv&#39;,
  &#39;sha&#39;: &#39;3379f2ad661859b90114bf78804f5c11ef84e3d7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Comparison of the coronavirus pandemic dynamics in Europe, USA and South Korea&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038133&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The pandemic cased by coronavirus COVID-19 is of great concern. A detailed scientific analysis of this phenomenon is still to come, but now it is urgently needed to evaluate the disease dynamics in order to organize the appropriate quarantine activities, to estimate the required number of places in hospitals, the level of individual protection, the rate of isolation of infected persons, etc. South Korea has achieved the stabilization of the number of cases at rather low level. The epidemic dynamics there can be compared with its development in other countries to make some preliminary, but very important conclusions. Here we provide a simple method of data comparison that can be useful for both governmental organizations and anyone.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Igor Nesteruk&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038133&#39;},
 {&#39;cord_uid&#39;: &#39;pm713jib&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Virome of New York Presbyterian Hospital pediatric emergency&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038166&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Viral infection of the respiratory tract is one of the major causes of hospital visits for young children. In this study, we report the occurrence and co-occurrence of different virus types and subtypes among the patients arriving at the pediatric emergency room of New York Presbyterian Hospital, a major urban hospital. Methods: We collected nasal swabs from the patients and their accompanying persons. We also recorded the levels (None, Mild, High, and Severe) of their symptoms (Fever, Chill, Muscle Pain, Watery Eye, Runny Nose, Sneezing, Sore throat and Cough). The collected swabs were tested for the presence of common viruses infecting the respiratory tract. Results: Human Rhinovirus was the most common virus among the patients, followed Influenza and Respiratory Syncytial Virus. Human Rhinovirus was most common in summer, autumn and spring. In contrast, influenza was more common in winter. Further, Influenza A virus was more likely to co-occur with Corona Virus 229E. In comparison, Influenza was less likely to co-occur with Human Rhinovirus. Moreover, Influenza, Parainfluenza and Corona virus were associated with more severe symptoms, while Human Rhinovirus was associated with less severe symptoms. In addition, we observed that Influenza and Respiratory Syncytial Virus were more likely to infect a patient when these viruses also infected the accompanying person. We also found that it was difficult to distinguish among viruses based on the symptoms. The inability to distinguish among different virus types and subtypes is explained by the fact that multiple viruses lead to similar symptoms. Conclusions: The findings of this study provide a better understanding of respiratory viral infections in small children presenting at a pediatric emergency room in New York.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Minhaz Ud-Dean; Ioan Filip; Marta Galanti; Ruthie Birger; Devon Comito; Greg A Freyer; Sadiat Ibrahim; Benjamin Lane; Chanel Ligon; Nelsa Matienzo; Haruka Morita; Atinuke Shittu; Eudosie Tagne; Peter Dayan; Jeffrey Shaman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038166&#39;},
 {&#39;cord_uid&#39;: &#39;t1wpujpm&#39;,
  &#39;sha&#39;: &#39;6131628f8273f4c73aa0a46e0ee71e359ba86fe5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038190&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Although a number of antiviral agents have been evaluated for coronaviruses there are no approved drugs available. To provide an overview of the landscape of therapeutic research for COVID-19, we conducted a review of registered clinical trials. Methods: A review of currently registered clinical trials was performed on registries, including the Chinese (chictr.org.cn) and US (clinicaltrials.gov) databases to identify relevant studies up to March, 7th 2020. The search was conducted using the search terms “2019-nCoV”, “COVID-19”, “SARS-CoV-2”, “Hcov-19”, “new coronavirus”, “novel coronavirus”. We included interventional clinical trials focusing on patients with COVID-19 and assessing antiviral drugs or agents. Findings: Out of the 353 studies identified, 115 clinical trials were selected for data extraction. Phase IV trials were the most commonly reported study type (n=27, 23%). However, 62 trials (54%) did not describe the phase of the study. Eighty percent (n=92) of the trials were randomized with parallel assignment and the median number of planned inclusions was 63 (IQR, 36-120). Open-label studies were the most frequent (46%) followed by double-blind (13%) and single blind studies (10%). The most frequently assessed therapies were: stem cells therapy (n=23 trials), lopinavir/ritonavir (n=15), chloroquine (n=11), umifenovir (n=9), hydroxychloroquine (n=7), plasma treatment (n=7), favipiravir (n=7), methylprednisolone (n=5), and remdesivir (n=5). Remdesivir was tested in 5 trials with a median of 400 (IQR, 394-453) planned inclusions per trial, while stem cells therapy was tested in 23 trials, but had a median of 40 (IQR, 23-60) planned inclusions per trial. Lopinavir/ritonavir was associated with the highest total number of planned inclusions (2606) followed by remdesivir (2155). Only 52% of the clinical trials reported the treatment dose (n=60) and only 34% (n=39) the duration. The primary outcome was clinical in 76 studies (66%), virological in 27 (23%); radiological in 9 (8%) or immunological in three studies (3%). Interpretation: Numerous clinical trials have been registered since the beginning of the COVID-19 outbreak, however, a number of information regarding drugs or trial design were lacking. Funding: None&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Drifa Belhadi; Nathan Peiffer-Smadja; Yazdan Yazdanpanah; France Mentré; Cédric Laouénan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038190&#39;},
 {&#39;cord_uid&#39;: &#39;adcptyfj&#39;,
  &#39;sha&#39;: &#39;b0aa77344d5726699361b0d3bf2dc9b45eff8b06&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing the Global Tendency of COVID-19 Outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038224&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;COVID-19 is now widely spreading around the world as a global pandemic, whereas it had been effecitvely under control in China by March 15. In this report, we estimate the global tendency of COVID-19 and analyze the associated global epidemic risk, given that the status quo is continued without further measures being taken. The results show that the global R_0, excluding China, is estimated to be 2.63 (95% CI: 1.67 - 4.69). Seven countries, including Italy and Republic of Korea, have peak values over 10,000. And the peak arrival time of all these countries is after May 12, 2020 (95 % CI: April 1, 2020 - June 24, 2020). We predict that there are four regional clusters of the outbreak: Southeast Asia extending southward to Oceania, the Middle East, Western Europe and North America. Among them, Western Europe will become the major center of the outbreak. The peak values in Spain and Italy are estimated to be 232,661 and 108,062, respectively. The total number of peak values in Europe will exceed 703,215 (95% CI: 209,670 - 23,095,507). Above all, if the current policy environment is maintained, the cumulative number of patients worldwide will exceed 892,950 (95% CI: 335,922 - 24,930,386).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Qinghe Liu; Zhicheng Liu; Junkai Zhu; Yuhao Zhu; Deqiang Li; Zefei Gao; Liuling Zhou; Junyan Yang; Qiao Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038224&#39;},
 {&#39;cord_uid&#39;: &#39;u5i935we&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Response and role of palliative care during the COVID-19 pandemic: a national telephone survey of hospices in Italy&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038448&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Palliative care is an essential component of healthcare in pandemics, contributing to symptom control, psychological support, and supporting triage and complex decision making. Aim To examine preparedness for, and impact of, the COVID-19 pandemic on hospices in Italy to inform the response in other countries. Design Cross-sectional telephone survey, carried out in March 2020. Setting Sixteen Italian hospices, purposively sampled according to COVID-19 risk into high (more than 25 COVID-19 cases per 100,000 inhabitants), medium (15-25 cases per 100,000), and low risk (fewer than 15 cases per 100,000) regions. A brief questionnaire was developed to guide the interviews. Descriptive analysis was undertaken. Results Seven high risk, five medium risk and four low risk hospices provided data. Two high risk hospices had experienced COVID-19 cases among both patients and staff. All hospices had implemented policy changes, and several had rapidly implemented changes in practice including transfer of staff from inpatient to community settings, change in admission criteria, and daily telephone support for families. Concerns included scarcity of personal protective equipment, a lack of hospice-specific guidance on COVID-19, anxiety about needing to care for children and other relatives, and poor integration of palliative care in the acute setting. Conclusion The hospice sector is capable of responding flexibly and rapidly to the COVID-19 pandemic. Governments must urgently recognise the essential contribution of hospice and palliative care to the COVID-19 pandemic, and ensure these services are integrated into the health care system response. Availability of personal protective equipment and setting-specific guidance is essential.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Massimo Costantini; Katherine E Sleeman; Carlo Peruselli; Irene J Higginson&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038448&#39;},
 {&#39;cord_uid&#39;: &#39;ei2b8oqn&#39;,
  &#39;sha&#39;: &#39;bf013151c39358fb0c19ad4dee802a9ccb7fb0ac&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038455&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: Coronavirus disease 2019 (COVID-19) is a respiratory infection caused by the severe acute respiratory syndrome coronavirus2 (SARSCoV-2). This virus uses the angiotensin converting enzyme II (ACE2) as the cellular entry receptor to infect the lower respiratory tract. Because individuals with chronic obstructive pulmonary disease (COPD) are at increased risk of severe COVID19, we determined whether ACE2 expression in the lower airways was related to COPD and cigarette smoking. Methods: Using RNAseq, we determined gene expression levels in bronchial epithelia obtained from cytologic brushings of 6th to 8th generation airways in individuals with and without COPD. We externally validated these results from two additional independent cohorts, which used microarray technologies to measure gene expression levels from 6th to 12th generation airways. Results: In the discovery cohort (n=42 participants), we found that ACE2 expression levels were increased by 48% in the airways of COPD compared with non-COPD subjects (COPD=2.52 (0.66) log2 counts per million reads (CPM) versus non-COPD= 1.70 (0.51) CPM , p=.000762). There was a significant inverse relationship between ACE2 gene expression and FEV1% of predicted (r=negative 0.24; p=0.035). Current smoking also significantly increased ACE2 expression levels compared with never smokers (never current smokers=2.77 (90.91) CPM versus smokers=1.78 (0.39) CPM, p=0.024). These findings were replicated in the two external cohorts. Conclusions: ACE2 expression in lower airways is increased in patients with COPD and with current smoking. These data suggest that these two subgroups are at increased risk of serious COVID19 infection and highlight the importance of smoking cessation in reducing the risk.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Janice M Leung; Chen Xi Yang; Anthony Tam; Tawimas Shaipanich; Tillie L Hackett; Gurpreet K Singhera; Delbert R Dorscheid; Don D Sin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038455&#39;},
 {&#39;cord_uid&#39;: &#39;wx0nhx8i&#39;,
  &#39;sha&#39;: &#39;3fb67d663ace1f95a9228d83bf11ff675d0fbff2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Dynamic profile of severe or critical COVID-19 cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038513&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In December 2019, a cluster of acute respiratory illness, now known as SARS-CoV-2 pneumonia, occurred in Wuhan, China. World Health Organization (WHO) declared the rapidly spreading coronavirus outbreak a pandemic on March 11, 2020, acknowledging what has seemed clear for some time-the virus will likely spread to all countries on the globe. As of February 11, 2020, the Chinese Center for Disease Control and Prevention (China CDC) has officially reported that there are 2.0% (889) asymptomatic cases, 2.3% (1,023) death cases, and 80.9% mild cases among 44,672 confirmed cases. 51.4% (22,981) were male and 48.6% (21,691) were female. Lymphopenia, in particular T lymphopenia, was common among patients with SARS-COV-2 in the observation. A notable drop in CD4 and CD8 lymphocyte counts occurred early in the course of the syndrome and was associated with adverse outcomes. The appearing a phenomenon of lymphocyte depletion (PLD) suggested severe adverse outcomes. The outcome observed: 60% had discharged and 20% had die.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Yang Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038513&#39;},
 {&#39;cord_uid&#39;: &#39;y7giefdw&#39;,
  &#39;sha&#39;: &#39;5b11e063f09adbfdaf7fd515ecc6c09a83674263&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Extended SIR prediction of the epidemics trend of COVID-19 in Italy and compared with Hunan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038570&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: Coronavirus Disease 2019 (COVID19) is currently a global public health threat. Outside of China, Italy is one of the most suffering countries with the COVID19 epidemic. It is important to predict the epidemics trend of COVID19 epidemic in Italy to help develop public health strategies. Methods: We used time series data of COVID 19 from Jan 22,2020 to Mar 16,2020. An infectious disease dynamic extended susceptible infected removed (eSIR) model, which covers the effects of different intervention measures in dissimilar periods, was applied to estimate the epidemic trend in Italy. The basic reproductive number was estimated using Markov Chain Monte Carlo methods and presented using the resulting posterior mean and 95% credible interval (CI). Hunan, with similar total number of populations in Italy, was used as a comparative item. Results: In the eSIR model, we estimated that the basic reproductive number for COVID 19 was respectively 4.10 (95% CI: 2.15 to 6.77) in Italy and 3.15(95% CI: 1.71 to 5.21) in Hunan. There would be totally 30 086 infected cases (95%CI:7920-81 869) under the current country blockade and the endpoint would be Apr 25 (95%CI: Mar 30 to Aug 07) in Italy. If the country blockade is imposed 5 day later, the total number of infected cases would expand the infection scale 1.50 times. Conclusion: Italy&#39;s current strict measures can efficaciously prevent the further spread of COVID19 and should be maintained. Necessary strict public health measures be implemented as soon as possible in other European countries with a high number of COVID19 cases. The most effective strategy needs to be confirmed in further studies.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Wangping Jia; Ke Han; Yang Song; Wenzhe Cao; Shengshu Wang; Shanshan Yang; Jianwei Wang; Fuyin Kou; Penggang Tai; Jing Li; Miao Liu; Yao He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038570&#39;},
 {&#39;cord_uid&#39;: &#39;yrq58n4k&#39;,
  &#39;sha&#39;: &#39;7c83c3afd2d5fd30b962694db7d9d4defd0c26a5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;PREDICTING THE EVOLUTION OF COVID-19 IN PORTUGAL USING AN ADAPTED SIR MODEL PREVIOUSLY USED IN SOUTH KOREA FOR THE MERS OUTBREAK&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038612&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since original reports in Wuhan, China, the new coronavirus covid-19 has spread very quickly worldwide, leading the World Health Organization (WHO) to declare a state of pandemic. Moreover, as of the 13th of March 2020, the WHO has announced that the European continent is now the main centre of the pandemic. Many European governments have already implemented harsh measures to attempt to contain the spread of the virus. In Portugal, there are, as of the 18th of March 2020, 642 confirmed cases. In this study, I applied an adapted SIR model previously used in South Korea to model the MERS outbrake, which is also caused by a coronavirus, to estimate the evolution of the curve of active cases in the case of the Portuguese situation. As some of the parameters were unknown, and the data for Portugal is still scarce, given that the outbreak started later (first case on the 2nd of March) I used Italian data (first reported case in Italy on the 31st of January) to predict them. I then construct two or three different scenarios for the evolution of covid-19 in Portugal, considering both the effectiveness of the mitigation measurements implemented by the government, and the self-protective measures taken by the population, as explained in the South Korean model. In the out of control scenario, the number of active cases could reach as much as ~40,000 people by the beginning of April. If measures are appropriately taken this number can be reduced to about 700 cases. In case that some measures are taken, but their implementation is not done appropriately, the active cases could reach circa 7,000 people. The actual figures probably lie between the interval (~700-7,000) and the peak will be reached between 9th and the 20nd of April 2020.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Pedro Teles&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038612&#39;},
 {&#39;cord_uid&#39;: &#39;901m6zw0&#39;,
  &#39;sha&#39;: &#39;397bb0bda9dfe7504abf8ba98bf628e3368f9329&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Spatial Visualization of Cluster-Specific COVID-19 Transmission Network in South Korea During the Early Epidemic Phase&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.20038638&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Coronavirus disease 2019 (COVID-19) has been rapidly spreading throughout China and other countries including South Korea. As of March 12, 2020, a total number of 7,869 cases and 66 deaths had been documented in South Korea. Although the first confirmed case in South Korea was identified on January 20, 2020, the number of confirmed cases showed a rapid growth on February 19, 2020 with a total number of 1,261 cases with 12 deaths based on the Korea Centers for Disease Control and Prevention (KCDC). Using the data of confirmed cases of COVID-19 in South Korea that are publicly available from the KCDC, this paper aims to create spatial visualizations of COVID-19 transmission between January 20, 2020 and February 19, 2020. Compared to various epidemiological measures such as the basic reproduction number, spatial visualizations of the cluster-specific transmission networks may be more useful to characterize super-spreaders and the spread of the virus especially in the early epidemic phase.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;James Yeongjun Park&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.20038638&#39;},
 {&#39;cord_uid&#39;: &#39;6blcrkat&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Single-cell transcriptome profiling an adult human cell atlas of 15 major organs&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.996975&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Background: As core units of organ tissues, cells of various types play their harmonious rhythms to maintain the homeostasis of human body. It is essential to identify characteristics of the cells in human organs and their regulatory networks for understanding biological mechanisms related to health and disease. However, a systematic and comprehensive single-cell transcriptional profile across multiple organs of normal human adults has been pending. Results: we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues. Conclusions: The AHCA reveals the inter- and intra-organ heterogeneity of cell characteristics and provides a useful resource to uncover key events during the development of human diseases such as the recent outbreak of coronavirus disease 2019 (COVID-19) in the context of heterogeneity of cells and organs.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Shuai He; Linhe Wang; Yang Liu; Yiqi Li; Haitian Chen; Jinghong Xu; Wan Peng; Guowang Lin; Panpan Wei; Bo Li; Xiaojun Xia; Dan Wang; Jin-Xin BEI; Xiaoshun He; Zhiyong Guo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.996975&#39;},
 {&#39;cord_uid&#39;: &#39;wzjyv8k6&#39;,
  &#39;sha&#39;: &#39;b9d429419f94f203f63439ffd95e3cac2a958fa6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Ancient RNA virus epidemics through the lens of recent adaptation in human genomes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.997346&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractOver the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as Yersinia pestis during the Great Plague.Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics.Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;David Enard; Dmitri A. Petrov&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.997346&#39;},
 {&#39;cord_uid&#39;: &#39;8prn86g6&#39;,
  &#39;sha&#39;: &#39;297f36eeb998b4e8729a0cbdf958671e9909edc1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Nucleotide Analogues as Inhibitors of SARS-CoV-2 Polymerase&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.18.997585&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SARS-CoV-2, a member of the coronavirus family, is responsible for the current COVID-19 pandemic. Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously demonstrated that three nucleotide analogues inhibit the SARS-CoV RNA-dependent RNA polymerase (RdRp). Here, using polymerase extension experiments, we have demonstrated that the active triphosphate form of Sofosbuvir (a key component of the FDA approved hepatitis C drug EPCLUSA), is incorporated by SARS-CoV-2 RdRp, and blocks further incorporation. Using the same molecular insight, we selected the active triphosphate forms of three other anti-viral agents, Alovudine, AZT (an FDA approved HIV/AIDS drug) and Tenofovir alafenamide (TAF, an FDA approved drug for HIV and hepatitis B) for evaluation as inhibitors of SARS-CoV-2 RdRp. We demonstrated the ability of these three viral polymerase inhibitors, 3′-fluoro-3′-deoxythymidine triphosphate, 3′-azido-3′-deoxythymidine triphosphate and Tenofovir diphosphate (the active triphosphate forms of Alovudine, AZT and TAF, respectively) to be incorporated by SARS-CoV-2 RdRp, where they also terminate further polymerase extension. These results offer a strong molecular basis for these nucleotide analogues to be evaluated as potential therapeutics for COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Minchen Chien; Thomas K. Anderson; Steffen Jockusch; Chuanjuan Tao; Shiv Kumar; Xiaoxu Li; James J. Russo; Robert Kirchdoerfer; Jingyue Ju&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.18.997585&#39;},
 {&#39;cord_uid&#39;: &#39;yp38t4yb&#39;,
  &#39;sha&#39;: &#39;70e1f2a0ff48a5d24e75fc2920d757968b224d79&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Identification of a super-spreading chain of transmission associated with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20026245&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Super-spreading events were associated with the outbreaks of SARS and MERS, but their association with the outbreak of COVID-19 remains unknown. Here, we report a super-spreading transmission chain of SARS-CoV-2 involving an index patient, seven cancer patients, 40 health care workers and four family members. Methods: We conducted a retrospective study to identify the index patient and the exposed individuals linked to a chain of transmission associated with COVID-19. We collected and analyzed the data on demographic features, exposure history, clinical presentation, laboratory investigation, radiological examination, and disease outcome of these patients. Results: We identified the index patient and another presumptive super-spreader, who initiated and amplified a super-spreading transmission chain associated with COVID-19, respectively. There were 31 female and 21 male patients in this cohort, and the median age was 37 years (range: 22-79 years). Each of them had an exposure history with the index patient or his close contacts. Approximately 87% (45/52) of the patients had fever or other symptoms, 96% (50/52) had abnormal chest CT-scan findings, 86% of the tested patients (39/45) were positive for SARS-CoV-2 in the nasopharyngeal or throat swab specimen, 85% of the tested patients (29/34) were positive for SARS-CoV-2-specific IgM and/or IgG, 15% of the RT-PCR positive patients were tested negative for the specific IgM and/or IgG at the convalescent phase, and 15% of the RT-PCR negative patients were tested positive for the specific IgM and/or IgG. The severe patients experienced a significant decrease in oximetry saturation, lymphocyte, and platelet counts, along with a significant increase in C-reactive protein, D-dimer, and lactate dehydrogenase. All six fatal cases had comorbidities and five of the seven cancer patients (71%) died within 2-20 days of the disease onset. Conclusions: The super-spreading events were associated with the outbreak of COVID-19 in Wuhan and its impact on disease transmission warrants further investigation. Cancer patients appeared highly vulnerable to COVID-19. The finding that a significant portion of SARS-CoV-2 infected patients were tested negative for the serum specific IgM and IgG at the convalescent phase should be addressed by additional studies.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Ke Hu; Yang Zhao; Mengmei Wang; Qiqi Zeng; Xiaorui Wang; Ming Wang; Zhishui Zheng; Xiaochen Li; Yunting Zhang; Tao Wang; Shaolin Zeng; Yan Jiang; Dan Liu; Wenzhen Yu; Fang Hu; Hongyu Qin; Jingcan Hao; Jian Yuan; Rui Shang; Meng Jiang; Xi Ding; Binghong Zhang; Bingyin Shi; Chengsheng Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20026245&#39;},
 {&#39;cord_uid&#39;: &#39;xsgxd5sy&#39;,
  &#39;sha&#39;: &#39;c29f7a8a7083893adb02de2cf6742c45ed2518ee&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of 74 Children with Coronavirus Disease 2019&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20027078&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a newly identified pathogen which mainly spreads by droplets. Most published studies focused on adult patients with coronavirus disease 2019 (COVID-19), but data concerning pediatric patients is limited. This study aimed to determine epidemiological characteristics and clinical features of pediatric patients with COVID-19. Methods: We reviewed and analyzed data on pediatric patients with laboratory-confirmed COVID-19, including basic information, epidemiological history, clinical manifestations, laboratory and radiologic findings, treatment, outcome and follow-up results. Results: From January 20th to February 27th of 2020, a total of 74 pediatric patients with COVID-19 were included in this study. Of the 68 cases whose epidemiological data were complete, 65 (65/68, 95.59%) cases were household contacts of adults whose symptoms developed earlier. Forty (59.46%) of the infected children were male, and the median age and body weight are 6 (0.10-15.08) years and 24 (4.20-87.00) kg, respectively. Except for one critically ill case, 20 (27.03%) patients did not show any symptoms of infection, 24 (32.43%) patients had acute upper respiratory tract infection and 29 (39.19%) patients were diagnosed with mild pneumonia. Cough (24/74, 32.43%) and fever (20, 27.03%) were the predominant symptoms of 44 (59.46%) symptomatic patients at onset of the illness. Abnormalities in leukocyte count were found in 23 (31.08%) children and 10 (13.51%) children presented with abnormal lymphocyte count. Of the 34 (45.95%) patients who had nucleic acid testing results for common respiratory pathogens, 19 (19 / 34, 51.35%) showed co-infection with other pathogens other than SARS-CoV-2.Ten (13.51%) children had RT-PCR analysis of SARS-CoV-2 for fecal specimens and 8 of them showed prolonged existence of SARS-CoV-2 RNA 11 (5-23) days after nasopharynx swabs turning negative. Abnormalities in chest imaging were observed in 37 (50.00%) patients and 28 (37.84%) of them only presented with nontypical changes.All children had good prognosis with a median hospitalization duration of 11 days and follow-up period of 16.5 days. During the follow-up period, all the patients remained in quarantine at designated site and home for two 14-day obervation periods and showed no clinical manifestation,but 3 of the 8 cases with prolonged fecal shedding of SARS-CoV-2 still showed positive result of feces test.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Qin Wu; Yuhan Xing; Lei Shi; Wenjie Li; Yang Gao; Sili Pan; Ying Wang; Wendi Wang; Quansheng Xing&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20027078&#39;},
 {&#39;cord_uid&#39;: &#39;hypyxzk2&#39;,
  &#39;sha&#39;: &#39;a400b0cfaa60df27b2ee3beea37dd58157912c34&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Geo temporal distribution of 1,688 Chinese healthcare workers infected with COVID-19 in severe conditions, a secondary data analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20032532&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction The COVID 19 outbreak is posing an unprecedented challenge to healthcare workers. This study analyzes the geo temporal effects on disease severity for the 1,688 Chinese healthcare workers infected with COVID 19. Method Using the descriptive results recently reported by the Chinese CDC, we compare the percentage of infected healthcare workers in severe conditions over time and across three areas in China, and the fatality rate of infected healthcare workers with all the infected individuals in China aged 22 to 59 years. Results Among the infected Chinese healthcare workers whose symptoms onset appeared during the same ten day period, the percentage of those in severe conditions decreased statistical significantly from 19.7% (Jan 11 to 20) to 14.4% (Jan 21 to 31) to 8.7% (Feb 1 to 11). Across the country, there was also a significant difference in the disease severity among patients symptoms onset during the same period, with Wuhan being the most severe (17%), followed by Hubei Province (10.4%), and the rest of China (7.0%). The case fatality rate for the 1,688 infected Chinese healthcare workers was significantly lower than that for the 29,798 infected patients aged 20 to 59 years 0.3% (5/1,688) vs. 0.65% (193/29,798), respectively. Conclusion The disease severity improved considerably over a short period of time in China. The more severe conditions in Wuhan compared to the rest of the country may be attributable to the draconian lockdown. The clinical outcomes of infected Chinese healthcare workers may represent a more accurate estimation of the severity of COVID 19 for those who have access to quality healthcare.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Wayne Gao; Mattia Sanna; Chi Pang Wen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20032532&#39;},
 {&#39;cord_uid&#39;: &#39;x3c7zjvg&#39;,
  &#39;sha&#39;: &#39;1eb2a0cf3757558a49f6ad91294a832b04ca6353&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Characteristics of patients with COVID-19 during epidemic ongoing outbreak in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20033175&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Since Dec 2019, SARS-CoV-2 has caused about fifty thousand patients and over two thousand deaths in Wuhan, China. We reported characteristics of patients with COVID-19 during epidemic ongoing outbreak in Wuhan. Methods Data of COVID-19 patients with clinical outcome in a designated hospital in Wuhan, were retrospectively collected from electronic medical records. Characteristics were compared between patients who died or recovered, and between patients with different disease severity. Results By Feb 25, 2020, 403 patients were enrolled, 100 died and 303 recovered. Most of non-survivors tended to be males, old aged, or with chronic diseases. Duration from illness onset to admission was 9 (7-12) days. Patients with severe or critical illness had more days from onset to admission compared to those with ordinary illness. Lymphopenia, anemia, hypoproteinemia, and abnormal serum sodium were presented in 52.6%, 54.6%, 69.8%, and 21.8% cases, respectively. Patients who died or with severe/critical illness showed increased white blood cell and neutrophil count, serum total bilirubin, creatinine, hypersensitive troponin I, D-dimer, procalcitonin, and C-reactive protein, and decreased red blood cell, lymphocyte, platelet count, and serum albumin on admission compared to those who recovered or with ordinary illness. Complications of acute organ injury and secondary infection were common in patients with COVID-19, especially in non-survivors. Conclusions Multiple homeostasis disturbances were common in patients with severe or critical illness at admission. Early support should be provided, especially for old men with chronic disease, which is vital to control disease progression and reduce mortality of COVID-19 during epidemic ongoing outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Xiaomin Luo; Hongxia Xia; Weize Yang; Benchao Wang; Tangxi Guo; Jun Xiong; Zongping Jiang; Yu Liu; Xiaojie Yan; Wei Zhou; Lu Ye; Bicheng Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20033175&#39;},
 {&#39;cord_uid&#39;: &#39;std4jddn&#39;,
  &#39;sha&#39;: &#39;e6af63ce394cb9c51c04251c99565fd0e6d58c1b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 Myocarditis and Severity Factors： An Adult Cohort Study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20034124&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background Notwithstanding the clinical hallmarks of COVID-19 patients were reported, several critical issues still remain mysterious, i.e., prognostic factors for COVID-19 including extrinsic factors as viral load of SARS-CoV-2 and intrinsic factors as individual&#39;s health conditions; myocarditis incidence rate and hallmarks. Methods Demographic, epidemiologic, radiologic and laboratory data were collected by medical record reviews of adult hospitalized patients diagnosed as COVID-19. Cycle threshold (Ct) value data of real-time PCR (RT-PCR) were collected. The time duration was from 21 January to 2 March, 2020. Pulmonary inflammation index (PII) values were used for chest CT findings. Multivariate logistic regression analysis was used to identify independent severity risk factors. RESULTS In total, 84 hospitalized adult patients diagnosed as COVID-19 were included, including 20 severe and 64 nonsevere cases. The viral load of the severe group was significantly higher than that of the non-severe group, regardless of the Ct values for N or ORF1ab gene of virus (all p&lt;0.05).Typical CT abnormalities was more likely existing in the severe group than in the nonsevere group in patchy shadows or ground glass opacities, consolidation, and interlobular septal thickening (all p&lt;0.05). In addition, the PII values in the severe group was significantly higher than that in the nonsevere group (52.5 [42.5-62.5] vs 20 [5.0-31.6]; p&lt;0.001). Amongst 84 patients, 13 patients (15.48%) were noted with abnormal electrocardiograms (ECGs) and serum myocardial enzyme levels; whereas 4 (4.8%) were clinically diagnosed as SARS-CoV-2 myocarditis. Multivariable logistic regress analysis distinguished three key independent risk factors for the severity of COVID-19, including age [OR 2.350; 95% CI (1.206 to 4.580); p=0.012], Ct value [OR 0.158; 95% CI (0.025 to 0.987); p=0.048] and PII [OR 1.912; 95% CI (1.187 to 3.079); p=0.008]. Interpretation Three key-independent risk factors of COVID-19 were identified, including age, PII, and Ct value. The Ct value is closely correlated with the severity of COVID-19, and may act as a predictor of clinical severity of COVID-19 in the early stage. SARS-CoV-2 myocarditis should be highlighted despite a relatively low incidence rate (4.8%). The oxygen pressure and blood oxygen saturation should not be neglected as closely linked with the altitude of epidemic regions.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Kun-Long Ma; Zhi-Heng Liu; Chun-feng Cao; Ming-Ke Liu; Juan Liao; Jing-Bo Zou; Ling-Xi Kong; Ke-Qiang Wan; Jun Zhang; Qun-Bo Wang; Wen-Guang Tian; Guang-Mei Qin; Lei Zhang; Fun-Jun Luan; Shi-Ling Li; Liang-Bo Hu; Qian-Lu Li; Hai-Qiang Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20034124&#39;},
 {&#39;cord_uid&#39;: &#39;6rs86u5v&#39;,
  &#39;sha&#39;: &#39;7ab3f4c86db5904e41edc0adefa84b797f97c510&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Potential biochemical markers to identify severe cases among COVID-19 patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20034447&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;There is a high mortality and long hospitalization period for severe cases with 2019 novel coronavirus disease (COVID-19) pneumonia. Therefore, it makes sense to search for a potential biomarker that could rapidly and effectively identify severe cases early. Clinical samples from 28 cases of COVID-19 (8 severe cases, 20 mild cases) in Zunyi District from January 29, 2020 to February 21, 2020 were collected and otherwise statistically analysed for biochemical markers. Serum urea, creatinine (CREA) and cystatin C (CysC) concentrations in severe COVID-19 patients were significantly higher than those in mild COVID-19 patients (P&lt;0.001), and there were also significant differences in serum direct bilirubin (DBIL), cholinesterase (CHE) and lactate dehydrogenase (LDH) concentrations between severe and mild COVID-19 patients (P&lt;0.05). Serum urea, CREA, CysC, DBIL, CHE and LDH could be used to distinguish severe COVID-19 cases from mild COVID-19 cases. In particular, serum biomarkers, including urea, CREA, CysC, which reflect glomerular filtration function, may have some significance as potential indicators for the early diagnosis of severe COVID-19 and to distinguish it from mild COVID-19. Glomerular filtration function injury in severe COVID-19 patients should also be considered by clinicians.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;JIANLIN XIANG; JING WEN; XIAOQING YUAN; Shun Xiong; XUE ZHOU; CHANGJIN LIU; XUN MIN&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20034447&#39;},
 {&#39;cord_uid&#39;: &#39;vsqaxqy4&#39;,
  &#39;sha&#39;: &#39;b9c285b9be5524a8d241c0a32a1cb1ef9df155f6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Trend analysis of the COVID-19 pandemic in China and the rest of the world&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20037192&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The recent epidemic of Coronavirus (COVID-19) that started in China has already been &#34;exported&#34; to more than 140 countries in all the continents, evolving in most of them by local spreading. In this contribution we analyze the trends of the cases reported in all the Chinese provinces, as well as in some countries that, until March 15th, 2020, have more than 500 cases reported. Notably and differently from other epidemics, the provinces did not show an exponential phase. The data available at the Johns Hopkins University site seem to fit well an algebraic sub-exponential growing behavior as was pointed out recently. All the provinces show a clear and consistent pattern of slowing down with growing exponent going nearly zero, so it can be said that the epidemic was contained in China. On the other side, the more recent spread in countries like, Italy, Iran, and Spain show a clear exponential growth, as well as other European countries. Even more recently, US -which was one of the first countries to have an individual infected outside China (Jan 21st, 2020)- seems to follow the same path. We calculate the exponential growth of the most affected countries, showing the evolution along time after the first local case. We identify clearly different patterns in the analyzed data and we give interpretations and possible explanations for them. The analysis and conclusions of our study can help countries that, after importing some cases, are not yet in the local spreading phase, or have just started&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Albertine Weber; Flavio Iannelli; Sebastian Goncalves&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20037192&#39;},
 {&#39;cord_uid&#39;: &#39;qljywb1t&#39;,
  &#39;sha&#39;: &#39;f8774cec00087a3d28f245c700f836e365684ef7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Insignificant effect of counter measure for coronavirus infectious disease -19 in Japan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20037945&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: To control COVID−19 outbreak in Japan, sports and entertainment events were canceled in Japan for two weeks from 26 February to 13 March. It has been designated as voluntary event cancellation (VEC). Object: This study predicts the effectiveness of VEC enduring and after its implementation. Method: We applied a simple susceptible−infected−recovery model to data of patients with symptoms in Japan during 14 January to VEC introduction and after VEC introduction to 8 March. We adjusted the reporting delay in the latest few days. Results: Results suggest that the basic reproduction number, R0, before VEC introduced as 2.29 with a 95% confidence interval (CI) was [2.19, 2.37] and the effective reproduction number, Rv, after VEC introduced as 1. 99; its 95% CI was [1.71,2.23]. Discussion and Conclusion: No significant effect of VEC was observed.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Yoshiyuki Sugishita; Junko Kurita; Tamie Sugawara; Yasushi Ohkusa&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20037945&#39;},
 {&#39;cord_uid&#39;: &#39;czh7xqph&#39;,
  &#39;sha&#39;: &#39;9c127198193a8b257fe885ad0fabf886019ae47e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Climatic influences on the worldwide spread of SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038158&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The rapid global spread of the novel, pathogenic, SARS-CoV-2 causing the severe acute respiratory disease COVID-19, becomes a major health problem worldwide and pose the need for international predictive programs. Given the lack of both specific drugs and an efficient preventive vaccine, the expectation that the SARS-CoV-2 transmission rate might decrease in temperate regions during summer, dominated in the social scene. Here, we attempted a prediction of the worldwide spread of the infections based on climatic data, expressed by bioclimatic variables. The calculated probability maps shown that potential areas of infection follow a shift from the Tropical to Temperate and Mediterranean Bioclimatic regions. Maps show an increased probability of infections in Europe, followed by an expansion covering areas of the Middle East and Northern Africa, as well as Eastern coastal areas of North America, South-Eastern coastal areas of Latin America and two areas of Southern Australia. Our approach may, therefore, be of value for the incorporation of climatic influences in the design and implementation of public health policies. Maps are available (constantly updated) at https://navaak.shinyapps.io/CVRisk/.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Michail Bariotakis; George Sourvinos; Elias Castanas; Stergios A Pirintsos&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038158&#39;},
 {&#39;cord_uid&#39;: &#39;9sr5b44k&#39;,
  &#39;sha&#39;: &#39;e15ee53ebb2b4b3e9c52cec4196927c973d112d1&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A prospect on the use of antiviral drugs to control local outbreaks of COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038182&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: Investigate the impact of using antiviral drugs to control local outbreaks of COVID-19. Methods: Using a simulation-based model of viral transmission we tested the impact of different intervention measures for the control of COVID-19. Results: The use of an antiviral drug, in combination with contact tracing, quarantine and isolation, results in a significant decrease of the mean final size and the peak incidence of local outbreaks of COVID-19, provided delays in contact tracing are small. Conclusions: Integrating antiviral drugs together with contact tracing and quarantine is predicted through this model to be an effective tool for the control of local outbreaks of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Andrea Torneri; Pieter Jules Karel Libin; Joris Vanderlocht; Anne-Mieke Vandamme; Johan Neyts; Niel Hens&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038182&#39;},
 {&#39;cord_uid&#39;: &#39;r2ad8nip&#39;,
  &#39;sha&#39;: &#39;bdad36874ec95a2edeed70a4d4484dd99b3fdd38&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038315&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract Background: Since December 2019, more than 100,000 coronavirus disease 2019 (COVID-19) patients have been confirmed globally based on positive viral nucleic acids with real-time reverse transcriptase-polymerase chain reaction (RT-PCR). However, the association between clinical, laboratory and CT characteristics and RT-PCR results is still unclear. We sought to examine this association in detail, especially in recovered patients. Methods: We analysed data from 52 confirmed patients who had been discharged with COVID-19. The clinical, laboratory, and radiological data were dynamically recorded and compared with the admission and follow-up RT-PCR results. Results: In this cohort, 52 admitted COVID-19 patients who had confirmed positive RT-PCR results were discharged after 2 rounds of consecutively negative RT-PCR results. Compared with admission levels, CRP levels (median 4.93 mg/L [IQR: 1.78-10.20]) decreased significantly (p&lt;0.001). and lymphocyte counts (median 1.50×109/L [IQR: 1.11-1.88]) increased obviously after obtaining negative RT-PCR results (p&lt;0.001). Additionally, substantially improved inflammatory exudation was observed on chest CT except for 2 progressed patients. At the two-week follow-up after discharge, 7 patients had re-positive RT-PCR results, including the abovementioned 2 progressed patients. Among the 7 patients, new GGO was demonstrated in 2 patients. There were no significant differences in CPR levels or lymphocyte counts when comparing the negative and re-positive PCT results (all p &gt;0.05). Conclusion: Heterogeneity between CT features and RT-PCR results was found in COVID-19, especially in some recovered patients with negative RT-PCR results. Our study highlights that both RT-PCR and chest CT should be considered as the key determinants for the diagnosis and management of COVID-19 patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Hang Fu; Huayan Xu; Na Zhang; Hong Xu; Zhenlin Li; Huizhu Chen; Rong Xu; Ran Sun; Lingyi Wen; Linjun Xie; Hui Liu; Kun Zhang; Chuan Fu; Keke Hou; Zhigang Yang; Ming Yang; Yingkun Guo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038315&#39;},
 {&#39;cord_uid&#39;: &#39;qdmuhxaw&#39;,
  &#39;sha&#39;: &#39;8e91912fad635341704d5df8f36415f6522e98e2&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical Characteristics of Coronavirus Disease 2019 in Hainan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038539&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Since January 2020, coronavirus disease 2019 (Covid-19) has spread rapidly and developing the pandemic model around the world. Data have been needed on the clinical characteristics of the affected patients in an imported cases as model in island outside Wuhan. Methods: We conducted a retrospective study included all 168 confirmed cases of Covid-19 in Hainan province from 22 January 2020 to 13 March 2020. Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, radiological and laboratory data. Results: Of 168 patients, 160 have been discharged, 6 have died and 2 remain hospitalized. The median age was 51.0 years and 51.8% were females. 129 (76.8%) patients were imported cases, and 118 (70.2%), 51 (30.4%) and 52 (31%) of patients lived in Wuhan or traveled to Wuhan, had contact with Covid-19 patients, or had contact with Wuhan residents, respectively. The most common symptoms at onset of illness were fever (65.5%), dry cough (48.8%) and expectoration (32.1%). On admission, ground-glass opacity was the most common radiologic finding on chest computed tomography (60.2%). The elderly people with diabetes, hypertension and CVD are more likely to develop severe cases. Follow-up of 160 discharged patients found that 20 patients (12.5%) had a positive RT-PCR test results of pharyngeal swabs or anal swabs or fecal. Conclusions: In light of the rapid spread of Covid-19 around the world, early diagnosis and quarantine is important to curb the spread of Covid-19 and intensive treatments in early stage is to prevent patients away from critical condition.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Shijiao Yan; Xingyue Song; Feng Lin; Haiyan Zhu; Xiaozhi Wang; Min Li; Jianwen Ruan; Changfeng Lin; Xiaoran Liu; Qiang Wu; Zhiqian Luo; Wenning Fu; Song Chen; Yong Yuan; Shengxing Liu; Jinjian Yao; Chuanzhu Lv&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038539&#39;},
 {&#39;cord_uid&#39;: &#39;fcnkwwpw&#39;,
  &#39;sha&#39;: &#39;ff0ca6f06e547051f1477fb0d652363b694b1882&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Fear, Access, and the Real-Time Estimation of Etiological Parameters for Outbreaks of Novel Pathogens&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038729&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Early analysis of outbreaks of novel pathogens to evaluate their likely public health impact depends on fitting predictive models to data gathered and updated in real-time. Both transmission rates and the critical threshold (i.e. the pathogen&#39;s &#39;reproductive number&#39;) are inferred by finding the values that provide the best model fit to reported case incidence. These models and inferred results are then the basic tools used for public health planning: how many people expected to be infected, at what scales of time and space, and whether potential intervention strategies impact disease transmission and spread. An underlying assumption, however, is that the ability to observe new cases is either constant, or at least constant relative to diagnostic test availability. We present a demonstration, discussion, and mathematical analysis of how this assumption of predictable observability in disease incidence can drastically impact model accuracy. We also demonstrate how to tailor estimations of these parameters to a few examples of different types of shifting influences acting on detection, depending on the likely sensitivity of surveillance systems to errors from sources such as clinical testing rates and differences in healthcare-seeking behavior from the public over time. Finally, we discuss the implications of these corrections for both historical and current outbreaks.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Nina H Fefferman; Eric Lofgren; Nianpeng Li; Pieter Blue; David Weber; Abdul-Aziz Yakubu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038729&#39;},
 {&#39;cord_uid&#39;: &#39;a7u8xdux&#39;,
  &#39;sha&#39;: &#39;2793131bcb54871487ad7183e6b19211b6d5b64f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Bibliometric Analysis of Global Scientific Research on SARSCoV-2 (COVID-19)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038752&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background and Aim: Since late 2019, an unknown-origin pneumonia outbreak detected in Wuhan city, Hubei Province, China. We aimed to build a model to qualitatively and quantitatively assess publications of research of COVID-19 from 2019 to 2020. Materials and Methods: Data were obtained from the Web of Science (WOS), PubMed, and Scopus Core Collection on March 02, 2020, and updated on March 10. We conducted a qualitative and quantitative analysis of publication outputs, journals, authors, institutions, countries, cited references, keywords, and terms according to bibliometric methods using VOS viewer c software packages. Results: Initially, we identified 227 papers, of which after an exclusion process, 92 studies were selected for statistical analyses. China accounted for the highest proportion of published research (44 papers, 40.48%), followed by the United States (21 papers, 19.32%), and Canada (7 papers, 6.44%). Adjusted by gross domestic product (GDP), ranked first, with 0.003 articles per billion GDP. In total, the top 10 journals published 47 articles, which accounted for 51.08% of all publications in this Feld. A total of 6 studies (05.52%) were supported by National Natural Science Foundation of China. Chinese Academy of Sciences ranked second 2, 2.76%). Conclusion: Bibliometric and visualized mapping may quantitatively monitor research performance in science and present predictions. The subject of this study was the fast growing publication on COVID-19. Most studies are published in journals with very high impact factors (IFs) and other journals are more interested in this type of research.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Fateme Rafiei Nasab; Fakher rahim&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038752&#39;},
 {&#39;cord_uid&#39;: &#39;ckay4ufw&#39;,
  &#39;sha&#39;: &#39;0e88d4fb23740a8d59ec3b36256db8f99231929d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;MATHEMATICAL PREDICTIONS FOR COVID-19 AS A GLOBAL PANDEMIC&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038794&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;This study shows that the disease free equilibrium (E_0 ) for COVID-19 coronavirus does not satisfy the criteria for a locally or globally asymptotic stability. This implies that as a pandemic as declared by WHO (2020) the COVID-19 coronavirus does not have a curative vaccine yet and precautionary measures are advised through quarantine and observatory procedures. Also, the Basic Reproductive number (R_0&lt;1) by Equation (33) shows that there is a chance of decline of secondary infections when the ratio between the incidence rate in the population and the total number of infected population quarantined with observatory procedure. The effort to evaluate the disease equilibrium shows that unless there is a dedicated effort from government, decision makers and stakeholders, the world would hardly be reed of the COVID-19 coronavirus and further spread is eminent and the rate of infection will continue to increase despite the increased rate of recovery because of the absence of vaccine at the moment.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Alexander Okhuese Victor&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038794&#39;},
 {&#39;cord_uid&#39;: &#39;lioj0tkn&#39;,
  &#39;sha&#39;: &#39;3f659542438aaa6eec647cd31b6123c31e8f1c0e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A framework for identifying regional outbreak and spread of COVID-19 from one-minute population-wide surveys&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038844&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Coronavirus infection spreads in clusters and therefore early identification of these clusters is critical for slowing down the spread of the virus. Here, we propose that daily population-wide surveys that assess the development of symptoms caused by the virus could serve as a strategic and valuable tool for identifying such clusters to inform epidemiologists, public health officials, and policy makers. We show preliminary results from a survey of over 38,000 Israelis and call for an international consortium to extend this concept in order to develop predictive models. We expect such data to allow: Faster detection of spreading zones and patients; Obtaining a current snapshot of the number of people in each area who have developed symptoms; Predicting future spreading zones several days before an outbreak occurs; Evaluating the effectiveness of the various social distancing measures taken, and their contribution to reduce the number of symptomatic people. Such information can provide a valuable tool for decision makers to decide which areas need strengthening of social distancing measures and which areas can be relieved. Researchers from the U.S, Spain, and Italy have adopted our approach and we are collaborating to further improve it. We call with urgency for other countries to join this international consortium, and to share methods and data collected from these daily, simple, one-minute surveys.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Hagai Rossman; Ayya Keshet; Smadar Shilo; Amir Gavrieli; Tal Bauman; Ori Cohen; Ran Balicer; Benjamin Geiger; Yuval Dor; Eran Segal&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038844&#39;},
 {&#39;cord_uid&#39;: &#39;p42cgpf0&#39;,
  &#39;sha&#39;: &#39;c013fb6790e1ca21d720911ee61532a0cd57408c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A model for COVID-19 prediction in Iran based on China parameters&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038950&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The rapid spread of COVID-19 virus from China to other countries and outbreaks of disease require an epidemiological analysis of the disease in the shortest time and an increased awareness of effective interventions. The purpose of this study was to estimate the COVID-19 epidemic in Iran based on the SIR model. The results of the analysis of the epidemiological data of Iran from January 22 to March 8, 2020 were investigated and the prediction was made until March 29, 2020. Methods: By estimating the three parameters of time-dependent transmission rate, time-dependent recovery rate, and time-dependent mortality rate from Covid-19 outbreak in China, and using the number of Covid-19 infections in Iran, we predicted the number of patients for the next month in Iran. Each of these parameters was estimated using GAM models. All analyses were conducted in R software using the mgcv package. Findings: On average, 925 people with COVID-19 are expected to be infected daily in Iran. The epidemic peaks within one week (15.03.2020 to 03.21.2020) and reaches its highest point on 03.18.2020 with 1126 infected cases. Conclusion: The most important point is to emphasize the timing of the epidemic peak, hospital readiness, government measures and public readiness to reduce social contact.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Bushra Zareie; Amin Roshani; Mohammad Ali Mansournia; Mohammad Aziz Rasouli; Ghobad Moradi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038950&#39;},
 {&#39;cord_uid&#39;: &#39;05m50voc&#39;,
  &#39;sha&#39;: &#39;25e4fed304695ca638f62756c6e79c23a5765977&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modelling the epidemiological trend and behavior of COVID-19 in Italy&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038968&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As of March 16, 2020, over 185,000 across the world, Italy became the red hotspot for the COVID-19 pandemic after China. With over 35,000 cases and 2900 deaths reported in the month of March in Italy, it is necessary to stimulate epidemic trend to understand the behavior of COVID-19 in Italy. By S.E.I.R. simulation, we estimated the most representative epidemic parameters occurred from March 1 to 14, 2020, thus being able to evaluate the consistency of the containment rules and identify possible Sars-Cov-2 local mutations. Our estimations are based on some assumptions and limitations exited.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Alessandro Rovetta; Akshaya Srikanth Bhagavathula&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038968&#39;},
 {&#39;cord_uid&#39;: &#39;9hknw4ws&#39;,
  &#39;sha&#39;: &#39;f6b9fd7dbed6516888eca71004caaa62aa0132d4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;An exploratory randomized, controlled study on the efficacy and safety of lopinavir/ritonavir or arbidol treating adult patients hospitalized with mild/moderate COVID-19 (ELACOI)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20038984&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: The novel coronavirus pneumonia (COVID-19) outbreak has caused a global pandemic, however, effective antiviral therapeutics are still unavailable. Methods: Our study (NCT04252885), designated as ELACOI, was an exploratory randomized (2:2:1) and controlled one, exploring the efficacy and safety of lopinavir/ritonavir (LPV/r) or arbidol monotherapy treating mild/moderate COVID-19 patients. Results: This study successful enrolled 44 patients with mild/moderate COVID-19, with 21 randomly assigned to receive LPV/r, 16 to arbidol and 7 to no antiviral medication as control. Baseline characteristics of three groups were comparable. The median time of positive-to-negative conversion of SARS-CoV-2 nucleic acid was 8.5 (IQR 3, 13) days in the LPV/r group, 7 (IQR 3, 10.5) days in the arbidol group and 4 (IQR 3, 10.5) days in the control group (P=0.751). The positive-to-negative conversion rates of SARS-CoV-2 nucleic acid at day 7 and 14 did not show significant differences in the LPV/r group (42.9%, 76.2%), the arbidol group (62.5%, 87.5%) and the control group (71.4%, 71.4%) (all P&gt;0.53). No statistical differences were found among three groups in the rates of antipyresis, cough alleviation, improvement of chest CT or the deterioration rate of clinical status (all P &gt; 0.05). Overall, 5 (23.8%) patients in the LPV/r group experienced adverse events during the follow-up period. No apparent adverse events occurred in the arbidol or control group. Conclusion: LPV/r or arbidol monotherapy seems little benefit for improving the clinical outcome of mild/moderate COVID-19. LPV/r might lead to more adverse events. Due to the limitation of small sample size, further verification is needed in the future.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Yueping Li; Zhiwei Xie; Weiyin Lin; Weiping Cai; Chunyan Wen; Yujuan Guan; Xiaoneng Mo; Jian Wang; Yaping Wang; Ping Peng; Xudan Chen; Wenxin Hong; Guangming Xiao; Jinxin Liu; Lieguang Zhang; Fengyu Hu; Feng Li; Feng Li; Fuchun Zhang; Xilong Deng; Linghua Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20038984&#39;},
 {&#39;cord_uid&#39;: &#39;ptnmtvzj&#39;,
  &#39;sha&#39;: &#39;d7a0d7d944131b85f308c65b3dfc51df9bcd512f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effectiveness and safety of antiviral or antibody treatments for coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039008&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: To identify safe and effective medical countermeasures (e.g., antivirals/antibodies) to address the current outbreak of a novel coronavirus (COVID-19) Methods: Comprehensive literature searches were developed by an experienced librarian for MEDLINE, EMBASE, the Cochrane Library, and biorxiv.org/medrxiv.org; additional searches for ongoing trials and unpublished studies were conducted in clinicaltrials.gov and the Global Infectious Diseases and Epidemiology Network (GIDEON). Title/abstract and full-text screening, data abstraction, and risk of bias appraisal were carried out by single reviewers. Results: 54 studies were included in the review: three controlled trials, 10 cohort studies, seven retrospective medical record/database studies, and 34 case reports or series. These studies included patients with severe acute respiratory syndrome (SARs, n=33), middle east respiratory syndrome (MERS, n=16), COVID-19 (n=3), and unspecified coronavirus (n=2). The most common treatment was ribavirin (n=41), followed by oseltamivir (n=10) and the combination of lopinavir/ritonavir (n=7). Additional therapies included broad spectrum antibiotics (n=30), steroids (n=39) or various interferons (n=12). No eligible studies examining monoclonal antibodies for COVID-19 were identified. One trial found that ribavirin prophylactic treatment statistically significantly reduced risk of MERS infection in people who had been exposed to the virus. Of the 21 studies reporting rates of ICU admission in hospitalized SARS or MERS patients, none reported statistically significant results in favour of or against antiviral therapies. Of the 40 studies reporting mortality rates in hospitalized SARS or MERS patients, one cohort study (MERS) and one retrospective study (SARS) found a statistically significant increase in the mortality rate for patients treated with ribavirin. Eighteen studies reported potential drug-related adverse effects including gastrointestinal symptoms, anemia, and altered liver function in patients receiving ribavirin. Conclusion: The current evidence for the effectiveness and safety of antiviral therapies for coronavirus is inconclusive and suffers from a lack of well-designed prospective trials or observational studies, preventing any treatment recommendations from being made. However, it is clear that the existing body of evidence is weighted heavily towards ribavirin (41/54 studies), which has not shown conclusive evidence of effectiveness and may cause harmful adverse events so future investigations may consider focusing on other candidates for antiviral therapy.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Patricia Rios; Amruta Radhakrishnan; Jesmin Antony; Sonia M. Thomas; Mathew Muller; Sharon E. Straus; Andrea C. Tricco&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039008&#39;},
 {&#39;cord_uid&#39;: &#39;i37ygb2z&#39;,
  &#39;sha&#39;: &#39;9063827ea238079db54e7ebddb628daad3d2b523&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Forecasting ultra-early intensive care strain from COVID-19 in England&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039057&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The COVID-19 pandemic has led to unprecedented strain on intensive care unit (ICU) admission in parts of the world. Strategies to create surge ICU capacity requires complex local and national service reconfiguration and reduction or cancellation of elective activity. Theses measures require time to implement and have an inevitable lag before additional capacity comes on-line. An accurate short-range forecast would be helpful in guiding such difficult, costly and ethically challenging decisions. At the time this work began, cases in England were starting to increase. Here we present an attempt at an agile short-range forecast based on published real-time COVID-19 case data from the seven National Health Service commissioning regions in England (East of England, London, Midlands, North East and Yorkshire, North West, South East and South West). We use a Monte Carlo approach to model the likely impact of current diagnoses on regional ICU capacity over a 14 day horizon. Our model is designed to be parsimonious and based on plausible epidemiological data from the literature available. On the basis of the modelling assumptions made, ICU occupancy is likely to increase dramatically in the the days following the time of modelling. If the current exponential growth continues, 5 out of 7 commissioning regions will have more critically ill COVID-19 patients than there are ICU beds within two weeks\\todo{last thing to do}. Despite variable growth in absolute patients, all commissioning regions are forecast to be heavily burdened under the assumptions used. Whilst, like any forecast model, there remain uncertainties both in terms of model specification and robust epidemiological data in this early prospective phase, it would seem that surge capacity will be required in the very near future. We hope that our model will help policy decision makers with their preparations. The uncertainties in the data highlight the urgent need for ongoing real-time surveillance to allow forecasts to be constantly updated using high quality local patient-facing data as it emerges.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Jacob Deasy; Emma Rocheteau; Katharina Kohler; Daniel J. Stubbs; Peitro Barbiero; Pietro Liò; Ari Ercole&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039057&#39;},
 {&#39;cord_uid&#39;: &#39;txt4jiy8&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Novel Triage Tool of Artificial Intelligence Assisted Diagnosis Aid System for Suspected COVID-19 pneumonia In Fever Clinics&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039099&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Currently, the prevention and control of COVID-19 outside Hubei province in China, and other countries has become more and more critically serious. We developed and validated a diagnosis aid model without CT images for early identification of suspected COVID-19 pneumonia (S-COVID-19-P) on admission in adult fever patients and made the validated model available via an online triage calculator. Patients admitted from Jan 14 to Feb 26, 2020 with the epidemiological history of exposure to COVID-19 were included [Model development (n = 132) and validation (n = 32)]. Candidate features included clinical symptoms, routine laboratory tests and other clinical information on admission. Features selection and model development were based on Lasso regression. The primary outcome is the development and validation of a diagnosis aid model for S-COVID-19-P early identification on admission. The development cohort contains 26 S-COVID-19-P and 7 confirmed COVID-19 pneumonia cases. The model performance in held-out testing set and validation cohort resulted in AUCs of 0.841 and 0.938, F-1 score of 0.571 and 0.667, recall of 1.000 and 1.000, specificity of 0.727 and 0.778, and the precision of 0.400 and 0.500. Based on this model, an optimized strategy for S-COVID-19-P early identification in fever clinics has also been designed. S-COVID-19-P could be identified early by a machine-learning model only used collected clinical information without CT images on admission in fever clinics with 100% recall score. The well performed and validated model has been deployed as an online triage tool, which is available at: https://intensivecare.shinyapps.io/COVID19/.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Cong Feng; Zhi Huang; Lili Wang; Xin Chen; Yongzhi Zhai; Feng Zhu; Hua Chen; Yingchan Wang; Xiangzheng Su; Sai Huang; Lin Tian; Weixiu Zhu; Wenzheng Sun; Liping Zhang; Qingru Han; Juan Zhang; Fei Pan; Li Chen; Zhihong Zhu; Hongju Xiao; Yu Liu; Gang Liu; Wei Chen; Tanshi Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039099&#39;},
 {&#39;cord_uid&#39;: &#39;y8o5j2be&#39;,
  &#39;sha&#39;: &#39;0a27cb2cd52229472fcfc3e49d3a3cb7179867e4&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Age profile of susceptibility, mixing, and social distancing shape the dynamics of the novel coronavirus disease 2019 outbreak in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039107&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Strict interventions were successful to control the novel coronavirus (COVID-19) outbreak in China. As transmission intensifies in other countries, the interplay between age, contact patterns, social distancing, susceptibility to infection and disease, and COVID-19 dynamics remains unclear. To answer these questions, we analyze contact surveys data for Wuhan and Shanghai before and during the outbreak and contact tracing information from Hunan Province. Daily contacts were reduced 7-9 fold during the COVID-19 social distancing period, with most interactions restricted to the household. Children 0-14 years were 59% (95% CI 7-82%) less susceptible than individuals 65 years and over. A transmission model calibrated against these data indicates that social distancing alone, as implemented in China during the outbreak, is sufficient to control COVID-19. While proactive school closures cannot interrupt transmission on their own, they reduce peak incidence by half and delay the epidemic. These findings can help guide global intervention policies.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Juanjuan Zhang; Maria Litvinova; Yuxia Liang; Yan Wang; Wei Wang; Shanlu Zhao; Qianhui Wu; Stefano Merler; Cecile Viboud; Alessandro Vespignani; Marco Ajelli; Hongjie Yu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039107&#39;},
 {&#39;cord_uid&#39;: &#39;h1b0s2wy&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Assessing the potential impacts of COVID-19 in Brasil: Mobility, Morbidity and Impact to the Health System&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039131&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The two largest Brazilian cities, Rio de Janeiro and Sao Paulo, have declared community transmission of COVID-19 on mid March, 2020. In this study we estimated the most vulnerable areas in Brazil for COVID-19, both in terms of risk of introduction and risk of mortality associated with social vulnerability. We explore the most likely routes of spread through the country, from the already exposed cities, according to human mobility statistics. The resulting maps should help authorities in their efforts to prioritize actions and if resource allocation to mitigate the effects of the pandemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Flavio Codeco Coelho; Raquel Martins Lana; Oswaldo G Cruz; Claudia T Codeco; Daniel Villela; Leonardo S Bastos; Ana Pastory y Piontti; Jessica T Davis; Alessandro Vespignani; Marcelo F.C. Gomes&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039131&#39;},
 {&#39;cord_uid&#39;: &#39;rgtv9t84&#39;,
  &#39;sha&#39;: &#39;ec35f3e1ae3ec647151c5832b9b8f741f1ae1822&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Role of meteorological temperature and relative humidity in the January-February 2020 propagation of 2019-nCoV in Wuhan, China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039164&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Identified in December 2019, the 2019-nCoV emerged in Wuhan, China, and its spread increased rapidly, with cases arising across Mainland China and several other countries. By January 2020, the potential risks imposed by 2019-nCoV in human health and economical activity were promptly highlighted. Considerable efforts have been devoted for understanding the transmission mechanisms aimed to pursue public policies oriented to mitigate the number of infected and deaths. An important question requiring some attention is the role of meteorological variables (e.g., temperature and humidity) in the 2019-nCoV transmission. Correlations between meteorological temperature and relative humidity with the number of daily confirmed cases were explored in this work for the epicenter city of Wuhan, China for the period from 29 January to March 6, 2020. Long-term trend of temperature and relative humidity was obtained with a 14-days adjacent-averaging filter, and lagged correlations of the number of daily confirmed cases were explored. The analysis showed negative correlations between temperatures with the number of daily confirmed cases. Maximum correlations were found for 6-day lagged temperatures, which is likely reflecting the incubation period of the virus. It was postulated that the indoor crowding effect is responsible of the high incidence of 2019-nCoV cases, where low absolute humidity and close human contact facilitate the transport of aerosol droplets.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Jose Alvarez-Ramirez; MONICA MERAZ&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039164&#39;},
 {&#39;cord_uid&#39;: &#39;cycc3fvl&#39;,
  &#39;sha&#39;: &#39;4346e63e692fcda7d3076d25df3e6f50b9c31407&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The local stability of a modified multi-strain SIR model for emerging viral strains&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039198&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We study a novel multi-strain SIR epidemic model with selective immunity by vaccination. A newer strain is made to emerge in the population when a preexisting strain has reached equilibrium. We assume that this newer strain does not exhibit cross-immunity with the original strain, hence those who are vaccinated and recovered from the original strain become susceptible to the newer strain. Recent events involving the COVID-19 virus demonstrates that it is possible for a viral strain to emerge from a population at a time when the influenza virus, a well-known virus with a vaccine readily available for some of its strains, is active in a population. We solved for four different equilibrium points and investigated the conditions for existence and local stability. The reproduction number was also determined for the epidemiological model and found to be consistent with the local stability condition for the disease-free equilibrium.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Miguel Fudolig; Reka Howard&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039198&#39;},
 {&#39;cord_uid&#39;: &#39;3tfp9hug&#39;,
  &#39;sha&#39;: &#39;e6e48650999bcbd06d5271bb52001457f82d67f9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Basic and effective reproduction numbers of COVID-19 cases in South Korea excluding Sincheonji cases&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039347&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;In February and March 2020, COVID-19 epidemic in South Korea met a large black swan effect by a Sincheonji cult mass infection in Daegu-Gyeongbuk area. The black swan made it difficult to evaluate that the current policies for infection prevention including social distancing, closing schools, hand washing, and wearing masks good enough or not. Therefore, in this study, we evaluated basic reproduction number (R0) and time-dependent reproduction number (Rt) of confirmed cases based on various kinds of populations, including total, Daegu-Geyoengbuk, except-Daegu-Gyeongbuk, Sincheonji, and except-Sincheonji. In total, it seems the infection is going to be under control, but this is never true because in the except-Sincheonji and except-Daegu-Geyongbuk cases, R0 is still above 1.0, and Rt is drifting around 1.0. This study could be used to determine government policies in the near future.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Jeongeun Hwang; Hyunjung Park; Jiwon Jung; Sung-Han Kim; Namkug Kim&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039347&#39;},
 {&#39;cord_uid&#39;: &#39;e6q92shw&#39;,
  &#39;sha&#39;: &#39;23b618de7564c9acf9d35c75ed18d7d15adac6e9&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;AI-assisted CT imaging analysis for COVID-19 screening: Building and deploying a medical AI system in four weeks&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039354&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The sudden outbreak of novel coronavirus 2019 (COVID-19) increased the diagnostic burden of radiologists. In the time of an epidemic crisis, we hoped artificial intelligence (AI) to help reduce physician workload in regions with the outbreak, and improve the diagnosis accuracy for physicians before they could acquire enough experience with the new disease. Here, we present our experience in building and deploying an AI system that automatically analyzes CT images to detect COVID-19 pneumonia features. Different from conventional medical AI, we were dealing with an epidemic crisis. Working in an interdisciplinary team of over 30 people with medical and / or AI background, geographically distributed in Beijing and Wuhan, we were able to overcome a series of challenges in this particular situation and deploy the system in four weeks. Using 1,136 training cases (723 positives for COVID-19) from five hospitals, we were able to achieve a sensitivity of 0.974 and specificity of 0.922 on the test dataset, which included a variety of pulmonary diseases. Besides, the system automatically highlighted all lesion regions for faster examination. As of today, we have deployed the system in 16 hospitals, and it is performing over 1,300 screenings per day.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Shuo Jin; Bo Wang; Haibo Xu; Chuan Luo; Lai Wei; Wei Zhao; Xuexue Hou; Wenshuo Ma; Zhengqing Xu; Zhuozhao Zheng; Wenbo Sun; Lan Lan; Wei Zhang; Xiangdong Mu; Chenxi Shi; Zhongxiao Wang; Jihae Lee; Zijian Jin; Minggui Lin; Hongbo Jin; Liang Zhang; Jun Guo; Benqi Zhao; Zhizhong Ren; Shuhao Wang; Zheng You; Jiahong Dong; Xinghuan Wang; Jianming Wang; Wei Xu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039354&#39;},
 {&#39;cord_uid&#39;: &#39;9032hh5c&#39;,
  &#39;sha&#39;: &#39;a9c98dc9001a93b57599a06a1145df12751ccad6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Chinese and Italian COVID-19 outbreaks can be correctly described by a modified SIRD model&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039388&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The COVID-19 disease is rapidly spreading in whole globe, affecting millions of people and pushing governments to take drastic measures to contain the outbreaks. The understanding of the dynamics of the epidemic is of great interest for the governments and health authorities that are facing COVID-19 outbreaks. The scarce presence of epidemiologic data, due to the still ongoing outbreaks, makes prediction difficult and mainly based on heuristic (fitting) models. However, these models with non-physical based parameters, can only give limited insight in the evolution of the outbreaks. In this work a SIRD compartmental model was developed to describe and predict the evolution of the Chinese and Italian outbreaks. Exploiting the similarities of the measures taken by the governments to contain the virus and of the total population number of Hubei province and Italy, the model was tuned on the Chinese outbreak (almost extinguished) and by perturbation the Italian outbreak was describe and predicted. With this approach the lack of data due to the still ongoing Italian epidemic was compensated by the Chinese experience, allowing to obtain a SIRD model for this relatively new COVID-19 outbreak. According to the modeling prediction the peak of infection should be on the 28th of March, with almost 42,000 infected people.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Diego Caccavo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039388&#39;},
 {&#39;cord_uid&#39;: &#39;4kzv06e6&#39;,
  &#39;sha&#39;: &#39;d83f238ec5aeaf6f08d20c4416c1ca9f631f698e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Healthcare worker absenteeism, child care costs, and COVID-19 school closures: a simulation analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.20039404&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;School closures have been enacted as a measure of mitigation during the ongoing COVID-19 pandemic due to their ability to reduce transmission. It has been shown that school closures could cause absenteeism amongst healthcare workers with dependent children, but there is a need for detailed high-resolution analysis of the relationship between school closures and healthcare worker absenteeism to inform local community preparedness. We provide national- and county-level simulations of school closures and absenteeism across the United States. At the national level, we estimate the projected absenteeism rate to range from 7.5% to 8.6%, and the effectiveness of school closures to range from 172 to 218 fewer hospital beds used per 100,000 people at peak demand. At the county-level, we find substantial variations of projected absenteeism and school closure effects, ranging from 2.0% to 18.6% absenteeism and 88 to 280 fewer hospital beds used per 100,000 people at peak demand. We also find significant associations between levels of absenteeism and COVID-19 complication factors. We observe from our models that an estimated 98.8% of counties would find it less expensive to provide child care to all healthcare workers with children than to bear the costs of healthcare worker absenteeism during school closures, identifying child care subsidization as a potential solution to help maintain healthcare systems during a pandemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Elizabeth T Chin; Benjamin Q Huynh; Nathan C Lo; Trevor Hastie; Sanjay Basu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.20039404&#39;},
 {&#39;cord_uid&#39;: &#39;t5iz3vdz&#39;,
  &#39;sha&#39;: &#39;c06a2e0865d3359e028815022b796a342cfba7f1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple endemic, epidemic and bat coronavirus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.997890&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronaviruses (CoVs) traffic frequently between species resulting in novel disease outbreaks, most recently exemplified by the newly emerged SARS-CoV-2. Herein, we show that the ribonucleoside analog β-D-N4-hydroxycytidine (NHC, EIDD-1931) has broad spectrum antiviral activity against SARS-CoV 2, MERS-CoV, SARS-CoV, and related zoonotic group 2b or 2c Bat-CoVs, as well as increased potency against a coronavirus bearing resistance mutations to another nucleoside analog inhibitor. In mice infected with SARS-CoV or MERS-CoV, both prophylactic and therapeutic administration of EIDD-2801, an orally bioavailable NHC-prodrug (b-D-N4-hydroxycytidine-5’-isopropyl ester), improved pulmonary function, and reduced virus titer and body weight loss. Decreased MERS-CoV yields in vitro and in vivo were associated with increased transition mutation frequency in viral but not host cell RNA, supporting a mechanism of lethal mutagenesis. The potency of NHC/EIDD-2801 against multiple coronaviruses, its therapeutic efficacy, and oral bioavailability in vivo, all highlight its potential utility as an effective antiviral against SARS-CoV-2 and other future zoonotic coronaviruses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Timothy P. Sheahan; Amy C. Sims; Shuntai Zhou; Rachel L. Graham; Collin S. Hill; Sarah R. Leist; Alexandra Schäfer; Kenneth H. Dinnon; Stephanie A. Montgomery; Maria L. Agostini; Andrea J. Pruijssers; James D. Chapell; Ariane J. Brown; Gregory R. Bluemling; Michael G. Natchus; Manohar Saindane; Alexander A. Kolykhalov; George Painter; Jennifer Harcourt; Azaibi Tamin; Natalie J. Thornburg; Ronald Swanstrom; Mark R. Denison; Ralph S. Baric&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.997890&#39;},
 {&#39;cord_uid&#39;: &#39;wnh6h9f0&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Molecular characterization of SARS-CoV-2 in the first COVID-19 cluster in France reveals an amino-acid deletion in nsp2 (Asp268Del)&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.998179&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWe present the first genetic characterization of a COVID-19 cluster in Europe using metagenomic next-generation sequencing (mNGS). Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England and in Netherlands, suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Antonin Bal; Grégory Destras; Alexandre Gaymard; Maude Bouscambert-Duchamp; Martine Valette; Vanessa Escuret; Emilie Frobert; Geneviève Billaud; Sophie Trouillet-Assant; Valérie Cheynet; Karen Brengel-Pesce; Florence Morfin; Bruno Lina; Laurence Josset&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.998179&#39;},
 {&#39;cord_uid&#39;: &#39;vlqd192g&#39;,
  &#39;sha&#39;: &#39;11bb388ea0dbb352e25fcbc8614bfd6e62831e8b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;All-in-One Dual CRISPR-Cas12a (AIOD-CRISPR) Assay: A Case for Rapid, Ultrasensitive and Visual Detection of Novel Coronavirus SARS-CoV-2 and HIV virus&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.998724&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;A recent outbreak of novel coronavirus (SARS-CoV-2), the causative agent of COVID-19, has spread rapidly all over the world. Human immunodeficiency virus (HIV) is another deadly virus and causes acquired immunodeficiency syndrome (AIDS). Rapid and early detection of these viruses will facilitate early intervention and reduce disease transmission risk. Here, we present an All-In-One Dual CRISPR-Cas12a (termed &#34;AIOD-CRISPR&#34;) assay method for simple, rapid, ultrasensitive, one-pot, and visual detection of coronavirus SARS-CoV-2 and HIV virus. In our AIOD CRISPR assay, a pair of crRNAs was introduced to initiate dual CRISPR-Cas12a detection and improve detection sensitivity. The AIOD-CRISPR assay system was successfully utilized to detect nucleic acids (DNA and RNA) of SARS-CoV-2 and HIV with a sensitivity of few copies. Also, it was evaluated by detecting HIV-1 RNA extracted from human plasma samples, achieving a comparable sensitivity with real-time RT-PCR method. Thus, our method has a great potential for developing next-generation point-of-care molecular diagnostics.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Xiong Ding; Kun Yin; Ziyue Li; Changchun Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.998724&#39;},
 {&#39;cord_uid&#39;: &#39;yi6yu5l1&#39;,
  &#39;sha&#39;: &#39;9bf79ddffb9b8f84da316dbb52d09d960f164739&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The first-in-class peptide binder to the SARS-CoV-2 spike protein&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.19.999318&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronavirus disease 19 (COVID-19) is an emerging global health crisis. With over 200,000 confirmed cases to date, this pandemic continues to expand, spurring research to discover vaccines and therapies. SARS-CoV-2 is the novel coronavirus responsible for this disease. It initiates entry into human cells by binding to angiotensin-converting enzyme 2 (ACE2) via the receptor binding domain (RBD) of its spike protein (S). Disrupting the SARS-CoV-2-RBD binding to ACE2 with designer drugs has the potential to inhibit the virus from entering human cells, presenting a new modality for therapeutic intervention. Peptide-based binders are an attractive solution to inhibit the RBD-ACE2 interaction by adequately covering the extended protein contact interface. Using molecular dynamics simulations based on the recently solved ACE2 and SARS-CoV-2-RBD co-crystal structure, we observed that the ACE2 peptidase domain (PD) α1 helix is important for binding SARS-CoV-2-RBD. Using automated fast-flow peptide synthesis, we chemically synthesized a 23-mer peptide fragment of the ACE2 PD α1 helix composed entirely of proteinogenic amino acids. Chemical synthesis of this human derived sequence was complete in 1.5 hours and after work up and isolation &gt;20 milligrams of pure material was obtained. Bio-layer interferometry revealed that this peptide specifically associates with the SARS-CoV-2-RBD with low nanomolar affinity. This peptide binder to SARS-CoV-2-RBD provides new avenues for COVID-19 treatment and diagnostic modalities by blocking the SARS-CoV-2 spike protein interaction with ACE2 and thus precluding virus entry into human cells.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;G. Zhang; S. Pomplun; A. R. Loftis; A. Loas; B. L. Pentelute&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.19.999318&#39;},
 {&#39;cord_uid&#39;: &#39;ld0vo1rl&#39;,
  &#39;sha&#39;: &#39;06d12dc5ac32d82387c65370d0a600e13059122d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 coronavirus vaccine design using reverse vaccinology and machine learning&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.000141&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;To ultimately combat the emerging COVID-19 pandemic, it is desired to develop an effective and safe vaccine against this highly contagious disease caused by the SARS-CoV-2 coronavirus. Our literature and clinical trial survey showed that the whole virus, as well as the spike (S) protein, nucleocapsid (N) protein, and membrane (M) protein, have been tested for vaccine development against SARS and MERS. However, these vaccine candidates might lack the induction of complete protection and have safety concerns. We then applied the Vaxign reverse vaccinology tool and the newly developed Vaxign-ML machine learning tool to predict COVID-19 vaccine candidates. By investigating the entire proteome of SARS-CoV-2, six proteins, including the S protein and five non-structural proteins (nsp3, 3CL-pro, and nsp8-10), were predicted to be adhesins, which are crucial to the viral adhering and host invasion. The S, nsp3, and nsp8 proteins were also predicted by Vaxign-ML to induce high protective antigenicity. Besides the commonly used S protein, the nsp3 protein has not been tested in any coronavirus vaccine studies and was selected for further investigation. The nsp3 was found to be more conserved among SARS-CoV-2, SARS-CoV, and MERS-CoV than among 15 coronaviruses infecting human and other animals. The protein was also predicted to contain promiscuous MHC-I and MHC-II T-cell epitopes, and linear B-cell epitopes localized in specific locations and functional domains of the protein. By applying reverse vaccinology and machine learning, we predicted potential vaccine targets for effective and safe COVID-19 vaccine development. We then propose that an &#34;Sp/Nsp cocktail vaccine&#34; containing a structural protein(s) (Sp) and a non-structural protein(s) (Nsp) would stimulate effective complementary immune responses.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Edison Ong; Mei U Wong; Anthony Huffman; Yongqun He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.000141&#39;},
 {&#39;cord_uid&#39;: &#39;wxagjqbt&#39;,
  &#39;sha&#39;: &#39;04dc6d9c1e3f28e2b57934301593cf3da58b9935&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.000794&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 15 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Marek Prachar; Sune Justesen; Daniel B Steen-Jensen; Stephan P Thorgrimsen; Erik Jurgons; Ole Winther; Frederik Otzen Bagger&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.000794&#39;},
 {&#39;cord_uid&#39;: &#39;7v5aln90&#39;,
  &#39;sha&#39;: &#39;ca950c8677b72b5c4812be3f7d082029df3c12ea&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.000885&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2&#39;s entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2&#39;s ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-22&#39;,
  &#39;authors&#39;: &#39;Xiaojun Li; Elena E. Giorgi; Manukumar Honnayakanahalli Marichann; Brian Foley; Chuan Xiao; Xiang-peng Kong; Yue Chen; Bette Korber; Feng Gao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.000885&#39;},
 {&#39;cord_uid&#39;: &#39;oc584x1k&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;RT-qPCR DETECTION OF SARS-CoV-2 RNA FROM PATIENT NASOPHARYNGEAL SWAB USING QIAGEN RNEASY KITS OR DIRECTLY VIA OMISSION OF AN RNA EXTRACTION STEP&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.001008&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The ongoing SARS-CoV-2 pandemic has caused an unprecedented need for rapid diagnostic screening [1]. The primary assay employed is a reverse transcription quantitative polymerase chain reaction (RT-qPCR) assay that requires the use of an RNA extraction kit [2,3]. In the United States, the Centers for Disease Control and Prevention (CDC) protocol requires the use of the Qiagen QIAamp DSP Viral RNA Mini kit. The current shortage of this RNA extraction kit during this pandemic has resulted in a severe bottleneck in testing capacity. To address this problem, we tested two alternative strategies: the use of alternative RNA extraction kits or a direct RT-qPCR assay that omits an RNA extraction step altogether. We found that the Qiagen RNeasy Mini kit and the Qiagen RNeasy Micro kit could be substituted for the QIAamp Viral RNA Mini kit. Importantly, we report here that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab, without the need for an RNA extraction step of any kind. Collectively, our findings provide viable options to circumvent supply chain issues in COVID-19 testing. Further, the ability to omit the RNA extraction step from RT-qPCR screening protocols would drastically ease supply chokepoints of COVID-19 screening and should be applicable throughout the world. We would note that our findings are preliminary and based on a single pooled nasopharyngeal swab sample from two previously confirmed positive COVID-19 patients. But due to the urgent need for high volume COVID-19 screening, we wanted to make these findings available immediately while we conduct replicate studies using additional patient samples.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Emily A Bruce; Scott Tighe; Jessica J Hoffman; Pheobe Laaguiby; Diana L Gerrard; Sean A. Diehl; Debra G. B. Leonard; Christopher D. Huston; Beth D. Kirkpatrick; Jessica W Crothers; Julie Dragon; Jason W Botten&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.001008&#39;},
 {&#39;cord_uid&#39;: &#39;50oy9qqy&#39;,
  &#39;sha&#39;: &#39;efd78d873d7c23d511d82524d6e6d9a0c2ec9551&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Predicting COVID-19 malignant progression with AI techniques&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20037325&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;Background: The coronavirus disease 2019 (COVID-19) has become a worldwide pandemic since mid-December 2019, which greatly challenge public medical systems. With limited medical resources, it is a natural strategy, while adopted, to access the severity of patients then determine the treatment priority. However, our work observes the fact that the condition of many mild outpatients quickly worsens in a short time, i.e. deteriorate into severe/critical cases. Hence, it has been crucial to early identify those cases and give timely treatment for optimizing treatment strategy and reducing mortality. This study aims to establish an AI model to predict mild patients with potential malignant progression. Methods: A total of 133 consecutively mild COVID-19 patients at admission who was hospitalized in Wuhan Pulmonary Hospital from January 3 to February 13, 2020, were selected in this retrospective IRB-approved study. All mild patients at admission were categorized into groups with or without malignant progression. The clinical and laboratory data at admission, the first CT, and the follow-up CT at severe/critical stage of the two groups were compared with Chi-square test, Fisher&#39;s exact test, and t test. Both traditional logistic regression and deep learning-based methods were used to build the prediction models. The area under ROC curve (AUC) was used to evaluate the models. Results: The deep learning-based method significantly outperformed logistic regression (AUC 0.954 vs. 0.893). The deep learning-based method achieved a prediction AUC of 0.938 by combining the clinical data and the CT data, significantly outperforming its counterpart trained with clinical data only by 0.141. By further considering the temporal information of the CT sequence, our model achieved the best AUC of 0.954. The proposed model can be effectively used for finding out the mild patients who are easy to deteriorate into severe/critical cases, so that such patients get timely treatments while alleviating the limitations of medical resources.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Xiang Bai; Cong Fang; Yu Zhou; Song Bai; Zaiyi Liu; Qianlan Chen; Yongchao Xu; Tian Xia; Shi Gong; Xudong Xie; Dejia Song; Ronghui Du; Chunhua Zhou; Chengyang Chen; Dianer Nie; Dandan Tu; Changzheng Zhang; Xiaowu Liu; Lixin Qin; Weiwei Chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20037325&#39;},
 {&#39;cord_uid&#39;: &#39;3wegrnyn&#39;,
  &#39;sha&#39;: &#39;b19bcc6ecbec9356e42d8309b9def1270a107fce&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;High COVID-19 incidence among Norwegian travellers returned from Lombardy: implications for travel restrictions&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20038406&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;On February 27th, three cases of COVID-19 were reported among Norwegians that had recently returned from Lombardy, Italy. Travellers from the region rapidly became the most common source of imported infections in the earliest stage of the Norwegian COVID-19 epidemic. The situation was exacerbated by the unfortunate temporal overlap between the Norwegian winter holidays and intense epidemic spread of COVID-19 in Northern Italy, resulting in a large number of infected travellers. Here we combined flight data on travels between Norway and Lombardy with patient-level data to determine the fraction of travellers returning to Norway that had been infected with SARS-CoV-2. Travellers returning to Norway from Lombardy contracted COVID-19 at incidence rates up to 0.02 per person-day in the period spanning February 21st and March 1st, with a clear uptick in transmission in the middle of the period. This shows an example of the infection risk in tourist destinations being several fold higher than elsewhere in the region. In Norway, this is also supported by high rates of infections among tourists returning from Austria in February and March, despite a low number of reported cases in the country at the time. The massive COVID-19 prevalence among travellers suggest that mandatory quarantine of returning travellers or suspension of non-essential international flights is essential if the aim is to control or suppress the COVID-19 pandemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Ola Brynildsrud; Vegard Eldholm&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20038406&#39;},
 {&#39;cord_uid&#39;: &#39;f3ikyl9d&#39;,
  &#39;sha&#39;: &#39;4a114f33b879a7440bdad6356e22435b4858a336&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating the reproduction number of COVID-19 in Iran using epidemic modeling&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20038422&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: As reported by Iranian governments, the first cases of coronavirus (COVID-19) infections confirmed in Qom, Iran on February 19, 2020 (30 Bahman 1398). The number of identified cases afterward increased rapidly and the novel coronavirus spread to all provinces of the country. This study aimed to fit an epidemic model to the reported case data to estimate the basic reproduction number (R_0) of COVID-19 in Iran. Methods: We used data from February 21, 2020, to March 18, 2020, on the number of cases reported by Iranian governments and we employed the SIR (Susceptible-Infectious-Removed) epidemic spreading model to fit the transmission model to the reported case data by tuning the parameters in order to estimate the basic reproduction number of COVID-19 in Iran. Results: The value of reproduction number was estimated 4.86 in the first week and 4.5 in the second and decreasing from 4.29 to 2.1 in the third and fourth weeks. Conclusions: The results indicate that the basic reproduction number of COVID-19 is significantly larger than one and higher than previous estimated R_0 of COVID-19 in the early stages. Based on the results, it is necessary to implement social distancing and a strict restriction on travelling to control the outbreak especially on Nowruz (Persian New Year).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Ebrahim Sahafizadeh; Samaneh Sartoli&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20038422&#39;},
 {&#39;cord_uid&#39;: &#39;nz02frdm&#39;,
  &#39;sha&#39;: &#39;52c10207ba9098e1f733dbb84c796bd321b33eb0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Screening and managing of suspected or confirmed novel coronavirus (COVID-19) patients: experiences from a tertiary hospital outside Hubei province&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20038679&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives: To report our experiences screening and managing patients with suspected or confirmed novel coronavirus (COVID-19) disease using a hospital-specific protocol. Design: Longitudinal cohort study. Setting: A 1,200 bed tertiary care teaching hospital in Chengdu, Sichuan, China. Participants: 802 adults presenting to hospital with concerns of having COVID-19, 1,246 inpatients and 2,531 hospital visitors. Interventions: Screening and management of patients using a hospital-specific protocol, which included fever triage, monitoring visitors and patients, emergency response, personnel training for healthcare team members, health education for patients and family, medical materials management, disinfection and wastes disposal protocols. Results: Between 23 January and 28 February 2020, 73 people were identified as having fever plus respiratory signs with/without a history of exposure and were tested for the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by our hospital lab using RT PCR. Forty-five of these 73 people were subsequently excluded based on one negative RT PCR result plus positive results to quick screening tests for flu or other respiratory viruses. The remaining 28 people received a second RT PCR test 24 h later. Three people (all hospital visitors) were confirmed positive for COVID-19 based on two consecutive positive RT PCR tests whilst 25 people were excluded based on two consecutive negative tests. The three COVID-19 confirmed cases received non-critical care. There were no new infections of medical staff or new infections of other hospital inpatients. Conclusions: All three cases were detected as a result of vigilant monitoring of hospital visitors. Whilst screening out-patients presenting to a fever clinic remains important, monitoring visitors must not be overlooked.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Hong Pu; Yujun Xu; Gordon S Doig; Yan Zhou&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20038679&#39;},
 {&#39;cord_uid&#39;: &#39;5242ns8i&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19: Modelling Local Transmission and Morbidity effects to provide an estimate of overall Relative Healthcare Resource Impact by General Practice Granularity&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039024&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) is the name given to the 2019 novel coronavirus. COVID-19 is the name given to the disease associated with the virus. SARS-CoV-2 is a new strain of coronavirus that has not been previously identified in humans. Methods Two key factors were analysed which when multiplied together would give an estimate of relative demand on healthcare utilisation. These factors were case incidence and case morbidity. GP Practice data was used as this provided the most geographically granular source of published public population data. To analyse case incidence, the latest values for indicators that could be associated with infection transmission rates were collected from the Office of National Statistics (ONS) and Quality Outcome Framework (QOF) sources. These included population density, % age &gt;16 at fulltime work/education, % age over 60, % BME ethnicity, social deprivation as IMD 2019, Location as latitude/longitude, and patient engagement as % self-confident in their own long term condition management. Average case morbidity was calculated by applying the international mortality Odds Ratio to the local population relevant age and disease prevalences and then summing and dividing by the equivalent national figure. To provide a comparative measure of overall healthcare resource impact, individual GP practice impact scores were compared against the median practice. Results The case incidence regression is a dynamic situation with the significance of specific factors moderating over time as the balance between external infection, community transmission and impact of mitigation measures feeds through to the number of cases. It showed that currently Urban, % Working and age &gt;60 were the strongest determinants of case incidence. The local population comorbidity remains unchanged. The range of relative HC impact was wide with 80% of practices falling between 20%-250% of the national median. Once practice population numbers were included it showed that the top 33% of GP practices supporting 45% of the patient population would require 68% of COVID-19 healthcare resources. The model provides useful information about the relative impact of Covid-19 on healthcare workload at GP practice granularity in all parts of England. Conclusion Covid-19 is impacting on the utilisation of health and social care resources across the country. This model provides a method for predicting relative local levels of disease burden based on defined criteria and thereby providing a method for targeting limited (and perhaps soon to be scarce) care resources to optimise national, regional and local responses to the COVID-19 outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Michael Stedman; Mark Lunt; Mark Davies; Martin Gibson; Adrian Heald&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039024&#39;},
 {&#39;cord_uid&#39;: &#39;mzcajw8c&#39;,
  &#39;sha&#39;: &#39;b07bb8f6466cb60a903182960e1e57f5cdbc7780&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Population Attributable Fraction (PAF) of cases due to gatherings and groups with relevance to COVID-19 mitigation strategies&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039537&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Many countries have banned groups and gatherings as part of their response to the pandemic caused by the coronavirus, SARS-CoV-2. Although there are outbreak reports involving mass gatherings, the contribution to overall transmission is unknown. Methods We used data from a survey of social contact behaviour that specifically asked about contact with groups to estimate the Population Attributable Fraction (PAF) due to groups as the relative change in the Basic Reproduction Number when groups are prevented. Findings We estimate that PAF due to groups of 50+ people is 2.2% (95%CI 1.1%, 3.6%); the PAF due to groups of 20+ people is 6.4% (5.0%, 8.0%); the PAF due to groups of 10+ is 11.3% (9.9%, 13.0%) Interpretation Large groups of individuals have a small epidemiological impact; small and medium sized groups between 10 and 50 people have a larger impact on an epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Ellen Brooks-Pollock; Jonathan M Read; Thomas House; Graham Medley; Matt J Keeling; Leon Danon&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039537&#39;},
 {&#39;cord_uid&#39;: &#39;zsra2yz3&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Covid-19 health care demand and mortality in Sweden in response to non-pharmaceutical (NPIs) mitigation and suppression scenarios&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039594&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;While the outbreak in China is now contained, since mid March 2020, the epicentre of the COVID-19 pandemic is in Europe, with Italy reporting more deaths than China. Various European countries have seen an exponential growth in daily new cases, and unless strong reductions in transmission, the wave is expected to hit all of Europe by the end of March 2020. There is an urgent need to reduce transmission and thus reduce the growth rate of this epidemic, reduce the height of the epidemic peak and the peak demand on healthcare services, as well as reduce the total number of infected persons. Here we estimate the impact of COVID-19 on the Swedish population at the municipality level, considering demography and human mobility under scenarios of mitigation and suppression. We estimate the timelines of incidence, hospitalization rates, the intensive care (ICU) need, and the mortality in relation to the current Swedish ICU capacity, and costs of care. We further investigate the sensitivity to various degrees of potential underreporting on the estimates.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Joacim Rocklov&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039594&#39;},
 {&#39;cord_uid&#39;: &#39;fb0nwixw&#39;,
  &#39;sha&#39;: &#39;b787353f562f7017c8f3131062c73cd45b86bccf&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A Tempo-geographic Analysis of Global COVID-19 Epidemic Outside of China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039602&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: Understanding the global epidemic trends, geographic distribution, and transmission patterns of COVID-19 contribute to providing timely information for the global response of the epidemic. This study aims to understand the global pandemic geospatial patterns and trends and identify new epicenters requiring urgent attention. Methods: Data on COVID-19 between 31st Dec. 2019 and 8th Mar. 2020 was included. The epidemic trend was analyzed using joinpoint regressions; the growth of affected countries was by descriptive analysis; and the global distribution and transmission trend by spatial analysis. Findings: The number of new cases in the regions outside of China slowly increased before 24th Feb. and rapidly accelerated after 24th Feb. Compared to China, other affected countries experienced a longer duration of a slow increase at the early stage and rapid growth at the latter stages. The first apparent increase in the number of affected countries occurred from 23rd Jan to 1st Feb, where most of the first confirmed cases originated from China; the second apparent increase started from 25th Feb., where the majority originated from Italy and Iran. The geographic distribution changed from single-center (13th Jan.-20th Feb.) to multi-centers pattern (20th Feb.-14th Mar.). More countries were affected by COVID-19 and developed local transmission. Interpretation: The joinpoint regression and geospatial analysis indicated a multi-center pandemic of COVID-19. Strategies to prevent the new multiple centers as well as prevent ongoing transmission are needed.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Huipeng Liao; Gifty Marley; Yafei Si; Zaisheng Wang; Yewei Xie; Cheng Wang; Weiming Tang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039602&#39;},
 {&#39;cord_uid&#39;: &#39;ts3llerc&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effectiveness and cost-effectiveness of public health measures to control COVID-19: a modelling study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039644&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) was first reported in China, which caused a respiratory disease known as Coronavirus Disease 2019 (COVID-19). Since its discovery, the virus has spread to over 100 countries and claimed more than 4000 deaths. This study aimed to assess the effectiveness and cost-effectiveness of various response public health measures. Method The stochastic agent-based model was used to simulate the process of COVID-19 outbreak in scenario I (imported one case) and II (imported four cases) with a series of public health measures, involving the personal protection, isolation-and-quarantine, gathering restriction, and community containment. The virtual community was constructed following the susceptible-latent-infectious-recovered framework. The epidemiological and economic parameters derived from the previous literature and field investigation. The main outcomes included avoided infectors, cost-effectiveness ratios (CERs), and incremental cost-effectiveness ratios (ICERs). The sensitivity analyses were undertaken to assess uncertainty. Findings In scenario I and II, the isolation-and-quarantine averted 1696 and 1990 humans infected respectively at the cost of US$12 428 and US$58 555, both with negative value of ICERs. The joint strategy of personal protection and isolation-and-quarantine could avert one more case than single isolation-and-quarantine with additional cost of US$ 166 871 and US$180 140 respectively. The effectiveness of isolation-and-quarantine decreased as lowering quarantine probability and increasing delay-time. Especially in scenario II, when the quarantine probability was less than 25%, the number of infections raised sharply; when the quarantine delay-time reached six days, more than a quarter of individuals would be infected in the community. The strategy including community containment could protect more lives and was cost-effective, when the number of imported cases was no less than 65, or the delay-time of quarantine was more than five days, or the quarantine probability was below 25%, based on current assumptions. Interpretation The isolation-and-quarantine was the most cost-effective intervention. However, personal protection and isolation-and-quarantine was the optimal strategy averting more infectors than single isolation-and-quarantine. Certain restrictions should be considered, such as more initial imported cases, longer quarantine delay-time and lower quarantine probability.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Qiang Wang; Naiyang Shi; Jinxin Huang; Tingting Cui; Liuqing Yang; Jing Ai; Hong Ji; Ke Xu; Tauseef Ahmad; Changjun Bao; Hui Jin&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039644&#39;},
 {&#39;cord_uid&#39;: &#39;rxv5lgq8&#39;,
  &#39;sha&#39;: &#39;8d07b32de2cd946099966ca7e46c3e9ee9659069&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Modelling the Potential Health Impact of the COVID-19 Pandemic on a Hypothetical European Country&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039776&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;A SEIR simulation model for the COVID-19 pandemic was developed (http://covidsim.eu) and applied to a hypothetical European country of 10 million population. Our results show which interventions potentially push the epidemic peak into the subsequent year (when vaccinations may be available) or which fail. Different levels of control (via contact reduction) resulted in 22% to 63% of the population sick, 0.2% to 0.6% hospitalised, and 0.07% to 0.28% dead (n=6,450 to 28,228).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Nick Wilson; Lucy Telfar Barnard; Amanda Kvalsig; Ayesha Verrall; Michael G Baker; Markus Schwehm&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039776&#39;},
 {&#39;cord_uid&#39;: &#39;tv9xsned&#39;,
  &#39;sha&#39;: &#39;94e62a756a8f77d56f465194b69e260afad78eea&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Potential Factors for Prediction of Disease Severity of COVID-19 Patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039818&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Abstract: Objective: Coronavirus disease 2019 (COVID-19) is an escalating global epidemic caused by SARS-CoV-2, with a high mortality in critical patients. Effective indicators for predicting disease severity in SARS-CoV-2 infected patients are urgently needed. Methods: In this study, 43 COVID-19 patients admitted in Chongqing Public Health Medical Center were involved. Demographic data, clinical features, and laboratory examinations were obtained through electronic medical records. Peripheral blood specimens were collected from COVID-19 patients and examined for lymphocyte subsets and cytokine profiles by flow cytometry. Potential contributing factors for prediction of disease severity were further analyzed. Results: A total of 43 COVID-19 patients were included in this study, including 29 mild patients and 14 sever patients. Severe patients were significantly older (61.9+/-9.4 vs 44.4+/-15.9) and had higher incidence in co-infection with bacteria compared to mild group (85.7%vs27.6%). Significantly more severe patients had the clinical symptoms of anhelation (78.6%) and asthma (71.4%). For laboratory examination, 57.1% severe cases showed significant reduction in lymphocyte count. The levels of Interluekin-6 (IL6), IL10, erythrocyte sedimentation rate (ESR) and D-Dimer (D-D) were significantly higher in severe patients than mild patients, while the level of albumin (ALB) was remarkably lower in severe patients. Further analysis demonstrated that ESR, D-D, age, ALB and IL6 were the major contributing factors for distinguishing severe patients from mild patients. Moreover, ESR was identified as the most powerful factor to predict disease progression of COVID-19 patients. Conclusion: Age and the levels of ESR, D-D, ALB and IL6 are closely related to the disease severity of COVID-19 patients. ESR can be used as a valuable indicator for distinguishing severe COVID-19 patients in early stage, so as to increase the survival of severe patients. Keyword: COVID-19, erythrocyte sedimentation rate, cytokines, lymphocytes&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;huizheng zhang; xiaoying wang; zongqiang fu; ming luo; zhen zhang; ke zhang; ying he; dongyong wan; liwen zhang; jing wang; xiaofeng yan; mei han; yaokai chen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039818&#39;},
 {&#39;cord_uid&#39;: &#39;k1lg8c7q&#39;,
  &#39;sha&#39;: &#39;9e0f7bae6dd63db5ccf38bd90d3337c72eb7958a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Development and Evaluation of an AI System for COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039834&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Early detection of COVID-19 based on chest CT will enable timely treatment of patients and help control the spread of the disease. With rapid spreading of COVID-19 in many countries, however, CT volumes of suspicious patients are increasing at a speed much faster than the availability of human experts. Here, we propose an artificial intelligence (AI) system for fast COVID-19 diagnosis with an accuracy comparable to experienced radiologists. A large dataset was constructed by collecting 970 CT volumes of 496 patients with confirmed COVID-19 and 260 negative cases from three hospitals in Wuhan, China, and 1,125 negative cases from two publicly available chest CT datasets. Trained using only 312 cases, our diagnosis system, which is based on deep convolutional neural network, is able to achieve an accuracy of 94.98%, an area under the receiver operating characteristic curve (AUC) of 97.91%, a sensitivity of 94.06%, and a specificity of 95.47% on an independent external verification dataset of 1,255 cases. In a reader study involving five radiologists, only one radiologist is slightly more accurate than the AI system. The AI system is two orders of magnitude faster than radiologists and the code is available at https://github.com/ChenWWWeixiang/diagnosis_covid19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Cheng Jin; Weixiang Chen; Yukun Cao; Zhanwei Xu; Xin Zhang; Lei Deng; Chuansheng Zheng; Jie Zhou; Heshui Shi; Jianjiang Feng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039834&#39;},
 {&#39;cord_uid&#39;: &#39;7h1wg0oa&#39;,
  &#39;sha&#39;: &#39;c6d525a29a5c98edfac25678a34e2d91aab461c7&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;COVID-19 outbreak in Algeria: A mathematical model to predict the incidence&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039891&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction: Since December 29, 2019 a pandemic of new novel coronavirus-infected pneumonia named COVID-19 has started from Wuhan, China, has led to 254 996 confirmed cases until midday March 20, 2020. Sporadic cases have been imported worldwide, in Algeria, the first case reported on February 25, 2020 was imported from Italy, and then the epidemic has spread to other parts of the country very quickly with 90 confirmed cases until March 19, 2020. Methods: It is crucial to estimate the cases number growth in the early stages of the outbreak, to this end, we have implemented the Alg-COVID-19 Model which allows to predict the incidence and the reproduction number R0 in the coming months in order to help decision makers. The Alg-COVIS-19 Model initial equation 1, estimates the cases number at t prediction time using two parameters: the reproduction number R0 and the serial interval SI. Results: We found R0=2.56 based on actual incidence at the first 23 days, using the serial interval SI= 4,4 and the prediction time t=23. The herd immunity HI estimated is HI=61%. Also, The Covid-19 incidence predicted with the Alg-COVID-19 Model fits closely the actual incidence during the first 23 days of the epidemic in Algeria Fig. 1.A. which allows us to use it. According to Alg-COVID-19 Model, the number of cases will exceed 5000 on the 40th day ,April 5th, and it will double to 10000 on 43th day of the epidemic, April 8th, thus, exponential phase will begin ,Table 2; Fig.1.B, and increases continuously until reaching a herd immunity of 61% unless serious preventive measures are considered. Discussion: This model is valid only when the majority of the population is vulnerable to COVID-19 infection, however, it can be updated to fit the new parameters values.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Mohamed HAMIDOUCHE&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039891&#39;},
 {&#39;cord_uid&#39;: &#39;ptgpxpad&#39;,
  &#39;sha&#39;: &#39;4919d5781ff33b7e946a7b24bb0068c340709b7b&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Financing Vaccines for Global Health Security&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20039966&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Recent outbreaks of infectious pathogens such as Zika, Ebola, and COVID-19 have underscored the need for the dependable availability of vaccines against emerging infectious diseases (EIDs). The cost and risk of R&amp;D programs and uniquely unpredictable demand for EID vaccines have discouraged vaccine developers, and government and nonprofit agencies have been unable to provide timely or sufficient incentives for their development and sustained supply. We analyze the economic returns of a portfolio of EID vaccine assets, and find that under realistic financing assumptions, the expected returns are significantly negative, implying that the private sector is unlikely to address this need without public-sector intervention. We have sized the financing deficit for this portfolio and propose several potential solutions, including price increases, enhanced public-private partnerships, and subscription models through which individuals would pay annual fees to obtain access to a portfolio of vaccines in the event of an outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Jonathan Vu; Benjamin Kaplan; Shomesh Chaudhuri; Monique Mansoura; Andrew Lo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20039966&#39;},
 {&#39;cord_uid&#39;: &#39;suygiew5&#39;,
  &#39;sha&#39;: &#39;53d0f495085900746ae8492a12b1283c056e7308&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Window of Opportunity for Mitigation to Prevent Overflow of ICU capacity in Chicago by COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20040048&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We estimate the growth in demand for ICU beds in Chicago during the emerging COVID-19 epidemic, using state-of-the-art computer simulations calibrated for the SARS-CoV-2 virus. The questions we address are these: (1) Will the ICU capacity in Chicago be exceeded, and if so by how much? (2) Can strong mitigation strategies, such as lockdown or shelter in place order, prevent the overflow of capacity? (3) When should such strategies be implemented? Our answers are as follows: (1) The ICU capacity may be exceeded by a large amount, probably by a factor of ten. (2) Strong mitigation can avert this emergency situation potentially, but even that will not work if implemented too late. (3) If the strong mitigation precedes April 1st, then the growth of COVID-19 can be controlled and the ICU capacity could be adequate. The earlier the strong mitigation is implemented, the greater the probability that it will be successful. After around April 1 2020, any strong mitigation will not avert the emergency situation. In Italy, the lockdown occurred too late and the number of deaths is still doubling every 2.3 days. It is difficult to be sure about the precise dates for this window of opportunity, due to the inherent uncertainties in computer simulation. But there is high confidence in the main conclusion that it exists and will soon be closed. Our conclusion is that, being fully cognizant of the societal trade-offs, there is a rapidly closing window of opportunity to avert a worst-case scenario in Chicago, but only with strong mitigation/lockdown implemented in the next week at the latest. If this window is missed, the epidemic will get worse and then strong mitigation/lockdown will be required after all, but it will be too late.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Sergei Maslov; Nigel Goldenfeld&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20040048&#39;},
 {&#39;cord_uid&#39;: &#39;qqsefagq&#39;,
  &#39;sha&#39;: &#39;d71e87ca58f191863eb830b026910c9f4eeaa290&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The Effectiveness of Social Distancing in Mitigating COVID-19 Spread: a modelling analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.20040055&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The novel coronavirus COVID19 has been classified by the World Health Organisation as a pandemic due to its worldwide spread. The ability of countries to contain and control transmission is critical in the absence of a vaccine. We evaluated a range of social distancing measures to determine which strategies are most effective in reducing the peak daily infection rate, and consequential pressure on the health care system. Methods Using COVID19 transmission data from the outbreak source in Hubei Province, China, collected prior to activation of containment measures, we adapted an established individual based simulation model of the city of Newcastle, Australia, population 272,409. Simulation of virus transmission in this community model without interventions provided a baseline from which to compare alternative social distancing strategies. The infection history of each individual was determined, as was the time infected. From this model generated data, the rate of growth in cases, the magnitude of the epidemic peak, and the outbreak duration were obtained. Findings The application of all four social distancing interventions: school closure, workplace non-attendance, increased case isolation, and community contact reduction is highly effective in flattening the epidemic curve, reducing the maximum daily case numbers, and lengthening outbreak duration. These were also found to be effective even after 10 weeks delay from index case arrivals. The most effective single intervention was found to be increasing case isolation, to 100 percent of children and 90 percent of adults. Interpretation As strong social distancing intervention strategies had the most effect in reducing the epidemic peak, this strategy may be considered when weaker strategies are first tried and found to be less effective. Questions arise as to the duration of strong social distancing measures, given they are highly disruptive to society. Tradeoffs may need to be made between the effectiveness of social distancing strategies and population willingness to adhere to them.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;George J Milne; Simon Xie&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.20040055&#39;},
 {&#39;cord_uid&#39;: &#39;1cc9ig04&#39;,
  &#39;sha&#39;: &#39;2634ab4b2f262ef28c5cdb34759ff3cae6c6187f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Replication of SARS-CoV-2 in human respiratory epithelium&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.999029&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SARS-CoV-2 emerged by the end of 2019 to rapidly spread in 2020. At present, it is of utmost importance to understand the virus biology and to rapidly assess the potential of existing drugs and develop new active compounds. While some animal models for such studies are under development, most of the research is carried out in the Vero E6 cells. Here, we propose fully differentiated human airway epithelium cultures as a model for studies on the SARS-CoV-2. Further, we also provide basic characteristics of the system.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Aleksandra Milewska; Anna Kula-Pacurar; Jakub Wadas; Agnieszka Suder; Artur Szczepanski; Agnieszka Dabrowska; Katarzyna Owczarek; Marek Ochman; Tomasz Stacel; Zenon Rajfur; Pawel Piotr Labaj; Wojciech Branicki; Krzysztof Pyrc&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.999029&#39;},
 {&#39;cord_uid&#39;: &#39;5f95gve3&#39;,
  &#39;sha&#39;: &#39;72bec44d56da538afeabe871b4069ffe9b0906da&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.20.999730&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;COVID-19 is an emerging infectious disease and was recently declared as a pandemic by WHO. Currently, there is no vaccine or therapeutic available for this disease. Drug repositioning represents the only feasible option to address this global challenge and a panel of fifty FDA-approved drugs that have been pre-selected by an assay of SARS-CoV was screened to identify potential antiviral drug candidates against SARS-CoV-2 infection. We found a total of 24 drugs which exhibited antiviral efficacy (0.1 μM &lt; IC50 &lt; 10 μM) against SARS-CoV-2. In particular, two FDA-approved drugs – niclosamide and ciclesonide – were notable in some respects. These drugs will be tested in an appropriate animal model for their antiviral activities. In near future, these already FDA-approved drugs could be further developed following clinical trials in order to provide additional therapeutic options for patients with COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-20&#39;,
  &#39;authors&#39;: &#39;Sangeun Jeon; Meehyun Ko; Jihye Lee; Inhee Choi; Soo Young Byun; Soonju Park; David Shum; Seungtaek Kim&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.20.999730&#39;},
 {&#39;cord_uid&#39;: &#39;tjmx6msm&#39;,
  &#39;sha&#39;: &#39;58fd342d5c70641bd4e72aa822f0e532c0127ca3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Comparative analyses of SAR-CoV2 genomes from different geographical locations and other coronavirus family genomes reveals unique features potentially consequential to host-virus interaction and pathogenesis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.001586&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The ongoing pandemic of the coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV2). We have performed an integrated sequence-based analysis of SARS-CoV2 genomes from different geographical locations in order to identify its unique features absent in SARS-CoV and other related coronavirus family genomes, conferring unique infection, facilitation of transmission, virulence and immunogenic features to the virus. The phylogeny of the genomes yields some interesting results. Systematic gene level mutational analysis of the genomes has enabled us to identify several unique features of the SARS-CoV2 genome, which includes a unique mutation in the spike surface glycoprotein (A930V (24351C&gt;T)) in the Indian SARS-CoV2, absent in other strains studied here. We have also predicted the impact of the mutations in the spike glycoprotein function and stability, using computational approach. To gain further insights into host responses to viral infection, we predict that antiviral host-miRNAs may be controlling the viral pathogenesis. Our analysis reveals nine host miRNAs which can potentially target SARS-CoV2 genes. Interestingly, the nine miRNAs do not have targets in SARS and MERS genomes. Also, hsa-miR-27b is the only unique miRNA which has a target gene in the Indian SARS-CoV2 genome. We also predicted immune epitopes in the genomes.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Rahila Sardar; Deepshikha Satish; Shweta Birla; Dinesh Gupta&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.001586&#39;},
 {&#39;cord_uid&#39;: &#39;ydm77mhk&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.001628&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;An outbreak of a novel coronavirus, now named SARS-CoV-2, causing respiratory disease and a ~2% case fatality rate started in Wuhan, China in December 2019. Following unprecedented rapid global spread, the World Health Organization declared COVID-19 a pandemic on March 11, 2020. Although data on disease in humans are emerging at a steady pace, certain aspects of the pathogenesis of SARS-CoV-2 can only be studied in detail in animal models, where repeated sampling and tissue collection is possible. Here, we show that SARS-CoV-2 causes respiratory disease in infected rhesus macaques, with disease lasting 8-16 days. Pulmonary infiltrates, a hallmark of human disease, were visible in lung radiographs of all animals. High viral loads were detected in swabs from the nose and throat of all animals as well as in bronchoalveolar lavages; in one animal we observed prolonged rectal shedding. Taken together, the rhesus macaque recapitulates moderate disease observed in the majority of human cases. The establishment of the rhesus macaque as a model of COVID-19 will increase our understanding of the pathogenesis of this disease and will aid development and testing of medical countermeasures.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-21&#39;,
  &#39;authors&#39;: &#39;Vincent Munster; Friederike Feldmann; Brandi Williamson; Neeltje van Doremalen; Lizzette Perez-Perez,; Jonathan Schultz; Kimberly Meade-White; Atsushi Okumura; Julie Callison; Beniah Brumbaugh; Victoria Avanzato; Rebecca Rosenke; Patrick Hanley; Greg Saturday; Dana Scott; Elizabeth Fischer; Emmie de Wit&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.001628&#39;},
 {&#39;cord_uid&#39;: &#39;q0yqnlyl&#39;,
  &#39;sha&#39;: &#39;b3e877acb1b2ec719f6e094f7900cc3bf865457c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.001933&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stability of the viral spike protein and its union with human receptor hACE2. Next, to gain further insights into factors associated with coronaviruses recognition of the human host receptor, we performed modeling studies of five different coronaviruses and their potential binding to hACE2. Modeling results indicate that interfering with the salt bridges at hot spot 353 could be an effective strategy for inhibiting binding, and hence for the prevention of coronavirus infections. We also propose that a glycine residue at the receptor binding domain of the spike glycoprotein can have a critical role in permitting bat variants of the coronaviruses to infect human cells.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Vinicio Armijos-Jaramillo; Justin Yeager; Claire Muslin; Yunierkis Perez-Castillo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.001933&#39;},
 {&#39;cord_uid&#39;: &#39;81m48udk&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Characterizing occupations that cannot work from home: a means to identify susceptible worker groups during the COVID-19 pandemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20031336&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Introduction As the COVID-19 pandemic spreads globally, public health guidance is advising all workers to work from home. However, not all workers are employed in occupations which can be done from home. These workers are at an increased risk for exposure to SARS-CoV-2due to increased interaction with the public and other workers, and also potentially at an increased risk for job displacement as more extreme public health measures (such as closing of retail operations or enforcing shelter in place) occur. Methods To characterize the occupations least likely to be able to work from home, national employment and wage data maintained by the United States Bureau of Labor Statistics (BLS) was merged with measures from the BLS O*NET survey data, which ranks occupations by a variety of physical, ergonomic, psychosocial, and structural exposures. Noting that the work that could most easily be done at home would be work done on a computer that does not rely on interaction with the public, O*NET measures quantifying the importance of computer use at work, and the importance of working with or performing for the public at work were utilized. Results From this analysis, 19.5% (28.2 M) of the United States workforce covered by O*NET are employed in occupations where working from home would be difficult, due to minimal use of computer at work, and a high level of interaction with the public. These workers tend to be in service occupations, including retail and food service, protective service occupations, and transportation occupations such as bus drivers. About 25% (35.6 M) of the United States workforce covered by O*NET are employed in occupations where working from home could likely be more easily accommodated, as these workers do much of their work on computers and interaction with the public is not important. These workers tend to be in technology, computer, management, administrative, financial, engineering, and some science occupations, which typically have higher pay than occupations which cannot be done at home. Conclusions The workers in occupations that have minimal computer use, and high interaction with the public are least likely to be able to work from home during a public health emergency. These workers could also be at an increased risk for job displacement if public-facing establishments close or alter their business model in the face of increased public health restrictions. Occupations where working from home is not possible tend to have lower annual median incomes than occupations where working from home is possible, increasing the vulnerability of these workers. Characterizing which occupational groups are least likely to be able to work from home can inform public health risk management and prioritize occupational sectors where additional workplace protections are necessary.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Marissa G. Baker&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20031336&#39;},
 {&#39;cord_uid&#39;: &#39;njo93g3h&#39;,
  &#39;sha&#39;: &#39;c7dc07727f9f0a09b060e1a3ca8acc3ee7d82bda&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Infection Control of 2019 Novel Corona Virus Disease (COVID-19) in Cancer Patients undergoing Radiotherapy in Wuhan&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20037051&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background A pandemic of 2019 novel corona virus disease (COVID-19), which was first reported in Wuhan city, has affected more than 100,000 patients worldwide. Patients with cancer are at a higher risk of COVID-19, but currently, there is no guidance on the management of cancer patients during this outbreak. Here, we report the infection control measures and early outcomes of patients who received radiotherapy (RT) at a tertiary cancer centre in Wuhan. Methods We reviewed all patients who were treated at the Zhongnan Hospital of Wuhan University (ZHWU) from Jan 20 to Mar 6, 2020. This preceded the city lock-down date of Jan 23, 2020. Infection control measures were implemented, which included a clinical pathway for managing suspect COVID-19 cases, on-site screening, modifications to the RT facility, and protection of healthcare workers. Primary end-point was infection rate among patients and healthcare staff. Diagnosis of COVID-19 was based on the 5th edition criteria. Findings 209 patients completed RT during the study period. Median age was 55 y (IQR = 48-64). Thoracic, head and neck, and lower gastrointestinal and gynaecological cancer patients consisted the majority of patients. Treatment sites included thoracic (38.3%), head and neck (25.4%), and abdomen and pelvis (25.8%); 47.4%, 27.3%, and 25.4% of treatments were for adjuvant, radical, and palliative indications, respectively. 188 treatments/day were performed prior to the lock-down, in contrast to 12.4 treatments/day post-lock-down. Only one (0.48%) patient was diagnosed with COVID-19 during the study period. No healthcare worker was infected. Interpretation Herein, we show that in a susceptible population to COVID-19, strict infection control measures can curb human-to-human transmission, and ensure timely delivery of RT to cancer patients.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Conghua Xie; Xiaoyong Wang; Hui Liu; Zhirong Bao; Jing Yu; Yahua Zhong; Melvin LK Chua&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20037051&#39;},
 {&#39;cord_uid&#39;: &#39;z7r4n0w7&#39;,
  &#39;sha&#39;: &#39;1ec9d8c7afcab0fb0478c33b3b6026356f6e0cad&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Effect of SARS-CoV-2 infection upon male gonadal function: A single center-based study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20037267&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Since SARS-CoV-2 infection was first identified in December 2019, it spread rapidly and a global pandemic of COVID-19 has occurred. ACE2, the receptor for entry into the target cells by SARS-CoV-2, was found to abundantly express in testes, including spermatogonia, Leydig and Sertoli cells. However, there is no clinical evidence about whether SARS-CoV-2 infection can affect male gonadal function so far. In this study, we compared the sex-related hormones between 81 reproductive-aged men with SARS-CoV-2 infection and 100 age-matched healthy men, and found that serum luteinizing hormone (LH) was significantly increased, but the ratio of testosterone (T) to LH and the ratio of follicle stimulating hormone (FSH) to LH were dramatically decreased in males with COVID-19. Besides, multivariable regression analysis indicated that c-reactive protein (CRP) level was significantly associated with serum T:LH ratio in COVID-19 patients. This study provides the first direct evidence about the influence of medical condition of COVID-19 on male sex hormones, alerting more attention to gonadal function evaluation among patients recovered from SARS-CoV-2 infection, especially the reproductive-aged men.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Ling Ma; Wen Xie; Danyang Li; Lei Shi; Yanhong Mao; Yao Xiong; Yuanzhen Zhang; Ming Zhang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20037267&#39;},
 {&#39;cord_uid&#39;: &#39;hix00vyd&#39;,
  &#39;sha&#39;: &#39;939cef57fa345bfdd5e27cecf90684dacb26273c&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Clinical characteristics and reasons of different duration from onset to release from quarantine for patients with COVID-19 Outside Hubei province, China.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20038778&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objective: To find out more characteristics and rules of COVID-19 by analyzing the clinical course of COVID-19 patients in a region outside Hubei province. Methods: 37 cases diagnosed adult COVID-19 cases of general characteristics, history of epidemiology, chronic underlying diseases,clinical symptoms and complications, chest CT, biochemical monitoring, severity assessment, treatment and outcome were retrospectively analyzed, and according to the duration from onset to release from quarantine were divided into 20-day group and &gt; 20-day group, compare the similarities and differences between the two groups. Results: Among the 37 patients, 5 were mild, 30 were moderate, 1 was severe and 1 was critical. All the patients were released from quarantine without death. The average duration from onset to release from quarantine was 20.2±6.6 days, The average length of stay from onset to hospitalization was 4.1±3.7 days, and hospitalization duration average 16.1± 6.2 days. The average age was 44.3±1.67 years. 78.4% of cases were caused by exposure to a confirmed patient or the workplace of a confirmed patient. The main symptoms were cough (67.6%), fever( 62.2%), shortness of breath (32.4%), fatigue (24.3%), sore throat (21.6%,) vomiting or diarrhea (21.6%). The white blood cell count was decreased in 27.0% of the patients, and the lymphocyte count was decreased in 62.2% of the patients, of which 43.5% patients were &lt;0.6 × 109/L. On admission, 86.5% of patients with chest CT showed pneumonia, including some asymptomatic patients. 68.8% of patients showed bilateral infiltration. In the &gt; 20-day group, the average age was 49.9±1.38 years old, and the duration from onset to hospitalization was 5.5±3.9 days. Compared with the ≤20-day group, the age was older and and duration was longer, P &lt; 0.05. All the 7 asymptomatic patients were ≤20-day group. When 37 patients were released from quarantine, the white blood cell count of 16.2% patients was &lt;4.0 × 109/L, and the lymphocyte count of 59.5% patients was &lt;1.1×109/L, and the absolute count of white blood cells and lymphocytes was 5.02 ± 1.34×109/L and 1.03± 0.34×109/L respectively , compared with those on admission, P &gt;0.05. Conclusion: The majority of COVID-19 cases in the study area were mild and moderate, with good clinical outcomes. There were some special characteristics in the clinical process. The reasons of duration from onset to release from quarantine were complex. There was no significant change in the number of granulocytes at the time of release from quarantine compared to the time of admission.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Suochen Tian; Zhenqin Chang; Yunxia Wang; Min Wu; Wenming Zhang; Guijie Zhou; Xiuli Zou; Hui Tian; Tingfang Xiao; Junmin Xing; Juan Chen; Jian Han; Kang Ning; Tiejun Wu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20038778&#39;},
 {&#39;cord_uid&#39;: &#39;zknngzt2&#39;,
  &#39;sha&#39;: &#39;72b41bbcaed1ccbe1e6a223f9e9b89a85ad5d4dd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Optimization of Microbiological Laboratory Detection Strategy for Patients in A Designated Hospital Treating Novel Coronavirus Pneumonia in Anhui Province&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20039065&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Novel coronavirus pneumonia (NCP) is an emerging, highly contagious community acquired pneumonia (CAP) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Highly efficient and accurate microbiological laboratory assay is essential to confirm the SARS-CoV-2 infection, rule out other pathogens that can cause CAP, and monitor secondary infections. Here, we enrolled and provide microbiological analysis for 129 suspected and 52 transferred confirmed NCP patients hospitalized in the First Affiliated Hospital of University of Science and Technology of China (USTC) from Jan 21 to Feb 29, 2020. By analyzing the dual swab samples (sputum and pharyngeal) from 129 suspected patients with realtime RT-PCR, we confirmed 33 SARS-CoV-2 infections, with two co-infection cases with adenovirus or rhinovirus. We also used multiplex PCR to detect 13 common respiratory tract pathogens in 96 non-NCP patients, and found that 30 patients (31.25%) were infected with at least one respiratory tract pathogen that may cause CAP. Further, we performed bacterial and fungal cultures as well as fungal serologic tests and found that there is no secondary bacterial/fungal infections in confirmed NCP patients. Our studies suggest that, during the epidemic of NCP in Anhui province, there was a certain proportion of infection and co-infection of other common pathogens of CAP, and the secondary bacterial and fungal infection is not detectable in NCP patients. In comparison with SARS-CoV-2 detection alone, this optimized strategy combining multiple pathogen detection for identification of NCP and other CAP patients as well as cultures and serologic tests for confirmed patients increases the diagnosis efficiency and facilitates the personalized medication.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Wenjiao Chang; Yuru Shi; Yingjie Qi; Jiaxing Liu; Ting Liu; Zhaowu Chen; Wenjing Zhang; Mengmeng Wang; Dongfeng Liu; Ming Yin; Jing Xu; Yun Yang; Xiaowu Zhu; Jing Ge; Shu Zhu; Yong Gao; Xiaoling Ma&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20039065&#39;},
 {&#39;cord_uid&#39;: &#39;1y0pl31i&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A New, Simple Projection Model for COVID-19 Pandemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20039867&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;With the worldwide outbreak of COVID-19, an accurate model to predict how the coronavirus pandemic will evolve becomes important and urgent to help policy makers in different countries address the epidemic outbreak and determine policies to control spread more efficiently and effectively. Unlike the classic public health and virus propagation models, this new projection model takes government intervention and public response into account to make reliable projections of the outbreak 10 days to 2 weeks in advance.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Jian Lu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20039867&#39;},
 {&#39;cord_uid&#39;: &#39;knt1f78p&#39;,
  &#39;sha&#39;: &#39;5b25c63b89586e93a09b5e213a20cfd9b0418b60&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A mathematical model for the spatiotemporal epidemic spreading of COVID19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040022&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;An outbreak of a novel coronavirus, named SARS-CoV-2, that provokes the COVID-19 disease, was first reported in Hubei, mainland China on 31 December 2019. As of 20 March 2020, cases have been reported in 166 countries/regions, including cases of human-to-human transmission around the world. The proportions of this epidemics is probably one of the largest challenges faced by our interconnected modern societies. According to the current epidemiological reports, the large basic reproduction number, R_0 ~ 2.3, number of secondary cases produced by an infected individual in a population of susceptible individuals, as well as an asymptomatic period (up to 14 days) in which infectious individuals are undetectable without further analysis, pave the way for a major crisis of the national health capacity systems. Recent scientific reports have pointed out that the detected cases of COVID19 at young ages is strikingly short and that lethality is concentrated at large ages. Here we adapt a Microscopic Markov Chain Approach (MMCA) metapopulation mobility model to capture the spread of COVID-19. We propose a model that stratifies the population by ages, and account for the different incidences of the disease at each strata. The model is used to predict the incidence of the epidemics in a spatial population through time, permitting investigation of control measures. The model is applied to the current epidemic in Spain, using the estimates of the epidemiological parameters and the mobility and demographic census data of the national institute of statistics (INE). The results indicate that the peak of incidence will happen in the first half of April 2020 in absence of mobility restrictions. These results can be refined with improved estimates of epidemiological parameters, and can be adapted to precise mobility restrictions at the level of municipalities. The current estimates largely compromises the Spanish health capacity system, in particular that for intensive care units, from the end of March. However, the model allows for the scrutiny of containment measures that can be used for health authorities to forecast with accuracy their impact in prevalence of COVID--19. Here we show by testing different epidemic containment scenarios that we urge to enforce total lockdown to avoid a massive collapse of the Spanish national health system.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Alex Arenas; Wesley Cota; Jesus Gomez-Gardenes; Sergio Gómez; Clara Granell; Joan T. Matamalas; David Soriano-Panos; Benjamin Steinegger&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040022&#39;},
 {&#39;cord_uid&#39;: &#39;zblitbo0&#39;,
  &#39;sha&#39;: &#39;1fe8e5f5709aa0e7fb3971ce721c27b879020e6e&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Myocardial injury is associated with in-hospital mortality of confirmed or suspected COVID-19 in Wuhan, China: A single center retrospective cohort study&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040121&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;[Background] Since December 2019, a cluster of coronavirus disease 2019 (COVID-19) occurred in Wuhan, Hubei Province, China and spread rapidly from China to other countries. In-hospital mortality are high in severe cases and cardiac injury characterized by elevated cardiac troponin are common among them. The mechanism of cardiac injury and the relationship between cardiac injury and in-hospital mortality remained unclear. Studies focused on cardiac injury in COVID-19 patients are scarce. [Objectives] To investigate the association between cardiac injury and in-hospital mortality of patients with confirmed or suspected COVID-19. [Methods] Demographic, clinical, treatment, and laboratory data of consecutive confirmed or suspected COVID-19 patients admitted in Wuhan No.1 Hospital from 25th December, 2019 to 15th February, 2020 were extracted from electronic medical records and were retrospectively reviewed and analyzed. Univariate and multivariate Cox regression analysis were used to explore the risk factors associated with in-hospital death. [Results] A total of 110 patients with confirmed (n=80) or suspected (n=30) COVID-19 were screened and 48 patients (female 31.3%, mean age 70.58±13.38 year old) among them with high-sensitivity cardiac troponin I (hs-cTnI) test within 48 hours after admission were included, of whom 17 (17/48, 35.4%) died in hospital while 31 (31/48, 64.6%) were discharged or transferred to other hospital. High-sensitivity cardiac troponin I was levated in 13 (13/48, 27.1%) patents. Multivariate Cox regression analysis showed pulse oximetry of oxygen saturation (SpO2) on admission (HR 0.704, 95% CI 0.546-0.909, per 1% decrease, p=0.007), elevated hs-cTnI (HR 10.902, 95% 1.279-92.927, p=0.029) and elevated d-dimer (HR 1.103, 95%CI 1.034-1.176, per 1mg/L increase, p=0.003) on admission were independently associated with in-hospital mortality. [Conclusions] Cardiac injury defined by hs-cTnI elevation and elevated d-dimer on admission were risk factors for in-hospital death, while higher SpO2 could be seen as a protective factor, which could help clinicians to identify patients with adverse outcome at the early stage of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Fan Zhang; Deyan Yang; Jing Li; Peng Gao; Taibo Chen; Zhongwei Cheng; Kangan Cheng; Quan Fang; Wan Pan; Chunfeng Yi; Hongru Fan; Yonghong Wu; Liwei Li; Yong Fang; Juan Liu; Guowei Tian; Liqun He&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040121&#39;},
 {&#39;cord_uid&#39;: &#39;fyh8gjjl&#39;,
  &#39;sha&#39;: &#39;ccbd77429a76452199ca99e5c436bc398578f78d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Tracking and forecasting milepost moments of the epidemic in the early-outbreak: framework and applications to the COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040139&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background. The outbreak of the 2019 novel coronavirus (COVID-19) has attracted global attention. In the early stage of the outbreak, the most important question concerns some meaningful milepost moments, including (1) the time when the number of daily confirmed cases decreases, (2) the time when the number of daily confirmed cases becomes smaller than that of the daily removed (recovered and death), (3) the time when the number of daily confirmed cases becomes zero, and (4) the time when the number of patients treated in hospital is zero, which indicates the end of the epidemic. Intuitively, the former two can be regarded as two important turning points which indicate the alleviation of epidemic to some extent, while the latter two as two ``zero&#34; points, respectively. Unfortunately, it is extremely difficult to make right and precise prediction due to the limited amount of available data at a early stage of the outbreak. Method. To address it, in this paper, we propose a flexible framework incorporating the effectiveness of the government control to forecast the whole process of a new unknown infectious disease in its early-outbreak. Specially, we first establish the iconic indicators to characterize the extent of epidemic spread, yielding four periods of the whole process corresponding to the four meaningful milepost moments: two turning points and two ``zero&#34; points. Then we develop the tracking and forecasting procedure with mild and reasonable assumption. Finally we apply it to analyze and evaluate the COVID-19 using the public available data for mainland China beyond Hubei Province from the China Centers for Disease Control (CDC) during the period of Jan 29th, 2020, to Feb 29th, 2020, which shows the effectiveness of the proposed procedure. Results. Results show that our model can clearly outline the development of the epidemic at a very early stage. The first prediction results on Jan 29th reveal that the first and second milepost moments for mainland China beyond Hubei Province would appear on Jan 31st and Feb 14th respectively, which are only one day and three days behind the real world situations. Forecasting results indicate that the number of newly confirmed cases will become zero in the mid-late March, and the number of patients treated in the hospital will become zero between mid-March and mid-April in mainland China beyond Hubei Province. The framework proposed in this paper can help people get a general understanding of the epidemic trends in counties where COVID-19 are raging as well as any other outbreaks of new and unknown infectious diseases in the future.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Huiwen Wang; Yanwen Zhang; Shan Lu; Shanshan Wang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040139&#39;},
 {&#39;cord_uid&#39;: &#39;9zeqigqa&#39;,
  &#39;sha&#39;: &#39;8213acea5f1a592dae49adae07380fea8203910d&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Predicting the number of reported and unreported cases for the COVID-19 epidemic in South Korea, Italy, France and Germany&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040154&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We model the COVID-19 coronavirus epidemic in South Korea, Italy, France, and Germany. We use early reported case data to predict the cumulative number of reported cases to a final size. The key features of our model are the timing of implementation of major public policies restricting social movement, the identification and isolation of unreported cases, and the impact of asymptomatic infectious cases.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;pierre magal; Glenn Webb&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040154&#39;},
 {&#39;cord_uid&#39;: &#39;vh9wreex&#39;,
  &#39;sha&#39;: &#39;7b78040738d551e30fc2b2b996cf4606c8739d62&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Initial Simulation of SARS-CoV2 Spread and Intervention Effects in the Continental US&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040303&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Here we use a metapopulation model applied at county resolution to simulate the spread and growth of COVID-19 incidence in the continental United States. We calibrate the model against county-level incidence data collected between February 21, 2020 and March 13, 2020, and estimate epidemiological parameters including the fraction of undocumented infections and their contagiousness1. Using the calibrated model, we project the outbreak in the continental US for 180 days after March 13, 2020, and evaluate the effects of social distancing and travel restrictions on the outbreak.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Jeffrey Shaman; Sen Pei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040303&#39;},
 {&#39;cord_uid&#39;: &#39;56zhxd6e&#39;,
  &#39;sha&#39;: &#39;1f80f32ab88715153f9690b096115e53f2fd6121&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040329&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background The outbreak of coronavirus disease 2019 (COVID-19) has been declared a pandemic by the World Health Organization, while several key epidemiological parameters of the disease remain to be clarified. This study aimed to obtain robust estimates of the incubation period, upper limit of latent period (interval between exposure of infector and infectee), serial interval, time point of exposure and basic reproduction number (R0) of COVID-19. Methods Between late February and early March of 2020, the individual data of laboratory confirmed cases of COVID-19 were retrieved from 10728 publicly available reports released by the health authorities of and outside China and from 1790 publications identified in PubMed and CNKI. To be eligible, a report had to contain the data that allowed for estimation of at least one parameter. As relevant data mainly came from clustering cases, the clusters for which no evidence was available to establish transmission order were all excluded to ensure accuracy of estimates. Additionally, only the cases with an exposure period spanning 3 days or less were included in the estimation of parameters involving exposure date, and a simple method for determining exposure date was adopted to ensure the error of estimates be small (&lt; 0.3 day). Depending on specific parameters, three or four of normal, lognormal, Weibull, and gamma distributions were fitted to the datasets and the results from appropriate models were presented. Findings In total, 1155 cases from China, Japan, Singapore, South Korea, Vietnam, Germany and Malaysia were included for the final analysis. The mean and standard deviation were 7.44 days and 4.39 days for incubation period, 2.52 days and 3.95 days for the upper limit of latent period, 6.70 days and 5.20 days for serial interval, and -0.19 day (i.e., 0.19 day before symptom onset of infector) and 3.32 days for time point of exposure. R0 was estimated to be 1.70 and 1.78 based on two different formulas. For 39 (6.64%) cases, the incubation periods were longer than 14 days. In 102 (43.78%) infector-infectee pairs, transmission occurred before the symptom onsets of infectors. In 27 (3.92%) infector-infectee pairs, symptom onsets of infectees occurred before those of infectors. Stratified analysis showed that incubation period and serial interval were consistently longer for those with less severe disease and for those whose primary cases had less severe disease. Asymptomatic transmission was also observed. Interpretation This study obtained robust estimates of several key epidemiological parameters of COVID-19. The findings support current practice of 14-day quarantine of persons with potential exposure, but also suggest that longer monitoring periods might be needed for selected groups. The estimates of serial interval, time point of exposure and latent period provide consistent evidence on pre-symptomatic transmission. This together with asymptomatic transmission and the generally longer incubation and serial interval of less severe cases suggests a high risk of long-term epidemic in the absence of appropriate control measures.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040329&#39;},
 {&#39;cord_uid&#39;: &#39;m76ic66s&#39;,
  &#39;sha&#39;: &#39;f5ad0b5495f2e970edb9d50944cf42543d8739f0&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Prognostic value of C-reactive protein in patients with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040360&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Elevated serum C-reactive protein (CRP) level was observed in most patients with COVID-19. Methods Data of COVID-19 patients with clinical outcome in a designated hospital in Wuhan, China, were retrospectively collected and analyzed from Jan 30 to Feb 20, 2020. The prognostic value of admission CRP was evaluated in patients with COVID-19. Results Out of 298 patients enrolled, 84 died and 214 recovered. Most non-survivors tended to be males, old aged, or with chronic diseases. Compared to survivors, non-survivors showed significantly higher white blood cell and neutrophil count, neutrophil to lymphocyte ratio (NLR), systemic immune-inflammation index (SII, defined by platelet count multiply by NLR), CRP, procalcitonin, and D-dimer, and lower red blood cell, lymphocyte, and platelet count. Age, neutrophil count, platelet count, and CRP were identified as independent predictors of adverse outcome. The area under the receiver operating characteristic (ROC) curve (AUC) of CRP (0.896) was significantly higher than that of age (0.833), neutrophil count (0.820), and platelet count (0.678) in outcome prediction (all p&lt;0.05). With a cut-off value of 41.4, CRP exhibited sensitivity 90.5%, specificity 77.6%, positive predictive value 61.3%, and negative predictive value 95.4%. Subgroup analysis revealed that CRP remained robust accuracy in adverse outcome prediction in patients with different disease severity (AUC 0.832, z=10.23, p&lt;0.001; AUC 0.989, z=44.04, p&lt;0.001). CRP was also an independent discriminator of severe/critical illness on admission (AUC 0.783, z=10.69, p&lt;0.001). Conclusions In patients with COVID-19, admission CRP correlated with disease severity and tended to be a good predictor of adverse outcome.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Xiaomin Luo; Wei Zhou; Xiaojie Yan; Tangxi Guo; Benchao Wang; Hongxia Xia; Lu Ye; Jun Xiong; Zongping Jiang; Yu Liu; Bicheng Zhang; Weize Yang&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040360&#39;},
 {&#39;cord_uid&#39;: &#39;udtj0lom&#39;,
  &#39;sha&#39;: &#39;6667f1bac8808c7b26f93f2ae41bec81b2fb6e76&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;From a single host to global spread. The global mobility based modelling of the COVID-19 pandemic implies higher infection and lower detection rates than current estimates.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040444&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Since the outbreak of the COVID-19 pandemic, multiple efforts of modelling of the geo-temporal transmissibility of the virus have been undertaken, but none succeeded in describing the pandemic at the global level. We propose a set of parameters for the first COVID-19 Global Epidemic and Mobility Model (GLEaM). The simulation starting with just a single pre-symptomatic, yet infectious, case in Wuhan, China, results in an accurate prediction of the number of diagnosed cases after 125 days in multiple countries across three continents. Methods We have built a modified SIR model and parameterized it analytically, according to the literature and by fitting the missing parameters to the observed dynamics of the virus spread. We compared our results with the number of diagnosed cases in sixeight selected countries which provide reliable statistics but differ substantially in terms of strength and speed of undertaken precautions. The obtained 95% confidence intervals for the predictions fit well to the empirical data. Findings The parameters that successfully model the pandemic are: the basic reproduction number R0, ~4.4; a latent non-infectious period of 1.1. days followed by 4.6 days of the presymptomatic infectious period; the probability of developing severe symptoms, 0.01; the probability of being diagnosed when presenting severe symptoms of 0.6; the probability of diagnosis for cases with mild symptoms or asymptomatic, 0.001. Also, the higher the testing rate per country, the lower the discrepancy between data (diagnosed cases) and model. Interpretation Parameters that successfully reproduce the observed number of cases indicate that both R0 and the prevalence of the virus might be underestimated. This is in concordance with the newest research on undocumented COVID-19 cases. Consequently, the actual mortality rate is putatively lower than estimated. Confirmation of the pandemic characteristic by further refinement of the model and screening tests is crucial for developing an effective strategy for the global epidemiological crisis.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Marlena M Siwiak; Pawel Szczesny; Marian P Siwiak&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040444&#39;},
 {&#39;cord_uid&#39;: &#39;6g34qwer&#39;,
  &#39;sha&#39;: &#39;d1c0a1bfbc2785ecfaac415612d354f2ffc3bfc5&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.21.20040691&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background: SARS-CoV-2 is a novel human coronavirus, there is no specific antiviral drugs. It has been proved that host-cell-expressed CD147 could bind spike protein of SARS-CoV-2 and involve in host cell invasion. Antibody against CD147 could block the infection of SARS-CoV-2. We aimed to assess the efficacy and safety of meplazumab, a humanized anti-CD147 antibody, as add-on therapy in patients with COVID-19 pneumonia. Methods: All patients received recommended strategy from Diagnosis and Treatment for 2019 Novel Coronavirus Diseases released by National Health Commission of China. Eligible patients were add-on administered 10 mg meplazumab intravenously at days 1, 2, and 5. Patients hospitalized in the same period were observed as concurrent control. The endpoints include virological clearance rate, case severity, chest radiographic, and laboratory test. This trial was approved by the Ethics Committee of Institution at the Tangdu hospital, and registered with ClinicalTrials.gov, NCT 04275245. Findings:17 patients were enrolled and assigned to meplazumab group between Feb 3, 2020 and Feb 10, 2020. 11 hospitalized patients served as concurrent control. Baseline characteristics were generally balanced across two groups. Compared to control group, meplazumab treatment significantly improved the discharged (p=0.006) and case severity (p=0.021) in critical and severe patients. The time to virus negative in meplazumab group was reduced than that in control group (median 3, 95%CI[1.5-4.5] vs. 13, [6.5-19.5]; p=0.014, HR=0.37, 95%CI[0.155-0.833]). The percentages of patients recovered to the normal lymphocyte count and CRP concentration were also increased remarkably and rapidly in meplazumab group. No adverse effect was found in meplazumab-treated patients. Interpretation:Meplazumab efficiently improved the recovery of patients with SARS-CoV-2 pneumonia with a favorable safety profile. Our results support to carry out a large-scale investigation of meplazumab as a treatment for COVID-19 pneumonia. Funding:National Science and Technology Major Project.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Huijie Bian; Zhao-Hui Zheng; Ding Wei; Zheng Zhang; Wen-Zhen Kang; Chun-Qiu Hao; Ke Dong; Wen Kang; Jie-Lai Xia; Jin-Lin Miao; Rong-Hua Xie; Bin Wang; Xiu-Xuan Sun; Xiang-Min Yang; Peng Lin; Jie-Jie Geng; Ke Wang; Hong-Yong Cui; Kui Zhang; Xiao-Chun Chen; Hao Tang; Hong Du; Na Yao; Shuang-Shuang Liu; Lin-Na Liu; Zhe Zhang; Zhao-Wei Gao; Gang Nan; Qing-Yi Wang; Jian-Qi Lian; Zhi-Nan Chen; Ping Zhu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.21.20040691&#39;},
 {&#39;cord_uid&#39;: &#39;x3472dvy&#39;,
  &#39;sha&#39;: &#39;14165ad017cc144d15168bc5c985d05b5be2d019&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Using directed attenuation to enhance vaccine immunity&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.002188&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;Many viral infections can be prevented by immunizing with live, attenuated vaccines. Early methods of attenuation were hit-and-miss, and these are now much improved by genetic engineering. But even current attenuation methods operate on the principle of genetic harm, reducing the virus&#39;s ability to grow, which in turn limits the host immune response below that of infection by wild-type. We use mathematical models of the dynamics of virus and its control by innate and adaptive immunity to explore the tradeoff between attenuation of virus growth and the generation of immunity. Our analysis suggests that directed attenuation that disables key viral defenses against the host immune responses may attenuate viral growth without compromising, and potentially even enhancing the generation of immunity. We explore which immune evasion pathways should be attenuated and how attenuating multiple pathways could lead to robust attenuation of pathology with enhancement of immunity.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Rustom Antia; Hasan Ahmed; James J Bull&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.002188&#39;},
 {&#39;cord_uid&#39;: &#39;txuthqgo&#39;,
  &#39;sha&#39;: &#39;9444dbf96374f2efc528fb4ede46ecf41a2b69be&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.002204&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Direct RNA sequencing using an Oxford Nanopore MinION characterised the transcriptome of SARS-CoV-2 grown in Vero E6 cells. This cell line is being widely used to propagate the novel coronavirus. The viral transcriptome was analysed using a recently developed ORF-centric pipeline. This revealed the pattern of viral transcripts, (i.e. subgenomic mRNAs), generally fitted the predicted replication and transcription model for coronaviruses. A 24 nt in-frame deletion was detected in subgenomic mRNAs encoding the spike (S) glycoprotein. This feature was identified in over half of the mapped transcripts and was predicted to remove a proposed furin cleavage site from the S glycoprotein. This motif directs cleavage of the S glycoprotein into functional subunits during virus entry or exit. Cleavage of the S glycoprotein can be a barrier to zoonotic coronavirus transmission and affect viral pathogenicity. Allied to this transcriptome analysis, tandem mass spectrometry was used to identify over 500 viral peptides and 44 phosphopeptides, covering almost all of the proteins predicted to be encoded by the SARS-CoV-2 genome, including peptides unique to the deleted variant of the S glycoprotein. Detection of an apparently viable deletion in the furin cleavage site of the S glycoprotein reinforces the point that this and other regions of SARS-CoV-2 proteins may readily mutate. This is of clear significance given the interest in the S glycoprotein as a potential vaccine target and the observation that the furin cleavage site likely contributes strongly to the pathogenesis and zoonosis of this virus. The viral genome sequence should be carefully monitored during the growth of viral stocks for research, animal challenge models and, potentially, in clinical samples. Such variations may result in different levels of virulence and morbidity and mortality.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Andrew D. Davidson; Maia Kavangh Williamson; Sebastian Lewis; Deborah Shoemark; Miles W Carroll; Kate Heesom; Maria Zambon; Joanna Ellis; Phillip A Lewis; Julian A Hiscox; David A Matthews&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.002204&#39;},
 {&#39;cord_uid&#39;: &#39;38d6gb7o&#39;,
  &#39;sha&#39;: &#39;b831e5c64437bbd350556f97f5f1ae4225bc3f1f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.002386&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 290,000 people since the end of 2019, killed over 12,000, and caused worldwide social and economic disruption. There are currently no antiviral drugs with proven efficacy nor are there vaccines for its prevention. Unfortunately, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To illuminate this, we cloned, tagged and expressed 26 of the 29 viral proteins in human cells and identified the human proteins physically associated with each using affinity-purification mass spectrometry (AP-MS), which identified 332 high confidence SARS-CoV-2-human protein-protein interactions (PPIs). Among these, we identify 67 druggable human proteins or host factors targeted by 69 existing FDA-approved drugs, drugs in clinical trials and/or preclinical compounds, that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral therapeutics against SARS-CoV-2 and other deadly coronavirus strains.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-22&#39;,
  &#39;authors&#39;: &#34;David E. Gordon; Gwendolyn M. Jang; Mehdi Bouhaddou; Jiewei Xu; Kirsten Obernier; Matthew J O&#39;Meara; Jeffrey Z. Guo; Danielle L. Swaney; Tia A. Tummino; Ruth Huttenhain; Robyn Kaake; Alicia L. Richards; Beril Tutuncuoglu; Helene Foussard; Jyoti Batra; Kelsey Haas; Maya Modak; Minkyu Kim; Paige Haas; Benjamin J. Polacco; Hannes Braberg; Jacqueline M. Fabius; Manon Eckhardt; Margaret Soucheray; Melanie Brewer; Merve Cakir; Michael J. McGregor; Qiongyu Li; Zun Zar Chi Naing; Yuan Zhou; Shiming Peng; Ilsa T. Kirby; James E. Melnyk; John S Chorba; Kevin Lou; Shizhong A. Dai; Wenqi Shen; Ying Shi; Ziyang Zhang; Inigo Barrio-Hernandez; Danish Memon; Claudia Hernandez-Armenta; Christopher J.P. Mathy; Tina Perica; Kala B. Pilla; Sai J. Ganesan; Daniel J. Saltzberg; Rakesh Ramachandran; Xi Liu; Sara B. Rosenthal; Lorenzo Calviello; Srivats Venkataramanan; Yizhu Lin; Stephanie A. Wankowicz; Markus Bohn; Phillip P. Sharp; Raphael Trenker; Janet M. Young; Devin A. Cavero; Joseph Hiatt; Theo Roth; Ujjwal Rathore; Advait Subramanian; Julia Noack; Mathieu Hubert; Ferdinand Roesch; Thomas Vallet; Björn Meyer; Kris M. White; Lisa Miorin; Oren S. Rosenberg; Kliment A. Verba; David Agard; Melanie Ott; Michael Emerman; Davide Ruggero; Adolfo Garc&amp;iacute-Sastre; Natalia Jura; Mark von Zastrow; Jack Taunton; Olivier Schwartz; Marco Vignuzzi; Christophe d&#39;Enfert; Shaeri Mukherjee; Matt Jacobson; Harmit S. Malik; Danica G Fujimori; Trey Ideker; Charles S Craik; Stephen Floor; James S. Fraser; John Gross; Andrej Sali; Tanja Kortemme; Pedro Beltrao; Kevan Shokat; Brian K. Shoichet; Nevan J. Krogan&#34;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.002386&#39;},
 {&#39;cord_uid&#39;: &#39;b3y9zxjr&#39;,
  &#39;sha&#39;: &#39;85e00fe8099f838d9eb455a7f62440d1bf7bc639&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naive and Experienced COVID-19 Patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20034041&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;As coronavirus disease 2019 (COVID-19) outbreak, caused by the severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), started in China in January, 2020, repurposing approved drugs is emerging as important therapeutic options. We reported here the first clinical study using hepatitis C virus (HCV) protease inhibitor, danoprevir, to treat COVID-19 patients. Danoprevir (Ganovo) is a potent HCV protease (NS3/4A) inhibitor (IC50 = 0.29 nM), which was approved and marketed in China since 2018 to treat chronic hepatitis C patients. Ritonavir is a CYP3A4 inhibitor to enhance plasma concentration of danoprevir while it also acts as a human immunodeficiency virus (HIV) protease inhibitor at high doses. The chymotrypsin-like protease of SARS-CoV-2 shares structure similarity with HCV and HIV proteases. In the current clinical study (NCT04291729) conducted at the Nineth Hospital of Nanchang, we evaluated therapeutic effects of danoprevir, boosted by ritonavir, on treatment naive and experienced COVID-19 patients. The data from this small-sample clinical study showed that danoprevir boosted by ritonavir is safe and well tolerated in all patients. After 4 to 12-day treatment of danoprevir boosted by ritonavir, all eleven patients enrolled, two naive and nine experienced, were discharged from the hospital as they met all four conditions as follows: (1) normal body temperature for at least 3 days; (2) significantly improved respiratory symptoms; (3) lung imaging shows obvious absorption and recovery of acute exudative lesion; and (4) two consecutive RT-PCR negative tests of SARS-CoV-2 nucleotide acid (respiratory track sampling with interval at least one day). Our findings suggest that repurposing danoprevir for COVID-19 is a promising therapeutic option.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Hongyi Chen; Zhicheng Zhang; Li Wang; Zhihua Huang; Fanghua Gong; Xiaodong Li; Yahong Chen; Jinzi J. WU&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20034041&#39;},
 {&#39;cord_uid&#39;: &#39;35ewwjfv&#39;,
  &#39;sha&#39;: &#39;396c98994ac34692a5fa7f8502fb33cebc54445a&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;High risk of infection caused posttraumatic stress symptoms in individuals with poor sleep quality: A study on influence of coronavirus disease (COVID-19) in China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20034504&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The influence of the outbreak of coronavirus disease (COVID-19) on mental health was poorly understood. The present study aimed to exam sleep problems and posttraumatic stress symptoms (PTSS) in Chinese immediately after the massive outbreak of COVID-19. A total of 2027 Chinese participated in the present study. Wuhan-expose history, sleep quality and PTSS were measured with self-rating scales. Results showed that there were significant differences of PCL-5 and of sleep quality scores in different data-collection dates (ps&lt;0.05). There were significant differences of PCL-5 scores (p&lt;0.05) and latency onset of sleep (p&lt;0.05) between participants with and without Wuhan-expose history. The interaction effect of Wuhan exposure history × sleep quality significantly influenced PCL-5 (ps&lt;0.05). These results indicate that keeping good sleep quality in individuals with high infectious risk is a way to prevent PTSS.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Fan Zhang; Zhilei Shang; Haiying Ma; Yanpu Jia; Luna Sun; Xin Guo; Lili Wu; Zhuoer Sun; Yaoguang Zhou; Yan Wang; Nianqi Liu; Weizhi Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20034504&#39;},
 {&#39;cord_uid&#39;: &#39;ddq2q1pg&#39;,
  &#39;sha&#39;: &#39;47a7579cf81744a0a83b1b64cc9017fb600452cd&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Epidemiological and clinical characteristics of COVID-19 in Shenzhen, the largest migrant city of China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20035246&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;We conducted a retrospective study among 417 confirmed COVID-19 cases from Jan 1 to Feb 28, 2020 in Shenzhen, the largest migrant city of China, to identify the epidemiological and clinical features in settings of high population mobility. We estimated the median incubation time to be 5.0 days. 342 (82.0%) cases were imported, 161 (38.6%) cases were identified by surveillance, and 247 (59.2%) cases were reported from cluster events. The main symptoms on admission were fever and dry cough. Most patients (91.4%) had mild or moderate illnesses. Age of 50 years or older, breathing problems, diarrhea, and longer time between the first medical visit and admission were associated with higher level of clinical severity. Surveillance-identified cases were much less likely to progress to severe illness. Although the COVID-19 epidemic has been contained in Shenzhen, close monitoring and risk assessments are imperative for prevention and control of COVID-19 in future.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-23&#39;,
  &#39;authors&#39;: &#39;Ying Wen; Lan Wei; Yuan Li; Xiujuan Tang; Shuo Feng; Kathy Leung; Xiaoliang Wu; Xiong-Fei Pan; Cong Chen; Junjie Xia; Xuan Zou; Tiejian Feng; Shujiang Mei&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20035246&#39;},
 {&#39;cord_uid&#39;: &#39;pm6ta0qy&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;The impact of temperature and absolute humidity on the coronavirus disease 2019 (COVID-19) outbreak - evidence from China&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20038919&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;OBJECTIVE To investigate the impact of temperature and absolute humidity on the coronavirus disease 2019 (COVID-19) outbreak. DESIGN Ecological study. SETTING 31 provincial-level regions in mainland China. MAIN OUTCOME MEASURES Data on COVID-19 incidence and climate between Jan 20 and Feb 29, 2020. RESULTS The number of new confirm COVID-19 cases in mainland China peaked on Feb 1, 2020. COVID-19 daily incidence were lowest at -10 ℃ and highest at 10 ℃,while the maximum incidence was observed at the absolute humidity of approximately 7 g/m3. COVID-19 incidence changed with temperature as daily incidence decreased when the temperature rose. No significant association between COVID-19 incidence and absolute humidity was observed in distributed lag nonlinear models. Additionally, A modified susceptible-exposed-infectious-recovered (M-SEIR) model confirmed that transmission rate decreased with the increase of temperature, leading to further decrease of infection rate and outbreak scale. CONCLUSION Temperature is an environmental driver of the COVID-19 outbreak in China. Lower and higher temperatures might be positive to decrease the COVID-19 incidence. M-SEIR models help to better evaluate environmental and social impacts on COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Peng Shi; Yinqiao Dong; Huanchang Yan; Xiaoyang Li; Chenkai Zhao; Wei Liu; Miao He; Shixing Tang; Shuhua Xi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20038919&#39;},
 {&#39;cord_uid&#39;: &#39;11hi1jel&#39;,
  &#39;sha&#39;: &#39;9b1e1c869a26b53973683d284250c2f4a8a339f6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Estimating excess 1- year mortality from COVID-19 according to underlying conditions and age in England: a rapid analysis using NHS health records in 3.8 million adults&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20040287&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;RAPID COMMUNICATION 22 March 2020 Estimating excess 1- year mortality from COVID-19 according to underlying conditions and age in England: a rapid analysis using NHS health records in 3.8 million adults Background: The medical, health service, societal and economic impact of the COVID-19 emergency has unknown effects on overall population mortality. Previous models of population mortality are based on death over days among infected people, nearly all of whom (to date at least) have underlying conditions. Models have not incorporated information on high risk conditions or their longer term background (pre-COVID-19) mortality. We estimated the excess number of deaths over 1 year under different COVID-19 incidence rates and differing mortality impacts. Methods: Using population based linked primary and secondary care electronic health records in England (HDR UK - CALIBER), we report the prevalence of underlying conditions defined by UK Public Health England COVID-19 guidelines (16 March 2020) in 3,862,012 individuals aged ≥30 years from 1997-2017. We used previously validated phenotypes, openly available (https://caliberresearch.org/portal), for each condition using ICD-10 diagnosis, Read, procedure and medication codes. We estimated the 1-year mortality in each condition, and developed simple models of excess COVID-19-related deaths assuming relative risk (RR) of the impact of the emergency (compared to background mortality) of 1.2, 1.5 and 2.0. Findings: 20.0% of the population are at risk according to current PHE guidelines, of which; 13.7% were age&gt;70 years and 6.3% aged ≤70 years with ≥1 underlying condition (cardiovascular disease (2.3%), diabetes (2.2%), steroid therapy (1.9%), severe obesity (0.9%), chronic kidney disease (0.6%) and chronic obstructive pulmonary disease, COPD (0.5%). Multimorbidity (co-occurrence of ≥2 conditions in an individual) was common (10.1%). The 1-year mortality in the at-risk population was 4.46%, and age and underlying conditions combine to influence background risk, varying markedly across conditions (5.9% in age&gt;70 years, 8.6% for COPD and 13.1% in those with ≥3 or more conditions). In a suppression scenario (at SARS CoV2 rates of 0.001% of the UK population), there would be minimal excess deaths (3 and 7 excess deaths at relative risk, RR, 1.5 and 2.0 respectively). At SARS CoV2 rates of 10% of the UK population (mitigation) the model estimates the numbers of excess deaths as: 13791, 34479 and 68957 (at RR 1.2, 1.5 and 2.0 respectively). At SARS CoV2 rates of 80% in the UK population (do-nothing), the model estimates the number of excess deaths as 110332, 275,830 and 551,659 (at RR 1.2, 1.5 and 2.0) respectively. Interpretation: We provide the public, researchers and policy makers a simple model to estimate the excess mortality over 1 year from COVID-19, based on underlying conditions at different ages. If the relative mortality impact of COVID-19 were to be about 20% (similar magnitude as the established winter vs summer mortality excess), then the excess deaths would be 0 when 1 in 100 000 (suppression), 13791 when 1 in 10 (mitigation) and 110332 when 8 in 10 are infected (do nothing) scenario. However, the relative impact of COVID-19 is unknown. If the emergency were to double the mortality risk, then we estimate 7, 68957 and 551,659 excess deaths in the same scenarios. These results may inform the need for more stringent suppression measures as well as efforts to target those at highest risk for a range of preventive interventions.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Amitava Banerjee; Laura Pasea; Steve Harris; Arturo Gonzalez-Izquierdo; Ana Torralbo; Laura Shallcross; Mahdad Noursadeghi; Deenan Pillay; Christina Pagel; Wai Keong Wong; Claudia Langenberg; Bryan Williams; Spiros Denaxas; Harry Hemingway&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20040287&#39;},
 {&#39;cord_uid&#39;: &#39;whnw19pc&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Hypertension and Diabetes Delay the Viral Clearance in COVID-19 Patients&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20040774&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Objectives: Comorbidities have significant indications for the disease outcome of COVID-19, however which underlying diseases that contribute the most to aggravate the conditions of COVID-19 patients is still largely unknown. SARS-CoV-2 viral clearance is a golden standard for defining the recovery of COVID-19 infections. To dissect the underlying diseases that could impact on viral clearance, we enrolled 106 COVID-19 patients who were hospitalized in the Zhongnan Hospital of Wuhan University, Wuhan, China between Jan 5 and Feb 25, 2020. Methodology: We comprehensively analyzed demographic, clinical and laboratory data, as well as patient treatment records. Survival analyses with Kaplan-Meier and Cox regression modelling were employed to identify factors influencing the viral clearance negatively. Results: We found that increasing age, male gender, and angiotensin-converting enzyme 2 (ACE2) associated factors (including hypertension, diabetes, and cardiovascular diseases) adversely affected the viral clearance. Furthermore, analysis by a random forest survival model pointed out hypertension, cortisone treatment, gender, and age as the four most important variables. Conclusions: We conclude that patients at old age, males, and/or having diseases associated with high expression of ACE2 will have worse prognosis during a COVID-19 infections.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Xiaoping Chen; Wenjia Hu; Jiaxin Ling; Pingzheng Mo; Yongxi Zhang; Qunqun Jiang; Zhiyong Ma; Qian Cao; Liping Deng; Shihui Song; Ruiying Zheng; Shicheng Gao; Hengning Ke; Xien Gui; Åke Lundkvist; Jinlin Li; Johanna F Lindahl; Yong Xiong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20040774&#39;},
 {&#39;cord_uid&#39;: &#39;j8odxs1h&#39;,
  &#39;sha&#39;: &#39;2e3c753682fba0d9c7a0462051a1287b9599ac92&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;A comparative multi-centre study on the clinical and imaging features of comfirmed and uncomfirmed patients with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20040782&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background Previous studies had described the differences in clinical characteristics between ICU and non-ICU patients. However, seldom study focused on confirmed and unconfirmed groups. Our aim was to compare clinical and imaging characteristics of COVID-19 patients outside Hubei province between confirmed and unconfirmed group. Methods We retrospectively enrolled 163 consecutive adult patients with suspected COVID-19 from three tertiary hospitals in two provinces outside Hubei province from January 12, 2020 to February 13, 2020 and the differences in epidemiological, clinical, laboratory and imaging characteristics between the two groups were compared. Results This study enrolled 163 patients with 62 confirmed cases and 101 unconfirmed cases. Most confirmed patients were clustered (31, 50.0%) and with definite epidemiological exposure. Symptoms of COVID-19 were nonspecific, largely fever and dry cough. Laboratory findings in confirmed group were characterized by normal or reduced white blood cell count, reduced the absolute value of lymphocytes, and elevated levels of C-reactive protein (CRP) and accelerated Erythrocyte sedimentation rate (ESR). The typical chest CT imaging features of patients with confirmed COVID-19 were peripherally distributed multifocal GGO with predominance in the lower lung lobe. Compared with unconfirmed patients, confirmed patients had significantly higher proportion of dry cough, leucopenia, lymphopenia and accelerated ESR (P&lt;0.05); but not with alanine aminotransferase, aspartate aminotransferase, D-dimer, lactic dehydrogenase, and myoglobin (P&gt;0.05). Proportion of peripheral, bilateral or lower lung distribution and multi-lobe involvement, GGO, crazy-paving pattern, air bronchogram and pleural thickening in the confirmed group were also higher (P&lt;0.05). Conclusions Symptoms of COVID-19 were nonspecific. Leukopenia, lymphopenia and ESR, as well as chest CT could be used as a clue for clinical diagnosis of COVID-19.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Congliang Miao; Jinqiang Zhuang; Mengdi Jin; Huanwen Xiong; Peng Huang; Qi Zhao; Li Miao; Jiang Du; Xinying Yang; Peijie Huang; Jiang Hong&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20040782&#39;},
 {&#39;cord_uid&#39;: &#39;gjfd1jhb&#39;,
  &#39;sha&#39;: &#39;77efa10bbf0d22795315f61ace2dc95804d269fb&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Analysis of psychological state and clinical psychological intervention model of patients with COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20040899&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;Background:Patients with the 2019 coronavirus disease (COVID-19) have different degrees of psychological pain, such as anxiety and depression, which may related to their prognosis. Psychological intervention can be conducted in different ways to improve psychological pain and improve the treatment effect. Objective:The present study aimed to investigate and analyze the psychological status of patients with COVID-19 during the course of illness, and to evaluate the effect and influencing factors of psychological crisis intervention, so as to explore the effective mode of clinical psychological intervention in acute patients under isolation environment. Methods:A total of 143 persons participated in the study, including 26 patients diagnosed with COVID-19 in the isolation ward (COVID-19 group), 87 patients with general pneumonia in the observation ward (General Pneumonia group) and 30 healthy volunteers (Normal group). All the patients in the ward received comprehensive psychological intervention, including telephone psychological counseling (active and passive), self-adjustment of written materials and one-to-one psychological crisis intervention. Hamilton depression scale (HAMD) and Hamilton anxiety scale (HAMA) were used to evaluate the mental health status of all patients on the day of admission and 1 week after treatment. Results:The scores of HAMA and HAMD of all patients (including isolation ward and observation ward) were significantly higher than the healthy volunteers at the time of admission. The total score of HAMA and HAMD in CVOID-19 group were both higher than that General Pneumonia group. After 1 week`s hospitalization with comprehensive psychological intervention, the scores of HAMA and HAMD in CVOID-19 group were significantly decreased. Conclusion:Patients those who diagnosed with COVID-19 in the isolation ward and/or general pneumonia in observation ward have different degrees of anxiety, depression and sleep problems. While receiving antiviral treatment, patients also need psychological intervention. Comprehensive psychological intervention model has been proved to be effective.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Lu Yang; Danjuan Wu; Yanbin Hou; Xunqiang Wang; Ni Dai; Guanjun Wang; Qing Yang; Wenhui Zhao; Zhongze Lou; Yunxin Ji; Liemin Ruan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20040899&#39;},
 {&#39;cord_uid&#39;: &#39;dqg8fkca&#39;,
  &#39;sha&#39;: &#39;71ee5dbc401b2af835c84275d436ff66ad6f337f&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Mechanistic-statistical SIR modelling for early estimation of the actual number of cases and mortality rate from COVID-19&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20040915&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#34;The first cases of COVID-19 in France were detected on January 24, 2020. The number of screening tests carried out and the methodology used to target the patients tested do not allow for a direct computation of the actual number of cases and the mortality rate. In this note, we develop a &#39;mechanistic-statistical&#39; approach coupling a SIR ODE model describing the unobserved epidemiological dynamics, a probabilistic model describing the data acquisition process and a statistical inference method. The objective of this model is not to make forecasts but to estimate the actual number of people infected with COVID-19 during the observation window in France and to deduce the mortality rate associated with the epidemic. Main results. The actual number of infected cases in France is probably much higher than the observations: we find here a factor x15 (95%-CI: 4-33), which leads to a 5.2/1000 mortality rate (95%-CI: 1.5/1000-11.7/1000) at the end of the observation period. We find a R0 of 4.8, a high value which may be linked to the long viral shedding period of 20 days.&#34;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Lionel Roques; Etienne Klein; Julien Papaix; Samuel Soubeyrand&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20040915&#39;},
 {&#39;cord_uid&#39;: &#39;nzat41wu&#39;,
  &#39;sha&#39;: &#39;79622641e002b2574af9dc7f99d4fbbe192606e6&#39;,
  &#39;source_x&#39;: &#39;medrxiv&#39;,
  &#39;title&#39;: &#39;Social distancing strategies for curbing the COVID-19 epidemic&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.22.20041079&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;medrvix&#39;,
  &#39;abstract&#39;: &#39;The SARS-CoV-2 pandemic is straining healthcare resources worldwide, prompting social distancing measures to reduce transmission intensity. The amount of social distancing needed to curb the SARS-CoV-2 epidemic in the context of seasonally varying transmission remains unclear. Using a mathematical model, we assessed that one-time interventions will be insufficient to maintain COVID-19 prevalence within the critical care capacity of the United States. Seasonal variation in transmission will facilitate epidemic control during the summer months but could lead to an intense resurgence in the autumn. Intermittent distancing measures can maintain control of the epidemic, but without other interventions, these measures may be necessary into 2022. Increasing critical care capacity could reduce the duration of the SARS-CoV-2 epidemic while ensuring that critically ill patients receive appropriate care.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Stephen M Kissler; Christine Tedijanto; Marc Lipsitch; Yonatan Grad&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.22.20041079&#39;},
 {&#39;cord_uid&#39;: &#39;7a3wdduq&#39;,
  &#39;sha&#39;: &#39;2e092f8f739aaa16ca632e9d2516f0ff2a94f781&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.23.002931&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor (i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3 °C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~0.35 USD per device. The system is operated digitally, portable and low power — capable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS900) of the genomic DNA of M. avium subsp. paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-Detection of 20 fg, equivalent to a single bacterium, at the 30th cycle. Using TriSilix, we also detected the cDNA from SARS-CoV-2 (1 pg), through PCR, with high specificity against SARS-CoV (2003).&#39;,
  &#39;publish_time&#39;: &#39;2020-03-25&#39;,
  &#39;authors&#39;: &#39;Estefania Nunez-Bajo; Michael Kasimatis; Yasin Cotur; Tarek Asfour; Alex Collins; Ugur Tanriverdi; Max Grell; Matti Kaisti; Guglielmo Senesi; Karen Stevenson; Firat Guder&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.23.002931&#39;},
 {&#39;cord_uid&#39;: &#39;mmkrwj8t&#39;,
  &#39;sha&#39;: &#39;9818f1bc227cf7e40f757ec36342c9824a21418d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A unifying structural and functional model of the coronavirus replication organelle: tracking down RNA synthesis&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.24.005298&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Zoonotic coronavirus (CoV) infections, like those responsible for the current SARS-CoV-2 epidemic, cause grave international public health concern. In infected cells, the CoV RNA-synthesizing machinery associates with modified endoplasmic reticulum membranes that are transformed into the viral replication organelle (RO). While double-membrane vesicles (DMVs) appear to be a pan -coronavirus RO element, studies to date describe an assortment of additional coronavirus-induced membrane structures. Despite much speculation, it remains unclear which RO element(s) accommodate viral RNA synthesis. Here we provide detailed 2D and 3D analyses of CoV ROs and show that diverse CoVs essentially induce the same membrane modifications, including the small open double-membrane spherules (DMSs) previously thought to be restricted to gamma- and delta-CoV infections and proposed as sites of replication. Metabolic labelling of newly-synthesized viral RNA followed by quantitative EM autoradiography revealed abundant viral RNA synthesis associated with DMVs in cells infected with the beta-CoVs MERS-CoV and SARS-CoV, and the gamma-CoV infectious bronchitis virus. RNA synthesis could not be linked to DMSs or any other cellular or virus-induced structure. Our results provide a unifying model of the CoV RO and clearly establish DMVs as the central hub for viral RNA synthesis and a potential drug target in coronavirus infection.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-24&#39;,
  &#39;authors&#39;: &#39;Eric J. Snijder; Ronald W.A.L. Limpens; Adriaan H. de Wilde; Anja W. M. de Jong; Jessika C. Zevenhoven-Dobbe; Helena J. Maier; F.G.A. Faas; Abraham J. Koster; Montserrat Bárcena&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.24.005298&#39;},
 {&#39;cord_uid&#39;: &#39;ebwxryai&#39;,
  &#39;sha&#39;: &#39;cda86bc966b1e5781579c49c8fc2179f85afa04c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Cellular senescence limits acute lung injury induced by mechanical ventilation&#39;,
  &#39;doi&#39;: &#39;10.1101/2020.03.24.005983&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Different forms of lung injury can activate senescence pathways. Senescence has been involved in chronic lung diseases, but its role in the acute phase is less known. Acute lung injury activates a large variety of pathogenetic mechanisms that can lead to respiratory insufficiency. In some cases, mechanical ventilation may help to maintain gas exchange, but positive airway pressures can cause regional overdistension of alveolar units and further damage. Here, we report that acute lung injury and alveolar overstretching activate a lung senescent program aimed to limit apoptosis. A systematic pooling of transcriptomic data from animal models of lung injury demonstrates the enrichment of a specific senescence signature. In a clinically relevant, murine model of acid aspiration and mechanical ventilation, we observed the activation of the Tp53/Cdkn1a (p21) senescence pathway, increases in senescence-associated heterochromatin foci in alveolar cells and markers of DNA damage, chromatin reorganization and changes in the nuclear envelope. Autopsy samples from lungs of critically ill patients showed the increase in senescence-associated heterochromatin foci. Cell stretch per se was also able to activate the senescence response in an ex-vivo model. Absence of Cdkn1a decreased the senescent response, but worsened lung injury by a significant increase in apoptosis. In opposite, treatment with lopinavir/ritonavir led to Cdkn1a overexpression and ameliorated cell apoptosis and lung injury. Collectively, these results suggest that acute injury promotes lung senescence. This response plays a protective role against acute lung injury in the short term, but its long-term consequences remain unexplored.&#39;,
  &#39;publish_time&#39;: &#39;2020-03-25&#39;,
  &#39;authors&#39;: &#39;Jorge Blazquez-Prieto; Covadonga Huidobro; Ines Lopez-Alonso; Laura Amado-Rodriguez; Paula Martin-Vicente; Cecilia Lopez-Martinez; Irene Crespo; Cristina Pantoja; Pablo J Fernandez-Marcos; Manuel Serrano; Jacob I Sznajder; Guillermo M Albaiceta&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/2020.03.24.005983&#39;},
 {&#39;cord_uid&#39;: &#39;begn4tsd&#39;,
  &#39;sha&#39;: &#39;7ea2f1ca7b3c2370d413ae990eb71b2b990c91fe&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Clearing the FoG: Antifungal tolerance is a subpopulation effect that is distinct from resistance and is associated with persistent candidemia&#39;,
  &#39;doi&#39;: &#39;10.1101/206359&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDrug susceptibility, defined by the minimal inhibitory concentration (MIC), often does not predict whether fungal infections will respond to therapy in the clinic. Tolerance at supra-MIC antifungal drug concentrations is rarely quantified and current clinical recommendations suggest it be ignored. Here, we measured and characterized drug-response variables that could influence the outcomes of fungal infections and be generalizable across major clades of Candida albicans, one of the most frequently isolated human fungal pathogens. We quantified antifungal tolerance as the fraction of growth (FoG) above the MIC and found that it is clearly distinct from susceptibility/resistance measured as MIC. Instead, tolerance is due to the slow growth of subpopulations of cells that overcome drug stress more efficiently than the rest of the population, and correlates inversely with the accumulation of intracellular drug. Importantly, many adjuvant drugs used together with fluconazole, a fungistatic drug, reduce tolerance without affecting resistance. These include inhibitors of major stress response hubs such as Hsp90, calcineurin, PKC1 and TOR. Accordingly, in an invertebrate infection model, adjuvant combination therapy was significantly more effective than fluconazole alone in treating highly tolerant isolates and did not improve the treatment of isolates with low tolerance levels. Furthermore, isolates recovered from immunocompetent patients with persistent candidemia displayed significantly higher tolerance than isolates that were readily cleared by fluconazole. Thus, tolerance correlates with the response to fluconazole therapy in patients and may help predict whether infections will respond to fluconazole alone. Similarly, measuring tolerance may provide a useful clinical parameter for choosing appropriate therapeutic strategies to overcome persistent clinical candidemia.&#39;,
  &#39;publish_time&#39;: &#39;2017-10-20&#39;,
  &#39;authors&#39;: &#39;Alexander Rosenberg; Iuliana V. Ene; Maayan Bibi; Shiri Zakin; Ella Shtifman Segal; Naomi Ziv; Alon M. Dahan; Arnaldo L. Colombo; Richard J. Bennett; Judith Berman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/206359&#39;},
 {&#39;cord_uid&#39;: &#39;0qmsripp&#39;,
  &#39;sha&#39;: &#39;f634f2b73197fdda1d444dcdd6bd84a96ea148ad&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Viral fitness predicts the magnitude and direction of perturbations in the infected host transcriptome&#39;,
  &#39;doi&#39;: &#39;10.1101/206789&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Determining the fitness of viral genotypes has become a standard practice in virology as it is essential to evaluate their evolutionary potential. Darwinian fitness, defined as the advantage of a given genotype with respect to a reference one, is a mesoscopic property that captures into a single figure differences in performance at every stage of viral infection. But to which extent viral fitness results from particular molecular interactions with host factors and regulatory networks during infection? Can we identify host genes, and then functional classes, whose expression depends on viral fitness? Here, we compared the transcriptomes of tobacco plants infected with seven genotypes of tobacco etch potyvirus (TEV) that differ in fitness. We found that the larger the fitness differences among genotypes, the more dissimilar the transcriptomic profiles are. Consistently, two different mutations, one in the viral RNA polymerase and another in the viral suppressor of RNA silencing, that led to close fitness values, also resulted in significantly similar gene expression profiles. Moreover, we identified host genes whose expression showed a significant correlation, positive or negative, with TEV fitness. Over-expression of genes with positive correlation activates hormone-and RNA silencing-mediated pathways of plant defense. By contrast, under-expression of genes negatively correlated reduces metabolism, growth, and development. Overall, these results reveal the high information content of viral fitness, and suggest its potential use to predict differences in genomic profiles of infected hosts.&#39;,
  &#39;publish_time&#39;: &#39;2017-10-20&#39;,
  &#39;authors&#39;: &#39;Héctor Cervera; Silvia Ambrós; Guillermo P. Bernet; Guillermo Rodrigo; Santiago F. Elena&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/206789&#39;},
 {&#39;cord_uid&#39;: &#39;ebqquj7i&#39;,
  &#39;sha&#39;: &#39;620d69e419e3c8756de8bf96d86f6bf0de7ed919&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Neuronal CCL2 expression drives inflammatory monocyte infiltration into the brain during acute virus infection&#39;,
  &#39;doi&#39;: &#39;10.1101/208587&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;AbstractBackgroundViral encephalitis is a dangerous compromise between the need to robustly clear pathogen from the brain and the need to protect neurons from bystander injury. Theiler&#39;s murine encephalomyelitis virus (TMEV) infection of C57Bl/6 mice is a model of viral encephalitis in which the compromise results in hippocampal damage and permanent neurological sequelae. We previously identified brain infiltrating inflammatory monocytes as the primary driver of this hippocampal pathology, but the mechanisms involved in recruiting these cells to the brain were unclear.MethodsChemokine expression levels in the hippocampus were assessed by microarray, ELISA, RT-PCR, and immunofluorescence. Monocyte infiltration during acute TMEV infection was measured by flow cytometry. CCL2 levels were manipulated by immunodepletion and by specific removal from neurons in mice generated by crossing a line expressing the Cre recombinase behind the synapsin promoter to animals with floxed CCL2.ResultsInoculation of the brain with TMEV induced hippocampal production of the proinflammatory chemokine CCL2 that peaked at 6 hours postinfection, whereas inoculation with UV-inactivated TMEV did not elicit this response. Immunofluorescence revealed that hippocampal neurons expressed high levels of CCL2 at this timepoint. Genetic deletion of CCR2 and systemic immunodepletion of CCL2 abrogated or blunted the infiltration of inflammatory monocytes into the brain during acute infection. Specific genetic deletion of CCL2 from neurons reduced serum and hippocampal CCL2 levels and inhibited inflammatory monocyte infiltration into the brain.ConclusionsWe conclude that intracranial inoculation with infectious TMEV rapidly induces the expression of CCL2 in neurons, and this cellular source is necessary for CCR2-dependent infiltration of inflammatory monocytes into the brain during the most acute stage of encephalitis. These findings highlight a unique role for neuronal production of chemokines in the initiation of leukocytic infiltration into the infected central nervous system.&#34;,
  &#39;publish_time&#39;: &#39;2017-10-25&#39;,
  &#39;authors&#39;: &#39;Charles L Howe; Reghann G. LaFrance-Corey; Emma N Goddery; Kanish Mirchia&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/208587&#39;},
 {&#39;cord_uid&#39;: &#39;i4yquw4k&#39;,
  &#39;sha&#39;: &#39;41fc7191573c2f01fdef9c8ebb072e5f54cbb598&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Coordination between discrete Mitotic Arrest Deficient 1 (MAD1) domains is required for efficient mitotic checkpoint signaling&#39;,
  &#39;doi&#39;: &#39;10.1101/212704&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTAs a sensitive signaling system, the mitotic checkpoint ensures faithful chromosome segregation by delaying anaphase onset when even a single kinetochore is unattached. The key signal amplification reaction for the checkpoint is the conformational conversion of open MAD2 (O-MAD2) into closed MAD2 (C-MAD2). The reaction was suggested to be catalyzed by an unusual catalyst, a MAD1:C-MAD2 tetramer, but how the catalysis is executed and regulated remains elusive. Here we report that in addition to the well-characterized middle region (MIM), both amino- and carboxyl-terminal domains (NTD and CTD) of MAD1 also contribute to the mitotic checkpoint. In contrast to MIM that stably associates with C-MAD2, MAD1-NTD and CTD surprisingly bind to both O-MAD2 and C-MAD2, suggesting their interactions with substrates and products of the O-C conversion. MAD1-NTD also interacts with CTD. MPS1 kinase interacts with and phosphorylates both NTD and CTD. The phosphorylation reduces the NTD:CTD interaction and CTD interaction with MPS1. Mutating CTD phosphorylation sites including Thr716 compromises MAD2 binding and the checkpoint responses. Ser610 and Tyr634 also contribute to the checkpoint. Our results have uncovered previously unknown interactions of MAD1-NTD and CTD with MAD2 conformers and their regulation by MPS1 kinase, providing novel insights into the mitotic checkpoint signaling.&#39;,
  &#39;publish_time&#39;: &#39;2017-11-01&#39;,
  &#39;authors&#39;: &#39;Wenbin Ji; Yibo Luo; Ejaz Ahmad; Song-Tao Liu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/212704&#39;},
 {&#39;cord_uid&#39;: &#39;ey4c74az&#39;,
  &#39;sha&#39;: &#39;3c70c99afc7a38df3c4807857856ea258d378429&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The molecular basis for Pompe disease revealed by the structure of human acid α-glucosidase&#39;,
  &#39;doi&#39;: &#39;10.1101/212837&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPompe disease results from a defect in human acid α-glucosidase (GAA), a lysosomal enzyme that cleaves terminal α1-4 and α1-6 glucose from glycogen. In Pompe disease (also known as Glycogen Storage Disorder type II), the accumulation of undegraded glycogen in lysosomes leads to cellular dysfunction, primarily in muscle and heart tissues. Pompe disease is an active candidate of clinical research, with pharmacological chaperone therapy tested and enzyme replacement therapy approved. Despite production of large amounts of recombinant GAA annually, the structure of GAA has not been reported until now. Here, we describe the first structure of GAA, at 1.7Å resolution. Three structures of GAA complexes reveal the molecular basis for the hundreds of mutations that lead to Pompe disease and for pharmacological chaperoning in the protein. The GAA structure reveals a surprising second sugar-binding site 34Å from the active site, suggesting a possible mechanism for processing of large glycogen substrates. Overall, the structure will assist in the design of next-generation treatments for Pompe disease.&#39;,
  &#39;publish_time&#39;: &#39;2017-11-01&#39;,
  &#39;authors&#39;: &#39;Derrick Deming; Karen Lee; Tracey McSherry; Ronnie R. Wei; Tim Edmunds; Scott C. Garman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/212837&#39;},
 {&#39;cord_uid&#39;: &#39;b2f3a8un&#39;,
  &#39;sha&#39;: &#39;d618a596def847bdfc19991c694e09bdeccf2287&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Stochastic dynamics of an epidemics with recurrent spillovers from an endemic reservoir&#39;,
  &#39;doi&#39;: &#39;10.1101/213579&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMost emerging human infectious diseases have an animal origin. Yet, while zoonotic diseases originate from a primary reservoir, most theoretical studies have principally focused on single-host processes, either exclusively humans or exclusively animals, without considering the importance of animal to human transmission for understanding the dynamics of emerging infectious diseases. Here we aim to investigate the importance of spillover transmission for explaining the number and the size of outbreaks. We propose a simple stochastic Susceptible-Infected-Recovered model with a recurrent infection of an incidental host from a reservoir (e.g. humans by a zoonotic species), considering two modes of transmission, (1) animal-to-human and (2) human-to-human. The model assumes that (i) epidemiological processes are faster than other processes such as demographics or pathogen evolution and (ii) that an epidemic occurs until there are no susceptible individuals left. The results show that during an epidemic, even when the pathogens are barely contagious, multiple outbreaks are observed due to spillover transmission. Overall, the findings demonstrate that the only consideration of direct transmission between individuals is not sufficient to explain the dynamics of zoonotic pathogens in an incidental host.&#39;,
  &#39;publish_time&#39;: &#39;2017-11-03&#39;,
  &#39;authors&#39;: &#39;Marina Voinson; Alexandra Alvergne; Sylvain Billiard; Charline Smadi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/213579&#39;},
 {&#39;cord_uid&#39;: &#39;zz4gut07&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Large protein complex production using the SmartBac System - Strategies and Applications&#39;,
  &#39;doi&#39;: &#39;10.1101/219246&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTRecent revolution of cryo-electron microscopy has opened a new door to solve high-resolution structures of macromolecule complexes without crystallization while how to efficiently obtain homogenous macromolecule complex sample is therefore becoming a bottleneck. Here we report SmartBac, an easy and versatile system for constructing large-sized transfer plasmids used to generate recombinant baculoviruses that express large multiprotein complexes in insect cells. The SmartBac system integrates the univector plasmid-fusion system, Gibson assembly method and polyprotein strategy to construct the final transfer plasmids. The fluorescent proteins are designed to be co-expressed with recombinant proteins to monitor transfection and expression efficiencies. A scheme of screening an optimal tagged subunit for effective purification is provided. Six large multiprotein complexes including the human exocyst complex and dynactin complex were successfully expressed, suggesting a great potential of SmartBac for its wide application in the future structural biology study.&#39;,
  &#39;publish_time&#39;: &#39;2017-11-14&#39;,
  &#39;authors&#39;: &#39;Yujia Zhai; Danyang Zhang; Leiye Yu; Fang Sun; Fei Sun&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/219246&#39;},
 {&#39;cord_uid&#39;: &#39;tenmmkh9&#39;,
  &#39;sha&#39;: &#39;4b1a08339bb5508f830e09f1a2c3a8022526cce0&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Airway IRF7hi versus IRF7lo molecular response patterns determine clinical phenotypes in children with acute wheezing&#39;,
  &#39;doi&#39;: &#39;10.1101/222950&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryAsthma exacerbations are triggered by rhinovirus infections. We employed a systems biology approach to delineate upper airway gene network patterns underlying asthma exacerbation phenotypes in children. Cluster analysis unveiled distinct IRF7hi versus IRF7lo molecular phenotypes, the former exhibiting robust upregulation of Th1/type I interferon responses and the latter an alternative signature marked by upregulation of cytokine and growth factor signalling and downregulation of interferon gamma. The two phenotypes also produced distinct clinical phenotypes. For IRF7lo versus IRF7hi: symptom duration prior to hospital presentation was more than twice as long from initial symptoms (p=0.011) and nearly three times as long for cough (p&lt;0.001); the odds ratio of admission to hospital was increased more than four-fold (p=0.018); and time to recurrence was shorter (p=0.015). In summary, our findings demonstrate that asthma exacerbations in children can be divided into IRF7hi versus IRF7lo phenotypes with associated differences in clinical phenotypes.AbbreviationsAHR, airway hyperresponsiveness; ARG1, Arginase 1, CSF3, Colony Stimulating Factor 3; CD38, Cluster of Differentiation 38; CD163, Cluster of Differentiation 163; cDCs, conventional (or myeloid) dendritic cells; DDX60, DExD/H-Box Helicase 60; ED, Emergency Department; EGF, Epidermal Growth Factor; ERK, Extracellular signal-Regulated Kinase; FCER1G, Fc Fragment Of IgE Receptor Ig; HMBS, Hydroxymethylbilane Synthase; IFNg, Interferon Gamma; IFNL1, Interferon Lambda 1; IL-1R2, Interleukin 1 Receptor Type 2; IRF7, Interferon Regulatory Factor 7; ISG15, Interferon-stimulated gene 15; MDA5, Melanoma Differentiation-Associated protein 5; MX1, Myxovirus Resistance Protein 1; NAD, nicotinamide adenine dinucleotide; NCR1, Natural cytotoxicity triggering receptor 1; OSM, Oncostatin M; PD-L1, Programmed Death-Ligand 1; PPIA, Peptidylprolyl Isomerase A; PPIB Peptidylprolyl Isomerase B; RSAD2, Radical S-adenosyl methionine domain-containing protein 2; RSV, respiratory syncytial virus; RT-qPCR, quantitative reverse transcription PCR; RV, rhinovirus; sPLA2, secretory Phospholipase A2; TGFb, Transforming Growth Factor beta; THBS1, Thrombospondin 1; TNF, Tumor Necrosis Factor; TLR2, Toll-like Receptor 2.&#39;,
  &#39;publish_time&#39;: &#39;2017-11-21&#39;,
  &#39;authors&#39;: &#39;Siew-Kim Khoo; James Read; Kimberley Franks; Guicheng Zhang; Joelene Bizzintino; Laura Coleman; Christopher McCrae; Lisa Öberg; Niamh Troy; Franciska Prastanti; Janet Everard; Stephen Oo; Meredith L Borland; Rose A Maciewicz; Peter N Le Souëf; Ingrid A Laing; Anthony Bosco&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/222950&#39;},
 {&#39;cord_uid&#39;: &#39;ithoxu6k&#39;,
  &#39;sha&#39;: &#39;b1e7752cbf528db99723a3a67b49b9b5c6e449af&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Host shifts result in parallel genetic changes when viruses evolve in closely related species&#39;,
  &#39;doi&#39;: &#39;10.1101/226175&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractHost shifts, where a pathogen invades and establishes in a new host species, are a major source of emerging infectious diseases. They frequently occur between related host species and often rely on the pathogen evolving adaptations that increase their fitness in the novel host species. To investigate genetic changes in novel hosts, we experimentally evolved replicate lineages of an RNA virus (Drosophila C Virus) in 19 different species of Drosophilidae and deep sequenced the viral genomes. We found a strong pattern of parallel evolution, where viral lineages from the same host were genetically more similar to each other than to lineages from other host species. When we compared viruses that had evolved in different host species, we found that parallel genetic changes were more likely to occur if the two host species were closely related. This suggests that when a virus adapts to one host it might also become better adapted to closely related host species. This may explain in part why host shifts tend to occur between related species, and may mean that when a new pathogen appears in a given species, closely related species may become vulnerable to the new disease.&#39;,
  &#39;publish_time&#39;: &#39;2017-11-28&#39;,
  &#39;authors&#39;: &#39;Ben Longdon; Jonathan P Day; Joel M Alves; Sophia CL Smith; Thomas M Houslay; John E McGonigle; Lucia Tagliaferri; Francis M Jiggins&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/226175&#39;},
 {&#39;cord_uid&#39;: &#39;ml5ra9x1&#39;,
  &#39;sha&#39;: &#39;75216c0238ddc17dec8ef66adcc84bd48922cae4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/228692&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPrecise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights.&#39;,
  &#39;publish_time&#39;: &#39;2017-12-04&#39;,
  &#39;authors&#39;: &#39;Xueting Qiu; Justin Bahl&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/228692&#39;},
 {&#39;cord_uid&#39;: &#39;9w2wjiik&#39;,
  &#39;sha&#39;: &#39;a7a4a5568e5b7c921bc616eedf532a393229decb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Gene birth contributes to structural disorder encoded by overlapping genes&#39;,
  &#39;doi&#39;: &#39;10.1101/229690&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe same nucleotide sequence can encode two protein products in different reading frames. Overlapping gene regions encode higher levels of intrinsic structural disorder (ISD) than non-overlapping genes (39% vs. 25% in our viral dataset). This might be because of the intrinsic properties of the genetic code, because one member per pair was recently born de novo in a process that favors high ISD, or because high ISD relieves increased evolutionary constraint imposed by dual-coding. Here we quantify the relative contributions of these three alternative hypotheses. We estimate that the recency of de novo gene birth explains 32% or more of the elevation in ISD in overlapping regions of viral genes. While the two reading frames within a same-strand overlapping gene pair have markedly different ISD tendencies that must be controlled for, their effects cancel out to make no net contribution to ISD. The remaining elevation of ISD in the older members of overlapping gene pairs, presumed due to the need to alleviate evolutionary constraint, was already present prior to the origin of the overlap. Same-strand overlapping gene birth events can occur in two different frames, favoring high ISD either in the ancestral gene or in the novel gene; surprisingly, most de novo gene birth events contained completely within the body of an ancestral gene favor high ISD in the ancestral gene (23 phylogenetically independent events vs. 1). This can be explained by mutation bias favoring the frame with more start codons and fewer stop codons.&#39;,
  &#39;publish_time&#39;: &#39;2017-12-06&#39;,
  &#39;authors&#39;: &#39;S. Willis; J. Masel&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/229690&#39;},
 {&#39;cord_uid&#39;: &#39;xpbgpsp8&#39;,
  &#39;sha&#39;: &#39;02201e4601ab0eb70b6c26480cf2bfeae2625193&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The impact of regular school closure on seasonal influenza epidemics: a data-driven spatial transmission model for Belgium&#39;,
  &#39;doi&#39;: &#39;10.1101/230565&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSchool closure is often considered as an option to mitigate influenza epidemics because of its potential to reduce transmission in children and then in the community. The policy is still however highly debated because of controversial evidence. Moreover, the specific mechanisms leading to mitigation are not clearly identified.We introduced a stochastic spatial age-specific metapopulation model to assess the role of holiday-associated behavioral changes and how they affect seasonal influenza dynamics. The model is applied to Belgium, parameterized with country-specific data on social mixing and travel, and calibrated to the 2008/2009 influenza season. It includes behavioral changes occurring during weekend vs. weekday, and holiday vs. school-term. Several experimental scenarios are explored to identify the relevant social and behavioral mechanisms.Stochastic numerical simulations show that holidays considerably delay the peak of the season and mitigate its impact. Changes in mixing patterns are responsible for the observed effects, whereas changes in travel behavior do not alter the epidemic. Weekends are important in slowing down the season by periodically dampening transmission. Christmas holidays have the largest impact on the epidemic, however later school breaks may help in reducing the epidemic size, stressing the importance of considering the full calendar. An extension of the Christmas holiday of 1 week may further mitigate the epidemic.Changes in the way individuals establish contacts during holidays are the key ingredient explaining the mitigating effect of regular school closure. Our findings highlight the need to quantify these changes in different demographic and epidemic contexts in order to provide accurate and reliable evaluations of closure effectiveness. They also suggest strategic policies in the distribution of holiday periods to minimize the epidemic impact.&#39;,
  &#39;publish_time&#39;: &#39;2017-12-07&#39;,
  &#39;authors&#39;: &#39;Giancarlo De Luca; Kim Van Kerckhove; Pietro Coletti; Chiara Poletto; Nathalie Bossuyt; Niel Hens; Vittoria Colizza&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/230565&#39;},
 {&#39;cord_uid&#39;: &#39;lcj64z9k&#39;,
  &#39;sha&#39;: &#39;c6039f8933305c9f44a44c81a15b321b6c2848dc&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Far-UVC light: A new tool to control the spread of airborne-mediated microbial diseases&#39;,
  &#39;doi&#39;: &#39;10.1101/240408&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAirborne-mediated microbial diseases such as influenza and tuberculosis represent major public health challenges. A direct approach to prevent airborne transmission is inactivation of airborne pathogens, and the airborne antimicrobial potential of UVC ultraviolet light has long been established; however, its widespread use in public settings is limited because conventional UVC light sources are both carcinogenic and cataractogenic. By contrast, we have previously shown that far-UVC light (207–222 nm) efficiently kills bacteria without harm to exposed mammalian skin. This is because, due to its strong absorbance in biological materials, far-UVC light cannot penetrate even the outer (non living) layers of human skin or eye; however, because bacteria and viruses are of micrometer or smaller dimensions, far-UVC can penetrate and inactivate them. We show for the first time that far-UVC efficiently kills airborne aerosolized viruses, a very low dose of 2 mJ/cm2 of 222-nm light inactivating &gt;95% of aerosolized H1N1 influenza virus. Continuous very low dose-rate far-UVC light in indoor public locations is a promising, safe and inexpensive tool to reduce the spread of airborne-mediated microbial diseases.&#39;,
  &#39;publish_time&#39;: &#39;2017-12-28&#39;,
  &#39;authors&#39;: &#39;David Welch; Manuela Buonanno; Veljko Grilj; Igor Shuryak; Connor Crickmore; Alan W. Bigelow; Gerhard Randers-Pehrson; Gary W. Johnson; David J. Brenner&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/240408&#39;},
 {&#39;cord_uid&#39;: &#39;gqhlw20n&#39;,
  &#39;sha&#39;: &#39;fec0c97b9bdc012b76f082b8bcd6ba0efbb4f5c3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Viral gain-of-function experiments uncover residues under diversifying selection in nature&#39;,
  &#39;doi&#39;: &#39;10.1101/242495&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;Viral gain-of-function mutations are commonly observed in the laboratory; however, it is unknown whether those mutations also evolve in nature. We identify two key residues in the host recognition protein of bacteriophage λ that are necessary to exploit a new receptor; both residues repeatedly evolved among homologs from environmental samples. Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-03&#39;,
  &#39;authors&#39;: &#39;Rohan Maddamsetti; Daniel T. Johnson; Stephanie J. Spielman; Katherine L. Petrie; Debora S. Marks; Justin R. Meyer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/242495&#39;},
 {&#39;cord_uid&#39;: &#39;pnd0zzdh&#39;,
  &#39;sha&#39;: &#39;1c06378f629bfa8870c89b3abfdecfc19db9b459&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The Viral Protein Corona Directs Viral Pathogenesis and Amyloid Aggregation&#39;,
  &#39;doi&#39;: &#39;10.1101/246785&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractArtificial nanoparticles accumulate a protein corona layer in biological fluids, which significantly influences their bioactivity. As nanosized obligate intracellular parasites, viruses share many biophysical properties with artificial nanoparticles in extracellular environments and here we show that respiratory syncytial virus (RSV) and herpes simplex virus 1 (HSV-1) accumulate a rich and distinctive protein corona in different biological fluids. Moreover, we show that corona pre-coating differentially affects viral infectivity and immune cell activation. Additionally, we demonstrate that viruses bind amyloidogenic peptides in their corona and catalyze amyloid formation via surface-assisted heterogeneous nucleation. Importantly, we show that HSV-1 catalyzes the aggregation of the amyloid beta peptide (Aβ42), a major constituent of amyloid plaques in Alzheimer’s disease, in-vitro and in animal models. Our results highlight the viral protein corona as an acquired structural layer that is critical for viral-host interactions and illustrate a mechanistic convergence between viral and amyloid pathologies.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-16&#39;,
  &#39;authors&#39;: &#39;Kariem Ezzat; Maria Pernemalm; Sandra Pålsson; Thomas C. Roberts; Peter Järver; Aleksandra Dondalska; Burcu Bestas; Michal J. Sobkowiak; Bettina Levänen; Magnus Sköld; Elizabeth A. Thompson; Osama Saher; Otto K. Kari; Tatu Lajunen; Eva Sverremark Ekström; Caroline Nilsson; Yevheniia Ishchenko; Tarja Malm; Matthew J.A. Wood; Ultan F. Power; Sergej Masich; Anders Lindén; Johan K. Sandberg; Janne Lehtiö; Anna-Lena Spetz; Samir EL Andaloussi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/246785&#39;},
 {&#39;cord_uid&#39;: &#39;bxqc9a7f&#39;,
  &#39;sha&#39;: &#39;4485b9462ebf52c0062692d7ab7e423c2ae05d9e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;HCMV glycoprotein B subunit vaccine efficacy was mediated by non-neutralizing antibody effector functions&#39;,
  &#39;doi&#39;: &#39;10.1101/246884&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractHuman cytomegalovirus (HCMV) is the most common congenital infection worldwide, frequently causing hearing loss and brain damage in afflicted infants. A vaccine to prevent maternal acquisition of HCMV during pregnancy is necessary to reduce the incidence of infant disease. The glycoprotein B (gB) + MF59 adjuvant subunit vaccine platform is the most successful HCMV vaccine tested to-date, demonstrating approximately 50% efficacy in preventing HCMV acquisition in phase II trials. However, the mechanism of vaccine protection remains unknown. Plasma from 33 gB/MF59 vaccinees at peak immunogenicity was tested for gB epitope specificity as well as neutralizing and non-neutralizing anti-HCMV effector functions, and compared to an HCMV-seropositive cohort. gB/MF59 vaccination elicited IgG responses with gB-binding magnitude and avidity comparable to natural infection. Additionally, IgG subclass distribution was similar with predominant IgG1 and IgG3 responses induced by gB vaccination and HCMV infection. However, vaccine-elicited antibodies exhibited limited neutralization of the autologous virus, negligible neutralization of multiple heterologous strains, and limited binding responses against gB structural motifs targeted by neutralizing antibodies including AD-1, AD-2, and Domain I. Interestingly, vaccinees had high-magnitude IgG responses against AD-3 linear epitopes, demonstrating immunodominance against this non-neutralizing, cytosolic region. Finally, vaccine-elicited IgG robustly bound trimeric, membrane-associated gB on the surface of transfected or HCMV-infected cells and mediated virion phagocytosis, though were poor mediators of NK cell activation. Altogether, these data suggest that non-neutralizing antibody functions, including virion phagocytosis, likely played a role in the observed 50% vaccine-mediated protection against HCMV acquisition.SignificanceThe CDC estimates that every hour, a child is born in the United States with permanent neurologic disability resulting from human cytomegalovirus (HCMV) infection – more than is caused by Down syndrome, fetal alcohol syndrome, and neural tube defects combined. A maternal vaccine to block transmission of HCMV to the developing fetus is a necessary intervention to prevent these adverse outcomes. The gB/MF59 vaccine is the most successful tested clinically to-date, achieving 50% reduction in HCMV acquisition. This manuscript establishes the function and epitope specificity of the humoral response stimulated by this vaccine that may explain the partial vaccine efficacy. Understanding the mechanism of gB/MF59-elicited protective immune responses will guide rational design and evaluation of the next generation of HCMV vaccines.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-11&#39;,
  &#39;authors&#39;: &#39;Cody S. Nelson; Tori Huffman; Eduardo Cisneros de la Rosa; Guanhua Xie; Nathan Vandergrift; Robert F. Pass; Justin Pollara; Sallie R. Permar&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/246884&#39;},
 {&#39;cord_uid&#39;: &#39;pxvapsdr&#39;,
  &#39;sha&#39;: &#39;4f392ef1aa2234dda52ab8e6540f275322a20697&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Coronavirus S protein-induced fusion is blocked prior to hemifusion by Abl kinase inhibitors&#39;,
  &#39;doi&#39;: &#39;10.1101/246991&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTEnveloped viruses gain entry into host cells by fusing with cellular membranes, a step required for virus replication. Coronaviruses, including the severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and infectious bronchitis virus (IBV), fuse at the plasma membrane or use receptor-mediated endocytosis and fuse with endosomes depending on the cell or tissue type. The virus Spike (S) protein mediates fusion with the host cell membrane. We have shown previously that an Abl kinase inhibitor, imatinib, significantly reduces SARS-CoV and MERS-CoV viral titers and prevents endosomal entry by HIV SARS S and MERS S pseudotyped virions. SARS-CoV and MERS-CoV are classified as BSL-3 viruses, which can make experimentation into the cellular mechanisms involved in infection more challenging. Here, we use IBV, a BSL-2 virus, as a model for studying the role of Abl kinase activity during coronavirus infection. We found that imatinib and two specific Abl kinase inhibitors, GNF2 and GNF5, reduce IBV titers by blocking the first round of virus infection. Additionally, all three drugs prevented IBV S-induced syncytia formation prior to the hemifusion step. Our results indicate that membrane fusion (both virus-cell and cell-cell) is blocked in the presence of Abl kinase inhibitors. Studying the effects of Abl kinase inhibitors on IBV will be useful in identifying host cell pathways required for coronavirus infection. This will provide insight into possible therapeutic targets to treat infections by current as well as newly emerging coronaviruses.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-12&#39;,
  &#39;authors&#39;: &#39;Jeanne M. Sisk; Matthew B. Frieman; Carolyn E. Machamer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/246991&#39;},
 {&#39;cord_uid&#39;: &#39;dzorkks5&#39;,
  &#39;sha&#39;: &#39;b5b029e65b963ae092dd88b21911e74b0cf557cc&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Microbiota-independent antiviral protection conferred by aminoglycoside antibiotics&#39;,
  &#39;doi&#39;: &#39;10.1101/248617&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAntibiotics are widely used to treat infections in humans. However, the impact of antibiotic use on host cells is understudied. We have identified a novel antiviral effect of commonly used aminoglycoside antibiotics. We show that mucosal application of aminoglycosides increased host resistance to a broad range of viral infections including herpes simplex viruses, influenza A virus and Zika virus. Aminoglycoside treatment also reduced viral replication in primary human cells. This antiviral activity was independent of the microbiota as aminoglycoside treatment protected germ-free mice. Microarray analysis uncovered a marked upregulation of transcripts for interferon-stimulated genes (ISGs) following aminoglycoside application. ISG induction was mediated by TLR3, and required TIR-domain-containing adapter-inducing interferon-β (TRIF), signaling adaptor, and interferon regulatory factors 3 (IRF3) and IRF7, transcription factors that promote ISG expression. XCR1+ dendritic cells, which uniquely express TLR3, were recruited to the vaginal mucosa upon aminoglycoside treatment and were required for ISG induction. These results highlight an unexpected ability of aminoglycoside antibiotics to confer broad antiviral resistance in vivo.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-16&#39;,
  &#39;authors&#39;: &#39;Smita Gopinath; Myoungjoo V. Kim; Tasfia Rakib; Patrick W. Wong; Michael van Zandt; Natasha A. Barry; Tsuneyasu Kaisho; Andrew L. Goodman; Akiko Iwasaki&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/248617&#39;},
 {&#39;cord_uid&#39;: &#39;maf96bof&#39;,
  &#39;sha&#39;: &#39;f8eee21acf147a7088fa376af6c3d85e606b47c4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences&#39;,
  &#39;doi&#39;: &#39;10.1101/249995&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMotivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.Availability: HCAtk and pyHCA are available at https://github.com/T-B-F/pyHCA under the CeCILL-C license.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-18&#39;,
  &#39;authors&#39;: &#39;Tristan Bitard-Feildel; Isabelle Callebaut&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/249995&#39;},
 {&#39;cord_uid&#39;: &#39;ox6japdt&#39;,
  &#39;sha&#39;: &#39;97a34a062a7ad09f7633d4a0e76d013b28b9b582&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The Evolutionary History and Impact of Bacterial tRNA Modifications&#39;,
  &#39;doi&#39;: &#39;10.1101/251322&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTAlong with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (“target”) tRNAs to recognize multiple codons and cover for other (“non-target”) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation.&#39;,
  &#39;publish_time&#39;: &#39;2018-01-22&#39;,
  &#39;authors&#39;: &#39;Gaurav D Diwan; Deepa Agashe&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/251322&#39;},
 {&#39;cord_uid&#39;: &#39;1vx62ofn&#39;,
  &#39;sha&#39;: &#39;c124701d09545e0117cca7a3a5f449584223899f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Generating genomic platforms to study Candida albicans pathogenesis&#39;,
  &#39;doi&#39;: &#39;10.1101/261628&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of Candida albicans, a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway™ donor vector, representing 83% of the currently annotated coding sequences of C. albicans. Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at http://candidaorfeome.eu. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the C. albicans ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway™-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in C. albicans and the elucidation of mechanisms that underlie its pathogenicity.&#39;,
  &#39;publish_time&#39;: &#39;2018-02-08&#39;,
  &#39;authors&#39;: &#39;Mélanie Legrand; Sophie Bachellier-Bassi; Keunsook K. Lee; Yogesh Chaudhari; Hélène Tournu; Laurence Arbogast; Hélène Boyer; Murielle Chauvel; Vitor Cabral; Corinne Maufrais; Audrey Nesseir; Irena Maslanka; Emmanuelle Permal; Tristan Rossignol; Louise A. Walker; Ute Zeidler; Sadri Znaidi; Floris Schoeters; Charlotte Majgier; Renaud A. Julien; Laurence Ma; Magali Tichit; Christiane Bouchier; Patrick Van Dijck; Carol A. Munro; Christophe d’Enfert&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/261628&#39;},
 {&#39;cord_uid&#39;: &#39;j97gul0w&#39;,
  &#39;sha&#39;: &#39;7d98a73314cd9f923aaa367f3f4de056b8cedec5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA&#39;,
  &#39;doi&#39;: &#39;10.1101/261776&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractInterferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5′ end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 in vitro, which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal ‘YxxxL’ motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important ‘missing link’ between IFIT interaction and function.&#39;,
  &#39;publish_time&#39;: &#39;2018-02-08&#39;,
  &#39;authors&#39;: &#39;Renata C Fleith; Harriet V Mears; Edward Emmott; Stephen C Graham; Daniel S Mansur; Trevor R Sweeney&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/261776&#39;},
 {&#39;cord_uid&#39;: &#39;8izuaesr&#39;,
  &#39;sha&#39;: &#39;46030e9b8451bf3b0c90833dbeacce286a263a09&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;An Augmented Aging Process in Brain White Matter in HIV&#39;,
  &#39;doi&#39;: &#39;10.1101/265199&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractObjectiveHIV infection and aging are both associated with neurodegeneration. However, whether the aging process alone or other factors associated with advanced age account for the progression of neurodegeneration in the aging HIV-positive (HIV+) population remains unclear.MethodsHIV+ (n=70) and HIV-negative (HIV-, n=34) participants underwent diffusion tensor imaging (DTI) and metrics of microstructural properties were extracted from regions of interest (ROIs). A support vector regression model was trained on two independent datasets of healthy adults across the adult life-span (n=765, Cam-CAN = 588; UiO = 177) to predict participant age from DTI metrics, and applied to the HIV dataset. Predicted brain age gap (BAG) was computed as the difference between predicted age and chronological age, and statistically compared between HIV groups. Regressions assessed the relationship between BAG and HIV severity/medical comorbidities. Finally, correlation analyses tested for associations between BAG and cognitive performance.ResultsBAG was significantly higher in the HIV+ group than the HIV-group F (1, 103) = 12.408, p = 0.001). HIV RNA viral load was significantly associated with BAG, particularly in older HIV+ individuals (R2 = 0.29, F(7, 70) = 2.66, p = 0.021). Further, BAG was negatively correlated with domain-level cognitive function (learning: r = −0.26, p = 0.008; memory: r = −0.21, p = 0.034).ConclusionsHIV infection is associated with augmented white matter aging, and greater brain aging is associated with worse cognitive performance in multiple domains.&#39;,
  &#39;publish_time&#39;: &#39;2018-02-14&#39;,
  &#39;authors&#39;: &#39;T. Kuhn; T. Kaufmann; N.T. Doan; L.T. Westlye; J. Jones; R.A. Nunez; S.Y. Bookheimer; E.J. Singer; C.H. Hinkin; A.D. Thames&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/265199&#39;},
 {&#39;cord_uid&#39;: &#39;o9decbb4&#39;,
  &#39;sha&#39;: &#39;4151e9b4ea5395d73bf33309dbca26551cc7c657&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Gene expression is stronger associated with behaviour than with age and fertility in ant workers&#39;,
  &#39;doi&#39;: &#39;10.1101/267336&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe ecological success of social insects is based on division of labour, not only between queens and workers, but also among workers. Whether a worker tends the brood or forages is strongly influenced by age, fertility and nutritional status, with brood carers being younger, more fecund and corpulent. Here, we experimentally disentangle behaviour from age and fertility in Temnothorax longispinosus ant workers and analyse how these parameters are linked to whole-body gene expression. Our transcriptome analysis reveals four times more genes associated with behaviour than with age and only few fertility-associated genes. Brood carers exhibited an upregulation of genes involved in lipid biosynthesis, whereas foragers invested in metabolism. Additional simulations revealed that the experimental disassociation of co-varying factors reduces transcriptomic noise, potentially explaining discrepancies between transcriptomic studies on worker behaviour in other social insects. Our study highlights the influence of nutritional status on task choice in ant workers.&#39;,
  &#39;publish_time&#39;: &#39;2018-02-18&#39;,
  &#39;authors&#39;: &#39;Philip Kohlmeier; Austin R Alleman; Romain Libbrecht; Susanne Foitzik; Barbara Feldmeyer&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/267336&#39;},
 {&#39;cord_uid&#39;: &#39;7jlh5xff&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Simultaneous detection of DNA and RNA virus species involved in bovine respiratory disease by PCR-free rapid tagmentation-based library preparation and MinION nanopore sequencing&#39;,
  &#39;doi&#39;: &#39;10.1101/269936&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstactThe Oxford Nanopore MinION Mk1B is a portable 90 g device that sequences DNA directly at 450 bases/second generating sequence reads in excess of 400 kb. Recent improvements in error rate and speed of library preparation mean that this device has considerable potential for rapid molecular bovine pathogen diagnostics. We tested the MinION for rapid untargeted detection of viral pathogens associated with bovine respiratory disease (BRD), an economically important disease often involving primary infection of the lung by one or more of a number of DNA and/or RNA viruses. We combined three foetal lung cell cultures which were infected with either Bovine Respiratory Syncytial Virus (BRSV), Bovine Herpes Virus 1 (BoHV1) or Bovine Parainfluenza Virus 3 (BPI-3). BoHV1 is a DNA virus and BPI-3 and BRSV are RNA viruses. The cell cultures were treated with DNase and RNase to deplete bovine nucleic acid prior to viral nucleic acid extraction and double-stranded cDNA synthesis. Sequencing libraries were generated by PCR-free tagmentation in under 10 minutes and loaded onto a MinION sequencer. Approximately 7,000 sequencing reads were generated and analysed using high-throughput local BLAST against the NCBI nr/nt database. The top BLAST hit for 2,937 of these reads was identified as a virus. Of these, 2,926 (99.6%) were correctly identified either as BoHV1, BRSV or BPI-3.&#39;,
  &#39;publish_time&#39;: &#39;2018-02-22&#39;,
  &#39;authors&#39;: &#39;Matthew S. McCabe; Paul Cormican; Dayle Johnston; Bernadette Earley&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/269936&#39;},
 {&#39;cord_uid&#39;: &#39;d3y0ni9q&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors&#39;,
  &#39;doi&#39;: &#39;10.1101/271171&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractReceptor mediated entry is the first step for viral infection. However, the relationship between viruses and receptors is still obscure. Here, by manually curating a high-quality database of 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we found the viral receptors were structurally and functionally diverse, yet they had several common features when compared to other cell membrane proteins: more protein domains, higher level of N-glycosylation, higher ratio of self-interaction and more interaction partners, and higher expression in most tissues of the host. Additionally, the receptors used by the same virus tended to co-evolve. Further correlation analysis between viral receptors and the tissue and host specificity of the virus shows that the virus receptor similarity was a significant predictor for mammalian virus cross-species. This work could deepen our understanding towards the viral receptor selection and help evaluate the risk of viral zoonotic diseases.&#39;,
  &#39;publish_time&#39;: &#39;2018-02-25&#39;,
  &#39;authors&#39;: &#39;Zheng Zhang; Zhaozhong Zhu; Wenjun Chen; Zena Cai; Beibei Xu; Zhiying Tan; Aiping Wu; Xingyi Ge; Xinhong Guo; Zhongyang Tan; Zanxian Xia; Haizhen Zhu; Taijiao Jiang; Yousong Peng&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/271171&#39;},
 {&#39;cord_uid&#39;: &#39;drqseaaa&#39;,
  &#39;sha&#39;: &#39;41d412415c46baf70ca91ebf049c0352ead19f18&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;BAR scaffolds drive membrane fission by crowding disordered domains&#39;,
  &#39;doi&#39;: &#39;10.1101/276147&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryCylindrical protein scaffolds are thought to stabilize membrane tubules, preventing membrane fission. In contrast, Snead et al. find that when scaffold proteins assemble, bulky disordered domains within them become acutely concentrated, generating steric pressure that destabilizes tubules, driving fission.AbstractCellular membranes are continuously remodeled. The crescent-shaped bin-amphiphysinrvs (BAR) domains remodel membranes in multiple cellular pathways. Based on studies of BAR domains in isolation, the current paradigm is that they polymerize into cylindrical scaffolds that stabilize lipid tubules, preventing membrane fission. But in nature BAR domains are often part of multi-domain proteins that contain large intrinsically-disordered regions. Using in vitro and live cell assays, here we show that full-length BAR domain-containing proteins, rather than stabilizing membrane tubules, are instead surprisingly potent drivers of membrane fission. Specifically, when BAR scaffolds assemble at membrane surfaces, their bulky disordered domains become crowded, generating steric pressure that destabilizes lipid tubules. More broadly, we observe this behavior with BAR domains that have a range of curvatures. These data challenge the idea that cellular membranes adopt the curvature of BAR scaffolds, suggesting instead that the ability to concentrate disordered domains is the key requirement for membrane remodeling and fission by BAR domain-containing proteins.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-04&#39;,
  &#39;authors&#39;: &#39;Wilton T. Snead; Wade F. Zeno; Grace Kago; Ryan W. Perkins; J Blair Richter; Chi Zhao; Eileen M. Lafer; Jeanne C. Stachowiak&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/276147&#39;},
 {&#39;cord_uid&#39;: &#39;a9lkhayg&#39;,
  &#39;sha&#39;: &#39;5a843f4299b04090ff76451decfe9f0b025cfe48&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Capturing diverse microbial sequence with comprehensive and scalable probe design&#39;,
  &#39;doi&#39;: &#39;10.1101/279570&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMetagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. We developed CATCH (Compact Aggregation of Targets for Comprehensive Hybridization), a computational method to enhance nucleic acid capture for enrichment of diverse microbial taxa. CATCH designs compact probe sets that achieve full coverage of known sequence diversity and that scale well with this diversity. To illustrate applications of CATCH, we focused on capturing viral genomes. We designed, synthesized, and validated multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriched unique viral content on average 18× and allowed us to assemble genomes that we could not otherwise recover, while accurately preserving within-sample diversity. We used this approach to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of viral infections in samples with unknown content. Together, this work demonstrates a path toward more sensitive, cost-effective metagenomic sequencing.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-12&#39;,
  &#39;authors&#39;: &#39;Hayden C. Metsky; Katherine J. Siddle; Adrianne Gladden-Young; James Qu; David K. Yang; Patrick Brehio; Andrew Goldfarb; Anne Piantadosi; Shirlee Wohl; Amber Carter; Aaron E. Lin; Kayla G. Barnes; Damien C. Tully; Björn Corleis; Scott Hennigan; Giselle Barbosa-Lima; Yasmine R. Vieira; Lauren M. Paul; Amanda L. Tan; Kimberly F. Garcia; Leda A. Parham; Ikponmwonsa Odia; Philomena Eromon; Onikepe A. Folarin; Augustine Goba; Etienne Simon-Lorière; Lisa Hensley; Angel Balmaseda; Eva Harris; Douglas Kwon; Todd M. Allen; Jonathan A. Runstadler; Sandra Smole; Fernando A. Bozza; Thiago M. L. Souza; Sharon Isern; Scott F. Michael; Ivette Lorenzana; Lee Gehrke; Irene Bosch; Gregory Ebel; Donald Grant; Christian Happi; Daniel J. Park; Andreas Gnirke; Pardis C. Sabeti; Christian B. Matranga&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/279570&#39;},
 {&#39;cord_uid&#39;: &#39;ajkhpw5f&#39;,
  &#39;sha&#39;: &#39;05082393ba4c7ec530190dd887d99c74fd72f6d6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Self-assembly of the RZZ complex into filaments drives kinetochore expansion in the absence of microtubule attachment&#39;,
  &#39;doi&#39;: &#39;10.1101/282707&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SUMMARYThe kinetochore is a dynamic multi-protein assembly that forms on each sister chromatid and interacts with microtubules of the mitotic spindle to drive chromosome segregation. In animals, kinetochores without attached microtubules expand their outermost layer into crescent and ring shapes to promote microtubule capture and spindle assembly checkpoint (SAC) signalling. Kinetochore expansion is an example of protein co-polymerization, but the mechanism is not understood. Here, we present evidence that kinetochore expansion is driven by oligomerization of the Rod-Zw10-Zwilch (RZZ) complex, an outer kinetochore component that recruits the motor dynein and the SAC proteins Mad1-Mad2. Depletion of ROD in human cells suppresses kinetochore expansion, as does depletion of Spindly, the adaptor that connects RZZ to dynein, while dynein itself is dispensable. Expansion is also suppressed by mutating ZWILCH residues implicated in Spindly binding. Conversely, supplying cells with excess ROD facilitates kinetochore expansion under otherwise prohibitive conditions. Using the C. elegans early embryo, we demonstrate that ROD-1 has a concentration-dependent propensity for oligomerizing into µm-scale filaments, and we identify the ROD-1 β-propeller as a key regulator of self-assembly. Finally, we show that a minimal ROD-1-Zw10 complex efficiently oligomerizes into filaments in vitro. Our results suggest that RZZ’s capacity for oligomerization is harnessed by kinetochores to assemble the expanded outermost domain, in which RZZ filaments serve as recruitment platforms for SAC components and microtubule-binding proteins. Thus, we propose that RZZ self-assembly into filaments underlies the adaptive change in kinetochore size that contributes to chromosome segregation fidelity.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-15&#39;,
  &#39;authors&#39;: &#39;Cláudia Pereira; Rita M. Reis; José B. Gama; Dhanya K. Cheerambathur; Ana X. Carvalho; Reto Gassmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/282707&#39;},
 {&#39;cord_uid&#39;: &#39;jb447ndz&#39;,
  &#39;sha&#39;: &#39;df6013a1630aa41627cc84c8e780d40ff93785cc&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A method for RNA structure prediction shows evidence for structure in lncRNAs&#39;,
  &#39;doi&#39;: &#39;10.1101/284869&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTo compare the secondary structures of RNA molecules we developed the CROSSalign method. CROSSalign is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied CROSSalign to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure CROSSalign accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. CROSSalign can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage http://service.tartaglialab.com//new_submission/CROSSalign.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-16&#39;,
  &#39;authors&#39;: &#39;Riccardo Delli ponti; Alexandros Armaos; Stefanie Marti; Gian Gaetano Tartaglia&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/284869&#39;},
 {&#39;cord_uid&#39;: &#39;ct2zhauz&#39;,
  &#39;sha&#39;: &#39;1eaa329f608055620a57e6273e9d1c409de1e9ee&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Structure of the chromatin remodelling enzyme Chd1 bound to a ubiquitinylated nucleosome&#39;,
  &#39;doi&#39;: &#39;10.1101/290874&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractATP-dependent chromatin remodelling proteins represent a diverse family of proteins that share ATPase domains that are adapted to regulate protein-DNA interactions. Here we present structures of the yeast Chd1 protein engaged with nucleosomes in the presence of the transition state mimic ADP-beryllium fluoride. The path of DNA strands through the ATPase domains indicates the presence of contacts conserved with single strand translocases and additional contacts with both strands that are unique to Snf2 related proteins. The structure provides connectivity between rearrangement of ATPase lobes to a closed, nucleotide bound state and the sensing of linker DNA. Two turns of linker DNA are prised off the surface of the histone octamer as a result of Chd1 binding, and both the histone H3 tail and ubiquitin conjugated to lysine 120 are re-orientated towards the unravelled DNA. This indicates how changes to nucleosome structure can alter the way in which histone epitopes are presented.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-30&#39;,
  &#39;authors&#39;: &#39;Ramasubramanian Sundaramoorthy; Amanda L. Hughes; Hassane El-Mkami; David Norman; Tom Owen-Hughes&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/290874&#39;},
 {&#39;cord_uid&#39;: &#39;dbd2ndxa&#39;,
  &#39;sha&#39;: &#39;b1ce6ca5960cb799403188ca303a364428f812d8&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Direct nucleic acid analysis of mosquitoes for high fidelity species identification and detection of Wolbachia using a cellphone&#39;,
  &#39;doi&#39;: &#39;10.1101/291849&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractManipulation of natural mosquito populations using the endosymbiotic bacteria Wolbachia is being investigated as a novel strategy to reduce the burden of mosquito-borne viruses. To evaluate the efficacy of these interventions, it will be critical to determine Wolbachia infection frequencies in Aedes aegypti mosquito populations. However, current diagnostic tools are not well-suited to fit this need. Morphological methods cannot identify Wolbachia, immunoassays often suffer from low sensitivity and poor throughput, while PCR and spectroscopy require complex instruments and technical expertise, which restrict their use to centralized laboratories. To address this unmet need, we have used loop-mediated isothermal amplification (LAMP) and oligonucleotide strand displacement (OSD) probes to create a one-pot sample-to-answer nucleic acid diagnostic platform for vector and symbiont surveillance. LAMP-OSD assays can directly amplify target nucleic acids from macerated mosquitoes without requiring nucleic acid purification and yield specific single endpoint yes/no fluorescence signals that are observable to eye or by cellphone camera. We demonstrate cellphone-imaged LAMP-OSD tests for two targets, the Aedes aegypti cytochrome oxidase I (coi) gene and the Wolbachia surface protein (wsp) gene, and show a limit of detection of 4 and 40 target DNA copies, respectively. In a blinded test of 90 field-caught mosquitoes, the coi LAMP-OSD assay demonstrated 98% specificity and 97% sensitivity in identifying Ae. aegypti mosquitoes even after 3 weeks of storage without desiccant at 37 °C. Similarly, the wsp LAMP-OSD assay readily identified the wAlbB Wolbachia strain in field-collected Aedes albopictus mosquitoes without generating any false positive signals. Modest technology requirements, minimal execution steps, simple binary readout, and robust accuracy make the LAMP-OSD-to-cellphone assay platform well suited for field vector surveillance in austere or resource-limited conditions.Author summaryMosquitoes spread many human pathogens and novel approaches are required to reduce the burden of mosquito-borne disease. One promising approach is transferring Wolbachia into Aedes aegypti mosquitoes where it blocks transmission of arboviruses like dengue, Zika and Yellow fever viruses and spreads through mosquito populations. For effective evaluation of this approach, regular surveillance of Wolbachia infections in Ae. aegypti is required, but current diagnostic tools are not well suited to support these critical surveillance needs. To fill this need we developed a simple, robust and inexpensive assay to identify Ae. aegypti mosquitoes and Wolbachia using our unique one-pot assay platform, LAMP-OSD, which uses loop-mediated isothermal amplification to amplify nucleic acid targets at a single temperature. Unlike other LAMP-based tests, our assays assure accuracy by coupling amplification with novel nucleic acid strand displacement (OSD) probes that hybridize to specific sequences in LAMP amplification products and thereby generate simple yes/no readout of fluorescence readable by human eye and by off-the-shelf cellphones. To facilitate field use, we developed our assays so they are compatible with crushed mosquito homogenate as the template, meaning no nucleic acid extraction is required. In blinded tests using field collected mosquitoes, LAMP-OSD-cellphone tests performed robustly to identify 29 of 30 Ae. aegypti even after 3 weeks of storage at 37 °C while producing only one false positive out of 60 non-specific mosquitoes. Similarly, our assay could identify Wolbachia in field-caught Aedes albopictus without producing any false positives. Our easy to use and easy to interpret assays should facilitate widespread field mosquito surveillance with minimal instrumentation and high accuracy.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-29&#39;,
  &#39;authors&#39;: &#39;Sanchita Bhadra; Timothy E. Riedel; Miguel A. Saldaña; Shivanand Hegde; Nicole Pederson; Grant L. Hughes; Andrew D. Ellington&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/291849&#39;},
 {&#39;cord_uid&#39;: &#39;28nlawnb&#39;,
  &#39;sha&#39;: &#39;331a7033bb8e6d4618e4e83b74aff1aec42764a6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Pulmonary Metagenomic Sequencing Suggests Missed Infections in Immunocompromised Children&#39;,
  &#39;doi&#39;: &#39;10.1101/291864&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTRATIONALEDespite improved diagnostics, pulmonary pathogens in immunocompromised children frequently evade detection, leading to significant morbidity and mortality.OBJECTIVESTo develop a highly sensitive metagenomic next generation sequencing (mNGS) assay capable of evaluating the pulmonary microbiome and identifying diverse pathogens in the lungs of immunocompromised children.METHODSWe collected 41 lower respiratory specimens from 34 immunocompromised children undergoing evaluation for pulmonary disease at 3 children’s hospitals from 2014-2016. Samples underwent mechanical homogenization, paired RNA/DNA extraction, and metagenomic sequencing. Sequencing reads were aligned to the NCBI nucleotide reference database to determine taxonomic identities. Statistical outliers were determined based on abundance within each sample and relative to other samples in the cohort.MEASUREMENTS &amp; MAIN RESULTSWe identified a rich cross-domain pulmonary microbiome containing bacteria, fungi, RNA viruses, and DNA viruses in each patient. Potentially pathogenic bacteria were ubiquitous among samples but could be distinguished as possible causes of disease by parsing for outlier organisms. Samples with bacterial outliers had significantly depressed alpha-diversity (median 0.58, IQR 0.33-0.62 vs. median 0.94, IQR 0.93-0.95, p&lt;0.001). Potential pathogens were detected in half of samples previously negative by clinical diagnostics, demonstrating increased sensitivity for missed pulmonary pathogens (p&lt;0.001).CONCLUSIONSAn optimized mNGS assay for pulmonary microbes demonstrates significant inoculation of the lower airways of immunocompromised children with diverse bacteria, fungi, and viruses. Potential pathogens can be identified based on absolute and relative abundance. Ongoing investigation is needed to determine the pathogenic significance of outlier microbes in the lungs of immunocompromised children with pulmonary disease.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-29&#39;,
  &#39;authors&#39;: &#39;MS Zinter; CC Dvorak; MY Mayday; K Iwanaga; NP Ly; ME McGarry; GD Church; LE Faricy; CM Rowan; JR Hume; ME Steiner; ED Crawford; C Langelier; K Kalantar; ED Chow; S Miller; K Shimano; A Melton; GA Yanik; A Sapru; JL DeRisi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/291864&#39;},
 {&#39;cord_uid&#39;: &#39;7w334xgt&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike&#39;,
  &#39;doi&#39;: &#39;10.1101/292672&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSevere acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S undergoes two proteolytic cleavages at S1/S2 and S2’ sites necessary for efficient membrane fusion. Here, we present a cryo-EM analysis of the trimeric SARS-CoV S interactions with ACE2 and of the trypsin-cleaved S. Surprisingly, neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within S or expose the secondary cleavage site, S2’. These observations suggest that S2’ cleavage does not occur in the S prefusion conformation and that additional triggers may be required.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-31&#39;,
  &#39;authors&#39;: &#39;Robert N. Kirchdoerfer; Nianshuang Wang; Jesper Pallesen; Daniel Wrapp; Hannah L. Turner; Christopher A. Cottrell; Kizzmekia S. Corbett; Barney S. Graham; Jason S. McLellan; Andrew B. Ward&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/292672&#39;},
 {&#39;cord_uid&#39;: &#39;hxugy10i&#39;,
  &#39;sha&#39;: &#39;7c17c6f5e434f1d8227c9d8c5f9da26200364491&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Probing the unfolded protein response to mouse hepatitis coronavirus infection through RNA sequencing and ribosome profiling&#39;,
  &#39;doi&#39;: &#39;10.1101/292979&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractCoronaviruses (CoVs) are enveloped, positive-sense RNA viruses with an unusually large RNA genome and a unique replication strategy. They cause important diseases in mammals and birds ranging from enteritis in cows and pigs and upper respiratory disease in chickens, to potentially lethal human respiratory infections. Here, we apply ribosome profiling and parallel RNA sequencing to analyse global changes in host cell transcriptome and translatome upon infection with mouse hepatitis virus strain A59 (MHV-A59), a model murine coronavirus in the same genus as the human pathogens severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). Amongst differentially-regulated cellular genes, we observed up-regulation of all arms of the unfolded protein response (UPR), including translational activation of transcription factors ATF4, ATF5 and Chop. Polysome profiling of infected-cells revealed an accumulation of empty 80S ribosomes, consistent with increased phosphorylation of eIF2α leading to translational shut-off via inhibited initiation. Ribosomal footprints on phosphorylated-eIF2α-resistant mRNAs revealed unambiguous upstream open reading frame (uORF) occupancy consistent with host maintenance of the UPR. Unexpectedly, an inhibitor of PERK that blocks the UPR and relieves translation inhibition was found to attenuate virus growth suggesting that MHV may subvert the UPR to its own advantage. This study sheds new light on the complex interactions between MHV and host during infection and provides new potential targets for antiviral intervention.&#39;,
  &#39;publish_time&#39;: &#39;2018-03-31&#39;,
  &#39;authors&#39;: &#39;Georgia M. Cook; Katherine Brown; Krzysztof Franaszek; Nathan A. Moore; Stuart G. Siddell; Ian Brierley; Andrew E. Firth; Nerea Irigoyen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/292979&#39;},
 {&#39;cord_uid&#39;: &#39;6s27v6at&#39;,
  &#39;sha&#39;: &#39;58bc0f2712ea1ced1f886c9fdd17b8cf014e66cb&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The Influence Of Social Behavior On Competition Between Virulent Pathogen Strains&#39;,
  &#39;doi&#39;: &#39;10.1101/293936&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;1AbstractInfectious disease interventions like contact precautions and vaccination have proven effective in disease control and elimination. The priority given to interventions can depend strongly on how virulent the pathogen is, and interventions may also depend partly for their success on social processes that respond adaptively to disease dynamics. However, mathematical models of competition between pathogen strains with differing natural history profiles typically assume that human behaviour is fixed. Here, our objective is to model the influence of social behaviour on the competition between pathogen strains with differing virulence. We couple a compartmental Susceptible-Infectious-Recovered model for a resident pathogen strain and a mutant strain with higher virulence, with a differential equation of a population where individuals learn to adopt protective behaviour from others according to the prevalence of infection of the two strains and the perceived severity of the respective strains in the population. We perform invasion analysis, time series analysis and phase plane analysis to show that perceived severities of pathogen strains and the efficacy of infection control against them can greatly impact the invasion of more virulent strain. We demonstrate that adaptive social behaviour enables invasion of the mutant strain under plausible epidemiological scenarios, even when the mutant strain has a lower basic reproductive number than the resident strain. Surprisingly, in some situations, increasing the perceived severity of the resident strain can facilitate invasion of the more virulent mutant strain. Our results demonstrate that for certain applications, it may be necessary to include adaptive social behaviour in models of the emergence of virulent pathogens, so that the models can better assist public health efforts to control infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-04&#39;,
  &#39;authors&#39;: &#39;Joe Pharaon; Chris T. Bauch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/293936&#39;},
 {&#39;cord_uid&#39;: &#39;1pbshnw9&#39;,
  &#39;sha&#39;: &#39;ec6adb9dc121bcbb8215cb16b880d6dc6497d209&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo&#39;,
  &#39;doi&#39;: &#39;10.1101/294488&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTMicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &gt; 350 pre-miRNAs in human cells and &gt; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3’ ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5’ UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-06&#39;,
  &#39;authors&#39;: &#39;Ian M. Silverman; Sager J. Gosai; Nicholas Vrettos; Shawn W. Foley; Nathan D. Berkowitz; Zissimos Mourelatos; Brian D. Gregory&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/294488&#39;},
 {&#39;cord_uid&#39;: &#39;tj1ohvh0&#39;,
  &#39;sha&#39;: &#39;a3dcd6b3f0551e607d0235616109ab8d506f81a2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Analyzing Vaccine Trials in Epidemics with Mild and Asymptomatic Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/295337&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTVaccine efficacy against susceptibility to infection (VES), regardless of symptoms, is an important endpoint of vaccine trials for pathogens with a high proportion of asymptomatic infection, as such infections may contribute to onward transmission and outcomes such as Congenital Zika Syndrome. However, estimating VESis resource-intensive. We aim to identify methods to accurately estimate VEswhen limited information is available and resources are constrained. We model an individually randomized vaccine trial by generating a network of individuals and simulating an epidemic. The disease natural history follows a Susceptible, Exposed, Infectious and Symptomatic or Infectious and Asymptomatic, Recovered model. We then use seven approaches to estimate VES, and we also estimate vaccine efficacy against progression to symptoms (VEP). A corrected relative risk and an interval censored Cox model accurately estimate VESand only require serologic testing of participants once, while a Cox model using only symptomatic infections returns biased estimates. Only acquiring serological endpoints in a 10% sample and imputing the remaining infection statuses yields unbiased VESestimates across values of R0and accurate estimates of VEPfor higher values. Identifying resource-preserving methods for accurately estimating VESis important in designing trials for diseases with a high proportion of asymptomatic infection.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-06&#39;,
  &#39;authors&#39;: &#39;Rebecca Kahn; Matt Hitchings; Rui Wang; Steven Bellan; Marc Lipsitch&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/295337&#39;},
 {&#39;cord_uid&#39;: &#39;iv8c6eda&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Changes in mRNA abundance drive shuttling of RNA binding proteins, linking cytoplasmic RNA degradation to transcription&#39;,
  &#39;doi&#39;: &#39;10.1101/295972&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAlterations in global mRNA decay broadly impact multiple stages of gene expression, although signals that connect these processes are incompletely defined. Here, we used tandem mass tag labeling coupled with mass spectrometry to reveal that changing the mRNA decay landscape, as frequently occurs during viral infection, results in subcellular redistribution of RNA binding proteins (RBPs) in human cells. Accelerating Xrn1-dependent mRNA decay through expression of a gammaherpesviral endonuclease drove nuclear translocation of many RBPs, including poly(A) tail-associated proteins. Conversely, cells lacking Xrn1 exhibited changes in the localization or abundance of numerous factors linked to mRNA turnover. Using these data, we uncovered a new role for relocalized cytoplasmic poly(A) binding protein in repressing recruitment of TATA binding protein and RNA polymerase II to promoters. Collectively, our results show that changes in cytoplasmic mRNA decay can directly impact protein localization, providing a mechanism to connect seemingly distal stages of gene expression.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-06&#39;,
  &#39;authors&#39;: &#39;Sarah Gilbertson; Joel D. Federspiel; Ella Hartenian; Ileana M. Cristea; Britt Glaunsinger&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/295972&#39;},
 {&#39;cord_uid&#39;: &#39;mozfm5ds&#39;,
  &#39;sha&#39;: &#39;b5e1d36298a78690140d5ccb9673dbd48a0ddcd3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The transcriptional and translational landscape of equine torovirus&#39;,
  &#39;doi&#39;: &#39;10.1101/296996&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe genus Torovirus (subfamily Torovirinae, family Coronaviridae, order Nidovirales) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5’ UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.ImportanceToroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames “hidden” within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-07&#39;,
  &#39;authors&#39;: &#39;Hazel Stewart; Katherine Brown; Adam M. Dinan; Nerea Irigoyen; Eric J. Snijder; Andrew E. Firth&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/296996&#39;},
 {&#39;cord_uid&#39;: &#39;pxvdx7p2&#39;,
  &#39;sha&#39;: &#39;4eb8b7fd0032816e4a29d65b06939266d6446624&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Encoding of odor fear memories in the mouse olfactory cortex&#39;,
  &#39;doi&#39;: &#39;10.1101/297226&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractOdor memories are exceptionally robust and essential for animal survival. The olfactory (piriform) cortex has long been hypothesized to encode odor memories, yet the cellular substrates for olfactory learning and memory remain unknown. Here, using intersectional, cFos-based genetic manipulations (“Fos-tagging”), we show that olfactory fear conditioning activates sparse and distributed ensembles of neurons in mouse piriform cortex. We demonstrate that chemogenetic silencing of these Fos-tagged piriform ensembles selectively interferes with odor fear memory retrieval, but does not compromise basic odor detection and discrimination. Furthermore, chemogenetic reactivation of piriform neurons that were Fos-tagged during olfactory fear conditioning causes a decrease in exploratory behavior, mimicking odor-evoked fear memory recall. Together, our experiments identify odor-specific ensembles of piriform neurons as necessary and sufficient for odor fear memory recall.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-07&#39;,
  &#39;authors&#39;: &#39;Claire Meissner-Bernard; Yulia Dembitskaya; Laurent Venance; Alexander Fleischmann&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/297226&#39;},
 {&#39;cord_uid&#39;: &#39;0c4cbwqq&#39;,
  &#39;sha&#39;: &#39;eb5e5088c41f976387784acd57dbb683932c6002&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The RZZ complex integrates spindle checkpoint maintenance with dynamic expansion of unattached kinetochores&#39;,
  &#39;doi&#39;: &#39;10.1101/297580&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;SummaryThe Mad1-Mad2 heterodimer is the catalytic hub of the spindle assembly checkpoint (SAC), which controls mitosis through assembly of a multi-subunit anaphase inhibitor, the mitotic checkpoint complex (MCC) [1, 2]. Mad1-Mad2 first catalyzes MCC assembly at interphase nuclear pores [3], then migrates to kinetochores at nuclear envelope breakdown (NEBD) and resumes MCC assembly until bipolar spindle attachment is complete [1, 2]. There is significant debate about the factor(s) involved in targeting Mad1-Mad2 to kinetochores in higher eukaryotes [4-9]. Through gene editing and live-cell imaging, we found that the human Rod-Zw10-Zwilch (RZZ) complex is dispensable for cell viability and initial recruitment of Mad1-Mad2 to kinetochores at NEBD, but then becomes necessary to tether Mad1-Mad2 at kinetochores and sustain SAC arrest in cells challenged with spindle poisons. We also show that RZZ forms the mesh-like fibrous corona, a structural expansion of the outer kinetochore important for timely chromosome congression [10-13] once Mps1 phosphorylates the N-terminus of Rod. Artificially tethering Mad1-Mad2 to kinetochores enabled long-term mitotic arrest in the absence of RZZ. Conversely, blocking early RZZ-independent recruitment of Mad1-Mad2 eliminated the transient SAC response in RZZ-null cells. We conclude that RZZ drives structural changes in the outer kinetochore that facilitate chromosome bi-orientation and chronic SAC transduction, a key determinant of cytotoxicity during anti-mitotic drug therapy [14-16].&#39;,
  &#39;publish_time&#39;: &#39;2018-04-09&#39;,
  &#39;authors&#39;: &#39;Jose-Antonio Rodriguez-Rodriguez; Kara L. McKinley; Vitali Sikirzhytski; Jennifer Corona; John Maciejowski; Alexey Khodjakov; Iain M. Cheeseman; Prasad V. Jallepalli&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/297580&#39;},
 {&#39;cord_uid&#39;: &#39;gwgflckq&#39;,
  &#39;sha&#39;: &#39;12c8ef5614d9a35bed3ff95a3b1dc971842514fe&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A planarian nidovirus expands the limits of RNA genome size&#39;,
  &#39;doi&#39;: &#39;10.1101/299776&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV’s greatly expanded genome, proteomic complexity, and unique features – impressive in themselves – attest to the likelihood of still-larger RNA genomes awaiting discovery.Significance StatementRNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-11&#39;,
  &#39;authors&#39;: &#39;Amir Saberi; Anastasia A. Gulyaeva; John L. Brubacher; Phillip A. Newmark; Alexander E. Gorbalenya&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/299776&#39;},
 {&#39;cord_uid&#39;: &#39;u1r0umpj&#39;,
  &#39;sha&#39;: &#39;0d7964c665ff0107c674bc6dab8f252cab08038e&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A transmissible RNA pathway in honey bees&#39;,
  &#39;doi&#39;: &#39;10.1101/299800&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTOne of the characteristics of RNA interference (RNAi) is systemic spread of the silencing signal among cells and tissues throughout the organism. Systemic RNAi, initiated by double-stranded RNA (dsRNA) ingestion, has been reported in diverse invertebrates, including honey bees, demonstrating environmental RNA uptake that undermines homologous gene expression. However, the question why any organism would take up RNA from the environment has remained largely unanswered. Here, we report on horizontal RNA flow among honey bees mediated by secretion and ingestion of worker and royal jelly diets. We show that ingested dsRNA spreads through the bee’s hemolymph associated with a protein complex. The systemic dsRNA is secreted with the jelly and delivered to larvae via ingestion. Furthermore, we demonstrate that transmission of jelly-secreted dsRNA to larvae is biologically active and triggers gene knockdown that lasts into adulthood. Finally, RNA extracted from worker and royal jellies harbor differential naturally occurring RNA populations. Some of these RNAs corresponded to honey bee protein coding genes, transposable elements, non-coding RNA as well as bacteria, fungi and viruses. These results reveal an inherent property of honey bees to share RNA among individuals and generations. Thus, our findings suggest a transmissible RNA pathway, playing a role in social immunity and epigenetic signaling between honey bees and potentially among other closely interacting organisms.SIGNIFICANCEHoney bees are eusocial insects, living in a colony that is often described as a superorganism. RNA mobility among cells of an organism has been documented in plants and animals. Here we show that RNA spreads further in honey bees, and is horizontally transferred between individuals and across generations. We found that honey bees share biologically active RNA through secretion and ingestion of worker and royal jellies. Such RNA initiates RNA interference, which is a known defense mechanism against viral infection. Furthermore, we characterized diverse RNA profiles of worker and royal jelly, including fragmented viral RNA. Our findings demonstrate a transmissible RNA pathway with potential roles in social immunity and epigenetic signaling among members of the hive.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-12&#39;,
  &#39;authors&#39;: &#39;Eyal Maori; Yael Garbian; Vered Kunik; Rita Mozes-Koch; Osnat Malka; Haim Kalev; Niv Sabath; Ilan Sela; Sharoni Shafir&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/299800&#39;},
 {&#39;cord_uid&#39;: &#39;mrpj8l3q&#39;,
  &#39;sha&#39;: &#39;5502de277b6ee34f29d813cc49d7cda1168c202d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;CHOP and IRE1α-XBP1/JNK signaling promote Newcastle Disease Virus induced apoptosis and benefit virus proliferation&#39;,
  &#39;doi&#39;: &#39;10.1101/300129&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTNewcastle disease virus (NDV) causes severe infectious disease in poultry, and selectively kills tumor cells by inducing apoptosis. In this report, we revealed the mechanisms underlying NDV-induced apoptosis via investigation of endoplasmic reticulum (ER) stress-related unfolded protein response (UPR) in HeLa cells. We found that NDV infection induced the expression of pro-apoptotic transcription factor CHOP via PKR-eIF2α pathway. Knock down and exogenous expression studies showed that CHOP promoted cell apoptosis by down-regulation of anti-apoptotic protein BCL-2 and MCL-1, promotion of pro-apoptotic JNK and p38 signaling, and suppression of pro-survival AKT signaling. Meanwhile, CHOP facilitated NDV proliferation. Furthermore, virus infection activated IRE1α, another ER stress sensor, thereby promoting the mRNA splicing of XBP1 and resulting in the translation of transcription factor XBP1s. XBP1s entered into cell nucleus, promoted the expression of ER chaperones and components of ER associated degradation (ERAD). Exogenous expression of XBP1s helped IBV proliferation, and silence of XBP1s reduced virus proliferation. Meanwhile, exogenous expression and knock down studies demonstrated that IRE1α activated pro-apoptotic JNK signaling, promoted apoptosis and inflammation. In conclusion, our current study demonstrates that the induction of CHOP and activation of IRE1α-XBP1/JNK signaling cascades promote apoptosis and benefit NDV proliferation.IMPORTANCEIt is well known that NDV kills host animal and tumor cells by inducing cell apoptosis. Although several studies investigate the apoptotic phenomena in NDV-infected tumor cells, the molecular mechanisms underlying this oncolytic virus induced apoptosis is not well understood yet. In this study, we focus on characterization of the ER stress responses in NDV-infected tumor cells, and find that virus induces apoptosis by up-regulation or activation of several unfolded protein responses (UPR) related transcription factors and signaling: such as ATF4, CHOP and XBP1s, and pro-apoptotic kinases (IRE1α, JNK, p38). Moreover, activation of these transcription factors and signaling cascades helps virus proliferation. Our study dissects the UPR induced apoptosis in NDV-infected tumor cells, and provides the evidence that UPR favors NDV proliferation.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-12&#39;,
  &#39;authors&#39;: &#39;Yanrong Li; Ying Liao; Qiaona Niu; Feng Gu; Yingjie Sun; Chunchun Meng; Lei Tan; Cuiping Song; Xusheng Qiu; Chan Ding&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/300129&#39;},
 {&#39;cord_uid&#39;: &#39;7gh1yzaa&#39;,
  &#39;sha&#39;: &#39;c8559b6ddd6213bc91524397933bce33aea7d50f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Improving early epidemiological assessment of emerging Aedes-transmitted epidemics using historical data&#39;,
  &#39;doi&#39;: &#39;10.1101/300954&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractModel-based epidemiological assessment is useful to support decision-making at the beginning of an emerging Aedes-transmitted outbreak. However, early forecasts are generally unreliable as little information is available in the first few incidence data points. Here, we show how past Aedes-transmitted epidemics help improve these predictions. The approach was applied to the 2015-2017 Zika virus epidemics in three islands of the French West Indies, with historical data including other Aedes-transmitted diseases (Chikungunya and Zika) in the same and other locations. Hierarchical models were used to build informative a priori distributions on the reproduction ratio and the reporting rates. The accuracy and sharpness of forecasts improved substantially when these a priori distributions were used in models for prediction. For example, early forecasts of final epidemic size obtained without historical information were 3.3 times too high on average (range: 0.2 to 5.8) with respect to the eventual size, but were far closer (1.1 times the real value on average, range: 0.4 to 1.5) using information on past CHIKV epidemics in the same places. Likewise, the 97.5% upper bound for maximal incidence was 15.3 times (range: 2.0 to 63.1) the actual peak incidence, and became much sharper at 2.4 times (range: 1.3 to 3.9) the actual peak incidence with informative a priori distributions. Improvements were more limited for the date of peak incidence and the total duration of the epidemic. The framework can adapt to all forecasting models at the early stages of emerging Aedes-transmitted outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-16&#39;,
  &#39;authors&#39;: &#39;Julien Riou; Chiara Poletto; Pierre-Yves Boëlle&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/300954&#39;},
 {&#39;cord_uid&#39;: &#39;2rzfuy33&#39;,
  &#39;sha&#39;: &#39;9af0b8e13e1e5ac714135dca8ba6d5216966066b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evolved sequence features within the intrinsically disordered tail influence FtsZ assembly and bacterial cell division&#39;,
  &#39;doi&#39;: &#39;10.1101/301622&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIntrinsically disordered regions (IDRs) challenge the well-established sequence-structure-function paradigm for describing protein function and evolution. Here, we direct a combination of biophysical and cellular studies to further our understanding of how the intrinsically disordered C-terminal tail of FtsZ contributes to cell division in rod-shaped bacteria. FtsZ is a modular protein that encompasses a conserved GTPase domain and a highly variable intrinsically disordered C-terminal tail (CTT). The CTT is essential for forming the cytokinetic Z-ring. Despite poor sequence conservation of the CTT, the patterning of oppositely charged residues, which refers to the extent of linear mixing / segregation of oppositely charged residues within CTT sequences is bounded within a narrow range. To assess the impact of evolutionary bounds on charge patterning within CTT sequences we performed experiments, aided by sequence design, to quantify the impact of changing the patterning of oppositely charged residues within the CTT on the functions of FtsZ from B. subtilis. Z-ring formation is robust if and only if the extent of linear mixing / segregation of oppositely charged residues within the CTT sequences is within evolutionarily observed bounds. Otherwise, aberrant, CTT-mediated, FtsZ assemblies impair Z-ring formation. The complexities of CTT sequences also have to be above a threshold value because FtsZ variants with low complexity CTTs are not tolerated in cells. Taken together, our results suggest that CTT sequences have evolved to be “just right” and that this is achieved through an optimal extent of charge patterning while maintaining the sequence complexity above a threshold value.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-14&#39;,
  &#39;authors&#39;: &#39;Megan C. Cohan; Ammon E. Posey; Steven J. Grigsby; Anuradha Mittal; Alex S. Holehouse; Paul J. Buske; Petra A. Levin; Rohit V. Pappu&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/301622&#39;},
 {&#39;cord_uid&#39;: &#39;4fuxbte0&#39;,
  &#39;sha&#39;: &#39;f905f78b32f63c6d14a79984dfb33f1b358b8ab4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Multimerization of HIV-1 integrase hinges on conserved SH3-docking platforms&#39;,
  &#39;doi&#39;: &#39;10.1101/301721&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTNew anti-AIDS treatments must be continually developed in order to overcome resistance mutations including those emerging in the newest therapeutic target, the viral integrase (IN). Multimerization of IN is functionally imperative and provides a forthcoming therapeutic target. Allosteric inhibitors of IN bind to non-catalytic sites and prevent correct multimerization not only restricting viral integration but also the assembly and maturation of viral particles. Here, we report an allosteric inhibitor peptide targeting an unexploited SH3-docking platform of retroviral IN. The crystal structure of the peptide in complex with the HIV-1 IN core domain reveals a steric interference that would inhibit conserved docking of SH3-containing domain with the core domain vital for IN multimerization, providing a template for the development of novel anti-IN allosteric inhibitors.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-16&#39;,
  &#39;authors&#39;: &#39;Meytal Galilee; Akram Alian&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/301721&#39;},
 {&#39;cord_uid&#39;: &#39;sab90rov&#39;,
  &#39;sha&#39;: &#39;3ddc987158400d3867939d3594da56dd546d6ddd&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A novel framework for inferring parameters of transmission from viral sequence data&#39;,
  &#39;doi&#39;: &#39;10.1101/302331&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractTransmission between hosts is a critical part of the viral lifecycle. Recent studies of viral transmission have used genome sequence data to evaluate the number of particles transmitted between hosts, and the role of selection as it operates during the transmission process. However, the interpretation of sequence data describing transmission events is a challenging task. We here present a novel and comprehensive framework for using short-read sequence data to understand viral transmission events. Our model describes transmission as an event involving whole viruses, rather than independent alleles. We demonstrate how selection and noisy sequence data may each affect inferences of the population bottleneck, and identify circumstances in which selection for increased viral transmission may or may not be identified. Applying our model to data from a previous experimental transmission study, we show that our approach grants a more quantitative insight into viral transmission, inferring that between 2 to 6 viruses initiated infection, and allowing for a more informed interpretation of transmission events. While our model is here applied to influenza transmission, the framework we present is highly generalisable to other systems. Our work provides new opportunities for studying viral transmission.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-18&#39;,
  &#39;authors&#39;: &#39;Casper K Lumby; Nuno R Nene; Christopher J R Illingworth&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/302331&#39;},
 {&#39;cord_uid&#39;: &#39;74k0fs3f&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Modeling site-specific amino-acid preferences deepens phylogenetic estimates of viral divergence&#39;,
  &#39;doi&#39;: &#39;10.1101/302703&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#34;Molecular phylogenetics is often used to estimate the time since the divergence of modern gene sequences. For highly diverged sequences, such phylogenetic techniques sometimes estimate surprisingly recent divergence times. In the case of viruses, independent evidence indicates that the estimates of deep divergence times from molecular phylogenetics are sometimes too recent. This discrepancy is caused in part by inadequate models of purifying selection leading to branch-length underestimation. Here we examine the effect on branch-length estimation of using models that incorporate experimental measurements of purifying selection. We find that models informed by experimentally measured site-specific amino-acid preferences estimate longer deep branches on phylogenies of influenza virus hemagglutinin. This lengthening of branches is due to more realistic stationary states of the models, and is mostly independent of the branch-length-extension from modeling site-to-site variation in amino-acid substitution rate. The branch-length extension from experimentally informed site-specific models is similar to that achieved by other approaches that allow the stationary state to vary across sites. However, the improvements from all of these site-specific but time-homogeneous and site-independent models are limited by the fact that a protein&#39;s amino-acid preferences gradually shift as it evolves. Overall, our work underscores the importance of modeling site-specific amino-acid preferences when estimating deep divergence times---but also shows the inherent limitations of approaches that fail to account for how these preferences shift over time.&#34;,
  &#39;publish_time&#39;: &#39;2018-04-17&#39;,
  &#39;authors&#39;: &#39;Sarah K Hilton; Jesse D Bloom&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/302703&#39;},
 {&#39;cord_uid&#39;: &#39;uu9l1uf4&#39;,
  &#39;sha&#39;: &#39;62298f90df017016d34e978befc29acbec201ccd&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The alkyl side chain of PACA nanoparticles dictates the impact on cellular stress responses and the mode of particle-induced cell death&#39;,
  &#39;doi&#39;: &#39;10.1101/304618&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractFor optimal exploitation of nanoparticles (NPs) in biomedicine, and to predict nanotoxicity, detailed knowledge on the cellular responses to cell-bound or internalized NPs is imperative. The outcome of NP-cell interaction is dictated by the type and magnitude of the NP insult and the cellular response. Here, we have systematically studied the impact of minor differences in NP composition on cellular stress responses and viability by using highly similar poly(alkylcyanoacrylate) (PACA) particles. Surprisingly, PACA particles differing only in their alkyl side chains; butyl (PBCA), ethylbutyl (PEBCA), or octyl (POCA), respectively, induced different stress responses and modes of cell death in human cell lines. POCA particles induced endoplasmic reticulum stress and apoptosis. In contrast, PBCA and PEBCA particles induced lipid peroxidation by depletion of the main cellular antioxidant glutathione (GSH), in a manner depending on the levels of the GSH precursor cystine, and transcription of the cystine transporter SLC7A11 regulated by ATF4 and Nrf2. Intriguingly, these particles activated the recently discovered cell death mechanism ferroptosis, which constitutes a promising alternative for targeting multidrug-resistant cancer stem-like cells. Of the two, PBCA was the strongest inducer. In summary, our findings highlight the cellular sensitivity to nanoparticle composition and have important implications for the choice of PACA monomer in therapeutical settings.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-19&#39;,
  &#39;authors&#39;: &#39;Marzena Szwed; Tonje Sønstevold; Anders Øverbye; Nikolai Engedal; Beata Grallert; Ýrr A Mørch; Tore-Geir Iversen; Tore Skotland; Kirsten Sandvig; Maria L Torgersen&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/304618&#39;},
 {&#39;cord_uid&#39;: &#39;n78fga3r&#39;,
  &#39;sha&#39;: &#39;e60640bfc445db0f53b5d92cb9f69cd943f1b73a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome&#39;,
  &#39;doi&#39;: &#39;10.1101/307678&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with Aplysia californica . Genome structure and domain architecture suggest that the identified virus is a novel member of Nidovirales . The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, Roniviridae , and Mesoniviridae . Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within Nidovirales . Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a Nidovirales in single specific neurons.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-24&#39;,
  &#39;authors&#39;: &#39;Humberto J Debat&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/307678&#39;},
 {&#39;cord_uid&#39;: &#39;cheiabv0&#39;,
  &#39;sha&#39;: &#39;2e7c17caecfe67b64f9b59df404f0761df96987f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;National and Regional Influenza-Like-Illness Forecasts for the USA&#39;,
  &#39;doi&#39;: &#39;10.1101/309021&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractHealth planners use forecasts of key metrics associated with influenza-like-illness (ILI); near-term weekly incidence, week of season onset, week of peak, and intensity of peak. Here, we describe our participation in a weekly prospective ILI forecasting challenge for the United States for the 2016-17 season and subsequent evaluation of our performance. We implemented a metapopulation model framework with 32 model variants. Variants differed from each other in their assumptions about: the force-of-infection (FOI); use of uninformative priors; the use of discounted historical data for not-yet-observed time points; and the treatment of regions as either independent or coupled. Individual model variants were chosen subjectively as the basis for our weekly forecasts; however, a subset of coupled models were only available part way through the season. Most frequently, during the 2016-17 season, we chose; FOI variants with both school vacations and humidity terms; uninformative priors; the inclusion of discounted historical data for not-yet-observed time points; and coupled regions (when available). Our near-term weekly forecasts substantially over-estimated incidence early in the season when coupled models were not available. However, our forecast accuracy improved in absolute terms and relative to other teams once coupled solutions were available. In retrospective analysis, we found that the 2016-17 season was not typical: on average, coupled models performed better when fit without historically augmented data. Also, we tested a simple ensemble model for the 2016-17 season and found that it underperformed our subjective choice for all forecast targets. In this study, we were able to improve accuracy during a prospective forecasting exercise by coupling dynamics between regions. Although reduction of forecast subjectivity should be a long-term goal, some degree of human intervention is likely to improve forecast accuracy in the medium-term in parallel with the systematic consideration of more sophisticated ensemble approaches.Author summaryIt is estimated that there are between 3 and 5 million worldwide annual seasonal cases of severe influenza illness, and between 290 000 and 650 000 respiratory deaths [1]. Influenza-like-illness (ILI) describes a set of symptoms and is a practical way for health-care workers to easily estimate likely influenza cases. The Centers for Disease Control (CDC) collects and disseminates ILI information, and has, for the last several years, run a forecasting challenge (the CDC Flu Challenge) for modelers to predict near-term weekly incidence, week of season onset, week of peak, and intensity of peak. We have developed a modeling framework that accounts for a range of mechanisms thought to be important for influenza transmission, such as climatic conditions, school vacations, and coupling between different regions. In this study we describe our forecast procedure for the 2016-17 season and highlight which features of our models resulted in better or worse forecasts. Most notably, we found that when the dynamics of different regions are coupled together, the forecast accuracy improves. We also found that the most accurate forecasts required some level of forecaster interaction, that is, the procedure could not be completely automated without a reduction in accuracy.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-27&#39;,
  &#39;authors&#39;: &#39;Michal Ben-Nun; Pete Riley; James Turtle; David P. Bacon; Steven Riley&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/309021&#39;},
 {&#39;cord_uid&#39;: &#39;3t75dbb7&#39;,
  &#39;sha&#39;: &#39;2da0a7ba851e53d21dba05042a397eafcdab093d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Combination attenuation offers strategy for live-attenuated coronavirus vaccines&#39;,
  &#39;doi&#39;: &#39;10.1101/309591&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWith an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2’O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2’O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2’O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live-attenuated coronavirus vaccines.SignificanceEmergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations like the aged and immune-compromised. This study illustrates that despite success in young models, the NSP16 mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the NSP16 mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for NSP16 attenuation as a major component of future live-attenuated coronavirus vaccines.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-28&#39;,
  &#39;authors&#39;: &#39;Vineet D. Menachery; Lisa E. Gralinski; Hugh D. Mitchell; Kenneth H. Dinnon; Sarah R. Leist; Boyd L. Yount; Eileen T. McAnarney; Rachel L. Graham; Katrina M. Waters; Ralph S. Baric&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/309591&#39;},
 {&#39;cord_uid&#39;: &#39;8p24gszj&#39;,
  &#39;sha&#39;: &#39;295036c0448244642eb6533de571def6cd8437e5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A speed-fidelity trade-off determines the mutation rate and virulence of an RNA virus&#39;,
  &#39;doi&#39;: &#39;10.1101/309880&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMutation rates can evolve through genetic drift, indirect selection due to genetic hitchhiking, or direct selection on the physicochemical cost of high fidelity. However, for many systems, it has been difficult to disentangle the relative impact of these forces empirically. In RNA viruses, an observed correlation between mutation rate and virulence has led many to argue that their extremely high mutation rates are advantageous, because they may allow for increased adaptability. This argument has profound implications, as it suggests that pathogenesis in many viral infections depends on rare or de novo mutations. Here we present data for an alternative model whereby RNA viruses evolve high mutation rates as a byproduct of selection for increased replicative speed. We find that a poliovirus antimutator, 3DG64S, has a significant replication defect and that wild type and 3DG64S populations have similar adaptability in two distinct cellular environments. Experimental evolution of 3DG64S under r-selection led to reversion and compensation of the fidelity phenotype. Mice infected with 3DG64S exhibited delayed morbidity at doses well above the LD50, consistent with attenuation by slower growth as opposed to reduced mutational supply. Furthermore, compensation of the 3DG64S growth defect restored virulence, while compensation of the fidelity phenotype did not. Our data are consistent with the kinetic proofreading model for biosynthetic reactions and suggest that speed is more important than accuracy. In contrast to what has been suggested for many RNA viruses, we find that within host spread is associated with viral replicative speed and not standing genetic diversity.Author SummaryMutation rate evolution has long been a fundamental problem in evolutionary biology. The polymerases of RNA viruses generally lack proofreading activity and exhibit extremely high mutation rates. Since most mutations are deleterious and mutation rates are tuned by natural selection, we asked why hasn’t the virus evolved to have a lower mutation rate? We used experimental evolution and a murine infection model to show that RNA virus mutation rates may actually be too high and are not necessarily adaptive. Rather, our data indicate that viral mutation rates are driven higher as a result of selection for viruses with faster replication kinetics. We suggest that viruses have high mutation rates, not because they facilitate adaption, but because it is hard to be both fast and accurate.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-27&#39;,
  &#39;authors&#39;: &#39;William Fitzsimmons; Robert J. Woods; John T. McCrone; Andrew Woodman; Jamie J. Arnold; Madhumita Yennawar; Richard Evans; Craig E. Cameron; Adam S. Lauring&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/309880&#39;},
 {&#39;cord_uid&#39;: &#39;rb9ozu56&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A functional different immune capacity in cattle is associated with higher mastitis incidence&#39;,
  &#39;doi&#39;: &#39;10.1101/311316&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTBovine neonatal pancytopenia (BNP) was a deadly disease transferred by antibodies from 5-10% of cows given a novel BVD vaccine. Disease was lethal in 90% of calves receiving colostrum with BNP antibodies. The cause of BNP is not fully understood to date. We revealed a profound difference in immune capacities between BNP dams and non-responders. Significant differences were detectable in response to in vitro stimulation of peripheral blood derived lymphocytes to several mitogens and IL-2. BNP cows regulated their immune proteomes completely different from controls with other immune response master regulators. Since we detected this response pattern also in 22% of cows that were never vaccinated at all, this immune deviant (ID) phenotype is still present in cattle and probably inherited. Immune response pattern of these cows was stable over an observation period of 38 months. Importantly, ID have a significant increased prevalence of mastitis underscoring the clinical importance.&#39;,
  &#39;publish_time&#39;: &#39;2018-04-30&#39;,
  &#39;authors&#39;: &#39;Karina Lutterberg; Kristina J. H. Kleinwort; Bernhard F. Hobmaier; Stefanie M. Hauck; Stefan Nüske; Armin M. Scholz; Cornelia A. Deeg&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/311316&#39;},
 {&#39;cord_uid&#39;: &#39;jry46itn&#39;,
  &#39;sha&#39;: &#39;975349ac1c79fc64525ee17e7d088c88c2c2071b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A practical generation interval-based approach to inferring the strength of epidemics from their speed&#39;,
  &#39;doi&#39;: &#39;10.1101/312397&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractInfectious-disease outbreaks are often characterized by the reproductive number and 𝓡 exponential rate of growth r. 𝓡 provides information about out-break control and predicted final size. Directly estimating 𝓡 is difficult, while r can often be estimated from incidence data. These quantities are linked by the generation interval – the time between when an individual is infected by an infector, and when that infector was infected. It is often infeasible to ob-tain the exact shape of a generation-interval distribution, and to understand how this shape affects estimates of 𝓡. We show that estimating generation interval mean and variance provides insight into the relationship between and 𝓡 r. We use examples based on Ebola, rabies and measles to explore approximations based on gamma-distributed generation intervals, and find that use of these simple approximations are often sufficient to capture the r–𝓡 relationship and provide robust estimates of 𝓡.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-02&#39;,
  &#39;authors&#39;: &#39;Sang Woo Park; David Champredon; Joshua S. Weitz; Jonathan Dushoff&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/312397&#39;},
 {&#39;cord_uid&#39;: &#39;32b5xysw&#39;,
  &#39;sha&#39;: &#39;64b327001f8fa95b83dc23259a2ad617be12498c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding&#39;,
  &#39;doi&#39;: &#39;10.1101/315366&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTOne unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC50 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found in vivo in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs (P &lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.IMPORTANCELarge-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus’s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-07&#39;,
  &#39;authors&#39;: &#39;Maria Nevot; Ana Jordan-Paiz; Glòria Martrus; Cristina Andrés; Damir García-Cehic; Josep Gregori; Sandra Franco; Josep Quer; Miguel Angel Martinez&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/315366&#39;},
 {&#39;cord_uid&#39;: &#39;rq18ey04&#39;,
  &#39;sha&#39;: &#39;&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird&#39;,
  &#39;doi&#39;: &#39;10.1101/315457&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe unique ornamental features and extreme sexual traits of Peacock have always intrigued the scientists. However, the genomic evidence to explain its phenotype are yet unknown. Thus, we report the first genome sequence and comparative analysis of peacock with the available high-quality genomes of chicken, turkey, duck, flycatcher and zebra finch. The candidate genes involved in early developmental pathways including TGF-β, BMP, and Wnt signaling pathway, which are also involved in feather patterning, bone morphogenesis, and skeletal muscle development, showed signs of adaptive evolution and provided useful clues on the phenotype of peacock. The innate and adaptive immune components such as complement system and T-cell response also showed signs of adaptive evolution in peacock suggesting their possible role in building a robust immune system which is consistent with the between species predictions of Hamilton-Zuk hypothesis. This study provides novel genomic and evolutionary insights into the molecular understanding towards the phenotypic evolution of Indian peacock.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-05&#39;,
  &#39;authors&#39;: &#39;Shubham K. Jaiswal; Ankit Gupta; Rituja Saxena; P. K. Vishnu Prasoodanan; Ashok K. Sharma; Parul Mittal; Ankita Roy; Aaron B.A. Shafer; Nagarjun Vijay; Vineet K. Sharma&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;False&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/315457&#39;},
 {&#39;cord_uid&#39;: &#39;298cbr1x&#39;,
  &#39;sha&#39;: &#39;532dc582abd41e833303742ca40976db63fe6fc1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Site specific target binding controls RNA cleavage efficiency by the Kaposi’s sarcoma-associated herpesvirus endonuclease SOX&#39;,
  &#39;doi&#39;: &#39;10.1101/320929&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDuring lytic replication of Kaposi’s sarcoma-associated herpesvirus (KSHV), the gene expression landscape of a cell is remodeled to evade the immune response and create an environment favorable to viral replication. A major driver of these gene expression changes is a virally encoded, messenger RNA (mRNA)-specific endonuclease termed SOX. SOX cleaves the majority of cytoplasmic mRNAs, but does so at specific internal sites loosely defined by a degenerate sequence motif. If and how RNA sequence directs SOX targeting remained unknown. To address these questions, we used recombinant, highly purified SOX endonuclease in a series of biochemical assays to reconstitute the cleavage reaction in vitro and gain significant insight into the biochemical mechanism of both SOX target recognition and endonucleolytic cleavage. Using this system, we determined that cut site specificity is preserved with purified SOX and a validated target RNA and thus does not require additional cellular cofactors. Furthermore, we showed that SOX displays robust, sequence-specific RNA binding to residues proximal to the cleavage site, which must be presented in a particular structural context. The strength of SOX binding dictates cleavage efficiency, providing an explanation for the breadth of target RNA susceptibility observed in cells.Significance StatementKaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic human virus that causes Kaposi’s sarcoma, primary effusion lymphoma, and multicentric Castleman disease. During viral replication, KSHV expresses an enzyme called SOX that cuts and inactivates the majority of cellular messenger RNAs, preventing their translation into proteins. Some mRNAs are efficiently cleaved by SOX, while others are poorly cleaved, but the mechanistic basis underlying this selectivity has remained largely unknown. Here, we reveal that the efficiency of RNA cleavage is heavily impacted by RNA sequences proximal to the cleavage site, which serve as a SOX binding platform. This helps explain both the range of RNA cleavage efficiency observed in SOX-expressing cells as well as the sequence specificity underlying SOX targeting.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-13&#39;,
  &#39;authors&#39;: &#39;Aaron S. Mendez; Carolin Vogt; Jens Bohne; Britt A. Glaunsinger&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/320929&#39;},
 {&#39;cord_uid&#39;: &#39;58y8mg8m&#39;,
  &#39;sha&#39;: &#39;317215c463463ddb39fa7564f67239dc484b86a9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Projecting the end of the Zika virus epidemic in Latin America: a modelling analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/323915&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackground Zika virus (ZIKV) emerged in Latin America &amp; the Caribbean (LAC) region in 2013, and has had serious implications for population health in the region. In 2016, the World Health Organization declared the ZIKV outbreak a Public Health Emergency of International Concern following a cluster of associated neurological disorders and neonatal malformations. In 2017, Zika cases declined, but future incidence in LAC remains uncertain due to gaps in our understanding, considerable variation in surveillance and a lack of a comprehensive collation of data from affected countries.Methods Our analysis combines information on confirmed and suspected Zika cases across LAC countries and a spatio-temporal dynamic transmission model for ZIKV infection to determine key transmission parameters and projected incidence in 91 major cities within 35 countries. Seasonality was determined by spatio-temporal estimates of Aedes aegypti vector capacity. We used country and state-level data from 2015 to mid-2017 to infer key model parameters, country-specific disease reporting rates, and the 2018 projected incidence. A 10-fold cross-validation approach was used to validate parameter estimates to out-of-sample epidemic trajectories.Results There was limited transmission in 2015, but in 2016 and 2017 there was sufficient opportunity for wide-spread ZIKV transmission in most cities, resulting in the depletion of susceptible individuals. We predict that the highest number of cases in 2018 within some Brazilian States (Sao Paulo and Rio de Janeiro), Colombia and French Guiana, but the estimated number of cases were no more than a few hundred. Model estimates of the timing of the peak in incidence were correlated (p&lt;0.05) with the reported peak in incidence. The reporting rate varied across countries, with lower reporting rates for those with only confirmed cases compared to those who reported both confirmed and suspected cases.Conclusions The findings suggest that the ZIKV epidemic is by and large over, with incidence projected to be low in most cities in LAC in 2018. Local low levels of transmission are probable but the estimated rate of infection suggests that most cities have a population with high levels of herd immunity.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-18&#39;,
  &#39;authors&#39;: &#39;Kathleen M O’Reilly; Rachel Lowe; W John Edmunds; Philippe Mayaud; Adam Kucharski; Rosalind M Eggo; Sebastian Funk; Deepit Bhatia; Kamran Khan; Moritz U Kraemar; Annelies Wilder-Smith; Laura C Rodrigues; Patricia Brasil; Eduardo Massad; Thomas Jaenisch; Simon Cauchemez; Oliver J Brady; Laith Yakob&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/323915&#39;},
 {&#39;cord_uid&#39;: &#39;6arj2i3m&#39;,
  &#39;sha&#39;: &#39;ff6bd29c8ecd325c4edddb2b050d5f4919db6112&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Deep sequencing of primary human lung epithelial cells challenged with H5N1 influenza virus reveals a proviral role for CEACAM1&#39;,
  &#39;doi&#39;: &#39;10.1101/324723&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTCurrent prophylactic and therapeutic strategies targeting human influenza viruses include vaccines and antivirals. Given variable rates of vaccine efficacy and antiviral resistance, alternative strategies are urgently required to improve disease outcomes. Here we describe the use of HiSeq deep sequencing to analyze host gene expression in primary human alveolar epithelial type II cells infected with highly pathogenic avian influenza H5N1 virus. At 24 hours post-infection, 623 host genes were significantly up-regulated, including the cell adhesion moleculeCEACAM1. The up-regulation ofCEACAM1was blocked in the presence of the reactive oxygen species inhibitor, apocynin. H5N1 virus infection stimulated significantly higher CEACAM1 protein expression when compared to low pathogenic PR8 H1N1 virus, suggesting a key role for CEACAM1 in influenza virus pathogenicity. Furthermore, silencing of endogenousCEACAM1resulted in reduced levels of proinflammatory cytokine/chemokine production, as well as reduced levels of virus replication following H5N1 infection. Our study provides evidence for the involvement of CEACAM1 in a clinically relevant model of H5N1 infection and may assist in the development of host-oriented antiviral strategies.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-17&#39;,
  &#39;authors&#39;: &#39;Siying Ye; Chris Cowled; Cheng-Hon Yap; John Stambas&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/324723&#39;},
 {&#39;cord_uid&#39;: &#39;foskvkwn&#39;,
  &#39;sha&#39;: &#39;ea94779ce166bcbf3813ccaab028524ff397089b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation&#39;,
  &#39;doi&#39;: &#39;10.1101/325316&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractLinear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from Kluyveromyces lactis as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5’ and 3’ RACE analysis of all the pGKL1,2 mRNAs and found them not 3’ polyadenylated and containing mostly uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that K2ORF3 encodes an mRNA cap guanine-N7-methyltransferase and that 5’ capped pGKL1,2 transcripts contain N7-methylated caps. Translation of pGKL1,2 transcripts is enhanced in Ism1Δ and pab1Δ strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5’ mRNA capping, degree of 5’ non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-17&#39;,
  &#39;authors&#39;: &#39;Václav Vopálenský; Michal Sýkora; Tomáš Mašek; Martin Pospíšek&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/325316&#39;},
 {&#39;cord_uid&#39;: &#39;xtibubra&#39;,
  &#39;sha&#39;: &#39;10abb3e977aa476a39bce4f1feda37c2e1250106&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Replication of MERS and SARS coronaviruses in bat cells offers insights to their ancestral origins&#39;,
  &#39;doi&#39;: &#39;10.1101/326538&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTPrevious findings of Middle East Respiratory Syndrome coronavirus (MERS-CoV)-related viruses in bats, and the ability of Tylonycteris-BatCoV HKU4 spike protein to utilize MERS-CoV receptor, human dipeptidyl peptidase 4 hDPP4, suggest a bat ancestral origin of MERS-CoV. We developed 12 primary bat cell lines from seven bat species, including Tylonycteris pachypus, Pipistrellus abramus and Rhinolophus sinicus (hosts of Tylonycteris-BatCoV HKU4, Pipistrellus-BatCoV HKU5 and SARS-related-CoV respectively), and tested their susceptibilities to MERS-CoVs, SARS-CoV and human coronavirus 229E (HCoV-229E). Five cell lines, including P. abramus and R. sinicus but not T. pachypus cells, were susceptible to human MERS-CoV EMC/2012. However, three tested camel MERS-CoV strains showed different infectivities, with only two strains capable of infecting three and one cell lines respectively. SARS-CoV can only replicate in R. sinicus cells, while HCoV-229E cannot replicate in any bat cells. Bat dipeptidyl peptidase 4 (DPP4) sequences were closely related to those of human and non-human primates but distinct from dromedary DPP4 sequence. Critical residues for binding to MERS-CoV spike protein were mostly conserved in bat DPP4. DPP4 was expressed in the five bat cells susceptible to MERS-CoV, with significantly higher mRNA expression levels than those in non-susceptible cells (P=0.0174), supporting that DPP4 expression is critical for MERS-CoV infection in bats. However, overexpression of T. pachypus DPP4 failed to confer MERS-CoV susceptibility in T. pachypus cells, suggesting other cellular factors in determining viral replication. The broad cellular tropism of MERS-CoV should prompt further exploration of host diversity of related viruses to identify its ancestral origin.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-20&#39;,
  &#39;authors&#39;: &#39;Susanna K. P. Lau; Rachel Y. Y. Fan; Hayes K. H. Luk; Longchao Zhu; Joshua Fung; Kenneth S. M. Li; Emily Y.M. Wong; Syed Shakeel Ahmed; Jasper F. W. Chan; Raven K. H. Kok; Kwok-Hung Chan; Ulrich Wernery; Kwok-Yung Yuen; Patrick C. Y. Woo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/326538&#39;},
 {&#39;cord_uid&#39;: &#39;y4qz8dez&#39;,
  &#39;sha&#39;: &#39;f32841f22efe8c2bccd0d04a0c0b47e6ecf86b61&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Attenuation of influenza A virus disease severity by viral co-infection in a mouse model&#39;,
  &#39;doi&#39;: &#39;10.1101/326546&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractInfluenza viruses and rhinoviruses are responsible for a large number of acute respiratory viral infections in human populations and are detected as co-pathogens within hosts. Clinical and epidemiological studies suggest that co-infection by rhinovirus and influenza virus may reduce disease severity and that they may also interfere with each other’s spread within a host population. To determine how co-infection by these two unrelated respiratory viruses affects pathogenesis, we established a mouse model using a minor serogroup rhinovirus (RV1B) and mouse-adapted influenza A virus (PR8). Infection of mice with RV1B two days before PR8 reduced pathogenesis of mild to moderate, but not severe PR8 infections. Disease attenuation was associated with an early inflammatory response in the lungs and enhanced clearance of PR8. However, co-infection by RV1B did not reduce PR8 viral loads early in infection or inhibit replication of PR8 within respiratory epithelia or in vitro. Inflammation in co-infected mice remained focal, in comparison to diffuse inflammation and damage in the lungs of mice infected by PR8. These findings suggest that RV1B stimulates an early immune response that clears PR8 while limiting excessive pulmonary inflammation. The timing of RV1B co-infection was a critical determinant of protection, suggesting that sufficient time is needed to induce this response. Finally, disease attenuation was not unique to RV1B: co-infection by a murine coronavirus two days before PR8 also reduced disease severity. This model will be critical for understanding the mechanisms responsible for attenuation of influenza disease during co-infection by unrelated respiratory viruses.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-19&#39;,
  &#39;authors&#39;: &#39;Andres J. Gonzalez; Emmanuel C. Ijezie; Onesmo B. Balemba; Tanya A. Miura&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/326546&#39;},
 {&#39;cord_uid&#39;: &#39;sojqdz8e&#39;,
  &#39;sha&#39;: &#39;1a9c229f4f866db4f9c23a77b8c1275b52407c64&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Early in planta detection of Xanthomonas axonopodis pv. punicae in pomegranate using enhanced loop-mediated isothermal amplification assay&#39;,
  &#39;doi&#39;: &#39;10.1101/328674&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBacterial blight in pomegranate caused by Xanthomonas axonopodis pv. punicae (Xap) is an increasing threat for pomegranate cultivation in India. To prevent the economic losses, it is pivotal to detect the infection in latent stages rather than in later stages. We have developed an enhanced method termed as loop-mediated isothermal amplification (LAMP) technique to evaluate for the latent detection of Xap in pomegranate using six set of specific primers. Three DNA intercalating dyes were used, such as Ethidium bromide, hydroxynaphthol blue (HNB) and SYBR Green resulted in visualising the positivity for LAMP assay. The reaction time and temperature were to be 65°C from 30 min onwards, for the dyes and its sensitivity was observed up to 10−7 ng in the LAMP assay. For field applicability, LAMP assay detected Xap on 7th day post infection while the PCR amplified Xap after 11th day post infection. Finally, the specificity of LAMP assay was validated to be positive with ten Xap isolates for its accuracy and 29 non-Xap bacterial isolates showed negative results. Moreover, this method could be used as a better alternative to PCR based methods, for early detection of the pathogens.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-22&#39;,
  &#39;authors&#39;: &#39;M.K. PrasannaKumar; P. Buela Parivallal; C. Manjunath; H.B. Mahesh; Karthik S Narayan; Gopal Venkatesh Babu; K. Priyanka; M.E. Puneeth; K.T. Rangaswamy&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/328674&#39;},
 {&#39;cord_uid&#39;: &#39;p0nb85e6&#39;,
  &#39;sha&#39;: &#39;57eb12d93057a4bc1a043420327251b2327c9b39&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Real-time projections of Ebola outbreak size and duration with and without vaccine use in Équateur, Democratic Republic of Congo, as of May 27, 2018&#39;,
  &#39;doi&#39;: &#39;10.1101/331447&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundAs of May 27, 2018, 54 cases of Ebola virus disease (EVD) were reported in Équateur Province, Democratic Republic of Congo. We used reported case counts and time series from prior outbreaks to estimate the current outbreak size and duration with and without vaccine use.MethodsWe modeled Ebola virus transmission using a stochastic branching process model with a negative binomial distribution, using both estimates of reproduction number R declining from supercritical to subcritical derived from past Ebola outbreaks, as well as a particle filtering method to generate a probabilistic projection of the future course of the outbreak conditioned on its reported trajectory to date; modeled using 0%, 44%, and 62% estimates of vaccination coverage. Additionally, we used the time series for 18 prior Ebola outbreaks from 1976 to 2016 to parameterize a regression model predicting the outbreak size from the number of observed cases from April 4 to May 27.ResultsWith the stochastic transmission model, we projected a median outbreak size of 78 EVD cases (95% credible interval: 52, 125.4), 86 cases (95% credible interval: 53, 174.3), and 91 cases (95% credible interval: 52, 843.5), using 62%, 44%, and 0% estimates of vaccination coverage. With the regression model, we estimated a median size of 85.0 cases (95% prediction interval: 53.5, 216.6).ConclusionsThis outbreak has the potential to be the largest outbreak in DRC since 2007. Vaccines are projected to limit outbreak size and duration but are only part of prevention, control, and care strategies.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-26&#39;,
  &#39;authors&#39;: &#39;J. Daniel Kelly; Lee Worden; Rae Wannier; Nicole A. Hoff; Patrick Mukadi; Cyrus Sinai; Sarah Ackley; Xianyun Chen; Daozhou Gao; Bernice Selo; Mathais Mossoko; Emile Okitolonda-Wemakoy; Eugene T. Richardson; George W. Rutherford; Thomas M. Lietman; Jean Jacques Muyembe-Tamfum; Anne W. Rimoin; Travis C. Porco&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/331447&#39;},
 {&#39;cord_uid&#39;: &#39;396rgxno&#39;,
  &#39;sha&#39;: &#39;23f1bab7d94bd0564e2319fddf75c1c7987ce75f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Complex dynamics in an SIS epidemic model induced by nonlinear incidence&#39;,
  &#39;doi&#39;: &#39;10.1101/331678&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWe study an epidemic model with nonlinear incidence rate, describing the saturated mass action as well as the psychological effect of certain serious diseases on the community. Firstly, the existence and local stability of disease-free and endemic equilibria are investigated. Then, we prove the occurrence of backward bifurcation, saddle-node bifurcation, Hopf bifurcation and cusp type Bogdanov-Takens bifurcation of codimension 3. Finally, numerical simulations, including one limit cycle, two limit cycles, unstable homoclinic loop and many other phase portraits are presented. These results show that the psychological effect of diseases and the behavior change of the susceptible individuals may affect the final spread level of an epidemic.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-25&#39;,
  &#39;authors&#39;: &#39;Ruixia Yuan; Zhidong Teng; Jinhui Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/331678&#39;},
 {&#39;cord_uid&#39;: &#39;8r1zb4oc&#39;,
  &#39;sha&#39;: &#39;4e3f4e31c370281c2d721d6113265c25d9c566c5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Host protein CD63 enhances viral RNA replication by interacting with human astrovirus nonstructural protein nsP1a/4&#39;,
  &#39;doi&#39;: &#39;10.1101/331835&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTHuman astrovirus nonstructural protein nsP1a/4, located at the C-terminal end of nsP1a, is thought to be involved in the regulation of RNA replication and capsid maturation;however, its rolesviral growth and virulence are not well understood. We investigated the intracellular host proteins that interact with nsP1a and explored the potential roles of the interaction in the pathogenesis of human astrovirus infection. We screened 14 independent proteins with a cDNA library derived from Caco-2 cells using a yeast two-hybrid technique. Deletion analysis revealed that interaction between the nsP1a/4 domain and the large extracellular loop (LEL) domain of the human protein CD63 is necessary for astrovirus replication. The interaction was confirmed by glutathione-S-transferase (GST) pull-down assays and co-immunoprecipitation assays. Confocal microscopy showed that nsP1a/4 and CD63 co-localized in the cytoplasm of infected cells. Over expression of CD63 promoted viral RNA synthesis, whereas knockdown of CD63 markedly decreased viral RNA levels. Those results suggest that CD63 plays a critical role in human astrovirus RNA replication. The interaction between CD63 and nsP1a/4 provides a channel to further understand the roles of interactions between host and virus proteins in astrovirus infection and release.IMPORTANCEHuman astroviruses cause gastroenteritis in young children and immunocompromised patients. In this study, we provide evidence that nsP1a/4, a nonstructural protein located at the C-terminal end of the human astrovirus nsP1a polyprotein, interacts with the host protein CD63. Over expression of CD63 promoted viral RNA replication, whereas knockdown of CD63 decreased virus RNA replication, indicating that CD63 plays a critical role in the human astrovirus life cycle.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-27&#39;,
  &#39;authors&#39;: &#39;wei zhao; Nian liu; Xiao Li Tao; Chun hong Zheng; Xiang yu Li; Man Yu; Yong gang Li&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/331835&#39;},
 {&#39;cord_uid&#39;: &#39;quu8c5l7&#39;,
  &#39;sha&#39;: &#39;9cfdce851951c817101b36f7b292e744906474c4&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Rapid Therapeutic Recommendations in the Context of a Global Public Health Crisis using Translational Bioinformatics Approaches: A proof-of-concept study using Nipah Virus Infection&#39;,
  &#39;doi&#39;: &#39;10.1101/333021&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractWe live in a world of emerging new diseases and old diseases resurging in more aggressive forms. Drug development by pharmaceutical companies is a market-driven and costly endeavor, and thus it is often a challenge when drugs are needed for diseases endemic only to certain regions or which affect only a few patients. However, biomedical open data is accessible and reusable for reanalysis and generation of a new hypotheses and discovery. In this study, we leverage biomedical data and tools to analyze available data on Nipah Virus (NiV) infection. NiV infection is an emerging zoonosis that is transmissible to humans and is associated with high mortality rates. In this study, explored the application of computational drug repositioning and chemogenomic enrichment analyses using host transcriptome data to match drugs that could reverse the virus-induced gene signature. We performed analyses using two gene signatures: i) A previously published gene signature (n=34), and ii) a gene signature generated using the characteristic direction method (n= 5,533). Our predictive framework suggests that several drugs including FDA approved therapies like beclometasone, trihexyphenidyl, S-propranolol etc. could modulate the NiV infection induced gene signatures in endothelial cells. A target specific analysis of CXCL10 also suggests the potential application of Eldelumab, an investigative therapy for Crohn’s disease and ulcerative colitis, as a putative candidate for drug repositioning. To conclude, we also discuss challenges and opportunities in clinical trials (n-of-1 and adaptive trials) for repositioned drugs. Further follow-up studies including biochemical assays and clinical trials are required to identify effective therapies for clinical use. Our proof-of-concept study highlights that translational bioinformatics methods including gene expression analyses and computational drug repositioning could augment epidemiological investigations in the context of an emerging disease with no effective treatment.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-29&#39;,
  &#39;authors&#39;: &#39;Khader Shameer; Kipp W. Johnson; Ben Readhead; Benjamin S. Glicksberg; Claire McCallum; Amjesh Revikumar; Jamie S. Hirsch; Kevin Bock; John Chelico; Negin Hajizadeh; Michael Oppenheim; Joel T. Dudley&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/333021&#39;},
 {&#39;cord_uid&#39;: &#39;pc4je19g&#39;,
  &#39;sha&#39;: &#39;4a24fcf8d2a7c69aef247ea12a740145daf68407&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Deconvoluting Virome-Wide Antiviral Antibody Profiling Data&#39;,
  &#39;doi&#39;: &#39;10.1101/333625&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe ability to comprehensively characterize exposures and immune responses to viral infections will be critical to better understanding human health and disease. We previously described the VirScan system, a phage-display based technology for profiling antibody binding to a comprehensive library of peptides designed to represent the human virome. The previous VirScan analytical approach did not fully account for disproportionate representation of viruses in the library or for antibody cross-reactivity among sequences shared by related viruses. Here we present the ‘AntiViral Antibody Response Deconvolution Algorithm’ (‘AVARDA’), a multi-module software package for analyzing VirScan datasets. AVARDA provides a probabilistic assessment of infection at species-level resolution by considering alignment of all library peptides to each other and to all human viruses. We employed AVARDA to analyze VirScan data from a cohort of encephalitis patients with either known viral infections or undiagnosed etiologies. By comparing acute and convalescent sera, AVARDA successfully confirmed or detected antibody responses to human herpesviruses 1, 3, 4, 5, and 6, thereby improving the rate of diagnosing viral encephalitis in this cohort by 62.5%. We further assessed AVARDA’s utility in the setting of an epidemiological study, demonstrating its ability to determine infections acquired in a child followed prospectively from infancy. We consider ways in which AVARDA’s conceptual framework may be further developed in the future and describe how its analyses may be extended beyond investigations of viral infection. AVARDA, in combination with VirScan and other pan-pathogen serological techniques, is likely to find broad utility in the epidemiology and diagnosis of infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-30&#39;,
  &#39;authors&#39;: &#39;Daniel R. Monaco; Sanjay V. Kottapalli; Tiezheng Yuan; Florian P. Breitwieser; Danielle E. Anderson; Limin Wijaya; Kevin Tan; Wan Ni Chia; Kai Kammers; Mario Caturegli; Kathleen Waugh; Marian Rewers; Lin-Fa Wang; H. Benjamin Larman&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/333625&#39;},
 {&#39;cord_uid&#39;: &#39;d0w40mkn&#39;,
  &#39;sha&#39;: &#39;d47377d0da60ea7aab2522c204ad65753b0545fe&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Genotypic diversity, circulation patterns, and co-detections among rhinoviruses in Queensland, 2001&#39;,
  &#39;doi&#39;: &#39;10.1101/334334&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRhinoviruses (RVs) occur more frequently than other viruses and more often in people displaying symptoms than in those without. RVs exacerbate chronic airway disease and confound the clinical diagnosis of influenza-like illness. We sought to estimate the spectrum of RV diversity, RV species seasonality and to breakdown RV involvement in respiratory virus co-detections by comprehensive molecular testing of a convenience collection of airway sample extracts from patients with suspected respiratory infections, collected during 2001.RVs were the most common virus detected. We were able to genotype ∼90% of RV detections, identifying 70 distinct RVs, spanning all three species. RV-Bs were under-represented. We found RV species co-circulated at times, although one species usually dominated. Each species displayed a bimodal distribution.Notably, RVs and influenza A viruses (IFAV) seldom co-occurred, supporting their roles as primary pathogens of the airway among acutely ill infants. Whether RV circulation has a moderating or controlling effect on the IFAV season or is controlled by it cannot be determined from these data.Despite the frequent perception that RVs commonly co-occur with another virus, our findings indicated this was not the case. Nearly 80% of RV detections occurred alone. Understanding more about population-level interference between viruses may allow us to harness aspects of it to generate a non-specific antiviral intervention that mimics a putative protective effect.For routine respiratory virus screening to best serve the patient, RV testing should be a principal component of any acute respiratory illness testing algorithm throughout the year.&#39;,
  &#39;publish_time&#39;: &#39;2018-05-30&#39;,
  &#39;authors&#39;: &#39;Katherine E. Arden; Ristan M. Greer; Claire Y.T. Wang; Ian M. Mackay&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/334334&#39;},
 {&#39;cord_uid&#39;: &#39;f79cd6l3&#39;,
  &#39;sha&#39;: &#39;e8b2e667f2145ec96fbb6400fb41b05cafa26e37&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Evaluation of the cytotoxic potential of extracts from the genus Passiflora cultived in Brazil against cancer cells&#39;,
  &#39;doi&#39;: &#39;10.1101/337253&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThis work aimed to evaluate the cytotoxic potential against cancer cells of Passiflora genus plant species cultivated in Brazil and identify the mechanism of cytotoxicity induced by the most promising extract. Leaf extracts from 14 Passiflora (P.) species were obtained ASE and in vitro cytotoxicity evaluated against cancer cell lines using MTT assay at a single concentration of 50 μg/mL. Additionally, the IC50 of the P. alata (ELPA) leaf extracts was determined against both tumor (HCT-116, SF-295, OVACAR-8, and HL-60), and non-tumor cells (PBMC). The ELPA flavonoids were identified by HPLC-DAD and UHPLC-MS/MS. The morphological analyses used light and fluorescence microscopy, and cell cycle and DNA fragmentation analyses used flow cytometry to determine the mechanism of cell death induced by ELPA in HL-60. Among the Passiflora leaf extracts evaluated; ELPA stood out with high cytotoxic activity, followed by P. capsularis and P. quadrangulares with varying high and low cytotoxic activity. ELPA presented high cytotoxic potency in HL-60 (IC50 19.37 μg/mL), yet without cytotoxic activity against PBMC, suggesting selectivity for tumor cells. The cytotoxic activity of ELPA may well be linked to the presence of ten identified flavonoids. Cells treated with ELPA presented the hallmarks typical of apoptosis and necrosis, with cell cycle arrest in the G2/M phase. Conclusion: From among the studied species, ELPA presented greater cytotoxic activity, possibly a consequence of synergistic flavonoid action which induces cell death by apoptosis and necrosis.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-01&#39;,
  &#39;authors&#39;: &#39;Ricardo Guimarães Amaral; Silvana Vieira Floresta Gomes; Ângelo Roberto Antoniolli; Maria Claudia dos Santos Luciano; Cláudia do Ó Pessoa; Luciana Nalone Andrade; Patrícia Severino; Geraldo Célio Brandão; Larissa Mendes Bomfim; Daniel Pereira Bezerra; Jorge Maurício David; Adriana Andrade Carvalho&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/337253&#39;},
 {&#39;cord_uid&#39;: &#39;ltb6l5xz&#39;,
  &#39;sha&#39;: &#39;558d318e1655da9f5e9507ccf24fc773a0d377ff&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Connectivity analyses of bioenergetic changes in schizophrenia: Identification of novel treatments&#39;,
  &#39;doi&#39;: &#39;10.1101/338392&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTWe utilized a cell-level approach to examine glycolytic pathways in the DLPFC of subjects with schizophrenia (n=16) and control (n=16) subjects and found decreased mRNA expression of glycolytic enzymes in pyramidal neurons, but not astrocytes. To replicate these novel bioenergetic findings, we probed independent datasets for bioenergetic targets and found similar abnormalities. Next, we used a novel strategy to build a schizophrenia bioenergetic profile by a tailored application of the Library of Integrated Network-Based Cellular Signatures data portal (iLINCS) and investigated connected cellular pathways, kinases, and transcription factors using Enrichr. Finally, with the goal of identifying drugs capable of “reversing” the bioenergetic schizophrenia signature, we performed a connectivity analysis with iLINCS and identified peroxisome proliferator-activated receptor (PPAR) agonists as promising therapeutic targets. We administered a PPAR agonist to the GluN1 knockdown model of schizophrenia and found it improved long-term memory. Taken together, our findings suggest that tailored bioinformatics approaches, coupled with the LINCS library of transcriptional signatures of chemical and genetic perturbagens may be employed to identify novel treatment strategies for schizophrenia and related diseases.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-05&#39;,
  &#39;authors&#39;: &#39;Courtney R. Sullivan; Catharine A. Mielnik; Sinead M. O’Donovan; Adam J. Funk; Eduard Bentea; Erica A.K. DePasquale; Zhexing Wen; Vahram Haroutunian; Pavel Katsel; Amy J. Ramsey; Jarek Meller; Robert E. McCullumsmith&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/338392&#39;},
 {&#39;cord_uid&#39;: &#39;ieh5cvw1&#39;,
  &#39;sha&#39;: &#39;982a6fe2fdba30430b547741811ff4411f1485f1&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Molecular basis for the evolution of species-specific hemoglobin capture by pathogenic Staphylococcus&#39;,
  &#39;doi&#39;: &#39;10.1101/339705&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTMetals are a limiting resource for pathogenic bacteria and must be scavenged from host proteins. Hemoglobin provides the most abundant source of iron in the human body and is required by several pathogens to cause invasive disease. However, the consequences of hemoglobin evolution for bacterial nutrient acquisition remain unclear. Here we show that the α- and β-globin genes exhibit strikingly parallel signatures of adaptive evolution across simian primates. Rapidly evolving sites in hemoglobin correspond to binding interfaces of IsdB, a bacterial hemoglobin receptor encoded by pathogenic Staphylococcus aureus. Using an evolution-guided experimental approach, we demonstrate that divergence between primates and staphylococcal isolates governs hemoglobin recognition and bacterial growth. Reintroducing putative adaptive mutations in α- or β-globin proteins is sufficient to impair S. aureus binding, providing a mechanism for the evolution of disease resistance. These findings suggest that bacterial hemoprotein capture has driven repeated evolutionary conflicts with hemoglobin during primate descent.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-07&#39;,
  &#39;authors&#39;: &#39;Jacob E. Choby; Hanna B. Buechi; Allison J. Farrand; Eric P. Skaar; Matthew F. Barber&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/339705&#39;},
 {&#39;cord_uid&#39;: &#39;d0eynz67&#39;,
  &#39;sha&#39;: &#39;0acc1f9a1c333a9a6b2dbba4a252d7576f024783&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation&#39;,
  &#39;doi&#39;: &#39;10.1101/341735&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMotivationUnderstanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.ResultsWe present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.AvailabilityThe database is available at https://life.bsc.es/pid/skempi2/&#39;,
  &#39;publish_time&#39;: &#39;2018-06-07&#39;,
  &#39;authors&#39;: &#39;Justina Jankauskaitė; Brian Jiménez-García; Justas Dapkūnas; Juan Fernández-Recio; Iain H. Moal&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/341735&#39;},
 {&#39;cord_uid&#39;: &#39;djmimi2c&#39;,
  &#39;sha&#39;: &#39;ebb553a859d31ebcfc57d489ca84af0df894dab8&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Choanoflagellate transfection illuminates their cell biology and the ancestry of animal septins&#39;,
  &#39;doi&#39;: &#39;10.1101/343111&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTAs the closest living relatives of animals, choanoflagellates offer unique insights into animal origins and core mechanisms underlying animal cell biology. However, unlike traditional model organisms, such as yeast, flies and worms, choanoflagellates have been refractory to DNA delivery methods for expressing foreign genes. Here we report the establishment of a robust method for expressing transgenes in the choanoflagellate Salpingoeca rosetta, overcoming barriers that have previously hampered DNA delivery and expression. To demonstrate how this method accelerates the study of S. rosetta cell biology, we engineered a panel of fluorescent protein markers that illuminate key features of choanoflagellate cells. We then investigated the localization of choanoflagellate septins, a family of GTP-binding cytoskeletal proteins that are hypothesized to regulate the multicellular rosette development in S. rosetta. Fluorescently tagged septins localized to the basal pole of S. rosetta single cells and rosettes in a pattern resembling septin localization in animal epithelia. The establishment of transfection in S. rosetta and its application to the study of septins represent critical advances in the growth of S. rosetta as an experimental model for investigating choanoflagellate cell biology, core mechanisms underlying animal cell biology, and the origin of animals.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-09&#39;,
  &#39;authors&#39;: &#39;David S. Booth; Heather Szmidt-Middleton; Nicole King&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/343111&#39;},
 {&#39;cord_uid&#39;: &#39;c3lmmll6&#39;,
  &#39;sha&#39;: &#39;4ed084719f40fac25989dbc99d08c2c609b33289&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;G-quadruplex forming sequences in the genome of all known human viruses: a comprehensive guide&#39;,
  &#39;doi&#39;: &#39;10.1101/344127&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTG-quadruplexes are non-canonical nucleic acid structures that control transcription, replication, and recombination in organisms. G-quadruplexes are present in eukaryotes, prokaryotes, and viruses. In the latter, mounting evidence indicates their key biological activity. Since data on viruses are scattered, we here present a comprehensive analysis of putative G-quadruplexes in the genome of all known viruses that can infect humans. We show that the presence, distribution, and location of G-quadruplexes are features characteristic of each virus class and family. Our statistical analysis proves that their presence within the viral genome is orderly arranged, as indicated by the possibility to correctly assign up to two-thirds of viruses to their exact class based on the G-quadruplex classification. For each virus we provide: i) the list of all G-quadruplexes formed by GG-, GGG- and GGGG-islands present in the genome (positive and negative strands), ii) their position in the viral genome along with the known function of that region, iii) the degree of conservation among strains of each G-quadruplex in its genome context, iv) the statistical significance of G-quadruplex formation. This information is accessible from a database (http://www.medcomp.medicina.unipd.it/main_site/doku.php?id=g4virus) to allow the easy and interactive navigation of the results. The availability of these data will greatly expedite research on G-quadruplex in viruses, with the possibility to accelerate finding therapeutic opportunities to numerous and some fearsome human diseases.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-11&#39;,
  &#39;authors&#39;: &#39;Enrico Lavezzo; Michele Berselli; Ilaria Frasson; Rosalba Perrone; Giorgio Palù; Alessandra R. Brazzale; Sara N. Richter; Stefano Toppo&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/344127&#39;},
 {&#39;cord_uid&#39;: &#39;ricb63xp&#39;,
  &#39;sha&#39;: &#39;c5bf5beb177c16aa29255e89e0cf47e4e9467cce&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The impact of news exposure on collective attention in the United States during the 2016 Zika epidemic&#39;,
  &#39;doi&#39;: &#39;10.1101/346411&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractIn recent years, many studies have drawn attention to the important role of collective awareness and human behaviour during epidemic outbreaks. A number of modelling efforts have investigated the interaction between the disease transmission dynamics and human behaviour change mediated by news coverage and by information spreading in the population. Yet, given the scarcity of data on public awareness during an epidemic, few studies have relied on empirical data. Here, we use fine-grained, geo-referenced data from three online sources – Wikipedia, the GDELT Project and the Internet Archive – to quantify population-scale information seeking about the 2016 Zika virus epidemic in the U.S., explicitly linking such behavioural signal to epidemiological data. Geolocalized Wikipedia pageview data reveal that visiting patterns of Zika-related pages in Wikipedia were highly synchronized across the United States and largely explained by exposure to national television broadcast. Contrary to the assumption of some theoretical models, news volume and Wikipedia visiting patterns were not significantly correlated with the magnitude or the extent of the epidemic. Attention to Zika, in terms of Zika-related Wikipedia pageviews, was high at the beginning of the outbreak, when public health agencies raised an international alert and triggered media coverage, but subsequently exhibited an activity profile that suggests nonlinear dependencies and memory effects in the relation between information seeking, media pressure, and disease dynamics. This calls for a new and more general modelling framework to describe the interaction between media exposure, public awareness and disease dynamics during epidemic outbreaks.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-17&#39;,
  &#39;authors&#39;: &#39;Michele Tizzoni; André Panisson; Daniela Paolotti; Ciro Cattuto&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/346411&#39;},
 {&#39;cord_uid&#39;: &#39;6pqjnkkw&#39;,
  &#39;sha&#39;: &#39;736b4793b11d01f4b55ea53eaf59f7325493b27b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Knowledge and attitudes of Ebola among the general public of Trinidad and Tobago during the 2014-15 West Africa outbreak&#39;,
  &#39;doi&#39;: &#39;10.1101/346999&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractObjectiveHealth system resilience and resilience of a country include the capacity of health personnel, institutions, and populations to prepare for and effectively respond to crises. This study investigates the knowledge and attitudes of the public concerning Ebola Virus Disease in Trinidad and Tobago.Design and MethodsA cross sectional study whereby respondents (n = 920) were sampled from public places. Data were collected via interviewer administered questionnaires. Data were analysed using SPSS version 23.ResultsThe response rate was 67.6 % (622/920). The main age category of responders was the 20 to 30 year age category (40.5%); responders were mostly female (58.0 %). Regarding knowledge, there were significant differences among occupational categories (F = 2.811, df1 = 6, df2 = 571, p-value = 0.011). Tukey’s HSD post hoc test revealed that the mean knowledge scores for professional and sales occupations differed significantly (p-value = 0.003). There was a significant association between being afraid to go for treatment and age (p-value = 0.001). Significant associations were also found between occupational grouping and education attainment with opinion about the preparedness of private medical facilities, likelihood to shun family members with Ebola, being afraid to go for treatment and preference for traditional medicine (p-value &lt;0.05).ConclusionThis study highlights opportunities for community engagement to enhance health system resilience during outbreaks which would maximise national and global health security.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-14&#39;,
  &#39;authors&#39;: &#39;S Pooransingh; S Mohammed; K Melville; C Mohammed; M Mohammed; R Mohammed; W Mootoo; M Morris; D Motilal; B Bhagwandeen; I Dialsingh&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/346999&#39;},
 {&#39;cord_uid&#39;: &#39;4fgz09cv&#39;,
  &#39;sha&#39;: &#39;dc3d8b94ae9acb1ec585507d0d77164a7f688d10&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Inflammation induced by influenza virus impairs innate control of human pneumococcal carriage&#39;,
  &#39;doi&#39;: &#39;10.1101/347161&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractSecondary bacterial pneumonia following influenza infection is a significant cause of mortality worldwide. Upper respiratory tract pneumococcal carriage is important as both determinants of disease and population transmission. The immunological mechanisms that contain pneumococcal carriage are well-studied in mice but remain unclear in humans. Loss of this control of carriage following influenza infection is associated with secondary bacterial pneumonia during seasonal and pandemic outbreaks. We used a human type 6B pneumococcal challenge model to show that carriage acquisition induces early degranulation of resident neutrophils and recruitment of monocytes to the nose. Monocyte function associated with clearance of pneumococcal carriage. Prior nasal infection with live attenuated influenza virus induced inflammation, impaired innate function and altered genome-wide nasal gene responses to pneumococcal carriage. Levels of the cytokine IP-10 promoted by viral infection at the time of pneumococcal encounter was positively associated with bacterial density. These findings provide novel insights in nasal immunity to pneumococcus and viral-bacterial interactions during co-infection.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-20&#39;,
  &#39;authors&#39;: &#39;Simon P. Jochems; Fernando Marcon; Beatriz F. Carniel; Mark Holloway; Elena Mitsi; Emma Smith; Jenna F. Gritzfeld; Carla Solórzano; Jesús Reiné; Sherin Pojar; Elissavet Nikolaou; Esther L. German; Angie Hyder-Wright; Helen Hill; Caz Hales; Wouter A.A de Steenhuijsen Piters; Debby Bogaert; Hugh Adler; Seher Zaidi; Victoria Connor; Jamie Rylance; Helder I. Nakaya; Daniela M. Ferreira&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/347161&#39;},
 {&#39;cord_uid&#39;: &#39;zj8gf45p&#39;,
  &#39;sha&#39;: &#39;f53e10f9cd4b97134ded58b3de4bcc36a55304d3&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes&#39;,
  &#39;doi&#39;: &#39;10.1101/349001&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractDefective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both in vitro and in vivo. However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of RSV cbDVGs. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus that selectively produced a specific cbDVG based on variations introduced in this sequence. The identified sequence was also found as a common site for cbDVG generation during natural RSV infections, and common cbDVGs generated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome are critical determinants of the location of cbDVG generation and, therefore, this is not a stochastic process. Most importantly, these findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.Author summaryCopy-back defective viral genomes (cbDVGs) regulate infection and pathogenesis of Mononegavirales. cbDVG are believed to arise from random errors that occur during virus replication and the predominant hypothesis is that the viral polymerase is the main driver of cbDVG generation. Here we describe a specific genomic sequence in the RSV genome that is necessary for the generation of a large proportion of the cbDVG population present during infection. We identified specific nucleotides that when modified altered cbDVG generation at this position, and we created a recombinant virus that selectively produced cbDVGs based on mutations in this sequence. These data demonstrate that the generation of RSV cbDVGs is regulated by specific viral sequences and that these sequences can be manipulated to alter the content and quality of cbDVG generated during infection.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-15&#39;,
  &#39;authors&#39;: &#39;Yan Sun; Eun Ji Kim; Sébastien A. Felt; Louis J. Taylor; Divyansh Agarwal; Gregory R. Grant; Carolina B. López&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/349001&#39;},
 {&#39;cord_uid&#39;: &#39;qgc8o6ps&#39;,
  &#39;sha&#39;: &#39;20154fa12ca525582cd1b42019927e8f7816fb51&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Validation and automation of a high-throughput multi-targeted method for semi-quantification of endogenous metabolites from different biological matrices using tandem mass spectrometry&#39;,
  &#39;doi&#39;: &#39;10.1101/352468&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe use of metabolomics profiling to understand metabolism under different physiological states has increased in recent years, which created the need for robust analytical platforms. Here, we present a validated method for targeted and semi-quantitative analysis of 102 polar metabolites that covers major metabolic pathways from 24 classes in a single 17.5-min assay. The method has been optimized for a wide range of biological matrices from various organisms, and involves automated sample preparation, and data processing using in-house developed R package. To ensure reliability, the method was validated for accuracy, precision, selectivity, specificity, linearity, recovery, and stability according to European Medicines Agency guidelines. We demonstrated excellent repeatability of the retention times (CV&lt;4%), calibration curves (R2≥0.980) in their respective wide dynamic concentration ranges (CV&lt;3%), and concentrations (CV&lt;25%) of quality control samples interspersed within 25 batches analyzed over a period of one-year. The robustness was demonstrated through high correlation between metabolite concentrations measured using our method and NIST reference values (R2=0.967), including cross-platform comparability against the BIOCRATES AbsoluteIDQp180 kit (R2=0.975) and NMR analyses (R2=0.884). We have shown that our method can be successfully applied in many biomedical research fields and clinical trials, including epidemiological studies for biomarker discovery. In summary, a thorough validation demonstrated that our method is reproducible, robust, reliable, and suitable for metabolomics studies.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-20&#39;,
  &#39;authors&#39;: &#39;Jatin Nandania; Gopal Peddinti; Alberto Pessia; Meri Kokkonen; Vidya Velagapudi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/352468&#39;},
 {&#39;cord_uid&#39;: &#39;q2ivrcya&#39;,
  &#39;sha&#39;: &#39;d1c77a93f69b5dbcf3806674bff9fe4677209ae5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Infectious bronchitis virus attaches to lipid rafts and enters cells via clathrin mediated endocytosis&#39;,
  &#39;doi&#39;: &#39;10.1101/352898&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTDue to its economic importance to in poultry industry, the biology and pathogenesis of infectious bronchitis virus (IBV) have been investigated extensively. However, the molecular mechanisms involved in IBV entry are not well characterized. In this study, systematic approaches were used to dissect IBV entry process in various susceptible cells. First, we observed that lipid rafts were involved in IBV attachment. Second, low pH in intracyplasmic vesicles was required for virus entry. By using the specific clathrin mediated endocytosis (CME) inhibitor or knock down of clathrin heavy chain (CHC), we demonstrated that IBV mainly utilized the CME for its entry. Furthermore, GTPase dynamin1 was involved in virus containing vesicle scission and internalization. Surprisingly, CME adaptor Eps15 had no effect on IBV internalization. Third, the penetration of IBV into cells led to active cytoskeleton rearrangement. After internalization, virus particles moved along with the classical endosome/lysosome track, as evidenced by co-localization of R18 labeled IBV with vehicle markers Rab5/Rab7/LAMP1 along with the infection time course. Functional inactivation of Rab5 and Rab7 significantly inhibited IBV infection. VCP, a protein helps early endosome maturation, was involved virus trafficking. Finally, by using the dual R18/DiOC labeled IBV, we observed that membrane fusion with late endosome/lysosome membranes was induced between 2-3 h.p.i.. Taken together, our findings demonstrate that IBV virions attach to lipid rafts and are internalized into cells via CME, move along with early/late endosomes-lysosomes, finally fuse with late endosome-lysosome membranes, release virus genome into cytoplasm. This study provides comprehensive images of IBV attachment-internalization-trafficking-fusion steps.IMPORTANCEIBV, the avian coronavirus isolated in 1937, infects chicken and causes economic loss in poultry industry. It has been reported that the entry of IBV requires low pH. However, the molecular mechanisms underlying IBV internalization and trafficking remain to be clarified. Therefore, we employed multiple chemical and molecular approaches to dissect the entry mechanisms of IBV in susceptible cells. Our results showed IBV entry was significantly inhibited when clathrin-mediated endocytosis (CME) was blocked by chemical inhibitor or depletion of clathrin protein. Moreover, by using R18-labeled IBV, we found that IBV particles attached to lipid rafts, led to actin rearrangement, and moved along with the entire endosomal system. R18/DiOC labeling method showed that IBV fused with late endosomes or lysosomes. This is the first report to describe the entire entry process of IBV, allowing for a better understanding of the infection process of group III avian coronavirus.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-21&#39;,
  &#39;authors&#39;: &#39;Huan Wang; Yingjie Sun; Xiang Mao; Chunchun Meng; Lei Tan; Cuiping Song; Xusheng Qiu; Chan Ding; Ying Liao&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/352898&#39;},
 {&#39;cord_uid&#39;: &#39;usrtbcj4&#39;,
  &#39;sha&#39;: &#39;ae4f1ac93bb55da1231f08eb1654f19352ab1a3f&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Viral etiology of Acute Respiratory Infections in Hospitalized Children in Novosibirsk City, Russia (2013 – 2017)&#39;,
  &#39;doi&#39;: &#39;10.1101/353037&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;&#39;,
  &#39;publish_time&#39;: &#39;2018-06-21&#39;,
  &#39;authors&#39;: &#39;Olga Kurskaya; Tatyana Ryabichenko; Natalya Leonova; Weifeng Shi; Hongtao Bi; Kirill Sharshov; Eugenia Kazachkova; Ivan Sobolev; Elena Prokopyeva; Tatyana Kartseva; Alexander Alekseev; Alexander Shestopalov&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/353037&#39;},
 {&#39;cord_uid&#39;: &#39;8dnyjuig&#39;,
  &#39;sha&#39;: &#39;906b616191eea5e5e724d47546f7d2d9d096ed7a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Ensemble Feature Selection and Meta-Analysis of Cancer miRNA Biomarkers&#39;,
  &#39;doi&#39;: &#39;10.1101/353201&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe role of microRNAs (miRNAs) in cellular processes captured the attention of many researchers, since their dysregulation is shown to affect the cancer disease landscape by sustaining proliferative signaling, evading program cell death, and inhibiting growth suppressors. Thus, miRNAs have been considered important diagnostic and prognostic biomarkers for several types of tumors. Machine learning algorithms have proven to be able to exploit the information contained in thousands of miRNAs to accurately predict and classify cancer types. Nevertheless, extracting the most relevant miRNA expressions is fundamental to allow human experts to validate and make sense of the results obtained by automatic algorithms. We propose a novel feature selection approach, able to identify the most important miRNAs for tumor classification, based on consensus on feature relevance from high-accuracy classifiers of different typologies. The proposed methodology is tested on a real-world dataset featuring 8,129 patients, 29 different types of tumors, and 1,046 miRNAs per patient, taken from The Cancer Genome Atlas (TCGA) database. A new miRNA signature is suggested, containing the 100 most important oncogenic miRNAs identified by the presented approach. Such a signature is proved to be sufficient to identify all 29 types of cancer considered in the study, with results nearly identical to those obtained using all 1,046 features in the original dataset. Subsequently, a meta-analysis of the medical literature is performed to find references to the most important biomarkers extracted by the methodology. Besides known oncomarkers, 15 new miRNAs previously not ranked as important biomarkers for diagnosis and prognosis in cancer pathologies are uncovered. Such miRNAs, considered relevant by the machine learning algorithms, but still relatively unexplored by specialized literature, could provide further insights in the biology of cancer.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-21&#39;,
  &#39;authors&#39;: &#39;Lopez-Rincon Alejandro; Martinez-Archundia Marlet; Martinez-Ruiz Gustavo Ulises; Tonda Alberto&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/353201&#39;},
 {&#39;cord_uid&#39;: &#39;n75siuwb&#39;,
  &#39;sha&#39;: &#39;e7ff9455c483c237c3ee08b9d230b7be65c064bf&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Imaging Striatal Dopamine Release Using a Non-Genetically Encoded Near-Infrared Fluorescent Catecholamine Nanosensor&#39;,
  &#39;doi&#39;: &#39;10.1101/356543&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractNeuromodulation plays a critical role in brain function in both health and disease. New optical tools, and their validation in biological tissues, are needed that can image neuromodulation with high spatial and temporal resolution, which will add an important new dimension of information to neuroscience research. Here, we demonstrate the use of a catecholamine nanosensor with fluorescent emission in the 1000-1300 nm near-infrared window to measure dopamine transmission in ex vivo brain slices. These near-infrared catecholamine nanosensors (nIRCats) represent a broader class of nanosensors that can be synthesized from non-covalent conjugation of single wall carbon nanotubes (SWNT) with single strand oligonucleotides. We show that nIRCats can be used to detect catecholamine efflux in brain tissue driven by both electrical stimulation or optogenetic stimulation. Spatial analysis of electrically-evoked signals revealed dynamic regions of interest approximately 2 microns in size in which transients scaled with simulation intensity. Optogenetic stimulation of dopaminergic terminals produced similar transients, whereas optogenetic stimulation of glutamatergic terminals showed no effect on nIRCat signal. Bath application of nomifensine prolonged nIRCat fluorescence signal, consistent with reuptake blockade of dopamine. We further show that the chemically synthetic molecular recognition elements of nIRCats permit measurement of dopamine dynamics in the presence of dopamine receptor agonists and antagonists. These nIRCat nanosensors may be advantageous for future use because i) they do not require virus delivery, gene delivery, or protein expression, ii) their near-infrared fluorescence facilitates imaging in optically scattering brain tissue and is compatible for use in conjunction with other optical neuroscience tool sets, iii) the broad availability of unique near-infrared colors have the potential for simultaneous detection of multiple neurochemical signals, and iv) they are compatible with pharmacology. Together, these data suggest nIRCats and other nanosensors of this class can serve as versatile new optical tools to report dynamics of extracellular neuromodulation in the brain.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-03&#39;,
  &#39;authors&#39;: &#39;Abraham G. Beyene; Kristen Delevich; Jackson Travis Del Bonis-O’Donnell; David J. Piekarski; Wan Chen Lin; A. Wren Thomas; Sarah J. Yang; Polina Kosillo; Darwin Yang; Linda Wilbrecht; Markita P. Landry&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/356543&#39;},
 {&#39;cord_uid&#39;: &#39;57xx291v&#39;,
  &#39;sha&#39;: &#39;83cca3779f8aeea47c5426e1d0fa65f1e9567e0c&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;G-quadruplex stabilization in the ions and maltose transporters inhibit Salmonella enterica growth and virulence&#39;,
  &#39;doi&#39;: &#39;10.1101/357046&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractThe G-quadruplex structure forming motifs have recently emerged as a novel therapeutic drug target in various human pathogens. Herein, we report three highly conserved G-quadruplex motifs (SE-PGQ-1, 2, and3) in genome of all the 412 strains of Salmonella enterica. Bioinformatics analysis inferred the presence of SE-PGQ-1 in the regulatory region of mgtA, presence of SE-PGQ-2 in the open reading frame of entA and presence of SE-PGQ-3 in the promoter region of malE and malK genes. The products of mgtA and entA are involved in transport and homeostasis of Mg2+ and Fe3+ ion and thereby required for bacterial survival in the presence of reactive nitrogen/oxygen species produced by the host macrophages, whereas, malK and malE genes are involved in transport of maltose sugar, that is one of the major carbon source in the gastrointestinal tract of human. The formation of stable intramolecular G-quadruplex structures by SE-PGQs was confirmed by employing CD, EMSA and NMR spectroscopy. Cellular studies revealed the inhibitory effect of 9-amino acridine on Salmonella enterica growth. Next, CD melting analysis demonstrated the stabilizing effect of 9-amino acridine on SE-PGQs. Further, polymerase inhibition and RT-qPCR assays emphasize the biological relevance of predicted G-quadruplex in the expression of PGQ possessing genes and demonstrate the G-quadruplexes as a potential drug target for the devolping novel therapeutics for combating Salmonella enterica infection.Author SummarySince last several decades’ scientific community has witnessed a rapid increase in number of such human pathogenic bacterial species that acquired resistant to multiple antibacterial agents. Currently, emergence of multidrug-resistant strains remain a major public health concern for clinical investigators that rings a global alarm to search for novel and highly conserved drug targets. Recently, G-quadruplex structure forming nucleic acid sequences were endorsed as highly conserved Drug target for preventing infection of several human pathogens including viral and protozoan species. Therefore, here we explored the presence G-quadruplex forming motif in genome of Salmonella enterica bacteria that causes food poisoning, and enteric fever in human. The formation of intra molecular G-quadruplex structure in four genes (mgtA, entA, malE and malK) was confirmed by NMR, CD and EMSA. The 9-amino acridine, a known G-quadruplex binder has been shown to stabilize the predicted G-quadruplex motif and decreases the expressioin of G-quadruplex hourbouring genes using RT-PCR and cellular toxicity assay. This study concludes the presence of G-quadruplex motifs in essential genes of Salmonella enterica genome as a novel and conserved drug target and 9-amino acridine as candidate small molecule for preventing the infection of Salmonella enterica using a G4 mediated inhibition mechanism.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-27&#39;,
  &#39;authors&#39;: &#39;Neha Jain; Subodh Kumar Mishra; Uma Shankar; Arpita Tawani; Ankit Jaiswal; Tarun Kumar Sharma; Prashant Kodgire; Amit Kumar&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/357046&#39;},
 {&#39;cord_uid&#39;: &#39;7tulgjhy&#39;,
  &#39;sha&#39;: &#39;b9b0cad2db02479624a3d087626811e4d2fbcdb9&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Changes in temperature alter susceptibility to a virus following a host shift&#39;,
  &#39;doi&#39;: &#39;10.1101/358564&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractHost shifts - where a pathogen jumps between different host species - are an important source of emerging infectious disease. With ongoing climate change there is an increasing need to understand the effect changes in temperature may have on emerging infectious disease. We investigated whether species’ susceptibilities change with temperature and ask if susceptibility is greatest at different temperatures in different species. We infected 45 species of Drosophilidae with an RNA virus and measured how viral load changes with temperature. We found the host phylogeny explained a large proportion of the variation in viral load at each temperature, with strong phylogenetic correlations between viral loads across temperature. The variance in viral load increased with temperature, whilst the mean viral load did not, such that as temperature increased the most susceptible species become more susceptible, and the least susceptible less so. We found no significant relationship between a species’ susceptibility across temperatures and proxies for thermal optima; critical thermal maximum and minimum or basal metabolic rate. These results suggest that whilst the rank order of species susceptibilities can remain the same with changes in temperature, the likelihood of host shifts into a given species may increase or decrease.Author SummaryEmerging infectious diseases are often the result of a host shift, where a pathogen jumps from one host species into another. Understanding the factors underlying host shifts is a major goal for infectious disease researchers. This effort has been further complicated by the fact that host-parasite interactions are now taking place in a period of unprecedented global climatic warming. Here, we ask how host shifts are affected by temperature by carrying out experimental infections using an RNA virus across a wide range of related species, at three different temperatures. We find that as temperature increases the most susceptible species become more susceptible, and the least susceptible less so. This has important consequences for our understanding of host shift events in a changing climate, and suggests that temperature changes may affect the likelihood of a host shift into certain species.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-28&#39;,
  &#39;authors&#39;: &#39;Katherine E. Roberts; Jarrod D. Hadfield; Manmohan D. Sharma; Ben Longdon&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/358564&#39;},
 {&#39;cord_uid&#39;: &#39;ivyp5cgz&#39;,
  &#39;sha&#39;: &#39;7b89a657dde3e5888c31d1d5ef3b90ceb0f66a62&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Beyond type 1 regulatory T cells: co-expression of LAG3 and CD49b in IL-10-producing T cell lineages&#39;,
  &#39;doi&#39;: &#39;10.1101/359547&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTType 1 regulatory CD4+ T (Tr1) cells express high levels of the immunosuppressive cytokine IL-10 but not the master transcription factor Foxp3, and can suppress inflammation and promote immune tolerance. In order to identify and obtain viable Tr1 cells for research and clinical applications, co-expression of CD49b and LAG3 has been proposed as a unique surface signature for both human and mouse Tr1 cells. However, recent studies have revealed that this pattern of co-expression is dependent on the stimulating conditions and the differentiation stage of the CD4+ T cells. Here, using an IL-10GFP/Foxp3RFP dual reporter transgenic murine model, we demonstrate that co-expression of CD49b and LAG3 is not restricted to the Foxp3− Tr1 cells, but is also observed in Foxp3+ T regulatory (Treg) cells and CD8+ T cells that produce IL-10. Our data indicate that IL-10-producing Tr1 cells, Treg cells and CD8+ T cells are all capable of co-expressing LAG3 and CD49b in vitro following differentiation under IL-10-inducing conditions, and in vivo following pathogenic insult or infection in the pulmonary mucosa. Our findings urge caution in the use of LAG3/CD49b co-expression to identify Tr1 cells, since it may mark IL-10-producing T cell lineages more broadly, including the Foxp3− Tr1 cells, Foxp3+ Treg cells and CD8+ T cells.&#39;,
  &#39;publish_time&#39;: &#39;2018-06-30&#39;,
  &#39;authors&#39;: &#39;Weishan Huang; Sabrina Solouki; Chavez Carter; Song-Guo Zheng; Avery August&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/359547&#39;},
 {&#39;cord_uid&#39;: &#39;7xb7nuw5&#39;,
  &#39;sha&#39;: &#39;10424da2599f0258bfaf71a3186b999ab2d747d0&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Risk of disease spillover from dogs to wild carnivores in Kanha Tiger Reserve, India&#39;,
  &#39;doi&#39;: &#39;10.1101/360271&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractMany mammalian carnivore species have been reduced to small, isolated populations by habitat destruction, fragmentation, poaching, and human conflict. Their limited genetic variability and increased exposure to domestic animals such as dogs place them at risk of further losses from infectious diseases. In India, domestic and feral dogs are associated with villages in and around protected areas, and may serve as reservoirs of pathogens to the carnivores within. India’s Kanha Tiger Reserve (KTR) is home to a number of threatened and endangered mammalian carnivores including tiger (Panthera tigris), leopard (Panthera pardus), wolf (Canis lupus), and dhole (Cuon alpinus). It also has more than 150 villages with associated dog populations. We found that dog populations ranged from 14 to 45/village (3.7 to 23.7/km2), and did not vary with village area, human population size, or distance from the KTR’s core area, though they all increased between summer 2014 and winter 2015, primarily through reproduction. No dog tested positive for rabies but seroprevalence levels to three other generalist viral pathogens were high in summer (N=67) and decreased somewhat by winter (N=168): canine parvovirus (83.6% to 68.4%), canine distemper virus (50.7% to 30.4%) and canine adenovirus (41.8% to 30.9%). The declines in seroprevalence were primarily due to new recruitments by birth and these were not yet exposed to the viruses. Wild carnivores frequently entered the villages, as shown by tracks, scats, kills and other indicators, and the dogs are known to leave the villages so that encounters between dogs and wild carnivores may be common. We conclude that there is a large population of unvaccinated dogs in and around Kanha Tiger Reserve, with high levels of seroprevalence to pathogens with broad host ranges and these dogs which interact with wild carnivores, therefore posing a high risk of disease spillover to the wild carnivores.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-03&#39;,
  &#39;authors&#39;: &#39;V. Chaudhary; N. Rajput; A. B. Shrivastav; D. W. Tonkyn&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/360271&#39;},
 {&#39;cord_uid&#39;: &#39;wqnvmvr0&#39;,
  &#39;sha&#39;: &#39;6cbefcf4c0fbb8466db7d00b8f53e1f37a6bf8b5&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Integrated Disease Surveillance and Response (IDSR) in Malawi: Implementation Gaps and Challenges for Timely Alert&#39;,
  &#39;doi&#39;: &#39;10.1101/363713&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractObjectiveThe emerging and recent 2014 Ebola Virus Disease (EVD) outbreaks rang the bell to call upon efforts from globe to assist resource-constrained countries to strengthen public health surveillance system for early response. Malawi adopted the Integrated Disease Surveillance and Response (IDSR) strategy to develop its national surveillance system since 2002 and revised its guideline to fulfill the International Health Regulation (IHR) requirements in 2014. This study aimed to understand the state of IDSR implementation and differences between guideline and practice for future disease surveillance system strengthening.MethodsThis was a mixed-method observational study. Quantitative data were to analyze completeness and timeliness of surveillance system performance from national District Health Information System 2 (DHIS2). Qualitative data were collected through interviews with 29 frontline health service providers from the selected district and key informants of the IDSR system implementation and administration at district and national levels.FindingsThe current IDSR system showed relatively good completeness (76.4%) but poor timeliness (41.5%) of total expected monthly reports nationwide and zero weekly reports. The challenges of IDSR implementation revealed through qualitative data included lack of supervision, inadequate resources for training and difficulty to implement weekly report due to overwhelming paperwork at frontline health services.ConclusionsThe differences between IDSR technical guideline and actual practice were huge. The developing information technology infrastructure in Malawi and emerging mobile health (mHealth) technology can be opportunities for the country to overcome these challenges and improve surveillance system to have better timeliness for the outbreaks and unusual events detection.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-06&#39;,
  &#39;authors&#39;: &#39;Tsung-Shu Joseph Wu; Matthew Kagoli; Jens Johan Kaasbøll; Gunnar Aksel Bjune&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/363713&#39;},
 {&#39;cord_uid&#39;: &#39;hdwn2fkj&#39;,
  &#39;sha&#39;: &#39;180c88b68816bbb6eb57aa515a3aec4020178729&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Inferring the presence of aflatoxin-producing Aspergillus flavus strains using RNA sequencing and electronic probes as a transcriptomic screening tool&#39;,
  &#39;doi&#39;: &#39;10.1101/365254&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractE-probe Diagnostic for Nucleic acid Analysis (EDNA) is a bioinformatic tool originally developed to detect plant pathogens in metagenomic databases. However, enhancements made to EDNA increased its capacity to conduct hypothesis directed detection of specific gene targets present in transcriptomic databases. To target specific pathogenicity factors used by the pathogen to infect its host or other targets of interest, e-probes need to be developed for transcripts related to that function. In this study, EDNA transcriptomics (EDNAtran) was developed to detect the expression of genes related to aflatoxin production at the transcriptomic level. E-probes were designed from genes up-regulated during A. flavus aflatoxin production. EDNAtran detected gene transcripts related to aflatoxin production in a transcriptomic database from corn, where aflatoxin was produced. The results were significantly different from e-probes being used in the transcriptomic database where aflatoxin was not produced (atoxigenic AF36 strain and toxigenic AF70 in Potato Dextrose Broth).&#39;,
  &#39;publish_time&#39;: &#39;2018-07-09&#39;,
  &#39;authors&#39;: &#39;Andres S. Espindola; William Schneider; Kitty F. Cardwell; Yisel Carrillo; Peter R. Hoyt; Stephen M. Marek; Hassan Melouk; Carla D. Garzon&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/365254&#39;},
 {&#39;cord_uid&#39;: &#39;d86o6j2s&#39;,
  &#39;sha&#39;: &#39;22264c65eb6c14ba96287ba4da536424dd083253&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Quality control implementation for universal characterization of DNA and RNA viruses in clinical respiratory samples using single metagenomic next-generation sequencing workflow&#39;,
  &#39;doi&#39;: &#39;10.1101/367367&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundIn recent years, metagenomic Next-Generation Sequencing (mNGS) has increasingly been used for an accurate assumption-free virological diagnosis. However, the systematic workflow evaluation on clinical respiratory samples and implementation of quality controls (QCs) is still lacking.MethodsA total of 3 QCs were implemented and processed through the whole mNGS workflow: a notemplate-control to evaluate contamination issues during the process; an internal and an external QC to check the integrity of the reagents, equipment, the presence of inhibitors, and to allow the validation of results for each sample. The workflow was then evaluated on 37 clinical respiratory samples from patients with acute respiratory infections previously tested for a broad panel of viruses using semi-quantitative real-time PCR assays (28 positive samples including 6 multiple viral infections; 9 negative samples). Selected specimens included nasopharyngeal swabs (n = 20), aspirates (n = 10), or sputums (n = 7).ResultsThe optimal spiking level of the internal QC was first determined in order to be sufficiently detected without overconsumption of sequencing reads. According to QC validation criteria, mNGS results were validated for 34/37 selected samples. For valid samples, viral genotypes were accurately determined for 36/36 viruses detected with PCR (viral genome coverage ranged from 0.6% to 100%, median = 67.7%). This mNGS workflow allowed the detection of DNA and RNA viruses up to a semi-quantitative PCR Ct value of 36. The six multiple viral infections involving 2 to 4 viruses were also fully characterized. A strong correlation between results of mNGS and real-time PCR was obtained for each type of viral genome (R2 ranged from 0.72 for linear single-stranded (ss) RNA viruses to 0.98 for linear ssDNA viruses).ConclusionsAlthough the potential of mNGS technology is very promising, further evaluation studies are urgently needed for its routine clinical use within a reasonable timeframe. The approach described herein is crucial to bring standardization and to ensure the quality of the generated sequences in clinical setting. We provide an easy-to-use single protocol successfully evaluated for the characterization of a broad and representative panel of DNA and RNA respiratory viruses in various types of clinical samples.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-11&#39;,
  &#39;authors&#39;: &#39;A. Bal; M. Pichon; C. Picard; JS. Casalegno; M. Valette; I. Schuffenecker; L. Billard; S. Vallet; G. Vilchez; V. Cheynet; G. Oriol; S. Assant; Y. Gillet; B. Lina; K. Brengel-Pesce; F. Morfin; L. Josset&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/367367&#39;},
 {&#39;cord_uid&#39;: &#39;d28hr3fv&#39;,
  &#39;sha&#39;: &#39;896cb03362facd2a670bee13dc5368b81768b2ba&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Zika Virus Infection at Different Pregnancy Stages: Anatomopathological Findings, Target Cells and Viral Persistence in Placental Tissues&#39;,
  &#39;doi&#39;: &#39;10.1101/370528&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTZika virus (ZIKV) infection in humans has been associated with congenital malformations and other neurological disorders, such as Guillain-Barré syndrome. The mechanism(s) of ZIKV intrauterine transmission, the cell types involved, the most vulnerable period of pregnancy for severe outcomes from infection and other physiopathological aspects remain unknown. In this study, we analyzed placental samples obtained at the time of delivery from a group of twenty-four women diagnosed with ZIKV infection during the first, second or third trimesters of pregnancy. Villous immaturity was the main histological finding in the placental tissues, although placentas without alterations were also frequently observed. Significant enhancement of the number of syncytial sprouts was observed in the placentas of women infected during the third trimester, indicating the development of placental abnormalities after ZIKV infection. Hyperplasia of Hofbauer cells (HCs) was also observed in these third-trimester placental tissues, and remarkably, HCs were the only ZIKV-positive fetal cells found in the placentas studied that persisted until birth, as revealed by immunohistochemical (IHC) analysis. Thirty-three percent of women infected during pregnancy delivered infants with congenital abnormalities, although no pattern correlating the gestational stage at infection, the IHC positivity of HCs in placental tissues and the presence of congenital malformations at birth was observed. Placental tissue analysis enabled us to confirm maternal ZIKV infection in cases where serum from the acute infection phase was not available, which reinforces the importance of this technique in identifying possible causal factors of birth defects. The results we observed in the samples from naturally infected pregnant women may contribute to the understanding of some aspects of the pathophysiology of ZIKV.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-16&#39;,
  &#39;authors&#39;: &#39;Lucia de Noronha; Camila Zanluca; Marion Burger; Andreia Akemi Suzukawa; Marina Azevedo; Patricia Z. Rebutini; Iolanda Maria Novadzki; Laurina Setsuko Tanabe; Mayra Marinho Presibella; Claudia Nunes Duarte dos Santos&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/370528&#39;},
 {&#39;cord_uid&#39;: &#39;4718pdtk&#39;,
  &#39;sha&#39;: &#39;e01948b342544a5fcd8c14d757501779029c6d6d&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Chronic infections can shape epidemic exposure: Pathogen co-occurrence networks in the Serengeti lions&#39;,
  &#39;doi&#39;: &#39;10.1101/370841&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractPathogens are embedded in a complex network of microparasites that can collectively or individually alter disease dynamics and outcomes. Chronic pathogens, for example, can either facilitate or compete with subsequent pathogens thereby exacerbating morbidity and mortality. Pathogen interactions are ubiquitous in nature, but poorly understood, particularly in wild populations. We report here on ten years of serological and molecular data in African lions, leveraging comprehensive demographic and behavioral data to utilize pathogen networks to test if chronic infections shape infection by acute pathogens. We combine network and community ecology approaches to assess broad network structure and characterize associations between pathogens across spatial and temporal scales. We found significant non-random structure in the lion-pathogen co-occurrence network and identified potential facilitative and competitive interactions between acute and chronic pathogens. Our results provide a novel insight for untangling the complex associations underlying pathogen co-occurrence networks.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-17&#39;,
  &#39;authors&#39;: &#39;Nicholas M. Fountain-Jones; Craig Packer; Maude Jacquot; F. Guillaume Blanchet; Karen Terio; Meggan E. Craft&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/370841&#39;},
 {&#39;cord_uid&#39;: &#39;dfgb0liu&#39;,
  &#39;sha&#39;: &#39;500c5ea0961b573ba5355d364ca94c00a0a26e4a&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The dengue virus non-structural protein 1 (NS1) is secreted from mosquito cells in association with the intracellular cholesterol transporter chaperone caveolin complex&#39;,
  &#39;doi&#39;: &#39;10.1101/370932&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTDengue virus (DENV) is a mosquito-borne virus of the family Flaviviridae. The RNA viral genome encodes for a polyprotein that is co-translationally processed into three structural proteins and seven non-structural proteins. The non-structural protein 1 (NS1) is a multifunctional viral protein actively secreted in vertebrate and mosquito cells during DENV infection. In mosquito cells, NS1 is secreted in a caveolin-1 (CAV-1) dependent manner by an unconventional pathway. The caveolin chaperone complex (CCC) is a cytoplasmic complex formed by caveolin-1 and the chaperones FKBP52, Cy40 and CyA which is responsible for cholesterol traffic inside the cell. In this work, we demonstrate that in infected mosquito cells, DENV NS1 is secreted by an early and unconventional route that bypasses the Golgi apparatus in close association with the CCC. Treatment of mosquito cells with classic secretion inhibitors such as brefeldin A, golgicide A and Fli-06 showed no effect on NS1 secretion, but significant reductions in recombinant luciferase secretion and virion release. Silencing the expression of CAV1, FKBP52 with siRNAs or the inhibition of CyA by cyclosporine A resulted in significant decrease in NS1 secretion without affecting virion release. Using co-localization, co-inmunoprecipitation and proximity ligation assays, NS1 was found to co-localize and interact with all the protein components of the CCC in mosquito infected cells. In addition, CAV-1 and FKBP52 expression was found augmented in DENV infected cells. Finally, the treatment of ZIKV infected mosquito cells with brefeldin A and golgicide A showed no effect on NS1 secretion, while affecting virion release. ZIKV NS1 was also found to co-localize with CAV-1 in infected mosquito cells. These results suggest that in mosquito cells, ZIKV NS1 follows the same secretory pathway observed for DENV NS1. The association of NS1 with the cholesterol transporter CCC agrees with the lipoprotein nature of secreted hexameric NS1.AUTHOR SUMMARYDengue protein NS1 is secreted in infected mosquito and vertebrate cells. In humans, secreted NS1 have been associated with pathogenesis. In mosquito cells, NS1 follows an unconventional secretion pathway that is dependent on Caveolin-1. This work shows that in mosquito cells, NS1 secretion is associated to the chaperone caveolin complex, a complex formed by caveolin-1 and several chaperones, in charge of cholesterol transport within the cells. Reduction of the expression or the activity of chaperone caveolin complex in mosquito infected cells, diminished the secretion of NS1 without affecting virion release. Direct interaction between NS1 and the chaperone caveolin complex proteins was demonstrated by several assays. Moreover, increased expression of the caveolin-1 and co-chaperone FKBP52 during dengue infection was found, presumably in response to the higher requirements of these proteins during dengue virus infection. Results obtained with ZIKV infected mosquito cells suggest that also ZIKV NS1 is released following an unconventional secretory route in association with the chaperone caveolin complex. The functions of secreted NS1 within mosquito are unclear. However, giving the importance of the soluble NS1 in the vertebrate host, manipulation of the NS1 secretory route may prove a valuable strategy for dengue mosquito control and patient treatment.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-17&#39;,
  &#39;authors&#39;: &#39;Romel Rosales Ramirez; Juan E. Ludert&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/370932&#39;},
 {&#39;cord_uid&#39;: &#39;cyxdy7hg&#39;,
  &#39;sha&#39;: &#39;e7c3ec3dcb3469ee4d608029e7aa3068a4c90c54&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The Role of Proton Transport in Gating Current in a Voltage Gated Ion Channel, as Shown by Quantum Calculations&#39;,
  &#39;doi&#39;: &#39;10.1101/371914&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTOver two-thirds of a century ago, Hodgkin and Huxley proposed the existence of voltage gated ion channels (VGIC) to carry Na+ and K+ ions across the cell membrane to create the nerve impulse, in response to depolarization of the membrane. The channels have multiple physiological roles, and play a central role in a wide variety of diseases when they malfunction. The first channel structure was found by MacKinnon and coworkers in 1998. Subsequently the structure of a number of VGIC was determined in the open (ion conducting) state. This type of channel consists of four voltage sensing domains (VSD), each formed from four transmembrane (TM) segments, plus a pore domain through which ions move. Understanding the gating mechanism (how the channel opens and closes) requires structures. One TM segment (S4) has an arginine in every third position, with one such segment per domain. It is usually assumed that these arginines are all ionized, and in the resting state are held toward the intracellular side of the membrane by voltage across the membrane. They are assumed to move outward (extracellular direction) when released by depolarization of this voltage, producing a capacitive gating current and opening the channel. We suggest alternate interpretations of the evidence that led to these models. Measured gating current is the total charge displacement of all atoms in the VSD; we propose that the prime, but not sole, contributor is proton motion, not displacement of the charges on the arginines of S4. It is known that the VSD can conduct protons. Quantum calculations on the Kv1.2 potassium channel VSD show how; the key is the amphoteric nature of the arginine side chain, which allows it to transfer a proton; this appears to be the first time the arginine side chain has had its amphoteric character considered. We have calculated one such proton transfer in detail: this proton starts from a tyrosine that can ionize, transferring to the NE of the third arginine on S4; that arginine’s NH then transfers a proton to a glutamate. The backbone remains static. A mutation predicted to affect the proton transfer has been qualitatively confirmed experimentally, from the change in the gating current-voltage curve. The total charge displacement in going from a normal closed potential of −70 mV across the membrane to 0 mV (open), is calculated to be approximately consistent with measured values, although the error limits on the calculation require caution in interpretation.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-19&#39;,
  &#39;authors&#39;: &#39;Alisher M Kariev; Michael E Green&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/371914&#39;},
 {&#39;cord_uid&#39;: &#39;zswddzm0&#39;,
  &#39;sha&#39;: &#39;69f473aee22a4ac8df15a54a6109b70b1ae9b7e7&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;HOPS-dependent endosomal fusion required for efficient cytosolic delivery of therapeutic peptides and small proteins&#39;,
  &#39;doi&#39;: &#39;10.1101/374926&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractProtein therapeutics represent a significant and growing component of the modern pharmacopeia, but their potential to treat human disease is limited because most proteins fail to traffic across biological membranes. Recently, we discovered that cell-permeant miniature proteins (CPMPs) containing a precisely defined, penta-arginine motif traffic readily to the cytosol and nucleus with efficiencies that rival those of hydrocarbon-stapled peptides active in animals and man. Like many cell-penetrating peptides (CPPs), CPMPs enter the endocytic pathway; the difference is that CPMPs are released efficiently from endosomes while other CPPs are not. Here, we seek to understand how CPMPs traffic from endosomes into the cytosol and what factors contribute to the efficiency of endosomal release. First, using two complementary cell-based assays, we exclude endosomal rupture as the primary means of endosomal escape. Next, using a broad spectrum of techniques, including an RNA interference (RNAi) screen, fluorescence correlation spectroscopy (FCS), and confocal imaging, we identify VPS39—a gene encoding a subunit of the homotypic fusion and protein sorting (HOPS) complex—as a critical determinant in the trafficking of CPMPs and hydrocarbon-stapled peptides to the cytosol. Although CPMPs neither inhibit nor activate HOPS function, HOPS activity is essential to efficiently deliver CPMPs to the cytosol. Subsequent multi-color confocal imaging studies identify CPMPs within the endosomal lumen, particularly within the intraluminal vesicles (ILVs) of Rab7+ and Lamp1+ endosomes that are the products of HOPS-mediated fusion. These results suggest that CPMPs require HOPS to reach ILVs—an environment that serves as a prerequisite for efficient endosomal escape.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-23&#39;,
  &#39;authors&#39;: &#39;Angela Steinauer; Jonathan R. LaRochelle; Rebecca Wissner; Samuel Berry; Alanna Schepartz&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/374926&#39;},
 {&#39;cord_uid&#39;: &#39;2k0g4hva&#39;,
  &#39;sha&#39;: &#39;110b4b089bd757bd54587c2ab3262a374462c098&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;The permeabilized SecY protein-translocation channel can serve as a nonspecific sugar transporter&#39;,
  &#39;doi&#39;: &#39;10.1101/378786&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractAs the initial step in carbohydrate catabolism in cells, the substrate-specific transporters via active transport and facilitated diffusion play a decisive role in passage of sugars through the plasma membrane into the cytoplasm. The SecY complex (SecYEG) in bacteria forms a membrane channel responsible for protein translocation. This work demonstrates that weakening the sealability of the SecY channel allowed free diffusion of sugars, including glucose, fructose, mannose, xylose, arabinose, and lactose, into the engineered cells, facilitating its rapid growth on a wide spectrum of monosaccharides and bypassing/reducing stereospecificity, transport saturation, competitive inhibition, and carbon catabolite repression (CCR), which are usually encountered with the specific sugar transporters. The SecY channel is structurally conserved in prokaryotes, thus it may be engineered to serve as a unique and universal transporter for bacteria to passage sugars as demonstrated in Escherichia coli and Clostridium acetobutylicum.&#39;,
  &#39;publish_time&#39;: &#39;2018-07-27&#39;,
  &#39;authors&#39;: &#39;Sen Mei; Chong Xie; Hao Mi; Chuang Xue; Qiang Guo; Guang-Qing Du; Guo-Bei Li; Cai-Xia Li; Ya-Nan Qu; Ming-Hao Xiong; Yang Jiang; Tian-Wei Tan; Shang-Tian Yang; Li-Hai Fan&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/378786&#39;},
 {&#39;cord_uid&#39;: &#39;yb5kpaq2&#39;,
  &#39;sha&#39;: &#39;1983cbc32412a9a7b1601b6778605f9e98ab69e0&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Glycosyltransferases promote development and prevent promiscuous cell aggregation in the choanoflagellate S. rosetta&#39;,
  &#39;doi&#39;: &#39;10.1101/384453&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;ABSTRACTThe mechanisms underlying multicellular development in the animal stem lineage may be reconstructed through the study of choanoflagellates, the closest living relatives of animals. To determine the genetic underpinnings of multicellularity in the emerging model choanoflagellate S. rosetta, we performed a screen for mutants with defects in multicellular rosette development. In two of the mutants, Jumble and Couscous, single cells failed to develop into orderly rosettes but instead aggregated promiscuously into amorphous clumps of cells. Both mutants mapped to lesions in genes encoding glycosyltransferases and the mutations perturbed glycosylation patterns in the extracellular matrix (ECM). In animals, glycosyltransferases transfer activated sugars to donor molecules and thereby sculpt the polysaccharide-rich ECM, regulate integrin and cadherin activity, and, when disrupted, contribute to tumorigenesis. The finding that glycosyltransferases promote proper rosette development and prevent cell aggregation in S. rosetta suggests a pre-metazoan role for glycosyltransferases in regulating development and preventing abnormal tumor-like multicellularity.IMPACT STATEMENTA genetic screen reveals that glycosyltransferases are required for proper rosette development and the prevention of cell clumping in one of the closest living relatives of animals, the choanoflagellate S. rosetta.&#39;,
  &#39;publish_time&#39;: &#39;2018-08-04&#39;,
  &#39;authors&#39;: &#39;Laura A. Wetzel; Tera C. Levin; Ryan E. Hulett; Daniel Chan; Grant A. King; Reef Aldayafleh; David S. Booth; Monika Abedin Sigg; Nicole King&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/384453&#39;},
 {&#39;cord_uid&#39;: &#39;myerb07d&#39;,
  &#39;sha&#39;: &#39;6f8a531b4b84d4daaadcff9d7e03ee24274baf0b&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Etiology of fever in Ugandan children: identification of microbial pathogens using metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic analysis&#39;,
  &#39;doi&#39;: &#39;10.1101/385005&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractBackgroundFebrile illness is a major burden in African children, and non-malarial causes of fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics platform and service to identify microbes from metagenomic next-generation sequencing of tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, Uganda.ResultsThe most common pathogens identified were Plasmodium falciparum (51.1% of samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx of a child with rash and sepsis. We also identified two novel human rhinovirus C species.ConclusionsThis exploratory pilot study demonstrates the utility of mNGS and the IDseq platform for defining the molecular landscape of febrile infectious diseases in resource limited areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases.&#39;,
  &#39;publish_time&#39;: &#39;2018-08-04&#39;,
  &#39;authors&#39;: &#39;Akshaya Ramesh; Sara Nakielny; Jennifer Hsu; Mary Kyohere; Oswald Byaruhanga; Charles de Bourcy; Rebecca Egger; Boris Dimitrov; Yun-Fang Juan; Jonathan Sheu; James Wang; Katrina Kalantar; Charles Langelier; Theodore Ruel; Arthur Mpimbaza; Michael R. Wilson; Philip J. Rosenthal; Joseph L. DeRisi&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/385005&#39;},
 {&#39;cord_uid&#39;: &#39;5isns5ka&#39;,
  &#39;sha&#39;: &#39;64478db449af8e4a5645c17f548649a0159da0c6&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Individual and temporal variation in pathogen load predicts long-term impacts of an emerging infectious disease&#39;,
  &#39;doi&#39;: &#39;10.1101/392324&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractEmerging infectious diseases increasingly threaten wildlife populations. Most studies focus on managing short-term epidemic properties, such as controlling early outbreaks. Predicting long-term endemic characteristics with limited retrospective data is more challenging. We used individual-based modelling informed by individual variation in pathogen load and transmissibility to predict long-term impacts of a lethal, transmissible cancer on Tasmanian devil (Sarcophilus harrisii) populations. For this, we employed Approximate Bayesian Computation to identify model scenarios that best matched known epidemiological and demographic system properties derived from ten years of data after disease emergence, enabling us to forecast future system dynamics. We show that the dramatic devil population declines observed thus far are likely attributable to transient dynamics. Only 21% of matching scenarios led to devil extinction within 100 years following devil facial tumour disease (DFTD) introduction, whereas DFTD faded out in 57% of simulations. In the remaining 22% of simulations, disease and host coexisted for at least 100 years, usually with long-period oscillations. Our findings show that pathogen extirpation or host-pathogen coexistence are much more likely than the DFTD-induced devil extinction, with crucial management ramifications. Accounting for individual-level disease progression and the long-term outcome of devil-DFTD interactions at the population-level, our findings suggest that immediate management interventions are unlikely to be necessary to ensure the persistence of Tasmanian devil populations. This is because strong population declines of devils after disease emergence do not necessarily translate into long-term population declines at equilibria. Our modelling approach is widely applicable to other host-pathogen systems to predict disease impact beyond transient dynamics.&#39;,
  &#39;publish_time&#39;: &#39;2018-08-15&#39;,
  &#39;authors&#39;: &#39;Konstans Wells; Rodrigo K. Hamede; Menna E. Jones; Paul A. Hohenlohe; Andrew Storfer; Hamish I. McCallum&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/392324&#39;},
 {&#39;cord_uid&#39;: &#39;zguy9ba7&#39;,
  &#39;sha&#39;: &#39;e4e5e67fd4f75548dffab7f75484bc14fa15c0c2&#39;,
  &#39;source_x&#39;: &#39;biorxiv&#39;,
  &#39;title&#39;: &#39;Existing host range mutations constrain further emergence of RNA viruses&#39;,
  &#39;doi&#39;: &#39;10.1101/394080&#39;,
  &#39;pmcid&#39;: &#39;&#39;,
  &#39;pubmed_id&#39;: &#39;&#39;,
  &#39;license&#39;: &#39;biorxiv&#39;,
  &#39;abstract&#39;: &#39;AbstractRNA viruses are capable of rapid host shifting, typically due to a point mutation that confers expanded host range. As additional point mutations are necessary for further expansions, epistasis among host range mutations can potentially affect the mutational neighborhood and frequency of niche expansion. We mapped the mutational neighborhood of host range expansion using three genotypes of the dsRNA bacteriophage phi6 (wildtype and two isogenic host range mutants) on the novel host Pseudomonas syringae pv. atrofaciens (PA). Sanger sequencing of fifty PA mutant clones for each genotype and population Illumina sequencing both revealed the same high frequency mutations allowing infection of PA. Wildtype phi6 had at least nine different ways of mutating to enter the novel host, eight of which are in p3 (host attachment protein gene), and 13/50 clones had unchanged p3 genes. However, the two isogenic mutants had dramatically restricted neighborhoods: only one or two mutations, all in p3. Deep sequencing revealed that wildtype clones without mutations in p3 likely had changes in p12 (morphogenic protein), a region that was not polymorphic for the two isogenic host range mutants. Sanger sequencing confirmed that 10/13 of the wildtype phi6 clones had nonsynonymous mutations in p12 and two others had point mutations in p9 and p5 – none of these genes had previously been associated with host range expansion in phi6. We demonstrate, for the first time, epistatic constraint in an RNA virus due to host range mutations themselves, which has implications for models of serial host range expansion.ImportanceRNA viruses mutate rapidly and frequently expand their host ranges to infect novel hosts, leading to serial host shifts. Using an RNA bacteriophage model system (Pseudomonas phage phi6), we studied the impact of pre-existing host range mutations on another host range expansion. Results from both clonal Sanger and Illumina sequencing show extant host range mutations dramatically narrow the neighborhood of potential host range mutations compared to wildtype phi6. This research suggests that serial host shifting viruses may follow a small number of molecular paths to enter additional novel hosts. We also identified new genes involved in phi6 host range expansion, expanding our knowledge of this important model system in experimental evolution.&#39;,
  &#39;publish_time&#39;: &#39;2018-08-17&#39;,
  &#39;authors&#39;: &#39;Lele Zhao; Mansha Seth Pasricha; Dragos Stemate; Alvin Crespo-Bellido; Jacqueline Gagnon; Siobain Duffy&#39;,
  &#39;journal&#39;: &#39;&#39;,
  &#39;Microsoft Academic Paper ID&#39;: &#39;&#39;,
  &#39;WHO #Covidence&#39;: &#39;&#39;,
  &#39;has_full_text&#39;: &#39;True&#39;,
  &#39;full_text_file&#39;: &#39;biorxiv_medrxiv&#39;,
  &#39;url&#39;: &#39;https://doi.org/10.1101/394080&#39;},
 ...]</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[10]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># length of the json_lines</span>
<span class="nb">len</span><span class="p">(</span><span class="n">json_lines</span><span class="p">)</span>

<span class="c1"># 45774 journals have been extracted</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[10]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>45774</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>How do I traverse on JSON file read by <strong>Method 1</strong>?</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[60]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Sample operation of json_lines</span>
<span class="c1"># How many journals are there in the file_path?</span>
<span class="c1"># Resolve File Format Exercise Problem 5</span>
<span class="c1"># append files first</span>
<span class="n">journals</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span><span class="nb">len</span><span class="p">(</span><span class="n">json_lines</span><span class="p">)):</span>
    <span class="n">journals</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">json_lines</span><span class="p">[</span><span class="n">i</span><span class="p">][</span><span class="s1">&#39;journal&#39;</span><span class="p">])</span>
    
<span class="c1"># read into a dataframe</span>
<span class="n">df_journals</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">journals</span><span class="p">)</span>
<span class="n">col_name</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;journal_name&#39;</span><span class="p">]</span>
<span class="n">df_journals</span><span class="o">.</span><span class="n">columns</span> <span class="o">=</span> <span class="n">col_name</span>

<span class="c1"># get count of journals by using .size()</span>
<span class="n">df_journal_count</span> <span class="o">=</span> <span class="n">df_journals</span><span class="o">.</span><span class="n">groupby</span><span class="p">(</span><span class="s1">&#39;journal_name&#39;</span><span class="p">)</span><span class="o">.</span><span class="n">size</span><span class="p">()</span>
<span class="n">df_journal_count_sorted</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">df_journal_count</span><span class="p">)</span><span class="o">.</span><span class="n">reset_index</span><span class="p">()</span>
<span class="n">col_names</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;journal_name&#39;</span><span class="p">,</span> <span class="s1">&#39;count&#39;</span><span class="p">]</span>
<span class="n">df_journal_count_sorted</span><span class="o">.</span><span class="n">columns</span> <span class="o">=</span> <span class="n">col_names</span>
<span class="n">df_journal_count_sorted</span> <span class="o">=</span> <span class="n">df_journal_count_sorted</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span>
    <span class="n">by</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;count&#39;</span><span class="p">,</span> <span class="s1">&#39;journal_name&#39;</span><span class="p">],</span> <span class="n">ascending</span> <span class="o">=</span> <span class="p">[</span><span class="kc">False</span><span class="p">,</span> <span class="kc">True</span><span class="p">])</span>

<span class="c1"># final answer</span>
<span class="n">long_cut</span> <span class="o">=</span> <span class="nb">list</span><span class="p">(</span>
    <span class="nb">zip</span><span class="p">(</span>
        <span class="n">df_journal_count_sorted</span><span class="p">[</span><span class="s1">&#39;journal_name&#39;</span><span class="p">],</span>
        <span class="n">df_journal_count_sorted</span><span class="p">[</span><span class="s1">&#39;count&#39;</span><span class="p">]))</span>
<span class="n">long_cut</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[60]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>[(&#39;&#39;, 4067),
 (&#39;Journal of Virology&#39;, 1740),
 (&#39;PLoS One&#39;, 1567),
 (&#39;Virology&#39;, 865),
 (&#39;Emerg Infect Dis&#39;, 749),
 (&#39;The Lancet&#39;, 597),
 (&#39;Viruses&#39;, 569),
 (&#39;Arch Virol&#39;, 504),
 (&#39;Virus Research&#39;, 495),
 (&#39;Sci Rep&#39;, 491),
 (&#39;Vaccine&#39;, 484),
 (&#39;Veterinary Microbiology&#39;, 443),
 (&#39;Journal of Virological Methods&#39;, 393),
 (&#39;Journal of Clinical Virology&#39;, 378),
 (&#39;PLoS Pathog&#39;, 361),
 (&#39;Virol J&#39;, 357),
 (&#39;Antiviral Research&#39;, 348),
 (&#39;The Lancet Infectious Diseases&#39;, 342),
 (&#39;Proceedings of the National Academy of Sciences&#39;, 341),
 (&#39;Journal of Clinical Microbiology&#39;, 335),
 (&#39;International Journal of Infectious Diseases&#39;, 254),
 (&#39;BMC Infect Dis&#39;, 246),
 (&#39;American Journal of Infection Control&#39;, 232),
 (&#39;Front Immunol&#39;, 213),
 (&#39;Front Microbiol&#39;, 203),
 (&#39;Veterinary Immunology and Immunopathology&#39;, 187),
 (&#39;Nucleic Acids Res&#39;, 185),
 (&#39;Journal of Hospital Infection&#39;, 170),
 (&#39;Journal of Neuroimmunology&#39;, 169),
 (&#39;Biochemical and Biophysical Research Communications&#39;, 168),
 (&#39;BMC Vet Res&#39;, 162),
 (&#39;mBio&#39;, 158),
 (&#39;Journal of Biological Chemistry&#39;, 157),
 (&#39;Influenza Other Respir Viruses&#39;, 150),
 (&#39;Clinical Microbiology and Infection&#39;, 149),
 (&#39;PLoS Negl Trop Dis&#39;, 148),
 (&#39;BMC Public Health&#39;, 146),
 (&#39;Journal of Infection&#39;, 146),
 (&#39;Virus Genes&#39;, 136),
 (&#39;Chest&#39;, 135),
 (&#39;Current Opinion in Virology&#39;, 128),
 (&#39;Emerg Microbes Infect&#39;, 121),
 (&#39;Infection, Genetics and Evolution&#39;, 121),
 (&#39;Int J Mol Sci&#39;, 118),
 (&#39;Travel Medicine and Infectious Disease&#39;, 118),
 (&#39;Research in Veterinary Science&#39;, 114),
 (&#39;Journal of Molecular Biology&#39;, 108),
 (&#39;Molecules&#39;, 105),
 (&#39;Journal of Allergy and Clinical Immunology&#39;, 103),
 (&#39;Preventive Veterinary Medicine&#39;, 103),
 (&#39;Clinical and Vaccine Immunology&#39;, 101),
 (&#39;The Veterinary Journal&#39;, 98),
 (&#39;Journal of Feline Medicine &amp; Surgery&#39;, 94),
 (&#39;International Journal of Antimicrobial Agents&#39;, 92),
 (&#39;Clinical Infectious Diseases&#39;, 90),
 (&#39;Comparative Immunology, Microbiology and Infectious Diseases&#39;, 89),
 (&#39;J Cell Biol&#39;, 88),
 (&#39;Journal of Infection and Public Health&#39;, 87),
 (&#39;Journal of Comparative Pathology&#39;, 85),
 (&#39;Annals of Emergency Medicine&#39;, 84),
 (&#39;Crit Care&#39;, 84),
 (&#39;Journal of Dairy Science&#39;, 84),
 (&#39;Vet Res&#39;, 84),
 (&#39;FEBS Letters&#39;, 83),
 (&#39;New Scientist&#39;, 83),
 (&#39;Trends in Microbiology&#39;, 83),
 (&#39;Cell&#39;, 82),
 (&#39;European Journal of Medicinal Chemistry&#39;, 81),
 (&#39;Advances in Virus Research&#39;, 80),
 (&#39;American Journal of Public Health&#39;, 79),
 (&#39;Antimicrobial Agents and Chemotherapy&#39;, 79),
 (&#39;Bioorganic &amp; Medicinal Chemistry&#39;, 78),
 (&#39;Building and Environment&#39;, 78),
 (&#39;Infectious Disease Clinics of North America&#39;, 77),
 (&#39;Journal of General Virology&#39;, 77),
 (&#39;Biomed Res Int&#39;, 76),
 (&#39;Int J Environ Res Public Health&#39;, 76),
 (&#39;PLoS Med&#39;, 76),
 (&#39;BMC Genomics&#39;, 73),
 (&#39;Médecine et Maladies Infectieuses&#39;, 73),
 (&#39;Veterinary Clinics of North America: Small Animal Practice&#39;, 73),
 (&#39;Nat Commun&#39;, 72),
 (&#39;Open Forum Infect Dis&#39;, 72),
 (&#39;Canadian Medical Association Journal&#39;, 70),
 (&#39;The Journal of Immunology&#39;, 70),
 (&#39;BMC Bioinformatics&#39;, 69),
 (&#39;Public Health&#39;, 69),
 (&#39;Nature&#39;, 68),
 (&#39;Diagnostic Microbiology and Infectious Disease&#39;, 65),
 (&#39;Eur J Clin Microbiol Infect Dis&#39;, 65),
 (&#39;J Vet Med Sci&#39;, 65),
 (&#39;Journal of Theoretical Biology&#39;, 65),
 (&#39;BMJ&#39;, 64),
 (&#39;Microbes and Infection&#39;, 64),
 (&#39;Veterinary Parasitology&#39;, 64),
 (&#39;Virol Sin&#39;, 64),
 (&#39;Journal of Medical Virology&#39;, 63),
 (&#39;Veterinary Clinics of North America: Food Animal Practice&#39;, 61),
 (&#39;Journal of Immunological Methods&#39;, 60),
 (&#39;Molecular Therapy&#39;, 60),
 (&#39;Genome Announc&#39;, 58),
 (&#39;Cell Host &amp; Microbe&#39;, 57),
 (&#39;Enfermedades Infecciosas y Microbiología Clínica&#39;, 57),
 (&#39;Human Vaccines &amp; Immunotherapeutics&#39;, 57),
 (&#39;J Exp Med&#39;, 57),
 (&#39;J Virol&#39;, 57),
 (&#39;Bioorganic &amp; Medicinal Chemistry Letters&#39;, 56),
 (&#39;The Journal of Infectious Diseases&#39;, 56),
 (&#39;The Lancet Respiratory Medicine&#39;, 56),
 (&#39;J Vet Intern Med&#39;, 55),
 (&#39;Archives de Pédiatrie&#39;, 54),
 (&#39;Biotechnology Advances&#39;, 54),
 (&#39;Microbial Pathogenesis&#39;, 54),
 (&#39;Gastroenterology&#39;, 53),
 (&#39;Oncotarget&#39;, 53),
 (&#39;BMJ Open&#39;, 52),
 (&#39;Bulletin of the World Health Organization&#39;, 52),
 (&#39;PLoS Comput Biol&#39;, 51),
 (&#39;Immunology&#39;, 49),
 (&#39;J Korean Med Sci&#39;, 49),
 (&#39;The American Journal of Pathology&#39;, 49),
 (&#39;Ecohealth&#39;, 48),
 (&#39;Encyclopedia of Virology&#39;, 48),
 (&#39;BMC Res Notes&#39;, 47),
 (&#39;Clinical Microbiology Reviews&#39;, 47),
 (&#39;Molecular Immunology&#39;, 46),
 (&#39;Tourism Management&#39;, 46),
 (&#39;Veterinary Clinics of North America: Exotic Animal Practice&#39;, 46),
 (&#39;Biophysical Journal&#39;, 45),
 (&#39;Euro Surveill&#39;, 45),
 (&#39;Front Vet Sci&#39;, 45),
 (&#39;RNA&#39;, 45),
 (&#39;International Encyclopedia of Public Health&#39;, 44),
 (&#39;Journal of The Royal Society Interface&#39;, 44),
 (&#39;Medicine - Programa de Formación Médica Continuada Acreditado&#39;, 44),
 (&#39;Science&#39;, 44),
 (&#39;Biology of Blood and Marrow Transplantation&#39;, 43),
 (&#39;Journal of Clinical Investigation&#39;, 43),
 (&#39;Medicine (Baltimore)&#39;, 43),
 (&#39;The American Journal of Tropical Medicine and Hygiene&#39;, 43),
 (&#39;Evid Based Complement Alternat Med&#39;, 42),
 (&#39;Intensive Care Med&#39;, 42),
 (&#39;J Biomed Sci&#39;, 42),
 (&#39;Journal of Ethnopharmacology&#39;, 42),
 (&#39;PeerJ&#39;, 42),
 (&#39;Respir Res&#39;, 42),
 (&#39;The Journal of Pediatrics&#39;, 42),
 (&#39;Thorax&#39;, 42),
 (&#39;Applied and Environmental Microbiology&#39;, 41),
 (&#39;Biosensors and Bioelectronics&#39;, 41),
 (&#39;Cytokine&#39;, 41),
 (&#39;Drug Discovery Today&#39;, 41),
 (&#39;Social Science &amp; Medicine&#39;, 41),
 (&#39;The American Journal of Medicine&#39;, 41),
 (&#39;World Journal of Gastroenterology&#39;, 41),
 (&#39;Current Opinion in Immunology&#39;, 40),
 (&#39;Immunology Letters&#39;, 40),
 (&#39;Physica A: Statistical Mechanics and its Applications&#39;, 40),
 (&#39;Appl Microbiol Biotechnol&#39;, 39),
 (&#39;PLoS Biol&#39;, 39),
 (&#39;Protein Science&#39;, 39),
 (&#39;Clinics in Chest Medicine&#39;, 38),
 (&#39;Current Opinion in Biotechnology&#39;, 38),
 (&#39;Journal of the Formosan Medical Association&#39;, 38),
 (&#39;New England Journal of Medicine&#39;, 38),
 (&#39;Revue des Maladies Respiratoires&#39;, 38),
 (&#39;Sci China Life Sci&#39;, 38),
 (&#39;Chin Sci Bull&#39;, 37),
 (&#39;EMBO reports&#39;, 37),
 (&#39;Advanced Drug Delivery Reviews&#39;, 36),
 (&#39;Molecular and Cellular Probes&#39;, 36),
 (&#39;Philosophical Transactions of the Royal Society B: Biological Sciences&#39;,
  36),
 (&#39;mSphere&#39;, 36),
 (&#39;Clinical Microbiology Newsletter&#39;, 35),
 (&#39;Current Opinion in Structural Biology&#39;, 35),
 (&#39;Journal of Infectious Diseases&#39;, 35),
 (&#39;Journal of Microbiology, Immunology and Infection&#39;, 35),
 (&#39;Trop Anim Health Prod&#39;, 35),
 (&#39;Clinical Immunology&#39;, 34),
 (&#39;Front Cell Infect Microbiol&#39;, 34),
 (&#39;J Neuroinflammation&#39;, 34),
 (&#39;JAMA&#39;, 34),
 (&#39;Paediatric Respiratory Reviews&#39;, 34),
 (&#39;Revue Francophone des Laboratoires&#39;, 34),
 (&#39;Acta Vet Scand&#39;, 33),
 (&#39;Current Opinion in Microbiology&#39;, 33),
 (&#39;Immunity&#39;, 33),
 (&#39;Vaccines (Basel)&#39;, 33),
 (&#34;Annales de l&#39;Institut Pasteur / Virologie&#34;, 32),
 (&#39;Epidemiology and Infection&#39;, 32),
 (&#39;Front Public Health&#39;, 32),
 (&#39;Anales de Pediatría&#39;, 31),
 (&#39;Developmental &amp; Comparative Immunology&#39;, 31),
 (&#39;Environ Health Perspect&#39;, 31),
 (&#39;Journal of Epidemiology &amp; Community Health&#39;, 31),
 (&#39;Laboratory Animal Medicine&#39;, 31),
 (&#39;Revue Française des Laboratoires&#39;, 31),
 (&#39;The EMBO Journal&#39;, 31),
 (&#39;BMC Health Serv Res&#39;, 30),
 (&#39;Biochimica et Biophysica Acta (BBA) - Biomembranes&#39;, 30),
 (&#39;Gene&#39;, 30),
 (&#39;Infectious Diseases&#39;, 30),
 (&#39;J Vet Sci&#39;, 30),
 (&#39;Medicine&#39;, 30),
 (&#39;Virus Evol&#39;, 30),
 (&#39;BMC Microbiol&#39;, 29),
 (&#39;Clinics in Laboratory Medicine&#39;, 29),
 (&#39;International Immunopharmacology&#39;, 29),
 (&#39;Medical Hypotheses&#39;, 29),
 (&#39;One Health&#39;, 29),
 (&#39;Pharmacology &amp; Therapeutics&#39;, 29),
 (&#39;Retrovirology&#39;, 29),
 (&#39;Trends in Immunology&#39;, 29),
 (&#39;Virologica Sinica&#39;, 29),
 (&#39;Cell Mol Life Sci&#39;, 28),
 (&#39;Epidemiol Health&#39;, 28),
 (&#39;Infection and Immunity&#39;, 28),
 (&#39;Journal of Infection and Chemotherapy&#39;, 28),
 (&#39;Option/Bio&#39;, 28),
 (&#39;Pathogens&#39;, 28),
 (&#39;Vet World&#39;, 28),
 (&#39;Biochemistry&#39;, 27),
 (&#39;British Veterinary Journal&#39;, 27),
 (&#39;Bull World Health Organ&#39;, 27),
 (&#39;Epidemiol Infect&#39;, 27),
 (&#39;Eur J Pediatr&#39;, 27),
 (&#39;Journal of Cystic Fibrosis&#39;, 27),
 (&#39;Parasit Vectors&#39;, 27),
 (&#39;Proceedings of the Royal Society B: Biological Sciences&#39;, 27),
 (&#39;Viral Immunology&#39;, 27),
 (&#39;American Journal of Respiratory and Critical Care Medicine&#39;, 26),
 (&#39;Blood&#39;, 26),
 (&#39;Cells&#39;, 26),
 (&#39;Curr Infect Dis Rep&#39;, 26),
 (&#39;Future Virology&#39;, 26),
 (&#39;Infect Chemother&#39;, 26),
 (&#39;Infect Dis Poverty&#39;, 26),
 (&#39;J Immunol Res&#39;, 26),
 (&#39;J Infect Dis&#39;, 26),
 (&#34;Mandell, Douglas, and Bennett&#39;s Principles and Practice of Infectious Diseases&#34;,
  26),
 (&#39;Online J Public Health Inform&#39;, 26),
 (&#39;Principles and Practice of Pediatric Infectious Diseases&#39;, 26),
 (&#39;Respiratory Medicine&#39;, 26),
 (&#39;Science of The Total Environment&#39;, 26),
 (&#39;The Pediatric Infectious Disease Journal&#39;, 26),
 (&#39;American Journal of Physiology-Lung Cellular and Molecular Physiology&#39;, 25),
 (&#39;Archivos de Bronconeumología&#39;, 25),
 (&#39;Clinical Medicine&#39;, 25),
 (&#39;F1000Res&#39;, 25),
 (&#39;Perspectives in Medical Virology&#39;, 25),
 (&#39;Protein Expression and Purification&#39;, 25),
 (&#39;The Journal of Emergency Medicine&#39;, 25),
 (&#39;Vet Res Commun&#39;, 25),
 (&#39;Zhonghua Jie He He Hu Xi Za Zhi&#39;, 25),
 (&#39;AORN Journal&#39;, 24),
 (&#39;BMC Biotechnol&#39;, 24),
 (&#39;Cell Death Dis&#39;, 24),
 (&#39;Chin Med J (Engl)&#39;, 24),
 (&#39;Hum Vaccin Immunother&#39;, 24),
 (&#39;Immunobiology&#39;, 24),
 (&#39;J Gen Virol&#39;, 24),
 (&#39;J R Soc Interface&#39;, 24),
 (&#39;Reference Module in Biomedical Sciences&#39;, 24),
 (&#39;ScientificWorldJournal&#39;, 24),
 (&#39;The Lancet Global Health&#39;, 24),
 (&#39;Trends in Biotechnology&#39;, 24),
 (&#39;Am J Trop Med Hyg&#39;, 23),
 (&#39;BMC Med&#39;, 23),
 (&#39;BMC Pulm Med&#39;, 23),
 (&#39;Disease-a-Month&#39;, 23),
 (&#39;FEMS Microbiology Letters&#39;, 23),
 (&#39;Global Health&#39;, 23),
 (&#39;International Journal of Biological Macromolecules&#39;, 23),
 (&#39;Journal of Aerosol Science&#39;, 23),
 (&#39;Journal of Biotechnology&#39;, 23),
 (&#39;Journal of Thoracic Disease&#39;, 23),
 (&#39;MMWR Morb Mortal Wkly Rep&#39;, 23),
 (&#39;Mayo Clinic Proceedings&#39;, 23),
 (&#39;Sensors and Actuators B: Chemical&#39;, 23),
 (&#39;VirusDisease&#39;, 23),
 (&#39;BMJ Glob Health&#39;, 22),
 (&#39;Biochimie&#39;, 22),
 (&#39;Clinical &amp; Experimental Immunology&#39;, 22),
 (&#39;Cytokine &amp; Growth Factor Reviews&#39;, 22),
 (&#39;Fish &amp; Shellfish Immunology&#39;, 22),
 (&#39;International Journal of Hospitality Management&#39;, 22),
 (&#39;J Biomed Biotechnol&#39;, 22),
 (&#39;J Med Internet Res&#39;, 22),
 (&#39;Mathematical Biosciences&#39;, 22),
 (&#39;Mediators Inflamm&#39;, 22),
 (&#39;Microbiology and Molecular Biology Reviews&#39;, 22),
 (&#39;Radiology&#39;, 22),
 (&#39;Tetrahedron&#39;, 22),
 (&#39;The Journal of Molecular Diagnostics&#39;, 22),
 (&#39;Theriogenology&#39;, 22),
 (&#39;Clin Infect Dis&#39;, 21),
 (&#39;Critical Care Clinics&#39;, 21),
 (&#39;Genomics Proteomics Bioinformatics&#39;, 21),
 (&#39;JFMS Open Rep&#39;, 21),
 (&#39;Lab Anim (NY)&#39;, 21),
 (&#39;Medicina Clínica&#39;, 21),
 (&#39;Molecular &amp; Cellular Proteomics&#39;, 21),
 (&#39;Mucosal Immunology&#39;, 21),
 (&#39;Nature Reviews Microbiology&#39;, 21),
 (&#39;Osong Public Health Res Perspect&#39;, 21),
 (&#39;Problem-Based Feline Medicine&#39;, 21),
 (&#39;Saudi Med J&#39;, 21),
 (&#39;Structure&#39;, 21),
 (&#39;Travel Medicine&#39;, 21),
 (&#39;Virulence&#39;, 21),
 (&#39;Adv Virol&#39;, 20),
 (&#39;American Journal of Kidney Diseases&#39;, 20),
 (&#39;Biochemical Pharmacology&#39;, 20),
 (&#39;Biomaterials&#39;, 20),
 (&#39;Handbook of Proteolytic Enzymes&#39;, 20),
 (&#39;Health Policy&#39;, 20),
 (&#39;Human Immunology&#39;, 20),
 (&#39;International Journal of Nursing Studies&#39;, 20),
 (&#39;Journal of Exotic Pet Medicine&#39;, 20),
 (&#39;Microbiol Resour Announc&#39;, 20),
 (&#39;PLoS Curr&#39;, 20),
 (&#39;Pathology&#39;, 20),
 (&#39;Pediatric Clinics of North America&#39;, 20),
 (&#39;Peptides&#39;, 20),
 (&#39;American Journal of Respiratory Cell and Molecular Biology&#39;, 19),
 (&#39;Annual Reports in Medicinal Chemistry&#39;, 19),
 (&#39;Autophagy&#39;, 19),
 (&#39;Biochimica et Biophysica Acta (BBA) - Molecular Cell Research&#39;, 19),
 (&#39;Biologicals&#39;, 19),
 (&#34;Bulletin de l&#39;Académie Nationale de Médecine&#34;, 19),
 (&#39;Cell Reports&#39;, 19),
 (&#39;Cellular &amp; Molecular Immunology&#39;, 19),
 (&#39;Epidemics&#39;, 19),
 (&#39;Ferrets, Rabbits, and Rodents&#39;, 19),
 (&#39;Genome Biol&#39;, 19),
 (&#39;Journal of Air Transport Management&#39;, 19),
 (&#39;Journal of Controlled Release&#39;, 19),
 (&#39;Journal of Emergency Nursing&#39;, 19),
 (&#39;Journal of Medicinal Chemistry&#39;, 19),
 (&#39;Medical Clinics of North America&#39;, 19),
 (&#39;PLoS Genet&#39;, 19),
 (&#39;Pediatr Res&#39;, 19),
 (&#39;Protein &amp; Cell&#39;, 19),
 (&#39;Sensors (Basel)&#39;, 19),
 (&#39;The American Journal of Emergency Medicine&#39;, 19),
 (&#39;Trends in Parasitology&#39;, 19),
 (&#39;Veterinary Clinics of North America: Equine Practice&#39;, 19),
 (&#39;Acta Neuropathol&#39;, 18),
 (&#39;Atmospheric Environment&#39;, 18),
 (&#39;Cell Rep&#39;, 18),
 (&#39;Emergency Medicine Clinics of North America&#39;, 18),
 (&#39;Encyclopedia of Environmental Health&#39;, 18),
 (&#39;Exp Ther Med&#39;, 18),
 (&#39;Front Pharmacol&#39;, 18),
 (&#39;Gene Ther&#39;, 18),
 (&#39;Genes (Basel)&#39;, 18),
 (&#39;Infect Ecol Epidemiol&#39;, 18),
 (&#39;Infection Control &amp; Hospital Epidemiology&#39;, 18),
 (&#39;Journal of Critical Care&#39;, 18),
 (&#39;La Presse Médicale&#39;, 18),
 (&#39;Microbial Forensics&#39;, 18),
 (&#39;Mol Biotechnol&#39;, 18),
 (&#39;Seminars in Pediatric Infectious Diseases&#39;, 18),
 (&#39;Tetrahedron Letters&#39;, 18),
 (&#39;Transfusion Medicine Reviews&#39;, 18),
 (&#39;Acta Crystallographica Section F Structural Biology and Crystallization Communications&#39;,
  17),
 (&#39;BMC Complement Altern Med&#39;, 17),
 (&#39;Braz J Microbiol&#39;, 17),
 (&#39;Cell Res&#39;, 17),
 (&#39;Clin Exp Vaccine Res&#39;, 17),
 (&#39;Cochrane Database of Systematic Reviews&#39;, 17),
 (&#39;Comput Math Methods Med&#39;, 17),
 (&#39;Current Biology&#39;, 17),
 (&#39;Current Opinion in Cell Biology&#39;, 17),
 (&#39;Current Opinion in Chemical Biology&#39;, 17),
 (&#39;Expert Review of Vaccines&#39;, 17),
 (&#39;Immunology and Allergy Clinics of North America&#39;, 17),
 (&#39;Indian J Med Res&#39;, 17),
 (&#39;Int J Nanomedicine&#39;, 17),
 (&#39;J Clin Med&#39;, 17),
 (&#39;Journal of Hazardous Materials&#39;, 17),
 (&#39;Journal of Travel Medicine&#39;, 17),
 (&#39;Korean J Pediatr&#39;, 17),
 (&#39;Life Sciences&#39;, 17),
 (&#39;Mar Drugs&#39;, 17),
 (&#39;Med Microbiol Immunol&#39;, 17),
 (&#39;Microb Cell Fact&#39;, 17),
 (&#39;Philos Trans R Soc Lond B Biol Sci&#39;, 17),
 (&#39;Poultry Science&#39;, 17),
 (&#39;Trends in Cell Biology&#39;, 17),
 (&#39;Trends in Molecular Medicine&#39;, 17),
 (&#39;American Journal of Preventive Medicine&#39;, 16),
 (&#39;Amino Acids&#39;, 16),
 (&#39;Ann Lab Med&#39;, 16),
 (&#39;Antimicrob Resist Infect Control&#39;, 16),
 (&#39;BMC Immunol&#39;, 16),
 (&#39;BMC Pediatr&#39;, 16),
 (&#39;Biomolecules&#39;, 16),
 (&#39;Canadian Journal of Public Health&#39;, 16),
 (&#39;Clinics in Perinatology&#39;, 16),
 (&#39;Encyclopedia of Microbiology&#39;, 16),
 (&#34;Fenner&#39;s Veterinary Virology&#34;, 16),
 (&#39;Food Environ Virol&#39;, 16),
 (&#39;Int J Chron Obstruct Pulmon Dis&#39;, 16),
 (&#39;J Microbiol&#39;, 16),
 (&#39;JMIR Public Health Surveill&#39;, 16),
 (&#39;Journal of Interferon &amp; Cytokine Research&#39;, 16),
 (&#39;Journal of Leukocyte Biology&#39;, 16),
 (&#39;Methods&#39;, 16),
 (&#39;Parasitol Res&#39;, 16),
 (&#39;Primary Care: Clinics in Office Practice&#39;, 16),
 (&#39;Advances in Immunology&#39;, 15),
 (&#39;Analytical Biochemistry&#39;, 15),
 (&#39;Biotechnol Lett&#39;, 15),
 (&#39;Bone Marrow Transplant&#39;, 15),
 (&#39;Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz&#39;, 15),
 (&#39;Canada Communicable Disease Report&#39;, 15),
 (&#39;Canine and Feline Infectious Diseases&#39;, 15),
 (&#39;Cellular Immunology&#39;, 15),
 (&#39;Chinese Journal of Laboratory Medicine&#39;, 15),
 (&#39;Clin Dev Immunol&#39;, 15),
 (&#39;Comp Clin Path&#39;, 15),
 (&#39;Dig Dis Sci&#39;, 15),
 (&#39;Disaster Medicine&#39;, 15),
 (&#39;Emerging Infectious Diseases&#39;, 15),
 (&#39;Emerging and Reemerging Viral Pathogens&#39;, 15),
 (&#39;Eur&#39;, 15),
 (&#39;Front Genet&#39;, 15),
 (&#39;Front Pediatr&#39;, 15),
 (&#39;Front Plant Sci&#39;, 15),
 (&#39;Genetics&#39;, 15),
 (&#39;Glob Health Action&#39;, 15),
 (&#39;Habitat International&#39;, 15),
 (&#39;Immunological Reviews&#39;, 15),
 (&#39;Indian J Pediatr&#39;, 15),
 (&#39;Journal of Veterinary Diagnostic Investigation&#39;, 15),
 (&#39;Journal of the Chinese Medical Association&#39;, 15),
 (&#39;Nature Medicine&#39;, 15),
 (&#39;Nature Reviews Immunology&#39;, 15),
 (&#39;Proc Natl Acad Sci U S A&#39;, 15),
 (&#39;Protein Cell&#39;, 15),
 (&#39;Research in Immunology&#39;, 15),
 (&#39;Science Translational Medicine&#39;, 15),
 (&#39;Springerplus&#39;, 15),
 (&#39;Toxicology and Applied Pharmacology&#39;, 15),
 (&#39;Vet Sci&#39;, 15),
 (&#39;Viral and Mycoplasmal of Laboratory Rodents&#39;, 15),
 (&#39;Yale J Biol Med&#39;, 15),
 (&#39;Acta Tropica&#39;, 14),
 (&#39;American Journal of Epidemiology&#39;, 14),
 (&#39;Animals (Basel)&#39;, 14),
 (&#39;Ann Thorac Med&#39;, 14),
 (&#39;Biochemical Journal&#39;, 14),
 (&#39;Biochimica et Biophysica Acta (BBA) - General Subjects&#39;, 14),
 (&#39;Biomedical and Environmental Sciences&#39;, 14),
 (&#39;Biomedicine &amp; Pharmacotherapy&#39;, 14),
 (&#39;Brain, Behavior, and Immunity&#39;, 14),
 (&#39;Clinical Veterinary Advisor&#39;, 14),
 (&#39;Disaster Management &amp; Response&#39;, 14),
 (&#39;Drug Des Devel Ther&#39;, 14),
 (&#39;Emerg Health Threats J&#39;, 14),
 (&#39;Emergency Medicine Journal&#39;, 14),
 (&#39;Experimental and Therapeutic Medicine&#39;, 14),
 (&#39;FMC - Formación Médica Continuada en Atención Primaria&#39;, 14),
 (&#39;Handbook of Clinical Neurology&#39;, 14),
 (&#39;Health Secur&#39;, 14),
 (&#39;Hepatology&#39;, 14),
 (&#39;Human Pathology&#39;, 14),
 (&#39;International Journal for Parasitology&#39;, 14),
 (&#39;J Mol Med (Berl)&#39;, 14),
 (&#39;J Prev Med Public Health&#39;, 14),
 (&#39;Journal of Hepatology&#39;, 14),
 (&#39;Journal of Obstetrics and Gynaecology Canada&#39;, 14),
 (&#39;Journal of Structural Biology&#39;, 14),
 (&#39;Krankenhaus-Hygiene + Infektionsverhütung&#39;, 14),
 (&#39;Mol Med Rep&#39;, 14),
 (&#39;Molecular Cell&#39;, 14),
 (&#39;Nursing Outlook&#39;, 14),
 (&#39;Porcine Health Manag&#39;, 14),
 (&#39;R Soc Open Sci&#39;, 14),
 (&#34;Revue Française d&#39;Allergologie et d&#39;Immunologie Clinique&#34;, 14),
 (&#39;The Brazilian Journal of Infectious Diseases&#39;, 14),
 (&#39;The Journal of the American Dental Association&#39;, 14),
 (&#39;TrAC Trends in Analytical Chemistry&#39;, 14),
 (&#39;Transgenic Res&#39;, 14),
 (&#39;Trop Med Infect Dis&#39;, 14),
 (&#39;Viral Pathogenesis&#39;, 14),
 (&#39;Zhonghua Yu Fang Yi Xue Za Zhi&#39;, 14),
 (&#39;Archives of Disease in Childhood&#39;, 13),
 (&#39;Asian Pacific Journal of Tropical Medicine&#39;, 13),
 (&#39;BMC Evol Biol&#39;, 13),
 (&#39;Biochimica et Biophysica Acta (BBA) - Molecular Basis of Disease&#39;, 13),
 (&#39;Biosci Rep&#39;, 13),
 (&#39;British Journal of Anaesthesia&#39;, 13),
 (&#39;Chaos, Solitons &amp; Fractals&#39;, 13),
 (&#34;Ciottone&#39;s Disaster Medicine&#34;, 13),
 (&#39;Critical Care Medicine&#39;, 13),
 (&#39;Curr Allergy Asthma Rep&#39;, 13),
 (&#39;Eurosurveillance&#39;, 13),
 (&#34;Fowler&#39;s Zoo and Wild Animal Medicine, Volume 8&#34;, 13),
 (&#39;Future Virol&#39;, 13),
 (&#39;Handbook of Biosurveillance&#39;, 13),
 (&#39;Heart &amp; Lung&#39;, 13),
 (&#39;Immunol Res&#39;, 13),
 (&#39;Influenza and Other Respiratory Viruses&#39;, 13),
 (&#39;Int J Health Geogr&#39;, 13),
 (&#39;Intensive Care Medicine&#39;, 13),
 (&#39;J Clin Immunol&#39;, 13),
 (&#39;Journal of Chromatography B&#39;, 13),
 (&#39;Journal of Epidemiology and Global Health&#39;, 13),
 (&#39;Journal of the Neurological Sciences&#39;, 13),
 (&#39;Med Sci Monit&#39;, 13),
 (&#39;Methods in Enzymology&#39;, 13),
 (&#39;Open Virol J&#39;, 13),
 (&#39;Seminars in Immunology&#39;, 13),
 (&#39;The Journal of Heart and Lung Transplantation&#39;, 13),
 (&#39;Trends in Biochemical Sciences&#39;, 13),
 (&#39;Tropical Infectious Diseases: Principles, Pathogens and Practice&#39;, 13),
 (&#39;Ultramicroscopy&#39;, 13),
 (&#39;Western Pacific Surveillance and Response Journal&#39;, 13),
 (&#39;Analytica Chimica Acta&#39;, 12),
 (&#39;Annals of Allergy, Asthma &amp; Immunology&#39;, 12),
 (&#39;Annals of Translational Medicine&#39;, 12),
 (&#39;Biosafety and Health&#39;, 12),
 (&#39;Biosecurity and Bioterrorism: Biodefense Strategy, Practice, and Science&#39;,
  12),
 (&#39;C&amp;EN Global Enterprise&#39;, 12),
 (&#39;Canine and Feline Cytology&#39;, 12),
 (&#39;Chemistry &amp; Biology&#39;, 12),
 (&#39;Chinese Journal of Natural Medicines&#39;, 12),
 (&#39;Clinica Chimica Acta&#39;, 12),
 (&#39;Current Opinion in Pharmacology&#39;, 12),
 (&#39;G3 (Bethesda)&#39;, 12),
 (&#39;Genome Research&#39;, 12),
 (&#39;Glia&#39;, 12),
 (&#39;Immunogenetics&#39;, 12),
 (&#39;International Congress Series&#39;, 12),
 (&#39;J Clin Microbiol&#39;, 12),
 (&#39;Journal of Autoimmunity&#39;, 12),
 (&#39;Journal of General Internal Medicine&#39;, 12),
 (&#39;Journal of Traditional Chinese Medicine&#39;, 12),
 (&#39;Journal of Visualized Experiments&#39;, 12),
 (&#34;Kendig&#39;s Disorders of the Respiratory Tract in Children&#34;, 12),
 (&#39;Mamm Genome&#39;, 12),
 (&#39;Mol Biol Rep&#39;, 12),
 (&#39;Molecular and Cellular Biology&#39;, 12),
 (&#39;NPJ Vaccines&#39;, 12),
 (&#39;Pathology of Wildlife and Zoo Animals&#39;, 12),
 (&#39;Pharmaceuticals (Basel)&#39;, 12),
 (&#39;Phytomedicine&#39;, 12),
 (&#39;Procedia in Vaccinology&#39;, 12),
 (&#39;Progress in Neuro-Psychopharmacology and Biological Psychiatry&#39;, 12),
 (&#39;Resuscitation&#39;, 12),
 (&#39;Saunders Manual of Small Animal Practice&#39;, 12),
 (&#39;Small Animal Critical Care Medicine&#39;, 12),
 (&#39;Studies in Natural Products Chemistry&#39;, 12),
 (&#39;The American Journal of the Medical Sciences&#39;, 12),
 (&#39;Trends in Genetics&#39;, 12),
 (&#39;Virus as Populations&#39;, 12),
 (&#39;Yonsei Med J&#39;, 12),
 (&#39;Zhonghua Liu Xing Bing Xue Za Zhi&#39;, 12),
 (&#39;Acta Pharmacologica Sinica&#39;, 11),
 (&#39;Advances in Parasitology&#39;, 11),
 (&#39;Advances in Veterinary Medicine&#39;, 11),
 (&#39;Annals of Tourism Research&#39;, 11),
 (&#39;Antibiotiques&#39;, 11),
 (&#39;Applied Mathematics and Computation&#39;, 11),
 (&#39;Archives of Biochemistry and Biophysics&#39;, 11),
 (&#39;Cancer Letters&#39;, 11),
 (&#39;Chin J Integr Med&#39;, 11),
 (&#39;Chinese Journal of Hospital Administration&#39;, 11),
 (&#39;Chinese Journal of Medical Science Research Management&#39;, 11),
 (&#39;Clinical Radiology&#39;, 11),
 (&#39;Clinical and Diagnostic Virology&#39;, 11),
 (&#39;EMC - Pediatría&#39;, 11),
 (&#39;Equine Infectious Diseases&#39;, 11),
 (&#39;Feline Internal Medicine Secrets&#39;, 11),
 (&#39;Handbook of Small Animal Practice&#39;, 11),
 (&#39;Health Security&#39;, 11),
 (&#39;Healthcare Management Forum&#39;, 11),
 (&#39;Infect Drug Resist&#39;, 11),
 (&#39;Int J Behav Med&#39;, 11),
 (&#39;International Journal of Pediatric Otorhinolaryngology&#39;, 11),
 (&#39;J Cell Mol Med&#39;, 11),
 (&#39;JCI Insight&#39;, 11),
 (&#39;Journal de Pédiatrie et de Puériculture&#39;, 11),
 (&#39;Journal of Clinical Laboratory Analysis&#39;, 11),
 (&#39;Journal of Equine Veterinary Science&#39;, 11),
 (&#39;Journal of Infection Prevention&#39;, 11),
 (&#39;Journal of Innate Immunity&#39;, 11),
 (&#39;Journal of Pediatric Surgery&#39;, 11),
 (&#39;Journal of Transport Geography&#39;, 11),
 (&#39;Journal of the American Academy of Dermatology&#39;, 11),
 (&#34;Jubb, Kennedy &amp; Palmer&#39;s Pathology of Domestic Animals&#34;, 11),
 (&#39;Medicina Intensiva&#39;, 11),
 (&#39;Molecular Virology of Human Pathogenic Viruses&#39;, 11),
 (&#39;Neurologic Clinics&#39;, 11),
 (&#39;Pathologic Basis of Veterinary Disease&#39;, 11),
 (&#39;Pathologie Biologie&#39;, 11),
 (&#39;Proc Biol Sci&#39;, 11),
 (&#39;Proteome Sci&#39;, 11),
 (&#39;RNA Biology&#39;, 11),
 (&#39;Research in Virology&#39;, 11),
 (&#39;Small Ruminant Research&#39;, 11),
 (&#39;Springer Semin Immunopathol&#39;, 11),
 (&#39;Textbook of Rabbit Medicine&#39;, 11),
 (&#39;The Cat&#39;, 11),
 (&#39;The Journal of Thoracic and Cardiovascular Surgery&#39;, 11),
 (&#39;The Lancet Psychiatry&#39;, 11),
 (&#39;The Lancet Public Health&#39;, 11),
 (&#39;The New Public Health&#39;, 11),
 (&#39;Transactions of the Royal Society of Tropical Medicine and Hygiene&#39;, 11),
 (&#39;Translational Research&#39;, 11),
 (&#39;Trends in Pharmacological Sciences&#39;, 11),
 (&#39;American Journal of Obstetrics and Gynecology&#39;, 10),
 (&#39;Anales de Pediatría (English Edition)&#39;, 10),
 (&#39;Annual Review of Virology&#39;, 10),
 (&#39;Archivos de Bronconeumología ((English Edition))&#39;, 10),
 (&#39;BMC Biol&#39;, 10),
 (&#39;Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms&#39;, 10),
 (&#39;Bull Math Biol&#39;, 10),
 (&#39;Chinese Journal of Emergency Medicine&#39;, 10),
 (&#39;Chinese Journal of Radiology&#39;, 10),
 (&#39;Chinese Medical Journal&#39;, 10),
 (&#39;Clinical Respiratory Medicine&#39;, 10),
 (&#39;Clinical Therapeutics&#39;, 10),
 (&#39;Cold Spring Harbor Perspectives in Biology&#39;, 10),
 (&#39;Consultations in Feline Internal Medicine&#39;, 10),
 (&#39;Diagnostic Pathology of Infectious Disease&#39;, 10),
 (&#39;EBioMedicine&#39;, 10),
 (&#39;Encyclopedia of Agriculture and Food Systems&#39;, 10),
 (&#39;Encyclopedia of Forensic and Legal Medicine&#39;, 10),
 (&#39;Encyclopedia of Immunobiology&#39;, 10),
 (&#39;Encyclopedia of Respiratory Medicine&#39;, 10),
 (&#39;Eur J Epidemiol&#39;, 10),
 (&#39;European Journal of Radiology&#39;, 10),
 (&#39;Expert Systems with Applications&#39;, 10),
 (&#39;Genome Med&#39;, 10),
 (&#39;Glycoconj J&#39;, 10),
 (&#34;Goldman&#39;s Cecil Medicine&#34;, 10),
 (&#39;Indian J Crit Care Med&#39;, 10),
 (&#39;Influenza Res Treat&#39;, 10),
 (&#39;Int J Gen Med&#39;, 10),
 (&#39;International Journal of Cardiology&#39;, 10),
 (&#39;International Review of Cell and Molecular Biology&#39;, 10),
 (&#39;International Review of Cytology&#39;, 10),
 (&#39;Journal of Biomedical Informatics&#39;, 10),
 (&#39;Journal of Chromatography A&#39;, 10),
 (&#39;Journal of Functional Foods&#39;, 10),
 (&#39;Journal of Occupational and Environmental Hygiene&#39;, 10),
 (&#39;Journal of the American College of Radiology&#39;, 10),
 (&#39;Kidney International&#39;, 10),
 (&#39;Mem Inst Oswaldo Cruz&#39;, 10),
 (&#39;Mol Biol Evol&#39;, 10),
 (&#39;Molecular Biology and Evolution&#39;, 10),
 (&#34;Murray and Nadel&#39;s Textbook of Respiratory Medicine&#34;, 10),
 (&#39;Oxid Med Cell Longev&#39;, 10),
 (&#39;Pharm Res&#39;, 10),
 (&#34;Plotkin&#39;s Vaccines&#34;, 10),
 (&#39;Pneumonia (Nathan)&#39;, 10),
 (&#39;Semin Immunopathol&#39;, 10),
 (&#39;Singapore Medical Journal&#39;, 10),
 (&#39;The Biology of Animal Viruses&#39;, 10),
 (&#39;The FASEB Journal&#39;, 10),
 (&#39;The International Journal of Biochemistry &amp; Cell Biology&#39;, 10),
 (&#39;Theranostics&#39;, 10),
 (&#39;Transfusion Clinique et Biologique&#39;, 10),
 (&#39;Trends in Ecology &amp; Evolution&#39;, 10),
 (&#39;Vaccines for Biodefense and Emerging and Neglected Diseases&#39;, 10),
 (&#39;Virus Infections of Ruminants&#39;, 10),
 (&#39;Virus Taxonomy&#39;, 10),
 (&#39;Waste Management&#39;, 10),
 (&#39;Water Research&#39;, 10),
 (&#39;West J Emerg Med&#39;, 10),
 (&#39;Zhonghua Er Ke Za Zhi&#39;, 10),
 (&#39;mAbs&#39;, 10),
 (&#39;Acta Pharm Sin B&#39;, 9),
 (&#39;Acta Pharmacol Sin&#39;, 9),
 (&#39;Advances in Protein Chemistry and Structural Biology&#39;, 9),
 (&#39;Advances in Small Animal Medicine and Surgery&#39;, 9),
 (&#39;Allergologia et Immunopathologia&#39;, 9),
 (&#39;American Journal of Physiology-Regulatory, Integrative and Comparative Physiology&#39;,
  9),
 (&#39;American Journal of Physiology-Renal Physiology&#39;, 9),
 (&#39;American Journal of Roentgenology&#39;, 9),
 (&#39;Anal Bioanal Chem&#39;, 9),
 (&#39;Ann Intern Med&#39;, 9),
 (&#39;Annals of Global Health&#39;, 9),
 (&#39;Aquaculture&#39;, 9),
 (&#39;Asian Pacific Journal of Tropical Biomedicine&#39;, 9),
 (&#39;Autoimmunity Reviews&#39;, 9),
 (&#39;BMC Med Inform Decis Mak&#39;, 9),
 (&#39;Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics&#39;, 9),
 (&#39;Biological Conservation&#39;, 9),
 (&#39;Br Dent J&#39;, 9),
 (&#39;British Journal of Pharmacology&#39;, 9),
 (&#39;Cell Tissue Res&#39;, 9),
 (&#39;Chinese Critical Care Medicine&#39;, 9),
 (&#39;Chinese Journal of Experimental Ophthalmology&#39;, 9),
 (&#39;Chinese Journal of Preventive Medicine&#39;, 9),
 (&#39;Clinics in Sports Medicine&#39;, 9),
 (&#39;Computational Biology and Chemistry&#39;, 9),
 (&#39;Current Opinion in Genetics &amp; Development&#39;, 9),
 (&#39;Cytotechnology&#39;, 9),
 (&#39;Deutsches Aerzteblatt Online&#39;, 9),
 (&#39;EMBO Mol Med&#39;, 9),
 (&#39;Encyclopedia of Dairy Sciences&#39;, 9),
 (&#39;Environ Health&#39;, 9),
 (&#39;Enzyme and Microbial Technology&#39;, 9),
 (&#39;European Journal of Immunology&#39;, 9),
 (&#39;Experimental Cell Research&#39;, 9),
 (&#39;Free Radical Biology and Medicine&#39;, 9),
 (&#39;Front Med&#39;, 9),
 (&#39;General Hospital Psychiatry&#39;, 9),
 (&#39;Genomics&#39;, 9),
 (&#39;Global Health Journal&#39;, 9),
 (&#39;Hypertension&#39;, 9),
 (&#39;Int J Biol Sci&#39;, 9),
 (&#39;International Encyclopedia of Human Geography&#39;, 9),
 (&#39;International Journal of Hygiene and Environmental Health&#39;, 9),
 (&#39;International Journal of Medical Microbiology&#39;, 9),
 (&#39;J Glob Infect Dis&#39;, 9),
 (&#39;Journal of Clinical Medicine&#39;, 9),
 (&#39;Journal of Integrative Agriculture&#39;, 9),
 (&#39;Journal of Neuroscience&#39;, 9),
 (&#39;Journal of Pharmaceutical and Biomedical Analysis&#39;, 9),
 (&#39;Journal of Proteomics&#39;, 9),
 (&#39;Journal of Psychosomatic Research&#39;, 9),
 (&#39;Journal of the American Association for Laboratory Animal Science&#39;, 9),
 (&#39;Journal of the American Medical Informatics Association&#39;, 9),
 (&#39;La Revue de Médecine Interne&#39;, 9),
 (&#39;Livestock Science&#39;, 9),
 (&#39;Malar J&#39;, 9),
 (&#39;Mol Divers&#39;, 9),
 (&#39;Nanomedicine: Nanotechnology, Biology and Medicine&#39;, 9),
 (&#39;Nat Med&#39;, 9),
 (&#39;Nature Microbiology&#39;, 9),
 (&#39;New Microbes New Infect&#39;, 9),
 (&#39;Nutrients&#39;, 9),
 (&#39;Parasitology International&#39;, 9),
 (&#39;Pharmaceutics&#39;, 9),
 (&#39;Pharmacological Research&#39;, 9),
 (&#34;Pratique Médicale et Chirurgicale de l&#39;Animal de Compagnie&#34;, 9),
 (&#39;Principles of Molecular Virology&#39;, 9),
 (&#39;Progress in Molecular Biology and Translational Science&#39;, 9),
 (&#39;Public Health Reports&#39;, 9),
 (&#39;Pulmonary Pharmacology &amp; Therapeutics&#39;, 9),
 (&#39;Reference Module in Life Sciences&#39;, 9),
 (&#39;Revista Clínica Española&#39;, 9),
 (&#39;Réanimation&#39;, 9),
 (&#39;Sci China C Life Sci&#39;, 9),
 (&#39;Sci China Chem&#39;, 9),
 (&#39;Small Animal Clinical Diagnosis by Laboratory Methods&#39;, 9),
 (&#39;Talanta&#39;, 9),
 (&#39;Technological Forecasting and Social Change&#39;, 9),
 (&#39;The Laboratory Rabbit, Guinea Pig, Hamster, and Other Rodents&#39;, 9),
 (&#39;Topics in Companion Animal Medicine&#39;, 9),
 (&#39;Toxicology Letters&#39;, 9),
 (&#39;Tropical Infectious Diseases&#39;, 9),
 (&#39;World Journal of Virology&#39;, 9),
 (&#39;Zentralblatt für Bakteriologie&#39;, 9),
 (&#39;Acta Crystallographica Section F Structural Biology Communications&#39;, 8),
 (&#39;Actualités Pharmaceutiques&#39;, 8),
 (&#39;Anaerobe&#39;, 8),
 (&#39;Animal Reproduction Science&#39;, 8),
 (&#39;Ann Intensive Care&#39;, 8),
 (&#39;Annual Review of Immunology&#39;, 8),
 (&#39;Antivir Chem Chemother&#39;, 8),
 (&#39;Arch Pharm Res&#39;, 8),
 (&#39;Asian Pacific Journal of Tropical Disease&#39;, 8),
 (&#39;Bioengineered&#39;, 8),
 (&#39;Bioinformatics&#39;, 8),
 (&#39;Biotechnol Bioprocess Eng&#39;, 8),
 (&#39;Build Simul&#39;, 8),
 (&#39;Can J Anaesth&#39;, 8),
 (&#39;Cancers (Basel)&#39;, 8),
 (&#39;Canine and Feline Gastroenterology&#39;, 8),
 (&#39;Case Reports&#39;, 8),
 (&#39;Chemical Reviews&#39;, 8),
 (&#39;Chinese Journal of Infectious Diseases&#39;, 8),
 (&#39;Chongqing Medicine&#39;, 8),
 (&#39;Clin Rev Allergy Immunol&#39;, 8),
 (&#39;Curr Microbiol&#39;, 8),
 (&#39;DNA and Cell Biology&#39;, 8),
 (&#39;Emerg Themes Epidemiol&#39;, 8),
 (&#39;Energy and Buildings&#39;, 8),
 (&#39;Ensuring National Biosecurity&#39;, 8),
 (&#39;Environment International&#39;, 8),
 (&#39;Environmental Health and Preventive Medicine&#39;, 8),
 (&#39;Equine Internal Medicine&#39;, 8),
 (&#39;Experimental Biology and Medicine&#39;, 8),
 (&#39;Experimental Neurology&#39;, 8),
 (&#39;Experimental and Molecular Pathology&#39;, 8),
 (&#34;Fenner and White&#39;s Medical Virology&#34;, 8),
 (&#34;Fowler&#39;s Zoo and Wild Animal Medicine Current Therapy, Volume 9&#34;, 8),
 (&#39;Front Bioeng Biotechnol&#39;, 8),
 (&#39;Fundamental and Applied Toxicology&#39;, 8),
 (&#39;Genomic and Personalized Medicine&#39;, 8),
 (&#39;Heliyon&#39;, 8),
 (&#39;Human Gene Therapy&#39;, 8),
 (&#34;Hunter&#39;s Tropical Medicine and Emerging Infectious Diseases&#34;, 8),
 (&#39;Infect Dis Model&#39;, 8),
 (&#39;Int J Equity Health&#39;, 8),
 (&#39;Int J Med Sci&#39;, 8),
 (&#39;International Journal of Medical Informatics&#39;, 8),
 (&#39;Internist (Berl)&#39;, 8),
 (&#39;J Family Med Prim Care&#39;, 8),
 (&#39;J Nanobiotechnology&#39;, 8),
 (&#39;J Transl Med&#39;, 8),
 (&#39;J Vis Exp&#39;, 8),
 (&#39;Journal of Animal Science&#39;, 8),
 (&#39;Journal of Molecular Graphics and Modelling&#39;, 8),
 (&#39;Journal of Pharmacological Sciences&#39;, 8),
 (&#39;Journal of Traditional and Complementary Medicine&#39;, 8),
 (&#39;Journal of the American Medical Directors Association&#39;, 8),
 (&#39;Journal of the Pediatric Infectious Diseases Society&#39;, 8),
 (&#39;Llama and Alpaca Care&#39;, 8),
 (&#34;Manson&#39;s Tropical Infectious Diseases&#34;, 8),
 (&#39;Med Chem Res&#39;, 8),
 (&#39;Medical Journal Armed Forces India&#39;, 8),
 (&#39;Mil Med Res&#39;, 8),
 (&#39;Mol Biol&#39;, 8),
 (&#39;Mol Biol Cell&#39;, 8),
 (&#39;Molecular Medicine&#39;, 8),
 (&#39;Monoclonal Antibodies in Immunodiagnosis and Immunotherapy&#39;, 8),
 (&#39;Nat Protoc&#39;, 8),
 (&#39;Nat Struct Mol Biol&#39;, 8),
 (&#39;Nature Biotechnology&#39;, 8),
 (&#39;Neurochem Res&#39;, 8),
 (&#39;Neuroscience&#39;, 8),
 (&#39;Nurse Education Today&#39;, 8),
 (&#39;Nursing Clinics of North America&#39;, 8),
 (&#39;Pathogens and Disease&#39;, 8),
 (&#39;Postgraduate Medical Journal&#39;, 8),
 (&#39;Procedia Computer Science&#39;, 8),
 (&#39;Psychiatry Research&#39;, 8),
 (&#39;Radiology of Infectious Diseases&#39;, 8),
 (&#34;Rebhun&#39;s Diseases of Dairy Cattle&#34;, 8),
 (&#39;Revista Argentina de Microbiología&#39;, 8),
 (&#34;Revue d&#39;Épidémiologie et de Santé Publique&#34;, 8),
 (&#39;Revue des Maladies Respiratoires Actualités&#39;, 8),
 (&#39;Sci Adv&#39;, 8),
 (&#39;Science Bulletin&#39;, 8),
 (&#39;Scientometrics&#39;, 8),
 (&#39;Small Animal Pediatrics&#39;, 8),
 (&#39;The Journal of Allergy and Clinical Immunology: In Practice&#39;, 8),
 (&#39;The Journal of Urology&#39;, 8),
 (&#39;The Kaohsiung Journal of Medical Sciences&#39;, 8),
 (&#39;The Laboratory Rat&#39;, 8),
 (&#39;The Lancet Gastroenterology &amp; Hepatology&#39;, 8),
 (&#39;The Travel and Tropical Medicine Manual&#39;, 8),
 (&#39;Theor Biol Med Model&#39;, 8),
 (&#39;Tuberc Respir Dis (Seoul)&#39;, 8),
 (&#39;Vector-Borne and Zoonotic Diseases&#39;, 8),
 (&#39;Vet Med Int&#39;, 8),
 (&#39;Viruses and Human Disease&#39;, 8),
 (&#39;Western Pac Surveill Response J&#39;, 8),
 (&#39;Zhonghua Yi Xue Za Zhi&#39;, 8),
 (&#39;ACS Omega&#39;, 7),
 (&#39;American Journal of Ophthalmology&#39;, 7),
 (&#39;American Journal of Transplantation&#39;, 7),
 (&#39;Analytical Chemistry&#39;, 7),
 (&#39;Ann Saudi Med&#39;, 7),
 (&#39;Antimicrob Agents Chemother&#39;, 7),
 (&#39;Arch Gesamte Virusforsch&#39;, 7),
 (&#39;Asthma and COPD&#39;, 7),
 (&#39;BMC Mol Biol&#39;, 7),
 (&#39;BMC Proc&#39;, 7),
 (&#39;Biosecurity and Bioterrorism&#39;, 7),
 (&#39;Bone&#39;, 7),
 (&#39;Breathe (Sheff)&#39;, 7),
 (&#39;Can J Infect Dis Med Microbiol&#39;, 7),
 (&#39;Cell Biochem Biophys&#39;, 7),
 (&#39;Cell Biosci&#39;, 7),
 (&#39;Cellular Signalling&#39;, 7),
 (&#39;Chinese Journal of Epidemiology&#39;, 7),
 (&#39;Chinese Journal of Perinatal Medicine&#39;, 7),
 (&#39;Chinese Journal of Trauma&#39;, 7),
 (&#39;Clinical Pediatric Emergency Medicine&#39;, 7),
 (&#39;Clinical Techniques in Small Animal Practice&#39;, 7),
 (&#39;Computer Methods and Programs in Biomedicine&#39;, 7),
 (&#39;Computers in Human Behavior&#39;, 7),
 (&#39;Drug Discovery Today: Technologies&#39;, 7),
 (&#39;Energy Policy&#39;, 7),
 (&#39;Enfermedades infecciosas y microbiologia clinica (English ed.)&#39;, 7),
 (&#39;Engineering&#39;, 7),
 (&#39;Environmental Research&#39;, 7),
 (&#39;Essential Human Virology&#39;, 7),
 (&#39;European Journal of Internal Medicine&#39;, 7),
 (&#39;Evol Appl&#39;, 7),
 (&#39;Food Animal Practice&#39;, 7),
 (&#39;Front Physiol&#39;, 7),
 (&#39;Genome Biol Evol&#39;, 7),
 (&#39;Haematologica&#39;, 7),
 (&#39;Health Res Policy Syst&#39;, 7),
 (&#39;Healthc Inform Res&#39;, 7),
 (&#39;IDCases&#39;, 7),
 (&#39;Immune Netw&#39;, 7),
 (&#39;Indian Journal of Virology&#39;, 7),
 (&#39;Industrial Crops and Products&#39;, 7),
 (&#39;Int J Mol Med&#39;, 7),
 (&#39;Interdiscip Perspect Infect Dis&#39;, 7),
 (&#39;International Journal of Disaster Risk Reduction&#39;, 7),
 (&#39;J Biomol NMR&#39;, 7),
 (&#39;J Biosci&#39;, 7),
 (&#39;J Cancer&#39;, 7),
 (&#39;J Mol Model&#39;, 7),
 (&#39;Journal of Affective Disorders&#39;, 7),
 (&#39;Journal of Biosafety and Biosecurity&#39;, 7),
 (&#39;Journal of Invertebrate Pathology&#39;, 7),
 (&#39;Journal of Microbiological Methods&#39;, 7),
 (&#39;Journal of Pharmaceutical Sciences&#39;, 7),
 (&#39;Journal of Zhejiang University-SCIENCE B&#39;, 7),
 (&#39;Journal of the American College of Cardiology&#39;, 7),
 (&#34;Kendig &amp; Chernick&#39;s Disorders of the Respiratory Tract in Children&#34;, 7),
 (&#39;Korean J Radiol&#39;, 7),
 (&#39;Manual of Exotic Pet Practice&#39;, 7),
 (&#39;Mathematics and Computers in Simulation&#39;, 7),
 (&#39;Medicina Clínica (English Edition)&#39;, 7),
 (&#39;Mol Cell Biochem&#39;, 7),
 (&#39;Mol Ther Nucleic Acids&#39;, 7),
 (&#39;Molecular Aspects of Medicine&#39;, 7),
 (&#39;Monatsh Chem&#39;, 7),
 (&#39;New Biotechnology&#39;, 7),
 (&#39;Nucleic Acids Research&#39;, 7),
 (&#39;Oman Medical Journal&#39;, 7),
 (&#39;Otolaryngology - Head and Neck Surgery&#39;, 7),
 (&#39;Pediatr Radiol&#39;, 7),
 (&#39;Pediatric Emergency Medicine&#39;, 7),
 (&#39;Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences&#39;,
  7),
 (&#39;Physiological Reviews&#39;, 7),
 (&#39;Phytochemistry&#39;, 7),
 (&#39;Principles and Practice of Clinical Research&#39;, 7),
 (&#39;Progress in Nucleic Acid Research and Molecular Biology&#39;, 7),
 (&#39;Qatar Med J&#39;, 7),
 (&#39;Seminars in Cell &amp; Developmental Biology&#39;, 7),
 (&#39;Spatial and Spatio-temporal Epidemiology&#39;, 7),
 (&#39;The AAPS Journal&#39;, 7),
 (&#39;The Geographical Distribution of Animal Viral Diseases&#39;, 7),
 (&#39;The ISME Journal&#39;, 7),
 (&#39;The Law of Emergencies&#39;, 7),
 (&#39;The Mouse in Biomedical Research&#39;, 7),
 (&#39;The Professional Animal Scientist&#39;, 7),
 (&#39;Toxicon&#39;, 7),
 (&#39;Toxins (Basel)&#39;, 7),
 (&#39;Veterinary Medicine&#39;, 7),
 (&#39;Veterinary Parasitology: Regional Studies and Reports&#39;, 7),
 (&#39;World Journal of Pediatrics&#39;, 7),
 (&#39;Zentralblatt für Bakteriologie, Mikrobiologie und Hygiene. Series A: Medical Microbiology, Infectious Diseases, Virology, Parasitology&#39;,
  7),
 (&#39;Zhonghua liu xing bing xue za zhi = Zhonghua liuxingbingxue zazhi&#39;, 7),
 (&#39;ACS Infectious Diseases&#39;, 6),
 (&#39;ACS Medicinal Chemistry Letters&#39;, 6),
 (&#39;Adenoviral Vectors for Gene Therapy&#39;, 6),
 (&#39;Aerobiologia (Bologna)&#39;, 6),
 (&#39;American Heart Journal&#39;, 6),
 (&#39;Arch Med Sci&#39;, 6),
 (&#39;Archives of Medical Research&#39;, 6),
 (&#39;Archives of Physical Medicine and Rehabilitation&#39;, 6),
 (&#39;Archivos de Bronconeumología (English Edition)&#39;, 6),
 (&#34;August&#39;s Consultations in Feline Internal Medicine, Volume 7&#34;, 6),
 (&#39;BMC Fam Pract&#39;, 6),
 (&#39;BMC Med Genet&#39;, 6),
 (&#39;BMC Med Res Methodol&#39;, 6),
 (&#39;BMJ Case Reports&#39;, 6),
 (&#39;Biomedicines&#39;, 6),
 (&#39;Brazilian Journal of Microbiology&#39;, 6),
 (&#39;Breastfeeding&#39;, 6),
 (&#39;British Journal of Ophthalmology&#39;, 6),
 (&#39;Cell Research&#39;, 6),
 (&#39;Chemical Engineering Journal&#39;, 6),
 (&#39;Circulation Research&#39;, 6),
 (&#39;Communicable Diseases Intelligence&#39;, 6),
 (&#39;Comprehensive Biotechnology&#39;, 6),
 (&#39;Comprehensive Medicinal Chemistry II&#39;, 6),
 (&#39;Comput Struct Biotechnol J&#39;, 6),
 (&#39;Critical Care Nursing Clinics of North America&#39;, 6),
 (&#39;Curr Genomics&#39;, 6),
 (&#39;Current Problems in Pediatrics&#39;, 6),
 (&#39;Energy&#39;, 6),
 (&#39;Epidemiology&#39;, 6),
 (&#39;European Journal of Pharmacology&#39;, 6),
 (&#39;Evol Bioinform Online&#39;, 6),
 (&#39;Facharztprüfung Allgemeinmedizin&#39;, 6),
 (&#39;Folia Microbiol (Praha)&#39;, 6),
 (&#39;Food Chemistry&#39;, 6),
 (&#39;Food and Chemical Toxicology&#39;, 6),
 (&#39;Front Cell Neurosci&#39;, 6),
 (&#39;Gaceta Sanitaria&#39;, 6),
 (&#39;Genes &amp; Development&#39;, 6),
 (&#39;Genetics and Evolution of Infectious Disease&#39;, 6),
 (&#39;Gut Pathog&#39;, 6),
 (&#39;High-Rise Security and Fire Life Safety&#39;, 6),
 (&#39;Histopathology of Preclinical Toxicity Studies&#39;, 6),
 (&#39;Human-Animal Medicine&#39;, 6),
 (&#39;Infectious Diseases Newsletter&#39;, 6),
 (&#39;Inquiry&#39;, 6),
 (&#39;Int J Parasitol Parasites Wildl&#39;, 6),
 (&#39;Intensive and Critical Care Nursing&#39;, 6),
 (&#39;International Emergency Nursing&#39;, 6),
 (&#39;International Journal of Food Microbiology&#39;, 6),
 (&#39;International Journal of Surgery&#39;, 6),
 (&#39;J Biol Chem&#39;, 6),
 ...]</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[103]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Alternative solution</span>
<span class="n">df_journal_count</span> <span class="o">=</span> <span class="n">df_journals</span><span class="o">.</span><span class="n">value_counts</span><span class="p">()</span>

<span class="c1"># note that the value_counts() returned a mutli-index index</span>
<span class="c1"># get only the first</span>
<span class="n">short_cut</span> <span class="o">=</span> <span class="nb">list</span><span class="p">(</span><span class="nb">zip</span><span class="p">(</span><span class="n">df_journal_count</span><span class="o">.</span><span class="n">index</span><span class="o">.</span><span class="n">get_level_values</span><span class="p">(</span><span class="mi">0</span><span class="p">),</span> <span class="n">df_journal_count</span><span class="p">))</span>
</pre></div>

    </div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[104]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># check whether the alternative solution is equivalent to longcut</span>
<span class="n">df_long</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">long_cut</span><span class="p">)</span>
<span class="n">df_short</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">short_cut</span><span class="p">)</span>
<span class="n">df_long</span> <span class="o">==</span> <span class="n">df_short</span>

<span class="c1"># note that value_counts() does not sort by lexicographical order</span>
<span class="c1"># the long cut is the way to go</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[104]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>0</th>
      <th>1</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <th>1</th>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <th>2</th>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <th>3</th>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <th>4</th>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <th>...</th>
      <td>...</td>
      <td>...</td>
    </tr>
    <tr>
      <th>4881</th>
      <td>False</td>
      <td>True</td>
    </tr>
    <tr>
      <th>4882</th>
      <td>False</td>
      <td>True</td>
    </tr>
    <tr>
      <th>4883</th>
      <td>False</td>
      <td>True</td>
    </tr>
    <tr>
      <th>4884</th>
      <td>False</td>
      <td>True</td>
    </tr>
    <tr>
      <th>4885</th>
      <td>False</td>
      <td>True</td>
    </tr>
  </tbody>
</table>
<p>4886 rows × 2 columns</p>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p><strong>Method 2</strong>: Read by using <code>.read().splitlines()</code> then load into a <code>pd.DataFrame</code>.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[114]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="n">file_path</span><span class="p">,</span> <span class="s1">&#39;r&#39;</span><span class="p">)</span> <span class="k">as</span> <span class="n">f</span><span class="p">:</span>
    <span class="n">lines</span> <span class="o">=</span> <span class="n">f</span><span class="o">.</span><span class="n">read</span><span class="p">()</span><span class="o">.</span><span class="n">splitlines</span><span class="p">()</span>
    <span class="n">df_in</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">lines</span><span class="p">)</span>
    <span class="n">df_out</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">json_normalize</span><span class="p">(</span><span class="n">df_in</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span><span class="o">.</span><span class="n">apply</span><span class="p">(</span><span class="n">json</span><span class="o">.</span><span class="n">loads</span><span class="p">))</span>
<span class="n">df_out</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[114]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cord_uid</th>
      <th>sha</th>
      <th>source_x</th>
      <th>title</th>
      <th>doi</th>
      <th>pmcid</th>
      <th>pubmed_id</th>
      <th>license</th>
      <th>abstract</th>
      <th>publish_time</th>
      <th>authors</th>
      <th>journal</th>
      <th>Microsoft Academic Paper ID</th>
      <th>WHO #Covidence</th>
      <th>has_full_text</th>
      <th>full_text_file</th>
      <th>url</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>vho70jcx</td>
      <td>f056da9c64fbf00a4645ae326e8a4339d015d155</td>
      <td>biorxiv</td>
      <td>SIANN: Strain Identification by Alignment to N...</td>
      <td>10.1101/001727</td>
      <td></td>
      <td></td>
      <td>biorxiv</td>
      <td>Next-generation sequencing is increasingly bei...</td>
      <td>2014-01-10</td>
      <td>Samuel Minot; Stephen D Turner; Krista L Ternu...</td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>biorxiv_medrxiv</td>
      <td>https://doi.org/10.1101/001727</td>
    </tr>
    <tr>
      <th>1</th>
      <td>i9tbix2v</td>
      <td>daf32e013d325a6feb80e83d15aabc64a48fae33</td>
      <td>biorxiv</td>
      <td>Spatial epidemiology of networked metapopulati...</td>
      <td>10.1101/003889</td>
      <td></td>
      <td></td>
      <td>biorxiv</td>
      <td>An emerging disease is one infectious epidemic...</td>
      <td>2014-06-04</td>
      <td>Lin WANG; Xiang Li</td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>biorxiv_medrxiv</td>
      <td>https://doi.org/10.1101/003889</td>
    </tr>
    <tr>
      <th>2</th>
      <td>62gfisc6</td>
      <td>f33c6d94b0efaa198f8f3f20e644625fa3fe10d2</td>
      <td>biorxiv</td>
      <td>Sequencing of the human IG light chain loci fr...</td>
      <td>10.1101/006866</td>
      <td></td>
      <td></td>
      <td>biorxiv</td>
      <td>Germline variation at immunoglobulin gene (IG)...</td>
      <td>2014-07-03</td>
      <td>Corey T Watson; Karyn Meltz Steinberg; Tina A ...</td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>biorxiv_medrxiv</td>
      <td>https://doi.org/10.1101/006866</td>
    </tr>
    <tr>
      <th>3</th>
      <td>058r9486</td>
      <td>4da8a87e614373d56070ed272487451266dce919</td>
      <td>biorxiv</td>
      <td>Bayesian mixture analysis for metagenomic comm...</td>
      <td>10.1101/007476</td>
      <td></td>
      <td></td>
      <td>biorxiv</td>
      <td>Deep sequencing of clinical samples is now an ...</td>
      <td>2014-07-25</td>
      <td>Sofia Morfopoulou; Vincent Plagnol</td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>biorxiv_medrxiv</td>
      <td>https://doi.org/10.1101/007476</td>
    </tr>
    <tr>
      <th>4</th>
      <td>wich35l7</td>
      <td>eccef80cfbe078235df22398f195d5db462d8000</td>
      <td>biorxiv</td>
      <td>Mapping a viral phylogeny onto outbreak trees ...</td>
      <td>10.1101/010389</td>
      <td></td>
      <td></td>
      <td>biorxiv</td>
      <td>Developing methods to reconstruct transmission...</td>
      <td>2014-11-11</td>
      <td>Stephen P Velsko; Jonathan E Allen</td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>biorxiv_medrxiv</td>
      <td>https://doi.org/10.1101/010389</td>
    </tr>
    <tr>
      <th>...</th>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
      <td>...</td>
    </tr>
    <tr>
      <th>45769</th>
      <td>4360s2yu</td>
      <td>289deae0b2050aa259a05ba84565a4df82fa099a</td>
      <td>Elsevier</td>
      <td>Personal Protective Equipment: Protecting Heal...</td>
      <td>10.1016/j.clinthera.2015.07.007</td>
      <td>PMC4661082</td>
      <td>26452427</td>
      <td>els-covid</td>
      <td>Abstract Purpose The recent Ebola epidemic tha...</td>
      <td>2015-11-01</td>
      <td>Fischer, William A.; Weber, David J.; Wohl, Da...</td>
      <td>Clinical Therapeutics</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>custom_license</td>
      <td>https://doi.org/10.1016/j.clinthera.2015.07.007</td>
    </tr>
    <tr>
      <th>45770</th>
      <td>66jumbir</td>
      <td>21a4369f83891bf6975dd916c0aa495d5df8709e</td>
      <td>Elsevier</td>
      <td>Viruses and asthma</td>
      <td>10.1016/j.bbagen.2011.01.012</td>
      <td>PMC3130828</td>
      <td>21291960</td>
      <td>els-covid</td>
      <td>Abstract Background Viral respiratory infectio...</td>
      <td>2011-11-30</td>
      <td>Dulek, Daniel E.; Peebles, R. Stokes</td>
      <td>Biochimica et Biophysica Acta (BBA) - General ...</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>custom_license</td>
      <td>https://doi.org/10.1016/j.bbagen.2011.01.012</td>
    </tr>
    <tr>
      <th>45771</th>
      <td>3wk36h9p</td>
      <td></td>
      <td>Elsevier</td>
      <td>Why the WHO won't use the p-word</td>
      <td>10.1016/s0262-4079(20)30474-7</td>
      <td></td>
      <td></td>
      <td>els-covid</td>
      <td>There are no criteria for a pandemic, but covi...</td>
      <td>2020-03-07</td>
      <td>MacKenzie, Debora</td>
      <td>New Scientist</td>
      <td></td>
      <td>#5716</td>
      <td>False</td>
      <td>custom_license</td>
      <td>https://doi.org/10.1016/s0262-4079(20)30474-7</td>
    </tr>
    <tr>
      <th>45772</th>
      <td>0ujw0gak</td>
      <td></td>
      <td>WHO</td>
      <td>Communication, transparency key as Canada face...</td>
      <td>10.1503/cmaj.1095846</td>
      <td>PMC7030882</td>
      <td>32071113</td>
      <td>unk</td>
      <td></td>
      <td>2020-02-17</td>
      <td>Glauser, Wendy</td>
      <td>Canadian Medical Association Journal</td>
      <td>1953687642</td>
      <td>#4117</td>
      <td>False</td>
      <td></td>
      <td>https://doi.org/10.1503/cmaj.1095846</td>
    </tr>
    <tr>
      <th>45773</th>
      <td>28vx9w58</td>
      <td>3369a14e1d116943f48b3a33597796c9802de279; f523...</td>
      <td>PMC</td>
      <td>Searching for animal models and potential targ...</td>
      <td>10.1016/j.onehlt.2017.03.001</td>
      <td>PMC5454147</td>
      <td>28616501</td>
      <td>cc-by-nc-nd</td>
      <td>Emerging and re-emerging pathogens represent a...</td>
      <td>2017-03-03</td>
      <td>Vergara-Alert, Júlia; Vidal, Enric; Bensaid, A...</td>
      <td>One Health</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>noncomm_use_subset</td>
      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>
    </tr>
  </tbody>
</table>
<p>45774 rows × 17 columns</p>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h3 id="How-to-read-multiple-json-files-into-one-dataframe">How to read multiple json files into one dataframe<a class="anchor-link" href="#How-to-read-multiple-json-files-into-one-dataframe">&#182;</a></h3>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
</div><div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>List first all files yuing <code>os.listdir()</code>.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[130]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># implementation</span>
<span class="kn">import</span> <span class="nn">os</span>

<span class="n">json_files</span> <span class="o">=</span> <span class="p">[</span><span class="n">i</span> <span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="n">os</span><span class="o">.</span><span class="n">listdir</span><span class="p">(</span><span class="s1">&#39;sensor&#39;</span><span class="p">)</span> <span class="k">if</span> <span class="n">i</span><span class="o">.</span><span class="n">endswith</span><span class="p">(</span><span class="s1">&#39;.json&#39;</span><span class="p">)]</span>
<span class="n">json_files</span> <span class="o">=</span> <span class="p">[</span><span class="n">i</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s1">&#39;.&#39;</span><span class="p">)[</span><span class="mi">0</span><span class="p">]</span> <span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="n">json_files</span><span class="p">]</span>
<span class="n">json_files</span><span class="o">.</span><span class="n">sort</span><span class="p">()</span> <span class="c1"># don&#39;t forget to sort if necessary; cannot be stored as a variable</span>
<span class="n">json_files</span>
<span class="c1"># one-liner</span>
<span class="c1"># # json_files = [x.split(&#39;.&#39;)[0] for x in os.listdir(&#39;sensor&#39;)</span>
<span class="c1">#                   if x.endswith(&#39;.json&#39;)]</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[130]:</div>




<div class="output_text output_subarea output_execute_result">
<pre>[&#39;2018-07-09&#39;,
 &#39;2018-07-10&#39;,
 &#39;2018-07-11&#39;,
 &#39;2018-07-12&#39;,
 &#39;2018-07-13&#39;,
 &#39;2018-07-14&#39;,
 &#39;2018-07-15&#39;]</pre>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[147]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Read all files using for loop</span>
<span class="n">dfs</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">d</span> <span class="ow">in</span> <span class="n">json_files</span><span class="p">:</span>
    <span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="s1">&#39;sensor/</span><span class="si">{}</span><span class="s1">.json&#39;</span><span class="o">.</span><span class="n">format</span><span class="p">(</span><span class="n">d</span><span class="p">),</span> <span class="s1">&#39;r&#39;</span><span class="p">)</span> <span class="k">as</span> <span class="n">f</span><span class="p">:</span>
        <span class="n">lines</span> <span class="o">=</span> <span class="n">f</span><span class="o">.</span><span class="n">read</span><span class="p">()</span><span class="o">.</span><span class="n">splitlines</span><span class="p">()</span>
    <span class="n">df_in</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">lines</span><span class="p">)</span>
    <span class="n">df_out</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">json_normalize</span><span class="p">(</span><span class="n">df_in</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span><span class="o">.</span><span class="n">apply</span><span class="p">(</span><span class="n">json</span><span class="o">.</span><span class="n">loads</span><span class="p">))</span>
    <span class="n">dfs</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">df_out</span><span class="p">)</span>
<span class="n">df_final</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">concat</span><span class="p">(</span><span class="n">dfs</span><span class="p">,</span> <span class="n">ignore_index</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="c1"># this is the final dataframe</span>
<span class="n">df_final</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[147]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>tag</th>
      <th>data.ots.male.child</th>
      <th>data.ots.male.youth</th>
      <th>data.ots.male.adult</th>
      <th>data.ots.male.senior</th>
      <th>data.ots.male.aged</th>
      <th>data.ots.female.child</th>
      <th>data.ots.female.youth</th>
      <th>data.ots.female.adult</th>
      <th>data.ots.female.senior</th>
      <th>...</th>
      <th>average.weeks.attention_time.female.child</th>
      <th>average.weeks.attention_time.female.youth</th>
      <th>average.weeks.attention_time.female.adult</th>
      <th>average.weeks.attention_time.female.senior</th>
      <th>average.weeks.attention_time.female.aged</th>
      <th>age_level.child</th>
      <th>age_level.youth</th>
      <th>age_level.adult</th>
      <th>age_level.senior</th>
      <th>age_level.aged</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>1531708528880</td>
      <td>1</td>
      <td>17</td>
      <td>87</td>
      <td>91</td>
      <td>24</td>
      <td>9</td>
      <td>29</td>
      <td>123</td>
      <td>117</td>
      <td>...</td>
      <td>[0, 18, 0, 0, 0, 0, 0]</td>
      <td>[0, 26, 0, 0, 0, 0, 0]</td>
      <td>[0, 652, 0, 0, 0, 0, 0]</td>
      <td>[0, 1064, 0, 0, 0, 0, 0]</td>
      <td>[0, 117, 0, 0, 0, 0, 0]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
    <tr>
      <th>1</th>
      <td>1531708531109</td>
      <td>1</td>
      <td>13</td>
      <td>92</td>
      <td>84</td>
      <td>23</td>
      <td>3</td>
      <td>40</td>
      <td>120</td>
      <td>113</td>
      <td>...</td>
      <td>[0, 0, 2, 0, 0, 0, 0]</td>
      <td>[0, 0, 60, 0, 0, 0, 0]</td>
      <td>[0, 0, 619, 0, 0, 0, 0]</td>
      <td>[0, 0, 990, 0, 0, 0, 0]</td>
      <td>[0, 0, 124, 0, 0, 0, 0]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
    <tr>
      <th>2</th>
      <td>1531708532896</td>
      <td>1</td>
      <td>15</td>
      <td>85</td>
      <td>84</td>
      <td>25</td>
      <td>6</td>
      <td>25</td>
      <td>106</td>
      <td>107</td>
      <td>...</td>
      <td>[0, 0, 0, 6, 0, 0, 0]</td>
      <td>[0, 0, 0, 44, 0, 0, 0]</td>
      <td>[0, 0, 0, 663, 0, 0, 0]</td>
      <td>[0, 0, 0, 851, 0, 0, 0]</td>
      <td>[0, 0, 0, 89, 0, 0, 0]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
    <tr>
      <th>3</th>
      <td>1531708534927</td>
      <td>0</td>
      <td>12</td>
      <td>94</td>
      <td>99</td>
      <td>30</td>
      <td>12</td>
      <td>48</td>
      <td>146</td>
      <td>121</td>
      <td>...</td>
      <td>[0, 0, 0, 0, 16, 0, 0]</td>
      <td>[0, 0, 0, 0, 66, 0, 0]</td>
      <td>[0, 0, 0, 0, 600, 0, 0]</td>
      <td>[0, 0, 0, 0, 961, 0, 0]</td>
      <td>[0, 0, 0, 0, 128, 0, 0]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
    <tr>
      <th>4</th>
      <td>1531708536993</td>
      <td>1</td>
      <td>12</td>
      <td>79</td>
      <td>64</td>
      <td>23</td>
      <td>3</td>
      <td>27</td>
      <td>106</td>
      <td>89</td>
      <td>...</td>
      <td>[0, 0, 0, 0, 0, 2, 0]</td>
      <td>[0, 0, 0, 0, 0, 77, 0]</td>
      <td>[0, 0, 0, 0, 0, 597, 0]</td>
      <td>[0, 0, 0, 0, 0, 783, 0]</td>
      <td>[0, 0, 0, 0, 0, 64, 0]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
    <tr>
      <th>5</th>
      <td>1531708538950</td>
      <td>1</td>
      <td>29</td>
      <td>92</td>
      <td>53</td>
      <td>21</td>
      <td>11</td>
      <td>43</td>
      <td>129</td>
      <td>71</td>
      <td>...</td>
      <td>[0, 0, 0, 0, 0, 0, 2]</td>
      <td>[0, 0, 0, 0, 0, 0, 101]</td>
      <td>[0, 0, 0, 0, 0, 0, 457]</td>
      <td>[0, 0, 0, 0, 0, 0, 481]</td>
      <td>[0, 0, 0, 0, 0, 0, 39]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
    <tr>
      <th>6</th>
      <td>1531708541171</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>...</td>
      <td>[0, 0, 0, 0, 0, 0, 0]</td>
      <td>[0, 0, 0, 0, 0, 0, 0]</td>
      <td>[0, 0, 0, 0, 0, 0, 0]</td>
      <td>[0, 0, 0, 0, 0, 0, 0]</td>
      <td>[0, 0, 0, 0, 0, 0, 0]</td>
      <td>[13, 17]</td>
      <td>[18, 29]</td>
      <td>[30, 49]</td>
      <td>[50, 64]</td>
      <td>[65, 99]</td>
    </tr>
  </tbody>
</table>
<p>7 rows × 128 columns</p>
</div>
</div>

</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[157]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># .filter may be used to filter dataframes based on column</span>
<span class="n">df_final</span><span class="o">.</span><span class="n">filter</span><span class="p">(</span><span class="n">like</span><span class="o">=</span><span class="s1">&#39;data.viewer&#39;</span><span class="p">,</span><span class="n">axis</span><span class="o">=</span><span class="mi">1</span><span class="p">)</span>
</pre></div>

    </div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area">

    <div class="prompt output_prompt">Out[157]:</div>



<div class="output_html rendered_html output_subarea output_execute_result">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>data.viewer.male.child</th>
      <th>data.viewer.male.youth</th>
      <th>data.viewer.male.adult</th>
      <th>data.viewer.male.senior</th>
      <th>data.viewer.male.aged</th>
      <th>data.viewer.female.child</th>
      <th>data.viewer.female.youth</th>
      <th>data.viewer.female.adult</th>
      <th>data.viewer.female.senior</th>
      <th>data.viewer.female.aged</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>1</td>
      <td>5</td>
      <td>66</td>
      <td>64</td>
      <td>14</td>
      <td>5</td>
      <td>8</td>
      <td>75</td>
      <td>98</td>
      <td>17</td>
    </tr>
    <tr>
      <th>1</th>
      <td>0</td>
      <td>5</td>
      <td>68</td>
      <td>62</td>
      <td>15</td>
      <td>1</td>
      <td>11</td>
      <td>70</td>
      <td>93</td>
      <td>16</td>
    </tr>
    <tr>
      <th>2</th>
      <td>0</td>
      <td>7</td>
      <td>59</td>
      <td>55</td>
      <td>16</td>
      <td>3</td>
      <td>8</td>
      <td>71</td>
      <td>86</td>
      <td>14</td>
    </tr>
    <tr>
      <th>3</th>
      <td>0</td>
      <td>4</td>
      <td>73</td>
      <td>71</td>
      <td>20</td>
      <td>5</td>
      <td>17</td>
      <td>78</td>
      <td>90</td>
      <td>18</td>
    </tr>
    <tr>
      <th>4</th>
      <td>0</td>
      <td>5</td>
      <td>59</td>
      <td>47</td>
      <td>14</td>
      <td>1</td>
      <td>7</td>
      <td>66</td>
      <td>74</td>
      <td>8</td>
    </tr>
    <tr>
      <th>5</th>
      <td>1</td>
      <td>9</td>
      <td>58</td>
      <td>34</td>
      <td>11</td>
      <td>1</td>
      <td>20</td>
      <td>64</td>
      <td>46</td>
      <td>5</td>
    </tr>
    <tr>
      <th>6</th>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
  </tbody>
</table>
</div>
</div>

</div>

</div>
</div>

</div>
    </div>
  </div>
</body>

 


</html>
